{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Webscraping qury"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n"
     ]
    }
   ],
   "source": [
    "import logging\n",
    "import sys\n",
    "\n",
    "logging.basicConfig(stream=sys.stdout, level=logging.INFO)\n",
    "logging.getLogger().addHandler(logging.StreamHandler(stream=sys.stdout))\n",
    "\n",
    "from IPython.display import Markdown, display\n",
    "\n",
    "from dotenv import load_dotenv\n",
    "\n",
    "load_dotenv('/workspaces/ml-learning/.env', override=True)\n",
    "\n",
    "import nest_asyncio\n",
    "nest_asyncio.apply()\n",
    "\n",
    "%load_ext autoreload\n",
    "%autoreload 2"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Test initialization"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/workspaces/ml-learning/src/phages/modules/library.py:69: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/\n",
      "  @validator('source', pre=True, always=True)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "embedded weaviate is already listening on port 8079\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "{\"action\":\"hnsw_vector_cache_prefill\",\"count\":1000,\"index_id\":\"docs_7hK4rmDaZn7X\",\"level\":\"info\",\"limit\":1000000000000,\"msg\":\"prefilled vector cache\",\"time\":\"2024-01-28T03:54:23Z\",\"took\":68200}\n",
      "{\"action\":\"hnsw_vector_cache_prefill\",\"count\":1000,\"index_id\":\"nodes_F9gFGkEajvtV\",\"level\":\"info\",\"limit\":1000000000000,\"msg\":\"prefilled vector cache\",\"time\":\"2024-01-28T03:54:23Z\",\"took\":68100}\n"
     ]
    }
   ],
   "source": [
    "from phages.modules.library import Library\n",
    "from llama_index.llms import OpenAI\n",
    "library = Library().withLLM(OpenAI(model='gpt-4'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.client.is_ready()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'classes': [{'class': 'Docs',\n",
       "   'description': 'Class for Docs',\n",
       "   'invertedIndexConfig': {'bm25': {'b': 0.75, 'k1': 1.2},\n",
       "    'cleanupIntervalSeconds': 60,\n",
       "    'stopwords': {'additions': None, 'preset': 'en', 'removals': None}},\n",
       "   'multiTenancyConfig': {'enabled': False},\n",
       "   'properties': [{'dataType': ['text'],\n",
       "     'description': 'Text property',\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'text',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': 'The ref_doc_id of the Node',\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'ref_doc_id',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': 'node_info (in JSON)',\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'node_info',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': 'The relationships of the node (in JSON)',\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'relationships',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:40 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'file_name',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:40 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'citation',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:40 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': '_node_content',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:40 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': '_node_type',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:40 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'document_id',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:40 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'doc_id',\n",
       "     'tokenization': 'word'}],\n",
       "   'replicationConfig': {'factor': 1},\n",
       "   'shardingConfig': {'virtualPerPhysical': 128,\n",
       "    'desiredCount': 1,\n",
       "    'actualCount': 1,\n",
       "    'desiredVirtualCount': 128,\n",
       "    'actualVirtualCount': 128,\n",
       "    'key': '_id',\n",
       "    'strategy': 'hash',\n",
       "    'function': 'murmur3'},\n",
       "   'vectorIndexConfig': {'skip': False,\n",
       "    'cleanupIntervalSeconds': 300,\n",
       "    'maxConnections': 64,\n",
       "    'efConstruction': 128,\n",
       "    'ef': -1,\n",
       "    'dynamicEfMin': 100,\n",
       "    'dynamicEfMax': 500,\n",
       "    'dynamicEfFactor': 8,\n",
       "    'vectorCacheMaxObjects': 1000000000000,\n",
       "    'flatSearchCutoff': 40000,\n",
       "    'distance': 'cosine',\n",
       "    'pq': {'enabled': False,\n",
       "     'bitCompression': False,\n",
       "     'segments': 0,\n",
       "     'centroids': 256,\n",
       "     'trainingLimit': 100000,\n",
       "     'encoder': {'type': 'kmeans', 'distribution': 'log-normal'}}},\n",
       "   'vectorIndexType': 'hnsw',\n",
       "   'vectorizer': 'none'},\n",
       "  {'class': 'Nodes',\n",
       "   'description': 'Class for Nodes',\n",
       "   'invertedIndexConfig': {'bm25': {'b': 0.75, 'k1': 1.2},\n",
       "    'cleanupIntervalSeconds': 60,\n",
       "    'stopwords': {'additions': None, 'preset': 'en', 'removals': None}},\n",
       "   'multiTenancyConfig': {'enabled': False},\n",
       "   'properties': [{'dataType': ['text'],\n",
       "     'description': 'Text property',\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'text',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': 'The ref_doc_id of the Node',\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'ref_doc_id',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': 'node_info (in JSON)',\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'node_info',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': 'The relationships of the node (in JSON)',\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'relationships',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['uuid'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:42 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': False,\n",
       "     'name': 'doc_id'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:42 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'file_name',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:42 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': 'citation',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:42 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': '_node_content',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['text'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:42 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': True,\n",
       "     'name': '_node_type',\n",
       "     'tokenization': 'word'},\n",
       "    {'dataType': ['uuid'],\n",
       "     'description': \"This property was generated by Weaviate's auto-schema feature on Sun Jan 28 04:09:42 2024\",\n",
       "     'indexFilterable': True,\n",
       "     'indexSearchable': False,\n",
       "     'name': 'document_id'}],\n",
       "   'replicationConfig': {'factor': 1},\n",
       "   'shardingConfig': {'virtualPerPhysical': 128,\n",
       "    'desiredCount': 1,\n",
       "    'actualCount': 1,\n",
       "    'desiredVirtualCount': 128,\n",
       "    'actualVirtualCount': 128,\n",
       "    'key': '_id',\n",
       "    'strategy': 'hash',\n",
       "    'function': 'murmur3'},\n",
       "   'vectorIndexConfig': {'skip': False,\n",
       "    'cleanupIntervalSeconds': 300,\n",
       "    'maxConnections': 64,\n",
       "    'efConstruction': 128,\n",
       "    'ef': -1,\n",
       "    'dynamicEfMin': 100,\n",
       "    'dynamicEfMax': 500,\n",
       "    'dynamicEfFactor': 8,\n",
       "    'vectorCacheMaxObjects': 1000000000000,\n",
       "    'flatSearchCutoff': 40000,\n",
       "    'distance': 'cosine',\n",
       "    'pq': {'enabled': False,\n",
       "     'bitCompression': False,\n",
       "     'segments': 0,\n",
       "     'centroids': 256,\n",
       "     'trainingLimit': 100000,\n",
       "     'encoder': {'type': 'kmeans', 'distribution': 'log-normal'}}},\n",
       "   'vectorIndexType': 'hnsw',\n",
       "   'vectorizer': 'none'}]}"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.client.schema.get()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'deprecations': [],\n",
       " 'objects': [{'class': 'Docs',\n",
       "   'creationTimeUnix': 1706414989023,\n",
       "   'id': '390ec987-3df0-4711-ad7e-dc6e7295dff6',\n",
       "   'lastUpdateTimeUnix': 1706414989023,\n",
       "   'properties': {'_node_content': '{\"id_\": \"390ec987-3df0-4711-ad7e-dc6e7295dff6\", \"embedding\": null, \"metadata\": {\"file_name\": \"/workspaces/ml-learning/src/phages/data/test3/fmicb-09-00850.pdf\", \"citation\": \"Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850\"}, \"excluded_embed_metadata_keys\": [], \"excluded_llm_metadata_keys\": [], \"relationships\": {}, \"text\": \"\", \"start_char_idx\": null, \"end_char_idx\": null, \"text_template\": \"{metadata_str}\\\\n\\\\n{content}\", \"metadata_template\": \"{key}: {value}\", \"metadata_seperator\": \"\\\\n\", \"class_name\": \"TextNode\"}',\n",
       "    '_node_type': 'TextNode',\n",
       "    'citation': 'Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850',\n",
       "    'doc_id': 'None',\n",
       "    'document_id': 'None',\n",
       "    'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-09-00850.pdf',\n",
       "    'ref_doc_id': 'None',\n",
       "    'text': 'Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850'},\n",
       "   'vectorWeights': None},\n",
       "  {'class': 'Docs',\n",
       "   'creationTimeUnix': 1706414980667,\n",
       "   'id': '5a680be7-b5cc-4d06-88cb-9965ae62c94d',\n",
       "   'lastUpdateTimeUnix': 1706414980667,\n",
       "   'properties': {'_node_content': '{\"id_\": \"5a680be7-b5cc-4d06-88cb-9965ae62c94d\", \"embedding\": null, \"metadata\": {\"file_name\": \"/workspaces/ml-learning/src/phages/data/test3/fmicb-08-02548.pdf\", \"citation\": \"Tikhe, Chinmay V., and Claudia Husseneder. \\\\\"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\\\\\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548.\"}, \"excluded_embed_metadata_keys\": [], \"excluded_llm_metadata_keys\": [], \"relationships\": {}, \"text\": \"\", \"start_char_idx\": null, \"end_char_idx\": null, \"text_template\": \"{metadata_str}\\\\n\\\\n{content}\", \"metadata_template\": \"{key}: {value}\", \"metadata_seperator\": \"\\\\n\", \"class_name\": \"TextNode\"}',\n",
       "    '_node_type': 'TextNode',\n",
       "    'citation': 'Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548.',\n",
       "    'doc_id': 'None',\n",
       "    'document_id': 'None',\n",
       "    'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-08-02548.pdf',\n",
       "    'ref_doc_id': 'None',\n",
       "    'text': 'Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548.'},\n",
       "   'vectorWeights': None},\n",
       "  {'class': 'Docs',\n",
       "   'creationTimeUnix': 1706415013018,\n",
       "   'id': '5eb07d54-a7f3-4e5b-b8dd-0677bc1ee2ee',\n",
       "   'lastUpdateTimeUnix': 1706415013018,\n",
       "   'properties': {'_node_content': '{\"id_\": \"5eb07d54-a7f3-4e5b-b8dd-0677bc1ee2ee\", \"embedding\": null, \"metadata\": {\"file_name\": \"/workspaces/ml-learning/src/phages/data/test3/fmicb-12-616548.pdf\", \"citation\": \"Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548\"}, \"excluded_embed_metadata_keys\": [], \"excluded_llm_metadata_keys\": [], \"relationships\": {}, \"text\": \"\", \"start_char_idx\": null, \"end_char_idx\": null, \"text_template\": \"{metadata_str}\\\\n\\\\n{content}\", \"metadata_template\": \"{key}: {value}\", \"metadata_seperator\": \"\\\\n\", \"class_name\": \"TextNode\"}',\n",
       "    '_node_type': 'TextNode',\n",
       "    'citation': 'Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548',\n",
       "    'doc_id': 'None',\n",
       "    'document_id': 'None',\n",
       "    'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-12-616548.pdf',\n",
       "    'ref_doc_id': 'None',\n",
       "    'text': 'Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548'},\n",
       "   'vectorWeights': None},\n",
       "  {'class': 'Docs',\n",
       "   'creationTimeUnix': 1706414996452,\n",
       "   'id': 'cf029be1-6e2a-4a25-adec-164f15261c21',\n",
       "   'lastUpdateTimeUnix': 1706414996452,\n",
       "   'properties': {'_node_content': '{\"id_\": \"cf029be1-6e2a-4a25-adec-164f15261c21\", \"embedding\": null, \"metadata\": {\"file_name\": \"/workspaces/ml-learning/src/phages/data/test3/fmicb-10-00627.pdf\", \"citation\": \"Liao, Yen-Te, et al. \\\\\"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\\\\\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.\"}, \"excluded_embed_metadata_keys\": [], \"excluded_llm_metadata_keys\": [], \"relationships\": {}, \"text\": \"\", \"start_char_idx\": null, \"end_char_idx\": null, \"text_template\": \"{metadata_str}\\\\n\\\\n{content}\", \"metadata_template\": \"{key}: {value}\", \"metadata_seperator\": \"\\\\n\", \"class_name\": \"TextNode\"}',\n",
       "    '_node_type': 'TextNode',\n",
       "    'citation': 'Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.',\n",
       "    'doc_id': 'None',\n",
       "    'document_id': 'None',\n",
       "    'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-10-00627.pdf',\n",
       "    'ref_doc_id': 'None',\n",
       "    'text': 'Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.'},\n",
       "   'vectorWeights': None},\n",
       "  {'class': 'Docs',\n",
       "   'creationTimeUnix': 1706415003972,\n",
       "   'id': 'e8ed01eb-9436-448c-9439-403330ffd6f9',\n",
       "   'lastUpdateTimeUnix': 1706415003972,\n",
       "   'properties': {'_node_content': '{\"id_\": \"e8ed01eb-9436-448c-9439-403330ffd6f9\", \"embedding\": null, \"metadata\": {\"file_name\": \"/workspaces/ml-learning/src/phages/data/test3/fmicb-10-01533.pdf\", \"citation\": \"Sharma, Ruchira, et al. \\\\\"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\\\\\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.\"}, \"excluded_embed_metadata_keys\": [], \"excluded_llm_metadata_keys\": [], \"relationships\": {}, \"text\": \"\", \"start_char_idx\": null, \"end_char_idx\": null, \"text_template\": \"{metadata_str}\\\\n\\\\n{content}\", \"metadata_template\": \"{key}: {value}\", \"metadata_seperator\": \"\\\\n\", \"class_name\": \"TextNode\"}',\n",
       "    '_node_type': 'TextNode',\n",
       "    'citation': 'Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.',\n",
       "    'doc_id': 'None',\n",
       "    'document_id': 'None',\n",
       "    'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-10-01533.pdf',\n",
       "    'ref_doc_id': 'None',\n",
       "    'text': 'Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.'},\n",
       "   'vectorWeights': None},\n",
       "  {'class': 'Docs',\n",
       "   'creationTimeUnix': 1706419153199,\n",
       "   'id': 'f47ef8b8-af3e-4e6d-9c21-df266deddc9e',\n",
       "   'lastUpdateTimeUnix': 1706419153199,\n",
       "   'properties': {'_node_content': '{\"id_\": \"f47ef8b8-af3e-4e6d-9c21-df266deddc9e\", \"embedding\": null, \"metadata\": {\"file_name\": \"/workspaces/ml-learning/src/phages/data/test3/aem.01468-21.pdf\", \"citation\": \"Xu, Zhenhe, et al. \\\\\"Isolation of a Virulent Aeromonas salmonicida subsp. masoucida Bacteriophage and Its Application in Phage Therapy in Turbot (Scophthalmus maximus).\\\\\" Journal of Virology, vol. 98, no. 2, 2024, pp. e01567-19.\"}, \"excluded_embed_metadata_keys\": [], \"excluded_llm_metadata_keys\": [], \"relationships\": {}, \"text\": \"\", \"start_char_idx\": null, \"end_char_idx\": null, \"text_template\": \"{metadata_str}\\\\n\\\\n{content}\", \"metadata_template\": \"{key}: {value}\", \"metadata_seperator\": \"\\\\n\", \"class_name\": \"TextNode\"}',\n",
       "    '_node_type': 'TextNode',\n",
       "    'citation': 'Xu, Zhenhe, et al. \"Isolation of a Virulent Aeromonas salmonicida subsp. masoucida Bacteriophage and Its Application in Phage Therapy in Turbot (Scophthalmus maximus).\" Journal of Virology, vol. 98, no. 2, 2024, pp. e01567-19.',\n",
       "    'doc_id': 'None',\n",
       "    'document_id': 'None',\n",
       "    'file_name': '/workspaces/ml-learning/src/phages/data/test3/aem.01468-21.pdf',\n",
       "    'ref_doc_id': 'None',\n",
       "    'text': 'Xu, Zhenhe, et al. \"Isolation of a Virulent Aeromonas salmonicida subsp. masoucida Bacteriophage and Its Application in Phage Therapy in Turbot (Scophthalmus maximus).\" Journal of Virology, vol. 98, no. 2, 2024, pp. e01567-19.'},\n",
       "   'vectorWeights': None}],\n",
       " 'totalResults': 6}"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.client.data_object.get(class_name='Docs', with_vector=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "library.docs_index.as_query_engine()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Blob 1 validation error for SourceType\n",
      "source\n",
      "  Value error, The source must be a valid URL or an existing local file path. [type=value_error, input_value='some/random/url', input_type=str]\n",
      "    For further information visit https://errors.pydantic.dev/2.5/v/value_error\n"
     ]
    }
   ],
   "source": [
    "try:    \n",
    "    library.add(\"some/random/url\")\n",
    "except (ValueError, Exception) as e:\n",
    "    print(f\"Blob {e}\")\n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Test querying 5 files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32487 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32487 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32487 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x327a8 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x327a8 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x327a8 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32a79 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32a79 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32a79 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32c82 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32c82 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32c82 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32e3c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32e3c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32e3c for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32fee for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32fee for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32fee for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x331d8 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x331d8 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x331d8 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x333ca for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x333ca for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x333ca for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x3365b for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x3365b for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x3365b for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x338ac for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x338ac for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x338ac for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x33a86 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33a86 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33a86 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x33c88 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33c88 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33c88 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x33e79 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33e79 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33e79 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x34102 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x34102 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x34102 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x34393 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x34393 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x34393 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x3453c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x3453c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x3453c for key /MediaBox\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: ORIGINAL RESEARCH\n",
      "published: 04 January 2018\n",
      "doi: 10.3389/fmicb.2017.02548\n",
      "Frontiers in Microbiology | www.frontiersin.org 1 January 2018 | Volume 8 | Article 2548Editedby:\n",
      "MichaelThomas-Poulsen,\n",
      "UniversityofCopenhagen,Denmark\n",
      "Reviewedby:\n",
      "RichardAllenWhiteIII,\n",
      "WashingtonStateUniversity,\n",
      "UnitedStates\n",
      "KarenDawnWeynberg,\n",
      "TheUniversityofQueensland,\n",
      "Australia\n",
      "AlejandroReyes,\n",
      "UniversityofLosAndes,Colombia\n",
      "*Correspondence:\n",
      "ChinmayV.Tikhe\n",
      "ctikhe1@jhu.edu\n",
      "†PresentAddress:\n",
      "ChinmayV.Tikhe,\n",
      "W.HarryFeinstoneDepartmentof\n",
      "MolecularMicrobiologyand\n",
      "Immunology,BloombergSchoolof\n",
      "PublicHealth,JohnsHopkins\n",
      "University,Baltimore,MD,\n",
      "UnitedStates\n",
      "Specialtysection:\n",
      "Thisarticlewassubmittedto\n",
      "MicrobialSymbioses,\n",
      "asectionofthejournal\n",
      "FrontiersinMicrobiology\n",
      "Received: 21September2017\n",
      "Accepted: 08December2017\n",
      "Published: 04January2018\n",
      "Citation:\n",
      "TikheCVandHussenederC(2018)\n",
      "MetaviromeSequencingofthe\n",
      "TermiteGutRevealsthePresenceof\n",
      "anUnexploredBacteriophage\n",
      "Community.Front.Microbiol.8:2548.\n",
      "doi:10.3389/fmicb.2017.02548Metavirome Sequencing of the\n",
      "Termite Gut Reveals the Presence of\n",
      "an Unexplored Bacteriophage\n",
      "Community\n",
      "ChinmayV.Tikhe *†andClaudiaHusseneder\n",
      "DepartmentofEntomology,LouisianaStateUniversityAgri culturalCenter,BatonRouge,LA,UnitedStates\n",
      "TheFormosansubterraneantermite; Coptotermesformosanus isnutritionallydependent\n",
      "on the complex and diverse community of bacteria and protozo a in their gut. Although,\n",
      "there have been many studies to decipher the taxonomic and fu nctional diversity of\n",
      "bacterial communities in the guts of termites, their bacter iophages remain unstudied.\n",
      "WesequencedthemetaviromeofthegutsofFormosansubterra neantermiteworkersto\n",
      "study thediversityof bacteriophagesandotherassociated viruses. Resultsshowed that\n",
      "the termites harbor a virome in their gut comprised of varied and previously unknown\n",
      "bacteriophages. Between 87–90% of the predicted dsDNA viru s genes by Metavir\n",
      "showed similarity to the tailed bacteriophages (Caudovirales) . Many predicted genes\n",
      "from the virome matched to bacterial prophage regions. Thes e data are suggestive\n",
      "of a virome dominated by temperate bacteriophages. We predi cted the genomes of\n",
      "seven novel Caudovirales bacteriophages from the termite g ut. Three of these predicted\n",
      "bacteriophage genomes were found in high proportions in all the three termite colonies\n",
      "tested. Two bacteriophages are predicted to infect endosym biotic bacteria of the gut\n",
      "protozoa.Thepresenceoftheseputativebacteriophagesin fectingendosymbiontsofthe\n",
      "gut protozoa, suggests a quadripartite relationship betwe en the termites their symbiotic\n",
      "protozoa, endosymbiotic bacteria of the protozoa and their bacteriophages. Other than\n",
      "Caudovirales, ss-DNA virus related genes were also present in the termite gut. We\n",
      "predicted the genomes of 12 novel Microviridae phages from the termite gut and seven\n",
      "of those possibly represent a new proposed subfamily. Circo virus like genomes were\n",
      "also assembled from the termite gut at lower relative abunda nce. We predicted 10\n",
      "novel circovirus\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x21878 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21878 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21878 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x21b0c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21b0c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21b0c for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x21e55 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21e55 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21e55 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22023 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22023 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22023 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x221d9 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x221d9 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x221d9 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x223aa for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x223aa for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x223aa for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22538 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22538 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22538 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x226f1 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x226f1 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x226f1 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22957 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22957 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22957 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22b98 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22b98 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22b98 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22e09 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22e09 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22e09 for key /MediaBox\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: ORIGINAL RESEARCH\n",
      "published: 30 April 2018\n",
      "doi: 10.3389/fmicb.2018.00850\n",
      "Frontiers in Microbiology | www.frontiersin.org 1 April 2018 | Volume 9 | Article 850Editedby:\n",
      "Xian-ZhiLi,\n",
      "HealthCanada,Canada\n",
      "Reviewedby:\n",
      "NinaChanishvili,\n",
      "GeorgeEliavaInstituteof\n",
      "Bacteriophage,Microbiologyand\n",
      "Virology,Georgia\n",
      "PaulHyman,\n",
      "AshlandUniversity,UnitedStates\n",
      "RemyA.Bonnin,\n",
      "UniversitéParis-Saclay,France\n",
      "*Correspondence:\n",
      "ZhiyongZong\n",
      "zongzhiy@scu.edu.cn\n",
      "Specialtysection:\n",
      "Thisarticlewassubmittedto\n",
      "Antimicrobials,Resistanceand\n",
      "Chemotherapy,\n",
      "asectionofthejournal\n",
      "FrontiersinMicrobiology\n",
      "Received: 06January2018\n",
      "Accepted: 13April2018\n",
      "Published: 30April2018\n",
      "Citation:\n",
      "ZhouW,FengYandZongZ(2018)\n",
      "TwoNewLyticBacteriophagesofthe\n",
      "MyoviridaeFamilyAgainst\n",
      "Carbapenem-ResistantAcinetobacter\n",
      "baumannii.Front.Microbiol.9:850.\n",
      "doi:10.3389/fmicb.2018.00850Two New Lytic Bacteriophages of the\n",
      "Myoviridae Family Against\n",
      "Carbapenem-Resistant\n",
      "Acinetobacterbaumannii\n",
      "WeilongZhou1,2,YuFeng1,2andZhiyongZong1,2,3,4*\n",
      "1CenterofInfectiousDiseases,WestChinaHospital,Sichuan University,Chengdu,China,2DivisionofInfectiousDiseases,\n",
      "StateKeyLaboratoryofBiotherapy,Chengdu,China,3DepartmentofInfectionControl,WestChinaHospital,Sichu an\n",
      "University,Chengdu,China,4CenterforPathogenResearch,WestChinaHospital,Sichuan University,Chengdu,China\n",
      "Two lytic bacteriophages, WCHABP1 and WCHABP12, were recover ed from hospital\n",
      "sewage and were able to infect 9 and 12 out of 18 carbapenem-re sistantAcinetobacter\n",
      "baumannii clinical strains, which belonged to different clones. Elec tron microscopy\n",
      "scan showed that both bacteriophages had the similar morpho logy as those of the\n",
      "Myoviridae family. Whole genomic sequencing revealed 45.4- or 45.8-kb g enome\n",
      "with a 37.6% GC content for WCHABP1 and WCHABP12, both of which s howed\n",
      "signiﬁcant DNA sequence similarity with bacteriophages of theAp22virus genus within\n",
      "theMyoviridae family.Taxonomicanalysiswasthereforeperformedfollow ingtheproposal\n",
      "approved by the International Committee on Taxonomy of Viru ses, which conﬁrmed that\n",
      "WCHABP1 and WCHABP12 represented two new species of the Ap22virus genus.\n",
      "No tRNAs but 88 and 89 open reading frames (ORFs) were predict ed for the two\n",
      "bacteriophages, among which 22 and 21 had known function and encoded proteins\n",
      "for morphogenesis, packaging, lysis, and nucleiotide meta bolism. The C-terminal amino\n",
      "acids of the large unit of ﬁber tail proteins varied between t he bacteriophages, which\n",
      "may explain their different host ranges. For most lytic bact eriophages, a set of holin\n",
      "and endolysin are required for lysis. However, no known holi n-encoding genes were\n",
      "identiﬁed in WCHABP1 and WCHABP12, suggesting that they may us e alternative,\n",
      "yet-to-be-identiﬁed, novel holins for host cell membrane l ysis. To test the efﬁcacy of the\n",
      "bacteriophages in protecting against A. baumannii infection, a Galleria mellonella larva\n",
      "model was used. Only <20%G. mellonella larvae survived at 96h after being infected\n",
      "bycarbapenem-resistant A.baumannii strains,fromwhichthetwobacter\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: fmicb-10-00627 March 30, 2019 Time: 16:26 # 1\n",
      "ORIGINAL RESEARCH\n",
      "published: 02 April 2019\n",
      "doi: 10.3389/fmicb.2019.00627\n",
      "Edited by:\n",
      "William Michael McShan,\n",
      "The University of Oklahoma Health\n",
      "Sciences Center, United States\n",
      "Reviewed by:\n",
      "Grzegorz Wegrzyn,\n",
      "University of Gda ´nsk, Poland\n",
      "Scott Van Nguyen,\n",
      "University College Dublin, Ireland\n",
      "*Correspondence:\n",
      "Vivian C. H. Wu\n",
      "vivian.wu@ars.usda.gov\n",
      "Specialty section:\n",
      "This article was submitted to\n",
      "Virology,\n",
      "a section of the journal\n",
      "Frontiers in Microbiology\n",
      "Received: 10 October 2018\n",
      "Accepted: 12 March 2019\n",
      "Published: 02 April 2019\n",
      "Citation:\n",
      "Liao Y-T, Sun X, Quintela IA,\n",
      "Bridges DF, Liu F, Zhang Y, Salvador A\n",
      "and Wu VCH (2019) Discovery\n",
      "of Shiga Toxin-Producing Escherichia\n",
      "coli (STEC)-Speciﬁc Bacteriophages\n",
      "From Non-fecal Composts Using\n",
      "Genomic Characterization.\n",
      "Front. Microbiol. 10:627.\n",
      "doi: 10.3389/fmicb.2019.00627\n",
      "Discovery of Shiga Toxin-Producing\n",
      "Escherichia coli (STEC)-Speciﬁc\n",
      "Bacteriophages From Non-fecal\n",
      "Composts Using Genomic\n",
      "Characterization\n",
      "Yen-Te Liao1, Xincheng Sun1,2,3, Irwin A. Quintela1, David F. Bridges1, Fang Liu1,4,\n",
      "Yujie Zhang1,5, Alexandra Salvador1and Vivian C. H. Wu1*\n",
      "1Produce Safety and Microbiology Research Unit, U.S. Department of Agriculture, Agricultural Research Service, Western\n",
      "Regional Research Center, Albany, CA, United States,2Henan Key Laboratory of Cold Chain Food Quality and Safety\n",
      "Control, Zhengzhou University of Light Industry, Zhengzhou, China,3Collaborative Innovation Center of Food Production\n",
      "and Safety, Zhengzhou, China,4College of Food Science and Engineering, Ocean University of China, Qingdao, China,\n",
      "5College of Food Science and Technology, Shanghai Ocean University, Shanghai, China\n",
      "Composting is a complex biodegradable process that converts organic materials\n",
      "into nutrients to facilitate crop yields, and, if well managed, can render bactericidal\n",
      "effects. Majority of research focused on detection of enteric pathogens, such as\n",
      "Shiga toxin-producing Escherichia coli (STEC) in fecal composts. Recently, attention\n",
      "has been emphasized on bacteriophages, such as STEC-speciﬁc bacteriophages,\n",
      "associated with STEC from the fecal-contaminated environment because they are able\n",
      "to sustain adverse environmental condition during composting process. However, little is\n",
      "known regarding the isolation of STEC-speciﬁc bacteriophages in non-fecal composts.\n",
      "Thus, the objectives were to isolate and genomically characterize STEC-speciﬁc\n",
      "bacteriophages, and to evaluate its association with STEC in non-fecal composts.\n",
      "For bacteriophage isolation, the samples were enriched with non-pathogenic E. coli\n",
      "(3 strains) and STEC (14 strains), respectively. After puriﬁcation, host range, plaque\n",
      "size, and phage morphology were examined. Furthermore, bacteriophage genomes\n",
      "were subjected to whole-genome sequencing using Illumina MiSeq and genomic\n",
      "analyses. Isolation of top six non-O157 and O157 STEC utilizing culture methods\n",
      "combined with PCR-based conﬁrmation was also conducted. The results showed\n",
      "that va\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: fmicb-10-01533 July 22, 2019 Time: 16:10 # 1\n",
      "ORIGINAL RESEARCH\n",
      "published: 23 July 2019\n",
      "doi: 10.3389/fmicb.2019.01533\n",
      "Edited by:\n",
      "Manuel Martinez Garcia,\n",
      "University of Alicante, Spain\n",
      "Reviewed by:\n",
      "Jeroen Wagemans,\n",
      "KU Leuven, Belgium\n",
      "Ana-Belen Martin-Cuadrado,\n",
      "University of Alicante, Spain\n",
      "Lars Fieseler,\n",
      "Zurich University of Applied Sciences,\n",
      "Switzerland\n",
      "*Correspondence:\n",
      "Julianne H. Grose\n",
      "grosejulianne@gmail.com\n",
      "Specialty section:\n",
      "This article was submitted to\n",
      "Virology,\n",
      "a section of the journal\n",
      "Frontiers in Microbiology\n",
      "Received: 08 March 2019\n",
      "Accepted: 19 June 2019\n",
      "Published: 23 July 2019\n",
      "Citation:\n",
      "Sharma R, Pielstick BA, Bell KA,\n",
      "Nieman TB, Stubbs OA, Yeates EL,\n",
      "Baltrus DA and Grose JH (2019) A\n",
      "Novel, Highly Related Jumbo Family\n",
      "of Bacteriophages That Were Isolated\n",
      "Against Erwinia.\n",
      "Front. Microbiol. 10:1533.\n",
      "doi: 10.3389/fmicb.2019.01533\n",
      "A Novel, Highly Related Jumbo\n",
      "Family of Bacteriophages That Were\n",
      "Isolated Against Erwinia\n",
      "Ruchira Sharma1, Brittany A. Pielstick1, Kimberly A. Bell1, Tanner B. Nieman1,\n",
      "Olivia A. Stubbs1, Edward L. Yeates1, David A. Baltrus2and Julianne H. Grose1*\n",
      "1Department of Microbiology and Molecular Biology, Brigham Young University, Provo, UT, United States,2School of Plant\n",
      "Sciences, The University of Arizona, Tucson, AZ, United States\n",
      "Erwinia amylovora is a plant pathogen from the Erwiniaceae family and a causative\n",
      "agent of the devastating agricultural disease ﬁre blight. Here we characterize eight\n",
      "lytic bacteriophages of E. amylovora that we isolated from the Wasatch front (Utah,\n",
      "United States) that are highly similar to vB_EamM_Ea35-70 which was isolated in\n",
      "Ontario, Canada. With the genome size ranging from 271 to 275 kb, this is a novel jumbo\n",
      "family of bacteriophages. These jumbo bacteriophages were further characterized\n",
      "through genomic and proteomic comparison, mass spectrometry, host range and burst\n",
      "size. Their proteomes are highly unstudied, with over 200 putative proteins with no\n",
      "known homologs. The production of 27 of these putative proteins was conﬁrmed\n",
      "by mass spectrometry analysis. These bacteriophages appear to be most similar to\n",
      "bacteriophages that infect Pseudomonas andRalstonia rather than Enterobacteriales\n",
      "bacteria by protein similarity, however, we were only able to detect infection of Erwinia\n",
      "and the closely related strains of Pantoea .\n",
      "Keywords: jumbo bacteriophage, Agrican357virus , burst size, novel, Erwinia ,Pantoea , genome, proteome\n",
      "INTRODUCTION\n",
      "Whitman et al. (1998) estimated that there are approximately 5 ×1030bacteria on earth, which is\n",
      "more than the number of plants and animals combined. Most, or likely all, bacteria are subject to\n",
      "infection by one or more viruses or “bacteriophages, ” making bacteriophages the most common\n",
      "and diverse biological entity at an estimated 1032(Bergh et al., 1989; Wommack and Colwell,\n",
      "2000; Hambly and Suttle, 2005). Bacteriophages were likely ﬁrst reported in 1896, when Ernest\n",
      "Hanbury Hankin discovered antibacterial activity against cholera in the waters of two large\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: fmicb-12-616548 March 4, 2021 Time: 17:51 # 1\n",
      "ORIGINAL RESEARCH\n",
      "published: 10 March 2021\n",
      "doi: 10.3389/fmicb.2021.616548\n",
      "Edited by:\n",
      "Indrani Karunasagar,\n",
      "Nitte University, India\n",
      "Reviewed by:\n",
      "Ahmed Askora,\n",
      "Zagazig University, Egypt\n",
      "D. Ipek Kurtböke,\n",
      "University of the Sunshine Coast,\n",
      "Australia\n",
      "*Correspondence:\n",
      "Chia Wanq Tan\n",
      "chiawanq@gmail.com\n",
      "Yaya Rukayadi\n",
      "yaya_rukayadi@upm.edu.my\n",
      "Specialty section:\n",
      "This article was submitted to\n",
      "Virology,\n",
      "a section of the journal\n",
      "Frontiers in Microbiology\n",
      "Received: 12 October 2020\n",
      "Accepted: 19 February 2021\n",
      "Published: 10 March 2021\n",
      "Citation:\n",
      "Tan CW, Rukayadi Y, Hasan H,\n",
      "Abdul-Mutalib N-A, Jambari NN,\n",
      "Hara H, Thung TY, Lee E and Radu S\n",
      "(2021) Isolation and Characterization\n",
      "of Six Vibrio parahaemolyticus Lytic\n",
      "Bacteriophages From\n",
      "Seafood Samples.\n",
      "Front. Microbiol. 12:616548.\n",
      "doi: 10.3389/fmicb.2021.616548\n",
      "Isolation and Characterization of Six\n",
      "Vibrio parahaemolyticus Lytic\n",
      "Bacteriophages From\n",
      "Seafood Samples\n",
      "Chia Wanq Tan1*, Yaya Rukayadi1*, Hanan Hasan1, Noor-Azira Abdul-Mutalib1,2,\n",
      "Nuzul Noorahya Jambari1,2, Hirofumi Hara3, Tze Young Thung4, Epeng Lee2and\n",
      "Son Radu1,2\n",
      "1Department of Food Science, Faculty of Food Science and Technology, Universiti Putra Malaysia, UPM Serdang, Selangor,\n",
      "Malaysia,2Food Safety and Food Integrity, Institute of Tropical Agriculture and Food Security (ITAFoS), Universiti Putra\n",
      "Malaysia, UPM Serdang, Selangor, Malaysia,3Department of Environmental Engineering and Green Technology,\n",
      "Malaysia-Japan International Institute of Technology, Universiti Teknologi Malaysia, Kuala Lumpur, Malaysia,4Infection\n",
      "and Immunity Program, Department of Microbiology, Biomedicine Discovery Institute, Monash University, Clayton, VIC,\n",
      "Australia\n",
      "Vibrio parahaemolyticus is a foodborne pathogen that is frequently isolated from a\n",
      "variety of seafood. To control this pathogenic Vibrio spp., the implementation of\n",
      "bacteriophages in aquaculture and food industries have shown a promising alternative\n",
      "to antibiotics. In this study, six bacteriophages isolated from the seafood samples\n",
      "demonstrated a narrow host range speciﬁcity that infecting only the V. parahaemolyticus\n",
      "strains. Morphological analysis revealed that bacteriophages Vp33, Vp22, Vp21, and\n",
      "Vp02 belong to the Podoviridae family, while bacteriophages Vp08 and Vp11 were\n",
      "categorized into the Siphoviridae family. All bacteriophages were composed of DNA\n",
      "genome and showed distinctive restriction fragment length polymorphism. The optimal\n",
      "MOI for bacteriophage propagation was determined to be 0.001 to 1. One-step growth\n",
      "curve revealed that the latent period ranged from 10 to 20 min, and the burst size\n",
      "of bacteriophage was approximately 17 to 51 PFU/cell. The inﬂuence of temperature\n",
      "and pH levels on the stability of bacteriophages showed that all bacteriophages\n",
      "were optimally stable over a wide range of temperatures and pH levels. In vitro\n",
      "lytic activity of all bacteriophages demonstrated to have a signiﬁcant effect against\n",
      "V. parahaemolyticus . Besides, the application of a b\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    }
   ],
   "source": [
    "# add 3 microbiology pdf\n",
    "from pathlib import Path\n",
    "\n",
    "def find_files(directory, ext = '*.pdf'):\n",
    "    # Convert the directory string to a Path object\n",
    "    path = Path(directory)\n",
    "\n",
    "    # Check if the provided path is a directory\n",
    "    if not path.is_dir():\n",
    "        raise ValueError(\"Provided path is not a directory\")\n",
    "\n",
    "    # Loop through the directory\n",
    "    for file in path.glob(ext):\n",
    "        # Each 'file' is a Path object representing a PDF file\n",
    "        yield str(file)\n",
    "\n",
    "\n",
    "for file in find_files(\"/workspaces/ml-learning/src/phages/data/test3/\"):\n",
    "    library.add(file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "query = 'What are the bacteriophages(s) that show promising results in aquaculture and food industries and what is the pathogen that each can control?'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    }
   ],
   "source": [
    "response_adoc_match = await library._adoc_match(query)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[NodeWithScore(node=TextNode(id_='5eb07d54-a7f3-4e5b-b8dd-0677bc1ee2ee', embedding=[-0.0067372723, -0.018468704, -0.005094457, -0.024804784, -0.0040084696, 0.012457728, -0.024514263, 0.0041675633, -0.010507101, -0.013481462, 0.016905434, 0.03962125, -0.0132186115, -0.0048108553, -0.010763035, 0.0020837816, 0.029383915, -0.029549927, 0.021941097, -0.03638404, 0.006685394, 0.009946815, -0.0061493176, 0.00017249562, 0.0026233168, 0.012990346, 0.008279789, -0.019049741, -0.014844134, 0.0014223323, 0.0019454393, -0.0030590952, -0.007954685, 0.011752183, -0.016933104, -0.026824582, 0.02855386, -0.006132025, 0.029162567, 0.009580208, 0.006996664, -0.008964584, -0.0002842503, 0.0041191434, 0.005606324, 0.00957329, 0.010832206, -0.007062377, -0.020460833, 0.012637573, 0.05334481, -0.014525947, 0.0014474068, 0.03148672, -0.009607876, -0.0034135976, 0.016241392, 0.004548005, -0.0063395384, -0.01936793, -0.0018243897, -0.008487303, -0.019312592, -0.025966858, 0.0012113602, -0.012312469, -0.0027080516, 0.0109912995, -0.0037525361, 0.0042920713, 0.013481462, 0.014041749, 0.0067303553, -0.005346932, 0.026769245, -0.017306628, -0.018122848, 0.0031299957, -0.006342997, 0.0072975587, 0.011447829, -0.00777484, -0.0049353633, -0.0011707222, 0.021539904, -0.010610858, 0.026700074, 0.020073475, -0.009428031, -0.008881579, 0.03560932, 0.049415886, 0.02747479, 0.011427078, -0.016352065, 0.017458804, -0.030130964, 0.030186301, 0.0013159816, -0.039095547, -0.004561839, -0.008203701, -0.013910323, -0.02178892, -0.039482906, 0.003541564, 0.008846994, -0.0037559948, 0.028498525, -0.022480631, -0.027336448, 0.007864763, -0.009047589, -0.048945524, 0.02793132, 0.022577472, 0.02838785, -0.010970549, -0.022950996, -0.008660231, 0.008369712, 0.031569723, -0.02476328, -0.009566373, 0.037850466, -0.0004751195, -0.017043777, -0.028235674, -0.0010289212, -0.007038167, 0.027128935, 0.01202195, 0.024016231, 0.001708528, -0.011357907, 0.021180214, -0.058712494, -0.025261313, -0.009206683, -0.014553615, -0.0036384037, 0.038763527, -0.0007033844, -0.001336733, -0.0068479464, -0.0084250495, 0.016905434, 0.031348377, -0.0032112717, -0.001655785, 4.2772645e-05, -0.017043777, -0.006796068, 0.011648426, -0.018039843, 0.011413244, -0.0099814, 0.004337033, -0.01972762, -0.038099483, -0.029522257, -0.00098396, 0.0031784156, -0.0010877168, 0.035387974, 0.036024347, 0.017915335, -0.0025091844, -0.008985336, 0.005606324, -0.008480386, 0.011683011, -0.016393568, 0.007844011, 0.0003899525, 0.025247479, 0.012298635, 0.0028152668, 0.011122725, -0.024873955, -0.018205853, 0.008791656, 0.033672526, 0.01161384, 0.009144429, 0.02603603, 0.029909616, -0.03740777, 0.02378105, -0.026575565, 0.015784862, 0.0026855709, 0.019589277, 0.014954808, -0.63482535, -0.032316774, -0.006771858, 0.0012753436, -0.0046898057, 0.006841029, -0.0024106153, -0.006581637, -0.006111273, 0.022120941, -0.018219687, 0.0032320232, -0.0048696506, -0.03596901, -0.01862088, -0.027972823, -0.01701611, -0.02205177, 0.021705914, 0.006872156, -0.0250538, -0.0024797865, 0.0073874816, 0.016767092, 0.018302692, 0.013529882, 0.032565787, -0.003417056, 0.0046033417, 0.014954808, -0.01898057, 0.023905559, -0.003856293, -0.012589154, 0.046925724, -0.01989363, 0.024680275, 0.022715814, 0.034806937, -0.0042920713, -0.011067388, -0.0225083, 0.012775916, 0.024873955, -0.016213723, 0.023172343, 0.044075873, 0.010832206, -0.027543962, -0.0060144337, -0.0034066804, 0.0022861073, -0.0034741221, 0.012174127, 0.020848192, -0.028166503, 0.025247479, -0.03646704, 0.011157311, 0.02070985, -0.0069551617, 0.017140618, 0.0081691155, -0.025579501, -0.001869351, 0.011925111, -0.020004304, 0.005257009, 0.016642585, -0.014705792, -0.0050114514, 0.008106861, -0.01120573, -0.012796667, 0.0039946353, 0.039316896, 0.008701734, 0.0031161613, -0.0006670695, 0.0049491976, -0.0041813976, 0.0027149685, -0.015328333, 0.019022074, 0.016711757, 0.0050252858, -0.02800049, 0.005384976, 0.012347055, -0.032289103, 0.007816343, 0.017583312, -0.001188015, -0.011371741, -0.025759345, 0.00803769, -0.001473346, -0.021194048, -0.0043716184, -0.005509484, -0.014048666, 0.0019731077, 0.030047959, -0.001794992, 0.023130842, -0.014290765, -0.023850221, -0.0073390617, 0.027391786, -0.016490407, -0.009690882, -0.01571569, -0.015120819, 0.0052708434, 0.0078094257, -0.035858337, 0.019630779, 0.022909492, 0.004278237, -0.015342167, 0.020613011, 0.004492668, -0.004765894, 0.009697799, 0.016407402, 0.016186055, -0.002829101, -0.007242222, -0.013101021, -0.0014007163, 0.014691957, 0.007532741, 0.022217782, -0.018482538, 0.016448906, 0.020917363, 0.027972823, 0.013972578, 0.012768999, -0.029992621, -0.0261052, -0.00818295, 0.015522012, 0.000490683, -0.007228388, -0.010970549, -0.011046637, -4.890618e-06, 0.006083605, -0.004406204, 0.02025332, 0.028415518, 0.004011928, 0.0072768074, 0.01130257, -0.021014202, -0.024804784, -0.041530374, 0.009683965, -0.00040421906, 0.0027063221, 0.010783786, 0.00038087377, -0.004592966, 0.0029484213, 0.0008179491, 0.015203824, 0.0240439, -0.010970549, -0.014581284, 0.01969995, -0.007304476, -0.020530004, 0.022632807, -0.0045825904, 0.041724052, -0.01370281, 0.011683011, -0.0135575505, -0.010790703, -0.008784739, -0.02051617, 0.008072276, 0.028332513, 0.011489332, -0.008051525, 0.0071661337, -0.004575673, 0.031237703, 0.039510574, -0.003330592, -0.0013211694, 0.015798697, -0.0035260005, -0.0018174726, 0.015342167, 0.015176156, 0.011731431, -0.002488433, 0.010458682, 0.006827195, 0.0026025653, 0.01216721, -0.0214984, 0.021954931, -0.031514388, 0.004433872, -0.036633056, 0.028277176, 0.028208004, 0.020059641, -0.01763865, -0.028360182, 0.014193925, 0.0031023272, 0.0076849177, 0.0016160116, -0.002630234, -0.01682243, 0.01238164, -0.0025351236, 0.008646397, 0.02142923, 0.0065643443, -0.047645107, 0.01456745, 0.005986765, 0.02241146, 0.0051532527, 0.010500184, 0.005416103, 0.01823352, 0.012810501, 0.016421236, 0.031735733, 0.004278237, 0.016573414, -0.014290765, -0.003565774, 0.014719626, -0.006951703, 0.019630779, 0.016988441, -0.012921175, 0.008501138, 0.00393584, 0.015729526, 0.014104002, -0.0077402545, -0.005250092, -0.011129642, 0.0058276714, -0.032233767, 0.0026267753, 0.0015010145, -0.021277053, -0.0026198581, -0.007581161, 0.0088539105, 0.0067026867, 0.009262021, 0.009040672, -0.0043543256, 0.0027184272, 0.013882655, -0.01473346, -0.028664535, 0.006505549, -0.03306382, -0.00950412, -0.022162445, -0.005817296, 0.0057723345, -0.002680383, 0.0033963048, -0.0038908785, 2.3088582e-05, 0.010285754, -0.005450689, -0.013861903, -0.038376167, -0.02874754, 0.0037802048, 0.030684333, 0.015106984, 0.0044788336, -0.019243421, 0.008210618, -0.02721194, 0.007899349, 0.015729526, 0.015480509, 0.015978541, 0.0069344104, -0.022328455, 0.009054507, 0.01900824, -0.007601912, 0.006419085, 0.009600959, 0.0028446645, -0.0026976757, -0.016435072, -0.012132624, 0.025593335, -0.013695893, -0.011793685, -0.0083420435, 0.0050356616, -0.031127028, 0.037739795, -0.0061251074, 0.004586049, 0.0072975587, -0.020267155, -0.0018935609, -0.021595242, 0.03450258, 0.008860827, -0.023504365, -0.005094457, -0.025800848, -0.0072768074, 0.0031680397, 0.043135144, 0.010700781, 0.014069417, 0.018178185, 0.04811547, -0.009289688, -0.017237457, -0.014844134, 0.006796068, -0.01999047, 0.008833159, -0.0043335743, 0.0051082913, 0.004734767, 0.0106523605, 0.0039220056, -0.008051525, -0.008826242, 0.0009078717, 0.013612887, 0.0038597516, -0.010057488, 0.020599175, 0.023006333, 0.011952779, -0.0012822606, 0.044601575, 0.022369958, 0.015757194, -0.01665642, -0.026838416, -0.0018019091, 0.0078094257, -0.0005490462, 0.034142893, -0.00024685464, -0.0042194417, 0.0014024456, 0.004413121, 0.024721777, 0.0039946353, 0.030407649, 0.0027149685, -0.002618129, -0.006128566, 0.013937992, -0.014332267, 0.006747648, -0.019257255, -0.022286953, 0.04866884, -0.0051670866, 0.012948844, -0.028415518, 0.012928093, 0.0019834833, -0.020640679, -0.0005953044, -0.0066646426, -0.020488502, 0.0035450228, -0.008577226, 0.0010159516, 4.2799664e-05, -0.027087431, 0.0026267753, -0.026727742, -0.017721655, -0.028277176, -0.0034464537, -0.008369712, -0.010804538, -0.03635637, 0.0049561146, -0.004115685, -0.00662314, 0.010348008, -0.006771858, 0.004136436, -0.0013021474, -0.014083251, 0.005620158, 0.010686946, -0.041530374, -0.002836018, 0.0153560005, 0.010832206, -0.00931044, -0.01574336, -0.006785692, 0.0033011944, 0.033893876, 0.019782957, -0.0011620758, 0.0020007761, 0.006381041, 0.005149794, -0.0033150285, -0.011980448, -0.003491415, 0.010776869, -0.038071815, -0.0060593947, -0.003769829, -2.4101832e-05, -0.018012173, -0.016988441, 0.0042678616, -0.019284923, -0.0035726912, 0.01626906, -0.010085157, 0.0014128212, 0.018800726, 0.03450258, 0.024099238, 0.009607876, -0.004240193, 0.020585341, 0.004921529, 0.0049734074, 0.00031840356, -0.009822307, -0.00011251124, -0.026838416, 0.022452964, -0.00950412, -0.030490654, -0.028277176, 0.0073874816, -0.011295653, 0.023476698, -0.005384976, 0.0008862557, -0.046925724, -0.0127067445, 0.03204009, 0.013370788, -0.0033254041, -0.036771398, -0.01823352, -0.0041502705, 0.017915335, 0.000716354, 0.003614194, -0.04360551, 0.00662314, 0.010251168, -0.016850099, 0.022286953, -0.0012883132, -0.007242222, 0.00011953644, 0.013377706, -0.041475035, -0.02515064, -0.0065505104, 0.0113371555, 0.024196077, 0.013578301, 0.028830545, -0.014138588, -0.0047382256, 0.029715937, 0.011883608, -0.0004303744, -0.015992375, 0.0056097824, -0.016448906, 0.0068859905, 0.016130717, 0.004890402, -0.008840076, -0.028968887, 0.007761006, 0.027903652, 0.0144982785, -0.009808472, 0.0063118697, -0.01845487, -0.006740731, -0.0074428185, -0.012554568, -0.00957329, -0.009338109, -0.028443187, 0.030628996, -0.017126784, 0.0085149715, -0.0019955884, 0.019589277, -0.014304599, 0.015023979, -0.004077641, 0.01823352, -0.024721777, -0.002792786, -0.03383854, -0.0058899256, 0.025275147, 0.024223745, 0.0007422932, 0.031403713, -0.008459635, 0.020018138, 0.01089446, -0.014802631, -0.011454746, -0.017154451, -0.00079417153, -0.024251414, -0.01691927, -0.014429107, -0.02646489, -0.00034174885, -0.0075465753, -0.024514263, 0.014401439, -0.014145506, -0.026160538, 0.017168285, 0.0111642275, 0.04086633, -0.023836387, 0.0363287, 0.036079686, -0.002203102, -0.019257255, -0.0046828887, -0.019478602, -0.015854033, 0.04623401, 0.01780466, -0.028180337, -0.030822676, -0.0024867037, 0.03577533, -0.016033879, -0.027779143, 0.033589523, -0.0020197984, -0.0045237946, -0.015480509, -0.0052327993, -0.05262543, 0.014968642, -0.019215751, 0.036577716, -0.0021287429, -0.0029328577, -0.0072629736, 0.008736319, 0.0024970793, 0.032814804, 0.02315851, -0.011247233, 0.017223623, -0.010458682, -0.027516292, 0.0056547434, -0.014318434, 0.006944786, 0.0060075168, -0.0017284147, -0.0033150285, 0.0077679227, 0.013640556, 0.005049496, 0.00042583505, 0.03655005, -0.014581284, 0.0040153866, -0.006176986, 0.018703885, 0.0075189066, -0.014996311, 0.0039462154, -0.008141447, -0.013709727, -0.0029570677, 0.032953147, 0.0065020905, 0.015798697, -0.021346224, -0.012782834, -0.032261435, -0.011274901, -0.016435072, -0.004624093, -0.02178892, 0.0077886744, 0.0006735543, -0.0072145537, -0.004375077, -0.006688853, -0.0012226005, -0.011558503, 0.016670253, -0.026160538, 0.009545622, 0.010015986, -0.008833159, -0.03688207, -0.0047174743, 0.015259162, -0.021927262, -0.0015407879, -0.004146812, -0.034640923, -0.0332575, -0.0125269, 0.045708314, -0.020557674, -0.018607046, 0.017776992, -0.013377706, 0.008743237, 0.0031801448, -0.0149271395, 0.03959358, -0.0035381056, 0.010998216, 0.0013791003, -0.0039669666, 0.032399777, -0.004402745, -0.008840076, -0.01089446, -0.014913306, -0.035553984, -0.0020319032, 0.014097085, 0.009746218, -0.01780466, -0.000117482916, -0.012042701, -0.0052708434, -0.017195955, 0.0026717365, -0.030795006, 0.0028014325, 0.013868821, 0.011786768, -0.009587125, -0.0035761497, -0.018205853, -0.010451765, -0.0029726312, -0.012270967, -0.0047243913, -0.024735613, 0.031597394, -0.02035016, -0.028968887, -0.023435194, -0.011724514, -0.025690174, -0.0029069185, -0.004049972, 0.0006307546, 0.025205975, -0.0154390065, 0.008259038, 0.015037813, -0.009538705, -0.0013194402, 0.0052812193, -0.021595242, 0.019976635, 0.007781757, 0.015535845, -0.0016108238, -0.027779143, 0.008217536, 0.038486842, 0.019575443, 0.0028498524, 0.01574336, -0.010147411, 0.0036971993, -0.004720933, 0.005564821, 0.0013574843, 0.005364225, 0.015369835, -0.037878137, -0.01989363, 0.024016231, 0.0014984206, 0.0033945753, -0.011095056, -0.001101551, 0.010458682, 0.019271089, -0.021512235, -0.023075504, 0.013571384, -0.01447061, 0.026630903, 0.004264403, 0.013771981, 0.008577226, 0.0103964275, -0.031874076, -0.0134745445, 0.01662875, -0.019035907, 0.009234352, 0.012955761, 0.0060628536, 0.00940728, -0.018288858, 0.0016384922, -0.019188084, 0.012077287, -0.0010669654, -0.014968642, -0.021996433, 0.020862026, 0.013135606, -0.012111872, -0.049498893, 0.008784739, -0.022701979, -0.003738702, -0.0048558163, 0.016518077, -0.017209789, 0.015784862, 0.018925233, -0.016227558, 0.018178185, 0.009262021, -0.005602865, -0.011655343, -0.0012269237, 0.17320462, 0.001108468, 0.0091928495, 0.033036154, -0.0046967226, 0.0125269, -0.0057896273, 0.0049180705, 0.0046828887, 0.012672159, 0.045957327, 0.0012848546, 0.015010145, 0.006010975, 0.00046906702, 4.4096625e-05, -0.03455792, -0.021705914, -0.016780928, -0.017721655, 0.0016160116, -0.015065482, -0.009054507, 0.0009485097, 0.031044023, -0.011157311, -0.031763405, 0.0017569478, 0.0063844994, -0.015245327, -0.013529882, -0.018634714, 0.013675141, 0.014871802, -0.0003315893, 0.002190997, 0.0005222424, 0.013024932, 0.021678247, -0.0084458, -0.021415396, -0.019354094, -0.008694816, -0.01969995, -0.008321293, 0.007878597, -0.021124877, 0.004091475, -0.012215629, 0.0012986888, -0.02793132, 0.0001726037, -0.0042747785, 0.015757194, 0.010410261, -0.042415764, 0.029079562, 0.027281111, -0.005001076, 0.027184272, -0.012602988, 0.018593213, -0.03002029, 0.010105909, -0.023725713, 0.020723684, -0.0072491392, -0.0055509866, 0.010880626, -0.013903406, 0.0071661337, -0.0062530744, -0.019478602, -0.010514018, -0.01545284, -0.027889818, 0.033008486, 0.013931074, 0.029798942, 0.0424711, -0.001782887, 0.0117660165, -0.01396566, -0.01283817, -0.0022376876, -0.038929537, 0.019381763, -0.000266309, -0.010804538, -0.005402269, -0.014788797, -0.024901623, -0.016808596, 0.013564467, 0.015231493, 0.006197737, -0.01247848, -0.021816589, 0.00782326, -0.010783786, -0.017597146, 0.045652974, 0.014968642, -0.0064709634, -0.022812653, 0.0012355701, -0.020931197, 0.0043335743, 0.023794884, -0.018081345, -0.0005563956, -0.0036314868, 0.006256533, -0.017292794, 0.0080031045, 0.02946692, 0.023047835, -0.0388742, 0.025441159, 0.006505549, -0.002481516, -0.016960772, 0.0014136859, 0.0055682794, -0.0036314868, -0.01907741, -0.03234444, 0.0065574273, 0.0038424588, -0.026921421, -0.016407402, 0.020391662, 0.009268938, -0.0065332176, -0.016061546, -0.007553492, 0.0030400732, -0.01166226, -0.01130257, -0.005336556, -0.015632685, 0.006553969, 0.008452717, 0.010472516, -0.009386528, -0.02639572, 0.030933348, 0.017195955, -0.020004304, -0.02730878, -0.012914258, -0.012609906, -0.026354218, -0.006374124, 0.020654513, 0.0016082298, -0.020308657, -0.029383915, -0.011689928, -0.01783233, -0.04844749, -0.004852358, 0.020806689, -0.0034689344, 0.010181997, -0.019478602, -0.17729956, 0.028636867, 0.0009355401, -0.002044008, 0.030712001, 0.016670253, 0.03107169, 0.0013332744, -0.00292767, 0.0033755533, 0.024652606, 0.026575565, -0.030296974, -0.0113371555, -0.0011957966, 0.018026007, -0.01799834, 0.025302816, 0.04053431, 0.009849975, 0.029826611, -0.027059764, 0.0137304785, -0.013225528, -0.0016505972, 0.02584235, -0.019672282, 0.026049864, -0.009345026, -0.021609075, -0.0059348866, 0.002778952, 0.0031542056, -0.015466674, 0.01823352, 0.013467628, 0.0060248096, 0.0199628, -0.0062738257, 0.018122848, 0.015618851, 0.017776992, 0.0005477492, -0.0070277913, -0.00041826945, 0.016033879, 0.007940851, -0.0063395384, 0.026949089, -0.034225896, -0.005056413, -0.020751353, 0.0139864115, -0.0032977357, 0.026534062, 0.018399533, -0.02593919, 0.0123608885, -0.0021131793, 0.007761006, -0.013751229, 0.001696423, 0.00590376, -0.0154390065, -0.0029328577, -0.032870144, -0.041502707, -0.0062219473, -0.027654635, 0.00037979297, -0.009262021, 0.0026319632, 6.9495414e-05, 0.0019938592, -0.0002384244, 0.01293501, -0.020640679, 0.005613241, 0.017749323, 0.0049111536, -0.020184148, 0.016642585, -0.027460957, 0.00895075, 0.011184979, 0.007110797, -0.002018069, 0.016006209, -0.018275024, 0.0012104956, 0.002116638, -0.0032787137, 0.010970549, -0.024057735, 0.004883485, 0.022203946, -0.008923082, 0.015272995, -0.010610858, 0.0014162798, 0.010417179, -0.021678247, -0.0021633285, 0.050577965, 0.019423265, -0.0016065006, -0.03635637, -0.0041329777, 0.048724174, 0.0005957367, -0.032455117, -0.0029415041, 0.015217658, 0.0041675633, 0.0047693527, 0.000200056, 0.00077039393, -0.030490654, 0.008314375, -0.0055509866, 0.055613626, 0.0008417267, -0.0076157465, 0.041696385, -0.016946938, -0.015591183, -0.10464215, 0.023476698, -0.014608952, 0.0327318, -0.029743604, 0.014166256, 0.0036245696, 0.0033963048, 0.0032579624, 0.023656541, -0.007885514, 0.0014690228, -0.012111872, 0.0021823505, 0.0067683994, 0.015923204, -0.004758977, -0.010590107, -0.013522965, 0.020419331, 0.0021149088, -0.0009459158, 0.0040395968, -0.009511037, 0.00019638128, -0.0045272536, -0.022646643, 0.03143138, -0.0029553385, 0.012402392, 0.013502213, -0.0138204, 0.024735613, -0.015203824, -0.022217782, -0.00020038025, -0.017887665, 0.0015667271, 0.01862088, -0.033036154, 0.011295653, 0.0061078146, 0.014650455, -3.5733938e-06, 0.009919146, -0.022688145, -0.008667149, 0.0007059783, 0.011447829, -0.023670375, -0.03403222, -0.012810501, -0.031846408, 0.0030175925, 0.0332575, -0.04316281, -0.014940973, 0.018593213, -0.021567572, -0.010354925, -0.022674311, 0.01053477, -0.035000615, 0.02070985, -0.0102234995, -0.00772642, -0.0041710217, -0.026949089, 0.023767216, -0.03383854, 0.003534647, -0.0016505972, -0.0106523605, -0.011544669, -0.027862148, -0.009345026, -0.037601452, -0.009144429, 0.008016939, 0.013301617, 0.0020906988, -0.01672559, -0.018510206, -0.0023604664, 0.009870727, 0.005329639, 0.010195831, 0.026575565, 0.016158387, -0.0042851544, 0.0021200965, 0.033672526, -0.010548604, -0.015231493, -0.011904359, 0.027128935, 0.0019765664, -0.0011646696, 0.0076641664, 0.016186055, -0.041143015, 0.0022117484, -0.08820708, 0.047257748, 0.0063983337, -0.026381886, -0.0090199215, -0.02205177, -0.0137304785, 0.0031784156, 0.013177109, 0.03037998, -0.02629888, 0.035858337, -0.033921544, -0.01907741, -0.03292548, -0.017776992, 0.023130842, 0.0020405496, 0.017984506, 0.025358154, -0.0029293993, 0.013101021, 0.011288736, 0.026907587, -0.015784862, 0.017929168, -0.00935886, 0.033202164, -0.01718212, 0.00019800248, 0.014346101, -0.012001199, -0.0031784156, 0.025026131, -0.0018105555, -0.013004181, -0.01610305, 0.008840076, -9.175989e-05, 0.02649256, -0.032095425, -0.023213847, 0.007885514, -0.010583189, 0.0072975587, 0.013101021, -0.010797621, 0.010431013, -0.0046586785, 0.018676218, 0.026838416, 0.00470364, -0.009856893, -0.03270413, 0.021346224, -0.006699228, 0.009144429, 0.003256233, 0.0022117484, -0.0084250495, 0.025551831, 0.009905312, 0.029411584, 0.008245204, -0.018925233, -0.012118789, 0.014069417, 0.006201196, -0.009545622, -0.057993114, -0.02207944, -0.01960311, 0.005070247, 0.015245327, 0.0045549218, 0.008397381, -0.022107108, 0.0225083, 0.0144982785, -0.0044304137, 0.012630656, -0.001900478, -0.024085402, -0.016573414, 0.027972823, 0.0059383456, 0.015563514, 0.019520106, -0.01607538, -0.00078984833, -0.018344196, 0.02277115, 0.021249386, -0.0066196816, -0.02551033, 0.019160416, 0.003918547, -0.014871802, 0.03073967, 0.02476328, 0.005070247, 0.011620757, 0.018482538, -0.029688269, -0.0049941586, 0.0042540273, -0.02910723, -0.03364486, 0.0132186115, -0.005267385, -0.013737395, 0.005564821, -0.0063775824, 0.0072975587, 0.004177939, 0.0013047413, 0.0016047712, -0.0020578424, -0.027654635, 0.054506887, 0.014000245, 0.013038767, 0.01574336, 0.0001903288, 0.0016384922, 0.02430675, 0.037684456, -0.011475498, 0.022535969, -0.01094288, 0.00664735, 0.020640679, -0.026119035, -0.0076849177, -0.0069551617, 0.004599883, 0.024984628, -0.0022446045, -0.027696138, 0.07841244, 0.0025904605, -0.018759223, 0.012907341, -0.035387974, 0.0037559948, 0.015674189, 0.013875738, -0.0017880748, -0.026893754, 0.010188914, -0.0048800264, -0.0011594818, -0.043135144, -0.040727988, 0.017389633, -0.0134745445, 0.010721532, -0.031846408, -0.006944786, 0.01545284, 0.01130257, 0.029881947, 0.027875984, -0.017707821, -0.010029821, 0.015812531, -0.015411338, 0.011378658, -0.06214338, 0.013578301, -0.0022394168, -0.039648917, -0.03309149, 0.01610305, -0.027294945, 0.005713539, -0.023130842, 0.033534184, 0.02740562, 0.0048419824, -0.0031455592, -0.014802631, -0.010431013, 0.011890525, -0.0016843181, -0.04119835, -0.0064640464, -0.016753258], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-12-616548.pdf', 'citation': 'Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.83827609),\n",
      " NodeWithScore(node=TextNode(id_='cf029be1-6e2a-4a25-adec-164f15261c21', embedding=[0.00097649783, -0.013648757, -0.005952963, -0.02813134, 0.0065851663, 0.016867869, -0.027065137, -0.014339055, -0.012452696, -0.0004152038, 0.032095425, 0.024618339, -0.005703499, 0.016061382, -0.0019734995, 0.01700456, 0.020435546, -0.0061340807, 0.0129857985, -0.0013387332, -0.003073875, 0.022294564, 0.004732982, 0.007176362, -0.002185373, 0.016867869, 0.028268032, -0.029826328, -0.036742974, -0.0004562116, 0.0016368941, -0.0019205312, -0.04294882, 0.001327627, -0.027994648, -0.03499331, 0.03387243, -0.00656808, 0.035950158, 0.0068551344, 0.0072105355, 0.015227557, -0.002211003, 0.0069747404, 0.0018248464, -3.1636864e-05, 0.014926833, 0.0067252764, -0.015965696, 0.009213082, 0.037262406, -0.008263068, 0.002185373, 0.019027611, 0.00416229, 0.004842336, 0.023210406, -0.009643664, 0.0067867883, -0.020681592, 0.0051772324, -0.013908473, -0.026422681, -0.018166449, -0.0029252218, -0.013512065, -0.006902977, 0.0034702837, -0.013662427, 0.008597965, 0.020968646, 0.006677434, 5.510421e-05, 0.00314393, 0.017441977, -0.023005366, -0.0090763895, -0.0007851282, 0.010764543, 0.008160549, -0.00379322, -0.00069798663, -0.008809838, 0.0046612183, 0.017701693, -0.013300191, 0.020230506, 0.02793997, -0.010074246, 0.005963215, 0.037645146, 0.030755838, 0.010662024, 0.028350048, -0.008481776, 0.009370279, -0.01662182, 0.030783176, 0.010545835, -0.02977165, -0.013676096, -0.00495169, 0.0021648693, -0.020025468, -0.035403386, 0.012459531, 0.023798184, -0.01943769, -0.006233183, -0.018822573, -0.03163067, 0.022267226, -0.003936747, -0.027898962, 0.028432064, 0.017141253, 0.027502554, -0.030373098, -0.024194593, -0.011899092, 0.013354868, 0.018357817, -0.0063767103, -0.016526137, 0.023538468, 0.009971726, -0.029088188, -0.01570598, 0.004271644, -0.013177168, 0.020080145, 0.0073677315, 0.025930589, 0.0077983136, -0.022896012, 0.022267226, -0.050193526, -0.013006302, -0.00776414, -0.02755723, 0.011126778, 0.036305558, -0.009527475, 0.004165707, -0.023183066, 0.007292551, 0.022855004, 0.018139109, 0.004001676, 0.017441977, 0.012240823, -0.009008043, -0.015760658, 0.00990338, -0.012548381, 0.017701693, -0.008570626, 0.01868588, -0.008967035, -0.042374708, -0.02348379, -0.003314796, 0.018098101, -0.008351918, 0.035622094, 0.026368005, 0.0061204117, 0.008379257, 0.00739507, -0.0055975625, -0.015104533, 0.005942711, -0.017715363, 0.015787996, -0.0048799263, 0.021611102, 0.015610296, 0.0068346304, 0.017141253, -0.018795233, -0.0003494205, 0.023743507, 0.050658282, 0.0091515705, -0.0051943194, -0.008126376, 0.036907006, -0.017893063, 0.0063698757, -0.018644873, 0.008194722, 0.008201556, 0.011270306, -0.000374196, -0.6364408, -0.0484712, -0.0025014747, -0.0062400177, 0.0041896286, 0.0065954183, -0.005570224, -0.010491158, -0.019916113, 0.0249464, -0.011003755, 0.009862372, 0.009780357, -0.016006704, -0.018016087, -0.028404724, -0.008645807, -0.0076069436, 0.021747794, 0.02795364, -0.023757176, 0.011085771, -0.0036018502, 0.017086577, 0.002351113, 0.0005788078, 0.026778083, -0.0045928718, 0.0026125375, 0.017469315, -0.028350048, 0.032450825, -0.0014634653, -0.020941308, 0.047268305, -0.011215629, 0.0059119556, 0.018248463, 0.02035353, 0.0126577355, -0.0091515705, -0.03436452, 0.0072447085, 0.020312522, -0.026326997, 0.017113915, 0.02886948, 0.01515921, -0.022472266, -0.01461244, 0.009930718, 0.013375372, -0.0019547043, 0.01323868, 0.021173686, 0.004777407, 0.02977165, -0.022499604, 0.012445862, 0.00563857, 0.0006381836, 0.009431791, -0.0023203571, -0.017141253, -0.015254895, 0.013395876, -0.032122765, -0.006533907, 0.024632009, -0.026395343, 0.006072569, 0.020886632, 0.0076889596, -0.009835034, -0.007695794, 0.046147425, -0.006243435, 0.010873897, -0.00089619093, 0.011085771, -0.0126645705, 0.0007471106, -0.0076137786, 0.014024662, 0.011844415, 0.005484791, -0.027311184, 0.00039341842, 0.005102052, -0.016676499, 0.0033968114, 0.023893869, -0.008338249, -0.033599045, -0.022362912, -0.0029662296, -0.0010559504, 0.0031644339, 0.017332623, -0.016703837, -0.01701823, -0.017414639, 0.014093008, -0.0062707737, 0.011482179, 0.0029252218, -0.0051328074, -0.0014224575, 0.02497374, -0.008269903, -0.001946161, -0.007832486, -0.021720456, -0.016293759, -0.007545432, -0.033462353, 0.026614051, 0.030783176, -0.012978964, -0.014421071, 0.022690974, 0.004972194, -0.01793407, 0.001956413, 0.01210413, 0.024057899, 0.014598771, -0.013163499, -0.013628254, -0.006298112, 0.0013182294, -0.0060213096, 0.04573735, -0.00859113, 0.020490222, 0.014626109, 0.029552942, -0.008003352, 0.006212679, -0.035294034, -0.025452165, -0.008468107, 0.026244981, 0.001000419, -0.008215226, -0.014257039, 0.0021563258, 0.0011473636, 0.010415977, -0.0023733254, 0.019027611, 0.00568983, -0.006222931, -0.0013720521, 0.0018248464, -0.019177973, -0.029908344, -0.05016619, 0.006773119, -0.013184003, -0.0006347663, 0.018959265, -0.007524928, 0.013395876, -0.011707722, 0.005724003, 0.017852055, 0.012432193, -0.027160821, -0.019957121, 0.010867063, -0.030291082, -0.010258781, 0.025192449, -0.0066125053, 0.034692585, -0.012548381, 0.007600109, -0.02813134, -0.00944546, -0.002954269, -0.025561519, 0.00022896013, 0.02070893, 0.024167255, -0.0031712686, 0.017893063, -0.014940502, 0.02439963, 0.01589735, -0.0126030585, -0.008844012, 0.0089396965, 0.002351113, -0.0013635088, 0.012145138, 0.021665778, 0.017154923, 0.0062912772, 0.015787996, 0.011242967, -0.012022115, -0.009773522, -0.027270176, 0.019013941, -0.04423373, -0.007565936, -0.03351703, 0.029525604, 0.03868401, 0.010826055, -0.014940502, -0.021515418, 0.016703837, 0.00036736138, 0.010798716, -0.010163096, -0.0022485934, -0.00029944224, 0.00656808, -0.008768831, 0.0064963163, 0.016143396, 0.02035353, -0.023210406, 0.015104533, 0.0066705993, 0.016239082, -0.016198074, -0.0067252764, 0.0069405674, 0.025370149, 0.023005366, 0.003618937, 0.031193255, 0.006902977, 0.010887566, -0.0023852861, 0.022267226, 0.013142995, -0.028650772, 0.016731177, 0.025247125, -0.005809436, -0.0022229636, 0.019847767, 0.014776471, 0.010846559, -0.0076684556, 0.01664916, -0.01869955, 0.006803875, -0.024139915, 0.0020623498, 0.008160549, -0.01869955, -0.0063869623, 0.015186549, 0.0277486, 0.006212679, -0.002070893, -0.0090900585, 0.0092267515, 0.0013652175, 0.011953769, -0.0077914787, -0.017592339, -0.015610296, -0.033407673, 0.0049585244, -0.03496597, 0.006879056, 0.002670632, 0.014270709, 0.008584296, 0.0009995648, 0.0075932746, 0.009329271, 0.007900833, -0.015746988, -0.028787464, -0.023237744, 0.0090285465, 0.032122765, 0.02275932, 0.025438495, -0.017660685, -0.016512467, -0.021324048, -0.007196866, 0.004920934, 0.025711881, 0.0054130275, 0.010286119, -0.01081922, 0.0128696095, 0.0029030093, -0.025780227, 0.0086048, 0.029197542, 0.013997324, -0.0129652945, -0.021611102, -0.007470251, 0.019888775, -0.022896012, 0.009315602, -0.006978158, 0.0026962617, -0.029088188, 0.032696873, -0.0004895304, -0.008010187, 0.0011934973, -0.019601721, 0.007709463, -0.031111239, 0.017633347, 0.019683735, -0.01647146, 0.0041383686, -0.042374708, -0.009876041, 0.012357012, 0.026928445, 0.02941625, 0.011892257, 0.0054164445, 0.038820703, 0.006246852, -0.016676499, -0.0091379015, 0.020134822, -0.028186016, -0.006872221, -0.0046441313, 0.0027030965, -0.0040700226, -0.002894466, 0.0027167657, -0.009773522, -0.014776471, -0.0017411221, 0.010791882, -0.003847897, -0.009561649, 0.027803278, 0.006663765, 0.0033113784, -0.02349746, 0.02832271, 0.025985265, 0.013771781, -0.00867998, -0.021775132, -0.01313616, 0.00850228, 0.010586843, 0.02979899, 0.0010046907, -0.0128559405, -0.006609088, -0.0029218046, 0.019506035, 0.026491027, 0.024385963, 0.004702226, -0.001332753, -0.008447603, -0.0003171696, 0.00090815156, 0.01850818, -0.020093814, -0.017428309, 0.04051569, -0.008290407, 0.005659074, -0.0207226, 0.012151973, -0.005061044, -0.024290277, -0.0021922078, -0.019150635, -0.01359408, -0.012931121, -0.018726887, -0.0019427438, 0.00012313014, -0.028459402, 0.0124732, -0.017305285, -0.015063526, -0.024372293, -0.0021682866, -0.008276737, -0.01924632, -0.03545806, -0.0014822605, -0.00033831422, 0.00016659412, 0.016034042, -0.016498799, -0.011728226, 0.007340393, -0.02295069, -0.0030414106, 0.0011764108, -0.036524266, 0.029142864, 0.016129728, 0.014885825, -0.0072993855, -0.035895478, 0.0029218046, 0.014024662, 0.013129326, 0.010600512, 0.006598836, -0.01515921, -0.0091037275, 0.010791882, 0.0052079884, -0.010504827, 0.00046005606, -0.0016949883, -0.033599045, -0.013177168, -0.00084920286, 0.005016619, -0.011372825, 0.00034365378, 0.00823573, -0.022991696, 0.00647923, 0.024344955, -0.01118829, 0.008789334, 0.007880329, 0.035649434, 0.03884804, 0.024878055, -0.0053480985, 0.022663634, -0.0068961424, 0.006048648, 0.013231845, 0.0067355284, 0.013600915, -0.015036187, 0.022991696, 0.00591879, -0.014298047, -0.04084375, -0.010456985, -0.017318955, 0.021255702, 0.011653045, -0.0010986668, -0.043413572, -0.005324177, 0.018603865, 0.016020374, 0.0019000273, -0.04267543, -0.01662182, 0.0021717038, -0.00067705556, -0.014817479, 0.00081844704, -0.039312795, -0.0045586987, 0.0054266965, 0.010682528, 0.015993036, 0.0077026286, 0.0005074713, -0.00867998, 0.018549187, -0.027270176, -0.03567677, 0.00079068134, -0.0021050663, 0.026737075, 0.023155728, 0.027816946, -0.0017359961, 0.022089526, 0.022731982, 0.0090558855, 0.0031986071, -0.012076792, -0.0074087395, -0.013908473, 0.012521043, 0.0029423086, -0.016170736, -0.013682931, -0.008174218, -0.0031165916, 0.022157872, 0.008351918, 0.0006864532, -0.003718039, -0.030017698, -0.029826328, 0.0043878327, -0.0026415847, 0.021898156, -0.042757448, -0.03422783, 0.022362912, -0.008208391, 0.00953431, -0.0041793766, 0.020790946, -0.0103613, 0.023073712, -0.0076274476, 0.010949078, -0.013471057, -0.0074497475, -0.023647822, -0.01165988, 0.028896818, 0.015596627, -0.0022673886, 0.023032704, -0.007032835, 0.008987539, 0.0006351935, -0.0063220332, -0.01248687, -0.024454309, -0.019109627, -0.018275803, -0.029170204, 0.027311184, -0.024317615, -0.003434402, -0.014229701, -0.01793407, 0.009582152, -0.0121178, -0.014120347, 0.0069303154, 0.025370149, 0.029334234, -0.019929783, 0.025001079, 0.037453774, 0.008912358, -0.024591, 0.012145138, -0.020449216, -0.006181923, 0.031712685, 0.024864387, -0.012876444, -0.033380337, 0.020886632, 0.030373098, -0.014585102, -0.020941308, 0.02368883, 0.016225412, -0.0003178103, -0.032286793, -0.01027245, -0.020872962, 0.0048799263, 0.013580411, 0.021392394, -0.010648354, 0.0044083367, -0.014913164, 0.003618937, -0.0063903797, 0.026067281, 0.0035266695, -0.016430452, 0.0051088864, -0.029142864, -0.019670067, 0.010327127, 0.002091397, 0.011598368, -0.0007778664, 0.008891854, 0.020080145, 0.0042887307, -0.00015142978, -0.002884214, 0.004606541, 0.035758786, 0.0016966971, 0.0029354738, -0.024905393, 0.020476554, 0.012288665, -0.01719593, 0.0035164175, -0.004380998, -0.00379322, -0.012445862, 0.031357285, 0.0038854876, 0.016020374, -0.022431258, -0.012411689, -0.011694053, -0.01406567, -0.03996892, 0.008003352, -0.03608685, 0.013669261, 0.00693715, 0.008338249, 0.01905495, 0.0051430594, 0.003943582, -0.015090864, 0.03239615, -0.023183066, -0.00034258584, 0.004845753, -0.020763608, -0.024153585, 0.00076120696, -0.0012276705, -0.004876509, 0.004216967, -0.0027834033, -0.010573174, -0.030291082, -0.021036994, 0.018098101, 0.005324177, -0.03089253, 0.014585102, -0.00266038, 0.018836241, -0.0128354365, -0.010067411, 0.028104002, -0.009274594, 0.004808163, -0.0016360397, -0.004808163, 0.038191915, 0.005310508, -0.019095957, -0.014899495, -0.009691506, -0.046393473, -0.0016206618, 0.009588987, 0.012534712, -0.013423215, 0.0068107094, -0.009500137, -0.011174621, -0.014530425, -0.0047158953, -0.007921337, -0.008837177, -0.006762867, -0.0017940905, 0.0063425372, -0.002395538, -0.016854199, 0.0090285465, -0.0049892804, -0.026969451, 0.006069152, -0.018972935, 0.026737075, -0.039941583, -0.020613246, -0.034173153, -0.009240421, -0.0373991, -0.001644583, -0.0091925785, -0.004797911, 0.016731177, -0.0073677315, -0.004886761, 0.015227557, -0.013259184, 0.011167786, 0.000662532, -0.0035164175, 0.007401905, 0.0042887307, 0.00065954187, 0.0014609023, -0.031220593, -0.005211406, 0.03436452, 0.03204075, 0.002540774, 0.021173686, -0.008153714, 0.009753018, -0.001657398, -0.007969179, -0.013901639, 0.011523187, 0.0063322852, -0.050084174, -0.012671405, 0.018070763, -0.010853393, -0.01257572, -0.005898286, -7.3525676e-05, -0.0019991295, 0.030974546, -0.0048047453, -0.021406064, 0.009889711, -0.0069849924, 0.032997597, 0.0045689507, 0.0034702837, 0.018535517, 0.016375775, -0.026491027, -0.044698484, 0.019547043, -0.0091515705, -0.0014104969, -0.0012960169, -0.01009475, 0.014462078, -0.013019972, -0.006181923, 0.0073540625, 0.008351918, -0.0006928607, -0.001831681, -0.006752615, 0.008440768, 0.0310839, -0.0381099, -0.04696758, 0.0022776406, -0.026805421, -0.010949078, -0.009240421, -0.00040986424, -0.02628599, 0.021788802, -0.00040409752, -0.01257572, 0.0063527892, -0.0042784787, -0.0074839205, -0.004206715, -0.014120347, 0.17070173, -0.02147441, 0.014858487, 0.03204075, -0.0025783645, -0.008823508, 0.005744507, -0.0053788545, -0.009411287, 0.013006302, 0.042046648, 0.00832458, 0.010245111, 0.003923078, 0.011133613, -0.0041349516, -0.031220593, -0.014284378, -0.012439027, -0.022909682, 0.004336573, -0.0053788545, -0.011625706, -0.0038000545, 0.022321904, -0.01979309, -0.033188965, 0.014503086, 0.027912632, -0.015555619, -0.030263744, 0.0016676498, 0.016225412, 0.0063186157, -0.012042619, 0.0021272788, 0.009219917, 0.024194593, 0.014284378, -0.00037013798, 0.0023681994, -0.017428309, -0.007121685, -0.02515144, -0.024003223, 0.0018573109, -0.030017698, -0.0022332156, -0.020503892, -0.004982446, -0.03165801, 0.01682686, -0.019601721, 0.016908877, -0.006127246, -0.031958733, 0.011249802, 0.022786658, 0.009739349, 0.03242349, -0.011967438, 0.025465833, -0.017059239, 0.020886632, -0.009411287, 0.00786666, -0.010709866, -0.009575318, 0.022253556, -0.0054027755, 0.010005899, -0.0016949883, -0.002211003, -0.026409013, -0.023524798, -0.021201024, 0.022335572, 0.011372825, 0.03567677, 0.048744585, 0.002720183, 0.006868804, -0.021788802, -0.0025834905, -0.0015283943, -0.026203973, 0.042784788, 0.0028278283, -0.019765751, -0.008379257, -0.014202362, 0.0068414654, -0.022226218, -0.0072652125, 0.014134016, -0.0035300867, -0.009459129, -0.0069200634, 0.024303947, -0.00758644, -0.025766557, 0.06440956, -0.00795551, -0.0005211406, -0.016717507, 0.023702499, -0.025438495, 0.012261327, 0.014298047, -0.016785853, 0.005471122, -0.006728694, -0.0026689232, -0.025192449, 0.025711881, 0.038082562, 0.030619144, -0.015651304, 0.031357285, -0.02681909, -0.017660685, -0.006171671, -0.0050747134, 0.023073712, -0.019547043, -0.01515921, -0.008850846, -0.010299788, -0.0089191925, -0.025042087, -0.0010858519, 0.015842674, 0.0070191654, 0.0025612777, -0.007545432, -0.00018560294, 0.0070260004, -0.020982316, -0.010955913, 0.011782903, -0.015213887, 0.0075932746, 0.027434207, -0.006947402, 0.00859113, -0.019888775, 0.032696873, 0.007360897, -0.0042032977, -0.022786658, -0.001384867, -0.029443588, -0.012316004, 0.0056180665, 0.01628009, -0.025233457, -0.01570598, -0.043878328, -0.016362106, -0.008515949, -0.0418006, 0.006923481, 0.034801938, -0.0015002014, 0.00065099855, -0.0031986071, -0.17452912, 0.017223269, -0.0089191925, 0.00038850604, 0.022813996, 0.017674355, 0.0024775537, -0.009794026, 0.011318148, -0.0027082223, 0.03942215, 0.025766557, -0.042429388, -0.023880199, 0.0024997662, 0.025561519, -0.015186549, 0.017428309, 0.03124793, 0.009910215, 0.034911294, -0.015610296, 0.003359221, -0.032478165, 0.0070943465, 0.042019308, -0.013737608, 0.023552136, -0.031357285, -0.013566742, -0.020476554, 0.0011020842, 0.0125825545, -0.002200751, 0.0093224365, 0.0010790173, 0.0054369485, 0.0038103065, -0.015938358, 0.017988747, 0.0074634166, 0.031220593, 0.008454438, -0.014571432, -0.00481158, 0.020613246, 0.008174218, -0.0044425097, 0.019601721, -0.02813134, -0.004175959, -0.012548381, 0.007579605, 0.014639779, 0.017961409, 0.00023686266, -0.015186549, 0.012110965, 0.010306623, 0.0021580346, -0.0044527617, -0.03961352, 0.007914502, -0.021242032, -0.014229701, -0.02979899, -0.022896012, -0.0029628123, -0.017852055, 0.007716298, -0.0058162706, -0.0067765363, 0.0011251511, -0.012541547, 0.011229298, 0.0010286119, -0.005754759, 0.015746988, 0.009698341, -0.009629995, -0.013115656, 0.015145541, -0.031958733, -0.013580411, 0.015036187, 0.006503151, -0.01304731, 0.0009935844, 0.00226568, 0.0076274476, 0.011741895, -0.004107613, 0.011577864, -0.0042477227, 0.010580008, 0.021542756, 0.0056727435, 0.020804616, -0.011085771, 0.00046518203, 0.02591692, -0.016963553, -0.022472266, 0.052025206, 0.02979899, 0.015610296, -0.02922488, 0.002005964, 0.05735622, -0.006544159, -0.021255702, -0.00035091556, 0.012452696, 0.020640584, 0.0043434077, 0.025411157, -0.008078533, -0.034528553, 0.006793623, -0.014571432, 0.045764685, -0.009178909, -0.0060896557, 0.03365372, -0.032505505, -0.019383011, -0.11274406, 0.008556957, -0.015829004, 0.022417588, -0.015118203, 0.002795364, 0.004336573, 0.010497993, -0.0021819558, 0.020380868, 0.009288263, -0.021050662, -0.013320695, -0.00813321, 0.0011157534, 0.016143396, -0.006509986, -0.010696197, -0.011201959, 0.030947207, 0.01027245, -0.008905523, 0.016498799, -0.00776414, 0.014858487, 0.0008389509, -0.02091397, 0.020667924, -0.003697535, 0.014557763, 0.02422193, -0.024659347, 0.011427502, -0.011345487, -0.01663549, -0.011174621, -0.0012114382, -0.0058026016, 0.021009654, -0.02331976, 0.012240823, 0.012172477, 0.008290407, 3.914561e-06, 0.02165211, -0.03493863, -0.0129857985, 0.018016087, 0.0001933987, -0.024495317, -0.025356479, -0.017961409, -0.03961352, -0.015993036, 0.026955783, -0.024331285, -0.016717507, 0.021050662, -0.012596224, -0.02627232, -0.0045108562, 0.0016403113, -0.035430726, 0.008556957, 0.001449796, -0.005765011, -0.030291082, -0.027816946, 0.023962215, -0.04267543, 0.02053123, -0.00079110847, -0.014120347, 0.0071831965, -0.024440639, -0.0072447085, -0.02070893, -0.010026403, 0.016430452, 0.0022246724, 0.006028144, -0.01962906, -0.011242967, -0.03165801, 0.017523993, 0.019492367, -0.004186211, 0.022472266, 0.00076761446, -0.01591102, -0.0037658815, 0.028596094, -0.008386091, -0.017168593, -0.010764543, 0.023743507, -0.022513272, -0.021337716, 0.022362912, 0.020230506, -0.020654254, -0.013682931, -0.06998662, 0.01663549, 0.00730622, -0.033079613, -0.007162693, -0.011584699, 0.01589735, 0.0050439574, -0.00044211515, 0.023811853, -0.020599576, 0.040050935, -0.029306896, -0.013710269, -0.018303141, -0.0072447085, 0.014298047, 0.002844915, 0.026203973, 0.020599576, -0.012240823, 0.0020948143, 0.0060144747, 0.022267226, -0.0067321113, 0.03573145, -0.016498799, 0.038027883, -0.034173153, -0.013894804, 0.011448006, -0.020271514, -0.009547979, 0.01480381, 0.0027236002, -0.007053339, -0.01610239, 0.016047712, 0.01515921, 0.013081483, -0.03050979, -0.021925494, -0.0008876476, -0.031220593, 0.0020743103, 0.009930718, -0.01341638, 0.0050644614, 0.0003829529, 0.006513403, 0.028240694, 0.003408772, -0.0070260004, -0.029525604, 0.008017021, -0.0050781304, -0.0016351853, -0.0028773795, 0.007436078, -0.012138304, 0.00867998, 0.011905926, 0.03362638, 0.021529086, -0.022403918, -0.01999813, -0.011249802, -0.016485129, 0.0031405129, -0.051724482, -0.0073198895, -0.013682931, 0.0004476683, 0.012480035, 0.005590728, 0.0069849924, -0.003946999, 0.0022981446, 0.013067814, 0.01812544, 0.0034480712, 0.007456582, -0.010142592, 0.016403113, 0.034337185, 0.0021973336, 0.0039709206, 0.009363444, -0.015063526, -0.005214823, -0.019588051, 0.01906862, 0.016676499, 0.006674017, -0.0063288677, 0.032724213, -0.01850818, 0.0034070634, 0.027461546, 0.012363846, -0.0039504166, 0.016034042, 0.0005536051, -0.016225412, -0.010627851, 0.013833292, -0.020763608, -0.0321501, 0.008816673, 0.0061409157, -0.01406567, 0.0068448824, -0.0207226, 0.0049790284, -0.017606009, 0.018767895, -0.0087825, -0.0041349516, -0.0127397515, 0.03387243, 0.010491158, -0.0007791479, 0.007989683, -0.013498396, 0.03570411, 0.009978561, 0.052271254, -0.01889092, 0.021214694, -0.011885422, 0.0048389183, 0.022485934, -0.040816415, -0.016006704, -0.008775665, 0.014120347, 0.012876444, 0.002016216, -0.0347746, 0.08464006, 0.010415977, -0.005529216, 0.013436884, -0.027680254, 0.012158807, 0.020804616, 0.020298854, 0.012124634, -0.030564468, 0.013662427, -0.0068756384, 0.009506971, -0.047760397, -0.027529892, 0.016949885, 0.017633347, -0.0026381675, -0.01887725, -0.0021494913, 0.018043425, 0.010286119, 0.03534871, 0.01700456, -0.026203973, 0.023333428, 0.028541418, -0.0030209068, 0.008488611, -0.05221658, -0.0059666326, 0.0013267726, -0.024987409, -0.015213887, 0.015241226, -0.004425423, -0.016731177, -0.027516223, 0.062058445, 0.010491158, 0.024249269, -0.010709866, -0.019218981, -0.022827666, -0.0062946947, -0.009575318, -0.0270378, 0.0041998806, -0.01664916], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-10-00627.pdf', 'citation': 'Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.8263269),\n",
      " NodeWithScore(node=TextNode(id_='e8ed01eb-9436-448c-9439-403330ffd6f9', embedding=[-0.012636852, -0.027198661, -0.016853753, 0.0041234223, 0.013689347, 0.02095294, -0.00021400423, -0.0013433148, -0.01469337, -0.017684668, 0.0037598964, 0.0051170597, 0.0007309467, 0.007360534, -0.00988098, 0.018903347, 0.01161898, -0.037363537, 0.014887251, -0.02114682, 0.0014800697, 0.025661465, -0.007073175, -0.014568733, 0.004005709, 0.022323951, 0.020842152, -0.03237804, -0.025384493, -0.008821562, 0.0022244323, 0.0016852021, -0.016064383, 0.00709741, -0.023861147, -0.030854693, 0.0213407, -0.016826056, 0.041047268, 0.004601199, 0.024567425, -0.005605223, 0.00135976, 0.0075821113, -0.0005193227, -0.015316557, 0.02064827, -0.012442972, -0.015635075, 0.020980638, 0.020911394, -0.0046288962, 0.0039260797, 0.010261817, -0.0026606633, -0.0054078805, 0.018737163, 0.008115282, -0.010871155, 0.0016168247, -0.009513992, -0.010116406, -0.021631522, -0.010691123, -0.00015157732, -0.004756996, -0.013287737, -0.0013216763, 0.0016566394, 0.011224295, 0.017033784, 0.011528964, -0.009735569, 0.008523816, 0.013308509, -0.036117163, -0.016369052, 0.014998039, 0.011757466, -0.0006491534, 0.008399179, -0.0017111683, -0.004088801, -0.0071666534, 0.023598023, -0.0012602231, 0.0046808287, 0.025855346, -0.009389355, -0.012983068, 0.0347323, 0.019914294, 0.009126231, 0.0026364282, -0.015621227, 0.019318804, 0.0018574442, 0.011930574, 0.0057125497, -0.031713307, -0.022503983, -0.008468422, -0.011902876, -0.012996916, -0.02541219, 0.023999633, -0.0023386832, -0.015219618, 0.013377753, -0.010102558, -0.0140078645, 0.014928796, -0.00094343623, -0.06525463, 0.042155154, 0.005224386, 0.02591074, -0.0075405654, -0.008018343, -0.000111546186, 0.018058581, 0.03315356, -0.0013329283, -0.018944891, 0.0262985, 0.0046358206, -0.023653418, -0.04525724, -0.007066251, 0.00029168624, 0.04337383, 0.023501083, 0.016382901, 0.017006088, -0.02391654, 0.01757388, -0.038886882, -0.019388048, 0.004736223, -0.023182565, 0.0041372706, 0.04340153, -0.01945729, -0.009396278, -0.021354549, 0.00667849, 0.012629928, 0.02513522, 0.010566486, -0.003360018, -0.0045007966, -0.012768414, -0.0006569432, 0.010677274, 0.01152204, -0.008413028, -0.017933944, 0.006273418, -0.030494628, -0.021091426, -0.026893992, 0.014471793, 0.014139426, -0.004919717, 0.036394134, 0.021728462, 0.0048089284, 0.014333307, -0.0033496313, -0.0055982987, -0.027877243, 0.01965117, -0.019277258, -0.00046133166, 0.012235243, 0.018501736, 0.024858246, 0.0031090118, 0.017933944, -0.0144163985, -0.0039883982, 0.00933396, 0.047334533, 0.010628805, -0.022171617, 0.003957239, 0.023390295, -0.027641816, 0.00543904, -0.011002717, 0.02103603, 0.0019336115, 0.004687753, -0.012872279, -0.6385869, -0.028860493, -0.010711896, -0.0069762347, -0.011106581, 0.017116876, -0.012505291, 0.00042649376, 0.003064004, 0.024872094, -0.014748764, 0.009015442, -0.0030605418, -0.025744557, -0.034704603, -0.02550913, 0.010677274, -0.012803036, 0.0158982, 0.001454969, -0.023598023, 0.016050534, -0.0014592968, -0.003112474, 0.0037806693, -0.00039814741, 0.036394134, -0.009680175, 0.0012403157, -0.00047994073, -0.030079171, 0.016105928, 0.0046600555, -0.013218494, 0.04201667, -0.010587259, 0.023611872, 0.008136055, 0.026533928, -0.013287737, -0.008572287, -0.022587076, 0.012934597, 0.01876486, -0.012387577, 0.020094326, 0.029968383, -0.01235988, -0.0060310676, 0.003043231, -0.022407044, 0.022116223, -0.018141674, -0.006491534, 0.015912047, -0.00056433066, 0.0253291, -0.02969141, 0.0025342947, -0.005075514, 0.0066923387, 0.0087246215, 0.003891458, -0.004251522, 0.01509498, 0.014125577, -0.014734916, -0.007069713, 0.028777402, -0.011916725, -0.0036525696, 0.017061481, 0.011459721, -0.0055671395, -0.0072358963, 0.04329074, 0.0013346593, 0.02174231, -0.011418175, 0.013883227, -0.007305139, -0.009354733, -0.011556661, 0.026741657, 0.025979983, 0.009098534, -0.022614773, 0.010864231, -0.004351924, -0.023279505, 0.017740063, -0.00031527216, -0.020052781, -0.027018629, -0.018404797, -0.012643777, -0.016105928, -0.0067373463, 0.017089179, -0.012103681, -0.008821562, -0.023224112, 0.017407697, -0.019706566, -0.0013813984, -0.0073397607, 0.024581274, 0.0058406494, 0.027988032, -0.013315434, 0.0011511653, 0.009181625, -0.02391654, 0.00938243, -0.007983721, -0.03686499, 0.017546182, 0.018806405, -0.007859084, -0.010213346, 0.022227012, 0.0004747475, -0.01668757, -0.012075984, 0.021326853, 0.006702725, 0.017712366, -0.025758406, 0.005764482, -0.016133625, 0.012304486, -0.005979135, 0.03187949, -0.0051586055, 0.014250215, 0.020025084, 0.028777402, 0.002792225, 0.00883541, -0.023778055, -0.026852446, 0.00484355, 0.029442135, 0.011951346, -0.002286751, -0.01895874, -0.0047500716, 0.004961263, 0.00494049, -0.011058112, 0.007374382, 0.02462282, -0.008295314, -0.0022954063, 0.0068723704, -0.020468239, -0.039246947, -0.047500715, 0.0011710727, -0.012719944, 0.014444095, 0.023293354, -0.0024373545, -0.0023023307, 0.017241513, -0.0022071216, 0.013433147, -0.0025983446, -0.026201561, -0.03395678, 0.0125122145, -0.02204698, -0.016673721, 0.02402733, -0.0021932728, 0.03852682, -0.023307202, 0.0122283185, -0.0054355776, -0.0035902509, -0.00992945, -0.018460192, 0.0063080397, 0.009043139, 0.029026676, -0.009860207, 0.008371482, -0.0035019661, 0.028666614, 0.0357294, -0.017255362, 0.0081914505, 0.0019820817, -0.0026000757, -0.0047950796, -0.0087592425, 0.01350239, 0.014679521, 0.008246845, 0.026173864, 0.006266494, 0.0023317588, -0.0013770707, -0.014250215, 0.020274358, -0.039856285, -0.0004412079, -0.030466931, 0.018141674, 0.02442894, 0.014998039, 0.0017276135, -0.017989337, 0.009063913, 0.006242259, -0.0011182749, 0.009673251, 0.015247314, -0.01699224, 0.00036936827, -0.013446996, 0.016798358, 0.01728306, 0.018681768, -0.02024666, -6.12368e-05, -0.006633482, 0.03118706, 0.0012160806, 0.006470761, -0.0010914432, 0.022891745, 0.011563585, 0.019027984, 0.024816701, 0.01449949, 0.027904939, -0.020994486, 0.012969219, 0.00080538297, -0.012373729, 0.029497528, 0.0233349, -9.942866e-05, 0.0018020498, -0.012893052, 0.010047163, 0.020052781, 0.0036006374, 0.011418175, -0.017809307, 0.0026797052, -0.011591283, 0.0069416133, 0.01678451, -0.017837003, -0.018834103, 0.00019420504, 0.02303023, 0.020925242, 0.0033911772, 0.012075984, -0.011487418, -0.010905777, 0.022060828, -0.008177602, -0.024055028, -0.006169554, -0.015178071, 0.014180972, -0.018834103, -0.00140823, 0.0016635637, -0.007852159, -0.0032700018, -0.008343785, 0.0037148884, 0.019928142, 0.0012991723, -0.00040182594, -0.040770292, -0.026880143, -0.0017345378, 0.023694964, 0.009250868, 0.01815552, -0.008856183, -0.005193227, -0.011092733, 0.018931044, 0.021022182, 0.03525855, 0.009673251, 0.015551983, -0.017726215, 0.014457944, 0.027821848, -0.020135872, 0.009520916, 0.014831856, 0.008544589, 0.0041061114, -0.016313657, 0.0034638825, 0.035590917, -0.022282407, -0.0018193605, -0.007346685, -0.00883541, -0.030660812, 0.011348932, -0.005580988, 0.00424806, 0.008530741, -0.017061481, -0.001992468, -0.016119776, 0.01321157, 0.024650518, -0.014174048, 0.007782916, -0.038055968, -0.017504636, -0.004376159, 0.048996367, 0.032350343, -0.0067823543, 0.016798358, 0.04262601, 0.0054494264, -0.022019282, -0.0058441116, 0.010677274, -0.023473386, -0.0053663347, -0.010275665, 0.042404428, -0.0015302709, -0.0059618247, 0.003084777, 0.006384207, -0.011009641, -0.0079421755, 0.013571633, 0.009410127, -0.0019820817, 0.014956494, 0.02422121, 0.010808837, -0.019484987, 0.04290298, 0.028777402, 0.008073737, -0.020675968, -0.011501267, -0.0011156782, 0.0022780956, 0.018404797, 0.01176439, -0.0056086853, -0.0010170069, 0.015261163, 0.0050720517, 0.028860493, 0.013246191, 0.016922995, 0.0014298685, -0.003527932, -0.010310287, 0.01007486, -0.0042826813, 0.018044733, -0.014804159, -0.011632828, 0.039939377, -0.003477731, 0.02600768, -0.005965287, 0.010109482, 0.019221865, -0.010324135, 0.0011641483, -0.02591074, -0.027364844, -0.0033981015, -0.020412844, 0.0016999163, -0.011992892, -0.03326435, -0.000960747, -0.015219618, -0.025190612, -0.02114682, -0.0018003187, -0.0034846554, -0.025204461, -0.023182565, 0.011556661, 0.012816885, -0.01057341, 0.021229912, 0.010746518, 0.006702725, 0.013481617, -0.01459643, -0.013073083, -0.00092785654, -0.036781896, -0.007249745, 0.014651825, 0.0059860596, -0.0028406952, -0.026035378, 0.01569047, 0.011459721, 0.019512685, 0.016022837, -0.00048123903, 0.0048954817, -0.010601107, 0.0051343706, -0.0028095357, -0.023584174, -0.016742963, 0.0111689, -0.03127015, 0.0016384631, -0.0027610657, 0.000172783, 0.0016090348, 0.00042519547, 0.009160852, -0.028237306, -0.0043138405, 0.0034136812, -0.017089179, 0.008579211, 0.027808, 0.045645002, 0.034510724, 0.029996078, -0.0154827405, 0.019886598, -0.0020720975, -0.002838964, 0.008413028, 0.0026018068, 0.008980821, -0.01797549, 0.024581274, -0.006861984, -0.042542916, -0.025841497, 0.006245721, -0.015579681, 0.024872094, 0.007505944, 0.0048850956, -0.03833294, -0.021673067, 0.015247314, 0.026201561, -0.012754565, -0.033596717, -0.010961171, -0.012235243, 0.01577356, 0.007298215, -0.0100402385, -0.03315356, 0.009299339, 0.022157768, 0.009659402, 0.0144163985, 0.010234119, -0.016812207, -0.0060933866, 0.0049301037, -0.032599617, -0.017712366, -0.009541689, 0.012505291, 0.031020876, 0.028348096, 0.026367744, -0.024442788, -0.0069658486, 0.036698803, 0.014444095, 0.0041961274, -0.015745863, -0.0119444225, -0.029331347, 0.006491534, 0.024941338, -0.01152204, -0.0044938726, -0.02145149, 0.010580335, 0.021409944, -0.005328251, 0.00019323132, -0.013696271, -0.0288051, -0.021285307, -0.0054528886, -0.013897075, 0.01295537, -0.02116067, -0.041684303, 0.013398525, -0.016673721, -0.00014962985, -0.006868908, 0.011099657, -0.014430246, 0.030273052, -0.0069935457, 0.013537012, -0.026575474, 0.009202398, -0.035230853, 0.0057506333, 0.023362597, 0.015496589, 0.0065053822, 0.012740717, -0.0039918604, 0.00883541, 0.0046565933, -0.012387577, -0.015053434, -0.018127825, -0.010885004, -0.022850199, -0.03187949, 0.010704972, -0.040105563, 0.0010014273, -0.0053628725, -0.023861147, 0.0020946015, -0.016521387, -0.014956494, 0.011335083, 0.023958087, 0.048525512, -0.01936035, 0.01414635, 0.04550652, 0.009015442, -0.030023776, -0.007073175, -0.008046039, -0.0023629183, 0.03356902, 0.0268109, -0.015912047, -0.029829895, 0.013640876, 0.01235988, -0.0152334655, -0.035618614, 0.030356143, -0.00022439068, -0.004916255, -0.01037953, -0.008655379, -0.029165164, 0.03210107, 0.007526717, 0.028777402, -0.010400303, -0.006152243, -0.0050443546, 0.0017232859, 0.0118197845, 0.021271458, 0.00814298, -0.041961275, 0.0028857032, -0.022864047, -0.015884351, -0.0046185097, -0.008419952, 0.022573227, 0.00048210457, -0.013897075, 0.02224086, -0.016147474, -0.0051412946, 0.0012108875, -0.009423976, 0.03487079, -0.024553576, 0.012138302, -0.0037564342, 0.01895874, 0.017837003, -0.010137179, -0.006242259, -0.0056536933, 0.0032838506, 0.002721251, 0.041933578, 0.004192665, 0.016535236, -0.019554231, -0.018792557, -0.0018193605, -0.013426223, -0.01678451, -0.00709741, -0.04420475, 0.0074297767, 0.0050685895, 0.010400303, 0.01285843, 0.007360534, 0.0005885657, -0.008260693, 0.014250215, -0.039551616, 0.0054944344, -0.0025239084, -0.019692717, -0.030716207, -0.0066854144, -0.007464398, -0.012442972, 0.013093856, -0.0298022, -0.022282407, -0.01974811, -0.01707533, 0.030826995, -0.0065780873, -0.029552924, 0.019097228, 0.004511183, 0.013516239, -0.010531864, -0.020495936, 0.041656606, 0.0023646494, 0.001144241, 0.00640498, -0.0040195575, 0.021382246, 0.0051274463, -0.020634422, -0.0070108566, -0.022587076, -0.04221055, 0.0058129523, 0.024442788, 0.0053490237, 0.0004178384, -0.0031020876, -0.023750357, -0.0092231715, -0.0023663805, 0.0042376732, -0.019041833, -0.010012542, 0.0035798645, 0.011937498, -0.007713673, -0.019291107, -0.021229912, 0.0022452052, -0.024581274, -0.022407044, -0.004480024, -0.0017094372, 0.02124376, -0.026160015, -0.015551983, -0.03653262, -0.009423976, -0.038803793, 0.0048677847, -0.0022330876, -0.0006733884, 0.018487887, -0.012678398, 0.006498458, 0.0020876774, -0.013904, 0.005421729, 0.0013095587, -0.010587259, -0.0027368306, 0.018598678, -0.0023265658, -0.0044696373, -0.032045674, 0.007575187, 0.028126517, 0.025896892, 0.0084338, 0.019014135, -0.027780302, 0.008406104, -0.012747642, 0.01627211, -0.0030051474, 0.015136526, 0.019609625, -0.03068851, -0.03625565, 0.026173864, -0.00978404, -0.009264717, -0.018972589, 0.0053040157, 0.0005115329, 0.04306916, -0.0051863026, -0.028071122, 0.016895298, -0.008828486, 0.026520079, -0.00088631077, -0.0021811554, 0.0073120636, 0.002742024, -0.033707507, -0.007360534, 0.020385146, -0.012242167, -0.011992892, 0.014208669, 0.0049127927, 0.0033479002, -0.029829895, 0.01171592, -4.173515e-05, 0.01206906, -0.00039858016, -0.011466646, -0.005179378, -0.00012517841, 0.025024429, 0.0045458046, -0.045534216, 0.011757466, -0.011688223, -0.0025793028, 0.003704502, 0.00888388, -0.015150374, 0.01355086, 0.019900447, -0.009763267, -0.01400094, -0.018030884, 0.016950693, -0.002612193, 0.008565363, 0.17859161, -0.0034725377, 0.017296908, 0.03287659, -0.0057367845, 0.0014385239, -0.0069900835, -0.006449988, 0.0023213725, 0.0037356613, 0.024484335, 0.01926341, 0.0012870547, 0.0035867887, 0.006609247, 0.0038152908, -0.03395678, -0.010026391, -0.01469337, -0.020606725, -0.006294191, -0.01519192, 0.0004191367, -0.00026009412, 0.020689817, -0.013301586, -0.052403122, 0.015815107, 0.02055133, -0.011909801, -0.025259856, -0.01321157, 0.0009520916, 0.009631705, -0.012532988, 0.006543466, -0.006387669, 0.020066628, 0.0090085175, 0.00011306088, -0.007568263, -0.036920384, 0.0022348186, -0.016590629, -0.008953123, 0.007782916, -0.02016357, 0.0016445218, -0.008364557, -0.011065036, -0.023639569, 0.01221447, -0.008295314, 0.027988032, 2.2503984e-05, -0.045838885, 0.008676151, 0.020218965, -0.0056433068, 0.0029791812, -0.0027264443, 0.014111728, -0.019277258, 0.0129761435, -0.023417993, 0.022296255, -0.011999817, -0.0042272867, 0.014374852, -0.0057575577, -0.004774307, -0.02153458, 0.0016090348, -0.013523163, 0.0020201653, -0.01509498, 0.027531028, 0.018640222, 0.03068851, 0.033292048, 0.0036664184, 0.010365681, -0.023750357, -0.010234119, -0.00048080625, -0.02195004, 0.031464033, -0.0070766374, -0.009216247, -0.013419298, -0.014347155, -0.004376159, -0.012844581, -0.014845705, 0.013910924, 0.0024183127, -0.013024613, 0.005947976, 0.011840558, -0.0013035, -0.025065975, 0.052957065, 0.009243944, -0.00918855, -0.029165164, 0.0021586514, -0.009147004, 0.0055359798, 0.027254056, -0.024055028, 0.0034448407, -0.017726215, -0.0015579681, -0.02679705, 0.023501083, 0.028611219, 0.012422199, -0.029746804, 0.010247968, -0.002766259, -0.012346032, -0.01335698, -0.0002854111, 0.025010582, -0.017352302, -0.009250868, -0.017823156, 0.011840558, -0.0011788625, -0.014250215, -0.0011606863, -0.014748764, 0.011189673, -0.003545243, 0.0041268845, 0.00038776096, -0.008994669, -0.015011888, -0.01137663, -0.0127614895, -0.011051187, 0.0019838128, 0.01726921, 0.004348462, -0.025107522, -0.0047777686, 0.04029944, 0.006979697, -0.005168992, -0.022697864, -0.012172924, -0.026990931, -0.027226359, 0.0028908963, 0.012498367, -0.00086770166, -0.024761306, -0.024941338, -0.007769068, -0.016964542, -0.032045674, -0.0063634343, 0.025370644, -0.005092825, -0.00799757, -0.011411251, -0.17837004, 0.016147474, 0.008039115, -0.0024789004, 0.026824748, 0.021091426, 0.020772908, -0.0031211295, -0.0071251076, 0.007921402, 0.029857593, 0.0036906532, -0.013412374, -0.030439235, 0.021396095, 0.013717043, 0.0025360258, 0.016798358, 0.034012176, 0.0042896057, 0.014374852, -0.0253291, 0.003683729, -0.017116876, 0.00759596, 0.03642183, -0.01995584, 0.01906953, -0.013841681, -0.022365497, -0.02154843, 0.008828486, 0.019554231, -0.016646024, 0.019138772, 0.0050201192, 0.008032192, 0.024345849, -0.0068966052, 0.009860207, 0.019595776, 0.013959394, -0.0027472172, -0.005677928, -0.01598129, -0.0009745956, 0.01826631, -0.014174048, 0.008267618, -0.032488827, -0.00907776, -0.011598207, 0.020800605, 0.012138302, 0.032433435, 0.008994669, -0.012740717, 0.009950223, 0.010580335, -0.016853753, -0.016673721, -0.018390948, 0.011736693, -0.02165922, -0.0019249561, -0.041573513, -0.025398342, -0.012027514, -0.022199314, 0.0031401713, -0.014998039, 0.0032353804, 0.009867132, 0.009237019, -0.007852159, 0.013772438, -0.017089179, 0.01699224, 0.0003366942, 0.0041303467, -0.016258262, 0.023958087, -0.017310757, -0.0072843665, 0.025841497, 0.012782263, -0.012159076, -0.0018816793, 0.0025706473, 0.000112952686, 0.017823156, -0.0068585216, 0.01082961, -0.008703848, 0.021105275, 0.010448772, -0.017158423, 0.0035262012, -0.016632175, -0.008516892, 0.014430246, -0.007727522, -0.0080668125, 0.049605705, 0.044841785, -0.0042272867, -0.008780016, -0.0035833267, 0.062817276, -0.011999817, -0.025010582, -0.013876302, 0.009472446, 0.012796111, -0.0036768047, 0.01599514, 0.0020478626, -0.033818293, 0.015247314, -0.018100128, 0.059327424, 0.0073120636, -0.0077621434, 0.028361944, -0.026589323, -0.02491364, -0.09522301, 0.0049543385, -0.015704319, 0.028348096, -0.03295968, 0.0008728949, -0.021202214, 0.020869847, -0.003228456, 0.014305609, 0.0021430717, -0.02679705, -0.00092352886, 0.007201275, -0.0065711634, 0.013966318, -0.0168676, -0.0137932105, -0.022725562, 0.023501083, 0.0031055498, -0.012048286, -0.009084685, -0.007415928, 0.023985785, -0.0030051474, -0.023210263, 0.019498836, 0.0158982, 0.0071666534, 0.009804812, -0.015053434, 0.015053434, -0.01181286, -0.030522326, -0.012373729, -0.013280813, -0.024650518, 0.015635075, -0.046752892, 0.008378406, 0.0158982, 0.0079421755, -0.020218965, 0.0013805329, -0.015745863, -0.009216247, 0.008032192, 0.0016869332, -0.014721068, -0.02631235, -0.007042016, -0.027489481, -0.0073536094, 0.035701703, -0.036394134, -0.0007123376, 0.016673721, -0.017449243, -0.02771106, -0.0092577925, 0.007069713, -0.029857593, 0.021825401, -0.0062491833, -0.010871155, -0.004369235, -0.034649212, 0.018640222, -0.025952287, 0.010995792, -0.0074297767, -0.01599514, -0.0016895299, -0.01285843, 0.0066438685, -0.04390008, -0.020579027, 0.022587076, -0.0016774123, -0.0010100827, -0.017753912, -0.0013796673, 0.009112382, 0.026076924, 0.009292414, -0.0059722112, 0.021631522, 0.012775338, -0.021423792, 0.0016488496, 0.033181258, -0.00918855, -0.01126584, -0.013578557, 0.038886882, -0.024955187, -0.008253769, 0.014804159, 0.01749079, -0.03442763, -0.0068896813, -0.080100335, 0.03761281, 0.0070212428, -0.020205116, -0.0028233845, -0.01866792, 0.009063913, -0.00053533516, 0.0097563425, 0.043955475, -0.038083665, 0.039883982, -0.022268558, -0.010393378, -0.028666614, -0.0047189123, 0.021770008, -0.005719474, 0.02244859, 0.014236366, -0.005470199, 0.010857306, 0.03287659, 0.03822215, -0.010407227, 0.010469546, 0.002470245, 0.027475633, -0.029137466, -0.0073813065, -2.8184075e-05, -0.02899898, -0.014374852, 0.030522326, 0.018847952, 0.0028614681, -0.021077577, 0.0050997487, 0.0104141515, 0.017712366, -0.018183218, -0.03883149, 0.0020876774, -0.010649578, 0.0129761435, 0.020385146, -0.0078106136, -0.005335175, 0.0035971752, 0.011411251, 0.035895586, 0.0105041675, -0.017837003, -0.05071359, -0.0019318805, 0.009133155, 0.0011728038, 0.008994669, 0.0055498285, -0.0012351224, 0.022019282, 0.031353243, 0.019706566, 0.027143266, -0.014014789, -0.015122677, -0.0012039631, 0.0063011153, 0.0006063093, -0.05849651, -0.011909801, -0.00543904, 0.0011901145, 0.0137585895, -0.0041822786, 0.0031938346, -0.014679521, 0.013460845, -5.301419e-06, 0.0053628725, 0.028472733, -0.002240012, -0.026727809, -0.013156175, 0.028971283, 0.009451673, 0.020205116, 0.008696924, -0.017684668, 0.011542813, -0.012256016, 0.02679705, 0.0014013058, 0.016798358, -0.0013545667, 0.02830655, -0.01569047, -0.0010282589, 0.026520079, 0.02404118, -0.0020236275, 0.017726215, 0.00603453, -0.015468892, -0.003228456, -0.010033314, -0.0072220475, -0.012394502, 0.016244413, 0.012498367, 0.00019701805, 0.010822685, -0.01007486, 0.010781139, -0.008849259, 0.010961171, 0.0017431933, -0.018834103, -0.030633114, 0.058219537, 0.026104622, 0.01797549, 0.015551983, 0.0034621514, 0.015316557, 0.024802852, 0.018238613, -0.008627681, 0.026561625, -0.013994016, 0.0020703664, 0.006415366, -0.017407697, 0.011646677, -0.014319458, -0.004396932, 0.001704244, -0.0042896057, -0.01876486, 0.08708003, -0.0062318724, -0.01747694, 0.024844399, -0.041351937, -0.0023473387, 0.03860991, 0.013350056, 0.0017449242, -0.025966136, -0.006882757, -0.0048677847, 0.00080365187, -0.042044364, -0.03185179, 0.024110422, 1.5214532e-05, -0.004601199, -0.023417993, -0.0096871, 0.0036214103, 0.009555537, 0.021977738, 0.00997792, -0.01404941, 0.0013813984, 0.012505291, -0.007866008, 0.017795458, -0.057942566, 0.017504636, -0.009361657, -0.020925242, -0.021783857, 0.015441195, -0.016119776, -0.02124376, -0.006675028, 0.031076271, -0.007845235, 0.01598129, -0.005352486, -0.025578374, -0.010822685, 0.016064383, -0.0022486672, -0.030411538, -0.0011849212, -0.027586421], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-10-01533.pdf', 'citation': 'Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.8256848999999999),\n",
      " NodeWithScore(node=TextNode(id_='390ec987-3df0-4711-ad7e-dc6e7295dff6', embedding=[-0.024463622, -0.007585244, -0.024546595, -0.0030406662, 0.020411704, 0.03457267, -0.0035298695, -0.0041625495, -0.0075022695, -0.008539449, 0.021144645, 0.022541381, -8.729599e-05, -0.004093404, -0.007827252, -0.0013103042, 0.01854478, -0.019346867, 0.00096198457, -0.024491278, 0.010786673, 0.009887784, -0.01142281, -0.010710613, -0.0021193046, 0.021255277, 0.021628663, -0.038610756, -0.015751308, 0.0041487203, -0.0003997465, 0.0008837639, -0.005825495, 0.02728475, -0.021601005, -0.03545773, 0.010924963, -0.01972025, 0.042068023, 0.005230845, 0.030534582, 0.007896398, -0.001773578, 0.011595674, -0.005137499, -0.01840649, 0.013013153, -0.0066275806, -0.020010661, 0.0043872716, 0.0131307, -0.0017977789, -0.004349242, 0.027990034, 0.009735663, -0.008221381, 0.014810932, 0.013600889, -0.0076613035, -0.0142854275, -0.010655297, -0.012591367, -0.013407282, -0.0091755865, 0.004152178, 0.004764114, 0.0025583776, 0.019056456, 0.0035817286, 0.0055212555, 0.017715037, 0.0043976437, 0.015198146, 0.0060156444, 0.024601912, -0.03341103, -0.028321931, 0.007930971, -0.0051063835, 0.012570622, 0.0055765714, 0.006333713, -0.026150767, 0.009846296, 0.02893041, -0.005797837, 0.013759922, 0.021020183, -0.012217982, -0.021725466, 0.021849927, 0.021891415, 0.043450933, 0.0137806665, -0.017037412, 0.031115403, -0.017479941, 0.009396852, -0.0006646596, -0.009161757, -5.1696945e-05, -0.001648252, -0.013773751, -0.02060531, -0.01942984, 0.008650082, 0.0016335587, -0.003688904, 0.012466905, -0.013925872, -0.017853327, 0.01839266, 0.014575837, -0.05326957, 0.030147368, 0.013123785, 0.020674456, -0.021559518, -0.019374523, 0.006634495, 0.008394244, 0.04732307, 0.0047122547, -0.009210159, 0.032249387, 0.0010665669, -0.0113813225, -0.050946288, -0.00085005554, -0.013068469, 0.03233236, 0.041459545, 0.020342559, 0.001768392, -0.016083205, 0.021324422, -0.04226163, -0.02862617, -0.015488557, -0.021255277, 0.008090004, 0.04292543, -0.015944915, 0.014202453, -0.021227619, 0.006409773, 0.024491278, 0.012923264, 0.0037753356, 0.014271598, 0.003419237, -0.018337345, -0.0025359052, 0.0009602559, -0.008470304, -0.0043665282, -0.010517006, 0.008186808, -0.045663584, -0.039246894, -0.03341103, 0.0029231193, 0.01662254, -0.0064547174, 0.022956252, 0.014479035, 0.017258678, 0.02136591, -0.0062369094, -0.0101851085, -0.0051617, 0.011637161, -0.011526528, -0.004570507, 0.0074262093, 0.028985726, 0.013504085, -0.004622366, 0.010039903, -0.015557702, 0.0031478417, 0.010897306, 0.031170718, 0.021849927, -0.010973365, 0.016525736, 0.0258327, -0.031640906, 0.01706507, -0.03260894, 0.017977789, 0.015861942, 0.020107465, -0.0018842105, -0.62706524, -0.034627985, -0.010717528, 0.005683747, -0.00852562, 0.011249946, 0.0006551521, -0.013186016, -0.010973365, 0.02211268, -0.012584452, -0.0035678996, -0.004349242, -0.032221727, -0.027270922, -0.028681487, 0.005732149, -0.0104340315, 0.0008046789, -0.0013025253, -0.007799594, 0.014084905, -0.010883477, -0.010198938, -0.0007644882, 0.009459082, 0.04198505, 0.006354457, 0.0134003665, 0.02194673, -0.03283021, 0.030230343, 0.0068937903, -0.018738387, 0.053186595, -0.006098619, 0.0025635634, 0.008380415, 0.028653828, -0.017452285, -0.012211067, -0.01764589, 0.0014157508, 0.019125601, -0.01676083, 0.012743486, 0.01676083, 0.0032550169, -0.010523921, -0.0021072042, -0.0064478028, 0.028460221, -0.022486065, 0.004069203, 0.02106167, 0.0118860835, 0.02967718, -0.0362045, 0.017562917, 0.015848111, 0.0058704396, -0.0014295799, 0.007882569, -0.029732496, -0.005853153, 0.016359787, -0.022527551, 0.007972457, 0.015571531, -0.014741787, 0.008816031, 0.007419295, 0.0081660645, -0.012508392, 0.009763322, 0.040021323, -0.007993202, 0.015460898, -0.0045255623, -0.0007964679, 0.0034002222, 0.0051236697, -0.023384955, 0.011146229, 0.031945147, 0.016650198, -0.017853327, 0.0041902075, -0.0021728922, -0.022209482, 0.023855142, 0.015032196, -0.02255521, -0.03504286, -0.012356272, 0.00489549, -0.002537634, -0.013891298, 0.009141014, 0.0012584452, -0.021863757, -0.0051478706, 0.026510322, -0.006433974, 0.013981188, -0.008318184, 0.003094254, -0.0023993433, 0.02032873, -0.011533443, -0.015364095, -0.01795013, -0.017894814, 0.0052792467, 0.010074476, -0.035430074, 0.022085022, 0.026178425, 0.017673548, -0.025348682, 0.011201545, -0.012874862, -0.014133307, -0.003522955, 0.014271598, 0.010593066, 0.018987311, -0.022223312, -0.011395152, -0.014202453, 0.0024823176, 0.0027848287, 0.023924287, -0.014382231, 0.026219914, 0.029262308, 0.025556117, 0.00986704, 0.00578055, -0.017701207, -0.028404906, -0.0087330565, 0.013331221, 0.010579237, 0.00097408495, -0.016193839, 0.011090913, 0.011464298, 0.017327823, 0.008117663, 0.0089888945, 0.028985726, 0.010579237, -0.0053656786, 0.023357296, -0.014382231, -0.023509415, -0.03767038, -0.0036958184, -0.009479826, 0.009383023, 0.006399401, 0.0021694351, -0.006254196, 0.018295856, -0.0033691067, 0.020301072, 0.009106441, 0.0015160115, -0.03247065, -0.0035575277, -0.021849927, -0.02595716, 0.013953529, -0.00075325207, 0.025791211, -0.013946615, 0.028653828, -0.0051617, -0.005580029, 0.006420145, -0.021158474, 0.009376108, 0.0230254, 0.016290642, 0.0051340414, 0.011775452, -0.016525736, 0.023246663, 0.0358726, -0.020964867, 0.008124578, -0.0038133655, -0.003438252, -0.005825495, 0.002560106, 0.023011569, 0.010793587, 0.008338928, 0.019830883, 0.013483341, 0.019637277, 0.011540357, -0.0065999227, 0.012183409, -0.049922936, 0.0015626847, -0.028819777, 0.03816823, 0.02895807, 0.0012869677, 0.002769271, -0.030202685, 0.0042558955, 0.0046016225, -0.0055005117, 0.020010661, -0.0038825108, -0.00083104055, 0.0018340801, -0.027381554, 0.016373616, 0.02493381, 0.01883519, -0.0314473, 0.007834167, -0.0028660744, 0.041238282, -0.0030959826, 0.0021694351, -0.0050303233, 0.020107465, 0.030175027, 0.00047494206, 0.0255008, 0.0113813225, 0.026842222, -0.006019102, 0.009770237, -0.004684597, -0.0033345341, 0.009009638, 0.018461805, 0.00042848501, 0.006824645, 0.009147928, 0.016180009, 0.035568364, -0.010710613, -0.0027502559, -0.021020183, -0.002286982, -0.020411704, -0.007599073, 0.019070284, -0.023785997, -0.011166972, 0.004017344, 0.029400598, 0.019540474, -0.0030354804, 0.0066759824, -0.021559518, -0.00452902, 0.0258327, -0.011740879, -0.020881893, -0.016968267, -0.021435056, -0.0014710671, -0.03260894, -0.007018252, 0.022195654, 0.0020691743, 0.014520521, -0.005472854, -0.017590575, -0.009320792, 0.00040601278, 0.004131434, -0.035264123, -0.03454501, 0.018738387, 0.019180918, 0.0077373637, 0.0007225688, -0.015460898, 0.00014272031, -0.015322607, 0.016221497, 0.028003862, 0.007993202, -0.00022623492, 0.026441177, -0.0075645, 0.0033051472, 0.02551463, -0.01676083, 0.017410796, 0.008242125, -0.0023025398, 0.0044944473, -0.016968267, -0.0026724674, 0.02729858, -0.011111656, 0.009251647, -0.0073363204, 0.00024633028, -0.03042395, 0.02153186, 0.0015713278, 0.0035246836, -0.0041106907, -0.01705124, 0.0021020183, -0.023965774, 0.024671057, 0.013483341, -0.021020183, -0.0075022695, -0.035430074, -0.010793587, -0.010392545, 0.041929733, 0.024256185, 0.009569715, 0.021587174, 0.03816823, 0.008954322, -0.006924906, -0.011450469, 0.01543324, -0.017576745, -0.01291635, -0.0016499807, 0.015391753, -0.0091341, 0.009950015, -0.003211801, -0.0035886432, -0.0145343505, -0.012985495, 0.0034140511, 0.0050856397, 0.0037857073, 0.04450194, 0.010862733, 0.017383138, -0.021725466, 0.03147496, 0.01929155, 0.005839324, -0.023647707, -0.010074476, -0.017839497, 0.0040208013, 0.020619141, 0.024117894, -0.006126277, 0.0012221439, 0.01009522, 0.019319208, 0.027257092, 0.0203149, 0.020494679, 0.006693269, 0.0035056688, -0.012743486, 0.012625939, -0.0119483145, 0.015709821, -0.011402067, -0.028709145, 0.040325563, -0.0023785997, 0.008795287, -0.032885525, 0.02816981, 0.012411589, -0.03100477, -0.011547272, -0.0063233413, -0.025334852, 0.002305997, -0.019692592, 0.009721835, -0.0079448, -0.022610527, 0.017120386, -0.017023582, -0.014071077, -0.029124018, -0.0013120328, -0.01810225, -0.020231927, -0.030036736, 0.012667426, 0.0025929501, 0.010593066, 0.016581053, -0.006357914, -0.0019464414, 0.02536251, -0.017687378, 0.00092827616, -0.0030389377, -0.04881661, 0.02227863, 0.0087330565, 0.0047468278, -0.004273182, -0.007986287, 0.0023526703, 0.011277605, 0.021421226, 0.02314986, 0.004919691, 0.006126277, -0.01913943, 0.000870367, 0.017410796, -0.00589464, -0.012660512, 0.014949222, -0.026289059, -0.003958571, -0.016857633, -0.012003631, -0.006769329, -0.005815123, 0.019374523, -0.020356389, 0.00076362386, 0.00082282955, -0.014741787, 0.004041545, 0.009064954, 0.03830652, 0.022154167, 0.0073224916, -0.011491955, 0.0074123805, -0.0073501496, 0.01195523, 0.006859218, -0.0045808787, 0.0027329696, -0.013587059, 0.03753209, -0.013552487, -0.02998142, -0.023896629, 0.0029127474, -0.026012477, 0.03473862, 0.02386897, -0.0035540704, -0.045718897, -0.025155075, 0.019457499, 0.018793704, -0.001961999, -0.031668566, -0.0061331918, -0.005918841, 0.013109957, 0.009797894, -0.0017312265, -0.03872139, 0.0040208013, 0.028404906, -0.010523921, -0.0033708352, 0.008504877, -0.010053732, -0.025888015, -0.00045419845, -0.057750188, -0.014714128, -0.0065999227, 0.014700299, 0.018779874, 0.014866248, 0.023620049, -0.010738271, -0.0086708255, 0.027367726, 0.012577537, 0.010399459, -0.011111656, -0.00028371197, -0.024919981, 0.015502385, 0.024449792, -0.007516098, -0.00067676004, -0.015972573, 0.008892091, 0.015958745, -0.004390729, -0.014797103, -0.006551521, -0.030617556, -0.027091144, -0.014921565, -0.016802318, -0.0076820473, -0.041653153, -0.031032428, 0.022320116, -0.010931878, 0.011810024, -0.001004336, 0.019761737, -0.0034278801, 0.023924287, 0.014271598, 0.007294833, -0.020702114, 0.0054348237, -0.028847435, 0.0050960113, 0.023371125, 0.0031616706, 0.00815915, 0.009583544, -0.028570855, 0.010226596, 0.010178194, -0.018033104, -0.011173887, -0.024684886, -0.012238725, -0.014575837, -0.020301072, 0.011471212, -0.03794696, 0.0065169483, -0.0043457844, -0.023246663, -0.0053691356, -0.017023582, -0.02078509, 0.025735896, 0.013918957, 0.03930221, -0.03460033, 0.032083437, 0.03780867, 0.0023457557, -0.015668334, -0.009645775, -0.020702114, -0.0050856397, 0.042676505, 0.03902563, -0.022015877, -0.023841314, 0.0025825785, 0.0222648, -0.009666518, -0.028059179, 0.039357528, -0.015087513, -0.006551521, -0.01291635, -0.013448768, -0.035430074, 0.017562917, -0.03130901, 0.02565292, -0.012418503, -0.0014606953, -0.011485041, 0.0009991502, 0.020978697, 0.018060762, 0.020273414, -0.036702346, 0.014271598, -0.019872371, 0.00075022696, 0.011906828, -0.023924287, 0.031115403, 0.015377924, 0.0031011684, 0.009887784, -0.0075299274, -0.009286219, 0.0006841067, 0.00326366, 0.040242586, -0.012045118, 0.0021936358, -0.0123286145, 0.015709821, 0.0186969, -0.007834167, -0.008567108, -0.026551811, -0.0017130758, -0.00079257844, 0.030340975, 0.013974274, -0.003702733, -0.024878493, -0.018710729, -0.009369194, 0.0027709994, -0.016498078, -0.010510092, -0.02639969, -0.0018046934, 0.006696726, -0.0026482665, 0.0018997682, 0.004923148, 0.00021705154, -0.015682163, 0.010468604, -0.036425766, 0.010945708, 9.0051e-05, -0.029068701, -0.024698716, 0.00071176485, -0.007523013, -0.009092612, 0.0030026364, -0.026593298, -0.0038099082, -0.031198377, -0.025348682, 0.026676273, -0.0038064509, -0.037172534, 0.022679672, -0.003937827, 0.019526644, -0.026316717, 0.00017642866, 0.041459545, -0.010558493, 0.014880077, 0.014230111, 7.346692e-06, 0.017396968, 0.0017329551, -0.008062347, -0.0059845294, -0.024007263, -0.03885968, 0.0027139545, 0.016041718, 0.0050856397, -0.014479035, 0.010447861, -0.022057364, -0.008449561, -0.01897348, 0.02196056, -0.007813423, 0.0033777498, -0.019678764, -0.014935394, -0.013255161, -0.016581053, 0.00034097297, -0.0062887687, -0.018724559, -0.021642491, -0.006420145, -0.028308103, 0.02313603, -0.018517122, -0.001981014, -0.020397875, -0.0077511924, -0.025431655, 0.018641584, -0.009701091, 0.0076958765, 0.0060398453, -0.017728865, 0.0072879186, -0.0014304442, -0.0059741572, 0.009528228, -0.016276812, -0.004674225, 0.0114435535, 0.0029870786, 0.032996155, -0.013849812, -0.033466343, -0.009417595, 0.020065978, 0.0131307, 0.009777151, 0.02612311, -0.026468836, 0.007861825, 0.0017312265, -0.0072395173, 0.008276697, 0.015018368, 0.015018368, -0.027934717, -0.01142281, 0.023633877, 0.011602588, -0.0051997295, -0.028100666, 0.022071192, -0.0034935684, 0.03916392, -0.012619024, -0.019180918, 0.0064063156, -0.0117616225, 0.024242356, 0.011547272, 0.012570622, -0.00079344277, 0.012466905, -0.02313603, 0.002141777, 0.02373068, -0.015695993, -0.015585359, 0.0015678706, 0.018613925, 0.014326914, -0.01900114, 0.030506924, -0.010738271, 0.0040104296, 0.008546364, -0.014147136, -0.0015549058, -5.9313737e-05, 0.017410796, -0.00025843072, -0.051195208, 0.009721835, -0.029345281, 0.025611434, 0.004045002, 0.0096803475, -0.0052204733, 0.016733171, 0.033853557, 0.0024512024, 0.0036301303, -0.012363187, 0.0016257798, 0.0026517238, 0.0064408886, 0.18276496, -0.01201746, 0.0056388024, 0.014617325, -0.012992409, 0.011339836, -0.010634554, -0.01573748, -0.0015263833, 0.007384722, 0.03125369, 0.017880986, 0.0036716175, 0.0032273587, -0.0023094544, -0.005189358, -0.03310679, -0.0072326027, -0.026164597, -0.018129908, -0.015308778, -0.01915326, 0.01647042, -0.0022178367, 0.024726374, -0.0051478706, -0.045442317, 0.009728749, 0.0042109513, -0.007135799, -0.0142854275, -0.010440947, -0.00046370595, 0.015322607, -0.00048531385, 0.002349213, -0.0032774892, 0.0045359344, 0.014465205, -0.008179894, -0.0052204733, -0.01974791, -0.002848788, -0.009763322, -0.009514399, 0.007028624, -0.012052033, -0.00667944, -0.01868307, 0.00242873, -0.024449792, 0.0134003665, -0.0034071365, 0.03424077, 0.0012722743, -0.028653828, 0.013587059, 0.023288151, -0.010828161, 0.025445485, -0.0044909897, 0.006465089, -0.026344374, 0.021711636, -0.008408073, 0.019028798, -0.0068419315, 0.009590459, 0.030396292, -0.010413288, -0.011028682, -0.029068701, 0.004591251, -0.007516098, -4.6132907e-05, -0.021241449, 0.017120386, 0.02909636, 0.021227619, 0.025182733, 0.009362279, 0.0015583631, -0.013013153, -0.012681255, -0.011464298, -0.038804363, 0.011270691, -0.002414901, -0.009583544, 0.0005337156, -0.01705124, -0.008000116, -0.015709821, 0.008802202, 0.026067793, -0.0011633703, -0.03396419, -0.010973365, 0.022361603, -0.0068903333, -0.020411704, 0.05293767, 0.013317392, -0.011166972, -0.016857633, -0.00919633, -0.017161874, 0.008318184, 0.014202453, -0.017853327, 0.012342443, -0.00934845, -0.005583486, -0.019457499, 0.017535258, 0.032138754, 0.025915673, -0.028238956, 0.033908874, -0.010952622, -0.0040726606, -0.007578329, -0.0056007723, 0.017300164, -0.010807416, -0.011892999, -0.025708238, -0.002990536, -0.00077486, -0.02657947, -0.007329406, 0.00549014, 0.0122663835, -0.0108765615, -0.004739913, 0.010517006, 0.00445296, -0.013981188, -0.020135123, -0.0029888074, -0.00957663, 0.0060294736, 0.018212883, 0.013995017, 0.0009896427, -0.019014968, 0.03454501, 0.00500958, -0.0017666634, -0.029345281, -0.0117616225, -0.017023582, -0.02493381, -2.4795087e-05, 0.011243032, 0.0026309802, -0.023108372, -0.014382231, -0.020162782, -0.0076751327, -0.033936534, -0.0076751327, 0.014077991, -0.0033604635, -0.001637016, -0.02610928, -0.17690144, 0.016581053, -0.00021327016, 0.0015393482, 0.02895807, 0.014631154, 0.007930971, 0.004024259, 0.0014900821, 0.01483859, 0.026178425, 0.0047952295, -0.012370101, -0.009037296, 0.017977789, 0.0057528922, -0.011104741, 0.033300396, 0.031364325, 0.007253346, 0.022361603, -0.018572439, 0.0066033797, -0.028792119, -0.0014477306, 0.04143189, -0.009521313, 0.025528459, -0.034849253, -0.020384045, -0.023205176, 0.003688904, 0.021614833, -0.013697692, 0.028543197, 0.010959537, 0.009403766, 0.01973408, -0.007343235, 0.01468647, 0.006482376, 0.020231927, -0.00971492, -0.008318184, -0.010634554, 0.01291635, 0.019111771, -0.011270691, 0.018046934, -0.020729773, -0.0073224916, 0.004923148, 0.029594205, 0.0039309124, 0.028225128, -0.0037373055, -0.0099292705, 0.008926664, -0.0060813325, -0.01142281, -0.00957663, -0.008269783, 0.017327823, 0.0011538628, 0.0068834187, -0.041874416, -0.02493381, 0.0041348916, -0.035817288, 0.012950922, -0.029953761, -0.0021521486, 0.013635461, -0.0018755674, 0.02153186, 0.02493381, -0.007080483, 0.0022282086, 0.03844481, -0.0060329307, -0.0047364556, 0.014064162, -0.022071192, -0.013960444, 0.017134216, 0.009611202, -0.0071911155, 0.011844597, 0.00033470665, -0.002796929, 0.0038513953, -0.01886285, 0.009880869, -0.024408305, 0.017549086, 0.01387747, -0.00034745532, 0.002447745, -0.001858281, 0.0079448, 0.008214466, -0.012217982, -0.025639093, 0.05028249, 0.02729858, -0.0030683244, -0.021144645, -0.0013690777, 0.046659276, -0.024947638, -0.023080714, 0.005732149, 0.013912043, 0.024560425, -0.00042913327, 0.008560193, -0.00971492, -0.033328053, -0.012681255, -0.019014968, 0.055897094, 0.008041603, -0.004632738, 0.04226163, -0.020812748, -0.022762645, -0.10797736, 0.007495355, -0.004127977, 0.03280255, -0.028003862, 0.0046915114, -0.01305464, 0.02479552, -0.006344085, 0.018212883, 0.007778851, -0.017120386, -0.020453192, 0.004650024, 0.003778793, 0.006112448, 0.015115172, -0.022665842, -0.03410248, 0.010406374, 0.0026655528, 0.0023336553, 0.0028729888, -0.00028824963, 0.027907059, -0.023177518, -0.019305378, 0.03562368, 0.008034688, 0.019969175, 0.02017661, -0.021697808, 0.0282113, -0.005023409, -0.012155751, -0.009832467, -0.0035333268, -0.021766953, 0.026178425, -0.04823579, 0.006541149, 0.019595789, 0.020301072, -0.007688962, 0.0108765615, -0.01454818, -8.9348745e-05, 0.007972457, 0.0050925543, -0.02063297, -0.030313317, -0.021863757, -0.02685605, -0.015460898, 0.029538888, -0.04986762, -0.008214466, 0.043119036, -0.016110864, -0.028985726, -0.006022559, -0.0020207725, -0.036453422, 0.0015808353, -0.011395152, -0.022887107, -0.00667944, -0.021683978, 0.012197238, -0.027229436, 0.01468647, -0.001967185, -0.01499071, -0.0070147947, -0.0326366, -0.0010173008, -0.031281352, -0.017438455, 0.026814563, 0.0013777209, -0.0070493673, -0.009777151, 0.01186534, -0.0038652245, 0.01781184, 0.003547156, -0.0054624816, 0.031530276, 0.0005108112, -0.011962144, -0.009950015, 0.035540704, -0.021919074, -0.030064393, -0.012854119, 0.034517355, -0.016636368, -0.01015745, 0.004898947, 0.024131723, -0.025251878, 0.0025428198, -0.0883954, 0.023384955, 0.0031046257, -0.029649522, -0.013441854, -0.0142854275, 0.005465939, -0.007135799, -0.0038168228, 0.027326237, -0.039827716, 0.025998648, -0.014465205, -0.011630246, -0.0155438725, -0.013552487, 0.032304704, 0.010572323, 0.022209482, 0.015502385, -0.009908527, 0.0019481699, -0.0030959826, 0.0274507, -0.029870788, 0.021974389, -0.008899005, 0.03814057, -0.01090422, -0.009562801, 0.0041487203, -0.01942984, -0.000799493, 0.025390169, -0.014617325, -0.011277605, -0.02568058, -0.0056664604, 0.0030700532, 0.015958745, -0.010510092, -0.040740434, 0.00012835104, -0.010012245, -0.00024654635, 0.0069836793, -0.0038652245, -0.005953414, -0.00382028, 0.00031547563, 0.026786905, 0.0043699853, -0.025943331, -0.039053287, 0.0019792854, -0.0062956833, 0.00986704, -0.006786615, -0.0089197485, 0.0140295895, 0.024117894, 0.021739295, 0.031530276, 0.027810255, -0.011803109, -0.01634596, -0.01282646, 0.006070961, -0.012805717, -0.05398868, -0.000121652585, -0.0014848962, -0.0060018157, 0.007211859, -0.010530835, 0.0210755, 0.0033120618, 0.025044441, 0.008352757, -0.0015600917, 0.009113356, -0.02314986, -0.016110864, -0.012923264, 0.004622366, 0.012729657, -0.008477218, 0.018959653, -0.013365794, -0.0040069725, -0.0056802896, 0.033162106, -0.002537634, 0.00030035008, -0.013822153, 0.017853327, -0.014852419, 0.00246676, 0.023661535, 0.018766046, 0.009037296, 0.016857633, 0.0046534813, -0.027990034, -0.0038271945, 0.006368286, -0.015613018, -0.014617325, 0.0017441912, 0.010779759, -0.012930178, 0.031419642, -0.012508392, 0.006769329, -0.0017122115, 0.011243032, -0.009735663, -0.02595716, -0.03446204, 0.05769487, 0.025486972, 0.024781689, 0.0107521005, -0.004245524, 0.009777151, 0.026966682, 0.02937294, -0.010150536, 0.020951038, -0.014562009, 0.005828952, 0.006987137, -0.039827716, -0.0031426556, 0.0016119507, -0.007315577, -0.00057390635, -0.014893906, -0.02432533, 0.07451102, 0.0045013614, -0.019817054, 0.02565292, -0.039717082, 0.01706507, 0.01915326, 0.016733171, 0.0054313666, -0.038085252, -0.01401576, -0.0134625975, -0.022002047, -0.024435963, -0.02298391, 0.021116987, -0.0012186866, 0.0004245956, -0.027035829, -0.0009784065, -0.010067562, 0.019042626, 0.016235325, -0.001459831, -0.015557702, -0.002338841, 0.013448768, -0.012404674, 0.011906828, -0.053463176, -0.011291434, 0.0019913858, -0.029400598, -0.023993433, 0.018005446, -0.014396059, -0.025168903, -0.019236233, 0.019236233, 0.010378716, 0.0041487203, 0.0071703717, -0.03637045, -0.024159381, 0.025210392, 0.00023142081, -0.022071192, -0.007446953, -0.017244847], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-09-00850.pdf', 'citation': 'Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.82501578),\n",
      " NodeWithScore(node=TextNode(id_='5a680be7-b5cc-4d06-88cb-9965ae62c94d', embedding=[-0.014390817, -0.016151834, -0.02001782, -0.006989039, 0.012739862, 0.024186477, -0.0141706895, -0.023195906, -0.02081578, -0.018573234, 0.016633363, 0.017486356, -0.0032331184, -0.0036492965, -0.0038728632, -0.009272859, 0.018807119, -0.012712346, 0.009424197, -0.031808384, 0.006273626, 0.0064834342, -0.006373371, -0.013723556, 0.01000203, 0.036651183, 0.014982409, -0.033294242, -0.024888134, 0.009114643, -0.0023990427, -0.00624267, -0.027364565, -0.005444709, -0.011453494, -0.021201003, 0.035000227, -0.016605847, 0.032000992, 0.0014205085, 0.015285083, -0.00014230881, -0.0063768104, 0.0044472576, -0.011439736, -0.010077699, 0.013338334, 0.002510826, -0.013579098, 0.012106997, 0.026759215, -0.0056854733, -0.010731202, 0.013551582, 0.008433624, 0.0094517125, 0.018091705, -0.006074135, -0.015670307, -0.017692724, -0.0076631787, -0.0124784615, -0.028148768, -0.010366616, -0.0074705672, -0.008364834, -0.013572219, 0.011790564, 0.0029424818, 0.023140874, 0.023250937, 0.018119222, 0.0037559206, -0.003403373, 0.029882269, -0.025218325, -0.029662142, 0.012801774, -0.000497006, 0.015367631, 0.0129668685, -0.009204069, 0.00096993556, -0.0056235623, 0.0269931, 0.003546112, 0.045428753, 0.022494249, 0.007601268, -0.018050432, 0.033734497, 0.018490687, 0.022934504, 0.019880239, -0.02242546, 0.019453743, -0.017266229, 0.023250937, 0.008536808, -0.029634627, -0.030707747, 0.0003056845, -0.009355406, -0.013441518, -0.03136813, 0.01335897, 0.016949795, -0.017156165, 0.012808653, -0.017527629, -0.012175786, 0.021393614, -0.0065487847, -0.05855384, 0.027213227, 0.0036699334, 0.044107992, -0.012671073, -0.015532726, -0.01827056, 0.015395147, 0.020375526, 0.009891966, -0.012849926, 0.023484822, 0.03189093, -0.0034945195, -0.042071816, -0.0011995214, -0.01463846, 0.026222654, 0.020568136, 0.028754117, -0.006552224, -0.019082278, 0.032386217, -0.047767606, -0.010442285, -0.005479104, -0.01865578, -0.0026466858, 0.044630792, -0.0141019, -0.0070062364, -0.01932992, 0.00051334355, 0.013379607, 0.0077182106, 0.011343431, -0.0063940077, 0.022893231, -0.010215279, -0.003546112, 0.0151199885, 0.0007149835, -0.020760749, 0.011563558, 0.027364565, -0.020265462, -0.032523796, -0.01732126, -0.005881524, 0.024819344, -0.011508526, 0.018958457, 0.02014164, 0.018022915, -0.00034029433, 0.0005924518, -0.015601517, 0.004663945, 0.031863414, -0.016674636, 0.007972732, -0.013235149, 0.016619604, 0.007848911, 0.011040756, 0.018091705, -0.026511572, -0.006796428, 0.02806622, 0.036733728, 0.025300872, -0.020540621, 0.013001264, 0.01699107, -0.01866954, 0.025066987, -0.034945194, 0.008763814, -0.008695025, 0.013503429, 0.001591623, -0.63308585, -0.036816277, -0.0035667487, -0.009272859, -0.0008899674, -0.0033001886, -0.00014177138, 0.00086847064, -0.0006681205, 0.018559476, -0.007009676, -0.012767378, -0.013923046, -0.022851957, -0.029056793, -0.026085075, 0.015532726, -0.023044568, -0.0037008887, -0.013757951, -0.024557943, 0.009898846, 0.006810186, 0.010304705, 0.009960757, -0.00059804093, 0.03753169, -0.017706484, -0.0069408864, 0.004041398, -0.031313095, 0.017059859, -0.0062357914, -0.012395914, 0.043695252, -0.024365332, 0.01671591, 0.0077870004, 0.021338582, 0.003296749, -0.022012722, -0.0165233, 0.008412987, 0.025273357, -0.016055528, 0.027089406, 0.032193605, -6.6478846e-05, -0.0011229927, -0.011790564, -0.01578037, 0.008942668, -0.0009974515, 0.022686861, 0.015739096, 0.004323436, 0.026649151, -0.02216406, 0.0074499305, 0.007229803, -0.007553115, 0.014652218, 0.015794128, -0.038714875, -0.013180117, 0.032798953, -0.016908523, -0.00893579, 0.032138575, -0.013200754, 0.02216406, 0.01456967, 0.0012545532, -0.00617732, 0.0058574476, 0.043805316, 0.013668524, 0.012251455, -0.010462922, 0.026511572, -0.009623687, -0.006669167, -0.024544185, 0.009754388, 0.012437188, -0.00932789, -0.039595384, -0.0057577025, -0.004240888, 0.0011462093, 0.016303172, 0.021668773, -0.0021256034, -0.020939602, -0.015697822, 0.02148992, 0.0020550939, -0.012602283, 0.03048762, -0.023292212, -0.0077044526, -0.01806419, 0.032688893, -0.007608147, 0.020719474, -0.010084578, 0.008550567, -0.018036673, 0.03637602, 0.004309678, -0.025011955, 0.0030164307, -0.0294145, 0.0005812734, -0.01389553, -0.037504174, 0.023250937, 0.009830056, -0.020334251, -0.01067617, 0.0068996125, 0.00564076, -0.015629033, 0.005203945, 0.04829041, 0.006882415, 0.023388516, -0.025727369, -0.012457824, 0.0005752543, 0.002571017, -0.010930692, 0.047107223, -0.021861384, 0.01806419, 0.02048559, 0.038935002, 0.004440379, -0.0056717154, -0.035357933, -0.0038350287, -0.0019140749, 0.027818577, 0.00429936, -0.010923813, -0.014253237, -0.0021307627, -0.0041996147, 0.011515405, 0.0020189793, 0.019481258, 0.017623935, -0.0020430558, -0.0014007314, 0.011054514, -0.0165233, -0.014211963, -0.04531869, -0.009651203, -0.012258334, 0.0012459544, 0.017527629, -0.009025216, -0.0052520977, -0.004804964, 0.0010645215, 0.0058264923, 0.0014394256, -0.006056938, -0.027694756, 0.011364068, -0.023168389, -0.02739208, 0.027543418, -0.0049941363, 0.031533223, -0.016193109, 0.018476928, -0.021118455, -0.015876675, -0.005823053, -0.00812407, 0.018724572, 0.0012614322, 0.024475396, -0.0016681516, 0.037366595, -0.0095273815, 0.032193605, 0.034202266, 0.005795537, 0.0025228644, -0.0009983113, 0.019192342, -0.016193109, -0.00016917981, 0.030460104, 0.012292729, -0.0100914575, 0.0152988415, 0.01249222, 0.0054343906, -0.006397447, -0.036486086, 0.0067310776, -0.024200236, -0.011391584, -0.025589788, 0.015532726, 0.012526615, 0.0055582123, -0.017912852, -0.031037938, 0.008674388, 0.014693492, 0.0015331517, 0.003303628, -0.0046708244, -0.021379856, -0.0009854132, -0.0047292956, 0.00094241963, 0.014473365, 0.007979612, -0.02121476, 0.013173238, 0.008151586, 0.012265214, 0.0010438845, -0.0049700597, 0.014390817, 0.022439217, 0.01188687, -0.0016939477, 0.06636836, 0.0010223878, 0.014500881, -0.017252471, 0.0045882766, 0.0071747713, -0.015794128, 0.019178584, 0.029992333, -0.020375526, 0.018683298, -0.009492986, 0.01208636, 0.021930173, -0.019192342, -0.0076838154, -0.019852724, -0.0008641713, -0.00523834, -0.0036940097, -0.0025022274, -0.02275565, -0.013510308, -0.001705126, 0.043199968, 0.010614259, -0.010469801, -0.0055960463, -0.010614259, -0.0114191, 0.0192061, -0.005633881, -0.006283944, -0.00839235, -0.031780865, -0.019316163, -0.02638775, -0.0051936265, -0.003215921, -0.014115658, 0.008990821, 0.0021445206, -0.005792097, 0.0050078942, 0.0141019, -0.004681143, -0.0065763006, -0.020967117, 0.012306487, 0.019426227, 0.007057829, -0.0023336925, 0.0007829134, -0.017651452, -0.010772476, -0.005551333, 0.014803555, 0.027103163, -0.00032137713, -0.0070062364, -0.026828004, 0.009479228, 0.014390817, -0.02612635, 0.008495535, 0.016193109, -0.00570611, 0.004457576, -0.012932474, -0.010249673, 0.031973477, -0.009788782, 0.015629033, -0.0013242027, -0.012058844, -0.017761515, 0.010511074, 0.0067379563, -0.0019312723, -0.005358722, -0.010593622, 0.0027739468, -0.029221889, 0.01866954, 0.0024196797, -0.019687628, -0.0030835008, -0.036100864, -0.013008143, 0.008337318, 0.04518111, 0.0347801, -0.0071334974, 0.019274889, 0.03068023, 0.0067861094, -0.013056296, -0.0179954, -0.012898079, -0.030845325, -0.00483592, 0.0012098398, 0.015353873, 0.013090691, 0.0074086566, -0.013180117, -0.0043406337, -0.011302156, -0.022535523, -0.013874894, -0.009651203, -0.0069787204, 0.01745884, 0.006989039, 0.02223285, -0.016743427, 0.03343182, 0.027914882, 0.0083235605, -0.012918716, -0.015037441, 0.0018899986, 0.013647887, 0.03142316, 0.015092473, -0.0029270041, 0.0033603795, 0.009369165, 0.026442781, 0.010827508, 0.0030250296, 0.03001985, 0.0041789776, -0.013269544, -0.01242343, 0.011460373, 0.00134226, 0.020292977, -0.0109650865, -0.022494249, 0.035220355, 0.007945216, -0.0034790419, -0.02282444, 0.02088457, 0.022494249, -0.0015202536, 0.012829289, -0.03604583, -0.033734497, 0.0004578818, -0.030597683, -0.012918716, 2.1080297e-05, -0.01778903, -0.0066244532, 6.798362e-05, -0.020361768, -0.025589788, 0.0029631187, -0.0064456, -0.027213227, -0.016344447, -0.0036905704, 0.00947235, -0.0031608893, 0.018504445, -0.018999731, -0.0037421626, 0.017541388, -0.02095336, -0.015257568, 0.017568903, -0.031808384, 0.008309802, -0.014514638, -0.0038522263, -0.015656548, -0.01758266, 0.019797692, -0.013193875, 0.012437188, 0.01584916, 0.017527629, -0.0016328968, 0.021503678, 0.017830305, 0.008805089, -0.018793361, -0.026580362, 0.0070131156, -0.024585458, -0.005860887, -0.012609162, -0.012313366, -0.0018848393, -0.0009037254, 0.016770942, -0.021833869, -0.00987133, 0.015285083, -0.013173238, -0.0020430558, 0.030845325, 0.046887096, 0.02356737, 0.0130425375, -0.010572986, -6.7446206e-05, -0.009761266, 0.0081171915, -0.00049829576, -0.0076769367, 0.00678267, 0.006366492, 0.03310163, -0.011900628, -0.029359467, -0.027433354, 0.0041377037, -0.007284835, 0.02713068, 0.0007270217, -0.018105464, -0.026580362, -0.013207633, 0.01941247, -0.0103391, -0.0136066135, -0.02671794, -0.021820111, -0.00932789, -0.01335897, -0.001047324, -0.010634896, -0.037201498, 0.004075793, 0.00879821, 5.653658e-05, 0.017637692, -0.012526615, 0.008412987, -0.013331454, 0.007339867, -0.031478193, -0.015395147, 0.00091834326, 0.0020121003, 0.03604583, 0.025617305, 0.021311065, -0.013070053, 0.021008391, 0.023319727, 0.015161262, -0.0102978265, -0.009142159, 0.0036630544, -0.013544703, -0.0024987878, 0.013771709, -0.015202536, 0.00017089954, 0.003215921, 0.016936038, 0.02109094, 0.00028311284, -0.003786876, 0.00037619396, -0.024874376, -0.0394578, -0.004454137, -0.013255786, 0.012953111, -0.023581129, -0.027736029, 0.03208354, 0.012794894, 0.006634772, 0.010098336, 0.034587488, -0.023925077, 0.03678876, -0.013276422, 0.025342146, -0.02752966, 0.014597186, -0.025273357, -0.0023268135, 0.035550546, -0.0012407952, 0.02303081, 0.013475913, -0.0015297122, 0.014762281, 0.021930173, -0.008777573, -0.03370698, -0.011735532, -0.0011152539, -0.022893231, -0.00070208544, 0.014266995, -0.029111825, 0.00063630525, -0.026098832, -0.009953878, 0.01827056, -0.019825207, 0.0013981517, 0.033459336, 0.0027412716, 0.04297984, -0.020581895, 0.030239977, 0.017169923, 0.0011505086, -0.024145205, -0.0022941383, -0.013936805, 0.0036011436, 0.045869008, 0.030707747, -0.027048131, -0.038054492, 0.03552303, 0.025094504, -0.01121273, -0.039430287, 0.02027922, 0.004478213, -0.017362535, -0.018545719, 3.3830587e-05, -0.039980605, 0.025686095, -0.009190312, 0.016729668, -0.013675403, -0.0059056003, 0.01865578, 0.0066450904, 0.012058844, 0.02713068, 0.009912604, -0.015739096, 0.021696288, -0.014445849, -0.01624814, 0.009885088, 0.0058402503, 0.024158962, -0.018972214, -0.007635663, 0.00651439, 0.0009639164, -0.0018831196, 0.010586743, -0.011804322, 0.038494747, -0.0119900545, 0.0138129825, -0.01269171, 0.0088876365, 0.023209663, -0.0023938834, 0.008846362, -0.012863684, 0.021173486, -0.016619604, 0.019082278, 0.010483558, 0.0066726063, -0.018353106, -0.007071587, -0.0019519093, -0.011797443, -0.014927377, 0.012774258, -0.027832335, 0.00597783, -0.0015004765, 0.017224954, 0.026415266, 0.014886103, 0.00037898857, -0.023966352, 0.017899094, -0.03343182, 0.0074912044, 0.010827508, -0.03216609, -0.020912085, -0.00016004367, -0.0044334996, -0.008213497, 0.0043543917, -0.0365136, -0.02377374, -0.004808404, 0.0012425149, 0.04336506, -0.017568903, -0.028891698, 0.012849926, 0.0040654745, 0.015739096, 0.0038419077, 0.0065487847, 0.044080473, -0.019838965, 0.007965853, 0.012347761, 7.920226e-06, 0.031065453, 0.01355846, -0.013372729, 8.93192e-05, -0.021723805, -0.048648115, 0.005427512, 0.019054763, -0.0004600315, -0.016578332, -0.0044954107, -0.020705717, -0.009341649, -0.007284835, 0.0019329921, -0.0043887864, -0.013551582, 0.012774258, 0.007635663, -0.0167847, -0.019481258, -0.010524833, -0.0035873856, -0.015518969, -0.018710813, -0.0118112005, -0.015904192, 0.030102396, -0.024805585, -0.023278452, -0.0365136, -0.0135309445, -0.031065453, -0.0014273875, 0.018642023, 0.005740505, 0.021173486, -0.016936038, -0.004158341, -0.010779355, -0.012203302, 0.010559227, -0.012024449, -0.008894515, 0.009568655, 0.0077044526, 0.026098832, -0.0032778317, -0.020980876, 0.0044059837, 0.022686861, 0.00543783, -0.015752854, 0.023443548, -0.02894673, 0.018091705, -0.007814516, 0.00076227647, -0.009995151, 0.0018831196, 0.025548516, -0.02202648, -0.025204565, 0.009376043, -0.014418333, -0.00228382, -0.028644053, 0.0027705075, -0.011240246, 0.031478193, -0.016949795, -0.03780685, 0.0141019, -0.023663675, 0.02787361, -0.008584961, 0.0033930547, -0.0053484035, 0.021985205, -0.014322027, -0.01839438, 0.0267317, -0.005197066, -0.0049115885, 0.012664194, 0.014184448, 0.0018917182, -0.02652533, 0.0076975734, -0.002134202, 0.016702153, -0.00570611, -0.010999482, -0.021971447, -0.005296811, 0.033404306, -0.0021995525, -0.040063154, 0.016371962, -0.015312599, -0.005001015, 0.015188778, -0.0010782795, -0.006184199, 0.033624433, 0.025232082, -0.0061738803, -0.007332988, -0.013063175, 0.0077594845, -0.015904192, -0.0040861117, 0.17731248, -0.022893231, -0.00097337505, 0.03431233, -0.0027223546, -0.0009811139, -0.000920063, 0.006345855, 0.006524708, -0.006204836, 0.03296405, -0.001423088, 0.017046101, 0.0039038185, 0.014184448, 0.004237449, -0.015587758, -0.01612432, -0.017224954, -0.02410393, -0.009080248, -0.019467501, 0.003955411, 0.0015417504, 0.013978078, -0.024213994, -0.01604177, 0.005217703, 0.0077182106, -0.0056098043, -0.009891966, 0.005228021, 0.007979612, 0.0018693616, 0.0032589147, 0.007910822, 0.00651439, 0.018476928, 0.0230996, -0.0029785964, 0.012340882, -0.017926611, -9.09852e-05, -0.013778588, -0.00731923, 0.0006406046, 0.0074499305, 0.0034721629, -0.005296811, -0.005964072, -0.012051965, 0.0048531173, 0.0020980875, 0.020444315, -0.014776039, -0.04215436, 0.017844062, 0.012382156, -0.0038591053, 0.02450291, -0.017142408, 0.014253237, -0.0054619065, 0.02867157, -0.022783168, 0.028368896, 0.0055169384, 0.016936038, 0.021462403, -0.013331454, 0.023471065, -0.014039989, -0.008825726, -0.025493484, -0.011095787, -0.020059092, 0.02323718, 0.008179102, 0.027226985, 0.03524787, -0.004244328, 0.0012304768, -0.021985205, -0.021778837, -0.0076494208, -0.034670036, 0.031918447, -0.007498083, -0.034532458, -0.012203302, -0.014583428, -0.013957441, -0.007890185, 0.004216812, 0.024158962, 0.00026613037, -0.01141222, 0.00031643288, -0.000705095, -0.022508008, -0.032386217, 0.043475125, -0.004103309, -0.012574767, -7.75497e-05, 0.0147347655, -0.0075462363, 0.0068514594, 0.01194878, -0.02256304, 0.004694901, -0.012402792, 0.0039382135, -0.018435655, 0.013462155, 0.030129913, 0.0054756645, -0.031588256, 0.029854754, -0.00040650446, -0.012898079, -0.0112471245, -0.0026346475, 0.023883803, -0.020361768, -0.012939353, -0.028341379, -0.004619232, -0.004130825, -0.015161262, 0.005678594, 0.02048559, 0.021599984, -0.0024953485, -0.0041239457, -0.013393365, -0.0034257297, -0.024833102, -0.030295009, 0.0012046805, -0.0042924806, -0.008206618, 0.01040789, 0.0029132462, -0.013420881, -0.018229285, 0.020320494, 0.0066175745, -0.011969417, -0.03156074, 0.005606365, -0.018903425, -0.019825207, 0.012870563, 0.027832335, -0.018229285, -0.017527629, -0.046171684, -0.025108261, -0.01282241, -0.04097118, -0.012609162, 0.021544952, -0.009719992, 0.013049416, 0.014803555, -0.17599171, 0.021696288, 0.0022236288, -0.0077182106, 0.018353106, 0.018600749, 0.01369604, -0.015725339, 0.011969417, 0.0030164307, 0.03766927, 0.008523051, -0.022783168, -0.02021043, 0.026098832, 0.025878705, -0.024200236, 0.017981643, 0.02671794, 0.013689161, 0.022920746, -0.023801256, 0.003807513, -0.012533493, 0.0046708244, 0.03351437, -0.017266229, 0.041549012, -0.027983673, -0.021847626, -0.02021043, 0.01127464, 0.021063423, -0.016096802, 0.019962788, -0.0037043283, 0.006352734, 0.014211963, -0.0038625447, 0.00083579554, 0.0378894, 0.020967117, 0.003215921, -0.017885337, 0.009561776, -0.005386238, 0.015518969, -0.00090200565, 0.012533493, -0.026332717, 0.0049528624, -0.0005524677, 0.012664194, 0.005510059, 0.009059611, 0.0020671322, -0.019302405, 0.01242343, 0.018614508, -0.009389802, -0.0030370676, -0.0044163023, 0.015395147, -0.016702153, -0.014528397, -0.021723805, -0.020774506, -0.00926598, -0.014487122, 0.005424072, 0.0014291072, 0.007030313, 0.002517705, 0.0053415243, 0.0012106997, 0.020114124, -0.016220624, 0.018009158, 0.0071472554, -0.005076684, -0.006607256, 0.021943932, -0.029827237, 0.00017025464, 0.024681764, 0.00039188663, 0.011522284, 0.0016096802, -0.0018418457, 0.00037232455, 0.0027911444, -0.02223285, -0.0017034063, -0.015986739, 0.025465967, 0.018132979, -0.006775791, -0.0037180863, -0.004973499, -0.010916934, 0.025011955, -0.013138843, -0.018889667, 0.040696017, 0.014349543, 0.0055444543, -0.04432812, 0.006682925, 0.05040913, 0.008275407, -0.03389959, -0.013799225, 0.019219857, 0.0083992295, 0.004075793, 0.0079177, -0.005231461, -0.046006586, 0.0076769367, -0.0075462363, 0.047574993, 0.0030267492, -0.010125852, 0.041989267, -0.0149411345, -0.024117688, -0.10967839, 0.014363301, -0.0030473862, 0.013517187, -0.013874894, 0.009754388, -0.007766363, 0.006036301, -0.003135093, 0.009259101, -0.007305472, 0.0017429604, 0.0014892982, -0.004774009, -0.021503678, 0.012829289, 0.0032382777, -0.009609928, -0.022053996, 0.021297308, 0.016757185, -0.01787158, 0.007986491, -0.002841017, 0.012230818, -0.005520378, -0.013448397, 0.019219857, 0.0042340094, -0.0083992295, 0.008213497, -0.019096036, 0.0158354, -0.01787158, -0.027956156, -0.012127634, -0.006115409, -0.023608644, 0.010524833, -0.031313095, -0.009410438, 0.008089676, 0.013269544, -0.012526615, 0.0050113336, -0.020100366, -0.019274889, 0.006283944, 0.0020172596, -0.029827237, -0.018036673, -0.01886215, -0.032331184, -0.018022915, 0.022686861, -0.015518969, -0.008440503, 0.013675403, -0.010332221, -0.020912085, -0.0035564303, -0.0003645857, -0.030652715, 0.03469755, -0.0071541346, -0.018559476, 0.0016741706, -0.04042086, 0.016812216, -0.034890164, 0.025259597, 0.0024592336, -0.0048531173, -0.02713068, -0.01604177, -0.0114191, -0.02303081, -0.02685552, 0.014514638, -0.00024721317, 6.728498e-05, -0.018339349, 0.01073808, 0.0121964235, 0.010916934, 0.018999731, -0.0030731822, 0.018490687, 0.008330439, -0.019687628, 0.012375277, 0.031203033, -0.016757185, -0.026896793, -0.019646354, 0.02685552, -0.015216294, -0.034670036, 0.0038969396, 0.016867248, -0.034202266, -0.0061876383, -0.07495332, 0.01827056, -0.002120444, -0.025754884, -0.0045710793, -0.02322342, -0.013070053, -0.0037352836, 0.0012476742, 0.040448375, -0.016096802, 0.03747666, -0.023512337, -0.0044163023, -0.028974244, -0.004660506, 0.02531463, -0.015904192, 0.031340614, 0.030047365, -0.0014058906, 0.0189447, 0.014776039, 0.029827237, -0.014899861, 0.007023434, -0.005795537, 0.037504174, -0.016633363, -0.014982409, 0.015821643, -0.01778903, 0.002424839, 0.017101133, 0.010284068, 0.006641651, -0.009431075, -0.002182355, 0.0060053454, 0.033074114, -0.022975778, -0.028754117, 0.0034859208, -0.0098094195, 0.015986739, 0.005719868, -0.013572219, -0.020320494, 0.016413236, 0.0051661106, 0.03208354, 0.009898846, -0.021008391, -0.033486854, 0.0021307627, -0.008089676, -0.0035873856, 0.009176553, 0.003417131, -0.011322794, 0.022205332, 0.036981374, 0.022191575, 0.005554773, -0.0041239457, -0.0052933716, -0.005792097, -0.0007850631, 0.001222738, -0.060204793, -0.006431842, -0.008873878, -0.006156683, 0.01578037, 0.006565982, -0.0027154756, -0.0063286573, 0.012402792, 0.010380374, 0.015395147, 0.019109793, -0.009940119, -0.009905725, -0.005541015, 0.03618341, 0.0029235645, 0.011577316, 0.005661397, -0.007615026, 0.0011083749, -0.01496865, 0.016770942, 0.024213994, 0.0045848372, -0.023993867, 0.010036426, -0.004718977, 0.0028427367, 0.031148002, 0.02349858, 0.0036699334, 0.010648654, 0.0077113314, -0.013874894, -0.007326109, 0.000644474, -0.018353106, -0.01981145, 0.021613741, 0.020389283, 0.0008005408, 0.015986739, -0.015890433, 0.008007128, -0.015931707, 0.014927377, -0.020237945, -0.0082203755, -0.03048762, 0.030322524, 0.013819862, 0.034890164, 0.027708514, -0.0069855996, 0.013689161, 0.005606365, 0.017692724, 0.014253237, 0.018793361, -0.01269171, 0.016261898, 0.02254928, -0.03604583, -0.011171456, -0.012870563, 0.015009925, 0.0033792967, -0.002806622, -0.007883306, 0.07621905, 0.018077947, -0.00041123375, 0.024158962, -0.043144934, 0.012045086, 0.036486086, 0.027515901, -0.0078007584, -0.004873754, 0.0010163686, -0.009238464, 0.0040517165, -0.03389959, -0.020155398, 0.008474898, 0.0018762406, 0.005458467, -0.016729668, -0.0012433748, 0.015986739, -0.0050801234, 0.02713068, -0.010270311, -0.022604313, 0.018999731, 0.017747756, -0.013702919, -0.002142801, -0.050849386, 0.01732126, -0.012533493, -0.034615006, -0.0081171915, 0.01564279, -0.0028822906, -0.021035908, -0.045153596, 0.052335247, -0.00040392485, 0.0014884383, -0.015959224, -0.02631896, -0.01960508, -0.0057989764, -0.00084783375, -0.025699852, -0.0068411413, -0.018628266], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-08-02548.pdf', 'citation': 'Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548.'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.80738497)]\n"
     ]
    }
   ],
   "source": [
    "from pprint import pprint\n",
    "pprint(response_adoc_match)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "5"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(response_adoc_match)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548\n",
      "{'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-12-616548.pdf', 'citation': 'Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548'}\n",
      "0.83827609\n",
      "Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.\n",
      "{'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-10-00627.pdf', 'citation': 'Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.'}\n",
      "0.8263269\n",
      "Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.\n",
      "{'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-10-01533.pdf', 'citation': 'Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.'}\n",
      "0.8256848999999999\n",
      "Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850\n",
      "{'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-09-00850.pdf', 'citation': 'Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850'}\n",
      "0.82501578\n",
      "Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548.\n",
      "{'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-08-02548.pdf', 'citation': 'Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548.'}\n",
      "0.80738497\n"
     ]
    }
   ],
   "source": [
    "for r in response_adoc_match:\n",
    "    print(r.text)\n",
    "    print(r.metadata)\n",
    "    print(r.score)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "# library.docs_index.docstore.persist('./storage/phages_docs_index_docstore.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "# library.docs_index.vector_store.persist('./storage/phages_docs_index_vector_store.json')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Testing aquery"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
     ]
    }
   ],
   "source": [
    "response_aquery = await library.aquery(query)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('What are the bacteriophages(s) that show promising results in aquaculture '\n",
      " 'and food industries and what is the pathogen that each can control?')\n",
      "('Bacteriophages Vp33, Vp22, Vp21, Vp02, Vp08, and Vp11 have shown promising '\n",
      " 'results in controlling the pathogen Vibrio parahaemolyticus, which is a '\n",
      " 'leading foodborne pathogen often isolated from a variety of seafood. These '\n",
      " 'bacteriophages were isolated from seafood samples and characterized based on '\n",
      " 'host specificity, morphological, genomic character, protein profile, and '\n",
      " 'temperature and pH stability. The in vitro lytic activity of these '\n",
      " 'bacteriophages demonstrated significant effects against V. parahaemolyticus. '\n",
      " 'Furthermore, the application of a bacteriophage cocktail instead of a single '\n",
      " 'bacteriophage suspension was observed to have better efficiency in '\n",
      " 'controlling the growth of V. parahaemolyticus (Tan et al., 2021).')\n",
      "{'Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., '\n",
      " 'Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and '\n",
      " 'Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From '\n",
      " 'Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: '\n",
      " '10.3389/fmicb.2021.616548'}\n"
     ]
    }
   ],
   "source": [
    "from pprint import pprint\n",
    "pprint(response_aquery.query)\n",
    "pprint(response_aquery.answer)\n",
    "pprint(set([c.node.metadata['citation'] for c in response_aquery.contexts]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: Isolation of a Virulent Aeromonas salmonicida subsp. masoucida\n",
      "Bacteriophage and Its Application in Phage Therapy in Turbot\n",
      "(Scophthalmus maximus )\n",
      "Zhenhe Xu ,aPeng Jin ,aXiangshan Zhou ,aYuanxing Zhang ,a,bQiyao Wang ,a,cXiaohong Liu ,a,cShuai Shao ,a,c\n",
      "Qin Liua,c\n",
      "aState Key Laboratory of Bioreactor Engineering, East China University of Science and Technology, Shanghai, China\n",
      "bSouthern Marine Science and Engineering Guangdong Laboratory (Zhuhai), Zhuhai, China\n",
      "cShanghai Engineering Research Center of Maricultured Animal Vaccines, Shanghai, China\n",
      "ABSTRACT Aeromonas salmonicida is an aquatic pathogen that can infect a variety of\n",
      "ﬁsh. Phage therapy has been applied to treat bacterial infections. In this study, we\n",
      "obtained three A. salmonicida subsp. masoucida phage isolates from sewage, and one\n",
      "phage (vB_AsM_ZHF) exhibited the best antibacterial effect, based on in vitro kinetics\n",
      "experiments. Sequencing indicated that the vB_AsM_ZHF genome is 161,887 bp (41.24%C1G content) with 237 predicted open reading frames. No antibiotic resistance or viru-\n",
      "lence genes were detected in the complete genome, which is a requirement for phage\n",
      "therapy safety. Intraperitoneal injection of phage vB_AsM_ZHF into turbot at 8 /C210\n",
      "4\n",
      "PFU/ﬁsh rescued turbot from A. salmonicida subsp. masoucida injection and reduced the\n",
      "bacterial burden by 1 order of magnitude. Injection of vB_AsM_ZHF also decreased levels\n",
      "of inﬂammatory cell in ﬁltration in muscle tissue, cytokines interleukin-1 b(IL-1b), tumor\n",
      "necrosis factor alpha (TNF- a), and gamma interferon (IFN- g) in serum and the expression\n",
      "of the in ﬂammatory factors IL-1 b, IL-6, IFN- g, transforming growth factor b,T N F - a,a n d\n",
      "hepcidin in the liver, spleen, and head kidney of turbot. Phage vB_AsM_ZHF demon-\n",
      "strated antibacterial ability in vitro and in vivo and signi ﬁcantly reduced mortality in tur-\n",
      "bot challenged by A. salmonicida subsp. masoucida . This study revealed that phage\n",
      "vB_AsM_ZHF can effectively treat the infection caused by A. salmonicida subsp. masou-\n",
      "cida in turbot.\n",
      "IMPORTANCE A. salmonicida is an aquatic pathogen that can infect different ﬁsh and\n",
      "causes economic loss to the global aquaculture industry. Clinical strains of A. salmoni-\n",
      "cida have developed multidrug resistance, and phage therapy is being evaluated for\n",
      "controlling bacterial infections. Phages are biological antibacterial agents and have the\n",
      "potential to be therapeutic agents against multidrug-resistant bacteria. In this study,three A. salmonicida subsp. masoucida phages were isolated from sewage, and their\n",
      "biological behaviors were characterized. The newly isolated phage vB_AsM_ZHF could\n",
      "inhibit A. salmonicida subsp. masoucida infection in vitro and in vivo ,s u g g e s t i n gt h a ti t\n",
      "may be an alternative strategy to antibiotics for protecting ﬁsh against multidrug-re-\n",
      "sistant A. salmonicida subsp. masoucida in the aquaculture industry.\n",
      "KEYWORDS Aeromonas salmonicida subsp. masoucida , antibacterial, aquaculture,\n",
      "phage therapy, turbot, Sc\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Xu, Zhenhe, et al. \"Isolation of a Virulent Aeromonas salmonicida subsp. masoucida Bacteriophage and Its Application in Phage Therapy in Turbot (Scophthalmus maximus).\" Journal of Virology, vol. 98, no. 2, 2024, pp. e01567-19.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.add('/workspaces/ml-learning/src/phages/data/test3/aem.01468-21.pdf')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
     ]
    }
   ],
   "source": [
    "response_aquery = await library.aquery(query)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('What are the bacteriophages(s) that show promising results in aquaculture '\n",
      " 'and food industries and what is the pathogen that each can control?')\n",
      "('Bacteriophages have shown promising results in controlling bacterial '\n",
      " 'pathogens in the aquaculture and food industries. A study isolated three A. '\n",
      " 'salmonicida subsp. masoucida phages from sewage, one of which, vB_AsM_ZHF, '\n",
      " 'was found to inhibit A. salmonicida subsp. masoucida infection in vitro and '\n",
      " 'in vivo, suggesting it could be an alternative to antibiotics for protecting '\n",
      " 'fish against multidrug-resistant A. salmonicida subsp. masoucida (Xu, '\n",
      " 'Zhenhe, et al., 2024). Commercial phage products such as Salmonlex, Listex, '\n",
      " 'Listshield, and Ecoshield have been reported to significantly reduce the '\n",
      " 'number of pathogens including Salmonella, Escherichia coli, and Listeria '\n",
      " 'monocytogenes during food storage (Xu, Zhenhe, et al., 2024). Additionally, '\n",
      " 'six Vibrio parahaemolyticus lytic bacteriophages isolated from seafood '\n",
      " 'samples showed significant effect against V. parahaemolyticus in vitro (Tan, '\n",
      " 'C. W., et al., 2021).')\n",
      "{'Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., '\n",
      " 'Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and '\n",
      " 'Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From '\n",
      " 'Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: '\n",
      " '10.3389/fmicb.2021.616548',\n",
      " 'Xu, Zhenhe, et al. \"Isolation of a Virulent Aeromonas salmonicida subsp. '\n",
      " 'masoucida Bacteriophage and Its Application in Phage Therapy in Turbot '\n",
      " '(Scophthalmus maximus).\" Journal of Virology, vol. 98, no. 2, 2024, pp. '\n",
      " 'e01567-19.'}\n"
     ]
    }
   ],
   "source": [
    "from pprint import pprint\n",
    "pprint(response_aquery.query)\n",
    "pprint(response_aquery.answer)\n",
    "pprint(set([c.node.metadata['citation'] for c in response_aquery.contexts]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[NodeWithScore(node=TextNode(id_='57c9d900-e1b0-4704-8e01-f0a011f9db7f', embedding=[-0.0067372723, -0.018468704, -0.005094457, -0.024804784, -0.0040084696, 0.012457728, -0.024514263, 0.0041675633, -0.010507101, -0.013481462, 0.016905434, 0.03962125, -0.0132186115, -0.0048108553, -0.010763035, 0.0020837816, 0.029383915, -0.029549927, 0.021941097, -0.03638404, 0.006685394, 0.009946815, -0.0061493176, 0.00017249562, 0.0026233168, 0.012990346, 0.008279789, -0.019049741, -0.014844134, 0.0014223323, 0.0019454393, -0.0030590952, -0.007954685, 0.011752183, -0.016933104, -0.026824582, 0.02855386, -0.006132025, 0.029162567, 0.009580208, 0.006996664, -0.008964584, -0.0002842503, 0.0041191434, 0.005606324, 0.00957329, 0.010832206, -0.007062377, -0.020460833, 0.012637573, 0.05334481, -0.014525947, 0.0014474068, 0.03148672, -0.009607876, -0.0034135976, 0.016241392, 0.004548005, -0.0063395384, -0.01936793, -0.0018243897, -0.008487303, -0.019312592, -0.025966858, 0.0012113602, -0.012312469, -0.0027080516, 0.0109912995, -0.0037525361, 0.0042920713, 0.013481462, 0.014041749, 0.0067303553, -0.005346932, 0.026769245, -0.017306628, -0.018122848, 0.0031299957, -0.006342997, 0.0072975587, 0.011447829, -0.00777484, -0.0049353633, -0.0011707222, 0.021539904, -0.010610858, 0.026700074, 0.020073475, -0.009428031, -0.008881579, 0.03560932, 0.049415886, 0.02747479, 0.011427078, -0.016352065, 0.017458804, -0.030130964, 0.030186301, 0.0013159816, -0.039095547, -0.004561839, -0.008203701, -0.013910323, -0.02178892, -0.039482906, 0.003541564, 0.008846994, -0.0037559948, 0.028498525, -0.022480631, -0.027336448, 0.007864763, -0.009047589, -0.048945524, 0.02793132, 0.022577472, 0.02838785, -0.010970549, -0.022950996, -0.008660231, 0.008369712, 0.031569723, -0.02476328, -0.009566373, 0.037850466, -0.0004751195, -0.017043777, -0.028235674, -0.0010289212, -0.007038167, 0.027128935, 0.01202195, 0.024016231, 0.001708528, -0.011357907, 0.021180214, -0.058712494, -0.025261313, -0.009206683, -0.014553615, -0.0036384037, 0.038763527, -0.0007033844, -0.001336733, -0.0068479464, -0.0084250495, 0.016905434, 0.031348377, -0.0032112717, -0.001655785, 4.2772645e-05, -0.017043777, -0.006796068, 0.011648426, -0.018039843, 0.011413244, -0.0099814, 0.004337033, -0.01972762, -0.038099483, -0.029522257, -0.00098396, 0.0031784156, -0.0010877168, 0.035387974, 0.036024347, 0.017915335, -0.0025091844, -0.008985336, 0.005606324, -0.008480386, 0.011683011, -0.016393568, 0.007844011, 0.0003899525, 0.025247479, 0.012298635, 0.0028152668, 0.011122725, -0.024873955, -0.018205853, 0.008791656, 0.033672526, 0.01161384, 0.009144429, 0.02603603, 0.029909616, -0.03740777, 0.02378105, -0.026575565, 0.015784862, 0.0026855709, 0.019589277, 0.014954808, -0.63482535, -0.032316774, -0.006771858, 0.0012753436, -0.0046898057, 0.006841029, -0.0024106153, -0.006581637, -0.006111273, 0.022120941, -0.018219687, 0.0032320232, -0.0048696506, -0.03596901, -0.01862088, -0.027972823, -0.01701611, -0.02205177, 0.021705914, 0.006872156, -0.0250538, -0.0024797865, 0.0073874816, 0.016767092, 0.018302692, 0.013529882, 0.032565787, -0.003417056, 0.0046033417, 0.014954808, -0.01898057, 0.023905559, -0.003856293, -0.012589154, 0.046925724, -0.01989363, 0.024680275, 0.022715814, 0.034806937, -0.0042920713, -0.011067388, -0.0225083, 0.012775916, 0.024873955, -0.016213723, 0.023172343, 0.044075873, 0.010832206, -0.027543962, -0.0060144337, -0.0034066804, 0.0022861073, -0.0034741221, 0.012174127, 0.020848192, -0.028166503, 0.025247479, -0.03646704, 0.011157311, 0.02070985, -0.0069551617, 0.017140618, 0.0081691155, -0.025579501, -0.001869351, 0.011925111, -0.020004304, 0.005257009, 0.016642585, -0.014705792, -0.0050114514, 0.008106861, -0.01120573, -0.012796667, 0.0039946353, 0.039316896, 0.008701734, 0.0031161613, -0.0006670695, 0.0049491976, -0.0041813976, 0.0027149685, -0.015328333, 0.019022074, 0.016711757, 0.0050252858, -0.02800049, 0.005384976, 0.012347055, -0.032289103, 0.007816343, 0.017583312, -0.001188015, -0.011371741, -0.025759345, 0.00803769, -0.001473346, -0.021194048, -0.0043716184, -0.005509484, -0.014048666, 0.0019731077, 0.030047959, -0.001794992, 0.023130842, -0.014290765, -0.023850221, -0.0073390617, 0.027391786, -0.016490407, -0.009690882, -0.01571569, -0.015120819, 0.0052708434, 0.0078094257, -0.035858337, 0.019630779, 0.022909492, 0.004278237, -0.015342167, 0.020613011, 0.004492668, -0.004765894, 0.009697799, 0.016407402, 0.016186055, -0.002829101, -0.007242222, -0.013101021, -0.0014007163, 0.014691957, 0.007532741, 0.022217782, -0.018482538, 0.016448906, 0.020917363, 0.027972823, 0.013972578, 0.012768999, -0.029992621, -0.0261052, -0.00818295, 0.015522012, 0.000490683, -0.007228388, -0.010970549, -0.011046637, -4.890618e-06, 0.006083605, -0.004406204, 0.02025332, 0.028415518, 0.004011928, 0.0072768074, 0.01130257, -0.021014202, -0.024804784, -0.041530374, 0.009683965, -0.00040421906, 0.0027063221, 0.010783786, 0.00038087377, -0.004592966, 0.0029484213, 0.0008179491, 0.015203824, 0.0240439, -0.010970549, -0.014581284, 0.01969995, -0.007304476, -0.020530004, 0.022632807, -0.0045825904, 0.041724052, -0.01370281, 0.011683011, -0.0135575505, -0.010790703, -0.008784739, -0.02051617, 0.008072276, 0.028332513, 0.011489332, -0.008051525, 0.0071661337, -0.004575673, 0.031237703, 0.039510574, -0.003330592, -0.0013211694, 0.015798697, -0.0035260005, -0.0018174726, 0.015342167, 0.015176156, 0.011731431, -0.002488433, 0.010458682, 0.006827195, 0.0026025653, 0.01216721, -0.0214984, 0.021954931, -0.031514388, 0.004433872, -0.036633056, 0.028277176, 0.028208004, 0.020059641, -0.01763865, -0.028360182, 0.014193925, 0.0031023272, 0.0076849177, 0.0016160116, -0.002630234, -0.01682243, 0.01238164, -0.0025351236, 0.008646397, 0.02142923, 0.0065643443, -0.047645107, 0.01456745, 0.005986765, 0.02241146, 0.0051532527, 0.010500184, 0.005416103, 0.01823352, 0.012810501, 0.016421236, 0.031735733, 0.004278237, 0.016573414, -0.014290765, -0.003565774, 0.014719626, -0.006951703, 0.019630779, 0.016988441, -0.012921175, 0.008501138, 0.00393584, 0.015729526, 0.014104002, -0.0077402545, -0.005250092, -0.011129642, 0.0058276714, -0.032233767, 0.0026267753, 0.0015010145, -0.021277053, -0.0026198581, -0.007581161, 0.0088539105, 0.0067026867, 0.009262021, 0.009040672, -0.0043543256, 0.0027184272, 0.013882655, -0.01473346, -0.028664535, 0.006505549, -0.03306382, -0.00950412, -0.022162445, -0.005817296, 0.0057723345, -0.002680383, 0.0033963048, -0.0038908785, 2.3088582e-05, 0.010285754, -0.005450689, -0.013861903, -0.038376167, -0.02874754, 0.0037802048, 0.030684333, 0.015106984, 0.0044788336, -0.019243421, 0.008210618, -0.02721194, 0.007899349, 0.015729526, 0.015480509, 0.015978541, 0.0069344104, -0.022328455, 0.009054507, 0.01900824, -0.007601912, 0.006419085, 0.009600959, 0.0028446645, -0.0026976757, -0.016435072, -0.012132624, 0.025593335, -0.013695893, -0.011793685, -0.0083420435, 0.0050356616, -0.031127028, 0.037739795, -0.0061251074, 0.004586049, 0.0072975587, -0.020267155, -0.0018935609, -0.021595242, 0.03450258, 0.008860827, -0.023504365, -0.005094457, -0.025800848, -0.0072768074, 0.0031680397, 0.043135144, 0.010700781, 0.014069417, 0.018178185, 0.04811547, -0.009289688, -0.017237457, -0.014844134, 0.006796068, -0.01999047, 0.008833159, -0.0043335743, 0.0051082913, 0.004734767, 0.0106523605, 0.0039220056, -0.008051525, -0.008826242, 0.0009078717, 0.013612887, 0.0038597516, -0.010057488, 0.020599175, 0.023006333, 0.011952779, -0.0012822606, 0.044601575, 0.022369958, 0.015757194, -0.01665642, -0.026838416, -0.0018019091, 0.0078094257, -0.0005490462, 0.034142893, -0.00024685464, -0.0042194417, 0.0014024456, 0.004413121, 0.024721777, 0.0039946353, 0.030407649, 0.0027149685, -0.002618129, -0.006128566, 0.013937992, -0.014332267, 0.006747648, -0.019257255, -0.022286953, 0.04866884, -0.0051670866, 0.012948844, -0.028415518, 0.012928093, 0.0019834833, -0.020640679, -0.0005953044, -0.0066646426, -0.020488502, 0.0035450228, -0.008577226, 0.0010159516, 4.2799664e-05, -0.027087431, 0.0026267753, -0.026727742, -0.017721655, -0.028277176, -0.0034464537, -0.008369712, -0.010804538, -0.03635637, 0.0049561146, -0.004115685, -0.00662314, 0.010348008, -0.006771858, 0.004136436, -0.0013021474, -0.014083251, 0.005620158, 0.010686946, -0.041530374, -0.002836018, 0.0153560005, 0.010832206, -0.00931044, -0.01574336, -0.006785692, 0.0033011944, 0.033893876, 0.019782957, -0.0011620758, 0.0020007761, 0.006381041, 0.005149794, -0.0033150285, -0.011980448, -0.003491415, 0.010776869, -0.038071815, -0.0060593947, -0.003769829, -2.4101832e-05, -0.018012173, -0.016988441, 0.0042678616, -0.019284923, -0.0035726912, 0.01626906, -0.010085157, 0.0014128212, 0.018800726, 0.03450258, 0.024099238, 0.009607876, -0.004240193, 0.020585341, 0.004921529, 0.0049734074, 0.00031840356, -0.009822307, -0.00011251124, -0.026838416, 0.022452964, -0.00950412, -0.030490654, -0.028277176, 0.0073874816, -0.011295653, 0.023476698, -0.005384976, 0.0008862557, -0.046925724, -0.0127067445, 0.03204009, 0.013370788, -0.0033254041, -0.036771398, -0.01823352, -0.0041502705, 0.017915335, 0.000716354, 0.003614194, -0.04360551, 0.00662314, 0.010251168, -0.016850099, 0.022286953, -0.0012883132, -0.007242222, 0.00011953644, 0.013377706, -0.041475035, -0.02515064, -0.0065505104, 0.0113371555, 0.024196077, 0.013578301, 0.028830545, -0.014138588, -0.0047382256, 0.029715937, 0.011883608, -0.0004303744, -0.015992375, 0.0056097824, -0.016448906, 0.0068859905, 0.016130717, 0.004890402, -0.008840076, -0.028968887, 0.007761006, 0.027903652, 0.0144982785, -0.009808472, 0.0063118697, -0.01845487, -0.006740731, -0.0074428185, -0.012554568, -0.00957329, -0.009338109, -0.028443187, 0.030628996, -0.017126784, 0.0085149715, -0.0019955884, 0.019589277, -0.014304599, 0.015023979, -0.004077641, 0.01823352, -0.024721777, -0.002792786, -0.03383854, -0.0058899256, 0.025275147, 0.024223745, 0.0007422932, 0.031403713, -0.008459635, 0.020018138, 0.01089446, -0.014802631, -0.011454746, -0.017154451, -0.00079417153, -0.024251414, -0.01691927, -0.014429107, -0.02646489, -0.00034174885, -0.0075465753, -0.024514263, 0.014401439, -0.014145506, -0.026160538, 0.017168285, 0.0111642275, 0.04086633, -0.023836387, 0.0363287, 0.036079686, -0.002203102, -0.019257255, -0.0046828887, -0.019478602, -0.015854033, 0.04623401, 0.01780466, -0.028180337, -0.030822676, -0.0024867037, 0.03577533, -0.016033879, -0.027779143, 0.033589523, -0.0020197984, -0.0045237946, -0.015480509, -0.0052327993, -0.05262543, 0.014968642, -0.019215751, 0.036577716, -0.0021287429, -0.0029328577, -0.0072629736, 0.008736319, 0.0024970793, 0.032814804, 0.02315851, -0.011247233, 0.017223623, -0.010458682, -0.027516292, 0.0056547434, -0.014318434, 0.006944786, 0.0060075168, -0.0017284147, -0.0033150285, 0.0077679227, 0.013640556, 0.005049496, 0.00042583505, 0.03655005, -0.014581284, 0.0040153866, -0.006176986, 0.018703885, 0.0075189066, -0.014996311, 0.0039462154, -0.008141447, -0.013709727, -0.0029570677, 0.032953147, 0.0065020905, 0.015798697, -0.021346224, -0.012782834, -0.032261435, -0.011274901, -0.016435072, -0.004624093, -0.02178892, 0.0077886744, 0.0006735543, -0.0072145537, -0.004375077, -0.006688853, -0.0012226005, -0.011558503, 0.016670253, -0.026160538, 0.009545622, 0.010015986, -0.008833159, -0.03688207, -0.0047174743, 0.015259162, -0.021927262, -0.0015407879, -0.004146812, -0.034640923, -0.0332575, -0.0125269, 0.045708314, -0.020557674, -0.018607046, 0.017776992, -0.013377706, 0.008743237, 0.0031801448, -0.0149271395, 0.03959358, -0.0035381056, 0.010998216, 0.0013791003, -0.0039669666, 0.032399777, -0.004402745, -0.008840076, -0.01089446, -0.014913306, -0.035553984, -0.0020319032, 0.014097085, 0.009746218, -0.01780466, -0.000117482916, -0.012042701, -0.0052708434, -0.017195955, 0.0026717365, -0.030795006, 0.0028014325, 0.013868821, 0.011786768, -0.009587125, -0.0035761497, -0.018205853, -0.010451765, -0.0029726312, -0.012270967, -0.0047243913, -0.024735613, 0.031597394, -0.02035016, -0.028968887, -0.023435194, -0.011724514, -0.025690174, -0.0029069185, -0.004049972, 0.0006307546, 0.025205975, -0.0154390065, 0.008259038, 0.015037813, -0.009538705, -0.0013194402, 0.0052812193, -0.021595242, 0.019976635, 0.007781757, 0.015535845, -0.0016108238, -0.027779143, 0.008217536, 0.038486842, 0.019575443, 0.0028498524, 0.01574336, -0.010147411, 0.0036971993, -0.004720933, 0.005564821, 0.0013574843, 0.005364225, 0.015369835, -0.037878137, -0.01989363, 0.024016231, 0.0014984206, 0.0033945753, -0.011095056, -0.001101551, 0.010458682, 0.019271089, -0.021512235, -0.023075504, 0.013571384, -0.01447061, 0.026630903, 0.004264403, 0.013771981, 0.008577226, 0.0103964275, -0.031874076, -0.0134745445, 0.01662875, -0.019035907, 0.009234352, 0.012955761, 0.0060628536, 0.00940728, -0.018288858, 0.0016384922, -0.019188084, 0.012077287, -0.0010669654, -0.014968642, -0.021996433, 0.020862026, 0.013135606, -0.012111872, -0.049498893, 0.008784739, -0.022701979, -0.003738702, -0.0048558163, 0.016518077, -0.017209789, 0.015784862, 0.018925233, -0.016227558, 0.018178185, 0.009262021, -0.005602865, -0.011655343, -0.0012269237, 0.17320462, 0.001108468, 0.0091928495, 0.033036154, -0.0046967226, 0.0125269, -0.0057896273, 0.0049180705, 0.0046828887, 0.012672159, 0.045957327, 0.0012848546, 0.015010145, 0.006010975, 0.00046906702, 4.4096625e-05, -0.03455792, -0.021705914, -0.016780928, -0.017721655, 0.0016160116, -0.015065482, -0.009054507, 0.0009485097, 0.031044023, -0.011157311, -0.031763405, 0.0017569478, 0.0063844994, -0.015245327, -0.013529882, -0.018634714, 0.013675141, 0.014871802, -0.0003315893, 0.002190997, 0.0005222424, 0.013024932, 0.021678247, -0.0084458, -0.021415396, -0.019354094, -0.008694816, -0.01969995, -0.008321293, 0.007878597, -0.021124877, 0.004091475, -0.012215629, 0.0012986888, -0.02793132, 0.0001726037, -0.0042747785, 0.015757194, 0.010410261, -0.042415764, 0.029079562, 0.027281111, -0.005001076, 0.027184272, -0.012602988, 0.018593213, -0.03002029, 0.010105909, -0.023725713, 0.020723684, -0.0072491392, -0.0055509866, 0.010880626, -0.013903406, 0.0071661337, -0.0062530744, -0.019478602, -0.010514018, -0.01545284, -0.027889818, 0.033008486, 0.013931074, 0.029798942, 0.0424711, -0.001782887, 0.0117660165, -0.01396566, -0.01283817, -0.0022376876, -0.038929537, 0.019381763, -0.000266309, -0.010804538, -0.005402269, -0.014788797, -0.024901623, -0.016808596, 0.013564467, 0.015231493, 0.006197737, -0.01247848, -0.021816589, 0.00782326, -0.010783786, -0.017597146, 0.045652974, 0.014968642, -0.0064709634, -0.022812653, 0.0012355701, -0.020931197, 0.0043335743, 0.023794884, -0.018081345, -0.0005563956, -0.0036314868, 0.006256533, -0.017292794, 0.0080031045, 0.02946692, 0.023047835, -0.0388742, 0.025441159, 0.006505549, -0.002481516, -0.016960772, 0.0014136859, 0.0055682794, -0.0036314868, -0.01907741, -0.03234444, 0.0065574273, 0.0038424588, -0.026921421, -0.016407402, 0.020391662, 0.009268938, -0.0065332176, -0.016061546, -0.007553492, 0.0030400732, -0.01166226, -0.01130257, -0.005336556, -0.015632685, 0.006553969, 0.008452717, 0.010472516, -0.009386528, -0.02639572, 0.030933348, 0.017195955, -0.020004304, -0.02730878, -0.012914258, -0.012609906, -0.026354218, -0.006374124, 0.020654513, 0.0016082298, -0.020308657, -0.029383915, -0.011689928, -0.01783233, -0.04844749, -0.004852358, 0.020806689, -0.0034689344, 0.010181997, -0.019478602, -0.17729956, 0.028636867, 0.0009355401, -0.002044008, 0.030712001, 0.016670253, 0.03107169, 0.0013332744, -0.00292767, 0.0033755533, 0.024652606, 0.026575565, -0.030296974, -0.0113371555, -0.0011957966, 0.018026007, -0.01799834, 0.025302816, 0.04053431, 0.009849975, 0.029826611, -0.027059764, 0.0137304785, -0.013225528, -0.0016505972, 0.02584235, -0.019672282, 0.026049864, -0.009345026, -0.021609075, -0.0059348866, 0.002778952, 0.0031542056, -0.015466674, 0.01823352, 0.013467628, 0.0060248096, 0.0199628, -0.0062738257, 0.018122848, 0.015618851, 0.017776992, 0.0005477492, -0.0070277913, -0.00041826945, 0.016033879, 0.007940851, -0.0063395384, 0.026949089, -0.034225896, -0.005056413, -0.020751353, 0.0139864115, -0.0032977357, 0.026534062, 0.018399533, -0.02593919, 0.0123608885, -0.0021131793, 0.007761006, -0.013751229, 0.001696423, 0.00590376, -0.0154390065, -0.0029328577, -0.032870144, -0.041502707, -0.0062219473, -0.027654635, 0.00037979297, -0.009262021, 0.0026319632, 6.9495414e-05, 0.0019938592, -0.0002384244, 0.01293501, -0.020640679, 0.005613241, 0.017749323, 0.0049111536, -0.020184148, 0.016642585, -0.027460957, 0.00895075, 0.011184979, 0.007110797, -0.002018069, 0.016006209, -0.018275024, 0.0012104956, 0.002116638, -0.0032787137, 0.010970549, -0.024057735, 0.004883485, 0.022203946, -0.008923082, 0.015272995, -0.010610858, 0.0014162798, 0.010417179, -0.021678247, -0.0021633285, 0.050577965, 0.019423265, -0.0016065006, -0.03635637, -0.0041329777, 0.048724174, 0.0005957367, -0.032455117, -0.0029415041, 0.015217658, 0.0041675633, 0.0047693527, 0.000200056, 0.00077039393, -0.030490654, 0.008314375, -0.0055509866, 0.055613626, 0.0008417267, -0.0076157465, 0.041696385, -0.016946938, -0.015591183, -0.10464215, 0.023476698, -0.014608952, 0.0327318, -0.029743604, 0.014166256, 0.0036245696, 0.0033963048, 0.0032579624, 0.023656541, -0.007885514, 0.0014690228, -0.012111872, 0.0021823505, 0.0067683994, 0.015923204, -0.004758977, -0.010590107, -0.013522965, 0.020419331, 0.0021149088, -0.0009459158, 0.0040395968, -0.009511037, 0.00019638128, -0.0045272536, -0.022646643, 0.03143138, -0.0029553385, 0.012402392, 0.013502213, -0.0138204, 0.024735613, -0.015203824, -0.022217782, -0.00020038025, -0.017887665, 0.0015667271, 0.01862088, -0.033036154, 0.011295653, 0.0061078146, 0.014650455, -3.5733938e-06, 0.009919146, -0.022688145, -0.008667149, 0.0007059783, 0.011447829, -0.023670375, -0.03403222, -0.012810501, -0.031846408, 0.0030175925, 0.0332575, -0.04316281, -0.014940973, 0.018593213, -0.021567572, -0.010354925, -0.022674311, 0.01053477, -0.035000615, 0.02070985, -0.0102234995, -0.00772642, -0.0041710217, -0.026949089, 0.023767216, -0.03383854, 0.003534647, -0.0016505972, -0.0106523605, -0.011544669, -0.027862148, -0.009345026, -0.037601452, -0.009144429, 0.008016939, 0.013301617, 0.0020906988, -0.01672559, -0.018510206, -0.0023604664, 0.009870727, 0.005329639, 0.010195831, 0.026575565, 0.016158387, -0.0042851544, 0.0021200965, 0.033672526, -0.010548604, -0.015231493, -0.011904359, 0.027128935, 0.0019765664, -0.0011646696, 0.0076641664, 0.016186055, -0.041143015, 0.0022117484, -0.08820708, 0.047257748, 0.0063983337, -0.026381886, -0.0090199215, -0.02205177, -0.0137304785, 0.0031784156, 0.013177109, 0.03037998, -0.02629888, 0.035858337, -0.033921544, -0.01907741, -0.03292548, -0.017776992, 0.023130842, 0.0020405496, 0.017984506, 0.025358154, -0.0029293993, 0.013101021, 0.011288736, 0.026907587, -0.015784862, 0.017929168, -0.00935886, 0.033202164, -0.01718212, 0.00019800248, 0.014346101, -0.012001199, -0.0031784156, 0.025026131, -0.0018105555, -0.013004181, -0.01610305, 0.008840076, -9.175989e-05, 0.02649256, -0.032095425, -0.023213847, 0.007885514, -0.010583189, 0.0072975587, 0.013101021, -0.010797621, 0.010431013, -0.0046586785, 0.018676218, 0.026838416, 0.00470364, -0.009856893, -0.03270413, 0.021346224, -0.006699228, 0.009144429, 0.003256233, 0.0022117484, -0.0084250495, 0.025551831, 0.009905312, 0.029411584, 0.008245204, -0.018925233, -0.012118789, 0.014069417, 0.006201196, -0.009545622, -0.057993114, -0.02207944, -0.01960311, 0.005070247, 0.015245327, 0.0045549218, 0.008397381, -0.022107108, 0.0225083, 0.0144982785, -0.0044304137, 0.012630656, -0.001900478, -0.024085402, -0.016573414, 0.027972823, 0.0059383456, 0.015563514, 0.019520106, -0.01607538, -0.00078984833, -0.018344196, 0.02277115, 0.021249386, -0.0066196816, -0.02551033, 0.019160416, 0.003918547, -0.014871802, 0.03073967, 0.02476328, 0.005070247, 0.011620757, 0.018482538, -0.029688269, -0.0049941586, 0.0042540273, -0.02910723, -0.03364486, 0.0132186115, -0.005267385, -0.013737395, 0.005564821, -0.0063775824, 0.0072975587, 0.004177939, 0.0013047413, 0.0016047712, -0.0020578424, -0.027654635, 0.054506887, 0.014000245, 0.013038767, 0.01574336, 0.0001903288, 0.0016384922, 0.02430675, 0.037684456, -0.011475498, 0.022535969, -0.01094288, 0.00664735, 0.020640679, -0.026119035, -0.0076849177, -0.0069551617, 0.004599883, 0.024984628, -0.0022446045, -0.027696138, 0.07841244, 0.0025904605, -0.018759223, 0.012907341, -0.035387974, 0.0037559948, 0.015674189, 0.013875738, -0.0017880748, -0.026893754, 0.010188914, -0.0048800264, -0.0011594818, -0.043135144, -0.040727988, 0.017389633, -0.0134745445, 0.010721532, -0.031846408, -0.006944786, 0.01545284, 0.01130257, 0.029881947, 0.027875984, -0.017707821, -0.010029821, 0.015812531, -0.015411338, 0.011378658, -0.06214338, 0.013578301, -0.0022394168, -0.039648917, -0.03309149, 0.01610305, -0.027294945, 0.005713539, -0.023130842, 0.033534184, 0.02740562, 0.0048419824, -0.0031455592, -0.014802631, -0.010431013, 0.011890525, -0.0016843181, -0.04119835, -0.0064640464, -0.016753258], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-12-616548.pdf', 'citation': 'Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.8382479),\n",
      " NodeWithScore(node=TextNode(id_='7504803c-db0a-4e5c-8601-6a9887087f05', embedding=[0.00097649783, -0.013648757, -0.005952963, -0.02813134, 0.0065851663, 0.016867869, -0.027065137, -0.014339055, -0.012452696, -0.0004152038, 0.032095425, 0.024618339, -0.005703499, 0.016061382, -0.0019734995, 0.01700456, 0.020435546, -0.0061340807, 0.0129857985, -0.0013387332, -0.003073875, 0.022294564, 0.004732982, 0.007176362, -0.002185373, 0.016867869, 0.028268032, -0.029826328, -0.036742974, -0.0004562116, 0.0016368941, -0.0019205312, -0.04294882, 0.001327627, -0.027994648, -0.03499331, 0.03387243, -0.00656808, 0.035950158, 0.0068551344, 0.0072105355, 0.015227557, -0.002211003, 0.0069747404, 0.0018248464, -3.1636864e-05, 0.014926833, 0.0067252764, -0.015965696, 0.009213082, 0.037262406, -0.008263068, 0.002185373, 0.019027611, 0.00416229, 0.004842336, 0.023210406, -0.009643664, 0.0067867883, -0.020681592, 0.0051772324, -0.013908473, -0.026422681, -0.018166449, -0.0029252218, -0.013512065, -0.006902977, 0.0034702837, -0.013662427, 0.008597965, 0.020968646, 0.006677434, 5.510421e-05, 0.00314393, 0.017441977, -0.023005366, -0.0090763895, -0.0007851282, 0.010764543, 0.008160549, -0.00379322, -0.00069798663, -0.008809838, 0.0046612183, 0.017701693, -0.013300191, 0.020230506, 0.02793997, -0.010074246, 0.005963215, 0.037645146, 0.030755838, 0.010662024, 0.028350048, -0.008481776, 0.009370279, -0.01662182, 0.030783176, 0.010545835, -0.02977165, -0.013676096, -0.00495169, 0.0021648693, -0.020025468, -0.035403386, 0.012459531, 0.023798184, -0.01943769, -0.006233183, -0.018822573, -0.03163067, 0.022267226, -0.003936747, -0.027898962, 0.028432064, 0.017141253, 0.027502554, -0.030373098, -0.024194593, -0.011899092, 0.013354868, 0.018357817, -0.0063767103, -0.016526137, 0.023538468, 0.009971726, -0.029088188, -0.01570598, 0.004271644, -0.013177168, 0.020080145, 0.0073677315, 0.025930589, 0.0077983136, -0.022896012, 0.022267226, -0.050193526, -0.013006302, -0.00776414, -0.02755723, 0.011126778, 0.036305558, -0.009527475, 0.004165707, -0.023183066, 0.007292551, 0.022855004, 0.018139109, 0.004001676, 0.017441977, 0.012240823, -0.009008043, -0.015760658, 0.00990338, -0.012548381, 0.017701693, -0.008570626, 0.01868588, -0.008967035, -0.042374708, -0.02348379, -0.003314796, 0.018098101, -0.008351918, 0.035622094, 0.026368005, 0.0061204117, 0.008379257, 0.00739507, -0.0055975625, -0.015104533, 0.005942711, -0.017715363, 0.015787996, -0.0048799263, 0.021611102, 0.015610296, 0.0068346304, 0.017141253, -0.018795233, -0.0003494205, 0.023743507, 0.050658282, 0.0091515705, -0.0051943194, -0.008126376, 0.036907006, -0.017893063, 0.0063698757, -0.018644873, 0.008194722, 0.008201556, 0.011270306, -0.000374196, -0.6364408, -0.0484712, -0.0025014747, -0.0062400177, 0.0041896286, 0.0065954183, -0.005570224, -0.010491158, -0.019916113, 0.0249464, -0.011003755, 0.009862372, 0.009780357, -0.016006704, -0.018016087, -0.028404724, -0.008645807, -0.0076069436, 0.021747794, 0.02795364, -0.023757176, 0.011085771, -0.0036018502, 0.017086577, 0.002351113, 0.0005788078, 0.026778083, -0.0045928718, 0.0026125375, 0.017469315, -0.028350048, 0.032450825, -0.0014634653, -0.020941308, 0.047268305, -0.011215629, 0.0059119556, 0.018248463, 0.02035353, 0.0126577355, -0.0091515705, -0.03436452, 0.0072447085, 0.020312522, -0.026326997, 0.017113915, 0.02886948, 0.01515921, -0.022472266, -0.01461244, 0.009930718, 0.013375372, -0.0019547043, 0.01323868, 0.021173686, 0.004777407, 0.02977165, -0.022499604, 0.012445862, 0.00563857, 0.0006381836, 0.009431791, -0.0023203571, -0.017141253, -0.015254895, 0.013395876, -0.032122765, -0.006533907, 0.024632009, -0.026395343, 0.006072569, 0.020886632, 0.0076889596, -0.009835034, -0.007695794, 0.046147425, -0.006243435, 0.010873897, -0.00089619093, 0.011085771, -0.0126645705, 0.0007471106, -0.0076137786, 0.014024662, 0.011844415, 0.005484791, -0.027311184, 0.00039341842, 0.005102052, -0.016676499, 0.0033968114, 0.023893869, -0.008338249, -0.033599045, -0.022362912, -0.0029662296, -0.0010559504, 0.0031644339, 0.017332623, -0.016703837, -0.01701823, -0.017414639, 0.014093008, -0.0062707737, 0.011482179, 0.0029252218, -0.0051328074, -0.0014224575, 0.02497374, -0.008269903, -0.001946161, -0.007832486, -0.021720456, -0.016293759, -0.007545432, -0.033462353, 0.026614051, 0.030783176, -0.012978964, -0.014421071, 0.022690974, 0.004972194, -0.01793407, 0.001956413, 0.01210413, 0.024057899, 0.014598771, -0.013163499, -0.013628254, -0.006298112, 0.0013182294, -0.0060213096, 0.04573735, -0.00859113, 0.020490222, 0.014626109, 0.029552942, -0.008003352, 0.006212679, -0.035294034, -0.025452165, -0.008468107, 0.026244981, 0.001000419, -0.008215226, -0.014257039, 0.0021563258, 0.0011473636, 0.010415977, -0.0023733254, 0.019027611, 0.00568983, -0.006222931, -0.0013720521, 0.0018248464, -0.019177973, -0.029908344, -0.05016619, 0.006773119, -0.013184003, -0.0006347663, 0.018959265, -0.007524928, 0.013395876, -0.011707722, 0.005724003, 0.017852055, 0.012432193, -0.027160821, -0.019957121, 0.010867063, -0.030291082, -0.010258781, 0.025192449, -0.0066125053, 0.034692585, -0.012548381, 0.007600109, -0.02813134, -0.00944546, -0.002954269, -0.025561519, 0.00022896013, 0.02070893, 0.024167255, -0.0031712686, 0.017893063, -0.014940502, 0.02439963, 0.01589735, -0.0126030585, -0.008844012, 0.0089396965, 0.002351113, -0.0013635088, 0.012145138, 0.021665778, 0.017154923, 0.0062912772, 0.015787996, 0.011242967, -0.012022115, -0.009773522, -0.027270176, 0.019013941, -0.04423373, -0.007565936, -0.03351703, 0.029525604, 0.03868401, 0.010826055, -0.014940502, -0.021515418, 0.016703837, 0.00036736138, 0.010798716, -0.010163096, -0.0022485934, -0.00029944224, 0.00656808, -0.008768831, 0.0064963163, 0.016143396, 0.02035353, -0.023210406, 0.015104533, 0.0066705993, 0.016239082, -0.016198074, -0.0067252764, 0.0069405674, 0.025370149, 0.023005366, 0.003618937, 0.031193255, 0.006902977, 0.010887566, -0.0023852861, 0.022267226, 0.013142995, -0.028650772, 0.016731177, 0.025247125, -0.005809436, -0.0022229636, 0.019847767, 0.014776471, 0.010846559, -0.0076684556, 0.01664916, -0.01869955, 0.006803875, -0.024139915, 0.0020623498, 0.008160549, -0.01869955, -0.0063869623, 0.015186549, 0.0277486, 0.006212679, -0.002070893, -0.0090900585, 0.0092267515, 0.0013652175, 0.011953769, -0.0077914787, -0.017592339, -0.015610296, -0.033407673, 0.0049585244, -0.03496597, 0.006879056, 0.002670632, 0.014270709, 0.008584296, 0.0009995648, 0.0075932746, 0.009329271, 0.007900833, -0.015746988, -0.028787464, -0.023237744, 0.0090285465, 0.032122765, 0.02275932, 0.025438495, -0.017660685, -0.016512467, -0.021324048, -0.007196866, 0.004920934, 0.025711881, 0.0054130275, 0.010286119, -0.01081922, 0.0128696095, 0.0029030093, -0.025780227, 0.0086048, 0.029197542, 0.013997324, -0.0129652945, -0.021611102, -0.007470251, 0.019888775, -0.022896012, 0.009315602, -0.006978158, 0.0026962617, -0.029088188, 0.032696873, -0.0004895304, -0.008010187, 0.0011934973, -0.019601721, 0.007709463, -0.031111239, 0.017633347, 0.019683735, -0.01647146, 0.0041383686, -0.042374708, -0.009876041, 0.012357012, 0.026928445, 0.02941625, 0.011892257, 0.0054164445, 0.038820703, 0.006246852, -0.016676499, -0.0091379015, 0.020134822, -0.028186016, -0.006872221, -0.0046441313, 0.0027030965, -0.0040700226, -0.002894466, 0.0027167657, -0.009773522, -0.014776471, -0.0017411221, 0.010791882, -0.003847897, -0.009561649, 0.027803278, 0.006663765, 0.0033113784, -0.02349746, 0.02832271, 0.025985265, 0.013771781, -0.00867998, -0.021775132, -0.01313616, 0.00850228, 0.010586843, 0.02979899, 0.0010046907, -0.0128559405, -0.006609088, -0.0029218046, 0.019506035, 0.026491027, 0.024385963, 0.004702226, -0.001332753, -0.008447603, -0.0003171696, 0.00090815156, 0.01850818, -0.020093814, -0.017428309, 0.04051569, -0.008290407, 0.005659074, -0.0207226, 0.012151973, -0.005061044, -0.024290277, -0.0021922078, -0.019150635, -0.01359408, -0.012931121, -0.018726887, -0.0019427438, 0.00012313014, -0.028459402, 0.0124732, -0.017305285, -0.015063526, -0.024372293, -0.0021682866, -0.008276737, -0.01924632, -0.03545806, -0.0014822605, -0.00033831422, 0.00016659412, 0.016034042, -0.016498799, -0.011728226, 0.007340393, -0.02295069, -0.0030414106, 0.0011764108, -0.036524266, 0.029142864, 0.016129728, 0.014885825, -0.0072993855, -0.035895478, 0.0029218046, 0.014024662, 0.013129326, 0.010600512, 0.006598836, -0.01515921, -0.0091037275, 0.010791882, 0.0052079884, -0.010504827, 0.00046005606, -0.0016949883, -0.033599045, -0.013177168, -0.00084920286, 0.005016619, -0.011372825, 0.00034365378, 0.00823573, -0.022991696, 0.00647923, 0.024344955, -0.01118829, 0.008789334, 0.007880329, 0.035649434, 0.03884804, 0.024878055, -0.0053480985, 0.022663634, -0.0068961424, 0.006048648, 0.013231845, 0.0067355284, 0.013600915, -0.015036187, 0.022991696, 0.00591879, -0.014298047, -0.04084375, -0.010456985, -0.017318955, 0.021255702, 0.011653045, -0.0010986668, -0.043413572, -0.005324177, 0.018603865, 0.016020374, 0.0019000273, -0.04267543, -0.01662182, 0.0021717038, -0.00067705556, -0.014817479, 0.00081844704, -0.039312795, -0.0045586987, 0.0054266965, 0.010682528, 0.015993036, 0.0077026286, 0.0005074713, -0.00867998, 0.018549187, -0.027270176, -0.03567677, 0.00079068134, -0.0021050663, 0.026737075, 0.023155728, 0.027816946, -0.0017359961, 0.022089526, 0.022731982, 0.0090558855, 0.0031986071, -0.012076792, -0.0074087395, -0.013908473, 0.012521043, 0.0029423086, -0.016170736, -0.013682931, -0.008174218, -0.0031165916, 0.022157872, 0.008351918, 0.0006864532, -0.003718039, -0.030017698, -0.029826328, 0.0043878327, -0.0026415847, 0.021898156, -0.042757448, -0.03422783, 0.022362912, -0.008208391, 0.00953431, -0.0041793766, 0.020790946, -0.0103613, 0.023073712, -0.0076274476, 0.010949078, -0.013471057, -0.0074497475, -0.023647822, -0.01165988, 0.028896818, 0.015596627, -0.0022673886, 0.023032704, -0.007032835, 0.008987539, 0.0006351935, -0.0063220332, -0.01248687, -0.024454309, -0.019109627, -0.018275803, -0.029170204, 0.027311184, -0.024317615, -0.003434402, -0.014229701, -0.01793407, 0.009582152, -0.0121178, -0.014120347, 0.0069303154, 0.025370149, 0.029334234, -0.019929783, 0.025001079, 0.037453774, 0.008912358, -0.024591, 0.012145138, -0.020449216, -0.006181923, 0.031712685, 0.024864387, -0.012876444, -0.033380337, 0.020886632, 0.030373098, -0.014585102, -0.020941308, 0.02368883, 0.016225412, -0.0003178103, -0.032286793, -0.01027245, -0.020872962, 0.0048799263, 0.013580411, 0.021392394, -0.010648354, 0.0044083367, -0.014913164, 0.003618937, -0.0063903797, 0.026067281, 0.0035266695, -0.016430452, 0.0051088864, -0.029142864, -0.019670067, 0.010327127, 0.002091397, 0.011598368, -0.0007778664, 0.008891854, 0.020080145, 0.0042887307, -0.00015142978, -0.002884214, 0.004606541, 0.035758786, 0.0016966971, 0.0029354738, -0.024905393, 0.020476554, 0.012288665, -0.01719593, 0.0035164175, -0.004380998, -0.00379322, -0.012445862, 0.031357285, 0.0038854876, 0.016020374, -0.022431258, -0.012411689, -0.011694053, -0.01406567, -0.03996892, 0.008003352, -0.03608685, 0.013669261, 0.00693715, 0.008338249, 0.01905495, 0.0051430594, 0.003943582, -0.015090864, 0.03239615, -0.023183066, -0.00034258584, 0.004845753, -0.020763608, -0.024153585, 0.00076120696, -0.0012276705, -0.004876509, 0.004216967, -0.0027834033, -0.010573174, -0.030291082, -0.021036994, 0.018098101, 0.005324177, -0.03089253, 0.014585102, -0.00266038, 0.018836241, -0.0128354365, -0.010067411, 0.028104002, -0.009274594, 0.004808163, -0.0016360397, -0.004808163, 0.038191915, 0.005310508, -0.019095957, -0.014899495, -0.009691506, -0.046393473, -0.0016206618, 0.009588987, 0.012534712, -0.013423215, 0.0068107094, -0.009500137, -0.011174621, -0.014530425, -0.0047158953, -0.007921337, -0.008837177, -0.006762867, -0.0017940905, 0.0063425372, -0.002395538, -0.016854199, 0.0090285465, -0.0049892804, -0.026969451, 0.006069152, -0.018972935, 0.026737075, -0.039941583, -0.020613246, -0.034173153, -0.009240421, -0.0373991, -0.001644583, -0.0091925785, -0.004797911, 0.016731177, -0.0073677315, -0.004886761, 0.015227557, -0.013259184, 0.011167786, 0.000662532, -0.0035164175, 0.007401905, 0.0042887307, 0.00065954187, 0.0014609023, -0.031220593, -0.005211406, 0.03436452, 0.03204075, 0.002540774, 0.021173686, -0.008153714, 0.009753018, -0.001657398, -0.007969179, -0.013901639, 0.011523187, 0.0063322852, -0.050084174, -0.012671405, 0.018070763, -0.010853393, -0.01257572, -0.005898286, -7.3525676e-05, -0.0019991295, 0.030974546, -0.0048047453, -0.021406064, 0.009889711, -0.0069849924, 0.032997597, 0.0045689507, 0.0034702837, 0.018535517, 0.016375775, -0.026491027, -0.044698484, 0.019547043, -0.0091515705, -0.0014104969, -0.0012960169, -0.01009475, 0.014462078, -0.013019972, -0.006181923, 0.0073540625, 0.008351918, -0.0006928607, -0.001831681, -0.006752615, 0.008440768, 0.0310839, -0.0381099, -0.04696758, 0.0022776406, -0.026805421, -0.010949078, -0.009240421, -0.00040986424, -0.02628599, 0.021788802, -0.00040409752, -0.01257572, 0.0063527892, -0.0042784787, -0.0074839205, -0.004206715, -0.014120347, 0.17070173, -0.02147441, 0.014858487, 0.03204075, -0.0025783645, -0.008823508, 0.005744507, -0.0053788545, -0.009411287, 0.013006302, 0.042046648, 0.00832458, 0.010245111, 0.003923078, 0.011133613, -0.0041349516, -0.031220593, -0.014284378, -0.012439027, -0.022909682, 0.004336573, -0.0053788545, -0.011625706, -0.0038000545, 0.022321904, -0.01979309, -0.033188965, 0.014503086, 0.027912632, -0.015555619, -0.030263744, 0.0016676498, 0.016225412, 0.0063186157, -0.012042619, 0.0021272788, 0.009219917, 0.024194593, 0.014284378, -0.00037013798, 0.0023681994, -0.017428309, -0.007121685, -0.02515144, -0.024003223, 0.0018573109, -0.030017698, -0.0022332156, -0.020503892, -0.004982446, -0.03165801, 0.01682686, -0.019601721, 0.016908877, -0.006127246, -0.031958733, 0.011249802, 0.022786658, 0.009739349, 0.03242349, -0.011967438, 0.025465833, -0.017059239, 0.020886632, -0.009411287, 0.00786666, -0.010709866, -0.009575318, 0.022253556, -0.0054027755, 0.010005899, -0.0016949883, -0.002211003, -0.026409013, -0.023524798, -0.021201024, 0.022335572, 0.011372825, 0.03567677, 0.048744585, 0.002720183, 0.006868804, -0.021788802, -0.0025834905, -0.0015283943, -0.026203973, 0.042784788, 0.0028278283, -0.019765751, -0.008379257, -0.014202362, 0.0068414654, -0.022226218, -0.0072652125, 0.014134016, -0.0035300867, -0.009459129, -0.0069200634, 0.024303947, -0.00758644, -0.025766557, 0.06440956, -0.00795551, -0.0005211406, -0.016717507, 0.023702499, -0.025438495, 0.012261327, 0.014298047, -0.016785853, 0.005471122, -0.006728694, -0.0026689232, -0.025192449, 0.025711881, 0.038082562, 0.030619144, -0.015651304, 0.031357285, -0.02681909, -0.017660685, -0.006171671, -0.0050747134, 0.023073712, -0.019547043, -0.01515921, -0.008850846, -0.010299788, -0.0089191925, -0.025042087, -0.0010858519, 0.015842674, 0.0070191654, 0.0025612777, -0.007545432, -0.00018560294, 0.0070260004, -0.020982316, -0.010955913, 0.011782903, -0.015213887, 0.0075932746, 0.027434207, -0.006947402, 0.00859113, -0.019888775, 0.032696873, 0.007360897, -0.0042032977, -0.022786658, -0.001384867, -0.029443588, -0.012316004, 0.0056180665, 0.01628009, -0.025233457, -0.01570598, -0.043878328, -0.016362106, -0.008515949, -0.0418006, 0.006923481, 0.034801938, -0.0015002014, 0.00065099855, -0.0031986071, -0.17452912, 0.017223269, -0.0089191925, 0.00038850604, 0.022813996, 0.017674355, 0.0024775537, -0.009794026, 0.011318148, -0.0027082223, 0.03942215, 0.025766557, -0.042429388, -0.023880199, 0.0024997662, 0.025561519, -0.015186549, 0.017428309, 0.03124793, 0.009910215, 0.034911294, -0.015610296, 0.003359221, -0.032478165, 0.0070943465, 0.042019308, -0.013737608, 0.023552136, -0.031357285, -0.013566742, -0.020476554, 0.0011020842, 0.0125825545, -0.002200751, 0.0093224365, 0.0010790173, 0.0054369485, 0.0038103065, -0.015938358, 0.017988747, 0.0074634166, 0.031220593, 0.008454438, -0.014571432, -0.00481158, 0.020613246, 0.008174218, -0.0044425097, 0.019601721, -0.02813134, -0.004175959, -0.012548381, 0.007579605, 0.014639779, 0.017961409, 0.00023686266, -0.015186549, 0.012110965, 0.010306623, 0.0021580346, -0.0044527617, -0.03961352, 0.007914502, -0.021242032, -0.014229701, -0.02979899, -0.022896012, -0.0029628123, -0.017852055, 0.007716298, -0.0058162706, -0.0067765363, 0.0011251511, -0.012541547, 0.011229298, 0.0010286119, -0.005754759, 0.015746988, 0.009698341, -0.009629995, -0.013115656, 0.015145541, -0.031958733, -0.013580411, 0.015036187, 0.006503151, -0.01304731, 0.0009935844, 0.00226568, 0.0076274476, 0.011741895, -0.004107613, 0.011577864, -0.0042477227, 0.010580008, 0.021542756, 0.0056727435, 0.020804616, -0.011085771, 0.00046518203, 0.02591692, -0.016963553, -0.022472266, 0.052025206, 0.02979899, 0.015610296, -0.02922488, 0.002005964, 0.05735622, -0.006544159, -0.021255702, -0.00035091556, 0.012452696, 0.020640584, 0.0043434077, 0.025411157, -0.008078533, -0.034528553, 0.006793623, -0.014571432, 0.045764685, -0.009178909, -0.0060896557, 0.03365372, -0.032505505, -0.019383011, -0.11274406, 0.008556957, -0.015829004, 0.022417588, -0.015118203, 0.002795364, 0.004336573, 0.010497993, -0.0021819558, 0.020380868, 0.009288263, -0.021050662, -0.013320695, -0.00813321, 0.0011157534, 0.016143396, -0.006509986, -0.010696197, -0.011201959, 0.030947207, 0.01027245, -0.008905523, 0.016498799, -0.00776414, 0.014858487, 0.0008389509, -0.02091397, 0.020667924, -0.003697535, 0.014557763, 0.02422193, -0.024659347, 0.011427502, -0.011345487, -0.01663549, -0.011174621, -0.0012114382, -0.0058026016, 0.021009654, -0.02331976, 0.012240823, 0.012172477, 0.008290407, 3.914561e-06, 0.02165211, -0.03493863, -0.0129857985, 0.018016087, 0.0001933987, -0.024495317, -0.025356479, -0.017961409, -0.03961352, -0.015993036, 0.026955783, -0.024331285, -0.016717507, 0.021050662, -0.012596224, -0.02627232, -0.0045108562, 0.0016403113, -0.035430726, 0.008556957, 0.001449796, -0.005765011, -0.030291082, -0.027816946, 0.023962215, -0.04267543, 0.02053123, -0.00079110847, -0.014120347, 0.0071831965, -0.024440639, -0.0072447085, -0.02070893, -0.010026403, 0.016430452, 0.0022246724, 0.006028144, -0.01962906, -0.011242967, -0.03165801, 0.017523993, 0.019492367, -0.004186211, 0.022472266, 0.00076761446, -0.01591102, -0.0037658815, 0.028596094, -0.008386091, -0.017168593, -0.010764543, 0.023743507, -0.022513272, -0.021337716, 0.022362912, 0.020230506, -0.020654254, -0.013682931, -0.06998662, 0.01663549, 0.00730622, -0.033079613, -0.007162693, -0.011584699, 0.01589735, 0.0050439574, -0.00044211515, 0.023811853, -0.020599576, 0.040050935, -0.029306896, -0.013710269, -0.018303141, -0.0072447085, 0.014298047, 0.002844915, 0.026203973, 0.020599576, -0.012240823, 0.0020948143, 0.0060144747, 0.022267226, -0.0067321113, 0.03573145, -0.016498799, 0.038027883, -0.034173153, -0.013894804, 0.011448006, -0.020271514, -0.009547979, 0.01480381, 0.0027236002, -0.007053339, -0.01610239, 0.016047712, 0.01515921, 0.013081483, -0.03050979, -0.021925494, -0.0008876476, -0.031220593, 0.0020743103, 0.009930718, -0.01341638, 0.0050644614, 0.0003829529, 0.006513403, 0.028240694, 0.003408772, -0.0070260004, -0.029525604, 0.008017021, -0.0050781304, -0.0016351853, -0.0028773795, 0.007436078, -0.012138304, 0.00867998, 0.011905926, 0.03362638, 0.021529086, -0.022403918, -0.01999813, -0.011249802, -0.016485129, 0.0031405129, -0.051724482, -0.0073198895, -0.013682931, 0.0004476683, 0.012480035, 0.005590728, 0.0069849924, -0.003946999, 0.0022981446, 0.013067814, 0.01812544, 0.0034480712, 0.007456582, -0.010142592, 0.016403113, 0.034337185, 0.0021973336, 0.0039709206, 0.009363444, -0.015063526, -0.005214823, -0.019588051, 0.01906862, 0.016676499, 0.006674017, -0.0063288677, 0.032724213, -0.01850818, 0.0034070634, 0.027461546, 0.012363846, -0.0039504166, 0.016034042, 0.0005536051, -0.016225412, -0.010627851, 0.013833292, -0.020763608, -0.0321501, 0.008816673, 0.0061409157, -0.01406567, 0.0068448824, -0.0207226, 0.0049790284, -0.017606009, 0.018767895, -0.0087825, -0.0041349516, -0.0127397515, 0.03387243, 0.010491158, -0.0007791479, 0.007989683, -0.013498396, 0.03570411, 0.009978561, 0.052271254, -0.01889092, 0.021214694, -0.011885422, 0.0048389183, 0.022485934, -0.040816415, -0.016006704, -0.008775665, 0.014120347, 0.012876444, 0.002016216, -0.0347746, 0.08464006, 0.010415977, -0.005529216, 0.013436884, -0.027680254, 0.012158807, 0.020804616, 0.020298854, 0.012124634, -0.030564468, 0.013662427, -0.0068756384, 0.009506971, -0.047760397, -0.027529892, 0.016949885, 0.017633347, -0.0026381675, -0.01887725, -0.0021494913, 0.018043425, 0.010286119, 0.03534871, 0.01700456, -0.026203973, 0.023333428, 0.028541418, -0.0030209068, 0.008488611, -0.05221658, -0.0059666326, 0.0013267726, -0.024987409, -0.015213887, 0.015241226, -0.004425423, -0.016731177, -0.027516223, 0.062058445, 0.010491158, 0.024249269, -0.010709866, -0.019218981, -0.022827666, -0.0062946947, -0.009575318, -0.0270378, 0.0041998806, -0.01664916], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-10-00627.pdf', 'citation': 'Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.82634664),\n",
      " NodeWithScore(node=TextNode(id_='bc468ca4-0117-49c8-b76f-169a551c43ee', embedding=[-0.012636852, -0.027198661, -0.016853753, 0.0041234223, 0.013689347, 0.02095294, -0.00021400423, -0.0013433148, -0.01469337, -0.017684668, 0.0037598964, 0.0051170597, 0.0007309467, 0.007360534, -0.00988098, 0.018903347, 0.01161898, -0.037363537, 0.014887251, -0.02114682, 0.0014800697, 0.025661465, -0.007073175, -0.014568733, 0.004005709, 0.022323951, 0.020842152, -0.03237804, -0.025384493, -0.008821562, 0.0022244323, 0.0016852021, -0.016064383, 0.00709741, -0.023861147, -0.030854693, 0.0213407, -0.016826056, 0.041047268, 0.004601199, 0.024567425, -0.005605223, 0.00135976, 0.0075821113, -0.0005193227, -0.015316557, 0.02064827, -0.012442972, -0.015635075, 0.020980638, 0.020911394, -0.0046288962, 0.0039260797, 0.010261817, -0.0026606633, -0.0054078805, 0.018737163, 0.008115282, -0.010871155, 0.0016168247, -0.009513992, -0.010116406, -0.021631522, -0.010691123, -0.00015157732, -0.004756996, -0.013287737, -0.0013216763, 0.0016566394, 0.011224295, 0.017033784, 0.011528964, -0.009735569, 0.008523816, 0.013308509, -0.036117163, -0.016369052, 0.014998039, 0.011757466, -0.0006491534, 0.008399179, -0.0017111683, -0.004088801, -0.0071666534, 0.023598023, -0.0012602231, 0.0046808287, 0.025855346, -0.009389355, -0.012983068, 0.0347323, 0.019914294, 0.009126231, 0.0026364282, -0.015621227, 0.019318804, 0.0018574442, 0.011930574, 0.0057125497, -0.031713307, -0.022503983, -0.008468422, -0.011902876, -0.012996916, -0.02541219, 0.023999633, -0.0023386832, -0.015219618, 0.013377753, -0.010102558, -0.0140078645, 0.014928796, -0.00094343623, -0.06525463, 0.042155154, 0.005224386, 0.02591074, -0.0075405654, -0.008018343, -0.000111546186, 0.018058581, 0.03315356, -0.0013329283, -0.018944891, 0.0262985, 0.0046358206, -0.023653418, -0.04525724, -0.007066251, 0.00029168624, 0.04337383, 0.023501083, 0.016382901, 0.017006088, -0.02391654, 0.01757388, -0.038886882, -0.019388048, 0.004736223, -0.023182565, 0.0041372706, 0.04340153, -0.01945729, -0.009396278, -0.021354549, 0.00667849, 0.012629928, 0.02513522, 0.010566486, -0.003360018, -0.0045007966, -0.012768414, -0.0006569432, 0.010677274, 0.01152204, -0.008413028, -0.017933944, 0.006273418, -0.030494628, -0.021091426, -0.026893992, 0.014471793, 0.014139426, -0.004919717, 0.036394134, 0.021728462, 0.0048089284, 0.014333307, -0.0033496313, -0.0055982987, -0.027877243, 0.01965117, -0.019277258, -0.00046133166, 0.012235243, 0.018501736, 0.024858246, 0.0031090118, 0.017933944, -0.0144163985, -0.0039883982, 0.00933396, 0.047334533, 0.010628805, -0.022171617, 0.003957239, 0.023390295, -0.027641816, 0.00543904, -0.011002717, 0.02103603, 0.0019336115, 0.004687753, -0.012872279, -0.6385869, -0.028860493, -0.010711896, -0.0069762347, -0.011106581, 0.017116876, -0.012505291, 0.00042649376, 0.003064004, 0.024872094, -0.014748764, 0.009015442, -0.0030605418, -0.025744557, -0.034704603, -0.02550913, 0.010677274, -0.012803036, 0.0158982, 0.001454969, -0.023598023, 0.016050534, -0.0014592968, -0.003112474, 0.0037806693, -0.00039814741, 0.036394134, -0.009680175, 0.0012403157, -0.00047994073, -0.030079171, 0.016105928, 0.0046600555, -0.013218494, 0.04201667, -0.010587259, 0.023611872, 0.008136055, 0.026533928, -0.013287737, -0.008572287, -0.022587076, 0.012934597, 0.01876486, -0.012387577, 0.020094326, 0.029968383, -0.01235988, -0.0060310676, 0.003043231, -0.022407044, 0.022116223, -0.018141674, -0.006491534, 0.015912047, -0.00056433066, 0.0253291, -0.02969141, 0.0025342947, -0.005075514, 0.0066923387, 0.0087246215, 0.003891458, -0.004251522, 0.01509498, 0.014125577, -0.014734916, -0.007069713, 0.028777402, -0.011916725, -0.0036525696, 0.017061481, 0.011459721, -0.0055671395, -0.0072358963, 0.04329074, 0.0013346593, 0.02174231, -0.011418175, 0.013883227, -0.007305139, -0.009354733, -0.011556661, 0.026741657, 0.025979983, 0.009098534, -0.022614773, 0.010864231, -0.004351924, -0.023279505, 0.017740063, -0.00031527216, -0.020052781, -0.027018629, -0.018404797, -0.012643777, -0.016105928, -0.0067373463, 0.017089179, -0.012103681, -0.008821562, -0.023224112, 0.017407697, -0.019706566, -0.0013813984, -0.0073397607, 0.024581274, 0.0058406494, 0.027988032, -0.013315434, 0.0011511653, 0.009181625, -0.02391654, 0.00938243, -0.007983721, -0.03686499, 0.017546182, 0.018806405, -0.007859084, -0.010213346, 0.022227012, 0.0004747475, -0.01668757, -0.012075984, 0.021326853, 0.006702725, 0.017712366, -0.025758406, 0.005764482, -0.016133625, 0.012304486, -0.005979135, 0.03187949, -0.0051586055, 0.014250215, 0.020025084, 0.028777402, 0.002792225, 0.00883541, -0.023778055, -0.026852446, 0.00484355, 0.029442135, 0.011951346, -0.002286751, -0.01895874, -0.0047500716, 0.004961263, 0.00494049, -0.011058112, 0.007374382, 0.02462282, -0.008295314, -0.0022954063, 0.0068723704, -0.020468239, -0.039246947, -0.047500715, 0.0011710727, -0.012719944, 0.014444095, 0.023293354, -0.0024373545, -0.0023023307, 0.017241513, -0.0022071216, 0.013433147, -0.0025983446, -0.026201561, -0.03395678, 0.0125122145, -0.02204698, -0.016673721, 0.02402733, -0.0021932728, 0.03852682, -0.023307202, 0.0122283185, -0.0054355776, -0.0035902509, -0.00992945, -0.018460192, 0.0063080397, 0.009043139, 0.029026676, -0.009860207, 0.008371482, -0.0035019661, 0.028666614, 0.0357294, -0.017255362, 0.0081914505, 0.0019820817, -0.0026000757, -0.0047950796, -0.0087592425, 0.01350239, 0.014679521, 0.008246845, 0.026173864, 0.006266494, 0.0023317588, -0.0013770707, -0.014250215, 0.020274358, -0.039856285, -0.0004412079, -0.030466931, 0.018141674, 0.02442894, 0.014998039, 0.0017276135, -0.017989337, 0.009063913, 0.006242259, -0.0011182749, 0.009673251, 0.015247314, -0.01699224, 0.00036936827, -0.013446996, 0.016798358, 0.01728306, 0.018681768, -0.02024666, -6.12368e-05, -0.006633482, 0.03118706, 0.0012160806, 0.006470761, -0.0010914432, 0.022891745, 0.011563585, 0.019027984, 0.024816701, 0.01449949, 0.027904939, -0.020994486, 0.012969219, 0.00080538297, -0.012373729, 0.029497528, 0.0233349, -9.942866e-05, 0.0018020498, -0.012893052, 0.010047163, 0.020052781, 0.0036006374, 0.011418175, -0.017809307, 0.0026797052, -0.011591283, 0.0069416133, 0.01678451, -0.017837003, -0.018834103, 0.00019420504, 0.02303023, 0.020925242, 0.0033911772, 0.012075984, -0.011487418, -0.010905777, 0.022060828, -0.008177602, -0.024055028, -0.006169554, -0.015178071, 0.014180972, -0.018834103, -0.00140823, 0.0016635637, -0.007852159, -0.0032700018, -0.008343785, 0.0037148884, 0.019928142, 0.0012991723, -0.00040182594, -0.040770292, -0.026880143, -0.0017345378, 0.023694964, 0.009250868, 0.01815552, -0.008856183, -0.005193227, -0.011092733, 0.018931044, 0.021022182, 0.03525855, 0.009673251, 0.015551983, -0.017726215, 0.014457944, 0.027821848, -0.020135872, 0.009520916, 0.014831856, 0.008544589, 0.0041061114, -0.016313657, 0.0034638825, 0.035590917, -0.022282407, -0.0018193605, -0.007346685, -0.00883541, -0.030660812, 0.011348932, -0.005580988, 0.00424806, 0.008530741, -0.017061481, -0.001992468, -0.016119776, 0.01321157, 0.024650518, -0.014174048, 0.007782916, -0.038055968, -0.017504636, -0.004376159, 0.048996367, 0.032350343, -0.0067823543, 0.016798358, 0.04262601, 0.0054494264, -0.022019282, -0.0058441116, 0.010677274, -0.023473386, -0.0053663347, -0.010275665, 0.042404428, -0.0015302709, -0.0059618247, 0.003084777, 0.006384207, -0.011009641, -0.0079421755, 0.013571633, 0.009410127, -0.0019820817, 0.014956494, 0.02422121, 0.010808837, -0.019484987, 0.04290298, 0.028777402, 0.008073737, -0.020675968, -0.011501267, -0.0011156782, 0.0022780956, 0.018404797, 0.01176439, -0.0056086853, -0.0010170069, 0.015261163, 0.0050720517, 0.028860493, 0.013246191, 0.016922995, 0.0014298685, -0.003527932, -0.010310287, 0.01007486, -0.0042826813, 0.018044733, -0.014804159, -0.011632828, 0.039939377, -0.003477731, 0.02600768, -0.005965287, 0.010109482, 0.019221865, -0.010324135, 0.0011641483, -0.02591074, -0.027364844, -0.0033981015, -0.020412844, 0.0016999163, -0.011992892, -0.03326435, -0.000960747, -0.015219618, -0.025190612, -0.02114682, -0.0018003187, -0.0034846554, -0.025204461, -0.023182565, 0.011556661, 0.012816885, -0.01057341, 0.021229912, 0.010746518, 0.006702725, 0.013481617, -0.01459643, -0.013073083, -0.00092785654, -0.036781896, -0.007249745, 0.014651825, 0.0059860596, -0.0028406952, -0.026035378, 0.01569047, 0.011459721, 0.019512685, 0.016022837, -0.00048123903, 0.0048954817, -0.010601107, 0.0051343706, -0.0028095357, -0.023584174, -0.016742963, 0.0111689, -0.03127015, 0.0016384631, -0.0027610657, 0.000172783, 0.0016090348, 0.00042519547, 0.009160852, -0.028237306, -0.0043138405, 0.0034136812, -0.017089179, 0.008579211, 0.027808, 0.045645002, 0.034510724, 0.029996078, -0.0154827405, 0.019886598, -0.0020720975, -0.002838964, 0.008413028, 0.0026018068, 0.008980821, -0.01797549, 0.024581274, -0.006861984, -0.042542916, -0.025841497, 0.006245721, -0.015579681, 0.024872094, 0.007505944, 0.0048850956, -0.03833294, -0.021673067, 0.015247314, 0.026201561, -0.012754565, -0.033596717, -0.010961171, -0.012235243, 0.01577356, 0.007298215, -0.0100402385, -0.03315356, 0.009299339, 0.022157768, 0.009659402, 0.0144163985, 0.010234119, -0.016812207, -0.0060933866, 0.0049301037, -0.032599617, -0.017712366, -0.009541689, 0.012505291, 0.031020876, 0.028348096, 0.026367744, -0.024442788, -0.0069658486, 0.036698803, 0.014444095, 0.0041961274, -0.015745863, -0.0119444225, -0.029331347, 0.006491534, 0.024941338, -0.01152204, -0.0044938726, -0.02145149, 0.010580335, 0.021409944, -0.005328251, 0.00019323132, -0.013696271, -0.0288051, -0.021285307, -0.0054528886, -0.013897075, 0.01295537, -0.02116067, -0.041684303, 0.013398525, -0.016673721, -0.00014962985, -0.006868908, 0.011099657, -0.014430246, 0.030273052, -0.0069935457, 0.013537012, -0.026575474, 0.009202398, -0.035230853, 0.0057506333, 0.023362597, 0.015496589, 0.0065053822, 0.012740717, -0.0039918604, 0.00883541, 0.0046565933, -0.012387577, -0.015053434, -0.018127825, -0.010885004, -0.022850199, -0.03187949, 0.010704972, -0.040105563, 0.0010014273, -0.0053628725, -0.023861147, 0.0020946015, -0.016521387, -0.014956494, 0.011335083, 0.023958087, 0.048525512, -0.01936035, 0.01414635, 0.04550652, 0.009015442, -0.030023776, -0.007073175, -0.008046039, -0.0023629183, 0.03356902, 0.0268109, -0.015912047, -0.029829895, 0.013640876, 0.01235988, -0.0152334655, -0.035618614, 0.030356143, -0.00022439068, -0.004916255, -0.01037953, -0.008655379, -0.029165164, 0.03210107, 0.007526717, 0.028777402, -0.010400303, -0.006152243, -0.0050443546, 0.0017232859, 0.0118197845, 0.021271458, 0.00814298, -0.041961275, 0.0028857032, -0.022864047, -0.015884351, -0.0046185097, -0.008419952, 0.022573227, 0.00048210457, -0.013897075, 0.02224086, -0.016147474, -0.0051412946, 0.0012108875, -0.009423976, 0.03487079, -0.024553576, 0.012138302, -0.0037564342, 0.01895874, 0.017837003, -0.010137179, -0.006242259, -0.0056536933, 0.0032838506, 0.002721251, 0.041933578, 0.004192665, 0.016535236, -0.019554231, -0.018792557, -0.0018193605, -0.013426223, -0.01678451, -0.00709741, -0.04420475, 0.0074297767, 0.0050685895, 0.010400303, 0.01285843, 0.007360534, 0.0005885657, -0.008260693, 0.014250215, -0.039551616, 0.0054944344, -0.0025239084, -0.019692717, -0.030716207, -0.0066854144, -0.007464398, -0.012442972, 0.013093856, -0.0298022, -0.022282407, -0.01974811, -0.01707533, 0.030826995, -0.0065780873, -0.029552924, 0.019097228, 0.004511183, 0.013516239, -0.010531864, -0.020495936, 0.041656606, 0.0023646494, 0.001144241, 0.00640498, -0.0040195575, 0.021382246, 0.0051274463, -0.020634422, -0.0070108566, -0.022587076, -0.04221055, 0.0058129523, 0.024442788, 0.0053490237, 0.0004178384, -0.0031020876, -0.023750357, -0.0092231715, -0.0023663805, 0.0042376732, -0.019041833, -0.010012542, 0.0035798645, 0.011937498, -0.007713673, -0.019291107, -0.021229912, 0.0022452052, -0.024581274, -0.022407044, -0.004480024, -0.0017094372, 0.02124376, -0.026160015, -0.015551983, -0.03653262, -0.009423976, -0.038803793, 0.0048677847, -0.0022330876, -0.0006733884, 0.018487887, -0.012678398, 0.006498458, 0.0020876774, -0.013904, 0.005421729, 0.0013095587, -0.010587259, -0.0027368306, 0.018598678, -0.0023265658, -0.0044696373, -0.032045674, 0.007575187, 0.028126517, 0.025896892, 0.0084338, 0.019014135, -0.027780302, 0.008406104, -0.012747642, 0.01627211, -0.0030051474, 0.015136526, 0.019609625, -0.03068851, -0.03625565, 0.026173864, -0.00978404, -0.009264717, -0.018972589, 0.0053040157, 0.0005115329, 0.04306916, -0.0051863026, -0.028071122, 0.016895298, -0.008828486, 0.026520079, -0.00088631077, -0.0021811554, 0.0073120636, 0.002742024, -0.033707507, -0.007360534, 0.020385146, -0.012242167, -0.011992892, 0.014208669, 0.0049127927, 0.0033479002, -0.029829895, 0.01171592, -4.173515e-05, 0.01206906, -0.00039858016, -0.011466646, -0.005179378, -0.00012517841, 0.025024429, 0.0045458046, -0.045534216, 0.011757466, -0.011688223, -0.0025793028, 0.003704502, 0.00888388, -0.015150374, 0.01355086, 0.019900447, -0.009763267, -0.01400094, -0.018030884, 0.016950693, -0.002612193, 0.008565363, 0.17859161, -0.0034725377, 0.017296908, 0.03287659, -0.0057367845, 0.0014385239, -0.0069900835, -0.006449988, 0.0023213725, 0.0037356613, 0.024484335, 0.01926341, 0.0012870547, 0.0035867887, 0.006609247, 0.0038152908, -0.03395678, -0.010026391, -0.01469337, -0.020606725, -0.006294191, -0.01519192, 0.0004191367, -0.00026009412, 0.020689817, -0.013301586, -0.052403122, 0.015815107, 0.02055133, -0.011909801, -0.025259856, -0.01321157, 0.0009520916, 0.009631705, -0.012532988, 0.006543466, -0.006387669, 0.020066628, 0.0090085175, 0.00011306088, -0.007568263, -0.036920384, 0.0022348186, -0.016590629, -0.008953123, 0.007782916, -0.02016357, 0.0016445218, -0.008364557, -0.011065036, -0.023639569, 0.01221447, -0.008295314, 0.027988032, 2.2503984e-05, -0.045838885, 0.008676151, 0.020218965, -0.0056433068, 0.0029791812, -0.0027264443, 0.014111728, -0.019277258, 0.0129761435, -0.023417993, 0.022296255, -0.011999817, -0.0042272867, 0.014374852, -0.0057575577, -0.004774307, -0.02153458, 0.0016090348, -0.013523163, 0.0020201653, -0.01509498, 0.027531028, 0.018640222, 0.03068851, 0.033292048, 0.0036664184, 0.010365681, -0.023750357, -0.010234119, -0.00048080625, -0.02195004, 0.031464033, -0.0070766374, -0.009216247, -0.013419298, -0.014347155, -0.004376159, -0.012844581, -0.014845705, 0.013910924, 0.0024183127, -0.013024613, 0.005947976, 0.011840558, -0.0013035, -0.025065975, 0.052957065, 0.009243944, -0.00918855, -0.029165164, 0.0021586514, -0.009147004, 0.0055359798, 0.027254056, -0.024055028, 0.0034448407, -0.017726215, -0.0015579681, -0.02679705, 0.023501083, 0.028611219, 0.012422199, -0.029746804, 0.010247968, -0.002766259, -0.012346032, -0.01335698, -0.0002854111, 0.025010582, -0.017352302, -0.009250868, -0.017823156, 0.011840558, -0.0011788625, -0.014250215, -0.0011606863, -0.014748764, 0.011189673, -0.003545243, 0.0041268845, 0.00038776096, -0.008994669, -0.015011888, -0.01137663, -0.0127614895, -0.011051187, 0.0019838128, 0.01726921, 0.004348462, -0.025107522, -0.0047777686, 0.04029944, 0.006979697, -0.005168992, -0.022697864, -0.012172924, -0.026990931, -0.027226359, 0.0028908963, 0.012498367, -0.00086770166, -0.024761306, -0.024941338, -0.007769068, -0.016964542, -0.032045674, -0.0063634343, 0.025370644, -0.005092825, -0.00799757, -0.011411251, -0.17837004, 0.016147474, 0.008039115, -0.0024789004, 0.026824748, 0.021091426, 0.020772908, -0.0031211295, -0.0071251076, 0.007921402, 0.029857593, 0.0036906532, -0.013412374, -0.030439235, 0.021396095, 0.013717043, 0.0025360258, 0.016798358, 0.034012176, 0.0042896057, 0.014374852, -0.0253291, 0.003683729, -0.017116876, 0.00759596, 0.03642183, -0.01995584, 0.01906953, -0.013841681, -0.022365497, -0.02154843, 0.008828486, 0.019554231, -0.016646024, 0.019138772, 0.0050201192, 0.008032192, 0.024345849, -0.0068966052, 0.009860207, 0.019595776, 0.013959394, -0.0027472172, -0.005677928, -0.01598129, -0.0009745956, 0.01826631, -0.014174048, 0.008267618, -0.032488827, -0.00907776, -0.011598207, 0.020800605, 0.012138302, 0.032433435, 0.008994669, -0.012740717, 0.009950223, 0.010580335, -0.016853753, -0.016673721, -0.018390948, 0.011736693, -0.02165922, -0.0019249561, -0.041573513, -0.025398342, -0.012027514, -0.022199314, 0.0031401713, -0.014998039, 0.0032353804, 0.009867132, 0.009237019, -0.007852159, 0.013772438, -0.017089179, 0.01699224, 0.0003366942, 0.0041303467, -0.016258262, 0.023958087, -0.017310757, -0.0072843665, 0.025841497, 0.012782263, -0.012159076, -0.0018816793, 0.0025706473, 0.000112952686, 0.017823156, -0.0068585216, 0.01082961, -0.008703848, 0.021105275, 0.010448772, -0.017158423, 0.0035262012, -0.016632175, -0.008516892, 0.014430246, -0.007727522, -0.0080668125, 0.049605705, 0.044841785, -0.0042272867, -0.008780016, -0.0035833267, 0.062817276, -0.011999817, -0.025010582, -0.013876302, 0.009472446, 0.012796111, -0.0036768047, 0.01599514, 0.0020478626, -0.033818293, 0.015247314, -0.018100128, 0.059327424, 0.0073120636, -0.0077621434, 0.028361944, -0.026589323, -0.02491364, -0.09522301, 0.0049543385, -0.015704319, 0.028348096, -0.03295968, 0.0008728949, -0.021202214, 0.020869847, -0.003228456, 0.014305609, 0.0021430717, -0.02679705, -0.00092352886, 0.007201275, -0.0065711634, 0.013966318, -0.0168676, -0.0137932105, -0.022725562, 0.023501083, 0.0031055498, -0.012048286, -0.009084685, -0.007415928, 0.023985785, -0.0030051474, -0.023210263, 0.019498836, 0.0158982, 0.0071666534, 0.009804812, -0.015053434, 0.015053434, -0.01181286, -0.030522326, -0.012373729, -0.013280813, -0.024650518, 0.015635075, -0.046752892, 0.008378406, 0.0158982, 0.0079421755, -0.020218965, 0.0013805329, -0.015745863, -0.009216247, 0.008032192, 0.0016869332, -0.014721068, -0.02631235, -0.007042016, -0.027489481, -0.0073536094, 0.035701703, -0.036394134, -0.0007123376, 0.016673721, -0.017449243, -0.02771106, -0.0092577925, 0.007069713, -0.029857593, 0.021825401, -0.0062491833, -0.010871155, -0.004369235, -0.034649212, 0.018640222, -0.025952287, 0.010995792, -0.0074297767, -0.01599514, -0.0016895299, -0.01285843, 0.0066438685, -0.04390008, -0.020579027, 0.022587076, -0.0016774123, -0.0010100827, -0.017753912, -0.0013796673, 0.009112382, 0.026076924, 0.009292414, -0.0059722112, 0.021631522, 0.012775338, -0.021423792, 0.0016488496, 0.033181258, -0.00918855, -0.01126584, -0.013578557, 0.038886882, -0.024955187, -0.008253769, 0.014804159, 0.01749079, -0.03442763, -0.0068896813, -0.080100335, 0.03761281, 0.0070212428, -0.020205116, -0.0028233845, -0.01866792, 0.009063913, -0.00053533516, 0.0097563425, 0.043955475, -0.038083665, 0.039883982, -0.022268558, -0.010393378, -0.028666614, -0.0047189123, 0.021770008, -0.005719474, 0.02244859, 0.014236366, -0.005470199, 0.010857306, 0.03287659, 0.03822215, -0.010407227, 0.010469546, 0.002470245, 0.027475633, -0.029137466, -0.0073813065, -2.8184075e-05, -0.02899898, -0.014374852, 0.030522326, 0.018847952, 0.0028614681, -0.021077577, 0.0050997487, 0.0104141515, 0.017712366, -0.018183218, -0.03883149, 0.0020876774, -0.010649578, 0.0129761435, 0.020385146, -0.0078106136, -0.005335175, 0.0035971752, 0.011411251, 0.035895586, 0.0105041675, -0.017837003, -0.05071359, -0.0019318805, 0.009133155, 0.0011728038, 0.008994669, 0.0055498285, -0.0012351224, 0.022019282, 0.031353243, 0.019706566, 0.027143266, -0.014014789, -0.015122677, -0.0012039631, 0.0063011153, 0.0006063093, -0.05849651, -0.011909801, -0.00543904, 0.0011901145, 0.0137585895, -0.0041822786, 0.0031938346, -0.014679521, 0.013460845, -5.301419e-06, 0.0053628725, 0.028472733, -0.002240012, -0.026727809, -0.013156175, 0.028971283, 0.009451673, 0.020205116, 0.008696924, -0.017684668, 0.011542813, -0.012256016, 0.02679705, 0.0014013058, 0.016798358, -0.0013545667, 0.02830655, -0.01569047, -0.0010282589, 0.026520079, 0.02404118, -0.0020236275, 0.017726215, 0.00603453, -0.015468892, -0.003228456, -0.010033314, -0.0072220475, -0.012394502, 0.016244413, 0.012498367, 0.00019701805, 0.010822685, -0.01007486, 0.010781139, -0.008849259, 0.010961171, 0.0017431933, -0.018834103, -0.030633114, 0.058219537, 0.026104622, 0.01797549, 0.015551983, 0.0034621514, 0.015316557, 0.024802852, 0.018238613, -0.008627681, 0.026561625, -0.013994016, 0.0020703664, 0.006415366, -0.017407697, 0.011646677, -0.014319458, -0.004396932, 0.001704244, -0.0042896057, -0.01876486, 0.08708003, -0.0062318724, -0.01747694, 0.024844399, -0.041351937, -0.0023473387, 0.03860991, 0.013350056, 0.0017449242, -0.025966136, -0.006882757, -0.0048677847, 0.00080365187, -0.042044364, -0.03185179, 0.024110422, 1.5214532e-05, -0.004601199, -0.023417993, -0.0096871, 0.0036214103, 0.009555537, 0.021977738, 0.00997792, -0.01404941, 0.0013813984, 0.012505291, -0.007866008, 0.017795458, -0.057942566, 0.017504636, -0.009361657, -0.020925242, -0.021783857, 0.015441195, -0.016119776, -0.02124376, -0.006675028, 0.031076271, -0.007845235, 0.01598129, -0.005352486, -0.025578374, -0.010822685, 0.016064383, -0.0022486672, -0.030411538, -0.0011849212, -0.027586421], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-10-01533.pdf', 'citation': 'Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.82566845),\n",
      " NodeWithScore(node=TextNode(id_='4694bc56-2afc-45df-a026-79ccca7944f8', embedding=[-0.024463622, -0.007585244, -0.024546595, -0.0030406662, 0.020411704, 0.03457267, -0.0035298695, -0.0041625495, -0.0075022695, -0.008539449, 0.021144645, 0.022541381, -8.729599e-05, -0.004093404, -0.007827252, -0.0013103042, 0.01854478, -0.019346867, 0.00096198457, -0.024491278, 0.010786673, 0.009887784, -0.01142281, -0.010710613, -0.0021193046, 0.021255277, 0.021628663, -0.038610756, -0.015751308, 0.0041487203, -0.0003997465, 0.0008837639, -0.005825495, 0.02728475, -0.021601005, -0.03545773, 0.010924963, -0.01972025, 0.042068023, 0.005230845, 0.030534582, 0.007896398, -0.001773578, 0.011595674, -0.005137499, -0.01840649, 0.013013153, -0.0066275806, -0.020010661, 0.0043872716, 0.0131307, -0.0017977789, -0.004349242, 0.027990034, 0.009735663, -0.008221381, 0.014810932, 0.013600889, -0.0076613035, -0.0142854275, -0.010655297, -0.012591367, -0.013407282, -0.0091755865, 0.004152178, 0.004764114, 0.0025583776, 0.019056456, 0.0035817286, 0.0055212555, 0.017715037, 0.0043976437, 0.015198146, 0.0060156444, 0.024601912, -0.03341103, -0.028321931, 0.007930971, -0.0051063835, 0.012570622, 0.0055765714, 0.006333713, -0.026150767, 0.009846296, 0.02893041, -0.005797837, 0.013759922, 0.021020183, -0.012217982, -0.021725466, 0.021849927, 0.021891415, 0.043450933, 0.0137806665, -0.017037412, 0.031115403, -0.017479941, 0.009396852, -0.0006646596, -0.009161757, -5.1696945e-05, -0.001648252, -0.013773751, -0.02060531, -0.01942984, 0.008650082, 0.0016335587, -0.003688904, 0.012466905, -0.013925872, -0.017853327, 0.01839266, 0.014575837, -0.05326957, 0.030147368, 0.013123785, 0.020674456, -0.021559518, -0.019374523, 0.006634495, 0.008394244, 0.04732307, 0.0047122547, -0.009210159, 0.032249387, 0.0010665669, -0.0113813225, -0.050946288, -0.00085005554, -0.013068469, 0.03233236, 0.041459545, 0.020342559, 0.001768392, -0.016083205, 0.021324422, -0.04226163, -0.02862617, -0.015488557, -0.021255277, 0.008090004, 0.04292543, -0.015944915, 0.014202453, -0.021227619, 0.006409773, 0.024491278, 0.012923264, 0.0037753356, 0.014271598, 0.003419237, -0.018337345, -0.0025359052, 0.0009602559, -0.008470304, -0.0043665282, -0.010517006, 0.008186808, -0.045663584, -0.039246894, -0.03341103, 0.0029231193, 0.01662254, -0.0064547174, 0.022956252, 0.014479035, 0.017258678, 0.02136591, -0.0062369094, -0.0101851085, -0.0051617, 0.011637161, -0.011526528, -0.004570507, 0.0074262093, 0.028985726, 0.013504085, -0.004622366, 0.010039903, -0.015557702, 0.0031478417, 0.010897306, 0.031170718, 0.021849927, -0.010973365, 0.016525736, 0.0258327, -0.031640906, 0.01706507, -0.03260894, 0.017977789, 0.015861942, 0.020107465, -0.0018842105, -0.62706524, -0.034627985, -0.010717528, 0.005683747, -0.00852562, 0.011249946, 0.0006551521, -0.013186016, -0.010973365, 0.02211268, -0.012584452, -0.0035678996, -0.004349242, -0.032221727, -0.027270922, -0.028681487, 0.005732149, -0.0104340315, 0.0008046789, -0.0013025253, -0.007799594, 0.014084905, -0.010883477, -0.010198938, -0.0007644882, 0.009459082, 0.04198505, 0.006354457, 0.0134003665, 0.02194673, -0.03283021, 0.030230343, 0.0068937903, -0.018738387, 0.053186595, -0.006098619, 0.0025635634, 0.008380415, 0.028653828, -0.017452285, -0.012211067, -0.01764589, 0.0014157508, 0.019125601, -0.01676083, 0.012743486, 0.01676083, 0.0032550169, -0.010523921, -0.0021072042, -0.0064478028, 0.028460221, -0.022486065, 0.004069203, 0.02106167, 0.0118860835, 0.02967718, -0.0362045, 0.017562917, 0.015848111, 0.0058704396, -0.0014295799, 0.007882569, -0.029732496, -0.005853153, 0.016359787, -0.022527551, 0.007972457, 0.015571531, -0.014741787, 0.008816031, 0.007419295, 0.0081660645, -0.012508392, 0.009763322, 0.040021323, -0.007993202, 0.015460898, -0.0045255623, -0.0007964679, 0.0034002222, 0.0051236697, -0.023384955, 0.011146229, 0.031945147, 0.016650198, -0.017853327, 0.0041902075, -0.0021728922, -0.022209482, 0.023855142, 0.015032196, -0.02255521, -0.03504286, -0.012356272, 0.00489549, -0.002537634, -0.013891298, 0.009141014, 0.0012584452, -0.021863757, -0.0051478706, 0.026510322, -0.006433974, 0.013981188, -0.008318184, 0.003094254, -0.0023993433, 0.02032873, -0.011533443, -0.015364095, -0.01795013, -0.017894814, 0.0052792467, 0.010074476, -0.035430074, 0.022085022, 0.026178425, 0.017673548, -0.025348682, 0.011201545, -0.012874862, -0.014133307, -0.003522955, 0.014271598, 0.010593066, 0.018987311, -0.022223312, -0.011395152, -0.014202453, 0.0024823176, 0.0027848287, 0.023924287, -0.014382231, 0.026219914, 0.029262308, 0.025556117, 0.00986704, 0.00578055, -0.017701207, -0.028404906, -0.0087330565, 0.013331221, 0.010579237, 0.00097408495, -0.016193839, 0.011090913, 0.011464298, 0.017327823, 0.008117663, 0.0089888945, 0.028985726, 0.010579237, -0.0053656786, 0.023357296, -0.014382231, -0.023509415, -0.03767038, -0.0036958184, -0.009479826, 0.009383023, 0.006399401, 0.0021694351, -0.006254196, 0.018295856, -0.0033691067, 0.020301072, 0.009106441, 0.0015160115, -0.03247065, -0.0035575277, -0.021849927, -0.02595716, 0.013953529, -0.00075325207, 0.025791211, -0.013946615, 0.028653828, -0.0051617, -0.005580029, 0.006420145, -0.021158474, 0.009376108, 0.0230254, 0.016290642, 0.0051340414, 0.011775452, -0.016525736, 0.023246663, 0.0358726, -0.020964867, 0.008124578, -0.0038133655, -0.003438252, -0.005825495, 0.002560106, 0.023011569, 0.010793587, 0.008338928, 0.019830883, 0.013483341, 0.019637277, 0.011540357, -0.0065999227, 0.012183409, -0.049922936, 0.0015626847, -0.028819777, 0.03816823, 0.02895807, 0.0012869677, 0.002769271, -0.030202685, 0.0042558955, 0.0046016225, -0.0055005117, 0.020010661, -0.0038825108, -0.00083104055, 0.0018340801, -0.027381554, 0.016373616, 0.02493381, 0.01883519, -0.0314473, 0.007834167, -0.0028660744, 0.041238282, -0.0030959826, 0.0021694351, -0.0050303233, 0.020107465, 0.030175027, 0.00047494206, 0.0255008, 0.0113813225, 0.026842222, -0.006019102, 0.009770237, -0.004684597, -0.0033345341, 0.009009638, 0.018461805, 0.00042848501, 0.006824645, 0.009147928, 0.016180009, 0.035568364, -0.010710613, -0.0027502559, -0.021020183, -0.002286982, -0.020411704, -0.007599073, 0.019070284, -0.023785997, -0.011166972, 0.004017344, 0.029400598, 0.019540474, -0.0030354804, 0.0066759824, -0.021559518, -0.00452902, 0.0258327, -0.011740879, -0.020881893, -0.016968267, -0.021435056, -0.0014710671, -0.03260894, -0.007018252, 0.022195654, 0.0020691743, 0.014520521, -0.005472854, -0.017590575, -0.009320792, 0.00040601278, 0.004131434, -0.035264123, -0.03454501, 0.018738387, 0.019180918, 0.0077373637, 0.0007225688, -0.015460898, 0.00014272031, -0.015322607, 0.016221497, 0.028003862, 0.007993202, -0.00022623492, 0.026441177, -0.0075645, 0.0033051472, 0.02551463, -0.01676083, 0.017410796, 0.008242125, -0.0023025398, 0.0044944473, -0.016968267, -0.0026724674, 0.02729858, -0.011111656, 0.009251647, -0.0073363204, 0.00024633028, -0.03042395, 0.02153186, 0.0015713278, 0.0035246836, -0.0041106907, -0.01705124, 0.0021020183, -0.023965774, 0.024671057, 0.013483341, -0.021020183, -0.0075022695, -0.035430074, -0.010793587, -0.010392545, 0.041929733, 0.024256185, 0.009569715, 0.021587174, 0.03816823, 0.008954322, -0.006924906, -0.011450469, 0.01543324, -0.017576745, -0.01291635, -0.0016499807, 0.015391753, -0.0091341, 0.009950015, -0.003211801, -0.0035886432, -0.0145343505, -0.012985495, 0.0034140511, 0.0050856397, 0.0037857073, 0.04450194, 0.010862733, 0.017383138, -0.021725466, 0.03147496, 0.01929155, 0.005839324, -0.023647707, -0.010074476, -0.017839497, 0.0040208013, 0.020619141, 0.024117894, -0.006126277, 0.0012221439, 0.01009522, 0.019319208, 0.027257092, 0.0203149, 0.020494679, 0.006693269, 0.0035056688, -0.012743486, 0.012625939, -0.0119483145, 0.015709821, -0.011402067, -0.028709145, 0.040325563, -0.0023785997, 0.008795287, -0.032885525, 0.02816981, 0.012411589, -0.03100477, -0.011547272, -0.0063233413, -0.025334852, 0.002305997, -0.019692592, 0.009721835, -0.0079448, -0.022610527, 0.017120386, -0.017023582, -0.014071077, -0.029124018, -0.0013120328, -0.01810225, -0.020231927, -0.030036736, 0.012667426, 0.0025929501, 0.010593066, 0.016581053, -0.006357914, -0.0019464414, 0.02536251, -0.017687378, 0.00092827616, -0.0030389377, -0.04881661, 0.02227863, 0.0087330565, 0.0047468278, -0.004273182, -0.007986287, 0.0023526703, 0.011277605, 0.021421226, 0.02314986, 0.004919691, 0.006126277, -0.01913943, 0.000870367, 0.017410796, -0.00589464, -0.012660512, 0.014949222, -0.026289059, -0.003958571, -0.016857633, -0.012003631, -0.006769329, -0.005815123, 0.019374523, -0.020356389, 0.00076362386, 0.00082282955, -0.014741787, 0.004041545, 0.009064954, 0.03830652, 0.022154167, 0.0073224916, -0.011491955, 0.0074123805, -0.0073501496, 0.01195523, 0.006859218, -0.0045808787, 0.0027329696, -0.013587059, 0.03753209, -0.013552487, -0.02998142, -0.023896629, 0.0029127474, -0.026012477, 0.03473862, 0.02386897, -0.0035540704, -0.045718897, -0.025155075, 0.019457499, 0.018793704, -0.001961999, -0.031668566, -0.0061331918, -0.005918841, 0.013109957, 0.009797894, -0.0017312265, -0.03872139, 0.0040208013, 0.028404906, -0.010523921, -0.0033708352, 0.008504877, -0.010053732, -0.025888015, -0.00045419845, -0.057750188, -0.014714128, -0.0065999227, 0.014700299, 0.018779874, 0.014866248, 0.023620049, -0.010738271, -0.0086708255, 0.027367726, 0.012577537, 0.010399459, -0.011111656, -0.00028371197, -0.024919981, 0.015502385, 0.024449792, -0.007516098, -0.00067676004, -0.015972573, 0.008892091, 0.015958745, -0.004390729, -0.014797103, -0.006551521, -0.030617556, -0.027091144, -0.014921565, -0.016802318, -0.0076820473, -0.041653153, -0.031032428, 0.022320116, -0.010931878, 0.011810024, -0.001004336, 0.019761737, -0.0034278801, 0.023924287, 0.014271598, 0.007294833, -0.020702114, 0.0054348237, -0.028847435, 0.0050960113, 0.023371125, 0.0031616706, 0.00815915, 0.009583544, -0.028570855, 0.010226596, 0.010178194, -0.018033104, -0.011173887, -0.024684886, -0.012238725, -0.014575837, -0.020301072, 0.011471212, -0.03794696, 0.0065169483, -0.0043457844, -0.023246663, -0.0053691356, -0.017023582, -0.02078509, 0.025735896, 0.013918957, 0.03930221, -0.03460033, 0.032083437, 0.03780867, 0.0023457557, -0.015668334, -0.009645775, -0.020702114, -0.0050856397, 0.042676505, 0.03902563, -0.022015877, -0.023841314, 0.0025825785, 0.0222648, -0.009666518, -0.028059179, 0.039357528, -0.015087513, -0.006551521, -0.01291635, -0.013448768, -0.035430074, 0.017562917, -0.03130901, 0.02565292, -0.012418503, -0.0014606953, -0.011485041, 0.0009991502, 0.020978697, 0.018060762, 0.020273414, -0.036702346, 0.014271598, -0.019872371, 0.00075022696, 0.011906828, -0.023924287, 0.031115403, 0.015377924, 0.0031011684, 0.009887784, -0.0075299274, -0.009286219, 0.0006841067, 0.00326366, 0.040242586, -0.012045118, 0.0021936358, -0.0123286145, 0.015709821, 0.0186969, -0.007834167, -0.008567108, -0.026551811, -0.0017130758, -0.00079257844, 0.030340975, 0.013974274, -0.003702733, -0.024878493, -0.018710729, -0.009369194, 0.0027709994, -0.016498078, -0.010510092, -0.02639969, -0.0018046934, 0.006696726, -0.0026482665, 0.0018997682, 0.004923148, 0.00021705154, -0.015682163, 0.010468604, -0.036425766, 0.010945708, 9.0051e-05, -0.029068701, -0.024698716, 0.00071176485, -0.007523013, -0.009092612, 0.0030026364, -0.026593298, -0.0038099082, -0.031198377, -0.025348682, 0.026676273, -0.0038064509, -0.037172534, 0.022679672, -0.003937827, 0.019526644, -0.026316717, 0.00017642866, 0.041459545, -0.010558493, 0.014880077, 0.014230111, 7.346692e-06, 0.017396968, 0.0017329551, -0.008062347, -0.0059845294, -0.024007263, -0.03885968, 0.0027139545, 0.016041718, 0.0050856397, -0.014479035, 0.010447861, -0.022057364, -0.008449561, -0.01897348, 0.02196056, -0.007813423, 0.0033777498, -0.019678764, -0.014935394, -0.013255161, -0.016581053, 0.00034097297, -0.0062887687, -0.018724559, -0.021642491, -0.006420145, -0.028308103, 0.02313603, -0.018517122, -0.001981014, -0.020397875, -0.0077511924, -0.025431655, 0.018641584, -0.009701091, 0.0076958765, 0.0060398453, -0.017728865, 0.0072879186, -0.0014304442, -0.0059741572, 0.009528228, -0.016276812, -0.004674225, 0.0114435535, 0.0029870786, 0.032996155, -0.013849812, -0.033466343, -0.009417595, 0.020065978, 0.0131307, 0.009777151, 0.02612311, -0.026468836, 0.007861825, 0.0017312265, -0.0072395173, 0.008276697, 0.015018368, 0.015018368, -0.027934717, -0.01142281, 0.023633877, 0.011602588, -0.0051997295, -0.028100666, 0.022071192, -0.0034935684, 0.03916392, -0.012619024, -0.019180918, 0.0064063156, -0.0117616225, 0.024242356, 0.011547272, 0.012570622, -0.00079344277, 0.012466905, -0.02313603, 0.002141777, 0.02373068, -0.015695993, -0.015585359, 0.0015678706, 0.018613925, 0.014326914, -0.01900114, 0.030506924, -0.010738271, 0.0040104296, 0.008546364, -0.014147136, -0.0015549058, -5.9313737e-05, 0.017410796, -0.00025843072, -0.051195208, 0.009721835, -0.029345281, 0.025611434, 0.004045002, 0.0096803475, -0.0052204733, 0.016733171, 0.033853557, 0.0024512024, 0.0036301303, -0.012363187, 0.0016257798, 0.0026517238, 0.0064408886, 0.18276496, -0.01201746, 0.0056388024, 0.014617325, -0.012992409, 0.011339836, -0.010634554, -0.01573748, -0.0015263833, 0.007384722, 0.03125369, 0.017880986, 0.0036716175, 0.0032273587, -0.0023094544, -0.005189358, -0.03310679, -0.0072326027, -0.026164597, -0.018129908, -0.015308778, -0.01915326, 0.01647042, -0.0022178367, 0.024726374, -0.0051478706, -0.045442317, 0.009728749, 0.0042109513, -0.007135799, -0.0142854275, -0.010440947, -0.00046370595, 0.015322607, -0.00048531385, 0.002349213, -0.0032774892, 0.0045359344, 0.014465205, -0.008179894, -0.0052204733, -0.01974791, -0.002848788, -0.009763322, -0.009514399, 0.007028624, -0.012052033, -0.00667944, -0.01868307, 0.00242873, -0.024449792, 0.0134003665, -0.0034071365, 0.03424077, 0.0012722743, -0.028653828, 0.013587059, 0.023288151, -0.010828161, 0.025445485, -0.0044909897, 0.006465089, -0.026344374, 0.021711636, -0.008408073, 0.019028798, -0.0068419315, 0.009590459, 0.030396292, -0.010413288, -0.011028682, -0.029068701, 0.004591251, -0.007516098, -4.6132907e-05, -0.021241449, 0.017120386, 0.02909636, 0.021227619, 0.025182733, 0.009362279, 0.0015583631, -0.013013153, -0.012681255, -0.011464298, -0.038804363, 0.011270691, -0.002414901, -0.009583544, 0.0005337156, -0.01705124, -0.008000116, -0.015709821, 0.008802202, 0.026067793, -0.0011633703, -0.03396419, -0.010973365, 0.022361603, -0.0068903333, -0.020411704, 0.05293767, 0.013317392, -0.011166972, -0.016857633, -0.00919633, -0.017161874, 0.008318184, 0.014202453, -0.017853327, 0.012342443, -0.00934845, -0.005583486, -0.019457499, 0.017535258, 0.032138754, 0.025915673, -0.028238956, 0.033908874, -0.010952622, -0.0040726606, -0.007578329, -0.0056007723, 0.017300164, -0.010807416, -0.011892999, -0.025708238, -0.002990536, -0.00077486, -0.02657947, -0.007329406, 0.00549014, 0.0122663835, -0.0108765615, -0.004739913, 0.010517006, 0.00445296, -0.013981188, -0.020135123, -0.0029888074, -0.00957663, 0.0060294736, 0.018212883, 0.013995017, 0.0009896427, -0.019014968, 0.03454501, 0.00500958, -0.0017666634, -0.029345281, -0.0117616225, -0.017023582, -0.02493381, -2.4795087e-05, 0.011243032, 0.0026309802, -0.023108372, -0.014382231, -0.020162782, -0.0076751327, -0.033936534, -0.0076751327, 0.014077991, -0.0033604635, -0.001637016, -0.02610928, -0.17690144, 0.016581053, -0.00021327016, 0.0015393482, 0.02895807, 0.014631154, 0.007930971, 0.004024259, 0.0014900821, 0.01483859, 0.026178425, 0.0047952295, -0.012370101, -0.009037296, 0.017977789, 0.0057528922, -0.011104741, 0.033300396, 0.031364325, 0.007253346, 0.022361603, -0.018572439, 0.0066033797, -0.028792119, -0.0014477306, 0.04143189, -0.009521313, 0.025528459, -0.034849253, -0.020384045, -0.023205176, 0.003688904, 0.021614833, -0.013697692, 0.028543197, 0.010959537, 0.009403766, 0.01973408, -0.007343235, 0.01468647, 0.006482376, 0.020231927, -0.00971492, -0.008318184, -0.010634554, 0.01291635, 0.019111771, -0.011270691, 0.018046934, -0.020729773, -0.0073224916, 0.004923148, 0.029594205, 0.0039309124, 0.028225128, -0.0037373055, -0.0099292705, 0.008926664, -0.0060813325, -0.01142281, -0.00957663, -0.008269783, 0.017327823, 0.0011538628, 0.0068834187, -0.041874416, -0.02493381, 0.0041348916, -0.035817288, 0.012950922, -0.029953761, -0.0021521486, 0.013635461, -0.0018755674, 0.02153186, 0.02493381, -0.007080483, 0.0022282086, 0.03844481, -0.0060329307, -0.0047364556, 0.014064162, -0.022071192, -0.013960444, 0.017134216, 0.009611202, -0.0071911155, 0.011844597, 0.00033470665, -0.002796929, 0.0038513953, -0.01886285, 0.009880869, -0.024408305, 0.017549086, 0.01387747, -0.00034745532, 0.002447745, -0.001858281, 0.0079448, 0.008214466, -0.012217982, -0.025639093, 0.05028249, 0.02729858, -0.0030683244, -0.021144645, -0.0013690777, 0.046659276, -0.024947638, -0.023080714, 0.005732149, 0.013912043, 0.024560425, -0.00042913327, 0.008560193, -0.00971492, -0.033328053, -0.012681255, -0.019014968, 0.055897094, 0.008041603, -0.004632738, 0.04226163, -0.020812748, -0.022762645, -0.10797736, 0.007495355, -0.004127977, 0.03280255, -0.028003862, 0.0046915114, -0.01305464, 0.02479552, -0.006344085, 0.018212883, 0.007778851, -0.017120386, -0.020453192, 0.004650024, 0.003778793, 0.006112448, 0.015115172, -0.022665842, -0.03410248, 0.010406374, 0.0026655528, 0.0023336553, 0.0028729888, -0.00028824963, 0.027907059, -0.023177518, -0.019305378, 0.03562368, 0.008034688, 0.019969175, 0.02017661, -0.021697808, 0.0282113, -0.005023409, -0.012155751, -0.009832467, -0.0035333268, -0.021766953, 0.026178425, -0.04823579, 0.006541149, 0.019595789, 0.020301072, -0.007688962, 0.0108765615, -0.01454818, -8.9348745e-05, 0.007972457, 0.0050925543, -0.02063297, -0.030313317, -0.021863757, -0.02685605, -0.015460898, 0.029538888, -0.04986762, -0.008214466, 0.043119036, -0.016110864, -0.028985726, -0.006022559, -0.0020207725, -0.036453422, 0.0015808353, -0.011395152, -0.022887107, -0.00667944, -0.021683978, 0.012197238, -0.027229436, 0.01468647, -0.001967185, -0.01499071, -0.0070147947, -0.0326366, -0.0010173008, -0.031281352, -0.017438455, 0.026814563, 0.0013777209, -0.0070493673, -0.009777151, 0.01186534, -0.0038652245, 0.01781184, 0.003547156, -0.0054624816, 0.031530276, 0.0005108112, -0.011962144, -0.009950015, 0.035540704, -0.021919074, -0.030064393, -0.012854119, 0.034517355, -0.016636368, -0.01015745, 0.004898947, 0.024131723, -0.025251878, 0.0025428198, -0.0883954, 0.023384955, 0.0031046257, -0.029649522, -0.013441854, -0.0142854275, 0.005465939, -0.007135799, -0.0038168228, 0.027326237, -0.039827716, 0.025998648, -0.014465205, -0.011630246, -0.0155438725, -0.013552487, 0.032304704, 0.010572323, 0.022209482, 0.015502385, -0.009908527, 0.0019481699, -0.0030959826, 0.0274507, -0.029870788, 0.021974389, -0.008899005, 0.03814057, -0.01090422, -0.009562801, 0.0041487203, -0.01942984, -0.000799493, 0.025390169, -0.014617325, -0.011277605, -0.02568058, -0.0056664604, 0.0030700532, 0.015958745, -0.010510092, -0.040740434, 0.00012835104, -0.010012245, -0.00024654635, 0.0069836793, -0.0038652245, -0.005953414, -0.00382028, 0.00031547563, 0.026786905, 0.0043699853, -0.025943331, -0.039053287, 0.0019792854, -0.0062956833, 0.00986704, -0.006786615, -0.0089197485, 0.0140295895, 0.024117894, 0.021739295, 0.031530276, 0.027810255, -0.011803109, -0.01634596, -0.01282646, 0.006070961, -0.012805717, -0.05398868, -0.000121652585, -0.0014848962, -0.0060018157, 0.007211859, -0.010530835, 0.0210755, 0.0033120618, 0.025044441, 0.008352757, -0.0015600917, 0.009113356, -0.02314986, -0.016110864, -0.012923264, 0.004622366, 0.012729657, -0.008477218, 0.018959653, -0.013365794, -0.0040069725, -0.0056802896, 0.033162106, -0.002537634, 0.00030035008, -0.013822153, 0.017853327, -0.014852419, 0.00246676, 0.023661535, 0.018766046, 0.009037296, 0.016857633, 0.0046534813, -0.027990034, -0.0038271945, 0.006368286, -0.015613018, -0.014617325, 0.0017441912, 0.010779759, -0.012930178, 0.031419642, -0.012508392, 0.006769329, -0.0017122115, 0.011243032, -0.009735663, -0.02595716, -0.03446204, 0.05769487, 0.025486972, 0.024781689, 0.0107521005, -0.004245524, 0.009777151, 0.026966682, 0.02937294, -0.010150536, 0.020951038, -0.014562009, 0.005828952, 0.006987137, -0.039827716, -0.0031426556, 0.0016119507, -0.007315577, -0.00057390635, -0.014893906, -0.02432533, 0.07451102, 0.0045013614, -0.019817054, 0.02565292, -0.039717082, 0.01706507, 0.01915326, 0.016733171, 0.0054313666, -0.038085252, -0.01401576, -0.0134625975, -0.022002047, -0.024435963, -0.02298391, 0.021116987, -0.0012186866, 0.0004245956, -0.027035829, -0.0009784065, -0.010067562, 0.019042626, 0.016235325, -0.001459831, -0.015557702, -0.002338841, 0.013448768, -0.012404674, 0.011906828, -0.053463176, -0.011291434, 0.0019913858, -0.029400598, -0.023993433, 0.018005446, -0.014396059, -0.025168903, -0.019236233, 0.019236233, 0.010378716, 0.0041487203, 0.0071703717, -0.03637045, -0.024159381, 0.025210392, 0.00023142081, -0.022071192, -0.007446953, -0.017244847], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-09-00850.pdf', 'citation': 'Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.8250469),\n",
      " NodeWithScore(node=TextNode(id_='75af58d4-ff08-4553-89fc-999ace72db77', embedding=[-0.013249093, -0.014452306, -0.019333912, -0.0058132373, 0.0110970605, 0.024875566, -0.014851085, -0.022276627, -0.020516498, -0.01971894, 0.019265156, 0.017670041, -0.0037712129, -0.0028327068, -0.006449221, -0.008766266, 0.022084113, -0.013716627, 0.008924401, -0.03203984, 0.0070886426, 0.006212016, -0.0066967392, -0.011406458, 0.010588273, 0.03886034, 0.017092498, -0.034102492, -0.023459213, 0.009935101, -0.0023376707, -0.0050466186, -0.027543262, -0.0066933013, -0.010244498, -0.020433992, 0.03344244, -0.016184932, 0.03297491, 0.0010648434, 0.01491984, 0.0009677269, -0.0066554863, 0.004293751, -0.0111245625, -0.009611952, 0.013310973, 0.0011456305, -0.014053526, 0.012760932, 0.02711698, -0.0045068916, -0.008965655, 0.013035952, 0.008333108, 0.008553125, 0.018192578, -0.005342265, -0.016652467, -0.016748723, -0.005785735, -0.012726555, -0.02685571, -0.009316306, -0.008903775, -0.009570699, -0.016033672, 0.011812112, 0.0046890923, 0.021822844, 0.022634152, 0.019017639, 0.004183743, -0.0045962734, 0.02983968, -0.026608193, -0.0301147, 0.01157147, -0.0013080643, 0.014562313, 0.01349661, -0.0082437275, 0.0026023777, -0.0058441767, 0.026649447, 0.0018684177, 0.044690765, 0.021671582, 0.006256707, -0.020447744, 0.033744965, 0.017725045, 0.023197945, 0.019072643, -0.023032932, 0.020915277, -0.017711293, 0.025631873, 0.009728836, -0.029949687, -0.029922185, 0.0011095342, -0.010278876, -0.012190266, -0.031269785, 0.01491984, 0.017711293, -0.014961093, 0.0116746025, -0.016927486, -0.013338475, 0.019773943, -0.0062051406, -0.057259183, 0.026580691, 0.0035408835, 0.043343164, -0.012059631, -0.015786152, -0.017601285, 0.014699823, 0.020296482, 0.008869397, -0.012142137, 0.024476787, 0.031874828, -0.0036096387, -0.040427953, -0.0018048193, -0.014204787, 0.026456932, 0.018728867, 0.027419502, -0.007652434, -0.018137574, 0.030967262, -0.04777099, -0.010753285, -0.0035151006, -0.018123824, -0.0047131567, 0.045653336, -0.012149013, -0.007879325, -0.020200225, -0.002016241, 0.013359101, 0.0076318076, 0.011550844, -0.0060710683, 0.023156691, -0.010911422, -0.003657767, 0.0143698, 5.5326305e-05, -0.020461494, 0.010416386, 0.027887037, -0.020035213, -0.033744965, -0.017230008, -0.0056241606, 0.02491682, -0.012616547, 0.018123824, 0.02026898, 0.019141398, 0.0007618916, 0.00050792773, -0.014686072, 0.005225382, 0.033469945, -0.016762475, 0.008381237, -0.013668498, 0.01725751, 0.009041285, 0.01204588, 0.017697543, -0.026388178, -0.0075217993, 0.0287396, 0.034625027, 0.025838137, -0.019127646, 0.012403406, 0.017876307, -0.017862555, 0.025961896, -0.035532594, 0.009055036, -0.0075217993, 0.011812112, 0.003398217, -0.6314461, -0.03875033, -0.004568771, -0.008683759, -0.0005281245, -0.0039671645, -0.000690988, 0.0006273896, 0.00013020482, 0.021410314, -0.00875939, -0.012465285, -0.012829687, -0.02369298, -0.028409574, -0.026071904, 0.014988595, -0.023074185, -0.0036852693, -0.014741077, -0.023610475, 0.00978384, 0.0073499116, 0.010051984, 0.010430137, 0.00040028314, 0.038310297, -0.017601285, -0.0047544097, 0.004960675, -0.032149848, 0.016817478, -0.005022554, -0.011619599, 0.042958137, -0.024119262, 0.017340016, 0.007679936, 0.021987855, 0.0014816708, -0.020502748, -0.017051246, 0.006806747, 0.024710555, -0.015799904, 0.027557014, 0.03368996, -0.0027003535, -0.0017240322, -0.011220819, -0.01575865, 0.01007261, -0.0023393896, 0.024298025, 0.015621141, 0.0030286587, 0.026264418, -0.020915277, 0.007762442, 0.00794808, -0.006449221, 0.016624965, 0.014699823, -0.03710021, -0.012788434, 0.034432516, -0.018687615, -0.010223872, 0.03135229, -0.01204588, 0.021410314, 0.014672321, -0.0002457992, -0.007068016, 0.0069889477, 0.04312315, 0.01349661, 0.011530217, -0.010017606, 0.029289639, -0.010038233, -0.004950362, -0.02532935, 0.009666956, 0.014466057, -0.009323181, -0.040565465, -0.003688707, -0.0055072773, 0.0003433454, 0.01589616, 0.022359133, -0.0016286345, -0.019663936, -0.013723502, 0.022964178, 0.0020059277, -0.011984, 0.031737316, -0.021836596, -0.007576803, -0.019512676, 0.032782394, -0.005469462, 0.019925205, -0.009336932, 0.0059473095, -0.01999396, 0.034790043, 0.002954747, -0.02395425, 0.0024064258, -0.02821706, -0.00057367474, -0.012960321, -0.037320226, 0.023527969, 0.010086361, -0.022015357, -0.008463743, 0.007611181, 0.0041871807, -0.014356049, 0.0055622812, 0.047330957, 0.0067311167, 0.02325295, -0.02546686, -0.012149013, -3.405522e-05, 0.0035580723, -0.011248321, 0.049036082, -0.02245539, 0.01875637, 0.020213977, 0.03941038, 0.0034446267, -0.0064526587, -0.03589012, -0.005964498, -0.002581751, 0.028299566, 0.0039018474, -0.009336932, -0.014424804, -0.0034790041, -0.0051738154, 0.012568418, 0.002306731, 0.018426346, 0.015772402, -0.0028137993, -0.0006574699, 0.009715085, -0.014837334, -0.014644819, -0.045790844, -0.009116916, -0.013132209, 0.0016363695, 0.018948885, -0.009983229, -0.004293751, -0.00557947, 0.002083277, 0.0047028433, 0.0028086426, -0.0055863457, -0.028024547, 0.010292627, -0.021479068, -0.028409574, 0.029124627, -0.0045515825, 0.029949687, -0.016308691, 0.018398844, -0.0212453, -0.015249864, -0.006105446, -0.0068136225, 0.018962635, 0.0015641768, 0.02421552, -0.0033260242, 0.03641266, -0.010148241, 0.03132479, 0.033662457, 0.0056551006, 0.0011215663, -0.0012332932, 0.019787695, -0.01589616, -6.520984e-05, 0.02997719, 0.013228466, -0.009543197, 0.013799133, 0.013180338, 0.0048541045, -0.0042353095, -0.035917625, 0.0076868115, -0.024600547, -0.009550072, -0.026484434, 0.015167357, 0.014122281, 0.006067631, -0.018000064, -0.031269785, 0.008683759, 0.0145210605, 0.0014713574, 0.0055210283, -0.006212016, -0.022744162, -0.00016383032, -0.0070370766, 0.0019853013, 0.014603566, 0.011028306, -0.019512676, 0.013111583, 0.0063907793, 0.013640996, -0.00080228515, -0.0041596787, 0.013324724, 0.023527969, 0.010602024, -0.0037540242, 0.06638985, 0.0014043213, 0.017147502, -0.017064996, 0.002901462, 0.0075355503, -0.014809832, 0.018838877, 0.029922185, -0.020557752, 0.015978668, -0.009866346, 0.012843438, 0.021561574, -0.01834384, -0.0070748916, -0.019457672, -0.0017575503, -0.0047165947, -0.0018735743, -0.0015762089, -0.021877848, -0.015841156, -0.0014352611, 0.044113223, 0.010375133, -0.010863293, -0.0066554863, -0.011289574, -0.011873992, 0.018632611, -0.0058097993, -0.0058338637, -0.008601253, -0.0315173, -0.019003889, -0.027598266, -0.0054900884, -0.0041218637, -0.013792258, 0.0072055263, 0.002127968, -0.0058991807, 0.006105446, 0.015538635, -0.0040806103, -0.008429365, -0.020956531, 0.012712804, 0.01943017, 0.007198651, -0.0021778154, 0.0013415824, -0.016817478, -0.0109458, -0.005249446, 0.014493559, 0.026745703, 0.00039512652, -0.005631036, -0.027130732, 0.0075493013, 0.013599743, -0.02575563, 0.009453815, 0.017670041, -0.00499849, 0.0039568515, -0.012334651, -0.010457639, 0.033854973, -0.011530217, 0.017092498, -0.0027725461, -0.012650924, -0.019732691, 0.0116814785, 0.0072192773, -0.0011181285, -0.006009189, -0.010636402, 0.002097028, -0.02821706, 0.017546281, 0.0017240322, -0.020035213, -0.0027691084, -0.036715183, -0.013647872, 0.0076318076, 0.046340883, 0.0342125, -0.006631422, 0.019127646, 0.031242281, 0.0071023935, -0.012307148, -0.018275084, -0.011543968, -0.03162731, -0.005500402, 0.00061492773, 0.015401125, 0.013276595, 0.007356787, -0.013056579, -0.003894972, -0.01149584, -0.023844242, -0.0145210605, -0.008374361, -0.008381237, 0.017601285, 0.006077944, 0.022565398, -0.016886232, 0.032507375, 0.028904611, 0.010065735, -0.014177285, -0.014686072, 0.0046787793, 0.014067277, 0.03107727, 0.0157449, -0.002624723, 0.0028481768, 0.008470619, 0.024903068, 0.012850313, 0.0026694138, 0.027983293, 0.0030200644, -0.013090956, -0.011385832, 0.009852595, 0.00013246085, 0.020956531, -0.008023711, -0.023211695, 0.034295004, 0.009536321, -0.0026281606, -0.023706732, 0.02271666, 0.021974105, -0.001484249, 0.013187213, -0.03721022, -0.0356426, 0.0006841125, -0.030444724, -0.014218538, 0.0021365623, -0.01738127, -0.004878169, 0.00095655426, -0.019333912, -0.02355547, 0.0032693013, -0.007638683, -0.026250668, -0.016514957, -0.0026161287, 0.00992135, -0.0030475664, 0.018591357, -0.019388916, -0.004878169, 0.019100145, -0.020048965, -0.017560033, 0.016886232, -0.032094844, 0.00794808, -0.013372852, -0.0040290444, -0.014149783, -0.017408771, 0.021987855, -0.012843438, 0.013675374, 0.014631068, 0.016363695, -0.00016705322, 0.022084113, 0.018852627, 0.0088075185, -0.019086394, -0.02491682, 0.0065936064, -0.023761736, -0.0070748916, -0.011440836, -0.0122590205, -0.0017721606, -0.0016286345, 0.015401125, -0.019457672, -0.010608899, 0.016446201, -0.013001574, -0.0029066186, 0.030472226, 0.047688484, 0.022276627, 0.01170898, -0.010952675, -0.00057238556, -0.010643277, 0.009508819, 0.0010132771, -0.0070198877, 0.0062498315, 0.008236852, 0.033359937, -0.011062683, -0.03066474, -0.030032193, 0.0047406587, -0.0064389077, 0.026058152, 0.00097288354, -0.018178828, -0.026759455, -0.012602796, 0.0198427, -0.008608129, -0.014617317, -0.028382072, -0.022194121, -0.009123791, -0.014204787, -0.00022689157, -0.0099626025, -0.037952773, 0.0071849, 0.0074942973, 0.0014240884, 0.01806882, -0.012644049, 0.009000032, -0.012925944, 0.008058089, -0.031022266, -0.016184932, 0.00026792972, 0.0015409719, 0.034955055, 0.024573045, 0.020901527, -0.011337703, 0.019746441, 0.02314294, 0.015291117, -0.011873992, -0.008449992, 0.0047991006, -0.01491984, -0.0023703293, 0.014424804, -0.014576064, -0.0010467952, 0.0021571887, 0.014672321, 0.021740338, 0.00045335342, -0.0036818313, -1.7000753e-05, -0.025398105, -0.039465383, -0.0057513574, -0.014074153, 0.014699823, -0.02465555, -0.027763277, 0.031379793, 0.012389655, 0.005940434, 0.010230747, 0.033854973, -0.024133012, 0.038502812, -0.0133797275, 0.024476787, -0.027419502, 0.015813654, -0.025040578, -0.0027845784, 0.034322508, -0.001719735, 0.024036756, 0.011763984, -0.0009978072, 0.013654747, 0.021699084, -0.0075424258, -0.033882476, -0.012905317, -0.0013759598, -0.021424064, -0.003410249, 0.01334535, -0.027983293, 0.0015022972, -0.026250668, -0.008725013, 0.019773943, -0.019512676, 0.0014481526, 0.035065062, 0.0028137993, 0.043838203, -0.02092903, 0.029262137, 0.017835053, 0.0009668675, -0.02532935, -0.00051179517, -0.012004627, 0.0019148274, 0.045130797, 0.031379793, -0.026099406, -0.037567742, 0.035367582, 0.026498185, -0.01143396, -0.04029044, 0.021011535, 0.005589783, -0.018550105, -0.01834384, -0.00027996185, -0.039795406, 0.025301848, -0.009357559, 0.017064996, -0.014005398, -0.006229205, 0.018412596, 0.00574792, 0.012087133, 0.029097125, 0.010306378, -0.015153606, 0.023087936, -0.014974844, -0.012575294, 0.012210892, 0.0052150683, 0.023307953, -0.02039274, -0.005455711, 0.0075493013, -0.00041811648, -0.0011860242, 0.010141366, -0.010739534, 0.038585316, -0.010794538, 0.013104707, -0.012699053, 0.0066348594, 0.024133012, -0.0047819116, 0.010657028, -0.012039004, 0.021520322, -0.017477527, 0.018976387, 0.01088392, 0.0057272934, -0.016886232, -0.0074117915, -0.001484249, -0.012430908, -0.014617317, 0.011268948, -0.026759455, 0.0055038393, -0.0013046266, 0.01738127, 0.026223166, 0.015579888, -0.00069270685, -0.023060434, 0.017367518, -0.033854973, 0.007879325, 0.011172691, -0.031929832, -0.021080289, -0.00061965466, -0.0057685464, -0.008291855, 0.0044931406, -0.03657767, -0.02356922, -0.0064285947, 0.0012083695, 0.04328816, -0.018275084, -0.027598266, 0.012513414, 0.0068033095, 0.014039775, 0.0056516626, 0.005830426, 0.044690765, -0.020764016, 0.009598201, 0.013592868, -0.0017687229, 0.03107727, 0.012513414, -0.014218538, 0.00082549, -0.020984033, -0.048981078, 0.0032830522, 0.019553928, 1.254242e-05, -0.01794506, -0.006212016, -0.021259053, -0.007893076, -0.006397655, 0.0023582974, -0.0046065864, -0.0128021855, 0.011805237, 0.0057376064, -0.01672122, -0.019113896, -0.012644049, -0.002953028, -0.014796081, -0.016377445, -0.012375904, -0.015194859, 0.029619664, -0.02491682, -0.023871744, -0.036027633, -0.013139085, -0.02983968, -0.0019457671, 0.019636434, 0.006157012, 0.021740338, -0.017188756, -0.005122249, -0.011619599, -0.014046651, 0.009577574, -0.011117687, -0.009653205, 0.010608899, 0.0067139277, 0.027034475, -0.0036508916, -0.019388916, 0.0032194539, 0.023074185, 0.0061020083, -0.014644819, 0.023307953, -0.030637238, 0.018013816, -0.007528675, -2.1875378e-05, -0.01047139, 0.00039276306, 0.025700627, -0.023239197, -0.025081832, 0.009357559, -0.013744129, -0.0025319036, -0.027309494, 0.0026625383, -0.010849542, 0.032259855, -0.017271262, -0.03685269, 0.015909912, -0.025659375, 0.028657094, -0.00814747, 0.0022895422, -0.004792225, 0.019471422, -0.015222362, -0.018165076, 0.027859535, -0.005424771, -0.00414249, 0.013111583, 0.014424804, 0.0009943695, -0.025494361, 0.008649382, -0.0015074539, 0.016831229, -0.005084434, -0.0099901045, -0.020475246, -0.0048953574, 0.031929832, -0.002182972, -0.041115504, 0.016391197, -0.017395021, -0.0046203374, 0.015153606, -0.0008718996, -0.004658153, 0.031984836, 0.026786957, -0.0043590683, -0.00773494, -0.012025253, 0.0075630522, -0.016789977, -0.005177253, 0.17821302, -0.023885494, -0.0014954217, 0.032892402, -0.0021881287, -0.0023015745, -0.002114217, 0.0045653335, 0.0069717593, -0.006019502, 0.032424867, -0.0007803695, 0.01629494, 0.0036852693, 0.013310973, 0.004919422, -0.017903809, -0.017546281, -0.01588241, -0.027199486, -0.011935872, -0.01943017, 0.0023015745, 0.00026255823, 0.014576064, -0.026360676, -0.015016097, 0.005648225, 0.008663133, -0.0057926103, -0.010278876, 0.007171149, 0.006751743, 0.0031438235, 0.00314898, 0.009460691, 0.007081767, 0.017450025, 0.022881672, -0.00229298, 0.011729606, -0.019086394, 0.00091873895, -0.013042828, -0.006906442, -0.0011946185, 0.008663133, 0.0032641448, -0.0037058957, -0.0059198076, -0.013152836, 0.0056241606, 0.001712, 0.020516498, -0.015043599, -0.041115504, 0.016954988, 0.014837334, -0.0032916467, 0.024724307, -0.016074924, 0.01465857, -0.0054797754, 0.027914539, -0.023087936, 0.027268242, 0.007040514, 0.016336193, 0.019966459, -0.013785382, 0.022661654, -0.015414876, -0.007693687, -0.024875566, -0.0116814785, -0.020172723, 0.023294201, 0.008704386, 0.026456932, 0.03600013, -0.0043418794, 0.0006273896, -0.02355547, -0.020420242, -0.007473671, -0.035120066, 0.03258988, -0.0067448677, -0.03503756, -0.012644049, -0.012540916, -0.012238394, -0.008339984, 0.004269687, 0.025645623, -0.0004185462, -0.013056579, 0.00012236246, -0.0009771808, -0.020475246, -0.030719744, 0.045625832, -0.0009840563, -0.012925944, 2.5581703e-05, 0.014768579, -0.0072674057, 0.006490474, 0.011427085, -0.021451566, 0.0044931406, -0.012437783, 0.004355631, -0.018838877, 0.012754057, 0.030829752, 0.007576803, -0.029949687, 0.027694523, -0.0027433254, -0.011860241, -0.010608899, -0.0029513093, 0.022826668, -0.020887775, -0.011942747, -0.02751576, -0.0037884016, -0.0038193415, -0.014947342, 0.005631036, 0.022744162, 0.022194121, -0.0033810283, -0.0034429077, -0.0116746025, -0.00428, -0.023665478, -0.029234635, 0.0011224258, -0.004915984, -0.009041285, 0.008979406, 0.003396498, -0.013792258, -0.017216258, 0.020076467, 0.00499849, -0.01163335, -0.03245237, 0.007968707, -0.019815197, -0.021204049, 0.013778507, 0.027900787, -0.01724376, -0.017188756, -0.046643406, -0.02603065, -0.012712804, -0.040592965, -0.012527165, 0.022826668, -0.00978384, 0.011990876, 0.014974844, -0.17579284, 0.021025285, 0.0018306024, -0.0073224097, 0.016886232, 0.020021463, 0.013317848, -0.013647872, 0.013049703, 0.0011611005, 0.03751274, 0.0072674057, -0.021080289, -0.016831229, 0.02667695, 0.026003148, -0.02325295, 0.017051246, 0.025810635, 0.01520861, 0.023404209, -0.022964178, 0.003367277, -0.013640996, 0.0026625383, 0.034707535, -0.017133752, 0.042435598, -0.030334715, -0.022551646, -0.019897703, 0.0105676465, 0.020076467, -0.014988595, 0.019017639, -0.0041218637, 0.005816675, 0.014576064, -0.0048403535, 0.0008938153, 0.037017703, 0.022262877, 0.0029736548, -0.016473703, 0.0076455586, -0.006432032, 0.01465857, -0.002526747, 0.013537864, -0.02435303, 0.0057376064, -0.0012453253, 0.011736482, 0.0038159038, 0.009673832, 0.0011593816, -0.019155148, 0.011619599, 0.019141398, -0.009735711, -0.0030286587, -0.003743711, 0.015937414, -0.01698249, -0.015263615, -0.022097863, -0.019980209, -0.007803695, -0.015662394, 0.005692916, -0.0001387992, 0.0070370766, 0.0030458474, 0.008319357, 0.0020798394, 0.017477527, -0.017023744, 0.015236113, 0.008415614, -0.0027141045, -0.0067001767, 0.021685334, -0.031819824, -0.0005298434, 0.023087936, -0.0004989036, 0.011598973, -0.00023634538, -0.0042868755, -0.00042305826, 0.0017523937, -0.022125365, -0.0009213173, -0.016748723, 0.024628049, 0.016652467, -0.0059576225, -0.0035305703, -0.005356016, -0.011791486, 0.025095582, -0.012967197, -0.021726586, 0.040455453, 0.0145073095, 0.0055210283, -0.04309565, 0.005397269, 0.050191164, 0.00862188, -0.033194926, -0.0139297675, 0.019911455, 0.007838072, 0.0027020723, 0.0077968193, -0.0053491406, -0.046203375, 0.008085591, -0.008388112, 0.048486043, 0.0041390522, -0.009481317, 0.040702973, -0.014397302, -0.02477931, -0.11105311, 0.014397302, -0.003425719, 0.011887743, -0.014122281, 0.009852595, -0.009213173, 0.006287647, -0.0014051808, 0.009838844, -0.0072674057, 0.0014481526, 0.0024390845, -0.0038434057, -0.021190297, 0.013462233, 0.004792225, -0.010051984, -0.0232667, 0.022070361, 0.017422523, -0.017312514, 0.0072467793, -0.003936225, 0.011970249, -0.005978249, -0.013599743, 0.017477527, 0.0046787793, -0.009687583, 0.008690635, -0.017573783, 0.01710625, -0.017023744, -0.02821706, -0.010120739, -0.0069648838, -0.02260665, 0.010560771, -0.031819824, -0.008566876, 0.009453815, 0.01410853, -0.013180338, 0.0040152934, -0.020846523, -0.020227727, 0.0066211084, 0.0012246988, -0.030059695, -0.01822008, -0.01848135, -0.03107727, -0.019196402, 0.02356922, -0.013668498, -0.010595148, 0.013723502, -0.009914475, -0.021575326, -0.0042146826, -0.0014859679, -0.03052723, 0.03657767, -0.0058269883, -0.016308691, 0.0007554458, -0.0396579, 0.016184932, -0.036027633, 0.0260169, 0.0017360643, -0.0062807715, -0.026800707, -0.01560739, -0.013029076, -0.022235375, -0.027405752, 0.014617317, -0.0006918474, 0.00049589557, -0.0198427, 0.0088212695, 0.011901494, 0.011323952, 0.017078748, -0.004166554, 0.019141398, 0.007226153, -0.020736514, 0.012623422, 0.029867182, -0.017422523, -0.02574188, -0.018041318, 0.027364498, -0.013599743, -0.033854973, 0.0037987148, 0.016212434, -0.0356151, -0.007906827, -0.07519049, 0.017161254, -0.0013261124, -0.025508113, -0.004517205, -0.0232667, -0.013187213, -0.002829269, 0.0011035181, 0.04064797, -0.016212434, 0.036550168, -0.022386635, -0.004902233, -0.029399646, -0.0026006587, 0.02587939, -0.015139855, 0.028712098, 0.028932113, -0.0013897109, 0.018261334, 0.01615743, 0.028547086, -0.016748723, 0.008030587, -0.0043040644, 0.03916286, -0.017848803, -0.016927486, 0.014672321, -0.019485174, 0.0016655904, 0.01575865, 0.010808289, 0.0047440967, -0.01013449, -0.003052723, 0.0045240805, 0.03448752, -0.02260665, -0.027694523, 0.004352193, -0.010045108, 0.017064996, 0.0031094458, -0.015016097, -0.022056611, 0.015194859, 0.005881992, 0.031737316, 0.00944694, -0.022221623, -0.034982555, 0.0053766426, -0.00911004, -0.0033518074, 0.01013449, 0.0028258313, -0.011179566, 0.021836596, 0.037732758, 0.020557752, 0.0067551807, -0.0038915342, -0.0049675503, -0.0063873413, -0.001835759, 0.0012307148, -0.059184324, -0.007026763, -0.010464515, -0.006270458, 0.015648643, 0.00516694, -0.0030733494, -0.00660392, 0.011812112, 0.011454587, 0.015978668, 0.019787695, -0.010581397, -0.010141366, -0.0052906987, 0.03613764, 0.0035546345, 0.010457639, 0.0052803857, -0.0071161445, 0.0025783132, -0.014149783, 0.0150848515, 0.0232667, 0.005445398, -0.022620402, 0.008498121, -0.0039155986, 0.0018701366, 0.03217735, 0.020874025, 0.005125687, 0.010842667, 0.007941205, -0.014589815, -0.008601253, -0.00026556628, -0.019498924, -0.020654008, 0.020997783, 0.02094278, 0.00027695383, 0.016776225, -0.01614368, 0.006476723, -0.017340016, 0.013015325, -0.019402668, -0.010045108, -0.030499728, 0.031407293, 0.015937414, 0.034790043, 0.0266907, -0.0067311167, 0.014039775, 0.004180305, 0.017147502, 0.012073382, 0.020736514, -0.014218538, 0.015442378, 0.023486715, -0.03685269, -0.012582169, -0.015222362, 0.014287294, 0.0042765625, -0.0025525303, -0.009474442, 0.07920579, 0.016446201, -0.0010098394, 0.024435535, -0.04502079, 0.010615775, 0.03737523, 0.027763277, -0.009275053, -0.0056275986, 0.0016896547, -0.007693687, 0.005593221, -0.033827472, -0.021176547, 0.010182619, 0.0036165142, 0.005469462, -0.016377445, 0.00045335342, 0.015456129, -0.0050156787, 0.027309494, -0.011042057, -0.022964178, 0.017738795, 0.02000771, -0.016033672, -0.00031390964, -0.050026152, 0.017161254, -0.01115894, -0.03220485, -0.008408739, 0.0157449, -0.0042731245, -0.020654008, -0.046780918, 0.05324389, 0.0020385864, 0.0039981045, -0.01520861, -0.025384353, -0.019251406, -0.004720032, -0.001063984, -0.025810635, -0.0049778637, -0.019801447], metadata={'file_name': '/workspaces/ml-learning/src/phages/data/test3/fmicb-08-02548.pdf', 'citation': 'Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, text='Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.8057611)]\n"
     ]
    }
   ],
   "source": [
    "pprint([n for n in response_aquery.selected_documents])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pprint([n.text for n in response_aquery.contexts])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of '\n",
      " 'Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in '\n",
      " 'Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.')\n",
      "('Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., '\n",
      " 'Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and '\n",
      " 'Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From '\n",
      " 'Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: '\n",
      " '10.3389/fmicb.2021.616548')\n"
     ]
    }
   ],
   "source": [
    "for cit in set([n.metadata['citation'] for n in response_aquery.contexts]):\n",
    "    pprint(cit)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: Mr. Speaker. Madam Vice President. Our First Lady and Second Gentleman.\n",
      "\n",
      "Members of Congress and the Cabinet. Leaders of our military.\n",
      "\n",
      "Mr. Chief Justice, Associate Justices, and retired Justices of the Supreme Court.\n",
      "\n",
      "And you, my fellow Americans.\n",
      "\n",
      "I start tonight by congratulating the members of the 118th Congress and the new Speaker of the House, Kevin McCarthy.\n",
      "\n",
      "Mr. Speaker, I look forward to working together.\n",
      "\n",
      "I also want to congratulate the new leader of the House Democrats and the first Black House Minority Leader in history, Hakeem Jeffries.\n",
      "\n",
      "Congratulations to the longest serving Senate Leader in history, Mitch McConnell.\n",
      "\n",
      "And congratulations to Chuck Schumer for another term as Senate Majority Leader, this time with an even bigger majority.\n",
      "\n",
      "And I want to give special recognition to someone who I think will be considered the greatest Speaker in the history of this country, Nancy Pelosi.\n",
      "\n",
      "The story of America is a story of progress and resilience. Of always moving forward. Of never giving up.\n",
      "\n",
      "A story that is unique among all nations.\n",
      "\n",
      "We are the only country that has emerged from every crisis stronger than when we entered it.\n",
      "\n",
      "That is what we are doing again.\n",
      "\n",
      "Two years ago, our economy was reeling.\n",
      "\n",
      "As I stand here tonight, we have created a record 12 million new jobs, more jobs created in two years than any president has ever created in four years.\n",
      "\n",
      "Two years ago, COVID had shut down our businesses, closed our schools, and robbed us of so much.\n",
      "\n",
      "Today, COVID no longer controls our lives.\n",
      "\n",
      "And two years ago, our democracy faced its greatest threat since the Civil War.\n",
      "\n",
      "Today, though bruised, our democracy remains unbowed and unbroken.\n",
      "\n",
      "As we gather here tonight, we are writing the next chapter in the great American story, a story of progress and resilience. When world leaders ask me to define America, I define our country in one word: Possibilities.\n",
      "\n",
      "You know, we’re often told that Democrats and Republicans can’t work together.\n",
      "\n",
      "But over these past two years, we proved the cynics and the naysayers wrong.\n",
      "\n",
      "Yes, we disagreed plenty. And yes, there were times when Democrats had to go it alone.\n",
      "\n",
      "But time and again, Democrats and Republicans came together.\n",
      "\n",
      "Came together to defend a stronger and safer Europe.\n",
      "\n",
      "Came together to pass a once-in-a-generation infrastructure law, building bridges to connect our nation and people.\n",
      "\n",
      "Came together to pass one of the most significant laws ever, helping veterans exposed to toxic burn pits.\n",
      "\n",
      "In fact, I signed over 300 bipartisan laws since becoming President. From reauthorizing the Violence Against Women Act, to the Electoral Count Reform Act, to the Respect for Marriage Act that protects the right to marry the person you love.\n",
      "\n",
      "To my Republican friends, if we could work together in the last Congress, there is no reason we can’t work together in this new Congress.\n",
      "\n",
      "The people sent us a clear message. Fighting for the sake of fighting, power for the sake of power, conflict for the sake of conflict, gets\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= \"Transcript: President Biden's Address to Congress.\" The New York Times, 24 Apr. 2024, www.nytimes.com/2024/04/24/us/politics/biden-speech-transcript.html.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "added = library.add(\"/workspaces/ml-learning/src/phages/data/test1/stateoftheunion.txt\")\n",
    "added"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[NodeWithScore(node=TextNode(id_='512a1f3c-ea8f-482a-bd6a-5a05cc169586', embedding=[-0.020383993, -0.010523272, -0.007901264, 0.009381429, 0.003897769, 0.00027554866, -0.007633425, -0.006512728, -0.028109048, 0.011918856, 0.046012, 0.015548785, -0.023386614, 0.008641347, -0.010678336, -0.0028687017, -0.002271349, -0.019495893, 0.008796412, -0.032817382, 0.008775267, 0.006502155, -0.015478302, 0.03653894, 0.008479234, -0.030872023, 0.018015727, -0.026769849, -0.008288926, -0.017057145, 0.0021074736, -0.019679151, -0.015295043, -0.010713578, -0.007795538, -0.022540806, -0.0038871963, -0.03216893, 0.0389354, -0.015548785, 0.009212268, -0.0043946817, -0.009395527, -0.0035206792, -0.029490534, 0.0030167182, 0.010628997, -0.021229802, -0.0066466476, 0.013159376, 0.014745267, 0.02194874, -0.016521465, -0.0025955758, -0.0004955911, 0.0028017417, -0.0072316653, 0.0047048116, -0.027883498, 0.0032986545, 0.01588711, -0.019834217, -0.003809664, -0.0013251005, -0.00703431, -0.031153958, -0.008584959, 0.0300826, -0.0010167327, 0.015675656, 0.02137077, 0.016154949, 0.0042501893, 0.0033338964, 0.024514358, -0.009029009, -0.020313509, 0.019848313, 0.0024334625, -0.0045955614, 0.002318926, -0.009769092, 0.00040858734, 0.0053285956, 0.019369023, 0.009444865, 0.013356731, 0.023302034, -0.029885245, -0.032704607, 0.024133746, 0.021384867, 0.031435896, 0.025035942, -0.006921536, 0.024880877, -0.010051028, 0.037948623, 0.025303781, -0.030223569, 0.017057145, 0.016126756, -0.02658659, -0.011531194, -0.03879443, 0.027502883, 0.008436943, -0.010015787, -0.004144463, -0.004133891, -0.0025074708, 0.004817586, -0.018833343, -0.03428345, -0.0077180057, 0.011763792, 0.008147959, -0.027347818, -0.019580474, -0.014491525, 0.011495952, 0.033578612, 0.008570863, -0.015591076, 0.011488903, 0.0077532474, -0.022117902, -0.03177422, -0.0059523797, 0.0034360983, 0.053144988, 0.010361158, 0.011397274, 0.0013515321, -0.015576979, 0.011510048, -0.024725812, -0.009078348, -0.019806024, -0.03481913, 0.024768101, 0.031379506, 0.01053032, 0.019890605, -0.012306519, 0.0078166835, 0.027404206, 0.005681016, 0.02614959, -0.0021744336, 0.021920545, -0.00893738, -0.0011286262, 0.027714336, -0.016507369, 0.018170793, 0.033296674, 0.019791927, 0.016479176, -0.027220948, -0.014096814, 0.041331857, 0.005973525, -0.009578785, -0.00550833, 0.029857052, 0.005888944, 0.00015936008, -0.012186696, -0.0075770374, -0.018156696, 0.0005392031, -0.022681773, 0.026769849, 0.006188501, 0.029180404, 0.014604299, 0.011087144, -0.022329353, -0.022512611, -0.02296371, 0.030505504, 0.033916935, 0.031802412, 0.009296848, -0.011897711, 0.035157453, -0.021835964, 0.007809635, -0.0113761285, 0.0261073, 0.011460709, 0.022244772, -0.014216636, -0.63243943, -0.026346944, -7.654129e-05, -0.007901264, 0.0019383118, 0.014449234, -0.011094192, 0.0007722411, -0.030646473, 0.03349403, -0.012708277, -0.0019788402, 0.006065154, -0.025162812, -0.025980428, -0.0014502098, -0.00831712, -0.0012343522, -0.00089470716, 0.007957651, -0.022258868, 0.010029883, -0.0061532594, 0.011467758, 0.000632154, 0.0042889556, 0.009578785, -0.006216695, 0.008197297, 0.016761111, -0.021709094, -0.005318023, 0.0194677, -0.012581406, 0.04505342, 0.0021233326, -0.03614423, 0.0038167124, 0.041331857, 0.0033779491, -0.00077973003, -0.028870275, -0.0041656084, 0.013434264, -0.0065902607, -0.008218442, 0.009874818, 0.0031717832, 0.012588454, -0.009212268, -0.022836838, 0.013948797, 0.0028299354, 0.0098325275, -0.006579688, -0.014562009, 0.04389748, -0.020496767, 0.020299412, -0.028813887, 0.016845692, 0.002981476, 0.0053462167, -0.014357605, -0.0067840917, 0.013991088, -0.019298539, 0.00053788157, 0.009705656, -0.0053426926, 0.0032827954, 0.031915188, 0.007999942, -0.02428881, 0.009021961, 0.013300343, 0.028306402, -0.0015700327, 0.0040422613, 0.02999802, -0.00068017124, -0.002773548, -0.015830722, 0.0050995224, -0.0055153784, 0.022568999, -0.024472069, 0.01256731, 0.008422846, -0.019101182, 0.01619724, 0.008542669, -0.015280946, -0.023682646, -0.0025691444, 0.015111784, 0.011587581, -0.004743578, 0.032450866, 0.004014068, -0.009317994, -0.006914487, 0.015816625, 0.02203332, 0.033916935, 0.02826411, -0.00827483, 0.02242803, 0.0185796, -0.009212268, -0.01818489, -0.00026783947, -0.028743403, -0.020031573, -0.02848966, -0.031661443, 0.0349601, 0.008993767, 0.0007251049, 0.015717948, 0.028010368, 0.00367222, -0.008958525, -0.022399837, 0.023132872, 0.012715326, 0.03684907, -0.026276462, -0.00969156, -0.007436069, 0.013695055, 0.012891537, 0.04460232, -0.0029638552, 0.011954098, -0.0056281528, 0.020694124, -0.011982292, 0.0067981887, -0.003999971, -0.009804334, 0.00014713549, 0.010784063, -0.0044052545, -0.014153201, -0.015450108, -0.0048986427, -0.002371789, -0.027643852, -0.01835405, -0.011453661, -0.020891478, 0.0065691154, 0.009973496, 0.0015259801, -0.011051902, -0.010502126, -0.01959457, 0.0059840973, -0.007929457, 5.236746e-05, 0.007605231, -0.01667653, 0.00183611, 0.025557524, 0.008549717, 0.0053285956, 0.023908196, 0.003099537, -0.030590085, 0.017057145, -0.011841323, -0.014660686, -0.011094192, 0.018621892, -0.0032792713, -0.032760996, -0.0172545, 0.0014334697, -0.005779694, -0.0033532795, 0.0367081, -0.017169919, 0.00088281295, 0.02323155, 0.017028952, 0.0132439565, -0.019975185, 0.0053039263, 0.0056563467, -0.007809635, 0.005194676, -0.02827821, 0.014463331, 0.009254558, 0.029744277, -0.013356731, 0.004662521, 0.012200792, -0.0073867305, 0.010685385, 0.01977783, 0.016535563, -0.010798159, -0.0036933653, -0.031661443, 0.00946601, -0.00911359, 0.012863343, 0.0025215675, 0.016916176, -0.023428904, -0.019439505, -0.00911359, 0.013765539, 0.023020096, -0.00955764, 0.009543543, -0.026177783, 0.009092445, -0.021384867, 0.004937409, -0.013814878, 0.0002581479, -0.030759247, 0.010537368, 0.00787307, 0.02813724, 0.0042501893, -0.0050008446, 0.020905575, 0.0007154133, 0.02645972, -0.0067277043, 0.015111784, -0.004733005, 0.008958525, -0.014759365, 0.022752257, 0.016535563, -0.011481855, -0.01385012, 0.024133746, -0.027982175, 0.021793675, -0.0001875537, 0.034001514, 0.02968789, -0.006220219, -0.0012334712, -0.009726802, -0.0071752784, -0.008105668, 0.0040563582, 0.011876565, -0.0010880978, -0.032535445, 0.030900216, 0.008401701, 0.029631503, 0.02176548, -0.009797285, -0.0059453314, 0.029180404, 0.017719695, -0.028151337, -0.030674666, 0.011432516, -0.017310888, -0.012250131, -0.028447371, 0.0136598125, 0.02428881, 0.0051206676, 0.01101666, -0.0030220044, 0.005980573, 0.008916235, 0.015069494, -0.002754165, -0.0247822, -0.022977807, 0.003880148, 0.021131124, -0.0019365498, 0.0026043863, 0.006932108, 0.005663395, 0.0003916271, 0.034227066, -0.008436943, 0.025176909, -0.015365527, 0.0017999869, -0.007922409, 0.021046543, 0.008972622, -0.0035982118, 0.0054660398, -0.004743578, -0.001783247, -0.0031277307, -0.0030202423, 0.014660686, 0.04623755, -0.026995398, 0.0062343157, -0.0012422818, -0.00060836563, -0.017381372, 0.025909944, -0.010988466, -0.020144347, 0.0044052545, 0.0022431556, 0.0037462283, -0.024049165, 0.0033656142, 0.034452613, 0.01818489, -0.0011744408, -0.042713344, -0.020905575, 0.016225433, 0.05083311, 0.023739034, 0.01619724, 0.0049092155, -0.009677462, 0.0073867305, 0.0031770694, -0.022822741, -0.009254558, -0.0035435865, -0.014477428, -0.0051911515, 0.017663307, -0.00023590137, 0.0047999653, -0.015125881, 0.005318023, -0.04141644, -0.00951535, -0.008253684, 0.009205219, -0.001975316, -0.00087928877, 0.023936389, 0.0058501777, -0.021088833, 0.017367275, 0.02013025, 0.03380416, -0.020144347, -0.00619555, 0.03907637, 0.02327384, 0.018551407, -0.0030554845, 0.017536437, 0.009874818, 0.0025991001, 0.007858974, -0.0053215474, 0.01154529, 0.003933011, -0.0064563407, -0.012088018, 0.011679211, 0.0058396054, 0.023569873, 0.029490534, -0.015802529, -0.026036816, 0.036623523, 0.0063611874, 0.012884487, -0.011559388, 0.0027101124, -0.00617088, 0.021173414, 0.012863343, -0.0190307, -0.017818373, -0.015732044, -0.031661443, -0.003756801, 0.002893371, -0.014632492, 0.006396429, -0.023132872, 0.0051382887, -0.025021845, 0.024584843, 0.013117085, -0.018692374, -0.027841207, 0.0058219843, 0.034903713, -0.008028136, 0.013666861, -0.011510048, -0.007746199, -0.018748762, -0.021737287, -0.036482554, 0.002200865, -0.015605173, 0.003568256, -0.0029215647, 0.0072598592, -0.010283626, -0.014900332, 0.008930332, -0.01119287, 0.004088076, -0.0015039538, 0.0076475213, 0.004725957, -0.0076968605, -0.0095364945, 0.001334792, -0.009381429, 0.0050642802, -0.008465136, -0.035777714, -0.014660686, 0.0016819261, 0.022146095, -0.00043215544, 0.014590202, 0.032281704, -0.0054096524, -0.025303781, 0.022131998, -0.02296371, 0.005233442, 5.333111e-05, 0.007640473, 0.022329353, 0.012137357, 0.016267722, 0.002163861, 0.000710127, 0.0036123085, -0.019340828, 0.011171725, 0.027404206, 0.013688006, 0.025783073, -0.019411312, -0.00955764, -0.01783247, 0.0022607765, -0.007108318, -0.0017347891, 0.009621075, -0.019185763, -0.024711715, -0.03605965, 0.012687133, 0.013638668, 0.004017592, -0.018706473, -0.0061356383, -0.030900216, -0.012510923, -0.0062096464, 0.007048407, -0.022850934, -0.007851926, 0.004814062, 0.008253684, 0.03222532, 0.011890663, 0.0035629696, -0.035354808, 0.0027999796, 0.0026960156, -0.033719577, -0.017804276, -0.0065550185, 0.019072989, -0.0039753015, 0.04023231, 0.005557669, -0.012560261, 0.034480806, 0.004902167, -0.00014944826, -0.008331217, -0.0072880527, -0.012954972, 0.0007145323, 0.014491525, 0.020595444, 0.00014526326, -0.0066607445, -0.020933768, -0.0005757667, 0.005881896, -0.013786684, -0.020496767, -0.030505504, -0.036820877, -0.0067277043, -0.023739034, 0.015774334, -0.028503757, -0.040006757, 0.027376013, -0.009712704, 0.010889789, -0.017804276, 0.030984797, -0.014576105, 0.019016601, 0.0172545, 0.00038656106, -0.020383993, 0.0031576862, -0.022794548, 0.014801655, 0.025952235, 0.024457972, 0.009007864, -0.015125881, -0.0058149355, 0.00911359, 0.001488976, -0.026361043, 0.0018043922, 0.0033197997, -0.025966331, -0.011051902, -0.03335306, 0.0020017475, -0.02544475, -0.0068580997, -0.013335586, 0.0017638638, 0.002526854, -0.008500379, -0.019961089, -3.287421e-05, -0.0053955554, 0.018509116, -0.010734724, 0.024091454, 0.030843828, -0.033860546, -0.036116038, -0.011594629, -0.008556766, -0.020327605, 0.023710841, 0.013984039, -0.02358397, -0.010988466, 0.02110293, 0.024542553, 0.010065125, -0.04756265, -0.007732102, -0.00078413525, 0.008345313, -0.008281878, -0.0075840857, -0.03527023, 0.024965458, -0.014139105, -0.0016828071, -0.006273082, 0.0014114435, -0.01464659, -0.012933827, -0.017536437, 0.021032447, 0.00893738, -0.002024655, 0.010445739, -0.0067840917, -0.010015787, -0.0115523385, -0.036426168, 0.010509174, -0.010121512, 0.005194676, -0.010361158, 0.0010352348, -0.01884744, -0.003205263, -0.01836815, 0.03552397, -0.060559914, -0.025106426, 0.017663307, 0.019904701, -0.0039400593, -0.0029638552, -0.023880003, -0.023302034, -0.02323155, -0.00070087594, 0.01119287, 0.002285446, 0.009606979, -0.026628882, -0.0027982176, -0.019298539, -0.011806082, -0.016648337, 0.002375313, -0.020341702, -0.011418419, 0.020947866, -0.0022308207, 0.011411371, 0.01787476, 0.005025514, -0.00041034943, 0.04857762, -0.025768975, -0.0016863314, -0.00679114, -0.00031409462, -0.031746026, -0.015633367, -0.02172319, 0.0037286072, 0.015703851, -0.029208599, -0.032930158, -0.0044827866, 0.002999097, 0.017578727, 0.02703769, 0.0030572466, 0.023950487, -0.008923283, 0.0016211336, 0.005074853, -0.00568454, 0.029124018, -0.00734444, -0.0018784004, 0.0062307916, -0.012327664, 0.014773461, 0.02871521, -0.011488903, 0.013138231, -0.04370012, -0.010537368, 0.012806956, 0.0061920253, -0.0021057115, -0.0140756685, -0.001109243, -0.030759247, -0.0056034834, 0.0125320675, -0.02017254, -0.015069494, -0.005381459, 0.020116154, 0.023513485, -0.0054202247, 0.0020898527, -0.022935515, -0.028856179, -0.029264985, -0.01760692, -0.028109048, 0.007323295, 0.021483544, 0.0036651716, -0.011439565, -0.029011242, -0.012095067, -0.013117085, 0.005592911, 0.01212326, 0.015732044, 0.025035942, 0.0062413644, -0.011284499, 0.026135493, -0.028644726, 0.0011911808, -0.032676414, -0.003933011, 0.0035788286, -0.0072528105, -0.00049294793, 0.007760296, 0.0026466767, 0.0025902896, 0.009148832, -0.009064252, 0.009155881, 0.029631503, -0.023344323, -0.0017286218, -0.0064492924, 0.0032951303, 0.015365527, 0.0013488889, 0.0057233064, -0.02411965, 0.002407031, 0.03233809, 0.010636046, -0.015196365, -0.02747469, -0.012905633, 0.0006876602, -0.010403449, -0.018128503, -0.014315315, -0.007009641, -0.025458846, 0.016394595, 0.01601398, -0.029434146, 0.013786684, -0.00033810327, -0.02331613, 0.008521524, 0.02468352, -0.0037427042, -0.013124133, -0.0071717543, -0.014315315, 0.008521524, 0.00955764, 0.029124018, -0.0047541503, -0.002889847, -0.018043922, -0.016916176, -0.03309932, -0.01216555, 0.021807771, -0.05100227, -0.039837595, 0.004958554, -0.016761111, 0.009769092, 0.0016070368, -0.023837712, -0.028743403, 0.03597507, 0.03233809, 0.0029850004, -0.009649269, -0.010621949, 0.02261129, -0.016775208, 0.020722317, 0.18461187, -0.0126096, 0.008126813, 0.03558036, -0.021737287, 0.002977952, 0.019538185, 0.0033162755, -0.019199861, 0.014082717, -0.009776141, 0.01611266, 0.014632492, 0.0006268677, 0.004327722, -0.006012291, -0.03907637, -0.019284442, -0.018791053, -0.016310014, -0.0145197185, -0.019453604, 0.010692433, -0.008211394, 0.028376887, -0.010593755, 0.0007510959, -0.001680164, 0.025289685, 0.0073726336, -0.035777714, -0.014089765, 0.010114464, 0.013138231, -0.008190249, -0.0034924857, -0.0012510922, -0.0032193598, 0.010579659, -0.022780452, 0.013991088, -0.023865907, -0.023908196, -0.018635988, 0.0013568184, 0.027996272, 0.015999883, -0.021004252, -0.0043841093, 0.021596318, -0.011629871, -0.01619724, 0.0069920197, 0.018819246, -0.003756801, -0.018100308, 0.010777014, 0.036341585, 0.0023171639, 0.021300286, 0.014956719, 0.022977807, -0.028602436, 0.030251762, 0.009226365, 0.027192755, 0.009712704, 2.822116e-05, 0.03109757, -0.02358397, 0.002063421, 0.0010466884, -0.012475681, -0.015901206, -0.016422788, -0.009092445, 0.044799674, 0.034452613, 0.009825479, 0.034142483, -0.021793675, -0.0017180492, 0.01406862, -0.002493374, 0.02884208, -0.04485606, 0.0143435085, -0.0043206736, -0.0049973205, 0.015732044, -0.019580474, -0.013215763, -0.025303781, 0.0029391856, -0.010480981, 0.010607853, -0.020947866, -0.005779694, -0.012623697, 0.008070426, -0.008662492, 0.042177666, 0.03171783, -0.00710127, 0.01008627, -0.03481913, 0.001749767, -0.008577911, 0.01181313, -0.011629871, 0.0139276525, 0.008155007, 0.0072316653, -0.00637176, 0.01070653, 0.016662434, -0.0040105437, -0.03949927, 0.021751383, -0.0022607765, -0.0050889496, -0.032535445, -0.012461583, -0.008486282, 0.0044052545, -0.01362457, -0.02848966, 0.0038026155, -0.023104677, -0.041303664, 0.01035411, 0.006301276, 0.028207725, 0.01597169, -0.015013107, -0.014392847, 0.004366488, -0.005666919, -0.02013025, 0.014745267, -0.002791169, -0.010036931, -0.006896866, -0.014632492, 0.007136512, -0.021342576, 0.028433273, -0.008584959, -0.0006510966, -0.009853673, -0.018241277, -0.0063541387, -0.012729423, 0.0044440203, 0.021427156, 0.016295917, -0.016718822, -0.014815751, -0.027460594, 0.015182269, -0.048634008, -0.0010273053, 0.019918798, -0.013631619, -0.0077814413, -0.020961963, -0.18246916, 0.03340945, 0.007985845, 0.017747888, 0.021243898, -0.009332091, 0.026967205, 0.00072246173, 0.0078166835, 0.0036369779, 0.0025867654, -0.006132114, -0.019101182, 0.014089765, -0.018128503, 0.01053032, -0.007640473, -0.018043922, 0.0015180507, 0.0070906975, 0.015139978, -0.03425526, 0.016253626, -0.02128619, 0.0043735364, 0.021835964, -0.0024828014, 0.049789947, -0.002754165, -0.035044678, -0.019481797, 0.003654599, -0.0022449177, -0.006893342, 0.015224559, 0.019326732, -0.009480108, -0.019791927, -0.0044757384, 0.037948623, 0.04167018, 0.0044722143, 0.002946234, -0.010903886, -0.0115241455, -0.00047180275, 0.014089765, 0.008415798, 0.019791927, -0.016761111, -0.0005973525, -0.039583854, -0.016169045, -0.013420166, 0.009550591, 0.019876508, -0.0060052425, 0.013286247, 0.014604299, 0.010600804, -0.0176915, -0.005906565, 0.013046601, -0.00012577, -0.0058396054, -0.011615775, -0.023414807, 0.0051171435, -0.019425409, 0.0033797112, -0.008303023, -0.017818373, -0.010593755, -0.011037805, 0.017804276, 0.0021761956, -0.048323877, 0.015929399, 0.006586736, 0.010791111, -0.005141813, 0.040993534, -0.017324984, -0.007009641, 0.0088246055, 0.013525893, -0.011629871, 0.011932953, 0.0027612133, -0.030251762, -0.0031118717, 0.00062818924, 0.006713608, -0.015548785, 0.034339838, 0.0022590144, 0.018058019, 0.0092404615, -0.0030167182, 0.00049471005, 0.026741656, -0.017860664, -0.017846566, 0.037638493, 0.027573368, 0.022907322, -0.029716084, 0.012080969, 0.033522222, 0.007464263, -0.036397975, 0.014773461, 0.010685385, 0.03614423, 0.0071470845, 0.0028017417, -0.015069494, -0.037666686, 0.007746199, -0.010854547, 0.040993534, 0.0038413818, 0.0065338733, 0.02252671, -0.01053032, -0.013060698, -0.0853703, -0.016126756, 0.0074783596, 0.028813887, 0.0064669135, 0.023034194, -0.007083649, 0.052552924, -0.0012440438, 0.016154949, 0.00073743955, -0.0125320675, 0.00955764, 0.021201609, 0.033747774, -0.009254558, -0.0047048116, -0.0036405022, -0.018043922, 0.016535563, 6.784092e-05, -0.025501136, 0.024979554, -0.015802529, -0.0025021844, -0.0021585745, -0.028503757, 0.041782957, 0.014660686, 0.007316246, 0.020595444, -0.014287121, 0.014047475, 0.0038413818, -0.010699482, 0.025966331, -0.034537192, -0.03498829, -0.0009982307, -0.04454593, -0.006604357, 0.007330343, 0.02141306, -0.022484418, -0.013716199, -0.037976816, -0.010502126, 0.019044796, -0.014970817, -0.02070822, -0.0043841093, 0.008859848, -0.021892352, -0.008021087, -0.008267782, 0.015999883, 0.0057197823, -0.011221064, -0.016817499, -0.029349566, -0.0043735364, -0.014258927, -0.022639483, -0.0030748674, 0.015929399, 0.0007977916, -0.028827984, -0.02362626, 0.017888857, -0.005853702, 0.006893342, 0.028870275, -0.009783189, 0.011009611, 0.008648396, -0.006706559, -0.031971574, -0.021892352, 0.008620202, -0.015224559, -0.0026713463, -0.010276577, -0.013934701, 0.021046543, 0.01375849, 0.023767227, -0.00679114, -0.018029826, 0.017367275, -0.0011356745, 0.002893371, 0.03738475, -0.004655473, -0.015168171, -0.0007881, 0.016056271, 0.0014739981, 0.014956719, -0.00026563683, 0.013631619, -0.04491245, -0.0029338994, -0.08018268, 0.012863343, 0.0066642687, -0.01818489, -0.014674783, 0.011714452, 0.005702161, -0.02929318, -0.009451914, 0.023569873, -0.036257006, 0.0001984567, -0.042459603, -0.005765597, -0.01004398, 0.005561193, 0.040908955, -0.025825363, 0.024824489, 0.03676449, 0.0138360225, 0.0037321316, 0.011263355, 0.008458088, -0.032507252, 0.027996272, -0.004637852, 0.023724938, 0.0121162115, -0.03879443, 0.00800699, -0.021398963, 0.012940875, 0.023287937, 0.0153796235, 0.0119681945, 0.0068580997, 0.020877382, 0.025994524, 0.031689636, -0.013025456, -0.023880003, -0.0010836925, -0.038089592, -0.0009436055, 0.019622765, -0.011686259, 0.008873944, 0.012377002, 0.014660686, 0.008584959, 0.021441255, -0.015407817, -0.014033378, 0.010114464, -0.0017347891, 0.027446497, 0.013145279, -0.008542669, -0.008789363, 0.037948623, 0.005547096, 0.015591076, 0.012764665, 0.014674783, -0.01499901, -0.0119681945, -0.0030079077, -0.015985787, -0.04124728, -0.0057303547, -0.026995398, 0.0017030713, 0.01526685, 0.015168171, 0.008859848, -0.028024467, 0.012059825, 0.011728549, 0.010699482, 0.03614423, -0.02199103, -0.0061250655, 0.02960331, 0.0019101183, 0.0349601, -4.4052544e-05, 0.00048545902, -0.0098325275, -0.008042232, -0.02929318, -0.0038308091, -0.0002517603, 0.007555892, 0.0023365468, 0.010593755, 0.0056316773, 0.015957594, 0.0069567775, 0.012595504, -0.006706559, -0.017155822, -0.015985787, -0.03636978, -0.024923166, -0.0043735364, -0.044517737, -0.026417429, 0.020849187, 0.009783189, -0.011235161, 0.022061514, 0.0058607506, 0.01818489, -0.045109805, 0.02128619, -0.0067840917, -0.010614901, -0.028207725, 0.036623523, 0.011418419, 0.01836815, 0.031182151, -0.02234345, 0.021004252, 0.014562009, 0.026346944, -0.0011814892, -0.009529446, 0.009374381, -0.006082775, 3.14425e-05, -0.028306402, 0.0012880964, -0.001975316, 0.0054801363, 0.0033145132, 0.013131182, 0.0046519483, 0.048126522, 0.030279957, -0.0038378576, 0.008761169, 0.00183611, 0.028813887, 0.0035242036, 0.012581406, -0.0036405022, -0.010290674, 0.016183142, -0.012285373, -0.013913555, -0.031435896, -0.018382246, 0.0032281703, 0.0031383033, 0.030223569, -0.026755752, 0.012461583, 0.009614027, 0.008479234, 0.023062387, -0.0016625429, -0.029913438, -0.0001060565, 0.01760692, -0.015210462, -0.0065338733, -0.048605815, 0.0010731199, 0.016803402, -0.016648337, 0.0018836868, 0.012320615, -0.008873944, -0.01097437, -0.0052052485, 0.020764606, -0.006593785, -0.00866954, -0.0057620727, -0.04136005, -0.010051028, -0.011982292, -0.006287179, -0.026473816, 0.00043325676, -0.048944138], metadata={'file_path': '/workspaces/ml-learning/src/phages/data/test1/stateoftheunion.txt', 'file_name': 'stateoftheunion.txt', 'file_type': 'text/plain', 'file_size': 43412, 'creation_date': '2024-01-07', 'last_modified_date': '2024-01-07', 'last_accessed_date': '2024-01-07', 'citation': '\"Transcript: President Biden\\'s Address to Congress.\" The New York Times, 24 Apr. 2024, www.nytimes.com/2024/04/24/us/politics/biden-speech-transcript.html.'}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, hash='7d83dc4c9fe677d569585c1a92e93e5f11fadd2e5647796870eed24eb0f454d4', text='\"Transcript: President Biden\\'s Address to Congress.\" The New York Times, 24 Apr. 2024, www.nytimes.com/2024/04/24/us/politics/biden-speech-transcript.html.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.8222056600000001),\n",
       " NodeWithScore(node=TextNode(id_='246155c0-df49-4228-be6c-d87a54efd6f1', embedding=[-0.014907048, 0.0013432145, -0.01851529, -0.031130973, -0.024322718, 0.007486446, -0.022966335, -0.0010074109, -0.012740785, -0.022452753, 0.025758116, 0.010857651, -0.033132628, -0.0037037167, 0.0058699795, 0.013879884, 0.0193976, -0.022821479, 0.018462617, 0.011265883, -0.0062156594, 0.008790154, 0.0066205994, -0.0081383, -0.01467001, -0.010963001, 0.018357266, -0.013577002, 0.01926591, -0.039242934, 0.0024905438, -0.004717712, -0.017791009, -0.014814867, 0.010344069, -0.00936958, 7.7057884e-05, -0.018291421, 0.024164693, -0.01959513, 0.009310321, -0.0012428027, 0.017501296, -0.021794315, -0.038136758, 0.0059094857, 0.0016296353, -0.02452025, -0.0020576203, 0.033791065, 0.029498046, -0.0045761475, -0.021083202, 0.008744064, -0.019753156, 0.020319413, -0.028233843, 0.021583615, 0.027101329, -0.020174555, 0.0026469228, 0.009336658, -0.019766325, 0.014511985, 0.0074271867, 0.008144884, 0.009856825, 0.002701244, -0.011134195, 0.0052082487, 0.020306244, 0.012668357, -0.015552318, -0.0084938565, 0.019516118, -0.00088477676, -0.025534246, -0.005626357, 0.0016773721, 0.004131702, 0.03855816, -0.032948263, -0.009665878, 0.005484793, 0.017856853, 0.007730068, -0.007960522, 0.024125187, -0.014116921, -0.016342444, 0.0053893193, 0.0035588602, 0.008796738, 0.0066798585, -0.011048598, 0.006074095, -0.010218966, 0.016763845, 0.00014516512, -0.030182822, -0.019107886, -0.012846136, -0.015038736, -0.009567112, -0.012832967, -0.009837071, 0.006959695, 0.0036148275, 0.02345358, 0.005448579, -0.016026393, 0.01851529, -0.008816492, -0.031447023, -0.0084938565, -0.004599193, -0.0074996147, -0.0049514575, -0.016197588, -0.022215715, 0.008000028, 0.02429638, 0.022110365, -0.01798854, 0.008586038, 0.0039045403, -0.048329387, -0.010567938, -0.0083490005, -0.018014878, 0.037847046, 0.006442821, 0.024691444, -0.011167117, -0.028734257, 0.015407462, -0.021241227, 0.018778667, -0.023348229, -0.019911181, -0.0015761371, 0.025613258, 0.015670836, -0.0055407602, -0.0037135934, -0.002373671, 0.013695521, 0.0038255278, 0.0019275787, -0.011713621, -0.0018617349, 0.005636234, 0.00492512, 0.009402502, 0.020082375, 0.017198414, -0.0031242908, 0.026903797, 0.0046650367, -0.009981927, 0.0008082333, 0.002786841, 0.0063440553, -0.034844566, 0.0069465265, 0.034949917, 0.023532592, 0.009586865, 0.0050798533, -0.026600916, -0.008862582, 0.013386055, -0.036161445, 0.016632156, -0.00015113222, 0.0051358202, 0.010844482, 0.032184474, -0.0061926143, -0.015723512, -0.027707092, 0.02322971, 0.007881509, 0.027970469, -0.002956389, -0.0027127666, 0.0072954986, 0.0028658537, 0.0072559924, -0.0077366526, 0.013017329, 0.03434415, 0.022663454, 0.014525154, -0.68941146, -0.01049551, 0.01001485, 0.0095934495, 0.00974489, 0.017711997, 0.010456003, 0.01959513, -0.005860103, 0.02398033, -0.0026551534, 0.012931732, 0.00093086745, -0.0135111585, -0.0086387135, -0.00985024, 0.008072456, -0.02130707, -0.019871674, 0.024335887, -0.01039016, 0.02623219, -0.023809137, -0.0058173044, -0.00072592846, 0.0051424047, -0.0001680047, -0.01650047, -0.014762191, 0.041007552, -0.02034575, 0.011331727, 0.009876578, 0.0008707849, 0.059996918, 0.017514465, -0.01605273, 0.007848587, 0.01076547, 0.039690673, -0.014143259, -0.017185245, 0.00430948, -0.0096988, 0.0019094717, 0.013083173, 0.008052703, -0.009422255, -0.006337471, -0.0045037195, 0.031842086, -0.0065020802, 0.018107058, 0.010330901, -0.007065045, -0.0102716405, 0.02130707, -0.0065712165, -0.0029284053, 0.013721859, -0.0032609168, 0.01809389, -0.0010995923, 0.005234586, -0.014432972, 0.01618442, -0.030709572, 0.0059720376, 0.0037168853, -0.005932531, 0.00088724593, 0.005175327, -0.025560584, -0.012174528, 0.013656015, 0.03584539, 0.017053558, -0.013735027, -0.00519508, 0.028628906, 0.020424763, 0.00803295, -0.019015705, -0.013458483, 0.017422283, -0.015921043, -0.032184474, -0.0045728553, -0.006083972, -0.00051399355, 0.03734663, 0.007921015, -0.01755397, -0.02731203, 0.027496392, 0.003983553, -0.005948992, 0.0046650367, 0.020740813, -0.0070452923, -0.004187669, 0.01039016, -0.007664224, 0.0076313023, 0.03842647, -0.00393417, -0.0046419916, 0.025758116, 0.017817346, -0.017922696, 0.0052708006, 0.00043703857, -0.028418206, 0.01381404, 0.0065909694, -0.029866772, 0.0025349883, 0.021030527, 0.024493912, -0.009672462, 0.032289825, -0.017000882, 0.028102156, 0.0010032956, -0.0017810762, 0.015907874, -0.0072691613, -0.008902089, -0.0095341895, -0.016579483, 0.01723792, -0.004938289, 0.01337947, -0.013550664, 0.013109511, -0.010666704, -0.0051292363, -0.015407462, 0.005860103, 0.0070716296, -0.0079736905, -0.007400849, -0.008296325, -0.0011893045, 0.01445931, -0.02623219, -0.023071686, -0.0052576317, 0.0052576317, 7.4434414e-05, 0.002939928, -0.010344069, -0.037451982, 0.02699598, -0.0066403523, -0.0049020746, 0.006614015, -0.024454407, -0.015038736, -0.029181994, 0.0047934325, 0.013563833, -0.015486474, 0.009949006, -0.0044675055, -0.032184474, -0.02355893, 0.020424763, 0.0011497983, -0.038452808, 0.003845281, 0.003247748, -0.011608271, 0.0049843797, -0.0026781987, 0.02280831, -0.01360334, -0.0043522785, 0.0040428126, -0.013471652, 0.015947381, -0.00701237, -0.01992435, 0.013471652, 0.010218966, 0.0014164658, 0.0023127652, 0.031262662, -0.014959723, 0.0023637943, 0.011924322, 0.0054419944, -0.012721032, -0.001950624, 0.002752273, -0.011437077, 0.009705383, -0.002205769, 0.017804177, 0.013748196, 0.050278366, 0.00663706, 0.028786931, -0.02581079, -0.0042403443, -0.024678275, -0.0060543423, -0.018870847, 0.017606646, 0.014182766, 0.003420588, -0.018554797, 0.0051720347, 0.016829688, 0.009507853, 0.033554025, -0.0038156512, -0.004671621, -0.01842311, 0.0037498074, 0.008697973, 0.00690702, -0.010054356, -0.008085625, 0.015697174, 0.028233843, 0.008546532, 0.044299744, 0.008651882, -0.010159707, -0.028576232, 0.006462574, 0.01435396, 0.004859276, -0.0021662628, 0.008065872, 0.011061767, -0.030393522, 0.030340847, -0.0052905534, -0.0055802665, 0.0074074334, 0.032289825, -0.025244532, -0.0054551633, 0.031209985, 0.018225579, -0.008737479, 0.0016337506, 0.04261414, -0.012115269, -0.0020905423, -0.017580308, 0.008790154, 0.009435424, -0.034844566, 0.0031242908, -0.002632108, 0.036266793, 0.025428895, 0.024006668, 0.008158053, 0.0036181198, -0.017211583, -0.00076996157, -0.021056864, 0.01162144, -0.008164638, -0.0008288095, 0.0013374533, -0.008487272, 0.0008477396, 0.015828863, -0.023519425, 0.011581933, -0.004000014, 0.0011333373, 0.0034831397, 0.0090140235, 0.004171208, -0.021991847, -0.022834647, 0.0017053557, 0.007776159, 0.0008362169, -0.030340847, -0.039637998, 0.002427992, 0.00089794555, 0.0036905478, -0.017791009, 0.0026650298, 0.024375394, -0.025981983, 0.0072954986, 0.00543541, 0.025995152, -0.0035851977, -2.6980548e-05, -0.012135022, 0.015157254, 0.00061975524, -0.009132543, -0.024941651, 0.010963001, -0.00037366385, -0.010502094, -0.023269217, -0.0018650271, -0.0003923882, -0.021451928, 0.001213996, -0.015618161, -0.010344069, 0.008961348, 0.0019456858, -0.010212381, 0.009837071, 0.029998459, -0.009422255, -0.006212367, -0.01809389, -0.023269217, -0.013116095, 0.048197698, 0.010956417, -0.0058173044, 0.004648576, -0.003384374, -0.006153108, -0.047249548, -0.022307897, 0.0023308725, -0.008164638, 0.0026288158, -0.007690562, -0.0038584496, 0.018133396, 0.025639595, 0.0055835587, 0.015657667, -0.022584442, -0.008987686, 0.010693042, -0.004237052, -0.0063835615, -0.0050008404, 0.016908702, 0.020925175, -0.002887253, 0.008276572, 0.0152889425, -0.006972864, -0.0076115495, 0.003967092, 0.0024000085, 0.019779492, 0.011983581, 0.017211583, 0.03178941, 0.014314453, 0.022294728, 0.006913604, -0.019502949, 0.0071572266, 0.009283983, -0.004032936, -0.01001485, 0.010153122, 0.013135849, -0.014077416, 0.012740785, 0.0028790224, -0.037530996, 0.032474186, -0.0017316933, -0.004869153, -0.015973719, -0.013893053, 0.016434625, -0.020095544, -0.011851894, -0.012115269, -0.004671621, -0.00010396127, -0.014630504, 0.0068345917, 0.00055514596, -0.026903797, -0.023637943, -0.03041986, 0.01992435, -0.01680335, 0.0013753135, -0.014432972, -0.02366428, -0.01456466, -0.002786841, 0.021162214, 0.010594276, -0.014261778, 0.0023209958, 0.010344069, 0.011990165, -0.009580281, -0.03286925, 0.0018304591, -0.0031341673, 0.008974517, 0.008559701, 0.008368754, 0.008322663, -0.012437903, 0.023400905, 0.0136691835, 0.010818145, 0.015236267, -0.015394293, 0.025942478, -0.0008559701, 0.008269988, -0.0054452866, -0.00803295, 0.0029876647, -0.002411531, -0.011529258, -0.0074996147, 0.0008625545, 0.0029975413, 0.008829661, 0.023269217, 0.026311204, -0.022163041, -0.016526807, 0.030709572, -0.02666676, 0.022860985, -0.007163811, 0.0010296332, 0.024704613, 0.011667531, 0.039216597, 0.0035292304, -0.016618988, -0.0070189545, -0.03268489, -0.0040757344, 0.021083202, 0.030024797, 0.0063703926, 0.0052082487, -0.018620642, -0.0055440525, -0.013642846, -0.016013226, 0.0132214455, -0.004256805, -0.018910354, -0.021623122, -0.012734201, -0.015776187, 0.0063506393, -0.003316884, -0.018054385, -0.019450273, -0.009494684, 0.009316905, -0.0047539263, -0.040085737, -0.039163925, -0.012062593, -0.00020370442, -0.0065745087, 0.009343242, -0.002003299, -0.000367491, -0.014867541, -0.017158907, 0.006205783, -0.012128438, 0.0030880766, -0.012220619, 0.032421514, 0.0132214455, 0.03437049, 0.009152295, 0.012944901, 0.013642846, -0.008401675, -0.013484821, -0.004375324, -0.0019901304, -0.015815694, 0.012444488, 0.036661856, 0.025902972, -0.007262577, 0.01370869, -0.002684783, -0.0028477465, -0.006271627, 0.008961348, -0.022149872, -0.017435452, -0.023374567, 0.0029695577, -0.002994249, -0.0006798378, 0.015868368, 0.0058403495, 0.016434625, 0.021280732, 0.01851529, -0.0037004245, 0.032263488, -0.02249226, 0.017277427, 0.00031666778, 0.010725964, -0.002737458, -0.0035292304, -0.0042140065, 0.0034271725, 0.012701279, 0.025033832, 0.020490607, -0.010409913, -0.010686457, -0.010383575, 0.02699598, -0.0026452767, -0.0035358148, 0.0054452866, -0.01819924, 0.005952284, -0.02388815, 0.008250235, -0.013010745, 0.002615647, -0.0065547554, -0.024006668, 0.017330103, -0.013879884, -0.029050307, -0.00530043, 0.0045794398, 0.04411538, 0.004717712, 0.013267536, 0.011884815, 0.0071703955, -0.008118547, 0.010311147, -0.01852846, -0.0061300625, 0.028154831, 0.014432972, -0.0042765583, -0.01007411, 0.009863409, 0.0021646167, -0.014472479, -0.029945783, 0.008401675, 0.022887323, -0.0065679243, -0.023190204, 0.012378644, -0.046880823, 0.02345358, -0.004184377, -0.012260125, -0.018317759, -0.0038518654, -0.022189379, 0.0076839775, -0.038136758, 0.019423936, 0.018581135, -0.016329275, -0.010133369, -0.00028271702, -0.018028047, 0.0019637928, -0.011937491, 0.015130918, -0.020161387, 0.0069465265, 0.02195234, 0.010363822, -0.028602568, -0.0006209898, 0.008460935, 0.014801698, -0.027101329, -0.005112775, -0.005952284, 0.010804976, 0.013827208, 0.007453524, -0.00861896, -0.0064197755, -0.009929253, -0.004934997, 0.019568793, 0.0045794398, 0.001437042, -0.014999229, -0.010060941, -0.024217369, -0.039374623, 0.0038782028, 0.0023209958, -0.01843628, -0.014446141, -0.012905395, -0.003486432, 0.017975371, 0.007828834, -0.015802525, 0.009106205, 0.03063056, -0.01851529, -0.0064164833, -0.0033662668, 0.0098239025, -0.02731203, 0.012826382, -0.005846934, -0.02579762, 0.00082798646, -0.018462617, -0.009758059, 0.0015086472, 0.015565487, 0.013695521, 0.0152889425, -0.00031851963, 0.028997632, 0.020688139, 0.0038683263, -0.008256819, -0.013577002, 0.027891455, -0.0160659, 0.00041728542, -0.0066403523, -0.0032921925, 0.0025251117, -0.0056033116, -0.0026535073, -0.006024712, -0.031763073, -0.015170423, 0.0011325142, 0.00782225, -0.0052740923, -0.003056801, -0.033896416, -0.015433799, -0.029076645, 0.010857651, 0.0113712335, -0.024125187, 0.013616509, 0.002528404, -0.0013621447, 0.0058666873, -0.006238705, -0.005846934, -0.027154004, -0.0062452895, -0.007130889, -0.014327622, -0.002668322, 0.02153094, 0.0012707864, -0.005158866, -0.018120227, -0.010376991, -0.027470054, -0.020069206, -0.014867541, 0.0054782084, 0.010462588, 0.015921043, -0.007130889, 0.023651112, 0.00755229, 0.009125958, 0.009856825, -0.021557277, 0.015091411, -0.0020987727, 0.012457657, 0.0077893278, -0.02591614, -0.003181904, 0.015552318, 0.01177288, -0.01064695, 0.017435452, 0.0062156594, -0.009619786, -0.012016503, -0.004721004, -0.0032724394, 0.007637887, 0.023716955, -0.010587691, 0.0051687425, -0.0013596755, -0.00028004212, -0.0007144058, -0.00031152373, 0.011851894, -0.011134195, 0.019015705, -0.018554797, -0.011167117, 0.0019786078, -0.021399252, 0.03300094, -0.019371262, 0.007940768, 0.03761001, -0.00041749116, -0.0016288123, -0.0019917765, -0.009132543, -0.012095516, 0.003111122, -0.0026370462, -0.020372087, -0.018567966, -0.0059292386, -0.009567112, -0.037083257, 0.005504546, -0.020846164, -0.005346521, 0.0022074152, 0.03147336, 0.013866715, -0.0319211, -0.011101274, -0.015025567, 0.012214035, 0.0027637957, -0.010844482, 0.0072954986, -0.0039868453, 0.0009670816, 0.020121882, -0.012767123, -0.008401675, 0.007710315, 0.014340791, 0.0038617419, 0.03265855, 0.21533573, -0.01819924, -0.0105613535, 0.03373839, 0.011667531, 0.016790181, 0.003726762, 0.003944047, 0.0010806622, 0.023374567, -0.020253569, 0.0064724507, -0.0212939, 0.0035786133, 0.0033234684, -0.0018617349, -0.019792661, -0.025376221, -0.036661856, -0.030393522, 0.0018436279, -0.02249226, -0.033764727, -0.021820653, 0.021886496, -0.0040592733, -0.015512811, -0.008816492, 0.03629313, 0.012325969, -0.002000007, -0.016434625, 0.011667531, 0.011799218, -0.01894986, -0.006874098, 0.02538939, -0.00447409, 0.041718666, 0.011575349, -0.010502094, -0.017093064, -0.0031786119, -0.006495496, -0.0049777953, 0.011884815, 0.0082041435, -0.033053614, 0.024388563, 0.026298035, -0.012299632, 0.010278225, 0.026745772, 0.03181575, -0.0013135849, -0.0027901332, 0.006205783, 0.010265056, -0.0063506393, -0.0008267519, -0.005234586, 0.039980385, 0.0010913619, 0.031842086, -0.017316934, 0.0027226433, -0.019726818, -0.0061070174, 0.0055177147, -0.021754809, -0.010693042, -0.005925947, 0.006939942, -0.0074074334, -0.024586095, -0.02376963, 0.017751502, 0.007196733, 0.026719434, 0.032395177, 0.0047539263, -0.0102584725, 0.012543254, -0.023493087, -0.039032236, -0.03210546, -0.011068352, 0.012253541, -0.005935823, -0.016434625, -0.0035423993, -0.02452025, -0.0024164694, -0.0011604979, 0.024770457, 0.026390215, 0.003641165, 0.015473305, -0.01851529, 0.015262605, -0.020200893, -0.01199675, 0.0024065927, -0.0024888976, 0.0030716157, 0.018278252, -0.010192628, -0.0022255222, -0.0026781987, -0.004836231, -0.012062593, -0.01510458, 0.009981927, -0.004220591, -0.0037991903, 0.010047772, -0.019884843, -0.009810734, 0.0046156542, -0.012009919, 0.005264216, -0.0109366635, -0.0066732746, -0.0029300514, -0.0031094758, -0.0037004245, -0.012701279, 0.00782225, 0.0072757457, -0.041586976, -0.000888892, -0.005504546, 0.017593477, 0.003539107, 0.00057531067, -0.006235413, 0.013478236, 0.0001653298, -0.023124361, 0.0031687354, -0.00053786195, -0.011759712, -0.009290568, 0.0013580294, -0.01001485, -0.0072954986, 0.03897956, -0.016408287, -0.014051078, 0.0016642034, -0.030762248, -0.01381404, -0.003743223, -0.017962202, 0.026495567, -0.01842311, -0.02826018, -0.029208332, 0.009560527, -0.002887253, -0.014604166, 0.0148412045, 0.0323425, -0.0120362565, -0.038268447, -0.022163041, -0.17151007, 0.022966335, 0.0076313023, -0.023190204, 0.024480743, 0.018673316, 0.029102983, 0.004921828, -0.002103711, 0.003249394, -0.0016625572, -0.013866715, -0.032000113, -0.010146538, -0.0009868348, -0.014577828, 0.012299632, 0.020595957, 0.03887421, 0.026390215, 0.02677211, -0.016842857, 0.020925175, -0.006887267, 0.0068082544, -0.014617335, 0.006235413, 0.028628906, 0.0015144086, -0.009659293, -0.018791836, -0.017593477, 0.020977851, 0.0072033172, 0.016513638, -0.009191802, 0.005211541, -0.030867597, -0.0018765498, 0.026614085, 0.026706267, 0.0126288505, 0.010943248, -0.022110365, -0.012378644, 0.006663398, 0.020503776, 0.0072691613, 0.01520993, 0.0066041383, 0.01671117, -0.011015676, -0.004460921, 0.005501254, 0.020740813, 0.0122667095, 0.0064856196, 0.024915313, 0.0073218364, -0.01734327, -0.006301257, -0.021767978, -0.00013775768, -0.00899427, -0.00030905457, 0.000290536, -0.009916084, 0.010126784, -0.01800171, 0.014590997, -0.0054551633, -0.029208332, -0.010791807, -0.032395177, 0.024217369, 0.004668329, -0.03063056, 0.021833822, -0.012253541, -0.0034732632, -0.00430948, 0.02226839, 0.0017695535, 0.010363822, -0.0092313085, 0.008171222, -0.0034963086, 0.009580281, -0.02720668, -0.014051078, 0.017949034, -0.014656842, -0.0013637908, -0.016658494, 0.004460921, 0.011588518, 0.0060082516, 0.014643673, -0.014749023, 0.0014148197, -0.004269974, 0.0067094886, -0.005497962, 0.00878357, 0.03105196, -0.0017201706, 0.02752273, 0.020003362, 0.018291421, -0.029603396, -0.025428895, 0.01435396, 0.020925175, 0.020819826, 0.015631331, 0.02773343, -0.011430493, -0.018594304, 0.002684783, -0.006337471, 0.03352769, -0.0018781959, -0.034449503, 0.0010921849, 0.003420588, 0.0055934354, -0.067002706, -0.03331699, 0.008006613, 0.01424861, -0.008487272, 0.029972121, -0.0015177007, 0.0025860174, 0.00017098825, 0.018107058, 0.010653535, -0.026877461, -0.01896303, -0.009382749, 0.018897185, -0.02354576, -0.011193455, -0.002479021, -0.020832995, 0.026100503, -0.0079012625, -0.011522674, 0.008428013, -0.00409878, -0.025744947, 0.004372032, -0.0130041605, 0.03684622, 0.014090585, -0.0003386843, -0.016658494, -0.0052740923, -0.0063177175, 7.0319176e-05, -0.0012979469, 0.0035226461, -0.04271949, 0.030893935, 0.011884815, -0.029498046, 0.041139238, 0.010429666, -0.006274919, -0.044484105, -0.0024576217, 0.00027037133, 0.0026255236, 0.0400594, 0.0034304648, -0.020069206, -0.019358093, -0.029181994, -0.026627254, 0.0049514575, 0.024599263, -0.00369384, 0.018712822, 0.039479975, -0.015460136, 0.00824365, 0.018778667, 0.0017744919, -0.02291366, 0.011766296, -0.0070057856, 0.00018672904, -0.017843684, 0.011338311, 0.023058517, 0.009112789, -0.016000057, 0.029208332, -0.009481515, 0.005998375, -0.028365532, 0.0005502077, -0.0045201806, -0.017870022, 0.014920217, -0.01768566, -0.037873384, -0.017975371, -0.006459282, -0.0057909666, 0.027575405, 0.013234614, -0.004293019, 0.0073942645, -0.0034436334, -0.030367184, -0.02226839, 0.026614085, 0.0121086845, -0.0070321234, 0.010265056, 0.001264202, -0.009119374, 0.0053662737, 0.013656015, 0.016263431, -0.01788319, -0.012306216, -0.0648957, 0.026627254, -0.013043667, -0.013945728, 0.015091411, 0.0026765526, 0.0075457054, -0.02055645, -0.0066963197, -0.01402474, -0.03210546, -0.0032345792, -0.00857287, 0.001095477, -0.033659376, -0.0059588687, 0.027970469, 0.012635435, 0.019621467, 0.014472479, 0.010442835, -0.029866772, -0.009316905, 0.006515249, -0.015894705, 0.0141037535, -0.025665933, 0.0041810847, -0.0052016643, -0.011641193, -0.00081276003, -0.03684622, -0.0020905423, 0.026653592, 0.008118547, -0.00067654555, 0.0008814846, 0.022755636, 0.0043489863, 0.030683234, -0.039269272, -0.017185245, 0.013425562, -0.0009827195, 0.0037135934, 0.0045563946, -0.00782225, -0.0036839636, 0.021056864, 0.007855171, -0.0012963008, 0.027496392, -0.016605819, -0.024572926, -0.019186899, -0.017382776, -0.0056000194, -0.0007333359, 0.0029596812, -0.033817403, 0.020530112, -0.008355585, 0.020398425, 0.0010740778, 0.0009193448, -0.0059193624, -0.017474959, -0.008763816, -0.0062189517, -0.021162214, -0.010034603, -0.011068352, 0.0004341579, 0.013307042, 0.025297208, 0.014011571, -0.01639512, 0.023967162, -0.036503833, 0.024112018, 0.016092237, 0.016118575, -0.022426415, 0.010758885, 0.032000113, 0.018291421, 1.1278332e-05, -0.00765764, -0.0040790266, 0.010297978, -0.03052521, -0.013465067, 0.0046782056, 0.021320239, -0.008414844, 0.0069794483, 0.015117749, 0.013307042, 0.030683234, 0.011983581, 0.01327412, 0.009020608, -0.014617335, -0.013642846, -0.012115269, -0.00034773783, -0.006528418, -0.03223715, -0.0021613245, 0.012029672, 0.0038057747, 0.018133396, 0.012516916, 0.014788529, -0.01681652, -0.0038716185, 0.020477438, -0.012800044, -0.041639652, 0.0340281, 0.00026769642, 0.0018995951, 0.02463877, -0.009632955, 0.019423936, 0.006120186, -0.0012518562, 0.004994256, 0.023387736, -0.005744876, 0.013642846, -0.005912778, -0.009343242, -0.027259355, -0.011469999, 0.007466693, 0.004961334, 0.030288171, -0.011009092, 0.07411384, -0.007604965, 0.0063835615, 0.01360334, 0.01917373, 0.02355893, 0.01863381, 0.018344097, -0.008928427, -0.033079952, -0.0031917808, 0.013201692, -0.0014642027, -0.027180342, -0.019318586, -0.008895504, -0.025573753, 0.0015242852, -0.011088105, 0.009112789, 0.01214819, 0.01022555, 0.01199675, 0.001555561, -0.01927908, -0.023190204, 0.009718552, 0.013326796, -0.02055645, -0.037530996, 0.002650215, 0.0024214077, -0.0022321066, -0.013037083, 0.0043028956, 0.009692215, -0.00840826, -0.0068609295, 0.015473305, 0.009343242, 0.0202404, 0.0042831427, -0.023269217, -0.03502893, 0.014235441, 0.00059053703, -0.030498872, -0.003420588, -0.010923495], metadata={}, excluded_embed_metadata_keys=[], excluded_llm_metadata_keys=[], relationships={}, hash='351b5b79b4cf0d96b24050f568933500d93160ec03e5e996aa31c29cf2d2f654', text='hello world', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.77306497)]"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ret = library.docs_index.as_retriever()\n",
    "ret.retrieve(\"Speech of Biden\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'VectorStoreIndex' object has no attribute 'todict'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[25], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m \u001b[43mlibrary\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mdocs_index\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mtodict\u001b[49m()\n",
      "\u001b[0;31mAttributeError\u001b[0m: 'VectorStoreIndex' object has no attribute 'todict'"
     ]
    }
   ],
   "source": [
    "library.docs_index.todict()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\"Transcript: President Biden's Address to Congress.\" The New York Times, 24 Apr. 2024, www.nytimes.com/2024/04/24/us/politics/biden-speech-transcript.html.\n"
     ]
    }
   ],
   "source": [
    "for i in library.documents:\n",
    "    print(i.metadata['citation'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(library.nodes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'citation': '\"Transcript: President Biden\\'s Address to Congress.\" The New '\n",
      "             'York Times, 24 Apr. 2024, '\n",
      "             'www.nytimes.com/2024/04/24/us/politics/biden-speech-transcript.html.',\n",
      " 'creation_date': '2024-01-07',\n",
      " 'file_name': 'stateoftheunion.txt',\n",
      " 'file_path': '/workspaces/ml-learning/src/phages/data/test1/stateoftheunion.txt',\n",
      " 'file_size': 43412,\n",
      " 'file_type': 'text/plain',\n",
      " 'last_accessed_date': '2024-01-07',\n",
      " 'last_modified_date': '2024-01-07'}\n"
     ]
    }
   ],
   "source": [
    "import pprint\n",
    "pprint.pprint(library.documents[0].metadata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'citation': '\"Transcript: President Biden\\'s Address to Congress.\" The New '\n",
      "             'York Times, 24 Apr. 2024, '\n",
      "             'www.nytimes.com/2024/04/24/us/politics/biden-speech-transcript.html.',\n",
      " 'creation_date': '2024-01-07',\n",
      " 'excerpt_keywords': 'Speaker, Vice President, First Lady, Second Gentleman, '\n",
      "                     'Congress, Cabinet, military, Chief Justice, Associate '\n",
      "                     'Justices, retired Justices, Supreme Court, Americans, '\n",
      "                     '118th Congress, Kevin McCarthy, House Democrats, Hakeem '\n",
      "                     'Jeffries, Senate Leader, Mitch McConnell, Chuck Schumer, '\n",
      "                     'Nancy Pelosi, progress, resilience, economy, jobs, '\n",
      "                     'COVID, democracy, crisis, possibilities, Democrats, '\n",
      "                     'Republicans, Europe, infrastructure law, veterans, '\n",
      "                     'bipartisan laws, soul of the nation, middle class, '\n",
      "                     'manufacturing jobs, pride, dignity, unemployment rate, '\n",
      "                     'Black workers, Hispanic workers, manufacturing growth.',\n",
      " 'file_name': 'stateoftheunion.txt',\n",
      " 'file_path': '/workspaces/ml-learning/src/phages/data/test1/stateoftheunion.txt',\n",
      " 'file_size': 43412,\n",
      " 'file_type': 'text/plain',\n",
      " 'last_accessed_date': '2024-01-07',\n",
      " 'last_modified_date': '2024-01-07',\n",
      " 'section_summary': 'The key topics of this section include the introduction '\n",
      "                    'of President Biden and the recognition of various '\n",
      "                    'political leaders. It also discusses the progress and '\n",
      "                    'resilience of America, the state of the economy, the '\n",
      "                    'impact of COVID-19, and the strength of democracy. '\n",
      "                    'Additionally, it highlights the ability of Democrats and '\n",
      "                    'Republicans to work together, the passing of bipartisan '\n",
      "                    'laws, and the importance of unity in Congress. The '\n",
      "                    'section also addresses the need to rebuild the middle '\n",
      "                    'class, create good-paying jobs, and prioritize the '\n",
      "                    'dignity and well-being of American workers.'}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(library.nodes[0].metadata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text:  1 Title: Nanopore sequencing of the phi X 174 genome  \n",
      " \n",
      "Authors: Andrew H. Laszlo1, Ian M. Derrington1, Brian C. Ross1, Henry Brinkerhoff1, Andrew  \n",
      "Adey2, Ian C. Nova1, Jonathan M. Craig1, Kyle W. Langford1, Jenny Mae Samson1, Riza Daza2, \n",
      "Kenji Doering1, Jay Shendure2, Jens H. Gundlach1,† \n",
      " \n",
      "Affiliations: 1Department of Physics, University of Washington, Seattle, WA 98195, USA  \n",
      "2Department of Genome Sci ences, University of Washington, Seattle, WA 98195, USA  \n",
      "†Corresponding author. E -mail: gundlach@uw.edu  \n",
      " \n",
      "Abstract : Nanopore sequencing of DNA is a single -molecule technique that may achieve long \n",
      "reads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we \n",
      "build on recent progress with respect to nanopore resolution and DNA control to interpret the \n",
      "procession of ion current levels observed during the translocation of DNA through the pore \n",
      "MspA. As approximately four nucleotides affect the ion current of each level, we measured the \n",
      "ion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer \n",
      "map is highly predictive of ion current levels of previously unmeasured sequences derived from \n",
      "the bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X \n",
      "174 up to 4,500 bases in length  that can be unambiguously aligned to the phi X 174 reference \n",
      "genome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and \n",
      "polymorphism detection. All methods and data are made fully available.  \n",
      " \n",
      "Main Text : DNA sequencing is revolutionizing biomedical and other life sciences research \n",
      "through its expanding scope (1) and has a rapidly growing presence in clinical medicine (2). \n",
      "These developments are driven in part by the successful completion of the Human Genome \n",
      "Project (3) and in part by the introduction of new sequencing technologies that have  dramatically \n",
      "reduc ed the cost of DNA sequencing (4). Although such ‘next -generation’ sequencing  \n",
      "technologies have matured considerably since  early proof -of-concept s (5-7), nearly all remain  \n",
      "limited to short sequence reads (with the exception of real -time sequencing from elongating \n",
      "polymerases (8)) and rely on  complex, expensive instrumentation. Most platforms are also \n",
      "limited with respect to speed and require extensive sample preparation steps prior to sequencing .  \n",
      " \n",
      "Nanopore sequencing, independently proposed by Church and Deamer in the mid -1990s, has \n",
      "tremendous potential to overcome these limitations and  achieve long reads, low cost, and high \n",
      "speed while requiring minimal sample preparation and instrumentation  (i.e. ‘tricorder’ -like DNA \n",
      "sequencing devices) (9-12). However, this promise has faced substantial technical challenges, \n",
      "such that despite nearl y 20 years of effort, nanopore -derived sequence reads that align to \n",
      "complex, natural DNA sequences have yet to be demonstrated.  \n",
      " \n",
      "In nanopore devices directed at DNA sequencing, a salt solution is divide\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.add(\"https://arxiv.org/ftp/arxiv/papers/1406/1406.4214.pdf\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: fmicb-13-938616 August 10, 2022 Time: 14:20 # 1\n",
      "TYPE Original Research\n",
      "PUBLISHED 16 August 2022\n",
      "DOI10.3389/fmicb.2022.938616\n",
      "OPEN ACCESS\n",
      "EDITED BY\n",
      "Hany Anany,\n",
      "Agriculture and Agri-Food Canada\n",
      "(AAFC), Canada\n",
      "REVIEWED BY\n",
      "Andrea Isabel Moreno Switt,\n",
      "Pontiﬁcia Universidad Católica\n",
      "de Chile, Chile\n",
      "Carlos E. Martinez-Soto,\n",
      "University of Guelph, Canada\n",
      "*CORRESPONDENCE\n",
      "Qiang Pan\n",
      "panqiang@phagepharm.com\n",
      "Huiying Ren\n",
      "renren0228@sina.com\n",
      "†These authors have contributed\n",
      "equally to this work\n",
      "SPECIALTY SECTION\n",
      "This article was submitted to\n",
      "Phage Biology,\n",
      "a section of the journal\n",
      "Frontiers in Microbiology\n",
      "RECEIVED 07 May 2022\n",
      "ACCEPTED 18 July 2022\n",
      "PUBLISHED 16 August 2022\n",
      "CITATION\n",
      "Liu W, Han L, Song P, Sun H, Zhang C,\n",
      "Zou L, Cui J, Pan Q and Ren H (2022)\n",
      "Complete genome sequencing of a\n",
      "Tequintavirus bacteriophage with\n",
      "a broad host range against Salmonella\n",
      "Abortus equi isolates from donkeys.\n",
      "Front. Microbiol. 13:938616.\n",
      "doi: 10.3389/fmicb.2022.938616\n",
      "COPYRIGHT\n",
      "© 2022 Liu, Han, Song, Sun, Zhang,\n",
      "Zou, Cui, Pan and Ren. This is an\n",
      "open-access article distributed under\n",
      "the terms of the Creative Commons\n",
      "Attribution License (CC BY). The use,\n",
      "distribution or reproduction in other\n",
      "forums is permitted, provided the\n",
      "original author(s) and the copyright\n",
      "owner(s) are credited and that the\n",
      "original publication in this journal is\n",
      "cited, in accordance with accepted\n",
      "academic practice. No use, distribution\n",
      "or reproduction is permitted which\n",
      "does not comply with these terms.Complete genome sequencing\n",
      "of aTequintavirus bacteriophage\n",
      "with a broad host range against\n",
      "Salmonella Abortus equi isolates\n",
      "from donkeys\n",
      "Wenhua Liu1†, Letian Han1†, Peng Song1†, Huzhi Sun2,\n",
      "Can Zhang1, Ling Zou1, Jiaqi Cui1, Qiang Pan2*and\n",
      "Huiying Ren1*\n",
      "1College of Veterinary Medicine, Qingdao Agricultural University, Qingdao, Shandong, China,\n",
      "2Qingdao Phagepharm Bio-tech Co., Ltd., Qingdao, Shandong, China\n",
      "Salmonella enterica subspecies enterica serovar abortus equi (S. Abortus equi)\n",
      "is the most common cause of abortion in mares. It has recently been found\n",
      "to cause abortion in donkeys more frequently in China. A novel virulent\n",
      "bacteriophage vB_SabS_Sds2 (hereafter designated as Sds2) was isolated from\n",
      "the feces of donkeys using a S. Abortus equi strain as a host. Phage Sds2\n",
      "had an isometric polyhedral head and an uncontracted long tail, belonging\n",
      "to the Tequintavirus, Markadamsvirinae, Demerecviridae, Caudovirales. The\n",
      "genome of phage Sds2 was 114,770 bp, with a GC content of 40.26%. The\n",
      "genome contained 160 open reading frames (ORFs), and no ORFs were\n",
      "associated with pathogenicity, drug resistance, or lysogenization by sequence\n",
      "analysis. Both genome annotation and phylogenetic analysis indicated that\n",
      "phage Sds2 was highly similar to T5-like bacteriophages. Phage Sds2 could\n",
      "lyse 100% (30/30) of S. Abortus equi strains, 25.3% (24/95) of other serotypes\n",
      "ofSalmonella strains, and 27.6% (8/29) of Escherichia coli strains using the\n",
      "double-layer agar plate method. The in vitro test showed that phage Sds2 had\n",
      "high ba\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Liu, Wenhua, et al. \"Complete genome sequencing of a Tequintavirus bacteriophage with a broad host range against Salmonella Abortus equi isolates from donkeys.\" Frontiers in Microbiology, vol. 13, 2022, article 938616. doi: 10.3389/fmicb.2022.938616.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.add(\"/workspaces/ml-learning/src/phages/data/test1/fmicb-13-938616.pdf\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
     ]
    }
   ],
   "source": [
    "nodes_engine = library.nodes_index.as_query_engine()\n",
    "response = nodes_engine.query(\"What do you know about nanopore sequencing?\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "Nanopore sequencing is a single-molecule technique for DNA sequencing that has the potential to achieve long reads, low cost, and high speed with minimal sample preparation and instrumentation. It involves the use of a nanopore device, which consists of a thin membrane with a nanometer-scale pore that separates two salt solution wells. When a voltage is applied across the membrane, ion current flows through the pore, and when single-stranded DNA enters the pore, it blocks some fraction of the ion current. The fraction of the ion current that is blocked depends on the identity of nucleotides within the pore. Recent advancements in nanopore resolution and DNA control have enabled single-nucleotide resolution and control of DNA translocation. However, despite nearly 20 years of effort, nanopore-derived sequence reads that align to complex, natural DNA sequences have yet to be demonstrated."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "display(Markdown(response.response))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[NodeWithScore(node=TextNode(id_='fcb50563-f2bb-4019-90e6-32c3f86f6c77', embedding=[-0.023770344, -0.016172517, -0.018628243, 0.005633924, -0.007645313, 0.021178942, -0.006203761, -0.0056576673, -0.040349886, -0.015901165, 0.0027966106, 0.015507706, -0.002874624, 0.012041198, 0.011898739, 0.0013414912, 0.011735928, -0.036225352, -0.002491341, -0.009639743, -0.032996275, -0.012366819, 0.010643741, -0.0025066044, -0.004677412, 0.0072654216, 0.023946721, -0.019659376, -0.034760058, -0.004785952, -0.0012185354, 0.011315335, -0.019808618, 0.0007881897, 0.019455863, -0.009972148, 0.015019275, -0.0022861317, 0.03391887, -0.03845043, 0.029007416, 0.0013915215, 0.010189228, 0.0032833465, -0.021178942, -0.018221216, 0.012346468, 0.008147312, -0.012509279, -0.007462151, 0.011993712, 0.01355398, 0.010589471, -0.00054694177, -0.023322614, -0.029143091, 0.014503708, -0.027514987, -0.0117020095, -0.013119819, 0.005386316, 0.0033342247, -0.029197361, 0.020921158, -0.028247634, 0.014557978, 0.015629815, 0.015019275, -0.0036598458, 0.013180872, 0.03359325, -0.005844221, 0.00082338054, 0.007903096, 0.017502137, -0.022155805, -0.0017553015, -0.0014050892, -0.012366819, 0.01936089, 0.01288917, -0.012007279, 0.004680804, 0.0037005486, 0.019279486, 0.018139811, 0.016226787, 0.017854892, 0.0058340454, -0.015032843, 0.049684357, 0.04037702, 0.014123817, 0.0159283, -0.012122603, 0.011776631, -0.009395527, 0.00801842, 0.0046909796, -0.027202932, -0.04287345, -0.0070279897, 0.0028542727, -0.009700797, -0.042629234, -0.011315335, 0.012305765, -0.021925157, 0.006393707, -0.00713653, -0.00641745, 0.021490995, -0.02638888, -0.023946721, 0.04371464, 0.0032833465, 0.029468713, -0.0053727487, 0.0014949741, -0.004233075, 0.04032275, 0.018370459, 0.010290985, -0.016457435, 0.03462438, 0.022481427, -0.019103106, -0.02683661, 0.012075117, -0.0169323, 0.02234575, -0.008181231, 0.009775418, 0.016389597, -0.025954718, 0.0023166586, -0.019727213, -0.015412734, -0.030825468, -0.06431018, 0.012482144, 0.0236211, -0.016674515, -0.0031442791, 0.017963432, 0.025791908, 0.013988141, 0.0015848591, 0.014042412, 0.0045451284, 0.004830047, 0.003385103, -0.0053998837, 0.012516063, 0.009897525, 0.015548409, -0.00040109063, 0.031042548, -0.0102231465, -0.01515495, -0.016389597, -0.0020622672, -0.013587899, 0.0015017579, 0.0055287755, 0.03041844, 0.020731213, 0.012204009, -0.019903593, -0.028356174, -0.027433582, 0.0016662644, -0.035438433, 0.0065395576, 0.0021368887, 0.0071093948, 6.439285e-05, -0.006770206, -0.0032273803, -0.024123099, 0.011227146, -0.0041448856, 0.027026555, 0.029088821, -0.037473567, 0.009117392, 0.03519422, 0.0055219918, 0.008493285, -0.017135812, 0.008737501, -0.012149739, -0.0009183534, 0.0046943715, -0.63083667, -0.035329893, -0.024964288, -0.032670654, -0.007984501, 0.014693654, 0.018166946, 0.019401593, -0.0042941286, 0.03014709, -0.017244352, -0.005962937, 0.009578688, -0.029957145, -0.03579119, -0.020934725, 0.011790198, -0.014842897, 0.030635523, -0.0010260458, -0.026510987, 0.0022606927, -0.0034275018, 0.018980999, 0.00068855303, -0.00408044, 0.0127670625, -0.014178087, -0.018492566, 0.0002219566, -0.036740918, 0.0030950967, -0.0009734715, -0.0007881897, 0.03207368, -0.011946226, 0.02018851, 0.03329476, 0.019116674, 0.021111103, -0.021545265, -0.007611394, -0.010094255, 0.009273418, -0.009164878, 0.0073400433, -0.0013881297, -0.0055864374, -0.004531561, -0.020894023, 0.0038531835, -0.0042432505, 0.0064547607, 0.019781483, -0.012183658, 0.0029882523, 0.024231639, -0.01963224, 0.03364752, -0.0033783193, 0.008791771, 0.0043212636, -0.00086450716, -0.028329039, -0.004941979, 0.012929873, -0.035546973, -0.000809389, 0.018560404, 0.008981716, 0.015751923, 0.036903728, -0.007529989, 0.0059595453, -0.008384745, 0.0318566, 0.009748283, 0.029658658, -0.0005494857, 0.0141916545, -0.0011795287, 0.0103588225, -0.028301904, 0.00052404654, 0.04186945, 0.017692082, -0.033946004, 0.019510133, 0.008696798, -0.016905164, -0.005688194, -0.005281168, -0.0012049678, -0.034895733, -0.010406309, -0.001290613, -0.01048093, -0.023851749, 0.0030408266, -0.038586106, -0.026565257, 0.009571904, 0.00092768105, -0.027012987, 0.046401013, -0.011003281, 0.01227863, -0.0011294984, 0.030282766, -0.026700933, -0.004467115, -0.013431872, -0.018438296, -0.006797341, -0.013038413, -0.031883735, 0.01659311, 0.02461153, -0.014802194, -0.019103106, 0.009151311, -0.013615034, -0.020568402, -0.01380498, 0.01792273, 0.015059978, 0.004616358, -0.0134250885, -0.009388743, -0.006169842, -0.008425447, -0.008852825, 0.018736783, -0.008513636, 0.0042093317, 0.018858891, 0.0013737142, 0.01653884, 0.0047011552, -0.036822323, 0.0074214484, 0.0070890435, 0.012760278, 0.0003353728, 0.005091222, -0.030554116, -0.024272343, -0.0074689346, 0.016226787, -0.01510068, 0.023254776, 0.009911093, 0.006654882, 0.03142244, 0.007930231, 0.013377601, -0.014707222, -0.036578108, -0.03280633, -0.02146386, 0.0050980058, 0.015534841, -0.01720365, -0.010521633, 0.02045986, -0.016565975, 0.031232495, 0.025954718, -0.0128349, -0.025954718, 0.028383309, -0.0008602673, -0.023675371, -0.017475002, -0.014625816, 0.035384163, 0.0107862, 0.004891101, -0.0036089676, -0.010107823, -0.013825331, -0.0068617873, 0.0036462783, 0.003714116, 0.005416843, 0.019279486, 0.033213355, 0.014286627, 0.02599542, 0.031775195, 0.0236211, -0.011885172, -0.0038226566, 0.026782338, 0.0047554253, 0.0048741414, 0.030906873, -0.008567906, -0.011159307, 0.019890025, 0.012468576, -0.0038871022, -0.0049657226, -0.025629098, 0.005118357, -0.0375007, 0.010969362, 0.009409094, 0.011613821, -0.009972148, -0.012855251, -0.020934725, -0.025072828, -0.016348895, -0.0030510023, 0.034163084, -0.009022419, 0.0077742045, -0.003924413, 0.0042466423, -0.0014898863, 0.02279348, 0.015439869, 0.015629815, -0.010820119, 0.01792273, 0.010033201, 0.0215724, 0.013771061, 0.0013321636, -0.014123817, -0.0052506407, 0.0022606927, 0.0054982486, 0.011593469, 0.025506988, 0.04442015, -0.003086617, 0.012848468, 0.010745497, -0.0034563327, 0.004402669, 0.004436588, -0.023607532, 0.006498855, -0.014042412, 0.013621817, 0.014245925, 0.005155668, -0.004250034, -0.0024506382, 0.008988501, -0.03381033, -0.010453795, -0.0028966714, -0.03364752, -0.011607037, 0.016579542, 0.020486997, 0.007251854, -0.00051344687, 0.0052947355, 0.007529989, -0.009904309, -0.00336136, -0.023783911, 0.016186085, -0.0064920713, -0.025927583, 0.01073193, -0.021382455, -0.01217009, 0.0076927994, -0.02090759, 0.0022369493, -0.00021326488, -0.00047783207, -0.0034665084, 0.020378456, -0.012373603, -0.008567906, -0.015629815, 0.021613102, 0.0124414405, -0.025344178, -0.04336188, -0.021287482, -0.01098293, -0.009443013, -0.011254281, 0.0044162367, 0.012041198, -0.0047520334, -0.0008564514, -0.045559824, -0.0055864374, 0.03722935, -0.023553262, -0.0013347075, -0.006319085, 0.01385925, -0.0059459778, -0.011457793, -0.010073904, 0.040132806, -0.0015416126, 0.00013260158, 0.00087001896, 0.012590684, -0.020392023, 0.015955435, -0.019008134, -0.0015526362, -0.00078182993, -0.014395168, 0.0065158145, -0.027691364, 0.012217577, 0.007021206, -0.019320188, -0.00057958864, -0.02101613, -0.011912307, 0.02854612, 0.041842315, 0.030961143, -9.385987e-05, 0.00611218, -0.00713653, -0.00037628744, -0.008961365, -0.011627388, 0.023173371, -0.0022132061, -0.0071433135, 0.01598257, 0.0064649363, 0.0014703829, 0.011403523, 0.008601825, -0.010949011, -0.0431448, -0.018763918, -0.007889529, -0.0047215065, -0.009599039, 0.015073545, 0.017000137, 0.0126042515, -0.014707222, 0.020894023, 0.014123817, -0.015317761, 0.003176502, -0.004579047, -0.013038413, 0.036252487, 0.023024129, -0.015399166, 0.005457546, 0.00882569, 0.0046672365, 0.024747206, 0.0096193915, -0.028953146, 0.0071229623, 0.007733502, 0.008839257, -0.001950335, -0.0060409503, -0.00082041265, 0.0143544655, -0.0055525186, 0.0101417415, 0.027460717, -0.024068829, -0.036008272, -0.010575904, -0.009788985, 0.018302621, -0.0048944927, -0.020826185, -0.012543198, -0.023458289, 0.0003122656, -0.038233347, 0.0055186, 0.00043967334, -0.028736066, -0.02549342, -0.019564403, -0.007129746, -0.03125963, -0.032996275, -0.0009565121, -0.030554116, -0.03568265, -0.006373355, 0.037310757, 0.007943799, 0.0070686922, 0.007577475, 0.020663375, 0.027840607, -0.020771915, 0.0071772328, 0.003893886, -0.024896448, 0.03343044, -0.0047520334, 0.015806193, -0.0043009124, -0.0046197497, 0.0025727462, 0.012651738, -0.009965364, 0.022820614, 0.040404156, -0.021721642, 0.024055261, 0.0026287122, 0.02279348, -0.01664738, -0.015277058, -0.00093276886, -0.032345034, -0.008472933, -0.011871604, -0.016335327, -0.017434299, 0.010616606, 0.008201582, -0.030581253, -0.023119101, 0.009158094, -0.02737931, 0.012142955, 0.009531202, 0.012807765, 0.03568265, -0.012129387, 0.0005940042, -0.02450299, 0.00020467918, -0.011491712, -0.0058611804, 0.029414443, 0.0025405232, 0.0020198687, 0.031938005, -0.0012159914, -0.016009705, 0.005264208, 0.010243499, 0.0048978846, 0.03142244, 0.012821333, -0.028111959, 0.01621322, -0.026293907, -0.0031035764, -0.0034868596, 0.024964288, -0.017285055, -0.005538951, 0.0067396793, 0.013771061, 0.02295629, -0.0062885582, -0.029197361, -0.021477427, -0.023363316, 0.016023273, 0.0046909796, -0.0047418578, 0.0016722002, -0.010324904, -0.026198935, -0.03153098, -0.024014559, -0.011084686, -0.020229213, 0.036686648, 0.004195764, 0.035872594, -0.020758348, -0.019198079, 0.034163084, 0.00981612, -0.0062275045, 0.0018384027, -0.0033783193, -0.007482502, 0.012563549, 0.034054544, -0.0075164214, -0.0086086085, -0.02588688, 0.015548409, 0.0006711696, 0.0062546395, -0.011661307, 0.01305198, -0.030988278, -0.03381033, -0.0015076937, -0.00616645, -0.017447866, -0.015846895, -0.01682376, 0.018248351, -0.018275486, 0.024163801, -0.00068770506, 0.035384163, -0.0070958273, 0.012733143, 0.007380746, 0.04425734, -0.013961006, -0.008479717, -0.0451528, 0.0056441, 0.027297905, 5.660211e-05, 0.018614676, -0.0011659611, -0.004606182, 0.0046672365, 0.0074417996, -0.008371177, -0.032670654, -0.0102774175, -0.015385599, 0.018506134, -0.012054766, 0.007428232, -0.056169648, -0.010107823, -0.006447977, -0.0032002453, 0.026307475, -0.008371177, -0.00760461, 0.025656233, 0.02522207, 0.008147312, -0.023539694, 0.040241346, 0.018709648, 0.008384745, -0.040621236, -0.009185229, -0.0030136914, 0.002369233, 0.05266922, 0.01626749, -0.009171662, -0.009002068, -0.008296555, -0.0043416154, -0.02527634, -0.04889744, 0.007075476, -0.00031883735, 0.0013279237, -0.0065531256, -0.00081023696, -0.020500565, 0.038965996, -0.0210704, 0.0051421006, -0.026266772, -0.023716073, -0.020486997, -0.012034414, 0.009008852, 0.010643741, 0.0040295618, -0.004935195, 0.0110439835, -0.004226291, 0.016023273, -0.016009705, 0.007401097, 0.029360173, -0.0037853457, -0.006892314, 0.002937374, 0.0077742045, 0.0055830455, -0.005240465, 0.000959056, 0.03728362, -0.035275623, 0.014042412, 0.004555304, -0.003476684, -0.00528456, 0.0047486415, -0.0029475496, -0.026918015, -0.0055186, 0.010399525, 0.002935678, 0.004005818, 0.00377517, -0.030554116, -0.0042364667, -0.012190442, -0.03381033, 0.0039040619, 0.028790336, -0.045397013, -0.010698011, -0.0031510629, 0.0013898256, 0.018478999, 0.013682871, 0.02434018, -0.020161375, 0.012455009, -0.017027272, 0.0049589383, 0.008113394, -0.034488704, -0.029495848, 0.00634622, -0.028898876, -0.004226291, 0.007937015, -0.01864181, -0.010684444, 0.00019111163, -0.014381601, 0.014937869, 0.0044264123, -0.001530589, 0.003178198, -0.0027406446, 0.008506852, -0.003598792, -0.019388026, 0.018940296, -0.017841324, 0.0019689903, 0.0041143587, -0.0037276836, -0.00023467618, 0.0012227752, -0.011573118, 0.01520922, -0.0025659625, -0.022549264, 0.0009548161, -0.004233075, 0.0042669936, 0.006169842, -0.010514849, -0.024964288, -0.015005708, -0.0022556048, -0.0036394945, 0.022427157, -0.021002563, -8.055731e-06, 0.021260347, -0.010284201, -0.02655169, 0.014842897, 0.012611035, -0.01217009, -0.030988278, 0.0072450703, -0.008669663, 0.025764773, 0.01598257, -0.0117562795, -0.010670876, 0.005840829, -0.024923585, -0.029658658, -0.01682376, 0.049738627, 0.014951437, 0.023960289, 0.02113824, 0.012020847, 0.00760461, -0.0023132667, -0.014897167, -0.0039481563, 0.021979427, -0.014707222, 0.016511705, 0.031069683, -0.029875739, 0.009531202, 0.017787054, 0.006241072, -0.019279486, 0.035329893, 0.008703582, 0.01687803, 0.0014924302, -0.002342098, 0.003931197, 0.027840607, 0.029007416, -0.034108814, -0.013988141, 0.006020599, -0.008601825, -0.022318615, 0.0086357435, -0.0047723847, 0.00027219893, -0.006068086, 0.010826902, -0.03291487, 0.012516063, -0.0005155668, 0.01610468, 0.0049250196, 0.003686981, 0.023688938, 0.0092666345, 0.0068380437, 0.0046875877, -0.012611035, -0.020975428, -0.0048232633, 0.011138956, 0.0010455492, -0.002041916, -0.012699224, -0.018872458, -0.016633812, -0.0043958854, 0.0074485834, -0.007638529, -0.023593964, 0.0110168485, 0.027786337, -0.012977359, -0.017678514, -0.012339684, -0.025181368, -0.0002726229, 0.009246283, -0.019062404, -0.056549538, 0.017882027, 0.0048944927, -0.0045756553, -0.023349749, -0.004585831, 0.0022132061, -0.0028559687, -0.013845682, 0.16704364, -0.016308192, -0.00056856504, 0.027867742, 0.012088684, -0.003805697, 0.020174943, -0.0019842538, -0.010650525, 0.0053218706, 0.0141373845, -0.0028339215, -0.023024129, 0.0012236232, 0.0225764, -0.0034631165, -0.03180233, -0.017610677, 0.0018468824, 0.011240713, 0.010345255, -0.019103106, 0.0006029079, 0.0010709882, 0.017977001, -0.0048673577, -0.02295629, -0.0014500316, -0.02218294, 0.019903593, -0.009829688, 0.036333892, 0.010094255, 0.01427306, -0.0070890435, 0.014883599, 0.019672943, -0.009022419, 0.008846041, 0.0052913437, -0.017732784, -0.0134522235, 0.0036801973, -0.010419876, 0.0052540326, 0.024530126, -0.009035987, 0.0030408266, -0.033321895, 0.018139811, -0.011593469, 0.014924302, 0.0029628132, 0.019374458, -0.040187076, -0.035546973, -0.005840829, 0.019767916, 0.0015695957, 0.02174878, 6.29619e-05, 0.005389708, -0.018655378, 0.015344896, -0.0071229623, 0.040078536, -0.024977855, 0.011267848, 0.027040122, -0.027623527, -0.0074892864, -0.00084076397, 0.007787772, -0.013655736, -0.028790336, -0.008506852, 0.05220792, 0.017217217, 0.034271624, 0.01319444, 0.016023273, 0.0009997586, -0.0050539114, -0.013533629, 0.018478999, -0.041679505, 0.009673662, -0.016253922, -0.009626175, -0.017773487, 0.0023624492, 0.002606665, -0.028030552, 0.0060613016, -0.0016696563, -0.023824614, 0.018560404, 0.020324185, -0.009022419, -0.0035580893, -0.010094255, 0.048164792, -0.030011415, 0.0020317403, -0.011118605, 0.010419876, -0.015222788, 0.044067394, 0.0069398005, -0.010833686, 0.019347323, -0.003863359, -0.00087001896, -0.024910018, 0.023865316, 0.045559824, 0.0048470064, -0.026985852, 0.04916879, -0.002698246, -0.034651514, -0.016199652, -0.018071974, 0.021531697, -0.017271487, -0.02135532, -0.0010421572, 0.011661307, -0.01775992, -0.0053523974, 0.006797341, 0.033783194, 0.00035826804, 0.0014059371, -0.028681796, -0.005359181, -0.018668946, -0.015127815, -0.010826902, -0.018940296, 0.0011422179, -0.017298622, 0.0072722053, 0.030988278, -0.01515495, -0.019808618, 0.012624603, 0.025927583, 0.0006321629, -0.009632959, -0.006068086, 0.010826902, 0.02151813, 0.009015636, 0.0024726854, -0.03353898, -0.029414443, -0.03174806, -0.025045693, 0.012556765, -0.0094565805, 0.0017357981, 0.005840829, -0.009409094, -0.013208007, 0.002783043, -0.17116818, -0.0010285897, 0.0023624492, -0.03014709, 0.022264345, 0.023146236, 0.025629098, -0.0030408266, 0.028274769, -0.0067532468, 0.018397594, 0.018126244, -0.0047181146, -0.008656095, 0.006505639, 0.014435871, -0.015141383, 0.010603039, 0.022861317, -0.0046875877, 0.004382318, -0.012319333, 0.03562838, 0.0028271377, 0.023051264, 0.023064831, 0.009992499, 0.023186939, 0.0013380994, -0.009205581, -0.0058917073, -0.019957863, 0.0236211, -0.018343324, 0.012265063, 0.0018994566, 0.019550836, -0.00623768, -0.008459366, 0.025303476, 0.032507844, 0.016131815, 0.017081542, -0.028681796, 0.0210704, -0.008289771, -0.0038226566, -0.027230067, 0.018546836, -0.020012133, 0.023078399, -0.016973002, 0.00995858, 0.010508065, 0.028898876, -0.0006788013, -0.00016683844, 0.027582824, -0.0019079363, -0.012617819, -0.026904447, 0.0024964288, -0.012095468, -0.006424234, -0.002493037, -0.028111959, -0.012543198, 0.017461434, -0.032100815, 0.01603684, -0.032345034, 0.002881408, 0.0043009124, -0.0017137509, 0.0068143005, 0.024313046, 0.0070958273, 0.018872458, 0.016023273, -0.0031527588, -0.012509279, 0.025045693, -0.011654523, -0.0078963125, 0.0073603946, 0.0009310729, -0.0141373845, 0.0031357994, 0.005342222, 0.02583261, 0.008947797, -0.010704795, 0.021423157, -0.00835761, 0.028329039, 0.00045196892, -0.0022386452, -0.0119190905, 8.0557315e-05, -0.011457793, 0.0055796537, -0.00087765075, -0.0012626299, 0.00992466, 0.035601243, 0.0007589347, 0.0061630583, 0.0101960115, 0.040892586, -0.01520922, -0.013377601, -0.017407164, -0.00020881303, 0.046916578, -0.004789344, 0.016457435, -0.008615392, -0.0032256844, -0.0033138734, 0.0037548188, 0.062139366, -0.009788985, -0.004378926, 0.025466286, 0.006068086, -0.03353898, -0.11765777, 0.0027660837, 0.008669663, 0.046211068, 0.002253909, -0.0007411273, -0.0043924935, 0.027202932, -0.044067394, 0.029550118, -0.011953009, -0.0070279897, 0.00016493049, 0.011912307, -0.013676087, -0.020283483, -0.0141916545, -0.012353252, -0.015670517, 0.02716223, 0.024123099, -0.0048470064, 0.015114248, 0.012821333, 0.0055593024, -0.0075706914, -0.027257202, 0.016688082, 0.030337036, 0.00755034, 0.0018791053, -0.00677699, 0.0055626943, -0.02140959, -0.014598681, -0.026850177, 0.003924413, -0.0021063618, 0.034651514, 0.0012227752, -0.012068333, 0.016050408, 0.007957366, 0.016253922, 0.0022708683, -0.018329756, -0.0045722635, 0.02069051, -0.03334903, -0.017990569, -0.017285055, -0.0019401592, -0.02079905, -0.029604388, -0.0010811639, -0.03424449, 0.0011820726, 0.01427306, -0.010704795, -0.020758348, -0.005162452, 0.014652951, -0.014747924, 0.008398312, 0.013635385, -0.000112038266, 0.011552767, -0.017366461, 0.012237928, -0.006020599, 0.003262995, 0.017434299, -0.0076249614, 0.008723933, -0.036713783, -0.0052302894, -0.024747206, -0.00092174526, 0.02035132, 0.020486997, -0.012536414, -0.018275486, 0.021368887, -0.032833464, 0.012957008, 0.020622673, 0.009700797, 0.029577253, 0.02196586, -0.022196507, -0.0065836525, 0.048924576, -0.022115102, 0.014734357, -0.006200369, 0.011294983, -0.023010561, -0.007197584, -0.0126042515, 0.004155061, -0.0126585215, -0.0029628132, -0.072342165, 0.015141383, -0.0060375584, -0.019672943, -0.0033545762, -0.021979427, 0.004830047, -0.015385599, 0.0063835313, 0.028220499, -0.029115956, 0.021504562, -0.023376884, -0.026863744, -0.033159085, -0.007672448, 0.0410554, -0.020988995, 0.00221999, 0.03381033, 0.0029797726, 0.010813335, 0.008852825, 0.027637094, -0.008154096, -0.007787772, -0.012421089, 0.028763201, 0.0079234475, 0.000929377, 0.008941013, -0.011430658, -0.0049284115, 0.0110439835, -0.009788985, -0.0013143561, -0.0060782614, -0.014598681, 0.018004136, 0.015358464, -0.020500565, -0.03429876, 0.006173234, 0.019008134, 0.009103824, 0.010847254, -0.012719575, -0.00020425518, 0.0038124807, -0.002013085, 0.033946004, 0.018804621, -0.0063292608, -0.012895954, -0.025696935, -0.0017603893, -0.0087714195, -0.011946226, 0.013079115, -0.016511705, 0.045776904, 0.01991716, -0.00073307153, -0.013961006, -0.0074689346, -0.02252213, -0.028871741, -0.0017603893, -0.029170226, -0.02727077, -0.012624603, -0.0046672365, 0.005664451, 0.0256698, -0.000959904, -0.028980281, -0.013011278, -0.016945867, -0.018261919, -0.0043585747, 0.0030476104, 0.024408018, -0.0112067945, -0.003357968, 0.019659376, -0.0022386452, 0.008703582, 0.011640956, -0.011247497, 0.010921876, 0.0017977001, 0.02511353, 0.012292198, 0.00647172, 0.011573118, 0.003237556, 0.013221575, 0.01263817, 0.018980999, 0.008601825, 0.0015390686, 0.029740063, 0.011227146, -0.010636957, -0.0050233845, 0.041516695, -0.026036123, -0.019672943, 0.028980281, -0.01305198, -0.0066718413, -0.004531561, -0.0076249614, 0.010582687, -0.035329893, 0.03348471, -0.014680087, -0.02218294, -0.02135532, 0.024028126, 0.0039379806, 0.029197361, 0.00041423418, -0.020785483, 0.018655378, -0.008805338, 0.044664364, 0.0118919555, 0.03391887, -0.02522207, 0.015738355, 0.01278063, -0.027569257, -0.013587899, 0.011966577, 0.019198079, -0.011824118, 0.009008852, -0.02229148, 0.07478432, 0.029088821, -0.02705369, 0.013703222, -0.02090759, 0.02180305, 0.012665305, -0.0023488817, -0.013459007, -0.024787908, 0.02849185, -0.01725792, -0.0014500316, -0.04431161, 0.004711331, 0.0251678, 0.0028627524, 0.023729641, -0.019428728, -0.011098254, 0.016783057, -0.016810192, 0.0064004906, -0.000569413, -0.028193364, 0.01603684, 0.0069431923, -0.0029746848, 0.03237217, -0.042249344, 0.009565121, -0.017977001, -0.046509553, -0.014028844, 0.0070347735, 0.019984998, 0.008086259, -0.01765138, 0.03158525, 0.023159804, -0.019944295, 0.017542839, -0.012868819, -0.019225216, 0.013574331, -0.0034291977, -0.01084047, 0.019754348, -0.0043924935], metadata={'file_name': 'data/1406.4214.pdf', 'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024.'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={<NodeRelationship.SOURCE: '1'>: RelatedNodeInfo(node_id='f000ec98-9631-440a-8f6c-4181172bc173', node_type=<ObjectType.DOCUMENT: '4'>, metadata={'file_name': 'data/1406.4214.pdf', 'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024.'}, hash='1cfd3cd76f12c5dae17e07638b6b4584e3c04282aae51284801629e0db25bb45'), <NodeRelationship.NEXT: '3'>: RelatedNodeInfo(node_id='29251e28-f9ea-443c-a9b8-0dc89102e06b', node_type=<ObjectType.TEXT: '1'>, metadata={}, hash='1d732c8ed00fc6c1a785480fefb3d6862eb577d49b8ab693c1c123cf167b49e9')}, hash='a0107c8821ed081319de3ddb37dc2023d52c0b2413f4a824cdefbc2378ee4998', text='1 Title: Nanopore sequencing of the phi X 174 genome  \\n \\nAuthors: Andrew H. Laszlo1, Ian M. Derrington1, Brian C. Ross1, Henry Brinkerhoff1, Andrew  \\nAdey2, Ian C. Nova1, Jonathan M. Craig1, Kyle W. Langford1, Jenny Mae Samson1, Riza Daza2, \\nKenji Doering1, Jay Shendure2, Jens H. Gundlach1,† \\n \\nAffiliations: 1Department of Physics, University of Washington, Seattle, WA 98195, USA  \\n2Department of Genome Sci ences, University of Washington, Seattle, WA 98195, USA  \\n†Corresponding author. E -mail: gundlach@uw.edu  \\n \\nAbstract : Nanopore sequencing of DNA is a single -molecule technique that may achieve long \\nreads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we \\nbuild on recent progress with respect to nanopore resolution and DNA control to interpret the \\nprocession of ion current levels observed during the translocation of DNA through the pore \\nMspA. As approximately four nucleotides affect the ion current of each level, we measured the \\nion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer \\nmap is highly predictive of ion current levels of previously unmeasured sequences derived from \\nthe bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X \\n174 up to 4,500 bases in length  that can be unambiguously aligned to the phi X 174 reference \\ngenome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and \\npolymorphism detection. All methods and data are made fully available.  \\n \\nMain Text : DNA sequencing is revolutionizing biomedical and other life sciences research \\nthrough its expanding scope (1) and has a rapidly growing presence in clinical medicine (2). \\nThese developments are driven in part by the successful completion of the Human Genome \\nProject (3) and in part by the introduction of new sequencing technologies that have  dramatically \\nreduc ed the cost of DNA sequencing (4). Although such ‘next -generation’ sequencing  \\ntechnologies have matured considerably since  early proof -of-concept s (5-7), nearly all remain  \\nlimited to short sequence reads (with the exception of real -time sequencing from elongating \\npolymerases (8)) and rely on  complex, expensive instrumentation. Most platforms are also \\nlimited with respect to speed and require extensive sample preparation steps prior to sequencing .  \\n \\nNanopore sequencing, independently proposed by Church and Deamer in the mid -1990s, has \\ntremendous potential to overcome these limitations and  achieve long reads, low cost, and high \\nspeed while requiring minimal sample preparation and instrumentation  (i.e. ‘tricorder’ -like DNA \\nsequencing devices) (9-12). However, this promise has faced substantial technical challenges, \\nsuch that despite nearl y 20 years of effort, nanopore -derived sequence reads that align to \\ncomplex, natural DNA sequences have yet to be demonstrated.  \\n \\nIn nanopore devices directed at DNA sequencing, a salt solution is divided into cis and trans  \\nwells  by a thin membrane. A singl e nanometer -scale pore in the membrane connects the cis and \\ntrans  wells electrically. When a voltage is applied across this membrane, ion current flows \\nthrough the pore; this current provides the primary signal. DNA is negatively charged and is \\nelectrophoretically attracted into the pore. When single -stranded (ss) DNA e nters the pore, it \\nblocks some fraction of the ion current. The fraction of the ion current that is blocked depends on \\nthe identity of nucleotides within the pore (13-15). Key challenges of this technique are single - 2 nucleotide resolution and control of the DNA translocation. Single -nucleotide resolution was \\nrecently enabled through the development of MspA , a protein pore with a short and narrow \\nconstriction (11, 13, 15, 16 ). DNA translocation control was also recently enabled through the \\nuse of molecular motors such as phi29 DNA Polymerase (DNAP) (11, 17 ) (Fig. 1 ). \\n \\nWe have found t hat the currents in the MspA pore are determined by about four nucleotides at \\nany given time (11, 15 ).', start_char_idx=1, end_char_idx=4095, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.87023771),\n",
       " NodeWithScore(node=TextNode(id_='9fe90d42-9f20-4967-b144-96163b8329ee', embedding=[-0.023894642, -0.0154500585, -0.018124629, 0.0059532956, -0.0083563365, 0.020907812, -0.0070054745, -0.004497216, -0.039208934, -0.0162918, 0.0029630715, 0.0150156105, -0.0025914146, 0.010847624, 0.011540026, 0.0022621844, 0.011092002, -0.036140647, -0.0023894643, -0.010216317, -0.03247499, -0.011859074, 0.010521788, -0.002109449, -0.004531157, 0.006587997, 0.024152596, -0.020378329, -0.03426709, -0.0060551194, -0.002413223, 0.010854413, -0.019794539, 0.0012940103, 0.02029687, -0.009686833, 0.0147576565, -0.0024709231, 0.035163138, -0.036059186, 0.027994746, 0.00073355535, 0.010487847, 0.0034348548, -0.019672351, -0.017133545, 0.012619358, 0.007691088, -0.0122324275, -0.005688554, 0.012524323, 0.013080959, 0.010515, 0.0007819216, -0.024451278, -0.029786844, 0.015993118, -0.027424533, -0.011220979, -0.012958771, 0.00649975, 0.004419151, -0.029623926, 0.020147528, -0.027994746, 0.014214597, 0.017092815, 0.014947728, -0.0051760413, 0.0131149, 0.032339226, -0.005335565, 0.00076919363, 0.00671358, 0.016196765, -0.023528077, -0.0035400726, -0.0013534075, -0.011709732, 0.021097884, 0.013019864, -0.012218851, 0.0048434166, 0.0035197078, 0.019685926, 0.017907405, 0.016468296, 0.01781237, 0.006167126, -0.013915914, 0.049988676, 0.040267903, 0.014608315, 0.016115308, -0.012483593, 0.0123614045, -0.011580756, 0.008152689, 0.0040186443, -0.027153002, -0.042521603, -0.0071412395, 0.0033500018, -0.010080552, -0.04311897, -0.011770827, 0.0125378985, -0.020391906, 0.00682898, -0.0058141365, -0.0070326277, 0.020867083, -0.02587681, -0.025550975, 0.04393356, 0.002190908, 0.02875503, -0.005664795, 0.0017106391, -0.004089921, 0.040267903, 0.017880252, 0.010535365, -0.015803047, 0.035760503, 0.02211612, -0.019414397, -0.026718555, 0.012436075, -0.017201427, 0.022211157, -0.008431007, 0.010250258, 0.015273564, -0.026379142, 0.0010861201, -0.020147528, -0.015762318, -0.029895457, -0.06304927, 0.013549348, 0.0239761, -0.01649545, -0.0034637048, 0.018599806, 0.025564551, 0.01354256, 0.0010275714, 0.013997373, 0.005216771, 0.004008462, 0.0027509385, -0.006394532, 0.011411049, 0.010684706, 0.016712673, -0.0006660971, 0.030139832, -0.011383897, -0.015925236, -0.018450465, -0.0005608792, -0.013440737, 0.0019329544, 0.006211249, 0.031063035, 0.020269716, 0.012585416, -0.019998185, -0.028972253, -0.026935779, 0.0019261661, -0.033886947, 0.0066728503, 0.002601597, 0.007093722, 3.7680093e-05, -0.006306285, -0.0035298902, -0.024532737, 0.010976601, -0.0033466076, 0.027057966, 0.027886134, -0.037959896, 0.01012807, 0.03546182, 0.007012263, 0.007813277, -0.016468296, 0.008254513, -0.011777615, -0.0010139949, 0.004826446, -0.6303841, -0.036520787, -0.026637096, -0.033533957, -0.0070326277, 0.01564013, 0.018640537, 0.019387243, -0.0025880206, 0.028510652, -0.017201427, -0.006061908, 0.010895142, -0.029705385, -0.03554328, -0.020867083, 0.0116690025, -0.014241749, 0.031171646, -0.00069749274, -0.025374481, 0.0021603608, -0.00459904, 0.019387243, 0.001015692, -0.0055765477, 0.0129248295, -0.013617231, -0.019414397, 0.00013374975, -0.036656555, 0.0032736338, -0.0015825109, -0.0005965175, 0.033533957, -0.011947321, 0.02098927, 0.03239353, 0.019319361, 0.02196678, -0.023134358, -0.007283793, -0.010732224, 0.008627866, -0.008424219, 0.005498483, -0.0012066115, -0.004982576, -0.005848078, -0.02192605, 0.0043275096, -0.0034586138, 0.0055052713, 0.021002848, -0.01175725, 0.002818821, 0.024614196, -0.020079644, 0.033968408, -0.0017904011, 0.008899396, 0.004419151, -0.001204066, -0.028021898, -0.004690681, 0.0129995, -0.036059186, 0.00019346514, 0.018952796, 0.008159477, 0.015925236, 0.036575094, -0.008044077, 0.00612979, -0.007562111, 0.031524636, 0.008268089, 0.03065574, 0.0001240977, 0.014526856, -0.0019125897, 0.011037695, -0.027220884, 0.00077301206, 0.041707013, 0.017920982, -0.033859793, 0.020473363, 0.007847218, -0.017486533, -0.0057326774, -0.0049995463, -0.0007445862, -0.034755845, -0.0101416465, -0.00065039925, -0.010257047, -0.0230529, 0.0040220385, -0.038584415, -0.025754623, 0.0115739675, 0.00021043577, -0.028890794, 0.046838928, -0.010250258, 0.011343167, -0.0020992665, 0.029895457, -0.027112273, -0.0041917446, -0.013013076, -0.017948134, -0.007202334, -0.01338643, -0.031008728, 0.016997779, 0.024234055, -0.014839116, -0.019550161, 0.00974114, -0.014716927, -0.02161379, -0.012979135, 0.017459381, 0.016278224, 0.005400053, -0.014554009, -0.010005881, -0.00574286, -0.007847218, -0.007894736, 0.018708419, -0.008057654, 0.0050606406, 0.019278632, 0.0017259127, 0.016115308, 0.0062757377, -0.038204275, 0.006367379, 0.0077861235, 0.011947321, -0.00095120363, 0.004334298, -0.030818658, -0.023853913, -0.0076028407, 0.015178529, -0.014703351, 0.022632027, 0.011024119, 0.0064454437, 0.031497482, 0.0076028407, 0.013447524, -0.015151375, -0.037199613, -0.03328958, -0.021464448, 0.004439516, 0.017160697, -0.016020272, -0.010813683, 0.020228988, -0.015477211, 0.031090187, 0.025849659, -0.014377515, -0.025741046, 0.027601028, -0.0016105125, -0.023568805, -0.01827397, -0.014431821, 0.036004882, 0.011845497, 0.004690681, -0.0029308272, -0.010623612, -0.01505634, -0.0056580068, 0.0041374387, 0.0033669723, 0.0056817657, 0.01908856, 0.033506803, 0.01505634, 0.027560297, 0.031361718, 0.023310853, -0.011512873, -0.002659297, 0.025999, 0.0030394392, 0.0048434166, 0.030981576, -0.0069172275, -0.0100126695, 0.020052493, 0.012402134, -0.0038693028, -0.004412363, -0.024763538, 0.005423812, -0.037335377, 0.011071636, 0.010372447, 0.012062721, -0.009754716, -0.0107050715, -0.020622706, -0.023962524, -0.016997779, -0.0036962025, 0.032502145, -0.008831514, 0.0074534994, -0.0035027373, 0.0041374387, -0.0008722902, 0.02382676, 0.016590485, 0.016210342, -0.010413176, 0.017459381, 0.0092252325, 0.02230619, 0.012829794, 0.0009749625, -0.014119561, -0.0036079553, 0.0020262927, 0.006740733, 0.01105806, 0.024627773, 0.044368006, -0.0010683009, 0.012463228, 0.010637188, -0.003713173, 0.00454134, 0.0042494447, -0.022564145, 0.007867582, -0.0133592775, 0.0145811625, 0.014391092, 0.0067848563, -0.0038523322, -0.0019685926, 0.008634655, -0.033153817, -0.009184503, -0.0031141099, -0.034755845, -0.0110105425, 0.017758064, 0.021287953, 0.007949041, 0.00041365903, 0.005478118, 0.005641036, -0.010189164, -0.004551522, -0.0230529, 0.015490788, -0.005983843, -0.02608046, 0.010698283, -0.021274377, -0.011424625, 0.0066185445, -0.021287953, 0.002654206, 0.00012579477, -0.000365717, -0.003814997, 0.021111459, -0.012632934, -0.009116621, -0.014798386, 0.021763131, 0.01098339, -0.02587681, -0.04268452, -0.02130153, -0.011112366, -0.009904058, -0.01008734, 0.0039371853, 0.011580756, -0.0031514454, -0.0010496333, -0.046078645, -0.0056546126, 0.03679232, -0.023813182, -0.0021230255, -0.006119608, 0.015273564, -0.0071072984, -0.010623612, -0.00954428, 0.04024075, -0.0015248108, 0.0010895142, -5.833653e-05, 0.012904464, -0.019509433, 0.016454719, -0.019767385, -0.0017411863, -0.0010767862, -0.014771233, 0.006146761, -0.027696062, 0.012850159, 0.0067101857, -0.020880658, -0.002162058, -0.021369413, -0.012911253, 0.02766891, 0.042548753, 0.03193193, 0.0008858667, 0.0066049676, -0.0059465077, -0.0010963025, -0.008295243, -0.010365658, 0.022781368, -0.0012125513, -0.0072905812, 0.014268903, 0.0072362754, 0.0015545094, 0.012578628, 0.007949041, -0.010331717, -0.042983204, -0.019265056, -0.005844684, -0.0031480512, -0.01109879, 0.013433948, 0.018966373, 0.012897676, -0.015042763, 0.020337598, 0.012870523, -0.017215004, 0.002302914, -0.0034687961, -0.012931617, 0.03627641, 0.023799608, -0.015504364, 0.0051862234, 0.009612163, 0.0044293334, 0.024220478, 0.00993121, -0.029596772, 0.0061162137, 0.0069477744, 0.009048738, -0.0033550928, -0.0059193545, 0.00020078373, 0.013148841, -0.0059091724, 0.008736478, 0.028510652, -0.024043983, -0.03443001, -0.009136985, -0.009652892, 0.018640537, -0.0060313605, -0.020391906, -0.012619358, -0.02284925, 0.00013841667, -0.037959896, 0.0038794852, 0.0013440736, -0.028809335, -0.026487755, -0.019604467, -0.0081459, -0.030682893, -0.034239937, -0.0005061489, -0.030384209, -0.036140647, -0.0072226985, 0.037091002, 0.008512466, 0.008797573, 0.0069172275, 0.019903151, 0.029623926, -0.021803861, 0.006584603, 0.0034603106, -0.024926456, 0.034918763, -0.005766619, 0.01478481, -0.003764085, -0.0041578035, 0.0024200114, 0.012578628, -0.010073764, 0.023351582, 0.038855948, -0.020731317, 0.024383396, 0.002683056, 0.022835676, -0.015531518, -0.015300717, -0.00086041074, -0.03372403, -0.007887947, -0.01191338, -0.016237495, -0.016658368, 0.010895142, 0.008824726, -0.031008728, -0.022835676, 0.006964745, -0.027180156, 0.011899804, 0.009795446, 0.012856946, 0.03581481, -0.012503957, 0.0005053004, -0.025252292, 0.0006588846, -0.011702944, -0.0053830827, 0.029895457, 0.0024284967, 0.002652509, 0.032502145, -0.00034132172, -0.016902745, 0.006543874, 0.009136985, 0.004527763, 0.03147033, 0.013216724, -0.028103357, 0.016454719, -0.025985423, -0.004337692, -0.002985133, 0.025401633, -0.017472956, -0.0058277133, 0.0073584635, 0.013060594, 0.023310853, -0.0072905812, -0.028999407, -0.02134226, -0.024342667, 0.015979541, 0.0043784217, -0.0060585137, 0.0022163638, -0.0087907845, -0.025062222, -0.031497482, -0.023921795, -0.011974474, -0.020147528, 0.037118156, 0.0030682893, 0.036004882, -0.019604467, -0.020120375, 0.03366972, 0.008804361, -0.0059974194, 0.0023640082, -0.003740326, -0.006818798, 0.014065255, 0.03510883, -0.007297369, -0.007894736, -0.027410956, 0.01564013, -0.00019198022, 0.0055663656, -0.0121034505, 0.013121689, -0.029786844, -0.03130741, -0.002002534, -0.006856133, -0.017133545, -0.016536178, -0.016454719, 0.01781237, -0.018654112, 0.024736386, -0.0017717334, 0.03592342, -0.007582476, 0.012870523, 0.007514593, 0.043716334, -0.012782276, -0.008756843, -0.044693843, 0.004758564, 0.02821197, -0.00037717217, 0.018233242, -0.0013720752, -0.0046397694, 0.003954156, 0.007718241, -0.0070733572, -0.033126663, -0.010643977, -0.01478481, 0.018436888, -0.013182783, 0.0073720403, -0.05669547, -0.01132959, -0.0068663154, -0.0041102855, 0.025863234, -0.008247725, -0.008118748, 0.027288767, 0.026745707, 0.0077997, -0.023528077, 0.04121826, 0.018192511, 0.008234148, -0.040539432, -0.008179842, -0.0034620077, 0.0033024838, 0.05175362, 0.016522601, -0.010508212, -0.009605374, -0.007881159, -0.0036520788, -0.025170833, -0.048984017, 0.007820065, -0.00022804279, 0.0021467842, -0.0067203683, -1.4371999e-05, -0.020541247, 0.03942616, -0.020554822, 0.0043309038, -0.026908625, -0.023487346, -0.019237902, -0.01160112, 0.0091437735, 0.01105806, 0.004795899, -0.0041815625, 0.011906591, -0.0054611475, 0.017310038, -0.014608315, 0.0069036507, 0.030519975, -0.0038930618, -0.00710051, 0.0022655786, 0.0069375923, 0.0052642883, -0.0055969125, 0.0011293952, 0.03706385, -0.034620076, 0.015246411, 0.005739466, -0.004629587, -0.0063266493, 0.00438521, -0.0019058014, -0.026351988, -0.0046465574, 0.009666469, 0.0022893376, 0.00428678, 0.0030801687, -0.030981576, -0.0034212782, -0.0122324275, -0.033995558, 0.0030207715, 0.029053712, -0.0461058, -0.011200613, -0.003329637, 0.00029931942, 0.01873557, 0.011940532, 0.023704572, -0.020079644, 0.011458567, -0.01664479, 0.004388604, 0.007181969, -0.035027374, -0.029895457, 0.0053932653, -0.028021898, -0.0042833863, 0.008736478, -0.019889574, -0.009992305, -0.00026856016, -0.014540433, 0.015762318, 0.0044530923, -0.0015604491, 0.0032125397, -0.0021773314, 0.008193418, -0.0022570933, -0.019305784, 0.018531924, -0.016862014, 0.0015723285, 0.0056003067, -0.0028782182, -0.0011370319, 0.0023758877, -0.010745801, 0.01618319, -0.0028544595, -0.021410143, 0.0012023689, -0.0047551696, 0.003706385, 0.0053219884, -0.009340633, -0.025279446, -0.015069916, -0.003363578, -0.0037980261, 0.02250984, -0.019998185, -0.0012880706, 0.020378329, -0.009924422, -0.027886134, 0.014486127, 0.012829794, -0.0133592775, -0.030845812, 0.0068323743, -0.00981581, 0.025347328, 0.016536178, -0.0113160135, -0.009761504, 0.005335565, -0.0235145, -0.028510652, -0.015517941, 0.0497443, 0.0139702195, 0.025795352, 0.020975694, 0.01202878, 0.008003348, -0.0030156802, -0.016128883, -0.004768746, 0.02153233, -0.015164952, 0.016970627, 0.030791504, -0.030818658, 0.008879032, 0.017676605, 0.0057801954, -0.019292207, 0.034565773, 0.0072362754, 0.016061, 0.001191338, -0.0021298137, 0.003219328, 0.026867896, 0.029678231, -0.03519029, -0.014445397, 0.0063334377, -0.008607502, -0.022564145, 0.008953703, -0.004711046, 0.00070725085, -0.0065167206, 0.011431414, -0.03138887, 0.011662215, -0.0005031791, 0.017948134, 0.005206588, 0.0038285733, 0.024071136, 0.009367785, 0.007609629, 0.005009729, -0.012076298, -0.021790285, -0.004819658, 0.012775487, -0.00030292568, -0.0014857784, -0.012721182, -0.018151782, -0.016033849, -0.0034620077, 0.007854006, -0.00665588, -0.024505585, 0.01132959, 0.028592112, -0.0129248295, -0.018029593, -0.010990177, -0.023718148, -0.00090962555, 0.010053399, -0.019495856, -0.05664116, 0.017133545, 0.0044666687, -0.0050776117, -0.024247631, -0.005433995, 0.0014535341, -0.003764085, -0.015436482, 0.16802278, -0.01726931, -0.0007280399, 0.027275192, 0.010956236, -0.0043784217, 0.019971034, -0.0020958723, -0.010005881, 0.005145494, 0.014906999, -0.0034077018, -0.021722402, 0.0015137799, 0.022360498, -0.0050843996, -0.03174186, -0.017282886, 0.0022723668, 0.011920168, 0.009238809, -0.01816536, -0.00082562095, 0.00044887306, 0.018694842, -0.0042630215, -0.022224732, -0.001978775, -0.022048239, 0.020731317, -0.010745801, 0.03752545, 0.01175725, 0.013515407, -0.009075891, 0.015124222, 0.02021541, -0.009123409, 0.008003348, 0.004795899, -0.019740233, -0.012850159, 0.0035027373, -0.010569306, 0.005552789, 0.02320224, -0.007962618, 0.0024573468, -0.032719366, 0.019115714, -0.0111870365, 0.0154500585, 0.0019142866, 0.020147528, -0.040892422, -0.036032036, -0.006316467, 0.02017468, 0.0017734305, 0.022686334, -9.042162e-06, 0.0066151503, -0.019183597, 0.015925236, -0.0069511686, 0.03967054, -0.023840336, 0.011261708, 0.027370226, -0.028239122, -0.0073245224, -0.00017882798, 0.0069274097, -0.013875185, -0.028836489, -0.008478525, 0.052079458, 0.016386837, 0.033886947, 0.013997373, 0.01626465, 0.0020839928, -0.0046838927, -0.014499703, 0.018219665, -0.041652706, 0.007711453, -0.016318955, -0.011078425, -0.016563332, 0.001868466, 0.0010674524, -0.028836489, 0.0064013205, -0.0016257861, -0.024817845, 0.020853506, 0.02075847, -0.008953703, -0.0029189477, -0.009530704, 0.048631027, -0.028619265, 0.0023690995, -0.011017331, 0.010548942, -0.015002034, 0.044911064, 0.0060449373, -0.011051272, 0.01722858, -0.004388604, -0.0019312573, -0.0257682, 0.02436982, 0.045155443, 0.0047517754, -0.027478838, 0.048712485, -0.0019431367, -0.035380363, -0.016128883, -0.017174274, 0.021274377, -0.017432228, -0.021396566, -0.00023949797, 0.010929083, -0.01781237, -0.004527763, 0.0058616544, 0.03318097, 0.00020640524, 0.00030144074, -0.028619265, -0.0053253826, -0.018124629, -0.013691901, -0.010100917, -0.018803455, 0.0009206565, -0.017744487, 0.008125536, 0.03022129, -0.017255733, -0.02048694, 0.013325336, 0.025075797, -0.00021595122, -0.010677918, -0.0063334377, 0.012001627, 0.022021085, 0.00845816, 0.002111146, -0.032665063, -0.028076205, -0.032800827, -0.026202647, 0.0128841, -0.009781869, 0.00093847566, 0.006818798, -0.009340633, -0.013712266, 0.0035163139, -0.17128114, -0.002031384, 0.0022078785, -0.029786844, 0.021871744, 0.024261208, 0.025075797, -0.002789971, 0.027207308, -0.0074670757, 0.018912066, 0.017459381, -0.0035129196, -0.008010136, 0.0062723435, 0.013345701, -0.016359683, 0.011193825, 0.02297144, -0.0035909845, 0.0044021807, -0.01137032, 0.036385022, 0.0035027373, 0.020772047, 0.023378735, 0.0094831865, 0.023582382, 0.002652509, -0.008566773, -0.006689821, -0.02052767, 0.021668095, -0.018341852, 0.01338643, 0.002952889, 0.018844184, -0.0070054745, -0.008437796, 0.027465262, 0.03274652, 0.016088154, 0.0162918, -0.029108018, 0.02048694, -0.008858667, -0.0046974695, -0.026718555, 0.019292207, -0.02021541, 0.022414804, -0.017595146, 0.010433542, 0.00915735, 0.028564958, -0.000737798, 0.00021934535, 0.027886134, -0.0019007102, -0.012252793, -0.027601028, 0.0020805988, -0.011689368, -0.0066287266, -0.001667364, -0.028347734, -0.0129248295, 0.018450465, -0.032040544, 0.0154500585, -0.03318097, 0.004415757, 0.0030241658, -0.0018803454, 0.006258767, 0.02460062, 0.006825586, 0.018654112, 0.01672625, -0.0030479245, -0.011417838, 0.024967186, -0.012388557, -0.0070869336, 0.006316467, 0.0018922249, -0.0138887605, 0.0021060547, 0.0053695063, 0.025931118, 0.007982982, -0.010949448, 0.020364752, -0.0073041576, 0.028347734, -0.00079040695, -0.002706815, -0.012307098, -0.000777679, -0.011628273, 0.004884146, -0.0031022306, -0.0009478095, 0.010066976, 0.035760503, 0.00035468608, 0.0055663656, 0.010080552, 0.041761316, -0.01420102, -0.012497169, -0.017065663, -0.0005897293, 0.04765352, -0.0046261926, 0.016848437, -0.008315607, -0.0031802955, -0.0031836894, 0.0029817391, 0.06228899, -0.009985517, -0.0042630215, 0.024234055, 0.0058854134, -0.03364257, -0.117735416, 0.004415757, 0.009238809, 0.04621441, 0.0014518371, -0.0019872603, -0.0035536492, 0.026379142, -0.04276598, 0.029678231, -0.012938405, -0.0065981797, -0.00028510654, 0.011288861, -0.014431821, -0.019292207, -0.014662622, -0.011451779, -0.015110645, 0.027153002, 0.023853913, -0.0054272064, 0.014554009, 0.013352489, 0.0054543596, -0.008376701, -0.026637096, 0.018178934, 0.030764353, 0.0065608444, 0.0025320174, -0.0070326277, 0.0051387055, -0.021179343, -0.014472551, -0.027410956, 0.0035129196, -0.0011921865, 0.033778336, 0.0020687194, -0.012137393, 0.016386837, 0.0076775113, 0.013725842, 0.0008731387, -0.017622298, -0.003984703, 0.021315107, -0.033425346, -0.018056747, -0.016481873, -0.0015349932, -0.02196678, -0.03011268, -9.0792855e-05, -0.03318097, 0.0010114494, 0.014839116, -0.012476805, -0.020283293, -0.003658867, 0.014852692, -0.014078831, 0.008152689, 0.014038103, 0.00018773756, 0.010162011, -0.015667282, 0.012449652, -0.0060347547, 0.0029189477, 0.015314293, -0.0069240155, 0.009781869, -0.037552603, -0.004201927, -0.02436982, -0.0015324475, 0.020161103, 0.020269716, -0.012578628, -0.018721996, 0.020853506, -0.032719366, 0.012565052, 0.020201834, 0.009734351, 0.028809335, 0.020459788, -0.021993931, -0.007779335, 0.04765352, -0.021491602, 0.013915914, -0.0061637317, 0.011709732, -0.023568805, -0.007636782, -0.013400007, 0.0036962025, -0.012008416, -0.0036452906, -0.072009765, 0.014947728, -0.0060109957, -0.019685926, -0.0022825492, -0.022102544, 0.004300357, -0.015748741, 0.005708919, 0.027831828, -0.028999407, 0.023568805, -0.023066476, -0.02654206, -0.032420684, -0.0073720403, 0.040756658, -0.021396566, 0.0026474178, 0.033995558, 0.0030054979, 0.010718647, 0.007976195, 0.02754672, -0.008641443, -0.007426346, -0.012178121, 0.028890794, 0.00725664, 0.0027780915, 0.008464948, -0.012449652, -0.005179435, 0.011397473, -0.009367785, -0.0022910347, -0.0071616042, -0.015680859, 0.01714712, 0.012741546, -0.021369413, -0.03407702, 0.0077861235, 0.017988864, 0.008851878, 0.009775081, -0.012463228, -0.00019898059, 0.004388604, -0.0025999, 0.03407702, 0.018844184, -0.006258767, -0.013603654, -0.025428787, -0.0008145901, -0.008471737, -0.0117979795, 0.012734758, -0.01664479, 0.045644198, 0.019427974, 0.000105536084, -0.015110645, -0.0077589704, -0.024111867, -0.029732538, -0.0026185676, -0.029461008, -0.027071543, -0.013087748, -0.0058888076, 0.0049384525, 0.026895048, -6.3215586e-05, -0.028483499, -0.012694028, -0.017052086, -0.01741865, -0.0030937453, 0.0026898442, 0.023650264, -0.011173461, -0.003984703, 0.019034255, -0.0017564598, 0.008281666, 0.0123614045, -0.012015204, 0.01218491, 0.0019550163, 0.024777114, 0.011879439, 0.006520115, 0.010610036, 0.0039405795, 0.012510746, 0.01152645, 0.01912929, 0.008200207, 0.0013279516, 0.031253107, 0.011458567, -0.010929083, -0.0060279667, 0.041734166, -0.025890388, -0.02025614, 0.029162325, -0.01284337, -0.006727156, -0.0043716333, -0.007195546, 0.009965152, -0.0339141, 0.033425346, -0.014092408, -0.022279039, -0.02173598, 0.024437701, 0.0032804222, 0.028592112, 0.0014925665, -0.021260802, 0.019332938, -0.009394939, 0.044802453, 0.01202878, 0.035380363, -0.0255374, 0.015246411, 0.0125378985, -0.028510652, -0.013060594, 0.011675791, 0.020188257, -0.012001627, 0.009028373, -0.022550568, 0.07429061, 0.029406702, -0.027831828, 0.012632934, -0.02071774, 0.021803861, 0.012626146, -0.0019974427, -0.012904464, -0.024899304, 0.027533144, -0.016033849, -0.0019855632, -0.04404217, 0.005607095, 0.024994338, 0.0031853865, 0.023528077, -0.018667689, -0.011390684, 0.016902745, -0.01595239, 0.006041543, 8.2466635e-05, -0.027383802, 0.015748741, 0.007609629, -0.0022842463, 0.031850472, -0.041707013, 0.009136985, -0.017391497, -0.046838928, -0.014486127, 0.005980449, 0.020595552, 0.008166266, -0.016101731, 0.030954422, 0.022754217, -0.019699503, 0.017703757, -0.01350183, -0.017948134, 0.015341447, -0.004008462, -0.011702944, 0.0207992, -0.005522242], metadata={'file_name': 'data/1406.4214.pdf', 'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024, pp. 1-4.'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={<NodeRelationship.SOURCE: '1'>: RelatedNodeInfo(node_id='df6b2f05-4b13-4e04-97c4-e58170e9b714', node_type=<ObjectType.DOCUMENT: '4'>, metadata={'file_name': 'data/1406.4214.pdf', 'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024, pp. 1-4.'}, hash='1cfd3cd76f12c5dae17e07638b6b4584e3c04282aae51284801629e0db25bb45'), <NodeRelationship.NEXT: '3'>: RelatedNodeInfo(node_id='4e83bfa9-e6bc-4638-ba11-75f16e41aad8', node_type=<ObjectType.TEXT: '1'>, metadata={}, hash='1d732c8ed00fc6c1a785480fefb3d6862eb577d49b8ab693c1c123cf167b49e9')}, hash='5d968b55704a07153d37bf73047a18f67efc8e7642e2921c80d28507939242e2', text='1 Title: Nanopore sequencing of the phi X 174 genome  \\n \\nAuthors: Andrew H. Laszlo1, Ian M. Derrington1, Brian C. Ross1, Henry Brinkerhoff1, Andrew  \\nAdey2, Ian C. Nova1, Jonathan M. Craig1, Kyle W. Langford1, Jenny Mae Samson1, Riza Daza2, \\nKenji Doering1, Jay Shendure2, Jens H. Gundlach1,† \\n \\nAffiliations: 1Department of Physics, University of Washington, Seattle, WA 98195, USA  \\n2Department of Genome Sci ences, University of Washington, Seattle, WA 98195, USA  \\n†Corresponding author. E -mail: gundlach@uw.edu  \\n \\nAbstract : Nanopore sequencing of DNA is a single -molecule technique that may achieve long \\nreads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we \\nbuild on recent progress with respect to nanopore resolution and DNA control to interpret the \\nprocession of ion current levels observed during the translocation of DNA through the pore \\nMspA. As approximately four nucleotides affect the ion current of each level, we measured the \\nion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer \\nmap is highly predictive of ion current levels of previously unmeasured sequences derived from \\nthe bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X \\n174 up to 4,500 bases in length  that can be unambiguously aligned to the phi X 174 reference \\ngenome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and \\npolymorphism detection. All methods and data are made fully available.  \\n \\nMain Text : DNA sequencing is revolutionizing biomedical and other life sciences research \\nthrough its expanding scope (1) and has a rapidly growing presence in clinical medicine (2). \\nThese developments are driven in part by the successful completion of the Human Genome \\nProject (3) and in part by the introduction of new sequencing technologies that have  dramatically \\nreduc ed the cost of DNA sequencing (4). Although such ‘next -generation’ sequencing  \\ntechnologies have matured considerably since  early proof -of-concept s (5-7), nearly all remain  \\nlimited to short sequence reads (with the exception of real -time sequencing from elongating \\npolymerases (8)) and rely on  complex, expensive instrumentation. Most platforms are also \\nlimited with respect to speed and require extensive sample preparation steps prior to sequencing .  \\n \\nNanopore sequencing, independently proposed by Church and Deamer in the mid -1990s, has \\ntremendous potential to overcome these limitations and  achieve long reads, low cost, and high \\nspeed while requiring minimal sample preparation and instrumentation  (i.e. ‘tricorder’ -like DNA \\nsequencing devices) (9-12). However, this promise has faced substantial technical challenges, \\nsuch that despite nearl y 20 years of effort, nanopore -derived sequence reads that align to \\ncomplex, natural DNA sequences have yet to be demonstrated.  \\n \\nIn nanopore devices directed at DNA sequencing, a salt solution is divided into cis and trans  \\nwells  by a thin membrane. A singl e nanometer -scale pore in the membrane connects the cis and \\ntrans  wells electrically. When a voltage is applied across this membrane, ion current flows \\nthrough the pore; this current provides the primary signal. DNA is negatively charged and is \\nelectrophoretically attracted into the pore. When single -stranded (ss) DNA e nters the pore, it \\nblocks some fraction of the ion current. The fraction of the ion current that is blocked depends on \\nthe identity of nucleotides within the pore (13-15). Key challenges of this technique are single - 2 nucleotide resolution and control of the DNA translocation. Single -nucleotide resolution was \\nrecently enabled through the development of MspA , a protein pore with a short and narrow \\nconstriction (11, 13, 15, 16 ). DNA translocation control was also recently enabled through the \\nuse of molecular motors such as phi29 DNA Polymerase (DNAP) (11, 17 ) (Fig. 1 ). \\n \\nWe have found t hat the currents in the MspA pore are determined by about four nucleotides at \\nany given time (11, 15 ).', start_char_idx=1, end_char_idx=4095, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.86771965)]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "response.source_nodes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{   'metadata': {   '9fe90d42-9f20-4967-b144-96163b8329ee': {   'citation': 'Laszlo, '\n",
      "                                                                            'Andrew '\n",
      "                                                                            'H., '\n",
      "                                                                            'et '\n",
      "                                                                            'al. '\n",
      "                                                                            '\"Nanopore '\n",
      "                                                                            'sequencing '\n",
      "                                                                            'of '\n",
      "                                                                            'the '\n",
      "                                                                            'phi '\n",
      "                                                                            'X '\n",
      "                                                                            '174 '\n",
      "                                                                            'genome.\" '\n",
      "                                                                            'Nanopore '\n",
      "                                                                            'sequencing '\n",
      "                                                                            'of '\n",
      "                                                                            'the '\n",
      "                                                                            'phi '\n",
      "                                                                            'X '\n",
      "                                                                            '174 '\n",
      "                                                                            'genome, '\n",
      "                                                                            '2024, '\n",
      "                                                                            'pp. '\n",
      "                                                                            '1-4.',\n",
      "                                                                'file_name': 'data/1406.4214.pdf'},\n",
      "                    'fcb50563-f2bb-4019-90e6-32c3f86f6c77': {   'citation': 'Laszlo, '\n",
      "                                                                            'Andrew '\n",
      "                                                                            'H., '\n",
      "                                                                            'et '\n",
      "                                                                            'al. '\n",
      "                                                                            '\"Nanopore '\n",
      "                                                                            'sequencing '\n",
      "                                                                            'of '\n",
      "                                                                            'the '\n",
      "                                                                            'phi '\n",
      "                                                                            'X '\n",
      "                                                                            '174 '\n",
      "                                                                            'genome.\" '\n",
      "                                                                            'Nanopore '\n",
      "                                                                            'sequencing '\n",
      "                                                                            'of '\n",
      "                                                                            'the '\n",
      "                                                                            'phi '\n",
      "                                                                            'X '\n",
      "                                                                            '174 '\n",
      "                                                                            'genome, '\n",
      "                                                                            '2024.',\n",
      "                                                                'file_name': 'data/1406.4214.pdf'}},\n",
      "    'response': 'Nanopore sequencing is a single-molecule technique for DNA '\n",
      "                'sequencing that has the potential to achieve long reads, low '\n",
      "                'cost, and high speed with minimal sample preparation and '\n",
      "                'instrumentation. It involves the use of a nanopore device, '\n",
      "                'which consists of a thin membrane with a nanometer-scale pore '\n",
      "                'that separates two salt solution wells. When a voltage is '\n",
      "                'applied across the membrane, ion current flows through the '\n",
      "                'pore, and when single-stranded DNA enters the pore, it blocks '\n",
      "                'some fraction of the ion current. The fraction of the ion '\n",
      "                'current that is blocked depends on the identity of '\n",
      "                'nucleotides within the pore. Recent advancements in nanopore '\n",
      "                'resolution and DNA control have enabled single-nucleotide '\n",
      "                'resolution and control of DNA translocation. However, despite '\n",
      "                'nearly 20 years of effort, nanopore-derived sequence reads '\n",
      "                'that align to complex, natural DNA sequences have yet to be '\n",
      "                'demonstrated.',\n",
      "    'source_nodes': [   NodeWithScore(node=TextNode(id_='fcb50563-f2bb-4019-90e6-32c3f86f6c77', embedding=[-0.023770344, -0.016172517, -0.018628243, 0.005633924, -0.007645313, 0.021178942, -0.006203761, -0.0056576673, -0.040349886, -0.015901165, 0.0027966106, 0.015507706, -0.002874624, 0.012041198, 0.011898739, 0.0013414912, 0.011735928, -0.036225352, -0.002491341, -0.009639743, -0.032996275, -0.012366819, 0.010643741, -0.0025066044, -0.004677412, 0.0072654216, 0.023946721, -0.019659376, -0.034760058, -0.004785952, -0.0012185354, 0.011315335, -0.019808618, 0.0007881897, 0.019455863, -0.009972148, 0.015019275, -0.0022861317, 0.03391887, -0.03845043, 0.029007416, 0.0013915215, 0.010189228, 0.0032833465, -0.021178942, -0.018221216, 0.012346468, 0.008147312, -0.012509279, -0.007462151, 0.011993712, 0.01355398, 0.010589471, -0.00054694177, -0.023322614, -0.029143091, 0.014503708, -0.027514987, -0.0117020095, -0.013119819, 0.005386316, 0.0033342247, -0.029197361, 0.020921158, -0.028247634, 0.014557978, 0.015629815, 0.015019275, -0.0036598458, 0.013180872, 0.03359325, -0.005844221, 0.00082338054, 0.007903096, 0.017502137, -0.022155805, -0.0017553015, -0.0014050892, -0.012366819, 0.01936089, 0.01288917, -0.012007279, 0.004680804, 0.0037005486, 0.019279486, 0.018139811, 0.016226787, 0.017854892, 0.0058340454, -0.015032843, 0.049684357, 0.04037702, 0.014123817, 0.0159283, -0.012122603, 0.011776631, -0.009395527, 0.00801842, 0.0046909796, -0.027202932, -0.04287345, -0.0070279897, 0.0028542727, -0.009700797, -0.042629234, -0.011315335, 0.012305765, -0.021925157, 0.006393707, -0.00713653, -0.00641745, 0.021490995, -0.02638888, -0.023946721, 0.04371464, 0.0032833465, 0.029468713, -0.0053727487, 0.0014949741, -0.004233075, 0.04032275, 0.018370459, 0.010290985, -0.016457435, 0.03462438, 0.022481427, -0.019103106, -0.02683661, 0.012075117, -0.0169323, 0.02234575, -0.008181231, 0.009775418, 0.016389597, -0.025954718, 0.0023166586, -0.019727213, -0.015412734, -0.030825468, -0.06431018, 0.012482144, 0.0236211, -0.016674515, -0.0031442791, 0.017963432, 0.025791908, 0.013988141, 0.0015848591, 0.014042412, 0.0045451284, 0.004830047, 0.003385103, -0.0053998837, 0.012516063, 0.009897525, 0.015548409, -0.00040109063, 0.031042548, -0.0102231465, -0.01515495, -0.016389597, -0.0020622672, -0.013587899, 0.0015017579, 0.0055287755, 0.03041844, 0.020731213, 0.012204009, -0.019903593, -0.028356174, -0.027433582, 0.0016662644, -0.035438433, 0.0065395576, 0.0021368887, 0.0071093948, 6.439285e-05, -0.006770206, -0.0032273803, -0.024123099, 0.011227146, -0.0041448856, 0.027026555, 0.029088821, -0.037473567, 0.009117392, 0.03519422, 0.0055219918, 0.008493285, -0.017135812, 0.008737501, -0.012149739, -0.0009183534, 0.0046943715, -0.63083667, -0.035329893, -0.024964288, -0.032670654, -0.007984501, 0.014693654, 0.018166946, 0.019401593, -0.0042941286, 0.03014709, -0.017244352, -0.005962937, 0.009578688, -0.029957145, -0.03579119, -0.020934725, 0.011790198, -0.014842897, 0.030635523, -0.0010260458, -0.026510987, 0.0022606927, -0.0034275018, 0.018980999, 0.00068855303, -0.00408044, 0.0127670625, -0.014178087, -0.018492566, 0.0002219566, -0.036740918, 0.0030950967, -0.0009734715, -0.0007881897, 0.03207368, -0.011946226, 0.02018851, 0.03329476, 0.019116674, 0.021111103, -0.021545265, -0.007611394, -0.010094255, 0.009273418, -0.009164878, 0.0073400433, -0.0013881297, -0.0055864374, -0.004531561, -0.020894023, 0.0038531835, -0.0042432505, 0.0064547607, 0.019781483, -0.012183658, 0.0029882523, 0.024231639, -0.01963224, 0.03364752, -0.0033783193, 0.008791771, 0.0043212636, -0.00086450716, -0.028329039, -0.004941979, 0.012929873, -0.035546973, -0.000809389, 0.018560404, 0.008981716, 0.015751923, 0.036903728, -0.007529989, 0.0059595453, -0.008384745, 0.0318566, 0.009748283, 0.029658658, -0.0005494857, 0.0141916545, -0.0011795287, 0.0103588225, -0.028301904, 0.00052404654, 0.04186945, 0.017692082, -0.033946004, 0.019510133, 0.008696798, -0.016905164, -0.005688194, -0.005281168, -0.0012049678, -0.034895733, -0.010406309, -0.001290613, -0.01048093, -0.023851749, 0.0030408266, -0.038586106, -0.026565257, 0.009571904, 0.00092768105, -0.027012987, 0.046401013, -0.011003281, 0.01227863, -0.0011294984, 0.030282766, -0.026700933, -0.004467115, -0.013431872, -0.018438296, -0.006797341, -0.013038413, -0.031883735, 0.01659311, 0.02461153, -0.014802194, -0.019103106, 0.009151311, -0.013615034, -0.020568402, -0.01380498, 0.01792273, 0.015059978, 0.004616358, -0.0134250885, -0.009388743, -0.006169842, -0.008425447, -0.008852825, 0.018736783, -0.008513636, 0.0042093317, 0.018858891, 0.0013737142, 0.01653884, 0.0047011552, -0.036822323, 0.0074214484, 0.0070890435, 0.012760278, 0.0003353728, 0.005091222, -0.030554116, -0.024272343, -0.0074689346, 0.016226787, -0.01510068, 0.023254776, 0.009911093, 0.006654882, 0.03142244, 0.007930231, 0.013377601, -0.014707222, -0.036578108, -0.03280633, -0.02146386, 0.0050980058, 0.015534841, -0.01720365, -0.010521633, 0.02045986, -0.016565975, 0.031232495, 0.025954718, -0.0128349, -0.025954718, 0.028383309, -0.0008602673, -0.023675371, -0.017475002, -0.014625816, 0.035384163, 0.0107862, 0.004891101, -0.0036089676, -0.010107823, -0.013825331, -0.0068617873, 0.0036462783, 0.003714116, 0.005416843, 0.019279486, 0.033213355, 0.014286627, 0.02599542, 0.031775195, 0.0236211, -0.011885172, -0.0038226566, 0.026782338, 0.0047554253, 0.0048741414, 0.030906873, -0.008567906, -0.011159307, 0.019890025, 0.012468576, -0.0038871022, -0.0049657226, -0.025629098, 0.005118357, -0.0375007, 0.010969362, 0.009409094, 0.011613821, -0.009972148, -0.012855251, -0.020934725, -0.025072828, -0.016348895, -0.0030510023, 0.034163084, -0.009022419, 0.0077742045, -0.003924413, 0.0042466423, -0.0014898863, 0.02279348, 0.015439869, 0.015629815, -0.010820119, 0.01792273, 0.010033201, 0.0215724, 0.013771061, 0.0013321636, -0.014123817, -0.0052506407, 0.0022606927, 0.0054982486, 0.011593469, 0.025506988, 0.04442015, -0.003086617, 0.012848468, 0.010745497, -0.0034563327, 0.004402669, 0.004436588, -0.023607532, 0.006498855, -0.014042412, 0.013621817, 0.014245925, 0.005155668, -0.004250034, -0.0024506382, 0.008988501, -0.03381033, -0.010453795, -0.0028966714, -0.03364752, -0.011607037, 0.016579542, 0.020486997, 0.007251854, -0.00051344687, 0.0052947355, 0.007529989, -0.009904309, -0.00336136, -0.023783911, 0.016186085, -0.0064920713, -0.025927583, 0.01073193, -0.021382455, -0.01217009, 0.0076927994, -0.02090759, 0.0022369493, -0.00021326488, -0.00047783207, -0.0034665084, 0.020378456, -0.012373603, -0.008567906, -0.015629815, 0.021613102, 0.0124414405, -0.025344178, -0.04336188, -0.021287482, -0.01098293, -0.009443013, -0.011254281, 0.0044162367, 0.012041198, -0.0047520334, -0.0008564514, -0.045559824, -0.0055864374, 0.03722935, -0.023553262, -0.0013347075, -0.006319085, 0.01385925, -0.0059459778, -0.011457793, -0.010073904, 0.040132806, -0.0015416126, 0.00013260158, 0.00087001896, 0.012590684, -0.020392023, 0.015955435, -0.019008134, -0.0015526362, -0.00078182993, -0.014395168, 0.0065158145, -0.027691364, 0.012217577, 0.007021206, -0.019320188, -0.00057958864, -0.02101613, -0.011912307, 0.02854612, 0.041842315, 0.030961143, -9.385987e-05, 0.00611218, -0.00713653, -0.00037628744, -0.008961365, -0.011627388, 0.023173371, -0.0022132061, -0.0071433135, 0.01598257, 0.0064649363, 0.0014703829, 0.011403523, 0.008601825, -0.010949011, -0.0431448, -0.018763918, -0.007889529, -0.0047215065, -0.009599039, 0.015073545, 0.017000137, 0.0126042515, -0.014707222, 0.020894023, 0.014123817, -0.015317761, 0.003176502, -0.004579047, -0.013038413, 0.036252487, 0.023024129, -0.015399166, 0.005457546, 0.00882569, 0.0046672365, 0.024747206, 0.0096193915, -0.028953146, 0.0071229623, 0.007733502, 0.008839257, -0.001950335, -0.0060409503, -0.00082041265, 0.0143544655, -0.0055525186, 0.0101417415, 0.027460717, -0.024068829, -0.036008272, -0.010575904, -0.009788985, 0.018302621, -0.0048944927, -0.020826185, -0.012543198, -0.023458289, 0.0003122656, -0.038233347, 0.0055186, 0.00043967334, -0.028736066, -0.02549342, -0.019564403, -0.007129746, -0.03125963, -0.032996275, -0.0009565121, -0.030554116, -0.03568265, -0.006373355, 0.037310757, 0.007943799, 0.0070686922, 0.007577475, 0.020663375, 0.027840607, -0.020771915, 0.0071772328, 0.003893886, -0.024896448, 0.03343044, -0.0047520334, 0.015806193, -0.0043009124, -0.0046197497, 0.0025727462, 0.012651738, -0.009965364, 0.022820614, 0.040404156, -0.021721642, 0.024055261, 0.0026287122, 0.02279348, -0.01664738, -0.015277058, -0.00093276886, -0.032345034, -0.008472933, -0.011871604, -0.016335327, -0.017434299, 0.010616606, 0.008201582, -0.030581253, -0.023119101, 0.009158094, -0.02737931, 0.012142955, 0.009531202, 0.012807765, 0.03568265, -0.012129387, 0.0005940042, -0.02450299, 0.00020467918, -0.011491712, -0.0058611804, 0.029414443, 0.0025405232, 0.0020198687, 0.031938005, -0.0012159914, -0.016009705, 0.005264208, 0.010243499, 0.0048978846, 0.03142244, 0.012821333, -0.028111959, 0.01621322, -0.026293907, -0.0031035764, -0.0034868596, 0.024964288, -0.017285055, -0.005538951, 0.0067396793, 0.013771061, 0.02295629, -0.0062885582, -0.029197361, -0.021477427, -0.023363316, 0.016023273, 0.0046909796, -0.0047418578, 0.0016722002, -0.010324904, -0.026198935, -0.03153098, -0.024014559, -0.011084686, -0.020229213, 0.036686648, 0.004195764, 0.035872594, -0.020758348, -0.019198079, 0.034163084, 0.00981612, -0.0062275045, 0.0018384027, -0.0033783193, -0.007482502, 0.012563549, 0.034054544, -0.0075164214, -0.0086086085, -0.02588688, 0.015548409, 0.0006711696, 0.0062546395, -0.011661307, 0.01305198, -0.030988278, -0.03381033, -0.0015076937, -0.00616645, -0.017447866, -0.015846895, -0.01682376, 0.018248351, -0.018275486, 0.024163801, -0.00068770506, 0.035384163, -0.0070958273, 0.012733143, 0.007380746, 0.04425734, -0.013961006, -0.008479717, -0.0451528, 0.0056441, 0.027297905, 5.660211e-05, 0.018614676, -0.0011659611, -0.004606182, 0.0046672365, 0.0074417996, -0.008371177, -0.032670654, -0.0102774175, -0.015385599, 0.018506134, -0.012054766, 0.007428232, -0.056169648, -0.010107823, -0.006447977, -0.0032002453, 0.026307475, -0.008371177, -0.00760461, 0.025656233, 0.02522207, 0.008147312, -0.023539694, 0.040241346, 0.018709648, 0.008384745, -0.040621236, -0.009185229, -0.0030136914, 0.002369233, 0.05266922, 0.01626749, -0.009171662, -0.009002068, -0.008296555, -0.0043416154, -0.02527634, -0.04889744, 0.007075476, -0.00031883735, 0.0013279237, -0.0065531256, -0.00081023696, -0.020500565, 0.038965996, -0.0210704, 0.0051421006, -0.026266772, -0.023716073, -0.020486997, -0.012034414, 0.009008852, 0.010643741, 0.0040295618, -0.004935195, 0.0110439835, -0.004226291, 0.016023273, -0.016009705, 0.007401097, 0.029360173, -0.0037853457, -0.006892314, 0.002937374, 0.0077742045, 0.0055830455, -0.005240465, 0.000959056, 0.03728362, -0.035275623, 0.014042412, 0.004555304, -0.003476684, -0.00528456, 0.0047486415, -0.0029475496, -0.026918015, -0.0055186, 0.010399525, 0.002935678, 0.004005818, 0.00377517, -0.030554116, -0.0042364667, -0.012190442, -0.03381033, 0.0039040619, 0.028790336, -0.045397013, -0.010698011, -0.0031510629, 0.0013898256, 0.018478999, 0.013682871, 0.02434018, -0.020161375, 0.012455009, -0.017027272, 0.0049589383, 0.008113394, -0.034488704, -0.029495848, 0.00634622, -0.028898876, -0.004226291, 0.007937015, -0.01864181, -0.010684444, 0.00019111163, -0.014381601, 0.014937869, 0.0044264123, -0.001530589, 0.003178198, -0.0027406446, 0.008506852, -0.003598792, -0.019388026, 0.018940296, -0.017841324, 0.0019689903, 0.0041143587, -0.0037276836, -0.00023467618, 0.0012227752, -0.011573118, 0.01520922, -0.0025659625, -0.022549264, 0.0009548161, -0.004233075, 0.0042669936, 0.006169842, -0.010514849, -0.024964288, -0.015005708, -0.0022556048, -0.0036394945, 0.022427157, -0.021002563, -8.055731e-06, 0.021260347, -0.010284201, -0.02655169, 0.014842897, 0.012611035, -0.01217009, -0.030988278, 0.0072450703, -0.008669663, 0.025764773, 0.01598257, -0.0117562795, -0.010670876, 0.005840829, -0.024923585, -0.029658658, -0.01682376, 0.049738627, 0.014951437, 0.023960289, 0.02113824, 0.012020847, 0.00760461, -0.0023132667, -0.014897167, -0.0039481563, 0.021979427, -0.014707222, 0.016511705, 0.031069683, -0.029875739, 0.009531202, 0.017787054, 0.006241072, -0.019279486, 0.035329893, 0.008703582, 0.01687803, 0.0014924302, -0.002342098, 0.003931197, 0.027840607, 0.029007416, -0.034108814, -0.013988141, 0.006020599, -0.008601825, -0.022318615, 0.0086357435, -0.0047723847, 0.00027219893, -0.006068086, 0.010826902, -0.03291487, 0.012516063, -0.0005155668, 0.01610468, 0.0049250196, 0.003686981, 0.023688938, 0.0092666345, 0.0068380437, 0.0046875877, -0.012611035, -0.020975428, -0.0048232633, 0.011138956, 0.0010455492, -0.002041916, -0.012699224, -0.018872458, -0.016633812, -0.0043958854, 0.0074485834, -0.007638529, -0.023593964, 0.0110168485, 0.027786337, -0.012977359, -0.017678514, -0.012339684, -0.025181368, -0.0002726229, 0.009246283, -0.019062404, -0.056549538, 0.017882027, 0.0048944927, -0.0045756553, -0.023349749, -0.004585831, 0.0022132061, -0.0028559687, -0.013845682, 0.16704364, -0.016308192, -0.00056856504, 0.027867742, 0.012088684, -0.003805697, 0.020174943, -0.0019842538, -0.010650525, 0.0053218706, 0.0141373845, -0.0028339215, -0.023024129, 0.0012236232, 0.0225764, -0.0034631165, -0.03180233, -0.017610677, 0.0018468824, 0.011240713, 0.010345255, -0.019103106, 0.0006029079, 0.0010709882, 0.017977001, -0.0048673577, -0.02295629, -0.0014500316, -0.02218294, 0.019903593, -0.009829688, 0.036333892, 0.010094255, 0.01427306, -0.0070890435, 0.014883599, 0.019672943, -0.009022419, 0.008846041, 0.0052913437, -0.017732784, -0.0134522235, 0.0036801973, -0.010419876, 0.0052540326, 0.024530126, -0.009035987, 0.0030408266, -0.033321895, 0.018139811, -0.011593469, 0.014924302, 0.0029628132, 0.019374458, -0.040187076, -0.035546973, -0.005840829, 0.019767916, 0.0015695957, 0.02174878, 6.29619e-05, 0.005389708, -0.018655378, 0.015344896, -0.0071229623, 0.040078536, -0.024977855, 0.011267848, 0.027040122, -0.027623527, -0.0074892864, -0.00084076397, 0.007787772, -0.013655736, -0.028790336, -0.008506852, 0.05220792, 0.017217217, 0.034271624, 0.01319444, 0.016023273, 0.0009997586, -0.0050539114, -0.013533629, 0.018478999, -0.041679505, 0.009673662, -0.016253922, -0.009626175, -0.017773487, 0.0023624492, 0.002606665, -0.028030552, 0.0060613016, -0.0016696563, -0.023824614, 0.018560404, 0.020324185, -0.009022419, -0.0035580893, -0.010094255, 0.048164792, -0.030011415, 0.0020317403, -0.011118605, 0.010419876, -0.015222788, 0.044067394, 0.0069398005, -0.010833686, 0.019347323, -0.003863359, -0.00087001896, -0.024910018, 0.023865316, 0.045559824, 0.0048470064, -0.026985852, 0.04916879, -0.002698246, -0.034651514, -0.016199652, -0.018071974, 0.021531697, -0.017271487, -0.02135532, -0.0010421572, 0.011661307, -0.01775992, -0.0053523974, 0.006797341, 0.033783194, 0.00035826804, 0.0014059371, -0.028681796, -0.005359181, -0.018668946, -0.015127815, -0.010826902, -0.018940296, 0.0011422179, -0.017298622, 0.0072722053, 0.030988278, -0.01515495, -0.019808618, 0.012624603, 0.025927583, 0.0006321629, -0.009632959, -0.006068086, 0.010826902, 0.02151813, 0.009015636, 0.0024726854, -0.03353898, -0.029414443, -0.03174806, -0.025045693, 0.012556765, -0.0094565805, 0.0017357981, 0.005840829, -0.009409094, -0.013208007, 0.002783043, -0.17116818, -0.0010285897, 0.0023624492, -0.03014709, 0.022264345, 0.023146236, 0.025629098, -0.0030408266, 0.028274769, -0.0067532468, 0.018397594, 0.018126244, -0.0047181146, -0.008656095, 0.006505639, 0.014435871, -0.015141383, 0.010603039, 0.022861317, -0.0046875877, 0.004382318, -0.012319333, 0.03562838, 0.0028271377, 0.023051264, 0.023064831, 0.009992499, 0.023186939, 0.0013380994, -0.009205581, -0.0058917073, -0.019957863, 0.0236211, -0.018343324, 0.012265063, 0.0018994566, 0.019550836, -0.00623768, -0.008459366, 0.025303476, 0.032507844, 0.016131815, 0.017081542, -0.028681796, 0.0210704, -0.008289771, -0.0038226566, -0.027230067, 0.018546836, -0.020012133, 0.023078399, -0.016973002, 0.00995858, 0.010508065, 0.028898876, -0.0006788013, -0.00016683844, 0.027582824, -0.0019079363, -0.012617819, -0.026904447, 0.0024964288, -0.012095468, -0.006424234, -0.002493037, -0.028111959, -0.012543198, 0.017461434, -0.032100815, 0.01603684, -0.032345034, 0.002881408, 0.0043009124, -0.0017137509, 0.0068143005, 0.024313046, 0.0070958273, 0.018872458, 0.016023273, -0.0031527588, -0.012509279, 0.025045693, -0.011654523, -0.0078963125, 0.0073603946, 0.0009310729, -0.0141373845, 0.0031357994, 0.005342222, 0.02583261, 0.008947797, -0.010704795, 0.021423157, -0.00835761, 0.028329039, 0.00045196892, -0.0022386452, -0.0119190905, 8.0557315e-05, -0.011457793, 0.0055796537, -0.00087765075, -0.0012626299, 0.00992466, 0.035601243, 0.0007589347, 0.0061630583, 0.0101960115, 0.040892586, -0.01520922, -0.013377601, -0.017407164, -0.00020881303, 0.046916578, -0.004789344, 0.016457435, -0.008615392, -0.0032256844, -0.0033138734, 0.0037548188, 0.062139366, -0.009788985, -0.004378926, 0.025466286, 0.006068086, -0.03353898, -0.11765777, 0.0027660837, 0.008669663, 0.046211068, 0.002253909, -0.0007411273, -0.0043924935, 0.027202932, -0.044067394, 0.029550118, -0.011953009, -0.0070279897, 0.00016493049, 0.011912307, -0.013676087, -0.020283483, -0.0141916545, -0.012353252, -0.015670517, 0.02716223, 0.024123099, -0.0048470064, 0.015114248, 0.012821333, 0.0055593024, -0.0075706914, -0.027257202, 0.016688082, 0.030337036, 0.00755034, 0.0018791053, -0.00677699, 0.0055626943, -0.02140959, -0.014598681, -0.026850177, 0.003924413, -0.0021063618, 0.034651514, 0.0012227752, -0.012068333, 0.016050408, 0.007957366, 0.016253922, 0.0022708683, -0.018329756, -0.0045722635, 0.02069051, -0.03334903, -0.017990569, -0.017285055, -0.0019401592, -0.02079905, -0.029604388, -0.0010811639, -0.03424449, 0.0011820726, 0.01427306, -0.010704795, -0.020758348, -0.005162452, 0.014652951, -0.014747924, 0.008398312, 0.013635385, -0.000112038266, 0.011552767, -0.017366461, 0.012237928, -0.006020599, 0.003262995, 0.017434299, -0.0076249614, 0.008723933, -0.036713783, -0.0052302894, -0.024747206, -0.00092174526, 0.02035132, 0.020486997, -0.012536414, -0.018275486, 0.021368887, -0.032833464, 0.012957008, 0.020622673, 0.009700797, 0.029577253, 0.02196586, -0.022196507, -0.0065836525, 0.048924576, -0.022115102, 0.014734357, -0.006200369, 0.011294983, -0.023010561, -0.007197584, -0.0126042515, 0.004155061, -0.0126585215, -0.0029628132, -0.072342165, 0.015141383, -0.0060375584, -0.019672943, -0.0033545762, -0.021979427, 0.004830047, -0.015385599, 0.0063835313, 0.028220499, -0.029115956, 0.021504562, -0.023376884, -0.026863744, -0.033159085, -0.007672448, 0.0410554, -0.020988995, 0.00221999, 0.03381033, 0.0029797726, 0.010813335, 0.008852825, 0.027637094, -0.008154096, -0.007787772, -0.012421089, 0.028763201, 0.0079234475, 0.000929377, 0.008941013, -0.011430658, -0.0049284115, 0.0110439835, -0.009788985, -0.0013143561, -0.0060782614, -0.014598681, 0.018004136, 0.015358464, -0.020500565, -0.03429876, 0.006173234, 0.019008134, 0.009103824, 0.010847254, -0.012719575, -0.00020425518, 0.0038124807, -0.002013085, 0.033946004, 0.018804621, -0.0063292608, -0.012895954, -0.025696935, -0.0017603893, -0.0087714195, -0.011946226, 0.013079115, -0.016511705, 0.045776904, 0.01991716, -0.00073307153, -0.013961006, -0.0074689346, -0.02252213, -0.028871741, -0.0017603893, -0.029170226, -0.02727077, -0.012624603, -0.0046672365, 0.005664451, 0.0256698, -0.000959904, -0.028980281, -0.013011278, -0.016945867, -0.018261919, -0.0043585747, 0.0030476104, 0.024408018, -0.0112067945, -0.003357968, 0.019659376, -0.0022386452, 0.008703582, 0.011640956, -0.011247497, 0.010921876, 0.0017977001, 0.02511353, 0.012292198, 0.00647172, 0.011573118, 0.003237556, 0.013221575, 0.01263817, 0.018980999, 0.008601825, 0.0015390686, 0.029740063, 0.011227146, -0.010636957, -0.0050233845, 0.041516695, -0.026036123, -0.019672943, 0.028980281, -0.01305198, -0.0066718413, -0.004531561, -0.0076249614, 0.010582687, -0.035329893, 0.03348471, -0.014680087, -0.02218294, -0.02135532, 0.024028126, 0.0039379806, 0.029197361, 0.00041423418, -0.020785483, 0.018655378, -0.008805338, 0.044664364, 0.0118919555, 0.03391887, -0.02522207, 0.015738355, 0.01278063, -0.027569257, -0.013587899, 0.011966577, 0.019198079, -0.011824118, 0.009008852, -0.02229148, 0.07478432, 0.029088821, -0.02705369, 0.013703222, -0.02090759, 0.02180305, 0.012665305, -0.0023488817, -0.013459007, -0.024787908, 0.02849185, -0.01725792, -0.0014500316, -0.04431161, 0.004711331, 0.0251678, 0.0028627524, 0.023729641, -0.019428728, -0.011098254, 0.016783057, -0.016810192, 0.0064004906, -0.000569413, -0.028193364, 0.01603684, 0.0069431923, -0.0029746848, 0.03237217, -0.042249344, 0.009565121, -0.017977001, -0.046509553, -0.014028844, 0.0070347735, 0.019984998, 0.008086259, -0.01765138, 0.03158525, 0.023159804, -0.019944295, 0.017542839, -0.012868819, -0.019225216, 0.013574331, -0.0034291977, -0.01084047, 0.019754348, -0.0043924935], metadata={'file_name': 'data/1406.4214.pdf', 'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024.'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={<NodeRelationship.SOURCE: '1'>: RelatedNodeInfo(node_id='f000ec98-9631-440a-8f6c-4181172bc173', node_type=<ObjectType.DOCUMENT: '4'>, metadata={'file_name': 'data/1406.4214.pdf', 'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024.'}, hash='1cfd3cd76f12c5dae17e07638b6b4584e3c04282aae51284801629e0db25bb45'), <NodeRelationship.NEXT: '3'>: RelatedNodeInfo(node_id='29251e28-f9ea-443c-a9b8-0dc89102e06b', node_type=<ObjectType.TEXT: '1'>, metadata={}, hash='1d732c8ed00fc6c1a785480fefb3d6862eb577d49b8ab693c1c123cf167b49e9')}, hash='a0107c8821ed081319de3ddb37dc2023d52c0b2413f4a824cdefbc2378ee4998', text='1 Title: Nanopore sequencing of the phi X 174 genome  \\n \\nAuthors: Andrew H. Laszlo1, Ian M. Derrington1, Brian C. Ross1, Henry Brinkerhoff1, Andrew  \\nAdey2, Ian C. Nova1, Jonathan M. Craig1, Kyle W. Langford1, Jenny Mae Samson1, Riza Daza2, \\nKenji Doering1, Jay Shendure2, Jens H. Gundlach1,† \\n \\nAffiliations: 1Department of Physics, University of Washington, Seattle, WA 98195, USA  \\n2Department of Genome Sci ences, University of Washington, Seattle, WA 98195, USA  \\n†Corresponding author. E -mail: gundlach@uw.edu  \\n \\nAbstract : Nanopore sequencing of DNA is a single -molecule technique that may achieve long \\nreads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we \\nbuild on recent progress with respect to nanopore resolution and DNA control to interpret the \\nprocession of ion current levels observed during the translocation of DNA through the pore \\nMspA. As approximately four nucleotides affect the ion current of each level, we measured the \\nion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer \\nmap is highly predictive of ion current levels of previously unmeasured sequences derived from \\nthe bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X \\n174 up to 4,500 bases in length  that can be unambiguously aligned to the phi X 174 reference \\ngenome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and \\npolymorphism detection. All methods and data are made fully available.  \\n \\nMain Text : DNA sequencing is revolutionizing biomedical and other life sciences research \\nthrough its expanding scope (1) and has a rapidly growing presence in clinical medicine (2). \\nThese developments are driven in part by the successful completion of the Human Genome \\nProject (3) and in part by the introduction of new sequencing technologies that have  dramatically \\nreduc ed the cost of DNA sequencing (4). Although such ‘next -generation’ sequencing  \\ntechnologies have matured considerably since  early proof -of-concept s (5-7), nearly all remain  \\nlimited to short sequence reads (with the exception of real -time sequencing from elongating \\npolymerases (8)) and rely on  complex, expensive instrumentation. Most platforms are also \\nlimited with respect to speed and require extensive sample preparation steps prior to sequencing .  \\n \\nNanopore sequencing, independently proposed by Church and Deamer in the mid -1990s, has \\ntremendous potential to overcome these limitations and  achieve long reads, low cost, and high \\nspeed while requiring minimal sample preparation and instrumentation  (i.e. ‘tricorder’ -like DNA \\nsequencing devices) (9-12). However, this promise has faced substantial technical challenges, \\nsuch that despite nearl y 20 years of effort, nanopore -derived sequence reads that align to \\ncomplex, natural DNA sequences have yet to be demonstrated.  \\n \\nIn nanopore devices directed at DNA sequencing, a salt solution is divided into cis and trans  \\nwells  by a thin membrane. A singl e nanometer -scale pore in the membrane connects the cis and \\ntrans  wells electrically. When a voltage is applied across this membrane, ion current flows \\nthrough the pore; this current provides the primary signal. DNA is negatively charged and is \\nelectrophoretically attracted into the pore. When single -stranded (ss) DNA e nters the pore, it \\nblocks some fraction of the ion current. The fraction of the ion current that is blocked depends on \\nthe identity of nucleotides within the pore (13-15). Key challenges of this technique are single - 2 nucleotide resolution and control of the DNA translocation. Single -nucleotide resolution was \\nrecently enabled through the development of MspA , a protein pore with a short and narrow \\nconstriction (11, 13, 15, 16 ). DNA translocation control was also recently enabled through the \\nuse of molecular motors such as phi29 DNA Polymerase (DNAP) (11, 17 ) (Fig. 1 ). \\n \\nWe have found t hat the currents in the MspA pore are determined by about four nucleotides at \\nany given time (11, 15 ).', start_char_idx=1, end_char_idx=4095, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.87023771),\n",
      "                        NodeWithScore(node=TextNode(id_='9fe90d42-9f20-4967-b144-96163b8329ee', embedding=[-0.023894642, -0.0154500585, -0.018124629, 0.0059532956, -0.0083563365, 0.020907812, -0.0070054745, -0.004497216, -0.039208934, -0.0162918, 0.0029630715, 0.0150156105, -0.0025914146, 0.010847624, 0.011540026, 0.0022621844, 0.011092002, -0.036140647, -0.0023894643, -0.010216317, -0.03247499, -0.011859074, 0.010521788, -0.002109449, -0.004531157, 0.006587997, 0.024152596, -0.020378329, -0.03426709, -0.0060551194, -0.002413223, 0.010854413, -0.019794539, 0.0012940103, 0.02029687, -0.009686833, 0.0147576565, -0.0024709231, 0.035163138, -0.036059186, 0.027994746, 0.00073355535, 0.010487847, 0.0034348548, -0.019672351, -0.017133545, 0.012619358, 0.007691088, -0.0122324275, -0.005688554, 0.012524323, 0.013080959, 0.010515, 0.0007819216, -0.024451278, -0.029786844, 0.015993118, -0.027424533, -0.011220979, -0.012958771, 0.00649975, 0.004419151, -0.029623926, 0.020147528, -0.027994746, 0.014214597, 0.017092815, 0.014947728, -0.0051760413, 0.0131149, 0.032339226, -0.005335565, 0.00076919363, 0.00671358, 0.016196765, -0.023528077, -0.0035400726, -0.0013534075, -0.011709732, 0.021097884, 0.013019864, -0.012218851, 0.0048434166, 0.0035197078, 0.019685926, 0.017907405, 0.016468296, 0.01781237, 0.006167126, -0.013915914, 0.049988676, 0.040267903, 0.014608315, 0.016115308, -0.012483593, 0.0123614045, -0.011580756, 0.008152689, 0.0040186443, -0.027153002, -0.042521603, -0.0071412395, 0.0033500018, -0.010080552, -0.04311897, -0.011770827, 0.0125378985, -0.020391906, 0.00682898, -0.0058141365, -0.0070326277, 0.020867083, -0.02587681, -0.025550975, 0.04393356, 0.002190908, 0.02875503, -0.005664795, 0.0017106391, -0.004089921, 0.040267903, 0.017880252, 0.010535365, -0.015803047, 0.035760503, 0.02211612, -0.019414397, -0.026718555, 0.012436075, -0.017201427, 0.022211157, -0.008431007, 0.010250258, 0.015273564, -0.026379142, 0.0010861201, -0.020147528, -0.015762318, -0.029895457, -0.06304927, 0.013549348, 0.0239761, -0.01649545, -0.0034637048, 0.018599806, 0.025564551, 0.01354256, 0.0010275714, 0.013997373, 0.005216771, 0.004008462, 0.0027509385, -0.006394532, 0.011411049, 0.010684706, 0.016712673, -0.0006660971, 0.030139832, -0.011383897, -0.015925236, -0.018450465, -0.0005608792, -0.013440737, 0.0019329544, 0.006211249, 0.031063035, 0.020269716, 0.012585416, -0.019998185, -0.028972253, -0.026935779, 0.0019261661, -0.033886947, 0.0066728503, 0.002601597, 0.007093722, 3.7680093e-05, -0.006306285, -0.0035298902, -0.024532737, 0.010976601, -0.0033466076, 0.027057966, 0.027886134, -0.037959896, 0.01012807, 0.03546182, 0.007012263, 0.007813277, -0.016468296, 0.008254513, -0.011777615, -0.0010139949, 0.004826446, -0.6303841, -0.036520787, -0.026637096, -0.033533957, -0.0070326277, 0.01564013, 0.018640537, 0.019387243, -0.0025880206, 0.028510652, -0.017201427, -0.006061908, 0.010895142, -0.029705385, -0.03554328, -0.020867083, 0.0116690025, -0.014241749, 0.031171646, -0.00069749274, -0.025374481, 0.0021603608, -0.00459904, 0.019387243, 0.001015692, -0.0055765477, 0.0129248295, -0.013617231, -0.019414397, 0.00013374975, -0.036656555, 0.0032736338, -0.0015825109, -0.0005965175, 0.033533957, -0.011947321, 0.02098927, 0.03239353, 0.019319361, 0.02196678, -0.023134358, -0.007283793, -0.010732224, 0.008627866, -0.008424219, 0.005498483, -0.0012066115, -0.004982576, -0.005848078, -0.02192605, 0.0043275096, -0.0034586138, 0.0055052713, 0.021002848, -0.01175725, 0.002818821, 0.024614196, -0.020079644, 0.033968408, -0.0017904011, 0.008899396, 0.004419151, -0.001204066, -0.028021898, -0.004690681, 0.0129995, -0.036059186, 0.00019346514, 0.018952796, 0.008159477, 0.015925236, 0.036575094, -0.008044077, 0.00612979, -0.007562111, 0.031524636, 0.008268089, 0.03065574, 0.0001240977, 0.014526856, -0.0019125897, 0.011037695, -0.027220884, 0.00077301206, 0.041707013, 0.017920982, -0.033859793, 0.020473363, 0.007847218, -0.017486533, -0.0057326774, -0.0049995463, -0.0007445862, -0.034755845, -0.0101416465, -0.00065039925, -0.010257047, -0.0230529, 0.0040220385, -0.038584415, -0.025754623, 0.0115739675, 0.00021043577, -0.028890794, 0.046838928, -0.010250258, 0.011343167, -0.0020992665, 0.029895457, -0.027112273, -0.0041917446, -0.013013076, -0.017948134, -0.007202334, -0.01338643, -0.031008728, 0.016997779, 0.024234055, -0.014839116, -0.019550161, 0.00974114, -0.014716927, -0.02161379, -0.012979135, 0.017459381, 0.016278224, 0.005400053, -0.014554009, -0.010005881, -0.00574286, -0.007847218, -0.007894736, 0.018708419, -0.008057654, 0.0050606406, 0.019278632, 0.0017259127, 0.016115308, 0.0062757377, -0.038204275, 0.006367379, 0.0077861235, 0.011947321, -0.00095120363, 0.004334298, -0.030818658, -0.023853913, -0.0076028407, 0.015178529, -0.014703351, 0.022632027, 0.011024119, 0.0064454437, 0.031497482, 0.0076028407, 0.013447524, -0.015151375, -0.037199613, -0.03328958, -0.021464448, 0.004439516, 0.017160697, -0.016020272, -0.010813683, 0.020228988, -0.015477211, 0.031090187, 0.025849659, -0.014377515, -0.025741046, 0.027601028, -0.0016105125, -0.023568805, -0.01827397, -0.014431821, 0.036004882, 0.011845497, 0.004690681, -0.0029308272, -0.010623612, -0.01505634, -0.0056580068, 0.0041374387, 0.0033669723, 0.0056817657, 0.01908856, 0.033506803, 0.01505634, 0.027560297, 0.031361718, 0.023310853, -0.011512873, -0.002659297, 0.025999, 0.0030394392, 0.0048434166, 0.030981576, -0.0069172275, -0.0100126695, 0.020052493, 0.012402134, -0.0038693028, -0.004412363, -0.024763538, 0.005423812, -0.037335377, 0.011071636, 0.010372447, 0.012062721, -0.009754716, -0.0107050715, -0.020622706, -0.023962524, -0.016997779, -0.0036962025, 0.032502145, -0.008831514, 0.0074534994, -0.0035027373, 0.0041374387, -0.0008722902, 0.02382676, 0.016590485, 0.016210342, -0.010413176, 0.017459381, 0.0092252325, 0.02230619, 0.012829794, 0.0009749625, -0.014119561, -0.0036079553, 0.0020262927, 0.006740733, 0.01105806, 0.024627773, 0.044368006, -0.0010683009, 0.012463228, 0.010637188, -0.003713173, 0.00454134, 0.0042494447, -0.022564145, 0.007867582, -0.0133592775, 0.0145811625, 0.014391092, 0.0067848563, -0.0038523322, -0.0019685926, 0.008634655, -0.033153817, -0.009184503, -0.0031141099, -0.034755845, -0.0110105425, 0.017758064, 0.021287953, 0.007949041, 0.00041365903, 0.005478118, 0.005641036, -0.010189164, -0.004551522, -0.0230529, 0.015490788, -0.005983843, -0.02608046, 0.010698283, -0.021274377, -0.011424625, 0.0066185445, -0.021287953, 0.002654206, 0.00012579477, -0.000365717, -0.003814997, 0.021111459, -0.012632934, -0.009116621, -0.014798386, 0.021763131, 0.01098339, -0.02587681, -0.04268452, -0.02130153, -0.011112366, -0.009904058, -0.01008734, 0.0039371853, 0.011580756, -0.0031514454, -0.0010496333, -0.046078645, -0.0056546126, 0.03679232, -0.023813182, -0.0021230255, -0.006119608, 0.015273564, -0.0071072984, -0.010623612, -0.00954428, 0.04024075, -0.0015248108, 0.0010895142, -5.833653e-05, 0.012904464, -0.019509433, 0.016454719, -0.019767385, -0.0017411863, -0.0010767862, -0.014771233, 0.006146761, -0.027696062, 0.012850159, 0.0067101857, -0.020880658, -0.002162058, -0.021369413, -0.012911253, 0.02766891, 0.042548753, 0.03193193, 0.0008858667, 0.0066049676, -0.0059465077, -0.0010963025, -0.008295243, -0.010365658, 0.022781368, -0.0012125513, -0.0072905812, 0.014268903, 0.0072362754, 0.0015545094, 0.012578628, 0.007949041, -0.010331717, -0.042983204, -0.019265056, -0.005844684, -0.0031480512, -0.01109879, 0.013433948, 0.018966373, 0.012897676, -0.015042763, 0.020337598, 0.012870523, -0.017215004, 0.002302914, -0.0034687961, -0.012931617, 0.03627641, 0.023799608, -0.015504364, 0.0051862234, 0.009612163, 0.0044293334, 0.024220478, 0.00993121, -0.029596772, 0.0061162137, 0.0069477744, 0.009048738, -0.0033550928, -0.0059193545, 0.00020078373, 0.013148841, -0.0059091724, 0.008736478, 0.028510652, -0.024043983, -0.03443001, -0.009136985, -0.009652892, 0.018640537, -0.0060313605, -0.020391906, -0.012619358, -0.02284925, 0.00013841667, -0.037959896, 0.0038794852, 0.0013440736, -0.028809335, -0.026487755, -0.019604467, -0.0081459, -0.030682893, -0.034239937, -0.0005061489, -0.030384209, -0.036140647, -0.0072226985, 0.037091002, 0.008512466, 0.008797573, 0.0069172275, 0.019903151, 0.029623926, -0.021803861, 0.006584603, 0.0034603106, -0.024926456, 0.034918763, -0.005766619, 0.01478481, -0.003764085, -0.0041578035, 0.0024200114, 0.012578628, -0.010073764, 0.023351582, 0.038855948, -0.020731317, 0.024383396, 0.002683056, 0.022835676, -0.015531518, -0.015300717, -0.00086041074, -0.03372403, -0.007887947, -0.01191338, -0.016237495, -0.016658368, 0.010895142, 0.008824726, -0.031008728, -0.022835676, 0.006964745, -0.027180156, 0.011899804, 0.009795446, 0.012856946, 0.03581481, -0.012503957, 0.0005053004, -0.025252292, 0.0006588846, -0.011702944, -0.0053830827, 0.029895457, 0.0024284967, 0.002652509, 0.032502145, -0.00034132172, -0.016902745, 0.006543874, 0.009136985, 0.004527763, 0.03147033, 0.013216724, -0.028103357, 0.016454719, -0.025985423, -0.004337692, -0.002985133, 0.025401633, -0.017472956, -0.0058277133, 0.0073584635, 0.013060594, 0.023310853, -0.0072905812, -0.028999407, -0.02134226, -0.024342667, 0.015979541, 0.0043784217, -0.0060585137, 0.0022163638, -0.0087907845, -0.025062222, -0.031497482, -0.023921795, -0.011974474, -0.020147528, 0.037118156, 0.0030682893, 0.036004882, -0.019604467, -0.020120375, 0.03366972, 0.008804361, -0.0059974194, 0.0023640082, -0.003740326, -0.006818798, 0.014065255, 0.03510883, -0.007297369, -0.007894736, -0.027410956, 0.01564013, -0.00019198022, 0.0055663656, -0.0121034505, 0.013121689, -0.029786844, -0.03130741, -0.002002534, -0.006856133, -0.017133545, -0.016536178, -0.016454719, 0.01781237, -0.018654112, 0.024736386, -0.0017717334, 0.03592342, -0.007582476, 0.012870523, 0.007514593, 0.043716334, -0.012782276, -0.008756843, -0.044693843, 0.004758564, 0.02821197, -0.00037717217, 0.018233242, -0.0013720752, -0.0046397694, 0.003954156, 0.007718241, -0.0070733572, -0.033126663, -0.010643977, -0.01478481, 0.018436888, -0.013182783, 0.0073720403, -0.05669547, -0.01132959, -0.0068663154, -0.0041102855, 0.025863234, -0.008247725, -0.008118748, 0.027288767, 0.026745707, 0.0077997, -0.023528077, 0.04121826, 0.018192511, 0.008234148, -0.040539432, -0.008179842, -0.0034620077, 0.0033024838, 0.05175362, 0.016522601, -0.010508212, -0.009605374, -0.007881159, -0.0036520788, -0.025170833, -0.048984017, 0.007820065, -0.00022804279, 0.0021467842, -0.0067203683, -1.4371999e-05, -0.020541247, 0.03942616, -0.020554822, 0.0043309038, -0.026908625, -0.023487346, -0.019237902, -0.01160112, 0.0091437735, 0.01105806, 0.004795899, -0.0041815625, 0.011906591, -0.0054611475, 0.017310038, -0.014608315, 0.0069036507, 0.030519975, -0.0038930618, -0.00710051, 0.0022655786, 0.0069375923, 0.0052642883, -0.0055969125, 0.0011293952, 0.03706385, -0.034620076, 0.015246411, 0.005739466, -0.004629587, -0.0063266493, 0.00438521, -0.0019058014, -0.026351988, -0.0046465574, 0.009666469, 0.0022893376, 0.00428678, 0.0030801687, -0.030981576, -0.0034212782, -0.0122324275, -0.033995558, 0.0030207715, 0.029053712, -0.0461058, -0.011200613, -0.003329637, 0.00029931942, 0.01873557, 0.011940532, 0.023704572, -0.020079644, 0.011458567, -0.01664479, 0.004388604, 0.007181969, -0.035027374, -0.029895457, 0.0053932653, -0.028021898, -0.0042833863, 0.008736478, -0.019889574, -0.009992305, -0.00026856016, -0.014540433, 0.015762318, 0.0044530923, -0.0015604491, 0.0032125397, -0.0021773314, 0.008193418, -0.0022570933, -0.019305784, 0.018531924, -0.016862014, 0.0015723285, 0.0056003067, -0.0028782182, -0.0011370319, 0.0023758877, -0.010745801, 0.01618319, -0.0028544595, -0.021410143, 0.0012023689, -0.0047551696, 0.003706385, 0.0053219884, -0.009340633, -0.025279446, -0.015069916, -0.003363578, -0.0037980261, 0.02250984, -0.019998185, -0.0012880706, 0.020378329, -0.009924422, -0.027886134, 0.014486127, 0.012829794, -0.0133592775, -0.030845812, 0.0068323743, -0.00981581, 0.025347328, 0.016536178, -0.0113160135, -0.009761504, 0.005335565, -0.0235145, -0.028510652, -0.015517941, 0.0497443, 0.0139702195, 0.025795352, 0.020975694, 0.01202878, 0.008003348, -0.0030156802, -0.016128883, -0.004768746, 0.02153233, -0.015164952, 0.016970627, 0.030791504, -0.030818658, 0.008879032, 0.017676605, 0.0057801954, -0.019292207, 0.034565773, 0.0072362754, 0.016061, 0.001191338, -0.0021298137, 0.003219328, 0.026867896, 0.029678231, -0.03519029, -0.014445397, 0.0063334377, -0.008607502, -0.022564145, 0.008953703, -0.004711046, 0.00070725085, -0.0065167206, 0.011431414, -0.03138887, 0.011662215, -0.0005031791, 0.017948134, 0.005206588, 0.0038285733, 0.024071136, 0.009367785, 0.007609629, 0.005009729, -0.012076298, -0.021790285, -0.004819658, 0.012775487, -0.00030292568, -0.0014857784, -0.012721182, -0.018151782, -0.016033849, -0.0034620077, 0.007854006, -0.00665588, -0.024505585, 0.01132959, 0.028592112, -0.0129248295, -0.018029593, -0.010990177, -0.023718148, -0.00090962555, 0.010053399, -0.019495856, -0.05664116, 0.017133545, 0.0044666687, -0.0050776117, -0.024247631, -0.005433995, 0.0014535341, -0.003764085, -0.015436482, 0.16802278, -0.01726931, -0.0007280399, 0.027275192, 0.010956236, -0.0043784217, 0.019971034, -0.0020958723, -0.010005881, 0.005145494, 0.014906999, -0.0034077018, -0.021722402, 0.0015137799, 0.022360498, -0.0050843996, -0.03174186, -0.017282886, 0.0022723668, 0.011920168, 0.009238809, -0.01816536, -0.00082562095, 0.00044887306, 0.018694842, -0.0042630215, -0.022224732, -0.001978775, -0.022048239, 0.020731317, -0.010745801, 0.03752545, 0.01175725, 0.013515407, -0.009075891, 0.015124222, 0.02021541, -0.009123409, 0.008003348, 0.004795899, -0.019740233, -0.012850159, 0.0035027373, -0.010569306, 0.005552789, 0.02320224, -0.007962618, 0.0024573468, -0.032719366, 0.019115714, -0.0111870365, 0.0154500585, 0.0019142866, 0.020147528, -0.040892422, -0.036032036, -0.006316467, 0.02017468, 0.0017734305, 0.022686334, -9.042162e-06, 0.0066151503, -0.019183597, 0.015925236, -0.0069511686, 0.03967054, -0.023840336, 0.011261708, 0.027370226, -0.028239122, -0.0073245224, -0.00017882798, 0.0069274097, -0.013875185, -0.028836489, -0.008478525, 0.052079458, 0.016386837, 0.033886947, 0.013997373, 0.01626465, 0.0020839928, -0.0046838927, -0.014499703, 0.018219665, -0.041652706, 0.007711453, -0.016318955, -0.011078425, -0.016563332, 0.001868466, 0.0010674524, -0.028836489, 0.0064013205, -0.0016257861, -0.024817845, 0.020853506, 0.02075847, -0.008953703, -0.0029189477, -0.009530704, 0.048631027, -0.028619265, 0.0023690995, -0.011017331, 0.010548942, -0.015002034, 0.044911064, 0.0060449373, -0.011051272, 0.01722858, -0.004388604, -0.0019312573, -0.0257682, 0.02436982, 0.045155443, 0.0047517754, -0.027478838, 0.048712485, -0.0019431367, -0.035380363, -0.016128883, -0.017174274, 0.021274377, -0.017432228, -0.021396566, -0.00023949797, 0.010929083, -0.01781237, -0.004527763, 0.0058616544, 0.03318097, 0.00020640524, 0.00030144074, -0.028619265, -0.0053253826, -0.018124629, -0.013691901, -0.010100917, -0.018803455, 0.0009206565, -0.017744487, 0.008125536, 0.03022129, -0.017255733, -0.02048694, 0.013325336, 0.025075797, -0.00021595122, -0.010677918, -0.0063334377, 0.012001627, 0.022021085, 0.00845816, 0.002111146, -0.032665063, -0.028076205, -0.032800827, -0.026202647, 0.0128841, -0.009781869, 0.00093847566, 0.006818798, -0.009340633, -0.013712266, 0.0035163139, -0.17128114, -0.002031384, 0.0022078785, -0.029786844, 0.021871744, 0.024261208, 0.025075797, -0.002789971, 0.027207308, -0.0074670757, 0.018912066, 0.017459381, -0.0035129196, -0.008010136, 0.0062723435, 0.013345701, -0.016359683, 0.011193825, 0.02297144, -0.0035909845, 0.0044021807, -0.01137032, 0.036385022, 0.0035027373, 0.020772047, 0.023378735, 0.0094831865, 0.023582382, 0.002652509, -0.008566773, -0.006689821, -0.02052767, 0.021668095, -0.018341852, 0.01338643, 0.002952889, 0.018844184, -0.0070054745, -0.008437796, 0.027465262, 0.03274652, 0.016088154, 0.0162918, -0.029108018, 0.02048694, -0.008858667, -0.0046974695, -0.026718555, 0.019292207, -0.02021541, 0.022414804, -0.017595146, 0.010433542, 0.00915735, 0.028564958, -0.000737798, 0.00021934535, 0.027886134, -0.0019007102, -0.012252793, -0.027601028, 0.0020805988, -0.011689368, -0.0066287266, -0.001667364, -0.028347734, -0.0129248295, 0.018450465, -0.032040544, 0.0154500585, -0.03318097, 0.004415757, 0.0030241658, -0.0018803454, 0.006258767, 0.02460062, 0.006825586, 0.018654112, 0.01672625, -0.0030479245, -0.011417838, 0.024967186, -0.012388557, -0.0070869336, 0.006316467, 0.0018922249, -0.0138887605, 0.0021060547, 0.0053695063, 0.025931118, 0.007982982, -0.010949448, 0.020364752, -0.0073041576, 0.028347734, -0.00079040695, -0.002706815, -0.012307098, -0.000777679, -0.011628273, 0.004884146, -0.0031022306, -0.0009478095, 0.010066976, 0.035760503, 0.00035468608, 0.0055663656, 0.010080552, 0.041761316, -0.01420102, -0.012497169, -0.017065663, -0.0005897293, 0.04765352, -0.0046261926, 0.016848437, -0.008315607, -0.0031802955, -0.0031836894, 0.0029817391, 0.06228899, -0.009985517, -0.0042630215, 0.024234055, 0.0058854134, -0.03364257, -0.117735416, 0.004415757, 0.009238809, 0.04621441, 0.0014518371, -0.0019872603, -0.0035536492, 0.026379142, -0.04276598, 0.029678231, -0.012938405, -0.0065981797, -0.00028510654, 0.011288861, -0.014431821, -0.019292207, -0.014662622, -0.011451779, -0.015110645, 0.027153002, 0.023853913, -0.0054272064, 0.014554009, 0.013352489, 0.0054543596, -0.008376701, -0.026637096, 0.018178934, 0.030764353, 0.0065608444, 0.0025320174, -0.0070326277, 0.0051387055, -0.021179343, -0.014472551, -0.027410956, 0.0035129196, -0.0011921865, 0.033778336, 0.0020687194, -0.012137393, 0.016386837, 0.0076775113, 0.013725842, 0.0008731387, -0.017622298, -0.003984703, 0.021315107, -0.033425346, -0.018056747, -0.016481873, -0.0015349932, -0.02196678, -0.03011268, -9.0792855e-05, -0.03318097, 0.0010114494, 0.014839116, -0.012476805, -0.020283293, -0.003658867, 0.014852692, -0.014078831, 0.008152689, 0.014038103, 0.00018773756, 0.010162011, -0.015667282, 0.012449652, -0.0060347547, 0.0029189477, 0.015314293, -0.0069240155, 0.009781869, -0.037552603, -0.004201927, -0.02436982, -0.0015324475, 0.020161103, 0.020269716, -0.012578628, -0.018721996, 0.020853506, -0.032719366, 0.012565052, 0.020201834, 0.009734351, 0.028809335, 0.020459788, -0.021993931, -0.007779335, 0.04765352, -0.021491602, 0.013915914, -0.0061637317, 0.011709732, -0.023568805, -0.007636782, -0.013400007, 0.0036962025, -0.012008416, -0.0036452906, -0.072009765, 0.014947728, -0.0060109957, -0.019685926, -0.0022825492, -0.022102544, 0.004300357, -0.015748741, 0.005708919, 0.027831828, -0.028999407, 0.023568805, -0.023066476, -0.02654206, -0.032420684, -0.0073720403, 0.040756658, -0.021396566, 0.0026474178, 0.033995558, 0.0030054979, 0.010718647, 0.007976195, 0.02754672, -0.008641443, -0.007426346, -0.012178121, 0.028890794, 0.00725664, 0.0027780915, 0.008464948, -0.012449652, -0.005179435, 0.011397473, -0.009367785, -0.0022910347, -0.0071616042, -0.015680859, 0.01714712, 0.012741546, -0.021369413, -0.03407702, 0.0077861235, 0.017988864, 0.008851878, 0.009775081, -0.012463228, -0.00019898059, 0.004388604, -0.0025999, 0.03407702, 0.018844184, -0.006258767, -0.013603654, -0.025428787, -0.0008145901, -0.008471737, -0.0117979795, 0.012734758, -0.01664479, 0.045644198, 0.019427974, 0.000105536084, -0.015110645, -0.0077589704, -0.024111867, -0.029732538, -0.0026185676, -0.029461008, -0.027071543, -0.013087748, -0.0058888076, 0.0049384525, 0.026895048, -6.3215586e-05, -0.028483499, -0.012694028, -0.017052086, -0.01741865, -0.0030937453, 0.0026898442, 0.023650264, -0.011173461, -0.003984703, 0.019034255, -0.0017564598, 0.008281666, 0.0123614045, -0.012015204, 0.01218491, 0.0019550163, 0.024777114, 0.011879439, 0.006520115, 0.010610036, 0.0039405795, 0.012510746, 0.01152645, 0.01912929, 0.008200207, 0.0013279516, 0.031253107, 0.011458567, -0.010929083, -0.0060279667, 0.041734166, -0.025890388, -0.02025614, 0.029162325, -0.01284337, -0.006727156, -0.0043716333, -0.007195546, 0.009965152, -0.0339141, 0.033425346, -0.014092408, -0.022279039, -0.02173598, 0.024437701, 0.0032804222, 0.028592112, 0.0014925665, -0.021260802, 0.019332938, -0.009394939, 0.044802453, 0.01202878, 0.035380363, -0.0255374, 0.015246411, 0.0125378985, -0.028510652, -0.013060594, 0.011675791, 0.020188257, -0.012001627, 0.009028373, -0.022550568, 0.07429061, 0.029406702, -0.027831828, 0.012632934, -0.02071774, 0.021803861, 0.012626146, -0.0019974427, -0.012904464, -0.024899304, 0.027533144, -0.016033849, -0.0019855632, -0.04404217, 0.005607095, 0.024994338, 0.0031853865, 0.023528077, -0.018667689, -0.011390684, 0.016902745, -0.01595239, 0.006041543, 8.2466635e-05, -0.027383802, 0.015748741, 0.007609629, -0.0022842463, 0.031850472, -0.041707013, 0.009136985, -0.017391497, -0.046838928, -0.014486127, 0.005980449, 0.020595552, 0.008166266, -0.016101731, 0.030954422, 0.022754217, -0.019699503, 0.017703757, -0.01350183, -0.017948134, 0.015341447, -0.004008462, -0.011702944, 0.0207992, -0.005522242], metadata={'file_name': 'data/1406.4214.pdf', 'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024, pp. 1-4.'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={<NodeRelationship.SOURCE: '1'>: RelatedNodeInfo(node_id='df6b2f05-4b13-4e04-97c4-e58170e9b714', node_type=<ObjectType.DOCUMENT: '4'>, metadata={'file_name': 'data/1406.4214.pdf', 'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 genome.\" Nanopore sequencing of the phi X 174 genome, 2024, pp. 1-4.'}, hash='1cfd3cd76f12c5dae17e07638b6b4584e3c04282aae51284801629e0db25bb45'), <NodeRelationship.NEXT: '3'>: RelatedNodeInfo(node_id='4e83bfa9-e6bc-4638-ba11-75f16e41aad8', node_type=<ObjectType.TEXT: '1'>, metadata={}, hash='1d732c8ed00fc6c1a785480fefb3d6862eb577d49b8ab693c1c123cf167b49e9')}, hash='5d968b55704a07153d37bf73047a18f67efc8e7642e2921c80d28507939242e2', text='1 Title: Nanopore sequencing of the phi X 174 genome  \\n \\nAuthors: Andrew H. Laszlo1, Ian M. Derrington1, Brian C. Ross1, Henry Brinkerhoff1, Andrew  \\nAdey2, Ian C. Nova1, Jonathan M. Craig1, Kyle W. Langford1, Jenny Mae Samson1, Riza Daza2, \\nKenji Doering1, Jay Shendure2, Jens H. Gundlach1,† \\n \\nAffiliations: 1Department of Physics, University of Washington, Seattle, WA 98195, USA  \\n2Department of Genome Sci ences, University of Washington, Seattle, WA 98195, USA  \\n†Corresponding author. E -mail: gundlach@uw.edu  \\n \\nAbstract : Nanopore sequencing of DNA is a single -molecule technique that may achieve long \\nreads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we \\nbuild on recent progress with respect to nanopore resolution and DNA control to interpret the \\nprocession of ion current levels observed during the translocation of DNA through the pore \\nMspA. As approximately four nucleotides affect the ion current of each level, we measured the \\nion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer \\nmap is highly predictive of ion current levels of previously unmeasured sequences derived from \\nthe bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X \\n174 up to 4,500 bases in length  that can be unambiguously aligned to the phi X 174 reference \\ngenome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and \\npolymorphism detection. All methods and data are made fully available.  \\n \\nMain Text : DNA sequencing is revolutionizing biomedical and other life sciences research \\nthrough its expanding scope (1) and has a rapidly growing presence in clinical medicine (2). \\nThese developments are driven in part by the successful completion of the Human Genome \\nProject (3) and in part by the introduction of new sequencing technologies that have  dramatically \\nreduc ed the cost of DNA sequencing (4). Although such ‘next -generation’ sequencing  \\ntechnologies have matured considerably since  early proof -of-concept s (5-7), nearly all remain  \\nlimited to short sequence reads (with the exception of real -time sequencing from elongating \\npolymerases (8)) and rely on  complex, expensive instrumentation. Most platforms are also \\nlimited with respect to speed and require extensive sample preparation steps prior to sequencing .  \\n \\nNanopore sequencing, independently proposed by Church and Deamer in the mid -1990s, has \\ntremendous potential to overcome these limitations and  achieve long reads, low cost, and high \\nspeed while requiring minimal sample preparation and instrumentation  (i.e. ‘tricorder’ -like DNA \\nsequencing devices) (9-12). However, this promise has faced substantial technical challenges, \\nsuch that despite nearl y 20 years of effort, nanopore -derived sequence reads that align to \\ncomplex, natural DNA sequences have yet to be demonstrated.  \\n \\nIn nanopore devices directed at DNA sequencing, a salt solution is divided into cis and trans  \\nwells  by a thin membrane. A singl e nanometer -scale pore in the membrane connects the cis and \\ntrans  wells electrically. When a voltage is applied across this membrane, ion current flows \\nthrough the pore; this current provides the primary signal. DNA is negatively charged and is \\nelectrophoretically attracted into the pore. When single -stranded (ss) DNA e nters the pore, it \\nblocks some fraction of the ion current. The fraction of the ion current that is blocked depends on \\nthe identity of nucleotides within the pore (13-15). Key challenges of this technique are single - 2 nucleotide resolution and control of the DNA translocation. Single -nucleotide resolution was \\nrecently enabled through the development of MspA , a protein pore with a short and narrow \\nconstriction (11, 13, 15, 16 ). DNA translocation control was also recently enabled through the \\nuse of molecular motors such as phi29 DNA Polymerase (DNAP) (11, 17 ) (Fig. 1 ). \\n \\nWe have found t hat the currents in the MspA pore are determined by about four nucleotides at \\nany given time (11, 15 ).', start_char_idx=1, end_char_idx=4095, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), score=0.86771965)]}\n"
     ]
    }
   ],
   "source": [
    "import pprint\n",
    "\n",
    "pp = pprint.PrettyPrinter(indent=4)\n",
    "pp.pprint(vars(response))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "fmicb-13-938616 August 10, 2022 Time: 14:20 # 1\n",
       "TYPE Original Research\n",
       "PUBLISHED 16 August 2022\n",
       "DOI10.3389/fmicb.2022.938616\n",
       "OPEN ACCESS\n",
       "EDITED BY\n",
       "Hany Anany,\n",
       "Agriculture and Agri-Food Canada\n",
       "(AAFC), Canada\n",
       "REVIEWED BY\n",
       "Andrea Isabel Moreno Switt,\n",
       "Pontiﬁcia Universidad Católica\n",
       "de Chile, Chile\n",
       "Carlos E. Martinez-Soto,\n",
       "University of Guelph, Canada\n",
       "*CORRESPONDENCE\n",
       "Qiang Pan\n",
       "panqiang@phagepharm.com\n",
       "Huiying Ren\n",
       "renren0228@sina.com\n",
       "†These authors have contributed\n",
       "equally to this work\n",
       "SPECIALTY SECTION\n",
       "This article was submitted to\n",
       "Phage Biology,\n",
       "a section of the journal\n",
       "Frontiers in Microbiology\n",
       "RECEIVED 07 May 2022\n",
       "ACCEPTED 18 July 2022\n",
       "PUBLISHED 16 August 2022\n",
       "CITATION\n",
       "Liu W, Han L, Song P, Sun H, Zhang C,\n",
       "Zou L, Cui J, Pan Q and Ren H (2022)\n",
       "Complete genome sequencing of a\n",
       "Tequintavirus bacteriophage with\n",
       "a broad host range against Salmonella\n",
       "Abortus equi isolates from donkeys.\n",
       "Front. Microbiol. 13:938616.\n",
       "doi: 10.3389/fmicb.2022.938616\n",
       "COPYRIGHT\n",
       "© 2022 Liu, Han, Song, Sun, Zhang,\n",
       "Zou, Cui, Pan and Ren. This is an\n",
       "open-access article distributed under\n",
       "the terms of the Creative Commons\n",
       "Attribution License (CC BY). The use,\n",
       "distribution or reproduction in other\n",
       "forums is permitted, provided the\n",
       "original author(s) and the copyright\n",
       "owner(s) are credited and that the\n",
       "original publication in this journal is\n",
       "cited, in accordance with accepted\n",
       "academic practice. No use, distribution\n",
       "or reproduction is permitted which\n",
       "does not comply with these terms.Complete genome sequencing\n",
       "of aTequintavirus bacteriophage\n",
       "with a broad host range against\n",
       "Salmonella Abortus equi isolates\n",
       "from donkeys\n",
       "Wenhua Liu1†, Letian Han1†, Peng Song1†, Huzhi Sun2,\n",
       "Can Zhang1, Ling Zou1, Jiaqi Cui1, Qiang Pan2*and\n",
       "Huiying Ren1*\n",
       "1College of Veterinary Medicine, Qingdao Agricultural University, Qingdao, Shandong, China,\n",
       "2Qingdao Phagepharm Bio-tech Co., Ltd., Qingdao, Shandong, China\n",
       "Salmonella enterica subspecies enterica serovar abortus equi (S. Abortus equi)\n",
       "is the most common cause of abortion in mares. It has recently been found\n",
       "to cause abortion in donkeys more frequently in China. A novel virulent\n",
       "bacteriophage vB_SabS_Sds2 (hereafter designated as Sds2) was isolated from\n",
       "the feces of donkeys using a S. Abortus equi strain as a host. Phage Sds2\n",
       "had an isometric polyhedral head and an uncontracted long tail, belonging\n",
       "to the Tequintavirus, Markadamsvirinae, Demerecviridae, Caudovirales. The\n",
       "genome of phage Sds2 was 114,770 bp, with a GC content of 40.26%. The\n",
       "genome contained 160 open reading frames (ORFs), and no ORFs were\n",
       "associated with pathogenicity, drug resistance, or lysogenization by sequence\n",
       "analysis. Both genome annotation and phylogenetic analysis indicated that\n",
       "phage Sds2 was highly similar to T5-like bacteriophages. Phage Sds2 could\n",
       "lyse 100% (30/30) of S. Abortus equi strains, 25.3% (24/95) of other serotypes\n",
       "ofSalmonella strains, and 27.6% (8/29) of Escherichia coli strains using the\n",
       "double-layer agar plate method. The in vitro test showed that phage Sds2 had\n",
       "high bactericidal activity against S. Abortus equi at a wide range of MOIs. The\n",
       "in vivo test indicated that phage Sds2 had an inhibitory effect on abortion in\n",
       "mice challenged with S. Abortus equi. In general, phage Sds2 is a novel lytic\n",
       "phage with a wide host range and has the potential to prevent abortion caused\n",
       "byS. Abortus equi.\n",
       "KEYWORDS\n",
       "S. Abortus equi, E. coli, bactericidal activity, complete genome, function of ORFs,\n",
       "abortive rate, phage vB_SabS_Sds2 (Sds2)\n",
       "Frontiers in Microbiology 01 frontiersin.org\n",
       "fmicb-13-938616 August 10, 2022 Time: 14:20 # 2\n",
       "Liu et al. 10.3389/fmicb.2022.938616\n",
       "Introduction\n",
       "Salmonella is a genus of the family Enterobacteriaceae that\n",
       "is important for public health and economic development\n",
       "worldwide. Diﬀerent serotypes of Salmonella often have\n",
       "diﬀerent host speciﬁcity (Uzzau et al., 2000). Salmonella enterica\n",
       "subspecies enterica serovar abortus equi (S. Abortus equi) is a\n",
       "serotype restricted to equines, which is regarded as the most\n",
       "common cause of abortion in mare"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "display(Markdown(library.documents[2].text[:4000]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## test nodes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "32"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(library.nodes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'citation': 'Laszlo, Andrew H., et al. \"Nanopore sequencing of the phi X 174 '\n",
      "             'genome.\" Nanopore sequencing of the phi X 174 genome, 2024, pp. '\n",
      "             '1-4.',\n",
      " 'excerpt_keywords': 'Alignment, scaffolded assembly, short DNA sequences, '\n",
      "                     'Illumina read, nanopore read, alignment quality, cutoff '\n",
      "                     'filter, viral alignment, viral identification, viral '\n",
      "                     'database, alignment speed, alignment confidence, DNA '\n",
      "                     'identification, nanopore reads, alignment scores, SNP '\n",
      "                     'calling, SNP detection, reference consensus, '\n",
      "                     'transversions, transitions, confusion matrix, detection '\n",
      "                     'efficiencies, resequencing.',\n",
      " 'file_name': 'data/1406.4214.pdf',\n",
      " 'section_summary': 'The key topics of this section are nanopore sequencing of '\n",
      "                    'the phi X 174 genome, alignment of short DNA sequences, '\n",
      "                    'viral alignment and identification, SNP calling workflow, '\n",
      "                    'and SNP detection efficiencies. The entities mentioned '\n",
      "                    'include the phi X 174 genome, Illumina reads, nanopore '\n",
      "                    'reads, viral database, alignment quality, alignment '\n",
      "                    'confidence, alignment scores, SNP detection methods, and '\n",
      "                    'confusion matrices.'}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(library.nodes[30].metadata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Document(id_='c2089090-ef71-4ec9-9a22-e55fbfc32a6d', embedding=None, metadata={'page_label': '2', 'file_name': '1406.4214.pdf', 'file_path': 'data/1406.4214.pdf', 'file_type': 'application/pdf', 'file_size': 2841196, 'creation_date': '2024-01-17', 'last_modified_date': '2024-01-17', 'last_accessed_date': '2024-01-17'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, hash='60fc5c86f922437eef7da25d912c7c9bcb5b797fb445ee542749f13580a17811', text=' 2 nucleotide resolution and control of the DNA translocation. Single -nucleotide resolution was \\nrecently enabled through the development of MspA , a protein pore with a short and narrow \\nconstriction (11, 13, 15, 16 ). DNA translocation control was also recently enabled through the \\nuse of molecular motors such as phi29 DNA Polymerase (DNAP) (11, 17 ) (Fig. 1 ). \\n \\nWe have found t hat the currents in the MspA pore are determined by about four nucleotides at \\nany given time (11, 15 ). Each four -nucleotide combination ( i.e. quadromer) has its own unique \\ncurrent value and in a few cases, nucleotides outside of a quadromer can have a small additional \\ninfluence on the current.  This prompted us to measure the ion current associated with each of the \\n256 possible quadromers.  \\n \\nWe constructed a 256 nucleotide -long cyclical de Bruijn sequence (18) containing all possible \\ncombinations of four nucleotides ( Table S1 ). We divided the de Bruijn sequence into eight \\nseparate strands (19) and synthesized these with appropriat e modifications to facilitate insertion \\ninto the pore, to initiate proper polymerase function, and to allow for ion current calibration (11, \\n17, 20 ) (Fig. 1a). Phi29 DNAP -based control of translocation results in two reads from each \\nDNA molecule: 1) ‘unzipping ‘ wherein one strand of the DNA moves 5’ to 3’ through the pore \\nas the polymera se is forced to unzip the complimentary strand, and 2) ’synthesis ’ wherein the \\nDNA moves 3’ to 5’ after the primer enters the DNAP’s active site and the DNAP begins \\nsynthesizing a second complimentary strand (11). As the polymerase moves along the strand, \\none nucleotide at a time, the identity of the quadromer within the MspA pore shifts in lock -step \\n(Fig 1b), resulting in discrete changes in the measured ion current ( Fig. 1 c). We performed \\nnanopore sequencing of all eight strands, averaging signals observed across multiple molecules \\nof each strand to estimate the current level of the 256 quadromers , i.e. a ‘quadromer map’  (Fig. \\n2a, Supplemental Fig. 2-4, Supplemental Table 2). \\n \\nWe next sought to evaluate whether the quadromer map constructed by nanopore sequencing of \\nthe de Bruijn sequence was predictive of current levels for previously unmeasured, natural DNA \\nsequences. To assess this, we constructed and nanopore sequenced a gen omic DNA sequencing \\nlibrary from the bacteriophage phi X 174. Specifically, we attached asymmetric adaptors (a \\nnicked hairpin adaptor and a cholesterol tailed adaptor; green and orange respectively in Fig. 1a) \\nto the ends of linearized phi X 174 dsDNA, a nd used phi29 DNAP to draw ssDNA through a \\nmutant MspA pore in single -nucleotide steps (11) (Fig. 1b,c). The measured current level \\nsequences were then compared to predicted current levels based on the quadromer map. Fig. 2b \\nshows quadromer map -based predicted current levels versus a consensus of 22 nanopore reads \\nfor a representative ~100 nucleotide (n t) region of the phi X 174 genome.    \\n \\nOverall, the predicted current levels from the de Bruijn sequence -based quadromer map strongly \\nmatch the observed current levels from nanopore sequencing of the phi X 174 DNA ( r = 0. 9905, \\n95% confidence bounds [0.9859 -0.9936] ). However, differences between prediction and \\nmeasurement are statistically significant. Close analysis suggests that this error is dominated by \\nshifts in the positioning of the DNA within the pore’s constriction due either to DNA secondary \\nstructu re within the vestibule or DNA interactions with the pore vestibule or constriction (19). \\nHowever, as independent reads of the same sequence yield extremely reproducible current values \\n(11, 20 ), we conclude that the small differences between prediction and observation are \\nsystematic and likely due to sequence context outside of the qua dromer itself.  ', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n')"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.documents[2]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "## Extractor for abstract for each document"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "from llama_index.llms import OpenAI\n",
    "\n",
    "llm = OpenAI(model='gpt-4')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "from llama_index import PromptTemplate\n",
    "\n",
    "extract_abstract_prompt_tpl = \"\"\"\n",
    "This is the first page of a scientifc paper. \n",
    "The task is to identify the paragraph corresponding to the abstract of the document. If no abstract found, try and identify the overview of the document.\n",
    "The output required is :\n",
    "    \"abstract\" : <abstract (identical to original text) or an overview of the document>\n",
    "    \"keywords\": <up to 10 keywords identified in the abstract>\n",
    "Example:\n",
    "=== Page ===\n",
    "1 Title: Nanopore sequencing of the phi X 174 genome\n",
    "\n",
    "Authors: Andrew H. Laszlo1, Ian M. Derrington1, Brian C. Ross1, Henry Brinkerhoff1, Andrew\n",
    "Adey2, Ian C. Nova1, Jonathan M. Craig1, Kyle W. Langford1, Jenny Mae Samson1, Riza Daza2, Kenji Doering1, Jay Shendure2, Jens H. Gundlach1,†\n",
    "\n",
    "Affiliations: 1Department of Physics, University of Washington, Seattle, WA 98195, USA\n",
    "2Department of Genome Sci ences, University of Washington, Seattle, WA 98195, USA\n",
    "†Corresponding author. E -mail: gundlach@uw.edu\n",
    "\n",
    "Abstract : Nanopore sequencing of DNA is a single -molecule technique that may achieve long reads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we build on recent progress with respect to nanopore resolution and DNA control to interpret the procession of ion current levels observed during the translocation of DNA through the pore MspA. As approximately four nucleotides affect the ion current of each level, we measured the ion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer map is highly predictive of ion current levels of previously unmeasured sequences derived from the bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X 174 up to 4,500 bases in length that can be unambiguously aligned to the phi X 174 reference genome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and polymorphism detection. All methods and data are made fully available.\n",
    "\n",
    "Main Text : DNA sequencing is revolutionizing biomedical and other life sciences research through its expanding scope (1) and has a rapidly growing presence in clinical medicine (2). These developments are driven in part by the successful completion of the Human Genome Project (3) and in part by the introduction of new sequencing technologies that have dramatically reduc ed the cost of DNA sequencing (4). Although such ‘next -generation’ sequencing\n",
    "technologies have matured considerably since early proof -of-concept s (5-7), nearly all remain\n",
    "limited to short sequence reads (with the exception of real -time sequencing from elongating polymerases (8)) and rely on complex, expensive instrumentation. Most platforms are also limited with respect to speed and require extensive sample preparation steps prior to sequencing .\n",
    "\n",
    "Nanopore sequencing, independently proposed by Church and Deamer in the mid -1990s, has tremendous potential to overcome these limitations and achieve long reads, low cost, and high speed while requiring minimal sample preparation and instrumentation (i.e. ‘tricorder’ -like DNA sequencing devices) (9-12). However, this promise has faced substantial technical challenges, such that despite nearl y 20 years of effort, nanopore -derived sequence reads that align to complex, natural DNA sequences have yet to be demonstrated.\n",
    "\n",
    "In nanopore devices directed at DNA sequencing, a salt solution is divide\n",
    "=== Abstract ===\n",
    "\"abstract\" : \"Nanopore sequencing of DNA is a single -molecule technique that may achieve long reads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we build on recent progress with respect to nanopore resolution and DNA control to interpret the procession of ion current levels observed during the translocation of DNA through the pore MspA. As approximately four nucleotides affect the ion current of each level, we measured the ion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer map is highly predictive of ion current levels of previously unmeasured sequences derived from the bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X 174 up to 4,500 bases in length that can be unambiguously aligned to the phi X 174 reference genome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and polymorphism detection. All methods and data are made fully available.\",\n",
    "\"keywords\" : \"Nanopore sequencing\", \"DNA\", \"translocation\", \"low cost\", \"quadromer map\", \"genome\", \"polymorphism\"\n",
    "\n",
    "Your turn\n",
    "=== Page ===\n",
    "{page}\n",
    "\n",
    "\"\"\"\n",
    "extract_abstract_prompt = PromptTemplate(extract_abstract_prompt_tpl)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'metadata': {'prompt_type': <PromptType.CUSTOM: 'custom'>},\n",
       " 'template_vars': ['page'],\n",
       " 'kwargs': {},\n",
       " 'output_parser': None,\n",
       " 'template_var_mappings': None,\n",
       " 'function_mappings': None,\n",
       " 'template': '\\nThis is the first page of a scientifc paper. \\nThe task is to identify the paragraph corresponding to the abstract of the document. If no abstract found, try and identify the overview of the document.\\nThe output required is :\\n    \"abstract\" : <abstract (identical to original text) or an overview of the document>\\n    \"keywords\": <up to 10 keywords identified in the abstract>\\nExample:\\n=== Page ===\\n1 Title: Nanopore sequencing of the phi X 174 genome\\n\\nAuthors: Andrew H. Laszlo1, Ian M. Derrington1, Brian C. Ross1, Henry Brinkerhoff1, Andrew\\nAdey2, Ian C. Nova1, Jonathan M. Craig1, Kyle W. Langford1, Jenny Mae Samson1, Riza Daza2, Kenji Doering1, Jay Shendure2, Jens H. Gundlach1,†\\n\\nAffiliations: 1Department of Physics, University of Washington, Seattle, WA 98195, USA\\n2Department of Genome Sci ences, University of Washington, Seattle, WA 98195, USA\\n†Corresponding author. E -mail: gundlach@uw.edu\\n\\nAbstract : Nanopore sequencing of DNA is a single -molecule technique that may achieve long reads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we build on recent progress with respect to nanopore resolution and DNA control to interpret the procession of ion current levels observed during the translocation of DNA through the pore MspA. As approximately four nucleotides affect the ion current of each level, we measured the ion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer map is highly predictive of ion current levels of previously unmeasured sequences derived from the bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X 174 up to 4,500 bases in length that can be unambiguously aligned to the phi X 174 reference genome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and polymorphism detection. All methods and data are made fully available.\\n\\nMain Text : DNA sequencing is revolutionizing biomedical and other life sciences research through its expanding scope (1) and has a rapidly growing presence in clinical medicine (2). These developments are driven in part by the successful completion of the Human Genome Project (3) and in part by the introduction of new sequencing technologies that have dramatically reduc ed the cost of DNA sequencing (4). Although such ‘next -generation’ sequencing\\ntechnologies have matured considerably since early proof -of-concept s (5-7), nearly all remain\\nlimited to short sequence reads (with the exception of real -time sequencing from elongating polymerases (8)) and rely on complex, expensive instrumentation. Most platforms are also limited with respect to speed and require extensive sample preparation steps prior to sequencing .\\n\\nNanopore sequencing, independently proposed by Church and Deamer in the mid -1990s, has tremendous potential to overcome these limitations and achieve long reads, low cost, and high speed while requiring minimal sample preparation and instrumentation (i.e. ‘tricorder’ -like DNA sequencing devices) (9-12). However, this promise has faced substantial technical challenges, such that despite nearl y 20 years of effort, nanopore -derived sequence reads that align to complex, natural DNA sequences have yet to be demonstrated.\\n\\nIn nanopore devices directed at DNA sequencing, a salt solution is divide\\n=== Abstract ===\\n\"abstract\" : \"Nanopore sequencing of DNA is a single -molecule technique that may achieve long reads, low cost, and high speed with minimal sample preparation and instrumentation. Here, we build on recent progress with respect to nanopore resolution and DNA control to interpret the procession of ion current levels observed during the translocation of DNA through the pore MspA. As approximately four nucleotides affect the ion current of each level, we measured the ion current corresponding to all 256 four -nucleotide comb inations (quadromers). This quadromer map is highly predictive of ion current levels of previously unmeasured sequences derived from the bacteriophage phi X 174 genome. Furthermore, we show nanopore sequencing reads of phi X 174 up to 4,500 bases in length that can be unambiguously aligned to the phi X 174 reference genome, and demonstrate proof -of-concept utility with respect to hybrid genome assembly and polymorphism detection. All methods and data are made fully available.\",\\n\"keywords\" : \"Nanopore sequencing\", \"DNA\", \"translocation\", \"low cost\", \"quadromer map\", \"genome\", \"polymorphism\"\\n\\nYour turn\\n=== Page ===\\n{page}\\n\\n'}"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "extract_abstract_prompt.dict()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "abstract_gen = llm.predict(extract_abstract_prompt, page = library.documents[3].text[:4000])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "\"abstract\" : \"Salmonella enterica subspecies enterica serovar abortus equi (S. Abortus equi) is the most common cause of abortion in mares. It has recently been found to cause abortion in donkeys more frequently in China. A novel virulent bacteriophage vB_SabS_Sds2 (hereafter designated as Sds2) was isolated from the feces of donkeys using a S. Abortus equi strain as a host. Phage Sds2 had an isometric polyhedral head and an uncontracted long tail, belonging to the Tequintavirus, Markadamsvirinae, Demerecviridae, Caudovirales. The genome of phage Sds2 was 114,770 bp, with a GC content of 40.26%. The genome contained 160 open reading frames (ORFs), and no ORFs were associated with pathogenicity, drug resistance, or lysogenization by sequence analysis. Both genome annotation and phylogenetic analysis indicated that phage Sds2 was highly similar to T5-like bacteriophages. Phage Sds2 could lyse 100% (30/30) of S. Abortus equi strains, 25.3% (24/95) of other serotypes ofSalmonella strains, and 27.6% (8/29) of Escherichia coli strains using the double-layer agar plate method. The in vitro test showed that phage Sds2 had high bactericidal activity against S. Abortus equi at a wide range of MOIs. The in vivo test indicated that phage Sds2 had an inhibitory effect on abortion in mice challenged with S. Abortus equi. In general, phage Sds2 is a novel lytic phage with a wide host range and has the potential to prevent abortion caused byS. Abortus equi.\",\n",
       "\"keywords\" : \"Salmonella enterica\", \"abortion\", \"donkeys\", \"bacteriophage vB_SabS_Sds2\", \"Tequintavirus\", \"genome\", \"open reading frames\", \"T5-like bacteriophages\", \"Escherichia coli\", \"bactericidal activity\""
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "display(Markdown(abstract_gen))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Test citation prompt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\"metadata\": {\"prompt_type\": \"custom\"}, \"template_vars\": [\"year\", \"text\"], \"kwargs\": {}, \"output_parser\": null, \"template_var_mappings\": null, \"function_mappings\": null, \"template\": \"Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:\"}\n",
      "DEBUG:openai._base_client:Request options: {'method': 'post', 'url': '/chat/completions', 'files': None, 'json_data': {'messages': [{'role': <MessageRole.USER: 'user'>, 'content': 'Provide the citation for the following text in MLA Format. The year is 2024\\n\\n    \\nProc. Natl. Acad. Sci. USA\\nVol. 80, pp. 4195-4199, July 1983\\nBiochemistry\\nIn vitro synthesis of bacteriophage 4X174 by purified components\\n(in vitro DNA synthesis/viral component/DNA polymerase Ill holoenzyme/rep protein/deoxyuridinetriphosphatase)\\nAKIRA AOYAMAt, ROBERT K. HAMATAKEt, AND MASAKI HAYASHIt\\nDepartments of tBiology and tChemistry, University of California at San Diego, La Jolla, California 92093\\nCommunicated by E. Peter Geiduschek, January 21, 1983\\nABSTRACT An in vitro system capable of synthesizing infectious 4X174 phage particles was reconstituted from purified components. The synthesis required 4X174 supercoiled replicative\\nform DNA, OX174-encoded proteins A, C, J, and prohead, Escherichia coli DNA polymerase III holoenzyme, rep protein, and\\ndeoxyuridinetriphosphatase (dUTPase, dUTP nucleotidohydrolase, EC 3.6.1.23) as well as MgCl2, four deoxyribonucleoside triphosphates, and ATP. Phage production was coupled to the synthesis of viral single-stranded DNA. More than 70% of the\\nsynthesized particles sedimented at the position of mature phage\\nin a sucrose gradient and associated with the infectivity. The simple requirement of the host proteins suggests that the mechanism\\nof viral strand synthesis in the phage-synthesizing reaction resembles that of viral strand synthesis during the replication of\\nreplicative form DNA.\\nDuring infection of Escherichia coli by the single-stranded (ss)\\nDNA bacteriophage 4X174, the circular ss DNA is replicated\\nvia three successive stages of DNA synthesis (for review, see\\nref. 1). Stage I is the conversion of ss DNA to double-stranded\\n\\n\\n\\n    Citation:'}], 'model': 'gpt-3.5-turbo', 'max_tokens': 256, 'stream': False, 'temperature': 0.0}}\n",
      "Request options: {'method': 'post', 'url': '/chat/completions', 'files': None, 'json_data': {'messages': [{'role': <MessageRole.USER: 'user'>, 'content': 'Provide the citation for the following text in MLA Format. The year is 2024\\n\\n    \\nProc. Natl. Acad. Sci. USA\\nVol. 80, pp. 4195-4199, July 1983\\nBiochemistry\\nIn vitro synthesis of bacteriophage 4X174 by purified components\\n(in vitro DNA synthesis/viral component/DNA polymerase Ill holoenzyme/rep protein/deoxyuridinetriphosphatase)\\nAKIRA AOYAMAt, ROBERT K. HAMATAKEt, AND MASAKI HAYASHIt\\nDepartments of tBiology and tChemistry, University of California at San Diego, La Jolla, California 92093\\nCommunicated by E. Peter Geiduschek, January 21, 1983\\nABSTRACT An in vitro system capable of synthesizing infectious 4X174 phage particles was reconstituted from purified components. The synthesis required 4X174 supercoiled replicative\\nform DNA, OX174-encoded proteins A, C, J, and prohead, Escherichia coli DNA polymerase III holoenzyme, rep protein, and\\ndeoxyuridinetriphosphatase (dUTPase, dUTP nucleotidohydrolase, EC 3.6.1.23) as well as MgCl2, four deoxyribonucleoside triphosphates, and ATP. Phage production was coupled to the synthesis of viral single-stranded DNA. More than 70% of the\\nsynthesized particles sedimented at the position of mature phage\\nin a sucrose gradient and associated with the infectivity. The simple requirement of the host proteins suggests that the mechanism\\nof viral strand synthesis in the phage-synthesizing reaction resembles that of viral strand synthesis during the replication of\\nreplicative form DNA.\\nDuring infection of Escherichia coli by the single-stranded (ss)\\nDNA bacteriophage 4X174, the circular ss DNA is replicated\\nvia three successive stages of DNA synthesis (for review, see\\nref. 1). Stage I is the conversion of ss DNA to double-stranded\\n\\n\\n\\n    Citation:'}], 'model': 'gpt-3.5-turbo', 'max_tokens': 256, 'stream': False, 'temperature': 0.0}}\n",
      "DEBUG:httpcore.connection:close.started\n",
      "close.started\n",
      "DEBUG:httpcore.connection:close.complete\n",
      "close.complete\n",
      "DEBUG:httpcore.connection:connect_tcp.started host='api.openai.com' port=443 local_address=None timeout=60.0 socket_options=None\n",
      "connect_tcp.started host='api.openai.com' port=443 local_address=None timeout=60.0 socket_options=None\n",
      "DEBUG:httpcore.connection:connect_tcp.complete return_value=<httpcore._backends.sync.SyncStream object at 0x7fe75172aed0>\n",
      "connect_tcp.complete return_value=<httpcore._backends.sync.SyncStream object at 0x7fe75172aed0>\n",
      "DEBUG:httpcore.connection:start_tls.started ssl_context=<ssl.SSLContext object at 0x7fe75226f800> server_hostname='api.openai.com' timeout=60.0\n",
      "start_tls.started ssl_context=<ssl.SSLContext object at 0x7fe75226f800> server_hostname='api.openai.com' timeout=60.0\n",
      "DEBUG:httpcore.connection:start_tls.complete return_value=<httpcore._backends.sync.SyncStream object at 0x7fe750c3a950>\n",
      "start_tls.complete return_value=<httpcore._backends.sync.SyncStream object at 0x7fe750c3a950>\n",
      "DEBUG:httpcore.http11:send_request_headers.started request=<Request [b'POST']>\n",
      "send_request_headers.started request=<Request [b'POST']>\n",
      "DEBUG:httpcore.http11:send_request_headers.complete\n",
      "send_request_headers.complete\n",
      "DEBUG:httpcore.http11:send_request_body.started request=<Request [b'POST']>\n",
      "send_request_body.started request=<Request [b'POST']>\n",
      "DEBUG:httpcore.http11:send_request_body.complete\n",
      "send_request_body.complete\n",
      "DEBUG:httpcore.http11:receive_response_headers.started request=<Request [b'POST']>\n",
      "receive_response_headers.started request=<Request [b'POST']>\n",
      "DEBUG:httpcore.http11:receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Date', b'Wed, 17 Jan 2024 04:10:19 GMT'), (b'Content-Type', b'application/json'), (b'Transfer-Encoding', b'chunked'), (b'Connection', b'keep-alive'), (b'access-control-allow-origin', b'*'), (b'Cache-Control', b'no-cache, must-revalidate'), (b'openai-model', b'gpt-3.5-turbo-0613'), (b'openai-organization', b'user-vyzy3sybphrf2xalgufcdzur'), (b'openai-processing-ms', b'3026'), (b'openai-version', b'2020-10-01'), (b'strict-transport-security', b'max-age=15724800; includeSubDomains'), (b'x-ratelimit-limit-requests', b'10000'), (b'x-ratelimit-limit-tokens', b'60000'), (b'x-ratelimit-remaining-requests', b'9999'), (b'x-ratelimit-remaining-tokens', b'59317'), (b'x-ratelimit-reset-requests', b'8.64s'), (b'x-ratelimit-reset-tokens', b'683ms'), (b'x-request-id', b'394db754f58c7fca09d863c556cf312c'), (b'CF-Cache-Status', b'DYNAMIC'), (b'Server', b'cloudflare'), (b'CF-RAY', b'846bbf57fff8dfbd-SYD'), (b'Content-Encoding', b'gzip'), (b'alt-svc', b'h3=\":443\"; ma=86400')])\n",
      "receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Date', b'Wed, 17 Jan 2024 04:10:19 GMT'), (b'Content-Type', b'application/json'), (b'Transfer-Encoding', b'chunked'), (b'Connection', b'keep-alive'), (b'access-control-allow-origin', b'*'), (b'Cache-Control', b'no-cache, must-revalidate'), (b'openai-model', b'gpt-3.5-turbo-0613'), (b'openai-organization', b'user-vyzy3sybphrf2xalgufcdzur'), (b'openai-processing-ms', b'3026'), (b'openai-version', b'2020-10-01'), (b'strict-transport-security', b'max-age=15724800; includeSubDomains'), (b'x-ratelimit-limit-requests', b'10000'), (b'x-ratelimit-limit-tokens', b'60000'), (b'x-ratelimit-remaining-requests', b'9999'), (b'x-ratelimit-remaining-tokens', b'59317'), (b'x-ratelimit-reset-requests', b'8.64s'), (b'x-ratelimit-reset-tokens', b'683ms'), (b'x-request-id', b'394db754f58c7fca09d863c556cf312c'), (b'CF-Cache-Status', b'DYNAMIC'), (b'Server', b'cloudflare'), (b'CF-RAY', b'846bbf57fff8dfbd-SYD'), (b'Content-Encoding', b'gzip'), (b'alt-svc', b'h3=\":443\"; ma=86400')])\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "DEBUG:httpcore.http11:receive_response_body.started request=<Request [b'POST']>\n",
      "receive_response_body.started request=<Request [b'POST']>\n",
      "DEBUG:httpcore.http11:receive_response_body.complete\n",
      "receive_response_body.complete\n",
      "DEBUG:httpcore.http11:response_closed.started\n",
      "response_closed.started\n",
      "DEBUG:httpcore.http11:response_closed.complete\n",
      "response_closed.complete\n",
      "DEBUG:openai._base_client:HTTP Request: POST https://api.openai.com/v1/chat/completions \"200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"200 OK\"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'Aoyama, Akira, et al. \"In vitro synthesis of bacteriophage 4X174 by purified components.\" Proceedings of the National Academy of Sciences of the United States of America, vol. 80, no. 14, 1983, pp. 4195-4199.'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from phages.modules.prompts import citation_prompt\n",
    "from datetime import datetime\n",
    "\n",
    "current_year = datetime.now().year\n",
    "\n",
    "cit = citation_prompt.partial_format(year=current_year)\n",
    "\n",
    "print(citation_prompt.json())\n",
    "\n",
    "response = llm.predict(cit, text = text)\n",
    "\n",
    "response"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['year', 'text']"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "citation_prompt.template_vars"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Test FullPDFReader"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from phages.modules.library import FullPDFReader\n",
    "\n",
    "# for SimpleDirectoryReader file_extractor param\n",
    "CUSTOM_FILE_READER_CLS = {\n",
    "    \".pdf\": FullPDFReader,\n",
    "}\n",
    "\n",
    "file_suffix=\".pdf\"\n",
    "\n",
    "file_extractor=CUSTOM_FILE_READER_CLS\n",
    "\n",
    "reader = file_extractor[file_suffix]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "\n",
    "response = reader.load_data(file='data/1406.4214.pdf')\n",
    "\n",
    "len(response)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Test adoc match"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "embedded weaviate is already listening on port 8079\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "{\"action\":\"restapi_management\",\"level\":\"info\",\"msg\":\"Shutting down... \",\"time\":\"2024-01-23T09:25:38Z\"}\n",
      "{\"action\":\"restapi_management\",\"level\":\"info\",\"msg\":\"Stopped serving weaviate at http://127.0.0.1:8079\",\"time\":\"2024-01-23T09:25:38Z\"}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32487 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32487 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x327a8 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x327a8 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32a79 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32a79 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32c82 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32c82 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32e3c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32e3c for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x32fee for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x32fee for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x331d8 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x331d8 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x333ca for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x333ca for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x3365b for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x3365b for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x338ac for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x338ac for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x33a86 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33a86 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x33c88 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33c88 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x33e79 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x33e79 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x34102 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x34102 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x34393 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x34393 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x3453c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x3453c for key /MediaBox\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: ORIGINAL RESEARCH\n",
      "published: 04 January 2018\n",
      "doi: 10.3389/fmicb.2017.02548\n",
      "Frontiers in Microbiology | www.frontiersin.org 1 January 2018 | Volume 8 | Article 2548Editedby:\n",
      "MichaelThomas-Poulsen,\n",
      "UniversityofCopenhagen,Denmark\n",
      "Reviewedby:\n",
      "RichardAllenWhiteIII,\n",
      "WashingtonStateUniversity,\n",
      "UnitedStates\n",
      "KarenDawnWeynberg,\n",
      "TheUniversityofQueensland,\n",
      "Australia\n",
      "AlejandroReyes,\n",
      "UniversityofLosAndes,Colombia\n",
      "*Correspondence:\n",
      "ChinmayV.Tikhe\n",
      "ctikhe1@jhu.edu\n",
      "†PresentAddress:\n",
      "ChinmayV.Tikhe,\n",
      "W.HarryFeinstoneDepartmentof\n",
      "MolecularMicrobiologyand\n",
      "Immunology,BloombergSchoolof\n",
      "PublicHealth,JohnsHopkins\n",
      "University,Baltimore,MD,\n",
      "UnitedStates\n",
      "Specialtysection:\n",
      "Thisarticlewassubmittedto\n",
      "MicrobialSymbioses,\n",
      "asectionofthejournal\n",
      "FrontiersinMicrobiology\n",
      "Received: 21September2017\n",
      "Accepted: 08December2017\n",
      "Published: 04January2018\n",
      "Citation:\n",
      "TikheCVandHussenederC(2018)\n",
      "MetaviromeSequencingofthe\n",
      "TermiteGutRevealsthePresenceof\n",
      "anUnexploredBacteriophage\n",
      "Community.Front.Microbiol.8:2548.\n",
      "doi:10.3389/fmicb.2017.02548Metavirome Sequencing of the\n",
      "Termite Gut Reveals the Presence of\n",
      "an Unexplored Bacteriophage\n",
      "Community\n",
      "ChinmayV.Tikhe *†andClaudiaHusseneder\n",
      "DepartmentofEntomology,LouisianaStateUniversityAgri culturalCenter,BatonRouge,LA,UnitedStates\n",
      "TheFormosansubterraneantermite; Coptotermesformosanus isnutritionallydependent\n",
      "on the complex and diverse community of bacteria and protozo a in their gut. Although,\n",
      "there have been many studies to decipher the taxonomic and fu nctional diversity of\n",
      "bacterial communities in the guts of termites, their bacter iophages remain unstudied.\n",
      "WesequencedthemetaviromeofthegutsofFormosansubterra neantermiteworkersto\n",
      "study thediversityof bacteriophagesandotherassociated viruses. Resultsshowed that\n",
      "the termites harbor a virome in their gut comprised of varied and previously unknown\n",
      "bacteriophages. Between 87–90% of the predicted dsDNA viru s genes by Metavir\n",
      "showed similarity to the tailed bacteriophages (Caudovirales) . Many predicted genes\n",
      "from the virome matched to bacterial prophage regions. Thes e data are suggestive\n",
      "of a virome dominated by temperate bacteriophages. We predi cted the genomes of\n",
      "seven novel Caudovirales bacteriophages from the termite g ut. Three of these predicted\n",
      "bacteriophage genomes were found in high proportions in all the three termite colonies\n",
      "tested. Two bacteriophages are predicted to infect endosym biotic bacteria of the gut\n",
      "protozoa.Thepresenceoftheseputativebacteriophagesin fectingendosymbiontsofthe\n",
      "gut protozoa, suggests a quadripartite relationship betwe en the termites their symbiotic\n",
      "protozoa, endosymbiotic bacteria of the protozoa and their bacteriophages. Other than\n",
      "Caudovirales, ss-DNA virus related genes were also present in the termite gut. We\n",
      "predicted the genomes of 12 novel Microviridae phages from the termite gut and seven\n",
      "of those possibly represent a new proposed subfamily. Circo virus like genomes were\n",
      "also assembled from the termite gut at lower relative abunda nce. We predicted 10\n",
      "novel circovirus\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Tikhe, Chinmay V., and Claudia Husseneder. \"Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community.\" Frontiers in Microbiology, vol. 8, 2018, article 2548. doi:10.3389/fmicb.2017.02548\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "Embedded weaviate wasn't listening on port 8079, so starting embedded weaviate again\n",
      "Embedded weaviate wasn't listening on port 8079, so starting embedded weaviate again\n",
      "Started ./weaviate/.cache/weaviate-embedded/: process ID 17092\n",
      "Started ./weaviate/.cache/weaviate-embedded/: process ID 17093\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "{\"action\":\"startup\",\"default_vectorizer_module\":\"none\",\"level\":\"info\",\"msg\":\"the default vectorizer modules is set to \\\"none\\\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer\",\"time\":\"2024-01-23T09:25:47Z\"}\n",
      "{\"action\":\"startup\",\"auto_schema_enabled\":true,\"level\":\"info\",\"msg\":\"auto schema enabled setting is set to \\\"true\\\"\",\"time\":\"2024-01-23T09:25:47Z\"}\n",
      "{\"action\":\"startup\",\"default_vectorizer_module\":\"none\",\"level\":\"info\",\"msg\":\"the default vectorizer modules is set to \\\"none\\\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer\",\"time\":\"2024-01-23T09:25:47Z\"}\n",
      "{\"action\":\"startup\",\"auto_schema_enabled\":true,\"level\":\"info\",\"msg\":\"auto schema enabled setting is set to \\\"true\\\"\",\"time\":\"2024-01-23T09:25:47Z\"}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "listen tcp :6060: bind: address already in use\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "{\"action\":\"hnsw_vector_cache_prefill\",\"count\":3000,\"index_id\":\"docs_SEbBamV3puOr\",\"level\":\"info\",\"limit\":1000000000000,\"msg\":\"prefilled vector cache\",\"time\":\"2024-01-23T09:25:47Z\",\"took\":174300}\n",
      "{\"level\":\"warning\",\"msg\":\"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.\",\"time\":\"2024-01-23T09:25:48Z\"}\n",
      "{\"action\":\"grpc_startup\",\"level\":\"info\",\"msg\":\"grpc server listening at [::]:50060\",\"time\":\"2024-01-23T09:25:48Z\"}\n",
      "{\"action\":\"restapi_management\",\"level\":\"info\",\"msg\":\"Serving weaviate at http://127.0.0.1:8079\",\"time\":\"2024-01-23T09:25:48Z\"}\n",
      "/usr/local/lib/python3.11/subprocess.py:1127: ResourceWarning: subprocess 17092 is still running\n",
      "  _warn(\"subprocess %s is still running\" % self.pid,\n",
      "ResourceWarning: Enable tracemalloc to get the object allocation traceback\n",
      "{\"action\":\"hnsw_vector_cache_prefill\",\"count\":3000,\"index_id\":\"nodes_yshPFQSIWFOC\",\"level\":\"info\",\"limit\":1000000000000,\"msg\":\"prefilled vector cache\",\"time\":\"2024-01-23T09:25:48Z\",\"took\":154211400}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x21878 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21878 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x21b0c for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21b0c for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x21e55 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x21e55 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22023 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22023 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x221d9 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x221d9 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x223aa for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x223aa for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22538 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22538 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x226f1 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x226f1 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22957 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22957 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22b98 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22b98 for key /MediaBox\n",
      "WARNING:pypdf.generic._data_structures:Multiple definitions in dictionary at byte 0x22e09 for key /MediaBox\n",
      "Multiple definitions in dictionary at byte 0x22e09 for key /MediaBox\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: ORIGINAL RESEARCH\n",
      "published: 30 April 2018\n",
      "doi: 10.3389/fmicb.2018.00850\n",
      "Frontiers in Microbiology | www.frontiersin.org 1 April 2018 | Volume 9 | Article 850Editedby:\n",
      "Xian-ZhiLi,\n",
      "HealthCanada,Canada\n",
      "Reviewedby:\n",
      "NinaChanishvili,\n",
      "GeorgeEliavaInstituteof\n",
      "Bacteriophage,Microbiologyand\n",
      "Virology,Georgia\n",
      "PaulHyman,\n",
      "AshlandUniversity,UnitedStates\n",
      "RemyA.Bonnin,\n",
      "UniversitéParis-Saclay,France\n",
      "*Correspondence:\n",
      "ZhiyongZong\n",
      "zongzhiy@scu.edu.cn\n",
      "Specialtysection:\n",
      "Thisarticlewassubmittedto\n",
      "Antimicrobials,Resistanceand\n",
      "Chemotherapy,\n",
      "asectionofthejournal\n",
      "FrontiersinMicrobiology\n",
      "Received: 06January2018\n",
      "Accepted: 13April2018\n",
      "Published: 30April2018\n",
      "Citation:\n",
      "ZhouW,FengYandZongZ(2018)\n",
      "TwoNewLyticBacteriophagesofthe\n",
      "MyoviridaeFamilyAgainst\n",
      "Carbapenem-ResistantAcinetobacter\n",
      "baumannii.Front.Microbiol.9:850.\n",
      "doi:10.3389/fmicb.2018.00850Two New Lytic Bacteriophages of the\n",
      "Myoviridae Family Against\n",
      "Carbapenem-Resistant\n",
      "Acinetobacterbaumannii\n",
      "WeilongZhou1,2,YuFeng1,2andZhiyongZong1,2,3,4*\n",
      "1CenterofInfectiousDiseases,WestChinaHospital,Sichuan University,Chengdu,China,2DivisionofInfectiousDiseases,\n",
      "StateKeyLaboratoryofBiotherapy,Chengdu,China,3DepartmentofInfectionControl,WestChinaHospital,Sichu an\n",
      "University,Chengdu,China,4CenterforPathogenResearch,WestChinaHospital,Sichuan University,Chengdu,China\n",
      "Two lytic bacteriophages, WCHABP1 and WCHABP12, were recover ed from hospital\n",
      "sewage and were able to infect 9 and 12 out of 18 carbapenem-re sistantAcinetobacter\n",
      "baumannii clinical strains, which belonged to different clones. Elec tron microscopy\n",
      "scan showed that both bacteriophages had the similar morpho logy as those of the\n",
      "Myoviridae family. Whole genomic sequencing revealed 45.4- or 45.8-kb g enome\n",
      "with a 37.6% GC content for WCHABP1 and WCHABP12, both of which s howed\n",
      "signiﬁcant DNA sequence similarity with bacteriophages of theAp22virus genus within\n",
      "theMyoviridae family.Taxonomicanalysiswasthereforeperformedfollow ingtheproposal\n",
      "approved by the International Committee on Taxonomy of Viru ses, which conﬁrmed that\n",
      "WCHABP1 and WCHABP12 represented two new species of the Ap22virus genus.\n",
      "No tRNAs but 88 and 89 open reading frames (ORFs) were predict ed for the two\n",
      "bacteriophages, among which 22 and 21 had known function and encoded proteins\n",
      "for morphogenesis, packaging, lysis, and nucleiotide meta bolism. The C-terminal amino\n",
      "acids of the large unit of ﬁber tail proteins varied between t he bacteriophages, which\n",
      "may explain their different host ranges. For most lytic bact eriophages, a set of holin\n",
      "and endolysin are required for lysis. However, no known holi n-encoding genes were\n",
      "identiﬁed in WCHABP1 and WCHABP12, suggesting that they may us e alternative,\n",
      "yet-to-be-identiﬁed, novel holins for host cell membrane l ysis. To test the efﬁcacy of the\n",
      "bacteriophages in protecting against A. baumannii infection, a Galleria mellonella larva\n",
      "model was used. Only <20%G. mellonella larvae survived at 96h after being infected\n",
      "bycarbapenem-resistant A.baumannii strains,fromwhichthetwobacter\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Zhou, W., Feng, Y., & Zong, Z. (2018). Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii. Frontiers in Microbiology, 9, 850. doi:10.3389/fmicb.2018.00850\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: fmicb-10-00627 March 30, 2019 Time: 16:26 # 1\n",
      "ORIGINAL RESEARCH\n",
      "published: 02 April 2019\n",
      "doi: 10.3389/fmicb.2019.00627\n",
      "Edited by:\n",
      "William Michael McShan,\n",
      "The University of Oklahoma Health\n",
      "Sciences Center, United States\n",
      "Reviewed by:\n",
      "Grzegorz Wegrzyn,\n",
      "University of Gda ´nsk, Poland\n",
      "Scott Van Nguyen,\n",
      "University College Dublin, Ireland\n",
      "*Correspondence:\n",
      "Vivian C. H. Wu\n",
      "vivian.wu@ars.usda.gov\n",
      "Specialty section:\n",
      "This article was submitted to\n",
      "Virology,\n",
      "a section of the journal\n",
      "Frontiers in Microbiology\n",
      "Received: 10 October 2018\n",
      "Accepted: 12 March 2019\n",
      "Published: 02 April 2019\n",
      "Citation:\n",
      "Liao Y-T, Sun X, Quintela IA,\n",
      "Bridges DF, Liu F, Zhang Y, Salvador A\n",
      "and Wu VCH (2019) Discovery\n",
      "of Shiga Toxin-Producing Escherichia\n",
      "coli (STEC)-Speciﬁc Bacteriophages\n",
      "From Non-fecal Composts Using\n",
      "Genomic Characterization.\n",
      "Front. Microbiol. 10:627.\n",
      "doi: 10.3389/fmicb.2019.00627\n",
      "Discovery of Shiga Toxin-Producing\n",
      "Escherichia coli (STEC)-Speciﬁc\n",
      "Bacteriophages From Non-fecal\n",
      "Composts Using Genomic\n",
      "Characterization\n",
      "Yen-Te Liao1, Xincheng Sun1,2,3, Irwin A. Quintela1, David F. Bridges1, Fang Liu1,4,\n",
      "Yujie Zhang1,5, Alexandra Salvador1and Vivian C. H. Wu1*\n",
      "1Produce Safety and Microbiology Research Unit, U.S. Department of Agriculture, Agricultural Research Service, Western\n",
      "Regional Research Center, Albany, CA, United States,2Henan Key Laboratory of Cold Chain Food Quality and Safety\n",
      "Control, Zhengzhou University of Light Industry, Zhengzhou, China,3Collaborative Innovation Center of Food Production\n",
      "and Safety, Zhengzhou, China,4College of Food Science and Engineering, Ocean University of China, Qingdao, China,\n",
      "5College of Food Science and Technology, Shanghai Ocean University, Shanghai, China\n",
      "Composting is a complex biodegradable process that converts organic materials\n",
      "into nutrients to facilitate crop yields, and, if well managed, can render bactericidal\n",
      "effects. Majority of research focused on detection of enteric pathogens, such as\n",
      "Shiga toxin-producing Escherichia coli (STEC) in fecal composts. Recently, attention\n",
      "has been emphasized on bacteriophages, such as STEC-speciﬁc bacteriophages,\n",
      "associated with STEC from the fecal-contaminated environment because they are able\n",
      "to sustain adverse environmental condition during composting process. However, little is\n",
      "known regarding the isolation of STEC-speciﬁc bacteriophages in non-fecal composts.\n",
      "Thus, the objectives were to isolate and genomically characterize STEC-speciﬁc\n",
      "bacteriophages, and to evaluate its association with STEC in non-fecal composts.\n",
      "For bacteriophage isolation, the samples were enriched with non-pathogenic E. coli\n",
      "(3 strains) and STEC (14 strains), respectively. After puriﬁcation, host range, plaque\n",
      "size, and phage morphology were examined. Furthermore, bacteriophage genomes\n",
      "were subjected to whole-genome sequencing using Illumina MiSeq and genomic\n",
      "analyses. Isolation of top six non-O157 and O157 STEC utilizing culture methods\n",
      "combined with PCR-based conﬁrmation was also conducted. The results showed\n",
      "that va\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: fmicb-10-01533 July 22, 2019 Time: 16:10 # 1\n",
      "ORIGINAL RESEARCH\n",
      "published: 23 July 2019\n",
      "doi: 10.3389/fmicb.2019.01533\n",
      "Edited by:\n",
      "Manuel Martinez Garcia,\n",
      "University of Alicante, Spain\n",
      "Reviewed by:\n",
      "Jeroen Wagemans,\n",
      "KU Leuven, Belgium\n",
      "Ana-Belen Martin-Cuadrado,\n",
      "University of Alicante, Spain\n",
      "Lars Fieseler,\n",
      "Zurich University of Applied Sciences,\n",
      "Switzerland\n",
      "*Correspondence:\n",
      "Julianne H. Grose\n",
      "grosejulianne@gmail.com\n",
      "Specialty section:\n",
      "This article was submitted to\n",
      "Virology,\n",
      "a section of the journal\n",
      "Frontiers in Microbiology\n",
      "Received: 08 March 2019\n",
      "Accepted: 19 June 2019\n",
      "Published: 23 July 2019\n",
      "Citation:\n",
      "Sharma R, Pielstick BA, Bell KA,\n",
      "Nieman TB, Stubbs OA, Yeates EL,\n",
      "Baltrus DA and Grose JH (2019) A\n",
      "Novel, Highly Related Jumbo Family\n",
      "of Bacteriophages That Were Isolated\n",
      "Against Erwinia.\n",
      "Front. Microbiol. 10:1533.\n",
      "doi: 10.3389/fmicb.2019.01533\n",
      "A Novel, Highly Related Jumbo\n",
      "Family of Bacteriophages That Were\n",
      "Isolated Against Erwinia\n",
      "Ruchira Sharma1, Brittany A. Pielstick1, Kimberly A. Bell1, Tanner B. Nieman1,\n",
      "Olivia A. Stubbs1, Edward L. Yeates1, David A. Baltrus2and Julianne H. Grose1*\n",
      "1Department of Microbiology and Molecular Biology, Brigham Young University, Provo, UT, United States,2School of Plant\n",
      "Sciences, The University of Arizona, Tucson, AZ, United States\n",
      "Erwinia amylovora is a plant pathogen from the Erwiniaceae family and a causative\n",
      "agent of the devastating agricultural disease ﬁre blight. Here we characterize eight\n",
      "lytic bacteriophages of E. amylovora that we isolated from the Wasatch front (Utah,\n",
      "United States) that are highly similar to vB_EamM_Ea35-70 which was isolated in\n",
      "Ontario, Canada. With the genome size ranging from 271 to 275 kb, this is a novel jumbo\n",
      "family of bacteriophages. These jumbo bacteriophages were further characterized\n",
      "through genomic and proteomic comparison, mass spectrometry, host range and burst\n",
      "size. Their proteomes are highly unstudied, with over 200 putative proteins with no\n",
      "known homologs. The production of 27 of these putative proteins was conﬁrmed\n",
      "by mass spectrometry analysis. These bacteriophages appear to be most similar to\n",
      "bacteriophages that infect Pseudomonas andRalstonia rather than Enterobacteriales\n",
      "bacteria by protein similarity, however, we were only able to detect infection of Erwinia\n",
      "and the closely related strains of Pantoea .\n",
      "Keywords: jumbo bacteriophage, Agrican357virus , burst size, novel, Erwinia ,Pantoea , genome, proteome\n",
      "INTRODUCTION\n",
      "Whitman et al. (1998) estimated that there are approximately 5 ×1030bacteria on earth, which is\n",
      "more than the number of plants and animals combined. Most, or likely all, bacteria are subject to\n",
      "infection by one or more viruses or “bacteriophages, ” making bacteriophages the most common\n",
      "and diverse biological entity at an estimated 1032(Bergh et al., 1989; Wommack and Colwell,\n",
      "2000; Hambly and Suttle, 2005). Bacteriophages were likely ﬁrst reported in 1896, when Ernest\n",
      "Hanbury Hankin discovered antibacterial activity against cholera in the waters of two large\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Sharma, Ruchira, et al. \"A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia.\" Frontiers in Microbiology, vol. 10, 2019, p. 1533. doi: 10.3389/fmicb.2019.01533.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: fmicb-12-616548 March 4, 2021 Time: 17:51 # 1\n",
      "ORIGINAL RESEARCH\n",
      "published: 10 March 2021\n",
      "doi: 10.3389/fmicb.2021.616548\n",
      "Edited by:\n",
      "Indrani Karunasagar,\n",
      "Nitte University, India\n",
      "Reviewed by:\n",
      "Ahmed Askora,\n",
      "Zagazig University, Egypt\n",
      "D. Ipek Kurtböke,\n",
      "University of the Sunshine Coast,\n",
      "Australia\n",
      "*Correspondence:\n",
      "Chia Wanq Tan\n",
      "chiawanq@gmail.com\n",
      "Yaya Rukayadi\n",
      "yaya_rukayadi@upm.edu.my\n",
      "Specialty section:\n",
      "This article was submitted to\n",
      "Virology,\n",
      "a section of the journal\n",
      "Frontiers in Microbiology\n",
      "Received: 12 October 2020\n",
      "Accepted: 19 February 2021\n",
      "Published: 10 March 2021\n",
      "Citation:\n",
      "Tan CW, Rukayadi Y, Hasan H,\n",
      "Abdul-Mutalib N-A, Jambari NN,\n",
      "Hara H, Thung TY, Lee E and Radu S\n",
      "(2021) Isolation and Characterization\n",
      "of Six Vibrio parahaemolyticus Lytic\n",
      "Bacteriophages From\n",
      "Seafood Samples.\n",
      "Front. Microbiol. 12:616548.\n",
      "doi: 10.3389/fmicb.2021.616548\n",
      "Isolation and Characterization of Six\n",
      "Vibrio parahaemolyticus Lytic\n",
      "Bacteriophages From\n",
      "Seafood Samples\n",
      "Chia Wanq Tan1*, Yaya Rukayadi1*, Hanan Hasan1, Noor-Azira Abdul-Mutalib1,2,\n",
      "Nuzul Noorahya Jambari1,2, Hirofumi Hara3, Tze Young Thung4, Epeng Lee2and\n",
      "Son Radu1,2\n",
      "1Department of Food Science, Faculty of Food Science and Technology, Universiti Putra Malaysia, UPM Serdang, Selangor,\n",
      "Malaysia,2Food Safety and Food Integrity, Institute of Tropical Agriculture and Food Security (ITAFoS), Universiti Putra\n",
      "Malaysia, UPM Serdang, Selangor, Malaysia,3Department of Environmental Engineering and Green Technology,\n",
      "Malaysia-Japan International Institute of Technology, Universiti Teknologi Malaysia, Kuala Lumpur, Malaysia,4Infection\n",
      "and Immunity Program, Department of Microbiology, Biomedicine Discovery Institute, Monash University, Clayton, VIC,\n",
      "Australia\n",
      "Vibrio parahaemolyticus is a foodborne pathogen that is frequently isolated from a\n",
      "variety of seafood. To control this pathogenic Vibrio spp., the implementation of\n",
      "bacteriophages in aquaculture and food industries have shown a promising alternative\n",
      "to antibiotics. In this study, six bacteriophages isolated from the seafood samples\n",
      "demonstrated a narrow host range speciﬁcity that infecting only the V. parahaemolyticus\n",
      "strains. Morphological analysis revealed that bacteriophages Vp33, Vp22, Vp21, and\n",
      "Vp02 belong to the Podoviridae family, while bacteriophages Vp08 and Vp11 were\n",
      "categorized into the Siphoviridae family. All bacteriophages were composed of DNA\n",
      "genome and showed distinctive restriction fragment length polymorphism. The optimal\n",
      "MOI for bacteriophage propagation was determined to be 0.001 to 1. One-step growth\n",
      "curve revealed that the latent period ranged from 10 to 20 min, and the burst size\n",
      "of bacteriophage was approximately 17 to 51 PFU/cell. The inﬂuence of temperature\n",
      "and pH levels on the stability of bacteriophages showed that all bacteriophages\n",
      "were optimally stable over a wide range of temperatures and pH levels. In vitro\n",
      "lytic activity of all bacteriophages demonstrated to have a signiﬁcant effect against\n",
      "V. parahaemolyticus . Besides, the application of a b\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    }
   ],
   "source": [
    "from phages.modules.library import Library\n",
    "from pathlib import Path\n",
    "\n",
    "library = Library()\n",
    "\n",
    "# Define the directory path\n",
    "directory_path = Path('/workspaces/ml-learning/src/phages/data/test3')\n",
    "\n",
    "# Loop over each file in the directory\n",
    "for file_path in directory_path.iterdir():\n",
    "    # Check if it's a file and not a directory\n",
    "    if file_path.is_file():\n",
    "        # Call the library.add() function with the file path as a string\n",
    "        library.add(str(file_path))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving object library_with_original_phages to pickle file ./library_with_original_phages.pkl\n"
     ]
    }
   ],
   "source": [
    "from phages.modules.helpers import pickleSave\n",
    "\n",
    "pickleSave(library, 'library_with_original_phages', '.')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "6"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(library.documents)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'file_path': '/workspaces/ml-learning/src/phages/notebooks/data/dataset_phages-microbiology-frontiers_2024-01-23_08-51-12-772.json',\n",
       " 'file_name': 'dataset_phages-microbiology-frontiers_2024-01-23_08-51-12-772.json',\n",
       " 'file_type': 'application/json',\n",
       " 'file_size': 993570,\n",
       " 'creation_date': '2024-01-23',\n",
       " 'last_modified_date': '2024-01-23',\n",
       " 'last_accessed_date': '2024-01-23',\n",
       " 'citation': '\"Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii.\" Frontiers in Microbiology, vol. 9, 2018, doi:10.3389/fmicb.2018.00850.'}"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.documents[0].metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    }
   ],
   "source": [
    "brol = await library.adoc_match('agricultural disease')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Liao, Yen-Te, et al. \"Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization.\" Frontiers in Microbiology, vol. 10, 2019, p. 627. doi: 10.3389/fmicb.2019.00627.'"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "brol[0].get_content()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Tan, C. W., Rukayadi, Y., Hasan, H., Abdul-Mutalib, N.-A., Jambari, N. N., Hara, H., Thung, T. Y., Lee, E., & Radu, S. (2021). Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples. Frontiers in Microbiology, 12, 616548. doi: 10.3389/fmicb.2021.616548'"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "{\"action\":\"read_disk_use\",\"level\":\"error\",\"msg\":\"failed to read disk usage: no such file or directory\",\"path\":\"./weaviate/.local/share/weaviate\",\"time\":\"2024-01-24T09:14:35Z\"}\n",
      "{\"action\":\"read_disk_use\",\"level\":\"error\",\"msg\":\"failed to read disk usage: no such file or directory\",\"path\":\"./weaviate/.local/share/weaviate\",\"time\":\"2024-01-24T09:14:36Z\"}\n",
      "{\"action\":\"read_disk_use\",\"level\":\"error\",\"msg\":\"failed to read disk usage: no such file or directory\",\"path\":\"./weaviate/.local/share/weaviate\",\"time\":\"2024-01-24T09:14:36Z\"}\n"
     ]
    }
   ],
   "source": [
    "brol[1].get_content()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Test json loader"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/workspaces/ml-learning/.venv/lib/python3.11/site-packages/pkg_resources/_vendor/jaraco/text/__init__.py:593: ResourceWarning: unclosed file <_io.TextIOWrapper name='/workspaces/ml-learning/.venv/lib/python3.11/site-packages/llama_index/download/llamahub_modules/requirements.txt' mode='r' encoding='UTF-8'>\n",
      "  for item in lines:\n",
      "ResourceWarning: Enable tracemalloc to get the object allocation traceback\n"
     ]
    }
   ],
   "source": [
    "from pathlib import Path\n",
    "from llama_index import download_loader\n",
    "\n",
    "JSONReader = download_loader(\"JSONReader\")\n",
    "\n",
    "loader = JSONReader()\n",
    "documents = loader.load_data()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{}"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "documents[0].metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "embedded weaviate is already listening on port 8079\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "New documents count: 1\n",
      "calling get_citation\n",
      "citation prompt: metadata={'prompt_type': <PromptType.CUSTOM: 'custom'>} template_vars=['year', 'text'] kwargs={'year': 2024} output_parser=None template_var_mappings=None function_mappings=None template='Provide the citation for the following text in MLA Format. The year is {year}\\n\\n    {text}\\n\\n\\n    Citation:'\n",
      "text: [{\n",
      "  \"url\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "  \"crawl\": {\n",
      "    \"loadedUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research#\",\n",
      "    \"loadedTime\": \"2024-01-23T08:48:06.942Z\",\n",
      "    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "    \"depth\": 0,\n",
      "    \"httpStatusCode\": 200\n",
      "  },\n",
      "  \"metadata\": {\n",
      "    \"canonicalUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research#\",\n",
      "    \"title\": \"Articles | Frontiers\",\n",
      "    \"description\": null,\n",
      "    \"author\": null,\n",
      "    \"keywords\": null,\n",
      "    \"languageCode\": \"en\"\n",
      "  },\n",
      "  \"screenshotUrl\": null,\n",
      "  \"text\": \"Original Research\\nPublished on 30 Apr 2018\\nTwo New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2018.00850\\n7,708 views \\n45 citations \\nOriginal Research\\nPublished on 4 Jan 2018\\nMetavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community\\nChinmay V. Tikhe\\nClaudia Husseneder\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2017.02548\\n13,451 views \\n31 citations \\nOriginal Research\\nPublished on 28 Sep 2017\\nNewly Isolated Bacteriophages from the Podoviridae, Siphoviridae, and Myoviridae Families Have Variable Effects on Putative Novel Dickeya spp.\\nŠpela Alič\\nTina Naglič\\nMagda Tušek-Žnidarič\\nMaja Ravnikar\\nNejc Rački\\nMatjaž Peterka\\nTanja Dreo\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2017.01870\\n9,561 views \\n27 citations \\nOriginal Research\\nPublished on 2 Apr 2019\\nDiscovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization\\nYen-Te Liao\\nXincheng Sun\\nIrwin A. Quintela\\nDavid F. Bridges\\nFang Liu\\nYujie Zhang\\nAlexandra Salvador\\nVivian C. H. Wu\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2019.00627\\n4,555 views \\n13 citations \\nOriginal Research\\nPublished on 10 Mar 2021\\nIsolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples\\nChia Wanq Tan\\nYaya Rukayadi\\nHanan Hasan\\nNoor-Azira Abdul-Mutalib\\nNuzul Noorahya Jambari\\nHirofumi Hara\\nTze Young Thung\\nEpeng Lee\\nSon Radu\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2021.616548\\n12,006 views \\n25 citations \\nOriginal Research\\nPublished on 23 Jul 2019\\nA Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia\\nRuchira Sharma\\nBrittany A. Pielstick\\nKimberly A. Bell\\nTanner B. Nieman\\nOlivia A. Stubbs\\nEdward L. Yeates\\nDavid A. Baltrus\\nJulianne H. Grose\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2019.01533\\n16,064 views \\n34 citations \\nOriginal Research\\nPublished on 13 Feb 2018\\nIdentification and Characterization of T5-Like Bacteriophages Representing Two Novel Subgroups from Food Products\\nDomonkos Sváb\\nLinda Falgenhauer\\nManfred Rohde\\nJudit Szabó\\\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n",
      "after llm call citation= \"Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii.\" Frontiers in Microbiology, vol. 9, 2018, doi:10.3389/fmicb.2018.00850.\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n",
      "HTTP Request: POST https://api.openai.com/v1/embeddings \"HTTP/1.1 200 OK\"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from phages.modules.library import Library\n",
    "library = Library()\n",
    "library.add('/workspaces/ml-learning/src/phages/notebooks/data/dataset_phages-microbiology-frontiers_2024-01-23_08-51-12-772.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'[{\\n  \"url\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research#\",\\n    \"loadedTime\": \"2024-01-23T08:48:06.942Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 0,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research#\",\\n    \"title\": \"Articles | Frontiers\",\\n    \"description\": null,\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Original Research\\\\nPublished on 30 Apr 2018\\\\nTwo New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2018.00850\\\\n7,708 views \\\\n45 citations \\\\nOriginal Research\\\\nPublished on 4 Jan 2018\\\\nMetavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community\\\\nChinmay V. Tikhe\\\\nClaudia Husseneder\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2017.02548\\\\n13,451 views \\\\n31 citations \\\\nOriginal Research\\\\nPublished on 28 Sep 2017\\\\nNewly Isolated Bacteriophages from the Podoviridae, Siphoviridae, and Myoviridae Families Have Variable Effects on Putative Novel Dickeya spp.\\\\nŠpela Alič\\\\nTina Naglič\\\\nMagda Tušek-Žnidarič\\\\nMaja Ravnikar\\\\nNejc Rački\\\\nMatjaž Peterka\\\\nTanja Dreo\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2017.01870\\\\n9,561 views \\\\n27 citations \\\\nOriginal Research\\\\nPublished on 2 Apr 2019\\\\nDiscovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization\\\\nYen-Te Liao\\\\nXincheng Sun\\\\nIrwin A. Quintela\\\\nDavid F. Bridges\\\\nFang Liu\\\\nYujie Zhang\\\\nAlexandra Salvador\\\\nVivian C. H. Wu\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.00627\\\\n4,555 views \\\\n13 citations \\\\nOriginal Research\\\\nPublished on 10 Mar 2021\\\\nIsolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples\\\\nChia Wanq Tan\\\\nYaya Rukayadi\\\\nHanan Hasan\\\\nNoor-Azira Abdul-Mutalib\\\\nNuzul Noorahya Jambari\\\\nHirofumi Hara\\\\nTze Young Thung\\\\nEpeng Lee\\\\nSon Radu\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2021.616548\\\\n12,006 views \\\\n25 citations \\\\nOriginal Research\\\\nPublished on 23 Jul 2019\\\\nA Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia\\\\nRuchira Sharma\\\\nBrittany A. Pielstick\\\\nKimberly A. Bell\\\\nTanner B. Nieman\\\\nOlivia A. Stubbs\\\\nEdward L. Yeates\\\\nDavid A. Baltrus\\\\nJulianne H. Grose\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.01533\\\\n16,064 views \\\\n34 citations \\\\nOriginal Research\\\\nPublished on 13 Feb 2018\\\\nIdentification and Characterization of T5-Like Bacteriophages Representing Two Novel Subgroups from Food Products\\\\nDomonkos Sváb\\\\nLinda Falgenhauer\\\\nManfred Rohde\\\\nJudit Szabó\\\\nTrinad Chakraborty\\\\nIstván Tóth\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2018.00202\\\\n6,673 views \\\\n32 citations \\\\nOriginal Research\\\\nPublished on 26 Apr 2022\\\\nOptimisation and Application of a Novel Method to Identify Bacteriophages in Maternal Milk and Infant Stool Identifies Host-Phage Communities Within Preterm Infant Gut\\\\nGregory R. Young\\\\nWen C. Yew\\\\nAndrew Nelson\\\\nSimon H. Bridge\\\\nJanet E. Berrington\\\\nNicholas D. Embleton\\\\nDarren L. Smith\\\\nFrontiers in Pediatrics\\\\ndoi 10.3389/fped.2022.856520\\\\n2,547 views \\\\n2 citations \\\\nOriginal Research\\\\nPublished on 16 Nov 2022\\\\nBacteriophage-based decontamination to control environmental colonization by Staphylococcus capitis in neonatal intensive care units: An in vitro proof-of-concept\\\\nMarie Chavignon\\\\nCamille Kolenda\\\\nMathieu Medina\\\\nMélanie Bonhomme\\\\nLeslie Blazere\\\\nTiphaine Legendre\\\\nAnne Tristan\\\\nFrédéric Laurent\\\\nMarine Butin\\\\nFrontiers in Cellular and Infection Microbiology\\\\ndoi 10.3389/fcimb.2022.1060825\\\\n2,238 views \\\\n1 citations \\\\nOriginal Research\\\\nPublished on 2 May 2018\\\\nBacteriophage Infectivity Against Pseudomonas aeruginosa in Saline Conditions\\\\nGiantommaso Scarascia\\\\nScott A. Yap\\\\nAnna H. Kaksonen\\\\nPei-Ying Hong\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2018.00875\\\\n8,420 views \\\\n11 citations \\\\nOriginal Research\\\\nPublished on 6 Oct 2020\\\\nIdentification and Characterization of New Bacteriophages to Control Multidrug-Resistant Pseudomonas aeruginosa Biofilm on Endotracheal Tubes\\\\nViviane C. Oliveira\\\\nFelipe L. Bim\\\\nRachel M. Monteiro\\\\nAna Paula Macedo\\\\nEmerson S. Santos\\\\nCláudia H. Silva-Lovato\\\\nHelena F. O. Paranhos\\\\nLuís D. R. Melo\\\\nSílvio B. Santos\\\\nEvandro Watanabe\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2020.580779\\\\n9,906 views \\\\n24 citations \\\\nOriginal Research\\\\nPublished on 23 Jan 2020\\\\nBacteriophages as an Up-and-Coming Alternative to the Use of Sulfur Dioxide in Winemaking\\\\nGustavo Cordero-Bueso\\\\nJavier Moraga\\\\nMaría Ríos-Carrasco\\\\nMarina Ruiz-Muñoz\\\\nJesús Manuel Cantoral\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.02931\\\\n5,238 views \\\\n3 citations \\\\nOriginal Research\\\\nPublished on 30 Aug 2017\\\\nEnvironmental Bacteriophages of the Emerging Enterobacterial Phytopathogen, Dickeya solani, Show Genomic Conservation and Capacity for Horizontal Gene Transfer between Their Bacterial Hosts\\\\nAndrew Day\\\\nJiyoon Ahn\\\\nXinzhe Fang\\\\nGeorge P. C. Salmond\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2017.01654\\\\n3,267 views \\\\n15 citations \\\\nOriginal Research\\\\nPublished on 12 Sep 2018\\\\nJumbo Bacteriophages Are Represented Within an Increasing Diversity of Environmental Viruses Infecting the Emerging Phytopathogen, Dickeya solani\\\\nAndrew Day\\\\nJiyoon Ahn\\\\nGeorge P. C. Salmond\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2018.02169\\\\n4,563 views \\\\n17 citations \\\\nOriginal Research\\\\nPublished on 22 Dec 2022\\\\nA lytic bacteriophage isolate reduced Clostridium perfringens induced lesions in necrotic enteritis challenged broilers\\\\nChake Keerqin\\\\nKatherine McGlashan\\\\nThi Thu Hao Van\\\\nHelene N. Chinivasagam\\\\nRobert J. Moore\\\\nMingan Choct\\\\nShu-Biao Wu\\\\nFrontiers in Veterinary Science\\\\ndoi 10.3389/fvets.2022.1058115\\\\n1,860 views \\\\n2 citations \\\\nOriginal Research\\\\nPublished on 21 Mar 2019\\\\nTherapeutic Characterization and Efficacy of Bacteriophage Cocktails Infecting Escherichia coli, Klebsiella pneumoniae, and Enterobacter Species\\\\nPrasanth Manohar\\\\nAshok J. Tamhankar\\\\nCecilia Stalsby Lundborg\\\\nRamesh Nachimuthu\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.00574\\\\n20,572 views \\\\n108 citations \\\\nOriginal Research\\\\nPublished on 28 Jun 2018\\\\nVectorized Delivery of Alpha-GalactosylCeramide and Tumor Antigen on Filamentous Bacteriophage fd Induces Protective Immunity by Enhancing Tumor-Specific T Cell Response\\\\nRossella Sartorius\\\\nLuciana D’Apice\\\\nPasquale Barba\\\\nDeborah Cipria\\\\nLaura Grauso\\\\nAdele Cutignano\\\\nPiergiuseppe De Berardinis\\\\nFrontiers in Immunology\\\\ndoi 10.3389/fimmu.2018.01496\\\\n4,170 views \\\\n18 citations \\\\nOriginal Research\\\\nPublished on 15 Nov 2017\\\\nPrion-Like Domains in Phagobiota\\\\nGeorge Tetz\\\\nVictor Tetz\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2017.02239\\\\n8,010 views \\\\n27 citations \\\\nOriginal Research\\\\nPublished on 20 Apr 2016\\\\nGenomics of Three New Bacteriophages Useful in the Biocontrol of Salmonella\\\\nCarlota Bardina\\\\nJoan Colom\\\\nDenis A. Spricigo\\\\nJennifer Otero\\\\nMiquel Sánchez-Osuna\\\\nPilar Cortés\\\\nMontserrat Llagostera\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2016.00545\\\\n10,249 views \\\\n38 citations \\\\nOriginal Research\\\\nPublished on 6 Apr 2022\\\\nThe Expansion of a Single Bacteriophage Leads to Bacterial Disturbance in Gut and Reduction of Larval Growth in Musca domestica\\\\nXinyu Zhang\\\\nShumin Wang\\\\nQian Zhang\\\\nKexin Zhang\\\\nWenjuan Liu\\\\nRuiling Zhang\\\\nZhong Zhang\\\\nFrontiers in Immunology\\\\ndoi 10.3389/fimmu.2022.885722\\\\n2,084 views \\\\n3 citations \\\\nOriginal Research\\\\nPublished on 21 Jul 2023\\\\nIsolation and characterization of a novel mycobacteriophage Kashi-VT1 infecting Mycobacterium species\\\\nTanmayee Nayak\\\\nAnuja Kakkar\\\\nRakesh Kumar Singh\\\\nLav Kumar Jaiswal\\\\nAnand Kumar Singh\\\\nLouise Temple\\\\nAnkush Gupta\\\\nFrontiers in Cellular and Infection Microbiology\\\\ndoi 10.3389/fcimb.2023.1173894\\\\n2,274 views \\\\nOriginal Research\\\\nPublished on 12 Mar 2013\\\\nAntiviral effect of cationic compounds on bacteriophages\\\\nMai Huong Chatain-Ly\\\\nSaliha Moussaoui\\\\nVeronique Rigobello\\\\nYann Demarigny\\\\nAnnabelle Vera\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2013.00046\\\\n8,076 views \\\\n47 citations \\\\nOriginal Research\\\\nPublished on 4 Apr 2019\\\\nBiodistribution of Liposome-Encapsulated Bacteriophages and Their Transcytosis During Oral Phage Therapy\\\\nJennifer Otero\\\\nAlba García-Rodríguez\\\\nMary Cano-Sarabia\\\\nDaniel Maspoch\\\\nRicard Marcos\\\\nPilar Cortés\\\\nMontserrat Llagostera\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.00689\\\\n8,053 views \\\\n43 citations \\\\nOriginal Research\\\\nPublished on 8 Nov 2016\\\\nUltrastructure and Viral Metagenome of Bacteriophages from an Anaerobic Methane Oxidizing Methylomirabilis Bioreactor Enrichment Culture\\\\nLavinia Gambelli\\\\nGeert Cremers\\\\nRob Mesman\\\\nSimon Guerrero\\\\nBas E. Dutilh\\\\nMike S. M. Jetten\\\\nHuub J. M. Op den Camp\\\\nLaura van Niftrik\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2016.01740\\\\n7,432 views \\\\n14 citations\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles?page=2\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles?page=2#\",\\n    \"loadedTime\": \"2024-01-23T08:48:29.060Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles?page=2#\",\\n    \"title\": \"Articles | Frontiers\",\\n    \"description\": null,\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"497,371 Article(s)\\\\nMost recent\\\\nMost viewed\\\\nMost cited\\\\nFilters\\\\nAll Domains\\\\nEngineering\\\\nHealth\\\\nHumanities and Social Sciences\\\\nScience\\\\nSustainability\\\\nYoung Minds\\\\nAny journal\\\\nAdvanced Optical Technologies\\\\nFrontiers for Young Minds\\\\nFrontiers in Acoustics\\\\nFrontiers in Adolescent Medicine\\\\nFrontiers in Aerospace Engineering\\\\nFrontiers in Aging\\\\nFrontiers in Aging Neuroscience\\\\nAny sections\\\\nAbdominal and Pelvic Pain\\\\nAbdominal Transplantation\\\\nAcoustic Materials, Noise Control and Sound Perception\\\\nAcoustic Metamaterials\\\\nAcoustic Remote Sensing\\\\nAcoustofluidics\\\\nAcute Stroke and Interventional Therapies\\\\nAny article type\\\\nAddendum\\\\nBook Review\\\\nBrief Research Report\\\\nClassification\\\\nClinical Case Study\\\\nClinical Study Protocol\\\\nClinical Trial\\\\nSince beginning\\\\nPast week\\\\nPast month\\\\nPast year\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850/full\",\\n    \"loadedTime\": \"2024-01-23T08:48:41.819Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850\",\\n    \"title\": \"Frontiers | Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii\",\\n    \"description\": \"Two lytic bacteriophages, WCHABP1 and WCHABP12, were recovered from hospital sewage and were able to infect 9 and 12 out of 18 carbapenem-resistant Acinetobacter baumannii clinical strains, which belonged to different clones. Electron microscopy scan showed that both bacteriophages had the similar morphology as those of the Myoviridae family. Whole genomic sequencing revealed 45.4- or 45.8-kb genome with a 37.6% GC content for WCHABP1 and WCHABP12, both of which showed significant DNA sequence similarity with bacteriophages of the Ap22virus genus within the Myoviridae family. Taxonomic analysis was therefore performed following the proposal approved by the International Committee on Taxonomy of Viruses, which confirmed that WCHABP1 and WCHABP12 represented two new species of the Ap22virus genus. No tRNAs but 88 and 89 open reading frames (ORFs) were predicted for the two bacteriophages, among which 22 and 21 had known function and encoded proteins for morphogenesis,packaging, lysis and nucleiotide metabolism. The C-terminal amino acids of the large unit of fiber tail proteins varied between the bacteriophages, which may explain their different host ranges. For most lytic bacteriophages, a set of holin and endolysin are required for lysis. However, no known holin-encoding genes were identified in WCHABP1 and WCHABP12, suggesting that they may use alternative, yet-to-be-identified, novel holins for host cell membrane lysis. To test the efficacy of the bacteriophages in protecting against A. baumannii infection, a Galleria mellonella larva model was used. Only <20% Galleria mellonella larvae survived at 96 h after being infected by carbapenem-resistant A. baumannii strains, from which the two bacteriophages were recovered. With the administration of WCHABP1 and WCHABP12, the survival of larvae increased to 75%, while the treatment of polymyxin B only slightly increased the survival rate to 25%. The isolation of two new lytic bacteriophages in this study could expand our sight on Acinetobacter bacteriophages and may offer new potential therapeutic alternatives against A. baumannii.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nOn 27 February 2017, World Health Organization (WHO) released a list of 12 types of antimicrobial-resistant “priority pathogens” that pose the greatest threat to human health (World Health Organization, 2017). In the list, Acinetobacter along with Pseudomonas and various species of the Enterobacteriaceae (e.g., Klebsiella, Escherichia coli, Serratia, and Proteus) are labeled as “Critical,” for which new antimicrobial agents are urgently needed. Acinetobacter baumannii is a non-fermentative, non-motile, catalase-positive, gram-negative bacterium. It is widely dispersed in soil and water as well as the healthcare environment with the ability to cause various infections such as bacteremia and pneumonia (Dijkshoorn et al., 2007; Peleg et al., 2008). In addition to its capability of survival on dry and moist surfaces (Berlau et al., 1999), a variety of intrinsic and acquired mechanisms for antimicrobial resistance and virulence (Di et al., 2011; Perez et al., 2011) have made A. baumannii a successful pathogen worldwide (Perez et al., 2007).\\\\nOne century ago, since the first day of their discovery, the compelling feature of bacteriophages as a natural enemy of bacteria has drawn attention of early researchers to exploit them as a mean to treat bacterial infections (d\\'herelle, 1917; Bruynoghe and Maisin, 1921; Helvoort, 2001). However, the successive discovery of penicillin and other antibiotics soon put bacteriophages on the shelf, with exception in the Soviet Union and Poland (Kutter et al., 2015). The emergence of multi-drug resistant (MDR) bacteria calls for alternatives of antibiotics and bacteriophages therefore gain interests for extensive studies again (Smith and Huggins, 1983; Soothill, 1992).\\\\nAs a potential alternative to antibiotics and a possible solution for impending infections caused by MDR bacteria, there are still constraints for bacteriophages to be widely applied in real clinical practice despite the presence of reported successful cases (Abubakar et al., 2016). The main challenges for such a bacteriophage therapy include that the relatively narrow lytic range against bacterial strains and the proneness of bacteria to develop resistance. To tackle the challenges, multiple lytic bacteriophages have been used in combination as cocktails (Merabishvili et al., 2009). The success of cocktail bacteriophage therapy against predominantly MDR bacteria such as A. baumannii relies on isolation of novel lytic bacteriophages. Here we report two newly isolated lytic bacteriophages named WCHABP1 and WCHABP12 of the Ap22virus genus within the Myoviridae family against carbapenem-resistant A. baumannii clinical isolates.\\\\nMethods\\\\nBacterial Strains\\\\nThis study included a total of 18 nonduplicate carbapenem-resistant (MICs of imipenem ≥ 8 mg/L) but polymyxin B-susceptible (MICs <2 mg/L) A. baumannii clinical strains recovered from blood, sputum, urine, ascites, bronchoalveolar lavage fluid (BALF), and drainages of different patients at West China Hospital, Sichuan University, China, from April to June 2016 (Table 1) as hosts for bacteriophage isolation. The species identification of them was established by partially sequencing the recA gene as described previously (Krawczyk et al., 2002). These isolates belonged to 18 different clones as determined using a pulsed field gel electrophoresis (PFGE) analysis (data not shown). Clones were defined for isolates with an 80% or above pattern similarity. Antimicrobial susceptibility testing was performed using the Vitek II automated microbiological system (bioMérieux, Marcy-l\\'Étoile, France) or broth microdilution method of the Clinical and Laboratory Standards Institute (CLSI) (CLSI, 2017) (for colistin, imipenem, polymycin B, and tigecycline). Breakpoints defined by U. S. Food and Drug Administration were used for tigecycline; otherwise, those defined by CLSI were applied. The results are shown in Table S1.\\\\nTable 1. Host ranges of WCHABP1 and WCHABP12.\\\\nBacteriophage Isolation, Titering, and Concentration\\\\nBacteriophages were isolated from sewage samples collected at the influx of the wastewater treatment plant at West China Hospital in September 2016. The isolation and propagation of bacteriophages were performed as described previously (Merabishvili et al., 2009) and titering was carried out using the agar overlay method (Kropinski et al., 2009). PEG 8000/NaCl precipitation was used to concentrate bacteriophages for further use as described previously (Peng et al., 2014).\\\\nDetermination of the Host Range\\\\nThe bacteriophage stock was diluted 1:106 with Luria-Bertani (LB) broth (Hopebio, Qingdao, China) and was adjusted to a final titer of 1 × 104 pfu/ml. Aliquots of 10 μl diluted bacteriophages were mixed with tested bacterial host strains or the original strains from which the bacteriophages were isolated. The mixtures were subjected to agar overlay method. The efficiency of plating (EOP) was calculated by dividing the bacteriophage titer of tested strain by that of original strain. A particular bacteriophage-bacterial strain combination was considered as high, medium and low production efficiency if the average EOP value was ≥ 0.5, ≥ 0.1 to < 0.5, and ≥ 0.001 to < 0.1, respectively; while a < 0.001 average EOP value suggests inefficient production (Khan and Nilsson, 2015).\\\\nElectron Microscopy\\\\nThe concentrated bacteriophage stocks were negatively stained with phosphotungstic acid (PTA) and were then imaged using a H-600 II transmission electron microscope at 75 kV (Hitachi, Tokyo, Japan) to acquire morphological information of single bacteriophage particles.\\\\nMultiplicity of Infection (MOI) Assay\\\\nBacteriophage stocks were diluted by LB broth (Hopebio, Qingdao, China) into 10-fold series. Aliquots of each dilution were mixed with their corresponding host cultures at 108 cfu/ml to result in different ratio (from 10−4 to 10 pfu/cfu) and were incubated at 37°C for 4 h. Bacteriophage progenies of each mixture ratio were titered to determine the highest production as the optimal MOI (Peng et al., 2014).\\\\nAdsorption Rate Assay\\\\nAliquots of bacteriophages were incubated with host strain cultures at the optimal MOI under 37°C. Samples (100 μl) were collected at 0, 3, 6, 9, 12, and 15 min, respectively, and were centrifuged to remove the bacterial cells. The supernatants were titered for determination of unabsorbed bacteriophages at each time interval (Peng et al., 2014).\\\\nOne-Step Growth Curve\\\\nBacteriophages (10 μl at a titer of 1 × 109 pfu/ml) were mixed with 5 ml of their host strain cultures at about 2 × 107 cfu/ml to reach a MOI of 0.1. The mixture was incubated at 37°C for 10 min to allow the complete adsorption and was then centrifuged at 15,000 g for 1 min to remove the unabsorbed bacteriophage particles by discarding the supernatants. The pellets were resuspended in LB broth and were incubated at 37°C. Aliquots (100 μl) were taken every 10 min from the beginning to 90 min and were titered for free bacteriophages after being centrifuged at 15,000 g for 1 min to remove bacterial cells. An additional 100 μl aliquot was taken at the beginning (0 min). A half of the aliquot (50 μl) was centrifuged and the remaining half was not centrifuged, both of which were titered for bacteriophages. Infected cells were calculated by subtracting the number of free bacteriophages of the centrifuged part from the number of bacteriophages of the un-centrifuged part, then the burst size could be calculated as dividing the maximal progeny counts by the number of infected cells (Peng et al., 2014).\\\\nBacteriophage Genome Sequencing and Bioinformatics Analysis\\\\nBacteriophage DNA was prepared using the standard phenol-chloroform extraction (Mandell and Hershey, 1960) after concentration. Whole genome sequencing was performed using the HiSeq 2500 Sequencer (Illumina, San Diego, CA, USA) with 150-bp paired-end (the final coverage was ~200x). De novo assembly was performed using SPAdes v3.10.1 (Bankevich et al., 2012) with auto-cutoff and careful mode. Annotation of the genomic sequence was carried out using Prokka v1.11 (Seemann, 2014) followed by manually confirmation via running BLASTp and PSI-BLAST (http://blast.ncbi.nlm.nih.gov/) (Altschul et al., 1990, 1997) against the non-redundant protein database with a significant E-value of <10−3. For annotated proteins, the conserved protein domains/motif and additional function inference were detected using InterProScan (http://www.ebi.ac.uk/interpro/search/sequence-search) (Quevillon et al., 2005), HMMER (https://www.ebi.ac.uk/Tools/hmmer), and Conserved Domain Database (https://www.ncbi.nlm.nih.gov/cdd) (Marchlerbauer et al., 2011). The ExPASy server (http://us.expasy.org/tools/protparam.html) was used to predict molecular weight and isoelectric point. Prediction of transmembrane helices was performed using TMHMM 2.0 (Krogh et al., 2001) and signal peptides were screened using SignalP (Petersen et al., 2011). Potential tRNA genes were identified using tRNAscan-SE (Lowe, 1997) and ARAGON (Laslett and Canback, 2004).\\\\nThe comparative genomic analysis between our bacteriophages and those deposited in GenBank was performed using BLASTn and BLAST Ring Image Generator (BRIG) (Alikhan et al., 2011). The newest proposal of International Committee on Taxonomy of Viruses (ICTV) (https://talk.ictvonline.org/files/ictv_official_taxonomy_updates_since_the_8th_report/m/prokaryote-official/5901) (Adams et al., 2017) was taken as the reference to determine the taxonomy of the bacteriophages. Mauve (http://asap.ahabs.wisc.edu/mauve/) and CoreGenes (Zafar et al., 2002; Mahadevan et al., 2009a,b) were used to compare the genomic organization and core genes for a given group of bacteriophages. Phylogenetic tree for taxonomic analysis was constructed via phylogeny.fr with “One Click mode” using MUSCLE for multiple alignments, PhyML for phylogeny, and Gblocks for eliminating poorly aligned positions and divergent regions (Anisimova and Gascuel, 2006).\\\\nNucleotide accession no. Genome sequences of bacteriophages WCHABP1 and WCHABP12 in the present study have been deposited into GenBank under accession no. KY829116 and KY670595.\\\\nAnalysis of the Genome Ends\\\\nGenome ends were determined as described previously (Casjens and Gilcrease, 2009). Briefly, 1 μg of each bacteriophage genome DNA was digested with the restriction enzyme HindIII (TaKaRa, Dalian, China). The mixture was heated at 80°C for 15 min to inactivate the digestion and was divided into two equal aliquots. One was rapidly chilled in ice water bath and the other was left to slowly cool to the room temperature. The resulting DNA bands were separated in a 1% agarose gel by electrophoresis. In parallel, 1 μg of Lambda DNA standard digested with the restriction enzyme EcoRV (TaKaRa) was used as control.\\\\nSodium Dodecyl Sulfate–Polyacrylamide Gel Electrophoresis (SDS-PAGE) Analysis\\\\nBacteriophage particles were boiled in loading buffer (50 mM tris-HCl, 2% Sodium dodecyl sulfate, 0.1% bromophenol blue, 10% glycerol, and 1% β-mercaptoethanol) for 5 min. Denatured proteins were then separated using sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS-PAGE) with a 5% concentration gel and a 12% separation gel and were stained with Coomassie Blue G-250 as described previously (Boulanger, 2009).\\\\nGalleria mellonella Bacteriophage Therapy Assay\\\\nG. mellonella infected by A. baumannii was used to study in vivo antibacterial efficacy of our bacteriophages (Peleg and Jara, 2009). Larvae of G. mellonella of 250–350 mg in weight (Huiyude, Tianjin, China) were stored at 4°C and were used within 1 week after delivery. Briefly, 16 randomly chosen larvae were used for each group. Host bacterial cultures of strain Ab1186 and Ab1262, from which the two bacteriophages were recovered, were washed by phosphate-buffered saline (PBS; Beyotime, Shanghai, China) and were then diluted 1:10 to an appropriate cell density (107 cfu/ml) as determined using a McFarland turbidimetry and direct plate colony counting. Aliquots of 10 μl PBS-washed bacterial cultures (105 cfu) were injected into the hemocoel of each larva through the last left proleg using a microsyringe (Gaoge, Shanghai, China), which led to a pre-determined LD80 dose for each inoculum (105 cfu/larva). Bacteriophage stocks were re-concentrated as described above and were re-suspended and diluted with PBS to the optimal MOI titer (for host cell at 105 cfu/ml, the bacteriophage dosage was 104 pfu/ml to reach a final MOI of 0.1 pfu/cfu). Aliquots of 10 μl bacteriophage suspensions (104 pfu/larva) were then injected into the larva via a different proleg within 30 min from the bacterial inoculation. In parallel, larvae injected with bacterial cells at the same concentration were treated with polymyxin B (Meilun, Dalian, China) at 2.5 mg/kg (Hornsey and Wareham, 2011) to compare the therapeutic efficacy between antibiotics and bacteriophages. For each bacteriophage, four control groups were set up including larvae infected with bacterial cells and treated with PBS solution to observe the bacterial virulence (group 1), those injected with either the bacteriophage suspension or polymyxin B to assess their potential toxicity (group 2 and 3); those injected with PBS only to observe the potential physical trauma from injection (group 4). Larvae were then incubated in plastic containers at 37°C and the number of dead, determining as no movement under touch, was counted at 24 h intervals up to 96 h after the incubation. When more than two larvae died in any negative control (group 2, 3, and 4) at the end of observation, the assay was considered invalid and was repeated. Assays were performed in triplicate using different batches of larvae.\\\\nThe statistical analysis was performed by GraphPad Prism v.7.0 (Software Inc., La Jolla, CA, USA) to plot the survival curves with the Kaplan-Meier method following a log-rank test to calculate the differences in survival. A P-value of < 0.05 was considered to be statistically significant.\\\\nResults\\\\nTwo Bacteriophages of the Myoviridae Family Were Recovered\\\\nTwo bacteriophages, designated WCHABP1 and WCHABP12, were obtained from hospital sewage with the ability to infect 9 and 12 out of the 18 host strains, exhibiting that both bacteriophages can infect multiple strains of A. baumannii with varied efficiency (Table 1). Upon infecting their original host strain, clear plaques of 8–9 and 5–7 mm in diameter with all surrounded by a 1–3 mm halo were formed by bacteriophage WCHABP1 and WCHABP12, respectively.\\\\nBoth bacteriophages produced the highest progeny at MOI of 0.1 (1.55 × 1010 pfu/ml for WCHABP1 and 2.2 × 1010 for WCHABP12), which was therefore used in subsequent experiments. The adsorption rate of them was comparable where more than 99% of bacteriophage particles could adsorb onto the host cell within 10 min (Figure 1). After the adsorption, for both WCHABP1 and WCHABP12, the eclipse period was between 10 and 20 min and the plateau was reached at 60 min with a slightly different final titer (1.5 × 109 pfu/ml for WCHABP1 and 2.1 × 109 pfu/ml for WCHABP1; Figure 1). Considering that the initial number of infected cells were 1.1 × 107 cfu/ml for WCHABP1 and 1.2 × 107 cfu/ml for WCHABP12, the burst size of the two bacteriophages was 136 and 175 pfu per infected cell, respectively.\\\\nFigure 1. Growth and lytic characteristics. (A) adsorption rate; (B) One-step growth curve. WCHABP1 and WCHABP12 are labeled by asterisk and circle, respectively.\\\\nThe electron microscopy revealed that both WCHABP1 and WCHABP12 had a ~75 nm icosahedral head and a ~105 nm tail, on which crossed striation could be seen (Figure 2). The morphologic features suggested that both WCHABP1 and WCHABP12 belong to the Myoviridae family.\\\\nFigure 2. Electron microscopy of bacteriophage morphology. (a) WCHABP1; (b) WCHABP12. The black scale bar represents 50 nm.\\\\nThe Two Bacteriophages Represent Two New Species of the Ap22virus Genus Within the Myoviridae Family\\\\nGenomes of the WCHABP1 and WCHABP12 were 45,888 and 45,415 bp in length with the same GC content of 37.6 mol%. The number of ORFs predicted was 88 for WCHABP1 and 89 for WCHABP12, respectively. Genomes of WCHABP1 and WCHABP12 were highly similar with each other (88.0% overall DNA sequence identity) and had 51.0 to 56.0% overall DNA sequence identity to a set of bacteriophages (Table 2) belonging to the Ap22virus genus of the Myoviridae family.\\\\nTable 2. Genomic properties of WCHABP1 and WCHABP12 compared with other bacteriophages of the Ap22virus genus.\\\\nTo investigate the taxonomy of WCHABP1 and WCHABP12, we followed the approach of the recently approved taxonomic proposal for the Ap22virus genus (Adams et al., 2017). Bacteriophages of this genus possess genome of ~45 kb (around 37.6 mol% G+C content) and encode ORFs close to 85 in number with no tRNAs. Members of the Ap22virus genus had an average of 53% overall DNA sequence identity and 64% homologous proteins compared with the reference species bacteriophage AP22. WCHABP1 and WCHABP12 exhibited a higher overall DNA sequence identity (58 and 62%) and homologous proteins (74 and 75%) with AP22, carrying almost the same number of ORFs as AP22 (88 and 89 vs. 89). The multiple genome alignment of WCHABP1, WCHABP12, and bacteriophages of the Ap22virus genus showed that all of these genomes were in a resembling gene cluster and order (Figure S1). Phylogenetic trees constructed with bacteriophage major capsid and baseplate J-like protein revealed that both WCHABP1 and WCHABP12 were clustered together within the Ap22virus genus (Figure S2). All above findings suggest that WCHABP1 and WCHABP12 belonged to the Ap22virus genus. In addition, WCHABP1 and WCHABP12 had <95.0% identity between each other and with any other bacteriophages of the Ap22virus genus. Therefore, WCHABP1 and WCHABP12 present two new species of the Ap22virus genus.\\\\nGenomes of the Two Bacteriophages Encode Proteins for Morphogenesis, Genome Packaging, Nucleotide Metabolism, and Host Lysis\\\\nAmong ORFs of WCHABP1 and WCHABP1, 22 (WCHABP1) and 21 (WCHABP12) could be assigned to known function including morphogenesis, genome packaging, nucleotide metabolism, and host lysis. Neither tRNAs nor genes involving in the lysogenic process such as integrase and repressor were identified in both bacteriophages (Table 2, Figure 3).\\\\nFigure 3. Multiple genome alignments of WCHABP1, WCHABP12, and bacteriophages of the Ap22virus genus. (A) the genome of WCHABP1 is used as the reference. (B) the genome of WCHABP12 is used as the reference. The alignment is a pairwise BLASTn performed using BRIG. Accession numbers of genomes of bacteriophages of the Ap22virus genus are listed in Table 2.\\\\nMorphology Module\\\\nThe SDS-PAGE analysis showed similar protein patterns between WCHABP1 and WCHABP12 (Figure S3). The protein bands mainly converged at 15, 35, 40–55, and 70 kDa, corresponding to the respective capsid, portal and tail fiber proteins of the two bacteriophages in light of the predicted molecular weight.\\\\nBioinformatics analysis predicted seven ORFs encoding structural proteins for the two bacteriophages. These structural proteins included two capsid proteins (gp27 and gp43 of WCHABP1; gp66 and gp82 of WCHABP12), one portal protein (gp44 of WCHABP1; gp65 of WCHABP12), two baseplate proteins (gp08 and gp10 of WCHABP1; gp11 and gp13 of WCHABP12), and two tail fiber proteins (gp05 and gp06 of WCHABP1; gp15 and gp16 of WCHABP12). The structural proteins encoded by each of the two bacteriophage genomes are highly similar (>60% overall amino acids identity, a detailed list is shown in Table 3) with the exception of the large tail fiber subunits (gp05 of WCHABP1 and gp16 of WCHABP12). The N-terminal 149 amino acids of the large tail fiber subunit of WCHABP1 and WCHABP12 were of 100% coverage and 76% identity, while the remaining C-terminal amino acids had no significant similarity.\\\\nTable 3. Genes with known function in bacteriophage WCHABP1 and WCHABP12.\\\\nIn addition, both WCHABP1 and WCHABP12 had a gene encoding a lysozyme-containing protein, which is also possessed by other bacteriophages of the Ap22virus genus. The lysozyme is a glycoside hydrolase that catalyzes the peptidoglycan of the bacterial cell wall (Strynadka and James, 1996). The lysozyme-containing protein lies close to the baseplate protein and tail fiber proteins, suggesting that it might be a part of tail. Lysozyme could be released from the inner tail tube to create a small hole in the cell wall peptidoglycan layer that allows the double-stranded DNA (dsDNA) genome to enter the cell (Duda, 2008). Nonetheless, the exact function of the lysozyme-containing protein in the bacteriophages warrants further studies.\\\\nPackaging Module and Genome Termini\\\\nTwo ORFs encoding terminase, which is the key component of the packaging machine, were identified in both bacteriophage genomes (gp45 and gp46 of WCHABP1; gp63 and gp64 of WCHABP12). Terminase for bacteriophages usually comprises two subunits, where the large subunit translocates bacteriophage DNA into empty capsids and cuts it at a unique and precise dsDNA sequence to accomplish the packaging process, while the small subunit contributes to binding of the packaging initiation site and regulates the ATPase activity for the large subunit (Feiss and Rao, 2012). Amino acid sequences of terminase subunits between WCHABP1 and WCHABP12 were identical (Table 3). The restriction map of genomes of WCHABP1 and WCHABP12 showed no change in band patterns after endonuclease restriction with rapid or slow cooling, suggesting the absence of cohesive ends in their genomes (Figure S4).\\\\nNucleotide Metabolism Module\\\\nBoth WCHABP1 and WCHABP12 had 10 genes involving in genome replication, transcription, establishment of infection, and other processes, including a RNA polymerase, a DNA helicase, a transcriptional regulator, a fis family transcriptional regulator, an ERF (essential recombination function) family protein, a nucleoside triphosphate pyrophosphohydrolase, a HNH endonuclease, an endodeoxyribonuclease, a superinfection immunity protein and a bacteriophage antirepressor protein (Table 2). Products of these genes were highly similar (93–100% overall amino acids identity) between the two bacteriophages except for the nucleotide triphosphate pyrophosphohydrolase (gp89 of WCHABP1 and gp21 of WCHABP12) with a 97% coverage but only 63% amino acid identity. Both gp89 of WCHABP1 and gp21 of WCHABP12 contained an NTP-PPase_u3 domain (also known as MazG domain, CCD accession cd11540), suggesting a similar function. Previous study suggests that the MazG domain-containing proteins might contribute to extending the logarithmic phase of bacterial growth, facilitating the production of bacteriophage progeny via the reactivation of metabolic pathways that are usually suppressed under nutrient starvation (Bryan et al., 2008).\\\\nBesides the identical HNH endonuclease seen in both WCHABP1 (gp33) and WCHABP12 (gp76), WCHABP1 had another HNH endonuclease (gp02), which is absent from WCHABP12 and any other members of the Ap22virus genus and has a relatively low similarity (31% coverage and 31% identity) to the aforementioned HNH endonuclease of WCHABP1 and WCHABP12. The gp02 product of WCHABP1 consisted of a HNH endonuclease domain (Pfam accession PF13392.5) and an AP2 domain (Pfam accession PF00847.19), while the gp33 product of WCHABP1 and gp76 product of WCHABP12 contained a HNH endonuclease domain (Pfam accession PF13392.5) and a NUMOD4 motif (Pfam accession PF07463.10). The HNH endonuclease domain is involved in the process of homing, while both AP2 domain and NUMOD4 motif play a role in DNA binding (Ohmetakagi and Shinshi, 1995; Sitbon and Pietrokovski, 2003) but without significant amino acid similarity.\\\\nLysis Control\\\\nWCHABP1 and WCHABP12 had one gene encoding the endolysin. However, endolysins of the two bacteriophages had no significant similarity in amino acid sequences. The endolysin of WCHABP1 (gp01) had 171 amino acids and contains a glycosyl hydrolase 108 family domain (Pfam accession PF05838.11) with catalytic activity as a N-acetylmuramidase (Stojković and Rothman-Denes, 2007) and a peptidoglycan domain for substrate binding (Pfam accession PF09374.9). By contrast, the endolysin of WCHABP12 (gp19) had 202 amino acids with a single domain belonging to the glycoside hydrolase family 19 (Pfam accession PF00182.18), which is able to hydrolyze the β-1,4-N-acetyl-D-glucosamine linkages in chitin polymers and leads to the breakage of chitin-containing cell walls (Eijsink et al., 2008). The different types of endolysins between WCHABP1 and WCHABP12 suggest that even closely-related bacteriophages may target different substrates of the host cell wall.\\\\nMost dsDNA bacteriophages with a genome size larger than 10 kb employ a complete set of a holin-endolysin system to release the phage progeny (Young and White, 2008). The holin first permeabilizes the host cell membrane and endolysin can therefore reach the periplasm to break down the peptidoglycan layer (Young et al., 2000). However, for both WCHABP1 and WCHABP12 no known holin-encoding genes were identified using both a BLASTp search and domain/motif prediction.\\\\nIn Vivo Therapeutic Efficacy\\\\nFor larvae infected by carbapenem-resistant A. baumannii strains, the treatment of WCHABP1 and WCHABP12 significantly improved the survival rates (Figure 4 and Table S2). For those infected by strain Ab1186, survival of larvae at 96 h was 75.00 and 18.75% (p < 0.01) with and without the treatment of WCHABP1, respectively. Similarly, for those infected by strain Ab1262, survival was 75 and 12.5% (p < 0.01) with and without the treatment of WCHABP12, respectively. By contrast, the treatment of polymyxin B only slightly increased the survival, which is not statistically significant. With and without the treatment of polymyxin B, the survival rate of those infected by strain Ab1186 was 25 and 18.75% (p = 0.46), respectively, and that of those infected by Ab1262 was 25 and 12.5% (p = 0.47), respectively. The treatment of the combination of bacteriophage and polymyxin B did not further enhance the survival of larvae infected by strain Ab1186 or Ab1262 compared with the treatment of bacteriophage alone (68.75 vs. 75%, p = 0.687).\\\\nFigure 4. Survival of G. mellonella larvae after infection by carbapenem-resistant A. baumannii clinical strains. (A) WCHABP1; (B) WCHABP12. Ab1186 and Ab1262 are two carbapenem-resistant A. baumannii clinical strains, from which bacteriophages WCHABP1, WCHABP12 were recovered. The detailed survival rates are listed in Table S2.\\\\nDiscussion\\\\nThe recent success in treating the infection caused by MDR A. baumannii by topical and systematic administration of varied bacteriophages together with antibiotics once again highlights the potential of bacteriophages as promising alternatives against bacterial infections that are difficult to be treated otherwise (https://www.sciencedaily.com/releases/2017/04/170425124826.htm). Nonetheless, constant works on isolation and characterization of new bacteriophages as well as their in vitro and in vivo test for both safety and therapeutic efficacy are still imperative. In this study, we reported two bacteriophages, WCHABP1 and WCHABP12, belonging to the Ap22virus genus of the Myoviridae family with a large burst size and short lytic life cycle against the carbapenem-resistant A. baumannii host strains. The bacteriophages identified in the present study together with others discovered in previous studies (Lee et al., 2011; Jin et al., 2012; Popova et al., 2012; Ghajavand et al., 2017) may expand our arsenal against A. baumannii.\\\\nThe injection of bacteriophages WCHABP1 and WCHABP12 did not cause any death of larvae, suggesting that they were not toxic for the larvae. The administration of WCHABP1 and WCHABP12 was able to rescue most larvae infected by carbapenem-resistant A. baumannii susceptible to the lysis of the bacteriophages, suggesting both bacteriophages new potent weapons against A. baumannii. By contrast, the treatment of polymyxin B alone was only able to rescue few larvae infected by these strains that were susceptible to the antibiotic, which is consistent with the findings of another study (Hornsey et al., 2013). A few factors may contribute to the failure of polymyxin B treatment such as emerging tolerance and resistance to the antibiotic after exposure (Li et al., 2006; Harding et al., 2018).\\\\nFor products encoded by the genomes of WCHABP1, WCHABP12, and other members of the Ap22virus genus, there are several noteworthy features. First, the C-terminal amino acid sequence of the large tail fiber subunit had no significant similarity between WCHABP1 and WCHABP12. Such highly variable C-terminal sequences of large tail fiber subunits are also present among other members of the Ap22virus genus. Previous studies have revealed that the large tail fiber subunit of various bacteriophages including those belonging to the Myoviridae family has a conserved N-terminus and a variable C-terminus, which determines the host specificity by binding the receptor on the surface of host cell (Rakhuba et al., 2010; Kurtböke, 2012). The variable C-terminal sequence of large tail fiber subunits may therefore account for the different host ranges of bacteriophages of the Ap22virus genus. Second, like those of other bacteriophages of the Ap22virus genus, the large terminase subunit of WCHABP1 and WCHABP12 had a terminase_3 family domain (Pfam accession pfam04466) and were distantly homologous (26% identity with a E-value of 2e-22) with that of bacteriophage SPP1 (GenBank accession NC_004166), which is a well-studied model system for headful DNA packaging mechanism (also known as pac-mechanism; Oliveira et al., 2013). This suggests that both WCHABP1 and WCHABP12 also employ a pac-mechanism for genome packaging and their genomes are circularly permuted and terminally redundant (Casjens and Gilcrease, 2009; Oliveira et al., 2013). Third, surprisingly, unlike the typical holin-endolysin set seen in most bacteriophages, no known holin-encoding genes were identified for bacteriophages of the Ap22virus genus. Two ORFs (gp3 and gp4 of WCHABP1; gp17 and gp18 of WCHABP12) were present between genes encoding endolysin and the large tail fiber subunit and were highly similar among members of the Ap22virus genus. The products encoded by the two ORFs had two and three transmembrane domains as identified using TMHMM. As most holins contain transmembrane domains (Young et al., 2000) and their encoding genes usually locate proximately to endolysin-encoding genes, it is reasonable to hypothesize that the two ORFs may encode a new, yet-to-be-characterized holin. Alternatively, as shown in a previous study there are means other than holin for bacteriophages to permeabilize the host cell membrane via inhibiting the specific host enzymes and impairing peptidoglycan biosynthesis (Bernhardt et al., 2001). Therefore, the exact lysis mechanism employed by bacteriophages of the Ap22virus genus remains unclear and warrants further studies.\\\\nAlthough genomes of bacteriophages WCHABP1 and WCHABP12 were highly similar, there were several differences, which have been demonstrated above and are summarized here. Endolysins of WCHABP1 and WCHABP12 had no significant similarity in amino acid sequences. WCHABP1 had one more HNH endonuclease than WCHABP12. The C-terminal amino acids of the large unit of tail fiber protein between the two bacteriophages had no significant similarity. In addition, WCHABP1 had three genes without known function that were absent from WCHABP12, while two genes with unknown function of WCHABP12 were not present in WCHABP1.\\\\nIn conclusion, two new bacteriophages, exhibiting activity to infect and kill multiple carbapenem-resistant A. baumannii clinical strains, were recovered. The two bacteriophages represent two new species of the Ap22virus genus within the Myoviridae family. Administration of the two bacteriophages was effective to treat carbapenem-resistant A. baumannii infections in the G. mellonella larvae model. The findings could expand our sight on Acinetobacter bacteriophages and offer new potential therapeutic alternatives.\\\\nAuthor Contributions\\\\nZZ designed the study. WZ performed the experiments and prepared figures. WZ, YF, and ZZ analyzed and interpreted the data. WZ and ZZ wrote the manuscript.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nThis work was supported by grants from the National Natural Science Foundation of China (project no. 81222025, 81572030, and 81661130159) and the Newton Advanced Fellowship, Royal Society, UK (NA150363).\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850/full#supplementary-material\\\\nReferences\\\\nAbubakar, S., Suleiman, B. H., Abbagana, B. A., Mustafa, I. A., and Musa, I. A. (2016). Novel uses of bacteriophages in the treatment of human infections and antibiotic resistance. Am. J. Biosci. 4, 34–40. doi: 10.11648/j.ajbio.20160403.13\\\\nCrossRef Full Text | Google Scholar\\\\nAdams, M. J., Lefkowitz, E. J., King, A. M. Q., Harrach, B., Harrison, R. L., Knowles, N. J., et al. (2017). Changes to taxonomy and the international code of virus classification and nomenclature ratified by the international committee on taxonomy of viruses (2017). Arch. Virol. 162, 1–34. doi: 10.1007/s00705-017-3358-5\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAlikhan, N. F., Petty, N. K., Ben Zakour, N. L., and Beatson, S. A. (2011). BLAST Ring Image Generator (BRIG): simple prokaryote genome comparisons. BMC Genomics 12:402. doi: 10.1186/1471-2164-12-402\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAltschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990). Basic local alignment search tool. J. Mol. Biol. 215, 403–410. doi: 10.1016/S0022-2836(05)80360-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAltschul, S. F., Madden, T. L., Schäffer, A. A., Zhang, J., Zhang, Z., Miller, W., et al. (1997). Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 25, 3389–3402. doi: 10.1093/nar/25.17.3389\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAnisimova, M., and Gascuel, O. (2006). Approximate likelihood-ratio test for branches: a fast, accurate, and powerful alternative. Syst. Biol. 55, 539–552. doi: 10.1080/10635150600755453\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455–477. doi: 10.1089/cmb.2012.0021\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBerlau, J., Aucken, H. M., Houang, E., and Pitt, T. L. (1999). Isolation of Acinetobacter spp. including A. baumannii from vegetables: implications for hospital-acquired infections. J. Hosp. Infect. 42, 201–204. doi: 10.1053/jhin.1999.0602\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBernhardt, T. G., Wang, I. N., Struck, D. K., and Young, R. (2001). A protein antibiotic in the phage Qβ virion: diversity in lysis targets. Science 292, 2326–2329. doi: 10.1126/science.1058289\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBoulanger, P. (2009). Purification of bacteriophages and SDS-PAGE analysis of phage structural proteins from ghost particles. Methods Mol. Biol. 502, 227. doi: 10.1007/978-1-60327-565-1_13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBruynoghe, R., and Maisin, J. (1921). Essais de thérapeutique au moyen du bacteriophage. CR Soc. Biol. 85, 1120–1121.\\\\nBryan, M. J., Burroughs, N. J., Spence, E. M., Clokie, M. R. J., Mann, N. H., and Bryan, S. J. (2008). Evidence for the intense exchange of mazG in marine cyanophages by horizontal gene transfer. PLoS ONE 3:e2048. doi: 10.1371/journal.pone.0002048\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCasjens, S. R., and Gilcrease, E. B. (2009). Determining DNA packaging strategy by analysis of the termini of the chromosomes in tailed-bacteriophage virions. Methods Mol. Biol. 502, 91–111. doi: 10.1007/978-1-60327-565-1_7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCLSI (2017). Performance Standards for Antimicrobial Susceptibility Testing; Twenty-Third Informational Supplement. M100-S27. (Wayne, PA: Clinical and Laboratory Standards Institute).\\\\nd\\'herelle, F. (1917). Sur un microbe invisible antagoniste des bacilles dysentériques. CR Acad. Sci. Paris 165, 373–375.\\\\nDi, P. A., Giannouli, M., Triassi, M., Brisse, S., and Zarrilli, R. (2011). Molecular epidemiological investigation of multidrug-resistant Acinetobacter baumannii strains in four Mediterranean countries with a multilocus sequence typing scheme. Clin. Microb. Infect. 17, 197–201. doi: 10.1111/j.1469-0691.2010.03254.x\\\\nCrossRef Full Text | Google Scholar\\\\nDijkshoorn, L., Nemec, A., and Seifert, H. (2007). An increasing threat in hospitals: multidrug-resistant Acinetobacter baumannii. Nat. Rev. Microbiol. 5, 939–951. doi: 10.1038/nrmicro1789\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nEijsink, V. G. H., Vaaje-Kolstad, G., Vårum, K. M., and Horn, S. J. (2008). Towards new enzymes for biofuels: lessons from chitinase research. Trends Biotechnol. 26, 228–235. doi: 10.1016/j.tibtech.2008.02.004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGhajavand, H., Esfahani, B. N., Havaei, A., Fazeli, H., Jafari, R., and Moghim, S. (2017). Isolation of bacteriophages against multidrug resistant Acinetobacter baumannii. Res. Pharm. Sci. 12, 373–380. doi: 10.4103/1735-5362.213982\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHarding, C. M., Hennon, S. W., and Feldman, M. F. (2018). Uncovering the mechanisms of Acinetobacter baumannii virulence. Nat. Rev. Microbiol. 16, 91–102. doi: 10.1038/nrmicro.2017.148\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHelvoort, T. V. (2001). Felix d\\'Herelle and the origins of molecular biology. J. Hist. Med. Allied Sci. 56, 305–307. doi: 10.1093/jhmas/56.3.305\\\\nCrossRef Full Text | Google Scholar\\\\nHornsey, M., and Wareham, D. W. (2011). In vivo efficacy of glycopeptide-colistin combination therapies in a Galleria mellonella model of Acinetobacter baumannii infection. Antimicrob. Agents Chemother. 55, 3534–3537. doi: 10.1128/AAC.00230-11\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHornsey, M., Phee, L., Longshaw, C., and Wareham, D. W. (2013). In vivo efficacy of telavancin/colistin combination therapy in a Galleria mellonella model of Acinetobacter baumannii infection. Int. J. Antimicrob. Agents 41, 285–287. doi: 10.1016/j.ijantimicag.2012.11.013\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJin, J., Li, Z. J., Wang, S. W., Wang, S. M., Huang, D. H., Li, Y. H., et al. (2012). Isolation and characterization of ZZ1, a novel lytic phage that infects Acinetobacter baumannii clinical isolates. BMC Microbiol. 12:156. doi: 10.1186/1471-2180-12-156\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKhan, M. M., and Nilsson, A. S. (2015). Isolation of phages for phage therapy: a comparison of spot tests and efficiency of plating analyses for determination of host range and efficacy. PLoS ONE 10:e0118557. doi: 10.1371/journal.pone.0118557\\\\nCrossRef Full Text | Google Scholar\\\\nKrawczyk, B., Lewandowski, K., and Kur, J. (2002). Comparative studies of the Acinetobacter genus and the species identification method based on the recA sequences. Mol. Cell. Probes 16, 1–11. doi: 10.1006/mcpr.2001.0388\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKrogh, A., Larsson, B., Von Heijne, G., and Sonnhammer, E. L. (2001). Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. J. Mol. Biol. 305, 567–580. doi: 10.1006/jmbi.2000.4315\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKropinski, A. M., Mazzocco, A., Waddell, T. E., Lingohr, E., and Johnson, R. P. (2009). Enumeration of bacteriophages by double agar overlay plaque assay. Methods Mol. Biol. 501, 69–76. doi: 10.1007/978-1-60327-164-6_7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLee, C. N., Tseng, T. T., Lin, J. W., Fu, Y. C., Weng, S. F., and Tseng, Y. H. (2011). Lytic myophage Abp53 encodes several proteins similar to those encoded by host Acinetobacter baumannii and phage phiKO2. Appl. Environ. Microbiol. 77, 6755–6762. doi: 10.1128/AEM.05116-11\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLi, J., Rayner, C. R., Nation, R. L., Owen, R. J., Spelman, D., Tan, K. E., et al. (2006). Heteroresistance to colistin in multidrug-resistant Acinetobacter baumannii. Antimicrob. Agents Chemother. 50, 2946–2950. doi: 10.1128/AAC.00103-06\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLowe, T. (1997). tRNAscan-SE: a program for improved transfer RNA detection in genomic sequence (release: 1.23). Nucleic Acids Res. 25, 955–964. doi: 10.1093/nar/25.5.0955\\\\nCrossRef Full Text | Google Scholar\\\\nMahadevan, P., King, J. F., and Seto, D. (2009a). CGUG: in silico proteome and genome parsing tool for the determination of “core” and unique genes in the analysis of genomes up to ca. 1.9 Mb. BMC Res. Notes 2:168. doi: 10.1186/1756-0500-2-168\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMahadevan, P., King, J. F., and Seto, D. (2009b). Data mining pathogen genomes using geneorder and coregenes and CGUG: gene order, synteny and in silico proteomes. Int. J. Comput. Biol. Drug Des. 2, 100–114. doi: 10.1504/IJCBDD.2009.027586\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMarchlerbauer, A., Lu, S., Anderson, J. B., Chitsaz, F., Derbyshire, M. K., Deweesescott, C., et al. (2011). CDD: a conserved domain database for the functional annotation of proteins. Nucleic Acids Res. 39, D225–D229. doi: 10.1093/nar/gkq1189\\\\nCrossRef Full Text | Google Scholar\\\\nMerabishvili, M., Pirnay, J.-P., Verbeken, G., Chanishvili, N., Tediashvili, M., Lashkhi, N., et al. (2009). Quality-controlled small-scale production of a well-defined bacteriophage cocktail for use in human clinical trials. PLoS ONE 4:e4944. doi: 10.1371/journal.pone.0004944\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nOhmetakagi, M., and Shinshi, H. (1995). Ethylene-inducible DNA binding proteins that interact with an ethylene-responsive element. Plant Cell 7, 173–182. doi: 10.1105/tpc.7.2.173\\\\nCrossRef Full Text | Google Scholar\\\\nPeleg, A. Y., and Jara, S. D. (2009). Galleria mellonella as a model system to study Acinetobacter baumannii pathogenesis and therapeutics. Antimicrob. Agents Chemother. 53, 2605–2609. doi: 10.1128/AAC.01533-08\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPeng, F., Mi, Z., Huang, Y., Yuan, X., Niu, W., Wang, Y., et al. (2014). Characterization, sequencing and comparative genomic analysis of vB_AbaM-IME-AB2, a novel lytic bacteriophage that infects multidrug-resistant Acinetobacter baumannii clinical isolates. BMC Microbiol. 14:181. doi: 10.1186/1471-2180-14-181\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPerez, F., Hujer, A. M., Hujer, K. M., Decker, B. K., Rather, P. N., and Bonomo, R. A. (2007). Global challenge of multidrug-resistant Acinetobacter baumannii. Antimicrob. Agents Chemother. 51, 3471–3484. doi: 10.1128/AAC.01464-06\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPerez, F., Ponceterashima, R., Adams, M. D., and Bonomo, R. A. (2011). Are we closing in on an “elusive enemy”? The current status of our battle with Acinetobacter baumannii. Virulence 2, 86–90. doi: 10.4161/viru.2.2.15748\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPetersen, T. N., Brunak, S., Von, H. G., and Nielsen, H. (2011). SignalP 4.0: discriminating signal peptides from transmembrane regions. Nat. Methods 8, 785–786. doi: 10.1038/nmeth.1701\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPopova, A. V., Zhilenkov, E. L., Myakinina, V. P., Krasilnikova, V. M., and Volozhantsev, N. V. (2012). Isolation and characterization of wide host range lytic bacteriophage AP22 infecting Acinetobacter baumannii. FEMS Microbiol. Lett. 332, 40–46. doi: 10.1111/j.1574-6968.2012.02573.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nQuevillon, E., Silventoinen, V., Pillai, S., Harte, N., Mulder, N., Apweiler, R., et al. (2005). InterProScan: protein domains identifier. Nucleic Acids Res. 33, W116–W120. doi: 10.1093/nar/gki442\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRakhuba, D. V., Kolomiets, E. I., Dey, E. S., and Novik, G. I. (2010). Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell. Pol. J. Microbiol. 59, 145–155. doi: 10.1016/j.micres.2015.01.008.1.94\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSitbon, E., and Pietrokovski, S. (2003). New types of conserved sequence domains in DNA-binding regions of homing endonucleases. Trends Biochem. Sci. 28, 473–477. doi: 10.1016/S0968-0004(03)00170-1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSmith, H. W., and Huggins, M. B. (1983). Effectiveness of phages in treating experimental Escherichia coli diarrhoea in calves, piglets and lambs. J. Gen. Microbiol. 129:2659. doi: 10.1099/00221287-129-8-2659\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nStojković, E. A., and Rothman-Denes, L. B. (2007). Coliphage N4 N-acetylmuramidase defines a new family of murein hydrolases. J. Mol. Biol. 366, 406–419. doi: 10.1016/j.jmb.2006.11.028\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWorld Health Organization (2017). Global Priority List of Antibiotic-Resistant Bacteria to Guide Research, Discovery, and Development of New Antibiotics. Geneva: World Health Organization.\\\\nYoung, R. F. III., and White, R. L. (2008). Lysis of the host by bacteriophage. Encycl. Virol. 248–258. doi: 10.1016/B978-012374410-4.00752-4\\\\nCrossRef Full Text | Google Scholar\\\\nZafar, N., Mazumder, R., and Seto, D. (2002). Coregenes: a computational tool for identifying and cataloging “core” genes in a set of small genomes. BMC Bioinform. 3:12. doi: 10.1186/1471-2105-3-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02548\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02548/full\",\\n    \"loadedTime\": \"2024-01-23T08:48:41.602Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02548\",\\n    \"title\": \"Frontiers | Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community\",\\n    \"description\": \"The Formosan subterranean termite; Coptotermes formosanus is nutritionally dependent on the complex and diverse community of bacteria and protozoa in their gut. Although, there have been many studies to decipher the taxonomic and functional diversity of bacterial community in the termite gut, their bacteriophages remain unstudied. In this study we sequenced the metavirome of the termite gut to study the diversity of bacteriophages and other associated viruses. Results showed that the termites harbor a diverse bacteriophage community in their gut. Taxonomically classifiable sequences from the virome, predominantly showed similarity to the tailed bacteriophages (Caudovirales). Many predicted genes from the virome matched to bacterial prophage regions. These data are suggestive of a virome dominated by temperate bacteriophages. We predicted the genomes of seven novel Caudovirales bacteriophages from the termite gut. Three of these predicted bacteriophage genomes were found in high proportions in all the three termite colonies tested. Two bacteriophages are predicted to infect endosymbiotic bacteria of the gut protozoa. The presence of these putative bacteriophages infecting endosymbionts of the gut protozoa, suggests a quadripartite relationship between the termites their symbiotic protozoa, endosymbiotic bacteria of the protozoa and their bacteriophages. Other than Caudovirales, ss-DNA virus related genes were also present in the termite gut. We predicted the genomes of 12 novel Microviridae phages from the termite gut and seven of which possibly represent a new proposed subfamily. Circovirus like genomes were also assembled from the termite gut at lower proportions. We predicted 10 novel circovirus genomes in this study. Whether these circoviruses infect the termites remains elusive at the moment. The functional and taxonomical annotations suggest that the termites may harbor a core virome comprised of the bacteriophages infecting endosymbionts of the gut protozoa.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nBacteriophages are considered as the most abundant biological entities on earth with their total number estimated to be 1031 (Wommack and Colwell, 2000; Suttle, 2005). Bacteriophages play an important role in the ecosystem by carrying out nutrient recycling via bacterial cell lysis (Wilhelm and Suttle, 1999; Rodriguez-Brito et al., 2010; Jasna et al., 2017). In the marine ecosystems bacteriophages have been shown to be responsible for metabolic reprograming (Hurwitz et al., 2013; Hurwitz and U\\'Ren, 2016). Bacteriophages also carry out horizontal gene transfer (Eggers et al., 2016; Touchon et al., 2017) and are considered as a driving force behind bacterial genome evolution (Canchaya et al., 2004; Zeng et al., 2017). Along with the horizontal gene transfer, bacterial-phage antagonistic co-evolution is hypothesized to have a major impact on bacterial and bacteriophage diversification (Buckling and Rainey, 2002; Paterson et al., 2010). Previous studies have shown that the majority of the differences observed in the genomes of different bacterial strains of the same species in the human gut were related to restriction-modification systems, and glycosyltransferases, both of which play a key role in phage defense (Zhu et al., 2015). Bacteriophage-resistant strains have been shown to possess modifications in their surface receptors, including-antigens and outer membrane proteins (Bassford et al., 1977; Labrie et al., 2010). These surface receptors also have an important role in cell surface adhesion (Shin et al., 2005) and thus may alter the interactions of bacterial strains with their host (Lerouge and Vanderleyden, 2002). Co-evolution of bacteriophages and bacteria has been shown to alter the metabolic capacities of bacteriophage-resistant strains including the ability to utilize certain carbon sources (Middelboe et al., 2009). All these observations show that phage-bacterial interactions can have a profound effect on the ecology of the microbial community and can in turn affect the host harboring the community.\\\\nTermites rely on their symbiotic gut microbial community for cellulose digestion and acquisition of essential nutrients (reviewed in Brune, 2014). The microbial community of the higher termites (family: Termitidae) is comprised of bacterial symbionts while the lower termites (Rhinotermitidae, Mastotermitidae, Hodotermitidae, Kalotermitidae, Termopsidae, Serritermitidae) harbor flagellated protozoa in addition to the bacterial symbionts. Workers of the Formosan subterranean termite (FST), Coptotermes formosanus (Family: Rhinotermitidae), harbor three species of protozoa in the hindgut paunch that are essential for the survival of the termite colony (Eutik et al., 1978). The worker caste specializes in providing nutrition for the entire termite colony by digesting lignocellulose, and the gut protozoa enhance the endogenous cellulolytic capability of the worker gut. The protozoa also harbor endo- and ecto-symbiotic bacteria which carry out nitrogen fixation and amino acid production (Hongoh et al., 2008; Desai and Brune, 2012). The termite gut microbial community is responsible for many complex biochemical processes, providing the termite host with essential nutrients. In a previous study based on cloning of 16S rRNA gene amplicons, at least 213 different bacterial ribotypes were reported from the gut of the FST (Husseneder et al., 2010). A closely related species, Coptotermes gestroi, was estimated to harbor a bacterial community of 1,460 different species using next-generation sequencing (Do et al., 2014). The termite gut presents a unique scenario in which the host is heavily dependent on the microbial community and any changes to this community can potentially have detrimental effects on the host (Rosengaus et al., 2011). The presence of a multifaceted complex bacterial community makes the termite gut a perfect ecological niche for the presence of a diverse bacteriophage community. Like many other ecosystems the extent of this diversity and the impact of bacteriophages on the microbial community in the termite gut remains unstudied.\\\\nBacteriophages have been previously shown to be associated with the gut bacteria in higher termites (Ottesen et al., 2006). We previously isolated and sequenced the first bacteriophage from a termite hindgut. This phage, CVT22, infects Citrobacter sp. found in the gut of the FST (Tikhe et al., 2015). Two additional novel bacteriophages, Tyrion and Arya (NC_031077.1 and NC_031048.1) infecting Enterobacter sp. were also isolated and sequenced from the termite gut (manuscripts under review). In a recent study by Pramano et al. a bacteriophage infecting “Candidatus Azobacteroides pseudotrichonymphae” was discovered, making it the first bacteriophage to be associated with an obligate intracellular mutualistic endosymbiont (Pramono et al., 2017). These reports of novel bacteriophages in the termite gut indicate the presence of an unexplored bacteriophage community. The main obstacle in studying the bacteriophages from the termite gut using conventional isolation techniques is the non-culturable bacterial community (Hongoh, 2010). Most of the bacteria from the termite gut are not culturable in the lab using conventional methods, which makes the study of their bacteriophages difficult. Bacteriophages also lack universal marker genes. It has been shown that virome sequencing can be used as a powerful tool to study and characterize viral communities (Sullivan, 2015). In recent years, advances in the field of marine viromics has established a foundation for all the other virome studies (Brum and Sullivan, 2015). Many marine virome studies have identified enormous diversity of previously unknown protein sequences whose functional roles still largely remains unknown (Roux et al., 2015b; Brum et al., 2016). Metavirome sequencing also circumvents the requirement of isolation and, therefore, we chose this approach to study the bacteriophage community in the FST gut in its entirety.\\\\nIn this study we report the virome sequencing of the guts of workers from three FST colonies with the intention of exploring the unstudied bacteriophage diversity. This study is the first effort focused entirely on uncovering the bacteriophages and any other possible viruses associated with a termite species. Studying the virome of the termite gut will help us to understand the complex quadripartite relationship between the termite host, protozoa, bacteria symbionts, and associated bacteriophages.\\\\nMaterials and Methods\\\\nTermite Collection\\\\nWorkers of the FST were collected from three different colonies in New Orleans, Louisiana, USA, using untreated in-ground bait stations. Termite colonies were designated as Colony 1 (collected from City Park, on 06/21/2013), Colony 2 (collected from Hayne Blvd., lakefront on 06/18/2013), and Colony 3 (collected from Cypress St. on 06/21/2013). The distance between each of the three colonies was more than 9 km, which by far exceeds the swarming distance of the winged reproductives (alates) of the FST (Messenger and Mullins, 2005) thereby ensuring that the three colonies were founded independently by unrelated alates. All the termites were brought back to the lab in a plastic container containing a moist filter paper. Termites were processed immediately for viral community DNA extraction.\\\\nExtraction of Viral Community DNA\\\\nA total of 500 worker termites from each colony were dissected and their guts were suspended in 3 ml sterile phosphate buffered saline (pH 7.5) kept on ice. The guts were homogenized vigorously using a sterile pestle until a uniform solution was formed. The homogenate was centrifuged at 10,000 g and the supernatant was filtered twice through a 0.22 μm syringe filter. The filtered homogenate was then treated with 2.5 units per μl of RNase A and DNase I at 37°C for 6 h. The filtrate was then mixed with 200 μl of 0.5 M EDTA, and DNA was isolated using phenol-chloroform-isoamyl alcohol extraction. Concentration and quality of the extracted DNA was checked with NanoDrop® ND1000. Bacterial DNA contamination was checked via PCR using 27f and 1492r universal 16S rRNA gene primers (Lane, 1991). The initial concentrations of the extracted DNA from Colony 1, Colony 2, and Colony 3 were 4, 1.8, and 2.5 ng/μl, respectively. The extracted DNA was amplified using illustra GenomiPhi V2 DNA Amplification Kit (GE Healthcare Life sciences, Pittsburgh, USA). The amplified DNA was then ethanol precipitated and dissolved in sterile distilled water.\\\\nNext Generation Sequencing and Bioinformatics Analysis\\\\nFrom each of the three colonies, 50 ng of DNA was used to prepare the libraries using Nextera DNA Sample Preparation Kit (Illumina). The insert size of the libraries was determined by Experion Automated Electrophoresis Station (Bio-Rad). The insert size of the libraries ranged from 300 to 850 bp (average 500 bp). Individual libraries were sequenced at Molecular Research LP, Shallowater, Texas, on the Illumina MiSeq platform (Colony 1: 2 × 250 bp, Colony 2 and Colony 3: 2 × 300 bp). Quality of the DNA reads was checked using FASTQC (Andrews, 2010). DNA reads were checked for Illumina adaptor contamination and reads below the Phred score of 20 were removed using Trim Galore (Krueger, 2015). DNA reads were assembled into contigs using SPAdes Genome Assembler (Version 3.0) available on the Illumina BaseSpace platform with the default parameters using error correction and assembly mode (Bankevich et al., 2012). The contigs obtained were uploaded on the Metavir server for taxonomic assignments of the predicted ORFs using RefSeq complete viral genome protein sequence database from NCBI (released on 01/11/2017) (Roux et al., 2014). All the predicted protein coding genes were also blasted against the protein sequences from bacteriophage ProJPt-1Bp1 (Pramono et al., 2017). The contigs were also analyzed using VirSorter to separate viral contigs from cellular contaminants (Roux et al., 2015a). Contigs available on Metavir were screened for the presence of VP1, TerL, and Rep genes. For the construction of phylogenetic trees, we used full length amino acid sequences of terminase large subunit TerL (terminase_1, terminase_6, terminase_3, terminase_GPA, and terminase_1), Microviridae VP1, and Circoviridae Rep proteins. The sequences were aligned using MUSCLE (Edgar, 2004). Maximum likelihood trees were constructed using PhyML algorithm with a WAG substitution model (Guindon et al., 2010). For Microviridae subfamily assignment, full-length amino acid sequences of VP1 protein from Quaiser et al. (2015) were used to construct a phylogenetic tree. Contigs containing VP1, TerL, or Rep genes were analyzed manually for the presence of other putative viral genes. Contigs were classified as of a viral origin using the parameters described previously, with the POG13 database was used instead of POG10 (Bellas et al., 2015). Putative partial or full phage genomes were annotated manually and comparative genomic diagrams were generated using Easyfig (Sullivan et al., 2011). Putative viral genomes were visualized using CGview (Grant and Stothard, 2008) and SnapGene® (from GSL Biotech; available at snapgene.com). PHACTS analysis was carried out to determine the lifestyle and host of the putative phages (McNair et al., 2012). Family assignment of putative phage genomes was performed with VIRFAM using the ACLAME database (Lopes et al., 2014). For contig LSPY100002, RNA polymerase beta and beta\\' subunit sequences from phiKZ-like bacteriophages were used to construct phylogenetic trees (Lavysh et al., 2016). Functional annotation was carried out using the MG-RAST automated pipeline with an integrated M5NR database (Keegan et al., 2016). Orthologous genes from the three colonies were compared against each other using Orthovenn (Wang et al., 2015). All the assembled contigs have been submitted to NCBI GenBank under the accession numbers LSPY0000, LSQA0000, and LSPZ0000. Fully annotated contigs from this study are publicly available on the Metavir server under the study named “termite gut metavirome” (http://metavir-meb.univ-bpclermont.fr).\\\\nResults and Discussion\\\\nDNA Extraction, Next Generation Sequencing, and Bioinformatics Analysis for Taxonomic Assignment\\\\nWorker termites from three different termite colonies were used for this study. The abundance and diversity of microbes in the worker termites\\' gut was the biggest challenge in separating the viral community DNA from all other microbial DNA to avoid contamination. High speed centrifugation and double filtration through 0.22 μm were carried out to remove any contaminating bacterial and protozoal cells along with other bigger particles. Other free contaminating nucleic acids in the filtrate were degraded by RNase and DNase and DNA was extracted from the filtrate. No bacterial contamination was detected by PCR with universal 16S rRNA primers. At this stage the amount of DNA to be used for next generation sequencing was extremely low. In many virome studies, multiple displacement amplification (MDA) has been used to increase the amount of DNA required for sequencing (Minot et al., 2011; Santiago-Rodriguez et al., 2015; Tangherlini et al., 2016). MDA, however is known to introduce bias in the DNA samples with preferential amplification of ssDNA viruses (Kim and Bae, 2011). While the presence of viruses presented in this study is undisputed, their relative abundance may not represent the actual viral composition in the termite gut. Many new methods are being established to overcome the bias of amplification techniques and present absolute quantitate estimation of viral communities (Roux et al., 2016).\\\\nThis DNA was then submitted for next generation sequencing using Illumina MiSeq platform. The sequencing data, predicted genes by Metavir and MG-RAST, and the number of circular contigs predicted by Metavir are summarized in Table 1.\\\\nTable 1. Sequencing data, gene prediction, and number of circular contigs from viral DNA isolated from the guts of the FST workers from three different colonies.\\\\nMetavir assigned 27.13% of the genes from Colony 1, 27.85% from Colony 2 and 28.9% from Colony 3 as virus affiliated genes with an e-value of 10−5 or less. Compared to Metavir, MG-RAST classified fewer genes as viral. Percentage of genes classified as viral for Colony 1 was 9.57%, for Colony 2 was 3.8%, and for Colony 3 was 3.53%. Taxonomically MG-RAST classified between 79 and 93% of the genes as bacterial. The alpha diversity calculated by MG-RAST for Colony 1, 2, and 3 was 551, 598, and 231 species respectively. The difference between the taxonomic assignments by MG-RAST and Metavir has been previously observed in the metavirome sequencing of Antarctic soils. These viromes were predicted to be dominated by temperate bacteriophages (Zablocki et al., 2014). The taxonomic assignment of phage DNA as bacterial is likely due to the fact that reference databases classify prophages as bacterial when they are integrated into a bacterial chromosome at the time of genome sequencing. To further analyze if our data was contaminated with bacterial cellular contaminants bearing prophages, we used VirSorter tool, which can effectively separate bacterial and viral contigs (Roux et al., 2015a). VirSorter did not predict prophage sequences of category 1, but it did predict a total of 14 contigs (0.0005%) as prophage-like (category 2 and category 3). These results suggests a minimal contamination by bacterial genomes containing prophages. The number of sequenced bacterial genomes is reaching the 100,000 mark while the bacteriophage genomes still remain poorly represented in the NCBI database (around 2000 Caudovirales genomes, as of 06/01/2017). Sequencing more phage genomes is paramount to improve recognition of prophage sequences in bacterial genomes and will improve taxonomic assignments in all virome studies.\\\\nPrevious studies showed that Bacteroidetes form around 70% of the bacterial flora of the guts of C. formosanus workers (Noda et al., 2005; Shinzato et al., 2005). For this reason, we expected the Bacteroidetes phages to dominate the virome composition. The virome composition however, was different from what we expected, with Proteobacteria and Firmicutes comprising at least 40% of the identified genes (MG-RAST). This difference could be explained by the small number of sequenced Bacteroidetes bacteriophages in reference databases. Since we sequenced filtrate from the termite gut it was not surprising that viral sequences were at least 42 times enriched in our data as compared to the unfiltered metagenome of a higher termite, Nasutitermes sp. (Warnecke et al., 2007). Functional annotation also showed that phage-related sequences were at least 10 times enriched in our data. The overall enrichment in phage-related genes and PCR results indicate a successful separation of bacterial contaminants during viral DNA purification.\\\\nTaxonomic assignment using Metavir showed that dsDNA virus-related genes were dominant amongst all the viral genes despite the MDA amplification and its likely bias towards ssDNA viruses (Figure 1A). The dsDNA viral genes predominantly belonged to the tailed bacteriophages from the order Caudovirales (Figure 1B). Genes related to all three families of the order Caudovirales, i.e., Myoviridae, Siphoviridae, and Podoviridae were present in all three termite colonies (Figure 1D). Apart from the genes related to Caudovirales, genes related to large eukaryotic dsDNA viruses and other unclassified viruses were also present in all the three termite colonies. The single stranded DNA viruses (ssDNA) contributed between 1 and 10% of the total virus related genes. Within the ssDNA virus related genes between 62 and 65% belonged to Microviridae phages (Figure 1C).\\\\nFigure 1. Classification and abundance (%) of various virus types observed in the guts of the termite workers from three different colonies. (A) Percent of taxonomic assignment of all viral genes (B) Percent of taxonomic assignment of dsDNA virus related genes. (C) Percent taxonomic assignment of ssDNA virus related genes. (D) Percent taxonomic assignment of Caudovirales related genes. The data were generated using Metavir-2 server by comparing the predicted proteins to the NCBI virus protein database. Only top BLAST hits with an e-value of 10−5 or less were used (normalized for each category).\\\\nGene Based Diversity of Tailed Bacteriophages\\\\nAmong all three termite colonies, Colony 3 was the most diverse in terms of tailed bacteriophages, with genes related to 712 different bacteriophages, followed by Colony 2 (598 bacteriophages) and Colony 1 (389 bacteriophages). Genes related to a total of 960 different tailed bacteriophages were observed across all three termite colonies. This number accounts for 48.65% of all tailed bacteriophages whose complete genomes are available in the NCBI Genbank database (04/12/2017). Of these 960, Siphoviridae-related bacteriophages were the most diverse, representing 483 different bacteriophages, followed by Myoviridae (335) and Podoviridae (142). A considerable number of bacteriophages (218) were shared by all three termite colonies, with Bacillus phage AR9 (Lavysh et al., 2016) and Azobacteroides phage ProJPt-1Bp1 (Pramono et al., 2017) related genes being present in the highest proportions (Figure 2). In Colony 1, 10% of all the classified sequences belonged to Bacillus phage AR9 followed by Colony 2 (6%) and Colony 3 (4%). Azobacteroides phage ProJPt-1Bp1 related genes constituted 6.45% of all the classified genes in Colony 1, 5.61% in Colony 2 and 3.92% in Colony 3. Of the top 20 most dominant tailed bacteriophage related genes from all the three termite colonies, 12 bacteriophages had Firmicutes as their host out of which 9 bacteriophages were infecting Bacillus spp.\\\\nFigure 2. Taxonomic distribution (%) of the predominant dsDNA bacteriophage genes from the guts of termite workers from three different colonies (normalized). The data were generated using Metavir-2 server by comparing the predicted proteins to the NCBI virus protein database. Only top BLAST hits with an e-value of 10−5 or less were used.\\\\nThe analysis of phage diversity of the three termite colonies suggests the presence of a conserved virome of tailed bacteriophages in the termite gut from the same geographical area. Although there is some degree of inter-colonial variation, nearly a quarter of all tailed bacteriophages (23%) were present in all three colonies. This hypothesis needs to be tested by a larger study including more colonies from the introduced and native distribution range of the FST (Husseneder et al., 2010).\\\\nPhylogenetic Analysis of Terminase Genes\\\\nFrom all three termite colonies together, 51 unique full length terminase large subunit amino acid sequences were predicted. Out of the 51 sequences, 25 contained terminase_6 (pfam03237) domains, 12 contained terminase_3 (pfam04466.8) domains, 9 had terminase_gpa (pfam05876) domains, and 5 had terminase_1 (pfam03354) domains. Our data of terminase diversity in the termite gut are comparable to the results from virome sequencing of the deep sea, where 52 unique terminase sequences were identified (Mizuno et al., 2016). Most of the terminase sequences from the termite gut matched closely to the terminase genes from prophage regions in the bacterial genomes (Figure 3).\\\\nFigure 3. A maximum likelihood phylogenetic tree of large terminase subunit of type terminase_6. The nodes with a bootstrap value of 70% or more are indicated by a circular symbol. Sequences from the termite gut are colored purple. Bacteriophages: red. Firmicutes: orange. Spirochetes: dark blue. Gammaproteobacteria: bright green. Bacteroidetes: dark green. Actinomycetes: pink. Alphaproteobacteria: light blue. Termite Group I bacterium: sky blue. Others: black.\\\\nPhylogenetic analysis of terminase_6 showed a vast diversity with matches to prophages associated with a wide range of bacterial taxa. These results were also similar to the deep sea sequencing data, where terminase_6 domain was the most abundant type (Mizuno et al., 2016). Eight terminases from the virome formed a separate clade comprising a terminase gene from a Termite Group 1 bacterium. Two terminase genes were clustered with two separate Firmicute clades, another two were clustered with two separate Gammaproteobacteria clades, and one was clustered with a Bacteroidetes clade. Another clade was comprised entirely of bacteriophages from the Myoviridae family and three genes from termite gut phages. The clade was further subdivided in phiKZ-like bacteriophages and T4-like bacteriophages (Figure 3). Remaining genes were present in multiple clades comprised of terminases from the genomes of taxonomically diverse bacterial phyla (putative prophages).\\\\nPhylogenetically, terminase_3 genes were divided into four clearly separated clades. The Alphaproteobacteria clade and the Bacteroidetes clade each contained three terminases associated with the termite gut virome, while both the Enterobacteriaceae and the Firmicute clade contained one. Two other terminases found in the termite gut virome formed separate branches while one was grouped with Clostridium sp. CAG 306 (Figure S1). Several subunits of terminase_GPA from the termite gut virome were assigned to two distinct clades of Spirochetes and Alphaproteobacteria. The Spirochetes cluster contained five terminase_GPA subunits from the termite gut while another subunit formed a sister clade with Alphaproteobacteria (Figure S1). Most of the terminase_1 subunits were associated with Firmicutes or bacteriophages infecting the Firmicutes and one sequence was clustered with Bacteroidetes (Figure S1).\\\\nThe phylogenetic analysis of the terminase genes indicated that most of the genes matched to prophage regions in the bacterial genomes rather than sequenced bacteriophage genomes. These results along with MG-RAST annotations and VirSorter analysis suggest that most of the termite gut bacteriophages might be temperate in nature.\\\\nPhylogenetic Analysis of the Integrase Genes\\\\nThe integrase gene is used by the temperate bacteriophages to enter the lysogenic life cycle. It has been shown that prophage integrates in the host genome with a minimum impact on the overall chromosomal architecture. The bacteriophage also undergoes numerous adaptations according to the host genome in order to establish a lysogenic life cycle (Brüssow et al., 2004). It can be assumed that temperate bacteriophages are likely to infect closely related bacteria or bacteria where the overall genome architecture is conserved. Thus, phylogenetic analysis of phage integrase is likely to yield more information about its host. A total of 31 unique phage integrase sequences were identified from the three termite colonies. Phylogenetic analysis of phage integrase genes showed that sequences from the termite gut are clustered with a wide range of bacterial taxa (Figure S2). Five termite gut integrases were clustered within a Spirochete clade, six were associated with Firmicutes, another six were distributed in two clades comprised of Spirochetes and Bacteroidetes, and four were associated in two clades comprised of Bacteroidetes and Firmicutes. The remaining genes were distributed in clades comprised of diverse bacterial phyla. The results suggest the presence of temperate bacteriophages capable of infecting all the major bacteria taxa in the termite gut.\\\\nPutative Contigs of dsDNA Bacteriophage Origin\\\\nWe compared individual contigs to Refseq virus, Pfam-A, POGs13, and ACLAME database as described previously (Bellas et al., 2015). Many contigs were considered to represent putative complete or partial bacteriophage genomes. The details of all the contigs described below are listed in Supplementary Table 1.\\\\nLSPY01000004 and LSPY01000006 Represent Genomes of Bacteriophages Infecting the Symbiotic Bacteria of the Gut Protozoa\\\\nBoth LSPY01000004 and LSPY01000006 were predicted as circular contigs in termite Colony 1. Generally, circular contigs are indicative of a complete genome. Contigs mapping onto LSPY01000004 and LSPY01000006 were present in all the three termite colonies, suggesting an inter-colonial conserved distribution. Out of the 68 predicted genes in LSPY01000004, 30 produced a match in NCBI nr protein database with an e value of 10−5 or less. Of those 30, 21 genes matched only to Azobacteroides phage ProJPt-1Bp1, a bacteriophage infecting an obligatory intracellular bacterium Candidatus A. pseudotrichonymphae of the termite gut protozoa (Pramono et al., 2017). Out of the remaining genes, five genes matched to two different plasmids from Ca. A. pseudotrichonymphae and one gene matched to the genome (Hongoh et al., 2008). A total of 65 genes were predicted in contig LSPY01000006, of which 36 produced a match in the NCBI database. Out of those 36, 22 matched to phage ProJPt-1Bp1, 7 genes matched to plasmid pCFPG3 from Ca. A. pseudotrichonymphae and 1 gene matched to the genome of Ca. A. pseudotrichonymphae. There was very little similarity at nucleotide level in the genomes of LSPY01000004 and LSPY01000006 (73% match over 5% of the genome); most of the similarity was observed in the region of conserved hypothetical proteins also found in the genome of ProJPt-1Bp1. Overall, the genome arrangement of LSPY01000004, phage ProJPt-1Bp1, and LSPY01000006 was alike with areas of high similarity and synteny (Figure 4A). LSPY01000004 and LSPY01000006 showed many differences in the hypothetical proteins. Notably, LSPY01000006 harbored a gene similar to dihydrofolate reductase (DHFR) which was absent in LSPY01000004 and phage ProJPt-1Bp1. T4 bacteriophage DHFR has been predicted to play an important role in DNA metabolism and was also predicted to be a part of the virion particle (Mosher et al., 1977). However, at this moment the function of DHFR in bacteriophages remains to be studied.\\\\nFigure 4. (A) Comparative genomic analysis of LSPY01000004 and LSPY01000006 with Azobacteroides phage ProJPt-1Bp1. All three circular genomes have been rearranged so that the start codon of a conserved hypothetical protein is the first base in the sequence. (B) Comparative genomic analysis of LSPZ01000002 with Bacillus phage AR9 and Yersinia phage phi R137. All three circular genomes have been rearranged so that the start codon of the large terminase subunit is the first base in the sequence. The figures were generated using Easyfig software with tblastx. The structural genes are indicated in red, DNA metabolism related genes are indicated in blue and RNA polymerase genes are indicated in orange. All the other genes are indicated in sky blue color.\\\\nVIRFAM analysis of ProJPt-1Bp1, LSPY01000004, and LSPY01000006 predicted them to be the members of Caudovirales; however, the contigs could not be assigned to any family. The genome of phage ProJPt-1Bp1 was captured while sequencing the host bacterial genome and a t-RNA detected in the phage genome also matched to the host (Pramono et al., 2017). We were not able to identify any t-RNA in LSPY01000004 and LSPY01000006 or any other genes, which could point toward the potential host of these phage-like genomes. However, based on the similarity of genes to phage ProJPt-1Bp1 and Ca. A. pseudotrichonymphae, LSPY01000004 and LSPY01000006 most likely infect Ca. A. pseudotrichonymphae or a closely related symbiont of the obligatory gut protozoa. Also, it should be noted that phage ProJPt-1Bp1 was sequenced from Ca. A. pseudotrichonymphae from the gut of Prorhinotermes japonicus from Japan collected in 2012. It has been demonstrated for Ca. A. pseudotrichonymphae that their protozoa hosts and the termites have co-diversified (Noda et al., 2007; Ikeda-Ohtsubo and Brune, 2009; Desai et al., 2010). This presents us with a possibility of co-diversification of bacteriophages along with their hosts. The observed differences and the conserved genome architecture between phage ProJPt-1Bp1, LSPY01000004, and LSPY01000006 can be explained by the bacteriophage co-diversification hypothesis. A study of more similar phage genomes from various termite species will shed more light on the association and co-diversification of bacteriophages, protozoa, their symbiotic bacteria, and the termite host. One interesting feature that was notable in LSPY01000004, LSPY01000006, and ProJPt-1Bp1 is the apparent absence of conserved phage genes like terminase, endolysins, and phage integrases. According to the best of our knowledge, no Caudovirales bacteriophage without the presence of a terminase gene has been found prior to our study. There are two explanations for the missing genes in these phages. One is that these bacteriophages have lost their signature genes and are maintained in the bacteria as plasmids. Whether this relationship is symbiotic, as hypothesized in the case of phage ProJPt-1Bp1 (Pramono et al., 2017), remains to be explored. The other possibility is that these types of bacteriophages have a completely new method of carrying out essential functions such as DNA packaging and host lysis. Gene expression analysis would answer the question whether these bacteriophages are dormant or play any role in the bacterial metabolism, or enter an infective cycle.\\\\nIn this study, we centrifuged the gut homogenate and also filtered it through a 0.22 μm to remove all possible bacterial contamination. The free nucleic acids were also degraded with RNase and DNase. If LSPY01000004, LSPY01000006 were lysogenic or pseudolysogenic, their DNA should not have been found in the filtrate. Since all the colonies were processed separately, the presence of the same phage genomes in the filtrate from all the three termite colonies, suggests strongly that LSPY01000004 and LSPY01000006 enter an infective cycle. If ProJPt-1Bp1-like phages enter an infective cycle, studying their life cycle would aid in finding the mechanism by which intracellular phages penetrate two different types of cells, in this case the protozoa and its bacterial symbiont. It has been shown that Wolbachia phage WO can form virions and it has been suggested that phage WO can infect other Wolbachia cells from neighboring infected cells (Bordenstein et al., 2006; Kent and Bordenstein, 2010). The mechanism by which phage WO penetrates both the insect and bacterial cells remains elusive.\\\\nLSPZ01000002 Represents the Genome of a phiKZ-Like Bacteriophage\\\\nThe large contig LSPZ01000002 of 252,037 bp harboring a terminase gene was assembled from Colony 2. Further analysis of all the three termite colonies showed multiple contigs mapping against LSPZ01000002. Phylogenetic analysis of the terminase gene showed that LSPZ01000002 clustered with phiKZ-like bacteriophages (Figure 3). Of the predicted 238 genes, 71 genes had a match in the NCBI database with an e-value of 10−5 or less. Of those 71 genes, 35 genes matched to the phiKZ-like bacteriophage Bacillus phage AR9 (Lavysh et al., 2016). In all three termite colonies, Bacillus phage AR9-like genes were present in the highest abundance, suggesting that the bacteriophage, possibly along with its host has an important role in the termite gut. phiKZ-like bacteriophages encode two multi-subunit RNA polymerases (RNAPs); one of them is packed in the virion while the other is expressed in infected cells (Krylov et al., 2007; Ceyssens et al., 2014). These RNAPs are considered as the signature genes of phiKZ-like bacteriophages. LSPZ01000002 had six predicted genes encoding RNAP; three were predicted to encode the beta subunit, while three encoded the beta\\' subunit. Phylogenetic analysis of predicted virion (Figure S3) and non-virion RNAPs (data not shown) along with RNAPs from the other phiKZ-like phages produced a similar phylogenetic topology as described previously (Lavysh et al., 2016). LSPZ01000002 RNAPs were clustered with Bacillus phage AR9 and Yersinia phage phiR137.\\\\nLSPZ01000002 RNAPs were clustered with Bacillus phage AR9 and Yersinia phage phiR137. Comparative genomic analysis of LSPZ01000002, Bacillus phage AR9, and Yersinia phage phiR137 showed considerable similarities but no synteny in the genomes (Figure 4B). Lack of synteny in the genomes of closely-related phiKZ-like bacteriophages has been observed previously (Cornelissen et al., 2012; Jang et al., 2013). Most of the phiKZ-like bacteriophages are considered lytic in nature. However, some phages have been shown to be pseudolysogenic (Pletnev et al., 2010; Lavysh et al., 2016). Whether LSPZ01000002 is lytic or pseudolysogenic remains unknown. Unlike the genome of Bacillus phage AR9 which had multiple introns, RFAM BLAST did not predict any intron sequences in LSPZ01000002. One open reading frame (ORF) encoded a Group I intron-like endonuclease which showed similarities to many Group I introns from Firmicutes. One of the most important differences observed between Bacillus phage AR9 and LSPZ01000002 is the difference between ribonucleotide reductase (RNR) genes. Bacillus phage AR9 contains an operon of class I RNR genes (nrdE and nrdF). This class of RNR is dependent on oxygen and is found in organisms which can grow aerobically (Dwivedi et al., 2013). LSPZ01000002, on the other hand, contains an operon of class III RNR genes. This class is sensitive to oxygen and bacteriophages infecting strict anaerobes like Clostridium sp. harbor only this class of RNR genes (Dwivedi et al., 2013). The RNR genes found in LSPZ01000002 showed a high degree of similarity to Treponema primitia RNR genes, which is a strict anaerobic spirochete isolated from the gut of a damp wood termite Zootermopsis angusticollis (Graber et al., 2004). Many spirochetes have been previously reported from the gut of various termite species and some have been known to be ectosymbionts of the gut protozoa (Noda et al., 2003; Hongoh et al., 2007). These data suggest that LSPZ01000002 most likely infects a strict anaerobe from the termite gut, possibly a spirochete.\\\\nLSPY01000009 and LSQA01000015 Represent Partial Genomes of Lactococcus Lactis Phage 1706-Like Phages\\\\nThe three contigs LSPY01000009, LSPZ01000022, and LSQA01000015 were identified in termite Colony 1, 2, and 3 respectively. LSPY01000009 and LSPZ01000022 showed 99% similarity at genome level and hence were considered as genomes from the same phage. A moderate nucleotide level similarity was observed between LSPY01000009 and LSQA01000015 mostly at the ends of the two contigs (46% query coverage, 67% identity). All the three contigs showed a high degree of similarity to proteins from Rhodococcus phage ReqiPepy6 (Summer et al., 2011) and Arthrobacter phage Mudcat proteins. As observed in the Rhodococcus phage ReqiPepy6 and Arthrobacter phage Mudcat, LSQA01000015 lacked reverse transcriptase in the genome. However, LSPY01000009 and LSPZ01000022 harbored a reverse transcriptase enzyme belonging to Group II introns. Another important difference observed between the three termite gut contigs, Rhodococcus phage ReqiPepy6 and Arthrobacter phage Mudcat, is the presence of anaerobic ribonucleotide reductase gene (nrdD). The nrdD gene was present in all the three termite gut contigs but not in ReqiPepy6 and Mudcat. The nrdD gene has been previously observed in the genomes of bacteriophages infecting anaerobes (Dwivedi et al., 2013). No other class of RNR gene was observed in any of the contigs. Interestingly, RNR genes in LSPY01000009 and LSQA01000015 showed very little similarity to each other at amino acid level. Comparative genomic analysis showed segments of synteny in structural, DNA metabolism-related genes and segments of variable small hypothetical proteins (Figure S4). It has been shown that Rhodococcus phage ReqiPepy6 along with other closely related phages from Lactococcus lactis phage 1706-like phages have segments of genome expansion (Summer et al., 2011). In that case, closely related bacteriophages have segments of conserved genes but differ from each other in genome segments where multiple small hypothetical proteins are observed (Lavigne et al., 2009). Another characteristic of L. lactis phage 1706-like phages is the enrichment of membrane related proteins (14–23%) (Garneau et al., 2008; Summer et al., 2011).\\\\nIn all the three contigs, between 20 and 23% of the predicted proteins were found to contain at least one transmembrane domain. Based on this analysis, LSPY01000009, LSPZ01000022, and LSQA01000015 represent partial genomes of L. lactis phage 1706-like phages which most likely infects a Firmicutes bacterium.\\\\nLSPZ01000027 Represents a Full Genome of a Temperate Phage\\\\nLSPZ01000027 was identified as a circular contig in termite Colony 2. Other multiple contigs could be mapped against LSPZ01000027 from all the three termite colonies. LSPZ01000027 genome showed high level of synteny to structural genes of Clostridium phage phiCDHM1 (Hargreaves et al., 2014) and Clostridium phage phiMMP01 (Boudry et al., 2015) (Figure S4). PhiCDHM1 genome was found to harbor a cassette of bacterial quorum sensing genes (Hargreaves et al., 2014), but no such cassette was identified in LSPZ01000027. PhiCDHM1 is considered a member of phiCD119-like bacteriophages with the presence of a signature DNA replication cassette. In LSPZ01000027 this signature DNA replication cassette was not observed. Also the G+C content of LSPZ01000027 was much higher (42%) than that of phiCDHM1 and phiMMP01the (G+C content 14–29%). Similar to the genome of phiCDHM1 and phiMMP01, the genome of LSPZ01000027 has many proteins annotated as putative anti-repressor proteins. The true identity of these anti-repressor proteins remains unknown at this moment. LSPZ01000027 also had a Group II intron encoded reverse transcriptase. The presence of an integrase and multiple anti-repressor-like proteins indicate that LSPZ01000027 is most likely a temperate bacteriophage.\\\\nLSQA01000020 Represents the Genome of a Lytic Siphovirus\\\\nLSQA01000020 was identified as a linear contig in termite Colony 3 and multiple contigs from Colony 2 could be mapped against LSQA01000020. The first and the last gene of the linear contig encoded the same partial gene, indicating an almost complete circular bacteriophage genome. Phylogenetic analysis of the terminase gene from LSQA01000020 placed it in the Bacteroidetes cluster in a sister clade with Flavobacter bacteriophages. Comparative genomic analysis of the LSQA01000020 genome shows a high level of similarity to the genomes of two bacteriophages, P2559S and P2559Y, both of which infect the Bacteroidetes species Croceibacter atlanticus (Kang et al., 2012, 2016). Even though both P2559S and P2559Y are lytic Siphoviruses infecting the same species they show similarity only in the structural module of the genome (Kang et al., 2016). LSQA01000020 showed similarity in the structural module to both the phages and similarity to some extent in the replication module to P2559Y (Figure S4). LSQA01000020 contained thymidylate synthase and asparagine synthase genes, which were absent in P2559S and P2559Y. Based on the similarity to P2559S and P2559Y and phylogenetic placement of the terminase gene, LSQA01000020 most likely infects a Bacteroidetes species.\\\\nContigs similar to the first bacteriophage (CVT22) isolated from the termite gut were also observed in Colony 3 suggesting some association with the termite gut (Tikhe et al., 2015). It has been suggested that CVT22 may represent a founding member of a new cluster of lytic bacteriophages (Casjens and Grose, 2016) and the termite gut might represent a niche of diversity of CVT22-like bacteriophages.\\\\nMany contigs observed in our virome, harbored signature phage genes. Large contigs of 200,000 bp or more were observed in all the three termite colonies. However, due to the presence of a large proportion of previously unknown genes, the origin of many contigs remains unclear. As more phage genomes will be sequenced to populate reference databases we believe that many more phage genomes will be uncovered from the termite gut.\\\\nLarge viruses like the Chlorella virus infecting the algal symbionts of the protozoa have been previously identified and sequenced (Yamada et al., 2006). In our study, we found ORFs similar to large eukaryotic dsDNA viruses. However, on further analysis we could not confidently assign the origin of these genes as viral. Further research is required to study the presence of large eukaryotic ds-DNA viruses from the termite gut.\\\\nTermite Gut Microviruses Represent a Putative New Sub-family\\\\nAs compared to double-stranded DNA viruses, single-stranded DNA (ssDNA) viruses were present in a lower abundance (1–10%). Colony 1 had the most diverse community of ssDNA viruses, with genes from 38 different types of ssDNA viruses followed by Colony 2 (26 types) and Colony 3 (15 types). Most of the dominant ssDNA viruses were conserved in all the three colonies along with some inter-colonial differences (Figure S5).\\\\nPhylogenetic analysis of VP1 major capsid gene indicated a diverse population of Microviridae in the termite gut. We were able to construct 12 novel complete genomes of Microviridae from all three termite colonies. Phylogenetically, VP1 from LSPY01000110 was placed in a cluster with Dysgonomonas macrotermitis, a bacterium of the phylum Bacteroidetes, which was previously isolated from the gut of a higher termite (Macrotermes barneyi) (Yang et al., 2014). Bacteria of this genus are also known to be a part of the gut community of the FST (Husseneder et al., 2009, 2010). LSPY01000110 showed synteny to a contig from the genome of a D. macrotermitis (Figure 5). It has been previously reported that Microviridae bacteriophage can undergo a temperate life cycle in Bacteroidetes (Krupovic and Forterre, 2011). The D. macrotermitis prophage-like sequence and LSPY01000110 showed the same gene order (VP1-ORF2-VP2-VP4) followed by five ORFs encoding hypothetical proteins in D. macrotermitis and four ORFs in case of LSPY01000110. The hypothetical proteins showed no similarity to each other. Although LSPY01000110 and D. macrotermitis prophage were clustered with the Alpavirinae subfamily, comparative genomics showed very little similarity between the two clusters.\\\\nFigure 5. A maximum likelihood phylogenetic tree of Microviridae VP1 from the guts of workers from three termite colonies along with the comparative genomic analysis of closely related Microviruses. Nodes with a bootstrap score of more than 70% are indicated by gray circles. For this figure, only VP1 genes from putative full Microviridae genomes were used. VP1 is colored red, VP2: green, VP4: blue. An extra ORF found after the VP1gene of Sukshmaviriane is colored pink.\\\\nPhylogenetic analysis of the VP1 sequences of LSPZ01000262 and LSPY01000148 placed them in the Gokushovirinae subfamily (Figure 5). The genome arrangements of LSPZ01000262 and LSPY01000148 were different from previously described Gokushovirinae sequences (Quaiser et al., 2015). No clear distinguishable VP5 sequences were observed in LSPZ01000262 and LSPY01000148.\\\\nVP1 from LSPY01000144 and LSPY01000166 could not be confidently placed in any cluster. VP1 from seven contigs formed a completely separate cluster from all of the other Microviridae. The genome arrangement also showed a conserved order (VP1-ORF1-VP2-VP4-ORF2). In this cluster, the ORF present after theVP1 gene encoded for a hypothetical protein. This protein did not match any known protein in the NCBI database but showed a high similarity to proteins in the cluster. The protein encoded by the ORF after VP4 from some contigs from the cluster showed similarity to Gokushovirinae VP5. Based on the genome arrangement and the VP1 phylogeny, we propose a new subfamily Sukshmavirinae (Sukshma is the Sanskrit word for “small”) for the sequences observed in the termite gut virome.\\\\nA Diverse Community of Circoviruses Is Present in the Termite Gut\\\\nCircoviruses are small ssDNA viruses known to infect a number of higher eukaryotes (Todd et al., 2001). In recent years Circovirus-like genomes have been identified to be associated with a variety of animals including many insects (Rosario et al., 2011; Garigliany et al., 2015). The exact role of Circoviruses associated with various animals is currently not understood completely.\\\\nFrom all three termite colonies, 10 novel Circoviridae-like genomes were assembled. The genome size ranged between 1,388 and 5,851 bp. All the genomes encoded the Circoviridae Rep protein which is considered as the signature gene of the family. Phylogenetic analysis of the Rep gene showed two distinct groups, one belonging to the Cycloviruses and the other to the Circoviruses (Figure 6). There was no correlation between the host of these viruses and the phylogenetic placement of the Rep proteins. Termite gut Rep proteins were distributed all over the phylogenetic tree with only one sequence clustered in the Cyclovirus group.\\\\nFigure 6. Unrooted maximum likelihood phylogenetic tree of Circoviridae replication initiation protein. The hosts of the Circoviruses are displayed in a picture next to the sequence. Environmental Circoviruses are shown in green. Insect related Circoviruses are shown in red.\\\\nSo far, there have been very limited data on viruses capable of infecting termites (Al Fazairy and Hassan, 1988). It has been suggested that a virus infecting termites would be an ideal candidate for biological control (Chouvenc et al., 2011). The origin of the Circoviruses found in the termite gut remains unclear. Whether these viruses are environmental and were ingested by the termites or they actually infect the termites is an area of future research.\\\\nFunctional Annotation Shows a High Degree of Conservation in the Termite Gut Virome\\\\nMG-RAST functional analysis assigned functions to 14.5% of the predicted proteins from Colony 1, 27.62% from Colony 2 and 33.17% from Colony 3. These results show that the termite virome is largely comprised of proteins whose function still remains unknown. Functional annotation of the assigned genes indicated that phage, prophages, transposable elements, and plasmids were present in the highest abundance in all the three termite colonies (Figure S6).\\\\nThe presence of conserved sequences was expected because most of the bacteriophages carry out the same core basic functions such as replication, lysis, packaging, and host integration during their life cycle. To study the conservation of functional genes in the gut virome, we analyzed the orthologous gene clusters present in the termite colonies. Orthologous genes may represent a difference in the sequence but the function is generally conserved. The orthovenn analysis of the gut virome from all the three termite colonies showed that all the 53,000 protein sequences formed a total of 9,625 clusters. Of these clusters, 8,317 orthologous clusters contained genes from at least two of the three termite colonies. A total of 3,823 clusters comprised of 12,295 protein sequences were shared by all three colonies. The data suggest that the FST gut virome we sequenced, has a core set of functional genes that is conserved between all the three termite colonies.\\\\nTermites Possibly Harbor a Core Virome Comprised of Bacteriophages Infecting Protozoal Endosymbiotic Bacteria\\\\nThe FST is native to China and was introduced into the USA (Hawaii) over a century ago (Husseneder et al., 2012). The first recorded report of FSTs on the US mainland is from 1957 (Chambers et al., 1988). A previous study showed that the bacterial composition of the FST gut between the native (China) and the introduced (New Orleans, USA) populations did not change significantly (Husseneder et al., 2010). The obligatory symbionts of the gut protozoa form the core of the conserved bacterial community of the FST gut (Noda et al., 2005). These symbionts are found exclusively in the termite gut. In this study we reported two bacteriophages LSPY01000004 and LSPY01000006, infecting the obligatory endosymbiotic bacteria. The similarity between LSPY01000004, LSPY01000006, and phage ProJPt-1Bp1 is remarkable despite the fact that they were sequenced independently at different time points in geographically well-separated areas (USA and Japan) and from two different species of termites. It has been speculated that phage related genes might be ubiquitous in the termite gut (Tadmor et al., 2011). In that study, closely related bacteriophage genes were found in different species of termites collected almost 20 years apart in two different geographic locations (Tadmor et al., 2011). Interestingly, we have also found LSPY01000004 genes in other FST gut samples collected from additional colonies in New Orleans at different time points (unpublished data).\\\\nA core virome has been previously observed in human saliva and lower respiratory tract (Willner et al., 2009; Pride et al., 2012). In a study of human virome, bacteriophage crAssphage related sequences were found globally distributed in many human fecal metagenome samples (Dutilh et al., 2014). Based on the taxonomic and functional overlap of phages among the three different termite colonies evidenced by shared phages and gene functions, we hypothesize that termites also harbor a highly conserved core virome. We are aware that the main limitation of this study is a small sample size from the same geographic area. It cannot be ruled out that similar environmental conditions may be responsible for observed similarities in the virome of the three termite colonies. It should be noted that at this point the “termite core virome” is a hypothesis and including more termite samples from various geographical areas will help to corroborate or refute our hypothesis.\\\\nPhylogenetic analysis of the terminase and integrase genes indicated that termite gut viruses show a high degree of similarity to prophage genes from diverse bacteria. Termites are highly dependent on their gut bacteria to complement their own metabolism, and changes in the bacterial population have been shown to negatively affect the termite host (Rosengaus et al., 2011; Brune, 2014). The impact of bacteriophage pressure on the bacteria is not only known to alter species composition but also their metabolic processes (Middelboe et al., 2009). It would be essential for the termite and the gut bacteria to maintain a functionally conserved set of biochemical pathways despite the presence of bacteriophage pressure. Hence, it would be advantageous to the termite host and the symbiotic gut community upon which it relies, if the bacteriophage adopts a temperate life style rather than lytic one. In turn, the phage replicates with a thriving bacterial host population and is transferred throughout the termite colony via social interactions. Manipulating the bacterial host abundance and studying the bacteriophage dynamics would further help us understand the lytic-lysogenic life style changes in the termite gut.\\\\nPrevious studies have shown that temperate bacteriophages can protect the bacterial host from other bacteriophages by superinfection immunity (Bondy-Denomy and Davidson, 2014). Termites are soil dwelling and the gut bacteria must be encountering a number of environmental bacteriophages. It would be interesting to study whether termite gut bacteriophages prevent the gut bacteria from environmental bacteriophages via superinfection immunity. In the future, we intend to develop termites as a model system to study the complicated quadripartite relationship between bacteria, bacteriophages, gut protozoa, and the termites themselves.\\\\nAuthor Contributions\\\\nCT designed and conducted the experiments and wrote the manuscript, CH designed the experiments, supervised the project, and edited the manuscript.\\\\nFunding\\\\nThe study was funded by the Louisiana Agricultural Experiment Station.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nWe would like to thank Dr. Carrie Cottone, Eric Guidry, and Barry Yokum, New Orleans Mosquito and Termite Control Board for providing the termites used in the study. We also would like to thank Dr. Chris Gissendanner, University of Louisiana at Monroe, for valuable comments on the manuscript.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2017.02548/full#supplementary-material\\\\nSupplementary Table 1. Table describing the characteristics of predicted tailed bacteriophage genomes. *LSPY01000009, LSPZ01000022 and LSQA01000015 represent similar bacteriophages. Only data for LSPY01000009 is shown here.\\\\nFigure S1. A maximum likelihood phylogenetic tree of large terminase subunit of type terminase_3 (A), terminase_GPA (B), terminase_1 (C). The nodes with a bootstrap value of 70% or more are indicated by a circular symbol. Sequences from the termite gut are colored purple. Bacteriophages: red. Firmicutes: orange. Spirochetes: dark blue. Gammaproteobacteria: bright green. Bacteroidetes: dark green. Alphaproteobacteria: gray. Others: black.\\\\nFigure S2. A maximum likelihood phylogenetic tree of phage integrase genes. The nodes with a bootstrap value of 70% or more are indicated by a circular symbol. Sequences from the termite gut are colored purple Firmicutes: orange. Actinobacteria: red. Spirochetes: dark blue. Gammaproteobacteria: bright green. Bacteroidetes: dark green. Planctomycetes: sky blue. Others: black.\\\\nFigure S3. Unrooted maximum likelihood phylogenetic tree of RNAP beta subunit of virion and non-virion subunits from phiKZ-like bacteriophages. RNAPs from LSPZ01000002 are shown in purple. RNAP from Thermus thermophilus is used as an outgroup.\\\\nFigure S4. (A) Comparative genomic analysis of LSPY01000009 and LSQA01000015 with Arthrobacter phage Mudcat and Rhodococcus phage ReqiPepy6. (B) Comparative genomic analysis of LSPZ01000027 with Clostridium phage cdhm1 and Clostridium phage phimmp01. (C) Comparative genomic analysis of LSQA01000020 with Croceibacter phage P2559S and Croceibacter phage P2559Y. All the genomes have been rearranged so that the start codon of the large terminase subunit is the first base in the sequence. The figures were generated using Easyfig software with tblastx. The structural genes are indicated in red color, DNA metabolism related genes in blue, cell lysis in green, virulence associated protein in pink, yellow anti-repressor, reverse transcriptase in purple. All the other genes are indicated in sky blue color.\\\\nFigure S5. Taxonomic distribution (%) of the predominant ssDNA bacteriophage genes (normalized) from the guts of the termites from three different colonies. The data were generated using Metavir-2 server by comparing the predicted proteins to NCBI virus protein database. Top BLAST hits with an e-value of 10−5 or less were used.\\\\nFigure S6. Normalized percent distribution of the functional categories of predominant genes from the guts of the termites from three different colonies. The data were generated using MG-RAST server by comparing the predicted proteins to MD5nr database.\\\\nReferences\\\\nAl Fazairy, A. A., and Hassan, F. (1988). Infection of termites by Spodoptera littoralis nuclear polyhedrosis virus. Int. J. Trop. Insect Sci. 9, 37–39. doi: 10.1017/S1742758400009991\\\\nCrossRef Full Text | Google Scholar\\\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455–477. doi: 10.1089/cmb.2012.0021\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBassford, P., Diedrich, D. L., Schnaitman, C. L., and Reeves, P. (1977). Outer membrane proteins of Escherichia coli. VI. Protein alteration in bacteriophage-resistant mutants. J. Bacteriol. 131, 608–622.\\\\nPubMed Abstract | Google Scholar\\\\nBellas, C. M., Anesio, A. M., and Barker, G. (2015). Analysis of virus genomes from glacial environments reveals novel virus groups with unusual host interactions. Front. Microbiol. 6:656. doi: 10.3389/fmicb.2015.00656\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBondy-Denomy, J., and Davidson, A. R. (2014). When a virus is not a parasite: the beneficial effects of prophages on bacterial fitness. J. Microbiol. 52, 235–242. doi: 10.1007/s12275-014-4083-3\\\\nCrossRef Full Text | Google Scholar\\\\nBordenstein, S. R., Marshall, M. L., Fry, A. J., Kim, U., and Wernegreen, J. J. (2006). The tripartite associations between bacteriophage, Wolbachia, and arthropods. PLoS Pathog. 2, 384–393. doi: 10.1371/journal.ppat.0020043\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBoudry, P., Semenova, E., Monot, M., Datsenko, K. A., Lopatina, A., Sekulovic, O., et al. (2015). Function of the CRISPR-cas system of the human pathogen Clostridium difficile. MBio 6, e01112–01115. doi: 10.1128/mBio.01112-15\\\\nCrossRef Full Text | Google Scholar\\\\nBrum, J. R., Ignacio-Espinoza, J. C., Kim, E.-H., Trubl, G., Jones, R. M., Roux, S., et al. (2016). Illuminating structural proteins in viral “dark matter” with metaproteomics. Proc. Natl. Acad. Sci. U.S.A. 113, 2436–2441. doi: 10.1073/pnas.1525139113\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBrum, J. R., and Sullivan, M. B. (2015). Rising to the challenge: accelerated pace of discovery transforms marine virology. Nat. Rev. Microbiol. 13, 147–159. doi: 10.1038/nrmicro3404\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBrüssow, H., Canchaya, C., and Hardt, W.-D. (2004). Phages and the evolution of bacterial pathogens: from genomic rearrangements to lysogenic conversion. Microbiol. Mol. Biol. Rev. 68, 560–602. doi: 10.1128/MMBR.68.3.560-602.2004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCasjens, S. R., and Grose, J. H. (2016). Contributions of P2-and P22-like prophages to understanding the enormous diversity and abundance of tailed bacteriophages. Virology 496, 255–276. doi: 10.1016/j.virol.2016.05.022\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCeyssens, P.-J., Minakhin, L., Van Den Bossche, A., Yakunina, M., Klimuk, E., Blasdel, B., et al. (2014). Development of giant bacteriophage φKZ is independent of the host transcription apparatus. J. Virol. 88, 10501–10510. doi: 10.1128/JVI.01347-14\\\\nCrossRef Full Text | Google Scholar\\\\nChambers, D., Zungoli, P., and Hill, Jr. H. (1988). Distribution and habitats of the Formosan subterranean termite (Isoptera: Rhinotermitidae) in South Carolina. J. Econ. Entomol. 81, 1611–1619. doi: 10.1093/jee/81.6.1611\\\\nCrossRef Full Text | Google Scholar\\\\nChouvenc, T., Su, N.-Y., and Grace, J. K. (2011). Fifty years of attempted biological control of termites–analysis of a failure. Biol. Control 59, 69–82. doi: 10.1016/j.biocontrol.2011.06.015\\\\nCrossRef Full Text | Google Scholar\\\\nCornelissen, A., Hardies, S. C., Shaburova, O. V., Krylov, V. N., Mattheus, W., Kropinski, A. M., et al. (2012). Complete genome sequence of the giant virus OBP and comparative genome analysis of the diverse φKZ-related phages. J. Virol. 86, 1844–1852. doi: 10.1128/JVI.06330-11\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDesai, M. S., and Brune, A. (2012). Bacteroidales ectosymbionts of gut flagellates shape the nitrogen-fixing community in dry-wood termites. ISME J. 6, 1302–1313. doi: 10.1038/ismej.2011.194\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDesai, M. S., Strassert, J. F., Meuser, K., Hertel, H., Ikeda-Ohtsubo, W., Radek, R., et al. (2010). Strict cospeciation of devescovinid flagellates and Bacteroidales ectosymbionts in the gut of dry-wood termites (Kalotermitidae). Environ. Microbiol. 12, 2120–2132. doi: 10.1111/j.1462-2920.2009.02080.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDo, T. H., Nguyen, T. T., Nguyen, T. N., Le, Q. G., Nguyen, C., Kimura, K., et al. (2014). Mining biomass-degrading genes through Illumina-based de novo sequencing and metagenomic analysis of free-living bacteria in the gut of the lower termite Coptotermes gestroi harvested in Vietnam. J. Biosci. Bioeng. 118, 665–671. doi: 10.1016/j.jbiosc.2014.05.010\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDutilh, B. E., Cassman, N., Mcnair, K., Sanchez, S. E., Silva, G. G., Boling, L., et al. (2014). A highly abundant bacteriophage discovered in the unknown sequences of human faecal metagenomes. Nat. Commun. 5:4498. doi: 10.1038/ncomms5498\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDwivedi, B., Xue, B., Lundin, D., Edwards, R. A., and Breitbart, M. (2013). A bioinformatic analysis of ribonucleotide reductase genes in phage genomes and metagenomes. BMC Evol. Biol. 13:33. doi: 10.1186/1471-2148-13-33\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nEggers, C. H., Gray, C. M., Preisig, A. M., Glenn, D. M., Pereira, J., Ayers, R. W., et al. (2016). Phage-mediated horizontal gene transfer of both prophage and heterologous DNA by φBB-1, a bacteriophage of Borrelia burgdorferi. Pathog. Dis. 74:ftw107. doi: 10.1093/femspd/ftw107\\\\nCrossRef Full Text | Google Scholar\\\\nEutik, M., Veivers, P., and O\\'Brien, R. and Slaytor, M. (1978). Dependence oí the higher termite, Nasutitermes exitiosus and the lower termite, Coptotermes lacteus on their gut flora. J. Insect Physiol. 24, 363–368. doi: 10.1016/0022-1910(78)90076-8\\\\nCrossRef Full Text | Google Scholar\\\\nGarigliany, M.-M., Börstler, J., Jöst, H., Badusche, M., Desmecht, D., Schmidt-Chanasit, J., et al. (2015). Characterization of a novel circo-like virus in Aedes vexans mosquitoes from Germany: evidence for a new genus within the family Circoviridae. J. Gen. Virol. 96, 915–920. doi: 10.1099/vir.0.000036\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGarneau, J. E., Tremblay, D. M., and Moineau, S. (2008). Characterization of 1706, a virulent phage from Lactococcus lactis with similarities to prophages from other Firmicutes. Virology 373, 298–309. doi: 10.1016/j.virol.2007.12.002\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGraber, J. R., Leadbetter, J. R., and Breznak, J. A. (2004). Description of Treponema azotonutricium sp. nov. and Treponema primitia sp. nov., the first spirochetes isolated from termite guts. Appl. Environ. Microbiol. 70, 1315–1320. doi: 10.1128/AEM.70.3.1315-1320.2004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGuindon, S., Dufayard, J.-F., Lefort, V., Anisimova, M., Hordijk, W., Gascuel, O., et al. (2010). New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Syst. Biol. 59, 307–321. doi: 10.1093/sysbio/syq010\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHargreaves, K. R., Kropinski, A. M., and Clokie, M. R. (2014). What does the talking?: quorum sensing signalling genes discovered in a bacteriophage genome. PLoS ONE 9:e85131. doi: 10.1371/journal.pone.0085131\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHongoh, Y., Sato, T., Noda, S., Ui, S., Kudo, T., and Ohkuma, M., et al. (2007). Candidatus Symbiothrix dinenymphae: bristle-like Bacteroidales ectosymbionts of termite gut protists. Environ. Microbiol. 9, 2631–2635. doi: 10.1111/j.1462-2920.2007.01365.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHongoh, Y., Sharma, V. K., Prakash, T., Noda, S., Toh, H., Taylor, T. D., et al. (2008). Genome of an endosymbiont coupling N2 fixation to cellulolysis within protist cells in termite gut. Science 322, 1108–1109. doi: 10.1126/science.1165578\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHusseneder, C., Berestecky, J., and Grace, J. (2009). Changes in composition of culturable bacteria community in the gut of the Formosan subterranean termite depending on rearing conditions of the host. Ann. Entomol. Soc. Am. 102, 498–507. doi: 10.1603/008.102.0321\\\\nCrossRef Full Text | Google Scholar\\\\nHusseneder, C., Ho, H.-Y., and Blackwell, M. (2010). Comparison of the bacterial symbiont composition of the Formosan subterranean termite from its native and introduced range. Open Microbiol. J. 4, 53–66. doi: 10.2174/1874285801004010053\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHusseneder, C., Simms, D. M., Delatte, J. R., Wang, C., Grace, J. K., and Vargo, E. L., et al. (2012). Genetic diversity and colony breeding structure in native and introduced ranges of the Formosan subterranean termite, Coptotermes formosanus. Biol. Invas. 14, 419–437. doi: 10.1007/s10530-011-0087-7\\\\nCrossRef Full Text | Google Scholar\\\\nIkeda-Ohtsubo, W., and Brune, A. (2009). Cospeciation of termite gut flagellates and their bacterial endosymbionts: Trichonympha species and “Candidatus Endomicrobium trichonymphae”. Mol. Ecol. 18, 332–342. doi: 10.1111/j.1365-294X.2008.04029.x\\\\nCrossRef Full Text | Google Scholar\\\\nJang, H. B., Fagutao, F. F., Nho, S. W., Park, S. B., Cha, I. S., Yu, J. E., et al. (2013). Phylogenomic network and comparative genomics reveal a diverged member of the φkz-related group, marine Vibrio phage φJM-2012. J. Virol. 87, 12866–12878. doi: 10.1128/JVI.02656-13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJasna, V., Parvathi, A., Pradeep Ram, A. S., Balachandran, K. K., Madhu, N. V., Nair, M., et al. (2017). Viral-induced mortality of prokaryotes in a tropical monsoonal estuary. Front. Microbiol. 8:895. doi: 10.3389/fmicb.2017.00895\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKang, I., Kang, D., and Cho, J.-C. (2016). Complete genome sequence of bacteriophage P2559Y, a marine phage that infects Croceibacter atlanticus HTCC2559. Mar. Genomics 29, 35–38. doi: 10.1016/j.margen.2016.07.001\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKeegan, K. P., Glass, E. M., and Meyer, F. (2016). MG-RAST, a metagenomics service for analysis of microbial community structure and function. Methods Mol. Biol. 1399, 207–233. doi: 10.1007/978-1-4939-3369-3_13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKim, K.-H., and Bae, J.-W. (2011). Amplification methods bias metagenomic libraries of uncultured single-stranded and double-stranded DNA viruses. Appl. Environ. Microbiol. 77, 7663–7668. doi: 10.1128/AEM.00289-11\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKrueger, F. (2015). Trim Galore. A Wrapper Tool Around Cutadapt and FastQC to Consistently Apply Quality and Adapter Trimming to FastQ Files. Available online at: https://github.com/FelixKrueger/TrimGalore\\\\nKrupovic, M., and Forterre, P. (2011). Microviridae goes temperate: microvirus-related proviruses reside in the genomes of Bacteroidetes. PLoS ONE 6:e19893. doi: 10.1371/journal.pone.0019893\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKrylov, V., Cruz, D. D., Hertveldt, K., and Ackermann, H.-W. (2007). “ϕKZ-like viruses,” a proposed new genus of myovirus bacteriophages. Arch. Virol. 152, 1955–1959. doi: 10.1007/s00705-007-1037-7\\\\nCrossRef Full Text | Google Scholar\\\\nLane, D. J. (1991). “16S/23S rRNA sequencing,” in Nucleic Acid Techniques in Bacterial Systematics, ed E. S. A. M. Goodfellow (New York, NY: John Wiley and Sons), 115–175.\\\\nGoogle Scholar\\\\nLavigne, R., Darius, P., Summer, E. J., Seto, D., Mahadevan, P., Nilsson, A. S., et al. (2009). Classification of Myoviridae bacteriophages using protein sequence similarity. BMC Microbiol. 9:224. doi: 10.1186/1471-2180-9-224\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLavysh, D., Sokolova, M., Minakhin, L., Yakunina, M., Artamonova, T., Kozyavkin, S., et al. (2016). The genome of AR9, a giant transducing Bacillus phage encoding two multisubunit RNA polymerases. Virology 495, 185–196. doi: 10.1016/j.virol.2016.04.030\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLerouge, I., and Vanderleyden, J. (2002). O-antigen structural variation: mechanisms and possible roles in animal/plant–microbe interactions. FEMS Microbiol. Rev. 26, 17–47. doi: 10.1111/j.1574-6976.2002.tb00597.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLopes, A., Tavares, P., Petit, M.-A., Guérois, R., and Zinn-Justin, S. (2014). Automated classification of tailed bacteriophages according to their neck organization. BMC Genomics 15:1027. doi: 10.1186/1471-2164-15-1027\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMcNair, K., Bailey, B. A., and Edwards, R. A. (2012). PHACTS, a computational approach to classifying the lifestyle of phages. Bioinformatics 28, 614–618. doi: 10.1093/bioinformatics/bts014\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMessenger, M. T., and Mullins, A. J. (2005). New flight distance recorded for Coptotermes formosanus (Isoptera: Rhinotermitidae). Fla. Entomol. 88, 99–100. doi: 10.1653/0015-4040(2005)088[0099:NFDRFC]2.0.CO;2\\\\nCrossRef Full Text | Google Scholar\\\\nMiddelboe, M., Holmfeldt, K., Riemann, L., Nybroe, O., and Haaber, J. (2009). Bacteriophages drive strain diversification in a marine Flavobacterium: implications for phage resistance and physiological properties. Environ. Microbiol. 11, 1971–1982. doi: 10.1111/j.1462-2920.2009.01920.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMinot, S., Sinha, R., Chen, J., Li, H., Keilbaugh, S. A., Wu, G. D., et al. (2011). The human gut virome: inter-individual variation and dynamic response to diet. Genome Res. 21, 1616–1625. doi: 10.1101/gr.122705.111\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMizuno, C. M., Ghai, R., Saghaï, A., López-García, P., and Rodriguez-Valera, F. (2016). Genomes of abundant and widespread viruses from the deep ocean. MBio 7, e00805–00816. doi: 10.1128/mBio.00805-16\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMosher, R. A., Direnzo, A. B., and Mathews, C. K. (1977). Bacteriophage T4 virion dihydrofolate reductase: approaches to quantitation and assessment of function. J. Virol. 23, 645–658.\\\\nPubMed Abstract | Google Scholar\\\\nNoda, S., Iida, T., Kitade, O., Nakajima, H., Kudo, T., and Ohkuma, M., et al. (2005). Endosymbiotic Bacteroidales bacteria of the flagellated protist Pseudotrichonympha grassii in the gut of the termite Coptotermes formosanus. Appl. Environ. Microbiol. 71, 8811–8817. doi: 10.1128/AEM.71.12.8811-8817.2005\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nNoda, S., Kitade, O., Inoue, T., Kawai, M., Kanuka, M., Hiroshima, K., et al. (2007). Cospeciation in the triplex symbiosis of termite gut protists (Pseudotrichonympha spp.), their hosts, and their bacterial endosymbionts. Mol. Ecol. 16, 1257–1266. doi: 10.1111/j.1365-294X.2006.03219.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nNoda, S., Ohkuma, M., Yamada, A., Hongoh, Y., and Kudo, T. (2003). Phylogenetic position and in situ identification of ectosymbiotic spirochetes on protists in the termite gut. Appl. Environ. Microbiol. 69, 625–633. doi: 10.1128/AEM.69.1.625-633.2003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nOttesen, E. A., Hong, J. W., Quake, S. R., and Leadbetter, J. R. (2006). Microfluidic digital PCR enables multigene analysis of individual environmental bacteria. Science 314, 1464–1467. doi: 10.1126/science.1131370\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPaterson, S., Vogwill, T., Buckling, A., Benmayor, R., Spiers, A. J., Thomson, N. R., et al. (2010). Antagonistic coevolution accelerates molecular evolution. Nature 464, 275–278. doi: 10.1038/nature08798\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPletnev, E. A., Krylov, S. V., Shaburova, O. V., Burkal\\'tseva, M. V., Miroshnikov, K. A., Krylov, V. N., et al. (2010). Pseudolysogeny of Pseudomonas aeruginosa bacteria infected with phiKZ-like bacteriophages. Genetika 46, 26–32. Available online at: http://europepmc.org/abstract/med/20198876\\\\nPubMed Abstract | Google Scholar\\\\nPramono, A. K., Kuwahara, H., Itoh, T., Toyoda, A., Yamada, A., and Hongoh, Y. (2017). Discovery and complete genome sequence of a bacteriophage from an obligate intracellular symbiont of a cellulolytic protist in the termite gut. Microbes Environ. 32, 112–117. doi: 10.1264/jsme2.ME16175\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPride, D. T., Salzman, J., Haynes, M., Rohwer, F., Davis-Long, C., White, R. A. III., et al. (2012). Evidence of a robust resident bacteriophage population revealed through analysis of the human salivary virome. ISME J. 6, 915–926. doi: 10.1038/ismej.2011.169\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nQuaiser, A., Dufresne, A., Ballaud, F., Roux, S., Zivanovic, Y., Colombet, J., et al. (2015). Diversity and comparative genomics of Microviridae in Sphagnum-dominated peatlands. Front. Microbiol. 6:375. doi: 10.3389/fmicb.2015.00375\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRodriguez-Brito, B., Li, L., Wegley, L., Furlan, M., Angly, F., Breitbart, M., et al. (2010). Viral and microbial community dynamics in four aquatic environments. ISME J. 4, 739–751. doi: 10.1038/ismej.2010.1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRosario, K., Marinov, M., Stainton, D., Kraberger, S., Wiltshire, E. J., Collings, D. A., et al. (2011). Dragonfly cyclovirus, a novel single-stranded DNA virus discovered in dragonflies (Odonata: Anisoptera). J. Gen. Virol. 92, 1302–1308. doi: 10.1099/vir.0.030338-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRosengaus, R. B., Zecher, C. N., Schultheis, K. F., Brucker, R. M., and Bordenstein, S. R. (2011). Disruption of the termite gut microbiota and its prolonged consequences for fitness. Appl. Environ. Microbiol. 77, 4303–4312. doi: 10.1128/AEM.01886-10\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRoux, S., Hallam, S. J., Woyke, T., and Sullivan, M. B. (2015b). Viral dark matter and virus–host interactions resolved from publicly available microbial genomes. Elife 4:e08490. doi: 10.7554/eLife.08490\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRoux, S., Solonenko, N. E., Dang, V. T., Poulos, B. T., Schwenck, S. M., Goldsmith, D. B., et al. (2016). Towards quantitative viromics for both double-stranded and single-stranded DNA viruses. PeerJ 4:e2777. doi: 10.7717/peerj.2777\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRoux, S., Tournayre, J., Mahul, A., Debroas, D., and Enault, F. (2014). Metavir 2: new tools for viral metagenome comparison and assembled virome analysis. BMC Bioinformatics 15:76. doi: 10.1186/1471-2105-15-76\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSantiago-Rodriguez, T. M., Ly, M., Bonilla, N., and Pride, D. T. (2015). The human urine virome in association with urinary tract infections. Front. Microbiol. 6:14. doi: 10.3389/fmicb.2015.00014\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nShin, S., Lu, G., Cai, M., and Kim, K.-S. (2005). Escherichia coli outer membrane protein A adheres to human brain microvascular endothelial cells. Biochem. Biophys. Res. Commun. 330, 1199–1204. doi: 10.1016/j.bbrc.2005.03.097\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nShinzato, N., Muramatsu, M., Matsui, T., and Watanabe, Y. (2005). Molecular phylogenetic diversity of the bacterial community in the gut of the termite Coptotermes formosanus. Biosci. Biotechnol. Biochem. 69, 1145–1155. doi: 10.1271/bbb.69.1145\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSullivan, M. B. (2015). Viromes, not gene markers, for studying double-stranded DNA virus communities. J. Virol. 89, 2459–2461. doi: 10.1128/JVI.03289-14\\\\nCrossRef Full Text | Google Scholar\\\\nSummer, E., Liu, M., Gill, J., Grant, M., Chan-Cortes, T., Ferguson, L., et al. (2011). Genomic and functional analyses of Rhodococcus equi phages ReqiPepy6, ReqiPoco6, ReqiPine5, and ReqiDocB7. Appl. Environ. Microbiol. 77, 669–683. doi: 10.1128/AEM.01952-10\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTadmor, A. D., Ottesen, E. A., Leadbetter, J. R., and Phillips, R. (2011). Probing individual environmental bacteria for viruses by using microfluidic digital PCR. Science 333, 58–62. doi: 10.1126/science.1200758\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTangherlini, M., Dell\\'anno, A., Allen, L. Z., Riccioni, G., and Corinaldesi, C. (2016). Assessing viral taxonomic composition in benthic marine ecosystems: reliability and efficiency of different bioinformatic tools for viral metagenomic analyses. Sci. Rep. 6:28428. doi: 10.1038/srep28428\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTikhe, C. V., Martin, T. M., Gissendanner, C. R., and Husseneder, C. (2015). Complete genome sequence of Citrobacter phage CVT22 isolated from the gut of the Formosan subterranean termite, Coptotermes formosanus Shiraki. Genome Announc. 3, e00408–00415. doi: 10.1128/genomeA.00408-15\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTouchon, M., Moura De Sousa, J. A., and Rocha, E. P. C. (2017). Embracing the enemy: the diversification of microbial gene repertoires by phage-mediated horizontal gene transfer. Curr. Opin. Microbiol. 38, 66–73. doi: 10.1016/j.mib.2017.04.010\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWang, Y., Coleman-Derr, D., Chen, G., and Gu, Y. Q. (2015). OrthoVenn: a web server for genome wide comparison and annotation of orthologous clusters across multiple species. Nucleic Acids Res. 43, W78–W84. doi: 10.1093/nar/gkv487\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWarnecke, F., Luginbühl, P., Ivanova, N., Ghassemian, M., Richardson, T. H., Stege, J. T., et al. (2007). Metagenomic and functional analysis of hindgut microbiota of a wood-feeding higher termite. Nature 450, 560–565. doi: 10.1038/nature06269\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWilhelm, S. W., and Suttle, C. A. (1999). Viruses and nutrient cycles in the sea viruses play critical roles in the structure and function of aquatic food webs. Bioscience 49, 781–788. doi: 10.2307/1313569\\\\nCrossRef Full Text | Google Scholar\\\\nWillner, D., Furlan, M., Haynes, M., Schmieder, R., Angly, F. E., Silva, J., et al. (2009). Metagenomic analysis of respiratory tract DNA viral communities in cystic fibrosis and non-cystic fibrosis individuals. PLoS ONE 4:e7370. doi: 10.1371/journal.pone.0007370\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYang, Y.-J., Zhang, N., Ji, S.-Q., Lan, X., Shen, Y.-L., Li, F.-L., et al. (2014). Dysgonomonas macrotermitis sp. nov., isolated from the hindgut of a fungus-growing termite. Int. J. Syst. Evol. Microbiol. 64, 2956–2961. doi: 10.1099/ijs.0.061739-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nZablocki, O., Van Zyl, L., Adriaenssens, E. M., Rubagotti, E., Tuffin, M., Cary, S. C., et al. (2014). High-level diversity of tailed phages, eukaryote-associated viruses, and virophage-like elements in the metaviromes of antarctic soils. Appl. Environ. Microbiol. 80, 6888–6897. doi: 10.1128/AEM.01525-14\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nZeng, H., Zhang, J., Li, C., Xie, T., Ling, N., Wu, Q., et al. (2017). The driving force of prophages and CRISPR-Cas system in the evolution of Cronobacter sakazakii. Sci. Rep. 7:40206. doi: 10.1038/srep40206\\\\nCrossRef Full Text | Google Scholar\\\\nZhu, A., Sunagawa, S., Mende, D. R., and Bork, P. (2015). Inter-individual differences in the gene content of human gut bacterial species. Genome Biol. 16, 82. doi: 10.1186/s13059-015-0646-9\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01870\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01870/full\",\\n    \"loadedTime\": \"2024-01-23T08:48:53.299Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01870\",\\n    \"title\": \"Frontiers | Newly Isolated Bacteriophages from the Podoviridae, Siphoviridae, and Myoviridae Families Have Variable Effects on Putative Novel Dickeya spp.\",\\n    \"description\": \"Soft rot pathogenic bacteria from the genus Dickeya cause severe economic losses in orchid nurseries worldwide, and there is no effective control currently available. In the last decade, the genus Dickeya has undergone multiple changes as multiple new taxa have been described, and just recently a new putative Dickeya species was reported. This study reports the isolation of three bacteriophages active against putative novel Dickeya spp. isolates from commercially produced infected orchids that show variable host-range profiles. Bacteriophages were isolated through enrichment from Dickeya-infected orchid tissue. Convective interaction media monolith chromatography was used to isolate bacteriophages from wastewaters, demonstrating its suitability for the isolation of infective bacteriophages from natural sources. Based on bacteriophage morphology, all isolated bacteriophages were classified as being in the order Caudovirales, belonging to three different families, Podoviridae, Myoviridae and Siphoviridae. The presence of three different groups of bacteriophages was confirmed by analysing the bacteriophage specificity of bacterial hosts, restriction fragment length polymorphism and plaque morphology. Bacteriophage BF25/12, the first reported Podoviridae bacteriophage effective against Dickeya spp., was selected for further characterisation. Its genome sequence determined by next-generation sequencing showed limited similarity to other characterised Podoviridae bacteriophages. Interactions among the bacteriophages and Dickeya spp. were examined using transmission electron microscopy, which revealed degradation of electron-dense granules in response to bacteriophage infection in some Dickeya strains. The temperature stability of the chosen Podoviridae bacteriophage monitored over one year showed a substantial decrease in the survival of bacteriophages stored at -20 °C over longer periods. It showed susceptibility to low pH and UV radiation but was stable in neutral and alkaline pH. Furthermore, the stability of the tested bacteriophage was also connected to the incubation medium and bacteriophage concentration at certain pH values. Finally, the emergence of bacteriophage-resistant bacterial colonies is highly connected to the concentration of bacteriophages in the bacterial environment. This is the first report on bacteriophages against Dickeya from the Podoviridae family to expand on potential bacteriophages to include in bacteriophage cocktails as biocontrol agents. Some of these bacteriophage isolates also showed activity against Dickeya solani, an aggressive strain that causes the soft rot of potatoes,\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nDickeya spp. (formerly Erwinia chrysanthemi) are plant pathogenic bacteria that can cause soft rot disease across a wide range of crops and ornamental plants worldwide (Czajkowski et al., 2011; Toth et al., 2011; Adriaenssens et al., 2012). These Gram-negative, facultative anaerobic bacteria of the γ-Proteobacteria subdivision cluster into the Enterobacteriaceae family. Dickeya characteristically produce cell wall-degrading enzymes that are secreted during their infection of plants, resulting in the plant–tissue maceration that is typical of soft rot disease (Czajkowski et al., 2011; Adriaenssens et al., 2012). Dickeya can be spread over long distances via infected plants and can also live as epiphytes or facultative saprophytes in soil and ground water (Reverchon and Nasser, 2013). They have been detected in a variety of water sources, and irrigation water has been reported as the probable source of their infection of potatoes in Australia (Toth et al., 2011). There are currently no effective chemical agents to control Dickeya soft rot infection, resulting in significant economic loss, particularly in terms of potato production (Adriaenssens et al., 2012; Czajkowski et al., 2014).\\\\nThe genus Dickeya currently encompasses eight species—Dickeya zeae, D. dadantii, D. chrysanthemi, D. solani, D. aquatica, D. dianthicola, D. paradisiaca, and D. fangzhongdai (Samson et al., 2005; Toth et al., 2011; Adriaenssens et al., 2012; Tian et al., 2016). The use of molecular tools has revealed further evolution of these species, with the occasional description of strains with characteristics different from the previously described species, such as unassigned lineages (UDLs), in the Dickeya phylogenetic backbone (Samson et al., 2005; Parkinson et al., 2009; Adriaenssens et al., 2012; Van Vaerenbergh et al., 2012). Van Vaerenbergh et al. (2012) reported the introduction of UDLs, mostly for Dickeya isolates from ornamental plants. Furthermore, some of the isolates corresponding to UDL-3 and UDL-4 based on the fliC sequence were described as putative new species in the genus Dickeya (Alič et al., 2017).\\\\nThe use of bacteriophages as biocontrol agents is an attractive option for controlling bacterial diseases of plants (Jones et al., 2007; Adriaenssens et al., 2012; Buttimer et al., 2017). Bacteriophages can show high specificity for the bacterial hosts they infect, and only lyse bacterial cells, thus providing targeted disease management. Bacteriophages are naturally present in the environment, such as in soil, water, plants, and animals, where they can persist through host-dependent self-replication over long periods. As bacteriophages cannot infect eukaryotic cells, they are safe to use (reviewed in Jones et al., 2007). While only lytic bacteriophages can be used for bacteriophage therapy and biocontrol (Jones et al., 2007, 2012), their relatively easy and inexpensive production means that they are of commercial interest (reviewed in Jones et al., 2007).\\\\nDespite these advantages, few bacteriophages have demonstrated repeated, successful applications in plant disease management. Apart from the obvious challenges, such as resistance development in bacteria (Katznelson, 1937; Levin and Bull, 2004), bacteriophage therapies must take into account the complex dynamics among bacteriophages, bacteria and their environment, which largely remain unexplored (Levin and Bull, 2004). In addition, suitable protective formulation and delivery of bacteriophages to affected plants must be optimized to ensure their efficient application and their survival during this application (Jones et al., 2007; Frampton et al., 2012).\\\\nThe possibility of plant disease control using bacteriophages has been studied for various plant pathogenic bacteria (e.g., E. amylovora, Agrobacterium tumefaciens, Ralstonia solanacearum, Streptomyces scabies, Pseudomonas sp., Xanthomonas sp., Pectobacterium sp., Xylella fastidiosa, and Dickeya sp.) (reviewed in Buttimer et al., 2017). The idea of using bacteriophages as controlling agents for Dickeya spp. has been proposed before, but only limited attempts have been made to isolate such lytic bacteriophages (Czajkowski et al., 2014). Adriaenssens et al. (2012) described the T4-related bacteriophages LIMEstone1 and LIMEstone2 that specifically infect D. solani. In their study, both bacteria and bacteriophages were isolated from soil taken from a potato field. The first reported experimental field trial for the bacteriophage treatment of potato tubers infected with D. solani resulted in improved yields, which led to the proposal of the bacteriophages LIMEstone1 and LIMEstone2 as effective therapeutics in an agricultural setting (Adriaenssens et al., 2012). Czajkowski et al. (2014) isolated and identified nine more bacteriophages that showed wider host ranges and that were all active against D. solani. All these bacteriophages prevented the growth of D. solani in vitro and protected potato tuber tissue from the maceration caused by D. solani.\\\\nHowever, further studies are required to determine the long-term effectiveness of bacteriophages and to perform field trials. Some bacteriophages with broader host ranges have been reported to infect some Pectobacterium strains as well as D. solani (Czajkowski et al., 2015). Recently, bacteriophages that are active against D. dadantii isolates were isolated from potato plants. These bacteriophage isolates are members of the Myoviridae and Siphoviridae families and were all isolated from the water of the Caspian Sea. Their effectiveness was tested in vitro, and they showed promising biocontrol potential. However, no field trials have been conducted to date (Soleimani-Delfan et al., 2015).\\\\nThe variously reported investigations and trials involving Dickeya spp. have so far only been carried out in terms of improving potato production (Adriaenssens et al., 2012; Czajkowski et al., 2014). However, ornamental plants with high added value, such as orchids, have significant economic importance in agricultural production, and due to extensive worldwide trade, if infected, these can facilitate the spread and possible expansion of the pathogenic bacteria host range. The aim of the study was to isolate and characterize bacteriophages active against putative new Dickeya spp. as a research tool to study the bacteria–bacteriophage system and as a potential biocontrol component.\\\\nHere, we report on the successful application of the convective interaction media (CIM) method for bacteriophage isolation from complex environmental samples. We show the activity of bacteriophage isolates against two previously determined putative Dickeya spp. (UDL-3 and UDL-4; Alič et al., 2017). For the isolates, we determine their host range and restriction fragment polymorphism (RFLP) profiles and characterize them according to their morphology using transmission electron microscopy. Finally, we describe the first reported Podoviridae bacteriophage effective against Dickeya spp., BF25/12, including its annotated genome sequence.\\\\nMaterials and Methods\\\\nBacterial Strains and Media\\\\nDickeya strains were isolated from Phalaenopsis orchid leaves that showed soft rot symptoms and were identified as a putative new species within the genus Dickeya (Alič et al., 2017). Based on the fliC gene sequence, the bacteria isolates were classified as members of UDL-3 and UDL-4 (Alič et al., 2017). These bacteria were routinely grown on Luria Bertani (LB) medium with 1.5% agar and 0.05% (w/v) NaCl at 28–30°C. For liquid preparations, the bacteria cultures were grown in LB broth at 28°C, with agitation at 200 rpm unless otherwise stated. LB with 0.4% agar was used for overlays. For long-term storage, the bacteria cultures and isolates were kept in a bacterial and fungal storage system (Microbank; bioTRADING, Mijdrecht, Netherlands) at ≤-76°C.\\\\nIsolation of Bacteriophages\\\\nIsolation of Bacteriophages from Plant Material\\\\nBacteriophages against Dickeya UDL-3 were isolated from Phalaenopsis orchid leaves that showed soft rot symptoms. The orchids were obtained from a commercial orchid production site. Extracts of the plant tissues were prepared by commuting or macerating plant material in 45 mL of Trypticase Soy Broth (30 g Trypticase Soy Broth [BD 211768], double-distilled water to 1 L, pH 9). Sterile 450 μL 1 M MgSO4 and 90 μL 1 M CaCl2 were added to the mixture. The extract was incubated for 1 h at room temperature with constant rotation (Heto Mastermix rotator), followed by centrifugation at 10,000 × g for 10 min. The resulting supernatant was carefully poured into new 50 mL tubes (Falcon). The clear extract was again centrifuged as above. Finally, the supernatant was sterile filtered through 0.2-μm membranes (Millipore) and stored at 4°C until enrichment. Enrichment of the bacteriophages in the extract was performed with a mixture of different Dickeya isolates (Table 1). For enrichment, 15 mL double-strength LB (20 g tryptone, 10 g yeast extract, 1 g NaCl, double-distilled water to 1 L) with 300 μL 1 M MgSO4 and 60 μL 1 M CaCl2 was mixed with 15 mL extract of bacteriophages. To this, 150 μL bacteria in log phase growth (12–14 h incubation at 28°C, with shaking at 200 rpm) was added at an equal ratio (Van Twest and Kropinski, 2009). This was incubated for approximately 24 h at 28°C, with shaking at 50 rpm.\\\\nTABLE 1. Dickeya spp. isolates used in the bacteriophage enrichment.\\\\nThe enriched suspension (10 μL) was spotted on a streak of bacteria in log phase growth used in the enrichment and on a plate with an LB overlay that contained the same bacteria isolates, following Van Twest and Kropinski (2009). After a 24-h incubation at 28°C, the resulting lysis cones were scratched into 200 μL saline-magnesium with gelatine (SMG; 5.8 g NaCl, 2.0 g MgSO4 × 7H2O, 7.88 g Tris HCl, pH 7.5, 0.1 g gelatine, double-distilled water to 1 L), and double agar overlay plaque assays were performed (Kropinski et al., 2009). The host bacterium for the plaque assays was prepared as follows: 1 μL inoculation loop of overnight bacteria grown on solid LB was transferred to 5 mL liquid LB with 50 μL 1 M MgSO4 and 10 μL 1 M CaCl2 and incubated for 12–14 h at 28°C, with shaking at 100 rpm. The overnight culture was then diluted 10-fold to obtain the optical density at 600 nm (OD600) of 0.5, which corresponded to approximately 108 cells per mL. Single plaques were selected and transferred to SMG for further purification using disposable pipette tips. The bacteriophages were purified by streaking 1 μL bacteriophage suspension on solid LB and pouring over the indicator bacterial lawn. Three successive single plaque isolations were performed as described above. For bacteriophage amplification, double agar overlay plaque assays on LB were carried out (enumeration of bacteriophages by double agar overlay plaque assay), following Kropinski et al. (2009) but with several modifications: (i) 2 mL of the top agar was dispensed into sterile tubes; (ii) tubes containing top agar were incubated at 52°C and (iii) 63 μL bacteria suspension and bacteriophage dilution were mixed and incubated for 20 min at room temperature prior to the transfer to the warm overlay medium, with agitation at 150 rpm. Confluent plates (plaques evenly distributed over the plates as single plaques and not touching others) were then poured with 3 mL SMG. The plates were sealed with Parafilm and incubated for approximately 20 h at +4°C, with gentle shaking at 100 rpm. The bacteriophages were sterile filtered over 0.2-μm membranes and stored at +4°C or at -80°C for longer periods.\\\\nIsolation of Bacteriophages from Wastewater\\\\nBacteriophages against the Dickeya UDL-3 and UDL-4 strains were isolated from wastewater concentrates. Effluent water was collected from a wastewater treatment plant and concentrated using a CIM quaternary amine monolithic column. The preparation and characterization of the water samples were as described in Steyer et al. (2015). The wastewater bacteriophage concentrates were diluted in double-distilled water (1:2; v/v) to a final 1% NaCl.\\\\nFor the enrichment, 3 mL double-strength LB with 120 μL 1 M MgSO4 and 24 μL 1 M CaCl2 was mixed with 3 mL diluted bacteriophage concentrate. To that, 30 μL of bacteria in log phase (12–14 h incubation at 28°C, with shaking at 200 rpm) was added at an equal ratio (Van Twest and Kropinski, 2009). The enrichment of UDL-3 and UDL-4 bacteria (Table 1) was preformed separately. The enrichments were incubated for approximately 24 h at 28°C, with shaking at 50 rpm and sterile filtering through 0.2-μm membranes.\\\\nThe bacteriophage suspension (7 μL) was used in the enrichment on plates with an LB overlay, with plaques isolated as described for the plant material isolation. The bacteriophages were purified by repeating the plaque assays three times. For bacteriophage amplification, double agar overlay plaque assays were carried out as described above. The bacteriophages were eluted from confluent plates with the change of the incubation temperature to room temperature (25°C) and were stored as described above.\\\\nHost Range and Specificity of Bacteriophages\\\\nFor host-range determination, 56 bacterial strains and isolates (Table 4) were tested. This included 14 Dickeya isolates from diseased Phalaenopsis, eight Dickeya spp. reference strains from a culture collection (D. solani, D. dieffenbachiae, D. dianthicola, D. zeae, D. dadanti, D. aquatica, two Dickeya spp.), seven Pectobacterium cypripedii from a culture collection, four non-Dickeya spp. isolates from diseased Phalaenopsis, 10 isolates from healthy Phalaenopsis leaves and 14 different bacterial strains from the Enterobacteriaceae family. The host range was tested by spotting 10 μL of a bacteriophage suspension onto an LB bacterial lawn, which was performed as three biological repeats. All positive results were confirmed by double agar overlay plaque assays (described above) to eliminate false positives.\\\\nTransmission Electron Microscopy\\\\nThe presence and morphology of the isolated bacteriophages were determined using transmission electron microscopy with a negative staining method. Here, 20-μL bacteriophage suspensions were deposited on Formvar-coated and carbon-stabilized copper grids and stained with a 1% (w/v) aqueous solution of uranyl acetate.\\\\nTo accompany the bacteria–bacteriophage interactions, bacterial suspensions (i.e., UDL-3 and UDL-4 separately) were mixed with the purified selected bacteriophage BF25/12, which was shown to be active against bacteria and was kept for 15 min at room temperature. The suspensions were centrifuged at 3,000 × g for 10 min. The supernatants were carefully removed, and the pellets were resuspended in 3% glutaraldehyde and 1% paraformaldehyde in 0.1 M Sörensen phosphate buffer (Merck), pH 7.2, for 20 h at 4°C. For UDL-3, pieces of soft agar that contained borders of plaques from the plaque assay and soft agar with bacteria from the control plate were fixed in the same fixative. All the samples were post-fixed in 2% OsO4 in 0.1 M Sörensen phosphate buffer (Merck) pH 7.2 and embedded in Agar 100 resin (Agar Scientific). Ultrathin sections were stained with 2% (w/v) aqueous uranyl acetate and Reynolds lead citrate (Agar Scientific).\\\\nThe grids were observed using transmission electron microscopy (CM 100; Philips, Netherlands) operated at 80 kV and equipped with a CCD camera (Orius SC 200) and Digital Micrograph software (Gatan Inc., United States).\\\\nBacteriophage Genome Characterization\\\\nBacteriophage DNA was isolated according to Pickard (2009) using the Phase Lock Gel system (5 PRIME, Germany). To determine the diversity of the isolated bacteriophages, RFLP analysis was performed according to Adriaenssens et al. (2012) using the Hind II (Thermo Scientific) restriction enzyme. Here, 0.5 μg bacteriophage DNA was used per reaction mixture. The restriction was performed at 37°C for 16 h. The fragments were analyzed on 0.8% agarose gels for 100–120 min at 90 V. Bacteriophage BF25/12 was selected from among Podoviridae isolates for further characterization.\\\\nWhole Genome Sequencing\\\\nThe whole genome of bacteriophage BF25/12 was sequenced using 454 sequencing technology. The paired-end library was sequenced at Microsynth (Balgach, Switzerland) following the manufacturer’s instructions to 36-fold coverage. Whole genome assembly was performed with the GS De Novo Assembler using default parameters (Microsynth). The fully assembled genome was annotated using Rapid Annotations using Subsystems Technology (RAST) (Aziz et al., 2008; Overbeek et al., 2014; Brettin et al., 2015) and the default setting options. Additionally, genome annotation was verified and curated by BLAST analysis (Altschul et al., 1990). The assembled genome sequence was compared to other characterized bacteriophages using the PASC web tool (Bao et al., 2014) and BLAST analysis (Altschul et al., 1990).\\\\nUV Radiation Stability\\\\nBacteriophage stability was tested for bacteriophage BF25/12. A bacteriophage suspension was prepared in SMG buffer, and the bacteriophage concentration was adjusted to the concentration of 108 pfu/mL. Testing was performed on 5 mL of prepared bacteriophage suspension in open Petri dishes (Φ = 55 mm, Golas) illuminated using double UV-C light (Philips TUV G30T8, UV dose approximately 100 mJ/cm2 30 cm from the light source) for 1 and 2 min. The Petri dishes were kept on ice covered with cloth to minimize the influence of heat produced by the UV light. The bacteriophage stability was tested by double agar overlay plaque assay (described above) in two biological repeats as two technical replicates.\\\\npH Stability\\\\nThe pH stability of the bacteriophage BF25/12 was tested for a selected range of pH values from 3 to 11. Testing was performed in SMG buffer and sterile demineralised water, adjusted with HCl or NaOH to the pH values of 3, 5, 7, 9, and 11. Phage suspensions were mixed with buffer or water (1 ml total volume) for final concentrations of 105 and 103 pfu/mL and were incubated at room temperature with shaking (100 rpm) for 24 h. The concentration of active bacteriophages was determined with a double agar overlay plaque assay (described above) in two biological repeats and two technical replicates.\\\\nTemperature Stability\\\\nBacteriophage stability was analyzed for the bacteriophage BF25/12 by incubating a bacteriophage suspension (100 μL) in SMG at different temperatures, +4, +28, -20, and -80°C. The stability was tested once per month for 1 year using double agar overlay plaque assays (described above) as three technical replicates.\\\\nDevelopment of Bacterial Resistance\\\\nThe emergence of bacterial resistance was measured for the combination of the selected bacteriophage BF25/12 and the bacteria Dickeya sp. B16. A double agar overlay assay was used to enumerate the emergence of resistant bacterial colonies at a bacterial concentration of 108 cfu/mL and four different bacteriophage concentrations, 103, 104, 105, and 106 pfu/mL. Resistant bacterial colonies were counted on plates, with double agar overlay plaque assays carried out after 27 h of incubation at 28°C. The test was performed in triplicate per bacteriophage concentration, with two technical repeats each.\\\\nResults\\\\nSourcing of Bacteriophages and Characterization\\\\nThe target Dickeya spp. bacteria used in this study were previously isolated from diseased Phalaenopsis orchids and identified as two distinct UDLs, UDL-3 and UDL-4, based on the partial sequencing of the fliC and dnaX genes (Alič et al., 2017). Together with a set of reference bacteria (Table 1), these were used as a mixture to enrich bacteriophages from extracts of rotting orchid leaves and wastewater. Altogether, 18 bacteriophages against Dickeya spp. were isolated, five from diseased Phalaenopsis leaves and 13 from wastewater. Bacteriophages active against UDL-3 bacteria were successfully isolated from plant tissue, but bacteriophages with strong activity against UDL-4 were only isolated from municipal wastewater far away from the orchid site of production. Bacteriophages were isolated from wastewater using a concentration of the bacteriophage particles with CIM monolith chromatography. Together with the subsequent plaque assays, this demonstrates the suitability of this approach for the isolation of infective bacteriophages from wastewater samples.\\\\nThe bacteriophage recovery from municipal wastewater varied during the season of monthly sampling and the isolation attempts. Bacteriophages active against Dickeya spp. were predominantly isolated in the warmer months (Table 2). After overnight incubation, all the bacteriophages active against Dickeya spp. from diseased Phalaenopsis isolates produced large clear plaques of approximately 6 mm in diameter with a halo effect, suggesting the presence of lysins (Figure 1A). Mixed bacteriophage populations were ruled out by purifying the bacteriophages separately from the clear plaque zones and the halo rings, which resulted in identical morphology with a halo effect for all of them. Furthermore, the restriction of bacteriophage DNA with Hind II (Figure 1E) showed no differences in the patterns between the bacteriophages isolated from these diseased Phalaenopsis orchids.\\\\nTABLE 2. Sampling timeline of the plant material and wastewater.\\\\nFIGURE 1. Representative bacteriophage plaque morphologies and restriction fragment length polymorphism profiles. Plaque morphologies for bacteriophages isolated from plant material (A) and wastewater (B,C). Bacteriophages against Dickeya spp. isolated from wastewater (D) and from diseased Phalaenopsis (E) digested with Hind II restriction enzyme. M, Gene Rule 1 kb DNA ladder.\\\\nThe bacteriophages active against Dickeya spp. that were isolated from wastewater showed two different plaque morphologies. The bacteriophages from the June samples produced large clear plaques that were approximately 5 mm in diameter with a halo effect, and those isolated in April, May, August, and October produced medium to small plaques of approximately 3 mm in diameter, with a clear zone encircled by a white ring and a halo effect (Figures 1B,C). Restriction of the bacteriophage DNA derived from the wastewater showed two different RFLP patterns (Figure 1D). The bacteriophages BF-CIM1/14 that were isolated in June had a uniform RFLP pattern that differed from all the other bacteriophages isolated from wastewater and instead showed a common RFLP pattern.\\\\nBased on the morphologies of the bacteriophage particles examined with transmission electron microscopy (using negative staining), two of five isolated bacteriophages from diseased Phalaenopsis were placed in the Podoviridae family and showed a head size of 55.8 ± 2.3 nm × 52.6 ± 3.1 nm (n = 20) (Figure 2A). The bacteriophage tail dimensions were difficult to determine accurately because whole tails were rarely seen and the size was highly dependent on the bacteriophage position on the grid. However, the tail size was estimated to be around 12.1 ± 2.4 nm × 16.3 ± 1.7 nm (n = 13) (Figure 2A). The bacteriophages from wastewater were classified as Siphoviridae (Figure 2B) and Myoviridae (Figure 2C), which are both in the order Caudovirales. Bacteriophages from the Siphoviridae family (Figure 2B) had icosahedral heads (54.6 ± 2.6 nm × 53.9 ± 2.2 nm, n = 24) and slightly flexible tails (131.7 ± 8.4 nm × 10.6 ± 0.6 nm, n = 23) that continued into thin terminal fibers. Bacteriophages characterized as members of the Myoviridae family (Figure 2C) had larger icosahedral heads (138.2 ± 3.7 nm × 136.6 ± 4.1 nm, n = 47) and rigid contractile tails (228.1 ± 7.0 nm × 24.8 ± 1.4 nm, n = 47). The base plates were rather large (approximate length of 30 nm), and the attached spikes were short and could be observed only criss-crossed in a netlike structure and were therefore impossible to count. In summary, all these bacteriophages active against Dickeya spp. belong to one of three groups, having common RFLP profiles and plaque and bacteriophage morphologies (Table 3).\\\\nFIGURE 2. Representative transmission electron microscopy micrographs of bacteriophages isolated from diseased Phalaenopsis leaves (A) and sewage water (B,C), all of which were only active against Dickeya spp.\\\\nTABLE 3. Bacteriophages isolated from infected Phalaenopsis tissue and wastewater concentrates using convective interaction media (CIM) monolith chromatography.\\\\nSpecificity of Isolated Bacteriophages against Dickeya spp.\\\\nThe specificities of the interactions between the bacteria and bacteriophages were examined using spot tests (i.e., isolation of bacteriophage via induction of lysogenesis) and were confirmed using plaque assays. The specificities of the bacteriophages included in the testing were limited to Dickeya spp., excluding D. aquatica and D. zeae. All the bacteriophages from orchid tissue showed identical host-range profiles. They were active against Dickeya sp. UDL-3 isolates, Dickeya sp. MK7 and D. dadantii NCPPB 898. However, the bacteriophages isolated from wastewater showed variability in their host specificities. The host ranges of the bacteriophages BF-CIM5/14, BF-CIM6/14, BF-CIM8/14, and BF-CIM10/15 were very similar, and their specificities were close for both Dickeya spp. UDL-3 and UDL-4. However, substantial differences in specificities against Dickeya spp. from the bacteria collection here were observed. The bacteriophages BF-CIM8/14 and BF-CIM10/14 did not only infect D. aquatica and D. zeae. Conversely, infection by the bacteriophages BF-CIM5/14 and BF-CIM6/14 was limited to Dickeya sp. MK7 and weakly to D. solani GBBC 500 and D. solani GBBC 2040. Finally, all three bacteriophage isolates from the BF-CIM1/14 group showed a uniform infection profile; they were active only against Dickeya sp. UDL-4 isolates D. solani GBBC 500 and D. dianthicola LMG 2458 (Table 4).\\\\nTABLE 4. Host range of bacteriophages isolated against UDL-3 and UDL-4 isolates.\\\\nBased on the plaque morphology, RFLP profiles, host ranges and transmission electron microscopy analysis, three different bacteriophages were isolated. Two were derived from the wastewater and were members of the Myoviridae and Siphoviridae families. The third bacteriophage belonged to the Podoviridae family and was obtained from the soft rot orchid tissue. Based on the characteristics of these bacteriophages, bacteriophage BF25/12 was chosen for further analysis.\\\\nTransmission Electron Microscopy of Bacteria–Bacteriophage Interactions\\\\nThe interactions of the selected bacteriophage BF25/12 with the host bacteria were examined at the ultrastructure level. The interactions of BF25/12 and the UDL-4 bacteria activated granule degradation in the bacteria cells without bacterial lysis. Most of these UDL-4 bacteria contained electron-lucent intracellular granules (20–70 nm in diameter), which were similar to intracellular inclusion bodies (Figure 3A; Alič et al., 2017). When bacteria UDL-4 and bacteriophages BF25/12 were incubated together for 15 min, these granules almost completely disappeared, and there were bacteriophages attached to the bacterial surface (Figure 3B). Mixtures of Dickeya sp. UDL-3 (Figure 3C) and BF25/12 also showed the attachment of the bacteriophages to the bacterial surface (Figure 3D). Based on the plaque assays, the borders of the plaques were analyzed. Most of the UDL-3 cells here were lysed, and the plasmalemmas of the resting cells were detached (Figure 3E). New bacteriophages could also be seen in the cytoplasm of some of the bacteria (Figure 3F).\\\\nFIGURE 3. Representative ultrastructure analyses of bacteria–bacteriophage interactions. (A,B) Strain UDL-4 before (A) and after (B) mixing with bacteriophage BF25/12 (arrows). (C,D) Strain UDL-3 before (C) and after (D) mixing with bacteriophage BF25/12 (arrows). (E,F) Lysed UDL-3 bacteria from the border of a plaque (E) and new bacteriophages in the cytoplasm (arrowheads) (F). G, intracellular granules; CW, cell wall; PL, plasmalemma; PS, periplasmic space.\\\\nGenome of Bacteriophage BF25/12\\\\nThe genome of bacteriophage BF25/12 was additionally characterized using genome sequencing and automatic annotation. BF25/12 contains linear double-stranded (ds)DNA that is 43,872 bp long with 52.2% GC, which supports its classification into the Podoviridae family. The genome of bacteriophage BF25/12 contains 52 open reading frames that were determined with RAST (Aziz et al., 2008; Overbeek et al., 2014; Brettin et al., 2015). They mainly coded for hypothetical proteins (27 genes), along with structural proteins and proteins involved in bacteriophage replication. Furthermore, a single putative endolysin gene was identified, corresponding to the halo effect observed in plaque morphology. However, no genes that encode antibiotic resistance or toxins were detected (Figure 4). The annotated BF25/12 genome sequence is available in GenBank under Accession No. KT240186.\\\\nFIGURE 4. Annotated BF25/12 bacteriophage genome. According to the annotation, the genome does not contain any genes encoding antibiotic resistance or toxins. The open reading frames coding for structural proteins, proteins involved in bacteriophage replication, and other conserved proteins are shown in yellow, and those coding for hypothetical proteins are shown in gray. Unit of the presented genome annotation is in bp.\\\\nThe genomic sequence of bacteriophage BF25/12 was compared to characterized genera from the Podoviridae family using the PASC tool (Bao et al., 2014). BF25/12 showed low identity (<3%) to any member of the Picovirinae or Sepvirinae subfamilies. However, it shared higher identity to the Autographivirinae subfamily, especially Klebsiella virus F19 from the Kp34 virus genus (33% identity). Based on the pairwise identity analysis conducted, the BF25/12 genome was 57–62% identical to three unassigned Pectobacterium bacteriophages, Pectobacterium phages PP16, PP90, and Peat1. The results correspond to the nBLAST analysis. The assembled BF25/12 genome showed substantial similarity (74–79% identity and 21–56% query coverage) to five bacteriophage genomes in the BLAST database. All five genomes belonged to Pectobacterium bacteriophages, four of which were assigned to the Podoviridae family, such as BF25/12, and the last was unclassified. The closest hit represented Pectobacterium bacteriophage PP16 (79% identity over 56% query coverage).\\\\nUV Radiation Stability\\\\nBacteriophage BF25/12 was not able to survive a 2-min exposure to UV-C light. The reduction of active bacteriophages was 105, corresponding to almost 100% in the first minute of exposure.\\\\npH Stability\\\\nBacteriophage pH stability analysis in SMG buffer and sterile demineralised water at two different bacteriophage concentrations (105 and 103 pfu/mL) showed that bacteriophage BF25/12 pH stability is affected by both its concentration and environment (Figure 6). Bacteriophage BF25/12 lost its infectivity completely at pH 3, regardless of the incubation media (SMG buffer or sterile demineralised water). It showed maximum stability at pH 7 and 9, which indicated the favorable pH for bacteriophage BF25/12 is neutral to mildly alkaline. Furthermore, no significant difference in the decrease of bacteriophage titer could be detected between the tested incubation media or bacteriophage concentrations after 24 h incubation at pH 7 and 9. At higher concentrations in SMG buffer, a mildly acidic pH (pH 5) did not cause significant phage de-activation. Bacteriophage incubated in SMG buffer completely lost its infectivity at pH 11 after 24 h incubation, but its infectivity was not affected in sterile demineralised water in the same conditions. The exposure of highly concentrated bacteriophage lysate (108 pfu/mL) to acidic (pH 3) and alkaline (pH 11) environments for 24 h had almost no detectable effect on bacteriophage survival and infectivity (data not shown).\\\\nTemperature Stability\\\\nThe stability of the bacteriophage BF25/12 active against Dickeya spp. was monitored at different temperatures over a year. It proved to be stable at 4°C, as the changes in the bacteriophage titers were negligible, which indicated this is a favorable temperature for long-term bacteriophage storage (Figure 5). The bacteriophage stability at -80°C appeared similar to that at 4°C. However, temperatures of -20°C proved to be inappropriate for long-term storage due to the notable decline of the bacteriophage titer. This unusual decrease in the bacteriophage titer at -20°C was seen after the first 3 months of storage. In the following months, the bacteriophage concentration returned to the initial bacteriophage titer. The least favorable temperature tested was 28°C.\\\\nFIGURE 5. Temperature stability with time of Dickeya spp. bacteriophage BF25/12 incubated at temperatures of +4, +28, –20 and –80°C (as indicated).\\\\nFIGURE 6. Effect of pH on stability of bacteriophage BF25/12. Stability was tested in SMG buffer and sterile demineralised water at two different bacteriophage concentrations, 105 and 103 pfu/mL. Concentration of bacteriophages presented in the graph was calculated to the level of bacteriophage stock, used for preparation of testing dilutions.\\\\nBacterial Resistance\\\\nThe emergence of resistance against bacteriophage BF25/12 was determined for host bacteria Dickeya sp. B16. The number of growth-resistant bacterial colonies depended greatly on the bacteriophage concentration in the medium (Figure 7). The highest rate of bacterial resistance development against bacteriophage BF25/12 was at the final bacteriophage concentration of between log 4 and log 5 and the initial bacteria concentration of log 8. Bacteriophage concentrations <log 4 and >log 5 provided lower selection pressure, which resulted in the decreased resistance of the bacterial colonies.\\\\nFIGURE 7. Emergence of Dickeya spp. B16 resistant to bacteriophage BF 25/12. Experiment was performed as two technical repeats, shown on graph as squares and diamonds.\\\\nDiscussion\\\\nIn the present study, we isolated bacteriophages active against the plant pathogens of Dickeya spp. that cause the soft rot of orchids, which is a destructive disease for Phalaenopsis production sites. Bacteriophages were isolated from two different sources, diseased Phalaenopsis tissue and wastewater. For the latter, CIM monolith chromatography was used for bacteriophage particle concentration. CIM monolith chromatography has been reported as an effective technique for virus concentration and for purification of virus particles (Steyer et al., 2015). However, we believe this is the first report that describes the use of CIM monolith chromatography as a technique for the isolation of infectious bacteriophages from environmental samples. It proved to be very efficient for the concentration of bacteriophages from environmental samples, which allowed more efficient bacteriophage enrichment on the host bacterium and therefore better yields in the bacteriophage isolation. Seventy-two percent of bacteriophages active against Dickeya spp. obtained in this study were isolated from wastewater using CIM monolith chromatography, indicating the presence of viable Dickeya spp. bacteria in wastewater or in plants that are in direct contact with wastewater. Furthermore, different sources of bacteriophage isolation resulted in different bacteriophages from different families and with different RFLP patterns, although they showed similar host-specificity profiles. The isolation of the bacteriophages from orchid samples was carried out in summer (i.e., July and September) when the highest concentrations of the bacteria are expected. The isolates from wastewater were obtained from late spring to early autumn, excluding July and September. The attempts to isolate bacteriophages from soil were not successful, regardless of the month when the sampling was carried out (data not shown). Therefore, these differences cannot be explained in terms of seasonal changes, which indicates the specificity of each of these bacteriophage sources.\\\\nHowever, the bacteriophages active against Dickeya spp. that showed high specificities can be used as biocontrol agents, as the treatment of a diseased orchid with these bacteriophages can specifically eliminate the pathogenic bacteria while leaving the normal microbiota unharmed. Adriaenssens et al. (2012) and Czajkowski et al. (2014, 2015) reported the isolation of bacteriophages specifically against the aggressive potato pathogen D. solani, bacteriophages with multiple hosts across Dickeya spp. and broad host-range bacteriophages against Pectobacterium spp. and Dickeya spp. All the bacteriophages active against Dickeya spp. described in the literature belong to the order Caudovirales as members of the Myoviridae family (Adriaenssens et al., 2012; Czajkowski et al., 2014, 2015; Soleimani-Delfan et al., 2015), and the first report about a bacteriophage from the Siphoviridae family was made recently (Soleimani-Delfan et al., 2015). However, the present study represents the first report of bacteriophages active against Dickeya spp. from the Podoviridae family. The bacteriophages from different families show some small differences in their specificities. All the bacteriophages from the Podoviridae (BF21/12 – BF25/12) and Myoviridae (BF-CIM5/14 – BF-CIM10/14) families infected all Dickeya isolates in the present study, as did the aggressive potato pathogen D. solani and other reference Dickeya strains, excluding D. paradisiaca and D. aquatica. However, as members of the Siphoviridae family, bacteriophages BF-CIM1/14 showed different host ranges. The activity against the Dickeya sp. UDL-3 isolates varied and was relatively low, although all the Dickeya sp. UDL-4 and D. solani strains were highly susceptible, which supports a relationship between those two Dickeya spp. (Alič et al., 2017).\\\\nThe selected bacteriophage here, BF25/12, was characterized at the genome level. It showed no similarities to any previously described bacteriophages that are active against Dickeya spp. However, homologous bacteriophages deposited in GenBank were identified using BLAST and PASC analysis. The BF25/12 genome showed the highest similarity to bacteriophages active against Pectobacterium species characterized as members of the Podoviridae family, such as BF25/12. Pectobacterium bacteriophages did not share similarity to BF25/12 across the genome; rather, the homology was limited to sections of the genome. This lack of genetically similar bacteriophages can be explained on the basis that this is the first member of the Podoviridae family among bacteriophages active against Dickeya spp. Bacteriophage BF25/12 did not show significant similarities to any currently described and characterized Podoviridae genus at the genome level. Based on the genomic and morphologic characteristics, it is closest to genera in the Autographivirinae subfamily, which includes the T7 virus genus and the Escherichia coli model bacteriophage T7. The bacteriophage T7 virion consists of a head with a diameter of 55 nm and a 23-nm long tail (Steven and Trus, 1986). The size of the T7 virus particle corresponds to the measurements of the BF25/12 virion (head approximately 56 nm × 52 nm and tail 16 nm × 12 nm). The BF25/12 genome size of 43.8 kbp and the 52.2% GC content matches the small T7 bacteriophage genome (around 40 kbp and 48.4% GC content; Dunn et al., 1983). Furthermore, T7 and BF25/12 have an almost identical number of coding sequences in the genome—50 and 52 genes, respectively (Dunn et al., 1983). A genome analysis showed the suitability of BF25/12 for field-application studies due to its lack of genes coding for the bacteriophage lysogenic cycle and of genes encoding toxins or antibiotic resistance.\\\\nBacteriophage BF25/12 showed differences in activity against the UDL-3 and UDL-4 bacteria, which are genetically very similar bacterial isolates. Differences in the infection of these two isolates were observed under transmission electron microscopy, which resulted in some phenomena that have not been reported previously. The interactions of the bacteriophage with the UDL-4 bacteria activated granule degradation in the bacterial cells, which are similar to intracellular storage granules, with no bacterial lysis. The structure and function of these observed inclusions remain unknown, although a type of intracellular metabolic reserve is most likely. Some metabolic inclusions that function as a metabolic reserve are directly connected to the persistence of bacteria. Glycogen metabolism has been linked to bacterial survival in the environment, as well as to colonization and virulence (reviewed in Wilson et al., 2010). Moreover, in Pseudomonas oleovorans, it was shown that the metabolism of intracellular polyhydroxyalkanoates granules contributes to bacterial cell survival. The wild-type strain P. oleovorans, which is able to degrade intracellular polyhydroxyalkanoates, showed enhanced resistance to stress agents compared to the corresponding mutant strain incapable of the depolymerisation of polyhydroxyalkanoates (Ruiz et al., 2001). This observed granule degradation in UDL-4 cells might represent a bacteria stress response to the bacteriophage infection process, whereby the metabolic inclusions might enhance the environmental survival of the bacteria and therefore limit the development of the bacteriophage infection. However, a correlation between bacterial resistance to bacteriophage lysis and intercellular storage granule degradation has never been described.\\\\nSome of the main challenges in bacteriophage applications are environmental factors, including temperature, pH and UV radiation. Temperature can affect the ability of bacteriophages to lyse bacteria and their survival on the plant surface (Jones et al., 2007). Monitoring bacteriophage stability here revealed decreased bacteriophage titers at 28°C. The observed outcome corresponds to a study by Jepson and March (2004), who reported a decrease in liquid bacteriophage λ stock at 37°C over 120 days to under the limit of detection. However, the decline in the viability of bacteriophage BF25/12 was only 27.34% in 1 year, which indicates its applicability due to this appreciable stability at the implementation temperature. As expected, the bacteriophage titer at 4°C remained stable over the observed period. Bacteriophage BF25/12 belongs to the tailed bacteriophages that are known to be extremely viable at 4°C, and some can retain viability for up to 10–12 years (Jepson and March, 2004; Jończyk et al., 2011). Storing bacteriophages at -20°C is not recommended, as the formation of ice crystals can damage or destroy bacteriophage particles (Jończyk et al., 2011). Furthermore, suboptimal pH can inactivate bacteriophages; therefore, the stability and pH optimum of bacteriophages is very important in plant disease control applications (Jones et al., 2007). Bacteriophage BF25/12 was sensitive to acid pH and showed maximum stability in neutral and alkaline conditions. Only bacteriophages incubated in sterile demineralised water were stable at pH 11, indicating the influence of the incubation medium to bacteriophage stability at certain pH. It has been reported that buffer and salt concentration can affect bacteriophage activity (Chow et al., 1971; Burnet and Stanley, 2013). Chow et al. (1971) reported higher stability of bacteriophage Xp12 in Tris buffer at a higher pH; the effect is more prominent at higher ionic strengths. However, the effect was studied only for the pH range from 7 to 9. The most damaging environmental factor for bacteriophage stability is UV radiation (Jones et al., 2007), and, as reported for other phages, BF25/12 is highly susceptible to UV radiation (Jones et al., 2007).\\\\nThe occurrence of resistant bacteria mutants represents a major concern when using bacteriophages as biocontrol agents (reviewed in Jones et al., 2007). Here, the dependence between bacteriophage concentration and the development of bacterial resistance was optimum from log 4 to log 5 pfu/mL. These data indicate the importance of using suitable bacteriophage concentrations for bacteriophage treatments and the danger of using a single bacteriophage in any active treatment approach. The active bacteriophage used in this treatment undergoes extensive reproduction above the threshold needed to kill the bacteria through secondary infection (Payne et al., 2000). Extensive enrichment of bacteriophage-resistant bacteria might occur if the initial bacteriophage concentration used in any active approach is below the observed optimum for resistance development; for Dickeya spp. B16 and BF25/12, this was below log 4. However, the emergence of resistant bacteria during bacteriophage biocontrol can be reduced by the use of bacteriophage cocktails instead of individual bacteriophages (reviewed in Jones et al., 2007).\\\\nIn terms of the applicability of the bacteriophages defined in the present study, it will be necessary to determine the infection parameters (e.g., adsorption rate, one-step growth assay, burst size) to determine the potential effectiveness of this bacteriophage as a biocontrol. The combination of these isolated bacteriophages could in future form an intriguing bacteriophage cocktail due to the combination of the three different bacteriophage families with similar specificities. However, the biocontrol properties of the bacteriophages should be evaluated by pot or field experiment. The dynamic approach of the bacteriophage biocontrol agents requires extensive knowledge of a specific bacteria–bacteriophage system. However, bacteriophages can also be used in combined biocontrol management strategies, as they enable a temporary decrease in bacterial inoculum and therefore the greater efficiency of other substances used in bacterial disease control and treatment. Presently, isolated phages represent an important research tool that can provide adequate knowledge of bacteria–bacteriophage systems and dynamics, which is essential for the applicability of bacteriophages.\\\\nAuthor Contributions\\\\nConception and design of the experiments: ŠA, TN, TD, and MR. Performing experiments, analysis and interpretation of the data: ŠA, TN, MT-Ž, and TD. Provided CIM virus concentrates: NR. Drafting the work and writing the paper: ŠA, TN, MT-Ž, and TD. Revising the work critically for important intellectual content: ŠA, TN, MT-Ž, MP, MR, and TD. All authors read and approved the final manuscript.\\\\nFunding\\\\nThis study was supported financially by the Slovenian Research Agency (contracts P4-0165, L7-5534, 1000-15-0105) and the Centre of Excellence for Biosensors, Instrumentation and Process Control, an operation financed by the European Union, European Regional Development Fund, and Republic of Slovenia, Ministry of Higher Education, Science and Technology.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nThe authors would like to acknowledge Nika Janež, Andreja Petan (COBIK) and Špela Prijatelj Novak, Lidija Matièiè (NIB) for technical assistance, Denis Kutnjak (NIB) for his support in sequencing data analysis and Dr. Christopher Berrie for critical reading and English language editing. We thank Johan Van Vaerenbergh of the Institute for Agricultural and Fisheries Research for reference bacterial strains, Ian Toth and Sonia Humphries of the James Hutton Institute for the Dickeya sp. MK7 strain and Tomaž Jevšnik of Ocean Orchids for the orchid plant material.\\\\nReferences\\\\nAdriaenssens, E. M., Van Vaerenbergh, J., Vandenheuvel, D., Dunon, V., Ceyssens, P.-J., De Proft, M., et al. (2012). T4-related bacteriophage LIMEstone isolates for the control of soft rot on potato caused by “Dickeya solani.”. PLOS ONE 7:e33227. doi: 10.1371/journal.pone.0033227\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAlič, Š., Naglič, T., Tušek-Žnidarič, M., Peterka, M., Ravnikar, M., and Dreo, T. (2017). Putative new species of the genus Dickeya as major soft rot pathogens in Phalaenopsis orchid production. Plant Pathol. 66, 1357–1368. doi: 10.1111/ppa.12677\\\\nCrossRef Full Text | Google Scholar\\\\nAltschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990). Basic local alignment search tool. J. Mol. Biol. 215, 403–410. doi: 10.1016/S0022-2836(05)80360-2\\\\nCrossRef Full Text | Google Scholar\\\\nAziz, R. K., Bartels, D., Best, A. A., DeJongh, M., Disz, T., Edwards, R. A., et al. (2008). The RAST Server: rapid annotations using subsystems technology. BMC Genomics 9:75. doi: 10.1186/1471-2164-9-75\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBao, Y., Chetvernin, V., and Tatusova, T. (2014). Improvements to pairwise sequence comparison (PASC): a genome-based web tool for virus classification. Arch. Virol. 159, 3293–3304. doi: 10.1007/s00705-014-2197-x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBrettin, T., Davis, J. J., Disz, T., Edwards, R. A., Gerdes, S., Olsen, G. J., et al. (2015). RASTtk: a modular and extensible implementation of the RAST algorithm for building custom annotation pipelines and annotating batches of genomes. Sci. Rep. 5:8365. doi: 10.1038/srep08365\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBurnet, F. M., and Stanley, W. M. (2013). General Virology: Biochemical, Biological, and Biophysical Properties. Cambridge, MA: Academic Press.\\\\nGoogle Scholar\\\\nButtimer, C., McAuliffe, O., Ross, R. P., Hill, C., O’Mahony, J., and Coffey, A. (2017). Bacteriophages and bacterial plant diseases. Front. Microbiol. 8:34. doi: 10.3389/fmicb.2017.00034\\\\nCrossRef Full Text | Google Scholar\\\\nChow, T. Y., Lin, Y. T., and Kuo, T. T. (1971). Stability of phage Xp12. Bot. Bull. Acad. Sin. 12, 57–65.\\\\nGoogle Scholar\\\\nCzajkowski, R., Ozymko, Z., de Jager, V., Siwinska, J., Smolarska, A., Ossowicki, A., et al. (2015). Genomic, proteomic and morphological characterization of two novel broad host lytic bacteriophages ΦPD10.3 and ΦPD23.1 infecting pectinolytic Pectobacterium spp. and Dickeya spp. PLOS ONE 10:e0119812. doi: 10.1371/journal.pone.0119812\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Ozymko, Z., and Lojkowska, E. (2014). Isolation and characterization of novel soilborne lytic bacteriophages infecting Dickeya spp. biovar 3 (“D. solani”). Plant Pathol. 63, 758–772. doi: 10.1111/ppa.12157\\\\nCrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Pérombelon, M. C. M., van Veen, J. A., and van der Wolf, J. M. (2011). Control of blackleg and tuber soft rot of potato caused by Pectobacterium and Dickeya species: a review. Plant Pathol. 60, 999–1013. doi: 10.1111/j.1365-3059.2011.02470.x\\\\nCrossRef Full Text | Google Scholar\\\\nDunn, J. J., Studier, F. W., and Gottesman, M. (1983). Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements. J. Mol. Biol. 166, 477–535.\\\\nPubMed Abstract | Google Scholar\\\\nJończyk, E., Kłak, M., Mibbbedzybrodzki, R., and Górski, A. (2011). The influence of external factors on bacteriophages–review. Folia Microbiol. 56, 191–200. doi: 10.1007/s12223-011-0039-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJones, J. B., Jackson, L. E., Balogh, B., Obradovic, A., Iriarte, F. B., and Momol, M. T. (2007). Bacteriophages for plant disease control. Annu. Rev. Phytopathol. 45, 245–262. doi: 10.1146/annurev.phyto.45.062806.094411\\\\nCrossRef Full Text | Google Scholar\\\\nJones, J. B., Vallad, G. E., Iriarte, F. B., Obradović, A., Wernsing, M. H., Jackson, L. E., et al. (2012). Considerations for using bacteriophages for plant disease control. Bacteriophage 2, 208–214. doi: 10.4161/bact.23857\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKropinski, A. M., Mazzocco, A., Waddell, T. E., Lingohr, E., and Johnson, R. P. (2009). Enumeration of bacteriophages by double agar overlay plaque assay. Methods Mol. Biol. 501, 69–76. doi: 10.1007/978-1-60327-164-6_7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nOverbeek, R., Olson, R., Pusch, G. D., Olsen, G. J., Davis, J. J., Disz, T., et al. (2014). The SEED and the rapid annotation of microbial genomes using subsystems technology (RAST). Nucleic Acids Res. 42, D206–D214. doi: 10.1093/nar/gkt1226\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nParkinson, N., Stead, D., Bew, J., Heeney, J., Tsror Lahkim, L., and Elphinstone, J. (2009). Dickeya species relatedness and clade structure determined by comparison of recA sequences. Int. J. Syst. Evol. Microbiol. 59, 2388–2393. doi: 10.1099/ijs.0.009258-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPayne, R. J., Phil, D., and Jansen, V. A. (2000). Phage therapy: the peculiar kinetics of self-replicating pharmaceuticals. Clin. Pharmacol. Ther. 68, 225–230. doi: 10.1067/mcp.2000.109520\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPickard, D. (2009). “Preparation of bacteriophage lysates and pure DNA,” in Bacteriophages Methods in Molecular BiologyTM, eds M. J. Clokie and A. Kropinski (Berlin: Humana Press), 3–9.\\\\nGoogle Scholar\\\\nRuiz, J. A., López, N. I., Fernández, R. O., and Méndez, B. S. (2001). Polyhydroxyalkanoate degradation is associated with nucleotide accumulation and enhances stress resistance and survival of Pseudomonas oleovorans in natural water microcosms. Appl. Environ. Microbiol. 67, 225–230. doi: 10.1128/AEM.67.1.225-230.2001\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSamson, R., Legendre, J. B., Christen, R., Saux, M. F.-L., Achouak, W., and Gardan, L. (2005). Transfer of Pectobacterium chrysanthemi (Burkholder et al., 1953) Brenner et al., 1973 and Brenneria paradisiaca to the genus Dickeya gen. nov. as Dickeya chrysanthemi comb. nov. and Dickeya paradisiaca comb. nov. and delineation of four novel species, Dickeya dadantii sp. nov., Dickeya dianthicola sp. nov., Dickeya dieffenbachiae sp. nov. and Dickeya zeae sp. nov. Int. J. Syst. Evol. Microbiol. 55, 1415–1427. doi: 10.1099/ijs.0.02791-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSoleimani-Delfan, A., Etemadifar, Z., Emtiazi, G., and Bouzari, M. (2015). Isolation of Dickeya dadantii strains from potato disease and biocontrol by their bacteriophages. Braz. J. Microbiol. 46, 791–797. doi: 10.1590/S1517-838246320140498\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSteven, A. C., and Trus, B. L. (1986). “The structure of bacteriophage T7,” in Electron Microscopy of Proteins, Viral Structure, eds J. R. Harris and R. W. Horne (London: Academic Press), 1–33.\\\\nGoogle Scholar\\\\nSteyer, A., Gutiérrez-Aguirre, I., Rački, N., Beigot Glaser, S., Brajer Humar, B., Stražar, M., et al. (2015). The detection rate of enteric viruses and Clostridium difficile in a waste water treatment plant effluent. Food Environ. Virol. 7, 164–172. doi: 10.1007/s12560-015-9183-7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTian, Y., Zhao, Y., Yuan, X., Yi, J., Fan, J., Xu, Z., et al. (2016). Dickeya fangzhongdai sp. nov., a plant-pathogenic bacterium isolated from pear trees (Pyrus pyrifolia). Int. J. Syst. Evol. Microbiol. 66, 2831–2835. doi: 10.1099/ijsem.0.001060\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nToth, I. K., van der Wolf, J. M., Saddler, G., Lojkowska, E., Hélias, V., Pirhonen, M., et al. (2011). Dickeya species: an emerging problem for potato production in Europe. Plant Pathol. 60, 385–399. doi: 10.1111/j.1365-3059.2011.02427.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nVan Vaerenbergh, J., Baeyen, S., De Vos, P., and Maes, M. (2012). Sequence diversity in the Dickeya fliC Gene: phylogeny of the Dickeya genus and TaqMan® PCR for “D. solani”, new biovar 3 variant on potato in Europe. PLOS ONE 7:e35738. doi: 10.1371/journal.pone.0035738\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWilson, W. A., Roach, P. J., Montero, M., Baroja-Fernández, E., Muñoz, F. J., Eydallin, G., et al. (2010). Regulation of glycogen metabolism in yeast and bacteria. FEMS Microbiol. Rev. 34, 952–985. doi: 10.1111/j.1574-6976.2010.00220.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00627\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00627/full\",\\n    \"loadedTime\": \"2024-01-23T08:49:03.307Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00627\",\\n    \"title\": \"Frontiers | Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization\",\\n    \"description\": \"Composting is a complex biodegradable process that converts organic materials into nutrients to facilitate crop yields, and, if well managed, can render bactericidal effects. Majority of research focused on detection of enteric pathogens, such as Shiga toxin-producing E. coli (STEC) in fecal composts. Recently, attention has been emphasized on bacteriophages, such as STEC-specific bacteriophages, associated with STEC from the fecal-contaminated environment because they are able to sustain adverse environment during composting process. However, little is known regarding the isolation of STEC-specific bacteriophages in non-fecal composts. Thus, the objectives were to isolate and genomically characterize STEC-specific bacteriophages, and to evaluate its association with STEC in non-fecal composts. For bacteriophage isolation, the samples were enriched with non-pathogenic E. coli (3 strains) and STEC (14 strains), respectively. After purification, host range, plaque size, and phage morphology were examined. Furthermore, bacteriophage genomes were subjected to whole-genome sequencing using Illumina MiSeq and genomic analyses. Isolation of top six non-O157 and O157 STEC utilizing culture methods combined with PCR-based confirmation was also conducted. The results showed that various STEC-specific bacteriophages, including vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw and vB_EcoM-Ro157lw, with different but complementary host ranges were isolated. Genomic analysis showed the genome sizes varied from 42kb to 149kb, and most bacteriophages were unclassified at the genus level, except vB_EcoM-Ro111lw as FelixO1-like viruses. Prokka predicted less than 25% of the ORFs coded for known functions, including those essential for DNA replication, bacteriophage structure, and host cell lysis. Moreover, none of the bacteriophages harbored lysogenic genes or virulence genes, such as stx or eae. Additionally, the presence of these lytic bacteriophages was likely attributed to zero isolation of STEC and could also contribute to additional antimicrobial effects in composts, if the composting process was insufficient. Current findings indicate that various STEC-specific bacteriophages were found in the non-fecal composts. In addition, the genomic characterization provides in-depth information to complement the deficiency of biological features regarding lytic cycle of the new bacteriophages. Most importantly, these bacteriophages have great potential to control various serogroups of STEC.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nIn recent years, produce outbreaks associated with contamination of enteric pathogens, such as Shiga toxin-producing Escherichia coli (STEC), have drawn significant attention of health administrations and the public. During 2011, a severe outbreak in Europe was associated with contamination of fenugreek sprouts with E. coli O104:H4. There were approximately 3800 cases reported in Germany alone, including 845 cases of hemolytic uremic syndrome and 54 deaths (European Food Safety Authority, 2011; Frank et al., 2011; Beutin and Martin, 2012). Recently, a massive recall occurred in the United States regarding contamination of romaine lettuce with E. coli O157:H7 in 22 states, causing illness in 98 people, 46 hospitalizations, and 10 cases of hemolytic uremic syndrome (CDC, 2018). Produce is often served raw and, thus, is susceptible to any contamination from farm to fork. Contamination of pathogenic STEC strains on produce firstly occurs on the farm mostly derived from the contact with fecal-contaminated sources, such as wild animal, irrigation water and composts (Walters et al., 2011; Cooley et al., 2013).\\\\nComposting is a biodegradable process by microorganisms to turn organic materials to humus-like components that are commonly used as fertilizers or soil amendment to improve crop yields (Ryckeboer et al., 2003). An efficiently managed composting process would attribute to the reduced levels of pathogens, such as Salmonella and pathogenic E. coli, first present in the raw organic materials after the process due to the change of temperature and pH (Ryckeboer et al., 2003; Heringa et al., 2010). Inadequate composting processes have resulted in the spread of foodborne pathogens in pre-harvest environment (Buck et al., 2003), which are commonly associated with the composting of organic material with higher levels of pathogens, particularly of fecal origin (Handley et al., 2012). A previous study demonstrated that Salmonella and E. coli O157:H7 were able to survive through the niche of composting process and sustain at low levels in dairy-based compost (Kim et al., 2009). Another study further indicated that the use of E. coli O157:H7-polluted compost for soil amendment was directly associated with contamination of outer lettuce leaves during irrigation (Oliveira et al., 2012). Due to the natural reservoir of STEC, additional interventions have been used to minimize the pathogens in feces-based composts prior to use to prevent the spread of the pathogens (Lemunier et al., 2005; Heringa et al., 2010).\\\\nBacteriophages (or phages) are key elements of intestinal microbiota of animals (Handley et al., 2012). Previous study indicated that various STEC-specific phages that were lytic against different serogroups of STEC strains, such as O26, O111 and O157, were isolated from cattle feces (Viazis et al., 2011). The authors found that these phages were free of virulence genes, such as stx, hlyA, and eaeA genes, and were potential biocontrol candidates to prevent STEC contamination. Another study, however, found that Stx phages – the phage containing stx genes – with lysogenic factors were present in the environment contaminated with animal feces and might be associated with emergence of new pathogens through the transfer of the virulence gene by the phages (Martinez-Castillo and Muniesa, 2014). Additionally, phages have been demonstrated to be able to be present at different stages of composting process (Johannessen et al., 2005; Partanen et al., 2010). The phages specific to Pseudomonas were isolated and characterized in composts (Amgarten et al., 2017); however, the information regarding the presence of STEC-specific phage or Stx phages in the composts, particularly non-fecal bases, is scarce. It is believed that the presence of STEC-specific phages or Stx phages (temperate phages) might have different impact on the biosafety level of compost upon application. Therefore, the objectives of this study were to isolate and genomically characterize free STEC-specific bacteriophages, and to evaluate its association with STEC strains in the non-fecal composts. To the best of the authors’ knowledge, this is the first study to isolate and characterize STEC-associated phages in non-fecal composts.\\\\nMaterials and Methods\\\\nSample Collection\\\\nTwo non-fecal compost samples (C1 and C2) were randomly collected from a composting operation in California. The raw ingredients used for the composting were consisted of food scraps, from restaurants and markets, and yard trimmings.\\\\nIsolation of STEC Strains\\\\nUsing an abridged version of the protocol outlined in Cooley et al. (2013), samples were enriched and streaked on MacConkey Sorbitol Agar supplemented with cefixime and potassium tellurite (CT-SMAC), Rainbow Agar O157 supplemented with novobiocin and potassium tellurite (NT-RA), BBL Blood Agar supplemented with calcium chloride, mitomycin and X-gal (mSBA), and Chromagar O157. Typical colonies of STEC found on each type of media were picked and grown overnight in Luria Broth (LB) at 37°C and plated on Luria Agar. DNA from each isolated colony streaked from the two enriched samples was extracted and screened for stx1, stx2abc, stx2f and stx2ex using multiplex PCR (hot start at 95°C for 10 min, 38 cycles of 95°C for 20 s and 60°C for 45 s) with 2.5 units of AmpliTAQ gold, 1× supplied PCR buffer, 3 mM MgCl2, 200 mM dNTP, 300 nM of each primer, and 200 nM each probe to make a 20 μl reaction (Cooley et al., 2013). An E. coli O157:H7 (ATCC 35150) positive control and a Salmonella Typhimurium (ATCC 14028) negative control were included in each run. A Ct value of ≤ 28 for any of the four stx genes was considered to be a positive result.\\\\nPreparation of Bacterial Host Strains for Phage Isolation\\\\nFourteen STEC strains, including 2 strains per serogroups of O26, O45, O103, O111, O121, O145, and O157, from the culture collections of Produce Safety and Microbiology Research Unit in USDA ARS were used as host strains in this study (Supplementary Table S1). Non-pathogenic strains, including ATCC 13706, ATCC 43888, and DH5α, were used for the isolation of phages encoding stx genes (or Stx phages). Overnight cultures were prepared by inoculating each non-pathogenic and STEC strain in an individual tube with 5 ml of Tryptic Soy Broth (TSB, Difco, Becton Dickinson, Sparks, MD, United States) and incubated overnight at 37°C. The cocktails of non-pathogenic and STEC strains were prepared separately by adding 0.1 ml of each overnight culture from each group together shortly prior to use.\\\\nIsolation of Free Bacteriophages\\\\nTwo non-fecal composts in 50 ml conical tubes were gently mixed by hand prior to centrifugation at 8,000 × g for 10 min to get rid of sediments. The supernatants were filtered through 0.22 μm syringe filter membrane, to get rid of background flora, and were used for phage isolation. In brief, an aliquot of 5 ml filtered supernatant from each sample was mixed with 25 ml TSB supplemented with 10 mM CaCl2 and inoculated with 0.3 ml of respective STEC and non-pathogenic E. coli cocktails for incubation at 37°C for 48 h. Next, chloroform (4% v/v) was added to the enriched samples and homogenized on a rotator (Thermo Fisher Scientific, Waltham, MA, United States) for 30 min to inactivate the residual host strains and background flora. The supernatants were obtained after centrifugation at 8,000 × g for 10 min and used to determine the presence of the phages against top six non-O157 STEC and O157 STEC strains by spotting 10 μl supernatants on Tryptic Soy Agar (TSA, Difco, Becton Dickinson, Sparks, MD, United States) containing one of the host strains (spot assay). After overnight incubation at 37°C, the plates with clearing on the spotted area, which indicated positive results, were further subjected to phage purification process as previously described (Liao et al., 2018). During purification process, two plaques with different plaque sizes were picked from the first single-layer agar plaque assay plates, and three runs of plaque-picking and enrichment process were conducted (Liao et al., 2018). After purification, each phage lysate was enriched in 40 ml TSB with the corresponding host culture at 37°C overnight for phage propagation. The propagated phages were filtered through 0.22 μm syringe filter membrane prior to further analysis.\\\\nDetermination of Host Range and Antimicrobial Effects Using Efficiency of Plating for the Isolated Phages\\\\nTo determine host range and antimicrobial effects of the isolated phages, the spot assay was used as described above to spot the lysate of the isolated phages on the TSA plates pre-mixed with individual strain of the selected host panel, the same group as those used for phage isolation, and the level of the clearing (opaque, transparent and complete clearing) on the spotted area was used to determine the lysis ability of the phages against different serogroups of STEC. Efficiency of plating (EOP) was used to evaluate the lytic activity of the phages against a variety of bacterial strains. In brief, fresh overnight cultures of the tested bacterial strains and the primary bacterial strain used for phage isolation were prepared in TSB at 37°C. After serial dilution, the diluted phage solutions (10-3 to 10-7) were used to obtain titers with each strain using a double-layer agar plaque assay, and the plates were incubated at 37°C overnight. EOP was calculated based on phage titer obtained from the tested bacteria versus the phage titer from the primary bacterium used for phage isolation. If EOP was 0.5 or more, it was considered as high production efficiency. An EOP of 0.1 or above, but below 0.5, was considered as medium production efficiency. Low production efficiency had EOP between 0.001 and 0.1, and any value equal to or under 0.001 represented inefficiency of phage production.\\\\nDetermination of Plaque Morphology of the Isolated Phages\\\\nA stereo microscope (Stemi 305 cam, Carl Zeiss Microscopy, Germany) was used to observe plaque morphology from the double-layer agar plaque assay plates. The images of plaques were captured by use of Zeiss Labscope Application (Version 2.5.1).\\\\nConcentration of Phages for Downstream Analysis\\\\nThe filtered lysates were concentrated using Amicon Ultra-15 Centrifugal Filter Unit (Merck Millipore, Billerica, MA, United States) with 100 KDA cut-off. An aliquot of 10 ml phage lysate was added to Amicon filter unit, which was subsequently centrifuged using a fixed angle rotor at 5,000 × g according to manufacturer’s instruction. The filter unit was then washed with addition of 200 μl sterile H2O and centrifuged at 5,000 × g to get rid of the water, followed by adding 200 μl of 10 μg/ml DNase I and incubating at 37°C for 30 min to eliminate bacterial DNA contamination. After spinning down the DNase I, the filter was then washed with 5 ml of sterile H2O with the procedure as previously described. Subsequently, an aliquot of 500 μl of sterile H2O was used to re-suspend the phages retained on the filter membrane, and the phage solution was transferred to a sterile microcentrifuge tube. Twenty microliters of the concentrated phages were spared for electron microscopy observation, while the rest of the lysates were used for DNA extraction.\\\\nTransmission Electron Microscopy\\\\nEach Amicon Filter-concentrated phage lysate with volume of 6 μl was placed on copper mesh PLECO grids (Ted Pella Inc., Redding, CA, United States) and incubated for 1 min at room temperature, followed by blotting on Whatman filter paper and subsequent negative staining with addition of 8 μl of 0.75% uranyl acetate (Sigma-Aldrich, Darmstadt, Germany) for incubation at room temperature (around 26°C) for 30 s. The phage morphology was then examined using a transmission electron microscope (Tecnai G2 F20 model FEI, United States).\\\\nDetection of stx Genes in the Phages\\\\nThe concentrated phage lysates were then subjected to DNA extraction using a phage DNA extraction kit (Norgen Biotek Corp., ON, Canada). Presence of stx1 and stx2 genes in the phages was screened by PCR as previously described (Liao et al., 2018).\\\\nWhole Genome Sequencing and Genomic Analyses\\\\nPrior to library construction, the purity and concentration of each phage’s DNA was determined by electrophoresis on 1% agarose gel and Nanodrop 8000, respectively. Two hundred nanograms of DNA were fragmented to 550 bp by Ultrasonic Cell Disruptor (M220 Covaris) and subsequently subjected to DNA library construction using a TruSeq® Nano DNA Library Prep Kit (Illumina, San Diego, CA, United States) according to manufacturer’s instruction. The libraries were sequenced on Illumina MiSeq sequencer using a MiSeq reagent kit v3 (Illumina, 600 cycles) with approximately 2 million paired-end (250-bp) sequence reads. The sequenced reads were subjected to FastQC, and the raw sequences were then trimmed using Trimmomatic on Galaxy online server1. A de novo assembly of the resultant quality reads was conducted using Unicycler v0.4.1 (SPAdes) with default parameters on Galaxy. The assembled sequences were subjected to NCBI (National Center for Biotechnology Information) BLAST to obtain taxonomic classification, including order, family, and genus. The phage termini and possible packaging mechanisms were predicted according to the in silico determination method proposed in PhageTerm (Garneau et al., 2017). The final contigs were annotated with Prokaryotic genome annotation (Prokka, v.1.12.0) with default parameters (Seemann, 2014). The open reading frames (ORFs) predicted and annotated by Prokka were further subjected to BLASTN search to confirm the annotation results. The reference phage genomes used in this study, including Escherichia phage ECML-117 (GenBank accession number JX128258.1), Escherichia phage ESCO13 (GenBank accession number KX552041.2), Escherichia phage K1ind1 (GU196279.1), and E. coli O157 typing phage 12 (GenBank accession number KP869110.1) were obtained based on the maximum score of the published sequences on NCBI nucleotide database after the BLASTN search using the assembled sequences of the new phages from this study. A visualized circular map with the predicted ORFs for each of vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw genomes were constructed using Geneious v11.0.4 (Biomatters Ltd., Auckland, New Zealand). The comparative analyses of phage nucleotide sequences, between the newly isolated phages and the corresponding reference phages, were conducted using EasyFig (Sullivan et al., 2011). ClustalW algorithm v2.1 (McWilliam et al., 2013) and the neighbor-joining method was used to evaluate evolutionary relationship of the phages based on the genes of interest (Saitou and Nei, 1987). The evolutionary distances were computed using the Maximum Composite Likelihood method (Tamura et al., 2004) and were in the units of the number of base substitutions per site. The analysis involved 4 nucleotide sequences. All positions containing gaps and missing data were eliminated. There was a total of 955 positions in the final dataset. Evolutionary analyses were conducted in MEGA7 (Kumar et al., 2016). The whole-genome sequences of Escherichia phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw were deposited to NCBI GenBank under accession number MH571750, MH160766, MH051334, and MH051335, respectively.\\\\nResults and Discussion\\\\nIsolation of STEC Strains and STEC-Specific Phages From the Composts\\\\nFor STEC isolation, a total of 17 and 23 presumptive positive colonies were picked from selective media after incubation of streak-plated enriched C1 and C2 samples, respectively. None of the colonies was positive of stx genes by PRC screening (Table 1). The results without isolation of STEC strains were possibly due to non-fecal materials of the compost samples used in this study.\\\\nTable 1. Isolation of STEC strains and STEC-specific bacteriophages from non-fecal composts.\\\\nWith regard to phage isolation, the spot assay results showed that both C1 and C2 contained phages lytic against the same serogroups of STEC strains, including O111, O121, O145, and O157, regardless of lysis levels on the plates. Additionally, the phages against non-pathogenic E. coli strains, including DH5α, ATCC 13706, and ATCC 43888 (E. coli O157:H7 without stx genes), were also detected by spot assay in the C2 (Table 1). The non-pathogenic hosts were used primarily for the isolation of Stx phages from the samples; however, after phage purification, none of these phages, including both STEC-specific phages and those lytic against non-pathogenic strains, were positive of stx genes, indicating that no Stx phages were isolated from the non-fecal composts (data not shown). Based on degree of lysis from spot assay, four phages, one of each against serogroup of O111, O121, O145, and O157 strains, were selected for further biological and genomic analyses. The four phages, including Escherichia phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw, were named after the phage morphology, the species name and serogroup of the major bacterial host accordingly (Adriaenssens and Brister, 2017). The TEM observation showed that vB_EcoS-Ro145lw belonged to the family Siphoviridae, while the other 3 phages were Myoviridae (Figure 1B). Furthermore, vB_EcoS-Ro145lw produced the largest size of plaques among all phages, indicating its strong lytic effect against its host strain, STEC O145 (Figure 1A). The host range test indicated that, in addition to generic E. coli, such as ATCC 13706 and DH5α strains, both phages vB_EcoM-Ro111lw and vB_EcoM-Ro121lw were lytic (antimicrobial) against multiple serogroups of STEC strains, whereas vB_EcoS-Ro145lw and vB_EcoM-Ro157lw had narrow host ranges (Table 2). In addition, EOP results revealed that vB_EcoM-Ro111lw was the only phage showing medium or high infection efficiency against different serogroups (O26 and O111) of STEC strains.\\\\nFigure 1. The morphology of plaques formed by phage vB_EcoM-Ro111lw (A1), phage vB_EcoM-Ro121lw (A2), phage vB_EcoS-Ro145lw (A3), and phage vB_EcoM-Ro157lw (A4) on their host strains of STEC O111, O121, O145, and O157, respectively, using plaque assay, and the transmission electron microscopy images of phage vB_EcoM-Ro111lw (B1), phage vB_EcoM-Ro121lw (B2), phage vB_EcoS-Ro145lw (B3), and phage vB_EcoM-Ro157lw (B4).\\\\nTable 2. Host range and antimicrobial effects of the isolated bacteriophages against various STEC and non-pathogenic strains by spot assayα and EOPβ (efficiency of plating).\\\\nA complete composting process is capable of reducing a significant amount of pathogens present in the raw organic materials (Erickson et al., 2009). However, if the composting organic materials containing high level of enteric pathogens, such fecal matters, was inefficiently managed, the final compost product may pose a risk of the spread of pathogens on a farm (Wu et al., 2009). Phages have been demonstrated to be more resistant to the environmental stress, such as the drastic change of pH and temperature, than bacteria and, thus, are likely to sustain through the composting process (Johannessen et al., 2005). Current findings revealed that various STEC-specific phages were isolated from the composting of non-fecal materials, such as yard trimmings or food scraps. Although the initial levels of pathogens in the raw materials prior to composting were unknown, the presence of STEC-specific phages was likely to add additional antimicrobial factors to reduce STEC to non-detectable levels during the process. Shiga toxin-producing E. coli (STEC)-specific phages were usually isolated from feces or fecal-contaminated environment (Imamovic et al., 2010; Wang et al., 2015; Amarillas et al., 2016); however, current study reported the presence of STEC-specific phages and evaluated its correlation with the STEC bacterial hosts from non-fecal composts.\\\\nGenomic Features of Four STEC-Specific Phages\\\\nThe genome sizes of phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw were 86,950bp, 149,803bp, 42,145bp, and 72,179bp, respectively (Table 3). According to taxonomy report of BLASTN search at NCBI nucleotide database, the phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, and vB_EcoM-Ro157lw belonged to the family Myoviridae and the vB_EcoS-Ro145lw belonged to Siphoviridae; the taxonomic results were in consensus to the morphologies observed under TEM of these phages. Additionally, the phage vB_EcoM-Ro111lw was further classified under sub-family Ounavirinae as FelixO1-like viruses; however, the rest of 3 phages were unclassified at genus level. The three myophages isolated in this study had genome sizes ranging from 72 k to 149 k bp in which the results were in agreement with the finding of the previous study indicating a wide range of genome sizes of the phages belonging to Myoviridae were isolated from cattle feces (Wang et al., 2015). The GC content of phage vB_EcoS-Ro145lw (50.6%) was not only higher than that of other new phages, with 46, 39.1, and 38.8% for vB_EcoM-Ro157lw, vB_EcoM-Ro121lw, and vB_EcoM-Ro111lw, respectively, but also close to that of E. coli strains with an average GC content of 50%. A previous study found that lytic phages usually had an average GC content 4% lower than that of its bacterial host, whereas temperate phages (with both lytic and lysogenic cycles) had GC content very close to the host (Rocha and Danchin, 2002). Although the GC content of phage vB_EcoS-Ro145lw was similar to that of its O145 STEC host strain, the phage expressed strong lysis, such as forming big plaques against the strain, without having any sign of lysogenic factor. On the other hand, the GC contents of both phages vB_EcoM-Ro121lw and vB_EcoM-Ro111lw were significant lower than that of their E. coli hosts (∼50.7%), suggesting the host strains, O121 and O111 STEC, used for the phage isolation in this study were not the optimal hosts for these two phages (Bahir et al., 2009). The small plaque sizes formed by phages vB_EcoM-Ro121lw and vB_EcoM-Ro111lw against its original strains of isolation –O121 and O111 STEC strain, respectively – might correlate with these findings.\\\\nTable 3. Genomic features of the STEC-specific bacteriophages from non-fecal composts.\\\\nPrediction of Genome Termini and Packaging Mechanisms\\\\nThe phage termini and possible packaging mechanism of the isolated phages were predicted according to the bioinformatic algorithm proposed in PhageTerm (Garneau et al., 2017). The analytical results indicated that both phages vB_EcoS-Ro145lw and vB_EcoM-Ro121lw had circularly permuted direct terminal repeats (without cohesive ends) with the packing mechanism similar to the well-known phage P1: headful with pac site (Oliveira et al., 2013). In the headful packaging mode, the terminase would recognize and cleave at a unique sequence region, called pac, to generate the first termini and initiate the packaging of the genomic DNA into the prohead (Li et al., 2014). The packing mechanisms for both phages vB_EcoM-Ro111lw and vB_EcoM-Ro157lw were predicted to be direct terminal repeats (DTR) at the genome termini without circular permutation: vB_EcoM-Ro111lw had a 613-bp terminal repeat region (similar to the phage T7, short direct terminal repeats) and vB_EcoM-Ro157lw had a 3533-bp terminal repeat region at both ends of the genome (similar to the phage T5, long direct terminal repeats) (Casjens and Gilcrease, 2009). Generally, in the mode of terminal repeats, the terminal sequences were generated by duplicating the direct repeat DNA sequence together with packaging; however, the mechanism regarding the generation of long repeats was unknown (Casjens and Gilcrease, 2009). Although the principles of several well-known packaging mechanisms were elucidated, identification of these packaging mechanisms from newly isolated phages traditionally require generation of restriction maps that are usually complicated with results not easy to interpret (Li et al., 2014). With the benefits of the computation of whole genome sequencing data using PhageTerm software, it was able to easily predict the terminal ends and types of the packaging mechanisms of these new phages (Garneau et al., 2017).\\\\nGenome Analysis of Four STEC-Specific Phages\\\\nThe BLASTN searches of the new phage genomes in this study against the reference sequences on NCBI database varied among different phages. The two best hits for vB_EcoS-Ro145lw genome included Escherichia phage P AB-2017 (67% coverage and 90% identity) and Escherichia phage K1ind1 (70% coverage and 89% identity); the low genome coverage of the new phages against its reference genomes indicated the difference of the phage as opposed to the known genomes among Siphoviridae phages from the public database. The BLASTN search results also showed that vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, and vB_EcoM-Ro157lw had the best hits against E. coli O157 typing phage 12, 11, and 8 (all with 95% coverage and 98% identity), Escherichia phage ESCO13 (98% coverage and 99% identity), and Escherichia phage ECML-117 (94% coverage and 97% identity), respectively. Although the phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, and vB_EcoM-Ro157lw had higher sequence coverage of genetic similarity than that of vB_EcoS-Ro145lw against their reference sequences, vB_EcoM-Ro111lw was the only phage further classified at genus level, FelixO1-like phages, based on NCBI taxonomy report. However, the phage vB_EcoM-Ro111lw was unable to infect Salmonella, unlike the Salmonella phage Felix O1, the core phage in the genus, with capability against different serovars of Salmonella strains (Whichard et al., 2010). BLAST comparison of the newly isolated phages and their reference phage genomes using EasyFig showed that that the genomes of vB_EcoM-Ro111lw and E. coli O157 typing phage 12 shared approximately over 90% nucleotide sequence similarity. However, one annotated ORF, coding for hypothetical protein, in phage vB_EcoM-Ro111lw was absent in the genome of E. coli O157 typing phage 12; two ORFs from the reference genome were lacking in vB_EcoM-Ro111lw (Figure 2A). The results also showed a high degree of nucleotide sequence similarity (> 90%) between the phages vB_EcoM-Ro121lw its reference genome – Escherichia phage ESCO1 – with 3 unshared ORFs, primarily coding for hypothetical proteins, from each other (Figure 2B). For vB_EcoS-Ro145lw genome, the results showed that low degree of nucleotide sequence similarity (68%) was observed on some shared ORFs as compared to the phage K1ind1 (Figure 2C). Additionally, there were 6 ORFs in vB_EcoS-Ro145lw that were absent in the counterpart genome and 3 ORFs from the reference genome that were not shared in the genome of vB_EcoS-Ro145lw. The comparison between the phage vB_EcoM-Ro157lw and Escherichia phage ECML-117 indicated a high nucleotide sequence similarity between the two genomes and all the annotated ORFs from the reference phage genome were shared in the vB_EcoM-Ro157lw; five ORFs in vB_EcoM-Ro157lw that coded for hypothetical proteins were absent in the reference phage (Figure 2D). These results showed that the genomes of the new phages and their counterparts contained some sequence regions with ORFs that were identified with the predicted hypothetical proteins in high similarity but in different organization in the genomes. The phenomenon, also known as mosaicism, likely deriving from horizontal genetic transfer may account for the diversity of phage genomes (Hatfull and Hendrix, 2011).\\\\nFigure 2. Whole genome comparison of Escherichia phage vB_EcoM-Ro111lw (A), Escherichia phage vB_EcoM-Ro121lw (B), Escherichia phage vB_EcoS-Ro145lw (C), and Escherichia phage vB_EcoM-Ro157lw (D) with their reference genomes from NCBI database using EasyFig. A gray-scaled shaded area indicates the different levels of sequence similarity between two phage sequences. The arrow shapes indicate the annotated ORFs in the genome, and the unshared ORFs are highlighted as red arrows.\\\\nAccording to genome annotation, the predicted number of ORFs for each of phage vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw were 153, 292, 71, and 109, respectively (Table 3). There were more than 75% of the ORFs annotated with unknown function, even including the FelixO1-like phage vB_EcoM-Ro111lw. The phage genomes that contained the ORFs with the predicted functions essential for phage structure, DNA metabolism and replication host lysis proteins, and host metabolism and regulation were illustrated with different colors in Figure 3. Additionally, only vB_EcoM-Ro111lw and vB_EcoM-Ro121lw contained tRNAs, with 22 and 10, respectively. Five of the tRNAs, #2, #10, #18, #20, and #21, in the vB_EcoM-Ro111lw genome and one, #4, in vB_EcoM-Ro121lw genome may be pseudo-tRNAs (Supplementary Table S2). Most tRNAs in these two phage genomes were located in the region adjacent to the DNA packing cluster, which might be associated with earlier stage of phage progenies formation (Bailly-Bechet et al., 2007). Bailly-Bechet et al. (2007) also indicated that lytic phages would contain more tRNAs, if present, than lysogenic phages. Moreover, it was demonstrated that the presence of tRNAs was more likely associated with the fitness of the phages, such as ability to infect more hosts, in order to adapt to different environment (Delesalle et al., 2016). Their findings were in consensus to the results of wide host ranges for the phages vB_EcoM-Ro111lw and vB_EcoM-Ro121lw in this study. In the genome of vB_EcoM-Ro157lw, the ORF #41 was predicted with the function of soluble lytic murein transglycosylase, which had the similar function as lysozyme/endolysin to catalyze the cell wall of its bacterial hosts, causing cell lysis (van Asselt et al., 1999), while the ORF #12, ORF #81 and ORF #11 were predicted as lysozyme/endolysin in the vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, and vB_EcoS-Ro145lw genomes, respectively (Supplementary Table S3–S6). Furthermore, in vB_EcoM-Ro111lw genome, two ORFs were annotated as putative dihydrofolate reductase (#77) and thymidylate synthase (#76) that were found to be the unique molecular markers for Bastille-like phages, a newly proposed genus of myophages in sub-family Spounavirinae, which were also known for the wide host ranges (Asare et al., 2015). The phage vB_EcoM-Ro111lw was classified as Felix O1-like phages according to NCBI taxonomy report, even though it was predicted to harbor the ORFs coded these two key markers, dihydrofolate reductase and thymidylate synthase, from other genus, Bastille-like phages, which demonstrated the evolutionary perspective of the phages likely through horizontal genetic transfer in the environment. A previous study found that a Salmonella-specific phage, UAB_Phi20, with detection of lysogenic genes still showed strong lytic infection against its Salmonella hosts; however, the lack of lysogenic cycle was derived from the suppression of the phage’s packaging mechanism, resulting in the dominant lytic cycle of the phage (Bardina et al., 2016). Though expression of strong lytic cycle, the presence of lysogenic genes in the phages may not be appropriate for the potential biocontrol use. In the current study, the genomic analysis showed that no lysogenic or virulence genes, such as stx or eae genes, were detected in the genomes of these four newly isolated phages.\\\\nFigure 3. Circular genome map of the phages vB_EcoM-Ro111lw (A), vB_EcoM-Ro121lw (B), vB_EcoS-Ro145lw (C), and vB_EcoM-Ro157lw (D). The outer circle represents genes as indicated by the arrows. Predicted functional categories of the genes were determined according to annotation and are represented with different colors. The central graph in blue shows genomic GC content variation.\\\\nComparative Genome Analysis\\\\nIt was hard to create a cluster of genes based on the sequences in this study because of the lack of similarity of these phage gnomes. Therefore, the ORFs encoded the proteins responsible for the function of host specificity and lytic effects, such as tail and lysozyme, were selected to construct phylogenetic analyses for a better understanding of the evolution of tailed phages with regard to its host range (Haggård-Ljungquist et al., 1992). The phylogenetic analysis based on ORFs encoding tail fiber showed both phages vB_EcoM-Ro111lw and vB_EcoM-Ro121lw were in the clade, indicating a closer evolutionary relationship with regard to the tail fiber (Figure 4A). The phages vB_EcoS-Ro145lw and vB_EcoM-Ro157lw were also in the same clade, which were closely related to Escherichia phage ST2 and phage ECML-117, respectively (Figure 4A). The results were in consensus to the previous biological features in which both phages vB_EcoM-Ro111lw and vB_EcoM-Ro121lw had a wider host range. With regard to endolysin/ lysozyme, the phylogenetic comparison indicated that phages vB_EcoM-Ro121lw and vB_EcoM-Ro157lw were in the same clade: the phage vB_EcoM-Ro157lw was closely related to phage ECML-117 whereas phage vB_EcoM-Ro121lw was further clustered into a single branch (Figure 4B). Similar results were also observed on the phages vB_EcoS-Ro145lw and vB_EcoM-Ro111lw that phage vB_EcoM-Ro111lw was closely related to phage KhF3 and phage vB_EcoS-Ro145lw was clustered into a single branch (Figure 4B).\\\\nFigure 4. Phylogenetic comparison based on Clustal Omega alignment of the genes coding for tail fiber (A) and endolysin/lysozyme (B) from the newly isolated phages (red underline) and other reference phage genomes obtained from NCBI database. The tree is drawn to scale, with branch lengths (next to the branches) in the same units as those of the evolutionary distances used to infer the phylogenetic tree.\\\\nConclusion\\\\nIn conclusion, the present study provides valuable information with regard to the isolation of various STEC-specific phages and the correlation of these phages with the STEC bacterial hosts from non-fecal compost samples. Whole-genome sequencing enabled us to comprehensively characterize the newly isolated phages, together with biological features, to determine further applications of the phages. Furthermore, genomic analyses of these newly isolated phages provide in-depth information regarding the evolution and diversity of STEC-specific phages isolated from non-fecal environment. The phages isolated in this study do not contain lysogenic factors potentially associated with the transfer of unwanted genes among bacterial population. The four new phages, including vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw, together are able to infect a wide range of STEC strains, including serogroups of O157 and top 6 non-O157, due to their different but complementary host ranges. Without the presence of virulence genes, such as stx or eae genes and lysogenic factors, these phages are potential candidates used to control different serogroups of STEC strains. Moreover, evidence of the phenotypic and genotypic characteristics associated with the mitigating effects of these phages may account for zero isolation of STEC strains in this study. These findings suggest that the lytic phages are of great potential in controlling various STEC strains and may provide additional antimicrobial effects against STEC that are present in the raw composting materials and sneaked through the composting process if not well-managed. Further studies regarding the reduction of STEC by these phages under various conditions are necessary in order to optimize the antimicrobial activities against STEC in different food-associated environments.\\\\nData Availability\\\\nThe datasets generated for this study can be found in Genbank, MH571750, MH160766, MH051334, and MH051335.\\\\nAuthor Contributions\\\\nY-TL was responsible for phage isolation with XS, data analyses, genome annotation, and manuscript preparation. FL and YZ were responsible for phage sequencing. DB was responsible for STEC strain isolation. IQ and AS were responsible for TEM and host range test. VW conceived the study, aided in experiment design, and prepared the manuscript. All authors reviewed the manuscript.\\\\nFunding\\\\nThis research was funded by USDA-ARS CRIS projects 2030-42000-050-00D. The collaboration was supported by Henan Province Key Research and Development Programs (International cooperation projects 182102410043) and Henan Key Laboratory of Cold Chain Food Quality and Safety Control (CCFQ2018-YB-14).\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nThe authors thank Leslie Harden for obtaining the composting samples.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2019.00627/full#supplementary-material\\\\nFootnotes\\\\n^https://usegalaxy.org\\\\nReferences\\\\nAmarillas, L., Chaidez, C., Gonzalez-Robles, A., Lugo-Melchor, Y., and Leon-Felix, J. (2016). Characterization of novel bacteriophage phiC119 capable of lysing multidrug-resistant Shiga toxin-producing Escherichia coli O157:H7. PeerJ 13:e2423. doi: 10.7717/peerj.2423\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAmgarten, D., Martins, L. F., Lombardi, K. C., Antunes, L. P., De Souza, A. P. S., Nicastro, G. G., et al. (2017). Three novel pseudomonas phages isolated from composting provide insights into the evolution and diversity of tailed phages. BMC Genomics 18:346. doi: 10.1186/s12864-12017-13729-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAsare, P. T., Jeong, T. Y., Ryu, S., Klumpp, J., Loessner, M. J., Merrill, B. D., et al. (2015). Putative type 1 thymidylate synthase and dihydrofolate reductase as signature genes of a novel bastille-like group of phages in the subfamily Spounavirinae. BMC Genomics 16:582. doi: 10.1186/s12864-015-1757-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBahir, I., Fromer, M., Prat, Y., and Linial, M. (2009). Viral adaptation to host: a proteome-based analysis of codon usage and amino acid preferences. Mol. Syst. Biol. 5:311. doi: 10.1038/msb.2009.71\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBardina, C., Colom, J., Spricigo, D. A., Otero, J., Sanchez-Osuna, M., Cortes, P., et al. (2016). Genomics of three new bacteriophages useful in the biocontrol of Salmonella. Front. Microbiol. 7:545. doi: 10.3389/fmicb.2016.00545\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBeutin, L., and Martin, A. (2012). Outbreak of Shiga toxin-producing Escherichia coli (STEC) O104:H4 infection in Germany causes a paradigm shift with regard to human pathogenicity of STEC strains. J. Food Prot. 75, 408–418. doi: 10.4315/0362-028X.JFP-11-452\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBuck, J. W., Walcott, R. R., and Beuchat, L. R. (2003). Recent trends in microbiological safety of fruits and vegetables. Plant Health Prog. 10, 1094–1103. doi: 10.1094/PHP-2003-0121-01-RV\\\\nCrossRef Full Text | Google Scholar\\\\nCasjens, S. R., and Gilcrease, E. B. (2009). Determining DNA packaging strategy by analysis of the termini of the chromosomes in tailed-bacteriophage virions. Methods Mol. Biol. 502, 91–111. doi: 10.1007/978-1-60327-565-1_7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCooley, M. B., Jay-Russell, M., Atwill, E. R., Carychao, D., Nguyen, K., Quinones, B., et al. (2013). Development of a robust method for isolation of Shiga toxin-positive Escherichia coli (STEC) from fecal, plant, soil and water samples from a leafy greens production region in California. PLoS One 8:e65716. doi: 10.1371/journal.pone.0065716\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDelesalle, V. A., Tanke, N. T., Vill, A. C., and Krukonis, G. P. (2016). Testing hypotheses for the presence of tRNA genes in mycobacteriophage genomes. Bacteriophage 6:e1219441. doi: 10.1080/21597081.2016.1219441\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nErickson, M. C., Liao, J., Ma, L., Jiang, X., and Doyle, M. P. (2009). Inactivation of Salmonella spp. in cow manure composts formulated to different initial C:N ratios. Bioresour. Technol. 100, 5898–5903. doi: 10.1016/j.biortech.2009.06.083\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nFrank, C., Werber, D., Cramer, J. P., Askar, M., Faber, M., An Der Heiden, M., et al. (2011). Epidemic profile of Shiga-toxin-producing Escherichia coli O104:H4 outbreak in Germany. N. Eng. J. Med. 365, 1771–1780. doi: 10.1056/NEJMoa1106483\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGarneau, J. R., Depardieu, F., Fortier, L. C., Bikard, D., and Monot, M. (2017). PhageTerm: a tool for fast and accurate determination of phage termini and packaging mechanism using next-generation sequencing data. Sci. Rep. 7:8292. doi: 10.1038/s41598-41017-07910-41595\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHaggård-Ljungquist, E., Halling, C., and Calendar, R. (1992). DNA sequences of the tail fiber genes of bacteriophage P2: evidence for horizontal transfer of tail fiber genes among unrelated bacteriophages. J. Bacteriol. 174, 1462–1477. doi: 10.1128/jb.174.5.1462-1477.1992\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHandley, S. A., Thackray, L. B., Zhao, G., Presti, R., Miller, A. D., Droit, L., et al. (2012). Pathogenic simian immunodeficiency virus infection is associated with expansion of the enteric virome. Cell 151, 253–266. doi: 10.1016/j.cell.2012.09.024\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHeringa, S. D., Kim, J., Jiang, X., Doyle, M. P., and Erickson, M. C. (2010). Use of a mixture of bacteriophages for biological control of Salmonella enterica strains in compost. Appl. Environ. Microbiol. 76, 5327–5332. doi: 10.1128/AEM.00075-00010\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nImamovic, L., Balleste, E., Jofre, J., and Muniesa, M. (2010). Quantification of Shiga toxin-converting bacteriophages in wastewater and in fecal samples by real-time quantitative PCR. Appl. Environ. Microbiol. 76, 5693–5701. doi: 10.1128/AEM.00107-10\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJohannessen, G. S., James, C. E., Allison, H. E., Smith, D. L., Saunders, J. R., and Mccarthy, A. J. (2005). Survival of a Shiga toxin-encoding bacteriophage in a compost model. FEMS Microbiol. Lett. 245, 369–375. doi: 10.1016/j.femsle.2005.03.031\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKim, J., Shepherd, M. W. J., Jiang, X., and Jiang, X. (2009). Evaluating the effect of environmental factors on pathogen regrowth in compost extract. Microb. Ecol. 58, 498–508. doi: 10.1007/s00248-009-9524-x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKumar, S., Stecher, G., and Tamura, K. (2016). MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets. Mol. Biol. Evol. 33, 1870–1874. doi: 10.1093/molbev/msw054\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLemunier, M., Francou, C., Rousseaux, S., Houot, S., Dantigny, P., Piveteau, P., et al. (2005). Long-term survival of pathogenic and sanitation indicator bacteria in experimental biowaste composts. Appl. Environ. Microbiol. 71, 5779–5786. doi: 10.1128/AEM.71.10.5779-5786.2005\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLi, S., Fan, H., An, X., Fan, H., Jiang, H., Chen, Y., et al. (2014). Scrutinizing virus genome termini by high-throughput sequencing. PLoS One 9:e85806. doi: 10.81371/journal.pone.0085806\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLiao, Y. T., Quintela, I. A., Nguyen, K., Salvador, A., Cooley, M. B., and Wu, V. C. H. (2018). Investigation of prevalence of free Shiga toxin- producing Escherichia coli (STEC)-specific bacteriophages and its correlation with STEC bacterial hosts in a produce-growing area in Salinas, California. PLoS One 13:e0190534. doi: 10.1371/journal.pone.0190534\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMartinez-Castillo, A., and Muniesa, M. (2014). Implications of free Shiga toxin-converting bacteriophages occurring outside bacteria for the evolution and the detection of Shiga toxin-producing Escherichia coli. Front. Cell. Infect. Microbiol. 4:46. doi: 10.3389/fcimb.2014.00046\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMcWilliam, H., Li, W., Uludag, M., Squizzato, S., Park, Y. M., Buso, N., et al. (2013). Analysis tool web services from the EMBL-EBI. Nucleic Acids Res. 41, W597–W600. doi: 10.1093/nar/gkt1376\\\\nCrossRef Full Text | Google Scholar\\\\nOliveira, M., Vinas, I., Usall, J., Anguera, M., and Abadias, M. (2012). Presence and survival of Escherichia coli O157:H7 on lettuce leaves and in soil treated with contaminated compost and irrigation water. Int. J. Food Microbiol. 156, 133–140. doi: 10.1016/j.ijfoodmicro.2012.03.014\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPartanen, P., Hultman, J., Paulin, L., Auvinen, P., and Romantschuk, M. (2010). Bacterial diversity at different stages of the composting process. BMC Microbiol. 10:94. doi: 10.1186/1471-2180-1110-94\\\\nCrossRef Full Text | Google Scholar\\\\nRyckeboer, J., Mergaert, J., Vaes, K., Klammer, S., De Clercq, D., Coosemans, J., et al. (2003). A survey of bacteria and fungi occurring during composting and self-heating process. Ann. Microbiol. 53, 349–410.\\\\nGoogle Scholar\\\\nSaitou, N., and Nei, M. (1987). The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol. Biol. Evol. 4, 406–425.\\\\nGoogle Scholar\\\\nSeemann, T. (2014). Prokka: rapid prokaryotic genome annotation. Bioinformatics 30, 2068–2069. doi: 10.1093/bioinformatics/btu2153\\\\nCrossRef Full Text | Google Scholar\\\\nTamura, K., Nei, M., and Kumar, S. (2004). Prospects for inferring very large phylogenies by using the neighbor-joining method. Proc. Natl. Acad. Sci. U.S.A. 101, 11030–11035. doi: 10.1073/pnas.0404206101\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nvan Asselt, E. J., Dijkstra, A. J., Kalk, K. H., Akacs, B., Keck, W., and Dijkstra, B. W. (1999). Crystal structure of Escherichia coli lytic transglycosylase Slt35 reveals a lysozyme-like catalytic domain with an EF-hand. Structure 7, 1167–1180. doi: 10.1016/S0969-2126(00)80051-9\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nViazis, S., Akhtar, M., Feirtag, J., Brabban, A. D., and Diez-Gonzalez, F. (2011). Isolation and characterization of lytic bacteriophages against enterohaemorrhagic Escherichia coli. J. Appl. Microbiol. 110, 1323–1331. doi: 10.1111/j.1365-2672.2011.04989.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWalters, S. P., Thebo, A. L., and Boehm, A. B. (2011). Impact of urbanization and agriculture on the occurrence of bacterial pathogens and stx genes in coastal waterbodies of central California. Water Res. 45, 1752–1762. doi: 10.1016/j.watres.2010.1711.1032\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWang, J., Niu, Y. D., Chen, J., Anany, H., Ackermann, H. W., Johnson, R. P., et al. (2015). Feces of feedlot cattle contain a diversity of bacteriophages that lyse non-O157 Shiga toxin-producing Escherichia coli. Can. J. Microbiol. 61, 467–475. doi: 10.1139/cjm-2015-0163\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWhichard, J. M., Weigt, L. A., Borris, D. J., Li, L. L., Zhang, Q., Kapur, V., et al. (2010). Complete genomic sequence of bacteriophage felix O1. Viruses 2, 710–730. doi: 10.3390/v2030710\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWu, S., Nishihara, M., Kawasaki, Y., Yokoyama, A., Matsuura, K., Koga, T., et al. (2009). Inactivation of Escherichia coli in soil by solarization. Soil Sci. Plant Nutr. 55, 258–263. doi: 10.1111/j.1747-0765.2009.00362.x\\\\nCrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2021.616548\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2021.616548/full\",\\n    \"loadedTime\": \"2024-01-23T08:49:19.798Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2021.616548\",\\n    \"title\": \"Frontiers | Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples\",\\n    \"description\": \"Vibrio parahaemolyticus is a foodborne pathogen that is frequently isolated from a variety of seafood. To control this pathogenic Vibrio spp., the implementation of bacteriophages in aquaculture and food industries have shown a promising alternative to antibiotics. In this study, six bacteriophages isolated from the seafood samples demonstrated a narrow host range specificity that infecting only the V. parahaemolyticus strains. Morphological analysis revealed that ɸVp33, ɸVp22, ɸVp21 and ɸVp02 belong to the Podoviridae family, while ɸVp08 and ɸVp11 were categorized into the Siphoviridae family. All bacteriophages were composed of DNA genome and showed distinctive restriction fragment length polymorphism. The optimal MOI for bacteriophage propagation was determined to be 0.001 to 1. One-step growth curve revealed that the latent period ranged from 10 to 20 min, and the burst size of bacteriophage was approximately 17 to 51 PFU/cell. The influence of temperature and pH levels on the stability of bacteriophages showed that all bacteriophages were optimally stable over a wide range of temperatures and pH levels. In vitro lytic activity of all bacteriophages demonstrated to have a significant effect against V. parahaemolyticus. Besides, the application of a bacteriophage cocktail instead of a single bacteriophage suspension was observed to have a better efficiency to control the growth of V. parahaemolyticus. Results from this study provided a basic understanding of the physiological and biological properties of the isolated bacteriophages before it can be readily used as a biocontrol agent against the growth of V. parahaemolyticus.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nVibrio parahaemolyticus is a Gram-negative halophilic bacterium, which occurs naturally in marine environments. It is recognized as a leading foodborne pathogen that is frequently isolated from a variety of seafood (Wang et al., 2015). Consumption of food contaminated with V. parahaemolyticus can cause gastroenteritis, which is often displayed through bouts of abdominal pain, diarrhea, fever, and nausea. An open wound in the skin which comes in contact with V. parahaemolyticus can also result in wound infection. In rare instances, V. parahaemolyticus can cause life-threatening conditions like rapid necrosis of subcutaneous tissue and septicemia (Tena et al., 2010; Ahmad et al., 2013; Zhang and Orth, 2013). However, not all V. parahaemolyticus strains are pathogenic and capable of causing illness in humans (Elmahdi et al., 2016). The presence of thermostable direct hemolysin (TDH) and the TDH-related hemolysin (TRH) has been identified as well-known pathogenic factors in V. parahaemolyticus which cause vibriosis. Both tdh and trh genes are therefore widely used as a virulence marker in many studies for the detection of pathogenic V. parahaemolyticus strains (Chen et al., 2016; Kang et al., 2017; Paria et al., 2019). However, infection caused by V. parahaemolyticus strains without the virulence tdh and trh genes have been reported in numerous studies (Bhoopong et al., 2007; Jones et al., 2012; Li et al., 2014; Pazhana et al., 2014). The pathogenesis of V. parahaemolyticus is therefore still open to question and has not been fully elucidated.\\\\nBacteriophages or phages are bacterial viruses that infect bacterial cells only. Bacteriophages can be found in ubiquitous environments and are recognized as the most abundant organisms with 1031 bacteriophage particles on the earth’s surface (Comeau et al., 2008; Keen, 2015). Bacteriophages are typically highly host-specific, which targets only specific species or even strains (Hagens and Loessner, 2007). Besides, high specificity levels of bacteriophages could minimize the disturbance of commensal bacteria (Viazis et al., 2011). The International Committee on Taxonomy of Viruses (ICTV) is responsible for the classification and nomenclature of viruses based on their morphological, genomic and proteomic characteristics (Adams et al., 2017). Tailed bacteriophages are grouped into the order of Caudovirales which consist of three families Myoviridae, Podoviridae, and Siphoviridae. Today, the order of Caudovirales includes six subfamilies, 80 genera, and 441 species (Krupovic et al., 2016). In 2018 and 2019, an update from the ICTV has proposed the creation of a new order (Tubulavirales), 10 new families (Autographiviridae, Chaseviridae, Demerecviridae, Drexlerviridae, Finnlakeviridae, Halspiviridae, Herelleviridae, Ovaliviridae, Plectroviridae, and Thaspiviridae), 22 subfamilies, 424 genera, and 964 species (Adriaenssens et al., 2020).\\\\nBacteriophages can be grouped into individual families based on their distinct morphological structure, as viewed through electron microscopy (Schramlová et al., 2010). Electron microscopy such as transmission electron microscopy (TEM) is typically used at the initial stage for rapid identification and characterization of viruses. Genetic materials of viruses can be either DNA or RNA, and the structure of the nucleic acid may be either single-stranded or double-stranded. Restriction enzyme cleaves the DNA of bacteriophages into distinct polymorphism, which is known as restriction fragment length polymorphism (RFLP). The use of RFLP analysis can facilitate the study of genetic diversity and differentiate between individual bacteriophages. On the other hand, protein profiles of bacteriophages can be analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). SDS-PAGE analysis of phage structural proteins will reveal the molecular weight of phage proteins and help to determine the presence of phage structural proteins.\\\\nIt is noteworthy that the discovery of bacteriophages occurred much earlier than antibiotics. The use of bacteriophage preparations in human trials have also been conducted for almost a century in Eastern Europe (Lin et al., 2017). Nevertheless, the development of a broad-spectrum of antibiotics later received more attention than the use of bacteriophages. Today, phage therapy involves the use of bacteriophages in clinical trials that have gained new attention due to the growing number of infections caused by multidrug-resistance bacteria (Rohde et al., 2018). Phage therapy against several bacterial pathogens in animal models have also been tested out and showed effective results in many studies (Biswas et al., 2002; Wang et al., 2006; Watanabe et al., 2007; Jun et al., 2014; Kalatzis et al., 2018).\\\\nThe application of bacteriophages is not only limited to clinical trials to fight against bacterial infections. It can be used in livestock protection to reduce the mortality rate of animals as well as enhance the food safety levels by controlling the growth of foodborne pathogens. Globally, efforts to control Vibrio spp. with the application of bacteriophages in aquaculture settings have been reported by researchers. For example, Wang et al. (2017) demonstrated that the use of bacteriophage therapy in controlling V. harveyi within greenlip abalone. Le et al. (2020b) showed that the application of lytic bacteriophages in the reduction of oyster larvae mortality rate caused by V. alginolyticus. Le et al. (2020a) demonstrated that the effectiveness of bacteriophages treatment in the decontamination of Vibrio spp. in commercially produced microalgae that used as the primary food source for oyster larvae during hatchery culture. In the future, it is believed that the use of bacteriophages could be a promising alternative to antibiotics for fighting against bacterial pathogens.\\\\nIn this study, the objective was to isolate V. parahaemolyticus bacteriophages from the seafood samples and the isolated bacteriophages were characterized based on host specificity, morphological, genomic character, protein profile, and temperature and pH stability. The in vitro lytic activity of single bacteriophage suspensions as well as the bacteriophage cocktails were also evaluated to determine their efficiency against the growth of V. parahaemolyticus.\\\\nMaterials and Methods\\\\nBacterial Strains\\\\nSix V. parahaemolyticus strains which consisted of 4 clinical isolates and 2 food source isolates (non-tdh and/or trh) were used as hosts for the isolation of bacteriophages (Table 1). PSU5333, PSU5322, and PSU4211 V. parahaemolyticus strains were kindly provided by Professor Dr. Varaporn Vuddhakul from the Prince of Songkla University, Thailand. V. parahaemolyticus ATCC 17802 was purchased from the American Type Culture Collection (ATCC), United States. Both VP8 and VP11 isolates were obtained from the laboratory stock strain at Food Safety Laboratory, Universiti Putra Malaysia. All bacterial strains used were confirmed by PCR amplification of species-specific toxR gene and pathogenicity tdh and trh genes with specific primers as mentioned in the previous study (Tan et al., 2017) and the amplified sequences were further confirmed by DNA sequencing.\\\\nTable 1. V. parahaemolyticus strains used for bacteriophages isolation.\\\\nSample Preparation\\\\nA total of 30 seafood samples consisting of 10 samples for each blood clam, shrimp, and surf clam were purchased from different wet markets in Selangor, Malaysia. Ten grams of each sample was then weighed and transferred into 10 mL of salt of magnesium (SM) buffer (100 mM NaCl, 8 mM MgSO4⋅7H2O, 50 mM pH 7.5 Tris-HCl and 0.01% gelatin [Merck, Germany]) in a 50 mL sterile centrifuge tube. The mixture was shaken at 150 rpm for 15 min in an orbital shaker. Large particles were pelleted in the tube via centrifugation at 10,000 × g for 5 min. The supernatant was filtered through a 0.2 μM pore size syringe filter (Pall, United States). A 200 μL of log-phase V. parahaemolyticus host culture (OD600nm = 0.4–0.6) was added into the 5 mL of filtrate and incubated at 37°C, 150 rpm in an orbital shaker overnight. After incubation, the mixture was centrifuged at 10,000 × g for 5 min to pelletize the bacterial cells and the supernatant was filtered with a 0.2 μM pore size syringe filter (Pall, United States).\\\\nDetection of the Presence of Bacteriophages\\\\nTen-fold serial dilutions (through 10–8) of the filtrate was prepared in SM buffer. Each of these different dilutions was screened for the presence of bacteriophage by spotting 10 μL of the filtrate on a double-layer agar plate with a bacterial host (Beck et al., 2009). The plate was allowed to dry for at least 15 min and the plate was incubated in an inverted position at 37°C overnight. Lysis zone was observed after the incubation. Filtrate from the highest dilution which showed a clear lysis zone on the double-layer agar plate was selected for isolation, propagation, and purification.\\\\nIsolation of Bacteriophage\\\\nIsolation of bacteriophages was carried out according to the double agar overlay plaque assay (Kropinski et al., 2009). A 100 μL of each dilution filtrate and 100 μL of the log-phase bacteria culture was aliquot into a 3 mL molten soft agar, which constituted of tryptic soy broth (TSB) (Merck, Germany) and 0.6% bacteriological agar (Merck, Germany), which was stored at 45°C. The molten soft agar was mixed gently and poured evenly onto tryptic soy agar (TSA) (Merck, Germany) base plate. The plate was allowed to solidify for at least 15 min and incubated in an inverted position at 37°C overnight. After incubation, a single colony plaque with clear lysis appearance from the highest dilution was picked using a sterile inoculation loop and transferred it into a 5 mL SM buffer with 200 μL log-phase bacteria culture. The mixture was incubated at 37°C, 150 rpm in an orbital shaker overnight. After incubation, the sample was centrifuged at 10,000 × g for 5 min, and the supernatant was filtered with a 0.2 μM pore size syringe filter.\\\\nPropagation and Precipitation of Bacteriophage\\\\nBacteriophage was amplified according to the phage propagation via liquid lysate as described by Bonilla et al. (2016) but with some modifications. Briefly, 200 mL of TSB supplemented with 0.001 M CaCl2 and MgCl2 was prepared in a 250 mL centrifuge bottle and spiked with 0.1 volumes of overnight bacteria host. A 200 μL of bacteriophage lysate (>108 PFU/mL) was then added into the centrifuge bottle and incubated at 37°C, 150 rpm in an orbital shaker until the lysate cleared. After incubation, the sample was centrifuged at 4°C and 4,000 × g for 20 min. The supernatant was filtered with a 0.2 μM pore size syringe filter into a sterile centrifuge bottle. The filtrate was precipitated by adding 10% (w/v) polyethylene glycol (PEG) 8000 (Sigma-Aldrich, United States) and kept undisturbed at 4°C for 24 h. After 24 h, the sample was centrifuged at 4°C and 4,000 × g for 20 min. The supernatant was removed, and the pellet was suspended in a 10 mL of SM buffer.\\\\nPurification of Bacteriophage\\\\nBacteriophage suspension was purified using caesium chloride (CsCl) (Santa Cruz Biotechnology, United States) with a density gradient according to the method described in the T7 Select® System Manual TB178 (Novagen, United States). A 5 mL of bacteriophage suspension was aliquot slowly onto the uppermost layer and the sample was subjected to ultracentrifugation at 210,000 × g and 4°C for 1 h in a SorvallTM WX ultracentrifuge with a swinging bucket TH-641 Rotor (Thermo Scientific, United States). After centrifugation, a sharp and turbid band containing the purified bacteriophage particles was collected carefully with a Pasteur pipet. The purified bacteriophage was subjected to dialysis to remove CsCl with an Amicon® Ultra-15 100K filter device (Merck, Germany) and SM buffer (Bonilla et al., 2016). After dialysis, the purified bacteriophage was assayed to determine the bacteriophage titer. The purified bacteriophage stock sample was stored at 4°C until further analysis.\\\\nHost Range Specificity\\\\nA total of 134 bacteria strains including 126 V. parahaemolyticus strains and 8 non-V. parahaemolyticus strains were used to study the host range of the isolated bacteriophages. Non-V. parahaemolyticus strains including of V. alginolyticus, V. cholerae, V. vulnificus, Escherichia coli, Listeria monocytogenes, Salmonella enterica serovar Typhimurium, Salmonella enterica serovar Enteritidis, and Staphylococcus aureus. Host range specificity was determined by spotting 20 μL of each bacteriophage (105 PFU/mL) on a double-layer agar plate seeded with a bacteria strain (Viazis et al., 2011). The agar plate with bacteriophage inoculum was allowed to dry for at least 15 min before being incubated at 37°C overnight. Lysis zones were observed on the plate after the incubation.\\\\nTransmission Electron Microscopy (TEM)\\\\nBacteriophage was visualized by a transmission electron microscope (TEM) as described by Kim et al. (2019) with some modifications. A carbon-coated copper grid was placed onto a drop of purified bacteriophage (1010 PFU/mL) for 2 min. The grid with the sample was negatively stained first with 2% (w/v) uranyl acetate for 2 min, followed by sterile water for 1 min. The excess sample was removed with a filter paper, and the grid was allowed to air dry for 10 min. The morphology of the bacteriophage was then visualized with a JEM-2100 200-kV transmission electron microscope (JEOL, Japan).\\\\nBacteriophage Genome Analysis\\\\nExtraction of nucleic acids of bacteriophage was performed according to the protocol provided with the Wizard DNA Clean-Up Kit (Promega, United States). For determination of the type of nucleic acids, 2 μL of extracted bacteriophage nucleic acid was treated separately with 2 μL of DNase I (1 U/μL) and RNase A (10 mg/mL) (Thermo Scientific, United States) for 30 min at 37°C (Topka et al., 2019). The treated sample was electrophoresed through 0.7% (w/v) agarose gel at 100 V for 30 min. The agarose gel was stained with ethidium bromide (0.5 μg/ml) (Merck, Germany) and visualized with Syngene GeneGenius BioImaging system, together with GeneSnap software (version 7.12.01) (Syngene, Cambridge, United Kingdom).\\\\nRestriction Fragment Length Polymorphism (RFLP)\\\\nGenomic DNA (1 μg/μL) of each bacteriophage was digested individually with EcoRI, HaeIII, HindIII, and SalI restriction enzymes (Promega, United States) according to the manufacturer’s protocols and incubated at 37°C for 1 h. The digested product was electrophoresed and visualized as mentioned in the above section.\\\\nSDS-PAGE Analysis\\\\nProtein profiles of bacteriophage were analyzed by one-dimensional SDS-PAGE. Each purified bacteriophage (1010 PFU/mL) sample was first solubilized by mixing an equal volume of 2× sample buffer solution (0.125 M Tris-HCl, 4% w/v SDS, 20% v/v glycerol, 0.01% w/v bromophenol blue, 10% v/v 2-mercaptoethanol) and boiled at 95°C for 10 min (Guan et al., 2004). The solubilized sample was loaded into 5–15% Bullet PAGE One Precast Gel (Nacalai Tesque, Japan) and electrophoresed in 1× running buffer solution (0.25 mol/L Tris, 1.92 mol/L Glycine, and 10 g/L SDS) at 200 V for 30 min. The gel was stained with CBB Stain One Super (Nacalai Tesque, Japan) for 30 min and de-stained several times with deionized water.\\\\nMultiplicity of Infection (MOI)\\\\nThe multiplicity of infection (MOI) of the bacteriophage was tested according to Liu et al. (2013) with some modifications. The log-phase bacteria host was first prepared and mixed with bacteriophage suspension at different ratios (0.0001, 0.001, 0.01, 0.1, 1, 10, and 100 PFU/CFU). After incubation at 37°C for 3.5 h, the sample was centrifuged at 10,000 × g for 5 min. The supernatant was filtered with a 0.2 μM pore size syringe filter and enumerated to determine the bacteriophage titer. Bacteria-free suspension and bacteriophage-free suspension was included in all the experiments as a control sample. An MOI ratio with the highest bacteriophage titer was considered as the optimal MOI.\\\\nOne-Step Growth Curve\\\\nOne-step growth curve of each bacteriophage was studied according to the protocol described by Hong et al. (2014). Briefly, bacteriophage suspension was mixed with 1 mL of log-phase bacteria host culture at the optimal MOI ratio. The mixture was incubated at 37°C and 150 rpm for 10 min to allow the adsorption of bacteriophages to the host cells before centrifuging at 10,000 × g for 30 s. The pellet was suspended in 10 mL of TSB and the suspension was incubated at 37°C and 150 rpm. A 100 μL of sample was withdrawn at 10 min intervals up to a total period of 1 h and enumerated by double agar overlay plaque assay in duplicate. The latent period and burst size of the bacteriophage was evaluated from the one-step growth curve.\\\\nTemperature and pH Stability\\\\nTemperature and pH stability were tested according to the method described by Thung et al. (2017) with some modifications. The temperature stability of bacteriophage suspension in the SM buffer was tested at different temperatures (−20, 25, 37, 50, 60, and 70°C) and incubated for 2 h. For pH stability, bacteriophage suspension was incubated at different pH ranges (pH 2, 3, 5, 7, 9, and 11) for 2 h. Bacteriophage suspension kept at 4°C in the SM buffer (pH7.5) was used as a control in this study. After incubation, the titer of bacteriophage was determined by double agar overlay plaque assay and the survival rate was calculated by dividing the bacteriophage titer after treatment with the initial bacteriophage concentration in the control sample (Liu et al., 2020).\\\\nIn vitro Lytic Activity\\\\nIn vitro lytic activity was determined individually for each bacteriophage (Vp33, Vp22, Vp21, Vp02, Vp08, and Vp11) against its respective bacterial host strain. Two bacteriophage cocktails (cocktails A and B) were also prepared and tested for their synergistic effect (Table 2). Bacteriophage cocktails were selected based on the host range specificity results in the section “Host Range Specificity.” Cocktails A and B were prepared for targeting V. parahaemolyticus clinical strains and environmental strains, respectively. A 500 μL of log-phase V. parahaemolyticus host culture was mixed with an equal volume of bacteriophage lysate (∼1 × 106 PFU/mL). The mixture was incubated immediately at 37°C. A 100 μL sample of the mixture was withdrawn at 0, 2, 4, 6, and 24 h, to determine the concentration of bacteria through spread plating onto a CHROMagarTM Vibrio (CV) plate. The plate was allowed to dry for at least 5 min before inverting the plate for incubation.\\\\nTable 2. Bacteriophage cocktail preparations.\\\\nResults\\\\nScreening of Bacteriophage\\\\nA total of six bacteriophages displayed an ability to lyse V. parahaemolyticus host strains and caused lysis zones on the double-layer agar plate was discovered in the blood clam, shrimp, and surf clam samples. Out of 10 shrimp samples, four samples were found to have the presence of V. parahaemolyticus bacteriophages. Only one sample of blood clam and surf clam was detected with the presence of V. parahaemolyticus bacteriophages.\\\\nHost Range Specificity\\\\nPurified bacteriophages were tested for their host range specificity. Lysis zones were observed on 61 out of 126 (48.4%) of the double-layer agar plate seeded with V. parahaemolyticus strains (Table 3). No lysis zones were observed on all non-V. parahaemolyticus (8/8) bacteria lawn agar plates. Bacteriophages Vp33, Vp22, Vp21, and Vp02, were found to be able to cause lysis zones on 18.3% (23/126) of V. parahaemolyticus strains agar plates, which included four clinical strains of V. parahaemolyticus. On the other hand, Vp08 and Vp11 were found to be able to cause lysis zones on 34.9% (44/126) of the environmental V. parahaemolyticus strains.\\\\nTable 3. Host range specificity on different V. parahaemolyticus strains.\\\\nTransmission Electron Microscopy (TEM)\\\\nElectron microscopy revealed that the morphological characteristics of bacteriophages Vp33, Vp22, Vp21, and Vp02 were resembled Podoviridae family, and bacteriophages Vp08 and Vp11 were resembled Siphoviridae family (Figures 1A–F). Bacteriophages Vp33, Vp22, Vp21, and Vp02 were characterized with a hexagonal head diameter between 41.28 ± 1.83 to 50.00 ± 3.21 nm and a short stumpy non-contractile tail diameter between 8.72 ± 2.29 to 12.44 ± 2.30 nm. Bacteriophages Vp08 and Vp11 were exhibited with an elongated hexagonal head diameter between 79.00 ± 5.94 × 44.98 ± 1.60 nm and 91.78 ± 4.11 × 49.54 ± 2.05 nm and a long flexible non-contractile tail diameter between 133.79 ± 15.98 nm and 142.24 ± 13.47 nm.\\\\nFigure 1. Electron micrography of negative stained Vibrio parahaemolyticus bacteriophages and the bar represents 50 nm. (A) Vp33 (head diameter: 44.70 ± 2.30 nm; tail diameter: 9.22 ± 0.92 nm); (B) Vp22 (head diameter: 41.28 ± 1.83 nm; tail diameter: 8.72 ± 2.29 nm); (C) Vp21 (head diameter: 42.63 ± 2.07 nm; tail diameter: 9.86 ± 1.15 nm); (D) Vp02 (head diameter: 50.00 ± 3.21 nm; tail diameter: 12.44 ± 2.30 nm); (E) Vp08 (head diameter: 91.78 ± 4.11 nm × 49.54 ± 2.05 nm; tail diameter: 142.24 ± 13.47 nm) (F) Vp11 (head diameter: 79.00 ± 5.94 nm × 44.98 ± 1.60 nm; tail diameter: 133.79 ± 15.98 nm).\\\\nBacteriophage Genome Analysis\\\\nFrom the determination of the nucleic acids, all the bacteriophage nucleic acids were digested by DNase I (1 U/μL) after 30 min incubation at 37°C. None of the bacteriophage nucleic acids was digested by RNase A (10 mg/mL) after 30 min of incubation at 37°C.\\\\nRestriction Fragment Length Polymorphism (RFLP)\\\\nFrom the restriction endonuclease digestion, the genomic DNA of all bacteriophages remained intact and unable to be digested by the EcoRI and SalI restriction enzymes. However, the genomic DNA of each bacteriophage treated with HindIII restriction enzyme produced a distinctive RFLP (Figure 2A). HaeIII restriction digestion profiles were also obtained for all bacteriophages except Vp08 and Vp11 (Figure 2B).\\\\nFigure 2. Lane M: 1 kb DNA ladder (Promega, United States); Lane 1: Vp33; Lane 2: Vp22; Lane 3: Vp21; Lane 4: Vp02; Lane 5: Vp08; Lane 6: Vp11. (A) RFLP patterns of bacteriophages genomic DNA digested with HindIII. (B) RFLP patterns of bacteriophages genomic DNA digested with HaeIII.\\\\nSDS-PAGE Analysis\\\\nStructural proteins of bacteriophages were analyzed by SDS-PAGE. Protein profiles of bacteriophage Vp33, Vp22, Vp21, and Vp02 demonstrated a major protein band at around 31 kDa, while bacteriophages Vp08 and Vp11 exhibited a major protein band at around 40 kDa (Figure 3). Bacteriophages Vp33, Vp22, and Vp02 were observed with similar protein profiles of 8 clear protein bands at around 36, 31, 26, 20, 17, 15, 12, and 10 kDa. Bacteriophages Vp08 and Vp11 exhibited similar patterns of protein profiles with seven clear protein bands at around 48, 40, 26, 20, 17, 10, and 5 kDa. Bacteriophage Vp21 only showed two clear protein bands at around 35 and 32 kDa.\\\\nFigure 3. Protein profiles of bacteriophages. Lane M: ExcelBandTM High Range Protein Marker, PM2610 (SMOBIO, Taiwan); Lane 1: Vp33; Lane 2: Vp22; Lane 3: Vp21; Lane 4: Vp02; Lane 5: Vp08; Lane 6: Vp11.\\\\nMultiplicity of Infection (MOI)\\\\nAt an MOI of 1, bacteriophages Vp33, Vp08, and Vp11 exhibited the highest bacteriophage titer of 8.34 ± 0.08, 10.05 ± 0.07, and 9.47 ± 0.15 log PFU/mL, respectively, after 3.5 h incubation at 37°C. The MOI ratio of 1 was then determined to be the optimal MOI for the bacteriophages Vp33, Vp08, and Vp11. Bacteriophage Vp22 was observed with an optimal MOI of 0.001 from the highest bacteriophage titer of 9.52 ± 0.06 log PFU/mL, while bacteriophage Vp02 was found with the optimal MOI of 0.1 from the highest bacteriophage titer of 10.02 ± 0.11 log PFU/mL. Lastly, the optimal MOI of bacteriophage Vp21 was determined to be 0.01 to 0.001 with the highest bacteriophage titer of 8.22 ± 0.01 and 8.22 ± 0.03 log PFU/mL.\\\\nOne-Step Growth Curve\\\\nFrom the one-step growth curve, all the bacteriophages were observed with a latent period of 20 min except bacteriophage Vp11 showed a faster latent period of 10 min in comparison to the others (Figures 4A–F). The average burst size of bacteriophages Vp33, Vp22, Vp21, Vp02, Vp08, and Vp11 estimated from the one-step growth curve were 22, 51, 34, 26, 37, and 17 plaque-forming units (PFU) per cell, respectively.\\\\nFigure 4. One-step growth curve of bacteriophages. (A) Vp33; (B) Vp22; (C) Vp21; (D) Vp02; (E) Vp08; (F) Vp11.\\\\nTemperature and pH Stability\\\\nAll bacteriophages were observed to remain relatively stable at temperatures −20, 25, 37, and 50°C, with survival rate ranged from 93.13 to 100.00% (Table 4). At a temperature of 60°C, a low survival rate of 31.16 and 49.96% was observed in bacteriophages Vp08 and Vp11, respectively. At a temperature of 70°C, all bacteriophages were found inactivated and loss of infectivity. For pH stability, all bacteriophages were observed to be able to withstand a wide range of pH levels from 5 to 11 and found with a percentage of survival of 95.82 to 99.92% (Table 5). Besides, the majority of bacteriophages were found to be resistant at pH3 except that bacteriophages Vp21 and Vp22 were observed with a low survival rate of 47.15 and 52.64%, respectively. At pH2, none of the bacteriophages was survived and recovered after a 2 h period of incubation.\\\\nTable 4. Temperature stability of bacteriophages.\\\\nTable 5. pH stability of bacteriophages.\\\\nIn vitro Lytic Activity\\\\nAll bacteriophages, including two bacteriophage cocktail preparations, were demonstrated to have significant (P < 0.05) in vitro lytic activity after 2 h incubation at 37°C (Table 6). The efficiency of single bacteriophage suspensions as well as the bacteriophage cocktails were also observed to increase significantly over time. The total reduction was reported to be in the range of 2.85 ± 0.07 to 4.60 ± 0.08 log CFU/mL, for individual bacteriophage suspension. Bacteriophage cocktails A and B achieved a better total reduction, with 3.69 ± 0.49 and 4.81 ± 0.14 log CFU/mL, respectively.\\\\nTable 6. Viable count (log CFU/mL) of V. parahaemolyticus after 0, 2, 4, 6, and 24 h incubation at 37°C with bacteriophage.\\\\nDiscussion\\\\nThe majority of studies were focused on the isolation of bacteriophages from environmental sources such as lake sediment, poultry farm, seawater, sewage, and soil (Salifu et al., 2013; Shende et al., 2017; Huang et al., 2018; Akhwale et al., 2019; Kim et al., 2019; Tang et al., 2019; Necel et al., 2020; Sevilla-Navarro et al., 2020). Reports of the isolation of these bacteriophages presence on fresh food samples were relatively limited in extent. According to Greer (2005), the successful isolation of bacteriophages from food samples was reliant on the occurrence of the target bacteria which was present in the food samples in relatively high concentrations.\\\\nShrimp samples purchased from the wet markets were found to be the primary source of V. parahaemolyticus bacteriophages in this study. Similarly, several studies reported the isolation of Vibrio spp. bacteriophages from a similar source such as shrimp aquaculture environments (Alagappan et al., 2010; Stalin and Srinivasan, 2017; Matamp and Bhat, 2020). For instance, Alagappan et al. (2010) reported that five bacteriophages with the ability to infect V. parahaemolyticus MTCC-451 strain were isolated from the shrimp pond. Stalin and Srinivasan (2017) reported that bacteriophage VVP1 capable of infecting V. parahaemolyticus as well as V. alginolyticus strains were isolated from the water suspended sediment samples collected from the shrimp grow-out pond. Matamp and Bhat (2020) also reported that bacteriophages ⌽VP-1 which able to lyse five strains of V. alginolyticus and 1 strain of V. parahaemolyticus were isolated from the shrimp pond water samples.\\\\nHost specificity of bacteriophage is one of the vital aspects which should be considered in a bacteriophage application. The specificity of bacteriophages isolated in this study were found to be of a narrow host range that infected only V. parahaemolyticus strains. A narrow host-specific bacteriophage seemed an attractive feature, especially when applying it in the gastrointestinal tract to target specific host bacteria as the narrow host range of the bacteriophage would not affect other endogenous bacteria (Viazis et al., 2011; Drulis-Kawa et al., 2012). Besides, different types of narrow host range bacteriophages can be mixed in a cocktail preparation for a more effective in vitro control of pathogenic bacteria (Tanji et al., 2004; Mapes et al., 2016; Bai et al., 2019).\\\\nBoth the Podoviridae and Siphoviridae families are classified under the same order of Caudovirales. A hexagonal head (41.28 to 50.00 nm) and a short stumpy non-contractile tail (8.72 to 12.44 nm) which resembled the Podoviridae family was observed in the negative staining of bacteriophages Vp33, Vp22, Vp21, and Vp02 under TEM. Similarly, Zhang et al. (2018) reported that a V. parahaemolyticus bacteriophage vB_VpaS_OMN isolated from the seafood sample belonged to the Podoviridae family and was observed to have a 55 nm isometric capsid and a short tail of about 16 nm. On the other hand, the electron micrographs of bacteriophages Vp08 and Vp11 showed an elongated hexagonal head (79.00 × 44.98 to 91.78 × 49.54 nm) and a long flexible non-contractile tail (133.79 to 142.24 nm). These morphological characteristics resemble those of the Siphoviridae family. A recent study by Yang et al. (2020) reported a similar finding that two V. parahaemolyticus bacteriophages, namely vB_VpS_BA3 and vB_VpS_CA8, isolated from the sewage were Siphoviridae bacteriophage with an icosahedral head (70 × 55 nm) and a long non-contractile tail (125 to 130 nm).\\\\nDion et al. (2020) reported that double-stranded DNA (dsDNA) tailed bacteriophages accounted for the majority of bacteriophages reported in the public database. This finding supports with Dion et al. (2020) statement as all bacteriophages in this study encapsulate a dsDNA genome. From the restriction endonuclease digestion, some genetic variation or polymorphisms were observed among bacteriophages. Distinctive restriction digestion profiles between each bacteriophage isolates were observed after the genome of bacteriophages was digested by the HindIII and HaeIII restriction enzymes. From the restriction endonuclease digestion profiles, it can be concluded that all bacteriophage isolates analyzed in this study present some variation in their genetic properties.\\\\nA prominent protein band at around 31 kDa was identified by SDS-PAGE analysis in all Podoviridae bacteriophages, Vp33, Vp22, Vp21, and Vp02. In contrast, bacteriophages Vp08 and Vp11 showed a prominent protein band at around 40 kDa. This prominent protein was assumed to be the major capsid protein of bacteriophage. In other studies, Drulis-Kawa et al. (2012) revealed a 39 kDa major capsid protein in the Klebsiella pneumoniae podovirus bacteriophage, and this protein structure was found highly homologous (78%) to the 37.8 kDa Vibrio bacteriophage VP93 capsid protein. Baudoux et al. (2012) revealed that a Vibrio siphovirus bacteriophage SIO-2 was identified with the presence of a major capsid protein with a molecular mass of 29 kDa. Other protein bands may indicate the presence of putative tail fiber protein (32.7 kDa), putative scaffolding protein (30.1 kDa), putative internal virion protein B (20.4 kDa), hypothetical and/or unknown proteins of Vibrio bacteriophage (Drulis-Kawa et al., 2012; Hu et al., 2020).\\\\nBacteriophages rely on the chance of meeting with bacteria to infect and kill bacteria (Huff et al., 2006). The MOI refers to the ratio of the virus particles to the host cells. In this study, the optimal MOI for bacteriophage propagation was determined to be 0.001 to 1. Different bacteriophages may infect and kill bacteria differently, thus having different optimal MOIs. For example, Yang et al. (2020) reported that two V. parahaemolyticus lytic bacteriophages, vB_VpS_BA3 and vB_VpS_CA8, were tested with optimal MOI of 0.1 from the maximum bacteriophage titer produced. A lytic bacteriophage PhVh6 showing the ability to lyse V. alginolyticus, V. harveyi, and V. parahaemolyticus was tested with an optimal MOI of 1 (Ibrahim et al., 2017).\\\\nFrom the one-step growth curve, the latent period of bacteriophages was estimated to be around 10 to 20 min and the average burst size was anticipated to be approximately 17 to 51 PFU/cell. Previously, vibriophage BA3 and CA8 were estimated as having a latent period of around 20 to 30 min and a burst size of about 100 to 180 PFU/cell (Yang et al., 2020). Another lytic bacteriophage VP06 of V. parahaemolyticus was estimated to have a latent period of approximately 30 min and 60 PFU per infected bacterial cell (Wong et al., 2019). Lal et al. (2016) also reported that a V. parahaemolyticus bacteriophage VpKK5 exhibited a latent period of 36 min and a burst size of 180 PFU/cell. From the results, bacteriophage isolates in this study demonstrated a shorter latent period but a smaller burst size than other reported vibrio bacteriophages.\\\\nPhysico-chemical factors such as temperature and pH level can affect the effectiveness of bacteriophage applications against pathogenic bacteria (Ly-Chatain, 2014). For example, higher temperatures may result in irreversible damage or denaturation of the virus particles (Ahmadi et al., 2017). The concentration of hydrogen ions in an acidic solution can cause a decline in bacteriophage concentration due to the aggregation process (Langlet et al., 2007; Jończyk et al., 2011). In the current study, the effect of temperature on bacteriophage stability revealed that all bacteriophages remained relatively stable at temperatures ranging from −20 to 50°C. At 60°C, bacteriophages Vp33, Vp22, Vp21, and Vp02 which belong to the Podoviridae family showed continuing stability. In contrast, bacteriophages Vp08 and Vp11 which belong to the Siphoviridae family showed a low survival rate at 60°C. Incubation at 70°C showed total inactivation of all bacteriophages. Previous studies reported a similar finding, in which V. parahaemolyticus bacteriophage VVP1 was found to be stable at temperatures 4 to 50°C but completely inactive at 60°C (Stalin and Srinivasan, 2017). Investigation of the effect of pH on bacteriophage stability revealed that all bacteriophages sustained well at pH ranges of 5 to 11. At pH3, bacteriophages Vp22 and Vp21 were observed with a low survival percentage. At pH2, none of the bacteriophages survived this highly acidic environment. Such characteristics align with results reported by Jun et al. (2014) in which a bacteriophage pVp-1, showed infectivity for multiple–antibiotic-resistant V. parahaemolyticus and V. vulnificus, was observed to be stable over a wide range of pH levels of 3 to 11.\\\\nOverall, the bacteriophage suspensions achieved an average log reduction of V. parahaemolyticus in the range of 2.85 ± 0.07 to 4.81 ± 0.14. Bacteriophage cocktail preparations were observed to have a better in vitro lytic activity as compared to a single bacteriophage suspension. However, other attributes such as competition between each V. parahaemolyticus strain and production of antagonistic compounds by V. parahaemolyticus strains that may exhibit antibacterial activity were excluded in the results and should be considered in the further study. Bacteriophage cocktail A achieved a total reduction rate of 3.69 ± 0.49 log CFU/mL, compared to its individual bacteriophage suspension with a reduction rate between 2.85 ± 0.07 and 3.38 ± 0.26 log CFU/mL. Bacteriophage cocktail B achieved a total reduction rate of 4.81 ± 0.14 log CFU/mL, compared to its individual bacteriophage suspension with a reduction rate between 4.22 ± 0.26 and 4.60 ± 0.08 log CFU/mL. According to these results, the application of a bacteriophage cocktail instead of a single bacteriophage suspension could therefore result in an improved reduction in the growth of V. parahaemolyticus.\\\\nMateus et al. (2014) stated that the use of bacteriophage cocktails mixed with two or three bacteriophages can increase the efficiency to control and inactivate the growth of V. parahaemolyticus better than the single bacteriophage suspensions. For example, Mateus et al. (2014) reported that the application of bacteriophage cocktails mixed with the VP-1 and VP-2 bacteriophages achieved a maximum of 4.0 log of V. parahaemolyticus inactivation after 2 h of incubation, compared to the use of VP-1 and VP-2 bacteriophage alone, with a reduction rate of only 0.8 log. Similarly, Ren et al. (2019) reported that a bacteriophage cocktail which consisted of PVP1 and PVP2 bacteriophages was superior in preventing V. parahaemolyticus infections in sea cucumbers than the use of single bacteriophages. It was also recently demonstrated that the benefit of bacteriophage cocktails can help to overcome the development of bacteriophage-resistant pathogen strains (Yang et al., 2020). For instance, Kim et al. (2020) demonstrated that a bacteriophage cocktail consists of three types of V. coralliilyticus-specific bacteriophages with greater efficiency against multiple-antibiotic-resistant (MAR) and phage-resistant (PR) V. coralliilyticus infection in oyster larvae. Yen et al. (2017) reported that a bacteriophage cocktail that targets different receptors on V. cholerae would reduce the emergence of bacteriophage-resistant V. cholerae isolates. Hence, bacteriophage cocktails consisting of multiple bacteriophage suspensions are more effective in the lytic ability.\\\\nConclusion\\\\nVibrio parahaemolyticus specific bacteriophages can be readily recovered from seafood samples. Bacteriophage Vp33, Vp22, Vp21, Vp02, Vp08, and Vp11 isolated from the seafood samples exhibited narrow host range and lytic activity against V. parahaemolyticus strains only. Characterization of bacteriophages in this study facilitates a better understanding of the physiological and biological features of bacteriophages. The results of in vitro lytic activity revealed the usability of bacteriophages as biocontrol agents for the control of V. parahaemolyticus.\\\\nData Availability Statement\\\\nThe raw data supporting the conclusions of this article will be made available by the authors, without undue reservation.\\\\nAuthor Contributions\\\\nNA-M, NJ, TT, and EL provided assistance and guidance throughout the research. YR, HaH, HiH, and SR were the mentor in the research study and assist in manuscript checking. All authors contributed to the article and approved the submitted version.\\\\nFunding\\\\nThis work was supported by the Fundamental Research Grant Scheme (FRGS) of the Ministry of Higher Education (MOHE), Malaysia (FRGS/1/2018/STG05/UPM/01/7; 01-01-18-2015FR) and Grant Putra IPS from Universiti Putra Malaysia (GP-IPS 9438703). Publication of this work was funded by UPM Journal Publication Fund/Dana Penerbitan Jurnal UPM from Research Management Centre (RMC) of Universiti Putra Malaysia.\\\\nConflict of Interest\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nReferences\\\\nAdams, M. J., Lefkowitz, E. J., King, A. M., Harrach, B., Harrison, R. L., Knowles, N. J., et al. (2017). 50 years of the international committee on taxonomy of viruses: progress and prospects. Arch. Virol. 162, 1441–1446. doi: 10.1007/s00705-016-3215-y\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAdriaenssens, E. M., Sullivan, M. B., Knezevic, P., van Zyl, L. J., Sarkar, B. L., Dutilh, B. E., et al. (2020). Taxonomy of prokaryotic viruses: 2018-2019: update from the ICTV bacterial and archaeal viruses subcommittee. Arch. Virol. 165, 1253–1260. doi: 10.1007/s00705-020-04577-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAhmad, A., Brumble, L., and Maniaci, M. (2013). Vibrio parahaemolyticus induced necrotising fasciitis: an atypical organism causing an unusual presentation. Case Rep. Infect. Dis. 2013:216854. doi: 10.1155/2013/216854\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAhmadi, H., Radford, D., Kropinski, A. M., Lim, L. T., and Balamurugan, S. (2017). Thermal-stability and reconstitution ability of Listeria phages P100 and A511. Front. Microbiol. 8:2375. doi: 10.3389/fmicb.2017.02375\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAkhwale, J. K., Rohde, M., Rohde, C., Bunk, B., Spröer, C., Boga, H. I., et al. (2019). Isolation, characterisation and analysis of bacteriophages from the haloalkaline lake Elmenteita, Kenya. PLoS One 14:e0215734. doi: 10.1371/journal.pone.0215734\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAlagappan, K. M., Deivasigamani, B., Somasundaram, S. T., and Kumaran, S. (2010). Occurrence of Vibrio parahaemolyticus and its specific phages from shrimp ponds in east coast of India. Curr. Opin. Microbiol. 61, 235–240. doi: 10.1007/s00284-010-9599-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBai, J., Jeon, B., and Ryu, S. (2019). Effective inhibition of Salmonella Typhimurium in fresh produce by a phage cocktail targeting multiple host receptors. J. Food Microbiol. 77, 52–60. doi: 10.1016/j.fm.2018.08.011\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBaudoux, A. C., Hendrix, R. W., Lander, G. C., Bailly, X., Podell, S., Paillard, C., et al. (2012). Genomic and functional analysis of Vibrio phage SIO-2 reveals novel insights into ecology and evolution of marine siphoviruses. Environ. Microbiol. 14, 2071–2086. doi: 10.1111/j.1462-2920.2011.02685.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBeck, N. K., Callahan, K., Nappier, S. P., Kim, H., Sobsey, M. D., and Meschke, J. S. (2009). Development of a spot-titer culture assay for quantifying bacteria and viral indicators. J. Rapid Methods Autom. Microbiol. 17, 455–464. doi: 10.1111/j.1745-4581.2009.00182.x\\\\nCrossRef Full Text | Google Scholar\\\\nBhoopong, P., Palittapongarnpim, P., Pomwised, P., Kiatkittipong, A., Kamruzzaman, M., Nakaguchi, Y., et al. (2007). Variability in properties of Vibrio parahaemolyticus strains isolated from single patients. J. Clin. Microbiol. 45, 1544–1550. doi: 10.1128/JCM.02371-06\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBiswas, B., Adhya, S., Washart, P., Paul, B., Trostel, A. N., Powell, B., et al. (2002). Bacteriophage therapy rescues mice bacteremic from a clinical isolate of vancomycin-resistant Enterococcus faecium. Infect. Immun. 70, 204–210. doi: 10.1128/iai.70.1.204-210.2002\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBonilla, N., Rojas, M. I., Netto Flores, Cruz, G., Hung, S. H., Rohwer, F., et al. (2016). Phage on tap-a quick and efficient protocol for the preparation of bacteriophage laboratory stocks. PeerJ. 4:e2261. doi: 10.7717/peerj.2261\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChen, Y., Chen, X., Yu, F., Wu, M., Wang, R., Zheng, S., et al. (2016). Serology, virulence, antimicrobial susceptibility and molecular characteristics of clinical Vibrio parahaemolyticus strains circulating in southeastern China from 2009 to 2013. Clin. Microbiol. Infect. 22, 258.e9–258.e16. doi: 10.1016/j.cmi.2015.11.003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nComeau, A. M., Hatfull, G. F., Krisch, H. M., Lindell, D., Mann, N. H., and Prangishvili, D. (2008). Exploring the prokaryotic virosphere. Res. Microbiol. 159, 306–313. doi: 10.1016/j.resmic.2008.05.001\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDrulis-Kawa, Z., Majkowska-Skrobek, G., Maciejewska, B., Delattre, A. S., and Lavigne, R. (2012). Learning from bacteriophages - advantages and limitations of phage and phage-encoded protein applications. Curr. Protein Pept. Sci. 13, 699–722. doi: 10.2174/138920312804871193\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nElmahdi, S., DaSilva, L. V., and Parveen, S. (2016). Antibiotic resistance of Vibrio parahaemolyticus and Vibrio vulnificus in various countries: a review. Food Microbiol. 57, 128–134. doi: 10.1016/j.fm.2016.02.008\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGuan, S. M., Nagata, H., Maeda, K., Kuboniwa, M., Minamino, N., and Shizukuishi, S. (2004). Purification and characterization of a hemoglobin-binding outer membrane protein of Prevotella intermedia. FEMS Microbiol. Lett. 235, 333–339. doi: 10.1111/j.1574-6968.2004.tb09607.x\\\\nCrossRef Full Text | Google Scholar\\\\nHagens, S., and Loessner, M. J. (2007). Application of bacteriophages for detection and control of foodborne pathogens. Appl. Microbiol. Biotechnol. 76, 513–519. doi: 10.1007/s00253-007-1031-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHong, Y., Pan, Y., and Ebner, P. D. (2014). Meat science and muscle biology symposium: development of bacteriophage treatments to reduce Escherichia coli O157:H7 contamination of beef products and produce1. J. Anim. Sci. 92, 1366–1377. doi: 10.2527/jas.2013-7272\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHu, M., Zhang, H., Gu, D., Ma, Y., and Zhou, X. (2020). Identification of a novel bacterial receptor that binds tail tubular proteins and mediates phage infection of Vibrio parahaemolyticus. Emerg. Microbes Infect. 9, 855–867. doi: 10.1080/22221751.2020.1754134\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHuang, C., Virk, S. M., Shi, J., Zhou, Y., Willias, S. P., Morsy, M. K., et al. (2018). Isolation, characterisation, and application of bacteriophage LPSE1 against Salmonella enterica in ready to eat (RTE) foods. Front. Microbiol. 9:1046. doi: 10.3389/fmicb.2018.01046\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHuff, W. E., Huff, G. R., Rath, N. C., and Donoghue, A. M. (2006). Evaluation of the influence of bacteriophage titer on the treatment of colibacillosis in broiler chickens. Poult. Sci. 85, 1373–1377. doi: 10.1093/ps/85.8.1373\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nIbrahim, W. N. W., Aznan, A. S., Saari, N. A., Leong, L. K., Musa, N., Razzak, L. A., et al. (2017). In-vitro characterization of lytic bacteriophage PhVh6 as potential biocontrol agent against pathogenic Vibrio harveyi. Aquac. Aquar. Conserv. Legis. 10, 64–76.\\\\nGoogle Scholar\\\\nJończyk, E., Kłak, M., Międzybrodzki, R., and Górski, A. (2011). The influence of external factors on bacteriophages - review. Folia Microbiol. 56, 191–200. doi: 10.1007/s12223-011-0039-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJones, J. L., Ludeke, C. H., Bowers, J. C., Garrett, N., Fischer, M., Parsons, M. B., et al. (2012). Biochemical, serological, and virulence characterization of clinical and oyster Vibrio parahaemolyticus isolates. J. Clin. Microbiol. 50, 2343–2352. doi: 10.1128/JCM.00196-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJun, J. W., Shin, T. H., Kim, J. H., Shin, S. P., Han, J. E., Heo, G. J., et al. (2014). Bacteriophage therapy of a Vibrio parahaemolyticus infection caused by a multiple-antibiotic–resistant O3:K6 pandemic clinical strain. J. Infect. Dis. 210, 72–78. doi: 10.1093/infdis/jiu059\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKalatzis, P. G., Castillo, D., Katharios, P., and Middelboe, M. (2018). Bacteriophage interactions with marine pathogenic vibrios: implications for phage therapy. Antibiot. Basel 7:15. doi: 10.3390/antibiotics7010015\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKang, C. H., Shin, Y., Jang, S., Yu, H., Kim, S., An, S., et al. (2017). Characterisation of Vibrio parahaemolyticus isolated from oysters in Korea: resistance to various antibiotics and prevalence of virulence genes. Mar. Pollut. Bull. 118, 261–266. doi: 10.1016/j.marpolbul.2017.02.070\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKim, H. J., Jun, J. W., Giri, S. S., Kim, S. G., Kim, S. W., Kwon, J., et al. (2020). Bacteriophage cocktail for the prevention of multiple-antibiotic-resistant and mono-phage-resistant Vibrio coralliilyticus infection in pacific oyster (Crassostrea gigas) larvae. Pathogens 9:813. doi: 10.3390/pathogens9100831\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKim, S. G., Jun, J. W., Giri, S. S., Yun, S., Kim, H. J., Kim, S. W., et al. (2019). Isolation and characterisation of pVa-21, a giant bacteriophage with anti-biofilm potential against Vibrio alginolyticus. Sci. Rep. 9:6284. doi: 10.1038/s41598-019-42681-1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKropinski, A. M., Mazzocco, A., Waddell, T. E., Lingohr, E., and Johnson, R. P. (2009). ““Enumeration of bacteriophages by double agar overlay plaque assay,”,” in Bacteriophages: Methods in Molecular Biology, eds M. R. Clokie and A. M. Kropinski (Totowa, NJ: Humana Press), 69–76.\\\\nGoogle Scholar\\\\nKrupovic, M., Dutilh, B. E., Adriaenssens, E. M., Wittmann, J., Vogensen, F. K., Sullivan, M. B., et al. (2016). Taxonomy of prokaryotic viruses: update from the ICTV bacterial and archaeal viruses subcommittee. Arch. Virol. 161, 1095–1099. doi: 10.1007/s00705-015-2728-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLal, T. M., Sano, M., and Ransangan, J. (2016). Genome characterization of a novel vibriophage VpKK5 (Siphoviridae) specific to fish pathogenic strain of Vibrio parahaemolyticus. J. Basic Microbiol. 56, 872–888. doi: 10.1002/jobm.201500611\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLanglet, J., Gaboriaud, F., and Gantzer, C. (2007). Effects of pH on plaque forming unit counts and aggregation of MS2 bacteriophage. J. Appl. Microbiol. 103, 1632–1638. doi: 10.1111/j.1365-2672.2007.03396.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLe, T. S., Southgate, P. C., O’Connor, W., Abramov, T., Shelley, D. V., Vu, S., et al. (2020a). Use of bacteriophages to control Vibrio contamination of microalgae used as a food source for oyster larvae during hatchery culture. Curr. Microbiol. 77, 1811–1820. doi: 10.1007/s00284-020-01981-w\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLe, T. S., Southgate, P. C., O’Connor, W., Vu, S. V., and Kurtböke, D. İ (2020b). Application of Bacteriophages to control Vibrio alginolyticus contamination in oyster (Saccostrea glomerata) larvae. Antibiotics 9:415. doi: 10.3390/antibiotics9070415\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLi, Y., Xie, X., Shi, X., Lin, Y., Qiu, Y., Mou, J., et al. (2014). Vibrio parahaemolyticus, southern coastal region of China, 2007-2012. Emerg. Infect. Dis. 20, 685–688. doi: 10.3201/eid2004.130744\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLin, D. M., Koskella, B., and Lin, H. C. (2017). Phage therapy: an alternative to antibiotics in the age of multi-drug resistance. World J. Gastrointest. Pharmacol. Ther. 8, 162–173. doi: 10.4292/wjgpt.v8.i3.162\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLiu, A., Liu, Y., Peng, L., Cai, X., Shen, L., Duan, M., et al. (2020). Characterization of the narrow-spectrum bacteriophage LSE7621 towards Salmonella Enteritidis and its biocontrol potential on lettuce and tofu. LWT Food Sci. Technol. 118:108791. doi: 10.1016/j.lwt.2019.108791\\\\nCrossRef Full Text | Google Scholar\\\\nLiu, W., Ye, D., Luo, Z., Yu, Y., and Gong, T. (2013). Isolation and characterisation of five lytic bacteriophages infecting a Vibrio strain closely related to Vibrio owensii. FEMS Microbiol. Lett. 348, 112–119. doi: 10.1111/1574-6968.12277\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMapes, A. C., Trautner, B. W., Liao, K. S., and Ramig, R. F. (2016). Development of expanded host range phage active on biofilms of multi-drug resistant Pseudomonas aeruginosa. Bacteriophage 6:e1096995. doi: 10.1080/21597081.2015.1096995\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMatamp, N., and Bhat, S. G. (2020). Genome characterisation of novel lytic Myoviridae bacteriophage ⌽VP-1 enhances its applicability against MDR-biofilm-forming Vibrio parahaemolyticus. Arch. Virol. 165, 387–396. doi: 10.1007/s00705-019-04493-6\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMateus, L., Costa, L., Silva, Y. J., Pereira, C., Cunha, A., and Almeida, A. (2014). Efficiency of phage cocktails in the inactivation of Vibrio in aquaculture. Aquaculture 42, 167–173. doi: 10.1016/j.aquaculture.2014.01.001\\\\nCrossRef Full Text | Google Scholar\\\\nNecel, A., Bloch, S., Nejman-Faleńczyk, B., Grabski, M., Topka, G., Dydecka, A., et al. (2020). Characterization of a bacteriophage, vB_Eco4M-7, that effectively infects many Escherichia coli O157 strains. Sci. Rep. 10:3743. doi: 10.1038/s41598-020-60568-4\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nParia, P., Kunal, S. P., Behera, B. K., Mohapatra, P. K. D., Das, A., Parida, P. K., et al. (2019). Molecular characterisation and genetic diversity study of Vibrio parahaemolyticus isolated from aquaculture farms in India. Aquaculture 509, 104–111. doi: 10.1016/j.aquaculture.2019.04.076\\\\nCrossRef Full Text | Google Scholar\\\\nPazhana, G. P., Bhowmik, S. K., Ghosh, S., Guin, S., Dutta, S., Rajendran, K., et al. (2014). Trends in the epidemiology of pandemic and non-pandemic strains of Vibrio parahaemolyticus isolated from diarrheal patients in Kolkata, India. PLoS Negl. Trop. Dis. 8:e2815. doi: 10.1371/journal.pntd.0002815\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRen, H., Li, Z., Xu, Y., Wang, L., and Li, X. (2019). Protective effectiveness of feeding phage cocktails in controlling Vibrio parahaemolyticus infection of sea cucumber Apostichopus japonicus. Aquaculture 503, 322–329. doi: 10.1016/j.aquaculture.2019.01.006\\\\nCrossRef Full Text | Google Scholar\\\\nSalifu, S. P., Casey, S. A. C., and Foley, S. (2013). Isolation and characterisation of soilborne virulent bacteriophages infecting the pathogen Rhodococcus equi. J. Appl. Microbiol. 114, 1625–1633. doi: 10.1111/jam.12194\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSchramlová, J., Arientová, S., and Hulínská, D. (2010). The role of electron microscopy in the rapid diagnosis of viral infections–review. Folia Microbiol. 55, 88–101. doi: 10.1007/s12223-010-0015-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSevilla-Navarro, S., Catalá-Gregori, P., García, C., Cortés, V., and Marin, C. (2020). Salmonella Infantis and Salmonella Enteritidis specific bacteriophages isolated form poultry faeces as a complementary tool for cleaning and disinfection against Salmonella. Comp. Immunol. Microbiol. Infect. Dis. 68:101405. doi: 10.1016/j.cimid.2019.101405\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nShende, R. K., Hirpurkar, S. D., Sannat, C., Rawat, N., and Pandey, V. (2017). Isolation and characterisation of bacteriophages with lytic activity against common bacterial pathogens. Vet. World 10, 973–978. doi: 10.14202/vetworld.2017.973-978\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nStalin, N., and Srinivasan, P. (2017). Efficacy of potential phage cocktails against Vibrio harveyi and closely related Vibrio species isolated from shrimp aquaculture environment in the south east coast of India. Vet. Microbiol. 207, 83–96. doi: 10.1016/j.vetmic.2017.06.006\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTan, C. W., Malcolm, T., Kuan, C. H., Thung, T. Y., Chang, W. S., Loo, Y. Y., et al. (2017). Prevalence and antimicrobial susceptibility of Vibrio parahaemolyticus isolated from short mackerels (Rastrelliger brachysoma) in Malaysia. Front. Microbiol. 8:1087. doi: 10.3389/fmicb.2017.01087\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTang, F., Zhang, P., Zhang, Q., Xue, F., Ren, J., Sun, J., et al. (2019). Isolation and characterisation of a broad-spectrum phage of multiple drug resistant Salmonella and its therapeutic utility in mice. Microb. Pathog. 126, 193–198. doi: 10.1016/j.micpath.2018.10.042\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTanji, Y., Shimada, T., Yoichi, M., Miyanaga, K., Hori, K., and Unno, H. (2004). Toward rational control of Escherichia coli O157:H7 by a phage cocktail. Appl. Microbiol. Biotechnol. 64, 270–274. doi: 10.1007/s00253-003-1438-9\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTena, D., Arias, M., Alvarez, B. T., Mauleon, C., Jimenez, M. P., and Bisquert, J. (2010). Fulminant necrotising fasciitis due to Vibrio parahaemolyticus. J. Med. Microbiol. 59, 235–238. doi: 10.1099/jmm.0.014654-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nThung, T. Y., Premarathne, J. M. K. J. K., Chang, W. S., Loo, Y. Y., Chin, Y. Z., Kuan, C. H., et al. (2017). Use of a lytic bacteriophage to control Salmonella Enteritidis in retail food. LWT Food Sci. Technol. 78, 222–225. doi: 10.1016/j.lwt.2016.12.044\\\\nCrossRef Full Text | Google Scholar\\\\nTopka, G., Bloch, S., Nejman-Faleńczyk, B., Gąsior, T., Jurczak-Kurek, A., Necel, A., et al. (2019). Characterization of bacteriophage vB-EcoS-95, isolated from urban sewage and revealing extremely rapid lytic development. Front. Microbiol. 9:3326. doi: 10.3389/fmicb.2018.03326\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nViazis, S., Akhtar, M., Feirtag, J., and Brabban, A. D. (2011). Isolation and characterisation of lytic bacteriophages against enterohaemorrhagic Escherichia coli. J. Appl. Microbiol. 110, 1323–1331. doi: 10.1111/j.1365-2672.2011.04989.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWang, J., Hu, B. E. I., Xu, M., Yan, Q. U. N., Liu, S., Zhu, X., et al. (2006). Therapeutic effectiveness of bacteriophages in the rescue of mice with extended spectrum ß-lactamase-producing Escherichia coli bacteremia. Int. J. Mol. Med. 17, 347–355. doi: 10.3892/ijmm.17.2.347\\\\nCrossRef Full Text | Google Scholar\\\\nWang, Y., Barton, M., Elliott, L., Li, X., Abraham, S., O’Dea, M., et al. (2017). Bacteriophage therapy for the control of Vibrio harveyi in greenlip abalone (Haliotis laevigata). Aquaculture 473, 251–258. doi: 10.1016/j.aquaculture.2017.01.003\\\\nCrossRef Full Text | Google Scholar\\\\nWatanabe, R., Matsumoto, T., Sano, G., Ishii, Y., Tateda, K., Sumiyama, Y., et al. (2007). Efficacy of bacteriophage therapy against gut-derived sepsis caused by Pseudomonas aeruginosa in mice. Antimicrob. Agents Chemother. 51, 446–452. doi: 10.1128/AAC.00635-06\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWong, H. C., Wang, T. Y., Yang, C. W., Tang, C. T., Ying, C., Wang, C. H., et al. (2019). Characterization of a lytic vibriophage VP06 of Vibrio parahaemolyticus. Res. Microbiol. 170, 13–23. doi: 10.1016/j.resmic.2018.07.003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYang, M., Liang, Y., Huang, S., Zhang, J., Wang, J., Chen, H., et al. (2020). Isolation and characterization of the novel phages vB_VpS_BA3 and vB_VpS_CA8 for lysing Vibrio parahaemolyticus. Front. Microbiol. 11:259. doi: 10.3389/fmicb.2020.00259\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYen, M., Cairns, L. S., and Camilli, A. (2017). A cocktail of three virulent bacteriophages prevents Vibrio cholerae infection in animal models. Nat. Commun. 8:14187. doi: 10.1038/ncomms14187\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nZhang, H., Yang, Z., Zhou, Y., Bao, H., Wang, R., Li, T., et al. (2018). Application of a phage in decontaminating Vibrio parahaemolyticus in oysters. Int. J. Food Microbiol. 275, 24–31. doi: 10.1016/j.ijfoodmicro.2018.03.027\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.01533\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.01533/full\",\\n    \"loadedTime\": \"2024-01-23T08:49:20.995Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.01533\",\\n    \"title\": \"Frontiers | A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia\",\\n    \"description\": \"Erwinia amylovora is a plant pathogen from the Erwiniaceae family and a causative agent of the devastating agricultural disease fire blight. Here we characterize eight lytic bacteriophages of E. amylovora that we isolated from the Wasatch front (Utah, USA) that are highly similar to vB_EamM_Ea35-70 which was isolated in Ontario, Canada. With the genome size ranging from 271-275 kb, this is a novel jumbo family of bacteriophages. These jumbo bacteriophages were further characterized through genomic and proteomic comparison, mass spectrometry, host range and burst size. Their proteomes are highly unstudied, with over 200 putative proteins with no known homologs. The production of 27 of these putative proteins was confirmed by mass spectrometry analysis. These bacteriophages appear to be most similar to bacteriophages that infect Pseudomonas and Ralstonia rather than Enterobacteriales bacteria by protein similarity, however we were only able to detect infection of Erwinia and the closely related strains of Pantoea. \",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nWhitman et al. (1998) estimated that there are approximately 5 × 1030 bacteria on earth, which is more than the number of plants and animals combined. Most, or likely all, bacteria are subject to infection by one or more viruses or “bacteriophages,” making bacteriophages the most common and diverse biological entity at an estimated 1032 (Bergh et al., 1989; Wommack and Colwell, 2000; Hambly and Suttle, 2005). Bacteriophages were likely first reported in 1896, when Ernest Hanbury Hankin discovered antibacterial activity against cholera in the waters of two large rivers in India, the Ganges and Yamuna (Abedon et al., 2011). They were independently characterized and named in the 1900s by bacteriologist Twort (1915, 1936) and microbiologist D’Herelle (1917, 2007), Keen (2015). During the infection process, bacteriophages can transfer foreign DNA to their host (including virulence factors), integrate into the host genome, and/or kill their host through cell lysis (Chen and Novick, 2009). The sheer number of bacteriophages combined with their clear evolutionary influence makes them an important target for understanding the ecology and evolution of bacteria, including pathogenic strains (Bollback and Huelsenbeck, 2009; Boyd, 2012). In addition, their specificity, genomic plasticity, and rapid multiplication rates make them a potential weapon to treat bacterial infections (Sulakvelidze, 2005; Shivaswamy et al., 2015).\\\\nOne such bacterial infection caused by a phytopathogen Erwinia amylovora (Zwet and Keil, 1979) is called fire blight and mainly affects ornamental plants of the Rosaceae family. The symptoms of the infected tissues include wilting, ooze production and death of blossoms, shoots branches and entire trees (Thomson, 2000). We have recently isolated and characterized twenty eight bacteriophages that infect E. amylovora (Esplin et al., 2017; Sharma et al., 2018). Out of these 28, there is a distinct group of eight highly related bacteriophages: vB_EamM_Special G (Special G), vB_EamM_Simmy50 (Simmy50), vB_EamM_RAY (RAY), vB_EamM_Deimos-Minion (Deimos-Minion or DM), vB_EamM_Bosolaphorus (Bosolaphorus), vB_EamM_Desertfox (Desertfox), vB_EamM_MadMel (MadMel), and vB_EamM_Mortimer (Mortimer) very similar to Erwinia bacteriophage Ea35-70 which was isolated in Ontario, Canada (Yagubi et al., 2014). These nine bacteriophages were recently added as the Agrican357virus genus of bacteriophages by the ICTV (Kuhn et al., 2013; Adams et al., 2017) and are considered jumbo bacteriophages due to their large genome (>200 kb) and particle size (Yuan and Gao, 2017).\\\\nAs reviewed in 2017, jumbo bacteriophages have diverse genome sizes (ranging from 208 to 497 kb) as well as diverse virion morphology and complex virion structure (Yuan and Gao, 2017). They often encode greater than 60 structural proteins with some displaying complex head structures composed of more than five proteins (Effantin et al., 2013) or long, wavy, curly tail fibers (Yuan and Gao, 2016). Jumbo bacteriophages were also found to be highly diverse, with over 11 clusters and five singleton bacteriophages suggested from 52 complete jumbo bacteriophage genomes analyzed in 2017, many of which are uncharacterized (Yuan and Gao, 2017). Only a few jumbo bacteriophage families have been characterized beyond sequence analysis and EM, including the phiKZ-like bacteriophages 201phi2-1 (Thomas et al., 2008), KTN4 (Danis-Wlodarczyk et al., 2016), phiPA3 (Monson et al., 2011), phiRSL2 (Bhunchoth et al., 2016), phiRSF1 (Bhunchoth et al., 2016), OBP (Shaburova et al., 2006), EL (Sokolova et al., 2014), and phiKZ (Lecoutere et al., 2009), related bacteriophages phiRSL1 (Yamada et al., 2010) and PaBG (Kurochkina et al., 2018), Cronobacter bacteriophage CR5 (Lee et al., 2016), Prochlorococcus bacteriophage P-SSM2 (Sullivan et al., 2005), related bacteriophages KVP40 (Miller et al., 2003) and Aeh1 (Gibb and Edgell, 2007), Aeromonas bacteriophage phiAS5 (Kim et al., 2012), Pectobacterium bacteriophage CBB (Buttimer et al., 2017), Caulobacter bacteriophage phiCbK (Gill et al., 2012), related Erwinia bacteriophages Joad and RisingSun (Arens et al., 2018), related bacteriophage RaK2 (Simoliunas et al., 2013) and GAP32 (Abbasifar et al., 2014), Bacillus bacteriophage 0305phi8-36 (Thomas et al., 2007), related Bacillus bacteriophages BpSp (Yuan and Gao, 2016) and AR9 (Lavysh et al., 2016, 2017). Herein we further analyze the genome, proteome, and host range of our eight Agrican357virus jumbo bacteriophages. Their lytic nature and plethora of novel genes makes them a unique entity to be studied further and analyzed. As a close relative of the animal pathogens Escherichia coli and Salmonella (Zhao and Qi, 2011), viruses that infect E. amylovora may help us understand the evolution of pathogenic strains in this family.\\\\nMaterials and Methods\\\\nBacteriophage Isolation and Genome Sequencing\\\\nEnvironmental samples of leaves, branches and soil surrounding infected trees were collected from around the state of Utah (United States) and used to create enrichment cultures with the host E. amylovora. To test the presence of amplified bacteriophages, the enrichment cultures were spun at 4000 rpm and 4°C for 20 min and the supernatant was removed and used without filtering. 50 μL of this supernatant was incubated at room temperature with 500 μL of E. amylovora ATCC 29780 bacteria for 30–45 min, mixed with 5 ml NBDYE top agar (at half concentration agar), plated on NBSYE agar plate, and incubated at 25°C overnight. Plaque presence on the plates was the primary indicator of bacteriophage presence in the environmental sample. Using a sterile needle or pipette tip, we picked a plaque from the initial bacteriophage identification plate and performed three rounds of plaque purification. All eight isolated bacteriophages: Special G (KU886222), Simmy50 (KU886223), RAY (KU886224), Deimos-Minion (KU886225), Bosolaphorus (MG655267), Desertfox (MG655268), MadMel (MG655269), and Mortimer (MG655270) were able to infect E. amylovora ATCC 29780 (Esplin et al., 2017; Sharma et al., 2018). Bacteriophage DNA was extracted using the Phage DNA isolation kit (Norgen Biotek Corporation), and was sequenced, assembled and annotated as previously described (Esplin et al., 2017; Sharma et al., 2018).\\\\nElectron Microscopy\\\\nElectron microscopy was performed at Brigham Young University in the Life Sciences Microscopy Lab using a FEI Helios NATOCAB 600i DualBeam FIB/SEM with STEM detector. The samples for SEM analysis were prepared by placing 15 μL of high-titer bacteriophage lysate on a 200-mesh copper carbon type-B electron microscope grid for one-two minutes. The lysate was wicked away and the grids were stained for 2 min using 15 μL of 2% phosphotungstic acid (pH = 7). Residual liquid was wicked away using Kimtech wipes and the grid was allowed to dry before being imaged. Bacteriophage structures in electron micrographs were measured using ImageJ (Abramoff et al., 2004). The average and standard deviation for each measurement was calculated from a minimum of four separate measurements.\\\\nBurst Size\\\\nBurst size was calculated by performing single-infection assay as described by Delbruck (1945). The bacteria-bacteriophage mixture was allowed to adsorb for 10 min at a multiplicity of infection (MOI) of 100. The lysate was then removed at different time-intervals ranging from 1 to 6 h and diluted to avoid secondary infection. Soft agar plaque method was used to determine titers and a graph of 10 separate readings was plotted with their average titers and time.\\\\nHost Range\\\\nHost range of all eight bacteriophages was determined using the soft agar plaque method (Hockett and Baltrus, 2017). For this, 50 μL of bacteriophage lysate dilutions were incubated with 500 μL of bacteria grown overnight for 30 min before plating in top agar. The plates were incubated with the top agar facing up at 25°C overnight for this assay. Seventeen bacterial strains including E. amylovora ATCC 29780 (Ritchie and Klos, 1976) as control were used including five other E. amylovora strains [Ea110 (Ritchie and Klos, 1976), GH9 (McGhee and Sundin, 2012), EaBH (McGhee and Sundin, 2012), RB02 (McGhee and Sundin, 2012), Ea273 (Sebaihia et al., 2010)], Pantoea agglomerans E325 (Pusey, 1997), Pantoea vagans C-91 (Ishimaru et al., 1987; Smits et al., 2010), E. coli K-12 BW 25113 (Datsenko and Wanner, 2000), Salmonella enterica LT2 (generously donated by John Roth lab), Yersinia pestis KIM6 (Fetherston et al., 1992; Gong et al., 2001), Enterobacter cloacae ATCC 13047 (Ren et al., 2010), Klebsiella pneumoniae ATCC 10031 (Otman et al., 2007), Bacillus subtilis ATCC 6033 (Nakamura et al., 1999), Cronobacter sakazakii ATCC 29544 (Iversen et al., 2007; Gicova et al., 2014), standard clinical isolate Pseudomonas aeruginosa PA100 (Cartagena et al., 2007) and Pseudomonas chlororaphis ATCC 13985 (Conway et al., 1956). An average of two readings was taken to obtain bacteriophage titers post-infection.\\\\nComputational Analysis and Genomic Comparison\\\\nBacteriophages with any similarities to Agrican357virus genus were identified using a blastx analysis of their putative major capsid and terminase proteins, and the corresponding bacteriophage for all retrieved hits with a cutoff e-value of less than 1.00E-04 and 33% similarity were downloaded from GenBank. In addition, any bacteriophages that showed up in at least three qblast hits while annotating were also retrieved. These sequences were then used in Gepard (Krumsiek et al., 2007) to generate the dot plots of nucleic acid and protein sequences. PhamDB, a web interface (Lamine et al., 2016) was used for creating databases and Phamerator, (Cresawn et al., 2011) an open-source program was used to generate pham map of bacteriophage genomes. PhamDB uses kClust (Hauser et al., 2013) to cluster large protein sequence databases. The default settings of PhamDB were used in this comparison. Splitstree (Huson and Bryant, 2006) protein analysis was produced from the exported pham table of conserved proteins converted to a Nexus file using Janus1. The Average Nucleotide Identity (ANI) percentages comparing each of the E. amylovora bacteriophage genomes were calculated using MAFFT (Katoh et al., 2002) plugin in Geneious R8.1 (Kearse et al., 2012).\\\\nThe evolutionary history was inferred by using the Maximum Likelihood method and Poisson correction model (Zuckerkandl and Pauling, 1965). The bootstrap consensus tree inferred from 100 replicates (Felsenstein, 1985) is taken to represent the evolutionary history of the taxa analyzed (Felsenstein, 1985). Branches corresponding to partitions reproduced in less than 50% bootstrap replicates are collapsed. The percentage of replicate trees in which the associated taxa clustered together in the bootstrap test (100 replicates) are shown next to the branches (Felsenstein, 1985). Initial tree(s) for the heuristic search were obtained automatically by applying Neighbor-Join and BioNJ algorithms to a matrix of pairwise distances estimated using a JTT model, and then selecting the topology with superior log likelihood value. This analysis involved 59 amino acid sequences. There were a total of 1302 positions in the final dataset. Evolutionary analyses were conducted in MEGA X (Kumar et al., 2018).\\\\nMass Spectrometry\\\\nSample preparation was performed (Guttman et al., 2009) by diluting crude lysates of RAY and Deimos-Minion in TNE (50 mM Tris pH 8.0, 100 mM NaCl, 1 mM EDTA) buffer and adding RapiGest SF reagent (Waters Corp.) to a final concentration of 0.1%. Samples were then boiled for 5 min followed by addition of 1 mM (final concentration) of TCEP [Tris (2-carboxyethyl) phosphine] and incubated at 37°C for 30 min. Afterward, carboxymethylation of samples was done with 0.5 mg/ml of iodoacetamide for 30 min at 37°C followed by neutralization with 2 mM TCEP (final concentration). Trypsin (trypsin: protein ratio – 1:50) was used overnight at 37°C to digest the crude lysates prepared as above. The samples were treated with 250 mM HCl at 37°C for 1 h followed by centrifugation at 14000 rpm for 30 min at 4°C to degrade and remove RapiGest. The soluble fraction was then added to a new tube and Aspire RP30 desalting columns (Thermo Fisher Scientific) were used for extraction and desalting of the peptides.\\\\nHigh pressure liquid chromatography (HPLC) coupled with tandem mass spectroscopy (LC-MS/MS) using nano spray ionization was used to analyze Trypsin-digested peptides (McCormack et al., 1997). A TripleT of 5600 hybrid mass spectrometer (ABSCIEX) interfaced with nano-scale reversed-phase HPLC (Tempo) using a 10 cm-100-micron ID glass capillary packed with 5-μm C18 ZorbaxTM beads (Agilent Technologies, Santa Clara, CA) was used to perform the nano-spray ionization experiments. By using a linear gradient (5–60%) of ACN (Acetonitrile) at a flow rate of 250 μl/min for 1 h, peptides were eluted from the C18 column into the mass spectrometer The ACN gradient was created using these buffers: buffer A (98% H2O, 2% ACN, 0.2% formic acid, and 0.005% TFA) and buffer B (100% ACN, 0.2% formic acid, and 0.005% TFA). In a data-dependent manner MS/MS data were acquired in which the MS1 data was acquired for 250 ms at m/z of 400 to 1250 Da and the MS/MS data was acquired from m/z of 50 to 2,000 Da. For Independent data acquisition (IDA) parameters MS1-TOF 250 ms, followed by 50 MS2 events of 25 ms each. The IDA criteria; over 200 counts threshold, charge state of plus 2–4 with 4 s exclusion window. Finally, MASCOT® (Matrix Sciences) was used to analyze the collected data and Protein Pilot 4.0 (ABSCIEX) was used for peptide identifications.\\\\nExtracellular Polymeric Substance (EPS) Depolymerase Mediated Biofilm Degradation Assay\\\\nSoft agar plaque method (Hockett and Baltrus, 2017), as described previously in host range method, was used to detect the presence of halo zone on P. vagans strain C9-1 and E. amylovora ATCC 29780. The putative EPS-depolymerase from bacteriophage RAY was PCR amplified from lysate using primers designed to amplify the full length gp76. It was cloned by digesting with enzymes NdeI/SalI into a similarly digested pET15b. The resulting plasmid (JG1700) was amplified by transforming into E. coli DH5α and plated on LB-amp. Resulting colonies were PCR checked and were used to start overnight cultures and DH5α without plasmid pJG1700 was grown as a control. The protein was induced using IPTG and extracted by lysing cells via sonication. Post-sonication, cell debris was removed from both cultures by centrifuging at 12000 rpm and 4°C for 2 × 20 min. 10 μl of resulting supernatant was spotted on bacterial lawns of P. vagans strain C9-1 and E. amylovora ATCC 29780 embedded in top agar after plating for 2 h.\\\\nMotif Identification and Analysis\\\\nMEME (Bailey et al., 2009) and FIMO (Grant et al., 2011) tools at public phage galaxy2 were used to scan bacteriophage genome of Agrican357virus for statistically significant motifs. Motifs found by MEME (Bailey et al., 2009) with e-value less than 1.00E-02 were selected by FIMO (Grant et al., 2011) to be searched for their coordinates and iterations in their respective genomes. User defined cut-off values (P-value <1.00E-03, Q-value <0.05), as described in Berg et al. (2016) were used to maximize the results. The location of the motifs within bacteriophage genomes was determined from the annotated GenBank files (Esplin et al., 2017).\\\\nResults and Discussion\\\\nIsolation and Characterization of Eight Closely Related Large Bacteriophages Infecting E. amylovora\\\\nEight novel bacteriophages (Deimos-Minion, Special G, RAY, Simmy50, Bosolaphorus, Desertfox, Mortimer, and MadMel) that infect E. amylovora were plaque isolated and their genomes were subsequently sequenced and annotated as previously described (Esplin et al., 2017; Sharma et al., 2018). All eight bacteriophages have relatively large genomes with genome sizes of 271 to 275 kb (Table 1), which are comparable to the related bacteriophage Ea35-70 (271084 bp). These bacteriophages have correspondingly large putative proteomes, with 317 to 324 predicted ORFs. A search for tRNA’s using tRNA ScanSE (Lowe and Chan, 2016) suggests that RAY, Simmy50, Bosolaphorus, and Mortimer have 1 tRNA each coding for Asparagine, whereas no tRNA’s were detected for Deimos-Minion, Special G, MadMel, and Desertfox. No lysogeny related genes were identified (including integrase, excisionase or repressors). Their clear plaque morphology and ease in obtaining higher titers (∼108–1010 pfu/ml) suggest they may be lytic bacteriophages, however, rigorous testing for bacterial lysogeny has not been performed.\\\\nTable 1. General characteristics of related bacteriophages Deimos-Minion (DM), RAY, Special G, Desertfox, MadMel, Mortimer, Bosolaphorus, Simmy50, and Ea35-70 that infect E. amylovora ATCC 29780.\\\\nElectron Microscopy Reveals Myovirus Structure for Five E. amylovora Bacteriophages\\\\nDeimos-Minion, Special G, RAY, Simmy50, Mortimer, MadMel, Desertfox, and Bosolaphorus were all found to be similarly sized Myoviridae, having contractile tails (average size 159 + −11.4 nm), a tail sheath (average size 78.5 + −9.28 nm), visible tail fibers, and large capsids (average size 128 + −5.96 nm) (Figure 1). This morphology is supported by their protein-based relationships to other jumbo Myoviridae discussed below. Due to apparent similarity within these bacteriophages, only RAY’s morphological calculations are listed but all eight of these bacteriophages were imaged extensively.\\\\nFigure 1. Electron microscopy STEM images of Deimos-Minion, Special G, RAY, Simmy50, Bosolaphorus, Desertfox, MadMel, and Mortimer revealed Myoviruses with long contractile tails.\\\\nHost Range and Burst Size\\\\nBacteriophages of the Agrican357virus family were tested for activity against seventeen different bacterial strains. Out of these, fifteen were from the Enterobacteriales- P. agglomerans E325 (Pusey, 1997), P. vagans C-91 (Ishimaru et al., 1987; Smits et al., 2010), E. coli K-12 BW 25113 (Datsenko and Wanner, 2000), S. enterica (generous donation by roth lab), Y. pestis KIM6 (Fetherston et al., 1992; Gong et al., 2001), E. cloacae ATCC 13047 (Ren et al., 2010), K. pneumoniae ATCC 10031 (Otman et al., 2007), B. subtilis ATCC 6033 (Nakamura et al., 1999), C. sakazakii ATCC 29544 (Iversen et al., 2007; Gicova et al., 2014), E. amylovora Ea110 (Ritchie and Klos, 1976), E. amylovora GH9 (McGhee and Sundin, 2012), E. amylovora EaBH (McGhee and Sundin, 2012), E. amylovora RB02 (McGhee and Sundin, 2012), E. amylovora Ea273 (Sebaihia et al., 2010), E. amylovora ATCC 29780 (control) (Ritchie and Klos, 1976), and two from Pseudomonadaceae- P. aeruginosa PA100 (Cartagena et al., 2007) and P. chlororaphis ATCC 13985 (Conway et al., 1956). Enterobacteriales strains were chosen due to being members of the same bacterial order as Erwinia, whereas Pseudomonadaceae strains were the hosts of bacteriophages related to the Agrican357virus bacteriophages based on protein BLAST. Our current analyses displayed that Agrican357virus bacteriophages infect all Erwinia strains (with the exception of Special G and Mortimer that failed to infect GH9 and EaBH, respectively) as well as closely related genera also commonly found on plants– P. agglomerans (Deletoile et al., 2009) and P. vagans (Palmer et al., 2016; Table 2).\\\\nTable 2. Host range analysis of eight Agrican357virus bacteriophages.\\\\nOwing to the large nature of Agrican357virus bacteriophages, we investigated the burst size of bacteriophage Deimos-Minion on E. amylovora strain ATCC 29780. Burst size studies suggested that when infected at MOI of 100 Deimos-Minion has burst size of 4.6–4.9 with latent period of 3–4 h before the first burst (Figure 2) under the laboratory growth conditions used herein, consistent with their large size. As seen in Figure 2, a second burst is appearing at the end of this 6 h period. The observed burst size (∼5) was confirmed with bacteriophage RAY (data not shown) and is consistent with other large Myoviridae in that P. aeruginosa bacteriophage KTN4 has a reported burst of 6–8 and may be due to the need to build internal cellular structures for the Jumbo viruses to be built (Danis-Wlodarczyk et al., 2016), or due to sub-optimal assay conditions for proliferation.\\\\nFigure 2. Growth Curve of Deimos-Minion with host E. amylovora ATCC 29780 by plaque assays shows first burst at ∼4 h and second burst at ∼6 h.\\\\nGenomic and Evolutionary Characteristics\\\\nTo determine the overall genomic and proteomic similarity of our eight novel bacteriophages to available bacteriophages in GenBank, related bacteriophages were identified by BLAST (qblast) using each of the putative gene products encoded by RAY. The bacteriophages with e-values below 1.00E-04 and above 33% identity that were identified in three or more BLAST searches were then compared using Gepard dot plot (Krumsiek et al., 2007), average nucleotide identity (ANI analysis) (Lassmann and Sonnhammer, 2005), and BLAST alignment (Altschul et al., 1990). Dot plots were constructed using whole genome sequences, major capsid protein amino acid, and terminase amino acid sequences (Figures 3A–C, respectively). While looking at the results of the whole genome dot plot, all eight of our bacteriophages show no similarity to any other bacteriophages used in the dot plot except for very close similarity to Ea35-70 (KF806589) (Yagubi et al., 2014), an Erwinia bacteriophage isolated in Canada in 2014 (see Figure 3A). In addition, their average nucleotide identity (ANI) using Geneious (Kearse et al., 2012) was remarkably high >94% (see Supplementary Table S1). These results indicate that these eight bacteriophages Deimos-Minion, Simmy50, RAY, Special G, Bosolaphorus, Desertfox, MadMel, and Mortimer along with Ea35-70 make a distinct family of bacteriophages, consistent with the International committee on taxonomy of viruses’ classification as new species of a new genus Agrican357virus in the family Myoviridae of order Caudovirales (Adams et al., 2017).\\\\nFigure 3. Whole-genome nucleotide (A) and protein terminase (B) or major capsid protein (C) dot plot analysis reveals a fairly isolated cluster of bacteriophages that includes Deimos-Minion, Special G, RAY, Simmy50, Bosolaphorus, Desertfox, MadMel, Mortimer and Ea35-70. Dot plots were constructed using Gepard.\\\\nThe major capsid protein (MCP) and terminase proteins are two of the most conserved proteins in bacteriophage genomes and have been used to group bacteriophages in families by single gene analysis (Smith et al., 2013). In order to identify distant bacteriophage relatives, a proteomic comparison of these bacteriophages was performed using terminases (see Figure 3B) and MCPs (see Figure 3C) by Gepard dot plot (Krumsiek et al., 2007). The same bacteriophage order from the whole genome dot plot (Figure 3A) was used in these dot plots. Whole genome and terminase dot plots both displayed limited synteny between Agrican357virus bacteriophages and Erwinia bacteriophage phiEaH1 (4.00E-155 from blastp of terminase) indicating this bacteriophage as the closest known relative from Erwiniaceae. In contrast, little similarity to Pseudomonas bacteriophages phiKZ (8.00E-156), KTN4 (8.00E-156), phiPA3 (2.00E-149), 201phi2-1 (5.00E-140), OBP (2.00E-101), EL (3.00E-77), and Ralstonia bacteriophages RSF1 (9.00E-122) and RSL2 (3.00E-120) can be seen in the terminase dot plot which was not apparent in the whole genome and major capsid protein dot plots. All of these bacteriophages are distantly related jumbo Myoviridae.\\\\nThe two subunits of terminase protein; large and small, are an essential part of DNA packaging (Kuebler and Rao, 1998; Mesyanzhinov et al., 2002). All eight of our Agrican357virus bacteriophages have a putative terminase gene with identical amino acid sequences: Deimos-Minion gp189, Special G gp185, RAY gp183, Simmy50 gp186, Desertfox gp184, Bosolaphorus gp185, MadMel gp185, and Mortimer gp188. This protein is also present in Ea35-70 gp181. This indicates that it is a highly conserved protein for this family. Considering the similarity between these bacteriophages, it can be inferred that all nine bacteriophages of Agrican357virus may have headful packaging (Figure 4). In support of this conclusion, blastp results demonstrated a match with Pseudomonas bacteriophage phiKZ with an e-value of 8.00E-156, a terminase large subunit from Erwinia bacteriophage PhiEaH2 with an e-value of 6.00E-122 and a terminase large subunit of Pseudomonas bacteriophage 201phi2-1 with an e-value of 5.00E-140. Bacteriophages phiKZ, phiEaH2 and 201phi2-1 are all known to have headful packaging (Merrill et al., 2016). In addition to blastp, bacteriophage termini and packaging mode for six bacteriophages (excluding Deimos-Minion and Special G) was also determined using randomly fragmented next-generation sequencing (NGS) data with the help of software PhageTerm (Garneau et al., 2017)3. PhageTerm analysis indicated that RAY, MadMel, Desertfox, Bosolaphorus, Simmy50, and Mortimer have headful packaging without a pac site. Thus, the headful packaging strategy is supported by terminase homology and NGS sequencing data.\\\\nFigure 4. Phylogenetic analyses of phage terminase proteins supports the relationships depicted by dot plot analysis of the Agrican357virus bacteriophages. The evolutionary history was inferred using the Maximum Likelihood method and Poisson correction model in MEGA X. Branches corresponding to partitions reproduced in less than 50% bootstrap replicates are collapsed where 100 was set to be initial bootstrapping value.\\\\nProteomic Analysis of the Agrican357virus Family\\\\nDue to great similarity between these bacteriophages we randomly chose RAY as a representative for the protein classification. Proteomic analysis of RAY reveals the novel nature of these bacteriophages in that of 318 proteins, 202 proteins were considered to be novel with no BLAST hit (the e-value cutoff was <1.00E-04), 50 were hypothetical proteins with BLAST hits, and 67 were proteins with a putative function based on their BLAST hit (Supplementary Figure S1A). Thus, over half of the proteins had no BLAST hit outside of the Agrican357virus bacteriophages. These proteins represent a considerable proteomic “dark matter” (Hatfull, 2015), and underscore the vast biological richness harbored in bacteriophages. Of the 67 proteins with predicted function, a majority appear to be structural proteins (∼41%), and DNA metabolism proteins (approximately 41%) (Supplementary Figure S1B).\\\\nThe computer program Phamerator (Cresawn et al., 2011) was used to compare the entire genomes of the nine Agrican357virus bacteriophages that infect E. amylovora: Deimos-Minion, Special G, RAY, Simmy50, Bosolaphorus, Desertfox, MadMel, Mortimer, and Ea35-70 (Figure 5). Despite their large size, these genomes display remarkable nucleotide sequence and proteomic conservation (>94% ANI, see Supplementary Table S1). The genomes encode recognizable structural and enzymatic bacteriophage proteins vital to the bacteriophage life cycle, including terminase proteins, major capsid proteins, and tail fiber proteins as well as proteins involved in DNA transcription and translation, such as helicase proteins, DNA polymerase, and RNA polymerase. Though the genomes of these nine bacteriophages are virtually identical, a few genes are differentially present across these bacteriophage genomes. Most of these are hypothetical proteins, however, HNH endonucleases also differed consistently between the Agrican357virus bacteriophages. HNH endonucleases are proteins that splice DNA and assist in the movement of introns and other intron-like sequences (Chevalier and Stoddard, 2001). Deimos-Minion has two such HNH endonucleases, gp93 and gp234 that do not appear to be homologs based on protein similarity. Protein BLAST results of gp93 show that the HNH endonuclease is also found in bacteriophages Bosolaphorus, Desertfox, MadMel, RAY, Simmy50, Special G and Ea35-70, and is similar to those found in some Pseudomonas bacteriophages (phiKZ and KTN4) as well as both Gram-negative and Gram-positive strains of bacteria. However, only the HNH endonuclease domain (∼amino acid 58-109 of bp93) is primarily conserved, the remaining 278 amino acid protein is not conserved in bacteria. On the other hand, homologs of HNH endonuclease gp234 are only found in Deimos-Minion and MadMel, as well as several Gram-positive and Gram-negative bacteria. Genomes of Deimos-Minion, Desertfox and MadMel also displayed a reversed order of two proteins (gp93-gp94 in Deimos-Minion, gp88-gp89 in Desertfox and gp90-gp91 in MadMel) when compared to similar proteins in other bacteriophages of this family. The proteins involved are HNH endonuclease and ribonucleotide reductase. To search for repetitive sequences in the genome which may be involved in recombination, MEME (Bailey et al., 2009) and FIMO (Grant et al., 2011) were used to locate motifs in the genomes of all eight of our Agrican357virus bacteriophages. Several common and unique motifs were discovered, however, they had poor e-values with little or no significance and were not followed further.\\\\nFigure 5. Whole genome Phamerator map of E. amylovora bacteriophages illustrates the high similarity of bacteriophages Mortimer, MadMel, Desertfox, Bosolaphorus, Deimos-Minion, Special G, RAY, Simmy50, and Ea35-70. Bacteriophages were mapped using Phamerator and arranged based on highest protein similarity. Violet shading between genomes indicates genome nucleotide homology (with standard e-value cutoff of 1.00E-04) and the ruler indicates genome base pairs, while white spaces indicate areas without significant nucleotide similarity. Boxes above and below the genome ruler indicate ORFs going in the forward and reverse direction, respectively. They are labeled with predicted function, occasionally numbered, and colored to indicate protein homologs between the bacteriophages.\\\\nDue to the large size of these bacteriophages, and their terminase similarity to bacteriophage phiKZ, these bacteriophages likely belong to the jumbo bacteriophages (Hertveldt et al., 2005; Yuan and Gao, 2017) making it no surprise that the structural proteins are found in other bacteriophages. Along with hypothetical proteins, the proteins that are conserved with other phiKZ-like jumbo bacteriophages include: RNA polymerase beta subunit, nuclease RtcB-like, SbcC like, helicase, virion structural proteins, tail fiber, tail sheath, lysozyme domain, terminase, and major capsid protein. A splitstree analysis showing the relationship of the related jumbo bacteriophages by protein conservation is displayed in Figure 6. This protein-based tree suggests seven groups of related jumbo Myoviridae bacteriophages, with the Agrican357virus group as the most distant group. It further confirms that proteins of Agrican357virus family are more similar to proteins from Pseudomonas bacteriophages EL and OBP and Ralstonia bacteriophage RSL2 than to other Enterobacteriales bacteriophages.\\\\nFigure 6. Protein-conservation analysis displayed by Splitstree of the Agrican357virus genus with related jumbo Myoviridae bacteriophages reveals Agrican357virus as a distant evolutionary group.\\\\nMass Spectrometry Validates 27 Hypothetical Proteins as Proteins of Unknown Function\\\\nFurther analysis of Deimos-Minion and RAY genomes via mass spectrometry (MS) detected several novel proteins, promoting the status of 27 proteins from hypothetical proteins to proteins of unknown function. In RAY and Deimos-Minion genomes collectively, MS analysis identified seventeen proteins with a putative function, eighteen novel hypothetical proteins specific to this bacteriophage family and nine hypothetical proteins (seven known bacteriophage proteins and two other) with blastp hits to other bacteriophages (Table 3). The majority of proteins found through MS are novel hypothetical proteins found only in this family, followed by putative bacteriophage structural proteins, hypothetical bacteriophage proteins, proteins with putative functions and other hypothetical proteins (see Table 3). This analysis agrees with our predicted conservation of proteins depicted through Phamerator analyses.\\\\nTable 3. Mass Spectrometry reveals 27 Hypothetical Proteins as Proteins of Unknown Function.\\\\nBiofilm Degradation (EPS) Assays Suggest Specificity for Pantoea\\\\nEnzymatic proteins like extracellular polysaccharide (EPS) depolymerase and phage-related lysozyme are few of the annotated proteins with putative functions which were also predicted via mass spectrometry. EPS depolymerase (Kim and Geider, 2000) is an enzyme that degrades EPS and phage-related lysozyme is shown to lyse the bacterial cell wall (Emrich and Streisinger, 1968). It has been shown that halo formation on the host could be a result of biofilm degradation activity (Cornelissen et al., 2012; Majkowska-Skrobek et al., 2016). The presence of halo zone after in infections of Agrican357virus family was first observed on P. vagans strain C9-1 (Figure 7A). To investigate further the EPS- depolymerase gene was cloned into a plasmid pJG1700, amplified using E. coli DH5α, and spotted on P. vagans stain C-91 and E. amylovora strain ATCC 29780 (Figure 7B). Lysate from a similarly grown and prepared DH5α culture was used as a control. The clearing is indicative of EPS depolymerase activity on P. vagans. This activity was not seen on E. amylovora ATCC 29780.\\\\nFigure 7. Halo formation on P. vagans by RAY (A) and bio-film degradation activity of gp76 on P. vagans (B).\\\\nStructural Prediction Supports the Putative Function of Several Proteins\\\\nTo further understand Agrican357virus and verify their protein functions, we studied proteins involved in DNA metabolism (∼45%), the largest group of functional proteins conserved in the Agrican357virus. Multiple mechanisms for DNA regulation and repair are evident with the presence of proteins that are hypothesized to aid DNA synthesis, repair, and recombination. These proteins may increase the stability and survival of these jumbo bacteriophages (Supplementary Table S2 and Supplementary Figure S2). In order to proliferate in host cells, bacteriophages need to be equipped with proteins that allow them to reproduce effectively. Although many bacteriophages harbor proteins for DNA damage repair and DNA reproduction inside a host bacteria cell, these large bacteriophages may require extremely viable progeny due to lower burst sizes (∼4.6 functional virions compared to thousands reported for other bacteriophages).\\\\nTwo proteins with a conserved domain found in the nine Agrican357virus bacteriophages are SbcC and a SbcCD nuclease (see Supplementary Figure S2A). The ability of SbcC and SbcCD to regulate and repair DNA has been shown to be essential for the stability and proliferation of some bacteriophages (Connelly et al., 1998). During DNA replication, palindromic sequences will create hairpin-like structures that can inhibit the progression of DNA polymerase (Leach et al., 1997). SbcC and SbcCD proteins work together to cleave both double- stranded and single-stranded DNA, and have been shown to recognize and specifically cleave hairpin structures. This breaks down the replication fork, allowing the genome to be repaired through recombination, so replication can proceed (Leach et al., 1997; Connelly et al., 1998). The proteins SbcC and SbcCD nucleases preserve the viability of the genome by allowing replication without excising the palindromic sequences (Leach et al., 1997). There are many types of DNA damage that may occur within a genome, making recombination and repair of DNA important, such as mutations due to UV damage. UV damage creates kinks or abnormalities within a genome, and prohibits proliferation. Exodeoxyribonuclease VIII breaks double stranded DNA, and degrades a genome on both 5′ ends (Joseph and Kolodner, 1983; Kolodner et al., 1994). This allows the kinked and abnormal portions of a genome to be straightened and repaired through homologous recombination. Additionally, exodeoxyribonuclease VIII does not require ATP to perform DNA repair, enabling repair of the genome even in low-energy environments where the bacteriophage does not have access to ATP (Kolodner et al., 1994). We hypothesize that exodeoxyribonuclease VIII enables the bacteriophages to remain stable despite mutations from UV damage. However, unique from our other predicted structure alignments, the protein from RAY does not match up well with other exodeoxyribonuclease VIII homologs (see Supplementary Figure S2B). It is possible that since these proteins do not have the same protein folding and alignment, they may not have the same function but a related, adapted function.\\\\nIn the Agrican357virus bacteriophages, there are several encoded proteins with conserved domains of the thymidylate kinase and thymidine kinase (see Supplementary Table S2). Structural prediction and alignment confirm these proteins as likely thymidine kinases (see Supplementary Figures S2C,D), a necessary step due to the distant relationship (low e-values) of Agrican357virus bacteriophage proteins when compare to other biological entities. Thymidine kinase is an enzyme that catalyzes the phosphorylation of thymidine monophosphate (Kokoris and Black, 2002). Thymidylate kinase then catalyzes the phosphorylation of thymidine diphosphate (Doharey et al., 2016), which is an essential precursor for DNA (Chaudhary et al., 2013). Therefore, these proteins are regulatory enzymes that make bacteriophage cell growth and survival possible by aiding proliferation through the synthesis of DNA (Chaudhary et al., 2013; Cui et al., 2013; Doharey et al., 2016). Other proteins shown in Supplementary Figures S2E,F are putative UvsX recombinase and a putative SF2 helicase with conserved helicase domain known as UvsW, which finishes the recombination (Madej et al., 1995; Gajewski et al., 2011). UvsX and UvsW are proteins that have been known to work together to repair broken replication forks through homologous recombination (Kadyrov and Drake, 2004; Maher and Morrical, 2013). Homologous recombination is one of the most efficient ways to have error free DNA repair, and is beneficial to bacteriophages to have this repair mechanism. These repair mechanisms would be important to the bacteriophages because it would not only help repair broken replication forks but it would also help repair damaged or broken DNA (Kadyrov and Drake, 2004; Maher and Morrical, 2013). It has been shown that the absence of UvsX increases UV sensitivity (Kadyrov and Drake, 2004).\\\\nConclusion\\\\nAgrican357virus genus of bacteriophages are Myoviridae with dsDNA, large capsids, long contractile tails and high GC content. Their genomes are nearly identical (>94% ANI). All three dot plots (whole genome, major capsid protein, and terminase protein) show no close similarity between the Agrican357virus family and any of the other bacteriophages on NCBI (see Figures 3A–C). We have also found that the Agrican357virus cluster is more closely related to bacteriophages infecting Pseudomonas and Ralstonia, than those infecting E. amylovora. The contrast that we observe between this cluster of bacteriophages and the distantly related bacteriophage analyzed by dot plot contributes valuable information about evolutionary relationships between these other clusters (see Figure 3), suggesting the distant relationship may emphasize the importance of ecological niche, since most other Enterobacteriales bacteriophages isolated infect animal pathogens rather than plant pathogens. It may also, however, simply indicate the abundance of unstudied bacteriophages. The Agrican357virus family of bacteriophages is a novel family, with very low similarity to any other viruses, providing approximately 250 novel proteins to add to the viral dark matter that have no homolog by blastp (Hatfull, 2015). To understand a bacteriophage, it is vital to understand the encoded proteome. A bacteriophage’s proteins determine how effectively it can infect bacteria, and how stable and safe it would be to use in a phage cocktail (a mixture of bacteriophages used together for phage therapy). Of the proteins with predicted function, this family encodes primarily DNA metabolism and repair proteins. Since the bacteriophage host, E. amylovora, is found primarily on the blossoms of fruit trees of the Rosaceae family, these proteins may be particularly vital due to the onset of UV radiation including putative thymidine and thymidylate kinases which aid the production of the nucleotide thymine for DNA synthesis (Doharey et al., 2016), putative SbcC and SbcCD proteins which protect against DNA damage by cleaving harmful hairpin structures during replication (Connelly et al., 1998), putative exodeoxyribonuclease VIII which makes double stranded DNA breaks to help repair DNA damage at low energy (Joseph and Kolodner, 1983), and putative UvsX recombinase and putative SF2 helicase which aid in repair and recombination of DNA (Maher and Morrical, 2013). The small burst size we report herein for these jumbo bacteriophages (∼4.6 functional virions), may require a high level of fidelity to ensure success in the environment.\\\\nA paper published in 2003 on evolutionary pathways of P. aeruginosa bacteria demonstrated that phiKZ-like bacteriophages have a very broad host range (Krylov et al., 2003). In 1995, Campbell et al. (Campbell et al., 1995) isolated bacteriophages from barley rhizosphere that infected Pseudomonas spp. other than P. aeruginosa. These bacteriophages displayed great morphological similarity to phiKZ-like bacteriophages despite low genomic similarity (Mesyanzhinov et al., 2002; Krylov et al., 2003; Sokolova et al., 2014). Similarly, Agrican357virus bacteriophages display proteomic similarity to phiKZ-like bacteriophages, particularly with their structural proteins, with little genomic synteny. These results suggest the phiKZ-like bacteriophages are highly divergent, derived from a common ancestor and successful in a wide range of ecological niches. It is highly likely that Agrican357virus family evolved through both mutational divergence and modular evolution (acquisition of larger regions of DNA, or modules), which is a common phenomenon in bacteriophages (Botstein, 1980), and yet there is extremely low variance in all isolates thus far (>94% ANI). Such high conservation in these large genomes may reflect selective forces on a majority of the genome, which is for the most part uncharacterized. The great challenge ahead is both the abundance of bacteriophages that are completely uncharacterized, and the abundance of novel proteins harbored in their genomes.\\\\nData Availability\\\\nAll datasets generated for this study are included in the manuscript and the Supplementary Files.\\\\nAuthor Contributions\\\\nRS designed and wrote the manuscript, performed genomic and proteomic analysis, executed host range, EPS depolymerase cloning, halo zone and burst size experiments, prepared samples for electron microscopy and mass spectrometry, and drew all the figures and tables. KB, BP, and OS performed the structural prediction of proteins and contributed to writing. TN and EY helped with Dotplots. DB contributed with funding and ideas and also edited the manuscript. JG is the corresponding author and principal investigator of this study and as such contributed to the design and execution of experiments, oversaw all students, edited the manuscript, and contributed with funding. All authors read and approved the submitted version.\\\\nFunding\\\\nDB was awarded a grant by the United States Dairy Association. Financial support by The University of Arizona, Brigham Young University, Department of Microbiology and Molecular Biology, and the College of Life Sciences in the form of student wage funds, support, and publication fees is gratefully acknowledged.\\\\nConflict of Interest Statement\\\\nJG is working on licensing a phage-based therapy for fire blight.\\\\nThe remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nThe authors extend a special thanks to the Science Education Alliance – Phage Hunters Advancing Genomics and Evolutionary Science (SEA-PHAGES) for support and training on bacteriophage analysis, Steven Cresawn for training on Phamerator, Majid Ghassemian and BMPMSF at UCSD for carrying out mass spectrometry, George W. Sundin for donating E. amylovora strains, Edward Wilcox from the BYU Sequencing Center, Michael Standing from the BYU Microscopy Lab and Daniel Thompson for helping with phage microscopy, David Parady and Brandon Ekins from Life Science IT at BYU for hosting DNAmaster and PhamDB on BYU server, Tsz Ching Tam for media preparation, Daniel Arens for helping with cloning and Yomesh Sharma for helping with the formatting of the manuscript. The authors also like to thank the following undergraduates from the BYU Phage Hunters’ program for their contributions to isolation of the bacteriophages: Garrett Jensen, Jared Kruger, Madison Melville, Trevon Galbraith, Savannah Grossarth, Hannah Ferguson, and Austin Simister.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2019.01533/full#supplementary-material\\\\nFootnotes\\\\n^ http://cobamide2.bio.pitt.edu\\\\n^ https://cpt.tamu.edu/galaxy-pub/\\\\n^ https://galaxy.pasteur.fr\\\\nReferences\\\\nAbbasifar, R., Griffiths, M. W., Sabour, P. M., Ackermann, H. W., Vandersteegen, K., Lavigne, R., et al. (2014). Supersize me: Cronobacter sakazakii phage Gap32. Virology 460-461, 138–146. doi: 10.1016/j.virol.2014.05.003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAbedon, S. T., Thomas-Abedon, C., Thomas, A., and Mazure, H. (2011). Bacteriophage prehistory: is or is not Hankin, 1896, a phage reference? Bacteriophage 3, 174–178. doi: 10.4161/bact.1.3.16591\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAbramoff, M. D., Magalhaes, P. J., and Ram, S. J. (2004). Image processing with imageJ. Biophoton. Int. 11, 36–42.\\\\nGoogle Scholar\\\\nAdams, M. J., Lefkowitz, E. J., King, A. M. Q., Harrach, B., Harrison, R. L., Knowles, N. J., et al. (2017). Changes to taxonomy and the international code of virus classification and nomenclature ratified by the international committee on taxonomy of viruses. Arch. Virol. 162, 2505–2538.\\\\nGoogle Scholar\\\\nAltschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990). Basic local alignment search tool. J. Mol. Biol. 215, 403–410. doi: 10.1006/jmbi.1990.9999\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nArens, D. K., Brady, T. S., Carter, J. L., Pape, J. A., Robinson, D. M., Russell, K. A., et al. (2018). Characterization of two related erwinia myoviruses that are distant relatives of the phikz-like jumbo phages. PLoS One 13:e0200202. doi: 10.1371/journal.pone.0200202\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBailey, T. L., Boden, M., Buske, F. A., Frith, M., Grant, C. E., Clementi, L., et al. (2009). Meme suite: tools for motif discovery and searching. Nucleic Acids Res. 37, W202–W208. doi: 10.1093/nar/gkp335\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBerg, J. A., Merrill, B. D., Crockett, J. T., Esplin, K. P., Evans, M. R., Heaton, K. E., et al. (2016). Characterization of five novel brevibacillus bacteriophages and genomic comparison of Brevibacillus phages. PLoS One 11:e0156838. doi: 10.1371/journal.pone.0156838\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBhunchoth, A., Blanc-Mathieu, R., Mihara, T., Nishimura, Y., Askora, A., Phironrit, N., et al. (2016). Two Asian jumbo phages, varphirsl2 and varphirsf1, infect ralstonia solanacearum and show common features of varphikz-related phages. Virology 494, 56–66. doi: 10.1016/j.virol.2016.03.028\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBollback, J. P., and Huelsenbeck, J. P. (2009). Parallel genetic evolution within and between bacteriophage species of varying degrees of divergence. Genetics 181, 225–234. doi: 10.1534/genetics.107.085225\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBoyd, E. F. (2012). Bacteriophage-encoded bacterial virulence factors and phage-pathogenicity island interactions. Adv. Virus Res. 82, 91–118. doi: 10.1016/B978-0-12-394621-8.00014-5\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nButtimer, C., Hendrix, H., Oliveira, H., Casey, A., Neve, H., McAuliffe, O., et al. (2017). Things are getting hairy: Enterobacteria bacteriophage Vb_Pcam_Cbb. Front. Microbiol. 8:44. doi: 10.3389/fmicb.2017.00044\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCampbell, J. I. A., Albrechtsen, M., and Sorensen, J. (1995). Large Pseudomonas phages isolated from barley rhizosphere. FEMS Microbiol. Ecol. 18, 63–74. doi: 10.1016/0168-6496(95)00043-a\\\\nCrossRef Full Text | Google Scholar\\\\nCartagena, E., Colom, O. A., Neske, A., Valdez, J. C., and Bardon, A. (2007). Effects of plant lactones on the production of biofilm of Pseudomonas Aeruginosa. Chem. Pharm. Bull. 55, 22–25. doi: 10.1248/cpb.55.22\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChaudhary, S. K., Jeyakanthan, J., and Sekar, K. (2013). Cloning, expression, purification, crystallization and preliminary X-Ray crystallographic study of thymidylate kinase (Ttha1607) from thermus thermophilus Hb8. Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun. 69(Pt 2), 118–121. doi: 10.1107/S1744309112050208\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChevalier, B. S., and Stoddard, B. L. (2001). Homing endonucleases: structural and functional insight into the catalysts of intron/intein mobility. Nucleic Acids Res. 29, 3757–3774. doi: 10.1093/nar/29.18.3757\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nConnelly, J. C., Kirkham, L. A., and Leach, D. R. (1998). The sbccd nuclease of Escherichia Coli is a structural maintenance of chromosomes (Smc) family protein that cleaves hairpin DNA. Proc. Natl. Acad. Sci. U.S.A. 95, 7969–7974. doi: 10.1073/pnas.95.14.7969\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nConway, H. F., Haynes, W. C., Jackson, R. W., Locke, J. M., Pridham, T. G., Sohns, V. E., et al. (1956). Pseudomonas aureofaciens kluyver and phenazine alpha-carboxylic acid, its characteristic pigment. J. Bacteriol. 72, 412–417.\\\\nGoogle Scholar\\\\nCornelissen, A., Ceyssens, P. J., Krylov, V. N., Noben, J. P., Volckaert, G., and Lavigne, R. (2012). Identification of Eps-degrading activity within the tail spikes of the novel Pseudomonas putida phage Af. Virology 434, 251–256. doi: 10.1016/j.virol.2012.09.030\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCresawn, S. G., Bogel, M., Day, N., Jacobs-Sera, D., Hendrix, R. W., and Hatfull, G. F. (2011). Phamerator: a bioinformatic tool for comparative bacteriophage genomics. BMC Bioinformatics 12:395. doi: 10.1186/1471-2105-12-395\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCui, Q., Shin, W. S., Luo, Y., Tian, J., Cui, H., and Yin, D. (2013). Thymidylate kinase: an old topic brings new perspectives. Curr. Med. Chem. 20, 1286–1305. doi: 10.2174/0929867311320100006\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDanis-Wlodarczyk, K., Vandenheuvel, D., Jang, H. B., Briers, Y., Olszak, T., Arabski, M., et al. (2016). A proposed integrated approach for the preclinical evaluation of phage therapy in Pseudomonas infections. Sci. Rep. 6:28115. doi: 10.1038/srep28115\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDatsenko, K. A., and Wanner, B. L. (2000). “One-step inactivation of chromosomal genes in Escherichia Coli K-12 using Pcr products. Proc. Natl. Acad. Sci. U.S.A. 97, 6640–6645. doi: 10.1073/pnas.120163297\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDelbruck, M. (1945). The burst size distribution in the growth of bacterial viruses (Bacteriophages). J. Bacteriol. 50, 131–135.\\\\nGoogle Scholar\\\\nDeletoile, A., Decre, D., Courant, S., Passet, V., Audo, J., Grimont, P., et al. (2009). Phylogeny and Identification of Pantoea species and typing of pantoea agglomerans strains by multilocus gene sequencing. J. Clin. Microbiol. 47, 300–310. doi: 10.1128/JCM.01916-08\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nD’Herelle, F. (1917). Sur un microbe invisible antagoniste des bacilles dysentériques. CR Acad. 165, 373–375.\\\\nGoogle Scholar\\\\nD’Herelle, F. (2007). On an invisible microbe antagonistic toward dysenteric bacilli: brief note by Mr. F. D’herelle, presented by Mr. Roux. 1917. Res. Microbiol. 158, 553–554. doi: 10.1016/j.resmic.2007.07.005\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDoharey, P. K., Singh, S. K., Verma, P., Verma, A., Rathaur, S., and Saxena, J. K. (2016). Insights into the structure-function relationship of Brugia Malayi Thymidylate Kinase (Bmtmk). Int. J. Biol. Macromol. 88, 565–571. doi: 10.1016/j.ijbiomac.2016.04.004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nEffantin, G., Hamasaki, R., Kawasaki, T., Bacia, M., Moriscot, C., Weissenhorn, W., et al. (2013). Cryo-electron microscopy three-dimensional structure of the jumbo phage phirsl1 infecting the phytopathogen Ralstonia solanacearum. Structure 21, 298–305. doi: 10.1016/j.str.2012.12.017\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nEmrich, J., and Streisinger, G. (1968). The role of phage lysozyme in the life cycle of phage T4. Virology 36, 387–391. doi: 10.1016/0042-6822(68)90163-3\\\\nCrossRef Full Text | Google Scholar\\\\nEsplin, I. N. D., Berg, J. A., Sharma, R., Allen, R. C., Arens, D. K., Ashcroft, C. R., et al. (2017). Genome sequences of 19 novel erwinia amylovora bacteriophages. Genome Announc. 5:e00931-17. doi: 10.1128/genomeA.00931-17\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nFetherston, J. D., Schuetze, P., and Perry, R. D. (1992). Loss of the pigmentation phenotype in Yersinia pestis is due to the spontaneous deletion of 102 Kb of chromosomal DNA which is flanked by a repetitive element. Mol. Microbiol. 6, 2693–2704. doi: 10.1111/j.1365-2958.1992.tb01446.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGajewski, S., Webb, M. R., Galkin, V., Egelman, E. H., Kreuzer, K. N., and White, S. W. (2011). “Crystal structure of the phage T4 recombinase Uvsx and its functional interaction with the T4 Sf2 helicase Uvsw. J. Mol. Biol. 405, 65–76. doi: 10.1016/j.jmb.2010.10.004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGarneau, J. R., Florence, D., Louis-Charles, F., David, B., and Marc, M. (2017). Phageterm: a fast and user-friendly software to determine bacteriophage termini and packaging mode using randomly fragmented Ngs data. bioRxiv\\\\nGoogle Scholar\\\\nGibb, E. A., and Edgell, D. R. (2007). Multiple controls regulate the expression of mobe, an hnh homing endonuclease gene embedded within a ribonucleotide reductase gene of phage Aeh1. J. Bacteriol. 189, 4648–4661. doi: 10.1128/jb.00321-07\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGicova, A., Orieskova, M., Oslanecova, L., Drahovska, H., and Kaclikova, E. (2014). Identification and characterization of cronobacter strains isolated from powdered infant foods. Lett. Appl. Microbiol. 58, 242–247. doi: 10.1111/lam.12179\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGill, J. J., Berry, J. D., Russell, W. K., Lessor, L., Escobar-Garcia, D. A., Hernandez, D., et al. (2012). The Caulobacter crescentus phage phicbk: genomics of a canonical phage. BMC Genomics 13:542. doi: 10.1186/1471-2164-13-542\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGong, S., Bearden, S. W., Geoffroy, V. A., Fetherston, J. D., and Perry, R. D. (2001). Characterization of the Yersinia pestis Yfu Abc inorganic iron transport system. Infect. Immun. 69, 2829–2837. doi: 10.1128/iai.67.5.2829-2837.2001\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGuttman, M., Betts, G. N., Barnes, H., Ghassemian, M., van der Geer, P., and Komives, E. A. (2009). Interactions of the Npxy microdomains of the low density lipoprotein receptor-related protein 1. Proteomics 9, 5016–5028. doi: 10.1002/pmic.200900457\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHertveldt, K., Lavigne, R., Pleteneva, E., Sernova, N., Kurochkina, L., Korchevskii, R., et al. (2005). “Genome comparison of Pseudomonas Aeruginosa large phages. J. Mol. Biol. 354, 536–545. doi: 10.1016/j.jmb.2005.08.075\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHockett, K. L., and Baltrus, D. A. (2017). Use of the soft-agar overlay technique to screen for bacterially produced inhibitory compounds. J. Vis. Exp. 119:55064. doi: 10.3791/55064\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nIshimaru, C. A., Klos, E. J., and Brubaker, R. R. (1987). Multiple antibiotic production by Erwinia herbicola. Phytopathology 78, 746–750.\\\\nGoogle Scholar\\\\nIversen, C., Lehner, A., Mullane, N., Bidlas, E., Cleenwerck, I., Marugg, J., et al. (2007). The taxonomy of Enterobacter sakazakii: proposal of a new genus cronobacter gen. nov. and descriptions of Cronobacter sakazakii comb. nov. Cronobacter sakazakii subsp. sakazakii, comb. nov., Cronobacter sakazakii subsp. Malonaticus subsp. nov., Cronobacter turicensis sp. nov., Cronobacter muytjensii sp. nov., Cronobacter dublinensis sp. nov. and cronobacter genomospecies 1. BMC Evol. Biol. 7:64. doi: 10.1186/1471-2148-7-64\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJoseph, J. W., and Kolodner, R. (1983). Exonuclease Viii of Escherichia Coli. I. purification and physical properties. J. Biol. Chem. 258, 10411–10417.\\\\nPubMed Abstract | Google Scholar\\\\nKadyrov, F. A., and Drake, J. W. (2004). Uvsx recombinase and dda helicase rescue stalled bacteriophage T4 DNA replication forks in vitro. J. Biol. Chem. 279, 35735–35740. doi: 10.1074/jbc.m403942200\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKatoh, K., Misawa, K., Kuma, K., and Miyata, T. (2002). Mafft: a novel method for rapid multiple sequence alignment based on fast fourier transform. Nucleic Acids Res. 30, 3059–3066. doi: 10.1093/nar/gkf436\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKearse, M., Moir, R., Wilson, A., Stones-Havas, S., Cheung, M., Sturrock, S., et al. (2012). Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics 28, 1647–1649. doi: 10.1093/bioinformatics/bts199\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKim, J. H., Son, J. S., Choi, Y. J., Choresca, C. H. Jr., Shin, S. P., Han, J. E., et al. (2012). Complete genome sequence and characterization of a broad-host range T4-Like bacteriophage phias5 infecting Aeromonas salmonicida Subsp. Salmonicida. Vet. Microbiol. 157, 164–171. doi: 10.1016/j.vetmic.2011.12.016\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKim, W. S., and Geider, K. (2000). Characterization of a viral eps-depolymerase, a potential tool for control of fire blight. Phytopathology 90, 1263–1268. doi: 10.1094/PHYTO.2000.90.11.1263\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKokoris, M. S., and Black, M. E. (2002). Characterization of herpes simplex virus type 1 thymidine kinase mutants engineered for improved ganciclovir or acyclovir activity. Protein Sci. 11, 2267–2272. doi: 10.1110/ps.2460102\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKolodner, R., Hall, S. D., and Luisi-DeLuca, C. (1994). Homologous pairing proteins encoded by the Escherichia Coli Rece and Rect Genes. Mol. Microbiol. 11, 23–30. doi: 10.1111/j.1365-2958.1994.tb00286.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKrumsiek, J., Arnold, R., and Rattei, T. (2007). Gepard: a rapid and sensitive tool for creating dotplots on genome scale. Bioinformatics 23, 1026–1028. doi: 10.1093/bioinformatics/btm039\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKrylov, V., Pleteneva, E., Bourkaltseva, M., Shaburova, O., Volckaert, G., Sykilinda, N., et al. (2003). Myoviridae bacteriophages of Pseudomonas Aeruginosa: a long and complex evolutionary pathway. Res. Microbiol. 154, 269–275. doi: 10.1016/s0923-2508(03)00070-6\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKuhn, J. H., Radoshitzky, S. R., Bavari, S., and Jahrling, P. B. (2013). The international code of virus classification and nomenclature (Icvcn): proposal for text changes for improved differentiation of viral taxa and viruses. Arch. Virol. 158, 1621–1629. doi: 10.1007/s00705-012-1582-6\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKumar, S., Stecher, G., Li, M., Knyaz, C., and Tamura, K. (2018). Mega X: molecular evolutionary genetics analysis across computing platforms. Mol. Biol. Evol. 35, 1547–1549. doi: 10.1093/molbev/msy096\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKurochkina, L. P., Semenyuk, P. I., Sykilinda, N. N., and Miroshnikov, K. A. (2018). The unique two-component tail sheath of giant Pseudomonas phage pabg. Virology 515, 46–51. doi: 10.1016/j.virol.2017.12.010\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLamine, J. G., DeJong, R. J., and Nelesen, S. M. (2016). Phamdb: a web-based application for building phamerator databases. Bioinformatics 32, 2026–2028. doi: 10.1093/bioinformatics/btw106\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLassmann, T., and Sonnhammer, E. L. (2005). Kalign–an accurate and fast multiple sequence alignment algorithm. BMC Bioinform. 6:298.\\\\nPubMed Abstract | Google Scholar\\\\nLavysh, D., Sokolova, M., Minakhin, L., Yakunina, M., Artamonova, T., Kozyavkin, S., et al. (2016). The genome of Ar9, a giant transducing bacillus phage encoding two multisubunit RNA polymerases. Virology 495, 185–196. doi: 10.1016/j.virol.2016.04.030\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLavysh, D., Sokolova, M., Slashcheva, M., Forstner, K. U., and Severinov, K. (2017). Transcription profiling of Bacillus subtilis cells infected with Ar9, a giant phage encoding two multisubunit RNA polymerases. MBio 8:e02041-16. doi: 10.1128/mBio.02041-16\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLeach, D. R., Okely, E. A., and Pinder, D. J. (1997). Repair by recombination of DNA containing a palindromic sequence. Mol. Microbiol. 26, 597–606. doi: 10.1046/j.1365-2958.1997.6071957.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLecoutere, E., Ceyssens, P. J., Miroshnikov, K. A., Mesyanzhinov, V. V., Krylov, V. N., Noben, J. P., et al. (2009). Identification and comparative analysis of the structural proteomes of phikz and El, two giant Pseudomonas Aeruginosa bacteriophages. Proteomics 9, 3215–3219. doi: 10.1002/pmic.200800727\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLee, J. H., Bai, J., Shin, H., Kim, Y., Park, B., Heu, S., et al. (2016). “A novel bacteriophage targeting Cronobacter sakazakii is a potential biocontrol agent in foods. Appl. Environ. Microbiol. 82, 192–201. doi: 10.1128/AEM.01827-15\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMajkowska-Skrobek, G., Latka, A., Berisio, R., Maciejewska, B., Squeglia, F., Romano, M., et al. (2016). Capsule-targeting depolymerase, derived from klebsiella Kp36 phage, as a tool for the development of anti-virulent strategy. Viruses 8:E324.\\\\nPubMed Abstract | Google Scholar\\\\nMcCormack, A. L., Schieltz, D. M., Goode, B., Yang, S., Barnes, G., Drubin, D., et al. (1997). Direct analysis and identification of proteins in mixtures by Lc/Ms/Ms and database searching at the low-femtomole level. Anal. Chem. 69, 767–776. doi: 10.1021/ac960799q\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMcGhee, G. C., and Sundin, G. W. (2012). Erwinia amylovora crispr elements provide new tools for evaluating strain diversity and for microbial source tracking. PLoS One 7:e41706. doi: 10.1371/journal.pone.0041706\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMerrill, B. D., Ward, A. T., Grose, J. H., and Hope, S. (2016). Software-based analysis of bacteriophage genomes, physical ends, and packaging strategies. BMC Genomics 17:679. doi: 10.1186/s12864-016-3018-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMesyanzhinov, V. V., Robben, J., Grymonprez, B., Kostyuchenko, V. A., Bourkaltseva, M. V., Sykilinda, N. N., et al. (2002). The genome of bacteriophage phikz of Pseudomonas Aeruginosa. J. Mol. Biol. 317, 1–19.\\\\nPubMed Abstract | Google Scholar\\\\nMiller, E. S., Heidelberg, J. F., Eisen, J. A., Nelson, W. C., Durkin, A. S., Ciecko, A., et al. (2003). Complete genome sequence of the broad-host-range vibriophage Kvp40: comparative genomics of a T4-related bacteriophage. J. Bacteriol. 185, 5220–5233. doi: 10.1128/jb.185.17.5220-5233.2003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMonson, R., Foulds, I., Foweraker, J., Welch, M., and Salmond, G. P. (2011). The Pseudomonas Aeruginosa generalized transducing phage phipa3 is a new member of the phikz-like group of ‘jumbo’ phages, and infects model laboratory strains and clinical isolates from cystic fibrosis patients. Microbiology 157(Pt 3), 859–867. doi: 10.1099/mic.0.044701-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nNakamura, L. K., Roberts, M. S., and Cohan, F. M. (1999). Relationship of Bacillus subtilis clades associated with strains 168 and W23: a proposal for Bacillus Subtilis Subsp. Subtilis subsp. nov. and Bacillus Subtilis Subsp. Spizizenii Subsp. Nov. Int. J. Syst. Bacteriol. 49(Pt 3), 1211–1215. doi: 10.1099/00207713-49-3-1211\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nOtman, J., Perugini, M. E., Tognim, M. C. B., and Vidotto, M. C. (2007). Atypical phenotypic characteristics of Klebsiella Pneumoniae isolates from an outbreak in a neonatal intensive care unit in Brazil. Brazil. J. Microbiol. 38, 273–277. doi: 10.1590/s1517-83822007000200016\\\\nCrossRef Full Text | Google Scholar\\\\nPalmer, M., de Maayer, P., Poulsen, M., Steenkamp, E. T., van Zyl, E., Coutinho, T. A., et al. (2016). Draft genome sequences of pantoea agglomerans and pantoea vagans isolates associated with termites. Stand Genomic Sci. 11:23. doi: 10.1186/s40793-016-0144-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRen, Y., Ren, Y., Zhou, Z., Guo, X., Li, Y., Feng, L., et al. (2010). Complete genome sequence of Enterobacter Cloacae subsp. cloacae type strain Atcc 13047. J. Bacteriol. 192, 2463–2464. doi: 10.1128/JB.00067-10\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRitchie, D. F., and Klos, E. J. (1976). Isolation of Erwinia amylovora bacteriophage from aerial parts of apple trees. Phytopathology 67, 101–104.\\\\nGoogle Scholar\\\\nSebaihia, M., Bocsanczy, A. M., Biehl, B. S., Quail, M. A., Perna, N. T., Glasner, J. D., et al. (2010). Complete genome sequence of the plant pathogen Erwinia amylovora Strain Atcc 49946. J. Bacteriol. 192, 2020–2021. doi: 10.1128/JB.00022-10\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nShaburova, O. V., Hertveldt, K., de la Crus, D. M., Krylov, S. V., Pleteneva, E. A., Burkaltseva, M. V., et al. (2006). Comparison of new giant bacteriophages Obp and Lu11 of soil pseudomonads with bacteriophages of phikz-supergroup of Pseudomonas Aeruginosa. Genetika 42, 1065–1074.\\\\nPubMed Abstract | Google Scholar\\\\nSharma, R., Berg, J. A., Beatty, N. J., Choi, M. C., Cowger, A. E., and Cozzens, B. J. R. (2018). Genome sequences of nine Erwinia amylovora bacteriophages. Microbiol. Resour. Announc. 7:e00944-18. doi: 10.1128/MRA.00944-18\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nShivaswamy, V. C., Kalasuramath, S. B., Sadanand, C. K., Basavaraju, A. K., Ginnavaram, V., Bille, S., et al. (2015). Ability of bacteriophage in resolving wound infection caused by multidrug-resistant Acinetobacter Baumannii in uncontrolled diabetic rats. Microb. Drug Resist. 21, 171–177. doi: 10.1089/mdr.2014.0120\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSimoliunas, E., Kaliniene, L., Truncaite, L., Zajanckauskaite, A., Staniulis, J., Kaupinis, A., et al. (2013). Klebsiella phage Vb_Klem-Rak2 - a giant singleton virus of the family myoviridae. PLoS One 8:e60717. doi: 10.1371/journal.pone.0060717\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSmith, K. C., Castro-Nallar, E., Fisher, J. N., Breakwell, D. P., Grose, J. H., and Burnett, S. H. (2013). Phage cluster relationships identified through single gene analysis. BMC Genomics 14:410. doi: 10.1186/1471-2164-14-410\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSmits, T. H., Rezzonico, F., Kamber, T., Goesmann, A., Ishimaru, C. A., Stockwell, V. O., et al. (2010). Genome sequence of the biocontrol agent pantoea vagans strain C9-1. J. Bacteriol. 192, 6486–6487. doi: 10.1128/JB.01122-10\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSokolova, O. S., Shaburova, O. V., Pechnikova, E. V., Shaytan, A. K., Krylov, S. V., Kiselev, N. A., et al. (2014). Genome packaging in El and Lin68, two giant phikz-like bacteriophages of P. Aeruginosa. Virology 468-470, 472–478. doi: 10.1016/j.virol.2014.09.002\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSulakvelidze, A. (2005). Phage therapy: an attractive option for dealing with antibiotic-resistant bacterial infections. Drug Discov. Today 10, 807–809. doi: 10.1016/s1359-6446(05)03441-0\\\\nCrossRef Full Text | Google Scholar\\\\nSullivan, M. B., Coleman, M. L., Weigele, P., Rohwer, F., and Chisholm, S. W. (2005). Three Prochlorococcus cyanophage genomes: signature features and ecological interpretations. PLoS Biol. 3:e144. doi: 10.1371/journal.pbio.0030144\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nThomas, J. A., Hardies, S. C., Rolando, M., Hayes, S. J., Lieman, K., Carroll, C. A., et al. (2007). Complete genomic sequence and mass spectrometric analysis of highly diverse, atypical bacillus thuringiensis phage 0305phi8-36. Virology 368, 405–421. doi: 10.1016/j.virol.2007.06.043\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nThomas, J. A., Rolando, M. R., Carroll, C. A., Shen, P. S., Belnap, D. M., Weintraub, S. T., et al. (2008). Characterization of Pseudomonas Chlororaphis myovirus 201varphi2-1 via genomic sequencing, mass spectrometry, and electron microscopy. Virology 376, 330–338. doi: 10.1016/j.virol.2008.04.004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nThomson, S. V. (2000). Epidemiology of Fire Blight. Fire Blight; the Disease and Its Causative Agent, Erwinia Amylovora, ed. J. Vanneste (Wallingford: CABI Publishing).\\\\nGoogle Scholar\\\\nTwort, F. W. (1915). An investigation on the nature of ultra-microscopic viruses. Lancet 186, 1241–1243. doi: 10.1016/s0140-6736(01)20383-3\\\\nCrossRef Full Text | Google Scholar\\\\nYagubi, A. I., Castle, A. J., Kropinski, A. M., Banks, T. W., and Svircev, A. M. (2014). Complete genome sequence of Erwinia amylovora bacteriophage Vb_Eamm_Ea35-70. Genome Announc.2:e00413-14. doi: 10.1128/genomeA.00413-14\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYamada, T., Satoh, S., Ishikawa, H., Fujiwara, A., Kawasaki, T., Fujie, M., et al. (2010). A jumbo phage infecting the phytopathogen Ralstonia solanacearum defines a new lineage of the Myoviridae family. Virology 398, 135–147. doi: 10.1016/j.virol.2009.11.043\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYuan, Y., and Gao, M. (2016). Proteomic analysis of a novel bacillus jumbo phage revealing glycoside hydrolase as structural component. Front. Microbiol. 7:745. doi: 10.3389/fmicb.2016.00745\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nZuckerkandl, E., and Pauling, L. (1965). “Evolutionary Divergence and Convergence in Proteins,” in Evolving Genes and Proteins, eds V. Bryson and H. J. Vogel (New York, NY: Academic Press).\\\\nGoogle Scholar\\\\nZwet, T. V. D., and Keil, H. L. (1979). Fireblight: A Bacterial Disease of Rosaceous Plants. Washington, DC: USDA.\\\\nGoogle Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00202\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00202/full\",\\n    \"loadedTime\": \"2024-01-23T08:49:35.867Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00202\",\\n    \"title\": \"Frontiers | Identification and Characterization of T5-Like Bacteriophages Representing Two Novel Subgroups from Food Products\",\\n    \"description\": \"During recent years, interest in the use of bacteriophages as biocontrol agents against foodborne pathogens has increased, particularly for members of the family Enterobacteriaceae, with pathogenic Escherichia coli, Shigella and Salmonella strains among them. Here, we report the isolation and characterisation of 12 novel T5-like bacteriophages from confiscated food samples. All bacterophages effectively lysed E. coli K-12 strains with an average burst size of 40 PFU/cell and were able to infect pathogenic E. coli strains representing enterohaemorrhagic (EHEC), enteropathogenic (EPEC), enterotoxigenic (ETEC) and enteroinvasive (EIEC) pathotypes, Shigella dysenteriae, S. sonnei strains, as well as multidrug-resistant (MDR) E. coli and multiple strains representing different Salmonella enterica serovars. All the bacteriophages exhibited Siphoviridae morphology. Whole genome sequencing of the novel T5-like bacteriophages showed that they represent two distinct groups, with the genome-based grouping correlating to the different host spectra. As these bacteriophages are of food origin, their stability and lack of any virulence genes, as well as their broad and mutually complementary host spectrum makes these new T5-like bacteriophages valuable candidates for use as biocontrol agents against food-borne pathogenic enterobacteria.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nSeveral members of the Enterobacteriaceae family are considered significant foodborne pathogens, with enterohemorrhagic Escherichia coli (EHEC) and Shigella strains capable of causing life-threatening disease even at a very small infectious dose (reviewed in Croxen et al., 2013; Anderson et al., 2016). Many of these infections are treated with antibiotic therapy, but this is threatened by the increasing occurrence of multidrug-resistant (MDR) strains (reviewed in Szmolka and Nagy, 2013). The development of novel antibiotics has significantly slowed down in the past decades, as it is becoming increasingly hard to find agents with new effect mechanisms or targets, and also because of limited economic returns (Hunter, 2012). More recently, attention has turned on the development of “evolution-proof” antibiotics (Bell and MacLean, 2017) and as an alternative answer to this challenge, there is also a renewed interest in the study of bacteriophages capable of lysing pathogenic bacteria.\\\\nAs biocontrol agents, phages can be applied either as a treatment of ongoing infections (phage therapy) as in the case of Staphylococcus aureus strains, or for foodborne pathogens, the treatment of alimentary food as a preventive measure (reviewed by Johnson et al., 2008 and Hagens and Loessner, 2010). There are already several examples of commercially available phage cocktails to be utilized against various foodborne pathogens. There is a phage cocktail aimed against E. coli O157 to be applied to food surfaces (Carter et al., 2012), as well as cocktails against Shigella (Soffer et al., 2017), Listeria monocytogenes and Salmonella enterica (reviewed by Abedon, 2017). There have also been clinical trials of phage cocktails against diarrheagenic E. coli on human subjects (Sarker et al., 2012, 2016), but these did not yield results regarding the efficacious use of phage cocktails for such infections.\\\\nBacteriophage T5 was one of the first enterobacterial phages to be characterized in-depth in the 1950s, and since then several similar bacteriophages have been reported. Their most unique feature is a two-step DNA transfer during infection (reviewed in Davison, 2015). Later, partly because of the renewed interest of utilizing bacteriophages as antibacterial agents, the bacteriophage T5 genome was fully sequenced (Wang et al., 2005) and new members of this bacteriophage group have also been characterized (Kim and Ryu, 2011; Golomidova et al., 2015). All T5 bacteriophages seem to share a common genomic structure where pre-early, early and late regions can be distinguished depending on the time of transcription during the infectious cycle (reviewed by Davison, 2015). In recent studies, several T5-like bacteriophages were proposed as potential candidates for use against various foodborne pathogens, including Salmonella enterica serovar Typhimurium (Kim and Ryu, 2011; Piya et al., 2015) as well as against E. coli O157:H7 strains (Niu et al., 2012; Hong et al., 2014), with promising in vivo results produced in live sheep with the phage CEV2 (Raya et al., 2011).\\\\nIn a previous study, we assessed the risk posed by foodborne pathogens present in illegally imported foodstuff in Europe, with a special emphasis on STEC (Nagy et al., 2015). We hypothesized that bacteriophages capable of lysing foodborne pathogenic bacteria should be also capable of surviving and replicating in the same food. Therefore, a potent source for bacteriophage candidates to be applied as biocontrol against foodborne bacteria could be the very same food samples. As pointed out by Abedon (2017), studies aiming toward phage therapy tend to focus on successes on therapeutic application, and neglect in-depth characterisation of phages. Studies on the potential therapeutic application of phages against enterobacterial infections (Sarker et al., 2016) suggest that a preventive approach, i.e., prior eradication of pathogens from food with phages before consumption could be a more succesful application.\\\\nIn light of these considerations we aimed to isolate bacteriophages from the food samples that are capable of lysing foodborne enteric pathogens to extensively characterize their host spectrum with other phenotypic features, and to characterize their genomes. This in-depth characterisation would serve to determine whether the isolated phages can be considered to serve as biocontrol agents against fooborne pathogenic Enterobacteria.\\\\nUsing E. coli K-12 strains for propagation, we isolated 12 new T5-like lytic bacteriophages from individual food samples, and proceeded with their detailed characterisation. Whole genome sequence analysis revealed that these bacteriophages do not harbor undesirable virulence-related genes. They represent two distinct new genotypes among T5-like bacteriophages and proved to be capable of lysing a wide array of pathogenic E. coli, Shigella, and Salmonella strains.\\\\nMaterials and Methods\\\\nIsolation of Bacteriophages\\\\nBacteriophages were isolated from food samples confiscated on the Hungarian border by the customs agents. Samples underwent the first steps of the ISO 16654:2001 method for isolating E. coli O157. Briefly, 5 g of the food samples were homogenized in 10 volumes of tryptic soy broth (TSB) supplemented with bile salts, and incubated for 24 h at 42°C. Bacterium-free samples of these precultures were spread or spotted onto layered 0.7% soft agar plates containing the E. coli K-12 derivative C600 as propagating strain, as described by Strauch et al. (2001). After overnight incubation at 37°C, single plaques were picked and purified by re-propagation on E. coli C600 as well as at E. coli MG1655 at least three times, until high titer (≥1011 PFU/ml) bacteriophage stocks were produced. The list of the isolated bacteriophages and their origin is given in Table 1.\\\\nTable 1. Origin and subgrouping of new T5-like bacteriophages.\\\\nTesting Host Specificity and Efficiency of Plating\\\\nThe host specificity of the isolated bacteriophages was investigated by spot assays on the strains listed in Table 2 as described before. In addition to reference strains and strains from our strain collection, extended-spectrum β-lactamase (ESBL)-producing multidrug-resistant (MDR) E. coli strains isolated from human clinical samples were investigated. MDR strains were isolated at the University of Debrecen and Salmonella strains were kindly provided by László Makrai (University of Veterinary Medicine, Budapest). Shigella flexneri M90T was kindly contributed by Zoltán Tigyi (University of Pécs). Enteroinvasive E. coli (EIEC) and Shigella strains (except for S. sonnei 866-F) originated from the Hungarian National Collection of Medical Bacteria (HNCMB), Budapest.\\\\nTable 2. Host specificity and efficiency of plating of the two subgroups of new T5-like bacteriophages.\\\\nThe ESBL resistant E. coli strains were isolated from human extraintestinal infections, the Shigella flexneri strain was isolated from human feces and the Salmonella strains were isolated from food or from animals. These isolates were identified biochemically and were serotyped using O specific immunsera (data not shown).\\\\nEfficiency of plating (EOP) was determined by applying serial dilutions of bacteriophage suspensions in spot assays. The ratio of bacteriophage titer on each strain to the titer measured on E. coli MG1655 was considered the EOP of the bacteriophage on the given strain.\\\\nOne Step Growth Experiments\\\\nOne step growth experiment to determine burst size was conducted on the representative bacteriophages chee24 and chee130_1 on E. coli MG1655, according to the description of E. Bassiri1 with some modifications. Briefly, 5 × 108 bacteria were mixed with 5 × 106 bacteriophages in Luria broth (LB), setting the multiplicity of infection (MOI) to 0.01. The mixture was incubated for 10 min at room temperature for initial adsorption, then diluted 104-fold, and in a total volume of 10 ml incubated at 37°C for 1 h. Hundred microliter aliquots were taken every 5 min and plated on layered soft agar for counting. Experiments were run at three independent times in two parallels each. Burst size was determined as a ratio of average bacteriophage count of the baseline and the average bacteriophage count after the burst.\\\\nHeat Stability Tests\\\\nHeat stability of the bacteriophages was tested as follows: 1 ml stocks containing a 100-fold dilution of a bacteriophage stock in LB were incubated for 1 h at 25°C, 37°C, 42°C, and 80°C. After incubation, the titer of the treated bacteriophage stocks was determined with spot assay on E. coli MG1655. Experiments were performed in two parallels on each temperature.\\\\npH Tolerance Tests\\\\nStability of the bacteriophages under different pH values was tested by incubating 100-fold dilutions of the bacteriophages for 1 h at 37°C in 1 ml LB pH-adjusted to 3, 5, 7, and 9 with HCl and NaOH solutions. After incubation, the titer of the treated bacteriophage stocks was determined with spot assay on E. coli MG1655. Experiments were performed in two parallels on each pH value.\\\\nDetermination of Bacteriophage Morphology\\\\nBacteriophages were investigated by transmission electron microscopy (TEM). Briefly, drops of a high titer bacteriophage suspensions were placed on parafilm, absorbed onto carbon film, washed in TE buffer (10 mM TRIS, 1 mM EDTA, pH 6.9) and negatively-stained with 2% aqueous uranyl acetate, pH 5.0. Carbon film was collected with 300 mesh copper grids and access negative-stain was removed with filter paper and subsequentely air-dried. Samples were examined in a TEM 910 transmission electron microscope (Carl Zeiss, Oberkochen) at an acceleration voltage of 80 kV. Images were recorded digitally at calibrated magnifications with a Slow-Scan CCD-Camera (ProScan, 1024 × 1024, Scheuring, Germany) with ITEM-Software (Olympus Soft Imaging Solutions, Münster, Germany). Contrast and brightness were adjusted with Adobe Photoshop CS3.\\\\nBacteriophage DNA Isolation\\\\nBacteriophage DNA was isolated from bacteriophage stocks with a concentration of at least 1011 PFU/ml. The phenol-chloroform method described by Sambrook et al. (1987) was used for isolation with the modifications outlined by Tóth et al. (2016). Briefly, to remove any non-bacteriophage related DNA and RNA from the 700 μl bacteriophage suspension, the sample was treated with amplification grade DNase I and RNase A (Sigma Aldrich) in 10 μg/ml final concentration for 30 min at 37°C. For bacteriophage lysis, proteinase K (Sigma Aldrich) was added in 6.6 μg/ml final concentration to the lysis buffer (50 mM Tris–HCl (pH 8.0), 10 mM EDTA, 0.5% SDS) and incubated at 65°C for 30 min. The suspension was cooled to room temperature and mixed with 750 μl of a 1:1 mixture of equilibrated phenol and chloroform. After 5 min incubation at room temperature, proteins were removed by centrifugation at 10,000 × g for 5 min. The aqueous phase was transferred to a clean Eppendorf tube and bacteriophage DNA was precipitated after adding 0.1 volume of 3 M potassium acetate, pH 5.5 with 0.7 volume of isopropanol at ice for 20 min. DNA was collected by centrifugation at 13,000 × g at 10 min, washed with 600 μ1 70% ethanol and dissolved in 30 μl 10 mM Tris–HCl, 1 mM EDTA, pH 8.0 buffer.\\\\nBacteriophage Genome Sequencing and Analysis\\\\nGenomic DNA sequencing libraries were prepared using the Nextera XT kit (Illumina, Eindhoven, NL). Sequencing was performed using Nextseq Mid-output reagent kit v2 (2 × 150 bp) on an Illumina NextSeq 500. Assembly was performed with SPAdes (Bankevich et al., 2012). The genome was annotated using the RAST server (Overbeek et al., 2014). The genes orf5c (T5.035) and the bacteriophage DNA polymerase (T5.122) were used for phylogenetic analysis. The DNA sequences of these two genes from all bacteriophages determined in this study and of T5 reference genomes (Supplementary Table 1) were aligned using Clustal Omega (Sievers et al., 2011). The phylogenetic tree was visualized using MEGA5.2 (Hall, 2013). For whole-bacteriophage sequence phylogeny, the program VICTOR (Meier-Kolthoff and Goeker, 2017) was used with default settings and all T5 reference genomes (Supplementary Table 1). Determination of sequence homologies and the identification of SNPs were conducted by using the BLAST tools available at the NCBI website. Genome alignments were visualized with Easyfig (Sullivan et al., 2011) and modified using Inkscape. The left and right repeats were defined using a pile-up analysis of raw sequencing reads after mapping with CLC workbench 9.0 (Qiagen, Hilden, Germany). The repeats were defined as >130% of average coverage at the particular site (30% more coverage than average coverage).\\\\nSequence Accession Numbers\\\\nThe whole genome sequences of all bacteriophages determined in the current study were deposited in GenBank database under accession nos. MF431730- MF431741.\\\\nResults\\\\nIsolation of Bacteriophages\\\\nBy using E. coli K-12 derivative strains MG1655 and C600 as indicator and propagating strain, we isolated 12 new bacteriophages from different food samples. The origin, as well as the subgrouping of the bacteriophages are summarized in Table 1. The bacteriophages were isolated from either meat, meat products (e.g., sausages) or cheese.\\\\nMorphology\\\\nAll the new isolated bacteriophages showed Siphoviridae morphology, with an average head length of ~85 nm, head width of ~75 nm and a flexible, non-contractile tail with a length of ~200 nm. As a representative, the morphology of T5-like bacteriophage chee24 is shown (Figure 1).\\\\nFigure 1. Transmission electron micrograph of bacteriophage 24 showing Siphoviridae morphology.\\\\nHost Spectrum, Efficiency of Plating, Burst Size, and Stability\\\\nTo assess the host specificity of the bacteriophages, we tested their lytic capacity on a large variety of enterobacterial pathogenic and ESBL multidrug-resistant clinical strains listed in Table 2. We found that the novel T5-like bacteriophages showed two distinct lysis patterns, which we denote by a representative bacteriophage of each: bacteriophage chee24 (group chee24) and bacteriophage chee130_1 (group chee130_1). The grouping of the bacteriophages is indicated in Table 1. EOP values were determined by spot assay for these two bacteriophages. The lysis spectrum and EOP values are given in Table 2.\\\\nApart from K-12 derivative strains, several E. coli, Shigella, and Salmonella strains representing different pathotypes and serovars proved to be susceptible to the bacteriophages. While bacteriophage chee130_1 lysed enteropathogenic E. coli (EPEC), EIEC, and Shigella sonnei strains, bacteriophage 24 induced lysis on Salmonella serovars, EHEC O103:H2, and Shigella dysenteriae strains. Both bacteriophages showed lysis on enterotoxigenic E. coli (ETEC) type strain H10407. It has to be noted however, that in each case EOP values were orders of magnitude lower when compared to K-12 strains, and on some wild type strains the plaque morphology was opaque, unlike the clear plaques of ~2 mm in diameter, observed with the strain MG1655.\\\\nBurst size was determined with one step growth experiments on E. coli MG1655 (Figure 2). The burst sizes of bacteriophage chee24 and chee130_1 proved to be different, in the case of the former, it was around 1,000 PFU/cell, while in the case of chee130_1, the burst size was around 44 PFU/cell. In both cases the burst occurred ~45 min after the end of the initial absorption.\\\\nFigure 2. One step growth curves of new T5-like bacteriophages on E. coli MG1655 strain. PFU values of bacteriophage chee24 are marked with triangles, those of bacteriophage chee130_1 are marked with squares. The error bars indicate standard deviations from the results of the three independent triplicate experiments.\\\\nHeat stability experiments showed that incubation of 1010 PFU bacteriophage stocks at 25°C, 37°C and 42°C for 1 h did not significantly affect the EOP of the bacteriophages (data not shown). Incubation at 80°C, however, caused 104-fold average decrease in the bacteriophage titer.\\\\nA pH value of 3 completely deactivated the bacteriophages, as no plaques were observable even in the spots of the concentrated suspensions. On the other hand, no significant decrease in titer could be observed at pH values 5, 7, and 9 after 1 h incubation (data not shown).\\\\nWhole Genome Sequence Analysis\\\\nTo examine the genomic structures and the potential diversity in the genomes, all the 12 newly isolated bacteriophage genome sequences were determined and detailed genomic comparison was conducted. Genome sequencing of the bacteriophages showed that all the bacteriophages have a linear double-stranded DNA genome, and belong to T5-like bacteriophages. The two subgroups with different host specificity also have minor genetic differences. The genome of the “group chee24” bacteriophages is 120,618 (bacteriophage pork27), 120,620 (bacteriophage saus111K), 120,622 (bacteriophage chee24, pork29 and saus47N) or 120,629 bp (bacteriophage poul124) long, and contains 165 (saus47N, saus111K, poul124), 166 (chee24, pork29) or 168 (pork27) protein-encoding CDS, as well as 18 tRNA genes, respectively. Its GC content is 39.3%. On the other hand, the genome size of the ‘group chee130_1’ bacteriophages is 121,986 bp, with 164 (saus132, poul149), 165 (cott162) or 167 (chee130_1, chee158, saus176N) protein-encoding CDS, 20 tRNA genes, and a GC content of 39.7%. These groups were congruent with the different host spectrum observable within the bacteriophages, indicating that the chee24-like and chee130_1-like bacteriophages form distinct genetic groups. The list of ORFs compared to each other and to the original T5 bacteriophage (GenBank AY543070.1) are shown in Supplementary Tables 2, 3. Members of both subgroups showed the characteristic functional regions in their genomes, which is a common feature of T5-like phages, having pre-early, early and late genomic regions named after the time of their transcription, as well as a terminal repeat region on the 5′ and 3′ end, which is 9,968 bp long in chee24-like phages, and 8,717 bp in group chee130_1 phages. For group chee24 bacteriophages, the bacteriophage pork27 differed by 1 SNP in the terminal repeat, while for group chee130_1 phages, all terminal repeats of other than chee130_1 differed from those present in the prototype, with saus132, chee158, cott162 and saus176N having 3 SNPs, and poul149 harboring 4 SNPs.\\\\nAt the nucleotide level, the newly identified T5-like bacteriophages not only have the same genome size within the two subgroups—group chee24 and group chee130_1-, but when compared to the “type bacteriophage” of their groups as reference bacteriophage chee24 or chee130_1, respectively, they differ in only 2–5 nucleotide positions. It has to be noted that in the case of bacteriophage pork27 and saus47N, one SNP causes a premature stop codon in bacteriophage protein genes of unknown function. In the case of bacteriophages saus132, poul149 and chee158 there are premature stop codons in a bacteriophage tail fiber gene. A summary of these genomic locations is given in Table 3.\\\\nTable 3. Single nucleotide polymorphisms shown by T5-like bacteriophages of the chee24-like subgroup (A) and the chee130_1-like subgroup (B) when compared to the type bacteriophage of each subgroup.\\\\nNone of the new characterized bacteriophage genomes contained any known virulence, toxin, antibiotic resistance-encoding and virulence regulator genes.\\\\nPhylogenetic Analysis\\\\nTo reveal the phylogenetic position of bacteriophages chee24 and chee130_1 among T5-like phages, several comparisons were conducted. Comparison based on the genes orf5c (T5.035) and the bacteriophage DNA polymerase (T5.122) of bacteriophage T5 (AY692264.1) showed that while bacteriophage 24 proved to be the closest relative of E. coli bacteriophages DT57C and DT571/2, bacteriophage chee130_1 are closely related to the Salmonella bacteriophages Stitch and SPC35 (Figure 3).\\\\nFigure 3. Phylogenetic tree of T5-like bacteriophages based on the Clustal Omega alignment of the genes orf5c (T5.035, A) and the phage DNA polymerase (T5.122, B) of bacteriophage T5 (AY692264.1). The scale represents homology %. The list of included T5-like phage genomes with their GenBank accession numbers is given in Supplementary Table 1.\\\\nWhole-genome based phylogeny of all T5-like bacteriophages clearly indicated that the phages isolated in the current study are members of the T5-like bacteriophages (Figure 4). The topology of this tree is entirely different from the one based on the single genes, putting the two representative phages next to the phage genomes that also proved closest homologs with the BLAST analysis. An alignment of T5-like genomes with mutual homologies including representatives of the two subgroups is presented in Figure 5, showing the 5′ end of the late region as the most variable, which contains mainly tail fiber genes.\\\\nFigure 4. Phylogenetic tree of T5-like bacteriophages made with the whole genome-based VICTOR analysis. The scale represents homology %. The list of included T5-like phage genomes with their GenBank accession numbers is given in Supplementary Table 1.\\\\nFigure 5. Alignment of T5-like bacteriophage whole genomes. GenBank accession numbers of the phages are given in Supplementary Table 1.Arrows with different colors represent ORFs associated with the genomic regions indicated at the top. The gray bar in the lower right corner shows the identity percentage associated with the color of the bars connecting homologous regions.\\\\nComparing the genomes of bacteriophage chee24 and chee130_1at the nucleotide-level, regions showing low homology to each other were the tail fiber region and an open reading frame encoding a bacteriophage-associated receptor binding protein gene. All these features seem to contribute to the differences observed in the host specificity of the two bacteriophage groups. At nucleotide level, the closest homolog to bacteriophage chee24 is bacteriophage SPC35 (GenBank no. HQ406778.). In addition, two hypothetical proteins, a thymidylate synthase gene, and two subunits of a HNH endonuclease, a receptor binding protein (CDS ADW80123.1) of SPC35 is also absent in chee24, suggesting different host specificity. Bacteriophage EPS7 (GenBank no. CP000917) is the closest homolog to bacteriophage chee130_1 of the previously sequenced T5-like bacteriophages, and carries homologs to all receptor-associated proteins, albeit the similarity is not 100% in each case. Furthermore, chee130_1 lacks a few hypothetical proteins and AGC_0146 putative tail protein as well. Genes showing low homology and those that are absent from bacteriophages of either the chee24 or chee130_1subgroup are listed in Table 4.\\\\nTable 4. List of genes and regions showing low homology or missing in chee24-like (A) and chee130_1-like (B) bacteriophages when compared to their closest homolog bacteriophages.\\\\nDiscussion\\\\nUsing E. coli K-12 C600 strain as the host bacterium, we isolated and characterized 12 novel T5-like bacteriophages from independent food samples of various animal origin from around the world These novel bacteriophages represented two subgroups based on their host specificity. While the most effective host of the bacteriophages were K-12 derivative E. coli strains, several strains representing important intestinal pathotypes of E. coli, Shigella, and Salmonella also proved susceptible to the bacteriophages. The Siphoviridae morphology and the genomic characteristics including the genome size, the genome architecture, the low GC content and the sequence homologies showed that the bacteriophages are T5-like bacteriophages.\\\\nAn important feature of these newly found phages is that they were isolated from food samples, unlike the previously isolated T5-like bacteriophages, which originated from sewage and fecal sources (Kim and Ryu, 2011; Golomidova et al., 2015). There is also one case where a T5-like phage was isolated and characterized from a commercially available bacteriophage cocktail designed against enteric pathogens (Piya et al., 2015). Our results indicate that food can be a rich source of bacteriophages effective against pathogenic bacteria potentially residing in the same environment. These phages maintain their infective capacity in this environment suggesting that foodborne phages are promising biocontrol candidates against foodborne bacteria.\\\\nOur results also show the huge diversity of bacteriophages, even from well-characterized groups such as T5-like bacteriophages, as both the chee24 and chee130_1 subgroups are different from the closest related bacteriophages. Our data indicate that the most crucial differences are within genes encoding receptor-binding proteins (Table 4, Figure 5). The closest colinear relative of bacteriophage chee24 is SPC35, which was reported to lyse Salmonella Typhimurium and Enteritidis strains, as well as a large set of K-12 derivative and commensal E. coli (Kim and Ryu, 2011). However, the host spectrum of group chee24 phages is particularly broad, and includes a strain representing one of the so-called “big six” serotypes of EHEC (O103:H2), as well as an ESBL E. coli strain of serogroup O73 isolated from an extraintestinal infection. The differences in ORFs 145 and 149, annotated as “phage tail fibers” and “receptor binding protein” could account for the discrepancy.\\\\nWe wanted to test the efficiency of the new T5-like phages on target strains, and the one step growth on E. coli MG1655 yielded burst sizes comparable to those of the T5-like bacteriophage phiLLS (Amarillas et al., 2017).\\\\nThe closest homolog of the chee130_1 subgroup, EPS7 is able to lyse the same Salmonella serovars and seven unnamed E. coli strains (Hong et al., 2008), however, phage chee130_1 did not lyse Salmonella strains. On the other hand, phage chee130_1 was able to propagate on Shigella sonnei, EIEC and EPEC strains. All of these differences in host range underline the potential role that genes showing low homology to closely related bacteriophages (Table 4) might have importance in host specificity. In the case of chee130_1, ORF134 that is annotated as “phage tail fibers” has very low homology to the corresponding gene in EPS7 and seems to support this notion. It is interesting to note however, that in ORF133, another gene annotated as phage tail fiber, there are premature stop codons in the case of bacteriophages saus132, poul149 and chee158. These mutations nevertheless did not seem to have any effect on the bacterial host specificity of these bacteriophages when compared to other members of their subgroup (data not shown).\\\\nLee et al. (2016) suggested that besides bacteriophage cocktails, a solution for simultaneous biocontrol of several foodborne pathogens could be the cloning of several receptor binding genes into a single bacteriophage. For either purpose, obtaining knowledge about new bacteriophages capable of infecting these pathogens and assessing their host spectrum is essential.\\\\nBesides the above-cited studies of promising candidate phages to be used as biocontrol agents against pathogenic bacteria, there are already several commercially available phage cocktails aimed at various foodborne enterobacterial pathogens (reviewed by Abedon, 2017). However, according to our knowledge except for ShigaShield, which is aimed against Shigellae (Soffer et al., 2017), and the Microgen ColiProteus aimed against E. coli and Proteus (McCallin et al., 2013), none of the component phages have been sequenced. In order to assess the applicability of a phage either for biocontrol of for therapy, extensive knowledge of its genome is essential, to determine whether the phage carries “undesirable” genes. In-depth knowledge of genes determining host specificity or those contributing to phage stability in different environments are also valuable information when selecting bacteriophages for a given therapeutic or biocontrol purpose.\\\\nAs it has been noted by Brüssow (2012), the development of an actual phage therapy treatment can take years from phage isolation to performing clinical trials. This process includes for exapmle the purification (Bourdin et al., 2014b), as well as characterisation of host specificity (Bourdin et al., 2014a) and testing the biological safety of the phages either in animals (Raya et al., 2011) or human subjects (Sarker et al., 2012, 2016). An extensive genetic database of phages isolated from different sources, representing different genuses and host spectra could speed up the process of selecting phages for clinical trials.\\\\nIn summary, we identified 12 new T5-like bacteriophages from food samples, which represent two new species within this bacteriophage group. Our study so far shows that they fulfill important criteria listed by Hagens and Loessner (2010) for bacteriophages to be used as biocontrol: their genome sequence was determined, they propagate well on a non-pathogenic host, they lack any virulence, virulence regulator, toxin, and antibiotic resistance genes. They also seem to be strictly lytic and were shown to be stable during storage and application temperatures, as well as a range of pH values usual in most of the food.\\\\nAll the above-mentioned properties, coupled with their food origin, as well as their broad and partially overlapping, supplementary host spectrum makes the new T5-like bacteriophages valuable candidates as effective bio-control agents against foodborne pathogenic enterobacteria.\\\\nAuthor Contributions\\\\nIT and TC conceived and elaborated the study. DS, LF, MR, JS, and IT conducted the phenotypic experiments. LF performed sequencing and provided bioinformatical tools. DS, LF, TC, and IT analyzed the results and composed the manuscript. All authors reviewed the manuscript.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nThis work was supported by the 7th EU Framework Programme PROMISE project (project n. 265877), the Hungarian National Research, Development and Innovative Office (NKFIH, grant 124335) and grants from the Bundesministerium fuer Bildung und Forschung (BMBF, Germany) within the German Center for Infection research (DZIF/grant number 8032808811 to TC).\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2018.00202/full#supplementary-material\\\\nFootnotes\\\\nReferences\\\\nAllué-Guardia, A., García-Aljaro, C., and Muniesa, M. (2011). Bacteriophage-encoding cytolethal distending toxin type V gene induced from nonclinical Escherichia coli isolates. Infect. Immun. 79, 3262–3272. doi: 10.1128/IAI.05071-11\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAmarillas, L., Rubí-Rangel, L., Chaidez, C., González-Robles, A., Lightbourn-Rojas, L., and León-Félix, J. (2017). Isolation and characterization of phiLLS, a novel phage with potential biocontrol agent against multidrug-resistant Escherichia coli. Front. Microbiol. 8:1355. doi: 10.3389/fmicb.2017.01355\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAnderson, M., Sansonetti, P. J., and Marteyn, B. S. (2016). Shigella diversity and changing landscape: insights for the twenty-first century. Front. Cell. Infect. Microbiol. 6:45. doi: 10.3389/fcimb.2016.00045\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455–477. doi: 10.1089/cmb.2012.0021\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBlattner, F. R., Plunkett, G., Bloch, C. A., Perna, N. T., Burland, V., Riley, M., et al. (1997). The complete genome sequence of Escherichia coli K-12. Science 277, 1453–1462. doi: 10.1126/science.277.5331.1453\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBourdin, G., Navarro, A., Sarker, S. A., Pittet, A.-C., Qadri, F., Sultana, S., et al. (2014a). Coverage of diarrhoea-associated Escherichia coli isolates from different origins with two types of phage cocktails. Microb. Biotechnol. 7, 165–176. doi: 10.1111/1751-7915.12113\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBourdin, G., Schmitt, B., Marvin Guy, L., Germond, J.-E., Zuber, S., Michot, L., et al. (2014b). Amplification and purification of T4-like Escherichia coli phages for phage therapy: from laboratory to pilot scale. Appl. Environ. Microbiol. 80, 1469–1476. doi: 10.1128/AEM.03357-13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCarter, C. D., Parks, A., Abuladze, T., Li, M., Woolston, J., Magnone, J., et al. (2012). Bacteriophage cocktail significantly reduces Escherichia coli O157: H7 contamination of lettuce and beef, but does not protect against recontamination. Bacteriophage 2, 178–185. doi: 10.4161/bact.22825\\\\nCrossRef Full Text | Google Scholar\\\\nCroxen, M. A., Law, R. J., Scholz, R., Keeney, K. M., Wlodarska, M., and Finlay, B. B. (2013). Recent advances in understanding enteric pathogenic Escherichia coli. Clin. Microbiol. Rev. 26, 822–880. doi: 10.1128/CMR.00022-13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nEvans, D. G., Evans, D. J., and Pierce, N. F. (1973). Differences in the response of rabbit small intestine to heat-labile and heat-stable enterotoxins of Escherichia coli. Infect. Immun. 7, 873–880.\\\\nPubMed Abstract | Google Scholar\\\\nGolomidova, A. K., Kulikov, E. E., Prokhorov, N. S., Guerrero-Ferreira, R. C., Ksenzenko, V. N., Tarasyan, K. K., et al. (2015). Complete genome sequences of T5-related Escherichia coli bacteriophages DT57C and DT571/2 isolated from horse feces. Arch. Virol. 160, 3133–3137. doi: 10.1007/s00705-015-2582-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHagens, S., and Loessner, M. J. (2010). Bacteriophage for biocontrol of foodborne pathogens: calculations and considerations. Curr. Pharm. Biotechnol. 11, 58–68. doi: 10.2174/138920110790725429\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHayashi, T., Makino, K., Ohnishi, M., Kurokawa, K., Ishii, K., Yokoyama, K., et al. (2001). Complete genome sequence of enterohemorrhagic Escherichia coli O157:H7 and genomic comparison with a laboratory strain K-12. DNA Res. 8, 11–22. doi: 10.1093/dnares/8.1.11\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHochhut, B., Wilde, C., Balling, G., Middendorf, B., Dobrindt, U., Brzuszkiewicz, E., et al. (2006). Role of pathogenicity island-associated integrases in the genome plasticity of uropathogenic Escherichia coli strain 536. Mol. Microbiol. 61, 584–595. doi: 10.1111/j.1365-2958.2006.05255.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHong, J., Kim, K.-P., Heu, S., Lee, S. J., Adhya, S., and Ryu, S. (2008). Identification of host receptor and receptor-binding module of a newly sequenced T5-like phage EPS7. FEMS Microbiol. Lett. 289, 202–209. doi: 10.1111/j.1574-6968.2008.01397.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHong, Y., Pan, Y., Harman, N. J., and Ebner, P. D. (2014). Complete genome sequences of two Escherichia coli O157:H7 phages effective in limiting contamination of food products. Genome Announc. 2:e00519–14. doi: 10.1128/genomeA.00519-14\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHunter, P. (2012). Where next for antibiotics?: the immune system and the nature of pathogenicity are providing vital clues in the fight against antibiotic-resistant bacteria. EMBO Rep. 13, 680–683. doi: 10.1038/embor.2012.101\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nIguchi, A., Thomson, N. R., Ogura, Y., Saunders, D., Ooka, T., Henderson, I. R., et al. (2009). Complete genome sequence and comparative genome analysis of enteropathogenic Escherichia coli O127:H6 strain E2348/69. J. Bacteriol. 191, 347–354. doi: 10.1128/JB.01238-08\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJohnson, R. P., Gyles, C. L., Huff, W. E., Ojha, S., Huff, G. R., Rath, N. C., et al. (2008). Bacteriophages for prophylaxis and therapy in cattle, poultry and pigs. Anim. Health Res. Rev. 9, 201–215. doi: 10.1017/S1466252308001576\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKim, M., and Ryu, S. (2011). Characterization of a T5-like coliphage, SPC35, and differential development of resistance to SPC35 in Salmonella enterica serovar typhimurium and Escherichia coli. Appl. Environ. Microbiol. 77, 2042–2050. doi: 10.1128/AEM.02504-10\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLee, H., Ku, H.-J., Lee, D.-H., Kim, Y.-T., Shin, H., Ryu, S., et al. (2016). Characterization and genomic study of the novel bacteriophage HY01 infecting both Escherichia coli O157:H7 and Shigella flexneri: potential as a biocontrol agent in food. PLoS ONE 11:e0168985. doi: 10.1371/journal.pone.0168985\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMarchès, O., Ledger, T. N., Boury, M., Ohara, M., Tu, X., Goffaux, F., et al. (2003). Enteropathogenic and enterohaemorrhagic Escherichia coli deliver a novel effector called Cif, which blocks cell cycle G2/M transition. Mol. Microbiol. 50, 1553–1567. doi: 10.1046/j.1365-2958.2003.03821.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMcCallin, S., Sarker, S. A., Barretto, C., Sultana, S., Berger, B., Huq, S., et al. (2013). Safety analysis of a Russian phage cocktail: from MetaGenomic analysis to oral application in healthy humans. Virology 443, 187–196. doi: 10.1016/j.virol.2013.05.022\\\\nCrossRef Full Text | Google Scholar\\\\nMoriel, D. G., Bertoldi, I., Spagnuolo, A., Marchi, S., Rosini, R., Nesta, B., et al. (2010). Identification of protective and broadly conserved vaccine antigens from the genome of extraintestinal pathogenic Escherichia coli. Proc. Natl. Acad. Sci. U.S.A. 107, 9072–9077. doi: 10.1073/pnas.0915077107\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nNagy, B., Szmolka, A., Smole MoŽina, S., Kovač, J., Strauss, A., Schlager, S., et al. (2015). Virulence and antimicrobial resistance determinants of verotoxigenic Escherichia coli (VTEC) and of multidrug-resistant E. coli from foods of animal origin illegally imported to the EU by flight passengers. Int. J. Food Microbiol. 209, 52–59. doi: 10.1016/j.ijfoodmicro.2015.06.026\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nNiu, Y. D., Stanford, K., Kropinski, A. M., Ackermann, H.-W., Johnson, R. P., She, Y.-M., et al. (2012). Genomic, proteomic and physiological characterization of a T5-like bacteriophage for control of shiga toxin-producing Escherichia coli O157:H7. PLoS ONE 7:e34585. doi: 10.1371/journal.pone.0034585\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nOverbeek, R., Olson, R., Pusch, G. D., Olsen, G. J., Davis, J. J., Disz, T., et al. (2014). The SEED and the Rapid Annotation of microbial genomes using Subsystems Technology (RAST). Nucleic Acids Res. 42, D206–D214. doi: 10.1093/nar/gkt1226\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPerna, N. T., Plunkett, G., Burland, V., Mau, B., Glasner, J. D., Rose, D. J., et al. (2001). Genome sequence of enterohaemorrhagic Escherichia coli O157:H7. Nature 409, 529–533. doi: 10.1038/35054089\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPiya, D., Xie, Y., Hernandez Morales, A. C., and Kuty Everett, G. F. (2015). Complete genome sequence of Salmonella enterica serovar Typhimurium siphophage shivani. Genome Announc. 3:e01443–14. doi: 10.1128/genomeA.01443-14\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRaya, R. R., Oot, R. A., Moore-Maley, B., Wieland, S., Callaway, T. R., Kutter, E. M., et al. (2011). Naturally resident and exogenously applied T4-like and T5-like bacteriophages can reduce Escherichia coli O157:H7 levels in sheep guts. Bacteriophage 1, 15–24. doi: 10.4161/bact.1.1.14175\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSambrook, J., Maniatis, T., Fritsch, E. F., and Laboratory, C. S. H. (1987). Molecular Cloning : A Laboratory Manual, 2nd Edn. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press. Available online at: http://trove.nla.gov.au/work/13615226 (Accessed June 1, 2017).\\\\nSarker, S. A., McCallin, S., Barretto, C., Berger, B., Pittet, A.-C., Sultana, S., et al. (2012). Oral T4-like phage cocktail application to healthy adult volunteers from Bangladesh. Virology 434, 222–232. doi: 10.1016/j.virol.2012.09.002\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSarker, S. A., Sultana, S., Reuteler, G., Moine, D., Descombes, P., Charton, F., et al. (2016). Oral phage therapy of acute bacterial diarrhea with two coliphage preparations: a randomized trial in children from Bangladesh. EBioMedicine 4, 124–137. doi: 10.1016/j.ebiom.2015.12.023\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSievers, F., Wilm, A., Dineen, D., Gibson, T. J., Karplus, K., Li, W., et al. (2011). Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Mol. Syst. Biol. 7, 539–539. doi: 10.1038/msb.2011.75\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSoffer, N., Woolston, J., Li, M., Das, C., and Sulakvelidze, A. (2017). Bacteriophage preparation lytic for Shigella significantly reduces Shigella sonnei contamination in various foods. PLoS ONE 12:e0175256. doi: 10.1371/journal.pone.0175256\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nStrauch, E., Lurz, R., and Beutin, L. (2001). Characterization of a shiga toxin-encoding temperate bacteriophage of Shigella sonnei. Infect. Immun. 69, 7588–7595. doi: 10.1128/IAI.69.12.7588-7595.2001\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTóth, I., Hérault, F., Beutin, L., and Oswald, E. (2003). Production of cytolethal distending toxins by pathogenic Escherichia coli strains isolated from human and animal sources: establishment of the existence of a new cdt variant (Type IV). J. Clin. Microbiol. 41, 4285–4291. doi: 10.1128/JCM.41.9.4285-4291.2003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTóth, I., Schmidt, H., Kardos, G., Lancz, Z., Creuzburg, K., Damjanova, I., et al. (2009). Virulence genes and molecular typing of different groups of Escherichia coli O157 strains in cattle. Appl. Environ. Microbiol. 75, 6282–6291. doi: 10.1128/AEM.00873-09\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTóth, I., Sváb, D., Bálint, B., Brown-Jaque, M., and Maróti, G. (2016). Comparative analysis of the Shiga toxin converting bacteriophage first detected in Shigella sonnei. Infect. Genet. Evol. 37, 150–157. doi: 10.1016/j.meegid.2015.11.022\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWang, J., Jiang, Y., Vincent, M., Sun, Y., Yu, H., Wang, J., et al. (2005). Complete genome sequence of bacteriophage T5. Virology 332, 45–65. doi: 10.1016/j.virol.2004.10.049\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWiles, S., Clare, S., Harker, J., Huett, A., Young, D., Dougan, G., et al. (2005). Organ-specificity, colonization and clearance dynamics in vivo following oral challenges with the murine pathogen Citrobacter rodentium. Cell. Microbiol. 7, 459–459. doi: 10.1111/j.1462-5822.2004.00479.x\\\\nCrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fped.2022.856520\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fped.2022.856520/full\",\\n    \"loadedTime\": \"2024-01-23T08:49:37.185Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fped.2022.856520\",\\n    \"title\": \"Frontiers | Optimisation and Application of a Novel Method to Identify Bacteriophages in Maternal Milk and Infant Stool Identifies Host-Phage Communities Within Preterm Infant Gut\",\\n    \"description\": \"Human milk oligosaccharides, proteins such as lactoferrin, and bacteria represent just some of the bioactive components of mothers’ breast milk (BM). Bacteriophages (viruses that infect bacteria) are an often-overlooked component of BM that can cause major changes in microbial composition and metabolism. BM bacteriophage composition has been explored in term and healthy infants, suggesting vertical transmission of bacteriophages occurs between mothers and their infants. Several important differences between term and very preterm infants (< 30 weeks gestational age) may limit this phenomenon in the latter. To better understand the link between BM bacteriophages and gut microbiomes of very preterm infants in health and disease, standardised protocols are required for isolation and characterisation from BM. In this study we use isolated nucleic acid content, bacteriophage richness and Shannon diversity to validate several parameters applicable during bacteriophage isolation from precious BM samples. Parameters validated include sample volume required; centrifugal sedimentation of microbes; hydrolysis of milk samples with digestive enzymes; induction of temperate bacteriophages and concentration / purification of isolated bacteriophage particles in donor milk (DM). Our optimised method enables characterisation of bacteriophages from as little as 0.1 mL BM. We identify viral families that were exclusively identified with inclusion of induction of temperate bacteriophages (Inoviridae) and hydrolysis of milk lipid processes (Iridoviridae and Baculoviridae). Once applied to a small clinical cohort we demonstrate vertical transmission of bacteriophages from mothers BM to the gut of very preterm infants at the species level. This optimised method will enable future research characterising the bacteriophage composition of BM in very preterm infants to determine their clinical relevance in the development of a healthy preterm infant gut microbiome.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nIt is widely accepted that mother\\'s breast milk (BM) is the gold standard of nutritional care for newborn infants (1, 2) due to the myriad of beneficial effects of BM for newborn infant growth and development (3, 4). Various studies have characterised the components present in BM, including human milk oligosaccharides (5), bioactive proteins, such as lactoferrin (6, 7), and bacteria (8–10), to elucidate potential modes of the improved health outcomes in infants who receive BM. There are several key differences between term and preterm infants, including length of hospitalisation and exposure of preterms, to necessary clinical interventions that impact microbiome development (11). These include chemical interventions, such as antibiotics and antivirals, or mechanical interventions, such as invasive catheters and feeding tubes, as well as exposure to microbes resident within the neonatal unit environment. In addition to these neonate-specific factors, evidence has emerged of longitudinal change to the nutritional composition of BM particularly the quantity and quality of proteins and lipids. Preterm BM, therefore, differs substantially from that of term BM, both in nutritional and microbial composition.\\\\nAnother bioactive component of BM is the bacteriophage content. Initial analysis of vertical bacteriophage transmission between BM and the neonatal GI tract performed by Breitbart et al. (12) in 2008 found no shared bacteriophages between the single mother-infant pair studied despite large quantities of identifiable virus in infant stool. Since this pioneering study improvements in metagenomic sequencing technology and bacteriophage genome databases have facilitated further investigations. In 2017, Duranti et al. (13) reported the presence of Bifidobacterium phages in both mother\\'s BM and their infant\\'s stools demonstrating the potential of bacteriophage transmission in term mother-infant pairs. Pannaraj et al. (14) compared alpha and beta diversity of BM and stool samples from a separate cohort of term infants, finding a greater alpha diversity in BM viromes than stools as well as several shared taxonomic features between BM and stool from the same mother-infant pairs. However, these taxonomic analyses were capped at the family level which limits our ability to determine whether there is a transmission of bacteriophages from mother to infant via BM. A comparison between the virome of BM given to infants delivered at term or preterm found contrasting taxonomic compositions (15). This study identified several bacteriophage species with host specificity for common skin commensals. Preterm infants enrolled in the study were a minimum of 32 weeks gestation at birth although BM bacteriophages were not compared with infant stool bacteriophages. A knowledge gap still exists to elucidate bacteriophage transmission between BM and neonatal stool in very preterm (<30 weeks gestation) neonates.\\\\nAlthough bacteriophages have now been revealed as one of the active components of BM, understanding their full impact in preterm BM, including their interactions with microbes in the preterm infant GIT, remains enigmatic. A major obstacle to overcome when interpreting the results of each study investigating bacteriophages in BM is the methodological differences between studies (12, 14, 16). Indeed, some methods appear to have limitations that lead to selection bias during viral isolation (17). For instance, the application of glass beads during sample homogenisation can reduce non-enveloped bacteriophages, membrane filtration with 0.2 and/or 0.45 μM filters may fail to recover giant viruses, and caesium chloride (CsCl) density-gradient ultracentrifugation fails to recover enveloped bacteriophages or those within particular buoyant densities (17). Such contrasting approaches in the literature prove an even greater issue when investigating preterm infant microbiome because sample volumes are so small and are therefore extremely precious.\\\\nThis study aims to develop a robust method for isolation of bacteriophages from the smallest possible volumes of BM from mothers to very preterm infants to promote accurate characterisation of these communities in future studies. We utilize pasteurised donor milk (DM) to validate bacteriophage isolation procedures, then apply the validated method to BM and neonatal stools collected from 4 mother-infant pairs to demonstrate its utility in identifying vertical transmission of bacteriophage species between clinical BM and stool samples.\\\\nMaterials and Methods\\\\nDevelopment and Validation of Bacteriophage Isolation Protocol\\\\nAll laboratory work was conducted in a Class II safety cabinet using nuclease-free or sterilised consumables and equipment. PROLACT HM® human donor milk (Prolacta Bioscience Inc., USA) was used for method validation experiments. Donor milk (DM) was stored at −80°C until the day of processing, when samples were defrosted at 4°C, gently mixed by inversion and dispensed into 1 mL aliquots. These were then further aliquoted, on ice, into 0.1, 0.2, 0.4, and 0.6 mL volumes to validate the required sample volume for bacteriophage isolation and DNA extraction. Six replicates of each sample volume were processed. DM aliquots were homogenised with cold sterile 1x PBS in a ratio of 1:5 (Supplementary Table 1).\\\\nExperiment 1: Sedimentation of Microbial Cells With Increased Centrifugation Speeds\\\\nTo assess the effect of centrifugal force in separating microbial cells for the induction of temperate bacteriophages, eight replicates of PBS-homogenised DM were split into three centrifugation groups (Supplementary Figure 1). In each group, samples were centrifuged at 400 xG at room temperature for 10 minutes to separate the DM lipid and skimmed milk, as previously described (18). The DM lipids were discarded, while the skimmed milk was centrifuged at 10, 15, and 20 k xG at 4°C for 10 min. The resulting microbial cell pellets were incubated with norfloxacin (1 μg/mL) at 37°C for 1 h to induce temperate bacteriophages, followed by centrifugation at 15 k xG at 4°C for 10 min to remove remaining microbial cell debris (19). Supernatants resulting from the first centrifugation (i.e., free bacteriophages) and the second centrifugation (i.e., induced temperate bacteriophages) were subjected to DNase (2 U/mL) and RNase (0.5 U/mL) treatments to remove human and microbial chromosomal DNA, as previously described (19). DNA of the isolated bacteriophages were then extracted using QIAamp MinElute Virus Spin Kit (QIAGEN, Germany), following the manufacturer\\'s protocol. Equal volumes of autoclaved 1x PBS were included as a kit negative control in every extraction batch.\\\\nExperiment 2: Release of Bacteriophages in DM and Milk-Lipids With Digestive Enzymes\\\\nDigestive enzymes similar to those in infant guts (20) were used to hydrolyse milk fat globule membrane (MFGM) and release bacteria and bacteriophages that are associated with the MFGM. For hydrolysis of DM (Supplementary Figure 2), four replicates of PBS-homogenised DM were incubated with digestive enzymes (0.4 mg/mL of pepsin and 0.86 mg/mL of lipase) at 37°C and shaking at 350 rpm for an hour. Hydrolysed samples were centrifuged at 400 xG at room temperature for 10 min to separate the DM lipid and skimmed milk, as previously described (18). The resulting DM lipids were carefully removed and were discarded, whilst the skimmed milk was centrifuged at 10 k xG at 4°C for 10 min. As described in Experiment 1, the separated microbial cell pellets were then subjected to norfloxacin-mediated bacteriophage induction and removal of remaining microbial cell debris. Both supernatants of free and temperate bacteriophages were ready for the subsequent steps of DNase and RNase treatments and DNA extraction.\\\\nFor hydrolysis of DM lipids (Supplementary Figure 3), four replicates of PBS-homogenised DM samples were centrifuged at 400 xG and room temperature for 10 min. The resulting DM lipids were carefully separated and homogenised in 1 mL of cold sterile 1x PBS. Four further DM samples were frozen at −20°C for 30 min following centrifugation before separating the DM lipids and homogenising in cold sterile 1x PBS. These frozen lipid samples were used to validate freezing the DM lipid fraction during bacteriophage isolation. PBS-homogenised lipid suspensions were incubated with digestive enzymes at 37°C and shaking at 350 rpm for an hour, then centrifuged at 10 k xG at 4°C for 10 min for the separation of free bacteriophages and microbial cell pellets. As described upfront in Experiment 1, the separated microbial cell pellets were then subjected to norfloxacin-mediated bacteriophage induction and removal of remaining microbial cell debris. Both supernatants of free and temperate bacteriophages were ready for the subsequent steps of DNase and RNase treatments and DNA extraction.\\\\nIn addition, four replicates of PBS-homogenised DM samples without the input of digestive enzymes were processed as a control (Supplementary Figure 4).\\\\nExperiment 3: Concentrating and Purifying Isolated Bacteriophages\\\\nIsolated bacteriophages from the DM (Supplementary Figures 2, 4), were subjected to further concentration steps using 10 % PEG8000 and purification steps using Centriprep® 50 kDa filter devices (Merck Millipore, USA), as previously described (21, 22).\\\\nIn addition, four replicates of PBS-homogenised DM samples without the application of 10 % PEG8000 and Centriprep® 50 kDa filter devices were processed as a control (Supplementary Figure 4).\\\\nApplication of Validated Bacteriophage Isolation Protocol to Mother-Infant Pair Samples\\\\nThe pilot study utilised BM and stool samples from a total of four very preterm (<30 weeks gestation) mother-infant pairs enrolled on the SERVIS study at the Neonatal Intensive Care Unit, Royal Victoria Infirmary, Newcastle upon Tyne, UK, as previously described (8, 23). Samples were collected with ethical permission granted by NRES Committee North East–Newcastle & North Tyneside 2 (10/H0908/39) and analysed with written parental consent. Expressed BM and stool samples were stored at −80°C until processing. Bacteriophage isolation and DNA extraction were performed using the optimised protocol in Figure 4.\\\\nBacteriophage Metagenome Sequencing\\\\nThe concentration and purity of extracted bacteriophage DNA were determined using Qubit 2.0 Fluorometer (Thermo Fisher Scientific, USA) and NanoDrop™ 1000 Spectrophotometer (Thermo Fisher Scientific, USA), respectively. Nucleic acid sequencing was performed by NU-OMICS DNA Sequencing Research Facility (Northumbria University, UK) using the Nextera XT kit and V3 600 cycle chemistry on the Illumina MiSeq (validation study) and V2.5 300 cycle mid-output chemistry on the Illumina NextSeq (pilot study) (Illumina Inc., USA). Raw sequencing data are available at ENA (study accession PRJEB49989).\\\\nInformatic Processing of Method Validation Data\\\\nRaw read data were de-multiplexed using MiSeq Reporter Software (Illumina, USA). Illumina adapter sequences were removed, sequences were trimmed with a sliding window (Q30) and any reads <50 nucleotides or >300 nucleotides were culled using FastQC (24) and Trimmomatic (25) in Galaxy (26). Trimmed paired and unpaired reads files were merged for further filtering with BBTools (27). Sequences identified in negative controls were culled from experimental samples. Low-complexity reads were removed using BBmask (entropy 0.7), and duplicated reads were removed with BBduk. Resulting reads were mapped to the human genome (GRCh38) using BBmap with high precision but low sensitivity parameter. Unmapped reads were aligned against NCBI viral reference database (accessed 16/07/2021 at https://ftp.ncbi.nlm.nih.gov/refseq/release/viral/) using the blastn option in BLAST (evalue 10-5, best-hit overhang 0.1, best-hit score edge 0.1). BLAST output reads were uploaded to MEGAN Community Edition (28), and taxonomy assignment was performed using a modified lowest common ancestor (LCA) algorithm (min score 100, E-value <10-5, min percent identity 80, min support 1, weighted 80). Enrichment of viral sequences in metagenomes was quantified using viromeQC (29).\\\\nInformatic Processing of Clinical Pilot Study\\\\nRaw paired end.fastq files were quality filtered with fastp (30) employing a 5 nucleotide sliding window and a minimum qscore >25. Reads <100 nucleotides were discarded. Kneaddata within the biobakery environment (31) was used to remove human reads. Contigs were assembled from cleaned reads using the default kmer steps in Megahit (32). Contigs with <5x coverage were culled with bwa mem (33), samtools (34), bamtools (35) and bedtools (35). Virsorter2 (36) was used to identify dsDNA phage, nucelocytoplasmic large-DNA viral, RNA viral and ssDNA viral contigs from the >5x contigs and CheckV (37) was used to estimate the completeness of viral assembled genomes. Taxonomic classification of >5x coverage viral contigs was performed with Kraken2 (38) by alignment to pre-built viral databases (accessed 04/01/2022, at https://benlangmead.github.io/aws-indexes/k2). Per-sample abundance of each classified viral contig was calculated using Bracken (39).\\\\nStatistical Analyses\\\\nValidation of bacteriophage isolation experiments was based on the concentration of DNA in nucleic acid extracts as well as the recovered viral richness and Shannon diversity index. R studio was used for all statistical analysis (40). The taxonomic richness and Shannon diversity measures of alpha diversity were calculated with the vegan package (41). Richness was calculated at a rarefied feature count below the minimum sample level where possible. Comparisons of continuous variables were performed by the Kruskal-Wallis test. Where significant (P > 0.05) differences were identified across experiment parameters the pairwise Wilcoxon test was used to perform pairwise comparisons between classes. Count and categorical data were compared by the Mann-Whitney test. In cases of multiple hypothesis testing P-values were adjusted using the Bonferroni method. The phyloseq (42) and ggplot (43) packages were used for data manipulation and visualisation.\\\\nResults\\\\nValidation of Bacteriophage Isolation\\\\nViral Composition of DM Samples\\\\nA median of 2.99 x 105 (IQR 8.82 x 104 – 4.81 x 105) raw reads were generated from 78 pasteurised DM samples (Supplementary Table 1). A total of 119 viral genera belonging to 18 families were identified (Supplementary Table 2). Bacteriophages accounted for 75 % of the total taxonomic reads. Predominant bacteriophage families included Siphoviridae (48%), Myoviridae (19%), Podoviridae (6%) and Herelleviridae (1%). The assigned bacteriophages are mainly associated with bacterial hosts Escherichia (17%), Salmonella (12%), Enterococcus (12 %), Klebsiella (7%), Staphylococcus (3%), Pseudomonas (3%), Enterobacter (3%), Acinetobacter (1%), Proteus (1%), Erwinia (1%), Lactococcus (1%) Shigella (1%) and Stenotrophomonas (1%) based on sequence homology. Unclassified bacteriophages constituted a further 1% of total reads.\\\\nEffect of Sample Volume on DM Bacteriophage Communities\\\\nNo significant difference in extracted DNA concentration (ng/μL) (P = 0.96), genus richness (P = 0.91) or Shannon diversity (P = 0.86) was observed between DM sample volumes 0.1, 0.2, 0.4, and 0.6 mL (Figure 1).\\\\nFigure 1. Comparisons of DNA concentration, genus richness and Shannon diversity between different volumes of milk samples. Boxes represent the 1st and 3rd quartiles. Middle lines represent the median. Whiskers extend to the full range of data. Points represent individual samples.\\\\nCentrifugal Separation, Inclusion of Digestive Enzymes and Concentration/Purification of Viral Particles\\\\nNo significant differences in genus richness (P = 0.26) or Shannon diversity (P = 0.09) was observed in DM with increased centrifugation speed for microbial cell pelleting (Figure 2A). DNA concentration was significantly affected by centrifugation speed (P = 0.01) with significantly greater DNA concentrations observed in samples spun at 10 k xG (median 2.0 ng/μL) than at 15 k xG (median 0.35 ng/μL) (pairwise Wilcoxon: P = 0.02) (Figure 2A). Significantly lower Shannon diversity (P = 0.03) was observed in DM hydrolysed with digestive enzymes for lipid hydrolysis despite no difference in genus richness (P = 0.09) or DNA concentration (P = 0.54) (Figure 2B). The application of PEG8000 for concentration of isolated bacteriophages had no significant impact on DNA concentration (P = 0.32), genus richness (P = 0.92) or Shannon diversity (P = 0.92) of DM (Figure 2C). Concentrating and purifying bacteriophage isolates with Centriprep® 50 kDa filter devices had no significant impact on observed genus richness (P = 0.25), or Shannon diversity (P = 1.00) (Figure 2D), but did marginally increase DNA concentration (P = 0.06) (Figure 2D). Separation of milk lipids from whole DM yielded no significant change in observed genus richness (P = 0.50), shannon diversity (P = 0.50), or isolated DNA concentration (P = 0.93) (Figure 2E).\\\\nFigure 2. Comparisons of DNA concentration, genus richness and Shannon diversity between centrifugal spin speeds for size-based separation (A), addition of digestive enzymes for milk hydrolysis (B), concentration of bacteriophage particles with PEG8000 (C), filtration of bacteriophage with Centriprep devices (D), and utilization of different milk fractions (E).\\\\nFurther optimisation of commercially available nucleic acid extraction kits found comparable results for both QIAamp MinElute Virus Spin Kit (QIAGEN, Germany) and Norgen Phage DNA Isolation Kits (Norgen Biotek Corp, Canada). Both yielded comparable bacteriophage richness and diversity (Supplementary Table 4, Supplementary Figure 5) (Figure 3A).\\\\nFigure 3. A higher viral richness was accounted for in samples at which (A) Norgen kit was used for DNA extraction, (B) norfloxacin-mediated phage induction was included or (C) digestive enzymes were added to the milk fat. The numbers within Venn diagrams represent the number of assigned families.\\\\nNorfloxacin-mediated induction of bacteriophages enabled identification of Inoviridae, which were not seen in any non-induced samples. Although hydrolysis with digestive enzymes had no significant impact on alpha diversity indices or isolated DNA concentration, an improved recovery of lipid-associated viruses, enabling exclusive identification of Iridoviridae and Baculoviridae was observed in samples subjected to lipid hydrolysis (Figure 3B). Freezing the lipid layer before separation enabled identification of Baculoviridae and Inoviridae but not Iridoviridae, suggesting lipid hydrolysis may be essential to isolate this family.\\\\nUnlike the pasteurised DM used here for method validation, bio-banked BM samples do not undergo pasteurisation, which disrupts interactions between DM fat globule membranes. We found that when processing bio-banked BM samples an extra step of intense horizontal agitation for 10 min at room temperature as proposed by Brewster and Paul (44) caused lipid aggregation against the side of sampling tubes which made removal of BM lipids easier prior to digestive enzyme treatment.\\\\nThe absence of any impact on DNA concentration, bacteriophage richness or diversity of increasing DM sample input volume validates our optimised protocol for bacteriophage community composition from as little as 0.1 mL DM. We found hydrolysis of DM lipids with digestive enzymes, 10 K xG centrifugal separation at 4°C for 20 min and norfloxacin-mediated induction of temperate bacteriophages from the microbial cell pellets yields best results. We find no evidence for the inclusion of a freezing step prior to separation of DM lipids from the whey, nor preferential use of either QIAamp MinElute or Norgen Phage DNA isolation kits. Our optimised protocol is illustrated in Figure 4.\\\\nFigure 4. Optimized phage isolation protocol generated from the validation of different tested variables.\\\\nOptimized Method Application in Mother-Infant Pair Samples\\\\nTo demonstrate the utility of the validated isolation methods we applied it to a small cohort of 4 mother-infant pairs. All were inpatients at the RVI, NICU, Newcastle upon Tyne Hospitals, and received expressed MM feeds, supplemented with Labinic probiotics (Biofloratech Ltd., UK). One BM milk and one neonatal stool sample taken within 48 h of BM feed were analysed from each mother-infant pair. Clinical details of the cohort are provided in Table 1.\\\\nTable 1. Recorded characteristics of clinical cohort used in pilot study.\\\\nBM yielded greater raw and viral read count than neonatal stools however this did not translate to a greater number or length of assembled contigs with a minimum of 5 x coverage (Supplementary Table 3). Kit and sequencing negative control samples each yielded zero contigs.\\\\nSignificantly greater bacteriophage species richness (P = 0.04) and diversity (P = 0.04) were observed in neonatal stool samples than BM (Figure 5A). A total of 78 bacteriophage species belonging to 4 families were identified in stool with only 18 bacteriophage species from 4 families identified in BM (Figure 5B). Despite these differences, greater richness and diversity in maternal BM were often observed alongside greater richness and diversity in the paired neonatal stool samples (Figure 5A). Furthermore, we identify 6 bacteriophage species that were shared between BM and neonatal stools in two different mother-infant pairs (Figures 5C,D). All 6 species were putative Staphylococcus infecting bacteriophage, 4 of which belonged to the Siphoviridae and 2 to the Podoviridae families. Several bacteriophage species remain exclusively identifiable in either BM or neonatal stool (Figure 5D).\\\\nFigure 5. Bacteriophage community of mother-infant pair BM and stool samples. BM has lower species richness and diversity than stool (A) despite links between BM and neonatal stool richness and diversity as illustrated points, colored by mother-infant pair. Several bacteriophage taxonomic features are shared between BM and stool samples at the family level (B). Bacteriophage species composition of BM and neonatal stool is illustrated across all species observed via venn diagram (C) and in more detail as a heatmap containing only the top 25 most abundant & prevalent bacteriophage species (D). Greater relative abundance of species/families in heatmaps (B,D) is represented by greater color intensity [transformed on a log(2) scale].\\\\nDiscussion\\\\nCharacterising the mechanisms of action of bioactive components in BM is key to understanding its beneficial effects. To maximise the reliability and generalisability of studies investigating these components, it is important to validate laboratory processes utilized in large-scale surveys. Despite this, few studies exist exploring methods for the isolation of complex bacteriophage communities from human BM.\\\\nThis study represents a validation and optimisation of methods to reliably characterise the bacteriophage component of BM from mothers of very preterm infants by maximising the ability to observe the full diversity of bacteriophages within the sample while minimizing inherent biases, protocol cost and time. Specifically, we justify the use of very small sample volumes (as little as 0.1 mL) for very preterm BM phage community characterisation. Previous studies investigating the milk virome have utilized as much as 1.5–10 mL of BM (13–15). Reducing required sample volumes is of particular importance because preterm delivery can have a negative effect on lactogenesis and therefore salvageable volumes can be small to allow as much BM as possible to be available for the infant.\\\\nSimilar to previous studies which applied viral purification protocols using CsCl density gradient ultracentrifugation on term BM (14), we identify Mimiviridae (a giant group of viruses) in DM from our sequencing data that were not previously reported in BM studies using micron-sized membrane filters (16). Our proposed method omits the time-consuming CsCl density gradient purification which has previously been reported to have low reproducibility and is not well-suited for high-throughput studies (45, 46).\\\\nPEG has been used as an alternative to CsCl density gradients, proving to be a cost-effective way to substantially increase nucleic acid yields and give reproducible results compared to other methods for concentrating bacteriophages from 1 to 50 mL adult human stool samples (21, 45). Importantly we did not see differences between samples when isolating phage nucleic acids from DM with PEG or our proposed method. We also did not find any increased diversity or richness of phage genera using PEG concentration protocols. This is possible because more mature individuals have greater community diversity (47, 48), requiring substantially greater sample volumes to fully characterize the overall community structure. These same volumes would be impossible to salvage from very preterm infants and mothers. This is of note as the use of PEG concentrations to purify phage particles may introduce inherent biases by preferentially selecting for specific viral particle sizes or shapes, especially in complex or mixed bacteriophage communities (49).\\\\nOur protocol enabled us to identify 6 viral families (Adenoviridae, Baculoviridae, Herelleviridae, Papillomaviridae, Parvoviridae and Retroviridae), that were not previously observed in studies using micron-sized membrane filters to explore term BM viromes. The inclusion of these families, which may well have been filtered out during unnecessary pre-processing steps highlights the impact filtration can have on observable bacteriophage communities. Indeed, filtration has been shown to reduce viral particles isolated from stools by almost 50% (50).\\\\nPrevious studies of the viral composition of BM utilized isolation directly from whole or skimmed milk without upstream intervention (51–53). Association of viruses with milk lipids are often excluded in these studies. Our data reveal the presence of bacteriophages belonging to the Inoviridae, and eukaryotic viruses belonging to the Iridoviridae and Baculoviridae families in the lipid fraction upon hydrolysis using digestive enzymes. This indicates the presence of microbes (particularly bacteriophages) associated with milk fat globule membranes that may have been previously overlooked. Furthermore, our data suggest norfloxacin-mediated induction of lysogens present in the microbial cell pellets following centrifugal separation is essential to identify the full complement of temperate bacteriophages within communities. In addition, we demonstrate the increased speed of centrifugal separation does not improve characterisation of the temperate bacteriophage community.\\\\nMany previous studies have utilized multiple displacement amplification (MDA) to increase nucleic acid yields for sequencing. MDA has previously been reported to preferentially amplify ssDNA phages introducing bias to community compositions by artificially inflating their numbers (54–56). As such we did not include any MDA step prior to sequencing bacteriophage nucleic acids in this study but still achieve suitable nucleic acid concentrations for sequencing library preparation and identifying ssDNA phages from the Inoviridae family.\\\\nWe incorporate nucleases prior to nucleic acid isolation and include no RT step afterward which may overlook the role of any RNA viruses in this system (46), although similar to other gut virome studies, we still identify RNA viruses, such as members of the Retroviridae.\\\\nThis study highlights the importance of validating protocols specifically tailored to the sample type and matrices before conducting large-scale ecological surveys.\\\\nAfter validating a method for bacteriophage community characterisation in DM we applied this same protocol to BM and matched neonatal stool samples from mother-infant pairs highlighting six shared bacteriophage species between mother\\'s BM and infant stools. The six shared species were all putative Staphylococcus infecting bacteriophages belonging to the Siphoviridae (4) and Podoviridae (2) families. Previous work by this group has identified Staphylococcus as the dominant member of bacterial communities of preterm infant\\'s stool and mother\\'s BM during early life (8, 57). Other highly abundant bacteriophages identified in this study, include Escherichia, Enterobacteria and Klebsiella phages. Although the host specificity of these phages is purely based on sequencing data, rather than host-range studies these results align with those of Liang et al. (16) who identified bacteriophage colonizing infant guts in the immediate post-natal period to be mostly comprised of temperate phages induced from bacterial community members. Previous studies have yielded conflicting results regarding the vertical transmission of bacteriophages between mothers and infants (12–15). This study is the first to our knowledge to provide evidence for the potential of this phenomenon in very preterm infants who, importantly, receive much smaller feed volumes than term infants.\\\\nDespite being adopted in previous studies as a measure of viral recovery efficiency (58), our method validation data suggest DNA yield is a poor predictor of this. In two experiments the protocol yielding the greatest DNA concentration did not translate to greater bacteriophage richness or diversity. This is likely because despite multiple bacteriophage isolation and purification steps we still cannot confirm extracted nucleic acids are free from sources of other contaminating organisms. Alternative methods could spike known compositions of viral particles into sterile matrices, however, acquisition of sterile matrices with similar complexity to human milk proves extremely difficult. Our selection of pasteurized DM was initially intended to facilitate this, but our data demonstrate the persistence of bacteriophage nucleic acids despite pasteurisation. In addition, bacteriophage communities in DM and BM are not well studied so it is difficult to select bacteriophage species/strains for effective validation purposes (45).\\\\nThe persistence of bacteriophage nucleic acids post-pasteurisation of DM highlights the need for further work to discriminate between viable and non-viable bacteriophage communities associated with the very preterm infant GIT.\\\\nThis study developed an optimised protocol for the isolation of a broad range of bacteriophages from small volumes of BM samples. Using the validated bacteriophage isolation protocol, we have successfully identified viral communities in pasteurized DM, as well as in bio-banked preterm BM. Shared bacteriophage species between BM and stool samples from the same mother-infant pairs were also identified. These data will serve as baseline information toward future studies to develop greater understanding of how bacteriophages may be implicated in the development of the preterm gut microbiome.\\\\nData Availability Statement\\\\nThe original contributions presented in the study are publicly available. This data can be found here: https://www.ebi.ac.uk/ena, PRJEB49989.\\\\nEthics Statement\\\\nThe studies involving human participants were reviewed and approved by NRES Committee North East–Newcastle and North Tyneside 2 (10/H0908/39). Written informed consent to participate in this study was provided by the participants\\' legal guardian/next of kin.\\\\nAuthor Contributions\\\\nWY, AN, SB, and DS conceived the method validation study. WY performed laboratory and bioinformatic work for the validation study. GY and DS conceived the application in the mother-infant pairs study. GY performed laboratory and bioinformatic work for this. AN sequenced the libraries. JB and NE enrolled participants in the clinical study and arranged and carried out sampling. GY and WY wrote the manuscript. All authors reviewed and approved the manuscript before submission.\\\\nFunding\\\\nThis work was funded by Children\\'s Charity Action Medical Research on grant number GN2730.\\\\nConflict of Interest\\\\nNE and JB declare research funding paid to their employing institution from Prolacta Biosciences US, and Danone Early Life Nutrition, and both declare lecture honoraria from Nestle Nutrition Institute.\\\\nThe remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nPublisher\\'s Note\\\\nAll claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.\\\\nAcknowledgments\\\\nWe thank The Great North Neonatal Biobank, a collaboration between Newcastle University and Newcastle Upon Tyne Hospitals NHS Foundation Trust, for providing access to bio-banked human milk. We also thank the research nurses, clinical staff and all participant families at the Royal Victoria Infirmary, Newcastle upon Tyne, UK, for their help in collecting maternally expressed milk and preterm infant stool samples. DM was from Prolacta Biosciences and milk was provided for a separate research study where the milk was beyond the expiry date but remained frozen. Prolacta Bioscience had no involvement in this study.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fped.2022.856520/full#supplementary-material\\\\nReferences\\\\n1. Victora CG, Bahl R, Barros AJD, França GVA, Horton S, Krasevec J, et al. Breastfeeding in the 21st century: epidemiology, mechanisms, and lifelong effect. Lancet. (2016) 387:475–90. doi: 10.1016/S0140-6736(15)01024-7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n2. Royal College of Paediatrics and Child Health. Breastfeeding in the UK - Position Statement. (2017). p. 1–10. \\\\nGoogle Scholar\\\\n4. Granger CL, Embleton ND, Palmer JM, Lamb CA, Berrington JE, Stewart CJ. Maternal breastmilk, infant gut microbiome and the impact on preterm infant health. Acta Paediatr. (2020) 110:450–7. doi: 10.1111/apa.15534\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n5. Masi AC, Embleton ND, Lamb CA, Young G, Granger CL, Najera J, et al. Human milk oligosaccharide DSLNT and gut microbiome in preterm infants predicts necrotising enterocolitis. Gut. (2021) 70:2273–82. doi: 10.1136/gutjnl-2020-322771\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n7. Embleton N, Berrington J, Cummings S, Dorling J, Ewer A, Frau A, et al. Lactoferrin impact on gut microbiota in preterm infants with late-onset sepsis or necrotising enterocolitis: the MAGPIE mechanisms of action study. Effic Mech Evaluat. (2021) 8:14. doi: 10.3310/eme08140\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n8. Young GR, van der Gast CJ, Smith DL, Berrington JE, Embleton ND, Lanyon C. Acquisition and development of the extremely preterm infant microbiota across multiple anatomical sites. J Pediatr Gastroenterol Nutr. (2020) 70:12–9. doi: 10.1097/MPG.0000000000002549\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n10. Cabrera-Rubio R, Collado MC, Laitinen K, Salminen S, Isolauri E, Mira A. The human milk microbiome changes over lactation and is shaped by maternal weight and mode of delivery. Am J Clin Nutr. (2012) 96:544–51. doi: 10.3945/ajcn.112.037382\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n11. Berrington JE, Stewart CJ, Embleton ND, Cummings SP. Gut microbiota in preterm infants: assessment and relevance to health and disease. Arch Dis Childhood-Fetal Neonatal Ed. (2013) 98:F286–90. doi: 10.1136/archdischild-2012-302134\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n12. Breitbart M, Haynes M, Kelley S, Angly F, Edwards RA, Felts B, et al. Viral diversity and dynamics in an infant gut. Res Microbiol. (2008) 159:367–73. doi: 10.1016/j.resmic.2008.04.006\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n13. Duranti S, Lugli GA, Mancabelli L, Armanini F, Turroni F, James K, et al. Maternal inheritance of bifidobacterial communities and bifidophages in infants through vertical transmission. Microbiome. (2017) 5:66. doi: 10.1186/s40168-017-0282-6\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n14. Pannaraj PS, Ly M, Cerini C, Saavedra M, Aldrovandi GM, Saboory AA, et al. Shared and distinct features of human milk and infant stool viromes. Front Microbiol. (2018) 9:1162. doi: 10.3389/fmicb.2018.01162\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n15. Dinleyici M, Pérez-Brocal V, Arslanoglu S, Aydemir O, Sevuk Ozumut S, Tekin N, et al. Human milk virome analysis: changing pattern regarding mode of delivery, birth weight, and lactational stage. Nutr. (2021) 13:1779. doi: 10.3390/nu13061779\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n16. Liang G, Zhao C, Zhang H, Mattei L, Sherrill-Mix S, Bittinger K, et al. The stepwise assembly of the neonatal virome is modulated by breastfeeding. Nature. (2020) 581:470–4. doi: 10.1038/s41586-020-2192-1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n18. Hamprecht K, Vochem M, Baumeister A, Boniek M, Speer CP, Jahn G. Detection of cytomegaloviral DNA in human milk cells and cell free milk whey by nested PCR1Parts of this study were presented at the 21ST herpesvirus workshop, Northern Illinois University, Dekalb, Illinois, USA, 1996.1. J Virol Methods. (1998) 70:167–76. doi: 10.1016/S0166-0934(97)00179-1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n19. Tariq MA, Everest FLC, Cowley LA, De Soyza AD, Holt GS, Bridge SH, et al. A metagenomic approach to characterize temperate bacteriophage populations from cystic fibrosis and non-cystic fibrosis bronchiectasis patients. Front Microbiol. (2015) 6:97. doi: 10.3389/fmicb.2015.00097\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n20. Luo J, Wang Z, Li Y, Chen C, Ren F, Guo H. The simulated in vitro infant gastrointestinal digestion of droplets covered with milk fat globule membrane polar lipids concentrate. J Dairy Sci. (2019) 102:2879–89. doi: 10.3168/jds.2018-15044\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n21. Castro-Mejía JL, Muhammed MK, Kot W, Neve H, Franz CMAP, Hansen LH, et al. Optimizing protocols for extraction of bacteriophages prior to metagenomic analyses of phage communities in the human gut. Microbiome. (2015) 3:64. doi: 10.1186/s40168-015-0131-4\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n22. Rasmussen TS, de Vries L, Kot W, Hansen LH, Castro-Mejía JL, Vogensen FK, et al. Mouse vendor influence on the bacterial and viral gut composition exceeds the effect of diet. Viruses. (2019) 11:435. doi: 10.3390/v11050435\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n23. Young GR, Smith DL, Embleton ND, Berrington JE, Schwalbe EC, Cummings SP, et al. Reducing viability bias in analysis of gut microbiota in preterm infants at risk of NEC and sepsis. Front Cell Infect Microbiol. (2017) 7:237.doi: 10.3389/fcimb.2017.00237\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n24. Andrews S. FastQC: A Quality Control Tool For High Throughput Sequence Data (2010). \\\\nGoogle Scholar\\\\n26. Afgan E, Baker D, van den Beek M, Blankenberg D, Bouvier D, Cech M, et al. The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update. Nucleic Acids Res. (2016) 44:W3–W10. doi: 10.1093/nar/gkw343\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n29. Zolfo M, Pinto F, Asnicar F, Manghi P, Tett A, Bushman FD, et al. Detecting contamination in viromes using viromeQC. Nat Biotechnol. (2019) 37:1408–12. doi: 10.1038/s41587-019-0334-5\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n31. McIver LJ, Abu-Ali G, Franzosa EA, Schwager R, Morgan XC, Waldron L, et al. bioBakery: a meta\\'omic analysis environment. Bioinformatics. (2018) 34:1235–7. doi: 10.1093/bioinformatics/btx754\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n32. Li D, Liu C-M, Luo R, Sadakane K, Lam T-W. MEGAHIT an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. Bioinformatics. (2015) 31:1674–6. doi: 10.1093/bioinformatics/btv033\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n34. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The Sequence alignment/map format and SAMtools. Bioinformatics. (2009) 25:2078–9. doi: 10.1093/bioinformatics/btp352\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n36. Guo J, Bolduc B, Zayed AA, Varsani A, Dominguez-Huerta G, Delmont TO, et al. VirSorter2: a multi-classifier, expert-guided approach to detect diverse DNA and RNA viruses. Microbiome. (2021) 9:37. doi: 10.1186/s40168-020-00990-y\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n37. Nayfach S, Camargo AP, Schulz F, Eloe-Fadrosh E, Roux S, Kyrpides NC. CheckV assesses the quality and completeness of metagenome-assembled viral genomes. Nat Biotechnol. (2021) 39:578–85. doi: 10.1038/s41587-020-00774-7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n39. Lu J, Breitwieser FP, Thielen P, Salzberg SL. Bracken: estimating species abundance in metagenomics data. PeerJ Comput Sci. (2017) 3:e104. doi: 10.7717/peerj-cs.104 \\\\nCrossRef Full Text | Google Scholar\\\\n42. McMurdie PJ, Holmes S. Phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. PLoS ONE. (2013) 8:e61217. doi: 10.1371/journal.pone.0061217\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n44. Brewster JD, Paul M. Short communication: Improved method for centrifugal recovery of bacteria from raw milk applied to sensitive real-time quantitative PCR detection of Salmonella spp1. J Dairy Sci. (2016) 99:3375–9. doi: 10.3168/jds.2015-9655\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n45. d\\'Humières C, Touchon M, Dion S, Cury J, Ghozlane A, Garcia-Garcera M, et al. A simple, reproducible and cost-effective procedure to analyse gut phageome: from phage isolation to bioinformatic approach. Sci Rep. (2019) 9:11331. doi: 10.1038/s41598-019-47656-w\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n46. Callanan J, Stockdale SR, Shkoporov A, Draper LA, Ross RP, Hill C. Biases in Viral metagenomics-based detection, cataloguing and quantification of bacteriophage genomes in human faeces, a review. Microorg. (2021) 9:524. doi: 10.3390/microorganisms9030524\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n47. Stewart CJ, Ajami NJ, O\\'Brien JL, Hutchinson DS, Smith DP, Wong MC, et al. Temporal development of the gut microbiome in early childhood from the TEDDY study. Nature. (2018) 562:583–8. doi: 10.1038/s41586-018-0617-x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n48. Yatsunenko T, Rey FE, Manary MJ, Trehan I, Dominguez-Bello MG, Contreras M, et al. Human gut microbiome viewed across age and geography. Nature. (2012) 486:222–7. doi: 10.1038/nature11053\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n49. Carroll-Portillo A, Coffman CN, Varga MG, Alcock J, Singh SB, Lin HC. Standard bacteriophage purification procedures cause loss in numbers and activity. Viruses. (2021) 13:328. doi: 10.3390/v13020328\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n50. Hoyles L, McCartney AL, Neve H, Gibson GR, Sanderson JD, Heller KJ, et al. Characterization of virus-like particles associated with the human faecal and caecal microbiota. Res Microbiol. (2014) 165:803–12. doi: 10.1016/j.resmic.2014.10.006\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n51. Lazar K, Rabe T, Goelz R, Hamprecht K. Human cytomegalovirus reactivation during lactation: impact of antibody kinetics and neutralization in blood and breast milk. Nutrients. (2020) 12:338. doi: 10.3390/nu12020338\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n52. Drago L, Toscano M, De Grandi R, Grossi E, Padovani EM, Peroni DG. Microbiota network and mathematic microbe mutualism in colostrum and mature milk collected in two different geographic areas: Italy versus Burundi. ISME J. (2017) 11:875–84. doi: 10.1038/ismej.2016.183\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n54. Kyoung-Ho K, Jin-Woo B. Amplification methods bias metagenomic libraries of uncultured single-stranded and double-stranded DNA viruses. Appl Environ Microbiol. (2011) 77:7663–8. doi: 10.1128/AEM.00289-11\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n55. DŽunková M, Garcia-Garcerà M, Martínez-Priego L, D\\'Auria G, Calafell F, Moya A. Direct Sequencing from the minimal number of DNA molecules needed to fill a 454 picotiterplate. PLoS ONE. (2014) 9:e97379. doi: 10.1371/journal.pone.0097379\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n56. Yilmaz S, Allgaier M, Hugenholtz P. Multiple displacement amplification compromises quantitative analysis of metagenomes. Nat Methods. (2010) 7:943–4. doi: 10.1038/nmeth1210-943\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n57. Stewart CJ, Embleton ND, Marrs ECL, Smith DP, Fofanova T, Nelson A, et al. Longitudinal development of the gut microbiome and metabolome in preterm neonates with late onset sepsis and healthy controls. Microbiome. (2017) 5:75. doi: 10.1186/s40168-017-0295-1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n58. Bergallo M, Costa C, Gribaudo G, Tarallo S, Baro S, Negro Ponzi A, et al. Evaluation of six methods for extraction and purification of viral DNA from urine and serum samples. New Microbiol. (2006) 29:111–9.\\\\nPubMed Abstract | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fcimb.2022.1060825\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fcimb.2022.1060825/full\",\\n    \"loadedTime\": \"2024-01-23T08:49:38.136Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fcimb.2022.1060825\",\\n    \"title\": \"Frontiers | Bacteriophage-based decontamination to control environmental colonization by Staphylococcus capitis in neonatal intensive care units: An in vitro proof-of-concept\",\\n    \"description\": \"In neonatal intensive care units (NICUs), the standard chemical-based disinfection procedures do not allow a complete eradication of pathogens from environmental surfaces. In particular, the clone Staphylococcus capitis NRCS-A, a significant pathogen in neonates, was shown to colonize neonatal incubators. The aim of this study was to evaluate the in vitro effect of a bacteriophage cocktail on NRCS-A eradication. Three bacteriophages were isolated, genetically characterized and assessed for their host range using a collection of representative clinical strains (n=31) belonging to the clone NRCS-A. The efficacy of a cocktail including these three bacteriophages to eradicate the reference strain S. capitis NRCS-A CR01 was determined in comparison or in combination with the chemical disinfectant Surfanios Premium on either dry inoculum or biofilm-embedded bacteria. The emergence of bacterial resistance against the bacteriophages alone or in cocktail was evaluated by growth kinetics. The three bacteriophages belonged to two families and genera, namely Herelleviridae/Kayvirus for V1SC01 and V1SC04 and Rountreeviridae/Andhravirus for V1SC05. They were active against 17, 25 and 16 of the 31 tested strains respectively. Bacteriophage cocktails decreased the bacterial inoculum of both dry spots and biofilms, with a dose dependent effect. The sequential treatment with bacteriophages then Surfanios Premium did not show enhanced efficacy. No bacterial resistance was observed when using the bacteriophage cocktail. This study established a proof-of-concept for the use of bacteriophages to fight against S. capitis NRCS-A. Further investigations are needed using a larger bacterial collection and in real-life conditions before being able to use such technology in NICUs.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nNosocomial infections in hospitalized patients are a well-known cause of mortality and morbidity all over the world (Sikora and Zahra, 2022). The prevalence of such infections is high, representing 6.5% of the patients in the European Economic Area and 3.2% patients in the United States (Magill et al., 2018; Suetens et al., 2018). Multidrug resistant bacteria from the hospital environment are frequently involved in nosocomial infections (Suleyman et al., 2018). Among the various hospital units, neonatal intensive care units (NICUs) are at high risk of nosocomial infections since preterm neonates are highly vulnerable given their global immaturity (Collins et al., 2018). Despite the strict hygiene measures implemented in NICUs, several studies have incriminated numerous pathogens in those infections including staphylococci, enterobacteria or enterococci originating from the environmental surfaces and notably from the incubators where preterm newborns are housed (Golan et al., 2005; Lin et al., 2017; Carter et al., 2018; Butin et al., 2019; Cadot et al., 2019; Ory et al., 2019; Chavignon et al., 2021).\\\\nThe clone Staphylococcus capitis NRCS-A is one of these pathogens. Its high prevalence in sepsis among low-birth weight preterm neonates in NICUs worldwide, its multidrug resistant profile and its high adaptation to the NICU environment have been reported (Butin et al., 2017; Wirth et al., 2020). This specialization has been related to its resistance pattern to antibiotics commonly used in NICUs including vancomycin or aminoglycosides (Rasigade et al., 2012; Wirth et al., 2020). Moreover, previous studies have highlighted the ability of NRCS-A to colonize inert surfaces, notably neonatal incubators, and to persist despite the usual chemical disinfection procedures fostering its continuous spread and dissemination after its initial introduction in a given NICU (Carter et al., 2018; Butin et al., 2019; Chavignon et al., 2021). Finally, the ability of S. capitis strains isolated from neonates or from the NICU environment to form biofilm has been highlighted and might explain, at least for a part, the ability of the strains to colonize abiotic surfaces (Cui et al., 2013; Carter et al., 2018; Qu et al., 2020). In this context, there is a need to develop alternative disinfection approaches to reduce the risk of environmental persistence of S. capitis and inter-patient transmission. One innovative approach could be the setup of bacteriophage-based decontamination. Bacteriophages are ubiquitous viruses able to specifically infect prokaryotic bacterial cells with a total innocuity for eucaryotic cells and so for humans (D’Accolti et al., 2021). Thus, lytic virulent bacteriophages present a bactericidal activity with a high target specificity at the species or infra-species level and constitute useful tools for biological control. The use of bacteriophages has already been considered as an alternative or a complementary method of decontamination in different contexts (D’Accolti et al., 2021). In food-processing industries, the use of bacteriophages as additive to control the contamination of food production by pathogenic bacteria has been already approved by the Food and Drug Administration (https://www.cfsanappsexternal.fda.gov/scripts/fdcc/?set=GRASNotices&sort=GRN_No&order=DESC&startrow=1&type=basic&search=phages, retrieved 9 August 2022). Bacteriophages have also been considered in decontamination of inanimate surfaces in the hospital environment. For instance, the use of an aerosolized bacteriophage suspension after the standard cleaning procedure (based on sodium hypochlorite 0.06% and alcohol 75%) in an intensive care unit in Taiwan has been reported to be able to decrease the rate of nosocomial infections caused by carbapenem-resistant Acinetobacter baumannii (Ho et al., 2016). In another study, D’Accolti et al. demonstrated the feasibility and effectiveness of hospital bathrooms nebulization with bacteriophages in addition to a probiotic-based sanitation to fight against Staphylococcus spp. contamination and nosocomial infections. Interestingly, daily nebulization of bacteriophages allowed a significant and rapid decrease of Staphylococcus on the surfaces that was up to 97% more effective than probiotic-based sanitation alone (D’Accolti et al., 2019).\\\\nDue to the difficulty to eradicate the clone S. capitis NRCS-A after its initial introduction in a NICU and to face its persistence, an in vitro preliminary study was conducted to determine if a bacteriophage-based decontamination could be an efficient approach. The aim of the present study was thus i) to isolate lytic bacteriophages presenting a bactericidal activity on clinical strains belonging to the clone S. capitis NRCS-A, ii) to explore the emergence of resistance against those bacteriophages, and finally iii) to evaluate the ability of those bacteriophages to impact biofilms or planktonic adhered bacteria in comparison with the usual disinfection molecule.\\\\nMaterials and methods\\\\nBacterial strains\\\\nThe Staphylococcus capitis strains used in this study were obtained from the collection of the French National Reference Centre for Staphylococci (Lyon, France). These strains were conserved in cryotubes containing glycerol at -20°C. They were grown on Columbia agar + 5% sheep blood (BioMérieux, Marcy l’Etoile, France) for 24 hours at 37°C. A collection of 31 strains was included in this study to isolate, produce and assess the host range of anti-S. capitis bacteriophages. These strains were selected to reflect the phylogenetic diversity of the S. capitis NRCS-A clone composed of three subgroups successively appeared: Proto-outbreak I, Proto-outbreak II and the most recent and specialized in NICU infections Outbreak (Table 1) (Wirth et al., 2020). The reference strain of the clone S. capitis NRCS-A, called CR01 (Lemriss et al., 2014) has been used to evaluate the effect of bacteriophages on dry spots and biofilms.\\\\nTable 1 Host range characterization of five anti-Staphylococcus capitis bacteriophages.\\\\nBacteriophages isolation and production\\\\nThree bacteriophages were isolated for the purpose of this study from three different wastewater samples in Lyon, France in 2020. Briefly, 5 mL of filtered water sample was incubated during 24 h at 37°C under agitation (180 rpm) with 500 µL of Tryptic Soy Broth (TSB) 10X (BD, Franklin Lakes, NJ, USA) and 10 µL of an overnight culture of the strain P2SC02, chosen for the intensity of the lysis produced by the bacteriophages when applied to it. The culture supernatant was filtered using a 0.22-µm syringe filter and diluted in double layer agars pouring a mix of 100 µL of this supernatant, 250 µL of P2SC02 culture and 5 mL of medium containing 0.75% of agar (called TSB soft-agar) over a Tryptic Soy Agar plate (BioMérieux, Marcy l’Etoile, France). Individual plaque forming units (PFU) were further purified by 5 rounds of serial passages and where required propagated in liquid medium. Strain P2SC16 was then used for bacteriophage production as amplification yields were higher with this strain. Bacteria were cultured in 50 mL of TSB in exponential phase then inoculated with bacteriophages to obtain a Multiplicity of Infection (MOI) of 10-3. Obtained bacteriophage lysates were filtered and stored at 4°C.\\\\nBacteriophage genome sequencing and bio-informatic analysis\\\\nAfter isolation, a volume of 6 mL of each bacteriophage was centrifuged at 14,000 g for 5 h and pellets were re-suspended in 50 µL of NaCl 0.9%. Enzymatic treatment with 100 mU of benzonase® nuclease (Merk, Darmstadt, Germany) at 37°C overnight was then used to degrade extracellular bacterial DNA, followed by benzonase heat-inactivation at 95°C for 30 min, a treatment with 4 µg of proteinase K (Merk, Darmstadt, Germany), and a proteinase K heat-inactivation at 95°C for 30 min. Bacteriophage DNA was extracted using the DNA Extractor® WB kit (Fujifilm, Osaka, Japan) and sequenced on an Illumina NextSeq 500 instrument using a 150-bp paired-end protocol. Reads were trimmed (cutadapt, v3.4; trimmomatic, v0.39) and the reads mapping against the genome of the P2SC16 strain used for the amplification of bacteriophages (bowtie2, v2.3.4.1; samtools, v1.15) were removed. The remaining reads were assembled (SPAdes, v3.13.0) and scaffolds smaller than 100 bp were removed. Taxonomic assignation was performed using kraken 1.1.1 with the minikraken database. Genomes were annotated using PATRIC (v3.6.12) with parameters for bacteriophage annotation. Finally, Abricate (v0.8.13) was used for resistance and virulence gene detection using all the databases available. The lytic nature of bacteriophages was assessed using the Phage AI repository (https://app.phage.ai/phages/). Bacteriophage genomes were deposited on the NCBI GenBank under the accession numbers OP323059, OP297178 and OP297179 for vB_ScaM-V1SC01, vB_ScaM-V1SC04, and vB_ScaM-V1SC05, respectively, which are named V1SC01, V1SC04, and V1SC05 in this study for simplification.\\\\nHost range assessment\\\\nHost range of each bacteriophage was assessed using the spot test assay on the panel of 31 S. capitis strains, selected as explained above (Table 1). Briefly, 5 µL of serial ten-fold dilutions of bacteriophages were spotted on an agar plate prepared extemporaneously by mixing 30 mL of TSB soft agar and 500 µL of S. capitis overnight culture in TSB broth. After overnight incubation at 37°C, PFU were enumerated. The efficiency of plating ratio (EOP) was calculated dividing the bacteriophage titer obtained on a test strain by the titer of the same bacteriophage suspension on the reference production strain P2SC16. A bacteriophage was considered active if EOP was ≥ 0.001 (Green et al., 2017). Experiments were performed in biological triplicates and mean EOP scores were calculated.\\\\nEvaluation of bacterial resistance emergence during bacteriophage treatment\\\\nGrowth kinetics were performed to determine if bacterial resistance occurred following the treatment with one bacteriophage alone compared to cocktails of the three bacteriophages. Briefly, CR01 was incubated in TSB until the exponential growth phase and then diluted in TSB to reach a concentration of 1.107 CFU/mL. A 96 well microplate (Thermo Fisher Scientific, Waltham, MA, USA) was then inoculated with 1.106 CFU of bacteria and either 1.106 PFU, 1.107 PFU or 1.108 PFU of each bacteriophage alone or in cocktail (to reach a MOI of respectively 1, 10 or 100) in a final TSB volume of 200 µL. Bacterial growth alone without bacteriophage addition and absence of bacterial contamination in bacteriophage suspensions and in TSB were also controlled in the microplate. The outlines of the microplate were filled with water to prevent evaporation then the microplate was incubated 24 h in a microplate reader Tecan infinite® 200 PRO (Tecan, Männedorf, Swiss) under agitation at 37°C with OD600 nm (optical density) measurement every 30 min. Emergence of bacterial resistance was considered when the OD600 nm exceeded the threshold of 0.01 (after TSB control media OD600 nm subtraction) that corresponded to the threshold of bacterial growth. The experiments were performed three times in technical triplicates and both bacteriophage titres and bacterial inoculum were verified each time.\\\\nActivity of a bacteriophage cocktail on dry spots and pre-formed biofilm of S. capitis NRCS-A\\\\nThe effect of a bacteriophage cocktail was tested on the reference strain CR01 in comparison with the disinfectant Surfanios Premium (SP) (composed of N-(3-aminopropyl)-N-dodécylpropane-1,3-diamine and chlorure de didécyldiméthylammonium) (ANIOS, Lezennes, France). Both treatments were tested on dry spots and on pre-formed biofilms using a previously published method with some modifications (Stachler et al., 2021). For bacteriophage cocktails preparation, the three bacteriophages were associated by diluting each of them in equal measure in either TSB or Phosphate Buffered Saline (PBS) (Thermo Fisher Scientific, Waltham, MA, USA) to obtain cocktails at three different final concentrations (3.104 PFU/mL, 3.106 PFU/mL or 3.108 PFU/mL). Bacteria were exposed to the bacteriophage cocktail for 6 h in wet chamber at 37°C. The chemical treatment consisted in exposition to SP at 0.25% in water at room temperature for 20 minutes, corresponding to the classical treatment used in NICUs for incubator disinfection (Butin et al., 2019). Untreated positive controls were treated with either TSB or PBS (controls for bacteriophage treatments) or water (control for SP treatment) alone.\\\\nDry spots were obtained by inoculating 10 µL drops of a 1.107 CFU/mL bacterial PBS suspension in a 96 well microplate (Greiner Bio-One, Cap Horn, France). The drops were dried at room temperature during 4.5 h under microbiological safety workstation. The dry spots treatment consisted of covering the adhered bacteria with 20 µL of bacteriophages cocktail, SP or control medium. Then, 100 µL of Dey Engley Neutralizing broth (Merk, Darmstadt, Germany) was applied during 5 min to inactivate the SP action. This same neutralising treatment was applied to dry spots exposed to bacteriophages to avoid experimental variations. Finally, the neutralising broth was discarded and the surviving bacteria were suspended in 100 µL of PBS by scraping and enumerated.\\\\nBiofilm was obtained by inoculating 100 µL of a 1.107 CFU/mL bacterial suspension in TSB in a 96 well microplate that was incubated 24 h at 37°C. After incubation, the biofilm was first rinsed using the steam method as previously described (Tasse et al., 2018) before being exposed to 150 µL of bacteriophage cocktails or SP 0.25% as described above. After incubation, the supernatant was carefully removed and 200 µL of Dey Engley Neutralizing broth was applied for 5 minutes. The biofilm was then rinsed with the steam method, suspended in 200 µL of PBS by scraping and enumerated.\\\\nIn addition, a sequential treatment with a bacteriophage cocktail at the highest bacteriophages concentration (3.108 PFU/mL) followed by a SP treatment was also evaluated on bacterial biofilm. For this purpose, after the step of bacteriophage treatment as described above, an additional steam rinse was performed and then SP was applied followed by inactivation as previously described.\\\\nLog10 reduction values (LRV) for each treatment were calculated as follows: Log10(N/N0) with N corresponding to the number of bacteria in the treated well and N0 corresponding to the number of bacteria in the untreated control well. The experiments were performed three times in triplicate for the simple treatments and three times in sextuplicate for the sequential treatment. Bacteriophage cocktails titres were checked each time.\\\\nGraphic representation and statistical analyses\\\\nThe graphics were drawn on GraphPad Prism 8 (La Jolla, CA, USA) using the scatter dot plot representation with the mean and the standard deviation displayed for each condition. Statistical analyses were performed in GraphPad Prism 8 on the LRV values. Because of the small sample size in the different experiments (experiments performed three times in triplicate or in sextuplicate), the Mann-Whitney nonparametric test was performed, with an α risk of 0.05.\\\\nResults\\\\nBacteriophages characteristics and host range against S. capitis NRCSA\\\\nThe three bacteriophages belonged to two different families and genera, namely Herelleviridae/Kayvirus for V1SC01 and V1SC04 or Rountreeviridae/Andhravirus for V1SC05. VISC04 had the widest host range as it was active against 25 out of the 31 tested strains while V1SC01 and V1SC05 were active against 17 and 16 of the strains respectively (Table 1). Of note, activities of bacteriophages were complementary: 28 out of the 31 strains were susceptible to at least one bacteriophage. However, bacteriophage activity depended on the NRCS-A subgroup: all strains of the Outbreak and Proto-outbreak I groups were susceptible to 1 to 3 bacteriophages, while 3 out of the 6 Proto-outbreak II strains were resistant to all bacteriophages.\\\\nEmergence of resistances against bacteriophages\\\\nAfter 24 h of growth in presence of bacteriophages, growth curves indicated that CR01 did not develop resistance against the bacteriophage V1SC01 while resistance to V1SC04 at MOI 1 and MOI 100 and to V1SC05 at MOI 1, MOI 10 and MOI 100 were detected (Table 2). The emergence of these resistances revealed by bacterial regrowth always occurred in the late phase of the experiment (18 h to 24 h). Interestingly, no resistance was observed whatever the MOI when bacteriophage cocktail of the three bacteriophages was tested.\\\\nTable 2 Development of bacterial resistance against bacteriophages using bacteriophage alone or in cocktail.\\\\nBacteriophage cocktail activity compared to Surfanios Premium on bacterial dry spots\\\\nWhile the SP 0.25% treatment was able to completely eradicate bacteria on the dry spots in the tested conditions, bacteriophage cocktail treatment significantly decreased the bacterial inoculum on dry spots both in TSB and in PBS at 3.106 PFU/mL (1 LRV (p < 0.0001) and 0,5 LRV (p <0.0001) respectively) and at 3.108 PFU/mL (4.7 LRV (p < 0.0001) and 2.4 LRV (p < 0.0001) respectively) but not at 3.104 PFU/mL (Figure 1A).\\\\nFigure 1 Effect of a bacteriophage cocktail and Surfanios Premium on dry spots and pre-formed biofilms of S. capitis NRCS-A. Results are presented as Log10 reduction values (LRV) calculated as Log10(N/N0) with N the remaining inoculum after treatment and N0 the remaining inoculum after treatment with diluent (TSB or PBS for treatment with bacteriophages and water for treatment with SP). The treatments consisted either of i) bacteriophage cocktails at three concentrations (3.104 PFU/mL, 3.106 PFU/mL or 3.108 PFU/mL) in TSB or in PBS, ii) Surfanios Premium (SP) at 0.25% in water or iii) a sequential treatment with the bacteriophage cocktail at the highest concentration (3.108 PFU/mL) in TSB followed by SP treatment. (A) Effect of the bacteriophage cocktails or SP individually on bacterial dry spots. (B) Effect of the bacteriophage cocktails or SP individually or sequentially on pre-formed biofilms.\\\\nBacteriophage cocktail activity compared to SP and sequential treatment on biofilm\\\\nBacteriophage cocktail showed a bactericidal effect on biofilm only when incubated with TSB (Figure 1B). This bactericidal effect was significantly higher at 3.106 PFU/mL and 3.108 PFU/mL than at 3.104 PFU/mL with 1.4 LRV and 1.8 LRV respectively versus 0.5 LRV (p = 0.0005 and p < 0.0001 respectively). The efficacy of SP 0.25% was not significantly different to that of bacteriophage cocktail at 3.108 PFU/mL with 1.9 LRV (p = 0.6314).\\\\nThe sequential treatment, consisting in a first exposition to bacteriophage cocktail at 3.108 PFU/mL in TSB followed by an exposition to SP 0.25%, showed a 1.8 LRV (Figure 1B). This was not significantly different from the effect of either SP 0.25% alone or the bacteriophage cocktail alone at the highest concentration (p = 0.6959 and p = 0.6314 respectively).\\\\nDiscussion\\\\nThe present study highlights the ability of a cocktail of three lytic bacteriophages to significantly reduce the bacterial inoculum of S. capitis NRCS-A on dry spots or in biofilm. Our data highlights that no emergence of resistance to the bacteriophage cocktail occurred during exposition of S. capitis NRCS-A. The sequential exposition to bacteriophages then to chemical disinfectant did not improve the final bactericidal effect observed.\\\\nBacteriophages have to be considered in the perspective of environmental decontamination because they present advantages in comparison to chemical disinfectants. First, in contrast to bacteriophages, chemical products are generally toxic both for humans and the environment (Dhama et al., 2021). Thus, quaternary ammonium compounds (QACs), that are components of SP, have been previously shown to be harmful for frequently exposed healthcare workers by causing allergic contact dermatitis and asthma (Purohit et al., 2000; Suneja and Belsito, 2008). QACs exposition has also been shown to increase secretion of inflammatory cytokines, alter mitochondrial function, and disrupt cholesterol homeostasis in humans (Hrubec et al., 2021). All these features are of concerns, all the more when such disinfectants are used to clean incubators housing very low-birth-weight preterm neonates. Secondly, conventional disinfectants have a corrosive effect on abiotic surfaces (Song et al., 2019). Such damages may induce porosity on equipment surfaces which fosters biofilm formation and so might favor persistence of some pathogenic bacteria. In contrast, bacteriophages are usually presented as having no impact on human cells and so to be safe for human health in addition to be rapidly cleared from the human body, as previously shown in reports of bacteriophage therapy (Wright et al., 2009). Furthermore, due to their narrow spectrum of antibacterial activity, it seems that bacteriophages do not damage the beneficial microbiota and limit the selection of multidrug resistance (D’Accolti et al., 2021). However, it should be noted that despite these general statements, recent studies question the real neutrality of bacteriophages for eukaryotic organisms. Notably, due to their nature of protein-nucleic particle, bacteriophages were suspected to activate the immune system resulting in the production of anti-bacteriophages antibodies or the production of pro-inflammatory or anti-inflammatory cytokines (Dąbrowska et al., 2014; Yıldızlı et al., 2020; Podlacha et al., 2021). In addition, bacteriophages are able of transcytosis across confluent epithelial cell layers but also to interact with eukaryotic proteins, in particular protein misfolding and prion formation (Nguyen et al., 2017; Tetz and Tetz, 2017; Tetz and Tetz, 2018). Moreover, recent works suggested that bacteriophages could alter the normal microbiota, since Tetz et al. reported a high abundance of lytic Lactococcus bacteriophages associated with a reduction of Lactococcus in the microbiota of patients with Parkinson disease (Tetz and Tetz, 2018). For these reasons, in a perspective of using these bacteriophages for decontamination in a NICU, it would be necessary to carefully assess the safety of products containing bacteriophages.\\\\nThe present study has been motivated by the need to explore innovative procedures for disinfection in NICU settings. The aim was to determine if bacteriophages could be used to fight against S. capitis NRCS-A environmental spread/persistence in NICUs. Our data documented a bactericidal activity of bacteriophages on both bacterial dry spots and biofilms. This action was dose dependant, as reported in previous studies (Zhang and Hu, 2013; Stachler et al., 2021). For instance, the bacterial decrease represented 4.7 LRV and 1.8 LRV respectively on dry spots and biofilms at 3.108 PFU/mL in TSB while no bacterial loss on dry spots and only 0.47 LRV on biofilms was observed at 3.104 PFU/mL. Our data demonstrated that the medium used for treatment impacted the efficacy of the bacteriophage cocktail on S. capitis including a better efficacy in TSB than in PBS, particularly on biofilm. This is consistent with the mechanism of action of bacteriophages that are known to better infect active bacteria (Azeredo and Sutherland, 2008; Harper et al., 2014). A rich medium, as TSB, could induce the re-growth of the dormant bacteria inside the biofilm and thus foster bacteriophages infection and replication inducing bacterial lysis. In addition, when bacteriophages greatly outnumber bacteria, they are able to kill it without intrabacterial replication, a phenomenon previously described as “lysis from without” (Abedon, 2011). This could explain the paradoxical efficacy of bacteriophages in PBS on dry spots at high bacteriophage concentration.\\\\nDespite all benefits of the use of bacteriophages, some points ought to be examined with particular attention. First, emergence of bacterial resistance against bacteriophages is frequent notably by avoiding bacteriophage adsorption or bacteriophage nucleic acid entry (based on alteration of bacteriophages nucleic acids or abortive infection systems) (Labrie et al., 2010; Oechslin, 2018). The emergence of bacterial resistance against bacteriophages was here prevented using a cocktail of three different bacteriophages, as already described in in vitro and in vivo studies (Yu et al., 2018; Yang et al., 2020). The choice of bacteriophages belonging to different genera (Herelleviridae/Kayvirus for V1SC01 and V1SC04 and Rountreeviridae/Andhravirus for V1SC05) may have been beneficial. Indeed, the Myoviridae including Kayvirus and the Podoviridae including Andhravirus are known to target different parts of wall teichoic acids in S. aureus (Moller et al., 2019). However, these are only hypotheses since in the present study, neither the mode of action of the bacteriophages nor their bacterial receptors have been investigated. It is likely that if these receptors are shared between two or three of these bacteriophages, it can increase the risk of bacterial resistance against the cocktail after a longer contact time than 24 h, as previously described (Oechslin, 2018). Moreover, our results demonstrate that the frequency of resistance increased for VISC05 used alone when the MOI increased. This result could be due to a faster selection of the mutant cells at higher bacteriophage concentration. This phenomenon has already been reported in vivo for Salmonella strains (Atterbury et al., 2007).\\\\nGiven that the bacteriophage cocktail at the higher concentration showed a similar effect than SP on S. capitis biofilm without resistance acquisition, its use in a protocol of decontamination seems achievable. However, because we observed only an incomplete effect on biofilm of either SP or bacteriophage cocktails, we aimed to potentiate the bactericidal effect by testing a sequential treatment using SP exposition followed by bacteriophage cocktail exposition. Unfortunately no additive effect was observed. We hypothesize that the biofilm structure offered a protection for bacteria against those two treatments, even when they were sequentially administered, as already described for antibiotics or disinfectant molecules (Smith and Hunter, 2008; Schilcher and Horswill, 2020). In a previous study, Melo et al. reported that an anti-Staphylococcus epidermidis bacteriophage was ineffective against the bacteria in biofilm whereas it showed efficacy on released biofilm cells and persisters, a subset of the biofilm population, highlighting the protective effect of the biofilm matrix against bacteriophages activity. Indeed, the authors demonstrated by confocal microscopy that bacteriophages had access only to biofilm cells not embedded in N-acetyl-D-glucosamine residues (PNAG) (Melo et al., 2020). The biofilm tridimensional structure and architecture was also shown to protect against bacteriophages in other bacterial species, i.e. Vibrio anguillarum (Tan et al., 2015). Contrariwise, many bacteriophages can also exert damages on biofilms notably thanks to enzymes that target components of the biofilm matrix (Harper et al., 2014; D’Accolti et al., 2021). For instance, exopolysaccharide degradation by a bacteriophage depolymerase was identified as the first step in the disruption of an Enterobacter agglomerans biofilm (Hughes et al., 1998). In that regard, the use of bacteriophages polysaccharide depolymerases in association with disinfectants was demonstrated to be effective to remove an Enterobacter biofilm (Tait et al., 2002). The combination of lytic bacteriophage with anti-biofilm bacteriophage could lead to a synergistic effect on biofilm. It should be noted that no genes corresponding to polysaccharide depolymerases were identified in the genomes of the three bacteriophages used in the present study (data not shown).\\\\nAccording to our results showing the absence of total eradication of biofilm-embedded bacteria by a bacteriophage cocktail, the necessity of using a rich medium that can lead to growth of other environmental bacteria and knowing their narrow antibacterial spectrum, bacteriophages alone would not be sufficient to eradicate bacteria from NICU surfaces. Thus, if used, bacteriophages must be combined with other complementary methods in a perspective of environmental disinfection of NICUs. Previous studies have evaluated the efficacy of bacteriophages in association with usual disinfectants in situations of hospital bacterial outbreaks and have shown the effectiveness of these combinations (Roy et al., 1993; Zhang and Hu, 2013; Stachler et al., 2021). Of note, when bacteriophages and disinfectant molecules are tested in combination, it is required to determine the effect of each disinfectant on bacteriophages activity and vice versa to avoid antagonistic effects (Agún et al., 2018; Stachler et al., 2021). Based on our data which did not show a better efficacy of the sequential treatment with bacteriophages then a chemical disinfectant compared to each treatment alone, but also due to the potential adverse effects of chemical molecules, other alternatives have to be explored. Disinfection of the incubators using steam pulverization is also a safe, efficient and simple method that has already been implemented in several NICU settings (Braux et al., 2008; Gillespie et al., 2017; Ory et al., 2019). If it seems more efficient than chemical disinfection, it did not allow for a complete decontamination of incubators. In that context, it might be interesting to develop a disinfection protocol including both the steam pulverization method, which allows non-selective killing of bacteria, and a bacteriophage cocktail to specifically target the persistent NRCS-A strains. However, the order in which the bacteriophage cocktail and steam should be used remains to be defined. Such a protocol deserves to be tested in future studies. Besides, in a perspective of using bacteriophages in real-life decontamination, additional questions concerning production and safety of bacteriophages have to be addressed in futures works. In particular, this project would require to setup a method allowing a large production of the three bacteriophages in controlled conditions ensuring the absence of bacterial residues as toxins and pyrogens in the final product. Another critical point to control is the stability of the bacteriophages that must be maintained during their storage; previous studies have reported that this stability would be obtained if bacteriophages are stored at high concentration (1.109 PFU/mL or more), or lyophilized in presence of sucrose or gelatin during months to years. (Pirnay et al., 2018; Manohar and Ramesh, 2019; Duyvejonck et al., 2021). Moreover, the safety of the treatment in the close environment of preterm neonates is another pivotal point but this will be easily performed since the product will be carefully rinsed after decontamination, to avoid the contact with neonates but also to avoid promoting the regrowth of other bacteria.\\\\nFinally, the present study presents several limitations. First, the tests confronting bacterial dry spots or biofilm to treatments were performed in a controlled laboratory setting using a single bacterial strain and a single type of material. Although CR01 is the reference strain for the NRCS-A clone, extended tests need to be performed to confirm the observed results on a larger collection of NRCS-A strains. Second, the comparison of the activity of bacteriophages and SP on dry spots has to be cautious given the differences in conditions during those two expositions. Indeed, during their treatment by bacteriophages, bacteria had grown in a rich medium (TSB) during 6 h versus water during 20 minutes for the SP treatment. Thus, the quantification of the effect using LRV is not fully comparable because the bacterial inoculum has evolved differently. Third, the high bacterial inoculum used for the treatments is not representative of the situation of the NICU settings and additional investigations are required before extrapolating our results to real-life practice although the likely lower bacterial inoculum in real life conditions could be in favor of a higher bacteriophage activity and bacterial eradication. Finally, the long-term effect of bacteriophages in the NICU environment but also their effect on eukaryotic cells could be interesting points to explore.\\\\nTo sum up, the present study constitutes a proof-of-concept for the use of bacteriophages on S. capitis NRCS-A environmental contamination. In the era of increasing resistance to antibiotics and disinfectants, the development of such innovative approach of decontamination using bacteriophages for biocontrol of specific pathogenic bacteria could find its full place in the landscape of infection prevention. Additional evaluations deserve to be performed to define the best and adequate use of biodecontamination and to evaluate its feasibility and efficacy in the NICU environment in conditions simulating real-life conditions.\\\\nData availability statement\\\\nThe data presented in the study are deposited in the data.world repository, following the link https://data.world/mariechavignon/phage. The genomes of bacteriophages vB_ScaM-V1SC01, vB_ScaM-V1SC04 and vB_ScaM-V1SC05 are deposited in the GenBank repository, accession number OP323059, OP297178 and OP297179 respectively.\\\\nAuthor contributions\\\\nMC and CK designed the study, analyzed the data and wrote the manuscript. MM, AT, FL, and MaB contributed to the design of the study. LB and TL contributed to the isolation, production and host range assessment of bacteriophages. MéB performed the bioinformatics analysis. MaB and FL reviewed the manuscript. All authors contributed to the article and approved the submitted version.\\\\nFunding\\\\nThis study was supported by the ANR (Agence Nationale de la Recherche) as part of the project NeoSCap [grant number ANR 19-CE17-0004-01] and the project PHAG-ONE [grant number ANR 20-PAMR-0009]. The authors thank the ANR (Agence Nationale de la Recherche) which supported this work as part of the project NeoSCap and the project PHAG-ONE and the Hospices Civils de Lyon Foundation which financed the PHAGEinLYON project.\\\\nConflict of interest\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nPublisher’s note\\\\nAll claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.\\\\nAbbreviations\\\\nLRV, Log10 reduction values; MOI, Multiplicity of Infection; NICU, Neonatal Intensive Care Unit; SP, Surfanios Premium.\\\\nReferences\\\\nAgún, S., Fernández, L., González-Menéndez, E., Martínez, B., Rodríguez, A., García, P. (2018). Study of the interactions between bacteriophage phiIPLA-RODI and four chemical disinfectants for the elimination of Staphylococcus aureus contamination. Viruses 10, E103. doi: 10.3390/v10030103\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAtterbury, R. J., Van Bergen, M. A. P., Ortiz, F., Lovell, M. A., Harris, J. A., De Boer, A., et al. (2007). Bacteriophage therapy to reduce Salmonella colonization of broiler chickens. App. Environ. Microbiol. 73, 4543–4549. doi: 10.1128/AEM.00049-07\\\\nCrossRef Full Text | Google Scholar\\\\nBraux, C., Lagier, A., Andrini, P., Debillon, T., Croizé, J. (2008). Entretien des incubateurs de néonatalogie à l’aide d’un générateur de vapeur. Hygienes. XVI. 16, 241–247.\\\\nGoogle Scholar\\\\nButin, M., Dumont, Y., Monteix, A., Raphard, A., Roques, C., Martins Simoes, P., et al. (2019). Sources and reservoirs of Staphylococcus capitis NRCS-a inside a NICU. Antimicrob. Resist. Infect. Control. 8, 157. doi: 10.1186/s13756-019-0616-1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nButin, M., Martins-Simões, P., Rasigade, J.-P., Picaud, J.-C., Laurent, F. (2017). Worldwide endemicity of a multidrug-resistant Staphylococcus capitis clone involved in neonatal sepsis. Emerg. Infect. Dis. 23, 538–539. doi: 10.3201/eid2303.160833\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCadot, L., Bruguière, H., Jumas-Bilak, E., Didelot, M.-N., Masnou, A., de Barry, G., et al. (2019). Extended spectrum beta-lactamase-producing Klebsiella pneumoniae outbreak reveals incubators as pathogen reservoir in neonatal care center. Eur. J. Pediatr. 178, 505–513. doi: 10.1007/s00431-019-03323-w\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCarter, G. P., Ussher, J. E., Da Silva, A. G., Baines, S. L., Heffernan, H., Riley, T. V., et al. (2018). Genomic analysis of multiresistant Staphylococcus capitis associated with neonatal sepsis. Antimicrob. Agents Chemother. 62, e00898–e00818. doi: 10.1128/AAC.00898-18\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChavignon, M., Reboux, M., Tasse, J., Tristan, A., Claris, O., Laurent, F., et al. (2021). Persistent microbial contamination of incubators despite disinfection. Pediatr. Res. 90, 1215–1220. doi: 10.1038/s41390-021-01407-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCollins, A., Weitkamp, J.-H., Wynn, J. L. (2018). Why are preterm newborns at increased risk of infection? Arch. Dis. Child. - Fetal. Neonatal. Ed. 103, F391–F394. doi: 10.1136/archdischild-2017-313595\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCui, B., Smooker, P. M., Rouch, D. A., Daley, A. J., Deighton, M. A. (2013). Differences between two clinical Staphylococcus capitis subspecies as revealed by biofilm, antibiotic resistance, and pulsed-field gel electrophoresis profiling. J. Clin. Microbiol. 51, 9–14. doi: 10.1128/JCM.05124-11\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDąbrowska, K., Miernikiewicz, P., Piotrowicz, A., Hodyra, K., Owczarek, B., Lecion, D., et al. (2014). Immunogenicity studies of proteins forming the T4 phage head surface. J. Virol. 88, 12551–12557. doi: 10.1128/JVI.02043-14\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nD’Accolti, M., Soffritti, I., Lanzoni, L., Bisi, M., Volta, A., Mazzacane, S., et al. (2019). Effective elimination of staphylococcal contamination from hospital surfaces by a bacteriophage-probiotic sanitation strategy: a monocentric study. Microb. Biotechnol. 12, 742–751. doi: 10.1111/1751-7915.13415\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nD’Accolti, M., Soffritti, I., Mazzacane, S., Caselli, E. (2021). Bacteriophages as a potential 360-degree pathogen control strategy. Microorganisms 9, 261. doi: 10.3390/microorganisms9020261\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDhama, K., Patel, S. K., Kumar, R., Masand, R., Rana, J., Yatoo, M. I., et al. (2021). The role of disinfectants and sanitizers during COVID-19 pandemic: advantages and deleterious effects on humans and the environment. Environ. Sci. pollut. Res. 28, 34211–34228. doi: 10.1007/s11356-021-14429-w\\\\nCrossRef Full Text | Google Scholar\\\\nDuyvejonck, H., Merabishvili, M., Vaneechoutte, M., de Soir, S., Wright, R., Friman, V.-P., et al. (2021). Evaluation of the stability of bacteriophages in different solutions suitable for the production of magistral preparations in Belgium. Viruses 13, 865. doi: 10.3390/v13050865\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGillespie, E., Tabbara, L., Scott, C., Lovegrove, A., Kotsanas, D., Stuart, R. L. (2017). Microfiber and steam for a neonatal service: An improved and safe cleaning methodology. Am. J. Infect. Control. 45, 98–100. doi: 10.1016/j.ajic.2016.06.041\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGolan, Y., Doron, S., Sullivan, B., Snydman, D. R. (2005). Transmission of vancomycin-resistant Enterococcus in a neonatal intensive care unit. Pediatr. Infect. Dis. J. 24, 566–567. doi: 10.1097/01.inf.0000164762.03930.0a\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGreen, S. I., Kaelber, J. T., Ma, L., Trautner, B. W., Ramig, R. F., Maresso, A. W. (2017). Bacteriophages from ExPEC reservoirs kill pandemic multidrug-resistant strains of clonal group ST131 in animal models of bacteremia. Sci. Rep. 7, 46151. doi: 10.1038/srep46151\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHarper, D. R., Parracho, H. M. R. T., Walker, J., Sharp, R., Hughes, G., Werthén, M., et al. (2014). Bacteriophages and biofilms. Antibiotics 3, 270–284. doi: 10.3390/antibiotics3030270\\\\nCrossRef Full Text | Google Scholar\\\\nHo, Y.-H., Tseng, C.-C., Wang, L.-S., Chen, Y.-T., Ho, G.-J., Lin, T.-Y., et al. (2016). Application of bacteriophage-containing aerosol against nosocomial transmission of carbapenem-resistant Acinetobacter baumannii in an intensive care unit. PloS One 11, e0168380. doi: 10.1371/journal.pone.0168380\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHrubec, T. C., Seguin, R. P., Xu, L., Cortopassi, G. A., Datta, S., Hanlon, A. L., et al. (2021). Altered toxicological endpoints in humans from common quaternary ammonium compound disinfectant exposure. Toxicol. Rep. 8, 646–656. doi: 10.1016/j.toxrep.2021.03.006\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHughes, K. A., Sutherland, I. W., Jones, M. V. Y. (1998). Biofilm susceptibility to bacteriophage attack: the role of phage-borne polysaccharide depolymerase. Microbiology 144, 3039–3047. doi: 10.1099/00221287-144-11-3039\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLemriss, H., Simões, P. M., Lemriss, S., Butin, M., Ibrahimi, A., El Kabbaj, S., et al. (2014). Non-contiguous finished genome sequence of Staphylococcus capitis CR01 (pulsetype NRCS-a). Stand. Genom. Sci. 9, 1118–1127. doi: 10.4056/sigs.5491045\\\\nCrossRef Full Text | Google Scholar\\\\nLin, D., Ou, Q., Lin, J., Peng, Y., Yao, Z. (2017). A meta-analysis of the rates of Staphylococcus aureus and methicillin-resistant S. aureus contamination on the surfaces of environmental objects that health care workers frequently touch. Am. J. Infect. Control. 45, 421–429. doi: 10.1016/j.ajic.2016.11.004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMagill, S. S., O’Leary, E., Janelle, S. J., Thompson, D. L., Dumyati, G., Nadle, J., et al. (2018). Changes in prevalence of health care–associated infections in U.S. hospitals. N. Engl. J. Med. 379, 1732–1744. doi: 10.1056/NEJMoa1801550\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMelo, L. D. R., Pinto, G., Oliveira, F., Vilas-Boas, D., Almeida, C., Sillankorva, S., et al. (2020). The protective effect of Staphylococcus epidermidis biofilm matrix against phage predation. Viruses 12, 1076. doi: 10.3390/v12101076\\\\nCrossRef Full Text | Google Scholar\\\\nNguyen, S., Baker, K., Padman, B. S., Patwa, R., Dunstan, R. A., Weston, T. A., et al. (2017). Bacteriophage transcytosis provides a mechanism to cross epithelial cell layers. mBio 8, e01874–e01817. doi: 10.1128/mBio.01874-17\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nOechslin, F. (2018). Resistance development to bacteriophages occurring during bacteriophage therapy. Viruses 10, 351. doi: 10.3390/v10070351\\\\nCrossRef Full Text | Google Scholar\\\\nOry, J., Cazaban, M., Richaud-Morel, B., Maio, M. D., Dunyach-Remy, C., Pantel, A., et al. (2019). Successful implementation of infection control measure in a neonatal intensive care unit to combat the spread of pathogenic multidrug resistant Staphylococcus capitis. Antimicrob. Resist. Infect. Control. 8, 1–6. doi: 10.1186/s13756-019-0512-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPirnay, J.-P., Merabishvili, M., Van Raemdonck, H., De Vos, D., Verbeken, G. (2018). “Bacteriophage production in compliance with regulatory requirements,” in Bacteriophage therapy: From Lab to clinical practice methods in molecular biology. Eds. Azeredo, J., Sillankorva, S. (New York, NY: Springer), 233–252. doi: 10.1007/978-1-4939-7395-8_18\\\\nCrossRef Full Text | Google Scholar\\\\nPodlacha, M., Grabowski, Ł., Kosznik-Kawśnicka, K., Zdrojewska, K., Stasiłojć, M., Węgrzyn, G., et al. (2021). Interactions of bacteriophages with animal and human organisms–safety issues in the light of phage therapy. Int. J. Mol. Sci. 22, 8937. doi: 10.3390/ijms22168937\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPurohit, A., Kopferschmitt-Kubler, M.-C., Moreau, C., Popin, E., Blaumeiser, M., Pauli, G. (2000). Quaternary ammonium compounds and occupational asthma. Int. Arch. Occup. Environ. Health 73, 423–427. doi: 10.1007/s004200000162\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nQu, Y., Li, Y., Cameron, D. R., Easton, C. D., Zhu, X., Zhu, M., et al. (2020). Hyperosmotic infusion and oxidized surfaces are essential for biofilm formation of Staphylococcus capitis from the neonatal intensive care unit. Front. Microbiol. 11. doi: 10.3389/fmicb.2020.00920\\\\nCrossRef Full Text | Google Scholar\\\\nRasigade, J.-P., Raulin, O., Picaud, J.-C., Tellini, C., Bes, M., Grando, J., et al. (2012). Methicillin-resistant Staphylococcus capitis with reduced vancomycin susceptibility causes late-onset sepsis in intensive care neonates. PloS One 7, e31548. doi: 10.1371/journal.pone.0031548\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRoy, B., Ackermann, H. W., Pandian, S., Picard, G., Goulet, J. (1993). Biological inactivation of adhering Listeria monocytogenes by listeriaphages and a quaternary ammonium compound. Appl. Environ. Microbiol. 59, 2914–2917. doi: 10.1128/aem.59.9.2914-2917.1993\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSchilcher, K., Horswill, A. R. (2020). Staphylococcal biofilm development: structure, regulation, and treatment strategies. Microbiol. Mol. Bio. Rev. 84, e00026–e00019. doi: 10.1128/MMBR.00026-19\\\\nCrossRef Full Text | Google Scholar\\\\nSikora, A., Zahra, F. (2022). “Nosocomial infections,” in StatPearls (Treasure Island (FL: StatPearls Publishing).\\\\nGoogle Scholar\\\\nSmith, K., Hunter, I. S. Y. (2008). Efficacy of common hospital biocides with biofilms of multi-drug resistant clinical isolates. J. Med. Microbiol. 57, 966–973. doi: 10.1099/jmm.0.47668-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSong, X., Vossebein, L., Zille, A. (2019). Efficacy of disinfectant-impregnated wipes used for surface disinfection in hospitals: a review. Antimicrob. Resist. Infect. Control. 8, 139. doi: 10.1186/s13756-019-0595-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nStachler, E., Kull, A., Julian, T. R. (2021). Bacteriophage treatment before chemical disinfection can enhance removal of plastic-surface-associated Pseudomonas aeruginosa. Appl. Environ. Microbiol. 87, e00980–e00921. doi: 10.1128/AEM.00980-21\\\\nCrossRef Full Text | Google Scholar\\\\nSuetens, C., Latour, K., Kärki, T., Ricchizzi, E., Kinross, P., Moro, M. L., et al. (2018). Prevalence of healthcare-associated infections, estimated incidence and composite antimicrobial resistance index in acute care hospitals and long-term care facilities: results from two European point prevalence surveys 2016 to 2017. Euro. surveill. 23, 1800516. doi: 10.2807/1560-7917.ES.2018.23.46.1800516\\\\nCrossRef Full Text | Google Scholar\\\\nSuleyman, G., Alangaden, G., Bardossy, A. C. (2018). The role of environmental contamination in the transmission of nosocomial pathogens and healthcare-associated infections. Curr. Infect. Dis. Rep. 20, 12. doi: 10.1007/s11908-018-0620-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSuneja, T., Belsito, D. V. (2008). Occupational dermatoses in health care workers evaluated for suspected allergic contact dermatitis. Contact. Derm. 58, 285–290. doi: 10.1111/j.1600-0536.2007.01315.x\\\\nCrossRef Full Text | Google Scholar\\\\nTait, K., Skillman, L. C., Sutherland, I. W. (2002). The efficacy of bacteriophage as a method of biofilm eradication. Biofouling 18, 305–311. doi: 10.1080/0892701021000034418\\\\nCrossRef Full Text | Google Scholar\\\\nTan, D., Dahl, A., Middelboe, M. (2015). Vibriophages differentially influence biofilm formation by Vibrio anguillarum strains. Appl. Environ. Microbiol. 81, 4489–4497. doi: 10.1128/AEM.00518-15\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTasse, J., Cara, A., Saglio, M., Villet, R., Laurent, F. (2018). A steam-based method to investigate biofilm. Sci. Rep. 8, 13040. doi: 10.1038/s41598-018-31437-y\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTetz, G., Tetz, V. (2018). Bacteriophages as new human viral pathogens. Microorganisms 6, 54. doi: 10.3390/microorganisms6020054\\\\nCrossRef Full Text | Google Scholar\\\\nWirth, T., Bergot, M., Rasigade, J.-P., Pichon, B., Barbier, M., Martins-Simoes, P., et al. (2020). Niche specialization and spread of Staphylococcus capitis involved in neonatal sepsis. Nat. Microbiol. 5, 735–745. doi: 10.1038/s41564-020-0676-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWright, A., Hawkins, C. H., Änggård, E. E., Harper, D. R. (2009). A controlled clinical trial of a therapeutic bacteriophage preparation in chronic otitis due to antibiotic-resistant Pseudomonas aeruginosa; a preliminary report of efficacy. Clin. Otolaryngol. 34, 349–357. doi: 10.1111/j.1749-4486.2009.01973.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYang, Y., Shen, W., Zhong, Q., Chen, Q., He, X., Baker, J. L., et al. (2020). Development of a bacteriophage cocktail to constrain the emergence of phage-resistant Pseudomonas aeruginosa. Front. Microbiol. 11. doi: 10.3389/fmicb.2020.00327\\\\nCrossRef Full Text | Google Scholar\\\\nYıldızlı, G., Coral, G., Ayaz, F. (2020). Immunostimulatory activities of coliphages on in vitro activated mammalian macrophages. Inflammation 43, 595–604. doi: 10.1007/s10753-019-01140-9\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYu, L., Wang, S., Guo, Z., Liu, H., Sun, D., Yan, G., et al. (2018). A guard-killer phage cocktail effectively lyses the host and inhibits the development of phage-resistant strains of Escherichia coli. Appl. Microbiol. Biotechnol. 102, 971–983. doi: 10.1007/s00253-017-8591-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"http://www.sas.upenn.edu/LabManuals/biol275/Table_of_Contents_files/13-PhageGrowth.pdf\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"http://www.sas.upenn.edu/LabManuals/biol275/Table_of_Contents_files/13-PhageGrowth.pdf\",\\n    \"depth\": 1,\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"loadedTime\": \"2024-01-23T08:49:52.298Z\",\\n    \"httpStatusCode\": 200\\n  },\\n  \"html\": null,\\n  \"text\": null,\\n  \"markdown\": null,\\n  \"metadata\": {\\n    \"mimeType\": \"application/pdf\",\\n    \"author\": null,\\n    \"canonicalUrl\": \"http://www.sas.upenn.edu/LabManuals/biol275/Table_of_Contents_files/13-PhageGrowth.pdf\",\\n    \"description\": null,\\n    \"keywords\": null,\\n    \"languageCode\": null,\\n    \"title\": null\\n  },\\n  \"screenshotUrl\": null,\\n  \"fileUrl\": \"https://api.apify.com/v2/key-value-stores/KVxLi5JzsdtaIIa9U/records/4a128da-13-PhageGrowth.pdf\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00875\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00875/full\",\\n    \"loadedTime\": \"2024-01-23T08:49:57.568Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00875\",\\n    \"title\": \"Frontiers | Bacteriophage Infectivity Against Pseudomonas aeruginosa in Saline Conditions\",\\n    \"description\": \"Pseudomonas aeruginosa is a ubiquitous member of marine biofilm, and reduces thiosulfate to produce toxic hydrogen sulfide gas. In this study, lytic bacteriophages of Podoviridae family were isolated and applied to inhibit the growth of P. aeruginosa in planktonic mode at different temperature, pH and salinity. Bacteriophages showed optimal infectivity at a multiplicity of infection of 10 in saline conditions, and demonstrated lytic abilities over all tested temperature (25, 30, 37 and 45 oC) and pH 6 to 9. Planktonic P. aeruginosa exhibited significantly longer lag phase and lower specific growth rates upon exposure to bacteriophages. Bacteriophages were subsequently applied to P. aeruginosa-enriched biofilm and were determined to lower the relative abundance of Pseudomonas-related taxa from 0.17-5.58% in controls to 0.01-0.61% in treated microbial communities. The relative abundance of Alphaproteobacteria, Pseudoalteromonas and Planococcaceae decreased, possibly due to the phage-induced disruption of the biofilm matrix. Lastly, when applied to mitigate biofouling of ultrafiltration membranes, bacteriophages were determined to reduce the transmembrane pressure increase by 18% when utilized alone, and by 49% when used in combination with citric acid. Collectively, the findings demonstrated that bacteriophages can be used as a biocidal agent to mitigate undesirable P. aeruginosa-associated problems in seawater applications.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nSeawater reverse osmosis (SWRO) desalination has had a great impact on the production of drinking water in the past 40 years (Goosen et al., 2005), and is particularly relied upon by sea-bordered countries that face water scarcity issues (Peñate and García-Rodríguez, 2012). Similar to all membrane-based technologies, SWRO is affected by biofouling. Biofilm formation on membrane detrimentally lowers the flux and salt rejection capacity. In order to maintain the desired desalination performance, plant operators usually apply higher transmembrane pressure which in turn increases energy consumption and economic costs (Fritzmann et al., 2007; Matin et al., 2011).\\\\nBacteria such as Pseudomonas, Bacillus, Mycobacterium, Acinetobacter are often detected on fouled RO membranes (Matin et al., 2011). In particular, Pseudomonas spp. and Desulfovibrio spp. were identified on fouled ultrafiltration (UF) and RO membranes sampled from a SWRO pilot plant located in the Arabian Gulf (Hong et al., 2016). Pseudomonas spp. create an optimal niche for Desulfovibrio spp. and other sulfate-reducing bacteria (SRB) by depleting oxygen. In addition, SRB and Pseudomonas aeruginosa are able to reduce sulfate and thiosulfate, respectively, to produce hydrogen sulfide, a corrosive and toxic gas (Hong et al., 2016). The presence of Pseudomonas spp. and SRB in SWRO are favored by the high sulfate concentration prevalent in seawater. In addition, sodium metabisulfite applied to neutralize chlorine within the SWRO (Hong et al., 2016) further provide a source of electron acceptors for these bacterial populations.\\\\nTo reduce both inorganic and organic foulants on membranes, oxidizing agents such as chlorine, permanganate, and ozone (Fritzmann et al., 2007; Gao et al., 2011) are used in large concentrations. However, the addition of these chemicals in seawater can result in the formation of carcinogenic and toxic disinfection byproducts that can result in public health concerns (Le Roux et al., 2015; Sanawar et al., 2017). Alternatively, regular cleaning of the membrane would have to be performed with acid cleaning (Greenlee et al., 2009). Citric acid cleaning is often used in the pretreatment stage of SWRO to chelate inorganic minerals and disrupt the stability of biofilm matrix attached on the pretreatment UF membranes (Lee et al., 2001; Porcelli and Judd, 2010). Despite their common usage, these biocidal agents have limited penetration through a biofilm matrix and are less effective against biofilm-associated bacteria compared to its effect on planktonic cells (Matin et al., 2011). Moreover, some chemicals e.g., chlorine, can detrimentally impact RO membrane integrity.\\\\nTo address the concerns arising from conventional biocides and cleaning agents, this study evaluates the use of lytic bacteriophages (i.e., viruses that infect bacteria at high host specificity) to inhibit the growth of planktonic P. aeruginosa in saline conditions. In addition, bacteriophages can also induce the release of depolymerases that degrade extracellular polymeric substances (EPS) and hence disrupt biofilm matrix (Harper et al., 2014). It is therefore hypothesized that bacteriophage would be effective to disrupt P. aeruginosa-associated biofilms formed on seawater filtration membranes. Although bacteriophage treatment against P. aeruginosa biofilm has been performed in therapeutic treatments (McVay et al., 2007; Fu et al., 2010; Alemayehu et al., 2012; Olszak et al., 2015), and as antifouling agents in water and wastewater membrane filtration systems (Zhang et al., 2013; Bhattacharjee et al., 2015), the conditions at which the bacteriophages were applied in those studies differ from those experienced in a SWRO system. Understanding how environmental conditions can impact bacteriophage infectivity is crucial for an effective application of bacteriophages in SWRO systems, where the conditions vary between high pH (i.e., the usual pH of seawater) to low pH (i.e., when citric acid cleaning is performed), high salinity (i.e., in raw seawater) to low salinity (i.e., after seawater is desalinated), and from low (i.e., in the raw seawater) to high temperatures (i.e., seawater retained within the desalination unit).\\\\nIn this study, the impact of various parameters, namely multiplicity of infection (MOI), pH, salinity and temperature, on the activity of seven lytic bacteriophages against planktonic P. aeruginosa cells was analyzed. Three bacteriophages were further applied against a P. aeruginosa-enriched seawater biofilm at various pH values and temperatures. Finally, a cocktail of the three bacteriophages was applied alone and in combination with citric acid cleaning to cross-flow seawater ultrafiltration setup, and the membrane modules were assessed for their transmembrane pressure and biofilm cell counts.\\\\nMaterials and Methods\\\\nBacteriophage Isolation\\\\nBacteriophages were isolated from an influent collected from a wastewater treatment plant located in KAUST. A 50 mL aliquot of the influent was centrifuged at 8,500 g for 20 min and thereafter supernatant was filtered through 0.22 μm cellulose acetate syringe filter (VWR, Radnor, PA) to remove bacterial cells. The filtrate was mixed with 50 mL Lennox broth (LB) supplemented with 35 g/L NaCl and 4 mM Ca2+, and 50 mL of exponentially growing P. aeruginosa strain DSM1117. The mixed culture was incubated for 24 h at 37°C. Thereafter, 1% v/v chloroform was added to lyse the bacterial cells, and incubated for 2 h at room temperature with constant agitation at 100 rpm. The culture was centrifuged at 8,500 g for 30 min at 4°C and the supernatant was filtered through 0.22 μm syringe filter to remove bacterial cells. Several dilutions, ranging from 10−3 to 10−5 fold, were performed in sodium magnesium (SM) buffer (5.8 g/L NaCl, 0.975 g/L MgSO4, 50 ml/L 50 mM Tris-Cl 7.5 pH) and 10 μL of each diluted sample was mixed with 100 μL of P. aeruginosa culture to detect the presence of plaques using the double layer plate method (Adams, 1959). Seven plaques were isolated with sterile inoculating loops, transferred to SM buffer and filtered through 0.22 μm syringe filter to remove bacterial cells. The bacteriophages from the seven plaques were named P1 through P7 and propagated 10 more times using the above mentioned procedure.\\\\nInfection of Planktonic Cultures With Bacteriophages at Various Conditions\\\\nAll bacteriophages were tested for their infectivity against other bacterial species using the soft agar plaque assay method. All the bacteria used in this study are listed in Table 1. After verifying their host specificity, bacteriophage activity against P. aeruginosa was analyzed at various multiplicity of infection (MOI) values, namely 0.1, 1, and 10. MOI represents the ratio between the number of bacteriophage particles determine as phage forming units (PFU/mL) and bacterial cells determined as colony forming units (CFU/mL) in a mixture of 1:1 LB broth and SM buffer, pH 7. MOI was calculated for each bacteriophage based on the number of plaques formed on agar (Figure S1). For each MOI, one control was prepared without bacteriophages. The treated cultures and controls were incubated at 37°C with constant shaking at 200 rpm for 24 h. Growth curves of P. aeruginosa were obtained with and without the presence of the seven phages. The best MOI value in terms of bacterial inhibition time (in h) and specific growth rate reduction (h−1) compared with the control was selected for evaluating the impact of other parameters. Subsequently, bacteriophage infectivity against P. aeruginosa cultures was evaluated at different temperatures, pH, and salinity, with the same procedure described above at a constant MOI of 10. All experiments were performed in triplicates.\\\\nTable 1. Name and origin of the bacteria used in this study.\\\\nInfection of Biofilm With Bacteriophages at Various Conditions\\\\nTo examine the effect of bacteriophages on the biofilm structure at various temperatures (25, 30, 37, and 45°C) and pH values (5, 6, 7, 8), 28 drip flow reactors (BioSurface Technologies Corp., Bozeman, MT) were assembled. Twenty eight cellulose acetate ultrafiltration membranes (Sterilitech Corporation, Kent, WA), each with surface area of 11.25 cm2, were secured on glass coupons and placed into the drip flow reactors to establish biofilm on the membranes. Reactors were covered with aluminum foil to avoid light exposure. Seawater from Red Sea was fed into each reactor at a continuous rate of 5 mL per minute. The feed was replaced with fresh seawater every 3 d. After 4 weeks, the feed solution was replaced with 5 L of P. aeruginosa-enriched seawater. Briefly, P. aeruginosa was cultivated overnight on Pseudomonas isolation agar (Sigma-Aldrich, St Louis, MO); single colonies were transferred into multiple tubes with 20 mL LB broth and grown at 37°C with constant shaking (200 rpm) to reach OD600 of 0.7. After centrifugation at 4,100 g for 30 min, bacterial pellet was resuspended in 5 L seawater to achieve a cell density of 3.5 × 107 CFU/mL of P. aeruginosa. This feed was also replaced every 3 d.\\\\nAfter 4 weeks, to calculate the starting number of Pseudomonas colonies, 4 membranes were removed from the reactor and washed twice in 1× phosphate-buffered saline (PBS) to remove loosely attached biofilm. Membranes were cut into 16 identical pieces and were placed individually into collection tubes containing 2 mL 1× PBS. Attached biofilm was sonicated for 5 min by a Q500 sonicator (Qsonica, Newton, CT, US) at 30% amplitude with 3 s pulsating steps to detach biomass from the membranes. The supernatant was serially diluted for spread plating on Pseudomonas isolation agar and incubated at 37°C for 1 d for colony counting. The remaining membranes were removed, rinsed twice in 1× PBS to remove loosely attached biofilm, and aseptically cut into four pieces for a total of 96 identical fragments. Each piece was transferred to an individual 15 mL tube containing LB broth and SM buffer (each in 50% v/v, salinity 3.5%). Bacteriophages P1, P5, and P7 were propagated as described above and were diluted with SM buffer to the adequate number of PFU/mL to attain an MOI value of 10. Each bacteriophage was then used individually to infect attached biofilm on the membrane at four different temperatures (25, 30, 37, and 45°C) and pH values (5, 6, 7, 8) for 10 h at constant shaking of 200 rpm. A control without bacteriophage inoculation was prepared for each temperature and pH condition. After 10 h of phage infection, biofilm was sonicated in 3.5 mL of 1× PBS and analyzed based on procedures as described in sections P. aeruginosa Colony and Bacteriophage Plaque Counts on Membranes and RNA Extraction and 16S rRNA Gene Based High-Throughput Sequencing. All experiments were conducted in triplicate.\\\\nPseudomonas aeruginosa Colony and Bacteriophage Plaque Counts on Membranes\\\\nA 100 μL aliquot of bacterial suspension was used to evaluate the number of colony forming units (CFU)/cm2 on Pseudomonas isolation agar, while 1 mL was filtered through 0.22 μm syringe filter to assess the number of plaque forming units (PFU)/cm2 recovered from the different tested conditions.\\\\nRNA Extraction and 16S rRNA Gene based High-Throughput Sequencing\\\\nA 2 mL aliquot of the biofilm suspension (as described in section Infection of Biofilm With Bacteriophages at Various Conditions) was utilized for RNA extraction. To avoid RNA degradation, 4 mL of RNAprotect® Cell Reagent (Qiagen, Hilden, Germany) was added to each bacterial suspension immediately after sampling. The mixture was incubated at room temperature for 5 min and then centrifuged for 10 min at 5,400 g. After centrifugation, the supernatant was removed and the pellet was stored at −80°C for ~2 weeks until RNA extraction.\\\\nRNA extraction from the biomass pellet was performed using the RNeasy® Midi Kit (Qiagen, Hilden, Germany) following manufacturer\\'s protocol and RNA concentration was measured with the Invitrogen RNA HS Qubit® 2.0 assay kit (Thermo Fisher Scientific, Carlsbad, CA). Extracted RNA was reverse transcribed into first-strand complementary DNA (cDNA) using the Invitrogen SuperScriptTM First-Strand Synthesis System (Thermo Fisher Scientific, Carlsbad, CA). The cDNA was then used as template to amplify for 16S rRNA genes with primer pair 515F (5′- Illumina overhang- GTG YCA GCM GCC GCG GTA A-3′) and 907R (5′- Illumina overhang- CCC CGY CAA TTC MTT TRA GT-3′) based on the procedure described earlier (Scarascia et al., 2017). Purified amplicons were submitted to KAUST Genomic Core lab for amplicon sequencing on Illumina MiSeq platform. All high-throughput sequencing files used in this study are deposited in the European Nucleotide Archive (ENA) and are accessible via accession number PRJEB23782.\\\\nBiofilm Microbial Community Data Analysis\\\\nAmplicon sequences were sorted on a Phred score >30 and primers, adaptors, and index sequences were removed. After removing any sequences longer than 280 nt, sequence chimeras were identified and removed by UCHIME (Edgar et al., 2011). Chimera-free sequences were split and a subset containing 100,000 sequences were further analyzed for each sample. Taxonomical assignment was obtained at 95% classification reliability level with copy number adjustment using the Ribosomal Database Project (RDP) Classifier (Wang et al., 2007). Relative abundance at genus and phylum level was calculated for each sample. Special emphasis was made to determine the relative abundance of Pseudomonas-related taxa in the presence of bacteriophages compared to the control. Chimera-removed sequence files were also sorted for unique operational taxonomic units (OTUs) at 97% 16S rRNA gene similarity [27]. Similarity percentages (SIMPER) analysis was performed using Primer-E version 7 to identify OTUs that showed a significant difference in their relative abundance between infected and non-infected biofilm (Clarke, 2015). The OTU identities were checked against the NCBI nucleotide database using the BLASTN algorithm.\\\\nEffect of Bacteriophage Treatment on UF Membrane Fouling\\\\nA UF filtration system was set up as illustrated in Figure S2. Five UF cellulose acetate membranes of pore size 8 kDa (Sterilitech Corporation, Kent, WA) were aseptically cut into dimensions of 10 by 2.5 cm each, soaked in deionized water for 24 h, and then rinsed in 80% ethanol. Sterile membranes were then individually placed inside cassette modules. Subsequently, deionized water was circulated through the modules to reach a stable trans-membrane pressure (TMP0). The modules were operated in cross-flow mode at room temperature with the concentrate recirculating into the feed tank. A constant flux of 12 L/m2/h (LMH) was maintained. Artificial seawater (26.29 g/L NaCl, 0.74 g/L KCl, 1.32 g/L CaCl2 dehydrated, 6.09 g/L MgCl2·7H2O, 1.92 g/L MgSO4, pH 7.8) was used as feed solution. Nutrient broth (HiMedia, Mumbai, India) was added to the feed solution to obtain a nutrient concentration of 24 mg/L as described previously (Oh et al., 2017). Feed bottle was inoculated with P. aeruginosa culture at a final concentration of 5 × 107 CFU/mL and this feed was replaced every 3 d. Biofilm was established on each membrane for 6 d with a recording of the transmembrane pressure (TMP) every 8 h.\\\\nOn the 7th day, the filtration system was kept in offline mode for four different cleaning treatments: (i) 5 × 108 PFU/mL phage cocktail application in SM buffer for 6 h followed by deionized water for 3 h, (ii) 0.1 M citric acid treatment alone for 3 h followed by deionized water for 6 h, (iii) 5 × 108 PFU/mL phage cocktail application in SM buffer for 6 h followed by 0.1 M citric acid treatment for 3 h, and (iv) 0.1 M citric acid treatment for 3 h followed by 5 × 108 PFU/mL phage cocktail application in SM buffer for 6 h. One membrane was used as non-treated control where deionized water was applied for the same 9 h duration as that of the treatments. All membranes were flushed for 10 min with deionized water at the end of the treatments, and the TMP monitoring was resumed after 1 h from the treatment. Cleaning cycles were repeated every 3 d for a total of three cycles.\\\\nAt the end of the third cycle, membranes were removed from the modules and three 2 × 2 cm pieces were cut from each membrane sheet and placed in 2 mL 1× PBS to enumerate for the Pseudomonas colony counts and total cell counts. Cells were stained with SYBR® green (Thermo Fisher Scientific, Waltham, MA, US) and counted by BD Accuri C6 flow cytometer (BD, Bioscience, NJ, US). The cleaning experiment on the filtration membranes was performed in duplicate as run 1 and run 2.\\\\nBacteriophages Morphology and Genome Size Characterization\\\\nPure bacteriophages P1, P5, and P7 in cultures were fixed with 2.5% v/v glutaraldehyde. A 5 μL aliquot of each bacteriophage culture was deposited on carbon-coated copper grids and negatively stained with 1% w/v uranyl acetate (pH 4.5). Samples were washed with sterile water, air-dried, and visualized through transmission electron microscopy (TEM) (Tecnai Spirit TWIN, FEI) operated at 120 kV and equipped with an ORIUS SC1000 camera (Gaitan). Image analysis was carried out using ImageJ Software.\\\\nDirect phage plaques DNA was extracted for the three bacteriophages as described previously (Kot et al., 2014). Briefly, plaques were resuspended in DNase I buffer, filtered and treated for 30 min with DNase I; DNase was inactivated by 10 μL of 50 mM EDTA and further treated with Proteinase K (Thermo Scientific, Waltham, USA) before DNA extraction using UltraClean Soil DNA Isolation Kit (MoBio Laboratories, Carlsbad, USA) based on manufacturer\\'s protocol. Bacterial DNA contamination was checked through 16S rRNA gene amplification and all plaque DNA samples showed no amplification for 16S rRNA genes. Extracted DNA was used to assess genome size through pulsed field gel electrophoresis (PFGE). DNA samples were run on 1% w/v agarose gel using the CHEF-DR III system (Bio-Rad, Hercules, CA, USA) (18 h, 6 V cm−1, 14°C, angle 120°, switch time 0.1–1). Lambda DNA Ladder (New England, Biolabs, Ipswich, MA, USA) was used as size marker.\\\\nStatistical Analysis\\\\nOne-way ANOVA was performed to evaluate for statistical differences between treatments. Statistical differences were considered significant at 95% confidence level (p < 0.05).\\\\nResults\\\\nBacteriophage Infectivity Against Planktonic Pseudomonas aeruginosa\\\\nPlaques were only observed when isolated bacteriophages were infected against P. aeruginosa but not against all other tested bacterial hosts (Table 1). Infectivity of the seven isolated bacteriophages against P. aeruginosa growing in planktonic phase was systematically analyzed at various MOIs, temperatures, pH values, and salinities based on lag phase duration and specific growth rates. Lag phase of non-infected control cultures ranged from 1 to 4 h (Figures 1A–D), while specific growth rates ranged from 0.21 to 0.40 h−1 (Table 2). In contrast, the lag phase duration of the infected bacterial culture was significantly longer compared to the control (p < 0.0001). The longer lag phase duration was observed regardless of the bacteriophage applied.\\\\nFigure 1. Average duration of the lag phase in infected and non-infected (control) planktonic Pseudomonas aeruginosa culture after phage treatment at various (A) multiplicities of infection (MOIs), (B) temperatures, (C) pH values, and (D) salinities. The standard test conditions were MOI 10, 37°C, pH 7, and salinity 3.5% when these parameters were not varied. Bars indicate the standard deviation among the three biological replicates. Letter a indicates statistical difference between each treatment and the respective control at p < 0.0006.\\\\nTable 2. Average specific bacterial growth rate (h−1) for infected Pseudomonas aeruginosa culture in the presence of bacteriophages (P1–P7) and in non-infected culture (C).\\\\nAt a MOI of 10, the difference in the lag phase duration between bacteriophage-treated culture and control was the highest (7.7 h; Figure 1A). Bacterial cultures infected with a MOI of 10 also showed significantly lower specific growth rates (p < 0.0003) compared with the control (Table 2). Infection at MOI 0.1 did not result in any specific growth rate reduction, while a MOI of 1 for phage P4 and P6 was able to reduce specific growth rates (Table 2). As such, subsequent experiments to test infectivity at various temperatures, pH values and salinities were conducted at a MOI value of 10.\\\\nLag phase was at least 3 times longer in bacteriophage-treated cultures compared with the control over all temperatures tested (p < 0.0001) (MOI 10, pH 7, salinity 3.5%), with particularly longer lag phase at 25 and 30°C (Figure 1B). The lag phase was followed by a bacterial growth with lower specific rates compared with the control. Significant reductions in specific growth rates were observed at 25, 30, 37°C (p < 0.0003) and 45°C (p < 0.03) compared with the respective controls.\\\\nA longer lag phase of ca. 9 h was observed in the presence of bacteriophage regardless of the pH at a constant MOI of 10, 37°C, and 3.5% salinity (p < 0.0001; Figure 1C), compared to the ca. 3 h lag phase for controls. Specific growth rate reduction was also observed when P. aeruginosa was infected at various pH values, with a higher decrease at pH 5 and 6 (p < 0.0005) compared with infection at pH 7 and 8 (p < 0.045; Table 2). Similarly, the lag phase duration was significantly prolonged to ca. 10 h (p < 0.0001) at various salinities in the presence of bacteriophages (Figure 1D). One-way ANOVA test showed significant reduction in specific growth rates in the presence of bacteriophages compared to controls at all tested salinities (p < 0.007).\\\\nBacteriophage Infectivity Against P. aeruginosa-Enriched Biofilm\\\\nPseudomonas aeruginosa-enriched seawater biofilm was infected separately with three bacteriophages (P1, P5, and P7) at four different temperatures (25, 30, 37, and 45°C) and pH values (5, 6, 7, and 8) for 10 h. At all tested temperatures and upon infection with bacteriophages, the number of viable bacterial cells recovered from Pseudomonas isolation agar decreased significantly to an average of 2.8 × 105 CFU/cm2 (p < 0.0001) compared with the initial number (1.37 × 106 CFU/cm2) recovered from the membranes (Figure 2A). In contrast, the controls increased significantly to an average of 6.0 × 106 CFU/cm2 and the number of bacterial colonies was significantly higher than the initial number (p < 0.04). Most of the bacteriophages, specifically P7, showed lower PFU recovery at high temperature of 45°C compared with the other temperatures (Figure 2B). Bacteriophage P7, in particular, had significantly lower PFU counts than the other bacteriophages at 45°C. The number of PFU counts for P7 was also significantly different from the PFU counts obtained after infection at 25, 30, and 37°C (p < 0.0001 Figure 2B).\\\\nFigure 2. Effect of phage treatment against Pseudomonas aeruginosa enriched biofilm. The effect was evaluated in terms of (A) viable colony counts on Pseudomonas isolation agar at various temperatures, (B) recovery of plaque counts after 10 h bacteriophage exposure at various temperatures, (C) viable colony counts on Pseudomonas isolation agar at various pH values, and (D) the recovery of plaque counts after 10 h bacteriophage exposure at various pH values. Dashed red lines indicate the number of CFU before infection or the number of PFU spiked for biofilm infection. The standard test conditions were MOI 10, 37°C, pH 7, and salinity 3.5% when these parameters were not varied. Bars indicate standard deviation among the three biological replicates. Letters indicate statistical difference between each treatment and the original spiked amount as indicated by red dash lines (a: p < 0.0006, b: p < 0.007, c: p < 0.05).\\\\nAt all tested pH values, the number of bacterial colonies in infected biofilm decreased significantly to an average of 4.9 × 104 CFU/cm2 compared with the initial spiked numbers of 1.7 × 106 CFU/cm2 (p < 0.001; 1.5–2 logs reduction). This is in contrast with the bacterial cell count observed for the controls, where the number of bacterial colonies was significantly higher than the initial numbers (p < 0.0001; Figure 2C). Phages were able to replicate by 1.6–13 times of the initial number (1.7 × 107 PFU/cm2) at all tested pH values (p < 0.05; Figure 2D).\\\\nThe Effect of Phage Treatment on Biofilm Microbial Community Based on 16S rRNA Gene-Based Amplicon Sequencing Data Analysis\\\\nOverall Proteobacteria was the most abundant phylum with an average relative abundance of 95%. At genus level, Vibrio and unclassified Vibrionaceae together accounted for ca. 34.5–65% of the microbial community regardless of the tested condition. Pseudoalteromonas was found to be present at high relative abundance of up to 37.3% of total microbial community, while Alteromonas, Arcobacter, and Thalassospira showed a relative abundance lower than 10%. Unclassified Pseudomonadaceae and Pseudomonas spp. were present in all samples at an average relative abundance of 0.75 and 0.62%, respectively. The relative abundance of these unclassified Pseudomonadaceae and Pseudomonas spp. in infected biofilm ranged from 0.01 to 0.61% of the total community, while in the controls they ranged from 0.17 to 5.58% (Table 3). Specifically, at 25, 30, and 45°C, the relative abundance of both Pseudomonas-related taxa in bacteriophage-treated samples was significantly lower compared with the control (p < 0.05). Similarly, when the same experiment was carried out at various pH values, the average of relative abundance of both taxa was significantly lower in bacteriophage-infected samples compared with the controls at all tested pH values (p < 0.03).\\\\nTable 3. Average of the relative abundance in percentage among the three biological replicates of two Pseudomonas-related taxa (Unclassified Pseudomonadaceae and Pseudomonas).\\\\nFrom SIMPER analysis, the average dissimilarity between infected biofilm and control ranged from 13.8 to 17.4%. Unclassified Pseudomonadaceae and Pseudomonas accounted for the three main populations impacted by bacteriophages (Table S1). Other genera affected by bacteriophages were Pseudoalteromonas, unclassified Alphaproteobacteria, unclassified Planococcaceae, and unclassified Rhodobacteraceae (Table S1). An OTU-based analysis of biofilm infected at various temperatures and pH values further showed that the only OTU associated with genus Pseudomonas that was significantly impacted by the phage treatment shared at least 98% similarity with P. aeruginosa (Table S2). Finally, an OTU of at least 98% similarity with Pseudoalteromonas shioyasakiensis along with other OTUs related to Thalassospira, Alcanivorax, and Aestuatiibacter genera were also affected by bacteriophage application (Table S2).\\\\nEffects of Ultrafiltration (UF) Membrane Cleaning Treatments on Transmembrane Pressure (TMP)\\\\nNon-treated UF membrane showed a rapid increase in its TMP, and reached critical fouling after 130 h of operation. When the first cleaning cycle was implemented at 144 h, the membrane treated with citric acid alone showed a 30% reduction in TMP, while bacteriophage treatment alone resulted in 18% reduction. Bacteriophage in combination with citric acid, regardless of the sequence of treatment, resulted in a TMP decrease of ca. 49%. The same trend was observed for all subsequent cleaning cycles, albeit with decreasing efficacy toward TMP drop with every cycle (Figure 3A). The slopes of TMP increase exhibited by membranes subjected to bacteriophage cleaning alone for all cleaning cycles were significantly lower than those of other treatments (Paired t-test, P < 0.01, Figure S3). There was no statistical difference between the slopes of TMP increase for the respective treatment among the three cleaning cycles (Figure S4).\\\\nFigure 3. Run 1 of the ultrafiltration membrane experiment. (A) Transmembrane pressure (TMP) profiles of ultrafiltration membranes. TMP was normalized against the initial TMP (TMP0) for each membrane. Different treatments were applied, namely CA, Citric acid; PH, Phage; CA+PH, Citric acid followed by phage; PH+CA, Phage followed by citric acid; C, no treatment. Dashed red lines indicate the point at which treatment was applied over three different cycles. (B) Bacterial sell numbers from plate counts in terms of CFU/mL, (C) and from flow cytometry in terms of cell number/mL, were measured on three 2 × 2 cm pieces for each membrane at the end of the experiment. Bars indicate standard deviation among the three biological replicates. Letters indicate statistical difference compared with the control (a: p < 0.0006, b: p < 0.007).\\\\nThe lower TMP was due to a lower average number of bacterial cells attached on the membranes after the cleaning treatment. To exemplify, the average number of bacterial cells on non-treated membrane fragment was 8.3 × 106 CFU/cm2. Instead, all treatments significantly reduced both the bacterial colony counts and total cell counts (p < 0.0001), with the highest log reduction of 1.1-logs observed when the combined cleaning was conducted (Figures 3B,C). Lower TMP and cell counts were also observed in the duplicate experiment (Figure S5).\\\\nBacteriophage Genome Size and Morphology\\\\nThe genome sizes of bacteriophages P1, P5, and P7 were determined through pulse field gel electrophoresis (Figure S6), and were observed to share a similar genome size of approximately 48 kbp. This genome size fits in the reported range of both Podoviridae and Siphoviridae families (41.6–79.4 and 34.5–61.1 kbp, respectively; Pires et al., 2015).\\\\nBacteriophages P1, P5, and P7 were further characterized for their morphology through transmission electron microscopy (TEM). Bacteriophages P1 and P5 had similar morphology, both had an icosahedral head of ~55 and ~47 nm, respectively, and short tail of ~12 and ~9 nm, respectively (Figures S7, S8). Phage P7 was often found in agglomerates (Figure S9) and showed ~60 nm icosahedral head and with no easily identifiable tail (Figure S10). Following the Ackermann classification (Ackermann, 2006), these morphological traits suggest that the three bacteriophages could possibly belong to the order Caudovirales, family Podoviridae, as characterized by icosahedral head with short or no tail.\\\\nDiscussion\\\\nIn this study, the feasibility of applying bacteriophages to mitigate biofouling caused by P. aeruginosa in applications involving seawater is investigated. Infectivity against P. aeruginosa in both planktonic phase and biofilm were systematically studied under a range of salinity, temperature, and pH conditions that are representative of different stages of seawater desalination process. In addition, bacteriophages when applied in combination with conventional citric acid cleaning procedure were found to be effective in reducing the transmembrane pressure increment.\\\\nTo infect bacterial hosts, bacteriophages first attach to host cells and integrate their DNA into the host genome without killing the hosts. Subsequently, in response to environmental or other signal cues from the bacterial hosts (Echols, 1972), bacteriophages then enter into lytic infection mode and result in the death of bacterial hosts through the induction of a suite of proteins including holins, endolysins, and spanins. These proteins collectively disrupt the cell membranes and achieve cell lysis (Young, 2013, 2014). Furthermore, bacteriophages possess enzymes that degrade the extracellular polymeric substances of biofilm matrix (Harper et al., 2014), in turn dispersing the biofilm structure to increase the susceptibility of bacterial cells to biocides or chemical cleaning. This is especially relevant for tackling membrane fouling issues in SWRO plants since traditional chemical cleaning and chlorination have limited effect in eradicating bacteria such as Pseudomonas spp. and SRB (Khan et al., 2015; Hong et al., 2016), both of which have been found to be associated with membrane fouling and production of corrosive hydrogen sulfide (Hong et al., 2016).\\\\nMost host-specific bacteriophages like the ones isolated in this study exhibit lytic actions that are specific against a limited number of hosts, and the use of host-specific bacteriophages can potentially minimize unintentional ecological impacts on the indigenous microbial community present within the ecosystem. However, to facilitate the use of bacteriophages as antifouling agent for seawater applications, parameters such as bacteriophage-host ratio, temperature, pH, and salinity need to be taken in consideration. In natural marine environments, bacteriophages distribution and proliferation are determined by the productivity and density of the specific host population. In our study, the highest lag phase extension and slowest specific growth rate were also observed for P. aeruginosa infected at a MOI of 10 (Figure 1A and Table 2) and in saline media (Figure 1D and Table 2). In natural marine environments, bacteriophages distribution and proliferation are determined by the productivity and density of the specific host population. In this study, a MOI value of 10 was the optimal for bacteriophage infection. It is likely that this MOI value maximized encounter rates between bacteriophage and bacterial cell, hence ensuring that nearly all host cells are infected by at least one bacteriophage particle.\\\\nBesides an optimal bacteriophage-host ratio, temperature plays a fundamental role in governing the attachment, proliferation, and cell lysis efficiencies. Attachment is dependent on capsid proteins of bacteriophages and surface proteins on bacterial hosts, both of which are sensitive to high temperatures. The findings of this work suggest that the isolated bacteriophages were able to infect within a broad temperature range of 25–45°C. However, the highest tested temperature resulted in a lower reduction in specific growth rates of P. aeruginosa compared to the other temperatures (Table 2). The lower lytic efficiency at 45°C can be accounted for by a lower bacterial growth and replication (Figure 2A), which may have provided lower surface area for bacteriophage adsorption (Sillankorva et al., 2004). Also, the high temperature of 45°C affected bacteriophage replication as evidenced from the lower PFU recovery compared to other temperatures (Figure 2B). Among the isolated bacteriophages, P7 seemed to be particularly sensitive to high temperature of 45°C. The PFU recovery for this bacteriophage showed a complex trend in which it first decreased at 30°C compared with the one at 25°C (p = 0.004) and it subsequently increased again at 37°C (p = 0.04) before decreasing at 45°C (p < 0.0001). The exact reasons to account for this cyclical behavior are not known. However, it may be possible that this bacteriophage was more affected than P1 and P5 by the different host metabolic activity at different temperatures. Bacterial density and metabolic activities are, in fact, key factors for an efficient bacteriophage proliferation (Chibani-Chennoufi et al., 2004). High temperatures can also result in lower infection efficiency due to alteration made in the capsid proteins and bacterial surface receptors, hence inhibiting the initial phage attachment (Sillankorva et al., 2004). Similarly, Kwiatek and colleagues observed a 1 order of magnitude reduction in the virus particles concentration when bacteriophages from Podoviridae family were maintained at 50°C (Kwiatek et al., 2015).\\\\nIn addition to temperature, pH is another important parameter that affects bacteriophage activity. Environmental pH can change the charge state of the amino acids in bacteriophage capsids, which in turn lead to modifications in the capsid structure (Nap et al., 2014). Moreover, at pH close to the isoelectric point, virus aggregation was observed in MS2 bacteriophage (Langlet et al., 2007), which can contribute to significant bias during PFU enumeration. Despite this, the pH range analyzed in this study exhibited no apparent effect on bacteriophage infectivity against both planktonic cells (Figure 1C, Table 2) and biofilm (Figures 2C,D), suggesting the applied pH was feasible for application in SWRO, which is generally operated within this pH range.\\\\nIn an earlier study, bacteriophages were stored at a similar pH range and were determined to be stable (Kwiatek et al., 2015). Coincidentally, both in this and the earlier study, isolated lytic bacteriophages were possibly associated with the Podoviridae family. Podoviridae were shown to survive large temperature fluctuations compared to bacteriophages with smaller genome sizes (Jonczyk et al., 2011). A complete genome sequencing of the isolated bacteriophages is not done for this study and it was therefore not possible to provide better and definitive information about bacteriophage classification and infection mechanisms at different conditions. However, based on morphological traits, Podoviridae are generally characterized by a big icosahedral head and short tail, which were observed for the bacteriophages in this study. These phenotypic characteristics observed may have accounted for the stability of the bacteriophages observed over the wide range of temperatures and pH values. A complete genomic and taxonomical characterization for all bacteriophages used in this study would be necessary to confirm this hypothesis.\\\\nWhen evaluated for their host specificities, it was determined that the bacteriophages were not only able to delay the growth of planktonic cells, but were also able to statistically reduce the relative abundance of Pseudomonas-related taxa within the biofilm (Table 3). Specifically, OTU analysis showed that P. aeruginosa was detrimentally impacted by the bacteriophages (Table S2). However, it was also observed that other taxa such as unclassified Alphaproteobacteria and Rhodobacteraceae decreased in their relative abundance. Bacteriophages used in this study showed no infectivity against non-P. aeruginosa bacterial hosts. This means that the effect on other bacteria within the biofilm matrix was likely due to an indirect action rather than a direct virus infection. Indeed, these two taxa are within the most abundant Proteobacteria phylum, and the reduction may be due to the loss of bacterial cells upon bacteriophage-induced biofilm disruption. Pseudoalteromonas was also detrimentally impacted by bacteriophage application. This genus has been reported to play an important role in marine biofilm by contributing to the EPS production (Dheilly et al., 2010; Liu et al., 2016) and its decrease in relative abundance reiterate the effect of bacteriophages on biofilm matrix. Finally, the decrease in relative abundance of Gram-positive Planococcaceae could be due to a higher susceptibility to phage lysins since in the absence of outer membrane, those proteins can make direct contact with cell wall carbohydrates and peptidoglycans even if Planococcaceae is not the target host (Fischetti, 2005).\\\\nDeveloping resistance to bacteriophage remains to be of a concern. This was exemplified when bacteria eventually overcame the bacteriophage to grow albeit at a lower specific growth rate compared to the same host grown in absence of bacteriophage (Table 2). Similar trend was also observed in a separate study for P. aeruginosa infected with Podoviridae phages (Alves et al., 2016). The bacterial growth after a prolonged lag phase indicates the evolution of resistance to bacteriophage infection after a certain period of bacteria-virus interaction. Small gene mutation or change/loss of receptor proteins among a subpopulation of cells can result in the loss of bacteriophage infectivity (Ly-Chatain, 2014). Bacterial resistance to bacteriophage infection can also occur by the production of competitive inhibitors that bind to phage receptor or by preventing phage DNA entry (Labrie et al., 2010). It was observed that P. aeruginosa phage resistance can affect also the expression of virulence factors (Hosseinidoust et al., 2013a,b) but incurs a fitness cost for the bacteria in terms of specific growth rate and metabolic activity (Hall et al., 2012). This can potentially account for the lower specific bacterial growth rates observed in presence of bacteriophages (Table 2). Besides gaining resistance, bacterial regrowth could possibly be related to the establishment of phage lysogeny which leads to the presence of prophages in the bacterial cells instead of cell lysis, hence allowing bacterial regrowth. Further studies would have to be carried out to determine the cause of this bacterial regrowth observed in presence of bacteriophages.\\\\nSubsequently, when bacteriophages were applied to the P. aeruginosa-enriched biofilm attached on UF membranes, it was observed that the starting TMP at each initial point of cleaning cycle was higher than the previous cycle (Figures S4, S5). This suggests a slight loss in bacteriophage efficacy with each application. Since the slopes of TMP increase after each cleaning cycle were not significantly different, the increase of initial TMP at each cleaning cycle was most likely not due to changes in metabolic rates or infectivity rates. Instead, this can potentially be explained by an increased production of EPS that would provide a physical barrier between bacteriophages and their receptors (Labrie et al., 2010). Earlier studies have reported a higher reduction in TMP decline compared to this study (i.e., ca. 18% decrease). For instance, Goldman and colleagues observed a 47% reduction in the UF permeability drop after phage application compared with the control (Goldman et al., 2009). However, bacteriophages were introduced in the feed solution and not directly on the formed biofilm, and the higher reduction may be due to a combined effect on both planktonic and biofilm-associated bacteria. In a full-scale SWRO, direct application of bacteriophage to a large volume of seawater would be impractical and costly as this means a large dose of bacteriophage would be required to maintain infectivity at an optimal MOI. In another study, Bhattacharjee et al. observed a 53 and 78% flux recovery after 1 and 2 days of phage application, respectively (Bhattacharjee et al., 2015). However, hollow-fiber membranes were conditioned with nutrient rich medium instead of environmental waters, and both bacteriophages and host may be more metabolically active to facilitate interactions.\\\\nIsolation of new bacteriophages or application of phages cocktail can be adopted to mitigate problems associated with phage resistance and lysogeny. However, isolating new bacteriophages can be challenging and time-consuming for long-term operations. Alternatively, this study demonstrated the feasibility of combining bacteriophages application with the common membrane cleaning procedure. Approximately 49% TMP recovery and 1.1 log biofilm removal was observed when UF membrane was treated with bacteriophages and citric acid, regardless of the order of application (Figure 3). This confirms that bacteriophages are not affected by the acidic pH and that they can be easily integrated with the common membrane treatment procedure to enhance the cleaning efficacy. Citric acid chelates inorganic minerals and disrupts the stability of biofilm matrix attached on the pretreatment UF membranes (Lee et al., 2001; Porcelli and Judd, 2010). Bacteriophages are able to further disrupt the biofilm structure, hence allowing for a better cleaning efficacy. Similarly, bacteriophages were applied successfully in combination with chlorine treatment against P. aeruginosa biofilm (Zhang and Hu, 2013). These findings suggest that bacteriophage can be applied synergistically with existing biocides or cleaning strategies to effectively mitigate biofouling associated with P. aeruginosa. Alternatively, other bacteriophages specific against other host targets, for example Pseudoalteromonas which is typically present in high abundance on a marine biofilm layer, can also be isolated and verified for their lytic efficiency against this new host based on the procedure described in this study. Finally, it could be interesting to analyze the effect of bacteriophages treatment in preventing biofilm formation rather than removing an established biofilm structure. In this context, the lower cell density could represent a limitation for bacteriophages activity and infectivity. A cocktail of bacteriophages from different taxonomical families targeting different bacterial host could also represent a better approach to maximize the impact of the infection and to further mitigate seawater membrane biofouling in the future for bigger scale operating systems.\\\\nConclusion\\\\nThis study demonstrated the ability of bacteriophages to infect planktonic P. aeruginosa and to reduce biofilm formation. Infectivity at different environmental conditions was systematically evaluated and bacteriophages showed great versatility to infect P. aeruginosa over a wide temperature and pH range. Although the bacteriophages were able to effectively reduce the relative abundance of Pseudomonas-related taxa, other taxa including Pseudoalteromonas and predominant Proteobacteria within a marine biofilm were also affected, suggesting that the bacteriophages lyse P. aeruginosa and disrupt the biofilm matrix. Finally, bacteriophages were demonstrated to be feasible for reducing P. aeruginosa biofouling on a lab-scale UF membrane. Specifically, the best reduction in transmembrane pressure was obtained when bacteriophage treatment was combined with citric acid cleaning. Collectively, the findings demonstrate that bacteriophages can be used as a biocidal agent to mitigate undesirable P. aeruginosa-associated problems in seawater applications.\\\\nAuthor Contributions\\\\nGS and P-YH: Designed the experiment; GS and SY: Performed the experiments and analyzed the data; GS, P-YH, and AK: Wrote and revised the manuscript.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nThe research reported in this publication was supported by CRG funding URF/1/2982-01-01 from King Abdullah University of Science and Technology (KAUST) awarded to P-YH. AK thanks KAUST and CSIRO Land and Water for financial support.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2018.00875/full#supplementary-material\\\\nTable S1. Similarity percentages (SIMPER) analysis of the average composition of the microbial communities in phage infected and non-infected biofilms.\\\\nTable S2. OTUs affected by bacteriophage infection are listed together with the closest match identified by the BLASTN algorithm.\\\\nFigure S1. Phage forming units for each bacteriophage propagated with the double layer plate method. Bars indicate standard deviation.\\\\nFigure S2. Cross-flow ultrafiltration membrane setup.\\\\nFigure S3. Values of TMP/TMP0 increment slopes, averaged from the three cycles of UF membrane cleaning. a, b represent homogenous subgroups by two-tailed t-test.\\\\nFigure S4. Values of TMP/TMP0 increment slopes for each cycle of membrane cleaning.\\\\nFigure S5. Run 2 of the ultrafiltration membrane experiment. (A) Transmembrane pressure (TMP) profiles of ultrafiltration membranes. TMP was normalized against the initial TMP (TMP0) for each membrane. Different treatments were applied, namely CA, Citric acid; PH, Phage; CA+PH, Citric acid followed by phage; PH+CA, Phage followed by citric acid; C, no treatment. Dashed red lines indicate the point at which treatment was applied over three different cycles. (B) Cell numbers from plate counts in terms of CFU/mL, (C) and from flow cytometry in terms of cells number/mL, were measured on three 2 × 2 cm pieces for each membrane at the end of the experiment. Bars indicate standard deviation among the three biological replicates. Letters indicate statistical difference compared with the control (a: p < 0.0006, b: p < 0.007 and c: p < 0.05).\\\\nFigure S6. Pulsed field gel electrophoresis images of P1, P5, and P7 genomes. λ and a indicate lambda and 1 kbp ladder respectively used as markers.\\\\nFigure S7. Transmission electron microscopy images of the isolated bacteriophage P1.\\\\nFigure S8. Transmission electron microscopy images of the isolated bacteriophage P5.\\\\nFigure S9. Transmission electron microscopy images of the isolated bacteriophage P7 in aggregate.\\\\nFigure S10. Transmission electron microscopy images of the isolated bacteriophage P7.\\\\nReferences\\\\nAckermann, H. W. (2006). “Classification of bacteriophages,” in The Bacteriophages, ed R. Calendar (New York, NY: Oxford University Press), 8–16.\\\\nGoogle Scholar\\\\nAdams, M. H. (1959). Bacteriophages. New York, NY: Interscience Publishers Inc.\\\\nGoogle Scholar\\\\nAlemayehu, D., Casey, P. G., McAuliffe, O., Guinane, C. M., Martin, J. G., Shanahan, F., et al. (2012). Bacteriophages φMR299-2 and φNH-4 can eliminate Pseudomonas aeruginosa in the murine lung and on cystic fibrosis lung airway cells. MBio 3, e00029–e00012. doi: 10.1128/mBio.00029-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAl-Jassim, N., Ansari, M. I., Harb, M., and Hong, P.-Y. (2015). Removal of bacterial contaminants and antibiotic resistance genes by conventional wastewater treatment processes in Saudi Arabia: is the treated wastewater safe to reuse for agricultural irrigation? Water Res. 73, 277–290. doi: 10.1016/j.watres.2015.01.036\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAlves, D. R., Perez-Esteban, P., Kot, W., Bean, J., Arnot, T., Hansen, L. H., et al. (2016). A novel bacteriophage cocktail reduces and disperses Pseudomonas aeruginosa biofilms under static and flow conditions. Microb. Biotechnol. 9, 61–74. doi: 10.1111/1751-7915.12316\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBhattacharjee, A. S., Choi, J., Motlagh, A. M., Mukherji, S. T., and Goel, R. (2015). Bacteriophage therapy for membrane biofouling in membrane bioreactors and antibiotic-resistant bacterial biofilms. Biotechnol. Bioeng. 112, 1644–1654. doi: 10.1002/bit.25574\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChibani-Chennoufi, S., Bruttin, A., Dillmann, M.-L., and Brüssow, H. (2004). Phage-host interaction: an ecological perspective. J. Bacteriol. 186, 3677–3686. doi: 10.1128/JB.186.12.3677-3686.2004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nClarke, K. G. R. (2015). PRIMER Version 7: User Manual/Tutorial. Plymouth: PRIMER-E.\\\\nDheilly, A., Soum-Soutéra, E., Klein, G. L., Bazire, A., Compère, C., Haras, D., et al. (2010). Antibiofilm activity of the marine bacterium Pseudoalteromonas sp. strain 3J6. Appl. Environ. Microbiol. 76, 3452–3461. doi: 10.1128/AEM.02632-09\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nEdgar, R. C., Haas, B. J., Clemente, J. C., Quince, C., and Knight, R. (2011). UCHIME improves sensitivity and speed of chimera detection. Bioinformatics 27, 2194–2200. doi: 10.1093/bioinformatics/btr381\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nFritzmann, C., Löwenberg, J., Wintgens, T., and Melin, T. (2007). State-of-the-art of reverse osmosis desalination. Desalination 216, 1–76. doi: 10.1016/j.desal.2006.12.009\\\\nCrossRef Full Text | Google Scholar\\\\nFu, W., Forster, T., Mayer, O., Curtin, J. J., Lehman, S. M., and Donlan, R. M. (2010). Bacteriophage cocktail for the prevention of biofilm formation by Pseudomonas aeruginosa on catheters in an in vitro model system. Antimicrob. Agents Chemother. 54, 397–404. doi: 10.1128/AAC.00669-09\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGao, W., Liang, H., Ma, J., Han, M., Chen, Z.-L., Han, Z.-S., et al. (2011). Membrane fouling control in ultrafiltration technology for drinking water production: a review. Desalination 272, 1–8. doi: 10.1016/j.desal.2011.01.051\\\\nCrossRef Full Text | Google Scholar\\\\nGoldman, G., Starosvetsky, J., and Armon, R. (2009). Inhibition of biofilm formation on UF membrane by use of specific bacteriophages. J. Memb. Sci. 342, 145–152. doi: 10.1016/j.memsci.2009.06.036\\\\nCrossRef Full Text | Google Scholar\\\\nGoosen, M., Sablani, S., Al-Hinai, H., Al-Obeidani, S., Al-Belushi, R., and Jackson, D. (2005). Fouling of reverse osmosis and ultrafiltration membranes: a critical review. Sep. Sci. Technol. 39, 2261–2297. doi: 10.1081/SS-120039343\\\\nCrossRef Full Text | Google Scholar\\\\nGreenlee, L. F., Lawler, D. F., Freeman, B. D., Marrot, B., and Moulin, P. (2009). Reverse osmosis desalination: water sources, technology, and today\\'s challenges. Water Res. 43, 2317–2348. doi: 10.1016/j.watres.2009.03.010\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHall, A. R., De Vos, D., Friman, V.-P., Pirnay, J.-P., and Buckling, A. (2012). Effects of sequential and simultaneous applications of bacteriophages on populations of Pseudomonas aeruginosa in vitro and in wax moth larvae. Appl. Environ. Microbiol. 78, 5646–5652. doi: 10.1128/AEM.00757-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHarper, D. R., Parracho, H. M., Walker, J., Sharp, R., Hughes, G., Werthén, M., et al. (2014). Bacteriophages and biofilms. Antibiotics 3, 270–284. doi: 10.3390/antibiotics3030270\\\\nCrossRef Full Text | Google Scholar\\\\nHong, P.-Y., Moosa, N., and Mink, J. (2016). Dynamics of microbial communities in an integrated ultrafiltration–reverse osmosis desalination pilot plant located at the Arabian Gulf. Desalination Water Treat. 57, 16310–16323. doi: 10.1080/19443994.2015.1083483\\\\nCrossRef Full Text | Google Scholar\\\\nHosseinidoust, Z., Tufenkji, N., and Van De Ven, T. G. (2013a). Predation in homogeneous and heterogeneous phage environments affects virulence determinants of Pseudomonas aeruginosa. Appl. Environ. Microbiol. 79, 2862–2871. doi: 10.1128/AEM.03817-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHosseinidoust, Z., Van De Ven, T. G., and Tufenkji, N. (2013b). Evolution of Pseudomonas aeruginosa virulence as a result of phage predation. Appl. Environ. Microbiol. 79, 6110–6116. doi: 10.1128/AEM.01421-13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKhan, M. T., Hong, P.-Y., Nada, N., and Croue, J. P. (2015). Does chlorination of seawater reverse osmosis membranes control biofouling? Water Res. 78, 84–97. doi: 10.1016/j.watres.2015.03.029\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKot, W., Vogensen, F. K., Sørensen, S. J., and Hansen, L. H. (2014). DPS–a rapid method for genome sequencing of DNA-containing bacteriophages directly from a single plaque. J. Virol. Methods 196, 152–156. doi: 10.1016/j.jviromet.2013.10.040\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKwiatek, M., Mizak, L., Parasion, S., Gryko, R., Olender, A., and Niemcewicz, M. (2015). Characterization of five newly isolated bacteriophages active against Pseudomonas aeruginosa clinical strains. Folia Microbiol. 60, 7–14. doi: 10.1007/s12223-014-0333-3\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLanglet, J., Gaboriaud, F., and Gantzer, C. (2007). Effects of pH on plaque forming unit counts and aggregation of MS2 bacteriophage. J. Appl. Microbiol. 103, 1632–1638. doi: 10.1111/j.1365-2672.2007.03396.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLee, H., Amy, G., Cho, J., Yoon, Y., Moon, S.-H., and Kim, I. S. (2001). Cleaning strategies for flux recovery of an ultrafiltration membrane fouled by natural organic matter. Water Res. 35, 3301–3308. doi: 10.1016/S0043-1354(01)00063-X\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLe Roux, J., Nada, N., Khan, M. T., and Croué, J.-P. (2015). Tracing disinfection byproducts in full-scale desalination plants. Desalination 359, 141–148. doi: 10.1016/j.desal.2014.12.035\\\\nCrossRef Full Text | Google Scholar\\\\nLiu, A., Mi, Z.-H., Zheng, X.-Y., Yu, Y., Su, H.-N., Chen, X.-L., et al. (2016). Exopolysaccharides play a role in the swarming of the benthic bacterium Pseudoalteromonas sp. SM9913. Front. Microbiol. 7:473. doi: 10.3389/fmicb.2016.00473\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMatin, A., Khan, Z., Zaidi, S., and Boyce, M. (2011). Biofouling in reverse osmosis membranes for seawater desalination: phenomena and prevention. Desalination 281, 1–16. doi: 10.1016/j.desal.2011.06.063\\\\nCrossRef Full Text | Google Scholar\\\\nMcVay, C. S., Velásquez, M., and Fralick, J. A. (2007). Phage therapy of Pseudomonas aeruginosa infection in a mouse burn wound model. Antimicrob. Agents Chemother. 51, 1934–1938. doi: 10.1128/AAC.01028-06\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nNap, R. J., BoŽič, A. L., Szleifer, I., and Podgornik, R. (2014). The role of solution conditions in the bacteriophage PP7 capsid charge regulation. Biophys. J. 107, 1970–1979. doi: 10.1016/j.bpj.2014.08.032\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nOh, H.-S., Tan, C. H., Low, J. H., Rzechowicz, M., Siddiqui, M. F., Winters, H., et al. (2017). Quorum quenching bacteria can be used to inhibit the biofouling of reverse osmosis membranes. Water Res. 112, 29–37. doi: 10.1016/j.watres.2017.01.028\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nOlszak, T., Zarnowiec, P., Kaca, W., Danis-Wlodarczyk, K., Augustyniak, D., Drevinek, P., et al. (2015). In vitro and in vivo antibacterial activity of environmental bacteriophages against Pseudomonas aeruginosa strains from cystic fibrosis patients. Appl. Microbiol. Biotechnol. 99, 6021–6033. doi: 10.1007/s00253-015-6492-6\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPeñate, B., and García-Rodríguez, L. (2012). Current trends and future prospects in the design of seawater reverse osmosis desalination technology. Desalination 284, 1–8. doi: 10.1016/j.desal.2011.09.010\\\\nCrossRef Full Text | Google Scholar\\\\nPires, D. P., Boas, D. V., Sillankorva, S., and Azeredo, J. (2015). Phage therapy: a step forward in the treatment of Pseudomonas aeruginosa infections. J. Virol. 89, 7449–7456. doi: 10.1128/JVI.00385-15\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPorcelli, N., and Judd, S. (2010). Chemical cleaning of potable water membranes: a review. Sep. Purif. Technol. 71, 137–143. doi: 10.1016/j.seppur.2009.12.007\\\\nCrossRef Full Text | Google Scholar\\\\nSanawar, H., Xiong, Y., Alam, A., Croue, J.-P., and Hong, P.-Y. (2017). Chlorination or monochloramination: balancing the regulated trihalomethane formation and microbial inactivation in marine aquaculture waters. Aquaculture 480, 94–102. doi: 10.1016/j.aquaculture.2017.08.014\\\\nCrossRef Full Text | Google Scholar\\\\nScarascia, G., Cheng, H., Harb, M., and Hong, P.-Y. (2017). Application of hierarchical oligonucleotide primer extension (HOPE) to assess relative abundances of ammonia-and nitrite-oxidizing bacteria. BMC Microbiol. 17:85. doi: 10.1186/s12866-017-0998-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSillankorva, S., Oliveira, R., Vieira, M. J., Sutherland, I., and Azeredo, J. (2004). Pseudomonas fluorescens infection by bacteriophage ΦS1: the influence of temperature, host growth phase and media. FEMS Microbiol. Lett. 241, 13–20. doi: 10.1016/j.femsle.2004.06.058\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWang, Q., Garrity, G. M., Tiedje, J. M., and Cole, J. R. (2007). Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy. Appl. Environ. Microbiol. 73, 5261–5267. doi: 10.1128/AEM.00062-07\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nZhang, Y., Hunt, H. K., and Hu, Z. (2013). Application of bacteriophages to selectively remove Pseudomonas aeruginosa in water and wastewater filtration systems. Water Res. 47, 4507–4518. doi: 10.1016/j.watres.2013.05.014\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2020.580779\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2020.580779/full\",\\n    \"loadedTime\": \"2024-01-23T08:49:58.786Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2020.580779\",\\n    \"title\": \"Frontiers | Identification and Characterization of New Bacteriophages to Control Multidrug-Resistant Pseudomonas aeruginosa Biofilm on Endotracheal Tubes\",\\n    \"description\": \"Studies involving antimicrobial-coated endotracheal tubes are scarce, and new approaches to control multidrug-resistant Pseudomonas aeruginosa biofilm on these devices should be investigated. In this study, five new P. aeruginosa bacteriophages from domestic sewage were isolated. All of them belong to the order Caudovirales, Myoviridae family. They are pH and heat stable and produce from 27 to 46 particles after a latent period of 30 min at 37oC. Their dsDNA genome (ranging from ⁓62 kb to ⁓65 kb), encode from 65 to 89 different putative proteins. They exhibit a broad lytic spectrum and infect 69.7% of the P. aeruginosa strains tested. All bacteriophages were able to reduce the growth of P. aeruginosa strains in the planktonic form. Bacteriophages were also able to reduce biofilm viability rates and metabolic activity of P. aeruginosa strains in a model of biofilms associated with endotracheal tubes. In addition, scanning electron microscopy micrographs showed disrupted biofilms and cell debris after treatment of bacteriophages revealing a remarkable biofilm reduction. The lytic activity on multidrug-resistant P. aeruginosa biofilm indicates that the isolated bacteriophages might be considered good candidates for therapeutic studies and for the application of bacteriophage-encoded products.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nMechanical ventilation through an endotracheal or tracheostomy tube is a supportive intervention to critically ill patients and represents a risk factor for the occurrence of ventilator-associated pneumonia (VAP; Haas et al., 2014). VAP is the most common healthcare-associated infection in critically ill patients, involves high financial costs, and has negative prognosis (Barbier et al., 2013; Zimlichman et al., 2013; Russotto et al., 2015). VAP etiologies have been widely discussed, and the factors associated with its development include endotracheal tube biofilm formation which plays an important role as a reservoir for microorganisms (Nseir et al., 2011).\\\\nKnowing the potential risk factor of multidrug-resistant Pseudomonas aeruginosa biofilms on endotracheal tubes, interest in the development of antimicrobial-coated endotracheal tubes has emerged. The presence of antimicrobial agents, such as silver particles and chlorhexidine, on the surface of endotracheal tubes demonstrated a decrease in biofilm formation on the device surface’s (Berra et al., 2004; Tokmaji et al., 2015). However, chemical compounds have been implicated in bacterial resistance (Cieplik et al., 2019) and frequently linked to potential cytotoxic hazards (Lessa et al., 2010; Mao et al., 2018). Therefore, new approaches to control multidrug-resistant P. aeruginosa biofilm on endotracheal tubes should be investigated.\\\\nThe applicability of bacteriophages in medical devices has aroused interest in several areas, including infections related to the urinary tract (Nzakizwanayo et al., 2015; Melo et al., 2016), venous catheters (Lungren et al., 2014), and orthopedic prostheses (Kaur et al., 2014; Barros et al., 2020). Recently, new lytic bacteriophages able to control multidrug-resistant P. aeruginosa planktonic cells and associated-biofilm forms were described (Yuan et al., 2019; Adnan et al., 2020). Although promising, there are few reports on bacteriophage applicability to control biofilms associated with endotracheal tubes (Aleshkin et al., 2016). Thus, we consider it essential to clarify whether newly isolated bacteriophages can be applied in order to control P. aeruginosa biofilms on the surface of endotracheal tubes. The null hypothesis of this study is that there would not be a difference in P. aeruginosa growth when challenged with bacteriophages.\\\\nMaterials and Methods\\\\nBacteriophage Isolation and Propagation\\\\nBacteriophages were isolated from domestic sewage, and P. aeruginosa (ATCC 27853) was used as the host strain. Isolation, sample enrichment, and purification methods followed the protocol described previously (Sillankorva et al., 2008). Briefly, 20 ml of domestic effluents, collected from March to November 2018 (Domestic Sewage Treatment Station, Ribeirão Preto, Brazil and Araxá, Brazil), was mixed to 20 ml of double-strength tryptic soy broth (TSB; BD Difco, Sparks, MN, United States) and 100 μl of exponentially grown P. aeruginosa. The presence of bacteriophage was confirmed by inhibition haloes on double-layer agar plating [Tryptic Soy Agar Soft (0.8% agar)—TSAS (BD Difco)] (Sambrook and Russell, 2001). Repeated double-layer agar plating was done until single-plaque morphology was observed. For propagation, one isolated plaque was collected through a sterile toothpick and diluted in 25 μl of elution buffer [1 M Tris–HCl, pH 7.5 (Sigma Aldrich, Saint Louis, MO, United States), 10 mM MgSO4 (Sigma Aldrich), 10 mM NaCl (Dinâmica, Indaiatuba, SP, Brazil), 0.002% (w/v) gelatin (Dinâmica)]. The suspension was dropped onto a TSAS medium, with P. aeruginosa lawns, and incubated at 37°C for 24 h. Afterward, 10 ml of elution buffer was added to each plate to elute the bacteriophages, which were purified with chloroform (Sigma Aldrich), titrated, and stored at 4°C.\\\\nTransmission Electron Microscopy\\\\nOne milliliter of bacteriophage suspension was centrifuged at 21,800 g for 60 min at 4°C. The sediment was washed twice in sterile ultrapure water, and 25 μl of each bacteriophage was applied on an electron microscopy grid. The grids were stained with 2% uranyl acetate for 5 min and then analyzed by transmission electron microscopy (JEOL JEM-100 CXII Akishima, Tokyo, Japan; Melo et al., 2014).\\\\nAcid–Base and Thermal Stability\\\\nIn order to assess acid–base stability, 100 μl of the bacteriophage suspension (108 plaque forming units per milliliter—PFU/ml) were added to 900 μl of elution buffer, with pH adjusted to 3.0, 5.0, 7.0, 9.0, and 11.0. The samples were incubated for 2 h at 37°C. To evaluate thermal stability, 100 μl of the bacteriophage suspension (108 PFU/ml) was added to 900 μl of elution buffer and incubated at different temperatures (37, 42, 50, 60, 70, and 80°C) for 2 h (20). After the incubation period, the phage titer was determined through the double-layer agar method (Cui et al., 2017). The assays were carried out at three independent times in triplicate.\\\\nOne-Step Growth Curve\\\\nAn exponential-phase P. aeruginosa culture was centrifuged at 4,200 g for 5 min, and OD600 was adjusted to achieve ∼1011 colony-forming units per milliliter (CFU/ml). Subsequently, 5 ml of bacteria suspension was added to 5 ml of bacteriophage suspension at 5 × 108 PFU/ml (multiplicity of infection—MOI: 0.005). The mixture was incubated (120 rpm, 37°C, 5 min) and centrifuged (4,200 g, 5 min), and 10 ml of sterile TSB was added to the pellet. Samples were incubated (120 rpm, 37°C), and aliquots were taken every 5 min until 60 min post-infection. Using the double-layer agar plating, the number of bacteriophage particles was determined at each time. Latent period was determined as the time required for the phage to increase in number and burst size as the ratio between the number of bacteriophages after reaching the plateau and the number of infected bacteria (Sillankorva et al., 2008). The assay was carried out at three independent times.\\\\nGenome Sequencing Analysis\\\\nOne milliliter of bacteriophage stock was incubated with 10 U DNAse I (Sigma Aldrich) and 10 μl RNAse A (20 mg/ml; Sigma Aldrich) at 37°C for 1 h. Enzymes were inactivated at 70°C for 15 min. Then, the suspensions were incubated with extraction buffer [40 μl EDTA 0.5 M (Sigma Aldrich), 5 μl proteinase K 10 mg/ml (Sigma Aldrich), and 50 μl SDS 10% (w/v; Sigma Aldrich)]. Extraction was performed with phenol/chloroform/isoamyl alcohol (25:24:1; Sigma Aldrich). Absence of bacterial contaminant DNA was confirmed by amplification of the 16S rRNA P. aeruginosa gene (Spilker et al., 2004).\\\\nLibraries were prepared using the NexteraXT kit (Illumina, San Diego, CA, United States), and genomes were sequenced using Illumina MiSeq paired-end sequencing (2 × 300 bp/coverage 100×), according to the manufacturer’s instructions. Reads were trimmed and assembled into a single contig using Velvet de novo tool (Illumina). Potential coding sequences (CDSs) were initially annotated using myRAST (Aziz et al., 2008), and alternative start codons or non-annotated CDSs were further analyzed using Geneious version 2019.2.1 (Biomatters Ltd., Auckland, New Zealand). Encoded proteins were queried against the National Center for Biotechnology Information (NCBI) non-redundant database with BLASTP (coverage > 80%; E-value ≤ 10–5; Altschul et al., 1990). Protein families, signatures, and structure prediction were searched with PFAM (Finn et al., 2014), InterPro (Finn et al., 2017), and HHpred (Söding, 2005), while the presence of transmembrane domains was checked using TMHMM (Käll and Sonnhammer, 2002), and Phobius (Käll et al., 2004). To predict signal peptide cleavage sites, SignalP (Petersen et al., 2011), and Spoctopus (Viklund et al., 2008) were used. Transfer RNAs (tRNAs) were scanned using tRNAscan-SE (Schattner et al., 2005).\\\\nHost Range Investigation and EOP Analysis\\\\nHost range was evaluated by the presence of clear or turbid inhibition haloes in a spot test on double-layer agar against 26 clinical isolates and seven multidrug-resistant P. aeruginosa strains (Panel ATCC® MP-23TM; Table 1). The susceptibility profile of clinical isolates was determined by the disk diffusion method. Susceptibility test, antimicrobial classes, antibiotic concentration, and zone diameter breakpoint for P. aeruginosa strains were defined according to the Clinical and Laboratory Standards Institute (Clinical and Laboratory Standards Institute [CLSI], 2019).\\\\nTable 1. Lytic spectrum and efficiency of plating of the bacteriophages against sensible and multidrug-resistant Pseudomonas aeruginosa isolated from different sources.\\\\nEfficiency of plating (EOP) was determined for all P. aeruginosa strains positive for bacteriophage lytic activity. Tenfold dilutions of the bacteriophages were spotted on double-layer agar inoculated with P. aeruginosa strains. EOP values were calculated as the ratio between the lysis plaques produced in each susceptible strain and the number of plaques produced in P. aeruginosa ATCC 27853. EOP was recorded as high, low, and inefficient, representing >1, 0.1–1, and <0.1%, respectively (Melo et al., 2014).\\\\nInfection of Planktonic Cells\\\\nTen clinical isolates with the highest EOP values, P. aeruginosa ATCC 27853, and multidrug-resistant P. aeruginosa ATCC 2108, ATCC 2110, ATCC 2112, and ATCC 2113 were selected for this assay. Then, 100 μl of P. aeruginosa strains, OD600 adjusted to achieve ∼109 CFU/ml, was inoculated with 100 μl of each bacteriophage at a concentration of 5 × 107 PFU/ml (MOI: 0.05). The positive and negative control experiments were performed using a dilution buffer instead of bacteriophages and sterile TSB instead of bacterial suspension, respectively. The plates were incubated at 37°C, 80 rpm. The OD600 was measured after 2, 4, 6, 8, and 24 h (Melo et al., 2014). The assays were carried out at three independent times in triplicate.\\\\nAntibiofilm Activity Measurement\\\\nThe same 15 P. aeruginosa strains applied in the infection of planktonic cells test were selected for this assay. The anti-biofilm activity of the bacteriophages was determined by means of biofilm viability rates (CFU/cm2), metabolic activity of the bacteria (XTT), and scanning electron microscopy (SEM). The assays were carried out at three independent times in triplicate.\\\\nFor biofilm infection, strains were grown in TSB medium (37°C, 24 h), and OD600 was adjusted to ∼109 CFU/ml. Circular endotracheal tube specimens (Ø 5 mm; Rüsh, Meridian, MS, United States) were sterilized by hydrogen peroxide plasma (Sterrad 100S, Irvine, CA, United States) and randomly distributed in 48-well tissue culture plates. Each well received 400 μl of TSB medium containing the standardized cell suspension (∼107 CFU/ml). Negative control was performed using sterile TSB. The plates were incubated for 48 h at 37°C and 80 rpm. Afterward, the specimens were removed carefully from the plate, rinsed in 10 ml of phosphate-buffered saline (PBS), and transferred to a new plate with 1 ml of sterile culture medium and 108 PFU/ml of bacteriophages. The control samples received culture medium without bacteriophages. The plates were incubated for 24 h at 37°C and 80 rpm. After the incubation time, the specimens were removed from the wells through sterilized tweezers and rinsed in 10 ml of PBS.\\\\nFor CFU quantification, the specimens were transferred to a tube containing 2 ml of PBS. In order to detach the remaining biofilm, the tubes were vortexed for 30 s, sonicated (200 W, 40 kHz; Altsonic, Clean 9CA, Ribeirão Preto, SP, Brazil) for 20 min, and vortexed again for 2 min. Afterward, 10-fold-dilution aliquots (100–10–6) were seeded in Petri dishes containing TSA medium. The Petri dishes were incubated at 37°C for 24 h, and the number of colonies was registered and expressed as log CFU/cm2.\\\\nFor the evaluation of metabolic activity, the specimens were transferred to another 48-well plate containing 316 μl PBS supplemented with 100 mM glucose (Sigma Aldrich), 80 μl XTT 1 mg/ml (Sigma Aldrich), and 4 μl of 0.4 mM menadione (Sigma Aldrich). The plates were statically incubated, protected from light, at 37°C for 2 h, and the absorbance of the resulting solution was measured (492 nm).\\\\nFor SEM, P. aeruginosa ATCC 27853 was analyzed. The biofilms were removed from the plates and washed twice in distilled water. Samples were fixed with 2.5% glutaraldehyde for 60 min and then dehydrated in a graded ethanol series (30, 50, 70, 90, and 100%). After chemical drying (hexamethyldisilazane; Sigma Aldrich), the specimens were mounted on a metal stub and gold-coated. The surface morphology of the biofilms was examined under high vacuum with a JEOL JSM-35CF microscope (Tokyo, Japan).\\\\nStatistical Analysis\\\\nStatistical tests were performed through the IBM SPSS Statistics 25.0 software (IBM Corp Armonk, NY, United States). The significance level was set to 0.05. All data showed normal distribution and homogeneous variance and were analyzed using two-way ANOVA, with independent levels and Bonferroni post hoc test.\\\\nResults\\\\nBacteriophage Isolation and Propagation\\\\nFive new lytic bacteriophages against P. aeruginosa were isolated from domestic sewage and designated as vB_PaeM_USP_1, vB_PaeM_USP_2, vB_PaeM_USP_3, vB_PaeM_USP_18, and vB_PaeM_USP_25. All bacteriophages were able to form clear plaques, having well-defined boundaries. It was observed that none of the bacteriophages produced double haloes, and it was not possible to differentiate lysis plaques among the different isolated bacteriophages (Figures 1A–E).\\\\nFigure 1. (A–E) Representative image of phage isolates depicting different plaque morphologies of vB_PaeM_USP_1, vB_PaeM_USP_2, vB_PaeM_USP_3, vB_PaeM_USP_18, and vB_PaeM_USP_25. (F–J) Transmission electron micrographs of individual bacteriophages. (K) Panel of plaque characteristics, phage morphology, latent period, and burst size.\\\\nElectron Microscopy\\\\nThe morphological aspects were slightly different among the different isolated viruses (Figures 1F–J). According to the International Committee on Taxonomy of Viruses, all bacteriophages belong to the order Caudovirales and are members of the Myoviridae family.\\\\nAcid–Base and Thermal Stability\\\\nThe values of pH 3.0 and pH 11.0 contributed to reduced bacteriophage viability (p < 0.05). Neutral pH (7.0) and small variations in hydrogenionic potential (pH 5.0 and pH 9.0) did not cause significant changes in viability. vB_PaeM_USP_2 and vB_PaeM_USP_3 were the most resistant to pH variation (Figure 2A). Reduced bacteriophage viability after exposure to 70 and 80°C was noticed. Bacteriophage vB_PaeM_USP_25 was the most resistant to temperature variation (Figure 2B).\\\\nFigure 2. Effects of different pH and temperature values (log10UFP/ml) on vB_PaeM_USP_1, vB_PaeM_USP_2, vB_PaeM_USP_3, vB_PaeM_USP_18, and vB_PaeM_USP_25.\\\\nOne-Step Growth Curve\\\\nA one-step growth curve revealed a similar behavior among the bacteriophages (Supplementary Figure 1). Latent period and yields at burst size are presented in Figure 1K.\\\\nGenome Sequencing Analysis\\\\nThe complete genome sequence of each bacteriophage was deposited in the NCBI database under the accession numbers MT491204–MT491208. Genome analysis revealed that the five bacteriophages are potentially virulent since they do not encode for any known protein associated to lysogeny. Moreover, no known virulence-associated or toxic proteins were identified, revealing that the isolated bacteriophages are potentially safe for therapeutic purposes. All bacteriophage genomes are composed of double-stranded DNA molecules, and no putative genes coding for tRNAs were found. The general genomic properties of the five bacteriophages are presented in Table 2. Two different subgroups of bacteriophages could be observed (Figure 3). The first one, composed of bacteriophages vB_PaeM_USP_1, vB_PaeM_USP_2, and vB_PaeM_USP_3, presents similar genome sizes (approximately 65 kb), G + C content (approximately 55.5%), a high nucleotide pairwise identity (94.6%), and proteome homology (sharing 83 protein homologs between them), with roughly 62% of the proteins with unknown function (Figure 3). Despite their similarity, they present some differences at the beginning (0–5 kb) and at the end of their genome (53 kb end; Figure 4A). Their genomes share high nucleotide identity with other P. aeruginosa bacteriophage members of the Pbunavirus genus, namely, Pseudomonas virus Pa193, Epa22, vB_PaeM_PAO1_Ab27, and BrSP1, with identities above 96% and coverages of up to 98%.\\\\nTable 2. Genomic properties of Pseudomonas aeruginosa bacteriophages.\\\\nFigure 3. Genome map of the five sequenced bacteriophages depicting the predicted open reading frames colored according to their predicted function (black for hypothetical proteins, green for genes involved in DNA replication and transcription, blue for DNA packaging and morphogenesis genes, and red for cell lysis genes). The blue line represents the guanine-cytosine content (%).\\\\nFigure 4. Whole genome alignment (using progressive Mauve algorithm) of the bacteriophage genomes demonstrating the high identity between the two groups. (A) Mauve alignment of the group containing bacteriophages vB_PaeM_USP_1, vB_PaeM_USP_2, and vB_PaeM_USP_3. (B) Mauve alignment of the group containing bacteriophages vB_PaeM_USP_18 and vB_PaeM_USP_25.\\\\nThe second subgroup, composed of vB_PaeM_USP_18 and vB_PaeM_USP_25, has nucleotide pairwise identity of 97.1% and shares 65 homologs, ca. 74% of them without an assigned function (Figure 3). A slightly lower homology is observed in the genomic regions 3–4.5, 12.5–14, and mainly from 39–46.5 kb (Figure 4B). The low homology with genomes from other phages deposited with the NCBI does not allow grouping these phage genomes with any known genus or subfamily.\\\\nHost Range Investigation and EOP Analysis\\\\nAll five bacteriophages exhibited a broad lytic spectrum and were able to infect 69.7% (n = 23) of the P. aeruginosa strains, of which seven were multidrug resistant. Furthermore, the bacteriophages showed lytic activity on clinical strains isolated from different infection sites (Table 1).\\\\nThe EOP values of vB_PaeM_USP_1 (high: 13; insufficient: 6; low: 4), vB_PaeM_USP_2 (high: 14; insufficient: 6; low: 3), and vB_PaeM_USP_3 (high: 13; insufficient: 7; low: 3) were higher than vB_PaeM_USP_18 (high: 8; insufficient: 11; low: 4) and vB_PaeM_USP_25 (high: 6; insufficient: 11; low: 6). The last two viruses had low or inefficient EOP for all multidrug-resistant strains (Table 1). Although during host range investigation the bacteriophages had lysed the same strains, inefficient EOP results indicate that the haloes observed previously refer to “lysis from without events.”\\\\nInfection of Planktonic Cells\\\\nAll five bacteriophages reduced bacterial growth up to 8 h of incubation in 13 strains (p < 0.05; Figure 5A). The reduction was above 60% after 6 and 8 h of incubation. After 24 h of incubation, all bacteriophages were able to maintain lower OD values in three strains, but with reductions in bacterial growth of 45% (p < 0.05). In five other strains, the growth rates were reduced by different bacteriophages, and the growth of five strains was not affected by the bacteriophages. Concerning multidrug-resistant P. aeruginosa, the bacteriophages promoted alteration in growth rates in one of the two strains. vB_PaeM_USP_1 and vB_PaeM_USP_3 promoted the greatest growth reductions (Supplementary Tables 1–5).\\\\nFigure 5. (A) Optical density of planktonic Pseudomonas aeruginosa strains after 2, 4, 6, 8, and 24 h of co-incubation with different bacteriophages. (B) Anti-biofilm activity of different bacteriophages on biofilm viability rates expressed in log10 CFU/cm2 (left y axis) and metabolic activity expressed in absorbance at 492 nm (right y axis). Inhibition of planktonic growth and anti-biofilm activity assays was carried out at three independent times in triplicate against 15 strains of P. aeruginosa. (C–H) Scanning electron micrographs after phage treatment: (C) control, (D) vB_PaeM_USP_1, (E) vB_PaeM_USP_2, (F) vB_PaeM_USP_3, (G) vB_PaeM_USP_18, and (H) vB_PaeM_USP_25. Scale bar = 100 nm. After the treatment of bacteriophages, it is possible to observe lower planktonic growth, biofilm viability rates, and metabolic activity. Disrupted biofilms and cell debris are also noticed in SEM images.\\\\nAnti-biofilm Activity Measurement\\\\nAll five bacteriophages reduced the biofilm viability rates in 12 strains (p < 0.05) in about 10-fold CFU/cm2 (Figure 5B). vB_PaeM_USP_2 and vB_PaeM_USP_18 presented increased anti-biofilm activity against two multidrug-resistant strains.\\\\nConcerning the metabolic activity (XTT), after treatment of the bacteriophages, the absorbance values reduced in about 50% in 11 strains (p < 0.05; Figure 5B). Bacteriophage vB_PaeM_USP_2 was able to significantly reduce the metabolic activity. The mean and the standard deviation values (CFU/cm2 and XTT) after exposure to different bacteriophages are shown in the Supplementary Data (Supplementary Tables 6, 7).\\\\nResults from SEM micrographs showed densely associated bacteria in the control biofilm (Figure 5C). In contrast, after exposure to bacteriophages, it is possible to observe disrupted biofilms and cell debris, demonstrating remarkable biofilm reduction (Figures 5D–H).\\\\nDiscussion\\\\nThis study comprises the isolation and the full characterization of five new P. aeruginosa bacteriophages isolated from domestic sewage. The bacteriophages showed a broad lytic spectrum, able to disrupt P. aeruginosa biofilms on endotracheal tube surfaces. Based on the results, the null hypothesis was rejected since there was a statistical difference on planktonic bacteria growth, viability rates, and metabolic activity of the biofilm.\\\\nThe vast majority of the already characterized bacteriophages are tailed (Hatfull and Hendrix, 2011). Through morphological and genomics analysis, here there is evidence that all the five bacteriophages have similar features, belonging to the order Caudovirales and Myoviridae family. Moreover, genome sequencing revealed that vB_PaeM_USP_1, vB_PaeM_USP_2, and vB_PaeM_USP_3 were most closely related to Epa22, sharing about 96% of identity. Their genomes are typically organized in clusters. Both guanine-cytosine content and genome length were similar to other Pbunaviruses as previously reported (de Melo et al., 2019; Yuan et al., 2019). Regarding vB_PaeM_USP_18 and vB_PaeM_USP_25, only ca. 20% of their genomes could be aligned with other phage genomes deposited at the NCBI. This low identity with other known genomes does not allow grouping these two phages with any known genus or subfamily. The inability to group these two genomes suggests that they belong to a new bacteriophage genus. Genomic bioinformatics analysis allowed identifying the endolysin gene on all bacteriophages. Commonly, this enzyme is responsible for the lytic ability of tailed bacteriophages (Wu et al., 2019). In addition, several open reading frames could only be annotated as coding for hypothetical proteins and thus with an unknown function, something that is common in bacteriophage studies (Guo et al., 2019).\\\\nThe one-step growth curve analysis revealed that the latent period was similar in the five bacteriophages (about 30 min; Figure 1H), with a progeny per infected cell varying from 22 to 46 bacteriophage particles. Regarding the latent period, the same result was reported previously. Nonetheless, comparing burst sizes with bacteriophages with a similar genome, we noticed that all bacteriophages produced lower burst sizes (Guo et al., 2019; Yuan et al., 2019). The reduction observed here could be explained by differences in bacterial growth rates since the burst size increases linearly with bacterial growth rate (Nabergoj et al., 2018). Adsorption and latent period also vary with bacterial growth rate, decreasing at higher growth rates.\\\\nThe infectivity of the bacteriophages remained stable for a pH range from 5.0 to 9.0 and temperature range from 37 to 60°C (Figure 2). Although pH and temperature do not undergo abrupt variation in physiological systems, comprehending this aspect is important in order to maintain bacteriophage viability and infectivity. Even though new antimicrobial agents should be investigated, the large pH and temperature tolerance indicate a broad workability range to search and apply bacteriophage-based products to setups of different infection models. For instance, skin becomes more alkaline by bacterial colonization, and wounds show pH variations in relation to healthy skin (Schneider et al., 2007).\\\\nThe lytic activity of bacteriophages was tested against 37 strains (33 P. aeruginosa and four different bacterial species). Our results reveal that all bacteriophages could efficiently lyse 69.7% of the P. aeruginosa strains, of which seven were multidrug resistant. Infection of multidrug-resistant P. aeruginosa indicates that bacteriophages or bacteriophage-based products could potentially be applied for therapeutic purposes. The lytic rates presented here are higher than in other Pbunavirus (de Melo et al., 2019), which can be related to the origin and the number of tested strains. In fact, host broad range and species specificity are desirable for bacteriophage application in phage therapy. Moreover, bacteriophages should be strictly lytic in order to avoid the risk of potential transmission of virulent bacterial genes (Canchaya et al., 2003). Here five lytic bacteriophages had promising results in disrupting mature P. aeruginosa biofilms and can be further explored in studies involving phage therapy.\\\\nEfficiency of plating assays indicated a considerable variation from low to high between the different bacteriophages on the different strains. Moreover, the bacteriophages showed inefficient EOP for six strains. Differences in EOP and absence of lysis can be explained by the recognition process, which can diverge in different strains according to the expression of bacteriophage receptors on the bacterial outer membrane (Uchiyama et al., 2016). Furthermore, the localization, volume, and density of surface molecules can affect bacteriophage adsorption (Rakhuba et al., 2010). Considering the mentioned variation in molecules for bacteriophage adsorption, the application of a bacteriophage cocktail, which retains distinct adsorption mechanisms, is important to maintain bacteriophage infectivity (Forti et al., 2018). Apart from the recognition process, well-known bacterial resistance systems against bacteriophages could have led to low EOP values. Literature has pointed out that many antiviral mechanisms may be employed by the host cell to evade infection from bacteriophages, such as restriction nucleases and CRISPR (Seed, 2015).\\\\nAssessing the efficacy of bacteriophages in reducing P. aeruginosa growth in planktonic form, we observed significant reductions for up to 8 h of co-incubation for almost all investigated strains. However, following 24 h of co-incubation, only three strains showed reduced growth rates. Two multidrug-resistant P. aeruginosa did not show reduced growth rates at the evaluated times. Guo et al. (2019) demonstrated that two new phages at low MOI (0.01) had good performance both on preventing biofilm formation and eradicating preformed biofilms. Moreover, Melo et al. (2014), using a MOI of 0.05, demonstrated reduction in the optical density of planktonic cells in about 83% of clinical isolates after 24 h of infection. We considered that antiviral mechanisms might result in the emergence of bacteriophage-insensitive bacterial strains through blocking of bacteriophage receptors, production of extracellular matrix, production of competitive inhibitors, prevention of bacteriophage DNA entry, slicing of bacteriophage nucleic acids, and abortive infection mechanisms (Labrie et al., 2010).\\\\nWe observed a great reduction in planktonic growth and anti-biofilm activity after treatment of bacteriophages, and it was widely coincident in both assays. Slight differences were detected among the strains, which can be related to distinct analysis methods. Planktonic assays were performed by OD, while anti-biofilm activity was evaluated by CFU counts and activity metabolic measurement. Comparing OD and CFU results, only one strain showed a discordant outcome. Considering the three analysis methods (OD, CFU, and XTT), the results of six strains did not seem to be concordant. The lytic activity of the bacteriophages was relevant to biofilm control on the endotracheal tube surface. Although no depolymerase was identified in these bacteriophage genomes, the anti-biofilm action could be related to the penetration of viruses at the deeper layers of the biofilm by means of the degradation of the polysaccharide matrix (Lin et al., 2017).\\\\nAnti-biofilm results support the hypothesis that bacteriophages might be applied as a biocontrol tool against multidrug-resistant P. aeruginosa on the endotracheal tube surface in the near future. Systemic treatment with antibiotics did not fully inhibit biofilm development on endotracheal tube surfaces (Fernández-Barat et al., 2012). Therefore, direct administration of non-antibiotic approaches, such as antimicrobial agents (Guillon et al., 2018) and bacteriophages, seem to be a suitable option for the fight against persistent biofilms. Indeed bacteriophages could be potentially applied on endotracheal tube biofilm as prophylactic and therapeutic agent. Bacteriophage-coated tubes could be attended to inhibit bacterial adhesion, whereas direct application may be useful in reducing microbial bioburden. Aleshkin et al. (2016) reported the intragastric administration of a phage cocktail in patients on mechanical ventilation, aiming for the treatment of acute infections caused by nosocomial pathogens. The authors demonstrated an important reduction in bacterial counts after phage therapy, indicating it as an alternative purpose to antibiotics to treat healthcare-associated infections.\\\\nLiterature has presented that bacteriophages or the synergistic action of bacteriophage and antibiotics constitute promising strategies against multidrug-resistant P. aeruginosa biofilm (Akturk et al., 2019; Adnan et al., 2020). Given the ability of new lytic bacteriophages to disrupt and reduce the microbial load of endotracheal-associated biofilms in vitro, several aspects must be taken into consideration before the transfer of phage therapy to clinical practice. As the device is not in contact with vascularized tissues, potential bacteriophage administration routes should be discussed. Moreover, in clinical settings, it is highly unlikely that the endotracheal tubes will continuously be exposed to the phages, as demonstrated in this in vitro study. Thus, we suggest that phage therapy could be applied during endotracheal suctioning, as recommended for ventilated patients. The instillation of bacteriophages during the aspiration of pulmonary secretions could favor its direct application (Guillon et al., 2018). Inhalation of aerosol-containing antibiotics and phage formulations is another potential administration route.\\\\nIn addition, it should be considered that in vivo biofilms are different from the in vitro ones studied here, probably due to the presence of body fluids, host cells, and different interactions among microorganisms. In light of the aforementioned assumptions, future investigations considering mixed biofilm models and the synergistic action of phage and antibiotic therapy should be conducted. Although treatment of bacteriophage reduced the concentration of P. aeruginosa both on planktonic and biofilm forms, further research is required to determine the correct concentration and the possible combinations of different bacteriophages to establish a cocktail. Based on EOP and anti-biofilm activity and considering the similarities among the five bacteriophages, we suggest that, in further studies, a cocktail composed of vB_PaeM_USP_2 and vB_PaeM_USP_18 should be used. Moreover, only one administration with a single bacteriophage concentration to treat biofilms was evaluated. Assessing different times of infection, a strong lytic activity could be evidenced. In addition, special attention should be paid to the potential development of bacteriophage resistance. In view of this, we also consider it necessary to evaluate the phenotype of bacteriophage-insensitive mutants after a long exposure to bacteriophages.\\\\nConclusion\\\\nBiofilms associated with endotracheal tubes are critical concerns in the development of VAP. Since antibiotic therapy is not profitable to eliminate biofilms on the surface of medical devices, bacteriophage therapy offers a possible alternative to reduce bacterial colonization on the surface of endotracheal tubes. The studied bacteriophages were efficient in controlling the growth of P. aeruginosa and promoted biofilm disruption on the surface of endotracheal tubes, including multidrug-resistant strains. Thus, they might be considered as good candidates for therapeutic studies and the application of bacteriophage-encoded proteins.\\\\nData Availability Statement\\\\nThe datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found in the article/ Supplementary Material.\\\\nAuthor Contributions\\\\nEW contributed to conceptualization and funding acquisition. VO carried out the main body of research and wrote the original manuscript. FB and RM contributed to the phage growth experiments. AM contributed to data analysis. CS-L and HP supervised the work’s progress and edited the manuscript. LM and SS contributed to bioinformatics analysis and edited the manuscript. All authors have read and agreed to the published version of the manuscript.\\\\nFunding\\\\nThis study was supported by the São Paulo Research Foundation (FAPESP) under grant numbers 2018/09757-0 and 2019/13271-9 and the National Council for Scientific and Technological Development (CNPq) under grant number 405622/2018-0.\\\\nConflict of Interest\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2020.580779/full#supplementary-material\\\\nReferences\\\\nAdnan, M., Shah, M. R. A., Jamal, M., Jalil, F., Andleeb, S., Nawaz, M. A., et al. (2020). Isolation and characterization of bacteriophage to control multidrug-resistant Pseudomonas aeruginosa planktonic cells and biofilm. Biologicals 63, 89–96. doi: 10.1016/j.biologicals.2019.10.003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAkturk, E., Oliveira, H., Santos, S. B., Costa, S., Kuyumcu, S., Melo, L. D. R., et al. (2019). Synergistic action of phage and antibiotics: parameters to enhance the killing efficacy against mono and dual-species biofilms. Antibiotics 8:103. doi: 10.3390/antibiotics8030103\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAleshkin, A. V., Ershova, O. N., Volozhantsev, N. V., Svetoch, E. A., Popova, A. V., Rubalskii, E. O., et al. (2016). Phagebiotics in treatment and prophylaxis of healthcare-associated infections. Bacteriophage 6:e1251379. doi: 10.1080/21597081.2016.1251379\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAltschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990). Basic local alignment search tool. J. Mol. Biol. 215, 403–410. doi: 10.1016/S0022-2836(05)80360-2\\\\nCrossRef Full Text | Google Scholar\\\\nAziz, R. K., Bartels, D., Best, A. A., DeJongh, M., Disz, T., Edwards, R. A., et al. (2008). The RAST Server: rapid annotations using subsystems technology. BMC Genom. 9:75. doi: 10.1186/1471-2164-9-75\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBarbier, F., Andremont, A., Wolff, M., and Bouadma, L. (2013). Hospital-acquired pneumonia and ventilator-associated pneumonia: recent advances in epidemiology and management. Curr. Opin. Pulm. Med. 19, 216–228. doi: 10.1097/MCP.0b013e32835f27be\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBarros, J. A. R., Melo, L. D. R., Silva, R. A. R., Ferraz, M. P., Azeredo, J. C. V. R., Pinheiro, V. M. C., et al. (2020). Encapsulated bacteriophages in alginate-nanohydroxyapatite hydrogel as a novel delivery system to prevent orthopedic implant-associated infections. Nanomedicine 24:102145. doi: 10.1016/j.nano.2019.102145\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBerra, L., De Marchi, L., Yu, Z. X., Laquerriere, P., Baccarelli, A., and Kolobow, T. (2004). Endotracheal tubes coated with antiseptics decrease bacterial colonization of the ventilator circuits, lungs, and endotracheal tube. Anesthesiology 100, 1446–1456. doi: 10.1097/00000542-200406000-00017\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCanchaya, C., Fournous, G., Chibani-Chennoufi, S., Dillmann, M. L., and Brussow, H. (2003). Phage as agents of lateral gene transfer. Curr. Opin. Microbiol. 6, 417–424. doi: 10.1016/s1369-5274(03)00086-9\\\\nCrossRef Full Text | Google Scholar\\\\nCieplik, F., Jakubovics, N. S., Buchalla, W., Maisch, T., Hellwig, E., and Al-Ahmad, A. (2019). Resistance toward Chlorhexidine in oral bacteria - is there cause for concern? Front. Microbiol. 10:587. doi: 10.3389/fmicb.2019.00587\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nClinical and Laboratory Standards Institute [CLSI] (2019). Performance Standards for Antimicrobial Susceptibility Testing, Twenty-Nine Informational Supplement. Wayne: CLSI.\\\\nGoogle Scholar\\\\nCui, Z. L., Feng, T. T., Gu, F. F., Li, Q. T., Dong, K., Zhang, Y., et al. (2017). Characterization and complete genome of the virulent Myoviridae phage JD007 active against a variety of Staphylococcus aureus isolates from different hospitals in Shanghai, China. Virol. J. 14:26. doi: 10.1186/s12985-017-0701-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nde Melo, A. C. C., Gomes, A. D., Melo, F. L., Ardisson-Araujo, D. M. P., de Vargas, A. P. C., Ely, V. L., et al. (2019). Characterization of a bacteriophage with broad host range against strains of Pseudomonas aeruginosa isolated from domestic animals. BMC Microbiol. 19:134. doi: 10.1186/s12866-019-1481-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nFernández-Barat, L., Ferrer, M., Sierra, J. M., Soy, D., Guerrero, L., Vila, J., et al. (2012). Linezolid limits burden of methicillin-resistant Staphylococcus aureus in biofilm of tracheal tubes. Crit. Care Med. 40, 2385–2389. doi: 10.1097/CCM.0b013e31825332fc\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nFinn, R. D., Attwood, T. K., Babbitt, P. C., Bateman, A., Bork, P., Bridge, A. J., et al. (2017). InterPro in 2017-beyond protein family and domain annotations. Nucleic Acids Res. 45, 190–199. doi: 10.1093/nar/gkw1107\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nFinn, R. D., Bateman, A., Clements, J., Coggill, P., Eberhardt, R. Y., Eddy, S. R., et al. (2014). Pfam: the protein families database. Nucleic Acids Res. 42, 222–230. doi: 10.1093/nar/gkt1223\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nForti, F., Roach, D. R., Cafora, M., Pasini, M. E., Horner, D. S., Fiscarelli, E. V., et al. (2018). Design of a broad-range bacteriophage cocktail that reduces Pseudomonas aeruginosa biofilms and treats acute infections in two animal models. Antimicrob. Agents Chemother. 62:e02573-17. doi: 10.1128/AAC.02573-17\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGuillon, A., Fouquenet, D., Morello, E., Henry, C., Georgeault, S., Si-Tahar, M., et al. (2018). Treatment of Pseudomonas aeruginosa biofilm present in endotracheal tubes by Poly-l-Lysine. Antimicrob. Agents Chemother. 62:e0564-18.\\\\nGoogle Scholar\\\\nGuo, Y. Y. J., Chen, P., Lin, Z. L., and Wang, T. T. (2019). Characterization of two Pseudomonas aeruginosa viruses vB_PaeM_SCUT-S1 and vB_PaeM_SCUT-S2. Viruses 11:e318. doi: 10.3390/v11040318\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKäll, L., Krogh, A., and Sonnhammer, E. L. L. (2004). A combined transmembrane topology and signal peptide prediction method. J. Mol. Biol. 338, 1027–1036. doi: 10.1016/j.jmb.2004.03.016\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKäll, L., and Sonnhammer, E. L. L. (2002). Reliability of transmembrane predictions in whole-genome data. FEBS Lett. 532, 415–418. doi: 10.1016/S0014-5793(02)03730-4\\\\nCrossRef Full Text | Google Scholar\\\\nKaur, S., Harjai, K., and Chhibber, S. (2014). Bacteriophage mediated killing of Staphylococcus aureus in vitro on orthopaedic K wires in presence of linezolid prevents implant colonization. PLoS One 9:e90411. doi: 10.1371/journal.pone.0090411\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLessa, F. C. R., Aranha, A. M. F., Nogueira, I., Giro, E. M. A., Hebling, J., and Costa, C. A. S. (2010). Toxicity of Chlorhexidine on odontoblast-like cells. J. Appl. Oral Sci. 18, 50–58. doi: 10.1590/s1678-77572010000100010\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLin, D. M., Koskella, B., and Lin, H. C. (2017). Phage therapy: an alternative to antibiotics in the age of multi-drug resistance. World J. Gastrointest. Pharmacol. Ther. 8:162. doi: 10.4292/wjgpt.v8.i3.162\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLungren, M. P., Donlan, R. M., Kankotia, R., Paxton, B. E., Falk, I., Christensen, D., et al. (2014). Bacteriophage K antimicrobial-lock technique for treatment of Staphylococcus aureus central venous catheter-related infection: a leporine model efficacy analysis. J. Vasc. Interv. Radiol. 25, 1627–1632. doi: 10.1016/j.jvir.2014.06.009\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMagiorakos, A. P., Srinivasan, A., Carey, R. B., Carmeli, Y., Falagas, M. E., Giske, C. G., et al. (2012). Multidrug-resistant, extensively drug-resistant and pandrug-resistant bacteria: an international expert proposal for interim standard definitions for acquired resistance. Clin. Microbiol. Infect. 18, 268–281. doi: 10.1111/j.1469-0691.2011.03570.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMao, B. H., Chen, Z. Y., Wang, Y. J., and Yan, S. J. (2018). Silver nanoparticles have lethal and sublethal adverse effects on development and longevity by inducing ROS-mediated stress responses. Sci. Rep. 8:2445. doi: 10.1038/s41598-018-20728-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMelo, L. D. R., Sillankorva, S., Ackermann, H. W., Kropinski, A. M., Azeredo, J., and Cerca, N. (2014). Isolation and characterization of a new Staphylococcus epidermidis broad-spectrum bacteriophage. J. Gen. Virol. 95, 506–515. doi: 10.1099/vir.0.060590-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMelo, L. D. R., Veiga, P., Cerca, N., Kropinski, A. M., Almeida, C., Azeredo, J., et al. (2016). Development of a phage cocktail to control Proteus mirabilis catheter-associated urinary tract infections. Front. Microbiol. 7:1024. doi: 10.3389/fmicb.2016.01024\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nNseir, S., Ader, F., Lubret, R., and Marquette, C. H. (2011). Pathophysiology of airway colonization in critically ill COPD patient. Curr. Drug Targets 12, 514–520. doi: 10.2174/138945011794751537\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nNzakizwanayo, J., Hanin, A., Alves, D. R., McCutcheon, B., Dedi, C., Salvage, J., et al. (2015). Bacteriophage can prevent encrustation and blockage of urinary catheters by Proteus mirabilis. Antimicrob. Agents Chemother. 60, 1530–1536. doi: 10.1128/AAC.02685-15\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPetersen, T. N., Brunak, S., Heijne, G. V., and Nielsen, H. (2011). SignalP 4.0: discriminating signal peptides from transmembrane regions. Nat. Methods 8, 785–786. doi: 10.1038/nmeth.1701\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRakhuba, D. V., Kolomiets, E. I., Dey, E. S., and Novik, G. I. (2010). Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell. Pol. J. Microbiol. 59, 145–155. doi: 10.33073/pjm-2010-023\\\\nCrossRef Full Text | Google Scholar\\\\nRussotto, V., Cortegiani, A., Raineri, S. M., and Giarratano, A. (2015). Bacterial contamination of inanimate surfaces and equipment in the intensive care unit. J. Intensive Care 3:54. doi: 10.1186/s40560-015-0120-125\\\\nCrossRef Full Text | Google Scholar\\\\nSambrook, J., and Russell, D. W. (2001). Molecular Cloning: A Laboratory Manual. New York, NY: Cold Spring Harbor Laboratory Press.\\\\nGoogle Scholar\\\\nSchattner, P., Brooks, A. N., and Lowe, T. M. (2005). The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs. Nucleic Acids Res. 33, W686–W689. doi: 10.1093/nar/gki366\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSchneider, L. A., Korber, A., Grabbe, S., and Dissemond, J. (2007). Influence of pH on wound-healing: a new perspective for wound-therapy? Arch. Dermatol. Res. 298, 413–420. doi: 10.1007/s00403-006-0713-x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSillankorva, S., Neubauer, P., and Azeredo, J. (2008). Isolation and characterization of a T7-like lytic phage for Pseudomonas fluorescens. BMC Biotechnol. 8:80. doi: 10.1186/1472-6750-8-80\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSpilker, T., Coenye, T., Vandamme, P., and LiPuma, J. J. (2004). PCR-Based assay for differentiation of Pseudomonas aeruginosa from other Pseudomonas species recovered from cystic fibrosis patients. J. Clin. Microbiol. 42, 2074–2079. doi: 10.1128/jcm.42.5.2074-2079.2004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTokmaji, G., Vermeulen, H., Muller, M. C. A., Kwakman, P. H. S., Schultz, M. J., and Zaat, S. A. J. (2015). Silver-coated endotracheal tubes for prevention of ventilator-associated pneumonia in critically ill patients. Cochrane Database Syst. Rev 8:CD009201. doi: 10.1002/14651858.CD009201\\\\nCrossRef Full Text | Google Scholar\\\\nUchiyama, J., Suzuki, M., Nishifuji, K., Kato, S. I., Miyata, R., Nasukawa, T., et al. (2016). Analyses of short-term antagonistic evolution of Pseudomonas aeruginosa strain PAO1 and phage KPP22 (Myoviridae Family, PB1-like virus Genus). Appl. Environ. Microbiol. 82, 4482–4491. doi: 10.1128/AEM.00090-16\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nViklund, H., Bernsel, A., Skwark, M., and Elofsson, A. (2008). Spoctopus: a combined predictor of signal peptides and membrane protein topology. Bioinformatics 24, 2928–2929. doi: 10.1093/bioinformatics/btn550\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWu, M. L., Hu, K. Y., Xie, Y. H., Liu, Y. L., Mu, D., Guo, H. M., et al. (2019). A novel phage PD-6A3, and its endolysin Ply6A3, with extended lytic activity against Acinetobacter baumannii. Front. Microbiol. 9:3302. doi: 10.3389/fmicb.2018.03302\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYuan, Y. Y., Qu, K. L., Tan, D. M., Li, X. Y., Wan, L. L., Cong, C., et al. (2019). Isolation and characterization of a bacteriophage and its potential to disrupt multi-drug resistant Pseudomonas aeruginosa biofilms. Microb. Pathog. 128, 329–336. doi: 10.1016/j.micpath.2019.01.032\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nZimlichman, E., Henderson, D., Tamir, O., Franz, C., Song, P., Yamin, C. K., et al. (2013). Health Care-associated infections a meta-analysis of costs and financial impact on the US health care system. JAMA Intern. Med. 173, 2039–2046. doi: 10.1001/jamainternmed.2013.9763\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.02931\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.02931/full\",\\n    \"loadedTime\": \"2024-01-23T08:50:15.677Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.02931\",\\n    \"title\": \"Frontiers | Bacteriophages as an Up-and-Coming Alternative to the Use of Sulfur Dioxide in Winemaking\",\\n    \"description\": \"Certain acetic and lactic acid bacteria are the major causes of quality defects in musts and wines, giving rise to defects such as \\\\\"vinegary\\\\\", \\\\\"sharp like nail polish remover\\\\\" or preventing alcoholic and/or malolactic fermentation. Sulfur dioxide is the major tool currently used in the control of these bacteria in wine. The aim of this work was to isolate bacteriophages from musts and wine of different grape varieties, able to eliminate lactic and acetic acid bacteria spoilages at laboratory scale. Musts obtained from grape-berries of Vitis vinifera cv. Chardonnay and Moscatel and a red wine made with Vitis vinifera cv. Tintilla de Rota were used to isolate bacteriophages. Bacteriophages were obtained from all musts and wine, belonging to the order Caudovirals and to the family Tectivirals. They were isolated by classical virology methods and identified by electron microscopy. Host-bacteria used in the study were the lactic acid bacteria of the species Lactobacillus hilgardii, L. plantarum and Oenococcus oeni and the acetic bacteria Acetobacter aceti. A comparative study was performed on musts and wines, previously inoculated with bacteria, by phage titration and SO2 addition to study the effectiveness of bacteriophages against bacteria. The comparative study showed that some bacteriophages were as effective as sulfur dioxide at low concentration.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nBacteria are part of the natural microbiota of wine, and they play a key role in winemaking by contributing to the aroma and flavor of wine while reducing acidity. Conversely, they can cause undesirable wine spoilage problems, reducing wine quality and value (Bartowsky, 2009). Some lactic acid and acetic acid bacteria species, such as Lactobacillus sp., Pediococcus sp., and Acetobacter sp., are primarily responsible for the loss of quality of musts and wines, leading to defects such as a “vinegary,” “sharp, like nail polish-remover” taste or preventing alcoholic and/or malolactic fermentation (Bartowsky and Henschke, 2008; Bartowsky, 2009; Fumi et al., 2010; Sengun, 2017).\\\\nSulfur dioxide (SO2) is the most commonly used antimicrobial compound in wine production due to its efficacy and low cost, being able to affect a wide variety of spoilage microorganisms. Despite being active against bacteria and some yeasts, it is innocuous to enological yeasts, since sulfur dioxide is combined with pyruvate, 2-ketoglutarate, and other compounds that are produced during alcoholic fermentation (Ough and Were, 2005). Thus, the main advantage is that Saccharomyces cerevisiae, the most common yeast in fermentation, is not significantly affected by sulfur dioxide at the levels used, so that it can perform the process normally (Lisanti et al., 2019). Moreover, SO2 protects wines against oxidation, and in adequate doses, stabilizes the color of wine and provides a complex flavor, especially in wines made with grape varieties that offer little in a sensory way (Guerrero and Cantos-Villar, 2015; Lisanti et al., 2019). The free sulfur dioxide in wine is dissociated in three chemical forms: molecular sulfite, bisulfite ion, and sulfite ion. The balance between the different chemical forms depends on the pH, the alcoholic strength, and the storage temperature of the wine. The only active form in terms of protection against contaminating microorganisms (e.g., Brettanomyces, Lactobacillus) is molecular sulfur dioxide, which is responsible for the pH value (Ough and Were, 2005). The more acidic the wine, the more active the free SO2 will be (Sturm et al., 2013). The initial dose depends on different factors, such as the grape maturity, the sanitary state of the grape harvest, its temperature, and, mainly, the type of wine to be obtained. Due to the great diversity of wines, there is not a pre-established dose, but winemakers adjust it according to the raw materials, technology, and products they have available.\\\\nAccording to Commission Regulation (EU) No. 53/2011 of January 21, 2011, amending Regulation (EC) No. 606/2009, the European Commission established an authorized total SO2 dose (mg/L) according to the type of wine. However, the number of people who are sensitive or allergic to this compound is increasing and, consequently, the European Commission has proposed to limit its use until its complete prohibition. This fact presents a problem for the wine industry, which may suffer major quality losses due to the uncontrolled proliferation of harmful microorganisms in wine. There has been a great deal of research on possible alternatives that can carry out the major functions of sulfur dioxide but, until now, none have been completely successful (Guerrero and Cantos-Villar, 2015). Chemical compounds such as colloidal silver complexes (Izquierdo-Cañas et al., 2012; García-Ruiz et al., 2015) and dimethyl dicarbonate (Costa et al., 2008) and even natural compounds such as lysozyme, grapevine seeds and shoots, chitosan, and bacteriocins (Lasanta et al., 2010; Garde-Cerdán et al., 2013; Liburdi et al., 2014; Dias et al., 2015; Roldán et al., 2017; Valera et al., 2017; Raposo et al., 2018; Marchante et al., 2019) are some examples. The use of polyphenolic extracts has also been investigated. Specifically, the use of enological tannins combined with lysozyme could reduce the amount of SO2 in wines, although the volatile composition is significantly modified (Sonni et al., 2009, 2011). In white wines, the use of almond skin and eucalyptus leaf extracts has also been proposed to reduce the SO2 content during barrel aging (González-Rompinelli et al., 2013).\\\\nBacteriophages, also known as phages, are viruses that infect bacteria and promote their lysis. Recently, they have been proposed as an alternative to the use of antibiotics in animals, as biopreservers of food, and as a tool for detecting pathogenic bacteria in food processing (Mathur et al., 2003; Clark and March, 2006; Premarathne et al., 2017). The characteristics of being innocuous to human cells and specific to their host range makes bacteriophages an innovative and safe way to replace the traditional chemical compounds used in the food industry, which can generate allergies and intolerances in humans (García et al., 2008). On the other hand, the production of these chemical agents, which include pesticides and herbicides, has an environmental impact on ecosystems, besides a socioeconomic impact, due to the tendency of the population to consume more natural products with fewer allergens and additive components. There is wide recognition that bacteriophage use as an antimicrobial poses new questions regarding its eventual impact on natural environments (Meaden and Koskella, 2013). It is important to evaluate the potential risk of altering the composition of natural microbial communities through the liberation of phages from medical or agricultural applications. This disruptive effect could be harmful to ecosystems, as the phages could alter the nutrient cycle (Fernández et al., 2019). However, these changes are expected to be much less severe than with synthetic antimicrobial agents due to the high specificity of bacteriophage activity.\\\\nIn this study, the aim was to search for bacteriophages that develop naturally in musts and wine. Thus, this research will allow us to identify phages able to hinder the development of the acetic and lactic acid bacteria that are responsible for quality losses in winemaking. If such viruses could be recognized as promising sulfur dioxide substitutes in combination with antioxidants, new strategies could be developed to deal with the wine spoilages caused by bacteria.\\\\nMaterials and Methods\\\\nStrains and Culture Media\\\\nLactic acid and acetic bacteria strains were obtained from the Colección Española de Cultivos Tipo (CECT). Lactobacillus hilgardii (CECT 4786), Lactobacillus plantarum (CECT 748T), and Oenococcus oeni (CECT 4742) were selected as lactic acid strains, while Acetobacter aceti (CECT 473) were chosen as the acetic bacteria.\\\\nSpecific culture media were used for the growth of each strain following CECT recommendations. For lactic acid bacteria from genus Lactobacillus, MRS liquid/agar was used (peptone 1% w/v, meat extract 1% w/v, yeast extract 0.5 w/v, glucose 2% w/v, ammonium citrate 0.2 w/v, sodium acetate 5% w/v, K2HPO4 2% w/v, pH 6.2–6.5). When agar was needed, 1.5% w/v (nutritive) and 0.75% w/v (layer) of agar was dissolved in 1 l of distilled water.\\\\nThe O. oeni strain was grown in MLO medium (tryptone 1% w/v, yeast extract 0.5% w/v, glucose 1% w/v, fructose 0.5% w/v, MgSO4⋅7H2O 0, 2% w/v, MnSO4⋅H2O 0.05% w/v, diamonium citrate 0.35% w/v, L-cysteine hydrochloride 0.5% w/v, Tween 80 1 ml, pH 4.8). When agar was needed, 1.5% w/v (nutritive) and 0.75% w/v (layer) of agar was dissolved in 900 ml of distilled water.\\\\nIn the case of the acetic bacteria, two culture media were used: (i) GYC medium (glucose 2% w/v, yeast extract 1% w/v, CaCO3 2% w/v, pH 6.8) and (ii) Müeller-Hinton medium (meat extract 0.2% w/v, hydrolyzed casein 1.75% w/v, starch 0.15% w/v, agar 1.7% w/v, pH 7.3 ± 0.1). When agar was needed, 1.5% w/v (nutritive) and 0.75% w/v (layer) of agar was dissolved in 1 l of distilled water. The most recommended culture medium is GYC, in which the main compound is calcium carbonate, used to give opacity (Bordons and Reguant, 2013; Philippe et al., 2018). This opacity enables the bacterial growth to be monitored, as the microorganism dissolves carbonate when colonies are developed, producing holes or differently colored areas in the plate.\\\\nPhage Plaque Assays for Titration and Isolation\\\\nBacteriophage Plaque Assays\\\\nHost bacteria were grown in liquid cultures of their specific media until the exponential phase was reached. Conic tubes (Eurotubo®) were inoculated with 15 ml of liquid media and 1 ml of fresh bacteria culture and were incubated at 30°C for L. hilgardii and L. plantarum, 26°C for O. oeni, and 28°C for A. aceti for 24–48 h for lactic acid bacteria and from 7 to 10 days for the acetic acid baterium.\\\\nThe presence of bacteriophages was determined by the double-layer agar (DLA) technique, as described by Sambrook and Russell (2001). In order to obtain the phage sample, a conic tube containing 25 ml of must or wine was centrifuged at 5000 rpm for 10 min and filtrated with a 0.2 μm filter (Millipore, MA, United States). Assays were carried out for each bacterial strain in wines and musts from three different varieties of grape: wine from Vitis vinifera ssp. cv. Tintilla de Rota (pH 3.53, ethanol content 14.5% v/v) and musts from V. vinifera ssp. cv. Moscatel (pH 4.06) and cv. Chardonnay (pH 3.88). Tenfold dilutions (from 10–1 to 10–7) of the phage samples were prepared in sterile 1× PBS solution (NaCl 138 mM, KCl 3 mM, Na2HPO4 8.1 mM, KH2PO4 1.5 mM). Then, 100 μl of diluted phage was added to 500 μl of the host-bacteria culture, including a control sample without phage. They were incubated for 10 min at the growth temperature of each strain. Each phage/bacteria solution was then combined with 6 ml of layer agar in a sterile 15 ml conic tube, and they were mixed briefly. Each mixture was poured onto a Petri dish containing solid nutritive agar, avoiding air bubble formation, and left to dry for 10 min in a laminar flow hood. Finally, the Petri dishes were incubated at the growth temperature of each strain for 24 h for lactic acid bacteria and from 5 to 6 days for acetic bacteria.\\\\nPlaque-Forming Units (PFU) were counted for each type of bacteria by applying the formula:\\\\nPFUml=no\\u2062plates\\u2062×\\u2062dilutionvolume\\u2062of\\u2062applied\\u2062virus\\u2062solution.\\\\nBacteriophage Isolation\\\\nBacteriophages were isolated from the lysis plaques formed in the Petri dishes after titration. These clear lysis zones on agar were cut with a sterile scalpel and placed in an Eppendorf microtube containing 1 ml of sterile 1× PBS solution. Three drops of chloroform were then added, and the mixture was shaken for 1 min using a vortex. It was centrifuged at 4°C at 3300 rpm and filtered with a 0.2 μm filter (Millipore, MA, United States). The process was repeated for the phage plaques obtained in each type of must and wine. The success of the isolation was checked by performing a second phage plaque assay following the same protocol as described before.\\\\nBacteriophage Characterization\\\\nTransmission Electron Microscopy (TEM)\\\\nThe sample preparation consisted of negative staining. A 2% solution of fosfotungstic acid (PTA, Electron Microscopy Sciences, Hatfield, PA, United States) was made up and adjusted to pH 8.1 using normal KOH. Ten microliters of each phage suspension in PBS was added to 10 μl of PTA, and the solution was then poured onto carbon-coated 300 MESH copper grids. After 10 min, the grids were dried with filter paper and left to dry at room temperature. The preparations were examined in a TEM JEOL 2100.\\\\nSaturated Phenol DNA Extraction and Quantification\\\\nPhage stocks (obtained by the double-layer agar technique and recovered from the top layer) were enriched by adding 100 μl of each virus to 20 mL of each host-bacteria culture to obtain high-titer (1010 to 1012 phage/ml) phage concentrates for DNA extraction. After 16 h of incubation at the growth temperature of each host bacterium, three drops of chloroform were added, and the cultures were then centrifuged at 14000 rpm at 4°C for 30 min. The supernatant was collected and filtered with a 0.20 μm Millipore filter.\\\\nA total of 400 μl of the filtered phage suspension was added to a solution containing 20 μl of EDTA (0.5 M Ethylenediaminetetraacetic Acid, pH 8, Sigma-Aldrich), 50 μl of SDS 10% (Sodium Dodecyl Sulfate, Pronadisa, Madrid, Spain), and 5 μl of proteinase K (Panreac, Glenview, IL, United States). The sample was incubated at 37°C for 1 h. Subsequently, 400 μl of saturated phenol in 0.5 M Tris–HCl (Sigma-Aldrich) was added and the mixture was centrifuged at 13000 rpm for 4 min.\\\\nThe aqueous phase was collected, and 200 μl of saturated phenol in Tris–HCl and 200 μl of phenol: chloroform:isoamyl alcohol (25:24:1) (Sigma-Aldrich) were added. They were briefly mixed and centrifuged at 13000 rpm for 4 min. The aqueous phase was transferred into a new Eppendorf microtube containing 400 μl of phenol:chloroform:isoamyl alcohol and then centrifuged at 12000 rpm for 4 min. The aqueous phase was transferred back to another 2-mL Eppendorf microtube, and 200 μl of potassium acetate (3 M, pH 7) and 600 μl of isopropanol were added. It was then centrifuged at 12000 rpm for 10 min. The supernatant was discarded, and the pellet was resuspended in 200 μl of cold ethanol 70% v/v. It was then centrifuged at 13000 rpm for 5 min, and ethanol was discarded. The pellet obtained was dehydrated at 45°C for 20 min using a Speed-Vac concentrator (Eppendorf, Germany). Finally, the pellet was resuspended in 50 μl of sterile ultrapure water. The extracted DNA concentration was measured using a Nanodrop spectrophotometer (Thermo Fisher Scientific, MA, United States) at a wavelength of 260/280.\\\\nRestriction Enzyme Digestion\\\\nThe phage genetic material was digested with restriction enzymes in order to determine its size. The enzymes used were EcoRI, EcoRV, HinfI, and HindIII, all of which were from Promega (Madison, WI, United States). Restricted DNA was electrophoresed on 0.7% (w/v) agarose gel in TAE buffer 1× (40 mM Tris-acetate, 1 mM EDTA) and visualized by UV photography after staining with ethidium bromide 5 μg/mL. The Lambda phage digested with HindIII enzyme was used as a marker.\\\\nPulsed-Field Gel Electrophoresis (PFGE)\\\\nPulsed-field gel electrophoresis consisted of taking 1 μg of non-digested viral DNA sample and heating it at 60°C for 5 min to denaturalize cohesive ends. The genetic material was then cooled in ice and quickly added into a low fusion point agarose solution (cf: 0.6%, Pronadisa). It was then loaded in an agarose gel 1% in TBE 0.5× buffer (Tris 0.89 M, EDTA 0.02 M, and boric acid 0.89 M) in order to electrophorese it.\\\\nThe electrophoresis conditions were an initial pulse of 0.1 s and a final pulse of 8 s. An electric field of 0.6 V/cm was applied for 15 h at 14°C and a 120° angle. After the electrophoresis had finished, the gel was stained with ethidium bromide 5 μg/mL and visualized with an imaging system with a UV transilluminator (Biorad). The genome size was determined using the Lambda phage genome digested with the restriction enzyme HindIII as a reference.\\\\nIn vitro Assay of Bacteriophage Efficacy\\\\nAn in vitro assay in a 96-well multiplate (NuncTM 96-well polystyrene conical bottom MicrowellTM, Thermo Fisher, Denmark) was carried out in order to compare the action of the bacteriophage against host bacteria with the addition of sulfur dioxide following the procedure established by Rojo-Bezares et al. (2007) with some modifications and using the purified phages as the antimicrobial agents. Assays were done in 200 μl of the different culture media according to the bacteria used (MRS, MLO, or GYC/MH). This assay was also carried out in musts and wines at the pH values mentioned above. The initial concentration of inoculated bacteria was 106 cells/ml after overnight growth in the indicated culture media. Pure bacteriophage solutions in PBS 1× were diluted from 10–1 to 10–8 UFP/mL and were co-inoculated with their host bacteria. On the other hand, sulfurous anhydrous dilutions were made from sodium metabisulfite (Sigma/Aldrich) from 100 to 450 mg/L, according to the doses recommended by Commission Regulation (EU) No. 53/2011, at pH 3.5 for Lactobacillus species and pH 4.5 for O. oenis. Blank and negative controls of infection were 200 μl of the correspondent culture media and 100 μl plus 100 μl of cells, respectively. Microplates were sealed with Breathe-Easy membranes (Sigma, Saint Louis, MI, United States) and placed into an automatic microplate reader (Multiskan go reader, Thermo Fisher Scientific, Waltham, MA, United States). The optical density of the mixture at 600 nm was read every hour (1 min of shaking just before automatic reading by the microplate reader) at the growth temperature of each host bacteria over 48–72 h. Assays were performed in triplicate, and the data obtained were processed with the software SkanIt RE for Multiskan GO 3.2. The absorbance data obtained were exported to Microsoft Excel for processing.\\\\nStatistical Analysis\\\\nAll statistical analysis was performed using SPSS software, version 25.0, and MS Excel for Windows. Data collected taken in triplicate and were analyzed using analysis of variance with Tukey’s multiple comparison test or by the non-parametric Mann–Whitney–Wilcoxon test.\\\\nResults\\\\nBacteriophage Titration and Isolation\\\\nPhage plaques were observed from L. plantarum as host bacteria from filtered samples of red wine (Tintilla de Rota) and musts (Moscatel and Chardonnay). In the case of L. hilgardii, bacteriophage plaques were only observed from red wine samples, while O. oeni showed to be infected by bacteriophages isolated from red wine and Moscatel and Chardonnay musts. Plaques from bacteriophages that infected lactic acid bacteria of genus Lactobacillus were small and displayed a uniform distribution (Figure 1A), while those associated with O. oeni showed a notable increase in the plaque size, and their distribution was very heterogeneous, occupying the entire plaque (Figure 1B).\\\\nFigure 1. Appearance of plaques formed by bacteriophages with the double layer agar (DLA) method. (A) Small size and homogeneous distribution of plaques from bacteriophages with L. plantarum as host bacteria and isolated from Chardonnay must. (B) Heterogeneous plaque formed by phage infecting O. oeni as host bacteria, isolated from red wine (Tintilla de Rota grape variety).\\\\nIn order to confirm the first results obtained, a second titration applying the DLA technique with purified phages isolated from the first screening was carried out. As a result, seven different bacteriophages were isolated and named according to their source, host bacteria, and plaque morphotype (Table 1). Significant differences were found between the PFU/ml in the titers made with the musts and the wine (p > 0.05). The number of plate-forming units (PFU) of the genus Lactobacillus titer was lower in red wine compared to white grape musts, which can be related to the acidity and ethanol content differences between wine and must (Table 1). As the optimum pH value for bacterial growth is near 4.5, musts formed a more suitable environment for the development of these bacterial populations and, therefore, for their predisposition to being infected by bacteriophages (Puertas, 1989; Bordons and Reguant, 2013).\\\\nTable 1. Count of plaques obtained in the second titration, applying the DLA technique.\\\\nRegarding A. aceti titration in GYC and MH media, lysis plaques were not observed using commercial vinegar samples, red wine (Tintilla de Rota), or white musts (Moscatel and Chardonnay) as the phage isolation source.\\\\nCharacterization of the Bacteriophages\\\\nTEM\\\\nThe bacteriophage morphology was analyzed by TEM, and they were taxonomically classified according to the International Committee on Taxonomy of Viruses (ICTV) (Ackermann, 2001). The images obtained were visually selected, and those with better resolution are featured in Figure 2.\\\\nFigure 2. Bacteriophages were taxonomically classified according to the International Committee on Taxonomy of Viruses (ICTV). Phages that infected genus Lactobacillus belonged to the order Caudovirals, which could be classified into three families: Myoviridae, Siphoviridae, and Podoviridae. Those that infected O.oeni belonged to poliedric phages and were classified into the Tectiviridae family.\\\\nPhages that infected genus Lactobacillus belonged to the order Caudovirals. According to Bradley’s classification (1987), three families have been described: Myoviridae (A), phages with a long contractile tail, Siphoviridae (B), phages with a long non-contractile tail, and Podoviridae (C), phages with a short tail (Figure 2). Each family has been divided into subgroups according to their head length: isometric (1), moderately elongated (2), and long-headed (3). Phages Tin1Lp and Tin1Lh belonged to the Siphoviridae (A) family, with morphotype 1 (isometric). Phages Mos1Lp and Char1Lp belonged to the Myoviridae (B) family, contractile tail phages, morphotype 1. On the other hand, those that infected O.oeni belonged to poliedric phages (D) and were classified into the Tectiviridae family (morphotype 4), phages with lipid vesicles and a pseudo-tail. The virus size went in decreasing order from the that which infected L. hilgardii, which was 330 nm in length, followed by those that infected L. plantarum, between 130 and 175 nm in length, and then those whose host was O. oeni, with a smaller size, between 80 and 100 nm length (Figure 2).\\\\nGenetic Material From Bacteriophages\\\\nAfter DNA extraction, the quantification showed high concentrations of viral genetic material (between 150 and 1000 ng/μL). The genetic material was extracted with a high purity level, as indicated by the 260/280 ratio, whose value should vary between 1.8 and 2.0. Regarding the 260/230 ratio, it is considered that DNA is pure when its value varies from 1.7 and 2, so the quality of the extracted DNA was proved.\\\\nThe treatment with EcoRI, EcoRV, HinfI, and HindIII did not digest the genomes of the phages. Several gels were made, applying different conditions, first varying the voltage intensity and then the agarose concentration used to prepare the gel. None of the tests showed a good resolution of the DNA, either the complete genome or the digested one, of any of the bacteriophage samples. However, after applying the PFGE technique, bacteriophages that infected bacteria of the genus Lactobacillus showed a genome of approximately 1300 bp in size, while those corresponding to O. oeni were over 1000 bp (Figure 3).\\\\nFigure 3. Results obtained from pulsed-field electrophoresis. Bacteriophages that infected bacteria of the genus Lactobacillus showed a genome approximately 1300 bp in size, while those corresponding to O. oeni were over 1000 bp.\\\\nIn vitro Assay: Co-culturing Bacteriophages Against Host Bacteria Related to Different Sulfur Dioxide Doses\\\\nIn the growth curve corresponding to L. plantarum, significant differences (p > 0.01) were found between each dilution made from 1 × 10–1 to 1 × 10–8 and the controls in red wine and Moscatel and Chardonnay musts (Figure 4). L. plantarum growth values in the presence of bacteriophage were smaller than the sulfur dioxide effect when added in any concentration to the red wine (Figure 4A). This proves that the isolated bacteriophage is an effective infectious agent under the experimental conditions. Regarding Moscatel must, the growth values of L. plantarum with some bacteriophage dilutions were larger than or similar to the results obtained with sulfur dioxide (Figure 4B). The only favorable result obtained was 10–1, which was able to decrease the bacterial growth to below that with anhydride addition, which indicates that the phage Mos1Lp would only be efficient in controlling a certain L. plantarum population. Regarding Chardonnay must, a 10–7 concentration was enough to equate the action of sulfur dioxide, and the efficacy of the isolated bacteriophage was confirmed in all the minor dilutions tested (Figure 4C).\\\\nFigure 4. Effect of specific bacteriophages diluted from 10– 1 to 10– 8 UFP/mL on L. plantarum compared to sulfurous action. The viral efficacy was assessed by taking, as a reference, the host bacteria growth in red wine (A), Moscatel must (B), and Chardonnay must (C), both in the presence of bacteriophage and of sulfur dioxide. Each point is the mean © SD of three assays.\\\\nIn the case of the growth kinetics of L. hilgardii, the phage Tin1Lh showed an insufficient activity compared to the dose of SO2 for red wine (Figure 5). No significant differences between the different dilutions from 1 × 10–1 to 1 × 10–8 and the control (p > 0.01) were found in red wine (Figure 5). All of the bacteriophage concentrations applied (from 10–1 to 10–8 PFU/ml) followed the same trend as the control curve, so it was deduced that these concentrations lacked infective power under the experimental conditions.\\\\nFigure 5. Effect of bacteriophages diluted from 10– 1 to 10– 8 UFP/mL on L. hilgardii compared to sulfurous action. The viral efficacy was assessed by taking, as a reference, the host bacteria growth in the presence of either bacteriophage or sulfur dioxide in red wine. Each point is the mean © SD of three assays.\\\\nConcerning the results obtained for the kinetics of O. oeni in red wine, bacteriophages displayed certain inhibition activity on growth in all the dilutions applied. However, this growth decrease was not strong compared to the effect of sulfur dioxide applied in the doses used for this type of wine. Significant differences (p > 0.01) were found between each dilution made from 1 × 10–1 to 1 × 10–8 and the control in red wine and in Moscatel must (Figure 6). Phage samples continued to be out of range either dilution used (Figure 6A). Bacteriophages that infected O. oeni in Moscatel must also showed efficacy in decreasing bacterial growth (Figure 6B). Unlike red wine, there was major infective power, as the difference between the control curve and those corresponding to different dilutions was bigger. However, the kinetics graph obtained from O. oeni as the host bacterium in Chardonnay must did not show conclusive results data not shown. This may be due to the inability of the bacteria to grow in must, preventing the establishment of a relationship between the control growth and that associated with the phage. Consequently, it could not be compared with the sulfur dioxide effect in this grape variety must.\\\\nFigure 6. Effect of specific bacteriophages diluted from 10-1 to 10– 8 UFP/mL on O. oeni compared to sulfurous action. The viral efficacy was assessed by taking as a reference the host bacteria growth in red wine (A) and in Moscatel must (B), either in presence of bacteriophage or of sulfur dioxide. Each point is the mean ± SD of three assays.\\\\nDiscussion\\\\nThe results obtained in this work evidenced the presence of antibacterial agents in the intrinsic microbiology of the grape musts and wine used under the experimental conditions harnessed in this research. This fact was confirmed with the appearance of phage plaques in titers made for lactic acid bacteria. As stated in the results, the plaques displayed diverse morphology and distributions, which indicated the existence of different infectious agents. The number of phage plaques ranged according to the source, wine or must.\\\\nThe Lactobacillus titer had a significantly higher PFU/ml value in white grape must than in red wine. This could be due to the ten-fold dilutions applied and to the host bacteria predisposition to grow and to be infected, as bacteriophages completely depend on their host cell, both for replicating their genome and for the production of new viral particles (Ackermann, 2003; González-García, 2014). Red wine confers a suitable environment for the growth of lactic acid bacteria because of malolactic fermentation. However, transformations produced during the alcoholic fermentation cause a pH drop and an increase in the organic acid content (Ocón, 2014). This can trigger an increase of acidity, which can lead to bacterial incompetence and, consequently, the loss of infective power of their bacteriophages. In contrast, musts have higher pH values than wine, closer to the optimum value for Lactobacillus to grow (pH 4.5), which could explain a greater ease of bacteriophage infection of their host. In fact, the first isolation of wine bacteriophages in 1985 by Davis et al. (1985) confirmed that pH values lower than 3.5 inhibited their growth, as verified by Henick-Kling et al. (1986a). In contrast with the results obtained in Lactobacillus, bacteriophages of O. oeni were able to infect and lyse it at red wine pH values, as stated by Henick-Kling et al. (1986b). Regarding the ethanol content in red wine used in our experiments, LAB strains (including O. oeni) were able to grow at 14.5% (v/v) of ethanol. Phages are usually resistant to high ethanol concentrations; Ebrecht et al. (2010) applied ethanol at different concentrations (from 10 to 100% v/v) to control the number of virulent and temperate Lactobacillus delbrueckii phages, and only ethanol 100% was effective against the LAB phages. In other studies, a concentration of 75% of ethanol was effective against other species of LAB phages (Binetti and Reinheimer, 2000; Suárez and Reinheimer, 2002; Quiberoni et al., 2003). Thus, the phages isolated in our study seemed to also be resistant at 14.5% of ethanol.\\\\nThis study was limited, as no bacteriophages for acetic acid bacteria were isolated, though these are involved both in the production of vinegar and loss of wine quality. The acidity from the vinegar (pH < 2.8) used as a precautionary measure to avoid phages in the industrial field could be crucial to avoid the development of bacteriophages (Sellmer et al., 1992; Gullo and Giudici, 2008). Firstly, A. aceti showed some difficulties in growing properly. Unlike lactic acid bacteria, those from genus Acetobacter took more than a week to grow in the liquid medium and between 5 and 6 days in the solid one. When titrations were carried out with these cultures, changes were observed after 5 days, in contrast with the 24 h needed by bacteriophages that infected the lactic acid bacteria used in this work. Different colored areas appeared, bacteria colonies were not appreciated, that gave a first hint of the possible existence of bacteriophages. However, the use of GYC medium with calcium carbonate posed a problem for elucidating the main cause of the PFU formation, as they could be inhibition halos produced by phages or holes caused by carbonate dissolution by the bacterial metabolism while growing. As an alternative, Müeller–Hinton medium was used, since it can facilitate the observation of lysis plates because it is translucent; it has been used to show antimicrobial activity in previous acetic bacteria experiments (Du Toit and Lambrechts, 2002; Mateluna, 2006). Acetic bacteria cultures took 14 days to reach the exponential phase in the liquid medium and a week in the solid one, which makes the titre results slower than for lactic acid bacteria. These titrations did not show any lysis plate or inhibition haloes, which proved the inexistence of bacteriophages able to infect the selected strain of A. aceti in vinegar, wine, and musts, and allowed it to be substantiated that the plates in the GYC medium were due to the bacterial metabolism itself. However, expected results were not obtained in the titer for commercial vinegar, wine, and grape must samples. This could also be related to vinegar acid conditions or to the acidity generated by A. aceti. However, the existence of specific phages of acetic bacteria was documented in 1971, when Schocher et al. (1979) isolated the first bacteriophage that infected an Acetobacter suboxidans culture. Its host range was analyzed using 24 Acetobacter and Gluconobacter strains, where only four of the genus Acetobacter were susceptible to being infected. Taking into account that A. suboxidans turned into Gluconobacter suboxidans and due to the closeness of the two genuses, in 1981, Jucker and Ettlinger (1981) did a host range analysis of this phage and suggested that none of the Acetobacter strains but Gluconobacter, according to the new nomenclature, were susceptible. The presence of specific bacteriophages of acetic bacteria was also confirmed by a study of different alcohol vinegar factories in different places in Europe (France, Sweden, and Germany). Four bacteriophages were isolated and showed varied effects on the bacterial population, all of them related to fermentation breaks (Stamm et al., 1989).\\\\nBacteriophage morphology was observed by electron microscopy. According to the ICTV guidelines, bacteriophages that infected genus Lactobacillus were classified as Caudovirals or tailed phages. Phages Tin1Lp and Tin1Lh belong to the Siphoviridae family, while Mos1Lp and Char1Lp belong to Myoviridae. Both families were obtained in the same proportion, which is in agreement with previous studies where the majority of families were Siphoviridae and Myoviridae, leaving Podoviridae as very rare (Ackermann, 1998). All of the phages isolated were icosahedral, which is the most frequent morphotype in Caudovirals (Ackermann, 2001). On the other hand, the phages that infected O. oeni belonged to the Tectiviridae family within the polyhedral phages. Although tectiviruses usually infect enteric species, bacilli, and species of the Thermus genus, previous studies have already demonstrated the existence of this family of viruses in wine, such as bacteriophage GC1, a new tectivirus that infects Gluconobacter cerinus in dry white wine-making (Ackermann and DuBow, 1987; Philippe et al., 2018).\\\\nBacteriophage characterization was carried out using molecular microbiology techniques. Viral genetic material extraction achieved high concentrations, so it was the basis for deducing that the bacteriophages were constituted of DNA. Visualization by electronic microscopy with DNA extraction allowed the phages to be classified as double-stranded DNA, matching the majority of previous studies of wine or vinegar bacteriophages (Davis et al., 1985; Henick-Kling et al., 1986a, b; Hernández-Magadán, 2007; Dini, 2011 Philippe et al., 2018). Despite EcoRI, EcoRV, HinfI, and HindIII endonucleases having been used in similar studies of bacteriophages in dairy products and wine (Moineau et al., 1993; Ackermann, 1998; Spricigo et al., 2013), in our study, the RFLP analysis showed that these enzymes did not digest the genome of the phages isolated. It is well known that the restriction enzymes need very clean and concentrated genetic material; however, according to the extraction and quantification data obtained in our study, the concentration and ratios of DNA were sufficiently satisfactory. The phage resistance to endonuclease activity could be attributed to several factors. The most prevalent is the natural loss of restriction sites during evolution (Moineau et al., 1993; González et al., 2005), although the integration of unusual bases in the bacteriophage genome (Jensen et al., 1998; Shivu et al., 2007) and a loss of validity when there is some methylation in phage DNA affecting the restriction sites that endonucleases could recognize (Sails et al., 1998; Dini, 2011) have also been described. Sequencing and comparative genomic analysis of the bacteriophages could provide us with further information about their genetic identity.\\\\nUnlike bacteria, there are no gene sequences conserved in any of the viruses or phages that can serve as universal primer sites for Polymerase Chain Reaction (PCR) amplification. PCR-based analyses of viral DNA must therefore target specific subsets of the total viral assemblage (Steward, 2001). Taking into account the results obtained by the RFLP, we used the PFGE as a tool for bacteriophage DNA fingerprinting. This molecular technique provides a quick and visual record of the genome size distribution that can be used for qualitative and quantitative comparisons among banding patterns of the viral DNA samples (Steward, 2001). Periodic changes of electric field orientation and duration were applied, and bands evidenced the presence of DNA and gave a phage genome size estimation. According to the size range of the Lambda phage marker fragments digested with the restriction enzyme HindIII, phages of lactic bacteria are about 1300 bp in size, larger than those corresponding to Oenococcus, which approach 1000 bp. It was found that the genome of the seven isolates was much smaller than that of other lactic phages, which usually vary between 18 and 55 Kb (Jarvis et al., 1991; Moineau et al., 1993).\\\\nRegarding the assays done to evaluate the efficacy of the bacteriophages compared to that of different doses of sulfur dioxide, the most promising results were obtained in the three bacteriophages of L. plantarum. In the three grape varieties tested, a very low phage concentration (10–7) was able to substantially decrease the host-bacteria growth rate. Control curves followed a common structure, showing an initial exponential phase (initial culture) followed by a stationary phase, in which growth stabilized. In the case of red wine, the exponential phase was notably shorter than in Moscatel and Chardonnay musts. This could be due to the low availability of nutrients in fermented wine in contrast to the white musts, which contained sugars available to be used by host bacteria. In contrast to the results obtained for L. plantarum, the only isolated bacteriophage that infected L. hilgardii, Tin1Lh, did not show the same or comparable efficacy as the SO2 dose applied. Control wells, containing only bacteria, followed the same kinetic curve as all the bacteriophage doses, from 10–1 to 10–8, from which it was deduced these concentrations lacked infective power under the experimental conditions.\\\\nIn general, the results of the phage titers may vary between trials, due to the resistance that can be generated by bacteria to bacteriophage infection (Ackermann, 2003). Although consideration of the influence of “Quorum Sensing” (QS) mechanisms of the host bacteria itself is out of the scope of this work, it could provide a possible explanation for why the bacteriophages would act in the plaque titration assay and not in the 96-well plate (Waters and Bassler, 2005). Thereby, in the plaque assay, titration L. hilgardii could reach the required cellular density to activate QS mechanisms and, therefore, the bacteriophage would have used the self-inducers to start lysis, while in the 96-well plaque, the phages remained under the lysogenic cycle. However, genetic assays would be needed to prove the existence of receptors or self-inducer molecules shared by L. hilgardii and its specific bacteriophage. If this were not proved, it would be necessary to study other factors related to the host bacteria growth. Another question that remains to be addressed is the difference between the ability of different phages to capture the same host bacteria when immobilized on a substrate (Double Layer Agar) or in a liquid media (Calendar, 2006; Hosseinidoust et al., 2011a, b). Additional research needs to be conducted in order to fully characterize the observed phenomena in wine of decreasing infection of phages in a liquid environment in contrast to a solid one.\\\\nFinally, bacteriophages that infected O. oeni showed an inhibitory effect on the growth of their host bacteria. Control curves had the same behavior as those of L. plantarum, with a quicker growth stabilization observed in red wine than in musts. Both isolated phages, Tin1Oo and Mos1Oo, showed a certain infective power. Nevertheless, the difference in growth between the control well and those with phage dilutions was significantly bigger in musts, which indicates that the host bacteria found must conditions more suitable to growth, promoting the infectious effect of the phage. The effect of these bacteriophages could not be compared with sulfur dioxide, since the bacteria growth was still much lower when the chemical compound was applied. Consequently, since phages showed a lytic effect on the host bacteria, it would only be necessary to increase the phage concentration to obtain a growth inhibition comparable to the sulfurous action. Results of the molecular analysis of O. oeni have established direct relationships between different bacteria strains and their bacteriophages. In 2005, the first genome sequence of O. oeni strain PSU-1 was published, and currently, information is available on the complete genome of 13 different strains. It was discovered that the main differences between the ORF (open reading frame) contents of bacteria strains are due to the presence of multiple bacteriophage sequences located at different points. At least four lysogenic phages can be integrated as complete elements or fragments in the O. oeni genome associated with the transfer of RNA binding sites (Borneman et al., 2012, 2013). The study of the conditions suitable for the activation of the lytic cycle of phages integrated in the bacterium genome could enhance the inhibitory effect on its growth.\\\\nThe best results obtained in this work were those for bacteriophages that infected L. plantarum, whose effects were similar to sulfurous addition, followed by those of O. oeni, which showed less inhibition than obtained with SO2, while phages that infected L. hilgardii did not display any infective power under the experimental conditions. Although preliminary observations were obtained in this research, the presence of bacteriophages in the three grape varieties used highlights the potential application of a “phage cocktail” combining several infectious agents against a specific bacteria or different species or pollutants that require elimination in combination with a natural antioxidant. The fact that phages from the same family are able to infect different species allows savings to be proposed in the production of cocktails, in which the minimum number of bacteriophages could be used to prevent contaminating bacteria growth in the process of winemaking in a single treatment. In order to reduce the likelihood of resistant bacteria, cocktails could be designed that contain phages that use different receptors. In that way, if the mutant generated resistance to one of the phages in the cocktail, it would remain sensitive to the others. Currently, the use of lytic bacteriophages is a growing alternative for the biological control of pathogens in food and the food industry (Hermoso et al., 2007; García et al., 2008; Spricigo et al., 2013; Sulakvelidze, 2013). In the present study, the objective was not to remove pathogen microorganisms but those that cause losses in wine quality, such as lactic acid and acetic bacteria. Thus, the application of phage cocktails made by specific bacteriophages for every grape variety, vinification stage, and contaminating bacterium, together with a natural antioxidant, could be proposed in further studies at a larger scale.\\\\nData Availability Statement\\\\nAll datasets generated for this study are included in the article/supplementary material.\\\\nAuthor Contributions\\\\nGC-B, JM, and MR-C contributed to the conception and design of experiments. MR-C, MR-M, GC-B, and JM performed the experiments. MR-C, MR-M, GC-B, JM, and JC analyzed the data. MR-C and GC-B contributed to writing the manuscript. MR-M revised the English. All authors critically revised the manuscript before submission.\\\\nFunding\\\\nThis work was financed by the project UCA18DGUEII02.\\\\nConflict of Interest\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nWe thank Olga Aliseda, Juan González, and José María Geraldía (Electron Microscopy Division, University of Cádiz, Cádiz, Spain) for assistance on the sample preparation, MET management, and image analysis.\\\\nReferences\\\\nAckermann, H. W., and DuBow, M. S. (1987). Viruses of Prokaryotes: General Properties of Bacteriophages, Vol. 1. Boca Raton, FL: CRC Press, 49–85.\\\\nGoogle Scholar\\\\nAckermann, W. (2003). Bacteriophage observations and evolution. J. Microbiol. 154, 245–251. doi: 10.1016/S0923-2508(03)00067-6\\\\nCrossRef Full Text | Google Scholar\\\\nBinetti, A. G., and Reinheimer, J. A. (2000). Thermal and chemical inactivation of indigenous Streptococcus thermophilus bacteriophages isolated from Argentinian dairy plants. J. Food Prot. 64, 509–515. doi: 10.4315/0362-028X-63.4.509\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBordons, A., and Reguant, C. (2013). Bioquímica de las bacterias lácticas del vino y la fermentación maloláctica. Acenología\\\\nGoogle Scholar\\\\nBorneman, A. R., McCarthy, J. M., Chambers, P. J., and Bartowsky, E. J. (2012). Comparative analysis of the Oenococcus oeni pan genome reveals genetic diversity in industrially-relevant pathways. BMC Genomics 13:373. doi: 10.1186/1471-2164-13-373\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCalendar, R. L. (2006). The Bacteriophages. New York, NY: Oxford University Press.\\\\nGoogle Scholar\\\\nCosta, A., Barata, A., Malfeito-Ferreira, M., and Loureiro, V. (2008). Evaluation of the inhibitory effect of dimethyl dicarbonate (DMDC) against wine microorganisms. Food Microbiol. 25, 422–427. doi: 10.1016/j.fm.2007.10.003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDavis, C., Silveira, N. F., and Fleet, G. H. (1985). Occurrence and properties of bacteriophages of Leuconostoc oenos in Australian wines. Appl. Environ. Microbiol. 50, 872–876.\\\\nPubMed Abstract | Google Scholar\\\\nDias, R., Volas-Boas, E., Campos, F. M., Hogg, T., and Couto, J. A. (2015). Activity of lysozyme on Lactobacillus hilgardii strains isolated from Port wines. Food Microbiol. 49, 6–11. doi: 10.1016/j.fm.2015.01.007\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDini, C. (2011). Aislamiento y Caracterización Molecular de Bacteriófagos de Bacterias Enteropatógenas Para Biocontrol de Enfermedades Transmitidas por Alimentos (ETA). Doctoral dissertation, Universidad Nacional de la Plata, Buenos Aires.\\\\nGoogle Scholar\\\\nDu Toit, W. J., and Lambrechts, M. G. (2002). The enumeration and identification of acetic acid bacteria from South African red wine fermentations. Int. J. Food Microbiol. 74, 57–64. doi: 10.1016/S0168-1605(01)00715-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nEbrecht, A., Guglielmotti, D. M., Tremmel, G., Reinheimer, J. A., and Suárez, V. B. (2010). Temperate and virulent Lactobacillus delbrueckii bacteriophages: comparison of their thermal and chemical resistance. Food Microbiol. 27, 515–520. doi: 10.1016/j.fm.2009.12.012\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nFernández, L., Gutiérrez, D., García, P., and Rodríguez, A. (2019). The perfect bacteriophage for therapeutic applications – a quick guide. Antibiotics 8:E126. doi: 10.3390/antibiotics8030126\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nFumi, M. D., Krieger-Weber, S., Déléris-Bou, M., Angela, S., and du Toit, M. (2010). Lactobacillus: the next generation of malolactic fermentation starter cultures-an overview. Food Bioprocess Technol. 4, 876–906. doi: 10.1007/s11947-010-0448-448\\\\nCrossRef Full Text | Google Scholar\\\\nGarcía, P., Martínez, B., Obeso, J. M., and Rodríguez, A. (2008). Bacteriophages and their application in food safety. Lett. Appl. Microbiol. 47, 479–485. doi: 10.1111/j.1472765X.2008.02458.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGarcía-Ruiz, A., Crespo, J., López-de-Luzuriaga, J. M., Olmos, M. E., Monge, M., Rodríguez-Álfaro, M. P., et al. (2015). Novel biocompatible silver nanoparticles for controlling the growth of lactic acid bacteria and acetic acid bacteria in wines. Food Control 50, 613–619. doi: 10.1016/j.foodcont.2014.09.035\\\\nCrossRef Full Text | Google Scholar\\\\nGarde-Cerdán, T., López, R., Garijo, P., González-Arenzana, L., Gutiérrez, A. R., López-Alfaro, I., et al. (2013). Application of colloidal silver versus sulfur dioxide during vinification and storage of Tempranillo red wines. Aust. J. Grape Wine Res. 20, 51–61. doi: 10.1111/ajgw.12050\\\\nCrossRef Full Text | Google Scholar\\\\nGonzález, Á, Hierro, N., Poblet, M., Mas, A., and Guillamón, J. M. (2005). Application of molecular methods to demonstrate species and strain evolution of acetic acid bacteria population during wine production. Int. J. Food Microbiol. 102, 295–304. doi: 10.1016/j.ijfoodmicro.2004.11.020\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nGonzález-García, V. (2014). Mecanismos Moleculares Implicados en la Infección del Bacteriófago T7. Doctoral dissertation, Universidad Autónoma de Madrid, España.\\\\nGoogle Scholar\\\\nGonzález-Rompinelli, E. M., Rodríguez-Bencomo, J. J., García-Ruiz, A., Sánchez-Patán, F., Martín-Álvarez, P. J., Bartolomé, B., et al. (2013). A winery-scale trial of the use of antimicrobial plant phenolic extracts as preservatives during wine ageing in barrels. Food Control 33, 440–447. doi: 10.1016/j.foodcont.2013.03.026\\\\nCrossRef Full Text | Google Scholar\\\\nGuerrero, R. F., and Cantos-Villar, E. (2015). Demonstrating the efficiency of sulfur dioxide replacements in wine: a parameter review. Trends Food Sci. Technol. 42, 27–43. doi: 10.1016/j.tifs.2014.11.004\\\\nCrossRef Full Text | Google Scholar\\\\nGullo, M., and Giudici, P. (2008). Acetic acid bacteria in traditional balsamic vinegar: phenotypic traits relevant for starter cultures selection. Int. J. Food Microbiol. 125, 46–53. doi: 10.1016/j.ijfoodmicro.2007.11.076\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHenick-Kling, T., Lee, T. H., and Nicholas, D. J. D. (1986a). Characterization of the lytic activity of bacteriophages of Leuconostoc oenos isolated from wine. J. Appl. Bacteriol. 61, 525–534. doi: 10.1111/j.1365-2672.1986.tb01725.x\\\\nCrossRef Full Text | Google Scholar\\\\nHenick-Kling, T., Lee, T. H., and Nicholas, D. J. D. (1986b). Inhibition of bacterial growth and malolactic fermentation in wine by bacteriophage. J. Appl. Bacteriol. 61, 287–293. doi: 10.1111/j.1365-2672.1986.tb04289.x\\\\nCrossRef Full Text | Google Scholar\\\\nHermoso, J., García, J., and García, P. (2007). Taking aim on bacterial pathogens: from phage therapy to enzybiotics. Curr. Opin. Microbiol. 10, 461–472. doi: 10.1016/j.mib.2007.08.002\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHernández-Magadán, A. (2007). Aislamiento, Caracterización y Detección Precoz de Bacteriófagos de Streptococcus thermophilus en la Industria Láctea. Doctoral dissertation, Universidad de Oviedo, Oviedo.\\\\nGoogle Scholar\\\\nHosseinidoust, Z., Tufenkji, N., and van de Ven, T. G. M. (2011a). Predation in homogeneous and heterogeneous phage environments affects virulence determinants of Pseudomonas aeruginosa. Appl. Environ. Microbiol. 79, 2862–2871. doi: 10.1128/AEM.03817-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHosseinidoust, Z., Van de Ven, T. G. M., and Tufenkji, N. (2011b). Bacterial capture efficiency and antimicrobial activity of phage-functionalized model surfaces. Langmuir 27, 5472–5480. doi: 10.1021/la200102z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nIzquierdo-Cañas, P. M., García-Romero, E., Huertas-Nebreda, B., and Gómez-Alonso, S. (2012). Colloidal silver complex as an alternative to sulphur dioxide in winemaking. Food Control 23, 73–81. doi: 10.1016/j.foodcont.2011.06.014\\\\nCrossRef Full Text | Google Scholar\\\\nJarvis, A. W., Fitzgerald, G. F., Mata, M., Mercenier, A., Neve, H., Powell, I. B., et al. (1991). Species and type phages of lactococcal bacteriophages. Intervirology 32, 2–9. doi: 10.1159/000150179\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nJensen, E. C., Schrader, H. S., Rieland, B., Thompson, T. L., Lee, K. W., Nickerson, K. W., et al. (1998). Prevalence of broad-host-range lytic bacteriophages of Sphaerotilus natans, Escherichia coli and Pseudomonas aeruginosa. Appl. Environ. Microbiol. 64, 575–580.\\\\nPubMed Abstract | Google Scholar\\\\nJucker, W., and Ettlinger, L. (1981). Host range of a bacteriophage of acetic acid bacteria. Int. J. Syst. Evol. Microbiol. 31, 245–246. doi: 10.1099/00207713-31-3-245\\\\nCrossRef Full Text | Google Scholar\\\\nLasanta, C., Roldán, A., Caro, I., Pérez, L., and Palacios, V. (2010). Use of lysozyme for the prevention and treatment of heterolactic fermentation in the biological aging of sherry wines. Food Control 21, 1442–1447. doi: 10.1016/j.foodcont.2010.03.013\\\\nCrossRef Full Text | Google Scholar\\\\nLiburdi, K., Benucci, I., and Esti, M. (2014). Lysozyme in wine: an overview of current and future applications. Compr. Rev. Food Sci. Food Saf. 13, 1062–1073. doi: 10.1111/1541-4337.12102\\\\nCrossRef Full Text | Google Scholar\\\\nLisanti, M. T., Blaiotta, G., Nioi, C., and Moio, L. (2019). Alternative methods to so2 for microbiological stabilization of wine. Compr. Rev. Food Sci. Food Saf. 18, 455–479. doi: 10.1111/1541-4337.12422\\\\nCrossRef Full Text | Google Scholar\\\\nMarchante, L., Izquierdo-Cañas, P. M., Gómez-Alonso, S., Alañón, M. E., García-Romero, E., Pérez-Coello, M. S., et al. (2019). Oenological potential of extracts from winery and cooperage by-products in combination with colloidal silver as natural substitutes to Sulphur dioxide. Food Chem. 276, 485–493. doi: 10.1016/j.foodchem.2018.10.072\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMateluna, R. A. (2006). Estudio de Actividad Antibacteriana de Potenciales Biocontroles Sobre Bacterias Acéticas Involucradas en la Pudrición Ácida de la uva. Master’s thesis, Universidad de Santiago de Chile, Santiago.\\\\nGoogle Scholar\\\\nMathur, M., Vidhani, S., and Mehndiratta, P. (2003). Bacteriophage therapy: an alternative to conventional antibiotics. J. Assoc. Physicians India 51, 593–596.\\\\nGoogle Scholar\\\\nMoineau, S., Pandian, S., and Klaenhammer, T. R. (1993). Restriction/modification systems and restriction endonucleases are more effective on lactococcal bacteriophages that have emerged recently in the dairy industry. Appl. Environ. Microbiol. 59, 197–202.\\\\nPubMed Abstract | Google Scholar\\\\nOcón, E. (2014). Diversidad de Levaduras no-Saccharomyces en Diferentes Ecosistemas Vitivinícolas. Doctoral dissertation, Universidad de La Rioja, Logroño.\\\\nGoogle Scholar\\\\nOugh, C. S., and Were, L. (2005). “Sulfur dioxide and Sulfites,” in Antimicrobials in Food, eds P. M. Davinson, J. N. Sofos, and A. L. Branen (Boca Raton, FL: Taylor & Francis), 143–160.\\\\nGoogle Scholar\\\\nPhilippe, C., Krupovic, M., Jaomanjaka, F., Claisse, O., Petrel, M., and le Marrec, C. (2018). Bacteriophage GC1, a novel tectivirus infecting Gluconobacter cerinus, an acetic acid bacterium associated with wine-making. Viruses 10:39. doi: 10.3390/v10010039\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPremarathne, J. M. K. J. K., Thung, T. Y., New, C. Y., Huat, J. T. Y., Basri, D. F., Rukayadi, Y., et al. (2017). Distribution of bacteriophages in food and environment samples. Int. Food Res. J. 24, 888–896.\\\\nGoogle Scholar\\\\nPuertas, B. (1989). El Potencial Vitícola y Enológico de Quince Variedades Blancas de vid en la Zona de Jerez. Doctoral dissertation, Universidad de Cádiz, Cádiz.\\\\nGoogle Scholar\\\\nQuiberoni, A., Guglielmotti, D. M., and Reinheimer, J. A. (2003). Inactivation of Lactobacillus delbrueckii bacteriophages by heat and biocides. Int. J. Food Microbiol. 84, 51–62. doi: 10.1016/s0168-1605(02)00394-x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRaposo, R., Chinnici, F., Ruíz-Romero, M. J., Puertas, B., Cuevas, F. J., Carbú, M., et al. (2018). Sulfur free red wines through the use of grapevine shoots: impact on the wine quality. Food Chem. 243, 453–460. doi: 10.1016/j.foodchem.2017.09.111\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRojo-Bezares, B., Sáenz, Y., Zarazaga, M., Torres, C., and Ruiz-Larrea, F. (2007). Antimicrobial activity of nisin against Oenococcus oeni and other wine bacteria. Int. J. Food Microbiol. 116, 32–36. doi: 10.1016/j.ijfoodmicro.2006.12.020\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRoldán, A., Lloret, I., and Palacios, V. (2017). Use of a submerged yeast culture and lysozyme for the treatment of bacterial contamination during biological aging of sherry wines. Food Control 71, 42–49. doi: 10.1016/j.foodcont.2016.06.016\\\\nCrossRef Full Text | Google Scholar\\\\nSails, D., Wareing, D. R., Bolton, F. J., Fox, J., and Curry, A. (1998). Characterization of 16 Campylobacter jejuni and E. coli typing bacteriophages. J. Med. Microbiol. 47, 123–128.\\\\nPubMed Abstract | Google Scholar\\\\nSambrook, J., and Russell, D. W. (2001). Molecular Cloning: A Laboratory Manual, 3rd Edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.\\\\nGoogle Scholar\\\\nSchocher, A. J., Kuhn, H., Schindler, B., Palleroni, N. J., Despreaux, C. W., Boublik, M., et al. (1979). Acetobacter bacteriophage A-1. Arch. Microbiol. 121, 193–197. doi: 10.1007/BF00689986\\\\nCrossRef Full Text | Google Scholar\\\\nSellmer, S., Sievers, M., and Teuber, M. (1992). Morphology, virulence and epidemiology of bacteriophage particles isolated from industrial vinegar fermentations. Syst. Appl. Microbiol. 15, 610–616. doi: 10.1016/S0723-2020(11)80123-2\\\\nCrossRef Full Text | Google Scholar\\\\nShivu, M. M., Rajeeva, B. C., Girisha, S. K., Karunasagar, I. I., and Krohne, G. (2007). Molecular characterization of Vibrio harveyi bacteriophages isolated from aquaculture environments along the coast of India. Environ. Microbiol. 9, 322–331. doi: 10.1111/j.1462-2920.2006.01140.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSonni, F., Cejudo Bastante, M. J., Chinnici, F., Natali, N., and Riponi, C. (2009). Replacement of sulfur dioxide by lysozyme and oenological tannins during fermentation: influence on volatile composition of white wines. J. Sci. Food Agric. 89, 688–696. doi: 10.1002/jsfa.3503\\\\nCrossRef Full Text | Google Scholar\\\\nSonni, F., Chinnici, F., Natali, N., and Riponi, C. (2011). Pre-fermentative replacement of sulphur dioxide by lysozyme and oenological tannins: effect on the formation and evolution of volatile compounds during the bottle storage of white wines. Food Chem. 129, 1193–1200. doi: 10.1016/j.foodchem.2011.05.104\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSpricigo, D. A., Bardina, C., Cortés, P., and Llagostera, M. (2013). Use of a bacteriophage cocktail to control Salmonella in food and the food industry. Int. J. Microbiol. 165, 169–174. doi: 10.1016/j.ijfoodmicro.2013.05.009\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nStamm, W. W., Kittelmann, M., Follmann, H., and Trüper, H. G. (1989). The occurrence of bacteriophages in spirit vinegar fermentation. Appl. Microbiol. Biotechnol. 30, 41–46. doi: 10.1007/BF00255994\\\\nCrossRef Full Text | Google Scholar\\\\nSteward, G. F. (2001). Fingerprinting viral assemblages by pulsed field gel electrophoresis (PFGE). Methods Microbiol. 30, 85–103. doi: 10.1016/s0580-9517(01)30041-7\\\\nCrossRef Full Text | Google Scholar\\\\nSturm, M. E., Arroyo-López, F. N., Garrido-Fernández, A., Querol, A., Mercado, L. A., Ramírez, M. L., et al. (2013). Probabilistic model for the spoilage wine yeast Dekkera bruxellensis as a function of pH, ethanol and free SO2 using time as a dummy variable. Int. J. Food Microbiol. 170, 83–90. doi: 10.1016/j.ijfoodmicro.2013.10.019\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSuárez, V. B., and Reinheimer, J. A. (2002). Effectiveness of thermal treatments and biocides in the inactivation of Argentinian Lactococcus lactis phages. J. Food Prot. 65, 1756–1759. doi: 10.4315/0362-028X-65.11.1756\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSulakvelidze, A. (2013). Using lytic bacteriophages to eliminate or significantly reduce contamination of food by foodborne bacterial pathogens. J. Sci. Food Agric. 93, 3137–3146. doi: 10.1002/jsfa.6222\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nValera, M. J., Sáinz, F., Mas, A., and Torija, M. J. (2017). Effect of chitosan and SO2 on viability of Acetobacter strains in wine. Int. J. Food Microbiol. 246, 1–4. doi: 10.1016/j.ijfoodmicro.2017.01.0220168.1605\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWaters, C. M., and Bassler, B. L. (2005). Quorum sensing: cell-to-cell communication in bacteria. Ann. Rev. Cell Dev. Biol. 21, 319–346. doi: 10.1146/annurev.cellbio.21.012704.131001\\\\nCrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.02169\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.02169/full\",\\n    \"loadedTime\": \"2024-01-23T08:50:30.641Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.02169\",\\n    \"title\": \"Frontiers | Jumbo Bacteriophages Are Represented Within an Increasing Diversity of Environmental Viruses Infecting the Emerging Phytopathogen, Dickeya solani\",\\n    \"description\": \"\\\\\\\\textit{Dickeya} species are economically important phytopathogens widespread in mainland Europe that can reduce crop yields by 25$\\\\\\\\%$. There are no effective environmentally-acceptable chemical systems available for diseases caused by \\\\\\\\textit{Dickeya}. Bacteriophages have been suggested for use in biocontrol of these pathogens in the field, and limited field trials have been conducted. To date the majority of bacteriophages capable of infecting \\\\\\\\textit{Dickeya solani}, one of the more aggressive species, are from the same family, the \\\\\\\\textit{Ackermannviridae}, many representatives of which have been shown to be unsuitable for use in the field due to their capacity for generalised transduction. Members of this family are also only capable of forming individual plaques on \\\\\\\\textit{Dickeya solani}. Here we describe novel bacteriophages from environmental sources isolated on \\\\\\\\textit{Dickeya solani}, including members of two other viral families; \\\\\\\\textit{Myoviridae} and \\\\\\\\textit{Podoviridae}, most of which are capable of forming plaques on multiple \\\\\\\\textit{Dickeya} species. Full genomic sequencing revealed that the \\\\\\\\textit{Myoviridae} family members form two novel clusters of jumbo bacteriophages with genomes over 250 kbp, with one cluster containing phages of another phytopathogen \\\\\\\\textit{Erwinia amylovora}. Transduction experiments showed that the majority of the new environmental bacteriophages are also capable of facilitating efficient horizontal gene transfer, however the single \\\\\\\\textit{Podoviridae} family member is not. This particular phage therefore has potential for use as a biocontrol agent against multiple species of \\\\\\\\textit{Dickeya}.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"1. Introduction\\\\nThe genus Dickeya, recently reclassified into the novel family Pectobacteriaceae (Adeolu et al., 2016), currently consists of 11 phytopathogenic species that can cause severe disease in economically important crops including tomato, orchid, and potato (Alic et al., 2017a). Until 2004, almost all European potato isolates of Dickeya were assigned as Dickeya dianthicola (Toth et al., 2011). In 2008/2009, a new clade of Dickeya in European potato isolates was identified (Laurila et al., 2008; Parkinson et al., 2009; Sławiak et al., 2009) and in 2014 a new species was proposed; Dickeya solani (van der Wolf et al., 2014).\\\\nDickeya solani is able to spread more easily through the plant vascular system and survive at higher temperatures than D. dianthicola (Toth et al., 2011). It is currently the predominant potato pathogen in Europe, with reductions in yield of up to 25% reported in potatoes exposed to Dickeya species (Tsror et al., 2009). Whilst there have been isolated cases of D. solani reported in England and Wales since 2007 (Cahill et al., 2010), these were all found in crops originating from outside of the UK (Toth et al., 2016). It is currently yet to become established in the UK, and to mitigate the significant economic cost inflicted by this virulent phytopathogen, the Scottish government has introduced specific legislation aimed at preventing the establishment of D. solani in its seed industry (Mansfield et al., 2012).\\\\nThe significant economic costs inflicted by Dickeya species have stimulated research interest in methods for control of these virulent phytopathogens. Bacterial viruses (bacteriophages; phages) have been suggested as potential tools for biocontrol due to their specificity, environmental persistence and biological “organic” nature (Iriarte et al., 2007; Czajkowski et al., 2017; Svircev et al., 2018). Several studies have isolated phages capable of infecting Dickeya species (Adriaenssens et al., 2012c; Czajkowski et al., 2014a,b, 2015a; Matilla et al., 2014; Alič et al., 2017b; Day et al., 2017). Their potential use as biocontrol agents has been trialed both in the lab and in the field (Adriaenssens et al., 2012c) and these studies showed a partially “therapeutic” outcome with reduced crop losses. There is a commercial product available, Biolyse™, from APS Biocontrol Ltd that is a phage cocktail able to target Pectobacterium as well as Dickeya species. Designed as a washing solution for potatoes during factory processing, to our knowledge it is the first, and currently the only, commercial Dickeya-targeting biocontrol product. It has been reported that Biolyse™ has been used by the UK supermarket chain Tesco (Branston, 2012). The identities of the phages contained within this cocktail however have not been reported.\\\\nAll of the Dickeya phages isolated so far, and 96% of all known phages, are members of the order Caudovirales (Fokine and Rossmann, 2014), which currently consists of four families. Apart from the Siphoviridae family member BF-CIM1/14 recently described by Alič et al. (2017b) and three Podoviridae family members reported in our recent publication (Day et al., 2017), the vast majority of D. solani phages characterized so far share a high degree of similarity and have been designated members of the Ackermannviridae family (formerly known as the Vi1virus genus; Adriaenssens et al., 2018) based on morphology and genomic comparisons. A summary of these phages is shown in Table 1. This has generated research interest, as these phages have been isolated from both soil and water samples and in three separate European countries; Belgium, Poland, and the United Kingdom. Host range testing has shown that the phages isolated in Belgium and the majority isolated in the UK are capable of forming plaques on strains of D. solani only (Adriaenssens et al., 2012c; Day et al., 2017). The phages isolated in Poland are reported to infect multiple species of Dickeya and Pectobacterium (Czajkowski et al., 2014a,b, 2015a), however, host range testing to the level of individual plaque formation has not been reported—an important criterion that allows exclusion of false positives (Khan Mirzaei and Nilsson, 2015). The high degree of morphological and genomic similarity between these phages and the other Ackermannviridae family members makes the reported broader host range that spans genera an intriguing prospect, assuming plaque formation data supporting this broader host range can be confirmed.\\\\nTable 1. Members of the Ackermannviridae family isolated on Dickeya solani.\\\\nSixty-seven phages were described in our recent publication (Day et al., 2017), 59 of which were only capable of forming plaques on D. solani species. When two were genomically sequenced they showed a high degree of similarity with the previously published D. solani phages of the Ackermannviridae family. The remaining eight phages were capable of forming plaques on other species of Dickeya, including Dickeya zeae, Dickeya chrysanthemi, and Dickeya paradisiaca. These particular phages warranted further investigation, as an expanded host range can be helpful for further application in phage therapy. The aim of this study therefore was to genomically sequence these phages to determine their similarity to previously published D. solani phages and their suitability for use in phage therapy.\\\\n2. Results\\\\n2.1. Host Range and Plaque Morphology\\\\nThe plaques of the previously described D. solani phages that are members of the Ackermannviridae family have tended to be clear, defined, and easy to distinguish from the bacterial top lawn (Adriaenssens et al., 2012c; Czajkowski et al., 2015a). This is the case for the other Ackermannviridae family members isolated in this laboratory, and is also true of one of the broader host range phages JA10. However, the other seven have an indistinct, turbid plaque morphology that is often hard to distinguish from the bacterial top lawn (data not shown). We believe this to be the reason why the host range of the eight D. solani phages shown in Table 2 is different from the host range reported in the previous publication (Day et al., 2017). This first became apparent due to confusing results generated by related, unpublished experiments that suggested a variation in the host range from that published in our previous paper (Day et al., 2017). Rigorous retesting has confirmed that the host range data presented in Table 2 are accurate and that the previous interpretations were incorrect. The efficiency of plating data in Table 2 shows that most of the phages are able to adsorb at a similar efficiency to all species of Dickeya, apart from JA29 which has an efficiency 10−4 lower on D. paradisiaca and D. dadantii subsp. dieffenbachiae.\\\\nTable 2. Broader host range of eight phages capable of infecting other species of Dickeya as well as Dickeya solani.\\\\n2.2. Morphological Classification\\\\nBacteriophages have traditionally been classified based on morphological characteristics viewed under electron microscopy (Ackermann, 2012). The majority of D. solani phages isolated to date are members of the Ackermannviridae family, which share common morphological characteristics. A representative of this family, XF4, is shown in Figure 1A. These phages possess an icosahedral head with a diameter of around 90 nm, a contractile tail around 110 nm in length and structures at the base of the tail that have been described as “stars” or “prongs” and have been identified as tail spikes (Adriaenssens et al., 2012a). Apart from these tail spikes, this is classical morphology of the phage family Myoviridae, therefore the combination of a contractile tail and tail spikes are the morphological markers of the family Ackermannviridae.\\\\nFigure 1. Transmission electron micrographs of six Dickeya solani phages. XF4 (A) is a member of the Ackermannviridae family, exhibiting an icosahedral head of around 90 nm, a tail length of 110 nm and tail spikes at the base of the tail. JA10 (B) is a member of the Podoviridae family, characterized by a short stubby tail. JA11, JA29, and AD1 (C–E) are members of the Myoviridae family with unclear tail appendages. AD2 (F) has a similar morphology to XF4 and has a partially-contracted tail.\\\\nAll eight of the expanded host range phages were viewed under transmission electron microscopy. Seven of them had an icosahedral head and long tail, with the structures at the base of the tail remaining unclear, with two representatives shown in Figures 1C,D. This marks them as either members of the Myoviridae or Ackermannviridae families, however, these phages were significantly larger than the previously viewed members of the Ackermannviridae family, which can be seen by comparing Figure 1A with Figure 1D. The head diameters were over 120 nm, with a tail length of around 150 nm, which suggested that these phages were not Ackermannviridae family members.\\\\nThe indistinct morphology of the tail appendages of the JA jumbo phages (best seen in Figure 1C) has also been identified in other phages. When first described in the Escherichia phage 121Q (Ackermann and Nguyen, 1983) this morphology was presumed to be an artifact of microscopy involving damage to the tail. It was also thought that the dimensions, at the time reported to be a head diameter of 150 nm and a tail length of 165 nm, were overstated. However, this morphology has since been directly reported in the Pseudomonas putida phage Lu11 (Adriaenssens et al., 2012b), the Pectobacterium carotovorum phage CBB (Buttimer et al., 2017), and the Erwinia amylovora phage Y3 (Buttimer et al., in press), and has been dubbed the “hairy Myoviridae” morphology (Buttimer et al., 2017).\\\\nUnexpectedly, as can be seen in Figure 1B, the phage JA10 could be classified as a member of the Podoviridae family when imaged, characterized by an icosahedral head and a short non-contractile tail. Whilst this is not the first member of this family that we have isolated (Day et al., 2017), this is the first isolate we have studied in further depth. The genome of JA10 was therefore sequenced to investigate the similarity between it and previously published Dickeya-infecting Podoviridae family members (Alič et al., 2017b).\\\\n2.3. Genome Sequence of Podoviridae Family Member JA10\\\\nThe genome of JA10 is 40,131 bp, has 50 predicted genes, and is shown in Figure 2. The closest match in the database is an as yet unpublished D. solani phage Ninurta (Genbank reference: MH059639) isolated from organic waste in Denmark that shares 95% DNA identity with JA10. The closest published phage is the Pectobacterium parmentieri phage PP74 isolated from potato washing waste water in Russia in 2015 (Kabanova et al., 2018), which shares less than 14% nucleotide identity. It shares no DNA sequence identity with the other sequenced Dickeya-infecting Podoviridae family member BF25/12 (Alič et al., 2017b). PP74 has been designated as a T7-like virus and a member of the Autographivirinae subfamily, with a conserved core genome. A translated nucleotide comparison of JA10 with the type phage T7 showed that most of the predicted genes are conserved (data not shown), including almost all the genes with a proposed function, such as the T3/T7-like RNA polymerase, structural capsid genes and DNA packaging machinery. JA10 is therefore also a member of the Autographivirinae subfamily.\\\\nFigure 2. Map of the genome of JA10. The outer gray ring marks open reading frames, with those highlighted in black discussed in more detail in the text, whilst the inner ring categorizes the proposed functions of these genes. The genome map was generated using Circos.\\\\nThere are several putative proteins encoded in the JA10 genome that do not have significant homology in the T7 genome and these are highlighted in Figure 2. Four of these seven putative proteins, the S-adenosyl-L-methionine hydrolase, bacterial RNA polymerase inhibitor, minor capsid protein, and the first tail assembly protein, have a protein of similar function encoded at this position in T7. The variation between JA10 and T7 in these putative proteins is therefore likely a determinant of host specificity. The marked tail fiber protein, which share a common N-terminal region but differ at the C-terminus between the two phages, is also likely a contributor to host range specificity. The DNA ligase highlighted in Figure 2 neighbors a conserved ligase and consists of fewer than 200 amino acids. This is therefore a possible result of a recombination duplication event and may not be a functional protein. The final tail assembly protein, close to the end of the JA10 genome, has no functional homologue in T7.\\\\n2.4. Novel Jumbo “Hairy Myoviridae” Phages\\\\nSequencing of the other seven genomes showed that, although the phages were isolated independently, several shared 100% identity at the nucleotide level. JA11, 31, and 32 grouped together, as did JA33 and 37. JA31, 32, and 37 were therefore excluded from further analysis. The genome size for the four remaining phages is between 253 and 256 kbp. A summary of the characteristics of these genomes, as well as JA10, is shown in Table 3. These phages are therefore jumbo phages, defined as phages with a genome over 200 kbp (Yuan and Gao, 2017). The genomes are significantly larger than the conserved size of the Ackermannviridae family genomes, which are around 150 kbp, and larger than most sequenced phages. As of July 2018, there were 9,351 recorded phage genome sequences in Genbank (http://millardlab.org/bioinformatics/bacteriophage-genomes/) and these JA phages would be the 62nd–65th largest sequenced. Many of the over 300 predicted open reading frames in each genome do not match any known genes; the majority of those that do share identity with known genes are from the E. amylovora phages Yoloswag (Esplin et al., 2017) and Y3 (Buttimer et al., in press). These are largely structural genes and genes involved in DNA metabolism and replication.\\\\nTable 3. Summary of the genomes of the broader host range phages.\\\\n2.5. Variation Within the JA Phages\\\\nThe gene order of the four JA jumbo phages is largely conserved. Over three quarters of the predicted ORFs are annotated as encoding hypothetical proteins, and many of the differences between the phages are contained within these ORFs as shown in Figure 3. JA29 is the most dissimilar to the others, sharing 86% nucleotide identity with JA11, whereas the nucleotide identity between JA11 and JA13 is 95%. JA11 and JA33 are 99% identical, with the major difference being the insertion of 126 bp in both genomes at different positions, and of different sequences. These insertions are in non-coding regions however, therefore their biological relevance is unclear. The only other differences are in two putative proteins. One is a hypothetical protein, whilst the other contains a putative discoidin domain, with the substitution between JA11 and JA33 (alanine to threonine) in the middle of the domain. Discoidin domains are present in eukaryotic agglutination factors and therefore the possible biological role for this in a phage genome, and the effect of the substitution, is not immediately obvious.\\\\nFigure 3. Map of the genome of JA11. The outer gray ring marks open reading frames, with those highlighted in black discussed in more detail in the text. The second ring categorizes the proposed functions of these genes, whilst the inner rings highlight the areas of the genome that differ from the genomes of JA29, JA13, and JA33 (third to fifth ring, respectively). The genome map was generated using Circos.\\\\nJA11, 13, and 29 have differing host ranges, as listed in Table 2. To investigate whether this was caused by variations within the tail fibers of the phages, the amino acid sequences of each of the three putative tail fiber proteins was compared between the phages. JA11 and JA13 possess identical tail fibers, whereas JA29 shows variations of several amino acids in each protein, as listed in Table 4. Whilst these variations could explain the differing host range of JA11 and JA29, it does not explain the difference in host range observed for JA11 and JA13.\\\\nTable 4. Summary of the annotated genes which differ between JA11, JA13, and JA29.\\\\nWhilst most of the differences between the phages are located in genes with no predicted function, there are a few annotated that are present in all of the JA phages. These include encoding a DNA helicase, two potential transcription factors and one structural protein, all highlighted in Figure 3 and listed in Table 4.\\\\nThere are variations in two DNA related genes: a DNA primase and a DNA helicase. The helicase shows the most variation between the phages, as it appears to have undergone insertion or deletion between some of the phages. A comparison of this region of the genome can be seen in Figure 4. There are two ORFs annotated as putative helicases in JA11 and JA33, which both share homology with one ORF in JA13 and JA29. Whether the two ORFs are able to function independently as a helicase, or whether this duplication has rendered them non-functional, is unknown. A DksA/TraR family protein and a ssDNA binding protein, both likely transcription factors, both differ in one amino acid between JA11 and JA13, and share lower identity with the JA29 homolog, particularly the ssDNA binding protein, which differs in 32 positions. A VgrG-like family protein, a component of the T6SS thought to be phage-derived as it is capable of assembling into a structure similar to a phage tail spike (Cianfanelli et al., 2016) shares, at best, only 33% amino acid identity with the closest hit in the E. amylovora phage Y3, and so these may define a relatively novel VgrG-like protein group. JA11 and JA13 differ by one conservative substitution in this protein, whilst JA29 differs in 87 amino acids, 13 of which are conservative substitutions. It is possible that differences in this predicted protein are a contributing factor to the differing host range of these phages.\\\\nFigure 4. Translated nucleotide comparison of a putative DNA helicase between JA11 (top), JA33 (second), JA13 (third), and JA29 (bottom). Red bars mark areas of amino acid identity, with darker colors showing higher identity. Figure produced using the Artemis Comparison Tool.\\\\nThe biological significance of the differences observed between the four JA phages is currently unclear. It is somewhat surprising to find variations in genes involved in transcription initiation and DNA replication, as the genomes of these phages are relatively similar in both size and GC content, as summarized in Table 3. It is therefore possible that these differences do not significantly alter the function of these proteins. The variation in the VgrG-like proteins is more logical, as the different host ranges of these phages may be related to differences in tail spikes and other host recognition factors. To determine the impact of these differences, and to investigate why JA11 and JA13 have a different host range despite having apparently identical tail fibers, requires further experimental work.\\\\n2.6. More Recent Isolates: AD Phages\\\\nAll of the JA phages were isolated from the River Cam in November 2014. Isolation of XF phages from the same location a year earlier produced mainly members of the Ackermannviridae family and a few Podoviridae family members (Day et al., 2017). Whilst there is clearly some maintenance of viral populations, as members of the two families have been isolated on both occasions, the jumbo phages presented here are a novel grouping. To gain further insight into the viral populations in the River Cam, further samples were taken in October 2017. Two phages were isolated on D. solani and are named AD1 and AD2. Whilst both were only capable of forming plaques on D. solani and not on strains of other species, microscopy showed that they had differing morphologies. AD1 (Figure 1E) appears to have a “hairy Myoviridae” morphology similar to that of the JA jumbo phages, with a head diameter of 120 nm, tail length of 150 nm, and unclear structures at the base of the tail. AD2 on the other hand (Figure 1F) has a head diameter of 90 nm and a (potentially partially-contracted) 70 nm tail, leading to a tentative classification as a member of the Ackermannviridae family. The structures at the end of the tail are inconclusive.\\\\nGenome sequencing of the two AD phages showed that, as suggested by microscopy, AD2 is a member of the Ackermannviridae family. It shares 99% nucleotide identity with previously published D. solani Ackermannviridae such as XF4 and LIMEstone1 (Day et al., 2017), although full coverage of the genome was not achieved (data not shown). AD1, as expected, has a large genome of 261,658 bp, confirming that it is a jumbo phage, shown in Figure 5. However, unexpectedly, it has low nucleotide sequence identity with the JA jumbo phages, despite sharing a similar gene order. In fact, a translated nucleotide comparison of JA11 and AD1, as shown in Figure 6C, shows that JA11 is about as similar to AD1 as it is to Y3 (Figure 6A), and a comparison of AD1 and Y3 (Figure 6B) shows them to be more similar to each other than to JA11 at the amino acid level.\\\\nFigure 5. Map of the genome of AD1. The outer gray ring marks open reading frames whilst the inner ring categorizes the proposed functions of these genes. The genome map was generated using Circos.\\\\nFigure 6. Translated nucleotide comparison of the genomes of JA11 (A), Y3 (B), and AD1 (C). Red bars mark areas of amino acid identity, with darker colors showing higher identity. Blue bars highlight areas of inversion. Figure produced using the Artemis Comparison Tool.\\\\n2.7. Phylogeny of the “Hairy Myoviridae” Phages\\\\nIn their recent publication, Buttimer et al. discussed the phylogenetic position of Y3 considering its low level of nucleotide identity to existing genomes (Buttimer et al., in press). Two potential subfamilies within the “hairy Myoviridae” have emerged; the Rak2-like phages, which includes the previously mentioned Pectobacterium phage CBB (Buttimer et al., 2017), and the as yet unnamed subfamily that encompasses the phage discussed here. This was established as it was found that Y3 had homologs including terminase, polymerase and helicase genes in several other phages reported or suspected to have the “hairy Myoviridae” morphology. A comparison of the large terminase subunit of these phages with those reported here shows clear clustering, and can be seen in Figure 7. As expected, the three JA phages cluster tightly with little variation between them. As reported by Buttimer et al. the Pseudomonas-infecting phages PaBG (Sykilinda et al., 2014) and Lu11 (Adriaenssens et al., 2012b) form a clade, whilst the Ralstonia solanacearum phage phiRSL1 (Yamada et al., 2010) and the metagenomically-derived NCTB (Pfreundt et al., 2017) are single nodes within the tree. As suggested by the translated nucleotide comparison in Figure 6B, Y3 and AD1 form a clade that puts AD1 closer to Erwinia-infecting phages than to the other D. solani phages. Intriguingly, AD1 is placed closer phylogenetically to Y3 than the other E. amylovora-infecting phage Yoloswag (Esplin et al., 2017). The same clustering is seen when using the sequence of the tail sheath proteins of the phages. All of the phages except phiRSL1 have two annotated tail sheath proteins, and the same phylogeny is seen with both (data not shown).\\\\nFigure 7. Phylogenetic tree of the large terminase subunit constructed using the Maximum Likelihood method in MEGA. All positions containing gaps and missing data were eliminated, with 642 positions in the final dataset. The tree shown has the highest log likelihood (−8677.52) and is drawn to scale, with branch lengths measured in the number of substitutions per site.\\\\nThe gene order between JA11, AD1, and Y3 is highly conserved. All three genomes contain over 300 open reading frames, with each containing only between one and three unique annotated genes. These unique genes are listed in Table 5 and are all DNA or metabolism-related. There are also five genes common to AD1 and Y3 that are not found in JA11. Whilst phylogenetic clustering, as shown in Figure 7, groups AD1 and Y3 closer than Y3 and Yoloswag, it is interesting to note that the two unique genes possessed by Y3 have homologs in Yoloswag. These two phages were both isolated from apple orchards using E. amylovora, therefore it is surprising that they differ phylogenetically. It is possible that these unique genes are a determinant of the host range of these phages. A phylogenetic comparison of three tail fiber genes found in each genome is shown in Figure 8. This again shows a definite separation between Yoloswag and the other two phages, particularly when comparing Yoloswag_102 with Y3_104 and AD1_102, which occupy the same genomic position. This also suggests the possibility that AD1 may be capable of forming plaques on Erwinia species, but we have not been able to test this as we do not have access to these hosts.\\\\nTable 5. Unique annotated genes found in the genomes of JA11, AD, and Y3, as well as genes common to AD1 and Y3 but not present in JA11.\\\\nFigure 8. Phylogenetic tree of three tail fiber proteins constructed using the Maximum Likelihood method in MEGA. All positions containing gaps and missing data were eliminated, with 158 positions in the final dataset. The tree shown has the highest log likelihood (−2426.84) and is drawn to scale, with branch lengths measured in the number of substitutions per site.\\\\n2.8. Jumbo Phages Are Capable of Transduction\\\\nIn a recent publication (Day et al., 2017) we tested the ability of the JA phages to facilitate transduction of chromosomal markers and plasmids between Dickeya cells. We can reconfirm that all of the JA jumbos are capable of transduction of a chromosomal marker between D. solani cells at a frequency of between 1 × 10−6 and 3 × 10−4 and report that the AD phages are capable of transduction of chromosomal markers at similar frequencies. The broader host range of the JA jumbo phages also allows transduction of plasmids between Dickeya species, as shown for a representative of each host range group in Table 6. JA10, the member of the Podoviridae family, proved incapable of transduction under the conditions tested. This makes JA10 a more promising candidate for phage therapy, and suggests the other phages may not be suitable due to the potential risk of transduction in the field.\\\\nTable 6. Transduction frequencies of the plasmid pECA1039-Km3 from donor Dickeya solani cells into other Dickeya species.\\\\n3. Discussion\\\\nAll seed crops imported into Scotland, along with 10% of Scottish-origin crops, are tested for D. solani each year. In 2017, the most recent year for which there are data, 663 samples were tested and none were positive for Dickeya species (Scottish Government, 2017), which has been the finding since 2010 when rigorous testing began. Whilst there have been isolated cases of D. solani infection reported in England and Wales since 2007 (Cahill et al., 2010), these have all originated in crops from outside of the UK (Toth et al., 2016), therefore it is not thought that Dickeya is established in the UK. This begs the question as to why we have been able to isolate phages capable of infecting D. solani with relative ease from the River Cam. We would hypothesize that either D. solani is present in the environment, but has not been confirmed by testing, or there is another permissive, but currently unknown, environmental host(s) for these phage. A novel species of Dickeya, Dickeya aquatica, was isolated from waterways in England (Parkinson et al., 2014), and so far has only been identified in waterway environments. It is, therefore, a formal possibility that this species could be an environmental host for the phages isolated here, but this has not yet been tested.\\\\nPreviously isolated phages of D. solani have been found almost exclusively to be members of the Ackermannviridae family. This has been a consistent feature of isolations spanning multiple European countries across the last decade, including from both soil and water samples, as shown in Table 1. We questioned in our recent publication whether this indicated a special relationship between Ackermannviridae and D. solani (Day et al., 2017). The phages presented here confirm that that was the result of an extrapolation from a limited viral sample set. There are at least four groups of D. solani phages present in waterways around Cambridge, spanning three of the four Caudovirales families. Phages that have been isolated on other species of Dickeya that are capable of forming plaques on D. solani have also been recently described (Alič et al., 2017b), including the fourth Caudovirales family, Siphoviridae. It is therefore apparent that we are only superficially defining the level of phage diversity present in the environment, consistent with the notion that double-stranded DNA phages alone have been predicted to outnumber their bacterial hosts by a factor of 10 to 1 (Chibani-Chennoufi et al., 2004).\\\\nAll the phages presented here, apart from JA10 and AD2, appear to be representatives of a recently described “hairy Myoviridae” subfamily (Buttimer et al., in press). To the best of our knowledge, these are the first reported members of this subfamily isolated using Dickeya species. Many of the previously reported members of this subfamily were also isolated on plant-associated bacteria such as Pseudomonas putida (Adriaenssens et al., 2012b) and E. amylovora (Esplin et al., 2017; Buttimer et al., in press). Whether there is a link between this group of phages and plant-associated bacteria, or whether the recent increase in isolation of phages using phytopathogens is skewing this view remains to be determined. The proteins responsible for the “hairs” that typify this grouping remain unknown, although the identified tail fiber proteins are likely candidates for further investigation.\\\\nInfection of seed crops with D. solani and related species inflicts a high economic burden, and therefore there is great interest in the use of virulent (lytic) bacteriophages as potential biocontrol agents. There have been multiple tests of the stability, environmental viability and efficacy of Dickeya phages (Adriaenssens et al., 2012c; Alič et al., 2017b; Czajkowski et al., 2017) in which promising results have been reported. The Dickeya phages able to form individual plaques through productive lytic cycle replication on multiple Dickeya species, reported both here and by Alič et al. (2017b), are potentially more promising for use as biocontrol agents, as they would be able to act against a wider set of phytopathogens. However, we have described here, and previously (Day et al., 2017), that the majority of the D. solani phages (including our phages and the LIMEstone phages) are able to facilitate generalized transduction between host cells. The phages isolated by Czajkowski et al. (2014a,b, 2015a) were not reported to have been tested for generalized transduction, but, due to their classification as Ackermannviridae family members, and the finding that all members of this family tested to date are capable of facilitating transduction, we predict that they are likely to be capable of doing so. Alič et al. did not report testing of their phages for transduction capacity, and it is possible that they may not be transducers, but we would echo the caution of the European Medicines Agency, among others, who have stated that it is “important to ensure that therapeutic phages do not carry out generalized transduction” (Pelfrene et al., 2016). However, the results presented here do suggest that JA10, a podovirus capable of infecting three Dickeya species other than D. solani, could offer some promise as a potentially therapeutic candidate, as it has not shown transduction capabilities when tested.\\\\nCzajkowski et al. (2014a) have reported that phages D3 and D5 are capable of infecting multiple species of Dickeya. This conclusion was based on simple spot test assays in which undiluted spots of phage lysate were tested on bacterial top lawns and incubated, with any resultant clearing taken to show infection. It is known that applications of high titre lysates of phage to bacterial cells can cause the phenomenon of “lysis-from-without,” in which cells lyse due to membrane disruption instead of productive phage infection (Khan Mirzaei and Nilsson, 2015). Consequently, confirmation of host range requires serial dilution of the phage lysate to visualize individual plaques on a host, and this confirmatory data would be helpful when assessing the reported host range of these phages. This reasonable caution is reinforced, particularly when we consider the genome identity of nearly 100 % with other Ackermannviridae family members.\\\\nThe phages PD10.3 and 23.1 (Czajkowski et al., 2015a) are also reported to infect both Dickeya and Pectobacterium species, although host range was determined by the same method as D3 and D5. However, adsorption and burst size data for both phages are reported on the two genera. The genomes of both phages have been sequenced and are reported as incomplete. Curiously, the largest scaffold of both is similar to the size of other Ackermannviridae family members (shown in Table 1) and these scaffolds share 99% identity with the full genome of LIMEstone1. The morphology of these two phages also clearly places them within the Ackermannviridae family. It is therefore intriguing that phages that are so similar have such different host ranges, and so further confirmatory data on the broader host range of these two phages could be biologically illuminating. Notwithstanding such observations, interpretation of Dickeya phage host range data should be treated with caution. We offer a salutary lesson, based on our own data, reported here, which more rigorously reinterprets previously reported host range data. Tests of the host range of our phages following the method of Czajkowski et al. have also suggested a much broader host range than we now know to be true (data not shown). Consequently, we would caution against assigning host range to phages without rigorous experimental data involving plaque formation in line with the comments of others (Khan Mirzaei and Nilsson, 2015).\\\\n4. Materials and Methods\\\\n4.1. Bacterial Strains, Phages, Culture Media, and Growth Conditions\\\\nAll bacterial strains used in this study are listed in Table 7. Dickeya species were routinely grown at 30°C in Luria broth (LB) or on LB agar plates (1.5%, wt/vol, agar). Phages were stored at 4°C in phage buffer (10 mM Tris-HCl, pH 7.4, 10 mM MgSO4, 0.01%, wt/vol, gelatine) over a few drops of NaHCO3 saturated chloroform.\\\\nTable 7. Bacterial strains and bacteriophage genomes used in this study.\\\\n4.2. Isolation of Phages\\\\nTreated sewage effluent was collected from a sewage treatment plant in Cambridge, United Kingdom (Matilla and Salmond, 2014). River water was collected from multiple sites along the River Cam. Samples were filter sterilized using a 0.22 μm filter before 5 mL of the sample was added to 2x LB along with 500 μL of an overnight culture of D. solani MK10. This mixture was incubated overnight in a 250 mL flask at 30°C with shaking at 250 rpm. One milliliter of the enriched sample was mixed with 100 μL of chloroform (saturated with NaHCO3) and vortexed to lyse bacterial cells. After sedimentation at 16,000 × g for 4 min, 10 μL of a serial dilution series of the supernatant was mixed with 200 μL of an overnight bacterial culture and 4 mL of LB top agar. This mixture was poured as an overlay on an LBA plate and incubated overnight at 30°C. Single phage plaques were picked with a sterile toothpick, placed into 100 μL phage buffer, and shaken with 40μL of chloroform to kill any bacteria. The phages obtained were plaque purified three times. High-titer phage lysates were then obtained as described previously (Petty et al., 2006). Briefly, 10-fold serial dilutions of the phage were incubated overnight in an agar overlay. Those plates exhibiting confluent lysis (seen as a mosaic-like effect in which the plaques are close to merging) were used for lysate preparation. The top agar was removed from the plate, vortexed with chloroform before sedimentation at 2,200 × g for 20 min at 4°C. The supernatant was removed and vortexed with a few drops of chloroform to produce the final lysate.\\\\n4.3. Determination of Host Range\\\\nThe host range of isolated phages was determined by plating out ten-fold serial dilutions of the phage lysates onto agar overlays containing host Dickeya cells and incubating overnight at 30°C. Following best practice to avoid potential confusion with “lysis from without,” only phages that produced individual plaques following serial dilution on three independent occasions were considered as being able to infect the respective host productively through a lytic cycle.\\\\n4.4. Transmission Electron Microscopy\\\\nHigh-titre lysates for transmission electron microscopy were obtained as described above using 0.35% (w/v) LB agarose instead of 0.35% (w/v) LB agar overlays. Ten μL of high-titre phage lysates were adsorbed onto 400-mesh copper grids with holey carbon support films (Agar Scientific, Stansted, United Kingdom) for 2 min. The copper grids were discharged in a Quorum/Emitech K100X system (Quorum, Ringmer, United Kingdom) prior to use. Excess phage suspension was removed with filter paper and phage samples were negatively stained by placing the grids for 30 s in ten μL of 2% uranyl acetate neutralized with NaOH. The grids were then blotted on filter paper to remove the excess solution and allowed to air dry. Phages were examined by transmission electron microscopy at Cambridge Advanced Imaging Centre (Department of Physiology, Development and Neuroscience, University of Cambridge) using an FEI Tecnai G2 transmission electron microscope (FEI, OR, USA). The accelerating voltage was 120.0 kV, and images were captured with an AMT XR60B digital camera running Deben software.\\\\n4.5. Genome Sequencing and Analysis\\\\nPhage genomes were sequenced on the Illumina MiSeq Sequencer at MicrobesNG (Birmingham, UK). The reads were trimmed using Trimmomatic (Bolger et al., 2014), assessed for quality using BWA-MEM (Li, 2013) and assembled using SPAdes 3.7.1 (Bankevich et al., 2012) with standard settings. Except for JA10 and AD2, all generated over 140,000 reads and had higher than 100x coverage of the full genome. JA10 generated 3,270 reads and had 26x coverage. AD2 generated 1,899 reads and had 4.79x coverage. All assembled into one contig except AD2. Annotation was performed using DNAMaster 5.23.2 (Lawrence, 2012). Genome maps were generated using Circos 0.69.6 (Krzywinski et al., 2009). Genomes were deposited in Genbank using BankIt (NCBI) and are available under accession numbers MH460459 (JA10), MH389777 (JA11), MH460460 (JA13), MH460461 (JA29), MH460462 (JA33), and MH460463 (AD1). Genomes were compared using NCBI Blast, MEGA 7.0.26 (Kumar et al., 2016) and the Artemis Comparison Tool 13.0.0 (Carver et al., 2005). Annotation tables are found in Tables S1–S6.\\\\n4.6. Transduction\\\\nTo test for transduction, phage lysates were generated as described above on donor bacterial strains carrying the desired plasmid or chromosomal marker. All the experiments used kanamycin as the antibiotic for selection. The chromosomal marker for the JA phages was a transposon stably inserted into a protease gene. Successful transduction was therefore confirmed by a protease-negative, kanamycin-resistant phenotype in the recipient cells. The chromosomal marker for the AD phages was a transposon stably inserted into the lacZ gene. Successful transduction was confirmed by kanamycin-resistant recipient colonies that were white on media containing X-gal. The plasmid marker was pECA1039-Km3 and successful transduction was confirmed by plasmid extraction and gel electrophoresis under standard conditions as described previously (Day et al., 2017).\\\\nTransduction was performed by mixing phage lysate with an overnight culture of the recipient cells to achieve a multiplicity of infection of 0.01, meaning that for each phage there were one hundred bacterial cells, apart from the transduction of pECA1039-Km3 into Dickeya dadantii subsp. dieffenbachiae, which required an MOI of 0.1. The mixture was left on the lab bench at room temperature for 20 min, followed by incubation on a rotary wheel at 30°C for 30 min. The infected culture was then centrifuged and the bacterial pellets washed with LB twice to eliminate any remaining non-adsorbed phage. The bacterial pellets were resuspended in 1 mL LB and 100 μL aliquots were spread onto LBA plates with selection for the chromosomal or plasmid marker. Appropriate standard controls, which were routinely negative, were used to score for any spontaneous resistance of the recipient strain. One hundred microliters of the phage lysate was also spread onto LBA plates to confirm lysate sterility.\\\\nAuthor Contributions\\\\nAD, JA, and GPCS conceived and designed the experiments, analyzed the data. AD and JA performed the experiments. AD wrote the paper. AD and GPCS edited the paper.\\\\nFunding\\\\nThis work was supported by the BBSRC, UK. AD was supported by a Cambridge Doctoral Training Partnership Award from the BBSRC, UK.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nSequencing was conducted by MicrobesNG (http://www.microbesng.uk), which is supported by the BBSRC (grant number BB/L024209/1). Microscopy was performed at Cambridge Advanced Imaging Centre with the help of Karin Müller and Lyn Carter. We are grateful to Sonia Humphris and Ian Toth (James Hutton Institute, Scotland) for generous provision of Dickeya strains. This work was done under DEFRA license: 50864/197900/3. We thank Alison Rawlinson, Sarah Barker, and Diana Breitmaier for technical support and Rita Monson for helpful advice.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2018.02169/full#supplementary-material\\\\nReferences\\\\nAckermann, H.-W. (2012). “Bacteriophage electron microscopy,” in Advances in Virus Research, eds M. Lobocka and W. T. Szybalski (Elsevier), 1–32.\\\\nGoogle Scholar\\\\nAckermann, H. W., and Nguyen, T. M. (1983). Sewage coliphages studied by electron microscopy. Appl. Environ. Microbiol. 45, 1049–59.\\\\nPubMed Abstract | Google Scholar\\\\nAdeolu, M., Alnajar, S., Naushad, S., and Gupta, R. S. (2016). Genome-based phylogeny and taxonomy of the “Enterobacteriales:” proposal for Enterobacterales ord. nov. divided into the families Enterobacteriaceae, Erwiniaceae fam. nov., Pectobacteriaceae fam. nov., Yersiniaceae fam. nov., Hafniaceae fam. nov., Morgane. Int. J. Syst. Evol. Microbiol. 66, 5575–5599. doi: 10.1099/ijsem.0.001485\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAdriaenssens, E. M., Ackermann, H. W., Anany, H., Blasdel, B., Connerton, I. F., Goulding, D., et al. (2012a). A suggested new bacteriophage genus: “Viunalikevirus.” Arch. Virol. 157, 2035–2046. doi: 10.1007/s00705-012-1360-5\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAdriaenssens, E. M., Mattheus, W., Cornelissen, A., Shaburova, O., Krylov, V. N., Kropinski, A. M., et al. (2012b). Complete genome sequence of the giant Pseudomonas phage Lu11. J. Virol. 86, 6369–6370. doi: 10.1128/JVI.00641-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAdriaenssens, E. M., Van Vaerenbergh, J., Vandenheuvel, D., Dunon, V., Ceyssens, P. J., De Proft, M., et al. (2012c). T4-related bacteriophage LIMEstone isolates for the control of soft rot on potato caused by “Dickeya solani.” PLoS ONE 7:et002. doi: 10.1371/journal.pone.0033227.t002\\\\nCrossRef Full Text | Google Scholar\\\\nAdriaenssens, E. M., Wittmann, J., Kuhn, J. H., Turner, D., Sullivan, M. B., Dutilh, B. E., et al. (2018). Taxonomy of prokaryotic viruses: 2017 update from the ICTV Bacterial and Archaeal Viruses Subcommittee. Arch. Virol. 163, 1125–1129. doi: 10.1007/s00705-018-3723-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAlic, Š., Naglič, T., Tušek-Žnidarič, M., Peterka, M., Ravnikar, M., and Dreo, T. (2017a). Putative new species of the genus Dickeya as major soft rot pathogens in Phalaenopsis orchid production. Plant Pathol. 66, 1357–1368. doi: 10.1111/ppa.12677\\\\nCrossRef Full Text | Google Scholar\\\\nAlič, Š., Naglič, T., Tušek-Žnidarič, M., Ravnikar, M., Rački, N., Peterka, M., etal. (2017b). Newly isolated bacteriophages from the Podoviridae, Siphoviridae, and Myoviridae families have variable effects on putative novel Dickeya spp. Front. Microbiol. 8:1870. doi: 10.3389/fmicb.2017.01870\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455–477. doi: 10.1089/cmb.2012.0021\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nButtimer, C., Hendrix, H., Oliveira, H., Casey, A., Neve, H., McAuliffe, O., et al. (2017). Things are getting hairy: enterobacteria bacteriophage vB_PcaM_CBB. Front. Microbiol. 8:44. doi: 10.3389/fmicb.2017.00044\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nButtimer, C., Born, Y., Lucid, A., Loessner, M. J., Fieseler, L., and Coffey, A. (in press). Erwinia amylovora phage vB_EamM_Y3 represents another lineage of hairy Myoviridae. Res. Microbiol. doi: 10.1016/j.resmic.2018.04.006\\\\nPubMed Abstract | CrossRef Full Text\\\\nCahill, G., Fraser, K., Kowalewska, M. J., Kenyon, D. M., and Saddler, G. S. (2010). “Recent findings from the Dickeya survey and monitoring programme,” in Proceedings Crop Protection in Northern Britain (Edinburgh), 171–176.\\\\nGoogle Scholar\\\\nCarver, T. J., Rutherford, K. M., Berriman, M., Rajandream, M.-A., Barrell, B. G., and Parkhill, J. (2005). ACT: the artemis comparison tool. Bioinformatics 21, 3422–3423. doi: 10.1093/bioinformatics/bti553\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChibani-Chennoufi, S., Bruttin, A., Dillmann, M.-L., and Brussow, H. (2004). Phage-host interaction: an ecological perspective. J. Bacteriol. 186, 3677–3686. doi: 10.1128/JB.186.12.3677-3686.2004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCianfanelli, F. R., Alcoforado Diniz, J., Guo, M., De Cesare, V., Trost, M., and Coulthurst, S. J. (2016). VgrG and PAAR proteins define distinct versions of a functional type VI secretion system. PLOS Pathogens 12:e1005735. doi: 10.1371/journal.ppat.1005735\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Ozymko, Z., de Jager, V., Siwinska, J., Smolarska, A., Ossowicki, A., et al. (2015a). Genomic, proteomic and morphological characterization of two novel broad host lytic bacteriophages ΦPD10.3 and ΦPD23.1 infecting pectinolytic Pectobacterium spp. and Dickeya spp. PloS ONE 10:e0119812. doi: 10.1371/journal.pone.0119812\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Ozymko, Z., and Lojkowska, E. (2014a). Isolation and characterization of novel soilborne lytic bacteriophages infecting Dickeya spp. biovar 3 (“D. solani”). Plant Pathol. 63, 758–772. doi: 10.1111/ppa.12157\\\\nCrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Ozymko, Z., Siwinska, J., Ossowicki, A., de Jager, V., Narajczyk, M., et al. (2015b). The complete genome, structural proteome, comparative genomics and phylogenetic analysis of a broad host lytic bacteriophage phiD3 infecting pectinolytic Dickeya spp. Stand. Genomic Sci. 10:68. doi: 10.1186/s40793-015-0068-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Ozymko, Z., Zwirowski, S., and Lojkowska, E. (2014b). Complete genome sequence of a broad-host-range lytic Dickeya spp. bacteriophage ΦD5. Arch. Virol. 159, 3153–3155. doi: 10.1007/s00705-014-2170-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Smolarska, A., and Ozymko, Z. (2017). The viability of lytic bacteriophage ΦD5 in potato-associated environments and its effect on Dickeya solani in potato (Solanum tuberosum L.) plants. PLoS ONE 12:e0183200. doi: 10.1371/journal.pone.0183200\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDay, A., Ahn, J., Fang, X., and Salmond, G. P. C. (2017). Environmental bacteriophages of the emerging enterobacterial Phytopathogen, Dickeya solani, show genomic conservation and capacity for horizontal gene transfer between their bacterial hosts. Front. Microbiol. 8:1654. doi: 10.3389/fmicb.2017.01654\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nEsplin, I. N. D., Berg, J. A., Sharma, R., Allen, R. C., Arens, D. K., Ashcroft, C. R., et al. (2017). Genome sequences of 19 novel Erwinia amylovora bacteriophages. Genome Announc. 5:e00931–17. doi: 10.1128/genomeA.00931-17\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nIriarte, F. B., Balogh, B., Momol, M. T., Smith, L. M., Wilson, M., and Jones, J. B. (2007). Factors affecting survival of bacteriophage on tomato leaf surfaces. Appl. Environ. Microbiol. 73, 1704–1711. doi: 10.1128/AEM.02118-06\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKabanova, A., Shneider, M., Bugaeva, E., Ha, V. T. N., Miroshnikov, K., Korzhenkov, A., et al. (2018). Genomic characteristics of vB_PpaP_PP74, a T7-like Autographivirinae bacteriophage infecting a potato pathogen of the newly proposed species Pectobacterium parmentieri. Arch. Virol. 163, 1691–1694. doi: 10.1007/s00705-018-3766-1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKhan Mirzaei, M., and Nilsson, A. S. (2015). Isolation of phages for phage therapy: a comparison of spot tests and efficiency of plating analyses for determination of host range and efficacy. PLoS ONE 10:e0118557. doi: 10.1371/journal.pone.0118557\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKrzywinski, M., Schein, J., Birol, I., Connors, J., Gascoyne, R., Horsman, D., et al. (2009). Circos: an information aesthetic for comparative genomics. Genome Res. 19, 1639–1645. doi: 10.1101/gr.092759.109\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKumar, S., Stecher, G., and Tamura, K. (2016). MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets. Mol. Biol. Evol. 33:msw054. doi: 10.1093/molbev/msw054\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLaurila, J., Ahola, V., Lehtinen, A., Joutsjoki, T., Hannukkala, A., Rahkonen, A., et al. (2008). Characterization of Dickeya strains isolated from potato and river water samples in Finland. Eur. J. Plant Pathol. 122, 213–225. doi: 10.1007/s10658-008-9274-5\\\\nCrossRef Full Text | Google Scholar\\\\nLi, H. (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint arXiv.\\\\nGoogle Scholar\\\\nMansfield, J., Genin, S., Magori, S., Citovsky, V., Sriariyanum, M., Ronald, P., et al. (2012). Top 10 plant pathogenic bacteria in molecular plant pathology. Mol. Plant Pathol. 13, 614–29. doi: 10.1111/j.1364-3703.2012.00804.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMatilla, M. A., Fang, X., and Salmond, G. P. C. (2014). Viunalikeviruses are environmentally common agents of horizontal gene transfer in pathogens and biocontrol bacteria. ISME J. 8, 2143–2147. doi: 10.1038/ismej.2014.150\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMatilla, M. A., and Salmond, G. P. C. (2014). Bacteriophage phiMAM1, a viunalikevirus, is a broad-host-range, high-efficiency generalized transducer that infects environmental and clinical isolates of the enterobacterial genera Serratia and Kluyvera. Appl. Environ. Microbiol. 80, 6446–6457. doi: 10.1128/AEM.01546-14\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nParkinson, N., DeVos, P., Pirhonen, M., and Elphinstone, J. (2014). Dickeya aquatica sp. nov., isolated from waterways. Int. J. Syst. Evol. Microbiol. 64(Pt 7), 2264–2266. doi: 10.1099/ijs.0.058693-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nParkinson, N., Stead, D., Bew, J., Heeney, J., Tsror, L., and Elphinstone, J. (2009). Dickeya species relatedness and clade structure determined by comparison of recA sequences. Int. J. Syst. Evol. Microbiol. 59, 2388–2393. doi: 10.1099/ijs.0.009258-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPelfrene, E., Willebrand, E., Cavaleiro Sanches, A., Sebris, Z., and Cavaleri, M. (2016). Bacteriophage therapy: a regulatory perspective. J. Antimicrob. Chemother. 71, 2071–2074. doi: 10.1093/jac/dkw083\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPetty, N. K., Foulds, I. J., Pradel, E., Ewbank, J. J., and Salmond, G. P. C. (2006). A generalized transducing phage (phiIF3) for the genomically sequenced Serratia marcescens strain Db11: a tool for functional genomics of an opportunistic human pathogen. Microbiology 152, 1701–1708. doi: 10.1099/mic.0.28712-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPfreundt, U., Spungin, D., Hou, S., Voß, B., Berman-Frank, I., and Hess, W. R. (2017). Genome of a giant bacteriophage from a decaying Trichodesmium bloom. Mar. Genomics 33, 21–25. doi: 10.1016/j.margen.2017.02.001\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPritchard, L., Humphris, S., Baeyen, S., Maes, M., Van Vaerenbergh, J., Elphinstone, J., et al. (2013a). Draft genome sequences of four Dickeya dianthicola and four Dickeya solani strains. Genome Announc. 1:e00087–12. doi: 10.1128/genomeA.00087-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPritchard, L., Humphris, S., Saddler, G. S., Elphinstone, J. G., Pirhonen, M., and Toth, I. K. (2013b). Draft genome sequences of 17 isolates of the plant pathogenic bacterium Dickeya. Genome Announc. 1:e00978–13. doi: 10.1128/genomeA.00978-13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSławiak, M., van Beckhoven, J. R. C. M., Speksnijder, A. G. C. L., Czajkowski, R., Grabe, G., and van der Wolf, J. M. (2009). Biochemical and genetical analysis reveal a new clade of biovar 3 Dickeya spp. strains isolated from potato in Europe. Eur. J. Plant Pathol. 125, 245–261. doi: 10.1007/s10658-009-9479-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSykilinda, N. N., Bondar, A. A., Gorshkova, A. S., Kurochkina, L. P., Kulikov, E. E., Shneider, M. M., et al. (2014). Complete genome sequence of the novel giant Pseudomonas phage PaBG. Genome Announc. 2:e00929–13. doi: 10.1128/genomeA.00929-13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nToth, I. K., Cahill, G., Elphinstone, J. G., Humphris, S., and Wale, S. J. (2016). “An update on the Potato Council/Scottish government-funded blackleg project - Year 2,” in Proceedings Crop Protection in Northern Britain 2016 (Edinburgh), 203–204.\\\\nGoogle Scholar\\\\nToth, I. K., van der Wolf, J. M., Saddler, G., Lojkowska, E., Hélias, V., Pirhonen, M., et al. (2011). Dickeya species: an emerging problem for potato production in Europe. Plant Pathol. 60, 385–399. doi: 10.1111/j.1365-3059.2011.02427.x\\\\nCrossRef Full Text | Google Scholar\\\\nTsror, L., Erlich, O., Lebiush, S., Hazanovsky, M., Zig, U., Slawiak, M., et al. (2009). Assessment of recent outbreaks of Dickeya sp. (syn. Erwinia chrysanthemi) slow wilt in potato crops in Israel. Eur. J. Plant Pathol. 123, 311–320. doi: 10.1007/s10658-008-9368-0\\\\nCrossRef Full Text | Google Scholar\\\\nvan der Wolf, J. M., Nijhuis, E. H., Kowalewska, M. J., Saddler, G. S., Parkinson, N., Elphinstone, J. G., et al. (2014). Dickeya solani sp. nov., a pectinolytic plant-pathogenic bacterium isolated from potato (Solanum tuberosum). Int. J. Syst. Evol. Microbiol. 64(Pt 3), 768–774. doi: 10.1099/ijs.0.052944-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYamada, T., Satoh, S., Ishikawa, H., Fujiwara, A., Kawasaki, T., Fujie, M., et al. (2010). A jumbo phage infecting the phytopathogen Ralstonia solanacearum defines a new lineage of the Myoviridae family. Virology 398, 135–147. doi: 10.1016/j.virol.2009.11.043\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01654\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01654/full\",\\n    \"loadedTime\": \"2024-01-23T08:50:28.875Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01654\",\\n    \"title\": \"Frontiers | Environmental Bacteriophages of the Emerging Enterobacterial Phytopathogen, Dickeya solani, Show Genomic Conservation and Capacity for Horizontal Gene Transfer between Their Bacterial Hosts\",\\n    \"description\": \"Dickeya solani is an economically important phytopathogen widespread in mainland Europe that can reduce potato crop yields by 25%. There are no effective environmentally-acceptable chemical systems available for diseases caused by Dickeya. Bacteriophages have been suggested for use in biocontrol of this pathogen in the field, and limited field trials have been conducted. To date only a small number of bacteriophages capable of infecting Dickeya solani have been isolated and characterised, and so there is a need to expand the repertoire of phages that may have potential utility in phage therapy strategies. Here we describe 67 bacteriophages from environmental sources, the majority of which are members of the viral family Myoviridae. Full genomic sequencing of two isolates revealed a high degree of DNA identity with Dickeya solani bacteriophages isolated in Europe in the past five years, suggesting a wide ecological distribution of this phage family. Transduction experiments showed that the majority of the new environmental bacteriophages are capable of facilitating efficient horizontal gene transfer. The possible risk of unintentional transfer of virulence or antibiotic resistance genes between hosts susceptible to transducing phages cautions against their environmental use for biocontrol, until specific phages are fully tested for transduction capabilities.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"1. Introduction\\\\nThe enterobacterial genus, Dickeya, currently consists of six phytopathogenic species that can cause severe disease in economically important crops, including tomato, chicory, and potato (Reverchon and Nasser, 2013). The first report of Dickeya (previously known as Erwinia chrysanthemi) infecting European potatoes came from the Netherlands in the 1970s (Maas Geesteranus HP, 1972). Until 2004, almost all European potato isolates of Dickeya were assigned as Dickeya dianthicola, which has a broad host range across both nutritional and ornamental species (Toth et al., 2011). In the past 10 years however, three groups have independently identified a new clade of Dickeya in European potato isolates (Laurila et al., 2008; Parkinson et al., 2009; Sławiak et al., 2009). In 2014 this led to the classification of a new species; Dickeya solani (van der Wolf et al., 2014).\\\\nDickeya solani is more aggressive than other Dickeya species, able to spread more easily through the plant vascular system and survive at higher temperatures than D. dianthicola (Toth et al., 2011). It is currently the predominant potato pathogen in Europe and in 2010 Scotland became the first country to introduce specific legislation aimed at preventing the establishment of D. solani in its seed industry (Mansfield et al., 2012).\\\\nIn Israel a reduction in yield of up to 25% was observed in potatoes exposed to Dickeya species (Tsror et al., 2009). This imposes a significant economic cost, and consequently has led to research into methods for control of these virulent phytopathogens. In the absence of any effective chemical control systems, bacterial viruses (bacteriophages; phages) have been suggested as potential biocontrol tools and several studies have isolated phages capable of infecting Dickeya species (Adriaenssens et al., 2012b; Czajkowski et al., 2014, 2015; Matilla et al., 2014). Their potential use as biocontrol agents has been trialed both in the lab and in the field (Adriaenssens et al., 2012b) and these studies showed a “therapeutic” outcome with an increase in yield of the potato crop. Because of the potential utility of specific phages as therapeutic agents in potato soft rot control experiments, there is value in investigating a wider range of Dickeya phages. However, prior work has shown that a previously isolated D. solani phage is capable of generalized transduction of both chromosomal and plasmid markers (Matilla et al., 2014). The European Medicines Agency, among others, has stated that it is “important to ensure that therapeutic phages do not carry out generalized transduction” (Pelfrene et al., 2016), and therefore this is an important consideration as some Dickeya phages may not have been fully tested for generalized transduction capacity before field trials. This study therefore aimed to isolate and characterize a larger repertoire of new environmental phages against D. solani and investigate their potential for generalized transduction.\\\\n2. Results\\\\n2.1. Isolation and Classification\\\\nSixty-seven phages were isolated using standard enrichment techniques from both treated sewage effluent and river water between 2013 and 2015 using D. solani MK10 as the host organism. Transmission electron microscopy (TEM) showed two different morphological groups, a selection of which are shown in Figure 1 alongside the previously characterized phage LIMEstone1 (Adriaenssens et al., 2012b).\\\\nFigure 1. Transmission electron micrographs of four Dickeya phages. LIMEstone1 is a previously characterized Dickeya phage (Adriaenssens et al., 2012b). ϕJA15 and ϕXF4 are phages of the Myoviridae family exhibiting tail spikes as opposed to tail fibers. ϕXF28 is a phage of the Podoviridae family. Scale bars represent 100 nm.\\\\nOf 24 phages imaged, all possessed an icosahedral head and a tail, placing them in the order Caudovirales. Three possessed short tails, characteristic of the family Podoviridae (such as ϕXF28 in the last panel of Figure 1) whilst the rest possessed longer contractile tails and belong to the family Myoviridae. The 21 Myoviridae members did not appear to possess the tail fibers characteristic of the family. Instead, short tail spikes were observed, (first three panels of Figure 1), and these are generally associated with the family Podoviridae.\\\\n2.2. Transduction\\\\nOther Dickeya phages with similar morphology have been described and were shown to be efficient generalized transducing phages (Matilla et al., 2014). Due to the transduction capability of certain phages shown by Matilla et al., the 67 newly-isolated phages were also tested for ability to affect horizontal gene transfer. Of these, 51 (including the 21 phages with the non-classical morphology) proved capable of transducing chromosomal markers. Twelve of the isolates, all of which had the non-classical morphology, were also tested for generalized transduction, and proved capable of transferring plasmids between Dickeya species. The results of transduction of the plasmid pBR322 by three of these phages are shown in Figure 2.\\\\nFigure 2. pBR322 was first used to transform Dickeya solani MK10 before the creation of lysates of ϕJA13, ϕJA29, or ϕJA37 on the recombinant host. These lysates were then used to infect Dickeya solani, D. dieffenbachiae, or D. zeae as appropriate and transductants selected on LB agar containing ampicillin. The plasmids from these transductants were extracted and analyzed by gel electrophoresis. Co-migration of the plasmid DNA samples and absence of the plasmid from the wild type (WT) controls confirmed successful plasmid transduction.\\\\n2.3. Host Range\\\\nBased on the bacterial strains tested, LIMEstone1 is capable of forming plaques on strains of D. solani only (Adriaenssens et al., 2012b). The phages isolated during this study were tested against a variety of Dickeya strains, listed in Table 1, to determine their host range. The majority of the phages presented here exhibited the same host range as LIMEstone1 and were only capable of forming plaques on D. solani strains but not isolated representatives of other Dickeya species used in this study. However, eight of the phage isolates had a wider host range extending to species such as Dickeya dieffenbachiae, Dickeya paradisiaca, and Dickeya zeae (Table 2).\\\\nTable 1. Bacterial strains, bacteriophages, and primers used in this study.\\\\nTable 2. Extended host range of three groups of the isolated phages capable of infecting other species of Dickeya.\\\\n2.4. Genetic Comparisons\\\\nTwo key genes known to be conserved between these phages, those for DNA polymerase (DNAP) and tail spike protein 1 (TSP1), were sequenced for several of the newly-isolated phages. These nucleotide sequences were then compared to those of LIMEstone1 as shown in Table 3. All of the phages in Table 3 were able to form plaques on D. solani. The corresponding amino acid sequences were compared between these phages and phylogenetic trees were created as shown in Figure 3 (DNAP) and Figure 4 (TSP1). These show that, in agreement with Table 3, the DNAP genes of ϕXF4 and ϕXF11 grouped together away from the other phages with a branch length of 0.053. The other phages formed two clusters that differed in a single amino acid. The TSP1 genes formed two distinct clusters, with the genes from ϕXF16 and ϕJA1 forming their own cluster with a branch length of 0.092, whereas the other phages all had identical TSP1 genes. The final 20 amino acids were trimmed from the TSP1 genes as the sequencing data for some of the phages was insufficient for tree construction.\\\\nTable 3. Nucleotide comparison of two conserved genes between the previously characterized LIMEstone1 and a selection of isolated phages.\\\\nFigure 3. Phylogenetic tree of the DNA Polymerase (DNAP) gene from 10 phages of Dickeya solani. Tree was constructed using the Maximum Likelihood method with 1,134 positions in the final dataset, and the tree shown has the highest log likelihood (−2033.82).\\\\nFigure 4. Phylogenetic tree of the Tail Spike Protein 1 (TSP1) gene from 10 phages of Dickeya solani. Tree was constructed using the Maximum Likelihood method with 1,593 positions in the final dataset, and the tree shown has the highest log likelihood (−3105.91). The final 20 amino acids of the TSP1 gene were trimmed from the alignment as the sequencing data for some of the phages was insufficient.\\\\n2.5. Genomic Sequencing of Two New Dickeya solani Phages\\\\nϕXF4 and ϕJA15 were isolated over a year apart yet they shared the same host range and PCR amplification and preliminary sequence analysis showed 100% DNA identity in the TSP1 genes, although they differ in their DNAP genes. The full genomes of both phages were then sequenced. Both consist of circular double-stranded DNA of 151,519 and 153,757 bp, respectively. The genome of ϕXF4 has a G+C content of 49.4% and contained 185 predicted genes with lengths ranging between 116 and 4,838 nucleotides, as shown in Figure 5. ϕJA15 has a G+C content of 49.2% and contained 188 predicted genes of lengths between 122 and 4,838 nucleotides, as shown in Figure 6. Full annotation tables for the two genomes can be found in Tables S1, S2, respectively.\\\\nFigure 5. Map of the genome of ϕXF4. The outer gray ring marks open reading frames whilst the middle ring categorizes the proposed functions of these genes. The inner ring highlights the areas of the ϕXF4 genome that differ from the genome of the LIMEstone1 phage. The genome map was generated using Circos.\\\\nFigure 6. Map of the genome of ϕJA15. The outer gray ring marks open reading frames whilst the middle ring categorizes the proposed functions of these genes. The inner ring highlights the areas of the ϕJA15 genome that differ from the genome of the LIMEstone1 phage. The genome map was generated using Circos.\\\\n2.6. Genomic Comparison\\\\nThe genomes of the two new phages shared 97% DNA identity, with the main areas of difference being regions encoding endonucleases and hypothetical proteins. A comparison of both of these phages with the previously published phage LIMEstone1, showed over 95% DNA identity, with the major areas of difference consisting of genes thought to be involved in DNA replication (such as homing endonucleases and polymerases) as well as introns located throughout all three genomes. These areas of difference are highlighted in Figures 5, 6.\\\\n3. Discussion\\\\nThe Scottish government tests all seed crops imported from outside Scotland plus 10% of Scottish-origin crops for D. solani and did not find any positive samples in 2016 (Scottish Government, 2016). These data are consistent with a view that D. solani is not yet environmentally widespread within the UK, although there have been isolated cases of D. solani reported in England and Wales since 2007 (Cahill et al., 2010) in crops originating from outside of the UK (Toth et al., 2016). The relative ease with which we have isolated environmental phages that infect D. solani therefore seems counter-intuitive given the reported paucity of the pathogen in the environment. In a restricted host range screen these phages did not form plaques on eight isolates of other Gram-negative laboratory strains such as Pectobacterium carotovorum, Pectobacterium atrosepticum, Serratia plymuthica, Serratia marcescens, Escherichia coli, and Pantoea agglomerans (data not shown). The apparently contradictory observations from phage isolations and host distribution beg the ecological question as to why D. solani phages are easy to find by simple enrichments. We would suggest that either D. solani is present in the environment around Cambridge and is not being detected, or that there is an, as yet unknown, alternative host(s) for these phages present in the environment.\\\\nAll but three of the phages imaged by TEM were morphologically characterized as Myoviridae due to the presence of an icosahedral head and a contractile tail. Classical Myoviridae members, such as the coliphage T4 possess long slender tail fibers attached to the baseplate that participate in adsorption of the phage to the bacterial host. The imaged phages do not appear to have tail fibers, but instead possess shorter, clustered structures more akin to the tail spikes present in members of the Podoviridae. Genome analysis of the phages ϕXF4 and ϕJA15 showed genes encoding potential tail spike proteins, which show 100% sequence identity to putative tail spike protein genes in LIMEstone1. This combination of a Myoviridae-like morphology with tail spikes has been reported previously as a feature of the novel viral genus termed viunalikevirus (Adriaenssens et al., 2012a), named after the ViI Salmonella typing phage, and includes virulent phages capable of infecting a wide range of Gram-negative hosts. Members of the genus share a high degree of genome order and identity, with the major region of divergence being the genes encoding the tail spike proteins. Although, we cannot state that all phages that exhibit this morphology are definitively members of the viunalikevirus genus, we conclude it is likely. This does pose the question of whether there is some particular connection between D. solani and viunalikeviruses, or whether the environment around Cambridge is a particularly abundant source of this genus of phages.\\\\nA comparison of two genes from a subset of the phages isolated here, along with LIMEstone1, showed that, whilst there were some differences in DNA polymerase and TSP1 genes, these did not translate into a difference in host range and that in general these phages are highly similar. Several D. solani phages have now been isolated from environmental sources, including the LIMEstone phages (Adriaenssens et al., 2012b) as well as ϕD3 (Czajkowski et al., 2015) and ϕD5 (Czajkowski et al., 2014). They are all remarkably similar upon comparison of their genomes, and the two phages discussed in this study show 95% DNA identity with LIMEstone1. Much of the variation comes within the many introns and homing endonucleases found throughout the genomes of the phages, whereas structural elements are largely conserved. These introns and homing endonucleases vary in their sequence, but their position within the genomes of the phages, and thus the gene order, remains the same. It is interesting that these phages have been isolated independently in countries across Europe, and even from different environments; soil (LIMEstone1 and 2, ϕD3, and ϕD5) and water (ϕXF4 and ϕJA15), and yet they share such conservation despite their wide geographical separation.\\\\nThe lytic activity of these phages, coupled with the high economic burden of D. solani crop phytopathogenesis, have highlighted the D. solani phages as potential biocontrol agents, and limited field trials have been performed (Adriaenssens et al., 2012b). Nevertheless, although these phages may have potential phage therapy features, we have shown that, 51 were capable of effecting horizontal gene transfer. We showed previously that eight candidate viunalikeviruses (including the LIMEstone phages and ϕXF4) were efficient generalized transducers of plasmid markers (Matilla et al., 2014). Consequently, we proposed that generalized transduction capacity is a characteristic trait of the viunalikevirus genus. This feature is important if these virulent phages are to be used therapeutically, as there could be potential (albeit low) for collateral transfer of bacterial virulence genes or drug resistance plasmids into unintended hosts, with unknown consequences—depending on ecological selection pressures. We therefore consider it prudent to caution against further field trials until the specific phage(s) involved are fully tested for generalized transduction capabilities.\\\\n4. Materials and Methods\\\\n4.1. Bacterial Strains, Phages, Culture Media, and Growth Conditions\\\\nAll bacterial strains used in this study are listed in Table 1. Dickeya species were routinely grown at 30°C in Luria broth (LB) or on LB agar plates (1.5%, wt/vol, agar). Phages were stored at 4°C in phage buffer (10 mM Tris-HCl, pH 7.4, 10 mM MgSO4, 0.01%, wt/vol, gelatin) over a few drops of NaHCO3− saturated chloroform.\\\\n4.2. Isolation of Phages\\\\nTreated sewage effluent was collected from a sewage treatment plant in Cambridge, United Kingdom (Matilla and Salmond, 2014). River water was collected from multiple sites along the River Cam. Samples were filter sterilized before 5 mL of the sample was added to 2x LB along with 500 μL of an overnight culture of D. solani MK10. This mixture was incubated overnight in a 250 mL flask at 30°C with shaking at 250 rpm. One milliliter of the enriched sample was mixed with 100 μL of chloroform (saturated with sodium hydrogen carbonate) and vortexed to lyse bacterial cells. The sample was centrifuged at 16,000 x g for 4 min and 10 μL of a serial dilution series of the supernatant was mixed with 200 μL of an overnight bacterial culture and 4 ml of LB top agar. This mixture was poured as an overlay on an LBA plate and incubated overnight at 30°C. Single phage plaques were picked with a sterile toothpick, placed into 100 μL phage buffer, and shaken with 40 μL of chloroform to kill any bacteria. The phages obtained were plaque purified three times. High-titre phage lysates were then obtained as described previously (Petty et al., 2006). Briefly, 10-fold serial dilutions of the phage were incubated overnight in an agar overlay as already described. Those plates exhibiting confluent lysis (seen as a mosaic-like effect in which the plaques are close to merging) were used for lysate preparation. The top agar was removed from the plate, vortexed with chloroform before sedimentation at 2,200 x g for 20 min at 4°C. The supernatant was removed and vortexed with a few drops of chloroform to produce the final lysate.\\\\n4.3. Transmission Electron Microscopy\\\\nHigh-titre lysates for transmission electron microscopy were obtained as described previously (Petty et al., 2006) using 0.35% (w/v) LB agarose instead of 0.35% (w/v) LB agar overlays. Ten microliters of high-titer phage lysates were adsorbed onto 400-mesh copper grids with holey carbon support films (Agar Scientific, Stansted, United Kingdom) for 30 min. The copper grids were discharged in a Quorum/Emitech K100X system (Quorum, Ringmer, United Kingdom) prior to use. After 1 min, excess phage suspension was removed with filter paper and phage samples were negatively stained by placing the grids for 5 min in 10 μL of 2% phosphotungstic acid (PTA) neutralized with sodium hydroxide, or with 10 μL of 2% uranyl acetate for 2 min. The grids were then blotted on filter paper to remove the excess solution and allowed to air dry for 10 min. Phages were examined by transmission electron microscopy in the Multi-Imaging Centre (Department of Physiology, Development and Neuroscience, University of Cambridge) using an FEI Tecnai G2 transmission electron microscope (FEI, OR, USA). The accelerating voltage was 120.0 kV, and images were captured with anAMT XR60B digital camera running Deben software.\\\\n4.4. Determination of Host Range\\\\nThe host range of isolated phages was determined by plating out 10-fold serial dilutions of the phage lysates, onto agar overlays containing the six species of Dickeya listed in Table 1. To avoid potential confusion with “lysis from without,” only phages that produced lysis at low dilution and individual plaques were considered as being able to infect the host.\\\\n4.5. Transduction\\\\nTo test for transduction, phage lysates were generated on donor bacterial strains carrying the desired plasmid or chromosomal marker as already described. Transduction was performed by mixing phage lysate with an overnight culture of the recipient host to achieve a multiplicity of infection of 0.1, meaning that for each phage there were 10 bacterial cells. The mixture was left on the lab bench at room temperature for 20 min, followed by incubation on a rotary wheel at 30°C for 30 min. The infected culture was then centrifuged and the bacterial pellets washed with LB twice to eliminate any remaining non-adsorbed phage. The bacterial pellets were resuspended in 1 mL LB and 100 μL aliquots were spread onto LBA plates with drug selection for the chromosomal or plasmid marker. Appropriate standard controls, which were routinely negative, were used to score for any spontaneous resistance of the recipient strain. One hundred microliters of the phage lysate was also spread onto LBA plates to confirm lysate sterility.\\\\n4.6. Gene Amplification and Analysis\\\\nGenomic DNA was extracted using Phase Lock Gel tubes (5 Prime, Hamburg, Germany) following manufacturer\\'s instructions for isolation of Lambda DNA. DNA Polymerase (DNAP) and Tail Spike Protein 1 (TSP1) genes were amplified using Phusion polymerase (ThermoScientific, MA, USA) following standard protocols. TSP1 was amplified using the primers oJA1 and oJA2, and DNAP by the primers oJA14 and oJA15, listed in Table 1. PCR products were sequenced by GATC Biotech AG (Konstanz, Germany). Sequences were compared using NCBI Blast and the Artemis Comparison Tool (Carver et al., 2005). Phylogenetic trees were constructed using MEGA 7.0.26 (Kumar et al., 2016).\\\\n4.7. Genome Sequencing and Analysis\\\\nϕXF4 was sequenced on the Illumina Bench Top MiSeq Sequencer (Illumina, CA, USA) at the DNA Sequencing Facility, Department of Biochemistry, University of Cambridge, UK. The resulting 138,803 reads were trimmed, quality assessed and assembled using Geneious 7.1.5 (Biomatters Ltd.), leading to higher than 100x coverage of the full genome. Gaps or single nucleotide polymorphisms were further filled or verified by Sanger sequencing to produce one contig. ϕJA15 was sequenced on the Illumina MiSeq Sequencer at MicrobesNG (Birmingham, UK). The 454,086 reads were trimmed using Trimmomatic (Bolger et al., 2014), assessed for quality using BWA-MEM (Li, 2013) and assembled using SPAdes 3.7.1 (Bankevich et al., 2012) with standard settings, leading to higher than 100x coverage of the full genome and producing one contig. Annotation of both genomes was performed using Prokka 1.11 (Seemann, 2014) using standard settings and LIMEstone1 (NC_019925.1) as a scaffold. Genome maps were generated using Circos (Krzywinski et al., 2009). Genomes were deposited in Genbank using Sequin (NCBI) and are available under accession numbers KY942057 (XF4) and KY942056 (JA15). Genomes were compared using NCBI Blast and the Artemis Comparison Tool (Carver et al., 2005).\\\\nAuthor Contributions\\\\nAnalyzed the data, conceived and designed the experiments: AD, JA, XF, and GS. Performed the experiments: AD, JA, and XF. Wrote the paper: AD and GS.\\\\nFunding\\\\nThis work was supported by the BBSRC, UK. AD was supported by a Cambridge Doctoral Training Partnership Award from the BBSRC, UK.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nAcknowledgments\\\\nSequencing of ϕJA15 was conducted by MicrobesNG (http://www.microbesng.uk), which is supported by the BBSRC (grant number BB/L024209/1). We are grateful to Ian Toth (James Hutton Institute, Scotland) for generous provision of Dickeya strains. This work was done under DEFRA license: 50864/197900/3. We thank Alison Rawlinson for technical support and Rita Monson and Miguel Matilla for helpful advice.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/article/10.3389/fmicb.2017.01654/full#supplementary-material\\\\nReferences\\\\nAdriaenssens, E. M., Ackermann, H. W., Anany, H., Blasdel, B., Connerton, I. F., Goulding, D., et al. (2012a). A suggested new bacteriophage genus: “Viunalikevirus”. Arch. Virol. 157, 2035–2046. doi: 10.1007/s00705-012-1360-5\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAdriaenssens, E. M., van Vaerenbergh, J., Vandenheuvel, D., Dunon, V., Ceyssens, P. J., de Proft, M., et al. (2012b). T4-related bacteriophage LIMEstone isolates for the control of soft rot on potato caused by ‘Dickeya solani.’ PLoS ONE 7:e33227. doi: 10.1371/journal.pone.0033227\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455–477. doi: 10.1089/cmb.2012.0021\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCahill, G., Fraser, K., Kowalewska, M. J., Kenyon, D. M., and Saddler, G. S. (2010). “Recent findings from the Dickeya survey and monitoring programme,” in Proceedings Crop Protection in Northern Britain (Dundee), 171–176.\\\\nGoogle Scholar\\\\nCarver, T. J., Rutherford, K. M., Berriman, M., Rajandream, M.-A., Barrell, B. G., and Parkhill, J. (2005). ACT: the artemis comparison tool. Bioinformatics 21, 3422–3423. doi: 10.1093/bioinformatics/bti553\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Ozymko, Z., Siwinska, J., Ossowicki, A., de Jager, V., Narajczyk, M., et al. (2015). The complete genome, structural proteome, comparative genomics and phylogenetic analysis of a broad host lytic bacteriophage phiD3 infecting pectinolytic Dickeya spp. Stand. Genomic Sci. 10:68. doi: 10.1186/s40793-015-0068-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCzajkowski, R., Ozymko, Z., Zwirowski, S., and Lojkowska, E. (2014). Complete genome sequence of a broad-host-range lytic Dickeya spp. bacteriophage phiD5. Arch. Virol. 159, 3153–3155. doi: 10.1007/s00705-014-2170-8\\\\nCrossRef Full Text | Google Scholar\\\\nKrzywinski, M., Schein, J., Birol, I., Connors, J., Gascoyne, R., Horsman, D., et al. (2009). Circos: an information aesthetic for comparative genomics. Genome Res. 19, 1639–1645. doi: 10.1101/gr.092759.109\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKumar, S., Stecher, G., and Tamura, K. (2016). MEGA7: Molecular Evolutionary Genetics Analysis version 7.0 for bigger datasets. Mol. Biol. Evol. 33:msw054. doi: 10.1093/molbev/msw054\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLaurila, J., Ahola, V., Lehtinen, A., Joutsjoki, T., Hannukkala, A., Rahkonen, A., et al. (2008). Characterization of Dickeya strains isolated from potato and river water samples in Finland. Eur. J. Plant Pathol. 122, 213–225. doi: 10.1007/s10658-008-9274-5\\\\nCrossRef Full Text | Google Scholar\\\\nLi, H. (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint, arXiv:1303.3997.\\\\nMaas Geesteranus HP (1972). Natrot en zwartbenigheid bij aardappelen. Bedrijfsontwikkeling 3, 941–945.\\\\nMansfield, J., Genin, S., Magori, S., Citovsky, V., Sriariyanum, M., Ronald, P., et al. (2012). Top 10 plant pathogenic bacteria in molecular plant pathology. Mol. Plant Pathol. 13, 614–629. doi: 10.1111/j.1364-3703.2012.00804.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMatilla, M. A., Fang, X., and Salmond, G. P. C. (2014). Viunalikeviruses are environmentally common agents of horizontal gene transfer in pathogens and biocontrol bacteria. ISME J. 8, 2143–2147. doi: 10.1038/ismej.2014.150\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMatilla, M. A., and Salmond, G. P. C. (2014). Bacteriophage phiMAM1, a viunalikevirus, is a broad-host-range, high-efficiency generalized transducer that infects environmental and clinical isolates of the enterobacterial genera Serratia and Kluyvera. Appl. Environ. Microbiol. 80, 6446–6457. doi: 10.1128/AEM.01546-14\\\\nCrossRef Full Text | Google Scholar\\\\nParkinson, N., Stead, D., Bew, J., Heeney, J., Tsror, L., and Elphinstone, J. (2009). Dickeya species relatedness and clade structure determined by comparison of recA sequences. Int. J. Syst. Evol. Microbiol. 59, 2388–2393. doi: 10.1099/ijs.0.009258-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPelfrene, E., Willebrand, E., Cavaleiro Sanches, A., Sebris, Z., and Cavaleri, M. (2016). Bacteriophage therapy: a regulatory perspective. J. Antimicrob. Chemother. 71, 2071–2074. doi: 10.1093/jac/dkw083\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPetty, N. K., Foulds, I. J., Pradel, E., Ewbank, J. J., and Salmond, G. P. C. (2006). A generalized transducing phage (phiIF3) for the genomically sequenced Serratia marcescens strain Db11: a tool for functional genomics of an opportunistic human pathogen. Microbiology 152, 1701–1708. doi: 10.1099/mic.0.28712-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPritchard, L., Humphris, S., Baeyen, S., Maes, M., Van Vaerenbergh, J., Elphinstone, J., et al. (2013a). Draft genome sequences of four Dickeya dianthicola and four Dickeya solani strains. Genome Announc. 1:e00087–12. doi: 10.1128/genomeA.00087-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPritchard, L., Humphris, S., Saddler, G. S., Elphinstone, J. G., Pirhonen, M., and Toth, I. K. (2013b). Draft genome sequences of 17 isolates of the plant pathogenic bacterium Dickeya. Genome Announc. 1:e00978–13. doi: 10.1128/genomeA.00978-13\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSławiak, M., van Beckhoven, J. R. C. M., Speksnijder, A. G. C. L., Czajkowski, R., Grabe, G., and van der Wolf, J. M. (2009). Biochemical and genetical analysis reveal a new clade of biovar 3 Dickeya spp. strains isolated from potato in Europe. Eur. J. Plant Pathol. 125, 245–261. doi: 10.1007/s10658-009-9479-2\\\\nCrossRef Full Text | Google Scholar\\\\nToth, I. K., Cahill, G., Elphinstone, J. G., Humphris, S., and Wale, S. J. (2016). “An update on the Potato Council/Scottish government-funded blackleg project - Year 2,” in Proceedings Crop Protection in Northern Britain 2016 (Dundee), 203–204.\\\\nGoogle Scholar\\\\nToth, I. K., van der Wolf, J. M., Saddler, G., Lojkowska, E., Hélias, V., Pirhonen, M., et al. (2011). Dickeya species: an emerging problem for potato production in Europe. Plant Pathol. 60, 385–399. doi: 10.1111/j.1365-3059.2011.02427.x\\\\nCrossRef Full Text | Google Scholar\\\\nTsror, L., Erlich, O., Lebiush, S., Hazanovsky, M., Zig, U., Slawiak, M., et al. (2009). Assessment of recent outbreaks of Dickeya sp. (syn. Erwinia chrysanthemi) slow wilt in potato crops in Israel. Eur. J. Plant Pathol. 123, 311–320. doi: 10.1007/s10658-008-9368-0\\\\nCrossRef Full Text | Google Scholar\\\\nvan der Wolf, J. M., Nijhuis, E. H., Kowalewska, M. J., Saddler, G. S., Parkinson, N., Elphinstone, J. G., et al. (2014). Dickeya solani sp. nov., a pectinolytic plant-pathogenic bacterium isolated from potato (Solanum tuberosum). Int. J. Syst. Evol. Microbiol. 64(Pt 3), 768–774. doi: 10.1099/ijs.0.052944-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fvets.2022.1058115\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fvets.2022.1058115/full\",\\n    \"loadedTime\": \"2024-01-23T08:50:43.881Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fvets.2022.1058115\",\\n    \"title\": \"Frontiers | A lytic bacteriophage isolate reduced Clostridium perfringens induced lesions in necrotic enteritis challenged broilers\",\\n    \"description\": \"Background: Bacteriophages are viral predators of bacteria and occur commonly in nature. Their host-specific infections against specific bacteria make them an attractive natural agent to control bacterial pathogens. Interest in the potential of bacteriophages as antibacterial agents in the production animal industries has increased. Methods: A total of 18 bacteriophages were isolated from Australian commercial poultry environments, from which three highly active phages were chosen for enrichment. Sequencing libraries were prepared using a Nextera XT kit (Illumina) and sequenced on an Illumina MiSeq instrument using 2×300 bp paired-end chemistry. The sequence data were then assembled and aligned with a2 bacteriophage as the reference. An animal trial was performed by oral gavaging Clostridium perfringens netB containing strain EHE-NE18 to the Ross 308 broiler chickens prior inoculation with Eimeria species. The chickens were raised following the management guide for Ross 308 from d 0 to d 21 and fed with starter and grower diets met the specific breed nutrient requirements. Body weight gain and feed intake were measured on d 9 and d 21 and FCR adjusted with mortality was calculated. Results: The isolated bacteriophages only had only 96.7% similarity to the most closely related, previously characterised, Clostridium bacteriophage indicated that they might represent a novel strain of bacteriophage. A ‘cocktail’ containing the three bacteriophages was capable of lysing four known disease-inducing C. perfringens strains in vitro. Oral administration of the bacteriophages cocktail to broilers challenged with necrotic enteritis markedly alleviated intestinal necrotic lesions in the duodenum and jejunum on day 16. The phage treatment significantly reduced the lesion scores of birds challenged with NE (P  0.05). However, no effect on the growth performance was apparent during the recorded period of days 9-21. Conclusions: These findings suggest that bacteriophage treatment is a promising approach to protect intestinal health from C. perfringens induced necrotic enteritis. Further research will be required on the dosing, route of administration, and large scale validation studies to further advance this approach to pathogen control.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nNecrotic enteritis (NE) is a bacterial enteric disease in poultry accountable for over US$6 billion profit loss annually to the industry worldwide in the treatments and containment measures (1). The etiology of NE has been well characterized and it is mainly due to the rapid proliferation of C. perfringens Type G (2) strains that express the NetB toxin (3), albeit NetB negative strains have also been reported to cause NE (4). Outbreaks of NE also depend on a number of predisposing factors that result in degradation of intestinal health and compromise of the host immune status (3, 5, 6). It is reported that change of feed, use of different cereal bases in diet formulation, i.e., non-viscous grains to viscous grains, the inclusion of low digestibility protein sources, Eimeria infection, and compromise of to the host immune status, all exacerbate the chances of NE outbreaks (7). Flocks affected by acute NE exhibit clinical symptoms coupled with a rapid increase in flock mortality and distress in the affected bird populations, but subclinical (mild) disease are more frequent with little mortality rate if any, and suboptimal growth performance. The subclinical form of necrotic enteritis leads to more significant economic consequences with prolonged deterioration of growth performance resulting from loss of feed efficiency over time (8).\\\\nFor decades, poultry and other animal production systems across the globe have depended on in-feed antibiotics to sustain growth performance and control enteric pathogens as they can effectively maintain intestinal microbiota uniformity and suppress bacterial infection (9). However, the emergence of antibiotic-resistant bacteria and the possibility of horizontal transfer of antibiotic resistance genes to human pathogens resulted in the ban of in-feed antibiotics and has ushered in antibiotic-free practices that require alternatives to conventional in-feed antibiotics to maintain flock performance and health. The suppression of specific intestinal bacterial populations is essential to manage opportunistic pathogens and stop them from reaching a disease-causing threshold (10). Following the 2006 EU\\'s legislative ban on antibiotic use in animal feed, numerous countries across the globe have followed suit and are dealing with the challenges of running antibiotic-free production systems. Early efforts to overcome these challenges focused on the adoption of antibiotic-free measures that provided some protection against pathogens (11), but the end results have been inconsistent.\\\\nThe lack of consistency in the efficacy of the current alternatives for maintaining performance means that novel solutions continue to be explored. The interest in bacteriophages stems from the fact that they naturally target specific bacteria and have narrow host ranges. On exposure to specific bacteriophages, a host bacterium undergoes lysis resulting in the death of the bacterium and the release of bacteriophages that can continue to infect other host bacteria; this self-replicating process is a very different process compared to other alternatives to antibiotics. Hence, bacteriophages are an attractive and feasible option for pathogen control. The use of phage therapy is not new. In fact, the clinical application of bacteriophages was first implemented in the early 1920s, with extensive therapeutic application in human medicine across the eastern European countries, and it has continued to the present time (12). Many known attributes of bacteriophages fit the criteria for an effective antibacterial alternative, including automatic dosing capability through lytic propagation of bacteriophages progeny, minimal inherent toxicity, low disruption to commensal microbiota, biofilm clearing, and variable routes of administration (13). Bacteriophage absorption on the surface of susceptible bacteria is mediated by specific receptors on the bacteria cell wall (14). Different types of bacteriophages that infect any one species of bacteria may recognize different receptors. Hence, the combination of several bacteriophages in the form of a cocktail may improve their efficacy against a target bacteria. Indeed, the use of multivalent bacteriophage cocktails was reported to be successful in reducing mortality and improving performance in chickens infected with C. perfringens resulting in clinical NE (15).\\\\nHowever, only limited research has been reported on phage prophylactic treatment against C. perfringens induced subclinical necrotic enteritis in chickens. This study was conducted to isolate and purify bacteriophages from poultry production environments that could infect and lyse C. perfringens, determine their lytic profile against a range of known virulent C. perfringens strains capable of inducing NE, and then the isolated bacteriophage was evaluated for prophylactic properties in vivo using a necrotic enteritis challenge model.\\\\nMaterials and methods\\\\nSample collection, bacterial hosts, and bacteriophage preparation\\\\nSamples from various sources within poultry environments were aseptically collected for bacteriophage isolation, which included soil samples from backyard chicken sheds and yards located in Armidale and Tamworth, NSW, Australia, sewage and offal wash from seven individual chicken farms, and intestinal content of chickens from experiments conducted at UNE (Armidale, New South Wales, Australia).\\\\nAll collected samples were kept on ice until processed. C. perfringens were isolated by plating serial dilutions in 0.1% peptone onto Perfringens Agar Base (Oxoid, Hampshire, UK) supplemented with Tryptose Sulphite Cycloserine (TSC) (Oxoid, Hampshire, UK), followed by incubation under anaerobic conditions using an anaerobic sachet (Oxoid Australia) at 39oC for 48 h. Putative C. perfringens colonies were re-streaked twice (to ensure purity) on fresh Horse Blood Agar (HBA) (Oxoid, Hampshire, UK) and incubated at 39oC for 18 h. Bacteria isolates were picked and suspended in the Brain Heart Infusion (BHI, Oxoid, Hampshire, UK) broth with sterile glycerol addition to 40% (v/v) and stored at −20oC until used.\\\\nSamples for bacteriophage isolation were prepared by gently shaking 1 g of solid samples (i.e., soil, gut content or litter) or 1 ml liquid sample (effluent) in 9 ml of SM buffer (0.1 M NaCl, 1 mM MgSO4, 0.2 M Tris-HCl, pH 7.5) for 12 h. The supernatant was then centrifuged at 11,000 × g at 4oC for 5 mins. The supernatant was then filtered through 0.22-μm-pore-size Millipore filters (MF-Millipore™, MERCK, Australia). Bacteriophages were isolated as per the methods of Smith (16) using the isolated C. perfringens host strains. In brief, overnight growth of C. perfringens from HBA agar was initially mixed in 3 ml brain-heart infusion (BHI) broth, and then further BHI was added to adjust the turbidity to 220 ± 15 Nephelometric Turbidity Unit (NTU). Serial dilutions of bacteriophage filtrates were prepared in SM buffer and 100 μl of each bacteriophage dilution was mixed with 200 μl of C. perfringens and incubated at 37oC for 30 min. Each set of the bacteriophages-host mixture was mixed into 7 ml of 0.5% BHI soft agar (supplemented with 1 mM MgCl2 and 1 mM Ca Cl2) and plated as an overlay on 1% BHI base agar. Plates were incubated anaerobically at 39oC for 48 h. Well separated plaques were picked and resuspended in 1 ml of SM buffer, then stored at 4°C overnight. Serial dilutions of the homogenized plaques were made and cultured with the susceptible host in three cycles of plaque purification (16, 17).\\\\nPhage stocks were prepared from three to five plates with confluent plaques from the third round of phage purification. SM buffer (5 ml) was added to confluent plates and shaken slowly at 10 rpm for 12 h to resuspend the bacteriophage. The supernatants were then filtrated through 0.22-μm-pore-size Millipore filters (MF-Millipore™, MERCK, Australia). The resulting filtrates were centrifuged at 40,000 × g at 4°C for 2h. The resulting bacteriophage pellets were resuspended in SM buffer to form 1 × 109 pfu/ml bacteriophage stock and stored in the dark at 4°C.\\\\nLytic profile of the bacteriophage\\\\nThe bacteriophages were tested for lytic ability against four pathogenic C. perfringens strains (NE14, NE18, NE21, and NE36) to investigate the bacteriophage\\'s virulence against target hosts in vitro. The characteristics of the tested C. perfringens strains are detailed in Table 1. Bacterial lawns were prepared using turbidity-adjusted C. perfringens strains (220 ± 15 NTU) mixed with 7 ml of 0.5% BHI soft agar (supplemented with 5mM MgSO4 and 10mM CaCl2) and set on the prepared 1% BHI base agar. The bacteriophage lysates were titered, and all were within the range of 1–7 × 109pfu/ml. A volume of 20 μl of each bacteriophage preparation was aseptically placed in duplicate on marked positions of the bacterial lawns and set to dry with minimal disturbance to retain the bacteriophage droplets at the designated positions. Plates were incubated anaerobically at 39°C up to 48 h and inspected for plaque formation. All positive tests were replicated three times with the relevant C. perfringens host.\\\\nTable 1. Characteristics of C. perfringens strains and susceptibility to the bacteriophages.\\\\nDNA sequencing of the bacteriophage\\\\nThe bacteriophage stocks were concentrated by precipitation with polyethylene glycol, and DNA was isolated using a phenol/chloroform extraction method. Sequencing libraries of 5 bacteriophages from different sources (A2, C2, D1, E1, and H1) were prepared using a Nextera XT kit (Illumina). The libraries were sequenced on an Illumina MiSeq instrument using 2 × 300 bp paired-end chemistry. The sequence data generated from the MiSeq were assembled using the A5-miseq assembly pipeline (21). The bacteriophage DNA sequences were aligned using Clone Manager Suite V8 (Scientific and Educational Software, NC) with a2 bacteriophage as the reference. Homology analysis of the bacteriophage a2 sequence was performed against the NCBI database using the BLASTn program. Gene prediction was performed by submitting the DNA sequence of bacteriophage a2 to the RAST server (https://rast.nmpdr.org/) for RAST annotations. The whole genome sequence data has been deposited in the NCBI database with the accesion numbers OP753449–OP753453.\\\\nPhage inoculants and in vivo study\\\\nEighteen bacteriophage isolates that had lytic activity against the C. perfringens EHE-NE18 strain were enriched for the preparation of a bacteriophage inoculate. Three selected bacteriophages were prepared in larger volumes using the procedure adopted for purification as described above. Pelleted bacteriophages were resuspended in SM buffer to a final concentration of 1 × 106 pfu/ml for the in vivo experiment.\\\\nDay-old birds (Ross 308 broilers) were obtained from a commercial hatchery (Baiada Hatchery, Tamworth, NSW, Australia). Birds were housed in 18 pens, each containing 11 birds. The pens (0.9 × 0.95 m) were equipped with feeders and drinkers and hardwood shavings were used as bedding materials. The temperature and lighting were controlled according to the Ross 308 broiler guideline. A standard wheat-soybean starter diet formulated to meet the 2014 Ross 308 nutrient specifications was fed to all birds from day 0 to 10 (Table 2). A grower diet consisting of the wheat-based formulation was fed from day 10 to the end of the trial on day 21 (Table 2). Feed and water were available ad libitum. Pen weight and feed intake were measured on days 9 and 21, and the feed conversion ratio (FCR) was calculated by pen feed intake divided by pen bird weight gain and adjusted with mortality during the rearing period of d 9 to d 21.\\\\nTable 2. Composition and nutrients of starter and grower.\\\\nThe experiment consisted of a control untreated and unchallenged group, and necrotic enteritis challenge groups with or without bacteriophage treatment. On day 9, the NE challenge groups were given 1 ml of a mixture of field strains of Eimeria species containing 5,000 oocysts each of E. acervulina and E. maxima, and 2,500 oocysts of E. brunetti (Eimeria Pty Ltd, Werribee, VIC, Australia) by oral gavage, and unchallenged control birds were orally gavaged with 1 ml phosphate buffered saline as a sham treatment. On days 14-15, birds in the challenged groups were inoculated by oral gavage with 1 × 108 CFU of C. perfringens EHE-NE18 strain as described by Wu et al. (22). The bacteriophage treatment birds were orally gavaged with 1 × 106 units of the C. perfringens specific bacteriophage inoculate approximately 1 h following C. perfringens gavage. On day 16, two birds per pen from all the groups were randomly selected, weighed, and euthanized by cervical dislocation to perform post-mortem analysis and intestinal lesion scoring at a 0–6 scale according to Keyburn et al. (19). On day 21, the experiment was completed, and birds were euthanized and disposed of according to the approved ethics protocol.\\\\nStatistical analysis\\\\nData were analyzed by one-way ANOVA with the pen as the experimental unit. Paired comparisons of means were performed using Tukey\\'s test when differences among treatments were detected by ANOVA. Significance at P < 0.05 was declared for paired comparison.\\\\nResults and discussion\\\\nLytic properties of C. perfringens specific bacteriophages and genomic characterization\\\\nThe current study isolated and purified eighteen C. perfringens bacteriophages isolates. Each bacteriophage isolate was capable of producing plaques (Figure 1A) on each of the C. perfringens hosts obtained from the same poultry environments (soil, sewage, offal wash or intestinal content) and corresponding geographic locations. Among 46 field C. perfringens isolates from the environment, only three C. perfringens isolates were susceptible to all the bacteriophages isolates and thus used as the hosts for the bacteriophages. Therefore, all 18 bacteriophages were subsequently enriched with these three common hosts for bacteriophage purification, and the resulting bacteriophage preparations were used to perform lytic tests individually against pathogenic C. perfringens strains derived from chickens with necrotic enteritis. Positive lytic activities were observed for all these bacteriophage strains against four previously described virulent C. perfringens strains (NE14, NE18, NE21, and NE36) in our spot-testing assay. However, other strains of C. perfringens (NE1, NE4, NE7, NE16, NE31, and NE38) were not susceptible to these bacteriophages (Table 1). As the bacteriophage preparation was lytic against NE18 strain, which has been reported to produce NetB toxin (3) and has been used in NE experimental disease induction models (23), this preparation was used as an inoculate in the subsequent in vivo experiment with EHE-NE18 as the challenge strain.\\\\nFigure 1. (A) Clostridium perfringens bacteriophage plaques. Arrows showing the plaques produced by the strains from sample E that was collected from chicken caecal content, (B) An evolutionary tree of Clostridium bacteriophage strains produced by NCBI online blast tools with the Fast Minimum Evolution tree method. The Clostridium phage A2 isolated in the current study is highlighted in yellow showing its close relation with Clostridium phage CPD4. The scale bar corresponds to 0.07 substitutions per nucleotide position.\\\\nA narrow spectrum of infectiousness against C. perfringens strains also presents the possibility of a controlled elimination of pathogenic C. perfringens strains but minimal harm to commensal C. perfringens. It has been demonstrated that C. perfringens exists as normal gut commensals with low abundance in the intestinal tract of healthy chickens (6).\\\\nFive bacteriophages genomes were randomly chosen for sequencing, and the BLAST analysis against the NCBI database indicated the bacteriophages isolated aligned with the Clostridium bacteriophages CPD4 with a homology of 97.7% which clustered closely as shown in Figure 1B and 77 genes were predicted, of which 19 had specific bacteriophage functions. The degree of homology to previously characterized bacteriophages indicates that the bacteriophages isolated in this study might represent a novel bacteriophage strain. C. perfringens specific bacteriophage isolates have been reported to belong to the family Siphoviridae or Podviridare, and bacteriophages endolysin has been shown to be responsible for the lytic activity (17, 24). The sequences of the bacteriophage isolates produced in the current study generated a single contig with the same GC content (34%) and showed a similar genome size with 99.98–100.00% identities (Table 3). The genomes of several C. perfringens specific bacteriophages have been characterized as reported previously, providing essential information for understanding the virulence of phages to their hosts and for downstream applications (17, 25).\\\\nTable 3. A5-miseq assemblies of the bacteriophages isolates.\\\\nBacteriophage treatment reduced NE induced lesions\\\\nNo mortality of birds due to NE and low lesion scores (0.8 out of a maximum of 6) were observed in the in vivo challenge experiment, indicating that subclinical NE had been induced. The lesion score results are shown in Table 4. Higher lesion scores were recorded in both the duodenum (P < 0.01) and jejunum (P < 0.01) of the challenged birds than those unchallenged. The phage treatment significantly reduced the lesion scores of birds challenged with NE (P < 0.01), such that the lesion scores between birds treated with the bacteriophages and the control unchallenged birds were not statistically different (P > 0.05). However, this significant effect of the bacteriophages on lesion scores did not fully manifest in the measures of bird performance. In the current study, the birds under NE challenge were significantly affected, showing reduced BWG, FI, and increased FCR in the challenged treatments compared to the control (Table 5). NE challenged birds with or without bacteriophage treatment had reduced BWG and FI and increased FCR during days 9 to 21 or 0 to 21 (P < 0.001 for all except FCR during d 0–21 being P = 0.002). The challenge model used in the current study experimentally reproduced the conditions of possible NE outbreaks in the field, of which coccidiosis serves as a predisposing factor to damage intestinal epithelium before C. perfringens infection (7, 22). The current study was not set out to apportion the impact of the Eimeria per se, and therefore, it is difficult to determine with any certainty to what extent the lack of performance improvement by the bacteriophage therapy could be due to its inability to counter any damage that resulted from coccidiosis. Eimeria negatively affects nutrient utilization by interfering with the integrity of epithelium cells, leading to compromised growth of the animals (6, 26).\\\\nTable 4. Lesion scores of the birds under treatments at day 16*.\\\\nTable 5. Performance of the birds in response to the NE and bacteriophages treatments*.\\\\nAlthough bacteriophage treatment did not translate into improved bird performance, less severity of intestinal lesions in the treated birds suggests that bacteriophages appropriately selected against the NetB producing strain of C. perfringens may be worthy of further exploration.\\\\nThe current study focused on the performance of the birds during days 9–21, when the NE challenge was applied, and the NE effect was expected. However, a prolonged rearing period might be useful to observe the bacteriophage treatment effect on later stage performance through improved intestinal health. In addition, the dosage and the route of bacteriophage administration may also be important factors for the effectiveness of the bacteriophage treatment to protect birds from NE infection. These will need to be addressed in future studies to maximize the protective effect of bacteriophages, such as water and feed delivery and higher doses. Further, more bacteriophage strains may be needed to control C. perfringens strains in chickens through the administration of a wider spectrum cocktail.\\\\nIn conclusion, 18 isolated C. perfringens specific bacteriophage strains were isolated. The finding that they had 96.7% similarity to the most closely related, previously characterized, Clostridium bacteriophage indicated that they represent a similar cluster of Clostridium bacteriophage. The result showed an encouraging protective effect of the bacteriophage application with alleviated lesions in the gut of NE-challenged birds despite the fact that no performance improvement was seen. Whether bacteriophage can be used in the poultry production to combat NE is yet to be assured. Further investigations are warranted to optimize the dosage and administration protocol and to isolate more NetB positive and even negative bacteriophage strains for a wider spectrum control of different C. perfringens pathogens.\\\\nData availability statement\\\\nThe data presented in the study are deposited in the NCBI GenBank repository, accession numbers OP753449 – OP753453.\\\\nEthics statement\\\\nThe animal study was reviewed and approved by the Animal Ethics Committee of the University of New England (UNE), Armidale, Australia.\\\\nAuthor contributions\\\\nCK performed data collection and analysis, interpreted results, and wrote the manuscript. KM performed isolation of the bacteriophages and analysis. TV sequenced the bacteriophage genome, analyzed data, and read the manuscript critically. HC participated the design of the project, isolated bacteriophages, and reviewed the manuscript critically. RM participated the design of the project, data analysis, and reviewed the manuscript critically. MC participated the conceptualization of the project and reviewed the manuscript critically. S-BW conceptualized, designed the project, performed the animal experiment, analyzed data, and reviewed the manuscript critically. All authors contributed to the article and approved the submitted version.\\\\nFunding\\\\nThis study was supported by the Poultry CRC (#1-3-4).\\\\nConflict of interest\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nPublisher\\'s note\\\\nAll claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.\\\\nReferences\\\\n1. Wade B, Keyburn, A. The true cost of necrotic enteritis. Poultry World. (2015) 31:16–17. \\\\nGoogle Scholar\\\\n2. Rood, JI, Adams V, Lacey J, Lyras D, McClane BA, et al. Expansion of the Clostridium perfringens toxin-based typing scheme. Anaerobe. (2018) 53:5–10. doi: 10.1016/j.anaerobe.2018.04.011\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n3. Keyburn AL, Boyce JD, Vaz P, Bannam TL, Ford ME, Parker D, et al. NetB, a new toxin that is associated with avian necrotic enteritis caused by Clostridium perfringens. PLoS Pathogens. (2008) 4:e26. doi: 10.1371/journal.ppat.0040026\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n4. Cooper KK, Songer JG. Virulence of Clostridium perfringens in an experimental model of poultry necrotic enteritis. Vet Microbiol. (2010) 142:323–8. doi: 10.1016/j.vetmic.2009.09.065\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n5. Keyburn AL, Bannam TL, Moore RJ, Rood JI. NetB, a pore-forming toxin from necrotic enteritis strains of Clostridium perfringens. Toxins. (2010) 2:1913–27. doi: 10.3390/toxins2071913\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n6. Van Immerseel F, Rood JI, Moore RJ, Titball RW. Rethinking our understanding of the pathogenesis of necrotic enteritis in chickens. Trends Microbiol. (2009) 17:32–6. doi: 10.1016/j.tim.2008.09.005\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n8. Skinner JT, Bauer S, Young V, Pauling G, Wilson J. An economic analysis of the impact of subclinical (mild) necrotic enteritis in broiler chickens. Avian Dis. (2010) 54:1237–40. doi: 10.1637/9399-052110-Reg.1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n9. Lanckriet A, Timbermont L, De Gussem M, Marien M, Vancraeynest D, Haesebrouck F, et al. The effect of commonly used anticoccidials and antibiotics in a subclinical necrotic enteritis model. Avian Pathol. (2010) 39:63–8. doi: 10.1080/03079450903505771\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n10. Eeckhaut V, Wang J, Van Parys A, Haesebrouck F, Joossens M, Falony G, et al. The probiotic Butyricicoccus pullicaecorum reduces feed conversion and protects from potentially harmful intestinal microorganisms and necrotic enteritis in broilers. Front Microbiol. (2016) 7:1416. doi: 10.3389/fmicb.2016.01416\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n11. Cogliani C, Goossens H, Greko C. Restricting antimicrobial use in food animals: lessons from Europe. Microbe. (2011) 6:274. \\\\nGoogle Scholar\\\\n13. Karthik K, Muneeswaran NS, Manjunathachar HV, Gopi M, Elamurugan A, Kalaiyarasu S. Bacteriophages: effective alternative to antibiotics. Adv Anim Vet Sci. (2014) 2:1–7. doi: 10.14737/journal.aavs/2014/2.3s.1.7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n14. Pleteneva EA, Shaburova OV, Sykilinda NN, Miroshnikov KA, Krylov SV, Mesianzhinov VV, Krylov, VN. Study of the diversity in a group of phages of Pseudomonas aeruginosa species PB1 (Myoviridae) and their behavior in adsorbtion-resistant bacterial mutants. Genetika. (2008) 44:185–94. doi: 10.1134/S1022795408020051\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n15. Miller RW, Skinner J, Sulakvelidze A, Mathis GF, Hofacre, CL. Bacteriophage therapy for control of necrotic enteritis of broiler chickens experimentally infected with Clostridium perfringens. Avian Dis. (2010) 54:33–40. doi: 10.1637/8953-060509-Reg.1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n17. Seal BS, Fouts DE, Simmons M, Garrish JK, Kuntz RL, Woolsey R, et al. Clostridium perfringens bacteriophages ΦCP39O and ΦCP26F: genomic organization and proteomic analysis of the virions. Arch Virol. (2011) 156:25–35. doi: 10.1007/s00705-010-0812-z\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n18. Sheedy SA, Ingham AB, Rood JI, Moore RJ. Highly conserved alpha-toxin sequences of avian isolates of Clostridium perfringens. J Clin Microbiol. (2004) 42:1345–7. doi: 10.1128/JCM.42.3.1345-1347.2003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n19. Keyburn AL, Sheedy SA, Ford ME, Williamson MM, Awad MM, Rood JI. Alpha-toxin of Clostridium perfringens is not an essential virulence factor in necrotic enteritis in chickens. Inf Immun. (2006) 74:6496–500. doi: 10.1128/IAI.00806-06\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n20. Keyburn AL, Portela RW, Sproat K, Ford ME, Bannam TL, Yan X, et al. Vaccination with recombinant NetB toxin partially protects broiler chickens from necrotic enteritis. Vet Res. (2013) 44:1. doi: 10.1186/1297-9716-44-54\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n21. Coil D, Jospin G, Darling AE. A5-miseq: an updated pipeline to assemble microbial genomes from Illumina MiSeq data. Bioinformatics. (2015) 31:587–9. doi: 10.1093/bioinformatics/btu661\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n22. Wu S-B, Rodgers N, Choct M. Optimized necrotic enteritis model producing clinical and subclinical infection of Clostridium perfringens in broiler chickens. Avian Dis. (2010) 54:1058–65. doi: 10.1637/9338-032910-Reg.1\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n23. Gharib-Naseri, K, Kheravii, S, Keerqin, C, Swick RA, Choct, M, Wu, S-B. Differential expression of intestinal genes in necrotic enteritis challenged broiler chickens with two different Clostridium perfringens strains. Poultry Sci. (2020) 100:100886. doi: 10.1016/j.psj.2020.11.063\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n25. Morales CA, Oakley BB, Garrish JK, Siragusa GR, Ard MB, Seal BS. Complete genome sequence of the podoviral bacteriophage ΦCP24R, which is virulent for Clostridium perfringens. Arch Virol. (2012) 157:769–72. doi: 10.1007/s00705-011-1218-2\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n26. Collier C, Hofacre C, Payne A, Anderson D, Kaiser P, Mackie R, et al. Coccidia-induced mucogenesis promotes the onset of necrotic enteritis by supporting Clostridium perfringens growth. Vet Immunol Immunopathol. (2008) 122:104–15. doi: 10.1016/j.vetimm.2007.10.014\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fimmu.2018.01496\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fimmu.2018.01496/full\",\\n    \"loadedTime\": \"2024-01-23T08:50:55.664Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fimmu.2018.01496\",\\n    \"title\": \"Frontiers | Vectorized Delivery of Alpha-GalactosylCeramide and Tumor Antigen on Filamentous Bacteriophage fd Induces Protective Immunity by Enhancing Tumor-Specific T Cell Response\",\\n    \"description\": \"We have exploited the properties of filamentous bacteriophage fd to deliver immunologically active lipids together with antigenic peptides. Filamentous bacteriophages resemble for size, capability to be permeable to blood vessels and high density antigen expression, a nature-made nanoparticle. In addition, their major coat protein pVIII, which is arranged to form a tubular shield surrounding the phage genome, has a high content of hydrophobic residues promoting lipid association. We conjugated bacteriophages to alpha-GalactosylCeramide (\\uf061-GalCer), a lipid antigen stimulating invariant Natural Killer T (iNKT) cells and capable of inducing their anti-tumoral activities. We found that bacteriophage fd/\\uf061-GalCer conjugates could repeatedly stimulate iNKT cells in vitro and in vivo, without inducing iNKT anergy. Moreover, co-delivery of alpha-GalactosylCeramide and a MHC class I restricted tumor associated antigenic determinant to APCs via bacteriophages strongly boosted adaptive CD8+ T cell response and efficiently delayed tumor progression. Co-delivery of a tumor antigen and iNKT-stimulatory lipid on the surface of filamentous bacteriophages is a novel approach to potentiate adaptive anti-cancer immune responses, overcoming the current limitations in the use of free alpha-GalactosylCeramide and may represent an attractive alternative to existing delivery methods, opening the path to a potential translational usage of this safe inexpensive and versatile tool.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nInvariant natural killer T (iNKT) cells represent a unique subpopulation of T lymphocytes with both innate-like and adaptive functions mainly found in spleen, liver, and bone marrow. iNKT cells express NK lineage receptors and a semi-invariant T cell receptor (TCR) composed of Vα14–Jα18 chain in mice and Vα24–Jα18 chain in humans, paired with β chains encoded by a limited number of Vβ genes (1).\\\\nInvariant natural killer T cells recognizes (glyco)lipid antigens (Ag) presented by the CD1d molecule expressed by several types of antigen-presenting cells (APCs), including dendritic cells (DCs) (2).\\\\nThe potential anti-tumoral function of iNKT cells was first discovered with the identification of alpha-GalactosylCeramide (α-GalCer), a synthetic derivative of agelasphin, a glycolipid originally isolated from the marine sponge Agelas mauritianus, as a strong stimulatory ligand during screening of anti-tumor compounds from natural sources (3–5).\\\\nIn response to α-GalCer, iNKT cells rapidly secrete large quantities of cytokines, including IFN-γ, IL-2, IL-4, IL-6, IL-10, IL-12, IL-13, IL-17, IL-21, IL-22, TGF-β, and TNF-α, that in turn activate a variety of other cell types, including NK cells, DCs, B, and T cells (6–9). Through this activation cascade, α-GalCer showed to exert potent anti-tumor and adjuvant activities in vivo in mouse models (5, 10, 11), rendering it a powerful candidate for adjuvant therapy in cancer.\\\\nBased on these pioneering reports, attempts have been made to exploit the anti-tumor property of α-GalCer. Although the drug was well tolerated, no or moderate clinical responses were observed among the patients repeatedly injected intravenously with α-GalCer (12). Studies performed in a mouse model demonstrated that α-GalCer induced a long-term anergy of iNKT cells, thus preventing proliferation and cytokine release upon a recall stimulation (13).\\\\nEven though the mechanism underlying iNKT cell anergy mediated by α-GalCer is still unknown, it is common knowledge that co-stimulatory signals and cytokines provided by antigen-presenting DCs are considered crucial for avoiding anergy (14).\\\\nIndeed, mouse studies demonstrated that the injection of α-GalCer-pulsed DCs induced a sustained cytokines production when compared with administration of free α-GalCer, suggesting that the type of α-GalCer administration is critical for iNKT cell stimulation (15).\\\\nA way to optimize iNKT cell responses may lie in actively directing α-GalCer to the appropriate APC using suitable delivery systems (16).\\\\nIn previous studies, we found that the filamentous bacteriophage is an efficient antigen-delivery system because it is internalized by DCs and activates innate and adaptive immune responses in the absence of classical adjuvants (17–19).\\\\nThe filamentous bacteriophage can be considered as a nature-made nanocarrier according to its nano-dimensions (5 nm in diameter and 1,000 nm in length), its capability to cross blood vessels and for the capacity of expressing very large amounts of recombinant protein antigen. Its major coat protein pVIII is present in 2,700 copies on the phage coat and is arranged to form a tubular shield surrounding the phage genome. The pVIII protein is composed of three specific domains: a hydrophobic core, an acidic N-terminal domain, and a basic C-terminal domain (20). Due to the high content of hydrophobic residues, the pVIII shows strong binding of lipids, a feature that we exploited by promoting association of the bacteriophage with α-GalCer. Here, we show that α-GalCer conjugated with bacteriophages could repeatedly stimulate iNKT cells in vitro and in vivo, without inducing iNKT cell anergy. Moreover, therapeutic vaccination with phages co-displaying α-GalCer and a tumor antigen delayed tumor progression in mice, showing improved adjuvanticity of the phage particles and leading to enhanced vaccine efficacy. These results suggest a novel approach to potentiate the efficacy of the bacteriophage delivery system as anti-tumor vaccine.\\\\nMaterials and Methods\\\\nCells and Reagents\\\\nCD1d-restricted Vα14i NKT hybridoma FF13 (21) cells were a kind gift of Dr. De Libero (Department of Biomedicine, University of Basel and University Hospital Basel, Basel, Switzerland) and were cultured in RPMI 1640 (Lonza) medium supplemented with 100 U/ml penicillin, 100 µg/ml streptomycin, and 10% fetal calf serum (FCS) (all from GIBCO, Milan, Italy).\\\\nOTI hybridoma cell line, recognizing the OVA257–264 SIINFEKL determinant, was produced by infection of 54ζ17 hybridoma T cells (22) with a retrovirus encoding the OTI TCR V alpha 2 and V beta 5 chains. Recombinant retroviral particles were produced in HEK293T cells (ATCC CRL 1573) by transfection with pMXOTI, pEcotropic, and pVSVg plasmids. The virus-containing supernatant was collected 48 h after transfection. 5 × 105 54ζ17 cells were infected with 10 ml of viral supernatant and TCR expression was verified after 5 days by APC-anti-CD3ε (145-2C11, Biolegend) staining and FACS analysis. Positive cells were then sorted using a FACS ARIA (Becton Dickinson, Fullerton, CA, USA) and amplified in DMEM supplemented with 10% FCS, 25 U/ml penicillin G, 25 µg/ml streptomycin, and 0.05 µM β-mercaptoethanol.\\\\nB16-OVA melanoma cells (H2Kb), stably expressing chicken ovalbumin, were a kind gift of Dr. Dellabona (San Raffaele Scientific Institute, Milan, Italy). Cells were cultured in RPMI 1640 medium supplemented with 100 U/ml penicillin, 100 µg/ml streptomycin, 10% FCS, and 100 µg/ml Hygromicin (Sigma-Aldrich).\\\\nThe synthetic peptide OVA257–264 (SIINFEKL) was purchased from Primm (Naples, Italy).\\\\nSynthetic α-GalCer (KRN7000) (2S,3S,4R)-1-O-(α-D-galactosyl)-N-hexacosanoyl-2-amino-1,3,4-octadecanetriol (BML-SL232-1000) was purchased from Vinci Biochem.\\\\nInternal standard (IS) for mass spectrometric analysis D-Galactosyl-β-1,1’-N-Palmitoyl-D-erythro sphingosine was purchased from Avanti Polar Lipids.\\\\nMeOH and water for liquid chromatography–mass spectrometric analysis were purchased from Merck and were LC–MS grade.\\\\nMice\\\\nSix- to eight-week-old female C57BL/6 were purchased from Charles River (Lecco, Italy) and housed in IGB “A. Buzzati-Traverso” Animal House Facility under standard pathogen-free conditions abiding institutional guidelines.\\\\nPurification of Bacteriophages Particles and Conjugation to α-GalCer\\\\nWild type (fdWT) and hybrid fdOVA (expressing the recombinant OVA257–264-pVIII proteins) filamentous bacteriophages were purified from the supernatant of transformed Escherichia coli TG1recO cells. Bacteria were grown in TY2X medium for 16 h and the bacteriophage virions were harvested from E. coli supernatant, Poly(ethylene glycol) 6000 (Sigma-Aldrich) precipitated, purified by ultracentrifugation (24,500 g) on cesium chloride (Sigma-Aldrich) gradient, and dialyzed against phosphate buffered saline (PBS) 1×. Elimination of lipopolysaccharides (LPS) from phage particles was performed using Triton X-114 (Sigma-Aldrich). Briefly, Triton X-114 was mixed to the phage preparations to a final concentration of 1% by vigorous vortexing. The mixture was incubated at 4°C for 5 min, then incubated for 5 min at 50°C, and centrifuged (20,000 g, 10 min) at 25°C. The upper aqueous phase containing the virions was carefully removed and subjected to Triton X-114 phase separation for more cycles. The resulting aqueous phase containing virions was subjected to cesium chloride gradient centrifugation, dialyzed against PBS 1×, and assayed for LPS contamination using the Limulus Amebocyte Lysate assay (Limulus Amebocyte Lysate QCL-1000 chromogenic modification, Lonza), according to the manufacturer’s instructions.\\\\nThe expression of the recombinant OVA257–264-pVIII proteins was induced adding 0.1 mM isopropyl-beta-D-thiogalactopyranoside (Sigma-Aldrich) to the bacteria growing in TY2X medium.\\\\nThe number of copies of pVIII displaying the OVA257–264 peptide was estimated by N-terminal sequence analysis of the purified virions and resulted in 15–20% for each phage preparation.\\\\nBacteriophages in PBS 1× pH 8 and KRN7000 in dimethyl sulfoxide (Sigma-Aldrich) were combined at a 10:1 ratio (μg phages: μg α-GalCer) and stirred at 4°C overnight. Virions were subjected to cesium chloride gradient ultracentrifugation (24,500 g), dialyzed against PBS 1× and the concentration of bacteriophages was determined using spectrophotometer. The presence of α-GalCer in the phage preparations was determined by the in vitro biological assay and its conjugated amount determined by mass spectrometric analysis, as described below.\\\\nRelease of α-GalCer From the Conjugated Bacteriophage by Solvent Extraction\\\\nA small aliquot (50 µl) of a PBS solution containing the bacteriophage conjugated to α-GalCer at a concentration of 1.5 mg/ml was diluted 1:10 to a final volume of 500 µl with ultrapure water; 200 ng of IS in 20 µl of methanol (MeOH) were added and the suspension was extracted with 2 ml of MeOH/CHCl3 (1:1) by sonication. After centrifugation, the organic phase was removed and the aqueous layer was re-extracted with MeOH/CHCl3. The combined organic phases were dried under nitrogen, reconstituted in 1 ml of MeOH, and subjected to LC–MS analysis.\\\\nQuantitative LC–MS/MS Analysis of α-GalCer\\\\nA quantitative method was developed on a UPLC system (Acquity, Waters) coupled to a triple quadrupole mass spectrometer (API 3200, SCIEX). The chromatographic analysis was performed on an Acquity BEH Phenyl column (Waters, 100 × 2.1 mm, 1.7 µm), eluted with a short gradient program from 95:5 MeOH/H2O to 100% MeOH in 1 min followed by an isocratic elution at 100% MeOH for 4 min. Flow rate was set at 0.4 ml/min and column temperature at 40°C. α-GalCer eluted at a Rt of 1.59 min, IS at 1.1 min.\\\\nA calibration curve was prepared by using five calibration points of α-GalCer standard (STD) (62.5, 125, 250, 500, and 1,000 ng/ml) spiked with 200 ng/ml IS and plotted as area ratio of STD/IS response vs concentration. Two MRM transitions were monitored for both STD and IS for quantitative purposes and to confirm analytical identification. The most intense transitions for each compound (i.e., m/z 856.7 > 178.9 for STD and m/z 698.5 > 89.2 for IS) were used as analytical responses.\\\\nIn Vitro α-GalCer Presentation on CD1d and Stimulation of iNKT and OTI Hybridoma Cells\\\\nMouse bone marrow-derived dendritic cells (BMDCs) were generated from C57BL/6 mice according to Ref. (23). At 7 day of culture, BMDCs were incubated in RPMI medium supplemented with 10% FCS, 5 µM 2-ME, 1 mM glutamine, and 1 mM sodium pyruvate for 2 h with different concentrations of free α-GalCer, fdWT bacteriophages, or fd/α-GalCer bacteriophages. The experiment OTI hybridoma cell experiment was performed by incubating BMDCs with different concentration of fdOVA, fdOVA/α-GalCer, or OVA257–264 synthetic peptide. After the incubation, cells were washed and stained with PE-conjugated anti mouse α-GalCer:CD1d complex (L363, Biolegend) or co-cultured (50,000/well) with the mouse Vα14 iNKT hybridoma FF13 or OTI hybridoma (50,000/well) for 40 h.\\\\nPE-conjugated anti-mouse α-GalCer:CD1d complex (L363, Biolegend) antibody was used to stain DCs and fluorescence of stained cells was analyzed by FACSCanto II flow-cytometer and DIVA (Data-Interpolating Variational Analysis) software (Becton Dickinson). The IL-2 released into cell co-culture supernatants was measured by ELISA. Supernatants (0.1 ml/well) were assayed in duplicate using mouse IL-2 ELISA MAX™ Standard (Biolegend), according to the manufacturer’s instructions.\\\\nMeasurement of In Vivo and In Vitro Recall Response to α-GalCer\\\\nMice were injected intravenously (i.v.) or intratumorally (i.t.) with 100 µl of PBS containing 5 µg α-GalCer, 50 µg of fd/α-GalCer bacteriophage conjugate, or with vehicle alone. Where indicated, 200 or 130 ng of α-GalCer was also used. After 24 h mice were sacrificed, spleens were harvested, and single-cell suspensions were prepared. Spleen cell suspensions were plated in U-bottomed 96-well plate at 2 × 105 cells per well in RPMI medium supplemented with 10% FCS, 5 µM 2-ME, 1 mM glutamine, and 1 mM sodium pyruvate in the presence of indicated doses of free α-GalCer or with medium alone. For proliferation assays, 1 μCi of [3H] thymidine (PerkinElmer Life Sciences) was added to the wells after 60 h of culture, and cells were cultured for an additional 12 h. Cells were then harvested using a semi-automatic cell harvester FilterMate (PerkinElmer, CA, USA) and uptake of radioactivity was measured using the Top Count NTX microplate scintillation counter (PerkinElmer). Cell proliferation was expressed as a cpm fold increase vs unstimulated cells. IL-2 secretion was evaluated by ELISA using Mouse IL-2 ELISA MAX™ Standard, collecting cell culture supernatants (0.1 ml/well) after 60 h of culture.\\\\nAnalysis of iNKT and NK In Vivo Activation\\\\nMice (n = 3/group) were injected i.v. with 100 µl of PBS containing 5 µg free or 50 µg of fd and fd/α-GalCer. Mice were sacrificed 3 h post-treatment. Splenic iNKT cells were analyzed for intracellular IFN-γ secretion by culturing 7.5 × 106 spleen cells overnight in the presence of 10 µg/ml of the secretion inhibitor Brefeldin-A (BFA, Sigma-Aldrich). Cells were then harvested and IFN-γ production was evaluated by intracellular staining on gated CD3+NK1.1+ cells or CD3−NK1.1+ cells using APC-conjugated anti mouse CD3 monoclonal antibody (mAb) (17A2, Biolegend), PE-conjugated anti-mouse NK1.1 mAb (PK136, Biolegend), FITC-conjugated anti-mouse IFN-γ mAb (XMG1.2, Biolegend), and Leucoperm fixation and permeabilization kit (AbD Serotec, Oxford, UK). The IFN-γ release induced by 30 ng/ml of Phorbol 12-myristate 13-acetate (PMA, Sigma-Aldrich) plus 1 µg/ml of ionomycin (Sigma-Aldrich) was used as a positive control. Cells were analyzed using FACSCanto II flow-cytometer and DIVA software.\\\\nIn Vivo OVA257–264 T Cell Response Evaluation\\\\nGroup of mice (n = 4) were primed (day 0) by subcutaneous injection with 100 µl of PBS containing fdOVA (SIINFEKL peptide) bacteriophages or fdOVA/α-GalCer bacteriophages (all delivering 1.5 µg of OVA257–264 peptide) and boosted (day 14) with the same amount of fdOVA bacteriophages delivering or not the α-GalCer. As control, mice were inoculated twice with vehicle alone (PBS).\\\\nAt day 21, splenocytes were isolated and the frequency of OVA257–264 specific CD8+ T cells was assessed using FITC or APC conjugated anti-mouse CD8a mAb (53–6.7, Biolegend) and PE-H2Kb SIINFEKL MHC dextramers (Immudex) staining. Stained cells were analyzed by flow cytometry using a FACSCantoII. Results are expressed as the percentage of CD8+ gated cells that are positive for the MHC I/peptide dextramers.\\\\nIFN-γ-producing effector cells were evaluated by culturing 7.5 × 106 spleen cells with OVA257–264 SIINFEKL synthetic peptide (10 μg/ml) for 5 h in the presence of BFA (Sigma-Aldrich). Cells were then harvested and IFN-γ production was evaluated by intracellular staining on gated CD8+ cells using FITC or APC conjugated anti-mouse CD8a mAb, PE conjugated anti-mouse IFN-γ mAb (XMG1.2 Biolegend), and Leucoperm fixation and permeabilization kit. The IFN-γ release induced by 30 ng/ml of PMA plus 1 µg/ml of ionomycin was used as a positive control. Data were acquired on FACSCanto II flow-cytometer and DIVA software.\\\\nTherapeutic Vaccination Against B16 Tumor Cells\\\\nNaïve C57BL/6 mice were engrafted with 2 × 105 B16 melanoma cells subcutaneously in the left flank. When tumors were palpable, mice were vaccinated with 100 µl volumes containing PBS, 2.5 µg α-GalCer, 50 µg of fdWT bacteriophages, or 50 µg of fd/α-GalCer. In the experiment with B16-OVA tumor cell line, mice were injected intratumorally twice, on day 0 and day 5, with 100 µl volumes containing PBS, 2.5 µg α-GalCer, 50 µg of fdOVA bacteriophages, 50 µg of fd/α-GalCer, or 50 µg of fdOVA/α-GalCer. Tumor growth was assessed three times weekly using caliper and recorded as tumor volume (mm3) according to the formula (d2 × D)/2, where d and D are the shortest and the longest diameters. Mice were culled once tumor size met or exceeded 1,500 mm3, in accordance with established guidelines. Survival was recorded as the percentage of surviving animals.\\\\nIn another set of experiments, mice were sacrificed on day 12 and tumors and spleens were collected and homogenized. Tumors and spleen cells were filtered through a 70-µm cell strainer nylon mesh, and erythrocytes were lysed. Cells were then washed once with medium, resuspended in PBS + 5% FCS to a concentration of 1.5 × 106 cells/ml and labeled with APC conjugated anti-mouse CD8a mAb and PE-H2Kb SIINFEKL MHC dextramers for flow cytometric analysis. 7-aminoactinomycin D (7-AAD) (BD) was used for live–dead cell discrimination.\\\\nStatistical Analysis\\\\nComparative analyses were performed using the analysis of variance (ANOVA) followed by a Dunnett’s multiple comparison test or Bonferroni post hoc comparisons. Analyses of survival were performed using the log-rank (Mantel–Cox) test. All the analyses were performed with the GraphPad Prism 5 program (GraphPad Software, CA, USA). In all cases, differences were considered statistically significant when p < 0.05.\\\\nResults\\\\nα-GalCer Conjugation to fd Filamentous Bacteriophage\\\\nIn order to exploit the ability of the major coat protein pVIII of the filamentous bacteriophage fd to bind hydrophobic lipids, α-GalCer was conjugated to LPS-purified bacteriophage particles (Figure 1A) in a 10:1 ratio (1.5 mg of bacteriophages: 150 µg α-GalCer), followed by ultracentrifugation on cesium chloride gradient to remove unbound lipids.\\\\nFigure 1. fd/alpha-GalactosylCeramide (α-GalCer) conjugate characterization. (A) Schematic image representing filamentous bacteriophage engineered for the expression of antigenic peptide and α-GalCer on the coat surface. (B) UPLC-multiple reaction monitoring profile of the organic extract of a solution containing 1.5 mg/ml of the bacteriophage conjugated with α-GalCer; IS, internal standard. The calibration curve obtained from five calibration points of α-GalCer STD (62.5, 125, 250, 500, and 1,000 ng/ml) spiked with fixed amount of IS (400 ng/ml) is reported along with equation parameters. (C,D) Dendritic cells (DCs) were cultured with free α-GalCer or fd/α-GalCer at different doses. After incubation, cells were stained with anti-α-GalCer:CD1d antibody and analyzed by flow cytometry. (C) Representative histogram overlay of DCs incubated with free α-GalCer or fd/α-GalCer at a dose of 130 ng/ml α-GalCer or fdWT. (D) Percentage of anti-α-GalCer:CD1d positive cells. Mean ± SD of two different experiments is reported. **p < 0.01 by unpaired two-tailed Student’s t-test. (E) The chart represents the IL-2 release of mouse Vα14 invariant natural killer T (iNKT) hybridoma cell line FF13 to α-GalCer delivered by phage particles. LPS-free filamentous bacteriophages were conjugated to α-GalCer, and presented by mouse DCs to stimulate iNKT hybridoma cells. Free form of α-GalCer was used as positive control. Supernatants were diluted 1:10 and assayed in duplicate. Mean ± SD is reported, one representative experiment of four is shown.\\\\nAlpha-GalactosylCeramide rapidly distributed on the pVIII protein thanks to hydrophobic interactions between the hydrophobic domains of the protein and the acyl chains of the lipid. The amount of α-GalCer conjugated to the bacteriophage vector was determined by quantitative mass analysis by a UPLC-MS/MS method. To this aim, the lipid was released from the conjugated vector by solvent extraction and the free glycosphingolipid was measured by multiple reaction monitoring (MRM) analysis. β-galactosylpalmitoylsphingosine was added as IS before extraction. The organic extract obtained as reported in the Section “Materials and Methods,” contained a measured absolute amount of α-GalCer of 0.196 µg/ml, indicating that 2.6% of the added galactosphingolipid was effectively loaded onto the phage particles (Figure 1B).\\\\nfd/α-GalCer Ability to Activate iNKT Cells\\\\nWe assessed the ability of α-GalCer loaded on phage particles to be presented by BMDCs and to activate iNKT cells in vitro. BMDCs were incubated with different doses of free α-GalCer or bacteriophage particles delivering α-GalCer. Cells were stained with a mAb recognizing the α-GalCer:CD1d complex and analyzed by flow cytometry. We found that fd/α-GalCer is able to be internalized by DCs and to efficiently induce α-GalCer presentation on CD1d molecule. In addition, at the dose of 130 ng only the α-GalCer carried by fd was displayed on CD1d molecule (Figures 1C,D).\\\\nMoreover, to investigate the efficiency of the antigen presentation of α-GalCer delivered by phage particles on CD1d, we co-cultured fd/α-GalCer pulsed BMDCs with the mouse Vα14 iNKT hybridoma cell line FF13. Bacteriophage-vectorized α-GalCer was presented by BMDCs, triggering activation of iNKT hybridoma, as assessed by IL-2 release (Figure 1E). According to mass analysis data, it is noteworthy that the highest used dose of bacteriophage (10,000 ng/ml) contains about 26 ng/ml of α-GalCer. The IL-2 production by hybridoma cells was due to the α-GalCer delivered on phage particles as demonstrated by the lack of capacity to induce IL-2 production from BMDCs pulsed with bacteriophage particles not-conjugated to the glycolipid (fdWT).\\\\nWe also tested the in vivo ability of bacteriophage/α-GalCer conjugates to activate a response, by injecting mice intravenously with 50 µg of fd/α-GalCer or 5 µg of free α-GalCer. We found that 3 h after the injection of fd/α-GalCer particles, CD3+NK1.1+ cells were activated, as measured by ex vivo analysis of IFN-γ production (Figure 2A). It is noteworthy that similar response was induced by the injection of the free α-GalCer and fd-conjugated α-GalCer even though the amount of lipid administered with the phage particles was less with respect to the free α-GalCer used. In fact, mass analysis demonstrated that approximately 130 ng of α-GalCer resulted bound to 50 µg of bacteriophages. Moreover, only the injection of vectorized glycosphingolipid was able to significantly increase the number of CD3+NK1.1+ in the spleen, compared to free α-GalCer injection (Figure 2B).\\\\nFigure 2. In vitro recall response of mice to intravenous alpha-GalactosylCeramide (α-GalCer) administration. Mice (n = 3/group) were injected intravenously with 50 µg of fd bacteriophages or 5 µg of free α-GalCer/mouse or vehicle (phosphate buffered saline, PBS). After 3 h, mice were sacrificed, spleen cells were collected and stained with the reported antibodies. (A) Percentage of INF-γ-producing CD3+NK1.1+ cells; average + SEM is reported. p < 0.001 by one-way analysis of variance (ANOVA) followed by a Dunnett’s multiple comparison test. (B) Percentage of CD3+NK1.1+ cells 3 h after intravenous injection of above-mentioned reagents. Average + SEM is reported. p < 0.01 by one-way ANOVA followed by a Dunnett’s multiple comparison test. (C) Percentage of INF-γ-producing CD3−NK1.1+. Average + SEM is reported. p < 0.001 by one-way ANOVA followed by a Dunnett’s multiple comparison test. *p < 0.05, **p < 0.01, ***p < 0.001. Abbreviation: ns, not significant. (D) In vitro recall response of mice to α-GalCer immunization. Mice (n = 3/group) were intravenously injected with 5 µg or 200 ng of free α-GalCer, 50 µg of fd/α-GalCer or vehicle alone (phosphate buffered saline, PBS). The day after, splenocytes were cultured with graded doses of α-GalCer or medium. After 3 days, proliferation was assessed by [3H] thymidine incorporation. Average + SEM is reported. p < 0.001 by two-way ANOVA and the Bonferroni multiple comparison test. *p < 0.05, **p < 0.01, ***p < 0.001.\\\\nImportantly, notwithstanding the lower amount of lipid administered with bacteriophage particles, we found that fd/α-GalCer induced a higher percentage of NK cells able to produce IFN-γ compared to cells isolated from mice injected with free α-GalCer (Figure 2C).\\\\nRecall Response to Intravenous α-GalCer Administration\\\\nIt is known that free α-GalCer administered intravenously to mice, causes TCR down regulation on iNKT, and splenocytes from α-GalCer-injected mice loose their capacity to proliferate and produce cytokines upon in vitro re-stimulation with α-GalCer (13). In agreement with these reports we found that spleen iNKT cells isolated from mice injected with 5 µg of free α-GalCer were unable to proliferate when re-stimulated in vitro with the same lipid (Figure 2D). Since mass experiments showed that the amounts of α-GalCer bound to the phage particles is low, we also used a low dose of free α-GalCer in these experiments and similar unresponsiveness was observed (Figure 2D). In contrast, we observed that splenocytes from mice injected with vectorized α-GalCer, by conjugation to bacteriophages, were still responsive to α-GalCer re-stimulation in vitro in a dose-dependent manner (Figure 2D, empty bars).\\\\nAntigen-Specific Immune Response Is Increased by the Adjuvant Effect of α-GalCer/Bacteriophage Conjugate\\\\nWe then investigated the adjuvant effect of α-GalCer delivered by bacteriophages on the immunogenicity of a displayed antigenic determinant. For this purpose, α-GalCer was conjugated on the surface of fd particles displaying the ovalbumin (OVA) determinant SIINFEKL (OVA257–264).\\\\nIn Figures 3A,B we show that OVA antigen and α-GalCer co-delivered by phage particles are presented by DCs. BMDCs were incubated with fdOVA/α-GalCer conjugate, or with fdOVA alone. As controls, DCs were incubated with free α-GalCer or synthetic OVA257–264 peptide in the same amount as estimated on phage surface (see methods). BMDCs pulsed with fdOVA/α-GalCer are able to stimulate IL-2 production by either OTI hybridoma cells or iNKT hybridoma cells (Figures 3A,B). Of relevance, iNKT hybridoma cells produced higher amount of IL-2 when stimulated by DCs pulsed with fdOVAα–GalCer with respect to free α-GalCer.\\\\nFigure 3. Adjuvant effect of vectorized delivery of alpha-GalactosylCeramide (α-GalCer) on bacteriophages in antigen-specific adaptive immune response. IL-2 release of mouse Vα14 invariant natural killer T hybridoma cell line FF13 (A) or OTI hybridoma cell line (B) in response to α-GalCer or OVA SIINFEKL peptide delivered by phage particles. LPS-free fdOVA filamentous bacteriophages were conjugated to α-GalCer, and fdOVA or fdOVA/α-GalCer were presented by mouse bone marrow-derived dendritic cells to stimulate FF13 or OTI hybridoma cells. Soluble form of α-GalCer was used as positive control in (A). Synthetic OVA257–264 peptide was used as positive control in (B). Supernatants were diluted 1:10 (A) or left undiluted (B) and assayed in duplicate. Mean ± SD is reported, one representative experiment of two is shown. (C,D) Group of mice (n = 4/group) were primed (day 0) and boosted (day 14) with fdOVA (SIINFEKL peptide) bacteriophages delivering or not the α-GalCer as indicated on the x-axis. As control, mice were inoculated twice with vehicle alone (phosphate buffered saline, PBS). At day 21, splenocytes were isolated and percentage of OVA257–264-specific CD8+ T cells producing IFN-γ (C) and percentage of H2Kb-SIINFEKL dextramer positive CD8+ T cells (D) were evaluated. Average + SEM is reported. Difference were statistical significant by one-way analysis of variance followed by a Dunnett’s multiple comparison test. *p < 0.05. Abbreviation: ns, not significant.\\\\nC57BL/6 mice were immunized with bacteriophages expressing OVA257–264 peptide (fdOVA) or with fdOVA delivering α-GalCer (fdOVA/α-GalCer) and after 14 days, mice were boosted with the same bacteriophage particles, delivering or not α-GalCer. As control, mice were inoculated twice with only vehicle (PBS). On day 21, mice were culled and spleen cells were isolated and assayed for OVA257–264-specific cell response.\\\\nWe found that immunization with fdOVA/α-GalCer particles developed an OVA257–264-specific IFN-γ-secreting T cell response. As illustrated in Figure 3, mice that received two fdOVA/α-GalCer administrations, showed increased OVA-specific CD8+ T cells producing IFN-γ compared to mice inoculated twice with fdOVA alone (Figures 3C,D).\\\\nInterestingly, the group of mice injected first with fdOVA and then with fdOVA/α-GalCer, showed similar results to two fdOVA/aGalCer injections, indicating that an adjuvant effect of α-GalCer administration via bacteriophage particles on the adaptive antigen-specific immune response can also be observed after a single administration of fd vectorized α-GalCer. As control, we treated a group of mice first with fdOVA/α-GalCer and then with fdOVA, obtaining comparable results to fdOVA/α-GalCer-treated group, suggesting that the fdOVA/α-GalCer administration either in priming or boosting is able to induce a higher CD8+ response.\\\\nTherapeutic Vaccination With Bacteriophages in B16 Tumor-Bearing Mice\\\\nWe also tested the ability of bacteriophage particles coated with α-GalCer to mediate protection in a therapeutic anti-tumor vaccination setting. For this purpose, C57BL/6 mice were injected subcutaneously with B16 melanoma cells (day −10), and when tumors were palpable (day 0), mice were treated intratumorally with free α-GalCer, fd particles (fdWT), or fd/α-GalCer bacteriophages (Figure 4A). Although intratumoral route of administration is not applicable to all the type of cancer, it has been demonstrated that intratumoral administration of drugs offers several advantages over traditional routes of immunization, as reduced systemic toxicity due to lower diffusion in the body and high local concentration, which permits use of smaller amount of drugs (24).\\\\nFigure 4. Invariant natural killer T response in mice injected intratumorally with fd-vectorized alpha-GalactosylCeramide (α-GalCer). Mice were inoculated with B16 melanoma cells and when tumor was palpable, mice were intratumorally injected with free α-GalCer, α-GalCer delivered by phage particles or vehicle alone, and sacrificed after 24 h. (A) Schematic representation of the experiment schedule. (B) CD3+NK1.1+ cells were evaluated in freshly isolated spleen of injected mice by FACS analysis and staining with anti-NK1.1 and anti-CD3 antibodies. Percentage of NK1.1+CD3+ positive cells ± SD is reported. p < 0.01 by one-way analysis of variance (ANOVA) followed by a Dunnett’s multiple comparison test. **p < 0.01. (C,D) Splenocytes were cultured with graded doses of α-GalCer. After 3 days, proliferation was assessed by [3H] thymidine incorporation (C), and culture supernatants were evaluated for IL-2 release by ELISA (D). Average + SEM is reported. p < 0.001 by two-way ANOVA and the Bonferroni multiple comparison test. *p < 0.05, ***p < 0.001.\\\\nWe found that the intratumoral injection of α-GalCer vectorized on phage particles (fd/α-GalCer) was able to increase the number of iNKT cells, as demonstrated by FACS analysis on spleen cells 24 h later, in comparison with the number of iNKT cells observed in the spleens isolated from mice treated with free α-GalCer (Figure 4B). We also found that spleen cells derived from mice primed in vivo with fd/α-GalCer still retained the capability to proliferate and to produce cytokines like IL-2 when re-stimulated in vitro with increasing doses of free α-GalCer (Figures 4C,D). This response was not observed with cells isolated from mice injected in vivo with free α-GalCer, and also re-stimulated in vitro with free α-GalCer. Moreover, we used for i.t. injection a dose of free α-GalCer (130 ng), normalized according to the amount retained in the phage particles. Similar unresponsiveness to in vitro restimulation was observed (Figure 4D).\\\\nAn important anti-tumor effect was also clearly assessed. fd/α-GalCer treatment resulted in a significant delay in the tumor growth, while administration of free α-GalCer or fd wild-type bacteriophages were less efficient in delaying tumor growth (Figures 5A,B). No significant differences were noted between groups of mice treated with fdWT or free α-GalCer. This protective effect was confirmed by the longer survival of mice receiving fd/α-GalCer (Figure 5C).\\\\nFigure 5. Therapeutic vaccination with filamentous bacteriophages fd/alpha-GalactosylCeramide (α-GalCer) is protective against a subcutaneous tumor challenge. C57BL/6 mice (n = 5/group) were engrafted with B16 melanoma cell line and when tumor was palpable, mice were injected two times subcutaneously into the tumor with phosphate buffered saline, free α-GalCer, fdWT, or fd/α-GalCer. (A) The chart shows the mean tumor size reached in each group. (B) Tumor volumes recorded at 9th day after vaccination are reported as box plot, showing the group median, quartiles, and extreme values. Median values are represented by the horizontal line. *p < 0.05 by one-way ANOVA followed by a Dunnett’s multiple comparison test. (C) Kaplan–Meier survival curves of B16-engrafted mice during the experiment. Differences among survival curves of α-GalCer-treated and fd/α-GalCer-treated mice are statistically different (p < 0.01) by log-rank (Mantel–Cox) test.\\\\nFinally, to investigate the capacity of fd/α-GalCer to induce a tumor-specific adaptive response in a vaccination model, mice were injected with hybrid bacteriophages co-expressing the OVA257–264 SIINFEKL peptide in the presence or absence of conjugated α-GalCer. C57BL/6 mice were engrafted (on day −10), with B16 melanoma cells engineered for the expression of ovalbumin protein. After tumor engraftment, mice received intratumorally fdOVA, fdOVA/α-GalCer, or fd/α-GalCer on day 0. The injection was repeated on day 5 and tumor growth was assessed over time (Figure 6A).\\\\nFigure 6. Filamentous bacteriophage fd co-delivering tumor antigen and alpha-GalactosylCeramide (α-GalCer) induces a stronger anti-tumor response. Mice (n = 5/group) were challenged with B16-OVA expressing melanoma cells. When tumor was palpable (day 0) mice were immunized intratumorally with fd/α-GalCer, fdOVA/α-GalCer, fdOVA, free α-GalCer, or with phosphate buffered saline. The injection was repeated at day 5, and tumor volume was monitored with a caliper over time. (A) Schematic representation of the experiment. (B) Tumor growth recorded during experiment. The chart shows the mean tumor size reached in each group. Line for each group ends once more than half of the group is culled. (C) Box plot of tumor volumes recorded at day 12 post vaccination. Median values are represented by the horizontal line. p < 0.001 by one-way analysis of variance (ANOVA) followed by a Dunnett’s multiple comparison test. ***p < 0.001 (D) Kaplan–Meier survival curves of B16-engrafted mice during the experiment. p < 0.05 between fdOVA/α-GalCer and free α-GalCer-treated mice group by log-rank (Mantel–Cox) test. (E-H) Groups of mice treated as above were sacrificed at day 12 post vaccination and tumor infiltrating lymphocytes were evaluated. Mean percentages (E,F) and absolute numbers (G,H) of CD8+ T cells gated on 7-AAD negative cells (E,G) and of H2Kb-SIINFEKL dextramer positive gated on CD8+ T cells (F,H) are reported. p < 0.01 by one-way ANOVA followed by a Dunnett’s multiple comparison test. *p < 0.05, **p < 0.01. Abbreviation: ns, not significant.\\\\nPriming and boosting mice with fdOVA/α-GalCer efficiently protected mice, as demonstrated by measuring tumor growth (Figures 6B,C). We observed a delay in tumor growth in mice immunized with fdOVA/α-GalCer phage particles, in comparison with mice treated with fdOVA or free α-GalCer.\\\\nMice engrafted with B16-OVA cells after 2 injections with fdOVA/α-GalCer showed 80% survival rate at the end of the experiment, in comparison with 40% survival of mice treated twice with fd/α-GalCer in the absence of the specific OVA257–264 peptide (Figure 6D). In this latter group of mice, a variable response was observed, suggesting the presence of responder and not responder mice (Figure S1 in Supplementary Material). No increased survival was observed in mouse groups treated with two injections of free α-GalCer or fdOVA particles not delivering the galactosylceramide.\\\\nThe analysis of tumor-infiltrating cells at day 12 from intratumor vaccination showed the presence of an increased percentage and absolute number of CD8+ T cells within the tumor bed in mice injected with fdOVA/α-GalCer compared to free α-GalCer and fdOVA-injected groups (Figures 6E,G), while the percentage of OVA257–264 specific T cells, as stained by SIINFEKL-MHC dextramer, was the same in fdOVA or fdOVA/α-GalCer-treated cells, both in tumors and in spleens (Figure 6F; Figure S2 in Supplementary Material). Importantly, the absolute number of tumor infiltrating OVA257–264-specific T cells was higher in mice treated with fdOVA/α-GalCer with respect to mice which received fdOVA (Figure 6H). These data indicate that fdOVA/α-GalCer bacteriophage particles are able to induce a general expansion of tumor-infiltrating T cells, probably due to the activity of α-GalCer, and then, an expansion of OVA257–264-specific cells, which are boosted by the phage particles co-displaying the OVA257–264 antigenic peptide.\\\\nDiscussion\\\\nFilamentous bacteriophage is a well characterized, powerful antigen delivery system that has been demonstrated to evoke long and sustained adaptive immune responses toward the antigens displayed on its surface (17–19). In addition, bacteriophage is able to activate also innate immune responses mainly via toll-like receptor pathways (23, 25).\\\\nTo further improve the immune responses elicited by bacteriophage particles delivering antigenic peptides, in this work we exploited the capacity of the partially hydrophobic pVIII structural protein expressed on the phage surface to interact and bind the immunostimulating glycolipid α-GalCer. We found that the activation of iNKT via fd/α-GalCer, also rapidly induced transactivation of NK cells, which is considered to enhance the anti-tumor effect of α-GalCer treatment (26).\\\\nFurthermore, iNKT cells activated in vivo by fd/α-GalCer remained responsive to repeated stimulations, in contrast to long-term anergy observed upon free α-GalCer injection (13). The reason of iNKT unresponsiveness remains unclear, although it is known that injection of free α-GalCer rapidly induces the overexpression of inhibitory co-stimulatory programmed death (PD)-1 receptor (27). Nevertheless, blockage of PD-1/PD-L1 interaction does not restore full iNKT functions, indicating the presence of additional inhibitory mechanisms.\\\\nCD1d is expressed by many types of APCs which might all stimulate iNKT cells in principle. However, the APCs expressing appropriate levels of costimulatory molecules are those more efficient in inducing full activation of iNKT cells (13, 15, 28, 29). Studies involving conditional depletion of CD1d molecule on selected types of APC have demonstrated that different APCs have different capability to prime iNKT cells to glycolipid-specific responses.\\\\nIn particular, when α-GalCer was presented by B cells a state of unresponsiveness was observed in iNKT cells (13, 30). In contrast, DCs resulted to be very active in stimulating iNKT cells with α-GalCer (31), and anergy could be further avoided by injection of α-GalCer-pulsed autologous DCs (15, 32). However, the dose of the injected lipid may be relevant in the induction of iNKT unresponsiveness by administration of α-GalCer-pulsed DCs (33). In addition, ex vivo manipulation of autologous human DCs appears to be a very expensive and time-consuming procedure.\\\\nFor this reason, vectorizing α-GalCer on the surface of nanoparticles might be an interesting strategy to efficiently deliver the galactosylceramide to DCs in order to decrease the possibility of iNKT anergy. Indeed, nanoparticles are preferentially uptaken by DCs, and α-GalCer-coupled nanoparticles have been described to activate DCs without inducing anergy (34). The filamentous bacteriophage is composed by repeated and ordered subunits and for its particulate nature can be assimilated to a nature-made nanoparticle, and its use prevent unresponsiveness of iNKT.\\\\nImportantly, the simultaneous delivery of glycolipid and a MHC class-I-presented peptide to the same APC enhanced CTL cross priming as compared to separate administration of α-GalCer and protein antigen (35). We confirmed in our model that the co-delivery of tumor peptide and α-GalCer by phage particles increased the induction of antigen-specific CD8+ T cells.\\\\nThe exact mechanism how α-GalCer facilitates priming and expansion of OVA-specific CD8+ T cells remain to be investigated. We envisage two, non-alternative mechanisms. The first is due to the recruitment of iNKT cells next to DCs that internalized fdOVA/α-GalCer. Recruited iNKT cells may immediately induce maturation of DCs and also the optimization of their Ag-presentation and priming capacities of CD8+ cells. These effects might be facilitated also by the large numbers of iNKT cells and their immediate response (within a few minutes) to α-GalCer stimulation. The activation of iNKT cells might explain the reports in which it was found superior to those elicited by TLR agonists in inducing specific anti-tumor CD8+ T cell responses (36).\\\\nThe second mechanism may be ascribed to the fact that bacteriophage, after internalization localizes into the endocytic cellular compartments (23), where loading of glycolipids on CD1d molecule also occurs.\\\\nOverall, our findings suggest that filamentous bacteriophage is a powerful tool to deliver immunogenic lipid molecules together with antigenic peptides.\\\\nGrowing interest is focused on the role of lipids as key stimulatory molecules of the immune response. Either naturally derived non-self lipids, microbial lipids (i.e., peptidoglycan, lipoteichoic acid, and lipopolysaccharides) or self lipids (i.e., phospholipids, sphingomyelin, and oxidized lipoproteins) are known to trigger the immune system (37–41), and co-administration of stimulatory lipid molecules and antigens of interest in vaccination seems a promising manner to formulate novel vaccines including anti-tumor vaccines.\\\\nThe use of filamentous bacteriophages complexed with lipid molecules might represent an efficient, fast, and cheap way to generate new vaccines capable of efficient priming and expansion of T cells and also devoid of chemical adjuvants. Our approach can be further improved by developing phage-based antigen delivery systems loaded with α-GalCer and simultaneously expressing antigens target of the vaccine and molecules delivering the phage to specific cell subsets. Bacteriophages are generally considered safe (42, 43) and cheap to prepare and purify (44, 45), thereby they might represent new tools for the next generation of vaccines.\\\\nEthics Statement\\\\nThis study was carried out in accordance with European Union Laws and guidelines (European Directive 2010/63/EU). The study was approved by our institutional review board and the animal procedures (i.e., immunization, sacrifice) were performed according to rules approved by the ethics committee (permission no. 79/2014-PR).\\\\nAuthor Contributions\\\\nConceptualization: RS and PB. Methodology: RS, AC, and PB. Investigation: RS, LA, PB, DC, LG, and AC. Writing: RS and PB. Review and editing: LA and AC. Funding acquisition: PB. Supervision: PB. All authors read and approved the final version of the manuscript.\\\\nConflict of Interest Statement\\\\nRS and PB are inventors of the patent no. PCT/IB2018/050525 “Phage conjugates and uses thereof.”\\\\nAcknowledgments\\\\nThe IGB-IBP FACS Facility and the IGB “A. Buzzati-Traverso” Animal House Facility at CNR, Naples are gratefully acknowledged. We also thank Gennaro De Libero (Department of Biomedicine, University of Basel and University Hospital Basel, Basel, Switzerland) for providing Vα14 iNKT hybridoma FF13 cells and for helpful scientific advices and for critically reviewing the manuscript and Angelo Fontana [Institute of Biomolecular Chemistry (ICB), CNR, Pozzuoli (NA), Italy] for helpful discussion.\\\\nFunding\\\\nThis work was supported by PO FESR Regione Campania: “Bando realizzazione della rete delle biotecnologie in Campania–progetto TIMING.”\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at https://www.frontiersin.org/articles/10.3389/fimmu.2018.01496/full#supplementary-material.\\\\nReferences\\\\n1. Brennan PJ, Brigl M, Brenner MB. Invariant natural killer T cells: an innate activation scheme linked to diverse effector functions. Nat Rev Immunol (2013) 13(2):101–17. doi:10.1038/nri3369\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n2. Kawano T, Cui J, Koezuka Y, Toura I, Kaneko Y, Motoki K, et al. CD1d-restricted and TCR-mediated activation of valpha14 NKT cells by glycosylceramides. Science (1997) 278(5343):1626–9. doi:10.1126/science.278.5343.1626\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n3. Natori T, Morita M, Akimoto K, Koezuka Y. Agelasphins, novel antitumor and immunostimulatory cerebrosides from the sponge Agelas mauritianus. Tetrahedron (1994) 50:2771. doi:10.1016/S0040-4020(01)86991-X\\\\nCrossRef Full Text | Google Scholar\\\\n4. Morita M, Motoki K, Akimoto K, Natori T, Sakai T, Sawa E, et al. Structure-activity relationship of α-galactosylceramides against B16-bearing mice. J Med Chem (1995) 38:2176. doi:10.1021/jm00012a018\\\\nCrossRef Full Text | Google Scholar\\\\n5. Kobayashi E, Motoki K, Uchida T, Fukushima H, Koezuka Y. KRN7000, a novel immunomodulator, and its antitumor activities. Oncol Res (1995) 7:529.\\\\nPubMed Abstract | Google Scholar\\\\n6. Michel ML, Keller AC, Paget C, Fujio M, Trottein F, Savage PB, et al. Identification of an IL-17-producing NK1.1(neg) iNKT cell population involved in airway neutrophilia. J Exp Med (2007) 204(5):995–1001. doi:10.1084/jem.20061551\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n7. Coquet JM, Kyparissoudis K, Pellicci DG, Besra G, Berzins SP, Smyth MJ, et al. IL-21 is produced by NKT cells and modulates NKT cell activation and cytokine production. J Immunol (2007) 178(5):2827–34. doi:10.4049/jimmunol.178.5.2827\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n8. Paget C, Ivanov S, Fontaine J, Renneson J, Blanc F, Pichavant M, et al. Interleukin-22 is produced by invariant natural killer T lymphocytes during influenza A virus infection: potential role in protection against lung epithelial damages. J Biol Chem (2012) 287(12):8816–29. doi:10.1074/jbc.M111.304758\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n9. Cavallari M, Stallforth P, Kalinichenko A, Rathwell DC, Gronewold TM, Adibekian A, et al. A semisynthetic carbohydrate-lipid vaccine that protects against S. pneumoniae in mice. Nat Chem Biol (2014) 10(11):950–6. doi:10.1038/nchembio.1650\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n10. Nakagawa R, Motoki K, Ueno H, Iijima R, Nakamura H, Kobayashi E, et al. Treatment of hepatic metastasis of the colon26 adenocarcinoma with an alpha-galactosylceramide, KRN7000. Cancer Res (1998) 58(6):1202–7.\\\\nPubMed Abstract | Google Scholar\\\\n11. Hayakawa Y, Rovero S, Forni G, Smyth MJ. Alpha−galactosylceramide (KRN7000) suppression of chemical- and oncogene-dependent carcinogenesis. Proc Natl Acad Sci U S A (2003) 100(16):9464–9. doi:10.1073/pnas.1630663100\\\\nCrossRef Full Text | Google Scholar\\\\n12. Giaccone G, Punt CJ, Ando Y, Ruijter R, Nishi N, Peters M, et al. A phase I study of the natural killer T-cell ligand alpha−galactosylceramide (KRN7000) in patients with solid tumors. Clin Cancer Res (2002) 8(12):3702–9.\\\\nGoogle Scholar\\\\n13. Parekh VV, Wilson MT, Olivares-Villagómez D, Singh AK, Wu L, Wang C-RR, et al. Glycolipid antigen induces long-term natural killer T cell anergy in mice. J Clin Invest (2005) 115:2572–83. doi:10.1172/JCI24762\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n15. Fujii S, Shimizu K, Kronenberg M, Steinman RM. Prolonged IFN-gamma−producing NKT response induced with alpha−galactosylceramide-loaded DCs. Nat Immunol (2002) 3(9):867–74. doi:10.1038/ni827\\\\nCrossRef Full Text | Google Scholar\\\\n16. Faveeuw C, Trottein F. Optimization of natural killer T cell-mediated immunotherapy in cancer using cell-based and nanovector vaccines. Cancer Res (2014) 74(6):1632–8. doi:10.1158/0008-5472.CAN-13-3504\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n17. Sartorius R, Bettua C, D’Apice L, Caivano A, Trovato M, Russo D, et al. Vaccination with filamentous bacteriophages targeting DEC-205 induces DC maturation and potent anti-tumor T-cell responses in the absence of adjuvants. Eur J Immunol (2011) 41(9):2573–84. doi:10.1002/eji.201141526\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n18. De Berardinis P, Sartorius R, Caivano A, Mascolo D, Domingo GJ, Del Pozzo G, et al. Use of fusion proteins and procaryotic display systems for delivery of HIV-1 antigens: development of novel vaccines for HIV-1 infection. Curr HIV Res (2003) 1(4):441–6. doi:10.2174/1570162033485168\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n19. Sartorius R, Pisu P, D’Apice L, Pizzella L, Romano C, Cortese G, et al. The use of filamentous bacteriophage fd to deliver MAGE-A10 or MAGE-A3 HLA−A2-restricted peptides and to induce strong antitumor CTL responses. J Immunol (2008) 180(6):3719–28. doi:10.4049/jimmunol.180.6.3719\\\\nCrossRef Full Text | Google Scholar\\\\n20. Stopar D, Spruijt RB, Wolfs CJ, Hemminga MA. Protein-lipid interactions of bacteriophage M13 major coat protein. Biochim Biophys Acta (2003) 1611:5–15. doi:10.1016/S0005-2736(03)00047-6\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n21. Schümann J, Facciotti F, Panza L, Michieletti M, Compostella F, Collmann A, et al. Differential alteration of lipid antigen presentation to NKT cells due to imbalances in lipid metabolism. Eur J Immunol (2007) 37:1431–41. doi:10.1002/eji.200737160\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n22. Caivano A, D’Apice L, Tiberio C, Prisco A, Acuto O, Guardiola J, et al. Design of cassette vectors permitting cloning of all types of human TCR variable alpha and beta regions. J Immunol Methods (2001) 255:125–34. doi:10.1016/S0022-1759(01)00420-3\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n23. Sartorius R, D’Apice L, Trovato M, Cuccaro F, Costa V, De Leo MG, et al. Antigen delivery by filamentous bacteriophage fd displaying an anti-DEC-205 single-chain variable fragment confers adjuvanticity by triggering a TLR9-mediated immune response. EMBO Mol Med (2015) 7(7):973–88. doi:10.15252/emmm.201404525\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n25. D’Apice L, Costa V, Sartorius R, Trovato M, Aprile M, De Berardinis P. Stimulation of innate and adaptive immunity by using filamentous bacteriophage fd targeted to DEC-205. J Immunol Res (2015) 2015:585078. doi:10.1155/2015/585078\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n26. Smyth MJ, Crowe NY, Pellicci DG, Kyparissoudis K, Kelly JM, Takeda K, et al. Sequential production of interferon-gamma by NK1.1(+) T cells and natural killer cells is essential for the antimetastatic effect of alpha-galactosylceramide. Blood (2002) 99(4):1259–66. doi:10.1182/blood.V99.4.1259\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n27. Parekh VV, Lalani S, Kim S, Halder R, Azuma M, Yagita H, et al. PD-1/PD-L blockade prevents anergy induction and enhances the anti-tumor activities of glycolipid-activated invariant NKT cells. J Immunol (2009) 182(5):2816–26. doi:10.4049/jimmunol.0803648\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n28. Fujii S, Shimizu K, Steinman RM, Dhodapkar MV. Detection and activation of human Valpha24+ natural killer T cells using alpha-galactosyl ceramide-pulsed dendritic cells. J Immunol Methods (2003) 272(1–2):147–59. doi:10.1016/S0022-1759(02)00497-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n29. Bezbradica JS, Stanic AK, Matsuki N, Bour-Jordan H, Bluestone JA, Thomas JW, et al. Distinct roles of dendritic cells and B cells in Va14Ja18 natural T cell activation in vivo. J Immunol (2005) 174(8):4696–705. doi:10.4049/jimmunol.174.8.4696\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n30. Bai L, Constantinides MG, Thomas SY, Reboulet R, Meng F, Koentgen F, et al. Distinct APCs explain the cytokine bias of α-galactosylceramide variants in vivo. J Immunol (2012) 188(7):3053–61. doi:10.4049/jimmunol.1102414\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n31. Chang DH, Osman K, Connolly J, Kukreja A, Krasovsky J, Pack M, et al. Sustained expansion of NKT cells and antigen-specific T cells after injection of alphagalactosyl-ceramide loaded mature dendritic cells in cancer patients. J Exp Med (2005) 201:1503–17. doi:10.1084/jem.20042592 [Erratum in: J Exp Med (2007) 204:2487],\\\\nCrossRef Full Text | Google Scholar\\\\n32. Ishikawa A, Motohashi S, Ishikawa E, Fuchida H, Higashino K, Otsuji M, et al. A phase I study of alpha-galactosylceramide (KRN7000)-pulsed dendritic cells in patients with advanced and recurrent non-small cell lung cancer. Clin Cancer Res (2005) 11(5):1910–7. doi:10.1158/1078-0432.CCR-04-1453\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n33. Iyoda T, Ushida M, Kimura Y, Minamino K, Hayuka A, Yokohata S, et al. Invariant NKT cell anergy is induced by a strong TCR-mediated signal plus co-stimulation. Int Immunol (2010) 22(11):905–13. doi:10.1093/intimm/dxq444\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n34. Thapa P, Zhang G, Xia C, Gelbard A, Overwijk WW, Liu C, et al. Nanoparticle formulated alpha-galactosylceramide activates NKT cells without inducing anergy. Vaccine (2009) 27(25–26):3484–8. doi:10.1016/j.vaccine.2009.01.047\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n35. Semmling V, Lukacs-Kornek V, Thaiss CA, Quast T, Hochheiser K, Panzer U, et al. Alternative cross-priming through CCL17-CCR4-mediated attraction of CTLs toward NKT cell-licensed DCs. Nat Immunol (2010) 11(4):313–20. doi:10.1038/ni.1848\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n36. Dölen Y, Kreutz M, Gileadi U, Tel J, Vasaturo A, van Dinther EA, et al. Co-delivery of PLGA encapsulated invariant NKT cell agonist with antigenic protein induce strong T cell-mediated antitumor immune responses. Oncoimmunology (2015) 5(1):e1068493. doi:10.1080/2162402X.2015.1068493\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n37. Dowds CM, Kornell SC, Blumberg RS, Zeissig S. Lipid antigens in immunity. Biol Chem (2014) 395(1):61–81. doi:10.1515/hsz-2013-0220\\\\nCrossRef Full Text | Google Scholar\\\\n39. Manzo E, Cutignano A, Pagano D, Gallo C, Barra G, Nuzzo G, et al. A new marine-derived sulfoglycolipid triggers dendritic cell activation and immune adjuvant response. Sci Rep (2017) 7(1):6286. doi:10.1038/s41598-017-05969-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n40. Perrin-Cocon L, Diaz O, André P, Lotteau V. Modified lipoproteins provide lipids that modulate dendritic cell immune function. Biochimie (2013) 95(1):103–8. doi:10.1016/j.biochi.2012.08.006\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n41. Zanoni I, Tan Y, Di Gioia M, Broggi A, Ruan J, Shi J, et al. An endogenous caspase-11 ligand elicits interleukin-1 release from living dendritic cells. Science (2016) 352(6290):1232–6. doi:10.1126/science.aaf3036\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\n43. Samoylova TI, Norris MD, Samoylov AM, Cochran AM, Wolfe KG, Petrenko VA, et al. Infective and inactivated filamentous phage as carriers for immunogenic peptides. J Virol Methods (2012) 183(1):63–8. doi:10.1016/j.jviromet.2012.03.032\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02239\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02239/full\",\\n    \"loadedTime\": \"2024-01-23T08:50:58.165Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02239\",\\n    \"title\": \"Frontiers | Prion-Like Domains in Phagobiota\",\\n    \"description\": \"Prions are molecules characterized by self-propagation, which can undergo a conformational switch leading to the creation of new prions. Prion proteins have originally been associated with the development of mammalian pathologies; however, recently they have been shown to contribute to the environmental adaptation in a variety of prokaryotic and eukaryotic organisms. Bacteriophages are widespread and represent the important regulators of microbiota homeostasis and have been shown to be diverse across various bacterial families. Here, we examined whether bacteriophages contain prion-like proteins and whether these prion-like protein domains are involved in the regulation of homeostasis. We used a computational algorithm, prion-like amino acid composition (PLAAC), to detect prion-like domains in 370,617 publicly available bacteriophage protein sequences, which resulted in the identification of 5040 putative prions. We analyzed a set of these prion-like proteins, and observed regularities in their distribution across different phage families, associated with their interactions with the bacterial host cells. We found that prion-like domains could be found across all phages of various groups of bacteria and archeae. The results obtained in this study indicate that bacteriophage prion-like proteins are predominantly involved in the interactions between bacteriophages and bacterial cell, such as those associated with the attachment and penetration of bacteriophage in the cell, and the release of the phage progeny. These data allow the identification of phage prion-like proteins as novel regulators of the interactions between bacteriophages and bacterial cells.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nPrions were demonstrated to play important roles in eukaryotes and prokaryotes, and they have been intensively studied during the last decade (Alberti et al., 2009). Prion proteins (PrPs) are characterized by self-propagation, undergoing a conformational switch from one conformational state to another, which leads to the creation of new prions (Bolton et al., 1982; Telling et al., 1995). Pathologically, prions are characterized by a process in which the infectious form of prion (PrPSc) interacts with the endogenous PrPs, catalyzing the transformation of the endogenous molecule into misfolded PrPSc aggregates (Ma and Lindquist, 2002). This was first observed in mammals, and the pathological deposition of insoluble protein aggregates was shown to be associated with the development of a number of human diseases, including scrapie, Creutzfeldt-Jakob disease, Alzheimer’s disease, Parkinson’s disease, amyloidosis, and other (Collinge, 2001; Wemheuer et al., 2017).\\\\nOne of the best-known forms of PrPs are amyloid proteins that, in addition to the role of infectious agents, were shown to be an important component of many physiological processes in eukaryotes and prokaryotes (Westergard et al., 2007; Sabate et al., 2015; Walker et al., 2016).\\\\nAlthough the amino acid sequences and function of these evolutionarily unrelated proteins are diverse, all amyloid fibrils can be misfolded into distinct conformers, forming highly ordered cross-β structures (Cheng et al., 2012).\\\\nRecently, these highly ordered parallel or antiparallel amyloid β-sheets were identified in yeasts and bacteria, suggesting that they may play an important physiological role (de Groot et al., 2012; Angarica et al., 2013; Malinovska et al., 2013). Unique mechanical and biological properties of amyloid sheets have multiple functions in a variety of bacterial species, and the secretion of amyloid proteins represents an important step during biofilm formation (Chapman et al., 2002; Fowler et al., 2007; Blanco et al., 2012). To discriminate between the physiological and pathological roles of amyloid, these forms that are involved in physiological process have been termed the functional amyloid (Chiti and Dobson, 2006). Therefore, the propagation of prions was shown to enable environmental adaptation, the activation of stress response, and in yeasts these molecules play important roles in the preservation of long-term memory (Shorter and Lindquist, 2005; Suzuki et al., 2012). The aggregation of PrPs is an amino-acid sequence-specific process, but the mechanism underlying the de novo appearance of prions remains unclear. PrPs are rich in asparagine (Q) and glutamine (N) domains, which was exploited for the development of an algorithm aimed at the identification of candidate prionogenic domains (PrDs) (Michelitsch and Weissman, 2000; Alberti et al., 2009). Currently, the most widespread PrD prediction algorithm is based on the hidden Markov model (HMM), the probabilistic sequence model based on the maximum likelihood estimation (Eddy, 1998; Angarica et al., 2013).\\\\nThe HMM can be found at the core of several bioinformatic approaches to the statistical representation of prion domains, which allow the scoring of protein sequences according to the likelihood that these proteins are prions (Toombs et al., 2010; Lancaster et al., 2014). To that end, the log-likelihood ratio (LLR) score is used to evaluate the sets of amino-acid interactions, with a large LLR indicating a high similarity between the examined interaction sets, whereas the LLR is nearing zero when comparing the sets of random interactions. Using these web-based algorithms, prion-forming domains were extensively studied in mammalian, bacterial, and fungal proteomes, demonstrating that prions are common in different organisms, and represent important global regulators (Iglesias et al., 2015; Yuan and Hochschild, 2017).\\\\nIn prokaryotes, PrPs were shown to play important roles in molecular transport, secretion, cell wall development, and other processes (Blanco et al., 2012; Yuan et al., 2014). Moreover, prion determinants were shown to participate in the inter-kingdom communication between prokaryotes and eukaryotes, leading to the alterations in the amyloidogenesis in Caenorhabditis elegans following its colonization with amyloid-producing bacteria (Chen et al., 2016).\\\\nHowever, the prion-like domains in viruses, in particular the bacterial viruses, bacteriophages, have not been thoroughly examined. Bacteriophages are ubiquitous organisms that are found almost everywhere, from the oceans, with the estimated 1031 viral particles, to mammalians, with over 1015 bacteriophages in the adult human gut (Dupuis et al., 2013; Dalmasso et al., 2014). The life cycle of bacteriophages can be lytic, resulting in bacterial death and de novo assembly of virions, or lysogenic, in which bacteriophage DNA is integrated into the bacterial genome. Both bacteriophage life cycles are important in the regulation of bacterial population. Phage-mediated horizontal gene transfer, occurring during the lysogenic cycle, is especially important for the bacterial ecological adaptation and is a common mechanism underlying the spreading of antibiotic resistance (Ochman et al., 2000). Moreover, we previously demonstrated that the bacteriophages, by altering mammalian gut flora, can induce an increase in gut permeability and facilitate endotoxemia, indirectly affecting the vital activities of microorganisms (Tetz and Tetz, 2016; Tetz et al., 2017). Previous studies revealed a correlation between the increased gut permeability and the development of a variety of human pathologies, including diabetes, neurodegenerative, and cardiovascular diseases. Therefore, the role of bacteriophages as the potential indirect causative agent inducing the development of these diseases requires further investigation (Bischoff et al., 2014).\\\\nHere, we studied in detail the putative prion domains in bacteriophages. We employed HHM algorithm for the analysis of bacteriophage proteomes, using the UniProtKB database. To the best of our knowledge, this is the most extensive effort aimed at the identification of candidate PrD sequences among bacteriophages. Furthermore, we analyzed the trends in the distribution of PrDs in different protein families and the correlations between bacteriophage families and the host bacterial host. The present predictive approach for the first time uncovers a large set of putative prionogenic proteins whose further experimental characterization might contribute significantly to understanding bacteriophages biology.\\\\nMaterials and Methods\\\\nProtein Sequences\\\\nTo identify the PrDs present in bacteriophage proteomes, protein sequences were obtained from the UniProt KnowledgeBase (Swiss-Prot and TrEMBL) (UniProt Consortium, 2012).\\\\nThe protein functions were manually curated using the information from the UniProt database, the National Center for Biotechnology Information (NCBI) protein sequence database1, and the literature.\\\\nIdentification of PrDs in Bacteriophage Proteomes\\\\nThe presence of PrDs was analyzed in all bacteriophage proteins using the prion-like amino acid composition (PLAAC) prion prediction program based on the HMM and trained using the known PrDs, by identifying the compositional bias toward N and Q residues. By employing a cutoff of 0.003 LLR, we identified 5040 PrDs (Supplementary Table S1). The enrichment values obtained for the proteins detected in the bacteriophage family subsets were compared with the total number of analyzed proteins.\\\\nWe analyzed the regularities in the likelihood of the identified PrDs to be prions, their distribution among bacteriophage families, host bacteria, and protein functions. The obtained data were used to generate a heatmap with R-statistical computing2, using the “levelplot” package. The color key indicates a range between the lowest (white) and the highest (red) LLR values.\\\\nStatistical Analyses\\\\nAll statistical analyses were performed using the statistics package Statistica for Windows (version 5.0). Data were compared between bacteriophage families by using a χ2 test or the Fisher’s exact test. Significant differences were calculated using one-way analysis of variance (ANOVA) with multiple comparisons and a standard confidence interval of 95%. A value of p < 0.001 was considered statistically significant in all tests. Correlations between the size of bacteriophage proteome and PrD enrichment were calculated using the Spearman’s rank correlation coefficient test.\\\\nResults\\\\nIdentification of PrDs in Bacteriophage Proteomes\\\\nTo identify the PrDs in bacteriophages using the PLAAC algorithm we analyzed 2111 bacteriophage proteomes comprising a total of 370,617 proteins retrieved from the UniProtKB database. We detected 5040 PrPs with the LLRs higher than 0.003 (1.35% of the total bacteriophage protein dataset) (Lancaster et al., 2014).\\\\nIn Table 1, the trends in the PrD distribution across different bacteriophage families are presented. PrD frequency ranged from 1.16 to 4.49% when analyzing bacteriophage families with at least 400 sequenced proteins. Due to the low representation of archaeal phages, it was not possible to compare them with the bacteriophage proteins with sufficient reliability. Therefore, we focused on the analysis of the PrDs in bacteriophages. Within Caudovirales, the most well-studied order of phages, the highest numbers of proteins containing the PrDs when analyzing bacteriophage families with more than 10,000 sequenced proteins were found among Podoviridae, with over 1.74% of the total Podoviridae proteins were shown to harbor PrDs (p < 0.001), constituting 4.21% of the total detected PrDs among Caudovirales (Table 1 and Supplementary Table S1).\\\\nTABLE 1. Summary of the prion predicted in different bacteriophage families.\\\\nThe analysis shown in Supplementary Table S1 demonstrates an interesting trend of representation of prionogenic domains among different bacteriophages families and host organisms.\\\\nNext, we determined the patterns of correlation between bacteriophage families and LLR scores (Figure 1 and Supplementary Table S2).\\\\nFIGURE 1. Log-likelihood ratio (LLR) score showing the predicted putative PrDs across bacteriophage families. Results were analyzed using one-way ANOVA.\\\\nThe average numbers of LLRs between three most abundant bacteriophage families, Myoviridae, Podoviridae, and Siphoviridae, were similar, with the highest LLR scores obtained for Podoviridae family, with an average LLR of 7.78 (p < 0.001) (Supplementary Tables S3A–C).\\\\nUnder the same conditions, we compared the LLRs between the remaining bacteriophage families, and Myoviridae were shown to have the highest number of PrDs, with the LLR score over 30. We identified 114 PrDs with the medium LLR score over 20 within Myoviridae family, yielding 3.29% of the total number of PrDs identified in this bacteriophage family (Supplementary Figure S1).\\\\nThe same trend was observed when analyzing top 100 scoring PrDs. Over 60% of these 100 PrDs were derived from Myoviridae bacteriophages, with the medium LLR score of 40.55 (Supplementary Table S4).\\\\nAfterward, we aimed to select bacteriophage proteins in bacteria that may have a greater prion-forming potential. To this end, we analyzed the correlation between LLR score of the PrD proteins and the host bacteria. The average LLR score varied significantly between the bacteriophages of different bacterial species. The lowest mean LLR values were detected among Cellulophaga and Escherichia bacteriophages (3.49 and 3.98, respectively) and the highest LLR score was obtained for the Bacillus bacteriophages, with the average LLR of 8.96 (p < 0.001).\\\\nNo direct correlation was observed between the functions of proteins containing PrDs and the LLR score. One of both highest and lowest LLR scores were obtained for the single-stranded DNA binding protein of Listeria bacteriophage LMSP-25 (LLR score, 45.89; Myoviridae) and the Streptococcus bacteriophage 5093 (LLR 1.97; Siphoviridae), respectively (Figures 2A,B).\\\\nFIGURE 2. Characterization of the candidate bacteriophage PrDs in the single-stranded DNA binding proteins. (A)Graphical representation of the highest LLR value in this group of proteins identified in the Listeria phage LMSP-25. (B) Graphical representation of the lowest LLR value in this group of proteins in the Streptococcus phage 5093. The predicted PrDs are highlighted in red.\\\\nAdditionally, we analyzed the proteome enrichment with PrDs in the bacteriophages of different bacteria. The highest enrichment rate, with at least five PrDs per proteome, was found among the bacteriophages of Bacillus, Cronobacter, Lactobacillus, Synechococcus, Staphylococcus, and other bacteria (Figure 4 and Supplementary Table S5). The majority of these bacteriophages containing more than five PrDs per one proteome was shown to belong to the Myoviridae family. Since the members of this family have larger proteomes compared with those of other bacteriophage families, we analyzed and obtained a direct correlation between the number of PrDs per bacteriophage and the proteome (p < 0.05), but no correlations were observed within any bacteriophage family (Supplementary Table S6). Notably, the particular enrichment with PrDs was found across the majority of Prochlorococcus and Synechococcus cyanophages identified as PrD carriers in this study.\\\\nStructural Domains Associated with to Bacteriophage PrDs\\\\nTo analyze the functions of bacteriophage proteins with PrDs, we clustered the identified domains into five groups, according to the major steps during the process of bacteriophage interactions with its host cell (Marvin, 1998). The proteins were grouped based on their functions during the interactions with the bacterial host cells into the following groups: attachment and penetration, replication of bacteriophage DNA and protein synthesis, assembly, release, and unknown functions (Laemmli, 1970; Zylicz et al., 1989; Marvin, 1998; Rakhuba et al., 2010; Aksyuk and Rossmann, 2011). The obtained correlations between the LLR scores, bacterial and bacteriophage families, and protein functions were analyzed and are displayed as a heatmap in Figure 3.\\\\nFIGURE 3. Heatmap showing PrD distribution in bacteriophages. The correlation between LLR score of the identified PrDs, their distribution across bacteriophage families, host bacteria, and their function in the bacteriophage-bacterial interaction are presented. The likelihood that an identified PrD is a prion is represented by a color scale, from white to red (maximum).\\\\nThe most abundant (2046 PrPs; medium LLR score 4.86) functional group was shown to be the attachment and penetration group, which included the proteins associated with the host cell binding and genome injection. We identified receptor binding proteins, tail fiber proteins, peptidase_M23 baseplate wedge components, lysins, and other proteins known to have crucial functions in the various aspects of phage infection, including adsorption, peptidoglycan hydrolysis, cell wall penetration, and DNA ejection, but the PrDs were predominantly found among tail component, tail fiber, and baseplate proteins (Rodríguez-Rubio et al., 2012; Stockdale et al., 2013). Furthermore, we identified a PrD in gp98/99 proteins that form the cell-puncturing device of Mycobacterium phages. Some of the identified proteins associated with surface structures, such as tail tube cap, are not involved in the direct contact with the bacterial cell, but they represent important structural components allowing the injection of DNA (Arisaka et al., 2003; Orlova, 2012). Heatmap analysis results showed that Podoviridae belonging to the Mycobacteriaceae family and Myoviridae of Listeriaceae have the highest LLR score (Figure 3).\\\\nThe second largest group was shown to contain 390 PrDs (medium LLR score, 7.93) involved in the replication of phage DNA and protein synthesis. We identified several Erf proteins, essential for bacteriophage DNA recombination, proteins belonging to glycosyltransferase family protecting bacteriophage DNA from degradation, and single-strand binding proteins (Markine-Goriaynoff et al., 2004). Other identified proteins included those involved in the regulation of host cell metabolism, such as serine/threonine phosphatases (Howard-Varona et al., 2017). The highest average LLR score was obtained for the members of Myoviridae family of Moraxellaceae and Listeriaceae (Figure 3).\\\\nFollowing this, we analyzed proteins with PrDs that are associated with the bacteriophage assembly in the host cells (170 proteins; medium LLR score, 6.16). These proteins were identified in all major families of Caudovirales. The highest average LLR score was obtained for the proteins of Myoviridae family of Moraxellaceae (Figure 3). These proteins are also associated with the organization and maturation of bacteriophages, and those involved in the tail assembly and DNA packaging (Linderoth et al., 1994; Boudko et al., 2002; Aksyuk and Rossmann, 2011).\\\\nThe identified PrDs in proteins that are involved in cell wall lysis and the release of bacteriophages from the host cells were less abundant and this group contained only 105 proteins (medium LLR score, 5.13). This group predominantly comprised lysins, including those with the amidase activity and proteins with peptidoglycan binding functions (hydrolases, D-ala-D-ala carboxypeptidase, endopeptidase CHAP domain protein, and others), which are involved in bacterial cell wall degradation, resulting in cell lysis and release of progeny viruses. They were predominantly detected among Siphoviridae infecting Streptococcaceae and Mycobacteriaceae. The highest average LLR score was obtained in Myoviridae family of Moraxellaceae (Figure 3).\\\\nFinally, we identified 2329 PrDs among the proteins with the still unknown functions. Among these proteins in Staphylococcus bacteriophages belonging to the Siphoviridae family, we identified Panton-Valentine leukocin (LukS-PV), a cytotoxin associated with the increased virulence of Staphylococcus aureus, which can induce a considerable tissue damage (Vandenesch et al., 2003). This is in agreement with previous observations that PrDs in viruses play important roles in the determination of virulence potential (Iglesias et al., 2015).\\\\nDiscussion\\\\nTo the best of our knowledge, this is the first analysis of PrDs among bacteriophages. The results of our study underline the necessity of characterizing bacteriophages and we propose here that the all bacteriophage genes should be known as the phagobiome.\\\\nHere, PrDs were more frequently found among archaeal phages, and the analysis of the three best-known Caudovirales families showed that these domains are more abundant among Podoviridae than Myoviridae or Siphoviridae.\\\\nFurthermore, our analyses demonstrated that approximately 50% of all sequenced bacteriophages available in public databases contain at least one PrD. The majority of bacteriophages contain less than five PrDs per proteome. Surprisingly, the vast majority of bacteriophages with more than five PrDs per proteome were shown to belong to the Myoviridae family. Since the total numbers of PrDs found in Myoviridae (2425) and Siphoviridae (1665) are of the same order of magnitude, these further supports the enrichment of PrDs in the Myoviridae proteomes. However, Myoviridae members have larger proteomes than the members of Siphoviridae family, which may explain the observed differences in the PrD enrichment (Sandaa, 2009). Further analyses indicated a direct correlation between the proteome size and the PrD enrichment, and therefore, we further explored whether the proteome size of bacteriophages in the Myoviridae family is related to the PrD enrichment. However, we have not observed a linear correlation between these parameters, suggesting that the PrD enrichment does not depend only on the size of a Myoviridae proteome.\\\\nMoreover, the majority of the top 100 scoring PrDs was also detected in the members of Myoviridae family, and among these, several short protein sequences were found to harbor a higher number of Q-rich domains than the longer ones. This indicates that the scores obtained in the analysis of protein sequences according to their prion-like characteristics do not correlate only with the size of a protein. However, the functional relevance of these findings remains unclear, since the majority of these proteins have not been characterized. The analysis of proteins with the known functions showed that the proteins involved in the replication of bacteriophage DNA and protein synthesis have the largest LLR scores.\\\\nMany PrD-containing proteins with known functions were identified among Bacillus phages that infect the spore-forming Bacillus spp. The identified PrDs were found in proteins responsible for attachment and penetration, replication of bacteriophage DNA, and release within the same Bacillus phages. Furthermore, the LLR scores of PrDs in the Bacillus phages were shown to have the highest meaning of the prion likelihood ratio compared with the phages associated with other bacterial genera. Previously, we reported a particular role of spore-forming bacteria in the global microbiota, which allowed their identification as the members of sporobiota (Tetz and Tetz, 2017). Taken together, previous results and the ones obtained in this study may suggest that the global distribution of sporobiota members and Bacillus spp., and their increased transfer between the ecological niches is associated with the presence of resistant transmissible spores due to their interactions with bacteriophages (von Wintersdorff et al., 2016).\\\\nHowever, the highest numbers of PrDs per proteome were found across Prochlorococcus and Synechococcus phages, whose hosts are represented by evolutionary ancient Cyanobacteria (Figure 4) (Schirrmeister et al., 2011). This finding along with the higher prevalence of PrDs in archaea suggests the possible association of the presence of PrDs and “evolutionary age,” however, this requires an independent analysis. Regardless, the enrichment of PrDs in the most abundant phages on Earth infecting marine Prochlorococcus and Synechococcus suggests the possible particular role of these elements (Sullivan et al., 2010).\\\\nFIGURE 4. Proteome enrichment for PrDs across bacteriophages. Bubble charts represent the medium enrichment rate with PrDs per bacteriophage proteome (phages with at least five PrDs per proteome have been considered) and the number of phages in which these PrDs were identified. The values in the bubbles are the median number of PrDs occurring per proteome in each bacteriophage group. Bigger bubble size indicates higher number of phages with at least five PrDs per proteome, and smaller size indicates lower number of phages with at least five PrDs per proteome.\\\\nTo determine the potential correlations between the functions of PrD-containing proteins, bacteriophage families, and host organisms, we generated a heatmap that showed that the highest LLRs were found among Myoviridae and Siphoviridae members of Moraxellaceae, Listeriaceae, and Pseudoalteromonadaceae, which were shown to be associated with replication of phage DNA, protein synthesis, and bacteriophage assembly. However, these data do not correspond to the frequency of PrD in bacteriophage proteomes. A vast majority of the bacteriophage PrD-containing proteins are involved in the interactions between a bacteriophage and bacterial cell wall, their attachment and penetration, and the release of progeny phages.\\\\nAttachment and penetration represent crucial steps in the bacteriophage–bacterium interaction, since these processes allow the entry of the bacteriophage DNA inside the host. We identified PrDs in the proteins forming different structural components of bacteriophages, including head, neck, sheath, baseplate and fibers, for tailed viruses, receptor-binding proteins for tailless Tectiviridae, and in the coat proteins III and IV of the filamentous Inoviridae (Hulo et al., 2017). This indicates the role of the identified PrDs in the formation of surface structures and specific and non-specific attachments to bacterial cells (Spinelli et al., 2014). Moreover, a similar trend in the distribution of prion-like domains in proteins responsible for interactions and virulence was defined by Gene Ontology GO terms across bacteria (Iglesias et al., 2015). Notably, PrDs were common across different bacterial and bacteriophage families, indicating the universal regulatory role of these putative prion domains (Iglesias et al., 2015).\\\\nFurthermore, since we identified PrDs in the interacting proteins in different bacteriophage families infecting a variety of bacterial hosts, this implies that these PrDs are conserved across species. However, since the PrDs in the surface proteins were not identified in all analyzed bacteriophages, and in those where they were identified, PrDs were found in some, but not all surface proteins of the given phage, this suggests that the presence of PrD-containing proteins may be beneficial, but it is not obligatory. The attachment of bacteriophages is a complex process, accompanied by a number of bacteriophage-resistance responses initiated in a bacterial cell (Labrie et al., 2010). Our findings may indicate that the proteins with the PrDs can alter their conformational states, adding further complexity and specificity in the phage-bacterial interactions, and this may serve as a mechanism to overcome some bacterial resistance mechanisms. Moreover, it can be speculated that the PrDs in the interacting proteins may be involved not only in the interactions with host bacteria, but also in the interactions of bacteriophages with other components of microbial communities. These observations agree with our recent studies showing that the oral administration of bacteriophages induces an increase in the intestinal permeability and endotoxemia in mammals, significantly changing microbiota composition, beyond the possible direct effect of bacteriophages, and because of this, it was suggested that these effects are exerted through yet unknown mechanisms (Tetz and Tetz, 2016; Tetz et al., 2017).\\\\nAdditionally, the results obtained in this study suggest that the presence of PrDs in bacteriophages may be associated with a number of mammalian diseases, in which the role of prions was demonstrated both in macroorganisms and in gut bacteria. Thus in the article by Chen et al. (2016), C. elegans fed with the prion-producing Escherichia coli was shown to have an enhanced prion aggregation in brain. It can be assumed that a similar process may be observed following the introduction of bacteriophages and that bacteriophages contribute to the observed processes.\\\\nProteins that are involved in the bacterial cell wall interactions and the release of progeny phages from the host were shown to harbor PrDs as well. The release stage involves bacterial cell wall degradation from within the cells with bacteriophage-encoded peptidoglycan hydrolases, synthesized at the end of the multiplication cycle. We identified PrDs in phage-encoded endolysins, and the majority of these proteins was found in Siphoviridae family, with only a few detected among the members of Podoviridae and Myoviridae.\\\\nSome of the proteins enriched in the PrDs are involved in the replication of bacteriophage DNA and protein synthesis. In eukaryotes, PrD-containing proteins were shown to play a role in the interactions and binding with the nucleic acids (King et al., 2012). Nucleic acids were shown to play a crucial role in the PrP conversion into PrPSc-like species as cofactors (Macedo and Cordeiro, 2017).\\\\nIn bacteriophages, we determined that the presence of PrDs can also be associated with the single-strand binding proteins, known as essential components of bacteriophage DNA synthesis. Therefore, this indicates that these PrDs can be easily misfolded into PrPSc through the interactions with the DNA, which was shown to be an important component inducing this misfolding (Cordeiro et al., 2001). This type of interaction with the host cell may represent an additional process through which the bacteriophages control bacterial intracellular functions (Wagner and Waldor, 2002; Davies and Davies, 2010).\\\\nBacteriophages are important vehicles facilitating the genetic exchange between microorganisms, including the spreading of virulence factors and the antibiotic resistant genes (Lina et al., 1999). Here, we identified a PrD in the LukS-PV protein of some Siphoviridae members and unclassified phages. This cytotoxin can be found among methicillin-resistant S. aureus and belongs to the group of β-pore-forming toxins (Vandenesch et al., 2003). The presence of LukS-PV is associated with the increased virulence of S. aureus, leading to abscess formation and severe necrotizing pneumonia. The detected PrDs located in LukS-PV indicates the role of these domains in the virulence potential of the cytotoxins and their role in the pathogenicity. Our results may indicate that the presence of PrDs in LukS-PV affects its interactions with the membranes of the targeted eukaryotic cells and induce the pore-forming potential, which is supported by previous studies (Iglesias et al., 2015).\\\\nTaken together, we identified numerous putative PrD-containing proteins in bacteriophages. We observed consistent patterns of the distribution of PrDs across different bacteriophage families. However, since the infectious agents of some bacteria were shown to lack PrDs, this may indicate that the diversity of phagobiota and phogobiome is underestimated. Although bacteriophage genomes are significantly smaller than bacterial genomes, currently less than 2,500 phage genomes are deposited in the NCBI database, compared with almost 90,000 bacterial whole genome sequences.\\\\nAlthough several trends in the frequency of PrDs across bacteriophage families have been detected, future studies are required (Pourcel et al., 2017), to elucidate further the functions and presence of PrP-containing proteins in bacteriophages. The predictive approach employed here revealed for the first time a large set of putative PrPs, and further experimental characterization of these proteins may contribute to the understanding of bacteriophage biology.\\\\nAuthor Contributions\\\\nGT and VT designed and conducted the experiments and analyzed data, GT wrote the manuscript.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2017.02239/full#supplementary-material\\\\nTABLE S1 | LLR score showing the predicted putative PrDs across bacteriophages.\\\\nTABLE S2 | Summary of the LLR score of prion predictions across different phage families.\\\\nTABLE S3 | Least squares means for effect of bacteriophage family on LLR score.\\\\nTABLE S4 | Top 100 scoring PrDs across bacteriophages.\\\\nTABLE S5 | Proteome enrichment for PrDs across bacteriophages.\\\\nTABLE S6 | Correlation between the number of PrD per phage and the proteome size.\\\\nFootnotes\\\\n^ http://www.ncbi.nlm.nih.gov/\\\\n^ www.r-project.org\\\\nReferences\\\\nAlberti, S., Halfmann, R., King, O., Kapila, A., and Lindquist, S. (2009). A systematic survey identifies prions and illuminates sequence features of prionogenic proteins. Cell 137, 146–158. doi: 10.1016/j.cell.2009.02.044\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nAngarica, V. E., Ventura, S., and Sancho, J. (2013). Discovering putative prion sequences in complete proteomes using probabilistic representations of Q/N-rich domains. BMC Genomics 14:316. doi: 10.1186/1471-2164-14-316\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nArisaka, F., Kanamaru, S., Leiman, P., and Rossmann, M. G. (2003). The tail lysozyme complex of bacteriophage T4. Int. J. Biochem. Cell Biol. 35, 16–21. doi: 10.1016/S1357-2725(02)00098-5\\\\nCrossRef Full Text | Google Scholar\\\\nBischoff, S. C., Barbara, G., Buurman, W., Ockhuizen, T., Schulzke, J. D., Serino, M., et al. (2014). Intestinal permeability–a new target for disease prevention and therapy. BMC Gastroenterol. 14:189. doi: 10.1186/s12876-014-0189-7\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBlanco, L. P., Evans, M. L., Smith, D. R., Badtke, M. P., and Chapman, M. R. (2012). Diversity, biogenesis and function of microbial amyloids. Trends microbial. 20, 66–73. doi: 10.1016/j.tim.2011.11.005\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nBoudko, S. P., Londer, Y. Y., Letarov, A. V., Sernova, N. V., Engel, J., and Mesyanzhinov, V. V. (2002). Domain organization, folding and stability of bacteriophage T4 fibritin, a segmented coiled-coil protein. FEBS J. 269, 833–841. doi: 10.1046/j.1432-1033.2002.02734.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChapman, M. R., Robinson, L. S., Pinkner, J. S., Roth, R., Heuser, J., Hammar, M., et al. (2002). Role of E. coli curli operons in directing amyloid fiber formation. Science 295, 851–855. doi: 10.1126/science.1067484\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChen, S. G., Stribinskis, V., Rane, M. J., Demuth, D. R., Gozal, E., Roberts, A. M., et al. (2016). Exposure to the functional bacterial amyloid protein curli enhances alpha-synuclein aggregation in aged fischer 344 rats and Caenorhabditis elegans. Sci. Rep. 6:34477. doi: 10.1038/srep34477\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCheng, P. N., Liu, C., Zhao, M., Eisenberg, D., and Nowick, J. S. (2012). Amyloid β-sheet mimics that antagonize protein aggregation and reduce amyloid toxicity. Nat. chem. 4, 927–933. doi: 10.1038/nchem.1433\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChiti, F., and Dobson, C. M. (2006). Protein misfolding, functional amyloid, and human disease. Annu. Rev. Biochem. 75, 333–366. doi: 10.1146/annurev.biochem.75.101304.123901\\\\nCrossRef Full Text | Google Scholar\\\\nCollinge, J. (2001). Prion diseases of humans and animals: their causes and molecular basis. Annu. Rev. Neurosci. 24, 519–550. doi: 10.1146/annurev.neuro.24.1.519\\\\nCrossRef Full Text | Google Scholar\\\\nCordeiro, Y., Machado, F., Juliano, L., Juliano, M. A., Brentani, R. R., Foguel, D., et al. (2001). DNA converts cellular prion protein into the β-sheet conformation and inhibits prion peptide aggregation. J. Biol. Chem. 276, 49400–49409. doi: 10.1074/jbc.M106707200\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nde Groot, N. S., Torrent, M., Villar-Piqué, A., Lang, B., Ventura, S., Gsponer, J., et al. (2012). Evolutionary selection for protein aggregation. Biochem. Soc. Trans. 40, 1032–1037. doi: 10.1042/BST20120160\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nDupuis, M. È., Villion, M., Magadán, A. H., and Moineau, S. (2013). CRISPR-Cas and restriction-modification systems are compatible and increase phage resistance. Nat. Commun. 4:2087. doi: 10.1038/ncomms3087\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHoward-Varona, C., Roux, S., Dore, H., Solonenko, N. E., Holmfeldt, K., Markillie, L. M., et al. (2017). Regulation of infection efficiency in a globally abundant marine Bacteriodetes virus. ISME J. 11, 284–285. doi: 10.1038/ismej.2016.81\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHulo, C., Masson, P., Toussaint, A., Osumi-Sutherland, D., de Castro, E., Auchincloss, A. H., et al. (2017). Bacterial virus ontology; coordinating across databases. Viruses 9:126. doi: 10.3390/v9060126\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nIglesias, V., de Groot, N. S., and Ventura, S. (2015). Computational analysis of candidate prion-like proteins in bacteria and their role. Front. Microbiol. 6:1123. doi: 10.3389/fmicb.2015.01123\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKing, O. D., Gitler, A. D., and Shorter, J. (2012). The tip of the iceberg: RNA-binding proteins with prion-like domains in neurodegenerative disease. Brain Res. 1462, 61–80. doi: 10.1016/j.brainres.2012.01.016\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLaemmli, U. K. (1970). Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227, 680–685. doi: 10.1038/227680a0\\\\nCrossRef Full Text | Google Scholar\\\\nLancaster, A. K., Nutter-Upham, A., Lindquist, S., and King, O. D. (2014). PLAAC: a web and command-line application to identify proteins with prion-like amino acid composition. Bioinformatics 30, 2501–2502. doi: 10.1093/bioinformatics/btu310\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLina, G., Piemont, Y., Godail-Gamot, F., Bes, M., Peter, M., Gauduchon, V., et al. (1999). Involvement of panton-valentine leukocidin-producing Staphylococcus aureus in primary skin infections and pneumonia. Clin. Infect. Dis. 29, 1128–1132. doi: 10.1086/313461\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLinderoth, N. A., Julien, B., Flick, K. E., Calendar, R., and Christle, G. E. (1994). Molecular cloning and characterization of bacteriophage P2 genes R and S involved in tail completion. Virology 200, 347–359. doi: 10.1006/viro.1994.1199\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMalinovska, L., Kroschwald, S., and Alberti, S. (2013). Protein disorder, prion propensities, and self-organizing macromolecular collectives. Biochim. Biophys. Acta 1834, 918–931. doi: 10.1016/j.bbapap.2013.01.003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMarkine-Goriaynoff, N., Gillet, L., Van Etten, J. L., Korres, H., Verma, N., and Vanderplasschen, A. (2004). Glycosyltransferases encoded by viruses. J. Gen. Virol. 85, 2741–2754. doi: 10.1099/vir.0.80320-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMarvin, D. A. (1998). Filamentous phage structure, infection and assembly. Curr. Opin. Struct. Biol. 8, 150–158. doi: 10.1016/S0959-440X(98)80032-8\\\\nCrossRef Full Text | Google Scholar\\\\nMichelitsch, M. D., and Weissman, J. S. (2000). A census of glutamine/asparagine-rich regions: implications for their conserved function and the prediction of novel prions. Proc. Natl. Acad. Sci. U.S.A. 97, 11910–11915. doi: 10.1073/pnas.97.22.11910\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPourcel, C., Midoux, C., Vergnaud, G., and Latino, L. (2017). A carrier state is established in Pseudomonas aeruginosa by phage LeviOr01, a newly isolated ssRNA levivirus. J. Gen. Virol. 98, 2181–2189. doi: 10.1099/jgv.0.000883\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRakhuba, D. V., Kolomiets, E. I., Dey, E. S., and Novik, G. I. (2010). Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell. Pol. J. Microbiol. 59, 145–155.\\\\nPubMed Abstract | Google Scholar\\\\nRodríguez-Rubio, L., Gutiérrez, D., Martínez, B., Rodríguez, A., Götz, F., and García, P. (2012). The tape measure protein of the Staphylococcus aureus bacteriophage vB_SauS-phiIPLA35 has an active muramidase domain. Appl. Environ. Microbiol. 78, 6369–6371. doi: 10.1128/AEM.01236-12\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSandaa, R. A. (2009). “Viral pool, environmental,” in Encyclopedia in Microbiology, 3rd Edn, ed. M. Schaechter (Cambridge: Elsevier Ltd), 553–567.\\\\nSpinelli, S., Veesler, D., Bebeacua, C., and Cambillau, C. (2014). Structures and host-adhesion mechanisms of lactococcal siphophages. Front. Microb. 5:3. doi: 10.3389/fmicb.2014.00003\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nStockdale, S. R., Mahony, J., Courtin, P., Chapot-Chartier, M. P., Van Pijkeren, J. P., Britton, R. A., et al. (2013). The lactococcal phages Tuc2009 and TP901-1 incorporate two alternate forms of their tail fiber into their virions for infection specialization. J. Biol. Chem. 288, 5581–5590. doi: 10.1074/jbc.M112.444901\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSullivan, M. B., Huang, K. H., Ignacio-Espinoza, J. C., Berlin, A. M., Kelly, L., Weigele, P. R., et al. (2010). Genomic analysis of oceanic cyanobacterial myoviruses compared with T4-like myoviruses from diverse hosts and environments. Environ. Microbiol. 12, 3035–3056. doi: 10.1111/j.1462-2920.2010.02280.x\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSuzuki, G., Shimazu, N., and Tanaka, M. (2012). A yeast prion, Mod5, promotes acquired drug resistance and cell survival under environmental stress. Science 336, 355–359. doi: 10.1126/science.1219491\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTelling, G. C., Scott, M., Mastrianni, J., Gabizon, R., Torchia, M., Cohen, F. E., et al. (1995). Prion propagation in mice expressing human and chimeric PrP transgenes implicates the interaction of cellular PrP with another protein. Cell 83, 79–90. doi: 10.1016/0092-8674(95)90236-8\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nTetz, G. V., Ruggles, K. V., Zhou, H., Heguy, A., Tsirigos, A., and Tetz, V. (2017). Bacteriophages as potential new mammalian pathogens. Sci. Rep. 7:7043. doi: 10.1038/s41598-017-07278-6\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nVandenesch, F., Naimi, T., Enright, M. C., Lina, G., Nimmo, G. R., and Heffernan, H. (2003). Community-acquired methicillin-resistant Staphylococcus aureus carrying Panton-Valentine leukocidin genes: worldwide emergence. Emerg. Infect. Dis. 9, 978–984. doi: 10.3201/eid0908.030089\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nvon Wintersdorff, C., Penders, J., van Niekerk, J., Mills, N., Majumder, S., van Alphen, L., et al. (2016). Dissemination of antimicrobial resistance in microbial ecosystems through horizontal gene transfer. Front. Microbiol. 7:173. doi: 10.3389/fmicb.2016.00173\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWagner, P. L., and Waldor, M. K. (2002). Bacteriophage control of bacterial virulence. Infect. Immun. 70, 3985–3993. doi: 10.1128/IAI.70.8.3985-3993.2002\\\\nCrossRef Full Text | Google Scholar\\\\nWalker, L. C., Schelle, J., and Jucker, M. (2016). The prion-like properties of amyloid-β assemblies: implications for Alzheimer’s disease. Cold Spring Harb. Perspect. Med. 6:a024398. doi: 10.1101/cshperspect.a024398\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWemheuer, W. M., Wrede, A., and Schulz-Schaeffer, W. J. (2017). Types and strains: their essential role in understanding protein aggregation in neurodegenerative diseases. Front. Aging Neurosci. 9:187. doi: 10.3389/fnagi.2017.00187\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nWestergard, L., Christensen, H. M., and Harris, D. A. (2007). The cellular prion protein (PrP C): its physiological function and role in disease. Biochim. Biophys. Acta 1772, 629–644. doi: 10.1016/j.bbadis.2007.02.011\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nYuan, A. H., Garrity, S. J., Nako, E., and Hochschild, A. (2014). Prion propagation can occur in a prokaryote and requires the ClpB chaperone. Elife 3:e02949. doi: 10.7554/eLife.02949\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nZylicz, M., Ang, D., Liberek, K., and Georgopoulos, C. (1989). Initiation of lambda DNA replication with purified host-and bacteriophage-encoded proteins: the role of the dnaK, dnaJ and grpE heat shock proteins. EMBO J. 8, 1601–1608.\\\\nPubMed Abstract | Google Scholar\"\\n},\\n{\\n  \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00574\",\\n  \"crawl\": {\\n    \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00574/full\",\\n    \"loadedTime\": \"2024-01-23T08:50:59.949Z\",\\n    \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n    \"depth\": 1,\\n    \"httpStatusCode\": 200\\n  },\\n  \"metadata\": {\\n    \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00574\",\\n    \"title\": \"Frontiers | Therapeutic Characterization and Efficacy of Bacteriophage Cocktails Infecting Escherichia coli, Klebsiella pneumoniae, and Enterobacter Species\",\\n    \"description\": \"Infections due to antibiotic resistant bacteria are increasing globally and this needs immediate attention. Bacteriophages are considered an effective alternative for the treatment of bacterial infections. The aim of this study was to isolate and characterize the bacteriophages that infect Escherichia coli, Klebsiella pneumoniae and Enterobacter species. For this, clinical bacterial isolates of the mentioned species were obtained from diagnostic centers located in Chennai, Tamil Nadu, India. The bacteriophages were isolated from sewage water samples collected from Tamil Nadu, India. Phage isolation was performed using the enrichment method and agar overlay method was used to confirm the presence of bacteriophages. All the phages were characterized for their life cycle parameters, genome analysis and in vitro phage cocktail activity. The three bacteriophages exhibited broad host range activity: Escherichia virus myPSH2311 infecting E. coli belonging to six different pathotypes, Klebsiella virus myPSH1235 infecting K. pneumoniae belonging to four different serotypes and Enterobacter virus myPSH1140 infecting four different species of Enterobacter. Morphological observations suggested that the bacteriophages belonged to, Phieco32virus (Escherichia virus myPSH2311), Podoviridae (Klebsiella virus myPSH1235) and Myoviridae (Enterobacter virus myPSH1140). The life cycles (adsorption, latent period and cell burst) of Escherichia virus myPSH2311, Klebsiella virus myPSH1235 and Enterobacter virus myPSH1140 were found to be 26 min, 40 min and 11 min respectively. Genomic analysis revealed that Escherichia virus myPSH2311 is closely related to Escherichia phage vB_EcoP_SU10, Klebsiella virus myPSH1235 is closely related to Klebsiella phage vB_KpnP_KpV48 and Enterobacter virus myPSH1140 is closely related to Enterobacter phage PG7 and Enterobacter phage CC31. When phage cocktail was used against multiple bacterial mixtures, there was a reduction in bacterial load from106 CFU/mL to 103 CFU/mL within 2 hours. All the three characterized phages were found to have a broad host range activity and the prepared phage cocktails were effective against a mixed bacterial population that is resistant to meropenem and colistin, two last resort antibiotics. Infections caused by drug resistant bacteria will be a serious threat in the future and the use of virulent bacteriophages in therapy may offer an effective solution.\",\\n    \"author\": null,\\n    \"keywords\": null,\\n    \"languageCode\": \"en\"\\n  },\\n  \"screenshotUrl\": null,\\n  \"text\": \"Introduction\\\\nBacteriophages are the viruses of bacteria that live in the same ecological niche, where their host bacteria are present (Rohwer, 2003). Phages are generally very specific (species-specific and strain-specific) to their bacterial host but some phages are polyvalent, and can infect more than one species or strain of bacteria (Chibani-Chennoufi et al., 2004). Phage therapy largely involves the treatment of bacterial infections using bacteriophages/phages (Levin and Bull, 2004). Phages with broad host range are mostly chosen for therapy, because of their broad spectrum host-range activity against multiple bacteria. The phages belonging to the order Caudovirales (Family-Myoviridae, Siphoviridae, and Podoviridae) with proteinaceous tail, that follow only lytic pathway, are preferred for therapy (Gill and Hyman, 2010). The use of bacteriophages for therapeutic purpose is an old concept that is re-emerging after about a century (Sulakvelidze et al., 2001). Antibiotic resistance has become a human health concern globally as the infections caused by resistant bacteria are becoming difficult to cure (World Health Organization [WHO], 2014; Ventola, 2015). Phage therapy can be one of the alternatives for combating antibiotic resistant bacterial infections (Rios et al., 2016).\\\\nEscherichia coli, Klebsiella pneumoniae, and Enterobacter cloacae are Gram-negative bacteria that belong to the family Enterobacteriaceae. All the three are enteric pathogens causing serious opportunistic infections in humans (Morens et al., 2004; Schlossberg, 2015). They cause hospital acquired and community acquired infections such as diarrhea, meningitis, urinary tract infections (UTIs), bacteremia, pneumonia, surgical site infections, and sepsis (Schlossberg, 2015). The increasing reports of resistance to carbapenem and colistin, two last resort drugs, among Enterobacteriaceae world over (World Health Organization [WHO], 2014) and more particularly so in the developing countries (Ventola, 2015) is a serious threat to their therapeutic use, which prompts search for alternative treatment options. Studies using bacteriophages as an antibacterial agent have shown promising outcomes in both in vitro and in vivo studies, and therefore phage therapy is being studied as a candidate to cure bacterial infections (McVay et al., 2007; Hung et al., 2011; Cao et al., 2015; Mirzaei and Nilsson, 2015). Phage cocktails have shown broad spectrum activity against many bacterial strains (Mendes et al., 2014; Yen et al., 2017). The characterization of phages for therapeutic purpose involves isolation of potential lytic phages, multi-step in vitro characterization, cocktail preparation and purification, dosing and in vivo studies. More than 50 Escherichia phages belonging to families Myoviridae, Siphoviridae, and Podoviridae have been reported with complete genome sequences1. Genome sequenced phages against Klebsiella (≈29) and Enterobacter (≈10) have been reported much lesser in number2. Here, we report characterization of three lytic bacteriophages that showed promising ability to lyse E. coli, K. pneumoniae, and Enterobacter species, report their host range specificity and also efficacy of phage cocktails made using these three phages in various permutations and combinations, in effectively killing combinations of host bacteria using in vitro phage killing assay.\\\\nMaterials and Methods\\\\nIsolation of Clinical Bacterial Strains for the Study\\\\nThis study does not include any human subjects; therefore, ethical approval was not required for this study according to national and institutional guidelines. The clinical isolates of Escherichia coli, Klebsiella pneumoniae, Enterobacter cloacae, E. hormaechei, E. asburiae, and E. aerogenes used in this study were collected from diagnostic centers in Chennai, Tiruchirappalli, and Madurai located in the state of Tamil Nadu in India, during December 2014- September 2016. All the isolates were preserved in 30% glycerol stocks at -20°C. The clinical samples used for bacterial isolation were urine, sputum, pus, blood, wound swab, and bronchial aspirate. Bacterial identification was performed using VITEK identification system (bioMèrieux, Inc., United States) and 16S rRNA analysis. Universal primers, 27F (5′-AGAGTTTGATCCTGGCTCAG-3′) and 1492R (5′-GGTTACCTTGTTACGACTT-3′), were adopted for gene amplification and sequences of 16S rRNA genes were deposited in GenBank. All the clinical isolates were studied for resistance against meropenem and colistin, two last resort antibiotics, using microbroth dilution method following Clinical and Laboratory Standards Institute [CLSI], 2018. For the study, a total of 150 non-repetitive, Gram-negative bacterial isolates belonging to three genera Escherichia, Klebsiella, and Enterobacter were used. The isolated clinical pathogens included 80 E. coli isolates, 44 Klebsiella pneumoniae isolates and in the case of Enterobacter isolates, there were four different species namely E. cloacae (n = 15), E. hormaechei (n = 4), E. asburiae (n = 4), and E. aerogenes (n = 3) (Table 1). The results for meropenem and colistin resistance screening of these isolates are presented in Supplementary Table S1.\\\\nTable 1. Host range infection and efficiency of plating (EOP) of the phages Escherichia virus myPSH2311, Klebsiella virus myPSH1235, and Enterobacter virus myPSH1140.\\\\nPhage Isolation and Enrichment\\\\nBacteriophages were isolated from water samples collected from Ganges River near Varanasi in Northern part of India and sewage water treatment plants (secondary treatment stage) from different locations in Chennai, Bangalore, Tirupathi, Vellore, Karur, and Trichy in Southern part of India. Initially, the isolated bacterial strains (one isolate at a time) were grown in Luria-Bertani broth (Himedia, India) medium and were used as a host for phage isolation. Briefly, to a 5 mL of exponentially grown bacterial culture (optical density at 600 nm = 0.6), 10 mL of phage containing water samples was added and incubated at 37°C for 24 h in shaking incubator to enrich the phages against the host bacterium. This mixture was centrifuged for 15 min at 12,000 ×g and the supernatant was filtered in 0.22-μm pore-sized membrane syringe filters for separation of phages from other contaminants. The filtrate was used for double-agar overlay method (Li et al., 2016a). Briefly, to the exponentially grown host bacterial culture (400 μL), the phage filtrate (200 μL) was added and incubated for 15 min. To the mixture, 3 mL of molten soft agar (0.75% agar) was added and over-laid onto prepared LB agar plate (1.5% agar). The plates were incubated at 37°C for 10 h and the appearance of clear plaques indicated the presence of phages against the host bacterium. The phage plaques were picked-up from the plate for further purification and the phage titer was determined. For spot test method, the bacterial (using exponentially grown host bacterial culture) lawn was prepared in LB agar plates and 10–50 μL of phage filtrate was placed as a spot over the target bacterial lawn to evaluate the phage activity. The development of clear spots indicated the phage activity and the time taken for bacterial clearance indicated the lytic activity of phages. The bacterium initially used for phage isolation was deemed as a host bacterium against the phage. Multiplicity of infection (MOI) was calculated using the number of phage particles against the potent host bacteria (PFU/CFU).\\\\nPurification of Lytic Phages\\\\nThe isolated phage lysates were prepared in high titers using phage multiplication strategy (propagation on host bacterium). Briefly, the phages were multiplied using host bacterium for 24 h (day 1) and centrifuged at 6000 ×g for 15 min. The collected supernatant was mixed with exponentially grown host bacterium (day 2) and allowed to multiply. Similar passages were carried out for 5 days and evaluated for phage activity/ titer by spot test and double agar overlay method against E. coli, K. pneumoniae, and Enterobacter species. The obtained high titer phages were precipitated using 10% PEG 6000 (polyethylene glycol) and 1 M NaCl. Briefly, to the phage lysate 10% PEG 6000+1 M NaCl was added, mixed gently (did not vortex) and the mixture was stored at 4°C for 24 h. The precipitated phage particles were centrifuged at 15,000 ×g for 45 min and the obtained pellet was resuspended in sterile SM buffer (For 1 L: 5.8 g, NaCl; 50 mL, 1 M Tris-HCl [pH 7.5]; 2 g, MgSO4.7H2O; 5 mL, 2% gelatin). The extraction was carried out by adding equal volume of chloroform and the aqueous phase was sedimented by centrifugation at 18,000 ×g for 80 min. The obtained phage particles were dialyzed against phosphate buffer saline (PBS) for 6 h by changing buffer every 2 h and the purified phage suspension was stored at 4°C for further analysis.\\\\nElectron Microscopic Analysis\\\\nThe purified phage particles (105 PFU/mL) were negatively stained using phosphotungstic acid, PTA (2% [w/v], pH 7.0). Briefly, 10 μL of phage lysate was added over the copper grid and the liquid was allowed to absorb for 10 min. The remaining liquid was removed using tissue paper and the prepared 2% PTA solution (staining solution) was added. After allowing it to stain for 5 min, the excess stain was removed and the grid was washed twice with sterile water. The negatively stained phage particles in the copper grid were allowed to dry at room temperature for 20–30 min and visualized under Transmission Electron Microscopy (FEI-TECNAI G2-20 TWIN, Bionand, Spain). The phage morphology was determined and head/ tail lengths (10 measurements each) were measured using ImageJ software.\\\\nHost-Range Specificity Determination and Efficiency of Plating (EOP)\\\\nThe lytic activity of isolated phages was tested against the target bacteria. Accordingly, Escherichia phage was tested against 80 E. coli isolates belonging to pathotypes: EPEC (enteropathogenic E. coli), EHEC (enterohemorrhagic E. coli), ETEC (enterotoxigenic E. coli), EIEC (enteroinvasive E. coli), EAEC (enteroaggregative E. coli), and UPEC (uropathogenic E. coli). Klebsiella phage was tested against 37 K. pneumoniae isolates that belonged to serotypes K1, K2, K5 and Enterobacter phage was tested against 15 E. cloacae, 4 E. hormaechei, 4 E. asburiae, and 3 E. aerogenes isolates. Spot test was carried out to assess the host-range specificity of phages against the test bacteria and the resulting positive isolates were again tested for their plaque forming ability in double agar overlay method for calculating efficiency of plating (EOP) (Mirzaei and Nilsson, 2015). EOP was calculated using the number of virus particles infecting the test bacterium against the same titer of virus particles infecting the host bacterium. Accordingly, all the test bacterial strains were grown overnight (16 h) and the concentration of 106–109 (CFU/mL) was used for double agar overlay method. For the assay, 200 μL of bacterial culture was mixed with 100 μL of phage lysate (MOI = 0.01) and EOP was determined using the formula, plaque forming units (PFUs) on the test bacterium/PFU on the host bacterium, evaluated by double agar overlay method. EOP was classified as ‘High,’ ‘Moderate,’ and ‘Low’ based on the productive infection on the target bacterium. EOP was deemed as ‘High’ only if the phage-bacterium combination against the test bacterium had a productive infection of at least 50% (EOP ≥ 0.5) compared to the host bacterium. EOP between >10 and <50% (EOP > 0.1- < 0.5) was considered ‘Moderate’ and EOP < 10% (EOP ≤ 0.1) was recorded as ‘Low’ (Mirzaei and Nilsson, 2015).\\\\nCharacterization: Adsorption Rate, Latency Period, and Burst Size\\\\nExponentially grown bacterial cells were mixed with the respective phages at a MOI of 0.001 and incubated at 37°C. Aliquots of 100 μL were removed after every 4 min interval for 40 min and diluted in 4.4 mL LB broth and 0.5 mL of chloroform was added. After incubating the mixture for 30 min at 37°C, the number of non-adsorbed phages was determined subsequently using double agar overlay method. The adsorption curve was constructed using the ratio of non-adsorbed phage particles at different time intervals to the number of initial phages. One-step growth experiment was performed to determine the latent period and burst size (14). Briefly, the bacterial cells (108 CFU/mL) were infected with the phage particles (MOI of 0.001) and allowed to adsorb (based on the adsorption time determined previously) at 37°C. The mixture was then centrifuged at 12,000 ×g for 5 min and the pellet was resuspended in 10 mL of LB broth and the incubation was continued at 37°C. The samples were taken at 5 min intervals for 80 min and titrated against the host bacterium. The latent-period was calculated as the duration between the phage adsorbed until the release of phage virions. The burst size of the phage was calculated using the final number of free phage particles to the initial number of phages.\\\\nPhage Stability Studies\\\\nStability studies were conducted at different pH and temperature. For thermal stability tests, phage lysates (108 PFU/mL) were incubated at 4, 20, 35, 45, 50, 55, 60, 70, and 80°C for 60 min in temperature-controlled water bath and immediately transferred to the ice cold condition (-20°C) which was further tested for phage activity using double agar overlay method. The pH stability studies were performed using SM buffer and pH was adjusted using 1N NaOH and 1N HCl. The phage lysates (108 PFU/mL) were incubated at pH 1–14 for 60 min and the aliquots were removed for stability analysis. The results were expressed as phage viability in terms of percentage of initial viral counts. All the stability studies were tested using E. coli ec311, K. pneumoniae kp235, and E. cloacae el140, and the experiments were repeated in triplicates.\\\\nDNA Isolation, Genome Sequencing, and Analysis\\\\nThe phage DNA was extracted from purified phage particles using phenol-chloroform (24:1) method and precipitated using ethanol (100%). The purified phage DNA was visualized on 0.8% agarose gels and the PE libraries were prepared using Illumina TruSeq Nano DNA library Prep kit. The prepared libraries were sequenced using Illumina Nextseq 500 system (using 2 bp × 150 bp chemistry) at Eurofins Genomics, Bangalore, India. The sequenced raw data was processed to obtain high quality clean reads using Trimmomaticv 0.35 to remove adapter sequences, ambiguous reads (reads with unknown nucleotides “N” larger than 5%), and low-quality sequences [reads with more than 10% quality threshold (QV) < 20 phred score]. The sequenced high quality reads were de novo assembled using CLC Genomics Workbench version 9.5.2. Protein-coding and tRNA genes were identified using the final assembly. The transfer-RNA (tRNA) genes were predicted using tRNAscan-SE 2.0 web server while the protein coding genes (CDS) were predicted using FGENESV web server. Functional annotation of the predicted proteins was performed using the amino acid sequences viaBLASTp program online against a custom database of viral proteins in NCBI. Gene ontology (GO) annotations of the genes were determined by the Blast2GO platform. Distribution of GO terms across the categories – Biological Process, Molecular Function and Cellular Component was obtained through WEGOportal3. The NCBI sequence was downloaded from NCBI2 for sequence comparison and the scaffolds were then subjected to reference-based assembly via CONTIGuator2. The final assembly generated by CONTIGuator was validated based on sequence homology to known bacteriophage sequences in NCBI via BlastN.\\\\nComposition and Preparation of Phage Cocktails\\\\nPhage cocktails containing different compositions of isolated phages under study were prepared and evaluated for activity against target species. For cocktail preparation, two or three different phages were mixed together in equal proportions to obtain a concentration of 106 PFU/mL. Briefly, cocktail EK1 contained Escherichia phage plus Klebsiella phage; KL2 contained Klebsiella phage plus Enterobacter phage; EL3 contained Escherichia phage plus Enterobacter phage and EKL4 contained Escherichia phage plus Klebsiella phage plus Enterobacter phage. The prepared cocktails were tested for in vitro phage-killing assay against respective bacterial strains. The results were compared to the activity of phages in cocktails to the activity of phages alone, and one bacterium from each genus was used for this study. Accordingly, 1 mL of host bacterium (6 × 107 CFU/mL) was diluted in LB broth to yield a final concentration of 6 × 106 CFU/mL. For each study, 100 μL (each phage at the concentration of 106 PFU/mL; MOI of 1.0) of bacteriophage suspension was added and the mixture was incubated at 37°C and the aliquots (100 μL) were removed at 0, 2, 4, 6, and 24 h to calculate the reduction in bacterial count. To test the activity of phage cocktails, bacteria were also used in combination similar to phage combinations (Supplementary Table S2). In the case of control experiments, bacteriophage buffer alone was used with the bacterial inoculum and bacterial growth was determined. All the cocktail studies were tested using E. coli ec311, K. pneumoniae kp235, and E. cloacae el140, and the experiments were repeated in triplicates.\\\\nResults\\\\nNomenclature and Morphological Characterization of Phages\\\\nThe phages were named as Escherichia virus myPSH2311, Klebsiella virus myPSH1235, and Enterobacter virus myPSH1140 following the bacteriophage nomenclature guidelines (Krupovic et al., 2016). Examination of phage morphology by TEM analysis showed that Escherichia virus myPSH2311 had an icosahedral head of 33 ± 3.0 nm, a non-contractile tail length of 65 ± 2.5 nm and belonged to genus Phieco32virus, Klebsiella virus myPSH1235 had the icosahedral head of 80 ± 4.5 nm and very short non-contractile tail that showed the phage belonged to family Podoviridae and Enterobacter virus myPSH1140 had an elongated head of approximately 80 ± 2.0 nm and long contractile tail of 101 ± 3.5 nm in length indicating that it belonged to the family Myoviridae (Figure 1).\\\\nFigure 1. Transmission electron microscopy (TEM) images of (A) Escherichia virus myPSH2311, (B) Enterobacter virus myPSH1140, (C) Klebsiella virus myPSH1235. The smaller images in (A,C) represent 100 nm.\\\\nHost-Range Activity Determination and Efficiency of Plating (EOP)\\\\nThe spot test assay showed that the Escherichia virus myPSH2311 had lytic activity against 73% of the tested E. coli isolates, Klebsiella virus myPSH1235 had activity against 52% K. pneumoniae isolates and Enterobacter virus myPSH1140 showed activity against 15 E. cloacae (n = 15), 3 E. hormaechei (n = 4), 2 E. asburiae (n = 4), and 2 E. aerogenes (n = 3) isolates. In the case of double agar overlay method, Escherichia virus myPSH2311 had plaques against 43/80 tested E. coli isolates, Klebsiella virus myPSH1235 had plaques against 17/44 K. pneumoniae isolates and Enterobacter virus myPSH1140 had plaques against 11/15 E. cloacae, 2/4 E. hormaechei, 2/4 E. asburiae, and 2/3 E. aerogenes isolates. The percentage activity difference between the spot test and double agar overlay method was found to be 29.7% for Escherichia virus myPSH2311, 30% for Klebsiella virus myPSH1235 and for Enterobacter virus myPSH1140, it was 30.7% for E. cloacae, 40% for E. hormaechei, 0% for both E. asburiae and E. aerogenes. The EOP analysis that was used to differentiate the phage infectivity between spot test and double agar overlay method, showed a different scenario (Table 1). Though, spot test results showed that Escherichia virus myPSH1311 produced clear zone (spot) against 58/80 tested E. coli isolates; the EOP analysis showed high productive infection in 34/80 E. coli isolates whereas against 9 E. coli isolates it was moderate or low productive infection and 15/80 E. coli isolates had no infection. Klebsiella virus myPSH1235 had high productive infection against 10/44 K. pneumoniae isolates with 23/44 in spot test assay and the percentage difference was 78.7%. Even if all the EOP results (High + Moderate + Low) were considered for Klebsiella phage myPSH1235, the number of isolates producing (17/44) plaques in double agar overlay assay was still lower than the spot test (23/44) results. Enterobacter virus myPSH1140 had high productive infection against 7/15 tested E. cloacae isolates compared to 15/15 in spot test assay. The same phage showed high productive infection against 2/4 E. hormaechei, 2/4 E. asburiae, and 2/3 E. aerogenes, respectively (Table 1).\\\\nPhage Characterization: One-Step and Stability Studies\\\\nThe multiplication capacity of phages was determined by one-step growth experiment to analyze the adsorption velocity, latency period and burst size (Figure 2). Accordingly, for Escherichia virus myPSH2311, the adsorption velocity was 1.1 × 10-9mL/min with latency period of 26 min and the burst size of approximately 110 phages/infected cell. The adsorption velocity for Klebsiella virus myPSH1235 was 4.35 × 10-9 and the latency period was 40 min with the burst size of 120 phages/infected cell. Enterobacter virus myPSH1140 had an adsorption velocity of 2.8 × 10-9, a very short latency period of 11 min and a burst size of 135 phages/infected cell (Figure 2). When sensitivity of phages to different pH conditions was determined by exposing them to varying range of pH from 1 to 14 for 60 min, all the three phages were found to be viable from pH 4 up to pH 11, but the phages were inactivated at pH ≤ 3 and ≥ 12 (Figure 3). In the case of thermal stability, all the phages were found to uphold their activity up to 55°C and reduction in activity was observed at higher temperatures (Figure 3). The complete characterization report is available in Supplementary Table S3.\\\\nFigure 2. (A) Adsorption rate of phages to their bacterial host, (A1) Escherichia virus myPSH2311, (A2) Enterobacter virus myPSH1140, (A3) Klebsiella virus myPSH1235, and (B) One-step growth curve experiment of phages, (B1) Escherichia virus myPSH2311, (B2) Enterobacter virus myPSH1140, (B3) Klebsiella virus myPSH1235. Legend within figure B: L-latency period; B-burst size.\\\\nFigure 3. (A) Stability of phages at varying temperatures; (A1) Escherichia virus myPSH2311, (A2) Enterobacter virus myPSH1140, (A3) Klebsiella virus myPSH1235 and (B) Stability of phages at varying pH at 37°C, (B1) Escherichia virus myPSH2311, (B2) Enterobacter virus myPSH1140, (B3) Klebsiella virus myPSH1235.\\\\nGenomic Analysis and Annotation\\\\nThe genome of Escherichia virus myPSH2311 measured 68,712 bp in size with a GC content of 42.4%. The genome contains 89 proteins or coding sequences (CDS) and it includes 27 proteins of known putative function and 62 hypothetical proteins. A total of 1.01 Gb data was assembled into scaffolds using CLC workbench version 9.5.2, and the assembly size was 5,945,203 bp with the average scaffold size of 12,133 bp. The arranged complete genome of Escherichia virus myPSH2311 is closely related to Escherichia phage vB_EcoP_SU10 (88%) and Escherichia virus phiEco32 (72%) (Figure 4A and Supplementary Figures S1, S2A). The NCBI accession number for this sequence is MG976803. The complete list of all the proteins is available in Supplementary Table S4. Klebsiella virus myPSH1235 was found to have a genome size of 45,135 bp with a GC content of 53.7%. The genome contains 49 proteins or CDS, of which 21 were found to have known putative function and 28 were hypothetical proteins. The obtained 1.37 Gb data was assembled into scaffolds using CLC workbench version 9.5.2, and the assembly size was 5,740,807 bp with the average scaffold size of 1,321 bp. The genome was closely related to Klebsiella phage vB_KpnP_KpV48 (95%) (Figure 4B and Supplementary Figures S1, S2B). The complete genome of Klebsiella virus myPSH1235 is free of toxins or toxin-related genes, and none of the proteins representing a temperate or lysogenic lifestyle was detected (Supplementary Table S4). The NCBI accession number for this phage is MG972768. The complete list of all the proteins is available in Supplementary Table S5. Enterobacter virus myPSH1140 was having a genome size of 172,614 bp with a GC content of 39.9%. The gene annotation studies showed 102 proteins with known function and 138 proteins were hypothetical proteins. The obtained 1.22 Gb data was assembled into scaffolds using CLC workbench version 9.5.2, and the assembly size was 5,193,726 bp with the average scaffold size of 6,059 bp. The complete genome was having 92% similarity with Enterobacter phage CC31 and 90% similarity with Enterobacter phage PG7 (Figure 4C and Supplementary Figures S1, S2C). The NCBI accession number is MG999954. The complete list of all the proteins is available in Supplementary Table S6.\\\\nFigure 4. Phylogenetic tree of (A) Escherichia virus myPSH2311, (B) Klebsiella virus myPSH1235, and (C) Enterobacter virus myPSH1140 constructed based on the complete genome sequences of selected bacteriophages in NCBI-BLAST. The tree was produced using BLAST pairwise alignment using Neighbor-Joining method. The query sequence is highlighted in yellow.\\\\nIn vitro Activity of Phage Cocktail\\\\nPhage cocktails were prepared to evaluate the activity of phages against multiple bacterial strains. When the phage cocktail containing all the three phages was tested against the three meropenem and colistin resistant test bacteria, the growth declined after 2 h from 106 to <105 CFU/mL and at the end of 24 h the bacterial density reached to zero with no viable cells. For EK1 cocktail, >2 fold decrease in bacterial cell count (both E. coli and K. pneumoniae) was observed after 2 h, for KL2 cocktail, the bacterial cell count (both K. pneumoniae and E. cloacae) decreased from 106 to 103 CFU/mL within 2 h and for EL3 cocktail, twofold reduction of bacterial cells (both E. coli and E. cloacae) was observed after 4 h. In phage cocktail containing all the three phages, EKL4, a twofold decrease in bacterial count (E. coli, K. pneumoniae, E. cloacae) was observed in 2 h as comparable to the phages alone. In the case of prepared phage cocktails, all the combinations had similar to better activity in comparison to the phages alone (Figure 5). Our experiment proved the activity of prepared phage cocktails against multiple bacterial genera and showed promising results against meropenem and colistin resistant bacteria in vitro.\\\\nFigure 5. In vitro phage activity of prepared phage cocktails. (A) activity of Escherichia virus myPSH2311 and Klebsiella virus myPSH1235 and their combination (EK1); (B) activity of Klebsiella virus myPSH1235 and Enterobacter virus myPSH1140 and their combination (KL2); (C) activity of Escherichia virus myPSH2311 and Enterobacter virus myPSH1140 and their combination (EL3); (D) activity of Escherichia virus myPSH2311, Klebsiella virus myPSH1235, Enterobacter virus myPSH1140 and their combination (EKL4) (Control for all the experiments-bacterial growth without antibacterial agents and phages).\\\\nDiscussion\\\\nPhage therapy is currently gaining attention in clinical medicine, because the infections caused by ‘superbugs – multi-drug resistant (MDR) bacteria’ are almost impossible to treat, using available antibiotics (Chan et al., 2013). From clinical point-of-view, bacteriophages are likely to be used in treatment, only when the patient is infected with multi-drug resistant bacteria or during the antibiotic crisis of non-curable infections (Lin et al., 2017). The increasing studies on bacteriophages and phage therapy could put forth phage therapy as one of the alternative treatment options to treat ‘superbug’ infections (Rios et al., 2016). This study is one such effort to characterize bacteriophages infecting Gram-negative bacteria, E. coli, K. pneumoniae, and Enterobacter species.\\\\nIn the present study, we characterized the phenotypic, morphological, and genomic properties of three bacteriophages that are independently specific in their activity against pathogenic E. coli, K. pneumoniae, and Enterobacter species. All the three characterized phages belong to order Caudovirales that are lytic phages which are generally considered for therapy (Podoviridae and Myoviridae). There are earlier studies on Escherichia phages that showed many of the studied phages are specific to a strain of E. coli (e.g., Escherichia phage OSYSP infecting only E. coli O157:H7) or had a narrow host range activity (Carter et al., 2012; Dalmasso et al., 2016; Baig et al., 2017; Lin et al., 2017). In our study, the characterized Escherichia virus myPSH2311 was found to have broad host range activity infecting E. coli isolates that belonged to six different pathotypes EPEC, EHEC, ETEC, EIEC, EAEC, and UPEC besides some unknown pathotypes. This is one of the rarest reports where an Escherichia phage infecting six different pathotypes of E. coli was characterized. But another study also report the broad host range activity of Escherichia phage against multiple pathotypes of E. coli (Manohar et al., 2018b). The studied Klebsiella virus myPSH1235 showed broad host range activity against K. pneumoniae belonging to serotypes K1, K2, K5 besides some unknown serotypes. An earlier study showed that a multi-host Klebsiella phage ϕK64-1 was capable to infect Klebsiella capsular types K1, KN4, KN5, K11, K21, K25, K30, K35, K64, and K69. It is stated that the multi-host infectivity of the bacteriophages is due to the presence of multiple depolymerases in the tail-fibers (Pan et al., 2017). The role of polyvalent phages in phage therapy needs more detailed investigation. The studies on Enterobacter phage thus far are very minimal or reports of established Enterobacter phages capable of infecting various strains or species are scarce (Li et al., 2016a; Pereira et al., 2016). The characterized Enterobacter virus myPSH1140 in our study belonging to Myoviridae family showed activity against four species of Enterobacter E. cloacae, E. hormaechei, E. asburiae, and E. aerogenes. Our study and those of some others strongly suggest that there are bacteriophages that can have broad host range activity against different strains or species or genera of bacteria (Hamdi et al., 2017). A recent study showed that bacteriophages SH6 and SH7 were capable of infecting bacteria from different genera – E. coli O157:H7, Shigella flexneri, Salmonella paratyphi, and Shigella dysenteriae (Hamdi et al., 2017). There are studies that showed ‘narrow’ host range specificity of bacteriophages mainly because of the use of standard isolation procedure where single host strain of bacteria is used (Li et al., 2016b; Ross et al., 2016; Hamdi et al., 2017). Future studies should focus on testing the host range activity of bacteriophages but it is also believed that host range activity is not a fixed property of bacteriophages and can evolve overtime (Ross et al., 2016; Hamdi et al., 2017). It should also be noted that we found dissimilarities in the results obtained between spot test and plaque assay in host range activity determination, therefore plaque assay is recommended in order to obtain productive infection for the determination of host range activity of phages (Mirzaei and Nilsson, 2015).\\\\nThe three characterized bacteriophages had different growth profile, which is one of the important characteristics of lytic bacteriophages. In this study, Escherichia virus myPSH2311 (Podovirus) had a growth profile with the latent period of 26 min and burst size of 110 phages/infected cell, which is beneficial for therapy. Earlier studies using Podoviridae phages showed latency period of ≈15–25 min and burst size of ≈50–80 phages/infected cell (Dalmasso et al., 2016). An earlier study on Podoviridae phages infecting Klebsiella species was found to have a growth profile with the latency period of 15 min and burst size of ≈50–60 phages/infected cell (Chaturongakul and Ounjai, 2014). In our study, Klebsiella virus myPSH1235 belonging to Podoviridae family was found to have a latent period of 40 min and burst size of 120 phages/infected cell, which is beneficial for therapy. It was very clear from this study that the growth profile of bacteriophages is not completely dependent on either the family or the host these bacteriophages infect. The bacteriophages belonging to Myoviridae family are known to have shorter life cycles and similarly the characterized Myoviridae phage (Enterobacter phage myPSH1140) had a latent period of 11 min and burst size of 135 phages/infected cell. It was also noted in this and earlier studies that the number of progeny phage particles (burst size) largely depends on the availability of host bacterial cells (Dalmasso et al., 2016). Life cycle parameters of bacteriophages will play significant role in determining both in vitro and in vivo phage activities (during therapy), because phage multiplication is directly proportional to reduction in bacteria.\\\\nThe genomes of all the three characterized phages showed 70–100% similarities to the already existing phage genomes in the database. Escherichia virus myPSH2311 (68,712 bp) was found to have 88% sequence similarity with Escherichia phage vB_EcoP_SU10 (77,327 bp). The endolysin gene (ORF 12) was found to have 98% sequence similarity with Escherichia virus phiEco32 and 97% sequence similarity with Escherichia phage vB_EcoP_SU10. The DNA injection protein (ORF 17) was detected to have 98% sequence similarity with Escherichia phage vB_EcoP_SU10. The genome does not have toxins or toxin-related genes, and none of the genomic markers representing a temperate or lysogenic lifestyle was detected (Supplementary Table S4). Escherichia phage vB_EcoP_SU10 is a C3 morphotype phage but Escherichia virus myPSH2311 does not have any elongated regions in its morphological characterization (Mirzaei et al., 2014) and related to T4-phages (Petrov et al., 2010). Klebsiella virus myPSH1235 was found to have 80% similarity with Klebsiella phage vB_KpnP_KpV48. Klebsiella virus myPSH1235 has a genome size of 45,135 bp with 49 CDS compared to Klebsiella phage vB_KpnP_KpV48, which has a genome size of 44,623 bp with 57 CDS. According to the genome morphology, phylogenetic analysis and sequence similarities, the Klebsiella virus myPSH1235 is classified within the genus Kp34virus, subfamily Autographivirinae, family Podoviridae. Klebsiella phage vB_KpnP_KpV48 has no capsular specificity and similarly, Klebsiella virus myPSH1235 was also found to infect K. pneumoniae of capsular types K1, K2 and K5 (Solovieva et al., 2018). From clinical point of view, the K. pneumoniae capsular types K1 and K2 are the most virulent strains and that the infections caused by these strains can be eliminated using Klebsiella virus myPSH1235 is a promising result, which needs further exploration. Enterobacter virus myPSH1140 was found to have a large genome size of 172,614 bp with 240 CDS with 92% similarity with Enterobacter phage CC31 (165,540 bp and 279 CDS) and 90% similarity to Enterobacter phage PG7 (173,276 and 294 bp). The genome contains 240 proteins or CDS and all the identified proteins were found to have 90–100% sequence similarity with the Enterobacter phage CC31 and Enterobacter phage PG7. Similarities in the phage genome showed that there is abundance of these phages in the environment and therefore they may be isolated for therapeutic purpose.\\\\nEarlier studies on phage cocktails showed promising results in reducing the bacterial load in both in vitro and in vivo models (Lin et al., 2017; Pan et al., 2017). In our in vitro experiments, a phage cocktail that was prepared using the three newly isolated bacteriophages infecting three different genera of bacteria (Escherichia virus myPSH2311 against E. coli, Klebsiella virus myPSH1235 against K. pneumoniae and Enterobacter virus myPSH1140 against Enterobacter species) showed promising results in totally suppressing the bacterial load within 24 h. The phage cocktails reported in this study consisting of multiple phage types could thus prove to be effective as clinical therapeutic agents (Chan et al., 2013; Kęsik-Szeloch et al., 2013). Ours is one of the rarest studies to use phage cocktails against different genera of bacteria. In vivo activity of these phage cocktails also showed promising results and the results are published elsewhere (Manohar et al., 2018a).\\\\nThe strength of our study is that we established the lytic effectiveness of the phages and their cocktail against clinical bacterial isolates of three genera that are known to be pathogenic. Further we also demonstrated the lytic effectiveness of these phages and their cocktail against pathogenic bacteria that were resistant to two last resort antibiotics, meropenem, and colistin. Thus further experiments can be taken up for in vivo studies in search of a therapeutic treatment against bacterial infections resistant to these drugs.\\\\nConclusion\\\\nThe emergence of multi-drug resistant bacterial strains is a global threat which needs to be addressed by using alternative therapies, such as phage therapy. The three highlighted bacteriophages (Escherichia virus myPSH2311 against E. coli, Klebsiella virus myPSH1235 against K. pneumoniae and Enterobacter virus myPSH1140 against Enterobacter species) and their cocktails showed promising in vitro results that may be extended for use as biocontrol agents in some clinical applications. The development of phage cocktails could be considered for the effective treatment of bacterial infections consisting of broad targets. Hopefully, in future, availability of increased repertoire of phages may allow the development of multi-phage cocktail therapy possible.\\\\nData Availability\\\\nAll the data that support the findings of this study have been deposited in GenBank with the accession numbers MG976803 (Escherichia phage myPSH2311), MG972768 (Klebsiella phage myPSH1235), and MG999954 (Enterobacter phage myPSH1140).\\\\nAuthor Contributions\\\\nPM and RN collected the isolates and did bacteriophage experiments, undertook the laboratory work, and wrote the initial manuscript. AT, CL, and RN interpreted the data. CL and AT revised and edited the manuscript. All authors approved the final version of the manuscript.\\\\nFunding\\\\nThe authors would like to thank Vellore Institute of Technology (VIT) for providing research facilities to complete the work. This research was partly supported by Swedish Research Council with grants to CL (Grant Nos. 2012-02889 and 2017-01327). The authors would also like to thank Council of Scientific and Industrial Research (CSIR) for providing financial assistance (SRF) to support this research.\\\\nConflict of Interest Statement\\\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\\\nSupplementary Material\\\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2019.00574/full#supplementary-material\\\\nFootnotes\\\\n^https://www.ncbi.nlm.nih.gov/genome/?term=escherichia+phage\\\\n^https://www.ncbi.nlm.nih.gov/\\\\n^http://wego.genomics.org.cn\\\\nReferences\\\\nBaig, A., Colom, J., Barrow, P., Schouler, C., Moodley, A., Lavigne, R., et al. (2017). Biology and genomics of an historic therapeutic Escherichia coli bacteriophage collection. Front. Microbiol. 8:1652. doi: 10.3389/fmicb.2017.01652\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCao, F., Wang, X., Wang, L., Li, Z., Che, J., Wang, L., et al. (2015). Evaluation of the efficacy of a bacteriophage in the treatment of pneumonia induced by multidrug resistance Klebsiella pneumoniae in mice. Biomed. Res. Int. 2015:752930. doi: 10.1155/2015/752930\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nCarter, C. D., Parks, A., Abuladze, T., Li, M., Woolston, J., Magnone, J., et al. (2012). Bacteriophage cocktail significantly reduces Escherichia coli O157: H7 contamination of lettuce and beef, but does not protect against recontamination. Bacteriophage 2, 178–185. doi: 10.4161/bact.22825\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nChaturongakul, S., and Ounjai, P. (2014). Phage–host interplay: examples from tailed phages and Gram-negative bacterial pathogens. Front. Microbiol. 5:442. doi: 10.3389/fmicb.2014.00442\\\\nCrossRef Full Text | Google Scholar\\\\nChibani-Chennoufi, S., Bruttin, A., Dillmann, M. L., and Brüssow, H. (2004). Phage-host interaction: an ecological perspective. J. Bacteriol. 186, 3677–3686. doi: 10.1128/JB.186.12.3677-3686.2004\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nClinical and Laboratory Standards Institute [CLSI] (2018). Performance Standards for Antimicrobial Susceptibility Testing; Twenty-Fifth Informational Supplements. Approved Standard M100-S28. Wayne, PA: Clinical and Laboratory Standards Institute [CLSI].\\\\nDalmasso, M., Strain, R., Neve, H., Franz, C. M., Cousin, F. J., Ross, R. P., et al. (2016). Three new Escherichia coli phages from the human gut show promising potential for phage therapy. PLoS One 11:e0156773. doi: 10.1371/journal.pone.0156773\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHamdi, S., Rousseau, G. M., Labrie, S. J., Tremblay, D. M., Kourda, R. S., Slama, K. B., et al. (2017). Characterization of two polyvalent phages infecting Enterobacteriaceae. Sci. Rep. 7:40349. doi: 10.1038/srep40349\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nHung, C. H., Kuo, C. F., Wang, C. H., Wu, C. M., and Tsao, N. (2011). Experimental phage therapy in treating Klebsiella pneumoniae-mediated liver abscesses and bacteremia in mice. Antimicrob. Agents Chemother. 55, 1358–1365. doi: 10.1128/AAC.01123-10\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKęsik-Szeloch, A., Drulis-Kawa, Z., Weber-Dęabrowska, B., Kassner, J., Majkowska-Skrobek, G., Augustyniak, D., et al. (2013). Characterising the biology of novel lytic bacteriophages infecting multidrug resistant Klebsiella pneumoniae. Virol. J. 10:100. doi: 10.1186/1743-422X-10-100\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nKrupovic, M., Dutilh, B. E., Adriaenssens, E. M., Wittmann, J., Vogensen, F. K., Sullivan, M. B., et al. (2016). Taxonomy of prokaryotic viruses: update from the ICTV bacterial and archaeal viruses subcommittee. Arch. Virol. 161, 1095–1099. doi: 10.1007/s00705-015-2728-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLi, E., Wei, X., Ma, Y., Yin, Z., Li, H., Lin, W., et al. (2016a). Isolation and characterization of a bacteriophage phiEap-2 infecting multidrug resistant Enterobacter aerogenes. Sci. Rep. 6:28338. doi: 10.1038/srep28338\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLi, E., Zhao, J., Ma, Y., Wei, X., Li, H., Lin, W., et al. (2016b). Characterization of a novel Achromobacter xylosoxidans specific siphoviruse: phiAxp-1. Sci. Rep. 6:21943. doi: 10.1038/srep21943\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nLin, D. M., Koskella, B., and Lin, H. C. (2017). Phage therapy: an alternative to antibiotics in the age of multi-drug resistance. World J. Gastrointest. Pharmacol. Therapeut. 8:162. doi: 10.4292/wjgpt.v8.i3.162\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nManohar, P., Nachimuthu, R., and Lopes, B. S. (2018a). The therapeutic potential of bacteriophages targeting gram-negative bacteria using Galleria mellonella infection model. BMC Microbiol. 18:97. doi: 10.1186/s12866-018-1234-4\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nManohar, P., Tamhankar, A. J., Lundborg, C. S., and Ramesh, N. (2018b). Isolation, characterization and in vivo efficacy of Escherichia phage myPSH1131. PLoS One 13:e0206278. doi: 10.1371/journal.pone.0206278\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMcVay, C. S., Velásquez, M., and Fralick, J. A. (2007). Phage therapy of Pseudomonas aeruginosa infection in a mouse burn wound model. Antimicrob. Agents Chemother. 51, 1934–1938. doi: 10.1128/AAC.01028-06\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMendes, J. J., Leandro, C., Mottola, C., Barbosa, R., Silva, F. A., Oliveira, M., et al. (2014). In vitro design of a novel lytic bacteriophage cocktail with therapeutic potential against organisms causing diabetic foot infections. J. Med. Microbiol. 63, 1055–1065. doi: 10.1099/jmm.0.071753-0\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMirzaei, M. K., Eriksson, H., Kasuga, K., Haggård-Ljungquist, E., and Nilsson, A. S. (2014). Genomic, proteomic, morphological, and phylogenetic analyses of vB_EcoP_SU10, a podoviridae phage with C3 morphology. PLoS One 9:e116294. doi: 10.1371/journal.pone.0116294\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nMirzaei, M. K., and Nilsson, A. S. (2015). Isolation of phages for phage therapy: a comparison of spot tests and efficiency of plating analyses for determination of host range and efficacy. PLoS One 10:e0118557. doi: 10.1371/journal.pone.0118557\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPan, Y. J., Lin, T. L., Chen, C. C., Tsai, Y. T., Cheng, Y. H., Chen, Y. Y., et al. (2017). Klebsiella phage ΦK64-1 encodes multiple depolymerases for multiple host capsular types. J. Virol. 91:e2457–6. doi: 10.1128/JVI.02457-16\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPereira, S., Pereira, C., Santos, L., Klumpp, J., and Almeida, A. (2016). Potential of phage cocktails in the inactivation of Enterobacter cloacae-an in vitro study in a buffer solution and in urine samples. Virus Res. 211, 199–208. doi: 10.1016/j.virusres.2015.10.025\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nPetrov, V. M., Ratnayaka, S., Nolan, J. M., Miller, E. S., and Karam, J. D. (2010). Genomes of the T4-related bacteriophages as windows on microbial genome evolution. Virol. J. 7:292. doi: 10.1186/1743-422X-7-292\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nRios, A. C., Moutinho, C. G., Pinto, F. C., Del Fiol, F. S., Jozala, A., Chaud, M. V., et al. (2016). Alternatives to overcoming bacterial resistances: state-of-the-art. Microbiol. Res. 191, 51–80. doi: 10.1016/j.micres.2016.04.008\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nSchlossberg, D. (ed.) (2015). Clinical Infectious Disease. Cambridge: Cambridge University Press. doi: 10.1017/CBO9781139855952\\\\nCrossRef Full Text | Google Scholar\\\\nSolovieva, E. V., Myakinina, V. P., Kislichkina, A. A., Krasilnikova, V. M., Verevkin, V. V., Mochalov, V. V., et al. (2018). Comparative genome analysis of novel podoviruses lytic for hypermucoviscous Klebsiella pneumoniae of K1, K2, and K57 capsular types. Virus Res. 243, 10–18. doi: 10.1016/j.virusres.2017.09.026\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\\\nVentola, C. L. (2015). The antibiotic resistance crisis: part 1: causes and threats. Pharm. Therapeut. 40, 277–283.\\\\nPubMed Abstract | Google Scholar\\\\nYen, M., Cairns, L. S., and Camilli, A. (2017). A cocktail of three virulent bacteriophages prevents Vibrio cholerae infection in animal models. Nat. Commun. 8:14187. doi: 10.1038/ncomms14187\\\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\\n}]'"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "library.documents[0].text"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### First load data to a dictionary"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "    # Replace 'your_file.txt' with the path to your text file\n",
    "file_path = '/workspaces/ml-learning/src/phages/notebooks/data/dataset_phages-microbiology-frontiers_2024-01-23_08-51-12-772.json'\n",
    "\n",
    "\n",
    "\n",
    "def jsonToDict(file_path: str)-> dict:\n",
    "\n",
    "\n",
    "    # Read the JSON object from the file\n",
    "    with open(file_path, 'r') as file:\n",
    "        data = file.read()\n",
    "\n",
    "    # Parse the JSON content\n",
    "    try:\n",
    "        json_data = json.loads(data)\n",
    "    except json.JSONDecodeError as e:\n",
    "        print(f\"Error parsing JSON: {e}\")\n",
    "        json_data = None\n",
    "\n",
    "    # Output the parsed data\n",
    "    if json_data is not None:\n",
    "        print(json.dumps(json_data, indent=4))\n",
    "\n",
    "    return json_data\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research#\",\n",
      "            \"loadedTime\": \"2024-01-23T08:48:06.942Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 0,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research#\",\n",
      "            \"title\": \"Articles | Frontiers\",\n",
      "            \"description\": null,\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Original Research\\nPublished on 30 Apr 2018\\nTwo New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2018.00850\\n7,708 views \\n45 citations \\nOriginal Research\\nPublished on 4 Jan 2018\\nMetavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community\\nChinmay V. Tikhe\\nClaudia Husseneder\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2017.02548\\n13,451 views \\n31 citations \\nOriginal Research\\nPublished on 28 Sep 2017\\nNewly Isolated Bacteriophages from the Podoviridae, Siphoviridae, and Myoviridae Families Have Variable Effects on Putative Novel Dickeya spp.\\n\\u0160pela Ali\\u010d\\nTina Nagli\\u010d\\nMagda Tu\\u0161ek-\\u017dnidari\\u010d\\nMaja Ravnikar\\nNejc Ra\\u010dki\\nMatja\\u017e Peterka\\nTanja Dreo\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2017.01870\\n9,561 views \\n27 citations \\nOriginal Research\\nPublished on 2 Apr 2019\\nDiscovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization\\nYen-Te Liao\\nXincheng Sun\\nIrwin A. Quintela\\nDavid F. Bridges\\nFang Liu\\nYujie Zhang\\nAlexandra Salvador\\nVivian C. H. Wu\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2019.00627\\n4,555 views \\n13 citations \\nOriginal Research\\nPublished on 10 Mar 2021\\nIsolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples\\nChia Wanq Tan\\nYaya Rukayadi\\nHanan Hasan\\nNoor-Azira Abdul-Mutalib\\nNuzul Noorahya Jambari\\nHirofumi Hara\\nTze Young Thung\\nEpeng Lee\\nSon Radu\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2021.616548\\n12,006 views \\n25 citations \\nOriginal Research\\nPublished on 23 Jul 2019\\nA Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia\\nRuchira Sharma\\nBrittany A. Pielstick\\nKimberly A. Bell\\nTanner B. Nieman\\nOlivia A. Stubbs\\nEdward L. Yeates\\nDavid A. Baltrus\\nJulianne H. Grose\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2019.01533\\n16,064 views \\n34 citations \\nOriginal Research\\nPublished on 13 Feb 2018\\nIdentification and Characterization of T5-Like Bacteriophages Representing Two Novel Subgroups from Food Products\\nDomonkos Sv\\u00e1b\\nLinda Falgenhauer\\nManfred Rohde\\nJudit Szab\\u00f3\\nTrinad Chakraborty\\nIstv\\u00e1n T\\u00f3th\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2018.00202\\n6,673 views \\n32 citations \\nOriginal Research\\nPublished on 26 Apr 2022\\nOptimisation and Application of a Novel Method to Identify Bacteriophages in Maternal Milk and Infant Stool Identifies Host-Phage Communities Within Preterm Infant Gut\\nGregory R. Young\\nWen C. Yew\\nAndrew Nelson\\nSimon H. Bridge\\nJanet E. Berrington\\nNicholas D. Embleton\\nDarren L. Smith\\nFrontiers in Pediatrics\\ndoi 10.3389/fped.2022.856520\\n2,547 views \\n2 citations \\nOriginal Research\\nPublished on 16 Nov 2022\\nBacteriophage-based decontamination to control environmental colonization by Staphylococcus capitis in neonatal intensive care units: An in vitro proof-of-concept\\nMarie Chavignon\\nCamille Kolenda\\nMathieu Medina\\nM\\u00e9lanie Bonhomme\\nLeslie Blazere\\nTiphaine Legendre\\nAnne Tristan\\nFr\\u00e9d\\u00e9ric Laurent\\nMarine Butin\\nFrontiers in Cellular and Infection Microbiology\\ndoi 10.3389/fcimb.2022.1060825\\n2,238 views \\n1 citations \\nOriginal Research\\nPublished on 2 May 2018\\nBacteriophage Infectivity Against Pseudomonas aeruginosa in Saline Conditions\\nGiantommaso Scarascia\\nScott A. Yap\\nAnna H. Kaksonen\\nPei-Ying Hong\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2018.00875\\n8,420 views \\n11 citations \\nOriginal Research\\nPublished on 6 Oct 2020\\nIdentification and Characterization of New Bacteriophages to Control Multidrug-Resistant Pseudomonas aeruginosa Biofilm on Endotracheal Tubes\\nViviane C. Oliveira\\nFelipe L. Bim\\nRachel M. Monteiro\\nAna Paula Macedo\\nEmerson S. Santos\\nCl\\u00e1udia H. Silva-Lovato\\nHelena F. O. Paranhos\\nLu\\u00eds D. R. Melo\\nS\\u00edlvio B. Santos\\nEvandro Watanabe\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2020.580779\\n9,906 views \\n24 citations \\nOriginal Research\\nPublished on 23 Jan 2020\\nBacteriophages as an Up-and-Coming Alternative to the Use of Sulfur Dioxide in Winemaking\\nGustavo Cordero-Bueso\\nJavier Moraga\\nMar\\u00eda R\\u00edos-Carrasco\\nMarina Ruiz-Mu\\u00f1oz\\nJes\\u00fas Manuel Cantoral\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2019.02931\\n5,238 views \\n3 citations \\nOriginal Research\\nPublished on 30 Aug 2017\\nEnvironmental Bacteriophages of the Emerging Enterobacterial Phytopathogen, Dickeya solani, Show Genomic Conservation and Capacity for Horizontal Gene Transfer between Their Bacterial Hosts\\nAndrew Day\\nJiyoon Ahn\\nXinzhe Fang\\nGeorge P. C. Salmond\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2017.01654\\n3,267 views \\n15 citations \\nOriginal Research\\nPublished on 12 Sep 2018\\nJumbo Bacteriophages Are Represented Within an Increasing Diversity of Environmental Viruses Infecting the Emerging Phytopathogen, Dickeya solani\\nAndrew Day\\nJiyoon Ahn\\nGeorge P. C. Salmond\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2018.02169\\n4,563 views \\n17 citations \\nOriginal Research\\nPublished on 22 Dec 2022\\nA lytic bacteriophage isolate reduced Clostridium perfringens induced lesions in necrotic enteritis challenged broilers\\nChake Keerqin\\nKatherine McGlashan\\nThi Thu Hao Van\\nHelene N. Chinivasagam\\nRobert J. Moore\\nMingan Choct\\nShu-Biao Wu\\nFrontiers in Veterinary Science\\ndoi 10.3389/fvets.2022.1058115\\n1,860 views \\n2 citations \\nOriginal Research\\nPublished on 21 Mar 2019\\nTherapeutic Characterization and Efficacy of Bacteriophage Cocktails Infecting Escherichia coli, Klebsiella pneumoniae, and Enterobacter Species\\nPrasanth Manohar\\nAshok J. Tamhankar\\nCecilia Stalsby Lundborg\\nRamesh Nachimuthu\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2019.00574\\n20,572 views \\n108 citations \\nOriginal Research\\nPublished on 28 Jun 2018\\nVectorized Delivery of Alpha-GalactosylCeramide and Tumor Antigen on Filamentous Bacteriophage fd Induces Protective Immunity by Enhancing Tumor-Specific T Cell Response\\nRossella Sartorius\\nLuciana D\\u2019Apice\\nPasquale Barba\\nDeborah Cipria\\nLaura Grauso\\nAdele Cutignano\\nPiergiuseppe De Berardinis\\nFrontiers in Immunology\\ndoi 10.3389/fimmu.2018.01496\\n4,170 views \\n18 citations \\nOriginal Research\\nPublished on 15 Nov 2017\\nPrion-Like Domains in Phagobiota\\nGeorge Tetz\\nVictor Tetz\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2017.02239\\n8,010 views \\n27 citations \\nOriginal Research\\nPublished on 20 Apr 2016\\nGenomics of Three New Bacteriophages Useful in the Biocontrol of Salmonella\\nCarlota Bardina\\nJoan Colom\\nDenis A. Spricigo\\nJennifer Otero\\nMiquel S\\u00e1nchez-Osuna\\nPilar Cort\\u00e9s\\nMontserrat Llagostera\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2016.00545\\n10,249 views \\n38 citations \\nOriginal Research\\nPublished on 6 Apr 2022\\nThe Expansion of a Single Bacteriophage Leads to Bacterial Disturbance in Gut and Reduction of Larval Growth in Musca domestica\\nXinyu Zhang\\nShumin Wang\\nQian Zhang\\nKexin Zhang\\nWenjuan Liu\\nRuiling Zhang\\nZhong Zhang\\nFrontiers in Immunology\\ndoi 10.3389/fimmu.2022.885722\\n2,084 views \\n3 citations \\nOriginal Research\\nPublished on 21 Jul 2023\\nIsolation and characterization of a novel mycobacteriophage Kashi-VT1 infecting Mycobacterium species\\nTanmayee Nayak\\nAnuja Kakkar\\nRakesh Kumar Singh\\nLav Kumar Jaiswal\\nAnand Kumar Singh\\nLouise Temple\\nAnkush Gupta\\nFrontiers in Cellular and Infection Microbiology\\ndoi 10.3389/fcimb.2023.1173894\\n2,274 views \\nOriginal Research\\nPublished on 12 Mar 2013\\nAntiviral effect of cationic compounds on bacteriophages\\nMai Huong Chatain-Ly\\nSaliha Moussaoui\\nVeronique Rigobello\\nYann Demarigny\\nAnnabelle Vera\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2013.00046\\n8,076 views \\n47 citations \\nOriginal Research\\nPublished on 4 Apr 2019\\nBiodistribution of Liposome-Encapsulated Bacteriophages and Their Transcytosis During Oral Phage Therapy\\nJennifer Otero\\nAlba Garc\\u00eda-Rodr\\u00edguez\\nMary Cano-Sarabia\\nDaniel Maspoch\\nRicard Marcos\\nPilar Cort\\u00e9s\\nMontserrat Llagostera\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2019.00689\\n8,053 views \\n43 citations \\nOriginal Research\\nPublished on 8 Nov 2016\\nUltrastructure and Viral Metagenome of Bacteriophages from an Anaerobic Methane Oxidizing Methylomirabilis Bioreactor Enrichment Culture\\nLavinia Gambelli\\nGeert Cremers\\nRob Mesman\\nSimon Guerrero\\nBas E. Dutilh\\nMike S. M. Jetten\\nHuub J. M. Op den Camp\\nLaura van Niftrik\\nFrontiers in Microbiology\\ndoi 10.3389/fmicb.2016.01740\\n7,432 views \\n14 citations\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles?page=2\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles?page=2#\",\n",
      "            \"loadedTime\": \"2024-01-23T08:48:29.060Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles?page=2#\",\n",
      "            \"title\": \"Articles | Frontiers\",\n",
      "            \"description\": null,\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"497,371 Article(s)\\nMost recent\\nMost viewed\\nMost cited\\nFilters\\nAll Domains\\nEngineering\\nHealth\\nHumanities and Social Sciences\\nScience\\nSustainability\\nYoung Minds\\nAny journal\\nAdvanced Optical Technologies\\nFrontiers for Young Minds\\nFrontiers in Acoustics\\nFrontiers in Adolescent Medicine\\nFrontiers in Aerospace Engineering\\nFrontiers in Aging\\nFrontiers in Aging Neuroscience\\nAny sections\\nAbdominal and Pelvic Pain\\nAbdominal Transplantation\\nAcoustic Materials, Noise Control and Sound Perception\\nAcoustic Metamaterials\\nAcoustic Remote Sensing\\nAcoustofluidics\\nAcute Stroke and Interventional Therapies\\nAny article type\\nAddendum\\nBook Review\\nBrief Research Report\\nClassification\\nClinical Case Study\\nClinical Study Protocol\\nClinical Trial\\nSince beginning\\nPast week\\nPast month\\nPast year\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:48:41.819Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850\",\n",
      "            \"title\": \"Frontiers | Two New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii\",\n",
      "            \"description\": \"Two lytic bacteriophages, WCHABP1 and WCHABP12, were recovered from hospital sewage and were able to infect 9 and 12 out of 18 carbapenem-resistant Acinetobacter baumannii clinical strains, which belonged to different clones. Electron microscopy scan showed that both bacteriophages had the similar morphology as those of the Myoviridae family. Whole genomic sequencing revealed 45.4- or 45.8-kb genome with a 37.6% GC content for WCHABP1 and WCHABP12, both of which showed significant DNA sequence similarity with bacteriophages of the Ap22virus genus within the Myoviridae family. Taxonomic analysis was therefore performed following the proposal approved by the International Committee on Taxonomy of Viruses, which confirmed that WCHABP1 and WCHABP12 represented two new species of the Ap22virus genus. No tRNAs but 88 and 89 open reading frames (ORFs) were predicted for the two bacteriophages, among which 22 and 21 had known function and encoded proteins for morphogenesis,packaging, lysis and nucleiotide metabolism. The C-terminal amino acids of the large unit of fiber tail proteins varied between the bacteriophages, which may explain their different host ranges. For most lytic bacteriophages, a set of holin and endolysin are required for lysis. However, no known holin-encoding genes were identified in WCHABP1 and WCHABP12, suggesting that they may use alternative, yet-to-be-identified, novel holins for host cell membrane lysis. To test the efficacy of the bacteriophages in protecting against A. baumannii infection, a Galleria mellonella larva model was used. Only <20% Galleria mellonella larvae survived at 96 h after being infected by carbapenem-resistant A. baumannii strains, from which the two bacteriophages were recovered. With the administration of WCHABP1 and WCHABP12, the survival of larvae increased to 75%, while the treatment of polymyxin B only slightly increased the survival rate to 25%. The isolation of two new lytic bacteriophages in this study could expand our sight on Acinetobacter bacteriophages and may offer new potential therapeutic alternatives against A. baumannii.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nOn 27 February 2017, World Health Organization (WHO) released a list of 12 types of antimicrobial-resistant \\u201cpriority pathogens\\u201d that pose the greatest threat to human health (World Health Organization, 2017). In the list, Acinetobacter along with Pseudomonas and various species of the Enterobacteriaceae (e.g., Klebsiella, Escherichia coli, Serratia, and Proteus) are labeled as \\u201cCritical,\\u201d for which new antimicrobial agents are urgently needed. Acinetobacter baumannii is a non-fermentative, non-motile, catalase-positive, gram-negative bacterium. It is widely dispersed in soil and water as well as the healthcare environment with the ability to cause various infections such as bacteremia and pneumonia (Dijkshoorn et al., 2007; Peleg et al., 2008). In addition to its capability of survival on dry and moist surfaces (Berlau et al., 1999), a variety of intrinsic and acquired mechanisms for antimicrobial resistance and virulence (Di et al., 2011; Perez et al., 2011) have made A. baumannii a successful pathogen worldwide (Perez et al., 2007).\\nOne century ago, since the first day of their discovery, the compelling feature of bacteriophages as a natural enemy of bacteria has drawn attention of early researchers to exploit them as a mean to treat bacterial infections (d'herelle, 1917; Bruynoghe and Maisin, 1921; Helvoort, 2001). However, the successive discovery of penicillin and other antibiotics soon put bacteriophages on the shelf, with exception in the Soviet Union and Poland (Kutter et al., 2015). The emergence of multi-drug resistant (MDR) bacteria calls for alternatives of antibiotics and bacteriophages therefore gain interests for extensive studies again (Smith and Huggins, 1983; Soothill, 1992).\\nAs a potential alternative to antibiotics and a possible solution for impending infections caused by MDR bacteria, there are still constraints for bacteriophages to be widely applied in real clinical practice despite the presence of reported successful cases (Abubakar et al., 2016). The main challenges for such a bacteriophage therapy include that the relatively narrow lytic range against bacterial strains and the proneness of bacteria to develop resistance. To tackle the challenges, multiple lytic bacteriophages have been used in combination as cocktails (Merabishvili et al., 2009). The success of cocktail bacteriophage therapy against predominantly MDR bacteria such as A. baumannii relies on isolation of novel lytic bacteriophages. Here we report two newly isolated lytic bacteriophages named WCHABP1 and WCHABP12 of the Ap22virus genus within the Myoviridae family against carbapenem-resistant A. baumannii clinical isolates.\\nMethods\\nBacterial Strains\\nThis study included a total of 18 nonduplicate carbapenem-resistant (MICs of imipenem \\u2265 8 mg/L) but polymyxin B-susceptible (MICs <2 mg/L) A. baumannii clinical strains recovered from blood, sputum, urine, ascites, bronchoalveolar lavage fluid (BALF), and drainages of different patients at West China Hospital, Sichuan University, China, from April to June 2016 (Table 1) as hosts for bacteriophage isolation. The species identification of them was established by partially sequencing the recA gene as described previously (Krawczyk et al., 2002). These isolates belonged to 18 different clones as determined using a pulsed field gel electrophoresis (PFGE) analysis (data not shown). Clones were defined for isolates with an 80% or above pattern similarity. Antimicrobial susceptibility testing was performed using the Vitek II automated microbiological system (bioM\\u00e9rieux, Marcy-l'\\u00c9toile, France) or broth microdilution method of the Clinical and Laboratory Standards Institute (CLSI) (CLSI, 2017) (for colistin, imipenem, polymycin B, and tigecycline). Breakpoints defined by U. S. Food and Drug Administration were used for tigecycline; otherwise, those defined by CLSI were applied. The results are shown in Table S1.\\nTable 1. Host ranges of WCHABP1 and WCHABP12.\\nBacteriophage Isolation, Titering, and Concentration\\nBacteriophages were isolated from sewage samples collected at the influx of the wastewater treatment plant at West China Hospital in September 2016. The isolation and propagation of bacteriophages were performed as described previously (Merabishvili et al., 2009) and titering was carried out using the agar overlay method (Kropinski et al., 2009). PEG 8000/NaCl precipitation was used to concentrate bacteriophages for further use as described previously (Peng et al., 2014).\\nDetermination of the Host Range\\nThe bacteriophage stock was diluted 1:106 with Luria-Bertani (LB) broth (Hopebio, Qingdao, China) and was adjusted to a final titer of 1 \\u00d7 104 pfu/ml. Aliquots of 10 \\u03bcl diluted bacteriophages were mixed with tested bacterial host strains or the original strains from which the bacteriophages were isolated. The mixtures were subjected to agar overlay method. The efficiency of plating (EOP) was calculated by dividing the bacteriophage titer of tested strain by that of original strain. A particular bacteriophage-bacterial strain combination was considered as high, medium and low production efficiency if the average EOP value was \\u2265 0.5, \\u2265 0.1 to < 0.5, and \\u2265 0.001 to < 0.1, respectively; while a < 0.001 average EOP value suggests inefficient production (Khan and Nilsson, 2015).\\nElectron Microscopy\\nThe concentrated bacteriophage stocks were negatively stained with phosphotungstic acid (PTA) and were then imaged using a H-600 II transmission electron microscope at 75 kV (Hitachi, Tokyo, Japan) to acquire morphological information of single bacteriophage particles.\\nMultiplicity of Infection (MOI) Assay\\nBacteriophage stocks were diluted by LB broth (Hopebio, Qingdao, China) into 10-fold series. Aliquots of each dilution were mixed with their corresponding host cultures at 108 cfu/ml to result in different ratio (from 10\\u22124 to 10 pfu/cfu) and were incubated at 37\\u00b0C for 4 h. Bacteriophage progenies of each mixture ratio were titered to determine the highest production as the optimal MOI (Peng et al., 2014).\\nAdsorption Rate Assay\\nAliquots of bacteriophages were incubated with host strain cultures at the optimal MOI under 37\\u00b0C. Samples (100 \\u03bcl) were collected at 0, 3, 6, 9, 12, and 15 min, respectively, and were centrifuged to remove the bacterial cells. The supernatants were titered for determination of unabsorbed bacteriophages at each time interval (Peng et al., 2014).\\nOne-Step Growth Curve\\nBacteriophages (10 \\u03bcl at a titer of 1 \\u00d7 109 pfu/ml) were mixed with 5 ml of their host strain cultures at about 2 \\u00d7 107 cfu/ml to reach a MOI of 0.1. The mixture was incubated at 37\\u00b0C for 10 min to allow the complete adsorption and was then centrifuged at 15,000 g for 1 min to remove the unabsorbed bacteriophage particles by discarding the supernatants. The pellets were resuspended in LB broth and were incubated at 37\\u00b0C. Aliquots (100 \\u03bcl) were taken every 10 min from the beginning to 90 min and were titered for free bacteriophages after being centrifuged at 15,000 g for 1 min to remove bacterial cells. An additional 100 \\u03bcl aliquot was taken at the beginning (0 min). A half of the aliquot (50 \\u03bcl) was centrifuged and the remaining half was not centrifuged, both of which were titered for bacteriophages. Infected cells were calculated by subtracting the number of free bacteriophages of the centrifuged part from the number of bacteriophages of the un-centrifuged part, then the burst size could be calculated as dividing the maximal progeny counts by the number of infected cells (Peng et al., 2014).\\nBacteriophage Genome Sequencing and Bioinformatics Analysis\\nBacteriophage DNA was prepared using the standard phenol-chloroform extraction (Mandell and Hershey, 1960) after concentration. Whole genome sequencing was performed using the HiSeq 2500 Sequencer (Illumina, San Diego, CA, USA) with 150-bp paired-end (the final coverage was ~200x). De novo assembly was performed using SPAdes v3.10.1 (Bankevich et al., 2012) with auto-cutoff and careful mode. Annotation of the genomic sequence was carried out using Prokka v1.11 (Seemann, 2014) followed by manually confirmation via running BLASTp and PSI-BLAST (http://blast.ncbi.nlm.nih.gov/) (Altschul et al., 1990, 1997) against the non-redundant protein database with a significant E-value of <10\\u22123. For annotated proteins, the conserved protein domains/motif and additional function inference were detected using InterProScan (http://www.ebi.ac.uk/interpro/search/sequence-search) (Quevillon et al., 2005), HMMER (https://www.ebi.ac.uk/Tools/hmmer), and Conserved Domain Database (https://www.ncbi.nlm.nih.gov/cdd) (Marchlerbauer et al., 2011). The ExPASy server (http://us.expasy.org/tools/protparam.html) was used to predict molecular weight and isoelectric point. Prediction of transmembrane helices was performed using TMHMM 2.0 (Krogh et al., 2001) and signal peptides were screened using SignalP (Petersen et al., 2011). Potential tRNA genes were identified using tRNAscan-SE (Lowe, 1997) and ARAGON (Laslett and Canback, 2004).\\nThe comparative genomic analysis between our bacteriophages and those deposited in GenBank was performed using BLASTn and BLAST Ring Image Generator (BRIG) (Alikhan et al., 2011). The newest proposal of International Committee on Taxonomy of Viruses (ICTV) (https://talk.ictvonline.org/files/ictv_official_taxonomy_updates_since_the_8th_report/m/prokaryote-official/5901) (Adams et al., 2017) was taken as the reference to determine the taxonomy of the bacteriophages. Mauve (http://asap.ahabs.wisc.edu/mauve/) and CoreGenes (Zafar et al., 2002; Mahadevan et al., 2009a,b) were used to compare the genomic organization and core genes for a given group of bacteriophages. Phylogenetic tree for taxonomic analysis was constructed via phylogeny.fr with \\u201cOne Click mode\\u201d using MUSCLE for multiple alignments, PhyML for phylogeny, and Gblocks for eliminating poorly aligned positions and divergent regions (Anisimova and Gascuel, 2006).\\nNucleotide accession no. Genome sequences of bacteriophages WCHABP1 and WCHABP12 in the present study have been deposited into GenBank under accession no. KY829116 and KY670595.\\nAnalysis of the Genome Ends\\nGenome ends were determined as described previously (Casjens and Gilcrease, 2009). Briefly, 1 \\u03bcg of each bacteriophage genome DNA was digested with the restriction enzyme HindIII (TaKaRa, Dalian, China). The mixture was heated at 80\\u00b0C for 15 min to inactivate the digestion and was divided into two equal aliquots. One was rapidly chilled in ice water bath and the other was left to slowly cool to the room temperature. The resulting DNA bands were separated in a 1% agarose gel by electrophoresis. In parallel, 1 \\u03bcg of Lambda DNA standard digested with the restriction enzyme EcoRV (TaKaRa) was used as control.\\nSodium Dodecyl Sulfate\\u2013Polyacrylamide Gel Electrophoresis (SDS-PAGE) Analysis\\nBacteriophage particles were boiled in loading buffer (50 mM tris-HCl, 2% Sodium dodecyl sulfate, 0.1% bromophenol blue, 10% glycerol, and 1% \\u03b2-mercaptoethanol) for 5 min. Denatured proteins were then separated using sodium dodecyl sulfate\\u2013polyacrylamide gel electrophoresis (SDS-PAGE) with a 5% concentration gel and a 12% separation gel and were stained with Coomassie Blue G-250 as described previously (Boulanger, 2009).\\nGalleria mellonella Bacteriophage Therapy Assay\\nG. mellonella infected by A. baumannii was used to study in vivo antibacterial efficacy of our bacteriophages (Peleg and Jara, 2009). Larvae of G. mellonella of 250\\u2013350 mg in weight (Huiyude, Tianjin, China) were stored at 4\\u00b0C and were used within 1 week after delivery. Briefly, 16 randomly chosen larvae were used for each group. Host bacterial cultures of strain Ab1186 and Ab1262, from which the two bacteriophages were recovered, were washed by phosphate-buffered saline (PBS; Beyotime, Shanghai, China) and were then diluted 1:10 to an appropriate cell density (107 cfu/ml) as determined using a McFarland turbidimetry and direct plate colony counting. Aliquots of 10 \\u03bcl PBS-washed bacterial cultures (105 cfu) were injected into the hemocoel of each larva through the last left proleg using a microsyringe (Gaoge, Shanghai, China), which led to a pre-determined LD80 dose for each inoculum (105 cfu/larva). Bacteriophage stocks were re-concentrated as described above and were re-suspended and diluted with PBS to the optimal MOI titer (for host cell at 105 cfu/ml, the bacteriophage dosage was 104 pfu/ml to reach a final MOI of 0.1 pfu/cfu). Aliquots of 10 \\u03bcl bacteriophage suspensions (104 pfu/larva) were then injected into the larva via a different proleg within 30 min from the bacterial inoculation. In parallel, larvae injected with bacterial cells at the same concentration were treated with polymyxin B (Meilun, Dalian, China) at 2.5 mg/kg (Hornsey and Wareham, 2011) to compare the therapeutic efficacy between antibiotics and bacteriophages. For each bacteriophage, four control groups were set up including larvae infected with bacterial cells and treated with PBS solution to observe the bacterial virulence (group 1), those injected with either the bacteriophage suspension or polymyxin B to assess their potential toxicity (group 2 and 3); those injected with PBS only to observe the potential physical trauma from injection (group 4). Larvae were then incubated in plastic containers at 37\\u00b0C and the number of dead, determining as no movement under touch, was counted at 24 h intervals up to 96 h after the incubation. When more than two larvae died in any negative control (group 2, 3, and 4) at the end of observation, the assay was considered invalid and was repeated. Assays were performed in triplicate using different batches of larvae.\\nThe statistical analysis was performed by GraphPad Prism v.7.0 (Software Inc., La Jolla, CA, USA) to plot the survival curves with the Kaplan-Meier method following a log-rank test to calculate the differences in survival. A P-value of < 0.05 was considered to be statistically significant.\\nResults\\nTwo Bacteriophages of the Myoviridae Family Were Recovered\\nTwo bacteriophages, designated WCHABP1 and WCHABP12, were obtained from hospital sewage with the ability to infect 9 and 12 out of the 18 host strains, exhibiting that both bacteriophages can infect multiple strains of A. baumannii with varied efficiency (Table 1). Upon infecting their original host strain, clear plaques of 8\\u20139 and 5\\u20137 mm in diameter with all surrounded by a 1\\u20133 mm halo were formed by bacteriophage WCHABP1 and WCHABP12, respectively.\\nBoth bacteriophages produced the highest progeny at MOI of 0.1 (1.55 \\u00d7 1010 pfu/ml for WCHABP1 and 2.2 \\u00d7 1010 for WCHABP12), which was therefore used in subsequent experiments. The adsorption rate of them was comparable where more than 99% of bacteriophage particles could adsorb onto the host cell within 10 min (Figure 1). After the adsorption, for both WCHABP1 and WCHABP12, the eclipse period was between 10 and 20 min and the plateau was reached at 60 min with a slightly different final titer (1.5 \\u00d7 109 pfu/ml for WCHABP1 and 2.1 \\u00d7 109 pfu/ml for WCHABP1; Figure 1). Considering that the initial number of infected cells were 1.1 \\u00d7 107 cfu/ml for WCHABP1 and 1.2 \\u00d7 107 cfu/ml for WCHABP12, the burst size of the two bacteriophages was 136 and 175 pfu per infected cell, respectively.\\nFigure 1. Growth and lytic characteristics. (A) adsorption rate; (B) One-step growth curve. WCHABP1 and WCHABP12 are labeled by asterisk and circle, respectively.\\nThe electron microscopy revealed that both WCHABP1 and WCHABP12 had a ~75 nm icosahedral head and a ~105 nm tail, on which crossed striation could be seen (Figure 2). The morphologic features suggested that both WCHABP1 and WCHABP12 belong to the Myoviridae family.\\nFigure 2. Electron microscopy of bacteriophage morphology. (a) WCHABP1; (b) WCHABP12. The black scale bar represents 50 nm.\\nThe Two Bacteriophages Represent Two New Species of the Ap22virus Genus Within the Myoviridae Family\\nGenomes of the WCHABP1 and WCHABP12 were 45,888 and 45,415 bp in length with the same GC content of 37.6 mol%. The number of ORFs predicted was 88 for WCHABP1 and 89 for WCHABP12, respectively. Genomes of WCHABP1 and WCHABP12 were highly similar with each other (88.0% overall DNA sequence identity) and had 51.0 to 56.0% overall DNA sequence identity to a set of bacteriophages (Table 2) belonging to the Ap22virus genus of the Myoviridae family.\\nTable 2. Genomic properties of WCHABP1 and WCHABP12 compared with other bacteriophages of the Ap22virus genus.\\nTo investigate the taxonomy of WCHABP1 and WCHABP12, we followed the approach of the recently approved taxonomic proposal for the Ap22virus genus (Adams et al., 2017). Bacteriophages of this genus possess genome of ~45 kb (around 37.6 mol% G+C content) and encode ORFs close to 85 in number with no tRNAs. Members of the Ap22virus genus had an average of 53% overall DNA sequence identity and 64% homologous proteins compared with the reference species bacteriophage AP22. WCHABP1 and WCHABP12 exhibited a higher overall DNA sequence identity (58 and 62%) and homologous proteins (74 and 75%) with AP22, carrying almost the same number of ORFs as AP22 (88 and 89 vs. 89). The multiple genome alignment of WCHABP1, WCHABP12, and bacteriophages of the Ap22virus genus showed that all of these genomes were in a resembling gene cluster and order (Figure S1). Phylogenetic trees constructed with bacteriophage major capsid and baseplate J-like protein revealed that both WCHABP1 and WCHABP12 were clustered together within the Ap22virus genus (Figure S2). All above findings suggest that WCHABP1 and WCHABP12 belonged to the Ap22virus genus. In addition, WCHABP1 and WCHABP12 had <95.0% identity between each other and with any other bacteriophages of the Ap22virus genus. Therefore, WCHABP1 and WCHABP12 present two new species of the Ap22virus genus.\\nGenomes of the Two Bacteriophages Encode Proteins for Morphogenesis, Genome Packaging, Nucleotide Metabolism, and Host Lysis\\nAmong ORFs of WCHABP1 and WCHABP1, 22 (WCHABP1) and 21 (WCHABP12) could be assigned to known function including morphogenesis, genome packaging, nucleotide metabolism, and host lysis. Neither tRNAs nor genes involving in the lysogenic process such as integrase and repressor were identified in both bacteriophages (Table 2, Figure 3).\\nFigure 3. Multiple genome alignments of WCHABP1, WCHABP12, and bacteriophages of the Ap22virus genus. (A) the genome of WCHABP1 is used as the reference. (B) the genome of WCHABP12 is used as the reference. The alignment is a pairwise BLASTn performed using BRIG. Accession numbers of genomes of bacteriophages of the Ap22virus genus are listed in Table 2.\\nMorphology Module\\nThe SDS-PAGE analysis showed similar protein patterns between WCHABP1 and WCHABP12 (Figure S3). The protein bands mainly converged at 15, 35, 40\\u201355, and 70 kDa, corresponding to the respective capsid, portal and tail fiber proteins of the two bacteriophages in light of the predicted molecular weight.\\nBioinformatics analysis predicted seven ORFs encoding structural proteins for the two bacteriophages. These structural proteins included two capsid proteins (gp27 and gp43 of WCHABP1; gp66 and gp82 of WCHABP12), one portal protein (gp44 of WCHABP1; gp65 of WCHABP12), two baseplate proteins (gp08 and gp10 of WCHABP1; gp11 and gp13 of WCHABP12), and two tail fiber proteins (gp05 and gp06 of WCHABP1; gp15 and gp16 of WCHABP12). The structural proteins encoded by each of the two bacteriophage genomes are highly similar (>60% overall amino acids identity, a detailed list is shown in Table 3) with the exception of the large tail fiber subunits (gp05 of WCHABP1 and gp16 of WCHABP12). The N-terminal 149 amino acids of the large tail fiber subunit of WCHABP1 and WCHABP12 were of 100% coverage and 76% identity, while the remaining C-terminal amino acids had no significant similarity.\\nTable 3. Genes with known function in bacteriophage WCHABP1 and WCHABP12.\\nIn addition, both WCHABP1 and WCHABP12 had a gene encoding a lysozyme-containing protein, which is also possessed by other bacteriophages of the Ap22virus genus. The lysozyme is a glycoside hydrolase that catalyzes the peptidoglycan of the bacterial cell wall (Strynadka and James, 1996). The lysozyme-containing protein lies close to the baseplate protein and tail fiber proteins, suggesting that it might be a part of tail. Lysozyme could be released from the inner tail tube to create a small hole in the cell wall peptidoglycan layer that allows the double-stranded DNA (dsDNA) genome to enter the cell (Duda, 2008). Nonetheless, the exact function of the lysozyme-containing protein in the bacteriophages warrants further studies.\\nPackaging Module and Genome Termini\\nTwo ORFs encoding terminase, which is the key component of the packaging machine, were identified in both bacteriophage genomes (gp45 and gp46 of WCHABP1; gp63 and gp64 of WCHABP12). Terminase for bacteriophages usually comprises two subunits, where the large subunit translocates bacteriophage DNA into empty capsids and cuts it at a unique and precise dsDNA sequence to accomplish the packaging process, while the small subunit contributes to binding of the packaging initiation site and regulates the ATPase activity for the large subunit (Feiss and Rao, 2012). Amino acid sequences of terminase subunits between WCHABP1 and WCHABP12 were identical (Table 3). The restriction map of genomes of WCHABP1 and WCHABP12 showed no change in band patterns after endonuclease restriction with rapid or slow cooling, suggesting the absence of cohesive ends in their genomes (Figure S4).\\nNucleotide Metabolism Module\\nBoth WCHABP1 and WCHABP12 had 10 genes involving in genome replication, transcription, establishment of infection, and other processes, including a RNA polymerase, a DNA helicase, a transcriptional regulator, a fis family transcriptional regulator, an ERF (essential recombination function) family protein, a nucleoside triphosphate pyrophosphohydrolase, a HNH endonuclease, an endodeoxyribonuclease, a superinfection immunity protein and a bacteriophage antirepressor protein (Table 2). Products of these genes were highly similar (93\\u2013100% overall amino acids identity) between the two bacteriophages except for the nucleotide triphosphate pyrophosphohydrolase (gp89 of WCHABP1 and gp21 of WCHABP12) with a 97% coverage but only 63% amino acid identity. Both gp89 of WCHABP1 and gp21 of WCHABP12 contained an NTP-PPase_u3 domain (also known as MazG domain, CCD accession cd11540), suggesting a similar function. Previous study suggests that the MazG domain-containing proteins might contribute to extending the logarithmic phase of bacterial growth, facilitating the production of bacteriophage progeny via the reactivation of metabolic pathways that are usually suppressed under nutrient starvation (Bryan et al., 2008).\\nBesides the identical HNH endonuclease seen in both WCHABP1 (gp33) and WCHABP12 (gp76), WCHABP1 had another HNH endonuclease (gp02), which is absent from WCHABP12 and any other members of the Ap22virus genus and has a relatively low similarity (31% coverage and 31% identity) to the aforementioned HNH endonuclease of WCHABP1 and WCHABP12. The gp02 product of WCHABP1 consisted of a HNH endonuclease domain (Pfam accession PF13392.5) and an AP2 domain (Pfam accession PF00847.19), while the gp33 product of WCHABP1 and gp76 product of WCHABP12 contained a HNH endonuclease domain (Pfam accession PF13392.5) and a NUMOD4 motif (Pfam accession PF07463.10). The HNH endonuclease domain is involved in the process of homing, while both AP2 domain and NUMOD4 motif play a role in DNA binding (Ohmetakagi and Shinshi, 1995; Sitbon and Pietrokovski, 2003) but without significant amino acid similarity.\\nLysis Control\\nWCHABP1 and WCHABP12 had one gene encoding the endolysin. However, endolysins of the two bacteriophages had no significant similarity in amino acid sequences. The endolysin of WCHABP1 (gp01) had 171 amino acids and contains a glycosyl hydrolase 108 family domain (Pfam accession PF05838.11) with catalytic activity as a N-acetylmuramidase (Stojkovi\\u0107 and Rothman-Denes, 2007) and a peptidoglycan domain for substrate binding (Pfam accession PF09374.9). By contrast, the endolysin of WCHABP12 (gp19) had 202 amino acids with a single domain belonging to the glycoside hydrolase family 19 (Pfam accession PF00182.18), which is able to hydrolyze the \\u03b2-1,4-N-acetyl-D-glucosamine linkages in chitin polymers and leads to the breakage of chitin-containing cell walls (Eijsink et al., 2008). The different types of endolysins between WCHABP1 and WCHABP12 suggest that even closely-related bacteriophages may target different substrates of the host cell wall.\\nMost dsDNA bacteriophages with a genome size larger than 10 kb employ a complete set of a holin-endolysin system to release the phage progeny (Young and White, 2008). The holin first permeabilizes the host cell membrane and endolysin can therefore reach the periplasm to break down the peptidoglycan layer (Young et al., 2000). However, for both WCHABP1 and WCHABP12 no known holin-encoding genes were identified using both a BLASTp search and domain/motif prediction.\\nIn Vivo Therapeutic Efficacy\\nFor larvae infected by carbapenem-resistant A. baumannii strains, the treatment of WCHABP1 and WCHABP12 significantly improved the survival rates (Figure 4 and Table S2). For those infected by strain Ab1186, survival of larvae at 96 h was 75.00 and 18.75% (p < 0.01) with and without the treatment of WCHABP1, respectively. Similarly, for those infected by strain Ab1262, survival was 75 and 12.5% (p < 0.01) with and without the treatment of WCHABP12, respectively. By contrast, the treatment of polymyxin B only slightly increased the survival, which is not statistically significant. With and without the treatment of polymyxin B, the survival rate of those infected by strain Ab1186 was 25 and 18.75% (p = 0.46), respectively, and that of those infected by Ab1262 was 25 and 12.5% (p = 0.47), respectively. The treatment of the combination of bacteriophage and polymyxin B did not further enhance the survival of larvae infected by strain Ab1186 or Ab1262 compared with the treatment of bacteriophage alone (68.75 vs. 75%, p = 0.687).\\nFigure 4. Survival of G. mellonella larvae after infection by carbapenem-resistant A. baumannii clinical strains. (A) WCHABP1; (B) WCHABP12. Ab1186 and Ab1262 are two carbapenem-resistant A. baumannii clinical strains, from which bacteriophages WCHABP1, WCHABP12 were recovered. The detailed survival rates are listed in Table S2.\\nDiscussion\\nThe recent success in treating the infection caused by MDR A. baumannii by topical and systematic administration of varied bacteriophages together with antibiotics once again highlights the potential of bacteriophages as promising alternatives against bacterial infections that are difficult to be treated otherwise (https://www.sciencedaily.com/releases/2017/04/170425124826.htm). Nonetheless, constant works on isolation and characterization of new bacteriophages as well as their in vitro and in vivo test for both safety and therapeutic efficacy are still imperative. In this study, we reported two bacteriophages, WCHABP1 and WCHABP12, belonging to the Ap22virus genus of the Myoviridae family with a large burst size and short lytic life cycle against the carbapenem-resistant A. baumannii host strains. The bacteriophages identified in the present study together with others discovered in previous studies (Lee et al., 2011; Jin et al., 2012; Popova et al., 2012; Ghajavand et al., 2017) may expand our arsenal against A. baumannii.\\nThe injection of bacteriophages WCHABP1 and WCHABP12 did not cause any death of larvae, suggesting that they were not toxic for the larvae. The administration of WCHABP1 and WCHABP12 was able to rescue most larvae infected by carbapenem-resistant A. baumannii susceptible to the lysis of the bacteriophages, suggesting both bacteriophages new potent weapons against A. baumannii. By contrast, the treatment of polymyxin B alone was only able to rescue few larvae infected by these strains that were susceptible to the antibiotic, which is consistent with the findings of another study (Hornsey et al., 2013). A few factors may contribute to the failure of polymyxin B treatment such as emerging tolerance and resistance to the antibiotic after exposure (Li et al., 2006; Harding et al., 2018).\\nFor products encoded by the genomes of WCHABP1, WCHABP12, and other members of the Ap22virus genus, there are several noteworthy features. First, the C-terminal amino acid sequence of the large tail fiber subunit had no significant similarity between WCHABP1 and WCHABP12. Such highly variable C-terminal sequences of large tail fiber subunits are also present among other members of the Ap22virus genus. Previous studies have revealed that the large tail fiber subunit of various bacteriophages including those belonging to the Myoviridae family has a conserved N-terminus and a variable C-terminus, which determines the host specificity by binding the receptor on the surface of host cell (Rakhuba et al., 2010; Kurtb\\u00f6ke, 2012). The variable C-terminal sequence of large tail fiber subunits may therefore account for the different host ranges of bacteriophages of the Ap22virus genus. Second, like those of other bacteriophages of the Ap22virus genus, the large terminase subunit of WCHABP1 and WCHABP12 had a terminase_3 family domain (Pfam accession pfam04466) and were distantly homologous (26% identity with a E-value of 2e-22) with that of bacteriophage SPP1 (GenBank accession NC_004166), which is a well-studied model system for headful DNA packaging mechanism (also known as pac-mechanism; Oliveira et al., 2013). This suggests that both WCHABP1 and WCHABP12 also employ a pac-mechanism for genome packaging and their genomes are circularly permuted and terminally redundant (Casjens and Gilcrease, 2009; Oliveira et al., 2013). Third, surprisingly, unlike the typical holin-endolysin set seen in most bacteriophages, no known holin-encoding genes were identified for bacteriophages of the Ap22virus genus. Two ORFs (gp3 and gp4 of WCHABP1; gp17 and gp18 of WCHABP12) were present between genes encoding endolysin and the large tail fiber subunit and were highly similar among members of the Ap22virus genus. The products encoded by the two ORFs had two and three transmembrane domains as identified using TMHMM. As most holins contain transmembrane domains (Young et al., 2000) and their encoding genes usually locate proximately to endolysin-encoding genes, it is reasonable to hypothesize that the two ORFs may encode a new, yet-to-be-characterized holin. Alternatively, as shown in a previous study there are means other than holin for bacteriophages to permeabilize the host cell membrane via inhibiting the specific host enzymes and impairing peptidoglycan biosynthesis (Bernhardt et al., 2001). Therefore, the exact lysis mechanism employed by bacteriophages of the Ap22virus genus remains unclear and warrants further studies.\\nAlthough genomes of bacteriophages WCHABP1 and WCHABP12 were highly similar, there were several differences, which have been demonstrated above and are summarized here. Endolysins of WCHABP1 and WCHABP12 had no significant similarity in amino acid sequences. WCHABP1 had one more HNH endonuclease than WCHABP12. The C-terminal amino acids of the large unit of tail fiber protein between the two bacteriophages had no significant similarity. In addition, WCHABP1 had three genes without known function that were absent from WCHABP12, while two genes with unknown function of WCHABP12 were not present in WCHABP1.\\nIn conclusion, two new bacteriophages, exhibiting activity to infect and kill multiple carbapenem-resistant A. baumannii clinical strains, were recovered. The two bacteriophages represent two new species of the Ap22virus genus within the Myoviridae family. Administration of the two bacteriophages was effective to treat carbapenem-resistant A. baumannii infections in the G. mellonella larvae model. The findings could expand our sight on Acinetobacter bacteriophages and offer new potential therapeutic alternatives.\\nAuthor Contributions\\nZZ designed the study. WZ performed the experiments and prepared figures. WZ, YF, and ZZ analyzed and interpreted the data. WZ and ZZ wrote the manuscript.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nThis work was supported by grants from the National Natural Science Foundation of China (project no. 81222025, 81572030, and 81661130159) and the Newton Advanced Fellowship, Royal Society, UK (NA150363).\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850/full#supplementary-material\\nReferences\\nAbubakar, S., Suleiman, B. H., Abbagana, B. A., Mustafa, I. A., and Musa, I. A. (2016). Novel uses of bacteriophages in the treatment of human infections and antibiotic resistance. Am. J. Biosci. 4, 34\\u201340. doi: 10.11648/j.ajbio.20160403.13\\nCrossRef Full Text | Google Scholar\\nAdams, M. J., Lefkowitz, E. J., King, A. M. Q., Harrach, B., Harrison, R. L., Knowles, N. J., et al. (2017). Changes to taxonomy and the international code of virus classification and nomenclature ratified by the international committee on taxonomy of viruses (2017). Arch. Virol. 162, 1\\u201334. doi: 10.1007/s00705-017-3358-5\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAlikhan, N. F., Petty, N. K., Ben Zakour, N. L., and Beatson, S. A. (2011). BLAST Ring Image Generator (BRIG): simple prokaryote genome comparisons. BMC Genomics 12:402. doi: 10.1186/1471-2164-12-402\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAltschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990). Basic local alignment search tool. J. Mol. Biol. 215, 403\\u2013410. doi: 10.1016/S0022-2836(05)80360-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAltschul, S. F., Madden, T. L., Sch\\u00e4ffer, A. A., Zhang, J., Zhang, Z., Miller, W., et al. (1997). Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 25, 3389\\u20133402. doi: 10.1093/nar/25.17.3389\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAnisimova, M., and Gascuel, O. (2006). Approximate likelihood-ratio test for branches: a fast, accurate, and powerful alternative. Syst. Biol. 55, 539\\u2013552. doi: 10.1080/10635150600755453\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455\\u2013477. doi: 10.1089/cmb.2012.0021\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBerlau, J., Aucken, H. M., Houang, E., and Pitt, T. L. (1999). Isolation of Acinetobacter spp. including A. baumannii from vegetables: implications for hospital-acquired infections. J. Hosp. Infect. 42, 201\\u2013204. doi: 10.1053/jhin.1999.0602\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBernhardt, T. G., Wang, I. N., Struck, D. K., and Young, R. (2001). A protein antibiotic in the phage Q\\u03b2 virion: diversity in lysis targets. Science 292, 2326\\u20132329. doi: 10.1126/science.1058289\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBoulanger, P. (2009). Purification of bacteriophages and SDS-PAGE analysis of phage structural proteins from ghost particles. Methods Mol. Biol. 502, 227. doi: 10.1007/978-1-60327-565-1_13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBruynoghe, R., and Maisin, J. (1921). Essais de th\\u00e9rapeutique au moyen du bacteriophage. CR Soc. Biol. 85, 1120\\u20131121.\\nBryan, M. J., Burroughs, N. J., Spence, E. M., Clokie, M. R. J., Mann, N. H., and Bryan, S. J. (2008). Evidence for the intense exchange of mazG in marine cyanophages by horizontal gene transfer. PLoS ONE 3:e2048. doi: 10.1371/journal.pone.0002048\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCasjens, S. R., and Gilcrease, E. B. (2009). Determining DNA packaging strategy by analysis of the termini of the chromosomes in tailed-bacteriophage virions. Methods Mol. Biol. 502, 91\\u2013111. doi: 10.1007/978-1-60327-565-1_7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCLSI (2017). Performance Standards for Antimicrobial Susceptibility Testing; Twenty-Third Informational Supplement. M100-S27. (Wayne, PA: Clinical and Laboratory Standards Institute).\\nd'herelle, F. (1917). Sur un microbe invisible antagoniste des bacilles dysent\\u00e9riques. CR Acad. Sci. Paris 165, 373\\u2013375.\\nDi, P. A., Giannouli, M., Triassi, M., Brisse, S., and Zarrilli, R. (2011). Molecular epidemiological investigation of multidrug-resistant Acinetobacter baumannii strains in four Mediterranean countries with a multilocus sequence typing scheme. Clin. Microb. Infect. 17, 197\\u2013201. doi: 10.1111/j.1469-0691.2010.03254.x\\nCrossRef Full Text | Google Scholar\\nDijkshoorn, L., Nemec, A., and Seifert, H. (2007). An increasing threat in hospitals: multidrug-resistant Acinetobacter baumannii. Nat. Rev. Microbiol. 5, 939\\u2013951. doi: 10.1038/nrmicro1789\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nEijsink, V. G. H., Vaaje-Kolstad, G., V\\u00e5rum, K. M., and Horn, S. J. (2008). Towards new enzymes for biofuels: lessons from chitinase research. Trends Biotechnol. 26, 228\\u2013235. doi: 10.1016/j.tibtech.2008.02.004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGhajavand, H., Esfahani, B. N., Havaei, A., Fazeli, H., Jafari, R., and Moghim, S. (2017). Isolation of bacteriophages against multidrug resistant Acinetobacter baumannii. Res. Pharm. Sci. 12, 373\\u2013380. doi: 10.4103/1735-5362.213982\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHarding, C. M., Hennon, S. W., and Feldman, M. F. (2018). Uncovering the mechanisms of Acinetobacter baumannii virulence. Nat. Rev. Microbiol. 16, 91\\u2013102. doi: 10.1038/nrmicro.2017.148\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHelvoort, T. V. (2001). Felix d'Herelle and the origins of molecular biology. J. Hist. Med. Allied Sci. 56, 305\\u2013307. doi: 10.1093/jhmas/56.3.305\\nCrossRef Full Text | Google Scholar\\nHornsey, M., and Wareham, D. W. (2011). In vivo efficacy of glycopeptide-colistin combination therapies in a Galleria mellonella model of Acinetobacter baumannii infection. Antimicrob. Agents Chemother. 55, 3534\\u20133537. doi: 10.1128/AAC.00230-11\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHornsey, M., Phee, L., Longshaw, C., and Wareham, D. W. (2013). In vivo efficacy of telavancin/colistin combination therapy in a Galleria mellonella model of Acinetobacter baumannii infection. Int. J. Antimicrob. Agents 41, 285\\u2013287. doi: 10.1016/j.ijantimicag.2012.11.013\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJin, J., Li, Z. J., Wang, S. W., Wang, S. M., Huang, D. H., Li, Y. H., et al. (2012). Isolation and characterization of ZZ1, a novel lytic phage that infects Acinetobacter baumannii clinical isolates. BMC Microbiol. 12:156. doi: 10.1186/1471-2180-12-156\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKhan, M. M., and Nilsson, A. S. (2015). Isolation of phages for phage therapy: a comparison of spot tests and efficiency of plating analyses for determination of host range and efficacy. PLoS ONE 10:e0118557. doi: 10.1371/journal.pone.0118557\\nCrossRef Full Text | Google Scholar\\nKrawczyk, B., Lewandowski, K., and Kur, J. (2002). Comparative studies of the Acinetobacter genus and the species identification method based on the recA sequences. Mol. Cell. Probes 16, 1\\u201311. doi: 10.1006/mcpr.2001.0388\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKrogh, A., Larsson, B., Von Heijne, G., and Sonnhammer, E. L. (2001). Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. J. Mol. Biol. 305, 567\\u2013580. doi: 10.1006/jmbi.2000.4315\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKropinski, A. M., Mazzocco, A., Waddell, T. E., Lingohr, E., and Johnson, R. P. (2009). Enumeration of bacteriophages by double agar overlay plaque assay. Methods Mol. Biol. 501, 69\\u201376. doi: 10.1007/978-1-60327-164-6_7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLee, C. N., Tseng, T. T., Lin, J. W., Fu, Y. C., Weng, S. F., and Tseng, Y. H. (2011). Lytic myophage Abp53 encodes several proteins similar to those encoded by host Acinetobacter baumannii and phage phiKO2. Appl. Environ. Microbiol. 77, 6755\\u20136762. doi: 10.1128/AEM.05116-11\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLi, J., Rayner, C. R., Nation, R. L., Owen, R. J., Spelman, D., Tan, K. E., et al. (2006). Heteroresistance to colistin in multidrug-resistant Acinetobacter baumannii. Antimicrob. Agents Chemother. 50, 2946\\u20132950. doi: 10.1128/AAC.00103-06\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLowe, T. (1997). tRNAscan-SE: a program for improved transfer RNA detection in genomic sequence (release: 1.23). Nucleic Acids Res. 25, 955\\u2013964. doi: 10.1093/nar/25.5.0955\\nCrossRef Full Text | Google Scholar\\nMahadevan, P., King, J. F., and Seto, D. (2009a). CGUG: in silico proteome and genome parsing tool for the determination of \\u201ccore\\u201d and unique genes in the analysis of genomes up to ca. 1.9 Mb. BMC Res. Notes 2:168. doi: 10.1186/1756-0500-2-168\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMahadevan, P., King, J. F., and Seto, D. (2009b). Data mining pathogen genomes using geneorder and coregenes and CGUG: gene order, synteny and in silico proteomes. Int. J. Comput. Biol. Drug Des. 2, 100\\u2013114. doi: 10.1504/IJCBDD.2009.027586\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMarchlerbauer, A., Lu, S., Anderson, J. B., Chitsaz, F., Derbyshire, M. K., Deweesescott, C., et al. (2011). CDD: a conserved domain database for the functional annotation of proteins. Nucleic Acids Res. 39, D225\\u2013D229. doi: 10.1093/nar/gkq1189\\nCrossRef Full Text | Google Scholar\\nMerabishvili, M., Pirnay, J.-P., Verbeken, G., Chanishvili, N., Tediashvili, M., Lashkhi, N., et al. (2009). Quality-controlled small-scale production of a well-defined bacteriophage cocktail for use in human clinical trials. PLoS ONE 4:e4944. doi: 10.1371/journal.pone.0004944\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nOhmetakagi, M., and Shinshi, H. (1995). Ethylene-inducible DNA binding proteins that interact with an ethylene-responsive element. Plant Cell 7, 173\\u2013182. doi: 10.1105/tpc.7.2.173\\nCrossRef Full Text | Google Scholar\\nPeleg, A. Y., and Jara, S. D. (2009). Galleria mellonella as a model system to study Acinetobacter baumannii pathogenesis and therapeutics. Antimicrob. Agents Chemother. 53, 2605\\u20132609. doi: 10.1128/AAC.01533-08\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPeng, F., Mi, Z., Huang, Y., Yuan, X., Niu, W., Wang, Y., et al. (2014). Characterization, sequencing and comparative genomic analysis of vB_AbaM-IME-AB2, a novel lytic bacteriophage that infects multidrug-resistant Acinetobacter baumannii clinical isolates. BMC Microbiol. 14:181. doi: 10.1186/1471-2180-14-181\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPerez, F., Hujer, A. M., Hujer, K. M., Decker, B. K., Rather, P. N., and Bonomo, R. A. (2007). Global challenge of multidrug-resistant Acinetobacter baumannii. Antimicrob. Agents Chemother. 51, 3471\\u20133484. doi: 10.1128/AAC.01464-06\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPerez, F., Ponceterashima, R., Adams, M. D., and Bonomo, R. A. (2011). Are we closing in on an \\u201celusive enemy\\u201d? The current status of our battle with Acinetobacter baumannii. Virulence 2, 86\\u201390. doi: 10.4161/viru.2.2.15748\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPetersen, T. N., Brunak, S., Von, H. G., and Nielsen, H. (2011). SignalP 4.0: discriminating signal peptides from transmembrane regions. Nat. Methods 8, 785\\u2013786. doi: 10.1038/nmeth.1701\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPopova, A. V., Zhilenkov, E. L., Myakinina, V. P., Krasilnikova, V. M., and Volozhantsev, N. V. (2012). Isolation and characterization of wide host range lytic bacteriophage AP22 infecting Acinetobacter baumannii. FEMS Microbiol. Lett. 332, 40\\u201346. doi: 10.1111/j.1574-6968.2012.02573.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nQuevillon, E., Silventoinen, V., Pillai, S., Harte, N., Mulder, N., Apweiler, R., et al. (2005). InterProScan: protein domains identifier. Nucleic Acids Res. 33, W116\\u2013W120. doi: 10.1093/nar/gki442\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRakhuba, D. V., Kolomiets, E. I., Dey, E. S., and Novik, G. I. (2010). Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell. Pol. J. Microbiol. 59, 145\\u2013155. doi: 10.1016/j.micres.2015.01.008.1.94\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSitbon, E., and Pietrokovski, S. (2003). New types of conserved sequence domains in DNA-binding regions of homing endonucleases. Trends Biochem. Sci. 28, 473\\u2013477. doi: 10.1016/S0968-0004(03)00170-1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSmith, H. W., and Huggins, M. B. (1983). Effectiveness of phages in treating experimental Escherichia coli diarrhoea in calves, piglets and lambs. J. Gen. Microbiol. 129:2659. doi: 10.1099/00221287-129-8-2659\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nStojkovi\\u0107, E. A., and Rothman-Denes, L. B. (2007). Coliphage N4 N-acetylmuramidase defines a new family of murein hydrolases. J. Mol. Biol. 366, 406\\u2013419. doi: 10.1016/j.jmb.2006.11.028\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWorld Health Organization (2017). Global Priority List of Antibiotic-Resistant Bacteria to Guide Research, Discovery, and Development of New Antibiotics. Geneva: World Health Organization.\\nYoung, R. F. III., and White, R. L. (2008). Lysis of the host by bacteriophage. Encycl. Virol. 248\\u2013258. doi: 10.1016/B978-012374410-4.00752-4\\nCrossRef Full Text | Google Scholar\\nZafar, N., Mazumder, R., and Seto, D. (2002). Coregenes: a computational tool for identifying and cataloging \\u201ccore\\u201d genes in a set of small genomes. BMC Bioinform. 3:12. doi: 10.1186/1471-2105-3-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02548\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02548/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:48:41.602Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02548\",\n",
      "            \"title\": \"Frontiers | Metavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community\",\n",
      "            \"description\": \"The Formosan subterranean termite; Coptotermes formosanus is nutritionally dependent on the complex and diverse community of bacteria and protozoa in their gut. Although, there have been many studies to decipher the taxonomic and functional diversity of bacterial community in the termite gut, their bacteriophages remain unstudied. In this study we sequenced the metavirome of the termite gut to study the diversity of bacteriophages and other associated viruses. Results showed that the termites harbor a diverse bacteriophage community in their gut. Taxonomically classifiable sequences from the virome, predominantly showed similarity to the tailed bacteriophages (Caudovirales). Many predicted genes from the virome matched to bacterial prophage regions. These data are suggestive of a virome dominated by temperate bacteriophages. We predicted the genomes of seven novel Caudovirales bacteriophages from the termite gut. Three of these predicted bacteriophage genomes were found in high proportions in all the three termite colonies tested. Two bacteriophages are predicted to infect endosymbiotic bacteria of the gut protozoa. The presence of these putative bacteriophages infecting endosymbionts of the gut protozoa, suggests a quadripartite relationship between the termites their symbiotic protozoa, endosymbiotic bacteria of the protozoa and their bacteriophages. Other than Caudovirales, ss-DNA virus related genes were also present in the termite gut. We predicted the genomes of 12 novel Microviridae phages from the termite gut and seven of which possibly represent a new proposed subfamily. Circovirus like genomes were also assembled from the termite gut at lower proportions. We predicted 10 novel circovirus genomes in this study. Whether these circoviruses infect the termites remains elusive at the moment. The functional and taxonomical annotations suggest that the termites may harbor a core virome comprised of the bacteriophages infecting endosymbionts of the gut protozoa.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nBacteriophages are considered as the most abundant biological entities on earth with their total number estimated to be 1031 (Wommack and Colwell, 2000; Suttle, 2005). Bacteriophages play an important role in the ecosystem by carrying out nutrient recycling via bacterial cell lysis (Wilhelm and Suttle, 1999; Rodriguez-Brito et al., 2010; Jasna et al., 2017). In the marine ecosystems bacteriophages have been shown to be responsible for metabolic reprograming (Hurwitz et al., 2013; Hurwitz and U'Ren, 2016). Bacteriophages also carry out horizontal gene transfer (Eggers et al., 2016; Touchon et al., 2017) and are considered as a driving force behind bacterial genome evolution (Canchaya et al., 2004; Zeng et al., 2017). Along with the horizontal gene transfer, bacterial-phage antagonistic co-evolution is hypothesized to have a major impact on bacterial and bacteriophage diversification (Buckling and Rainey, 2002; Paterson et al., 2010). Previous studies have shown that the majority of the differences observed in the genomes of different bacterial strains of the same species in the human gut were related to restriction-modification systems, and glycosyltransferases, both of which play a key role in phage defense (Zhu et al., 2015). Bacteriophage-resistant strains have been shown to possess modifications in their surface receptors, including-antigens and outer membrane proteins (Bassford et al., 1977; Labrie et al., 2010). These surface receptors also have an important role in cell surface adhesion (Shin et al., 2005) and thus may alter the interactions of bacterial strains with their host (Lerouge and Vanderleyden, 2002). Co-evolution of bacteriophages and bacteria has been shown to alter the metabolic capacities of bacteriophage-resistant strains including the ability to utilize certain carbon sources (Middelboe et al., 2009). All these observations show that phage-bacterial interactions can have a profound effect on the ecology of the microbial community and can in turn affect the host harboring the community.\\nTermites rely on their symbiotic gut microbial community for cellulose digestion and acquisition of essential nutrients (reviewed in Brune, 2014). The microbial community of the higher termites (family: Termitidae) is comprised of bacterial symbionts while the lower termites (Rhinotermitidae, Mastotermitidae, Hodotermitidae, Kalotermitidae, Termopsidae, Serritermitidae) harbor flagellated protozoa in addition to the bacterial symbionts. Workers of the Formosan subterranean termite (FST), Coptotermes formosanus (Family: Rhinotermitidae), harbor three species of protozoa in the hindgut paunch that are essential for the survival of the termite colony (Eutik et al., 1978). The worker caste specializes in providing nutrition for the entire termite colony by digesting lignocellulose, and the gut protozoa enhance the endogenous cellulolytic capability of the worker gut. The protozoa also harbor endo- and ecto-symbiotic bacteria which carry out nitrogen fixation and amino acid production (Hongoh et al., 2008; Desai and Brune, 2012). The termite gut microbial community is responsible for many complex biochemical processes, providing the termite host with essential nutrients. In a previous study based on cloning of 16S rRNA gene amplicons, at least 213 different bacterial ribotypes were reported from the gut of the FST (Husseneder et al., 2010). A closely related species, Coptotermes gestroi, was estimated to harbor a bacterial community of 1,460 different species using next-generation sequencing (Do et al., 2014). The termite gut presents a unique scenario in which the host is heavily dependent on the microbial community and any changes to this community can potentially have detrimental effects on the host (Rosengaus et al., 2011). The presence of a multifaceted complex bacterial community makes the termite gut a perfect ecological niche for the presence of a diverse bacteriophage community. Like many other ecosystems the extent of this diversity and the impact of bacteriophages on the microbial community in the termite gut remains unstudied.\\nBacteriophages have been previously shown to be associated with the gut bacteria in higher termites (Ottesen et al., 2006). We previously isolated and sequenced the first bacteriophage from a termite hindgut. This phage, CVT22, infects Citrobacter sp. found in the gut of the FST (Tikhe et al., 2015). Two additional novel bacteriophages, Tyrion and Arya (NC_031077.1 and NC_031048.1) infecting Enterobacter sp. were also isolated and sequenced from the termite gut (manuscripts under review). In a recent study by Pramano et al. a bacteriophage infecting \\u201cCandidatus Azobacteroides pseudotrichonymphae\\u201d was discovered, making it the first bacteriophage to be associated with an obligate intracellular mutualistic endosymbiont (Pramono et al., 2017). These reports of novel bacteriophages in the termite gut indicate the presence of an unexplored bacteriophage community. The main obstacle in studying the bacteriophages from the termite gut using conventional isolation techniques is the non-culturable bacterial community (Hongoh, 2010). Most of the bacteria from the termite gut are not culturable in the lab using conventional methods, which makes the study of their bacteriophages difficult. Bacteriophages also lack universal marker genes. It has been shown that virome sequencing can be used as a powerful tool to study and characterize viral communities (Sullivan, 2015). In recent years, advances in the field of marine viromics has established a foundation for all the other virome studies (Brum and Sullivan, 2015). Many marine virome studies have identified enormous diversity of previously unknown protein sequences whose functional roles still largely remains unknown (Roux et al., 2015b; Brum et al., 2016). Metavirome sequencing also circumvents the requirement of isolation and, therefore, we chose this approach to study the bacteriophage community in the FST gut in its entirety.\\nIn this study we report the virome sequencing of the guts of workers from three FST colonies with the intention of exploring the unstudied bacteriophage diversity. This study is the first effort focused entirely on uncovering the bacteriophages and any other possible viruses associated with a termite species. Studying the virome of the termite gut will help us to understand the complex quadripartite relationship between the termite host, protozoa, bacteria symbionts, and associated bacteriophages.\\nMaterials and Methods\\nTermite Collection\\nWorkers of the FST were collected from three different colonies in New Orleans, Louisiana, USA, using untreated in-ground bait stations. Termite colonies were designated as Colony 1 (collected from City Park, on 06/21/2013), Colony 2 (collected from Hayne Blvd., lakefront on 06/18/2013), and Colony 3 (collected from Cypress St. on 06/21/2013). The distance between each of the three colonies was more than 9 km, which by far exceeds the swarming distance of the winged reproductives (alates) of the FST (Messenger and Mullins, 2005) thereby ensuring that the three colonies were founded independently by unrelated alates. All the termites were brought back to the lab in a plastic container containing a moist filter paper. Termites were processed immediately for viral community DNA extraction.\\nExtraction of Viral Community DNA\\nA total of 500 worker termites from each colony were dissected and their guts were suspended in 3 ml sterile phosphate buffered saline (pH 7.5) kept on ice. The guts were homogenized vigorously using a sterile pestle until a uniform solution was formed. The homogenate was centrifuged at 10,000 g and the supernatant was filtered twice through a 0.22 \\u03bcm syringe filter. The filtered homogenate was then treated with 2.5 units per \\u03bcl of RNase A and DNase I at 37\\u00b0C for 6 h. The filtrate was then mixed with 200 \\u03bcl of 0.5 M EDTA, and DNA was isolated using phenol-chloroform-isoamyl alcohol extraction. Concentration and quality of the extracted DNA was checked with NanoDrop\\u00ae ND1000. Bacterial DNA contamination was checked via PCR using 27f and 1492r universal 16S rRNA gene primers (Lane, 1991). The initial concentrations of the extracted DNA from Colony 1, Colony 2, and Colony 3 were 4, 1.8, and 2.5 ng/\\u03bcl, respectively. The extracted DNA was amplified using illustra GenomiPhi V2 DNA Amplification Kit (GE Healthcare Life sciences, Pittsburgh, USA). The amplified DNA was then ethanol precipitated and dissolved in sterile distilled water.\\nNext Generation Sequencing and Bioinformatics Analysis\\nFrom each of the three colonies, 50 ng of DNA was used to prepare the libraries using Nextera DNA Sample Preparation Kit (Illumina). The insert size of the libraries was determined by Experion Automated Electrophoresis Station (Bio-Rad). The insert size of the libraries ranged from 300 to 850 bp (average 500 bp). Individual libraries were sequenced at Molecular Research LP, Shallowater, Texas, on the Illumina MiSeq platform (Colony 1: 2 \\u00d7 250 bp, Colony 2 and Colony 3: 2 \\u00d7 300 bp). Quality of the DNA reads was checked using FASTQC (Andrews, 2010). DNA reads were checked for Illumina adaptor contamination and reads below the Phred score of 20 were removed using Trim Galore (Krueger, 2015). DNA reads were assembled into contigs using SPAdes Genome Assembler (Version 3.0) available on the Illumina BaseSpace platform with the default parameters using error correction and assembly mode (Bankevich et al., 2012). The contigs obtained were uploaded on the Metavir server for taxonomic assignments of the predicted ORFs using RefSeq complete viral genome protein sequence database from NCBI (released on 01/11/2017) (Roux et al., 2014). All the predicted protein coding genes were also blasted against the protein sequences from bacteriophage ProJPt-1Bp1 (Pramono et al., 2017). The contigs were also analyzed using VirSorter to separate viral contigs from cellular contaminants (Roux et al., 2015a). Contigs available on Metavir were screened for the presence of VP1, TerL, and Rep genes. For the construction of phylogenetic trees, we used full length amino acid sequences of terminase large subunit TerL (terminase_1, terminase_6, terminase_3, terminase_GPA, and terminase_1), Microviridae VP1, and Circoviridae Rep proteins. The sequences were aligned using MUSCLE (Edgar, 2004). Maximum likelihood trees were constructed using PhyML algorithm with a WAG substitution model (Guindon et al., 2010). For Microviridae subfamily assignment, full-length amino acid sequences of VP1 protein from Quaiser et al. (2015) were used to construct a phylogenetic tree. Contigs containing VP1, TerL, or Rep genes were analyzed manually for the presence of other putative viral genes. Contigs were classified as of a viral origin using the parameters described previously, with the POG13 database was used instead of POG10 (Bellas et al., 2015). Putative partial or full phage genomes were annotated manually and comparative genomic diagrams were generated using Easyfig (Sullivan et al., 2011). Putative viral genomes were visualized using CGview (Grant and Stothard, 2008) and SnapGene\\u00ae (from GSL Biotech; available at snapgene.com). PHACTS analysis was carried out to determine the lifestyle and host of the putative phages (McNair et al., 2012). Family assignment of putative phage genomes was performed with VIRFAM using the ACLAME database (Lopes et al., 2014). For contig LSPY100002, RNA polymerase beta and beta' subunit sequences from phiKZ-like bacteriophages were used to construct phylogenetic trees (Lavysh et al., 2016). Functional annotation was carried out using the MG-RAST automated pipeline with an integrated M5NR database (Keegan et al., 2016). Orthologous genes from the three colonies were compared against each other using Orthovenn (Wang et al., 2015). All the assembled contigs have been submitted to NCBI GenBank under the accession numbers LSPY0000, LSQA0000, and LSPZ0000. Fully annotated contigs from this study are publicly available on the Metavir server under the study named \\u201ctermite gut metavirome\\u201d (http://metavir-meb.univ-bpclermont.fr).\\nResults and Discussion\\nDNA Extraction, Next Generation Sequencing, and Bioinformatics Analysis for Taxonomic Assignment\\nWorker termites from three different termite colonies were used for this study. The abundance and diversity of microbes in the worker termites' gut was the biggest challenge in separating the viral community DNA from all other microbial DNA to avoid contamination. High speed centrifugation and double filtration through 0.22 \\u03bcm were carried out to remove any contaminating bacterial and protozoal cells along with other bigger particles. Other free contaminating nucleic acids in the filtrate were degraded by RNase and DNase and DNA was extracted from the filtrate. No bacterial contamination was detected by PCR with universal 16S rRNA primers. At this stage the amount of DNA to be used for next generation sequencing was extremely low. In many virome studies, multiple displacement amplification (MDA) has been used to increase the amount of DNA required for sequencing (Minot et al., 2011; Santiago-Rodriguez et al., 2015; Tangherlini et al., 2016). MDA, however is known to introduce bias in the DNA samples with preferential amplification of ssDNA viruses (Kim and Bae, 2011). While the presence of viruses presented in this study is undisputed, their relative abundance may not represent the actual viral composition in the termite gut. Many new methods are being established to overcome the bias of amplification techniques and present absolute quantitate estimation of viral communities (Roux et al., 2016).\\nThis DNA was then submitted for next generation sequencing using Illumina MiSeq platform. The sequencing data, predicted genes by Metavir and MG-RAST, and the number of circular contigs predicted by Metavir are summarized in Table 1.\\nTable 1. Sequencing data, gene prediction, and number of circular contigs from viral DNA isolated from the guts of the FST workers from three different colonies.\\nMetavir assigned 27.13% of the genes from Colony 1, 27.85% from Colony 2 and 28.9% from Colony 3 as virus affiliated genes with an e-value of 10\\u22125 or less. Compared to Metavir, MG-RAST classified fewer genes as viral. Percentage of genes classified as viral for Colony 1 was 9.57%, for Colony 2 was 3.8%, and for Colony 3 was 3.53%. Taxonomically MG-RAST classified between 79 and 93% of the genes as bacterial. The alpha diversity calculated by MG-RAST for Colony 1, 2, and 3 was 551, 598, and 231 species respectively. The difference between the taxonomic assignments by MG-RAST and Metavir has been previously observed in the metavirome sequencing of Antarctic soils. These viromes were predicted to be dominated by temperate bacteriophages (Zablocki et al., 2014). The taxonomic assignment of phage DNA as bacterial is likely due to the fact that reference databases classify prophages as bacterial when they are integrated into a bacterial chromosome at the time of genome sequencing. To further analyze if our data was contaminated with bacterial cellular contaminants bearing prophages, we used VirSorter tool, which can effectively separate bacterial and viral contigs (Roux et al., 2015a). VirSorter did not predict prophage sequences of category 1, but it did predict a total of 14 contigs (0.0005%) as prophage-like (category 2 and category 3). These results suggests a minimal contamination by bacterial genomes containing prophages. The number of sequenced bacterial genomes is reaching the 100,000 mark while the bacteriophage genomes still remain poorly represented in the NCBI database (around 2000 Caudovirales genomes, as of 06/01/2017). Sequencing more phage genomes is paramount to improve recognition of prophage sequences in bacterial genomes and will improve taxonomic assignments in all virome studies.\\nPrevious studies showed that Bacteroidetes form around 70% of the bacterial flora of the guts of C. formosanus workers (Noda et al., 2005; Shinzato et al., 2005). For this reason, we expected the Bacteroidetes phages to dominate the virome composition. The virome composition however, was different from what we expected, with Proteobacteria and Firmicutes comprising at least 40% of the identified genes (MG-RAST). This difference could be explained by the small number of sequenced Bacteroidetes bacteriophages in reference databases. Since we sequenced filtrate from the termite gut it was not surprising that viral sequences were at least 42 times enriched in our data as compared to the unfiltered metagenome of a higher termite, Nasutitermes sp. (Warnecke et al., 2007). Functional annotation also showed that phage-related sequences were at least 10 times enriched in our data. The overall enrichment in phage-related genes and PCR results indicate a successful separation of bacterial contaminants during viral DNA purification.\\nTaxonomic assignment using Metavir showed that dsDNA virus-related genes were dominant amongst all the viral genes despite the MDA amplification and its likely bias towards ssDNA viruses (Figure 1A). The dsDNA viral genes predominantly belonged to the tailed bacteriophages from the order Caudovirales (Figure 1B). Genes related to all three families of the order Caudovirales, i.e., Myoviridae, Siphoviridae, and Podoviridae were present in all three termite colonies (Figure 1D). Apart from the genes related to Caudovirales, genes related to large eukaryotic dsDNA viruses and other unclassified viruses were also present in all the three termite colonies. The single stranded DNA viruses (ssDNA) contributed between 1 and 10% of the total virus related genes. Within the ssDNA virus related genes between 62 and 65% belonged to Microviridae phages (Figure 1C).\\nFigure 1. Classification and abundance (%) of various virus types observed in the guts of the termite workers from three different colonies. (A) Percent of taxonomic assignment of all viral genes (B) Percent of taxonomic assignment of dsDNA virus related genes. (C) Percent taxonomic assignment of ssDNA virus related genes. (D) Percent taxonomic assignment of Caudovirales related genes. The data were generated using Metavir-2 server by comparing the predicted proteins to the NCBI virus protein database. Only top BLAST hits with an e-value of 10\\u22125 or less were used (normalized for each category).\\nGene Based Diversity of Tailed Bacteriophages\\nAmong all three termite colonies, Colony 3 was the most diverse in terms of tailed bacteriophages, with genes related to 712 different bacteriophages, followed by Colony 2 (598 bacteriophages) and Colony 1 (389 bacteriophages). Genes related to a total of 960 different tailed bacteriophages were observed across all three termite colonies. This number accounts for 48.65% of all tailed bacteriophages whose complete genomes are available in the NCBI Genbank database (04/12/2017). Of these 960, Siphoviridae-related bacteriophages were the most diverse, representing 483 different bacteriophages, followed by Myoviridae (335) and Podoviridae (142). A considerable number of bacteriophages (218) were shared by all three termite colonies, with Bacillus phage AR9 (Lavysh et al., 2016) and Azobacteroides phage ProJPt-1Bp1 (Pramono et al., 2017) related genes being present in the highest proportions (Figure 2). In Colony 1, 10% of all the classified sequences belonged to Bacillus phage AR9 followed by Colony 2 (6%) and Colony 3 (4%). Azobacteroides phage ProJPt-1Bp1 related genes constituted 6.45% of all the classified genes in Colony 1, 5.61% in Colony 2 and 3.92% in Colony 3. Of the top 20 most dominant tailed bacteriophage related genes from all the three termite colonies, 12 bacteriophages had Firmicutes as their host out of which 9 bacteriophages were infecting Bacillus spp.\\nFigure 2. Taxonomic distribution (%) of the predominant dsDNA bacteriophage genes from the guts of termite workers from three different colonies (normalized). The data were generated using Metavir-2 server by comparing the predicted proteins to the NCBI virus protein database. Only top BLAST hits with an e-value of 10\\u22125 or less were used.\\nThe analysis of phage diversity of the three termite colonies suggests the presence of a conserved virome of tailed bacteriophages in the termite gut from the same geographical area. Although there is some degree of inter-colonial variation, nearly a quarter of all tailed bacteriophages (23%) were present in all three colonies. This hypothesis needs to be tested by a larger study including more colonies from the introduced and native distribution range of the FST (Husseneder et al., 2010).\\nPhylogenetic Analysis of Terminase Genes\\nFrom all three termite colonies together, 51 unique full length terminase large subunit amino acid sequences were predicted. Out of the 51 sequences, 25 contained terminase_6 (pfam03237) domains, 12 contained terminase_3 (pfam04466.8) domains, 9 had terminase_gpa (pfam05876) domains, and 5 had terminase_1 (pfam03354) domains. Our data of terminase diversity in the termite gut are comparable to the results from virome sequencing of the deep sea, where 52 unique terminase sequences were identified (Mizuno et al., 2016). Most of the terminase sequences from the termite gut matched closely to the terminase genes from prophage regions in the bacterial genomes (Figure 3).\\nFigure 3. A maximum likelihood phylogenetic tree of large terminase subunit of type terminase_6. The nodes with a bootstrap value of 70% or more are indicated by a circular symbol. Sequences from the termite gut are colored purple. Bacteriophages: red. Firmicutes: orange. Spirochetes: dark blue. Gammaproteobacteria: bright green. Bacteroidetes: dark green. Actinomycetes: pink. Alphaproteobacteria: light blue. Termite Group I bacterium: sky blue. Others: black.\\nPhylogenetic analysis of terminase_6 showed a vast diversity with matches to prophages associated with a wide range of bacterial taxa. These results were also similar to the deep sea sequencing data, where terminase_6 domain was the most abundant type (Mizuno et al., 2016). Eight terminases from the virome formed a separate clade comprising a terminase gene from a Termite Group 1 bacterium. Two terminase genes were clustered with two separate Firmicute clades, another two were clustered with two separate Gammaproteobacteria clades, and one was clustered with a Bacteroidetes clade. Another clade was comprised entirely of bacteriophages from the Myoviridae family and three genes from termite gut phages. The clade was further subdivided in phiKZ-like bacteriophages and T4-like bacteriophages (Figure 3). Remaining genes were present in multiple clades comprised of terminases from the genomes of taxonomically diverse bacterial phyla (putative prophages).\\nPhylogenetically, terminase_3 genes were divided into four clearly separated clades. The Alphaproteobacteria clade and the Bacteroidetes clade each contained three terminases associated with the termite gut virome, while both the Enterobacteriaceae and the Firmicute clade contained one. Two other terminases found in the termite gut virome formed separate branches while one was grouped with Clostridium sp. CAG 306 (Figure S1). Several subunits of terminase_GPA from the termite gut virome were assigned to two distinct clades of Spirochetes and Alphaproteobacteria. The Spirochetes cluster contained five terminase_GPA subunits from the termite gut while another subunit formed a sister clade with Alphaproteobacteria (Figure S1). Most of the terminase_1 subunits were associated with Firmicutes or bacteriophages infecting the Firmicutes and one sequence was clustered with Bacteroidetes (Figure S1).\\nThe phylogenetic analysis of the terminase genes indicated that most of the genes matched to prophage regions in the bacterial genomes rather than sequenced bacteriophage genomes. These results along with MG-RAST annotations and VirSorter analysis suggest that most of the termite gut bacteriophages might be temperate in nature.\\nPhylogenetic Analysis of the Integrase Genes\\nThe integrase gene is used by the temperate bacteriophages to enter the lysogenic life cycle. It has been shown that prophage integrates in the host genome with a minimum impact on the overall chromosomal architecture. The bacteriophage also undergoes numerous adaptations according to the host genome in order to establish a lysogenic life cycle (Br\\u00fcssow et al., 2004). It can be assumed that temperate bacteriophages are likely to infect closely related bacteria or bacteria where the overall genome architecture is conserved. Thus, phylogenetic analysis of phage integrase is likely to yield more information about its host. A total of 31 unique phage integrase sequences were identified from the three termite colonies. Phylogenetic analysis of phage integrase genes showed that sequences from the termite gut are clustered with a wide range of bacterial taxa (Figure S2). Five termite gut integrases were clustered within a Spirochete clade, six were associated with Firmicutes, another six were distributed in two clades comprised of Spirochetes and Bacteroidetes, and four were associated in two clades comprised of Bacteroidetes and Firmicutes. The remaining genes were distributed in clades comprised of diverse bacterial phyla. The results suggest the presence of temperate bacteriophages capable of infecting all the major bacteria taxa in the termite gut.\\nPutative Contigs of dsDNA Bacteriophage Origin\\nWe compared individual contigs to Refseq virus, Pfam-A, POGs13, and ACLAME database as described previously (Bellas et al., 2015). Many contigs were considered to represent putative complete or partial bacteriophage genomes. The details of all the contigs described below are listed in Supplementary Table 1.\\nLSPY01000004 and LSPY01000006 Represent Genomes of Bacteriophages Infecting the Symbiotic Bacteria of the Gut Protozoa\\nBoth LSPY01000004 and LSPY01000006 were predicted as circular contigs in termite Colony 1. Generally, circular contigs are indicative of a complete genome. Contigs mapping onto LSPY01000004 and LSPY01000006 were present in all the three termite colonies, suggesting an inter-colonial conserved distribution. Out of the 68 predicted genes in LSPY01000004, 30 produced a match in NCBI nr protein database with an e value of 10\\u22125 or less. Of those 30, 21 genes matched only to Azobacteroides phage ProJPt-1Bp1, a bacteriophage infecting an obligatory intracellular bacterium Candidatus A. pseudotrichonymphae of the termite gut protozoa (Pramono et al., 2017). Out of the remaining genes, five genes matched to two different plasmids from Ca. A. pseudotrichonymphae and one gene matched to the genome (Hongoh et al., 2008). A total of 65 genes were predicted in contig LSPY01000006, of which 36 produced a match in the NCBI database. Out of those 36, 22 matched to phage ProJPt-1Bp1, 7 genes matched to plasmid pCFPG3 from Ca. A. pseudotrichonymphae and 1 gene matched to the genome of Ca. A. pseudotrichonymphae. There was very little similarity at nucleotide level in the genomes of LSPY01000004 and LSPY01000006 (73% match over 5% of the genome); most of the similarity was observed in the region of conserved hypothetical proteins also found in the genome of ProJPt-1Bp1. Overall, the genome arrangement of LSPY01000004, phage ProJPt-1Bp1, and LSPY01000006 was alike with areas of high similarity and synteny (Figure 4A). LSPY01000004 and LSPY01000006 showed many differences in the hypothetical proteins. Notably, LSPY01000006 harbored a gene similar to dihydrofolate reductase (DHFR) which was absent in LSPY01000004 and phage ProJPt-1Bp1. T4 bacteriophage DHFR has been predicted to play an important role in DNA metabolism and was also predicted to be a part of the virion particle (Mosher et al., 1977). However, at this moment the function of DHFR in bacteriophages remains to be studied.\\nFigure 4. (A) Comparative genomic analysis of LSPY01000004 and LSPY01000006 with Azobacteroides phage ProJPt-1Bp1. All three circular genomes have been rearranged so that the start codon of a conserved hypothetical protein is the first base in the sequence. (B) Comparative genomic analysis of LSPZ01000002 with Bacillus phage AR9 and Yersinia phage phi R137. All three circular genomes have been rearranged so that the start codon of the large terminase subunit is the first base in the sequence. The figures were generated using Easyfig software with tblastx. The structural genes are indicated in red, DNA metabolism related genes are indicated in blue and RNA polymerase genes are indicated in orange. All the other genes are indicated in sky blue color.\\nVIRFAM analysis of ProJPt-1Bp1, LSPY01000004, and LSPY01000006 predicted them to be the members of Caudovirales; however, the contigs could not be assigned to any family. The genome of phage ProJPt-1Bp1 was captured while sequencing the host bacterial genome and a t-RNA detected in the phage genome also matched to the host (Pramono et al., 2017). We were not able to identify any t-RNA in LSPY01000004 and LSPY01000006 or any other genes, which could point toward the potential host of these phage-like genomes. However, based on the similarity of genes to phage ProJPt-1Bp1 and Ca. A. pseudotrichonymphae, LSPY01000004 and LSPY01000006 most likely infect Ca. A. pseudotrichonymphae or a closely related symbiont of the obligatory gut protozoa. Also, it should be noted that phage ProJPt-1Bp1 was sequenced from Ca. A. pseudotrichonymphae from the gut of Prorhinotermes japonicus from Japan collected in 2012. It has been demonstrated for Ca. A. pseudotrichonymphae that their protozoa hosts and the termites have co-diversified (Noda et al., 2007; Ikeda-Ohtsubo and Brune, 2009; Desai et al., 2010). This presents us with a possibility of co-diversification of bacteriophages along with their hosts. The observed differences and the conserved genome architecture between phage ProJPt-1Bp1, LSPY01000004, and LSPY01000006 can be explained by the bacteriophage co-diversification hypothesis. A study of more similar phage genomes from various termite species will shed more light on the association and co-diversification of bacteriophages, protozoa, their symbiotic bacteria, and the termite host. One interesting feature that was notable in LSPY01000004, LSPY01000006, and ProJPt-1Bp1 is the apparent absence of conserved phage genes like terminase, endolysins, and phage integrases. According to the best of our knowledge, no Caudovirales bacteriophage without the presence of a terminase gene has been found prior to our study. There are two explanations for the missing genes in these phages. One is that these bacteriophages have lost their signature genes and are maintained in the bacteria as plasmids. Whether this relationship is symbiotic, as hypothesized in the case of phage ProJPt-1Bp1 (Pramono et al., 2017), remains to be explored. The other possibility is that these types of bacteriophages have a completely new method of carrying out essential functions such as DNA packaging and host lysis. Gene expression analysis would answer the question whether these bacteriophages are dormant or play any role in the bacterial metabolism, or enter an infective cycle.\\nIn this study, we centrifuged the gut homogenate and also filtered it through a 0.22 \\u03bcm to remove all possible bacterial contamination. The free nucleic acids were also degraded with RNase and DNase. If LSPY01000004, LSPY01000006 were lysogenic or pseudolysogenic, their DNA should not have been found in the filtrate. Since all the colonies were processed separately, the presence of the same phage genomes in the filtrate from all the three termite colonies, suggests strongly that LSPY01000004 and LSPY01000006 enter an infective cycle. If ProJPt-1Bp1-like phages enter an infective cycle, studying their life cycle would aid in finding the mechanism by which intracellular phages penetrate two different types of cells, in this case the protozoa and its bacterial symbiont. It has been shown that Wolbachia phage WO can form virions and it has been suggested that phage WO can infect other Wolbachia cells from neighboring infected cells (Bordenstein et al., 2006; Kent and Bordenstein, 2010). The mechanism by which phage WO penetrates both the insect and bacterial cells remains elusive.\\nLSPZ01000002 Represents the Genome of a phiKZ-Like Bacteriophage\\nThe large contig LSPZ01000002 of 252,037 bp harboring a terminase gene was assembled from Colony 2. Further analysis of all the three termite colonies showed multiple contigs mapping against LSPZ01000002. Phylogenetic analysis of the terminase gene showed that LSPZ01000002 clustered with phiKZ-like bacteriophages (Figure 3). Of the predicted 238 genes, 71 genes had a match in the NCBI database with an e-value of 10\\u22125 or less. Of those 71 genes, 35 genes matched to the phiKZ-like bacteriophage Bacillus phage AR9 (Lavysh et al., 2016). In all three termite colonies, Bacillus phage AR9-like genes were present in the highest abundance, suggesting that the bacteriophage, possibly along with its host has an important role in the termite gut. phiKZ-like bacteriophages encode two multi-subunit RNA polymerases (RNAPs); one of them is packed in the virion while the other is expressed in infected cells (Krylov et al., 2007; Ceyssens et al., 2014). These RNAPs are considered as the signature genes of phiKZ-like bacteriophages. LSPZ01000002 had six predicted genes encoding RNAP; three were predicted to encode the beta subunit, while three encoded the beta' subunit. Phylogenetic analysis of predicted virion (Figure S3) and non-virion RNAPs (data not shown) along with RNAPs from the other phiKZ-like phages produced a similar phylogenetic topology as described previously (Lavysh et al., 2016). LSPZ01000002 RNAPs were clustered with Bacillus phage AR9 and Yersinia phage phiR137.\\nLSPZ01000002 RNAPs were clustered with Bacillus phage AR9 and Yersinia phage phiR137. Comparative genomic analysis of LSPZ01000002, Bacillus phage AR9, and Yersinia phage phiR137 showed considerable similarities but no synteny in the genomes (Figure 4B). Lack of synteny in the genomes of closely-related phiKZ-like bacteriophages has been observed previously (Cornelissen et al., 2012; Jang et al., 2013). Most of the phiKZ-like bacteriophages are considered lytic in nature. However, some phages have been shown to be pseudolysogenic (Pletnev et al., 2010; Lavysh et al., 2016). Whether LSPZ01000002 is lytic or pseudolysogenic remains unknown. Unlike the genome of Bacillus phage AR9 which had multiple introns, RFAM BLAST did not predict any intron sequences in LSPZ01000002. One open reading frame (ORF) encoded a Group I intron-like endonuclease which showed similarities to many Group I introns from Firmicutes. One of the most important differences observed between Bacillus phage AR9 and LSPZ01000002 is the difference between ribonucleotide reductase (RNR) genes. Bacillus phage AR9 contains an operon of class I RNR genes (nrdE and nrdF). This class of RNR is dependent on oxygen and is found in organisms which can grow aerobically (Dwivedi et al., 2013). LSPZ01000002, on the other hand, contains an operon of class III RNR genes. This class is sensitive to oxygen and bacteriophages infecting strict anaerobes like Clostridium sp. harbor only this class of RNR genes (Dwivedi et al., 2013). The RNR genes found in LSPZ01000002 showed a high degree of similarity to Treponema primitia RNR genes, which is a strict anaerobic spirochete isolated from the gut of a damp wood termite Zootermopsis angusticollis (Graber et al., 2004). Many spirochetes have been previously reported from the gut of various termite species and some have been known to be ectosymbionts of the gut protozoa (Noda et al., 2003; Hongoh et al., 2007). These data suggest that LSPZ01000002 most likely infects a strict anaerobe from the termite gut, possibly a spirochete.\\nLSPY01000009 and LSQA01000015 Represent Partial Genomes of Lactococcus Lactis Phage 1706-Like Phages\\nThe three contigs LSPY01000009, LSPZ01000022, and LSQA01000015 were identified in termite Colony 1, 2, and 3 respectively. LSPY01000009 and LSPZ01000022 showed 99% similarity at genome level and hence were considered as genomes from the same phage. A moderate nucleotide level similarity was observed between LSPY01000009 and LSQA01000015 mostly at the ends of the two contigs (46% query coverage, 67% identity). All the three contigs showed a high degree of similarity to proteins from Rhodococcus phage ReqiPepy6 (Summer et al., 2011) and Arthrobacter phage Mudcat proteins. As observed in the Rhodococcus phage ReqiPepy6 and Arthrobacter phage Mudcat, LSQA01000015 lacked reverse transcriptase in the genome. However, LSPY01000009 and LSPZ01000022 harbored a reverse transcriptase enzyme belonging to Group II introns. Another important difference observed between the three termite gut contigs, Rhodococcus phage ReqiPepy6 and Arthrobacter phage Mudcat, is the presence of anaerobic ribonucleotide reductase gene (nrdD). The nrdD gene was present in all the three termite gut contigs but not in ReqiPepy6 and Mudcat. The nrdD gene has been previously observed in the genomes of bacteriophages infecting anaerobes (Dwivedi et al., 2013). No other class of RNR gene was observed in any of the contigs. Interestingly, RNR genes in LSPY01000009 and LSQA01000015 showed very little similarity to each other at amino acid level. Comparative genomic analysis showed segments of synteny in structural, DNA metabolism-related genes and segments of variable small hypothetical proteins (Figure S4). It has been shown that Rhodococcus phage ReqiPepy6 along with other closely related phages from Lactococcus lactis phage 1706-like phages have segments of genome expansion (Summer et al., 2011). In that case, closely related bacteriophages have segments of conserved genes but differ from each other in genome segments where multiple small hypothetical proteins are observed (Lavigne et al., 2009). Another characteristic of L. lactis phage 1706-like phages is the enrichment of membrane related proteins (14\\u201323%) (Garneau et al., 2008; Summer et al., 2011).\\nIn all the three contigs, between 20 and 23% of the predicted proteins were found to contain at least one transmembrane domain. Based on this analysis, LSPY01000009, LSPZ01000022, and LSQA01000015 represent partial genomes of L. lactis phage 1706-like phages which most likely infects a Firmicutes bacterium.\\nLSPZ01000027 Represents a Full Genome of a Temperate Phage\\nLSPZ01000027 was identified as a circular contig in termite Colony 2. Other multiple contigs could be mapped against LSPZ01000027 from all the three termite colonies. LSPZ01000027 genome showed high level of synteny to structural genes of Clostridium phage phiCDHM1 (Hargreaves et al., 2014) and Clostridium phage phiMMP01 (Boudry et al., 2015) (Figure S4). PhiCDHM1 genome was found to harbor a cassette of bacterial quorum sensing genes (Hargreaves et al., 2014), but no such cassette was identified in LSPZ01000027. PhiCDHM1 is considered a member of phiCD119-like bacteriophages with the presence of a signature DNA replication cassette. In LSPZ01000027 this signature DNA replication cassette was not observed. Also the G+C content of LSPZ01000027 was much higher (42%) than that of phiCDHM1 and phiMMP01the (G+C content 14\\u201329%). Similar to the genome of phiCDHM1 and phiMMP01, the genome of LSPZ01000027 has many proteins annotated as putative anti-repressor proteins. The true identity of these anti-repressor proteins remains unknown at this moment. LSPZ01000027 also had a Group II intron encoded reverse transcriptase. The presence of an integrase and multiple anti-repressor-like proteins indicate that LSPZ01000027 is most likely a temperate bacteriophage.\\nLSQA01000020 Represents the Genome of a Lytic Siphovirus\\nLSQA01000020 was identified as a linear contig in termite Colony 3 and multiple contigs from Colony 2 could be mapped against LSQA01000020. The first and the last gene of the linear contig encoded the same partial gene, indicating an almost complete circular bacteriophage genome. Phylogenetic analysis of the terminase gene from LSQA01000020 placed it in the Bacteroidetes cluster in a sister clade with Flavobacter bacteriophages. Comparative genomic analysis of the LSQA01000020 genome shows a high level of similarity to the genomes of two bacteriophages, P2559S and P2559Y, both of which infect the Bacteroidetes species Croceibacter atlanticus (Kang et al., 2012, 2016). Even though both P2559S and P2559Y are lytic Siphoviruses infecting the same species they show similarity only in the structural module of the genome (Kang et al., 2016). LSQA01000020 showed similarity in the structural module to both the phages and similarity to some extent in the replication module to P2559Y (Figure S4). LSQA01000020 contained thymidylate synthase and asparagine synthase genes, which were absent in P2559S and P2559Y. Based on the similarity to P2559S and P2559Y and phylogenetic placement of the terminase gene, LSQA01000020 most likely infects a Bacteroidetes species.\\nContigs similar to the first bacteriophage (CVT22) isolated from the termite gut were also observed in Colony 3 suggesting some association with the termite gut (Tikhe et al., 2015). It has been suggested that CVT22 may represent a founding member of a new cluster of lytic bacteriophages (Casjens and Grose, 2016) and the termite gut might represent a niche of diversity of CVT22-like bacteriophages.\\nMany contigs observed in our virome, harbored signature phage genes. Large contigs of 200,000 bp or more were observed in all the three termite colonies. However, due to the presence of a large proportion of previously unknown genes, the origin of many contigs remains unclear. As more phage genomes will be sequenced to populate reference databases we believe that many more phage genomes will be uncovered from the termite gut.\\nLarge viruses like the Chlorella virus infecting the algal symbionts of the protozoa have been previously identified and sequenced (Yamada et al., 2006). In our study, we found ORFs similar to large eukaryotic dsDNA viruses. However, on further analysis we could not confidently assign the origin of these genes as viral. Further research is required to study the presence of large eukaryotic ds-DNA viruses from the termite gut.\\nTermite Gut Microviruses Represent a Putative New Sub-family\\nAs compared to double-stranded DNA viruses, single-stranded DNA (ssDNA) viruses were present in a lower abundance (1\\u201310%). Colony 1 had the most diverse community of ssDNA viruses, with genes from 38 different types of ssDNA viruses followed by Colony 2 (26 types) and Colony 3 (15 types). Most of the dominant ssDNA viruses were conserved in all the three colonies along with some inter-colonial differences (Figure S5).\\nPhylogenetic analysis of VP1 major capsid gene indicated a diverse population of Microviridae in the termite gut. We were able to construct 12 novel complete genomes of Microviridae from all three termite colonies. Phylogenetically, VP1 from LSPY01000110 was placed in a cluster with Dysgonomonas macrotermitis, a bacterium of the phylum Bacteroidetes, which was previously isolated from the gut of a higher termite (Macrotermes barneyi) (Yang et al., 2014). Bacteria of this genus are also known to be a part of the gut community of the FST (Husseneder et al., 2009, 2010). LSPY01000110 showed synteny to a contig from the genome of a D. macrotermitis (Figure 5). It has been previously reported that Microviridae bacteriophage can undergo a temperate life cycle in Bacteroidetes (Krupovic and Forterre, 2011). The D. macrotermitis prophage-like sequence and LSPY01000110 showed the same gene order (VP1-ORF2-VP2-VP4) followed by five ORFs encoding hypothetical proteins in D. macrotermitis and four ORFs in case of LSPY01000110. The hypothetical proteins showed no similarity to each other. Although LSPY01000110 and D. macrotermitis prophage were clustered with the Alpavirinae subfamily, comparative genomics showed very little similarity between the two clusters.\\nFigure 5. A maximum likelihood phylogenetic tree of Microviridae VP1 from the guts of workers from three termite colonies along with the comparative genomic analysis of closely related Microviruses. Nodes with a bootstrap score of more than 70% are indicated by gray circles. For this figure, only VP1 genes from putative full Microviridae genomes were used. VP1 is colored red, VP2: green, VP4: blue. An extra ORF found after the VP1gene of Sukshmaviriane is colored pink.\\nPhylogenetic analysis of the VP1 sequences of LSPZ01000262 and LSPY01000148 placed them in the Gokushovirinae subfamily (Figure 5). The genome arrangements of LSPZ01000262 and LSPY01000148 were different from previously described Gokushovirinae sequences (Quaiser et al., 2015). No clear distinguishable VP5 sequences were observed in LSPZ01000262 and LSPY01000148.\\nVP1 from LSPY01000144 and LSPY01000166 could not be confidently placed in any cluster. VP1 from seven contigs formed a completely separate cluster from all of the other Microviridae. The genome arrangement also showed a conserved order (VP1-ORF1-VP2-VP4-ORF2). In this cluster, the ORF present after theVP1 gene encoded for a hypothetical protein. This protein did not match any known protein in the NCBI database but showed a high similarity to proteins in the cluster. The protein encoded by the ORF after VP4 from some contigs from the cluster showed similarity to Gokushovirinae VP5. Based on the genome arrangement and the VP1 phylogeny, we propose a new subfamily Sukshmavirinae (Sukshma is the Sanskrit word for \\u201csmall\\u201d) for the sequences observed in the termite gut virome.\\nA Diverse Community of Circoviruses Is Present in the Termite Gut\\nCircoviruses are small ssDNA viruses known to infect a number of higher eukaryotes (Todd et al., 2001). In recent years Circovirus-like genomes have been identified to be associated with a variety of animals including many insects (Rosario et al., 2011; Garigliany et al., 2015). The exact role of Circoviruses associated with various animals is currently not understood completely.\\nFrom all three termite colonies, 10 novel Circoviridae-like genomes were assembled. The genome size ranged between 1,388 and 5,851 bp. All the genomes encoded the Circoviridae Rep protein which is considered as the signature gene of the family. Phylogenetic analysis of the Rep gene showed two distinct groups, one belonging to the Cycloviruses and the other to the Circoviruses (Figure 6). There was no correlation between the host of these viruses and the phylogenetic placement of the Rep proteins. Termite gut Rep proteins were distributed all over the phylogenetic tree with only one sequence clustered in the Cyclovirus group.\\nFigure 6. Unrooted maximum likelihood phylogenetic tree of Circoviridae replication initiation protein. The hosts of the Circoviruses are displayed in a picture next to the sequence. Environmental Circoviruses are shown in green. Insect related Circoviruses are shown in red.\\nSo far, there have been very limited data on viruses capable of infecting termites (Al Fazairy and Hassan, 1988). It has been suggested that a virus infecting termites would be an ideal candidate for biological control (Chouvenc et al., 2011). The origin of the Circoviruses found in the termite gut remains unclear. Whether these viruses are environmental and were ingested by the termites or they actually infect the termites is an area of future research.\\nFunctional Annotation Shows a High Degree of Conservation in the Termite Gut Virome\\nMG-RAST functional analysis assigned functions to 14.5% of the predicted proteins from Colony 1, 27.62% from Colony 2 and 33.17% from Colony 3. These results show that the termite virome is largely comprised of proteins whose function still remains unknown. Functional annotation of the assigned genes indicated that phage, prophages, transposable elements, and plasmids were present in the highest abundance in all the three termite colonies (Figure S6).\\nThe presence of conserved sequences was expected because most of the bacteriophages carry out the same core basic functions such as replication, lysis, packaging, and host integration during their life cycle. To study the conservation of functional genes in the gut virome, we analyzed the orthologous gene clusters present in the termite colonies. Orthologous genes may represent a difference in the sequence but the function is generally conserved. The orthovenn analysis of the gut virome from all the three termite colonies showed that all the 53,000 protein sequences formed a total of 9,625 clusters. Of these clusters, 8,317 orthologous clusters contained genes from at least two of the three termite colonies. A total of 3,823 clusters comprised of 12,295 protein sequences were shared by all three colonies. The data suggest that the FST gut virome we sequenced, has a core set of functional genes that is conserved between all the three termite colonies.\\nTermites Possibly Harbor a Core Virome Comprised of Bacteriophages Infecting Protozoal Endosymbiotic Bacteria\\nThe FST is native to China and was introduced into the USA (Hawaii) over a century ago (Husseneder et al., 2012). The first recorded report of FSTs on the US mainland is from 1957 (Chambers et al., 1988). A previous study showed that the bacterial composition of the FST gut between the native (China) and the introduced (New Orleans, USA) populations did not change significantly (Husseneder et al., 2010). The obligatory symbionts of the gut protozoa form the core of the conserved bacterial community of the FST gut (Noda et al., 2005). These symbionts are found exclusively in the termite gut. In this study we reported two bacteriophages LSPY01000004 and LSPY01000006, infecting the obligatory endosymbiotic bacteria. The similarity between LSPY01000004, LSPY01000006, and phage ProJPt-1Bp1 is remarkable despite the fact that they were sequenced independently at different time points in geographically well-separated areas (USA and Japan) and from two different species of termites. It has been speculated that phage related genes might be ubiquitous in the termite gut (Tadmor et al., 2011). In that study, closely related bacteriophage genes were found in different species of termites collected almost 20 years apart in two different geographic locations (Tadmor et al., 2011). Interestingly, we have also found LSPY01000004 genes in other FST gut samples collected from additional colonies in New Orleans at different time points (unpublished data).\\nA core virome has been previously observed in human saliva and lower respiratory tract (Willner et al., 2009; Pride et al., 2012). In a study of human virome, bacteriophage crAssphage related sequences were found globally distributed in many human fecal metagenome samples (Dutilh et al., 2014). Based on the taxonomic and functional overlap of phages among the three different termite colonies evidenced by shared phages and gene functions, we hypothesize that termites also harbor a highly conserved core virome. We are aware that the main limitation of this study is a small sample size from the same geographic area. It cannot be ruled out that similar environmental conditions may be responsible for observed similarities in the virome of the three termite colonies. It should be noted that at this point the \\u201ctermite core virome\\u201d is a hypothesis and including more termite samples from various geographical areas will help to corroborate or refute our hypothesis.\\nPhylogenetic analysis of the terminase and integrase genes indicated that termite gut viruses show a high degree of similarity to prophage genes from diverse bacteria. Termites are highly dependent on their gut bacteria to complement their own metabolism, and changes in the bacterial population have been shown to negatively affect the termite host (Rosengaus et al., 2011; Brune, 2014). The impact of bacteriophage pressure on the bacteria is not only known to alter species composition but also their metabolic processes (Middelboe et al., 2009). It would be essential for the termite and the gut bacteria to maintain a functionally conserved set of biochemical pathways despite the presence of bacteriophage pressure. Hence, it would be advantageous to the termite host and the symbiotic gut community upon which it relies, if the bacteriophage adopts a temperate life style rather than lytic one. In turn, the phage replicates with a thriving bacterial host population and is transferred throughout the termite colony via social interactions. Manipulating the bacterial host abundance and studying the bacteriophage dynamics would further help us understand the lytic-lysogenic life style changes in the termite gut.\\nPrevious studies have shown that temperate bacteriophages can protect the bacterial host from other bacteriophages by superinfection immunity (Bondy-Denomy and Davidson, 2014). Termites are soil dwelling and the gut bacteria must be encountering a number of environmental bacteriophages. It would be interesting to study whether termite gut bacteriophages prevent the gut bacteria from environmental bacteriophages via superinfection immunity. In the future, we intend to develop termites as a model system to study the complicated quadripartite relationship between bacteria, bacteriophages, gut protozoa, and the termites themselves.\\nAuthor Contributions\\nCT designed and conducted the experiments and wrote the manuscript, CH designed the experiments, supervised the project, and edited the manuscript.\\nFunding\\nThe study was funded by the Louisiana Agricultural Experiment Station.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nWe would like to thank Dr. Carrie Cottone, Eric Guidry, and Barry Yokum, New Orleans Mosquito and Termite Control Board for providing the termites used in the study. We also would like to thank Dr. Chris Gissendanner, University of Louisiana at Monroe, for valuable comments on the manuscript.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2017.02548/full#supplementary-material\\nSupplementary Table 1. Table describing the characteristics of predicted tailed bacteriophage genomes. *LSPY01000009, LSPZ01000022 and LSQA01000015 represent similar bacteriophages. Only data for LSPY01000009 is shown here.\\nFigure S1. A maximum likelihood phylogenetic tree of large terminase subunit of type terminase_3 (A), terminase_GPA (B), terminase_1 (C). The nodes with a bootstrap value of 70% or more are indicated by a circular symbol. Sequences from the termite gut are colored purple. Bacteriophages: red. Firmicutes: orange. Spirochetes: dark blue. Gammaproteobacteria: bright green. Bacteroidetes: dark green. Alphaproteobacteria: gray. Others: black.\\nFigure S2. A maximum likelihood phylogenetic tree of phage integrase genes. The nodes with a bootstrap value of 70% or more are indicated by a circular symbol. Sequences from the termite gut are colored purple Firmicutes: orange. Actinobacteria: red. Spirochetes: dark blue. Gammaproteobacteria: bright green. Bacteroidetes: dark green. Planctomycetes: sky blue. Others: black.\\nFigure S3. Unrooted maximum likelihood phylogenetic tree of RNAP beta subunit of virion and non-virion subunits from phiKZ-like bacteriophages. RNAPs from LSPZ01000002 are shown in purple. RNAP from Thermus thermophilus is used as an outgroup.\\nFigure S4. (A) Comparative genomic analysis of LSPY01000009 and LSQA01000015 with Arthrobacter phage Mudcat and Rhodococcus phage ReqiPepy6. (B) Comparative genomic analysis of LSPZ01000027 with Clostridium phage cdhm1 and Clostridium phage phimmp01. (C) Comparative genomic analysis of LSQA01000020 with Croceibacter phage P2559S and Croceibacter phage P2559Y. All the genomes have been rearranged so that the start codon of the large terminase subunit is the first base in the sequence. The figures were generated using Easyfig software with tblastx. The structural genes are indicated in red color, DNA metabolism related genes in blue, cell lysis in green, virulence associated protein in pink, yellow anti-repressor, reverse transcriptase in purple. All the other genes are indicated in sky blue color.\\nFigure S5. Taxonomic distribution (%) of the predominant ssDNA bacteriophage genes (normalized) from the guts of the termites from three different colonies. The data were generated using Metavir-2 server by comparing the predicted proteins to NCBI virus protein database. Top BLAST hits with an e-value of 10\\u22125 or less were used.\\nFigure S6. Normalized percent distribution of the functional categories of predominant genes from the guts of the termites from three different colonies. The data were generated using MG-RAST server by comparing the predicted proteins to MD5nr database.\\nReferences\\nAl Fazairy, A. A., and Hassan, F. (1988). Infection of termites by Spodoptera littoralis nuclear polyhedrosis virus. Int. J. Trop. Insect Sci. 9, 37\\u201339. doi: 10.1017/S1742758400009991\\nCrossRef Full Text | Google Scholar\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455\\u2013477. doi: 10.1089/cmb.2012.0021\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBassford, P., Diedrich, D. L., Schnaitman, C. L., and Reeves, P. (1977). Outer membrane proteins of Escherichia coli. VI. Protein alteration in bacteriophage-resistant mutants. J. Bacteriol. 131, 608\\u2013622.\\nPubMed Abstract | Google Scholar\\nBellas, C. M., Anesio, A. M., and Barker, G. (2015). Analysis of virus genomes from glacial environments reveals novel virus groups with unusual host interactions. Front. Microbiol. 6:656. doi: 10.3389/fmicb.2015.00656\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBondy-Denomy, J., and Davidson, A. R. (2014). When a virus is not a parasite: the beneficial effects of prophages on bacterial fitness. J. Microbiol. 52, 235\\u2013242. doi: 10.1007/s12275-014-4083-3\\nCrossRef Full Text | Google Scholar\\nBordenstein, S. R., Marshall, M. L., Fry, A. J., Kim, U., and Wernegreen, J. J. (2006). The tripartite associations between bacteriophage, Wolbachia, and arthropods. PLoS Pathog. 2, 384\\u2013393. doi: 10.1371/journal.ppat.0020043\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBoudry, P., Semenova, E., Monot, M., Datsenko, K. A., Lopatina, A., Sekulovic, O., et al. (2015). Function of the CRISPR-cas system of the human pathogen Clostridium difficile. MBio 6, e01112\\u201301115. doi: 10.1128/mBio.01112-15\\nCrossRef Full Text | Google Scholar\\nBrum, J. R., Ignacio-Espinoza, J. C., Kim, E.-H., Trubl, G., Jones, R. M., Roux, S., et al. (2016). Illuminating structural proteins in viral \\u201cdark matter\\u201d with metaproteomics. Proc. Natl. Acad. Sci. U.S.A. 113, 2436\\u20132441. doi: 10.1073/pnas.1525139113\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBrum, J. R., and Sullivan, M. B. (2015). Rising to the challenge: accelerated pace of discovery transforms marine virology. Nat. Rev. Microbiol. 13, 147\\u2013159. doi: 10.1038/nrmicro3404\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBr\\u00fcssow, H., Canchaya, C., and Hardt, W.-D. (2004). Phages and the evolution of bacterial pathogens: from genomic rearrangements to lysogenic conversion. Microbiol. Mol. Biol. Rev. 68, 560\\u2013602. doi: 10.1128/MMBR.68.3.560-602.2004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCasjens, S. R., and Grose, J. H. (2016). Contributions of P2-and P22-like prophages to understanding the enormous diversity and abundance of tailed bacteriophages. Virology 496, 255\\u2013276. doi: 10.1016/j.virol.2016.05.022\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCeyssens, P.-J., Minakhin, L., Van Den Bossche, A., Yakunina, M., Klimuk, E., Blasdel, B., et al. (2014). Development of giant bacteriophage \\u03c6KZ is independent of the host transcription apparatus. J. Virol. 88, 10501\\u201310510. doi: 10.1128/JVI.01347-14\\nCrossRef Full Text | Google Scholar\\nChambers, D., Zungoli, P., and Hill, Jr. H. (1988). Distribution and habitats of the Formosan subterranean termite (Isoptera: Rhinotermitidae) in South Carolina. J. Econ. Entomol. 81, 1611\\u20131619. doi: 10.1093/jee/81.6.1611\\nCrossRef Full Text | Google Scholar\\nChouvenc, T., Su, N.-Y., and Grace, J. K. (2011). Fifty years of attempted biological control of termites\\u2013analysis of a failure. Biol. Control 59, 69\\u201382. doi: 10.1016/j.biocontrol.2011.06.015\\nCrossRef Full Text | Google Scholar\\nCornelissen, A., Hardies, S. C., Shaburova, O. V., Krylov, V. N., Mattheus, W., Kropinski, A. M., et al. (2012). Complete genome sequence of the giant virus OBP and comparative genome analysis of the diverse \\u03c6KZ-related phages. J. Virol. 86, 1844\\u20131852. doi: 10.1128/JVI.06330-11\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDesai, M. S., and Brune, A. (2012). Bacteroidales ectosymbionts of gut flagellates shape the nitrogen-fixing community in dry-wood termites. ISME J. 6, 1302\\u20131313. doi: 10.1038/ismej.2011.194\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDesai, M. S., Strassert, J. F., Meuser, K., Hertel, H., Ikeda-Ohtsubo, W., Radek, R., et al. (2010). Strict cospeciation of devescovinid flagellates and Bacteroidales ectosymbionts in the gut of dry-wood termites (Kalotermitidae). Environ. Microbiol. 12, 2120\\u20132132. doi: 10.1111/j.1462-2920.2009.02080.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDo, T. H., Nguyen, T. T., Nguyen, T. N., Le, Q. G., Nguyen, C., Kimura, K., et al. (2014). Mining biomass-degrading genes through Illumina-based de novo sequencing and metagenomic analysis of free-living bacteria in the gut of the lower termite Coptotermes gestroi harvested in Vietnam. J. Biosci. Bioeng. 118, 665\\u2013671. doi: 10.1016/j.jbiosc.2014.05.010\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDutilh, B. E., Cassman, N., Mcnair, K., Sanchez, S. E., Silva, G. G., Boling, L., et al. (2014). A highly abundant bacteriophage discovered in the unknown sequences of human faecal metagenomes. Nat. Commun. 5:4498. doi: 10.1038/ncomms5498\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDwivedi, B., Xue, B., Lundin, D., Edwards, R. A., and Breitbart, M. (2013). A bioinformatic analysis of ribonucleotide reductase genes in phage genomes and metagenomes. BMC Evol. Biol. 13:33. doi: 10.1186/1471-2148-13-33\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nEggers, C. H., Gray, C. M., Preisig, A. M., Glenn, D. M., Pereira, J., Ayers, R. W., et al. (2016). Phage-mediated horizontal gene transfer of both prophage and heterologous DNA by \\u03c6BB-1, a bacteriophage of Borrelia burgdorferi. Pathog. Dis. 74:ftw107. doi: 10.1093/femspd/ftw107\\nCrossRef Full Text | Google Scholar\\nEutik, M., Veivers, P., and O'Brien, R. and Slaytor, M. (1978). Dependence o\\u00ed the higher termite, Nasutitermes exitiosus and the lower termite, Coptotermes lacteus on their gut flora. J. Insect Physiol. 24, 363\\u2013368. doi: 10.1016/0022-1910(78)90076-8\\nCrossRef Full Text | Google Scholar\\nGarigliany, M.-M., B\\u00f6rstler, J., J\\u00f6st, H., Badusche, M., Desmecht, D., Schmidt-Chanasit, J., et al. (2015). Characterization of a novel circo-like virus in Aedes vexans mosquitoes from Germany: evidence for a new genus within the family Circoviridae. J. Gen. Virol. 96, 915\\u2013920. doi: 10.1099/vir.0.000036\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGarneau, J. E., Tremblay, D. M., and Moineau, S. (2008). Characterization of 1706, a virulent phage from Lactococcus lactis with similarities to prophages from other Firmicutes. Virology 373, 298\\u2013309. doi: 10.1016/j.virol.2007.12.002\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGraber, J. R., Leadbetter, J. R., and Breznak, J. A. (2004). Description of Treponema azotonutricium sp. nov. and Treponema primitia sp. nov., the first spirochetes isolated from termite guts. Appl. Environ. Microbiol. 70, 1315\\u20131320. doi: 10.1128/AEM.70.3.1315-1320.2004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGuindon, S., Dufayard, J.-F., Lefort, V., Anisimova, M., Hordijk, W., Gascuel, O., et al. (2010). New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Syst. Biol. 59, 307\\u2013321. doi: 10.1093/sysbio/syq010\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHargreaves, K. R., Kropinski, A. M., and Clokie, M. R. (2014). What does the talking?: quorum sensing signalling genes discovered in a bacteriophage genome. PLoS ONE 9:e85131. doi: 10.1371/journal.pone.0085131\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHongoh, Y., Sato, T., Noda, S., Ui, S., Kudo, T., and Ohkuma, M., et al. (2007). Candidatus Symbiothrix dinenymphae: bristle-like Bacteroidales ectosymbionts of termite gut protists. Environ. Microbiol. 9, 2631\\u20132635. doi: 10.1111/j.1462-2920.2007.01365.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHongoh, Y., Sharma, V. K., Prakash, T., Noda, S., Toh, H., Taylor, T. D., et al. (2008). Genome of an endosymbiont coupling N2 fixation to cellulolysis within protist cells in termite gut. Science 322, 1108\\u20131109. doi: 10.1126/science.1165578\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHusseneder, C., Berestecky, J., and Grace, J. (2009). Changes in composition of culturable bacteria community in the gut of the Formosan subterranean termite depending on rearing conditions of the host. Ann. Entomol. Soc. Am. 102, 498\\u2013507. doi: 10.1603/008.102.0321\\nCrossRef Full Text | Google Scholar\\nHusseneder, C., Ho, H.-Y., and Blackwell, M. (2010). Comparison of the bacterial symbiont composition of the Formosan subterranean termite from its native and introduced range. Open Microbiol. J. 4, 53\\u201366. doi: 10.2174/1874285801004010053\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHusseneder, C., Simms, D. M., Delatte, J. R., Wang, C., Grace, J. K., and Vargo, E. L., et al. (2012). Genetic diversity and colony breeding structure in native and introduced ranges of the Formosan subterranean termite, Coptotermes formosanus. Biol. Invas. 14, 419\\u2013437. doi: 10.1007/s10530-011-0087-7\\nCrossRef Full Text | Google Scholar\\nIkeda-Ohtsubo, W., and Brune, A. (2009). Cospeciation of termite gut flagellates and their bacterial endosymbionts: Trichonympha species and \\u201cCandidatus Endomicrobium trichonymphae\\u201d. Mol. Ecol. 18, 332\\u2013342. doi: 10.1111/j.1365-294X.2008.04029.x\\nCrossRef Full Text | Google Scholar\\nJang, H. B., Fagutao, F. F., Nho, S. W., Park, S. B., Cha, I. S., Yu, J. E., et al. (2013). Phylogenomic network and comparative genomics reveal a diverged member of the \\u03c6kz-related group, marine Vibrio phage \\u03c6JM-2012. J. Virol. 87, 12866\\u201312878. doi: 10.1128/JVI.02656-13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJasna, V., Parvathi, A., Pradeep Ram, A. S., Balachandran, K. K., Madhu, N. V., Nair, M., et al. (2017). Viral-induced mortality of prokaryotes in a tropical monsoonal estuary. Front. Microbiol. 8:895. doi: 10.3389/fmicb.2017.00895\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKang, I., Kang, D., and Cho, J.-C. (2016). Complete genome sequence of bacteriophage P2559Y, a marine phage that infects Croceibacter atlanticus HTCC2559. Mar. Genomics 29, 35\\u201338. doi: 10.1016/j.margen.2016.07.001\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKeegan, K. P., Glass, E. M., and Meyer, F. (2016). MG-RAST, a metagenomics service for analysis of microbial community structure and function. Methods Mol. Biol. 1399, 207\\u2013233. doi: 10.1007/978-1-4939-3369-3_13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKim, K.-H., and Bae, J.-W. (2011). Amplification methods bias metagenomic libraries of uncultured single-stranded and double-stranded DNA viruses. Appl. Environ. Microbiol. 77, 7663\\u20137668. doi: 10.1128/AEM.00289-11\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKrueger, F. (2015). Trim Galore. A Wrapper Tool Around Cutadapt and FastQC to Consistently Apply Quality and Adapter Trimming to FastQ Files. Available online at: https://github.com/FelixKrueger/TrimGalore\\nKrupovic, M., and Forterre, P. (2011). Microviridae goes temperate: microvirus-related proviruses reside in the genomes of Bacteroidetes. PLoS ONE 6:e19893. doi: 10.1371/journal.pone.0019893\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKrylov, V., Cruz, D. D., Hertveldt, K., and Ackermann, H.-W. (2007). \\u201c\\u03d5KZ-like viruses,\\u201d a proposed new genus of myovirus bacteriophages. Arch. Virol. 152, 1955\\u20131959. doi: 10.1007/s00705-007-1037-7\\nCrossRef Full Text | Google Scholar\\nLane, D. J. (1991). \\u201c16S/23S rRNA sequencing,\\u201d in Nucleic Acid Techniques in Bacterial Systematics, ed E. S. A. M. Goodfellow (New York, NY: John Wiley and Sons), 115\\u2013175.\\nGoogle Scholar\\nLavigne, R., Darius, P., Summer, E. J., Seto, D., Mahadevan, P., Nilsson, A. S., et al. (2009). Classification of Myoviridae bacteriophages using protein sequence similarity. BMC Microbiol. 9:224. doi: 10.1186/1471-2180-9-224\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLavysh, D., Sokolova, M., Minakhin, L., Yakunina, M., Artamonova, T., Kozyavkin, S., et al. (2016). The genome of AR9, a giant transducing Bacillus phage encoding two multisubunit RNA polymerases. Virology 495, 185\\u2013196. doi: 10.1016/j.virol.2016.04.030\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLerouge, I., and Vanderleyden, J. (2002). O-antigen structural variation: mechanisms and possible roles in animal/plant\\u2013microbe interactions. FEMS Microbiol. Rev. 26, 17\\u201347. doi: 10.1111/j.1574-6976.2002.tb00597.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLopes, A., Tavares, P., Petit, M.-A., Gu\\u00e9rois, R., and Zinn-Justin, S. (2014). Automated classification of tailed bacteriophages according to their neck organization. BMC Genomics 15:1027. doi: 10.1186/1471-2164-15-1027\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMcNair, K., Bailey, B. A., and Edwards, R. A. (2012). PHACTS, a computational approach to classifying the lifestyle of phages. Bioinformatics 28, 614\\u2013618. doi: 10.1093/bioinformatics/bts014\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMessenger, M. T., and Mullins, A. J. (2005). New flight distance recorded for Coptotermes formosanus (Isoptera: Rhinotermitidae). Fla. Entomol. 88, 99\\u2013100. doi: 10.1653/0015-4040(2005)088[0099:NFDRFC]2.0.CO;2\\nCrossRef Full Text | Google Scholar\\nMiddelboe, M., Holmfeldt, K., Riemann, L., Nybroe, O., and Haaber, J. (2009). Bacteriophages drive strain diversification in a marine Flavobacterium: implications for phage resistance and physiological properties. Environ. Microbiol. 11, 1971\\u20131982. doi: 10.1111/j.1462-2920.2009.01920.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMinot, S., Sinha, R., Chen, J., Li, H., Keilbaugh, S. A., Wu, G. D., et al. (2011). The human gut virome: inter-individual variation and dynamic response to diet. Genome Res. 21, 1616\\u20131625. doi: 10.1101/gr.122705.111\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMizuno, C. M., Ghai, R., Sagha\\u00ef, A., L\\u00f3pez-Garc\\u00eda, P., and Rodriguez-Valera, F. (2016). Genomes of abundant and widespread viruses from the deep ocean. MBio 7, e00805\\u201300816. doi: 10.1128/mBio.00805-16\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMosher, R. A., Direnzo, A. B., and Mathews, C. K. (1977). Bacteriophage T4 virion dihydrofolate reductase: approaches to quantitation and assessment of function. J. Virol. 23, 645\\u2013658.\\nPubMed Abstract | Google Scholar\\nNoda, S., Iida, T., Kitade, O., Nakajima, H., Kudo, T., and Ohkuma, M., et al. (2005). Endosymbiotic Bacteroidales bacteria of the flagellated protist Pseudotrichonympha grassii in the gut of the termite Coptotermes formosanus. Appl. Environ. Microbiol. 71, 8811\\u20138817. doi: 10.1128/AEM.71.12.8811-8817.2005\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nNoda, S., Kitade, O., Inoue, T., Kawai, M., Kanuka, M., Hiroshima, K., et al. (2007). Cospeciation in the triplex symbiosis of termite gut protists (Pseudotrichonympha spp.), their hosts, and their bacterial endosymbionts. Mol. Ecol. 16, 1257\\u20131266. doi: 10.1111/j.1365-294X.2006.03219.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nNoda, S., Ohkuma, M., Yamada, A., Hongoh, Y., and Kudo, T. (2003). Phylogenetic position and in situ identification of ectosymbiotic spirochetes on protists in the termite gut. Appl. Environ. Microbiol. 69, 625\\u2013633. doi: 10.1128/AEM.69.1.625-633.2003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nOttesen, E. A., Hong, J. W., Quake, S. R., and Leadbetter, J. R. (2006). Microfluidic digital PCR enables multigene analysis of individual environmental bacteria. Science 314, 1464\\u20131467. doi: 10.1126/science.1131370\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPaterson, S., Vogwill, T., Buckling, A., Benmayor, R., Spiers, A. J., Thomson, N. R., et al. (2010). Antagonistic coevolution accelerates molecular evolution. Nature 464, 275\\u2013278. doi: 10.1038/nature08798\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPletnev, E. A., Krylov, S. V., Shaburova, O. V., Burkal'tseva, M. V., Miroshnikov, K. A., Krylov, V. N., et al. (2010). Pseudolysogeny of Pseudomonas aeruginosa bacteria infected with phiKZ-like bacteriophages. Genetika 46, 26\\u201332. Available online at: http://europepmc.org/abstract/med/20198876\\nPubMed Abstract | Google Scholar\\nPramono, A. K., Kuwahara, H., Itoh, T., Toyoda, A., Yamada, A., and Hongoh, Y. (2017). Discovery and complete genome sequence of a bacteriophage from an obligate intracellular symbiont of a cellulolytic protist in the termite gut. Microbes Environ. 32, 112\\u2013117. doi: 10.1264/jsme2.ME16175\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPride, D. T., Salzman, J., Haynes, M., Rohwer, F., Davis-Long, C., White, R. A. III., et al. (2012). Evidence of a robust resident bacteriophage population revealed through analysis of the human salivary virome. ISME J. 6, 915\\u2013926. doi: 10.1038/ismej.2011.169\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nQuaiser, A., Dufresne, A., Ballaud, F., Roux, S., Zivanovic, Y., Colombet, J., et al. (2015). Diversity and comparative genomics of Microviridae in Sphagnum-dominated peatlands. Front. Microbiol. 6:375. doi: 10.3389/fmicb.2015.00375\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRodriguez-Brito, B., Li, L., Wegley, L., Furlan, M., Angly, F., Breitbart, M., et al. (2010). Viral and microbial community dynamics in four aquatic environments. ISME J. 4, 739\\u2013751. doi: 10.1038/ismej.2010.1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRosario, K., Marinov, M., Stainton, D., Kraberger, S., Wiltshire, E. J., Collings, D. A., et al. (2011). Dragonfly cyclovirus, a novel single-stranded DNA virus discovered in dragonflies (Odonata: Anisoptera). J. Gen. Virol. 92, 1302\\u20131308. doi: 10.1099/vir.0.030338-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRosengaus, R. B., Zecher, C. N., Schultheis, K. F., Brucker, R. M., and Bordenstein, S. R. (2011). Disruption of the termite gut microbiota and its prolonged consequences for fitness. Appl. Environ. Microbiol. 77, 4303\\u20134312. doi: 10.1128/AEM.01886-10\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRoux, S., Hallam, S. J., Woyke, T., and Sullivan, M. B. (2015b). Viral dark matter and virus\\u2013host interactions resolved from publicly available microbial genomes. Elife 4:e08490. doi: 10.7554/eLife.08490\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRoux, S., Solonenko, N. E., Dang, V. T., Poulos, B. T., Schwenck, S. M., Goldsmith, D. B., et al. (2016). Towards quantitative viromics for both double-stranded and single-stranded DNA viruses. PeerJ 4:e2777. doi: 10.7717/peerj.2777\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRoux, S., Tournayre, J., Mahul, A., Debroas, D., and Enault, F. (2014). Metavir 2: new tools for viral metagenome comparison and assembled virome analysis. BMC Bioinformatics 15:76. doi: 10.1186/1471-2105-15-76\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSantiago-Rodriguez, T. M., Ly, M., Bonilla, N., and Pride, D. T. (2015). The human urine virome in association with urinary tract infections. Front. Microbiol. 6:14. doi: 10.3389/fmicb.2015.00014\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nShin, S., Lu, G., Cai, M., and Kim, K.-S. (2005). Escherichia coli outer membrane protein A adheres to human brain microvascular endothelial cells. Biochem. Biophys. Res. Commun. 330, 1199\\u20131204. doi: 10.1016/j.bbrc.2005.03.097\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nShinzato, N., Muramatsu, M., Matsui, T., and Watanabe, Y. (2005). Molecular phylogenetic diversity of the bacterial community in the gut of the termite Coptotermes formosanus. Biosci. Biotechnol. Biochem. 69, 1145\\u20131155. doi: 10.1271/bbb.69.1145\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSullivan, M. B. (2015). Viromes, not gene markers, for studying double-stranded DNA virus communities. J. Virol. 89, 2459\\u20132461. doi: 10.1128/JVI.03289-14\\nCrossRef Full Text | Google Scholar\\nSummer, E., Liu, M., Gill, J., Grant, M., Chan-Cortes, T., Ferguson, L., et al. (2011). Genomic and functional analyses of Rhodococcus equi phages ReqiPepy6, ReqiPoco6, ReqiPine5, and ReqiDocB7. Appl. Environ. Microbiol. 77, 669\\u2013683. doi: 10.1128/AEM.01952-10\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTadmor, A. D., Ottesen, E. A., Leadbetter, J. R., and Phillips, R. (2011). Probing individual environmental bacteria for viruses by using microfluidic digital PCR. Science 333, 58\\u201362. doi: 10.1126/science.1200758\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTangherlini, M., Dell'anno, A., Allen, L. Z., Riccioni, G., and Corinaldesi, C. (2016). Assessing viral taxonomic composition in benthic marine ecosystems: reliability and efficiency of different bioinformatic tools for viral metagenomic analyses. Sci. Rep. 6:28428. doi: 10.1038/srep28428\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTikhe, C. V., Martin, T. M., Gissendanner, C. R., and Husseneder, C. (2015). Complete genome sequence of Citrobacter phage CVT22 isolated from the gut of the Formosan subterranean termite, Coptotermes formosanus Shiraki. Genome Announc. 3, e00408\\u201300415. doi: 10.1128/genomeA.00408-15\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTouchon, M., Moura De Sousa, J. A., and Rocha, E. P. C. (2017). Embracing the enemy: the diversification of microbial gene repertoires by phage-mediated horizontal gene transfer. Curr. Opin. Microbiol. 38, 66\\u201373. doi: 10.1016/j.mib.2017.04.010\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWang, Y., Coleman-Derr, D., Chen, G., and Gu, Y. Q. (2015). OrthoVenn: a web server for genome wide comparison and annotation of orthologous clusters across multiple species. Nucleic Acids Res. 43, W78\\u2013W84. doi: 10.1093/nar/gkv487\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWarnecke, F., Luginb\\u00fchl, P., Ivanova, N., Ghassemian, M., Richardson, T. H., Stege, J. T., et al. (2007). Metagenomic and functional analysis of hindgut microbiota of a wood-feeding higher termite. Nature 450, 560\\u2013565. doi: 10.1038/nature06269\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWilhelm, S. W., and Suttle, C. A. (1999). Viruses and nutrient cycles in the sea viruses play critical roles in the structure and function of aquatic food webs. Bioscience 49, 781\\u2013788. doi: 10.2307/1313569\\nCrossRef Full Text | Google Scholar\\nWillner, D., Furlan, M., Haynes, M., Schmieder, R., Angly, F. E., Silva, J., et al. (2009). Metagenomic analysis of respiratory tract DNA viral communities in cystic fibrosis and non-cystic fibrosis individuals. PLoS ONE 4:e7370. doi: 10.1371/journal.pone.0007370\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYang, Y.-J., Zhang, N., Ji, S.-Q., Lan, X., Shen, Y.-L., Li, F.-L., et al. (2014). Dysgonomonas macrotermitis sp. nov., isolated from the hindgut of a fungus-growing termite. Int. J. Syst. Evol. Microbiol. 64, 2956\\u20132961. doi: 10.1099/ijs.0.061739-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nZablocki, O., Van Zyl, L., Adriaenssens, E. M., Rubagotti, E., Tuffin, M., Cary, S. C., et al. (2014). High-level diversity of tailed phages, eukaryote-associated viruses, and virophage-like elements in the metaviromes of antarctic soils. Appl. Environ. Microbiol. 80, 6888\\u20136897. doi: 10.1128/AEM.01525-14\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nZeng, H., Zhang, J., Li, C., Xie, T., Ling, N., Wu, Q., et al. (2017). The driving force of prophages and CRISPR-Cas system in the evolution of Cronobacter sakazakii. Sci. Rep. 7:40206. doi: 10.1038/srep40206\\nCrossRef Full Text | Google Scholar\\nZhu, A., Sunagawa, S., Mende, D. R., and Bork, P. (2015). Inter-individual differences in the gene content of human gut bacterial species. Genome Biol. 16, 82. doi: 10.1186/s13059-015-0646-9\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01870\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01870/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:48:53.299Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01870\",\n",
      "            \"title\": \"Frontiers | Newly Isolated Bacteriophages from the Podoviridae, Siphoviridae, and Myoviridae Families Have Variable Effects on Putative Novel Dickeya spp.\",\n",
      "            \"description\": \"Soft rot pathogenic bacteria from the genus Dickeya cause severe economic losses in orchid nurseries worldwide, and there is no effective control currently available. In the last decade, the genus Dickeya has undergone multiple changes as multiple new taxa have been described, and just recently a new putative Dickeya species was reported. This study reports the isolation of three bacteriophages active against putative novel Dickeya spp. isolates from commercially produced infected orchids that show variable host-range profiles. Bacteriophages were isolated through enrichment from Dickeya-infected orchid tissue. Convective interaction media monolith chromatography was used to isolate bacteriophages from wastewaters, demonstrating its suitability for the isolation of infective bacteriophages from natural sources. Based on bacteriophage morphology, all isolated bacteriophages were classified as being in the order Caudovirales, belonging to three different families, Podoviridae, Myoviridae and Siphoviridae. The presence of three different groups of bacteriophages was confirmed by analysing the bacteriophage specificity of bacterial hosts, restriction fragment length polymorphism and plaque morphology. Bacteriophage BF25/12, the first reported Podoviridae bacteriophage effective against Dickeya spp., was selected for further characterisation. Its genome sequence determined by next-generation sequencing showed limited similarity to other characterised Podoviridae bacteriophages. Interactions among the bacteriophages and Dickeya spp. were examined using transmission electron microscopy, which revealed degradation of electron-dense granules in response to bacteriophage infection in some Dickeya strains. The temperature stability of the chosen Podoviridae bacteriophage monitored over one year showed a substantial decrease in the survival of bacteriophages stored at -20 \\u00b0C over longer periods. It showed susceptibility to low pH and UV radiation but was stable in neutral and alkaline pH. Furthermore, the stability of the tested bacteriophage was also connected to the incubation medium and bacteriophage concentration at certain pH values. Finally, the emergence of bacteriophage-resistant bacterial colonies is highly connected to the concentration of bacteriophages in the bacterial environment. This is the first report on bacteriophages against Dickeya from the Podoviridae family to expand on potential bacteriophages to include in bacteriophage cocktails as biocontrol agents. Some of these bacteriophage isolates also showed activity against Dickeya solani, an aggressive strain that causes the soft rot of potatoes,\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nDickeya spp. (formerly Erwinia chrysanthemi) are plant pathogenic bacteria that can cause soft rot disease across a wide range of crops and ornamental plants worldwide (Czajkowski et al., 2011; Toth et al., 2011; Adriaenssens et al., 2012). These Gram-negative, facultative anaerobic bacteria of the \\u03b3-Proteobacteria subdivision cluster into the Enterobacteriaceae family. Dickeya characteristically produce cell wall-degrading enzymes that are secreted during their infection of plants, resulting in the plant\\u2013tissue maceration that is typical of soft rot disease (Czajkowski et al., 2011; Adriaenssens et al., 2012). Dickeya can be spread over long distances via infected plants and can also live as epiphytes or facultative saprophytes in soil and ground water (Reverchon and Nasser, 2013). They have been detected in a variety of water sources, and irrigation water has been reported as the probable source of their infection of potatoes in Australia (Toth et al., 2011). There are currently no effective chemical agents to control Dickeya soft rot infection, resulting in significant economic loss, particularly in terms of potato production (Adriaenssens et al., 2012; Czajkowski et al., 2014).\\nThe genus Dickeya currently encompasses eight species\\u2014Dickeya zeae, D. dadantii, D. chrysanthemi, D. solani, D. aquatica, D. dianthicola, D. paradisiaca, and D. fangzhongdai (Samson et al., 2005; Toth et al., 2011; Adriaenssens et al., 2012; Tian et al., 2016). The use of molecular tools has revealed further evolution of these species, with the occasional description of strains with characteristics different from the previously described species, such as unassigned lineages (UDLs), in the Dickeya phylogenetic backbone (Samson et al., 2005; Parkinson et al., 2009; Adriaenssens et al., 2012; Van Vaerenbergh et al., 2012). Van Vaerenbergh et al. (2012) reported the introduction of UDLs, mostly for Dickeya isolates from ornamental plants. Furthermore, some of the isolates corresponding to UDL-3 and UDL-4 based on the fliC sequence were described as putative new species in the genus Dickeya (Ali\\u010d et al., 2017).\\nThe use of bacteriophages as biocontrol agents is an attractive option for controlling bacterial diseases of plants (Jones et al., 2007; Adriaenssens et al., 2012; Buttimer et al., 2017). Bacteriophages can show high specificity for the bacterial hosts they infect, and only lyse bacterial cells, thus providing targeted disease management. Bacteriophages are naturally present in the environment, such as in soil, water, plants, and animals, where they can persist through host-dependent self-replication over long periods. As bacteriophages cannot infect eukaryotic cells, they are safe to use (reviewed in Jones et al., 2007). While only lytic bacteriophages can be used for bacteriophage therapy and biocontrol (Jones et al., 2007, 2012), their relatively easy and inexpensive production means that they are of commercial interest (reviewed in Jones et al., 2007).\\nDespite these advantages, few bacteriophages have demonstrated repeated, successful applications in plant disease management. Apart from the obvious challenges, such as resistance development in bacteria (Katznelson, 1937; Levin and Bull, 2004), bacteriophage therapies must take into account the complex dynamics among bacteriophages, bacteria and their environment, which largely remain unexplored (Levin and Bull, 2004). In addition, suitable protective formulation and delivery of bacteriophages to affected plants must be optimized to ensure their efficient application and their survival during this application (Jones et al., 2007; Frampton et al., 2012).\\nThe possibility of plant disease control using bacteriophages has been studied for various plant pathogenic bacteria (e.g., E. amylovora, Agrobacterium tumefaciens, Ralstonia solanacearum, Streptomyces scabies, Pseudomonas sp., Xanthomonas sp., Pectobacterium sp., Xylella fastidiosa, and Dickeya sp.) (reviewed in Buttimer et al., 2017). The idea of using bacteriophages as controlling agents for Dickeya spp. has been proposed before, but only limited attempts have been made to isolate such lytic bacteriophages (Czajkowski et al., 2014). Adriaenssens et al. (2012) described the T4-related bacteriophages LIMEstone1 and LIMEstone2 that specifically infect D. solani. In their study, both bacteria and bacteriophages were isolated from soil taken from a potato field. The first reported experimental field trial for the bacteriophage treatment of potato tubers infected with D. solani resulted in improved yields, which led to the proposal of the bacteriophages LIMEstone1 and LIMEstone2 as effective therapeutics in an agricultural setting (Adriaenssens et al., 2012). Czajkowski et al. (2014) isolated and identified nine more bacteriophages that showed wider host ranges and that were all active against D. solani. All these bacteriophages prevented the growth of D. solani in vitro and protected potato tuber tissue from the maceration caused by D. solani.\\nHowever, further studies are required to determine the long-term effectiveness of bacteriophages and to perform field trials. Some bacteriophages with broader host ranges have been reported to infect some Pectobacterium strains as well as D. solani (Czajkowski et al., 2015). Recently, bacteriophages that are active against D. dadantii isolates were isolated from potato plants. These bacteriophage isolates are members of the Myoviridae and Siphoviridae families and were all isolated from the water of the Caspian Sea. Their effectiveness was tested in vitro, and they showed promising biocontrol potential. However, no field trials have been conducted to date (Soleimani-Delfan et al., 2015).\\nThe variously reported investigations and trials involving Dickeya spp. have so far only been carried out in terms of improving potato production (Adriaenssens et al., 2012; Czajkowski et al., 2014). However, ornamental plants with high added value, such as orchids, have significant economic importance in agricultural production, and due to extensive worldwide trade, if infected, these can facilitate the spread and possible expansion of the pathogenic bacteria host range. The aim of the study was to isolate and characterize bacteriophages active against putative new Dickeya spp. as a research tool to study the bacteria\\u2013bacteriophage system and as a potential biocontrol component.\\nHere, we report on the successful application of the convective interaction media (CIM) method for bacteriophage isolation from complex environmental samples. We show the activity of bacteriophage isolates against two previously determined putative Dickeya spp. (UDL-3 and UDL-4; Ali\\u010d et al., 2017). For the isolates, we determine their host range and restriction fragment polymorphism (RFLP) profiles and characterize them according to their morphology using transmission electron microscopy. Finally, we describe the first reported Podoviridae bacteriophage effective against Dickeya spp., BF25/12, including its annotated genome sequence.\\nMaterials and Methods\\nBacterial Strains and Media\\nDickeya strains were isolated from Phalaenopsis orchid leaves that showed soft rot symptoms and were identified as a putative new species within the genus Dickeya (Ali\\u010d et al., 2017). Based on the fliC gene sequence, the bacteria isolates were classified as members of UDL-3 and UDL-4 (Ali\\u010d et al., 2017). These bacteria were routinely grown on Luria Bertani (LB) medium with 1.5% agar and 0.05% (w/v) NaCl at 28\\u201330\\u00b0C. For liquid preparations, the bacteria cultures were grown in LB broth at 28\\u00b0C, with agitation at 200 rpm unless otherwise stated. LB with 0.4% agar was used for overlays. For long-term storage, the bacteria cultures and isolates were kept in a bacterial and fungal storage system (Microbank; bioTRADING, Mijdrecht, Netherlands) at \\u2264-76\\u00b0C.\\nIsolation of Bacteriophages\\nIsolation of Bacteriophages from Plant Material\\nBacteriophages against Dickeya UDL-3 were isolated from Phalaenopsis orchid leaves that showed soft rot symptoms. The orchids were obtained from a commercial orchid production site. Extracts of the plant tissues were prepared by commuting or macerating plant material in 45 mL of Trypticase Soy Broth (30 g Trypticase Soy Broth [BD 211768], double-distilled water to 1 L, pH 9). Sterile 450 \\u03bcL 1 M MgSO4 and 90 \\u03bcL 1 M CaCl2 were added to the mixture. The extract was incubated for 1 h at room temperature with constant rotation (Heto Mastermix rotator), followed by centrifugation at 10,000 \\u00d7 g for 10 min. The resulting supernatant was carefully poured into new 50 mL tubes (Falcon). The clear extract was again centrifuged as above. Finally, the supernatant was sterile filtered through 0.2-\\u03bcm membranes (Millipore) and stored at 4\\u00b0C until enrichment. Enrichment of the bacteriophages in the extract was performed with a mixture of different Dickeya isolates (Table 1). For enrichment, 15 mL double-strength LB (20 g tryptone, 10 g yeast extract, 1 g NaCl, double-distilled water to 1 L) with 300 \\u03bcL 1 M MgSO4 and 60 \\u03bcL 1 M CaCl2 was mixed with 15 mL extract of bacteriophages. To this, 150 \\u03bcL bacteria in log phase growth (12\\u201314 h incubation at 28\\u00b0C, with shaking at 200 rpm) was added at an equal ratio (Van Twest and Kropinski, 2009). This was incubated for approximately 24 h at 28\\u00b0C, with shaking at 50 rpm.\\nTABLE 1. Dickeya spp. isolates used in the bacteriophage enrichment.\\nThe enriched suspension (10 \\u03bcL) was spotted on a streak of bacteria in log phase growth used in the enrichment and on a plate with an LB overlay that contained the same bacteria isolates, following Van Twest and Kropinski (2009). After a 24-h incubation at 28\\u00b0C, the resulting lysis cones were scratched into 200 \\u03bcL saline-magnesium with gelatine (SMG; 5.8 g NaCl, 2.0 g MgSO4 \\u00d7 7H2O, 7.88 g Tris HCl, pH 7.5, 0.1 g gelatine, double-distilled water to 1 L), and double agar overlay plaque assays were performed (Kropinski et al., 2009). The host bacterium for the plaque assays was prepared as follows: 1 \\u03bcL inoculation loop of overnight bacteria grown on solid LB was transferred to 5 mL liquid LB with 50 \\u03bcL 1 M MgSO4 and 10 \\u03bcL 1 M CaCl2 and incubated for 12\\u201314 h at 28\\u00b0C, with shaking at 100 rpm. The overnight culture was then diluted 10-fold to obtain the optical density at 600 nm (OD600) of 0.5, which corresponded to approximately 108 cells per mL. Single plaques were selected and transferred to SMG for further purification using disposable pipette tips. The bacteriophages were purified by streaking 1 \\u03bcL bacteriophage suspension on solid LB and pouring over the indicator bacterial lawn. Three successive single plaque isolations were performed as described above. For bacteriophage amplification, double agar overlay plaque assays on LB were carried out (enumeration of bacteriophages by double agar overlay plaque assay), following Kropinski et al. (2009) but with several modifications: (i) 2 mL of the top agar was dispensed into sterile tubes; (ii) tubes containing top agar were incubated at 52\\u00b0C and (iii) 63 \\u03bcL bacteria suspension and bacteriophage dilution were mixed and incubated for 20 min at room temperature prior to the transfer to the warm overlay medium, with agitation at 150 rpm. Confluent plates (plaques evenly distributed over the plates as single plaques and not touching others) were then poured with 3 mL SMG. The plates were sealed with Parafilm and incubated for approximately 20 h at +4\\u00b0C, with gentle shaking at 100 rpm. The bacteriophages were sterile filtered over 0.2-\\u03bcm membranes and stored at +4\\u00b0C or at -80\\u00b0C for longer periods.\\nIsolation of Bacteriophages from Wastewater\\nBacteriophages against the Dickeya UDL-3 and UDL-4 strains were isolated from wastewater concentrates. Effluent water was collected from a wastewater treatment plant and concentrated using a CIM quaternary amine monolithic column. The preparation and characterization of the water samples were as described in Steyer et al. (2015). The wastewater bacteriophage concentrates were diluted in double-distilled water (1:2; v/v) to a final 1% NaCl.\\nFor the enrichment, 3 mL double-strength LB with 120 \\u03bcL 1 M MgSO4 and 24 \\u03bcL 1 M CaCl2 was mixed with 3 mL diluted bacteriophage concentrate. To that, 30 \\u03bcL of bacteria in log phase (12\\u201314 h incubation at 28\\u00b0C, with shaking at 200 rpm) was added at an equal ratio (Van Twest and Kropinski, 2009). The enrichment of UDL-3 and UDL-4 bacteria (Table 1) was preformed separately. The enrichments were incubated for approximately 24 h at 28\\u00b0C, with shaking at 50 rpm and sterile filtering through 0.2-\\u03bcm membranes.\\nThe bacteriophage suspension (7 \\u03bcL) was used in the enrichment on plates with an LB overlay, with plaques isolated as described for the plant material isolation. The bacteriophages were purified by repeating the plaque assays three times. For bacteriophage amplification, double agar overlay plaque assays were carried out as described above. The bacteriophages were eluted from confluent plates with the change of the incubation temperature to room temperature (25\\u00b0C) and were stored as described above.\\nHost Range and Specificity of Bacteriophages\\nFor host-range determination, 56 bacterial strains and isolates (Table 4) were tested. This included 14 Dickeya isolates from diseased Phalaenopsis, eight Dickeya spp. reference strains from a culture collection (D. solani, D. dieffenbachiae, D. dianthicola, D. zeae, D. dadanti, D. aquatica, two Dickeya spp.), seven Pectobacterium cypripedii from a culture collection, four non-Dickeya spp. isolates from diseased Phalaenopsis, 10 isolates from healthy Phalaenopsis leaves and 14 different bacterial strains from the Enterobacteriaceae family. The host range was tested by spotting 10 \\u03bcL of a bacteriophage suspension onto an LB bacterial lawn, which was performed as three biological repeats. All positive results were confirmed by double agar overlay plaque assays (described above) to eliminate false positives.\\nTransmission Electron Microscopy\\nThe presence and morphology of the isolated bacteriophages were determined using transmission electron microscopy with a negative staining method. Here, 20-\\u03bcL bacteriophage suspensions were deposited on Formvar-coated and carbon-stabilized copper grids and stained with a 1% (w/v) aqueous solution of uranyl acetate.\\nTo accompany the bacteria\\u2013bacteriophage interactions, bacterial suspensions (i.e., UDL-3 and UDL-4 separately) were mixed with the purified selected bacteriophage BF25/12, which was shown to be active against bacteria and was kept for 15 min at room temperature. The suspensions were centrifuged at 3,000 \\u00d7 g for 10 min. The supernatants were carefully removed, and the pellets were resuspended in 3% glutaraldehyde and 1% paraformaldehyde in 0.1 M S\\u00f6rensen phosphate buffer (Merck), pH 7.2, for 20 h at 4\\u00b0C. For UDL-3, pieces of soft agar that contained borders of plaques from the plaque assay and soft agar with bacteria from the control plate were fixed in the same fixative. All the samples were post-fixed in 2% OsO4 in 0.1 M S\\u00f6rensen phosphate buffer (Merck) pH 7.2 and embedded in Agar 100 resin (Agar Scientific). Ultrathin sections were stained with 2% (w/v) aqueous uranyl acetate and Reynolds lead citrate (Agar Scientific).\\nThe grids were observed using transmission electron microscopy (CM 100; Philips, Netherlands) operated at 80 kV and equipped with a CCD camera (Orius SC 200) and Digital Micrograph software (Gatan Inc., United States).\\nBacteriophage Genome Characterization\\nBacteriophage DNA was isolated according to Pickard (2009) using the Phase Lock Gel system (5 PRIME, Germany). To determine the diversity of the isolated bacteriophages, RFLP analysis was performed according to Adriaenssens et al. (2012) using the Hind II (Thermo Scientific) restriction enzyme. Here, 0.5 \\u03bcg bacteriophage DNA was used per reaction mixture. The restriction was performed at 37\\u00b0C for 16 h. The fragments were analyzed on 0.8% agarose gels for 100\\u2013120 min at 90 V. Bacteriophage BF25/12 was selected from among Podoviridae isolates for further characterization.\\nWhole Genome Sequencing\\nThe whole genome of bacteriophage BF25/12 was sequenced using 454 sequencing technology. The paired-end library was sequenced at Microsynth (Balgach, Switzerland) following the manufacturer\\u2019s instructions to 36-fold coverage. Whole genome assembly was performed with the GS De Novo Assembler using default parameters (Microsynth). The fully assembled genome was annotated using Rapid Annotations using Subsystems Technology (RAST) (Aziz et al., 2008; Overbeek et al., 2014; Brettin et al., 2015) and the default setting options. Additionally, genome annotation was verified and curated by BLAST analysis (Altschul et al., 1990). The assembled genome sequence was compared to other characterized bacteriophages using the PASC web tool (Bao et al., 2014) and BLAST analysis (Altschul et al., 1990).\\nUV Radiation Stability\\nBacteriophage stability was tested for bacteriophage BF25/12. A bacteriophage suspension was prepared in SMG buffer, and the bacteriophage concentration was adjusted to the concentration of 108 pfu/mL. Testing was performed on 5 mL of prepared bacteriophage suspension in open Petri dishes (\\u03a6 = 55 mm, Golas) illuminated using double UV-C light (Philips TUV G30T8, UV dose approximately 100 mJ/cm2 30 cm from the light source) for 1 and 2 min. The Petri dishes were kept on ice covered with cloth to minimize the influence of heat produced by the UV light. The bacteriophage stability was tested by double agar overlay plaque assay (described above) in two biological repeats as two technical replicates.\\npH Stability\\nThe pH stability of the bacteriophage BF25/12 was tested for a selected range of pH values from 3 to 11. Testing was performed in SMG buffer and sterile demineralised water, adjusted with HCl or NaOH to the pH values of 3, 5, 7, 9, and 11. Phage suspensions were mixed with buffer or water (1 ml total volume) for final concentrations of 105 and 103 pfu/mL and were incubated at room temperature with shaking (100 rpm) for 24 h. The concentration of active bacteriophages was determined with a double agar overlay plaque assay (described above) in two biological repeats and two technical replicates.\\nTemperature Stability\\nBacteriophage stability was analyzed for the bacteriophage BF25/12 by incubating a bacteriophage suspension (100 \\u03bcL) in SMG at different temperatures, +4, +28, -20, and -80\\u00b0C. The stability was tested once per month for 1 year using double agar overlay plaque assays (described above) as three technical replicates.\\nDevelopment of Bacterial Resistance\\nThe emergence of bacterial resistance was measured for the combination of the selected bacteriophage BF25/12 and the bacteria Dickeya sp. B16. A double agar overlay assay was used to enumerate the emergence of resistant bacterial colonies at a bacterial concentration of 108 cfu/mL and four different bacteriophage concentrations, 103, 104, 105, and 106 pfu/mL. Resistant bacterial colonies were counted on plates, with double agar overlay plaque assays carried out after 27 h of incubation at 28\\u00b0C. The test was performed in triplicate per bacteriophage concentration, with two technical repeats each.\\nResults\\nSourcing of Bacteriophages and Characterization\\nThe target Dickeya spp. bacteria used in this study were previously isolated from diseased Phalaenopsis orchids and identified as two distinct UDLs, UDL-3 and UDL-4, based on the partial sequencing of the fliC and dnaX genes (Ali\\u010d et al., 2017). Together with a set of reference bacteria (Table 1), these were used as a mixture to enrich bacteriophages from extracts of rotting orchid leaves and wastewater. Altogether, 18 bacteriophages against Dickeya spp. were isolated, five from diseased Phalaenopsis leaves and 13 from wastewater. Bacteriophages active against UDL-3 bacteria were successfully isolated from plant tissue, but bacteriophages with strong activity against UDL-4 were only isolated from municipal wastewater far away from the orchid site of production. Bacteriophages were isolated from wastewater using a concentration of the bacteriophage particles with CIM monolith chromatography. Together with the subsequent plaque assays, this demonstrates the suitability of this approach for the isolation of infective bacteriophages from wastewater samples.\\nThe bacteriophage recovery from municipal wastewater varied during the season of monthly sampling and the isolation attempts. Bacteriophages active against Dickeya spp. were predominantly isolated in the warmer months (Table 2). After overnight incubation, all the bacteriophages active against Dickeya spp. from diseased Phalaenopsis isolates produced large clear plaques of approximately 6 mm in diameter with a halo effect, suggesting the presence of lysins (Figure 1A). Mixed bacteriophage populations were ruled out by purifying the bacteriophages separately from the clear plaque zones and the halo rings, which resulted in identical morphology with a halo effect for all of them. Furthermore, the restriction of bacteriophage DNA with Hind II (Figure 1E) showed no differences in the patterns between the bacteriophages isolated from these diseased Phalaenopsis orchids.\\nTABLE 2. Sampling timeline of the plant material and wastewater.\\nFIGURE 1. Representative bacteriophage plaque morphologies and restriction fragment length polymorphism profiles. Plaque morphologies for bacteriophages isolated from plant material (A) and wastewater (B,C). Bacteriophages against Dickeya spp. isolated from wastewater (D) and from diseased Phalaenopsis (E) digested with Hind II restriction enzyme. M, Gene Rule 1 kb DNA ladder.\\nThe bacteriophages active against Dickeya spp. that were isolated from wastewater showed two different plaque morphologies. The bacteriophages from the June samples produced large clear plaques that were approximately 5 mm in diameter with a halo effect, and those isolated in April, May, August, and October produced medium to small plaques of approximately 3 mm in diameter, with a clear zone encircled by a white ring and a halo effect (Figures 1B,C). Restriction of the bacteriophage DNA derived from the wastewater showed two different RFLP patterns (Figure 1D). The bacteriophages BF-CIM1/14 that were isolated in June had a uniform RFLP pattern that differed from all the other bacteriophages isolated from wastewater and instead showed a common RFLP pattern.\\nBased on the morphologies of the bacteriophage particles examined with transmission electron microscopy (using negative staining), two of five isolated bacteriophages from diseased Phalaenopsis were placed in the Podoviridae family and showed a head size of 55.8 \\u00b1 2.3 nm \\u00d7 52.6 \\u00b1 3.1 nm (n = 20) (Figure 2A). The bacteriophage tail dimensions were difficult to determine accurately because whole tails were rarely seen and the size was highly dependent on the bacteriophage position on the grid. However, the tail size was estimated to be around 12.1 \\u00b1 2.4 nm \\u00d7 16.3 \\u00b1 1.7 nm (n = 13) (Figure 2A). The bacteriophages from wastewater were classified as Siphoviridae (Figure 2B) and Myoviridae (Figure 2C), which are both in the order Caudovirales. Bacteriophages from the Siphoviridae family (Figure 2B) had icosahedral heads (54.6 \\u00b1 2.6 nm \\u00d7 53.9 \\u00b1 2.2 nm, n = 24) and slightly flexible tails (131.7 \\u00b1 8.4 nm \\u00d7 10.6 \\u00b1 0.6 nm, n = 23) that continued into thin terminal fibers. Bacteriophages characterized as members of the Myoviridae family (Figure 2C) had larger icosahedral heads (138.2 \\u00b1 3.7 nm \\u00d7 136.6 \\u00b1 4.1 nm, n = 47) and rigid contractile tails (228.1 \\u00b1 7.0 nm \\u00d7 24.8 \\u00b1 1.4 nm, n = 47). The base plates were rather large (approximate length of 30 nm), and the attached spikes were short and could be observed only criss-crossed in a netlike structure and were therefore impossible to count. In summary, all these bacteriophages active against Dickeya spp. belong to one of three groups, having common RFLP profiles and plaque and bacteriophage morphologies (Table 3).\\nFIGURE 2. Representative transmission electron microscopy micrographs of bacteriophages isolated from diseased Phalaenopsis leaves (A) and sewage water (B,C), all of which were only active against Dickeya spp.\\nTABLE 3. Bacteriophages isolated from infected Phalaenopsis tissue and wastewater concentrates using convective interaction media (CIM) monolith chromatography.\\nSpecificity of Isolated Bacteriophages against Dickeya spp.\\nThe specificities of the interactions between the bacteria and bacteriophages were examined using spot tests (i.e., isolation of bacteriophage via induction of lysogenesis) and were confirmed using plaque assays. The specificities of the bacteriophages included in the testing were limited to Dickeya spp., excluding D. aquatica and D. zeae. All the bacteriophages from orchid tissue showed identical host-range profiles. They were active against Dickeya sp. UDL-3 isolates, Dickeya sp. MK7 and D. dadantii NCPPB 898. However, the bacteriophages isolated from wastewater showed variability in their host specificities. The host ranges of the bacteriophages BF-CIM5/14, BF-CIM6/14, BF-CIM8/14, and BF-CIM10/15 were very similar, and their specificities were close for both Dickeya spp. UDL-3 and UDL-4. However, substantial differences in specificities against Dickeya spp. from the bacteria collection here were observed. The bacteriophages BF-CIM8/14 and BF-CIM10/14 did not only infect D. aquatica and D. zeae. Conversely, infection by the bacteriophages BF-CIM5/14 and BF-CIM6/14 was limited to Dickeya sp. MK7 and weakly to D. solani GBBC 500 and D. solani GBBC 2040. Finally, all three bacteriophage isolates from the BF-CIM1/14 group showed a uniform infection profile; they were active only against Dickeya sp. UDL-4 isolates D. solani GBBC 500 and D. dianthicola LMG 2458 (Table 4).\\nTABLE 4. Host range of bacteriophages isolated against UDL-3 and UDL-4 isolates.\\nBased on the plaque morphology, RFLP profiles, host ranges and transmission electron microscopy analysis, three different bacteriophages were isolated. Two were derived from the wastewater and were members of the Myoviridae and Siphoviridae families. The third bacteriophage belonged to the Podoviridae family and was obtained from the soft rot orchid tissue. Based on the characteristics of these bacteriophages, bacteriophage BF25/12 was chosen for further analysis.\\nTransmission Electron Microscopy of Bacteria\\u2013Bacteriophage Interactions\\nThe interactions of the selected bacteriophage BF25/12 with the host bacteria were examined at the ultrastructure level. The interactions of BF25/12 and the UDL-4 bacteria activated granule degradation in the bacteria cells without bacterial lysis. Most of these UDL-4 bacteria contained electron-lucent intracellular granules (20\\u201370 nm in diameter), which were similar to intracellular inclusion bodies (Figure 3A; Ali\\u010d et al., 2017). When bacteria UDL-4 and bacteriophages BF25/12 were incubated together for 15 min, these granules almost completely disappeared, and there were bacteriophages attached to the bacterial surface (Figure 3B). Mixtures of Dickeya sp. UDL-3 (Figure 3C) and BF25/12 also showed the attachment of the bacteriophages to the bacterial surface (Figure 3D). Based on the plaque assays, the borders of the plaques were analyzed. Most of the UDL-3 cells here were lysed, and the plasmalemmas of the resting cells were detached (Figure 3E). New bacteriophages could also be seen in the cytoplasm of some of the bacteria (Figure 3F).\\nFIGURE 3. Representative ultrastructure analyses of bacteria\\u2013bacteriophage interactions. (A,B) Strain UDL-4 before (A) and after (B) mixing with bacteriophage BF25/12 (arrows). (C,D) Strain UDL-3 before (C) and after (D) mixing with bacteriophage BF25/12 (arrows). (E,F) Lysed UDL-3 bacteria from the border of a plaque (E) and new bacteriophages in the cytoplasm (arrowheads) (F). G, intracellular granules; CW, cell wall; PL, plasmalemma; PS, periplasmic space.\\nGenome of Bacteriophage BF25/12\\nThe genome of bacteriophage BF25/12 was additionally characterized using genome sequencing and automatic annotation. BF25/12 contains linear double-stranded (ds)DNA that is 43,872 bp long with 52.2% GC, which supports its classification into the Podoviridae family. The genome of bacteriophage BF25/12 contains 52 open reading frames that were determined with RAST (Aziz et al., 2008; Overbeek et al., 2014; Brettin et al., 2015). They mainly coded for hypothetical proteins (27 genes), along with structural proteins and proteins involved in bacteriophage replication. Furthermore, a single putative endolysin gene was identified, corresponding to the halo effect observed in plaque morphology. However, no genes that encode antibiotic resistance or toxins were detected (Figure 4). The annotated BF25/12 genome sequence is available in GenBank under Accession No. KT240186.\\nFIGURE 4. Annotated BF25/12 bacteriophage genome. According to the annotation, the genome does not contain any genes encoding antibiotic resistance or toxins. The open reading frames coding for structural proteins, proteins involved in bacteriophage replication, and other conserved proteins are shown in yellow, and those coding for hypothetical proteins are shown in gray. Unit of the presented genome annotation is in bp.\\nThe genomic sequence of bacteriophage BF25/12 was compared to characterized genera from the Podoviridae family using the PASC tool (Bao et al., 2014). BF25/12 showed low identity (<3%) to any member of the Picovirinae or Sepvirinae subfamilies. However, it shared higher identity to the Autographivirinae subfamily, especially Klebsiella virus F19 from the Kp34 virus genus (33% identity). Based on the pairwise identity analysis conducted, the BF25/12 genome was 57\\u201362% identical to three unassigned Pectobacterium bacteriophages, Pectobacterium phages PP16, PP90, and Peat1. The results correspond to the nBLAST analysis. The assembled BF25/12 genome showed substantial similarity (74\\u201379% identity and 21\\u201356% query coverage) to five bacteriophage genomes in the BLAST database. All five genomes belonged to Pectobacterium bacteriophages, four of which were assigned to the Podoviridae family, such as BF25/12, and the last was unclassified. The closest hit represented Pectobacterium bacteriophage PP16 (79% identity over 56% query coverage).\\nUV Radiation Stability\\nBacteriophage BF25/12 was not able to survive a 2-min exposure to UV-C light. The reduction of active bacteriophages was 105, corresponding to almost 100% in the first minute of exposure.\\npH Stability\\nBacteriophage pH stability analysis in SMG buffer and sterile demineralised water at two different bacteriophage concentrations (105 and 103 pfu/mL) showed that bacteriophage BF25/12 pH stability is affected by both its concentration and environment (Figure 6). Bacteriophage BF25/12 lost its infectivity completely at pH 3, regardless of the incubation media (SMG buffer or sterile demineralised water). It showed maximum stability at pH 7 and 9, which indicated the favorable pH for bacteriophage BF25/12 is neutral to mildly alkaline. Furthermore, no significant difference in the decrease of bacteriophage titer could be detected between the tested incubation media or bacteriophage concentrations after 24 h incubation at pH 7 and 9. At higher concentrations in SMG buffer, a mildly acidic pH (pH 5) did not cause significant phage de-activation. Bacteriophage incubated in SMG buffer completely lost its infectivity at pH 11 after 24 h incubation, but its infectivity was not affected in sterile demineralised water in the same conditions. The exposure of highly concentrated bacteriophage lysate (108 pfu/mL) to acidic (pH 3) and alkaline (pH 11) environments for 24 h had almost no detectable effect on bacteriophage survival and infectivity (data not shown).\\nTemperature Stability\\nThe stability of the bacteriophage BF25/12 active against Dickeya spp. was monitored at different temperatures over a year. It proved to be stable at 4\\u00b0C, as the changes in the bacteriophage titers were negligible, which indicated this is a favorable temperature for long-term bacteriophage storage (Figure 5). The bacteriophage stability at -80\\u00b0C appeared similar to that at 4\\u00b0C. However, temperatures of -20\\u00b0C proved to be inappropriate for long-term storage due to the notable decline of the bacteriophage titer. This unusual decrease in the bacteriophage titer at -20\\u00b0C was seen after the first 3 months of storage. In the following months, the bacteriophage concentration returned to the initial bacteriophage titer. The least favorable temperature tested was 28\\u00b0C.\\nFIGURE 5. Temperature stability with time of Dickeya spp. bacteriophage BF25/12 incubated at temperatures of +4, +28, \\u201320 and \\u201380\\u00b0C (as indicated).\\nFIGURE 6. Effect of pH on stability of bacteriophage BF25/12. Stability was tested in SMG buffer and sterile demineralised water at two different bacteriophage concentrations, 105 and 103 pfu/mL. Concentration of bacteriophages presented in the graph was calculated to the level of bacteriophage stock, used for preparation of testing dilutions.\\nBacterial Resistance\\nThe emergence of resistance against bacteriophage BF25/12 was determined for host bacteria Dickeya sp. B16. The number of growth-resistant bacterial colonies depended greatly on the bacteriophage concentration in the medium (Figure 7). The highest rate of bacterial resistance development against bacteriophage BF25/12 was at the final bacteriophage concentration of between log 4 and log 5 and the initial bacteria concentration of log 8. Bacteriophage concentrations <log 4 and >log 5 provided lower selection pressure, which resulted in the decreased resistance of the bacterial colonies.\\nFIGURE 7. Emergence of Dickeya spp. B16 resistant to bacteriophage BF 25/12. Experiment was performed as two technical repeats, shown on graph as squares and diamonds.\\nDiscussion\\nIn the present study, we isolated bacteriophages active against the plant pathogens of Dickeya spp. that cause the soft rot of orchids, which is a destructive disease for Phalaenopsis production sites. Bacteriophages were isolated from two different sources, diseased Phalaenopsis tissue and wastewater. For the latter, CIM monolith chromatography was used for bacteriophage particle concentration. CIM monolith chromatography has been reported as an effective technique for virus concentration and for purification of virus particles (Steyer et al., 2015). However, we believe this is the first report that describes the use of CIM monolith chromatography as a technique for the isolation of infectious bacteriophages from environmental samples. It proved to be very efficient for the concentration of bacteriophages from environmental samples, which allowed more efficient bacteriophage enrichment on the host bacterium and therefore better yields in the bacteriophage isolation. Seventy-two percent of bacteriophages active against Dickeya spp. obtained in this study were isolated from wastewater using CIM monolith chromatography, indicating the presence of viable Dickeya spp. bacteria in wastewater or in plants that are in direct contact with wastewater. Furthermore, different sources of bacteriophage isolation resulted in different bacteriophages from different families and with different RFLP patterns, although they showed similar host-specificity profiles. The isolation of the bacteriophages from orchid samples was carried out in summer (i.e., July and September) when the highest concentrations of the bacteria are expected. The isolates from wastewater were obtained from late spring to early autumn, excluding July and September. The attempts to isolate bacteriophages from soil were not successful, regardless of the month when the sampling was carried out (data not shown). Therefore, these differences cannot be explained in terms of seasonal changes, which indicates the specificity of each of these bacteriophage sources.\\nHowever, the bacteriophages active against Dickeya spp. that showed high specificities can be used as biocontrol agents, as the treatment of a diseased orchid with these bacteriophages can specifically eliminate the pathogenic bacteria while leaving the normal microbiota unharmed. Adriaenssens et al. (2012) and Czajkowski et al. (2014, 2015) reported the isolation of bacteriophages specifically against the aggressive potato pathogen D. solani, bacteriophages with multiple hosts across Dickeya spp. and broad host-range bacteriophages against Pectobacterium spp. and Dickeya spp. All the bacteriophages active against Dickeya spp. described in the literature belong to the order Caudovirales as members of the Myoviridae family (Adriaenssens et al., 2012; Czajkowski et al., 2014, 2015; Soleimani-Delfan et al., 2015), and the first report about a bacteriophage from the Siphoviridae family was made recently (Soleimani-Delfan et al., 2015). However, the present study represents the first report of bacteriophages active against Dickeya spp. from the Podoviridae family. The bacteriophages from different families show some small differences in their specificities. All the bacteriophages from the Podoviridae (BF21/12 \\u2013 BF25/12) and Myoviridae (BF-CIM5/14 \\u2013 BF-CIM10/14) families infected all Dickeya isolates in the present study, as did the aggressive potato pathogen D. solani and other reference Dickeya strains, excluding D. paradisiaca and D. aquatica. However, as members of the Siphoviridae family, bacteriophages BF-CIM1/14 showed different host ranges. The activity against the Dickeya sp. UDL-3 isolates varied and was relatively low, although all the Dickeya sp. UDL-4 and D. solani strains were highly susceptible, which supports a relationship between those two Dickeya spp. (Ali\\u010d et al., 2017).\\nThe selected bacteriophage here, BF25/12, was characterized at the genome level. It showed no similarities to any previously described bacteriophages that are active against Dickeya spp. However, homologous bacteriophages deposited in GenBank were identified using BLAST and PASC analysis. The BF25/12 genome showed the highest similarity to bacteriophages active against Pectobacterium species characterized as members of the Podoviridae family, such as BF25/12. Pectobacterium bacteriophages did not share similarity to BF25/12 across the genome; rather, the homology was limited to sections of the genome. This lack of genetically similar bacteriophages can be explained on the basis that this is the first member of the Podoviridae family among bacteriophages active against Dickeya spp. Bacteriophage BF25/12 did not show significant similarities to any currently described and characterized Podoviridae genus at the genome level. Based on the genomic and morphologic characteristics, it is closest to genera in the Autographivirinae subfamily, which includes the T7 virus genus and the Escherichia coli model bacteriophage T7. The bacteriophage T7 virion consists of a head with a diameter of 55 nm and a 23-nm long tail (Steven and Trus, 1986). The size of the T7 virus particle corresponds to the measurements of the BF25/12 virion (head approximately 56 nm \\u00d7 52 nm and tail 16 nm \\u00d7 12 nm). The BF25/12 genome size of 43.8 kbp and the 52.2% GC content matches the small T7 bacteriophage genome (around 40 kbp and 48.4% GC content; Dunn et al., 1983). Furthermore, T7 and BF25/12 have an almost identical number of coding sequences in the genome\\u201450 and 52 genes, respectively (Dunn et al., 1983). A genome analysis showed the suitability of BF25/12 for field-application studies due to its lack of genes coding for the bacteriophage lysogenic cycle and of genes encoding toxins or antibiotic resistance.\\nBacteriophage BF25/12 showed differences in activity against the UDL-3 and UDL-4 bacteria, which are genetically very similar bacterial isolates. Differences in the infection of these two isolates were observed under transmission electron microscopy, which resulted in some phenomena that have not been reported previously. The interactions of the bacteriophage with the UDL-4 bacteria activated granule degradation in the bacterial cells, which are similar to intracellular storage granules, with no bacterial lysis. The structure and function of these observed inclusions remain unknown, although a type of intracellular metabolic reserve is most likely. Some metabolic inclusions that function as a metabolic reserve are directly connected to the persistence of bacteria. Glycogen metabolism has been linked to bacterial survival in the environment, as well as to colonization and virulence (reviewed in Wilson et al., 2010). Moreover, in Pseudomonas oleovorans, it was shown that the metabolism of intracellular polyhydroxyalkanoates granules contributes to bacterial cell survival. The wild-type strain P. oleovorans, which is able to degrade intracellular polyhydroxyalkanoates, showed enhanced resistance to stress agents compared to the corresponding mutant strain incapable of the depolymerisation of polyhydroxyalkanoates (Ruiz et al., 2001). This observed granule degradation in UDL-4 cells might represent a bacteria stress response to the bacteriophage infection process, whereby the metabolic inclusions might enhance the environmental survival of the bacteria and therefore limit the development of the bacteriophage infection. However, a correlation between bacterial resistance to bacteriophage lysis and intercellular storage granule degradation has never been described.\\nSome of the main challenges in bacteriophage applications are environmental factors, including temperature, pH and UV radiation. Temperature can affect the ability of bacteriophages to lyse bacteria and their survival on the plant surface (Jones et al., 2007). Monitoring bacteriophage stability here revealed decreased bacteriophage titers at 28\\u00b0C. The observed outcome corresponds to a study by Jepson and March (2004), who reported a decrease in liquid bacteriophage \\u03bb stock at 37\\u00b0C over 120 days to under the limit of detection. However, the decline in the viability of bacteriophage BF25/12 was only 27.34% in 1 year, which indicates its applicability due to this appreciable stability at the implementation temperature. As expected, the bacteriophage titer at 4\\u00b0C remained stable over the observed period. Bacteriophage BF25/12 belongs to the tailed bacteriophages that are known to be extremely viable at 4\\u00b0C, and some can retain viability for up to 10\\u201312 years (Jepson and March, 2004; Jo\\u0144czyk et al., 2011). Storing bacteriophages at -20\\u00b0C is not recommended, as the formation of ice crystals can damage or destroy bacteriophage particles (Jo\\u0144czyk et al., 2011). Furthermore, suboptimal pH can inactivate bacteriophages; therefore, the stability and pH optimum of bacteriophages is very important in plant disease control applications (Jones et al., 2007). Bacteriophage BF25/12 was sensitive to acid pH and showed maximum stability in neutral and alkaline conditions. Only bacteriophages incubated in sterile demineralised water were stable at pH 11, indicating the influence of the incubation medium to bacteriophage stability at certain pH. It has been reported that buffer and salt concentration can affect bacteriophage activity (Chow et al., 1971; Burnet and Stanley, 2013). Chow et al. (1971) reported higher stability of bacteriophage Xp12 in Tris buffer at a higher pH; the effect is more prominent at higher ionic strengths. However, the effect was studied only for the pH range from 7 to 9. The most damaging environmental factor for bacteriophage stability is UV radiation (Jones et al., 2007), and, as reported for other phages, BF25/12 is highly susceptible to UV radiation (Jones et al., 2007).\\nThe occurrence of resistant bacteria mutants represents a major concern when using bacteriophages as biocontrol agents (reviewed in Jones et al., 2007). Here, the dependence between bacteriophage concentration and the development of bacterial resistance was optimum from log 4 to log 5 pfu/mL. These data indicate the importance of using suitable bacteriophage concentrations for bacteriophage treatments and the danger of using a single bacteriophage in any active treatment approach. The active bacteriophage used in this treatment undergoes extensive reproduction above the threshold needed to kill the bacteria through secondary infection (Payne et al., 2000). Extensive enrichment of bacteriophage-resistant bacteria might occur if the initial bacteriophage concentration used in any active approach is below the observed optimum for resistance development; for Dickeya spp. B16 and BF25/12, this was below log 4. However, the emergence of resistant bacteria during bacteriophage biocontrol can be reduced by the use of bacteriophage cocktails instead of individual bacteriophages (reviewed in Jones et al., 2007).\\nIn terms of the applicability of the bacteriophages defined in the present study, it will be necessary to determine the infection parameters (e.g., adsorption rate, one-step growth assay, burst size) to determine the potential effectiveness of this bacteriophage as a biocontrol. The combination of these isolated bacteriophages could in future form an intriguing bacteriophage cocktail due to the combination of the three different bacteriophage families with similar specificities. However, the biocontrol properties of the bacteriophages should be evaluated by pot or field experiment. The dynamic approach of the bacteriophage biocontrol agents requires extensive knowledge of a specific bacteria\\u2013bacteriophage system. However, bacteriophages can also be used in combined biocontrol management strategies, as they enable a temporary decrease in bacterial inoculum and therefore the greater efficiency of other substances used in bacterial disease control and treatment. Presently, isolated phages represent an important research tool that can provide adequate knowledge of bacteria\\u2013bacteriophage systems and dynamics, which is essential for the applicability of bacteriophages.\\nAuthor Contributions\\nConception and design of the experiments: \\u0160A, TN, TD, and MR. Performing experiments, analysis and interpretation of the data: \\u0160A, TN, MT-\\u017d, and TD. Provided CIM virus concentrates: NR. Drafting the work and writing the paper: \\u0160A, TN, MT-\\u017d, and TD. Revising the work critically for important intellectual content: \\u0160A, TN, MT-\\u017d, MP, MR, and TD. All authors read and approved the final manuscript.\\nFunding\\nThis study was supported financially by the Slovenian Research Agency (contracts P4-0165, L7-5534, 1000-15-0105) and the Centre of Excellence for Biosensors, Instrumentation and Process Control, an operation financed by the European Union, European Regional Development Fund, and Republic of Slovenia, Ministry of Higher Education, Science and Technology.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nThe authors would like to acknowledge Nika Jane\\u017e, Andreja Petan (COBIK) and \\u0160pela Prijatelj Novak, Lidija Mati\\u00e8i\\u00e8 (NIB) for technical assistance, Denis Kutnjak (NIB) for his support in sequencing data analysis and Dr. Christopher Berrie for critical reading and English language editing. We thank Johan Van Vaerenbergh of the Institute for Agricultural and Fisheries Research for reference bacterial strains, Ian Toth and Sonia Humphries of the James Hutton Institute for the Dickeya sp. MK7 strain and Toma\\u017e Jev\\u0161nik of Ocean Orchids for the orchid plant material.\\nReferences\\nAdriaenssens, E. M., Van Vaerenbergh, J., Vandenheuvel, D., Dunon, V., Ceyssens, P.-J., De Proft, M., et al. (2012). T4-related bacteriophage LIMEstone isolates for the control of soft rot on potato caused by \\u201cDickeya solani.\\u201d. PLOS ONE 7:e33227. doi: 10.1371/journal.pone.0033227\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAli\\u010d, \\u0160., Nagli\\u010d, T., Tu\\u0161ek-\\u017dnidari\\u010d, M., Peterka, M., Ravnikar, M., and Dreo, T. (2017). Putative new species of the genus Dickeya as major soft rot pathogens in Phalaenopsis orchid production. Plant Pathol. 66, 1357\\u20131368. doi: 10.1111/ppa.12677\\nCrossRef Full Text | Google Scholar\\nAltschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990). Basic local alignment search tool. J. Mol. Biol. 215, 403\\u2013410. doi: 10.1016/S0022-2836(05)80360-2\\nCrossRef Full Text | Google Scholar\\nAziz, R. K., Bartels, D., Best, A. A., DeJongh, M., Disz, T., Edwards, R. A., et al. (2008). The RAST Server: rapid annotations using subsystems technology. BMC Genomics 9:75. doi: 10.1186/1471-2164-9-75\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBao, Y., Chetvernin, V., and Tatusova, T. (2014). Improvements to pairwise sequence comparison (PASC): a genome-based web tool for virus classification. Arch. Virol. 159, 3293\\u20133304. doi: 10.1007/s00705-014-2197-x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBrettin, T., Davis, J. J., Disz, T., Edwards, R. A., Gerdes, S., Olsen, G. J., et al. (2015). RASTtk: a modular and extensible implementation of the RAST algorithm for building custom annotation pipelines and annotating batches of genomes. Sci. Rep. 5:8365. doi: 10.1038/srep08365\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBurnet, F. M., and Stanley, W. M. (2013). General Virology: Biochemical, Biological, and Biophysical Properties. Cambridge, MA: Academic Press.\\nGoogle Scholar\\nButtimer, C., McAuliffe, O., Ross, R. P., Hill, C., O\\u2019Mahony, J., and Coffey, A. (2017). Bacteriophages and bacterial plant diseases. Front. Microbiol. 8:34. doi: 10.3389/fmicb.2017.00034\\nCrossRef Full Text | Google Scholar\\nChow, T. Y., Lin, Y. T., and Kuo, T. T. (1971). Stability of phage Xp12. Bot. Bull. Acad. Sin. 12, 57\\u201365.\\nGoogle Scholar\\nCzajkowski, R., Ozymko, Z., de Jager, V., Siwinska, J., Smolarska, A., Ossowicki, A., et al. (2015). Genomic, proteomic and morphological characterization of two novel broad host lytic bacteriophages \\u03a6PD10.3 and \\u03a6PD23.1 infecting pectinolytic Pectobacterium spp. and Dickeya spp. PLOS ONE 10:e0119812. doi: 10.1371/journal.pone.0119812\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCzajkowski, R., Ozymko, Z., and Lojkowska, E. (2014). Isolation and characterization of novel soilborne lytic bacteriophages infecting Dickeya spp. biovar 3 (\\u201cD. solani\\u201d). Plant Pathol. 63, 758\\u2013772. doi: 10.1111/ppa.12157\\nCrossRef Full Text | Google Scholar\\nCzajkowski, R., P\\u00e9rombelon, M. C. M., van Veen, J. A., and van der Wolf, J. M. (2011). Control of blackleg and tuber soft rot of potato caused by Pectobacterium and Dickeya species: a review. Plant Pathol. 60, 999\\u20131013. doi: 10.1111/j.1365-3059.2011.02470.x\\nCrossRef Full Text | Google Scholar\\nDunn, J. J., Studier, F. W., and Gottesman, M. (1983). Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements. J. Mol. Biol. 166, 477\\u2013535.\\nPubMed Abstract | Google Scholar\\nJo\\u0144czyk, E., K\\u0142ak, M., Mibbbedzybrodzki, R., and G\\u00f3rski, A. (2011). The influence of external factors on bacteriophages\\u2013review. Folia Microbiol. 56, 191\\u2013200. doi: 10.1007/s12223-011-0039-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJones, J. B., Jackson, L. E., Balogh, B., Obradovic, A., Iriarte, F. B., and Momol, M. T. (2007). Bacteriophages for plant disease control. Annu. Rev. Phytopathol. 45, 245\\u2013262. doi: 10.1146/annurev.phyto.45.062806.094411\\nCrossRef Full Text | Google Scholar\\nJones, J. B., Vallad, G. E., Iriarte, F. B., Obradovi\\u0107, A., Wernsing, M. H., Jackson, L. E., et al. (2012). Considerations for using bacteriophages for plant disease control. Bacteriophage 2, 208\\u2013214. doi: 10.4161/bact.23857\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKropinski, A. M., Mazzocco, A., Waddell, T. E., Lingohr, E., and Johnson, R. P. (2009). Enumeration of bacteriophages by double agar overlay plaque assay. Methods Mol. Biol. 501, 69\\u201376. doi: 10.1007/978-1-60327-164-6_7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nOverbeek, R., Olson, R., Pusch, G. D., Olsen, G. J., Davis, J. J., Disz, T., et al. (2014). The SEED and the rapid annotation of microbial genomes using subsystems technology (RAST). Nucleic Acids Res. 42, D206\\u2013D214. doi: 10.1093/nar/gkt1226\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nParkinson, N., Stead, D., Bew, J., Heeney, J., Tsror Lahkim, L., and Elphinstone, J. (2009). Dickeya species relatedness and clade structure determined by comparison of recA sequences. Int. J. Syst. Evol. Microbiol. 59, 2388\\u20132393. doi: 10.1099/ijs.0.009258-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPayne, R. J., Phil, D., and Jansen, V. A. (2000). Phage therapy: the peculiar kinetics of self-replicating pharmaceuticals. Clin. Pharmacol. Ther. 68, 225\\u2013230. doi: 10.1067/mcp.2000.109520\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPickard, D. (2009). \\u201cPreparation of bacteriophage lysates and pure DNA,\\u201d in Bacteriophages Methods in Molecular BiologyTM, eds M. J. Clokie and A. Kropinski (Berlin: Humana Press), 3\\u20139.\\nGoogle Scholar\\nRuiz, J. A., L\\u00f3pez, N. I., Fern\\u00e1ndez, R. O., and M\\u00e9ndez, B. S. (2001). Polyhydroxyalkanoate degradation is associated with nucleotide accumulation and enhances stress resistance and survival of Pseudomonas oleovorans in natural water microcosms. Appl. Environ. Microbiol. 67, 225\\u2013230. doi: 10.1128/AEM.67.1.225-230.2001\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSamson, R., Legendre, J. B., Christen, R., Saux, M. F.-L., Achouak, W., and Gardan, L. (2005). Transfer of Pectobacterium chrysanthemi (Burkholder et al., 1953) Brenner et al., 1973 and Brenneria paradisiaca to the genus Dickeya gen. nov. as Dickeya chrysanthemi comb. nov. and Dickeya paradisiaca comb. nov. and delineation of four novel species, Dickeya dadantii sp. nov., Dickeya dianthicola sp. nov., Dickeya dieffenbachiae sp. nov. and Dickeya zeae sp. nov. Int. J. Syst. Evol. Microbiol. 55, 1415\\u20131427. doi: 10.1099/ijs.0.02791-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSoleimani-Delfan, A., Etemadifar, Z., Emtiazi, G., and Bouzari, M. (2015). Isolation of Dickeya dadantii strains from potato disease and biocontrol by their bacteriophages. Braz. J. Microbiol. 46, 791\\u2013797. doi: 10.1590/S1517-838246320140498\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSteven, A. C., and Trus, B. L. (1986). \\u201cThe structure of bacteriophage T7,\\u201d in Electron Microscopy of Proteins, Viral Structure, eds J. R. Harris and R. W. Horne (London: Academic Press), 1\\u201333.\\nGoogle Scholar\\nSteyer, A., Guti\\u00e9rrez-Aguirre, I., Ra\\u010dki, N., Beigot Glaser, S., Brajer Humar, B., Stra\\u017ear, M., et al. (2015). The detection rate of enteric viruses and Clostridium difficile in a waste water treatment plant effluent. Food Environ. Virol. 7, 164\\u2013172. doi: 10.1007/s12560-015-9183-7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTian, Y., Zhao, Y., Yuan, X., Yi, J., Fan, J., Xu, Z., et al. (2016). Dickeya fangzhongdai sp. nov., a plant-pathogenic bacterium isolated from pear trees (Pyrus pyrifolia). Int. J. Syst. Evol. Microbiol. 66, 2831\\u20132835. doi: 10.1099/ijsem.0.001060\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nToth, I. K., van der Wolf, J. M., Saddler, G., Lojkowska, E., H\\u00e9lias, V., Pirhonen, M., et al. (2011). Dickeya species: an emerging problem for potato production in Europe. Plant Pathol. 60, 385\\u2013399. doi: 10.1111/j.1365-3059.2011.02427.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nVan Vaerenbergh, J., Baeyen, S., De Vos, P., and Maes, M. (2012). Sequence diversity in the Dickeya fliC Gene: phylogeny of the Dickeya genus and TaqMan\\u00ae PCR for \\u201cD. solani\\u201d, new biovar 3 variant on potato in Europe. PLOS ONE 7:e35738. doi: 10.1371/journal.pone.0035738\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWilson, W. A., Roach, P. J., Montero, M., Baroja-Fern\\u00e1ndez, E., Mu\\u00f1oz, F. J., Eydallin, G., et al. (2010). Regulation of glycogen metabolism in yeast and bacteria. FEMS Microbiol. Rev. 34, 952\\u2013985. doi: 10.1111/j.1574-6976.2010.00220.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00627\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00627/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:03.307Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00627\",\n",
      "            \"title\": \"Frontiers | Discovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization\",\n",
      "            \"description\": \"Composting is a complex biodegradable process that converts organic materials into nutrients to facilitate crop yields, and, if well managed, can render bactericidal effects. Majority of research focused on detection of enteric pathogens, such as Shiga toxin-producing E. coli (STEC) in fecal composts. Recently, attention has been emphasized on bacteriophages, such as STEC-specific bacteriophages, associated with STEC from the fecal-contaminated environment because they are able to sustain adverse environment during composting process. However, little is known regarding the isolation of STEC-specific bacteriophages in non-fecal composts. Thus, the objectives were to isolate and genomically characterize STEC-specific bacteriophages, and to evaluate its association with STEC in non-fecal composts. For bacteriophage isolation, the samples were enriched with non-pathogenic E. coli (3 strains) and STEC (14 strains), respectively. After purification, host range, plaque size, and phage morphology were examined. Furthermore, bacteriophage genomes were subjected to whole-genome sequencing using Illumina MiSeq and genomic analyses. Isolation of top six non-O157 and O157 STEC utilizing culture methods combined with PCR-based confirmation was also conducted. The results showed that various STEC-specific bacteriophages, including vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw and vB_EcoM-Ro157lw, with different but complementary host ranges were isolated. Genomic analysis showed the genome sizes varied from 42kb to 149kb, and most bacteriophages were unclassified at the genus level, except vB_EcoM-Ro111lw as FelixO1-like viruses. Prokka predicted less than 25% of the ORFs coded for known functions, including those essential for DNA replication, bacteriophage structure, and host cell lysis. Moreover, none of the bacteriophages harbored lysogenic genes or virulence genes, such as stx or eae. Additionally, the presence of these lytic bacteriophages was likely attributed to zero isolation of STEC and could also contribute to additional antimicrobial effects in composts, if the composting process was insufficient. Current findings indicate that various STEC-specific bacteriophages were found in the non-fecal composts. In addition, the genomic characterization provides in-depth information to complement the deficiency of biological features regarding lytic cycle of the new bacteriophages. Most importantly, these bacteriophages have great potential to control various serogroups of STEC.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nIn recent years, produce outbreaks associated with contamination of enteric pathogens, such as Shiga toxin-producing Escherichia coli (STEC), have drawn significant attention of health administrations and the public. During 2011, a severe outbreak in Europe was associated with contamination of fenugreek sprouts with E. coli O104:H4. There were approximately 3800 cases reported in Germany alone, including 845 cases of hemolytic uremic syndrome and 54 deaths (European Food Safety Authority, 2011; Frank et al., 2011; Beutin and Martin, 2012). Recently, a massive recall occurred in the United States regarding contamination of romaine lettuce with E. coli O157:H7 in 22 states, causing illness in 98 people, 46 hospitalizations, and 10 cases of hemolytic uremic syndrome (CDC, 2018). Produce is often served raw and, thus, is susceptible to any contamination from farm to fork. Contamination of pathogenic STEC strains on produce firstly occurs on the farm mostly derived from the contact with fecal-contaminated sources, such as wild animal, irrigation water and composts (Walters et al., 2011; Cooley et al., 2013).\\nComposting is a biodegradable process by microorganisms to turn organic materials to humus-like components that are commonly used as fertilizers or soil amendment to improve crop yields (Ryckeboer et al., 2003). An efficiently managed composting process would attribute to the reduced levels of pathogens, such as Salmonella and pathogenic E. coli, first present in the raw organic materials after the process due to the change of temperature and pH (Ryckeboer et al., 2003; Heringa et al., 2010). Inadequate composting processes have resulted in the spread of foodborne pathogens in pre-harvest environment (Buck et al., 2003), which are commonly associated with the composting of organic material with higher levels of pathogens, particularly of fecal origin (Handley et al., 2012). A previous study demonstrated that Salmonella and E. coli O157:H7 were able to survive through the niche of composting process and sustain at low levels in dairy-based compost (Kim et al., 2009). Another study further indicated that the use of E. coli O157:H7-polluted compost for soil amendment was directly associated with contamination of outer lettuce leaves during irrigation (Oliveira et al., 2012). Due to the natural reservoir of STEC, additional interventions have been used to minimize the pathogens in feces-based composts prior to use to prevent the spread of the pathogens (Lemunier et al., 2005; Heringa et al., 2010).\\nBacteriophages (or phages) are key elements of intestinal microbiota of animals (Handley et al., 2012). Previous study indicated that various STEC-specific phages that were lytic against different serogroups of STEC strains, such as O26, O111 and O157, were isolated from cattle feces (Viazis et al., 2011). The authors found that these phages were free of virulence genes, such as stx, hlyA, and eaeA genes, and were potential biocontrol candidates to prevent STEC contamination. Another study, however, found that Stx phages \\u2013 the phage containing stx genes \\u2013 with lysogenic factors were present in the environment contaminated with animal feces and might be associated with emergence of new pathogens through the transfer of the virulence gene by the phages (Martinez-Castillo and Muniesa, 2014). Additionally, phages have been demonstrated to be able to be present at different stages of composting process (Johannessen et al., 2005; Partanen et al., 2010). The phages specific to Pseudomonas were isolated and characterized in composts (Amgarten et al., 2017); however, the information regarding the presence of STEC-specific phage or Stx phages in the composts, particularly non-fecal bases, is scarce. It is believed that the presence of STEC-specific phages or Stx phages (temperate phages) might have different impact on the biosafety level of compost upon application. Therefore, the objectives of this study were to isolate and genomically characterize free STEC-specific bacteriophages, and to evaluate its association with STEC strains in the non-fecal composts. To the best of the authors\\u2019 knowledge, this is the first study to isolate and characterize STEC-associated phages in non-fecal composts.\\nMaterials and Methods\\nSample Collection\\nTwo non-fecal compost samples (C1 and C2) were randomly collected from a composting operation in California. The raw ingredients used for the composting were consisted of food scraps, from restaurants and markets, and yard trimmings.\\nIsolation of STEC Strains\\nUsing an abridged version of the protocol outlined in Cooley et al. (2013), samples were enriched and streaked on MacConkey Sorbitol Agar supplemented with cefixime and potassium tellurite (CT-SMAC), Rainbow Agar O157 supplemented with novobiocin and potassium tellurite (NT-RA), BBL Blood Agar supplemented with calcium chloride, mitomycin and X-gal (mSBA), and Chromagar O157. Typical colonies of STEC found on each type of media were picked and grown overnight in Luria Broth (LB) at 37\\u00b0C and plated on Luria Agar. DNA from each isolated colony streaked from the two enriched samples was extracted and screened for stx1, stx2abc, stx2f and stx2ex using multiplex PCR (hot start at 95\\u00b0C for 10 min, 38 cycles of 95\\u00b0C for 20 s and 60\\u00b0C for 45 s) with 2.5 units of AmpliTAQ gold, 1\\u00d7 supplied PCR buffer, 3 mM MgCl2, 200 mM dNTP, 300 nM of each primer, and 200 nM each probe to make a 20 \\u03bcl reaction (Cooley et al., 2013). An E. coli O157:H7 (ATCC 35150) positive control and a Salmonella Typhimurium (ATCC 14028) negative control were included in each run. A Ct value of \\u2264 28 for any of the four stx genes was considered to be a positive result.\\nPreparation of Bacterial Host Strains for Phage Isolation\\nFourteen STEC strains, including 2 strains per serogroups of O26, O45, O103, O111, O121, O145, and O157, from the culture collections of Produce Safety and Microbiology Research Unit in USDA ARS were used as host strains in this study (Supplementary Table S1). Non-pathogenic strains, including ATCC 13706, ATCC 43888, and DH5\\u03b1, were used for the isolation of phages encoding stx genes (or Stx phages). Overnight cultures were prepared by inoculating each non-pathogenic and STEC strain in an individual tube with 5 ml of Tryptic Soy Broth (TSB, Difco, Becton Dickinson, Sparks, MD, United States) and incubated overnight at 37\\u00b0C. The cocktails of non-pathogenic and STEC strains were prepared separately by adding 0.1 ml of each overnight culture from each group together shortly prior to use.\\nIsolation of Free Bacteriophages\\nTwo non-fecal composts in 50 ml conical tubes were gently mixed by hand prior to centrifugation at 8,000 \\u00d7 g for 10 min to get rid of sediments. The supernatants were filtered through 0.22 \\u03bcm syringe filter membrane, to get rid of background flora, and were used for phage isolation. In brief, an aliquot of 5 ml filtered supernatant from each sample was mixed with 25 ml TSB supplemented with 10 mM CaCl2 and inoculated with 0.3 ml of respective STEC and non-pathogenic E. coli cocktails for incubation at 37\\u00b0C for 48 h. Next, chloroform (4% v/v) was added to the enriched samples and homogenized on a rotator (Thermo Fisher Scientific, Waltham, MA, United States) for 30 min to inactivate the residual host strains and background flora. The supernatants were obtained after centrifugation at 8,000 \\u00d7 g for 10 min and used to determine the presence of the phages against top six non-O157 STEC and O157 STEC strains by spotting 10 \\u03bcl supernatants on Tryptic Soy Agar (TSA, Difco, Becton Dickinson, Sparks, MD, United States) containing one of the host strains (spot assay). After overnight incubation at 37\\u00b0C, the plates with clearing on the spotted area, which indicated positive results, were further subjected to phage purification process as previously described (Liao et al., 2018). During purification process, two plaques with different plaque sizes were picked from the first single-layer agar plaque assay plates, and three runs of plaque-picking and enrichment process were conducted (Liao et al., 2018). After purification, each phage lysate was enriched in 40 ml TSB with the corresponding host culture at 37\\u00b0C overnight for phage propagation. The propagated phages were filtered through 0.22 \\u03bcm syringe filter membrane prior to further analysis.\\nDetermination of Host Range and Antimicrobial Effects Using Efficiency of Plating for the Isolated Phages\\nTo determine host range and antimicrobial effects of the isolated phages, the spot assay was used as described above to spot the lysate of the isolated phages on the TSA plates pre-mixed with individual strain of the selected host panel, the same group as those used for phage isolation, and the level of the clearing (opaque, transparent and complete clearing) on the spotted area was used to determine the lysis ability of the phages against different serogroups of STEC. Efficiency of plating (EOP) was used to evaluate the lytic activity of the phages against a variety of bacterial strains. In brief, fresh overnight cultures of the tested bacterial strains and the primary bacterial strain used for phage isolation were prepared in TSB at 37\\u00b0C. After serial dilution, the diluted phage solutions (10-3 to 10-7) were used to obtain titers with each strain using a double-layer agar plaque assay, and the plates were incubated at 37\\u00b0C overnight. EOP was calculated based on phage titer obtained from the tested bacteria versus the phage titer from the primary bacterium used for phage isolation. If EOP was 0.5 or more, it was considered as high production efficiency. An EOP of 0.1 or above, but below 0.5, was considered as medium production efficiency. Low production efficiency had EOP between 0.001 and 0.1, and any value equal to or under 0.001 represented inefficiency of phage production.\\nDetermination of Plaque Morphology of the Isolated Phages\\nA stereo microscope (Stemi 305 cam, Carl Zeiss Microscopy, Germany) was used to observe plaque morphology from the double-layer agar plaque assay plates. The images of plaques were captured by use of Zeiss Labscope Application (Version 2.5.1).\\nConcentration of Phages for Downstream Analysis\\nThe filtered lysates were concentrated using Amicon Ultra-15 Centrifugal Filter Unit (Merck Millipore, Billerica, MA, United States) with 100 KDA cut-off. An aliquot of 10 ml phage lysate was added to Amicon filter unit, which was subsequently centrifuged using a fixed angle rotor at 5,000 \\u00d7 g according to manufacturer\\u2019s instruction. The filter unit was then washed with addition of 200 \\u03bcl sterile H2O and centrifuged at 5,000 \\u00d7 g to get rid of the water, followed by adding 200 \\u03bcl of 10 \\u03bcg/ml DNase I and incubating at 37\\u00b0C for 30 min to eliminate bacterial DNA contamination. After spinning down the DNase I, the filter was then washed with 5 ml of sterile H2O with the procedure as previously described. Subsequently, an aliquot of 500 \\u03bcl of sterile H2O was used to re-suspend the phages retained on the filter membrane, and the phage solution was transferred to a sterile microcentrifuge tube. Twenty microliters of the concentrated phages were spared for electron microscopy observation, while the rest of the lysates were used for DNA extraction.\\nTransmission Electron Microscopy\\nEach Amicon Filter-concentrated phage lysate with volume of 6 \\u03bcl was placed on copper mesh PLECO grids (Ted Pella Inc., Redding, CA, United States) and incubated for 1 min at room temperature, followed by blotting on Whatman filter paper and subsequent negative staining with addition of 8 \\u03bcl of 0.75% uranyl acetate (Sigma-Aldrich, Darmstadt, Germany) for incubation at room temperature (around 26\\u00b0C) for 30 s. The phage morphology was then examined using a transmission electron microscope (Tecnai G2 F20 model FEI, United States).\\nDetection of stx Genes in the Phages\\nThe concentrated phage lysates were then subjected to DNA extraction using a phage DNA extraction kit (Norgen Biotek Corp., ON, Canada). Presence of stx1 and stx2 genes in the phages was screened by PCR as previously described (Liao et al., 2018).\\nWhole Genome Sequencing and Genomic Analyses\\nPrior to library construction, the purity and concentration of each phage\\u2019s DNA was determined by electrophoresis on 1% agarose gel and Nanodrop 8000, respectively. Two hundred nanograms of DNA were fragmented to 550 bp by Ultrasonic Cell Disruptor (M220 Covaris) and subsequently subjected to DNA library construction using a TruSeq\\u00ae Nano DNA Library Prep Kit (Illumina, San Diego, CA, United States) according to manufacturer\\u2019s instruction. The libraries were sequenced on Illumina MiSeq sequencer using a MiSeq reagent kit v3 (Illumina, 600 cycles) with approximately 2 million paired-end (250-bp) sequence reads. The sequenced reads were subjected to FastQC, and the raw sequences were then trimmed using Trimmomatic on Galaxy online server1. A de novo assembly of the resultant quality reads was conducted using Unicycler v0.4.1 (SPAdes) with default parameters on Galaxy. The assembled sequences were subjected to NCBI (National Center for Biotechnology Information) BLAST to obtain taxonomic classification, including order, family, and genus. The phage termini and possible packaging mechanisms were predicted according to the in silico determination method proposed in PhageTerm (Garneau et al., 2017). The final contigs were annotated with Prokaryotic genome annotation (Prokka, v.1.12.0) with default parameters (Seemann, 2014). The open reading frames (ORFs) predicted and annotated by Prokka were further subjected to BLASTN search to confirm the annotation results. The reference phage genomes used in this study, including Escherichia phage ECML-117 (GenBank accession number JX128258.1), Escherichia phage ESCO13 (GenBank accession number KX552041.2), Escherichia phage K1ind1 (GU196279.1), and E. coli O157 typing phage 12 (GenBank accession number KP869110.1) were obtained based on the maximum score of the published sequences on NCBI nucleotide database after the BLASTN search using the assembled sequences of the new phages from this study. A visualized circular map with the predicted ORFs for each of vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw genomes were constructed using Geneious v11.0.4 (Biomatters Ltd., Auckland, New Zealand). The comparative analyses of phage nucleotide sequences, between the newly isolated phages and the corresponding reference phages, were conducted using EasyFig (Sullivan et al., 2011). ClustalW algorithm v2.1 (McWilliam et al., 2013) and the neighbor-joining method was used to evaluate evolutionary relationship of the phages based on the genes of interest (Saitou and Nei, 1987). The evolutionary distances were computed using the Maximum Composite Likelihood method (Tamura et al., 2004) and were in the units of the number of base substitutions per site. The analysis involved 4 nucleotide sequences. All positions containing gaps and missing data were eliminated. There was a total of 955 positions in the final dataset. Evolutionary analyses were conducted in MEGA7 (Kumar et al., 2016). The whole-genome sequences of Escherichia phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw were deposited to NCBI GenBank under accession number MH571750, MH160766, MH051334, and MH051335, respectively.\\nResults and Discussion\\nIsolation of STEC Strains and STEC-Specific Phages From the Composts\\nFor STEC isolation, a total of 17 and 23 presumptive positive colonies were picked from selective media after incubation of streak-plated enriched C1 and C2 samples, respectively. None of the colonies was positive of stx genes by PRC screening (Table 1). The results without isolation of STEC strains were possibly due to non-fecal materials of the compost samples used in this study.\\nTable 1. Isolation of STEC strains and STEC-specific bacteriophages from non-fecal composts.\\nWith regard to phage isolation, the spot assay results showed that both C1 and C2 contained phages lytic against the same serogroups of STEC strains, including O111, O121, O145, and O157, regardless of lysis levels on the plates. Additionally, the phages against non-pathogenic E. coli strains, including DH5\\u03b1, ATCC 13706, and ATCC 43888 (E. coli O157:H7 without stx genes), were also detected by spot assay in the C2 (Table 1). The non-pathogenic hosts were used primarily for the isolation of Stx phages from the samples; however, after phage purification, none of these phages, including both STEC-specific phages and those lytic against non-pathogenic strains, were positive of stx genes, indicating that no Stx phages were isolated from the non-fecal composts (data not shown). Based on degree of lysis from spot assay, four phages, one of each against serogroup of O111, O121, O145, and O157 strains, were selected for further biological and genomic analyses. The four phages, including Escherichia phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw, were named after the phage morphology, the species name and serogroup of the major bacterial host accordingly (Adriaenssens and Brister, 2017). The TEM observation showed that vB_EcoS-Ro145lw belonged to the family Siphoviridae, while the other 3 phages were Myoviridae (Figure 1B). Furthermore, vB_EcoS-Ro145lw produced the largest size of plaques among all phages, indicating its strong lytic effect against its host strain, STEC O145 (Figure 1A). The host range test indicated that, in addition to generic E. coli, such as ATCC 13706 and DH5\\u03b1 strains, both phages vB_EcoM-Ro111lw and vB_EcoM-Ro121lw were lytic (antimicrobial) against multiple serogroups of STEC strains, whereas vB_EcoS-Ro145lw and vB_EcoM-Ro157lw had narrow host ranges (Table 2). In addition, EOP results revealed that vB_EcoM-Ro111lw was the only phage showing medium or high infection efficiency against different serogroups (O26 and O111) of STEC strains.\\nFigure 1. The morphology of plaques formed by phage vB_EcoM-Ro111lw (A1), phage vB_EcoM-Ro121lw (A2), phage vB_EcoS-Ro145lw (A3), and phage vB_EcoM-Ro157lw (A4) on their host strains of STEC O111, O121, O145, and O157, respectively, using plaque assay, and the transmission electron microscopy images of phage vB_EcoM-Ro111lw (B1), phage vB_EcoM-Ro121lw (B2), phage vB_EcoS-Ro145lw (B3), and phage vB_EcoM-Ro157lw (B4).\\nTable 2. Host range and antimicrobial effects of the isolated bacteriophages against various STEC and non-pathogenic strains by spot assay\\u03b1 and EOP\\u03b2 (efficiency of plating).\\nA complete composting process is capable of reducing a significant amount of pathogens present in the raw organic materials (Erickson et al., 2009). However, if the composting organic materials containing high level of enteric pathogens, such fecal matters, was inefficiently managed, the final compost product may pose a risk of the spread of pathogens on a farm (Wu et al., 2009). Phages have been demonstrated to be more resistant to the environmental stress, such as the drastic change of pH and temperature, than bacteria and, thus, are likely to sustain through the composting process (Johannessen et al., 2005). Current findings revealed that various STEC-specific phages were isolated from the composting of non-fecal materials, such as yard trimmings or food scraps. Although the initial levels of pathogens in the raw materials prior to composting were unknown, the presence of STEC-specific phages was likely to add additional antimicrobial factors to reduce STEC to non-detectable levels during the process. Shiga toxin-producing E. coli (STEC)-specific phages were usually isolated from feces or fecal-contaminated environment (Imamovic et al., 2010; Wang et al., 2015; Amarillas et al., 2016); however, current study reported the presence of STEC-specific phages and evaluated its correlation with the STEC bacterial hosts from non-fecal composts.\\nGenomic Features of Four STEC-Specific Phages\\nThe genome sizes of phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw were 86,950bp, 149,803bp, 42,145bp, and 72,179bp, respectively (Table 3). According to taxonomy report of BLASTN search at NCBI nucleotide database, the phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, and vB_EcoM-Ro157lw belonged to the family Myoviridae and the vB_EcoS-Ro145lw belonged to Siphoviridae; the taxonomic results were in consensus to the morphologies observed under TEM of these phages. Additionally, the phage vB_EcoM-Ro111lw was further classified under sub-family Ounavirinae as FelixO1-like viruses; however, the rest of 3 phages were unclassified at genus level. The three myophages isolated in this study had genome sizes ranging from 72 k to 149 k bp in which the results were in agreement with the finding of the previous study indicating a wide range of genome sizes of the phages belonging to Myoviridae were isolated from cattle feces (Wang et al., 2015). The GC content of phage vB_EcoS-Ro145lw (50.6%) was not only higher than that of other new phages, with 46, 39.1, and 38.8% for vB_EcoM-Ro157lw, vB_EcoM-Ro121lw, and vB_EcoM-Ro111lw, respectively, but also close to that of E. coli strains with an average GC content of 50%. A previous study found that lytic phages usually had an average GC content 4% lower than that of its bacterial host, whereas temperate phages (with both lytic and lysogenic cycles) had GC content very close to the host (Rocha and Danchin, 2002). Although the GC content of phage vB_EcoS-Ro145lw was similar to that of its O145 STEC host strain, the phage expressed strong lysis, such as forming big plaques against the strain, without having any sign of lysogenic factor. On the other hand, the GC contents of both phages vB_EcoM-Ro121lw and vB_EcoM-Ro111lw were significant lower than that of their E. coli hosts (\\u223c50.7%), suggesting the host strains, O121 and O111 STEC, used for the phage isolation in this study were not the optimal hosts for these two phages (Bahir et al., 2009). The small plaque sizes formed by phages vB_EcoM-Ro121lw and vB_EcoM-Ro111lw against its original strains of isolation \\u2013O121 and O111 STEC strain, respectively \\u2013 might correlate with these findings.\\nTable 3. Genomic features of the STEC-specific bacteriophages from non-fecal composts.\\nPrediction of Genome Termini and Packaging Mechanisms\\nThe phage termini and possible packaging mechanism of the isolated phages were predicted according to the bioinformatic algorithm proposed in PhageTerm (Garneau et al., 2017). The analytical results indicated that both phages vB_EcoS-Ro145lw and vB_EcoM-Ro121lw had circularly permuted direct terminal repeats (without cohesive ends) with the packing mechanism similar to the well-known phage P1: headful with pac site (Oliveira et al., 2013). In the headful packaging mode, the terminase would recognize and cleave at a unique sequence region, called pac, to generate the first termini and initiate the packaging of the genomic DNA into the prohead (Li et al., 2014). The packing mechanisms for both phages vB_EcoM-Ro111lw and vB_EcoM-Ro157lw were predicted to be direct terminal repeats (DTR) at the genome termini without circular permutation: vB_EcoM-Ro111lw had a 613-bp terminal repeat region (similar to the phage T7, short direct terminal repeats) and vB_EcoM-Ro157lw had a 3533-bp terminal repeat region at both ends of the genome (similar to the phage T5, long direct terminal repeats) (Casjens and Gilcrease, 2009). Generally, in the mode of terminal repeats, the terminal sequences were generated by duplicating the direct repeat DNA sequence together with packaging; however, the mechanism regarding the generation of long repeats was unknown (Casjens and Gilcrease, 2009). Although the principles of several well-known packaging mechanisms were elucidated, identification of these packaging mechanisms from newly isolated phages traditionally require generation of restriction maps that are usually complicated with results not easy to interpret (Li et al., 2014). With the benefits of the computation of whole genome sequencing data using PhageTerm software, it was able to easily predict the terminal ends and types of the packaging mechanisms of these new phages (Garneau et al., 2017).\\nGenome Analysis of Four STEC-Specific Phages\\nThe BLASTN searches of the new phage genomes in this study against the reference sequences on NCBI database varied among different phages. The two best hits for vB_EcoS-Ro145lw genome included Escherichia phage P AB-2017 (67% coverage and 90% identity) and Escherichia phage K1ind1 (70% coverage and 89% identity); the low genome coverage of the new phages against its reference genomes indicated the difference of the phage as opposed to the known genomes among Siphoviridae phages from the public database. The BLASTN search results also showed that vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, and vB_EcoM-Ro157lw had the best hits against E. coli O157 typing phage 12, 11, and 8 (all with 95% coverage and 98% identity), Escherichia phage ESCO13 (98% coverage and 99% identity), and Escherichia phage ECML-117 (94% coverage and 97% identity), respectively. Although the phages vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, and vB_EcoM-Ro157lw had higher sequence coverage of genetic similarity than that of vB_EcoS-Ro145lw against their reference sequences, vB_EcoM-Ro111lw was the only phage further classified at genus level, FelixO1-like phages, based on NCBI taxonomy report. However, the phage vB_EcoM-Ro111lw was unable to infect Salmonella, unlike the Salmonella phage Felix O1, the core phage in the genus, with capability against different serovars of Salmonella strains (Whichard et al., 2010). BLAST comparison of the newly isolated phages and their reference phage genomes using EasyFig showed that that the genomes of vB_EcoM-Ro111lw and E. coli O157 typing phage 12 shared approximately over 90% nucleotide sequence similarity. However, one annotated ORF, coding for hypothetical protein, in phage vB_EcoM-Ro111lw was absent in the genome of E. coli O157 typing phage 12; two ORFs from the reference genome were lacking in vB_EcoM-Ro111lw (Figure 2A). The results also showed a high degree of nucleotide sequence similarity (> 90%) between the phages vB_EcoM-Ro121lw its reference genome \\u2013 Escherichia phage ESCO1 \\u2013 with 3 unshared ORFs, primarily coding for hypothetical proteins, from each other (Figure 2B). For vB_EcoS-Ro145lw genome, the results showed that low degree of nucleotide sequence similarity (68%) was observed on some shared ORFs as compared to the phage K1ind1 (Figure 2C). Additionally, there were 6 ORFs in vB_EcoS-Ro145lw that were absent in the counterpart genome and 3 ORFs from the reference genome that were not shared in the genome of vB_EcoS-Ro145lw. The comparison between the phage vB_EcoM-Ro157lw and Escherichia phage ECML-117 indicated a high nucleotide sequence similarity between the two genomes and all the annotated ORFs from the reference phage genome were shared in the vB_EcoM-Ro157lw; five ORFs in vB_EcoM-Ro157lw that coded for hypothetical proteins were absent in the reference phage (Figure 2D). These results showed that the genomes of the new phages and their counterparts contained some sequence regions with ORFs that were identified with the predicted hypothetical proteins in high similarity but in different organization in the genomes. The phenomenon, also known as mosaicism, likely deriving from horizontal genetic transfer may account for the diversity of phage genomes (Hatfull and Hendrix, 2011).\\nFigure 2. Whole genome comparison of Escherichia phage vB_EcoM-Ro111lw (A), Escherichia phage vB_EcoM-Ro121lw (B), Escherichia phage vB_EcoS-Ro145lw (C), and Escherichia phage vB_EcoM-Ro157lw (D) with their reference genomes from NCBI database using EasyFig. A gray-scaled shaded area indicates the different levels of sequence similarity between two phage sequences. The arrow shapes indicate the annotated ORFs in the genome, and the unshared ORFs are highlighted as red arrows.\\nAccording to genome annotation, the predicted number of ORFs for each of phage vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw were 153, 292, 71, and 109, respectively (Table 3). There were more than 75% of the ORFs annotated with unknown function, even including the FelixO1-like phage vB_EcoM-Ro111lw. The phage genomes that contained the ORFs with the predicted functions essential for phage structure, DNA metabolism and replication host lysis proteins, and host metabolism and regulation were illustrated with different colors in Figure 3. Additionally, only vB_EcoM-Ro111lw and vB_EcoM-Ro121lw contained tRNAs, with 22 and 10, respectively. Five of the tRNAs, #2, #10, #18, #20, and #21, in the vB_EcoM-Ro111lw genome and one, #4, in vB_EcoM-Ro121lw genome may be pseudo-tRNAs (Supplementary Table S2). Most tRNAs in these two phage genomes were located in the region adjacent to the DNA packing cluster, which might be associated with earlier stage of phage progenies formation (Bailly-Bechet et al., 2007). Bailly-Bechet et al. (2007) also indicated that lytic phages would contain more tRNAs, if present, than lysogenic phages. Moreover, it was demonstrated that the presence of tRNAs was more likely associated with the fitness of the phages, such as ability to infect more hosts, in order to adapt to different environment (Delesalle et al., 2016). Their findings were in consensus to the results of wide host ranges for the phages vB_EcoM-Ro111lw and vB_EcoM-Ro121lw in this study. In the genome of vB_EcoM-Ro157lw, the ORF #41 was predicted with the function of soluble lytic murein transglycosylase, which had the similar function as lysozyme/endolysin to catalyze the cell wall of its bacterial hosts, causing cell lysis (van Asselt et al., 1999), while the ORF #12, ORF #81 and ORF #11 were predicted as lysozyme/endolysin in the vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, and vB_EcoS-Ro145lw genomes, respectively (Supplementary Table S3\\u2013S6). Furthermore, in vB_EcoM-Ro111lw genome, two ORFs were annotated as putative dihydrofolate reductase (#77) and thymidylate synthase (#76) that were found to be the unique molecular markers for Bastille-like phages, a newly proposed genus of myophages in sub-family Spounavirinae, which were also known for the wide host ranges (Asare et al., 2015). The phage vB_EcoM-Ro111lw was classified as Felix O1-like phages according to NCBI taxonomy report, even though it was predicted to harbor the ORFs coded these two key markers, dihydrofolate reductase and thymidylate synthase, from other genus, Bastille-like phages, which demonstrated the evolutionary perspective of the phages likely through horizontal genetic transfer in the environment. A previous study found that a Salmonella-specific phage, UAB_Phi20, with detection of lysogenic genes still showed strong lytic infection against its Salmonella hosts; however, the lack of lysogenic cycle was derived from the suppression of the phage\\u2019s packaging mechanism, resulting in the dominant lytic cycle of the phage (Bardina et al., 2016). Though expression of strong lytic cycle, the presence of lysogenic genes in the phages may not be appropriate for the potential biocontrol use. In the current study, the genomic analysis showed that no lysogenic or virulence genes, such as stx or eae genes, were detected in the genomes of these four newly isolated phages.\\nFigure 3. Circular genome map of the phages vB_EcoM-Ro111lw (A), vB_EcoM-Ro121lw (B), vB_EcoS-Ro145lw (C), and vB_EcoM-Ro157lw (D). The outer circle represents genes as indicated by the arrows. Predicted functional categories of the genes were determined according to annotation and are represented with different colors. The central graph in blue shows genomic GC content variation.\\nComparative Genome Analysis\\nIt was hard to create a cluster of genes based on the sequences in this study because of the lack of similarity of these phage gnomes. Therefore, the ORFs encoded the proteins responsible for the function of host specificity and lytic effects, such as tail and lysozyme, were selected to construct phylogenetic analyses for a better understanding of the evolution of tailed phages with regard to its host range (Hagg\\u00e5rd-Ljungquist et al., 1992). The phylogenetic analysis based on ORFs encoding tail fiber showed both phages vB_EcoM-Ro111lw and vB_EcoM-Ro121lw were in the clade, indicating a closer evolutionary relationship with regard to the tail fiber (Figure 4A). The phages vB_EcoS-Ro145lw and vB_EcoM-Ro157lw were also in the same clade, which were closely related to Escherichia phage ST2 and phage ECML-117, respectively (Figure 4A). The results were in consensus to the previous biological features in which both phages vB_EcoM-Ro111lw and vB_EcoM-Ro121lw had a wider host range. With regard to endolysin/ lysozyme, the phylogenetic comparison indicated that phages vB_EcoM-Ro121lw and vB_EcoM-Ro157lw were in the same clade: the phage vB_EcoM-Ro157lw was closely related to phage ECML-117 whereas phage vB_EcoM-Ro121lw was further clustered into a single branch (Figure 4B). Similar results were also observed on the phages vB_EcoS-Ro145lw and vB_EcoM-Ro111lw that phage vB_EcoM-Ro111lw was closely related to phage KhF3 and phage vB_EcoS-Ro145lw was clustered into a single branch (Figure 4B).\\nFigure 4. Phylogenetic comparison based on Clustal Omega alignment of the genes coding for tail fiber (A) and endolysin/lysozyme (B) from the newly isolated phages (red underline) and other reference phage genomes obtained from NCBI database. The tree is drawn to scale, with branch lengths (next to the branches) in the same units as those of the evolutionary distances used to infer the phylogenetic tree.\\nConclusion\\nIn conclusion, the present study provides valuable information with regard to the isolation of various STEC-specific phages and the correlation of these phages with the STEC bacterial hosts from non-fecal compost samples. Whole-genome sequencing enabled us to comprehensively characterize the newly isolated phages, together with biological features, to determine further applications of the phages. Furthermore, genomic analyses of these newly isolated phages provide in-depth information regarding the evolution and diversity of STEC-specific phages isolated from non-fecal environment. The phages isolated in this study do not contain lysogenic factors potentially associated with the transfer of unwanted genes among bacterial population. The four new phages, including vB_EcoM-Ro111lw, vB_EcoM-Ro121lw, vB_EcoS-Ro145lw, and vB_EcoM-Ro157lw, together are able to infect a wide range of STEC strains, including serogroups of O157 and top 6 non-O157, due to their different but complementary host ranges. Without the presence of virulence genes, such as stx or eae genes and lysogenic factors, these phages are potential candidates used to control different serogroups of STEC strains. Moreover, evidence of the phenotypic and genotypic characteristics associated with the mitigating effects of these phages may account for zero isolation of STEC strains in this study. These findings suggest that the lytic phages are of great potential in controlling various STEC strains and may provide additional antimicrobial effects against STEC that are present in the raw composting materials and sneaked through the composting process if not well-managed. Further studies regarding the reduction of STEC by these phages under various conditions are necessary in order to optimize the antimicrobial activities against STEC in different food-associated environments.\\nData Availability\\nThe datasets generated for this study can be found in Genbank, MH571750, MH160766, MH051334, and MH051335.\\nAuthor Contributions\\nY-TL was responsible for phage isolation with XS, data analyses, genome annotation, and manuscript preparation. FL and YZ were responsible for phage sequencing. DB was responsible for STEC strain isolation. IQ and AS were responsible for TEM and host range test. VW conceived the study, aided in experiment design, and prepared the manuscript. All authors reviewed the manuscript.\\nFunding\\nThis research was funded by USDA-ARS CRIS projects 2030-42000-050-00D. The collaboration was supported by Henan Province Key Research and Development Programs (International cooperation projects 182102410043) and Henan Key Laboratory of Cold Chain Food Quality and Safety Control (CCFQ2018-YB-14).\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nThe authors thank Leslie Harden for obtaining the composting samples.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2019.00627/full#supplementary-material\\nFootnotes\\n^https://usegalaxy.org\\nReferences\\nAmarillas, L., Chaidez, C., Gonzalez-Robles, A., Lugo-Melchor, Y., and Leon-Felix, J. (2016). Characterization of novel bacteriophage phiC119 capable of lysing multidrug-resistant Shiga toxin-producing Escherichia coli O157:H7. PeerJ 13:e2423. doi: 10.7717/peerj.2423\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAmgarten, D., Martins, L. F., Lombardi, K. C., Antunes, L. P., De Souza, A. P. S., Nicastro, G. G., et al. (2017). Three novel pseudomonas phages isolated from composting provide insights into the evolution and diversity of tailed phages. BMC Genomics 18:346. doi: 10.1186/s12864-12017-13729-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAsare, P. T., Jeong, T. Y., Ryu, S., Klumpp, J., Loessner, M. J., Merrill, B. D., et al. (2015). Putative type 1 thymidylate synthase and dihydrofolate reductase as signature genes of a novel bastille-like group of phages in the subfamily Spounavirinae. BMC Genomics 16:582. doi: 10.1186/s12864-015-1757-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBahir, I., Fromer, M., Prat, Y., and Linial, M. (2009). Viral adaptation to host: a proteome-based analysis of codon usage and amino acid preferences. Mol. Syst. Biol. 5:311. doi: 10.1038/msb.2009.71\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBardina, C., Colom, J., Spricigo, D. A., Otero, J., Sanchez-Osuna, M., Cortes, P., et al. (2016). Genomics of three new bacteriophages useful in the biocontrol of Salmonella. Front. Microbiol. 7:545. doi: 10.3389/fmicb.2016.00545\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBeutin, L., and Martin, A. (2012). Outbreak of Shiga toxin-producing Escherichia coli (STEC) O104:H4 infection in Germany causes a paradigm shift with regard to human pathogenicity of STEC strains. J. Food Prot. 75, 408\\u2013418. doi: 10.4315/0362-028X.JFP-11-452\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBuck, J. W., Walcott, R. R., and Beuchat, L. R. (2003). Recent trends in microbiological safety of fruits and vegetables. Plant Health Prog. 10, 1094\\u20131103. doi: 10.1094/PHP-2003-0121-01-RV\\nCrossRef Full Text | Google Scholar\\nCasjens, S. R., and Gilcrease, E. B. (2009). Determining DNA packaging strategy by analysis of the termini of the chromosomes in tailed-bacteriophage virions. Methods Mol. Biol. 502, 91\\u2013111. doi: 10.1007/978-1-60327-565-1_7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCooley, M. B., Jay-Russell, M., Atwill, E. R., Carychao, D., Nguyen, K., Quinones, B., et al. (2013). Development of a robust method for isolation of Shiga toxin-positive Escherichia coli (STEC) from fecal, plant, soil and water samples from a leafy greens production region in California. PLoS One 8:e65716. doi: 10.1371/journal.pone.0065716\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDelesalle, V. A., Tanke, N. T., Vill, A. C., and Krukonis, G. P. (2016). Testing hypotheses for the presence of tRNA genes in mycobacteriophage genomes. Bacteriophage 6:e1219441. doi: 10.1080/21597081.2016.1219441\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nErickson, M. C., Liao, J., Ma, L., Jiang, X., and Doyle, M. P. (2009). Inactivation of Salmonella spp. in cow manure composts formulated to different initial C:N ratios. Bioresour. Technol. 100, 5898\\u20135903. doi: 10.1016/j.biortech.2009.06.083\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nFrank, C., Werber, D., Cramer, J. P., Askar, M., Faber, M., An Der Heiden, M., et al. (2011). Epidemic profile of Shiga-toxin-producing Escherichia coli O104:H4 outbreak in Germany. N. Eng. J. Med. 365, 1771\\u20131780. doi: 10.1056/NEJMoa1106483\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGarneau, J. R., Depardieu, F., Fortier, L. C., Bikard, D., and Monot, M. (2017). PhageTerm: a tool for fast and accurate determination of phage termini and packaging mechanism using next-generation sequencing data. Sci. Rep. 7:8292. doi: 10.1038/s41598-41017-07910-41595\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHagg\\u00e5rd-Ljungquist, E., Halling, C., and Calendar, R. (1992). DNA sequences of the tail fiber genes of bacteriophage P2: evidence for horizontal transfer of tail fiber genes among unrelated bacteriophages. J. Bacteriol. 174, 1462\\u20131477. doi: 10.1128/jb.174.5.1462-1477.1992\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHandley, S. A., Thackray, L. B., Zhao, G., Presti, R., Miller, A. D., Droit, L., et al. (2012). Pathogenic simian immunodeficiency virus infection is associated with expansion of the enteric virome. Cell 151, 253\\u2013266. doi: 10.1016/j.cell.2012.09.024\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHeringa, S. D., Kim, J., Jiang, X., Doyle, M. P., and Erickson, M. C. (2010). Use of a mixture of bacteriophages for biological control of Salmonella enterica strains in compost. Appl. Environ. Microbiol. 76, 5327\\u20135332. doi: 10.1128/AEM.00075-00010\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nImamovic, L., Balleste, E., Jofre, J., and Muniesa, M. (2010). Quantification of Shiga toxin-converting bacteriophages in wastewater and in fecal samples by real-time quantitative PCR. Appl. Environ. Microbiol. 76, 5693\\u20135701. doi: 10.1128/AEM.00107-10\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJohannessen, G. S., James, C. E., Allison, H. E., Smith, D. L., Saunders, J. R., and Mccarthy, A. J. (2005). Survival of a Shiga toxin-encoding bacteriophage in a compost model. FEMS Microbiol. Lett. 245, 369\\u2013375. doi: 10.1016/j.femsle.2005.03.031\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKim, J., Shepherd, M. W. J., Jiang, X., and Jiang, X. (2009). Evaluating the effect of environmental factors on pathogen regrowth in compost extract. Microb. Ecol. 58, 498\\u2013508. doi: 10.1007/s00248-009-9524-x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKumar, S., Stecher, G., and Tamura, K. (2016). MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets. Mol. Biol. Evol. 33, 1870\\u20131874. doi: 10.1093/molbev/msw054\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLemunier, M., Francou, C., Rousseaux, S., Houot, S., Dantigny, P., Piveteau, P., et al. (2005). Long-term survival of pathogenic and sanitation indicator bacteria in experimental biowaste composts. Appl. Environ. Microbiol. 71, 5779\\u20135786. doi: 10.1128/AEM.71.10.5779-5786.2005\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLi, S., Fan, H., An, X., Fan, H., Jiang, H., Chen, Y., et al. (2014). Scrutinizing virus genome termini by high-throughput sequencing. PLoS One 9:e85806. doi: 10.81371/journal.pone.0085806\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLiao, Y. T., Quintela, I. A., Nguyen, K., Salvador, A., Cooley, M. B., and Wu, V. C. H. (2018). Investigation of prevalence of free Shiga toxin- producing Escherichia coli (STEC)-specific bacteriophages and its correlation with STEC bacterial hosts in a produce-growing area in Salinas, California. PLoS One 13:e0190534. doi: 10.1371/journal.pone.0190534\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMartinez-Castillo, A., and Muniesa, M. (2014). Implications of free Shiga toxin-converting bacteriophages occurring outside bacteria for the evolution and the detection of Shiga toxin-producing Escherichia coli. Front. Cell. Infect. Microbiol. 4:46. doi: 10.3389/fcimb.2014.00046\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMcWilliam, H., Li, W., Uludag, M., Squizzato, S., Park, Y. M., Buso, N., et al. (2013). Analysis tool web services from the EMBL-EBI. Nucleic Acids Res. 41, W597\\u2013W600. doi: 10.1093/nar/gkt1376\\nCrossRef Full Text | Google Scholar\\nOliveira, M., Vinas, I., Usall, J., Anguera, M., and Abadias, M. (2012). Presence and survival of Escherichia coli O157:H7 on lettuce leaves and in soil treated with contaminated compost and irrigation water. Int. J. Food Microbiol. 156, 133\\u2013140. doi: 10.1016/j.ijfoodmicro.2012.03.014\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPartanen, P., Hultman, J., Paulin, L., Auvinen, P., and Romantschuk, M. (2010). Bacterial diversity at different stages of the composting process. BMC Microbiol. 10:94. doi: 10.1186/1471-2180-1110-94\\nCrossRef Full Text | Google Scholar\\nRyckeboer, J., Mergaert, J., Vaes, K., Klammer, S., De Clercq, D., Coosemans, J., et al. (2003). A survey of bacteria and fungi occurring during composting and self-heating process. Ann. Microbiol. 53, 349\\u2013410.\\nGoogle Scholar\\nSaitou, N., and Nei, M. (1987). The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol. Biol. Evol. 4, 406\\u2013425.\\nGoogle Scholar\\nSeemann, T. (2014). Prokka: rapid prokaryotic genome annotation. Bioinformatics 30, 2068\\u20132069. doi: 10.1093/bioinformatics/btu2153\\nCrossRef Full Text | Google Scholar\\nTamura, K., Nei, M., and Kumar, S. (2004). Prospects for inferring very large phylogenies by using the neighbor-joining method. Proc. Natl. Acad. Sci. U.S.A. 101, 11030\\u201311035. doi: 10.1073/pnas.0404206101\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nvan Asselt, E. J., Dijkstra, A. J., Kalk, K. H., Akacs, B., Keck, W., and Dijkstra, B. W. (1999). Crystal structure of Escherichia coli lytic transglycosylase Slt35 reveals a lysozyme-like catalytic domain with an EF-hand. Structure 7, 1167\\u20131180. doi: 10.1016/S0969-2126(00)80051-9\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nViazis, S., Akhtar, M., Feirtag, J., Brabban, A. D., and Diez-Gonzalez, F. (2011). Isolation and characterization of lytic bacteriophages against enterohaemorrhagic Escherichia coli. J. Appl. Microbiol. 110, 1323\\u20131331. doi: 10.1111/j.1365-2672.2011.04989.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWalters, S. P., Thebo, A. L., and Boehm, A. B. (2011). Impact of urbanization and agriculture on the occurrence of bacterial pathogens and stx genes in coastal waterbodies of central California. Water Res. 45, 1752\\u20131762. doi: 10.1016/j.watres.2010.1711.1032\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWang, J., Niu, Y. D., Chen, J., Anany, H., Ackermann, H. W., Johnson, R. P., et al. (2015). Feces of feedlot cattle contain a diversity of bacteriophages that lyse non-O157 Shiga toxin-producing Escherichia coli. Can. J. Microbiol. 61, 467\\u2013475. doi: 10.1139/cjm-2015-0163\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWhichard, J. M., Weigt, L. A., Borris, D. J., Li, L. L., Zhang, Q., Kapur, V., et al. (2010). Complete genomic sequence of bacteriophage felix O1. Viruses 2, 710\\u2013730. doi: 10.3390/v2030710\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWu, S., Nishihara, M., Kawasaki, Y., Yokoyama, A., Matsuura, K., Koga, T., et al. (2009). Inactivation of Escherichia coli in soil by solarization. Soil Sci. Plant Nutr. 55, 258\\u2013263. doi: 10.1111/j.1747-0765.2009.00362.x\\nCrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2021.616548\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2021.616548/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:19.798Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2021.616548\",\n",
      "            \"title\": \"Frontiers | Isolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples\",\n",
      "            \"description\": \"Vibrio parahaemolyticus is a foodborne pathogen that is frequently isolated from a variety of seafood. To control this pathogenic Vibrio spp., the implementation of bacteriophages in aquaculture and food industries have shown a promising alternative to antibiotics. In this study, six bacteriophages isolated from the seafood samples demonstrated a narrow host range specificity that infecting only the V. parahaemolyticus strains. Morphological analysis revealed that \\u0278Vp33, \\u0278Vp22, \\u0278Vp21 and \\u0278Vp02 belong to the Podoviridae family, while \\u0278Vp08 and \\u0278Vp11 were categorized into the Siphoviridae family. All bacteriophages were composed of DNA genome and showed distinctive restriction fragment length polymorphism. The optimal MOI for bacteriophage propagation was determined to be 0.001 to 1. One-step growth curve revealed that the latent period ranged from 10 to 20 min, and the burst size of bacteriophage was approximately 17 to 51 PFU/cell. The influence of temperature and pH levels on the stability of bacteriophages showed that all bacteriophages were optimally stable over a wide range of temperatures and pH levels. In vitro lytic activity of all bacteriophages demonstrated to have a significant effect against V. parahaemolyticus. Besides, the application of a bacteriophage cocktail instead of a single bacteriophage suspension was observed to have a better efficiency to control the growth of V. parahaemolyticus. Results from this study provided a basic understanding of the physiological and biological properties of the isolated bacteriophages before it can be readily used as a biocontrol agent against the growth of V. parahaemolyticus.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nVibrio parahaemolyticus is a Gram-negative halophilic bacterium, which occurs naturally in marine environments. It is recognized as a leading foodborne pathogen that is frequently isolated from a variety of seafood (Wang et al., 2015). Consumption of food contaminated with V. parahaemolyticus can cause gastroenteritis, which is often displayed through bouts of abdominal pain, diarrhea, fever, and nausea. An open wound in the skin which comes in contact with V. parahaemolyticus can also result in wound infection. In rare instances, V. parahaemolyticus can cause life-threatening conditions like rapid necrosis of subcutaneous tissue and septicemia (Tena et al., 2010; Ahmad et al., 2013; Zhang and Orth, 2013). However, not all V. parahaemolyticus strains are pathogenic and capable of causing illness in humans (Elmahdi et al., 2016). The presence of thermostable direct hemolysin (TDH) and the TDH-related hemolysin (TRH) has been identified as well-known pathogenic factors in V. parahaemolyticus which cause vibriosis. Both tdh and trh genes are therefore widely used as a virulence marker in many studies for the detection of pathogenic V. parahaemolyticus strains (Chen et al., 2016; Kang et al., 2017; Paria et al., 2019). However, infection caused by V. parahaemolyticus strains without the virulence tdh and trh genes have been reported in numerous studies (Bhoopong et al., 2007; Jones et al., 2012; Li et al., 2014; Pazhana et al., 2014). The pathogenesis of V. parahaemolyticus is therefore still open to question and has not been fully elucidated.\\nBacteriophages or phages are bacterial viruses that infect bacterial cells only. Bacteriophages can be found in ubiquitous environments and are recognized as the most abundant organisms with 1031 bacteriophage particles on the earth\\u2019s surface (Comeau et al., 2008; Keen, 2015). Bacteriophages are typically highly host-specific, which targets only specific species or even strains (Hagens and Loessner, 2007). Besides, high specificity levels of bacteriophages could minimize the disturbance of commensal bacteria (Viazis et al., 2011). The International Committee on Taxonomy of Viruses (ICTV) is responsible for the classification and nomenclature of viruses based on their morphological, genomic and proteomic characteristics (Adams et al., 2017). Tailed bacteriophages are grouped into the order of Caudovirales which consist of three families Myoviridae, Podoviridae, and Siphoviridae. Today, the order of Caudovirales includes six subfamilies, 80 genera, and 441 species (Krupovic et al., 2016). In 2018 and 2019, an update from the ICTV has proposed the creation of a new order (Tubulavirales), 10 new families (Autographiviridae, Chaseviridae, Demerecviridae, Drexlerviridae, Finnlakeviridae, Halspiviridae, Herelleviridae, Ovaliviridae, Plectroviridae, and Thaspiviridae), 22 subfamilies, 424 genera, and 964 species (Adriaenssens et al., 2020).\\nBacteriophages can be grouped into individual families based on their distinct morphological structure, as viewed through electron microscopy (Schramlov\\u00e1 et al., 2010). Electron microscopy such as transmission electron microscopy (TEM) is typically used at the initial stage for rapid identification and characterization of viruses. Genetic materials of viruses can be either DNA or RNA, and the structure of the nucleic acid may be either single-stranded or double-stranded. Restriction enzyme cleaves the DNA of bacteriophages into distinct polymorphism, which is known as restriction fragment length polymorphism (RFLP). The use of RFLP analysis can facilitate the study of genetic diversity and differentiate between individual bacteriophages. On the other hand, protein profiles of bacteriophages can be analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). SDS-PAGE analysis of phage structural proteins will reveal the molecular weight of phage proteins and help to determine the presence of phage structural proteins.\\nIt is noteworthy that the discovery of bacteriophages occurred much earlier than antibiotics. The use of bacteriophage preparations in human trials have also been conducted for almost a century in Eastern Europe (Lin et al., 2017). Nevertheless, the development of a broad-spectrum of antibiotics later received more attention than the use of bacteriophages. Today, phage therapy involves the use of bacteriophages in clinical trials that have gained new attention due to the growing number of infections caused by multidrug-resistance bacteria (Rohde et al., 2018). Phage therapy against several bacterial pathogens in animal models have also been tested out and showed effective results in many studies (Biswas et al., 2002; Wang et al., 2006; Watanabe et al., 2007; Jun et al., 2014; Kalatzis et al., 2018).\\nThe application of bacteriophages is not only limited to clinical trials to fight against bacterial infections. It can be used in livestock protection to reduce the mortality rate of animals as well as enhance the food safety levels by controlling the growth of foodborne pathogens. Globally, efforts to control Vibrio spp. with the application of bacteriophages in aquaculture settings have been reported by researchers. For example, Wang et al. (2017) demonstrated that the use of bacteriophage therapy in controlling V. harveyi within greenlip abalone. Le et al. (2020b) showed that the application of lytic bacteriophages in the reduction of oyster larvae mortality rate caused by V. alginolyticus. Le et al. (2020a) demonstrated that the effectiveness of bacteriophages treatment in the decontamination of Vibrio spp. in commercially produced microalgae that used as the primary food source for oyster larvae during hatchery culture. In the future, it is believed that the use of bacteriophages could be a promising alternative to antibiotics for fighting against bacterial pathogens.\\nIn this study, the objective was to isolate V. parahaemolyticus bacteriophages from the seafood samples and the isolated bacteriophages were characterized based on host specificity, morphological, genomic character, protein profile, and temperature and pH stability. The in vitro lytic activity of single bacteriophage suspensions as well as the bacteriophage cocktails were also evaluated to determine their efficiency against the growth of V. parahaemolyticus.\\nMaterials and Methods\\nBacterial Strains\\nSix V. parahaemolyticus strains which consisted of 4 clinical isolates and 2 food source isolates (non-tdh and/or trh) were used as hosts for the isolation of bacteriophages (Table 1). PSU5333, PSU5322, and PSU4211 V. parahaemolyticus strains were kindly provided by Professor Dr. Varaporn Vuddhakul from the Prince of Songkla University, Thailand. V. parahaemolyticus ATCC 17802 was purchased from the American Type Culture Collection (ATCC), United States. Both VP8 and VP11 isolates were obtained from the laboratory stock strain at Food Safety Laboratory, Universiti Putra Malaysia. All bacterial strains used were confirmed by PCR amplification of species-specific toxR gene and pathogenicity tdh and trh genes with specific primers as mentioned in the previous study (Tan et al., 2017) and the amplified sequences were further confirmed by DNA sequencing.\\nTable 1. V. parahaemolyticus strains used for bacteriophages isolation.\\nSample Preparation\\nA total of 30 seafood samples consisting of 10 samples for each blood clam, shrimp, and surf clam were purchased from different wet markets in Selangor, Malaysia. Ten grams of each sample was then weighed and transferred into 10 mL of salt of magnesium (SM) buffer (100 mM NaCl, 8 mM MgSO4\\u22c57H2O, 50 mM pH 7.5 Tris-HCl and 0.01% gelatin [Merck, Germany]) in a 50 mL sterile centrifuge tube. The mixture was shaken at 150 rpm for 15 min in an orbital shaker. Large particles were pelleted in the tube via centrifugation at 10,000 \\u00d7 g for 5 min. The supernatant was filtered through a 0.2 \\u03bcM pore size syringe filter (Pall, United States). A 200 \\u03bcL of log-phase V. parahaemolyticus host culture (OD600nm = 0.4\\u20130.6) was added into the 5 mL of filtrate and incubated at 37\\u00b0C, 150 rpm in an orbital shaker overnight. After incubation, the mixture was centrifuged at 10,000 \\u00d7 g for 5 min to pelletize the bacterial cells and the supernatant was filtered with a 0.2 \\u03bcM pore size syringe filter (Pall, United States).\\nDetection of the Presence of Bacteriophages\\nTen-fold serial dilutions (through 10\\u20138) of the filtrate was prepared in SM buffer. Each of these different dilutions was screened for the presence of bacteriophage by spotting 10 \\u03bcL of the filtrate on a double-layer agar plate with a bacterial host (Beck et al., 2009). The plate was allowed to dry for at least 15 min and the plate was incubated in an inverted position at 37\\u00b0C overnight. Lysis zone was observed after the incubation. Filtrate from the highest dilution which showed a clear lysis zone on the double-layer agar plate was selected for isolation, propagation, and purification.\\nIsolation of Bacteriophage\\nIsolation of bacteriophages was carried out according to the double agar overlay plaque assay (Kropinski et al., 2009). A 100 \\u03bcL of each dilution filtrate and 100 \\u03bcL of the log-phase bacteria culture was aliquot into a 3 mL molten soft agar, which constituted of tryptic soy broth (TSB) (Merck, Germany) and 0.6% bacteriological agar (Merck, Germany), which was stored at 45\\u00b0C. The molten soft agar was mixed gently and poured evenly onto tryptic soy agar (TSA) (Merck, Germany) base plate. The plate was allowed to solidify for at least 15 min and incubated in an inverted position at 37\\u00b0C overnight. After incubation, a single colony plaque with clear lysis appearance from the highest dilution was picked using a sterile inoculation loop and transferred it into a 5 mL SM buffer with 200 \\u03bcL log-phase bacteria culture. The mixture was incubated at 37\\u00b0C, 150 rpm in an orbital shaker overnight. After incubation, the sample was centrifuged at 10,000 \\u00d7 g for 5 min, and the supernatant was filtered with a 0.2 \\u03bcM pore size syringe filter.\\nPropagation and Precipitation of Bacteriophage\\nBacteriophage was amplified according to the phage propagation via liquid lysate as described by Bonilla et al. (2016) but with some modifications. Briefly, 200 mL of TSB supplemented with 0.001 M CaCl2 and MgCl2 was prepared in a 250 mL centrifuge bottle and spiked with 0.1 volumes of overnight bacteria host. A 200 \\u03bcL of bacteriophage lysate (>108 PFU/mL) was then added into the centrifuge bottle and incubated at 37\\u00b0C, 150 rpm in an orbital shaker until the lysate cleared. After incubation, the sample was centrifuged at 4\\u00b0C and 4,000 \\u00d7 g for 20 min. The supernatant was filtered with a 0.2 \\u03bcM pore size syringe filter into a sterile centrifuge bottle. The filtrate was precipitated by adding 10% (w/v) polyethylene glycol (PEG) 8000 (Sigma-Aldrich, United States) and kept undisturbed at 4\\u00b0C for 24 h. After 24 h, the sample was centrifuged at 4\\u00b0C and 4,000 \\u00d7 g for 20 min. The supernatant was removed, and the pellet was suspended in a 10 mL of SM buffer.\\nPurification of Bacteriophage\\nBacteriophage suspension was purified using caesium chloride (CsCl) (Santa Cruz Biotechnology, United States) with a density gradient according to the method described in the T7 Select\\u00ae System Manual TB178 (Novagen, United States). A 5 mL of bacteriophage suspension was aliquot slowly onto the uppermost layer and the sample was subjected to ultracentrifugation at 210,000 \\u00d7 g and 4\\u00b0C for 1 h in a SorvallTM WX ultracentrifuge with a swinging bucket TH-641 Rotor (Thermo Scientific, United States). After centrifugation, a sharp and turbid band containing the purified bacteriophage particles was collected carefully with a Pasteur pipet. The purified bacteriophage was subjected to dialysis to remove CsCl with an Amicon\\u00ae Ultra-15 100K filter device (Merck, Germany) and SM buffer (Bonilla et al., 2016). After dialysis, the purified bacteriophage was assayed to determine the bacteriophage titer. The purified bacteriophage stock sample was stored at 4\\u00b0C until further analysis.\\nHost Range Specificity\\nA total of 134 bacteria strains including 126 V. parahaemolyticus strains and 8 non-V. parahaemolyticus strains were used to study the host range of the isolated bacteriophages. Non-V. parahaemolyticus strains including of V. alginolyticus, V. cholerae, V. vulnificus, Escherichia coli, Listeria monocytogenes, Salmonella enterica serovar Typhimurium, Salmonella enterica serovar Enteritidis, and Staphylococcus aureus. Host range specificity was determined by spotting 20 \\u03bcL of each bacteriophage (105 PFU/mL) on a double-layer agar plate seeded with a bacteria strain (Viazis et al., 2011). The agar plate with bacteriophage inoculum was allowed to dry for at least 15 min before being incubated at 37\\u00b0C overnight. Lysis zones were observed on the plate after the incubation.\\nTransmission Electron Microscopy (TEM)\\nBacteriophage was visualized by a transmission electron microscope (TEM) as described by Kim et al. (2019) with some modifications. A carbon-coated copper grid was placed onto a drop of purified bacteriophage (1010 PFU/mL) for 2 min. The grid with the sample was negatively stained first with 2% (w/v) uranyl acetate for 2 min, followed by sterile water for 1 min. The excess sample was removed with a filter paper, and the grid was allowed to air dry for 10 min. The morphology of the bacteriophage was then visualized with a JEM-2100 200-kV transmission electron microscope (JEOL, Japan).\\nBacteriophage Genome Analysis\\nExtraction of nucleic acids of bacteriophage was performed according to the protocol provided with the Wizard DNA Clean-Up Kit (Promega, United States). For determination of the type of nucleic acids, 2 \\u03bcL of extracted bacteriophage nucleic acid was treated separately with 2 \\u03bcL of DNase I (1 U/\\u03bcL) and RNase A (10 mg/mL) (Thermo Scientific, United States) for 30 min at 37\\u00b0C (Topka et al., 2019). The treated sample was electrophoresed through 0.7% (w/v) agarose gel at 100 V for 30 min. The agarose gel was stained with ethidium bromide (0.5 \\u03bcg/ml) (Merck, Germany) and visualized with Syngene GeneGenius BioImaging system, together with GeneSnap software (version 7.12.01) (Syngene, Cambridge, United Kingdom).\\nRestriction Fragment Length Polymorphism (RFLP)\\nGenomic DNA (1 \\u03bcg/\\u03bcL) of each bacteriophage was digested individually with EcoRI, HaeIII, HindIII, and SalI restriction enzymes (Promega, United States) according to the manufacturer\\u2019s protocols and incubated at 37\\u00b0C for 1 h. The digested product was electrophoresed and visualized as mentioned in the above section.\\nSDS-PAGE Analysis\\nProtein profiles of bacteriophage were analyzed by one-dimensional SDS-PAGE. Each purified bacteriophage (1010 PFU/mL) sample was first solubilized by mixing an equal volume of 2\\u00d7 sample buffer solution (0.125 M Tris-HCl, 4% w/v SDS, 20% v/v glycerol, 0.01% w/v bromophenol blue, 10% v/v 2-mercaptoethanol) and boiled at 95\\u00b0C for 10 min (Guan et al., 2004). The solubilized sample was loaded into 5\\u201315% Bullet PAGE One Precast Gel (Nacalai Tesque, Japan) and electrophoresed in 1\\u00d7 running buffer solution (0.25 mol/L Tris, 1.92 mol/L Glycine, and 10 g/L SDS) at 200 V for 30 min. The gel was stained with CBB Stain One Super (Nacalai Tesque, Japan) for 30 min and de-stained several times with deionized water.\\nMultiplicity of Infection (MOI)\\nThe multiplicity of infection (MOI) of the bacteriophage was tested according to Liu et al. (2013) with some modifications. The log-phase bacteria host was first prepared and mixed with bacteriophage suspension at different ratios (0.0001, 0.001, 0.01, 0.1, 1, 10, and 100 PFU/CFU). After incubation at 37\\u00b0C for 3.5 h, the sample was centrifuged at 10,000 \\u00d7 g for 5 min. The supernatant was filtered with a 0.2 \\u03bcM pore size syringe filter and enumerated to determine the bacteriophage titer. Bacteria-free suspension and bacteriophage-free suspension was included in all the experiments as a control sample. An MOI ratio with the highest bacteriophage titer was considered as the optimal MOI.\\nOne-Step Growth Curve\\nOne-step growth curve of each bacteriophage was studied according to the protocol described by Hong et al. (2014). Briefly, bacteriophage suspension was mixed with 1 mL of log-phase bacteria host culture at the optimal MOI ratio. The mixture was incubated at 37\\u00b0C and 150 rpm for 10 min to allow the adsorption of bacteriophages to the host cells before centrifuging at 10,000 \\u00d7 g for 30 s. The pellet was suspended in 10 mL of TSB and the suspension was incubated at 37\\u00b0C and 150 rpm. A 100 \\u03bcL of sample was withdrawn at 10 min intervals up to a total period of 1 h and enumerated by double agar overlay plaque assay in duplicate. The latent period and burst size of the bacteriophage was evaluated from the one-step growth curve.\\nTemperature and pH Stability\\nTemperature and pH stability were tested according to the method described by Thung et al. (2017) with some modifications. The temperature stability of bacteriophage suspension in the SM buffer was tested at different temperatures (\\u221220, 25, 37, 50, 60, and 70\\u00b0C) and incubated for 2 h. For pH stability, bacteriophage suspension was incubated at different pH ranges (pH 2, 3, 5, 7, 9, and 11) for 2 h. Bacteriophage suspension kept at 4\\u00b0C in the SM buffer (pH7.5) was used as a control in this study. After incubation, the titer of bacteriophage was determined by double agar overlay plaque assay and the survival rate was calculated by dividing the bacteriophage titer after treatment with the initial bacteriophage concentration in the control sample (Liu et al., 2020).\\nIn vitro Lytic Activity\\nIn vitro lytic activity was determined individually for each bacteriophage (Vp33, Vp22, Vp21, Vp02, Vp08, and Vp11) against its respective bacterial host strain. Two bacteriophage cocktails (cocktails A and B) were also prepared and tested for their synergistic effect (Table 2). Bacteriophage cocktails were selected based on the host range specificity results in the section \\u201cHost Range Specificity.\\u201d Cocktails A and B were prepared for targeting V. parahaemolyticus clinical strains and environmental strains, respectively. A 500 \\u03bcL of log-phase V. parahaemolyticus host culture was mixed with an equal volume of bacteriophage lysate (\\u223c1 \\u00d7 106 PFU/mL). The mixture was incubated immediately at 37\\u00b0C. A 100 \\u03bcL sample of the mixture was withdrawn at 0, 2, 4, 6, and 24 h, to determine the concentration of bacteria through spread plating onto a CHROMagarTM Vibrio (CV) plate. The plate was allowed to dry for at least 5 min before inverting the plate for incubation.\\nTable 2. Bacteriophage cocktail preparations.\\nResults\\nScreening of Bacteriophage\\nA total of six bacteriophages displayed an ability to lyse V. parahaemolyticus host strains and caused lysis zones on the double-layer agar plate was discovered in the blood clam, shrimp, and surf clam samples. Out of 10 shrimp samples, four samples were found to have the presence of V. parahaemolyticus bacteriophages. Only one sample of blood clam and surf clam was detected with the presence of V. parahaemolyticus bacteriophages.\\nHost Range Specificity\\nPurified bacteriophages were tested for their host range specificity. Lysis zones were observed on 61 out of 126 (48.4%) of the double-layer agar plate seeded with V. parahaemolyticus strains (Table 3). No lysis zones were observed on all non-V. parahaemolyticus (8/8) bacteria lawn agar plates. Bacteriophages Vp33, Vp22, Vp21, and Vp02, were found to be able to cause lysis zones on 18.3% (23/126) of V. parahaemolyticus strains agar plates, which included four clinical strains of V. parahaemolyticus. On the other hand, Vp08 and Vp11 were found to be able to cause lysis zones on 34.9% (44/126) of the environmental V. parahaemolyticus strains.\\nTable 3. Host range specificity on different V. parahaemolyticus strains.\\nTransmission Electron Microscopy (TEM)\\nElectron microscopy revealed that the morphological characteristics of bacteriophages Vp33, Vp22, Vp21, and Vp02 were resembled Podoviridae family, and bacteriophages Vp08 and Vp11 were resembled Siphoviridae family (Figures 1A\\u2013F). Bacteriophages Vp33, Vp22, Vp21, and Vp02 were characterized with a hexagonal head diameter between 41.28 \\u00b1 1.83 to 50.00 \\u00b1 3.21 nm and a short stumpy non-contractile tail diameter between 8.72 \\u00b1 2.29 to 12.44 \\u00b1 2.30 nm. Bacteriophages Vp08 and Vp11 were exhibited with an elongated hexagonal head diameter between 79.00 \\u00b1 5.94 \\u00d7 44.98 \\u00b1 1.60 nm and 91.78 \\u00b1 4.11 \\u00d7 49.54 \\u00b1 2.05 nm and a long flexible non-contractile tail diameter between 133.79 \\u00b1 15.98 nm and 142.24 \\u00b1 13.47 nm.\\nFigure 1. Electron micrography of negative stained Vibrio parahaemolyticus bacteriophages and the bar represents 50 nm. (A) Vp33 (head diameter: 44.70 \\u00b1 2.30 nm; tail diameter: 9.22 \\u00b1 0.92 nm); (B) Vp22 (head diameter: 41.28 \\u00b1 1.83 nm; tail diameter: 8.72 \\u00b1 2.29 nm); (C) Vp21 (head diameter: 42.63 \\u00b1 2.07 nm; tail diameter: 9.86 \\u00b1 1.15 nm); (D) Vp02 (head diameter: 50.00 \\u00b1 3.21 nm; tail diameter: 12.44 \\u00b1 2.30 nm); (E) Vp08 (head diameter: 91.78 \\u00b1 4.11 nm \\u00d7 49.54 \\u00b1 2.05 nm; tail diameter: 142.24 \\u00b1 13.47 nm) (F) Vp11 (head diameter: 79.00 \\u00b1 5.94 nm \\u00d7 44.98 \\u00b1 1.60 nm; tail diameter: 133.79 \\u00b1 15.98 nm).\\nBacteriophage Genome Analysis\\nFrom the determination of the nucleic acids, all the bacteriophage nucleic acids were digested by DNase I (1 U/\\u03bcL) after 30 min incubation at 37\\u00b0C. None of the bacteriophage nucleic acids was digested by RNase A (10 mg/mL) after 30 min of incubation at 37\\u00b0C.\\nRestriction Fragment Length Polymorphism (RFLP)\\nFrom the restriction endonuclease digestion, the genomic DNA of all bacteriophages remained intact and unable to be digested by the EcoRI and SalI restriction enzymes. However, the genomic DNA of each bacteriophage treated with HindIII restriction enzyme produced a distinctive RFLP (Figure 2A). HaeIII restriction digestion profiles were also obtained for all bacteriophages except Vp08 and Vp11 (Figure 2B).\\nFigure 2. Lane M: 1 kb DNA ladder (Promega, United States); Lane 1: Vp33; Lane 2: Vp22; Lane 3: Vp21; Lane 4: Vp02; Lane 5: Vp08; Lane 6: Vp11. (A) RFLP patterns of bacteriophages genomic DNA digested with HindIII. (B) RFLP patterns of bacteriophages genomic DNA digested with HaeIII.\\nSDS-PAGE Analysis\\nStructural proteins of bacteriophages were analyzed by SDS-PAGE. Protein profiles of bacteriophage Vp33, Vp22, Vp21, and Vp02 demonstrated a major protein band at around 31 kDa, while bacteriophages Vp08 and Vp11 exhibited a major protein band at around 40 kDa (Figure 3). Bacteriophages Vp33, Vp22, and Vp02 were observed with similar protein profiles of 8 clear protein bands at around 36, 31, 26, 20, 17, 15, 12, and 10 kDa. Bacteriophages Vp08 and Vp11 exhibited similar patterns of protein profiles with seven clear protein bands at around 48, 40, 26, 20, 17, 10, and 5 kDa. Bacteriophage Vp21 only showed two clear protein bands at around 35 and 32 kDa.\\nFigure 3. Protein profiles of bacteriophages. Lane M: ExcelBandTM High Range Protein Marker, PM2610 (SMOBIO, Taiwan); Lane 1: Vp33; Lane 2: Vp22; Lane 3: Vp21; Lane 4: Vp02; Lane 5: Vp08; Lane 6: Vp11.\\nMultiplicity of Infection (MOI)\\nAt an MOI of 1, bacteriophages Vp33, Vp08, and Vp11 exhibited the highest bacteriophage titer of 8.34 \\u00b1 0.08, 10.05 \\u00b1 0.07, and 9.47 \\u00b1 0.15 log PFU/mL, respectively, after 3.5 h incubation at 37\\u00b0C. The MOI ratio of 1 was then determined to be the optimal MOI for the bacteriophages Vp33, Vp08, and Vp11. Bacteriophage Vp22 was observed with an optimal MOI of 0.001 from the highest bacteriophage titer of 9.52 \\u00b1 0.06 log PFU/mL, while bacteriophage Vp02 was found with the optimal MOI of 0.1 from the highest bacteriophage titer of 10.02 \\u00b1 0.11 log PFU/mL. Lastly, the optimal MOI of bacteriophage Vp21 was determined to be 0.01 to 0.001 with the highest bacteriophage titer of 8.22 \\u00b1 0.01 and 8.22 \\u00b1 0.03 log PFU/mL.\\nOne-Step Growth Curve\\nFrom the one-step growth curve, all the bacteriophages were observed with a latent period of 20 min except bacteriophage Vp11 showed a faster latent period of 10 min in comparison to the others (Figures 4A\\u2013F). The average burst size of bacteriophages Vp33, Vp22, Vp21, Vp02, Vp08, and Vp11 estimated from the one-step growth curve were 22, 51, 34, 26, 37, and 17 plaque-forming units (PFU) per cell, respectively.\\nFigure 4. One-step growth curve of bacteriophages. (A) Vp33; (B) Vp22; (C) Vp21; (D) Vp02; (E) Vp08; (F) Vp11.\\nTemperature and pH Stability\\nAll bacteriophages were observed to remain relatively stable at temperatures \\u221220, 25, 37, and 50\\u00b0C, with survival rate ranged from 93.13 to 100.00% (Table 4). At a temperature of 60\\u00b0C, a low survival rate of 31.16 and 49.96% was observed in bacteriophages Vp08 and Vp11, respectively. At a temperature of 70\\u00b0C, all bacteriophages were found inactivated and loss of infectivity. For pH stability, all bacteriophages were observed to be able to withstand a wide range of pH levels from 5 to 11 and found with a percentage of survival of 95.82 to 99.92% (Table 5). Besides, the majority of bacteriophages were found to be resistant at pH3 except that bacteriophages Vp21 and Vp22 were observed with a low survival rate of 47.15 and 52.64%, respectively. At pH2, none of the bacteriophages was survived and recovered after a 2 h period of incubation.\\nTable 4. Temperature stability of bacteriophages.\\nTable 5. pH stability of bacteriophages.\\nIn vitro Lytic Activity\\nAll bacteriophages, including two bacteriophage cocktail preparations, were demonstrated to have significant (P < 0.05) in vitro lytic activity after 2 h incubation at 37\\u00b0C (Table 6). The efficiency of single bacteriophage suspensions as well as the bacteriophage cocktails were also observed to increase significantly over time. The total reduction was reported to be in the range of 2.85 \\u00b1 0.07 to 4.60 \\u00b1 0.08 log CFU/mL, for individual bacteriophage suspension. Bacteriophage cocktails A and B achieved a better total reduction, with 3.69 \\u00b1 0.49 and 4.81 \\u00b1 0.14 log CFU/mL, respectively.\\nTable 6. Viable count (log CFU/mL) of V. parahaemolyticus after 0, 2, 4, 6, and 24 h incubation at 37\\u00b0C with bacteriophage.\\nDiscussion\\nThe majority of studies were focused on the isolation of bacteriophages from environmental sources such as lake sediment, poultry farm, seawater, sewage, and soil (Salifu et al., 2013; Shende et al., 2017; Huang et al., 2018; Akhwale et al., 2019; Kim et al., 2019; Tang et al., 2019; Necel et al., 2020; Sevilla-Navarro et al., 2020). Reports of the isolation of these bacteriophages presence on fresh food samples were relatively limited in extent. According to Greer (2005), the successful isolation of bacteriophages from food samples was reliant on the occurrence of the target bacteria which was present in the food samples in relatively high concentrations.\\nShrimp samples purchased from the wet markets were found to be the primary source of V. parahaemolyticus bacteriophages in this study. Similarly, several studies reported the isolation of Vibrio spp. bacteriophages from a similar source such as shrimp aquaculture environments (Alagappan et al., 2010; Stalin and Srinivasan, 2017; Matamp and Bhat, 2020). For instance, Alagappan et al. (2010) reported that five bacteriophages with the ability to infect V. parahaemolyticus MTCC-451 strain were isolated from the shrimp pond. Stalin and Srinivasan (2017) reported that bacteriophage VVP1 capable of infecting V. parahaemolyticus as well as V. alginolyticus strains were isolated from the water suspended sediment samples collected from the shrimp grow-out pond. Matamp and Bhat (2020) also reported that bacteriophages \\u233dVP-1 which able to lyse five strains of V. alginolyticus and 1 strain of V. parahaemolyticus were isolated from the shrimp pond water samples.\\nHost specificity of bacteriophage is one of the vital aspects which should be considered in a bacteriophage application. The specificity of bacteriophages isolated in this study were found to be of a narrow host range that infected only V. parahaemolyticus strains. A narrow host-specific bacteriophage seemed an attractive feature, especially when applying it in the gastrointestinal tract to target specific host bacteria as the narrow host range of the bacteriophage would not affect other endogenous bacteria (Viazis et al., 2011; Drulis-Kawa et al., 2012). Besides, different types of narrow host range bacteriophages can be mixed in a cocktail preparation for a more effective in vitro control of pathogenic bacteria (Tanji et al., 2004; Mapes et al., 2016; Bai et al., 2019).\\nBoth the Podoviridae and Siphoviridae families are classified under the same order of Caudovirales. A hexagonal head (41.28 to 50.00 nm) and a short stumpy non-contractile tail (8.72 to 12.44 nm) which resembled the Podoviridae family was observed in the negative staining of bacteriophages Vp33, Vp22, Vp21, and Vp02 under TEM. Similarly, Zhang et al. (2018) reported that a V. parahaemolyticus bacteriophage vB_VpaS_OMN isolated from the seafood sample belonged to the Podoviridae family and was observed to have a 55 nm isometric capsid and a short tail of about 16 nm. On the other hand, the electron micrographs of bacteriophages Vp08 and Vp11 showed an elongated hexagonal head (79.00 \\u00d7 44.98 to 91.78 \\u00d7 49.54 nm) and a long flexible non-contractile tail (133.79 to 142.24 nm). These morphological characteristics resemble those of the Siphoviridae family. A recent study by Yang et al. (2020) reported a similar finding that two V. parahaemolyticus bacteriophages, namely vB_VpS_BA3 and vB_VpS_CA8, isolated from the sewage were Siphoviridae bacteriophage with an icosahedral head (70 \\u00d7 55 nm) and a long non-contractile tail (125 to 130 nm).\\nDion et al. (2020) reported that double-stranded DNA (dsDNA) tailed bacteriophages accounted for the majority of bacteriophages reported in the public database. This finding supports with Dion et al. (2020) statement as all bacteriophages in this study encapsulate a dsDNA genome. From the restriction endonuclease digestion, some genetic variation or polymorphisms were observed among bacteriophages. Distinctive restriction digestion profiles between each bacteriophage isolates were observed after the genome of bacteriophages was digested by the HindIII and HaeIII restriction enzymes. From the restriction endonuclease digestion profiles, it can be concluded that all bacteriophage isolates analyzed in this study present some variation in their genetic properties.\\nA prominent protein band at around 31 kDa was identified by SDS-PAGE analysis in all Podoviridae bacteriophages, Vp33, Vp22, Vp21, and Vp02. In contrast, bacteriophages Vp08 and Vp11 showed a prominent protein band at around 40 kDa. This prominent protein was assumed to be the major capsid protein of bacteriophage. In other studies, Drulis-Kawa et al. (2012) revealed a 39 kDa major capsid protein in the Klebsiella pneumoniae podovirus bacteriophage, and this protein structure was found highly homologous (78%) to the 37.8 kDa Vibrio bacteriophage VP93 capsid protein. Baudoux et al. (2012) revealed that a Vibrio siphovirus bacteriophage SIO-2 was identified with the presence of a major capsid protein with a molecular mass of 29 kDa. Other protein bands may indicate the presence of putative tail fiber protein (32.7 kDa), putative scaffolding protein (30.1 kDa), putative internal virion protein B (20.4 kDa), hypothetical and/or unknown proteins of Vibrio bacteriophage (Drulis-Kawa et al., 2012; Hu et al., 2020).\\nBacteriophages rely on the chance of meeting with bacteria to infect and kill bacteria (Huff et al., 2006). The MOI refers to the ratio of the virus particles to the host cells. In this study, the optimal MOI for bacteriophage propagation was determined to be 0.001 to 1. Different bacteriophages may infect and kill bacteria differently, thus having different optimal MOIs. For example, Yang et al. (2020) reported that two V. parahaemolyticus lytic bacteriophages, vB_VpS_BA3 and vB_VpS_CA8, were tested with optimal MOI of 0.1 from the maximum bacteriophage titer produced. A lytic bacteriophage PhVh6 showing the ability to lyse V. alginolyticus, V. harveyi, and V. parahaemolyticus was tested with an optimal MOI of 1 (Ibrahim et al., 2017).\\nFrom the one-step growth curve, the latent period of bacteriophages was estimated to be around 10 to 20 min and the average burst size was anticipated to be approximately 17 to 51 PFU/cell. Previously, vibriophage BA3 and CA8 were estimated as having a latent period of around 20 to 30 min and a burst size of about 100 to 180 PFU/cell (Yang et al., 2020). Another lytic bacteriophage VP06 of V. parahaemolyticus was estimated to have a latent period of approximately 30 min and 60 PFU per infected bacterial cell (Wong et al., 2019). Lal et al. (2016) also reported that a V. parahaemolyticus bacteriophage VpKK5 exhibited a latent period of 36 min and a burst size of 180 PFU/cell. From the results, bacteriophage isolates in this study demonstrated a shorter latent period but a smaller burst size than other reported vibrio bacteriophages.\\nPhysico-chemical factors such as temperature and pH level can affect the effectiveness of bacteriophage applications against pathogenic bacteria (Ly-Chatain, 2014). For example, higher temperatures may result in irreversible damage or denaturation of the virus particles (Ahmadi et al., 2017). The concentration of hydrogen ions in an acidic solution can cause a decline in bacteriophage concentration due to the aggregation process (Langlet et al., 2007; Jo\\u0144czyk et al., 2011). In the current study, the effect of temperature on bacteriophage stability revealed that all bacteriophages remained relatively stable at temperatures ranging from \\u221220 to 50\\u00b0C. At 60\\u00b0C, bacteriophages Vp33, Vp22, Vp21, and Vp02 which belong to the Podoviridae family showed continuing stability. In contrast, bacteriophages Vp08 and Vp11 which belong to the Siphoviridae family showed a low survival rate at 60\\u00b0C. Incubation at 70\\u00b0C showed total inactivation of all bacteriophages. Previous studies reported a similar finding, in which V. parahaemolyticus bacteriophage VVP1 was found to be stable at temperatures 4 to 50\\u00b0C but completely inactive at 60\\u00b0C (Stalin and Srinivasan, 2017). Investigation of the effect of pH on bacteriophage stability revealed that all bacteriophages sustained well at pH ranges of 5 to 11. At pH3, bacteriophages Vp22 and Vp21 were observed with a low survival percentage. At pH2, none of the bacteriophages survived this highly acidic environment. Such characteristics align with results reported by Jun et al. (2014) in which a bacteriophage pVp-1, showed infectivity for multiple\\u2013antibiotic-resistant V. parahaemolyticus and V. vulnificus, was observed to be stable over a wide range of pH levels of 3 to 11.\\nOverall, the bacteriophage suspensions achieved an average log reduction of V. parahaemolyticus in the range of 2.85 \\u00b1 0.07 to 4.81 \\u00b1 0.14. Bacteriophage cocktail preparations were observed to have a better in vitro lytic activity as compared to a single bacteriophage suspension. However, other attributes such as competition between each V. parahaemolyticus strain and production of antagonistic compounds by V. parahaemolyticus strains that may exhibit antibacterial activity were excluded in the results and should be considered in the further study. Bacteriophage cocktail A achieved a total reduction rate of 3.69 \\u00b1 0.49 log CFU/mL, compared to its individual bacteriophage suspension with a reduction rate between 2.85 \\u00b1 0.07 and 3.38 \\u00b1 0.26 log CFU/mL. Bacteriophage cocktail B achieved a total reduction rate of 4.81 \\u00b1 0.14 log CFU/mL, compared to its individual bacteriophage suspension with a reduction rate between 4.22 \\u00b1 0.26 and 4.60 \\u00b1 0.08 log CFU/mL. According to these results, the application of a bacteriophage cocktail instead of a single bacteriophage suspension could therefore result in an improved reduction in the growth of V. parahaemolyticus.\\nMateus et al. (2014) stated that the use of bacteriophage cocktails mixed with two or three bacteriophages can increase the efficiency to control and inactivate the growth of V. parahaemolyticus better than the single bacteriophage suspensions. For example, Mateus et al. (2014) reported that the application of bacteriophage cocktails mixed with the VP-1 and VP-2 bacteriophages achieved a maximum of 4.0 log of V. parahaemolyticus inactivation after 2 h of incubation, compared to the use of VP-1 and VP-2 bacteriophage alone, with a reduction rate of only 0.8 log. Similarly, Ren et al. (2019) reported that a bacteriophage cocktail which consisted of PVP1 and PVP2 bacteriophages was superior in preventing V. parahaemolyticus infections in sea cucumbers than the use of single bacteriophages. It was also recently demonstrated that the benefit of bacteriophage cocktails can help to overcome the development of bacteriophage-resistant pathogen strains (Yang et al., 2020). For instance, Kim et al. (2020) demonstrated that a bacteriophage cocktail consists of three types of V. coralliilyticus-specific bacteriophages with greater efficiency against multiple-antibiotic-resistant (MAR) and phage-resistant (PR) V. coralliilyticus infection in oyster larvae. Yen et al. (2017) reported that a bacteriophage cocktail that targets different receptors on V. cholerae would reduce the emergence of bacteriophage-resistant V. cholerae isolates. Hence, bacteriophage cocktails consisting of multiple bacteriophage suspensions are more effective in the lytic ability.\\nConclusion\\nVibrio parahaemolyticus specific bacteriophages can be readily recovered from seafood samples. Bacteriophage Vp33, Vp22, Vp21, Vp02, Vp08, and Vp11 isolated from the seafood samples exhibited narrow host range and lytic activity against V. parahaemolyticus strains only. Characterization of bacteriophages in this study facilitates a better understanding of the physiological and biological features of bacteriophages. The results of in vitro lytic activity revealed the usability of bacteriophages as biocontrol agents for the control of V. parahaemolyticus.\\nData Availability Statement\\nThe raw data supporting the conclusions of this article will be made available by the authors, without undue reservation.\\nAuthor Contributions\\nNA-M, NJ, TT, and EL provided assistance and guidance throughout the research. YR, HaH, HiH, and SR were the mentor in the research study and assist in manuscript checking. All authors contributed to the article and approved the submitted version.\\nFunding\\nThis work was supported by the Fundamental Research Grant Scheme (FRGS) of the Ministry of Higher Education (MOHE), Malaysia (FRGS/1/2018/STG05/UPM/01/7; 01-01-18-2015FR) and Grant Putra IPS from Universiti Putra Malaysia (GP-IPS 9438703). Publication of this work was funded by UPM Journal Publication Fund/Dana Penerbitan Jurnal UPM from Research Management Centre (RMC) of Universiti Putra Malaysia.\\nConflict of Interest\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nReferences\\nAdams, M. J., Lefkowitz, E. J., King, A. M., Harrach, B., Harrison, R. L., Knowles, N. J., et al. (2017). 50 years of the international committee on taxonomy of viruses: progress and prospects. Arch. Virol. 162, 1441\\u20131446. doi: 10.1007/s00705-016-3215-y\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAdriaenssens, E. M., Sullivan, M. B., Knezevic, P., van Zyl, L. J., Sarkar, B. L., Dutilh, B. E., et al. (2020). Taxonomy of prokaryotic viruses: 2018-2019: update from the ICTV bacterial and archaeal viruses subcommittee. Arch. Virol. 165, 1253\\u20131260. doi: 10.1007/s00705-020-04577-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAhmad, A., Brumble, L., and Maniaci, M. (2013). Vibrio parahaemolyticus induced necrotising fasciitis: an atypical organism causing an unusual presentation. Case Rep. Infect. Dis. 2013:216854. doi: 10.1155/2013/216854\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAhmadi, H., Radford, D., Kropinski, A. M., Lim, L. T., and Balamurugan, S. (2017). Thermal-stability and reconstitution ability of Listeria phages P100 and A511. Front. Microbiol. 8:2375. doi: 10.3389/fmicb.2017.02375\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAkhwale, J. K., Rohde, M., Rohde, C., Bunk, B., Spr\\u00f6er, C., Boga, H. I., et al. (2019). Isolation, characterisation and analysis of bacteriophages from the haloalkaline lake Elmenteita, Kenya. PLoS One 14:e0215734. doi: 10.1371/journal.pone.0215734\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAlagappan, K. M., Deivasigamani, B., Somasundaram, S. T., and Kumaran, S. (2010). Occurrence of Vibrio parahaemolyticus and its specific phages from shrimp ponds in east coast of India. Curr. Opin. Microbiol. 61, 235\\u2013240. doi: 10.1007/s00284-010-9599-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBai, J., Jeon, B., and Ryu, S. (2019). Effective inhibition of Salmonella Typhimurium in fresh produce by a phage cocktail targeting multiple host receptors. J. Food Microbiol. 77, 52\\u201360. doi: 10.1016/j.fm.2018.08.011\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBaudoux, A. C., Hendrix, R. W., Lander, G. C., Bailly, X., Podell, S., Paillard, C., et al. (2012). Genomic and functional analysis of Vibrio phage SIO-2 reveals novel insights into ecology and evolution of marine siphoviruses. Environ. Microbiol. 14, 2071\\u20132086. doi: 10.1111/j.1462-2920.2011.02685.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBeck, N. K., Callahan, K., Nappier, S. P., Kim, H., Sobsey, M. D., and Meschke, J. S. (2009). Development of a spot-titer culture assay for quantifying bacteria and viral indicators. J. Rapid Methods Autom. Microbiol. 17, 455\\u2013464. doi: 10.1111/j.1745-4581.2009.00182.x\\nCrossRef Full Text | Google Scholar\\nBhoopong, P., Palittapongarnpim, P., Pomwised, P., Kiatkittipong, A., Kamruzzaman, M., Nakaguchi, Y., et al. (2007). Variability in properties of Vibrio parahaemolyticus strains isolated from single patients. J. Clin. Microbiol. 45, 1544\\u20131550. doi: 10.1128/JCM.02371-06\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBiswas, B., Adhya, S., Washart, P., Paul, B., Trostel, A. N., Powell, B., et al. (2002). Bacteriophage therapy rescues mice bacteremic from a clinical isolate of vancomycin-resistant Enterococcus faecium. Infect. Immun. 70, 204\\u2013210. doi: 10.1128/iai.70.1.204-210.2002\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBonilla, N., Rojas, M. I., Netto Flores, Cruz, G., Hung, S. H., Rohwer, F., et al. (2016). Phage on tap-a quick and efficient protocol for the preparation of bacteriophage laboratory stocks. PeerJ. 4:e2261. doi: 10.7717/peerj.2261\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChen, Y., Chen, X., Yu, F., Wu, M., Wang, R., Zheng, S., et al. (2016). Serology, virulence, antimicrobial susceptibility and molecular characteristics of clinical Vibrio parahaemolyticus strains circulating in southeastern China from 2009 to 2013. Clin. Microbiol. Infect. 22, 258.e9\\u2013258.e16. doi: 10.1016/j.cmi.2015.11.003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nComeau, A. M., Hatfull, G. F., Krisch, H. M., Lindell, D., Mann, N. H., and Prangishvili, D. (2008). Exploring the prokaryotic virosphere. Res. Microbiol. 159, 306\\u2013313. doi: 10.1016/j.resmic.2008.05.001\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDrulis-Kawa, Z., Majkowska-Skrobek, G., Maciejewska, B., Delattre, A. S., and Lavigne, R. (2012). Learning from bacteriophages - advantages and limitations of phage and phage-encoded protein applications. Curr. Protein Pept. Sci. 13, 699\\u2013722. doi: 10.2174/138920312804871193\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nElmahdi, S., DaSilva, L. V., and Parveen, S. (2016). Antibiotic resistance of Vibrio parahaemolyticus and Vibrio vulnificus in various countries: a review. Food Microbiol. 57, 128\\u2013134. doi: 10.1016/j.fm.2016.02.008\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGuan, S. M., Nagata, H., Maeda, K., Kuboniwa, M., Minamino, N., and Shizukuishi, S. (2004). Purification and characterization of a hemoglobin-binding outer membrane protein of Prevotella intermedia. FEMS Microbiol. Lett. 235, 333\\u2013339. doi: 10.1111/j.1574-6968.2004.tb09607.x\\nCrossRef Full Text | Google Scholar\\nHagens, S., and Loessner, M. J. (2007). Application of bacteriophages for detection and control of foodborne pathogens. Appl. Microbiol. Biotechnol. 76, 513\\u2013519. doi: 10.1007/s00253-007-1031-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHong, Y., Pan, Y., and Ebner, P. D. (2014). Meat science and muscle biology symposium: development of bacteriophage treatments to reduce Escherichia coli O157:H7 contamination of beef products and produce1. J. Anim. Sci. 92, 1366\\u20131377. doi: 10.2527/jas.2013-7272\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHu, M., Zhang, H., Gu, D., Ma, Y., and Zhou, X. (2020). Identification of a novel bacterial receptor that binds tail tubular proteins and mediates phage infection of Vibrio parahaemolyticus. Emerg. Microbes Infect. 9, 855\\u2013867. doi: 10.1080/22221751.2020.1754134\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHuang, C., Virk, S. M., Shi, J., Zhou, Y., Willias, S. P., Morsy, M. K., et al. (2018). Isolation, characterisation, and application of bacteriophage LPSE1 against Salmonella enterica in ready to eat (RTE) foods. Front. Microbiol. 9:1046. doi: 10.3389/fmicb.2018.01046\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHuff, W. E., Huff, G. R., Rath, N. C., and Donoghue, A. M. (2006). Evaluation of the influence of bacteriophage titer on the treatment of colibacillosis in broiler chickens. Poult. Sci. 85, 1373\\u20131377. doi: 10.1093/ps/85.8.1373\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nIbrahim, W. N. W., Aznan, A. S., Saari, N. A., Leong, L. K., Musa, N., Razzak, L. A., et al. (2017). In-vitro characterization of lytic bacteriophage PhVh6 as potential biocontrol agent against pathogenic Vibrio harveyi. Aquac. Aquar. Conserv. Legis. 10, 64\\u201376.\\nGoogle Scholar\\nJo\\u0144czyk, E., K\\u0142ak, M., Mi\\u0119dzybrodzki, R., and G\\u00f3rski, A. (2011). The influence of external factors on bacteriophages - review. Folia Microbiol. 56, 191\\u2013200. doi: 10.1007/s12223-011-0039-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJones, J. L., Ludeke, C. H., Bowers, J. C., Garrett, N., Fischer, M., Parsons, M. B., et al. (2012). Biochemical, serological, and virulence characterization of clinical and oyster Vibrio parahaemolyticus isolates. J. Clin. Microbiol. 50, 2343\\u20132352. doi: 10.1128/JCM.00196-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJun, J. W., Shin, T. H., Kim, J. H., Shin, S. P., Han, J. E., Heo, G. J., et al. (2014). Bacteriophage therapy of a Vibrio parahaemolyticus infection caused by a multiple-antibiotic\\u2013resistant O3:K6 pandemic clinical strain. J. Infect. Dis. 210, 72\\u201378. doi: 10.1093/infdis/jiu059\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKalatzis, P. G., Castillo, D., Katharios, P., and Middelboe, M. (2018). Bacteriophage interactions with marine pathogenic vibrios: implications for phage therapy. Antibiot. Basel 7:15. doi: 10.3390/antibiotics7010015\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKang, C. H., Shin, Y., Jang, S., Yu, H., Kim, S., An, S., et al. (2017). Characterisation of Vibrio parahaemolyticus isolated from oysters in Korea: resistance to various antibiotics and prevalence of virulence genes. Mar. Pollut. Bull. 118, 261\\u2013266. doi: 10.1016/j.marpolbul.2017.02.070\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKim, H. J., Jun, J. W., Giri, S. S., Kim, S. G., Kim, S. W., Kwon, J., et al. (2020). Bacteriophage cocktail for the prevention of multiple-antibiotic-resistant and mono-phage-resistant Vibrio coralliilyticus infection in pacific oyster (Crassostrea gigas) larvae. Pathogens 9:813. doi: 10.3390/pathogens9100831\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKim, S. G., Jun, J. W., Giri, S. S., Yun, S., Kim, H. J., Kim, S. W., et al. (2019). Isolation and characterisation of pVa-21, a giant bacteriophage with anti-biofilm potential against Vibrio alginolyticus. Sci. Rep. 9:6284. doi: 10.1038/s41598-019-42681-1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKropinski, A. M., Mazzocco, A., Waddell, T. E., Lingohr, E., and Johnson, R. P. (2009). \\u201c\\u201cEnumeration of bacteriophages by double agar overlay plaque assay,\\u201d,\\u201d in Bacteriophages: Methods in Molecular Biology, eds M. R. Clokie and A. M. Kropinski (Totowa, NJ: Humana Press), 69\\u201376.\\nGoogle Scholar\\nKrupovic, M., Dutilh, B. E., Adriaenssens, E. M., Wittmann, J., Vogensen, F. K., Sullivan, M. B., et al. (2016). Taxonomy of prokaryotic viruses: update from the ICTV bacterial and archaeal viruses subcommittee. Arch. Virol. 161, 1095\\u20131099. doi: 10.1007/s00705-015-2728-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLal, T. M., Sano, M., and Ransangan, J. (2016). Genome characterization of a novel vibriophage VpKK5 (Siphoviridae) specific to fish pathogenic strain of Vibrio parahaemolyticus. J. Basic Microbiol. 56, 872\\u2013888. doi: 10.1002/jobm.201500611\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLanglet, J., Gaboriaud, F., and Gantzer, C. (2007). Effects of pH on plaque forming unit counts and aggregation of MS2 bacteriophage. J. Appl. Microbiol. 103, 1632\\u20131638. doi: 10.1111/j.1365-2672.2007.03396.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLe, T. S., Southgate, P. C., O\\u2019Connor, W., Abramov, T., Shelley, D. V., Vu, S., et al. (2020a). Use of bacteriophages to control Vibrio contamination of microalgae used as a food source for oyster larvae during hatchery culture. Curr. Microbiol. 77, 1811\\u20131820. doi: 10.1007/s00284-020-01981-w\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLe, T. S., Southgate, P. C., O\\u2019Connor, W., Vu, S. V., and Kurtb\\u00f6ke, D. \\u0130 (2020b). Application of Bacteriophages to control Vibrio alginolyticus contamination in oyster (Saccostrea glomerata) larvae. Antibiotics 9:415. doi: 10.3390/antibiotics9070415\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLi, Y., Xie, X., Shi, X., Lin, Y., Qiu, Y., Mou, J., et al. (2014). Vibrio parahaemolyticus, southern coastal region of China, 2007-2012. Emerg. Infect. Dis. 20, 685\\u2013688. doi: 10.3201/eid2004.130744\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLin, D. M., Koskella, B., and Lin, H. C. (2017). Phage therapy: an alternative to antibiotics in the age of multi-drug resistance. World J. Gastrointest. Pharmacol. Ther. 8, 162\\u2013173. doi: 10.4292/wjgpt.v8.i3.162\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLiu, A., Liu, Y., Peng, L., Cai, X., Shen, L., Duan, M., et al. (2020). Characterization of the narrow-spectrum bacteriophage LSE7621 towards Salmonella Enteritidis and its biocontrol potential on lettuce and tofu. LWT Food Sci. Technol. 118:108791. doi: 10.1016/j.lwt.2019.108791\\nCrossRef Full Text | Google Scholar\\nLiu, W., Ye, D., Luo, Z., Yu, Y., and Gong, T. (2013). Isolation and characterisation of five lytic bacteriophages infecting a Vibrio strain closely related to Vibrio owensii. FEMS Microbiol. Lett. 348, 112\\u2013119. doi: 10.1111/1574-6968.12277\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMapes, A. C., Trautner, B. W., Liao, K. S., and Ramig, R. F. (2016). Development of expanded host range phage active on biofilms of multi-drug resistant Pseudomonas aeruginosa. Bacteriophage 6:e1096995. doi: 10.1080/21597081.2015.1096995\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMatamp, N., and Bhat, S. G. (2020). Genome characterisation of novel lytic Myoviridae bacteriophage \\u233dVP-1 enhances its applicability against MDR-biofilm-forming Vibrio parahaemolyticus. Arch. Virol. 165, 387\\u2013396. doi: 10.1007/s00705-019-04493-6\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMateus, L., Costa, L., Silva, Y. J., Pereira, C., Cunha, A., and Almeida, A. (2014). Efficiency of phage cocktails in the inactivation of Vibrio in aquaculture. Aquaculture 42, 167\\u2013173. doi: 10.1016/j.aquaculture.2014.01.001\\nCrossRef Full Text | Google Scholar\\nNecel, A., Bloch, S., Nejman-Fale\\u0144czyk, B., Grabski, M., Topka, G., Dydecka, A., et al. (2020). Characterization of a bacteriophage, vB_Eco4M-7, that effectively infects many Escherichia coli O157 strains. Sci. Rep. 10:3743. doi: 10.1038/s41598-020-60568-4\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nParia, P., Kunal, S. P., Behera, B. K., Mohapatra, P. K. D., Das, A., Parida, P. K., et al. (2019). Molecular characterisation and genetic diversity study of Vibrio parahaemolyticus isolated from aquaculture farms in India. Aquaculture 509, 104\\u2013111. doi: 10.1016/j.aquaculture.2019.04.076\\nCrossRef Full Text | Google Scholar\\nPazhana, G. P., Bhowmik, S. K., Ghosh, S., Guin, S., Dutta, S., Rajendran, K., et al. (2014). Trends in the epidemiology of pandemic and non-pandemic strains of Vibrio parahaemolyticus isolated from diarrheal patients in Kolkata, India. PLoS Negl. Trop. Dis. 8:e2815. doi: 10.1371/journal.pntd.0002815\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRen, H., Li, Z., Xu, Y., Wang, L., and Li, X. (2019). Protective effectiveness of feeding phage cocktails in controlling Vibrio parahaemolyticus infection of sea cucumber Apostichopus japonicus. Aquaculture 503, 322\\u2013329. doi: 10.1016/j.aquaculture.2019.01.006\\nCrossRef Full Text | Google Scholar\\nSalifu, S. P., Casey, S. A. C., and Foley, S. (2013). Isolation and characterisation of soilborne virulent bacteriophages infecting the pathogen Rhodococcus equi. J. Appl. Microbiol. 114, 1625\\u20131633. doi: 10.1111/jam.12194\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSchramlov\\u00e1, J., Arientov\\u00e1, S., and Hul\\u00ednsk\\u00e1, D. (2010). The role of electron microscopy in the rapid diagnosis of viral infections\\u2013review. Folia Microbiol. 55, 88\\u2013101. doi: 10.1007/s12223-010-0015-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSevilla-Navarro, S., Catal\\u00e1-Gregori, P., Garc\\u00eda, C., Cort\\u00e9s, V., and Marin, C. (2020). Salmonella Infantis and Salmonella Enteritidis specific bacteriophages isolated form poultry faeces as a complementary tool for cleaning and disinfection against Salmonella. Comp. Immunol. Microbiol. Infect. Dis. 68:101405. doi: 10.1016/j.cimid.2019.101405\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nShende, R. K., Hirpurkar, S. D., Sannat, C., Rawat, N., and Pandey, V. (2017). Isolation and characterisation of bacteriophages with lytic activity against common bacterial pathogens. Vet. World 10, 973\\u2013978. doi: 10.14202/vetworld.2017.973-978\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nStalin, N., and Srinivasan, P. (2017). Efficacy of potential phage cocktails against Vibrio harveyi and closely related Vibrio species isolated from shrimp aquaculture environment in the south east coast of India. Vet. Microbiol. 207, 83\\u201396. doi: 10.1016/j.vetmic.2017.06.006\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTan, C. W., Malcolm, T., Kuan, C. H., Thung, T. Y., Chang, W. S., Loo, Y. Y., et al. (2017). Prevalence and antimicrobial susceptibility of Vibrio parahaemolyticus isolated from short mackerels (Rastrelliger brachysoma) in Malaysia. Front. Microbiol. 8:1087. doi: 10.3389/fmicb.2017.01087\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTang, F., Zhang, P., Zhang, Q., Xue, F., Ren, J., Sun, J., et al. (2019). Isolation and characterisation of a broad-spectrum phage of multiple drug resistant Salmonella and its therapeutic utility in mice. Microb. Pathog. 126, 193\\u2013198. doi: 10.1016/j.micpath.2018.10.042\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTanji, Y., Shimada, T., Yoichi, M., Miyanaga, K., Hori, K., and Unno, H. (2004). Toward rational control of Escherichia coli O157:H7 by a phage cocktail. Appl. Microbiol. Biotechnol. 64, 270\\u2013274. doi: 10.1007/s00253-003-1438-9\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTena, D., Arias, M., Alvarez, B. T., Mauleon, C., Jimenez, M. P., and Bisquert, J. (2010). Fulminant necrotising fasciitis due to Vibrio parahaemolyticus. J. Med. Microbiol. 59, 235\\u2013238. doi: 10.1099/jmm.0.014654-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nThung, T. Y., Premarathne, J. M. K. J. K., Chang, W. S., Loo, Y. Y., Chin, Y. Z., Kuan, C. H., et al. (2017). Use of a lytic bacteriophage to control Salmonella Enteritidis in retail food. LWT Food Sci. Technol. 78, 222\\u2013225. doi: 10.1016/j.lwt.2016.12.044\\nCrossRef Full Text | Google Scholar\\nTopka, G., Bloch, S., Nejman-Fale\\u0144czyk, B., G\\u0105sior, T., Jurczak-Kurek, A., Necel, A., et al. (2019). Characterization of bacteriophage vB-EcoS-95, isolated from urban sewage and revealing extremely rapid lytic development. Front. Microbiol. 9:3326. doi: 10.3389/fmicb.2018.03326\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nViazis, S., Akhtar, M., Feirtag, J., and Brabban, A. D. (2011). Isolation and characterisation of lytic bacteriophages against enterohaemorrhagic Escherichia coli. J. Appl. Microbiol. 110, 1323\\u20131331. doi: 10.1111/j.1365-2672.2011.04989.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWang, J., Hu, B. E. I., Xu, M., Yan, Q. U. N., Liu, S., Zhu, X., et al. (2006). Therapeutic effectiveness of bacteriophages in the rescue of mice with extended spectrum \\u00df-lactamase-producing Escherichia coli bacteremia. Int. J. Mol. Med. 17, 347\\u2013355. doi: 10.3892/ijmm.17.2.347\\nCrossRef Full Text | Google Scholar\\nWang, Y., Barton, M., Elliott, L., Li, X., Abraham, S., O\\u2019Dea, M., et al. (2017). Bacteriophage therapy for the control of Vibrio harveyi in greenlip abalone (Haliotis laevigata). Aquaculture 473, 251\\u2013258. doi: 10.1016/j.aquaculture.2017.01.003\\nCrossRef Full Text | Google Scholar\\nWatanabe, R., Matsumoto, T., Sano, G., Ishii, Y., Tateda, K., Sumiyama, Y., et al. (2007). Efficacy of bacteriophage therapy against gut-derived sepsis caused by Pseudomonas aeruginosa in mice. Antimicrob. Agents Chemother. 51, 446\\u2013452. doi: 10.1128/AAC.00635-06\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWong, H. C., Wang, T. Y., Yang, C. W., Tang, C. T., Ying, C., Wang, C. H., et al. (2019). Characterization of a lytic vibriophage VP06 of Vibrio parahaemolyticus. Res. Microbiol. 170, 13\\u201323. doi: 10.1016/j.resmic.2018.07.003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYang, M., Liang, Y., Huang, S., Zhang, J., Wang, J., Chen, H., et al. (2020). Isolation and characterization of the novel phages vB_VpS_BA3 and vB_VpS_CA8 for lysing Vibrio parahaemolyticus. Front. Microbiol. 11:259. doi: 10.3389/fmicb.2020.00259\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYen, M., Cairns, L. S., and Camilli, A. (2017). A cocktail of three virulent bacteriophages prevents Vibrio cholerae infection in animal models. Nat. Commun. 8:14187. doi: 10.1038/ncomms14187\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nZhang, H., Yang, Z., Zhou, Y., Bao, H., Wang, R., Li, T., et al. (2018). Application of a phage in decontaminating Vibrio parahaemolyticus in oysters. Int. J. Food Microbiol. 275, 24\\u201331. doi: 10.1016/j.ijfoodmicro.2018.03.027\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.01533\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.01533/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:20.995Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.01533\",\n",
      "            \"title\": \"Frontiers | A Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia\",\n",
      "            \"description\": \"Erwinia amylovora is a plant pathogen from the Erwiniaceae family and a causative agent of the devastating agricultural disease fire blight. Here we characterize eight lytic bacteriophages of E. amylovora that we isolated from the Wasatch front (Utah, USA) that are highly similar to vB_EamM_Ea35-70 which was isolated in Ontario, Canada. With the genome size ranging from 271-275 kb, this is a novel jumbo family of bacteriophages. These jumbo bacteriophages were further characterized through genomic and proteomic comparison, mass spectrometry, host range and burst size. Their proteomes are highly unstudied, with over 200 putative proteins with no known homologs. The production of 27 of these putative proteins was confirmed by mass spectrometry analysis. These bacteriophages appear to be most similar to bacteriophages that infect Pseudomonas and Ralstonia rather than Enterobacteriales bacteria by protein similarity, however we were only able to detect infection of Erwinia and the closely related strains of Pantoea. \",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nWhitman et al. (1998) estimated that there are approximately 5 \\u00d7 1030 bacteria on earth, which is more than the number of plants and animals combined. Most, or likely all, bacteria are subject to infection by one or more viruses or \\u201cbacteriophages,\\u201d making bacteriophages the most common and diverse biological entity at an estimated 1032 (Bergh et al., 1989; Wommack and Colwell, 2000; Hambly and Suttle, 2005). Bacteriophages were likely first reported in 1896, when Ernest Hanbury Hankin discovered antibacterial activity against cholera in the waters of two large rivers in India, the Ganges and Yamuna (Abedon et al., 2011). They were independently characterized and named in the 1900s by bacteriologist Twort (1915, 1936) and microbiologist D\\u2019Herelle (1917, 2007), Keen (2015). During the infection process, bacteriophages can transfer foreign DNA to their host (including virulence factors), integrate into the host genome, and/or kill their host through cell lysis (Chen and Novick, 2009). The sheer number of bacteriophages combined with their clear evolutionary influence makes them an important target for understanding the ecology and evolution of bacteria, including pathogenic strains (Bollback and Huelsenbeck, 2009; Boyd, 2012). In addition, their specificity, genomic plasticity, and rapid multiplication rates make them a potential weapon to treat bacterial infections (Sulakvelidze, 2005; Shivaswamy et al., 2015).\\nOne such bacterial infection caused by a phytopathogen Erwinia amylovora (Zwet and Keil, 1979) is called fire blight and mainly affects ornamental plants of the Rosaceae family. The symptoms of the infected tissues include wilting, ooze production and death of blossoms, shoots branches and entire trees (Thomson, 2000). We have recently isolated and characterized twenty eight bacteriophages that infect E. amylovora (Esplin et al., 2017; Sharma et al., 2018). Out of these 28, there is a distinct group of eight highly related bacteriophages: vB_EamM_Special G (Special G), vB_EamM_Simmy50 (Simmy50), vB_EamM_RAY (RAY), vB_EamM_Deimos-Minion (Deimos-Minion or DM), vB_EamM_Bosolaphorus (Bosolaphorus), vB_EamM_Desertfox (Desertfox), vB_EamM_MadMel (MadMel), and vB_EamM_Mortimer (Mortimer) very similar to Erwinia bacteriophage Ea35-70 which was isolated in Ontario, Canada (Yagubi et al., 2014). These nine bacteriophages were recently added as the Agrican357virus genus of bacteriophages by the ICTV (Kuhn et al., 2013; Adams et al., 2017) and are considered jumbo bacteriophages due to their large genome (>200 kb) and particle size (Yuan and Gao, 2017).\\nAs reviewed in 2017, jumbo bacteriophages have diverse genome sizes (ranging from 208 to 497 kb) as well as diverse virion morphology and complex virion structure (Yuan and Gao, 2017). They often encode greater than 60 structural proteins with some displaying complex head structures composed of more than five proteins (Effantin et al., 2013) or long, wavy, curly tail fibers (Yuan and Gao, 2016). Jumbo bacteriophages were also found to be highly diverse, with over 11 clusters and five singleton bacteriophages suggested from 52 complete jumbo bacteriophage genomes analyzed in 2017, many of which are uncharacterized (Yuan and Gao, 2017). Only a few jumbo bacteriophage families have been characterized beyond sequence analysis and EM, including the phiKZ-like bacteriophages 201phi2-1 (Thomas et al., 2008), KTN4 (Danis-Wlodarczyk et al., 2016), phiPA3 (Monson et al., 2011), phiRSL2 (Bhunchoth et al., 2016), phiRSF1 (Bhunchoth et al., 2016), OBP (Shaburova et al., 2006), EL (Sokolova et al., 2014), and phiKZ (Lecoutere et al., 2009), related bacteriophages phiRSL1 (Yamada et al., 2010) and PaBG (Kurochkina et al., 2018), Cronobacter bacteriophage CR5 (Lee et al., 2016), Prochlorococcus bacteriophage P-SSM2 (Sullivan et al., 2005), related bacteriophages KVP40 (Miller et al., 2003) and Aeh1 (Gibb and Edgell, 2007), Aeromonas bacteriophage phiAS5 (Kim et al., 2012), Pectobacterium bacteriophage CBB (Buttimer et al., 2017), Caulobacter bacteriophage phiCbK (Gill et al., 2012), related Erwinia bacteriophages Joad and RisingSun (Arens et al., 2018), related bacteriophage RaK2 (Simoliunas et al., 2013) and GAP32 (Abbasifar et al., 2014), Bacillus bacteriophage 0305phi8-36 (Thomas et al., 2007), related Bacillus bacteriophages BpSp (Yuan and Gao, 2016) and AR9 (Lavysh et al., 2016, 2017). Herein we further analyze the genome, proteome, and host range of our eight Agrican357virus jumbo bacteriophages. Their lytic nature and plethora of novel genes makes them a unique entity to be studied further and analyzed. As a close relative of the animal pathogens Escherichia coli and Salmonella (Zhao and Qi, 2011), viruses that infect E. amylovora may help us understand the evolution of pathogenic strains in this family.\\nMaterials and Methods\\nBacteriophage Isolation and Genome Sequencing\\nEnvironmental samples of leaves, branches and soil surrounding infected trees were collected from around the state of Utah (United States) and used to create enrichment cultures with the host E. amylovora. To test the presence of amplified bacteriophages, the enrichment cultures were spun at 4000 rpm and 4\\u00b0C for 20 min and the supernatant was removed and used without filtering. 50 \\u03bcL of this supernatant was incubated at room temperature with 500 \\u03bcL of E. amylovora ATCC 29780 bacteria for 30\\u201345 min, mixed with 5 ml NBDYE top agar (at half concentration agar), plated on NBSYE agar plate, and incubated at 25\\u00b0C overnight. Plaque presence on the plates was the primary indicator of bacteriophage presence in the environmental sample. Using a sterile needle or pipette tip, we picked a plaque from the initial bacteriophage identification plate and performed three rounds of plaque purification. All eight isolated bacteriophages: Special G (KU886222), Simmy50 (KU886223), RAY (KU886224), Deimos-Minion (KU886225), Bosolaphorus (MG655267), Desertfox (MG655268), MadMel (MG655269), and Mortimer (MG655270) were able to infect E. amylovora ATCC 29780 (Esplin et al., 2017; Sharma et al., 2018). Bacteriophage DNA was extracted using the Phage DNA isolation kit (Norgen Biotek Corporation), and was sequenced, assembled and annotated as previously described (Esplin et al., 2017; Sharma et al., 2018).\\nElectron Microscopy\\nElectron microscopy was performed at Brigham Young University in the Life Sciences Microscopy Lab using a FEI Helios NATOCAB 600i DualBeam FIB/SEM with STEM detector. The samples for SEM analysis were prepared by placing 15 \\u03bcL of high-titer bacteriophage lysate on a 200-mesh copper carbon type-B electron microscope grid for one-two minutes. The lysate was wicked away and the grids were stained for 2 min using 15 \\u03bcL of 2% phosphotungstic acid (pH = 7). Residual liquid was wicked away using Kimtech wipes and the grid was allowed to dry before being imaged. Bacteriophage structures in electron micrographs were measured using ImageJ (Abramoff et al., 2004). The average and standard deviation for each measurement was calculated from a minimum of four separate measurements.\\nBurst Size\\nBurst size was calculated by performing single-infection assay as described by Delbruck (1945). The bacteria-bacteriophage mixture was allowed to adsorb for 10 min at a multiplicity of infection (MOI) of 100. The lysate was then removed at different time-intervals ranging from 1 to 6 h and diluted to avoid secondary infection. Soft agar plaque method was used to determine titers and a graph of 10 separate readings was plotted with their average titers and time.\\nHost Range\\nHost range of all eight bacteriophages was determined using the soft agar plaque method (Hockett and Baltrus, 2017). For this, 50 \\u03bcL of bacteriophage lysate dilutions were incubated with 500 \\u03bcL of bacteria grown overnight for 30 min before plating in top agar. The plates were incubated with the top agar facing up at 25\\u00b0C overnight for this assay. Seventeen bacterial strains including E. amylovora ATCC 29780 (Ritchie and Klos, 1976) as control were used including five other E. amylovora strains [Ea110 (Ritchie and Klos, 1976), GH9 (McGhee and Sundin, 2012), EaBH (McGhee and Sundin, 2012), RB02 (McGhee and Sundin, 2012), Ea273 (Sebaihia et al., 2010)], Pantoea agglomerans E325 (Pusey, 1997), Pantoea vagans C-91 (Ishimaru et al., 1987; Smits et al., 2010), E. coli K-12 BW 25113 (Datsenko and Wanner, 2000), Salmonella enterica LT2 (generously donated by John Roth lab), Yersinia pestis KIM6 (Fetherston et al., 1992; Gong et al., 2001), Enterobacter cloacae ATCC 13047 (Ren et al., 2010), Klebsiella pneumoniae ATCC 10031 (Otman et al., 2007), Bacillus subtilis ATCC 6033 (Nakamura et al., 1999), Cronobacter sakazakii ATCC 29544 (Iversen et al., 2007; Gicova et al., 2014), standard clinical isolate Pseudomonas aeruginosa PA100 (Cartagena et al., 2007) and Pseudomonas chlororaphis ATCC 13985 (Conway et al., 1956). An average of two readings was taken to obtain bacteriophage titers post-infection.\\nComputational Analysis and Genomic Comparison\\nBacteriophages with any similarities to Agrican357virus genus were identified using a blastx analysis of their putative major capsid and terminase proteins, and the corresponding bacteriophage for all retrieved hits with a cutoff e-value of less than 1.00E-04 and 33% similarity were downloaded from GenBank. In addition, any bacteriophages that showed up in at least three qblast hits while annotating were also retrieved. These sequences were then used in Gepard (Krumsiek et al., 2007) to generate the dot plots of nucleic acid and protein sequences. PhamDB, a web interface (Lamine et al., 2016) was used for creating databases and Phamerator, (Cresawn et al., 2011) an open-source program was used to generate pham map of bacteriophage genomes. PhamDB uses kClust (Hauser et al., 2013) to cluster large protein sequence databases. The default settings of PhamDB were used in this comparison. Splitstree (Huson and Bryant, 2006) protein analysis was produced from the exported pham table of conserved proteins converted to a Nexus file using Janus1. The Average Nucleotide Identity (ANI) percentages comparing each of the E. amylovora bacteriophage genomes were calculated using MAFFT (Katoh et al., 2002) plugin in Geneious R8.1 (Kearse et al., 2012).\\nThe evolutionary history was inferred by using the Maximum Likelihood method and Poisson correction model (Zuckerkandl and Pauling, 1965). The bootstrap consensus tree inferred from 100 replicates (Felsenstein, 1985) is taken to represent the evolutionary history of the taxa analyzed (Felsenstein, 1985). Branches corresponding to partitions reproduced in less than 50% bootstrap replicates are collapsed. The percentage of replicate trees in which the associated taxa clustered together in the bootstrap test (100 replicates) are shown next to the branches (Felsenstein, 1985). Initial tree(s) for the heuristic search were obtained automatically by applying Neighbor-Join and BioNJ algorithms to a matrix of pairwise distances estimated using a JTT model, and then selecting the topology with superior log likelihood value. This analysis involved 59 amino acid sequences. There were a total of 1302 positions in the final dataset. Evolutionary analyses were conducted in MEGA X (Kumar et al., 2018).\\nMass Spectrometry\\nSample preparation was performed (Guttman et al., 2009) by diluting crude lysates of RAY and Deimos-Minion in TNE (50 mM Tris pH 8.0, 100 mM NaCl, 1 mM EDTA) buffer and adding RapiGest SF reagent (Waters Corp.) to a final concentration of 0.1%. Samples were then boiled for 5 min followed by addition of 1 mM (final concentration) of TCEP [Tris (2-carboxyethyl) phosphine] and incubated at 37\\u00b0C for 30 min. Afterward, carboxymethylation of samples was done with 0.5 mg/ml of iodoacetamide for 30 min at 37\\u00b0C followed by neutralization with 2 mM TCEP (final concentration). Trypsin (trypsin: protein ratio \\u2013 1:50) was used overnight at 37\\u00b0C to digest the crude lysates prepared as above. The samples were treated with 250 mM HCl at 37\\u00b0C for 1 h followed by centrifugation at 14000 rpm for 30 min at 4\\u00b0C to degrade and remove RapiGest. The soluble fraction was then added to a new tube and Aspire RP30 desalting columns (Thermo Fisher Scientific) were used for extraction and desalting of the peptides.\\nHigh pressure liquid chromatography (HPLC) coupled with tandem mass spectroscopy (LC-MS/MS) using nano spray ionization was used to analyze Trypsin-digested peptides (McCormack et al., 1997). A TripleT of 5600 hybrid mass spectrometer (ABSCIEX) interfaced with nano-scale reversed-phase HPLC (Tempo) using a 10 cm-100-micron ID glass capillary packed with 5-\\u03bcm C18 ZorbaxTM beads (Agilent Technologies, Santa Clara, CA) was used to perform the nano-spray ionization experiments. By using a linear gradient (5\\u201360%) of ACN (Acetonitrile) at a flow rate of 250 \\u03bcl/min for 1 h, peptides were eluted from the C18 column into the mass spectrometer The ACN gradient was created using these buffers: buffer A (98% H2O, 2% ACN, 0.2% formic acid, and 0.005% TFA) and buffer B (100% ACN, 0.2% formic acid, and 0.005% TFA). In a data-dependent manner MS/MS data were acquired in which the MS1 data was acquired for 250 ms at m/z of 400 to 1250 Da and the MS/MS data was acquired from m/z of 50 to 2,000 Da. For Independent data acquisition (IDA) parameters MS1-TOF 250 ms, followed by 50 MS2 events of 25 ms each. The IDA criteria; over 200 counts threshold, charge state of plus 2\\u20134 with 4 s exclusion window. Finally, MASCOT\\u00ae (Matrix Sciences) was used to analyze the collected data and Protein Pilot 4.0 (ABSCIEX) was used for peptide identifications.\\nExtracellular Polymeric Substance (EPS) Depolymerase Mediated Biofilm Degradation Assay\\nSoft agar plaque method (Hockett and Baltrus, 2017), as described previously in host range method, was used to detect the presence of halo zone on P. vagans strain C9-1 and E. amylovora ATCC 29780. The putative EPS-depolymerase from bacteriophage RAY was PCR amplified from lysate using primers designed to amplify the full length gp76. It was cloned by digesting with enzymes NdeI/SalI into a similarly digested pET15b. The resulting plasmid (JG1700) was amplified by transforming into E. coli DH5\\u03b1 and plated on LB-amp. Resulting colonies were PCR checked and were used to start overnight cultures and DH5\\u03b1 without plasmid pJG1700 was grown as a control. The protein was induced using IPTG and extracted by lysing cells via sonication. Post-sonication, cell debris was removed from both cultures by centrifuging at 12000 rpm and 4\\u00b0C for 2 \\u00d7 20 min. 10 \\u03bcl of resulting supernatant was spotted on bacterial lawns of P. vagans strain C9-1 and E. amylovora ATCC 29780 embedded in top agar after plating for 2 h.\\nMotif Identification and Analysis\\nMEME (Bailey et al., 2009) and FIMO (Grant et al., 2011) tools at public phage galaxy2 were used to scan bacteriophage genome of Agrican357virus for statistically significant motifs. Motifs found by MEME (Bailey et al., 2009) with e-value less than 1.00E-02 were selected by FIMO (Grant et al., 2011) to be searched for their coordinates and iterations in their respective genomes. User defined cut-off values (P-value <1.00E-03, Q-value <0.05), as described in Berg et al. (2016) were used to maximize the results. The location of the motifs within bacteriophage genomes was determined from the annotated GenBank files (Esplin et al., 2017).\\nResults and Discussion\\nIsolation and Characterization of Eight Closely Related Large Bacteriophages Infecting E. amylovora\\nEight novel bacteriophages (Deimos-Minion, Special G, RAY, Simmy50, Bosolaphorus, Desertfox, Mortimer, and MadMel) that infect E. amylovora were plaque isolated and their genomes were subsequently sequenced and annotated as previously described (Esplin et al., 2017; Sharma et al., 2018). All eight bacteriophages have relatively large genomes with genome sizes of 271 to 275 kb (Table 1), which are comparable to the related bacteriophage Ea35-70 (271084 bp). These bacteriophages have correspondingly large putative proteomes, with 317 to 324 predicted ORFs. A search for tRNA\\u2019s using tRNA ScanSE (Lowe and Chan, 2016) suggests that RAY, Simmy50, Bosolaphorus, and Mortimer have 1 tRNA each coding for Asparagine, whereas no tRNA\\u2019s were detected for Deimos-Minion, Special G, MadMel, and Desertfox. No lysogeny related genes were identified (including integrase, excisionase or repressors). Their clear plaque morphology and ease in obtaining higher titers (\\u223c108\\u20131010 pfu/ml) suggest they may be lytic bacteriophages, however, rigorous testing for bacterial lysogeny has not been performed.\\nTable 1. General characteristics of related bacteriophages Deimos-Minion (DM), RAY, Special G, Desertfox, MadMel, Mortimer, Bosolaphorus, Simmy50, and Ea35-70 that infect E. amylovora ATCC 29780.\\nElectron Microscopy Reveals Myovirus Structure for Five E. amylovora Bacteriophages\\nDeimos-Minion, Special G, RAY, Simmy50, Mortimer, MadMel, Desertfox, and Bosolaphorus were all found to be similarly sized Myoviridae, having contractile tails (average size 159 + \\u221211.4 nm), a tail sheath (average size 78.5 + \\u22129.28 nm), visible tail fibers, and large capsids (average size 128 + \\u22125.96 nm) (Figure 1). This morphology is supported by their protein-based relationships to other jumbo Myoviridae discussed below. Due to apparent similarity within these bacteriophages, only RAY\\u2019s morphological calculations are listed but all eight of these bacteriophages were imaged extensively.\\nFigure 1. Electron microscopy STEM images of Deimos-Minion, Special G, RAY, Simmy50, Bosolaphorus, Desertfox, MadMel, and Mortimer revealed Myoviruses with long contractile tails.\\nHost Range and Burst Size\\nBacteriophages of the Agrican357virus family were tested for activity against seventeen different bacterial strains. Out of these, fifteen were from the Enterobacteriales- P. agglomerans E325 (Pusey, 1997), P. vagans C-91 (Ishimaru et al., 1987; Smits et al., 2010), E. coli K-12 BW 25113 (Datsenko and Wanner, 2000), S. enterica (generous donation by roth lab), Y. pestis KIM6 (Fetherston et al., 1992; Gong et al., 2001), E. cloacae ATCC 13047 (Ren et al., 2010), K. pneumoniae ATCC 10031 (Otman et al., 2007), B. subtilis ATCC 6033 (Nakamura et al., 1999), C. sakazakii ATCC 29544 (Iversen et al., 2007; Gicova et al., 2014), E. amylovora Ea110 (Ritchie and Klos, 1976), E. amylovora GH9 (McGhee and Sundin, 2012), E. amylovora EaBH (McGhee and Sundin, 2012), E. amylovora RB02 (McGhee and Sundin, 2012), E. amylovora Ea273 (Sebaihia et al., 2010), E. amylovora ATCC 29780 (control) (Ritchie and Klos, 1976), and two from Pseudomonadaceae- P. aeruginosa PA100 (Cartagena et al., 2007) and P. chlororaphis ATCC 13985 (Conway et al., 1956). Enterobacteriales strains were chosen due to being members of the same bacterial order as Erwinia, whereas Pseudomonadaceae strains were the hosts of bacteriophages related to the Agrican357virus bacteriophages based on protein BLAST. Our current analyses displayed that Agrican357virus bacteriophages infect all Erwinia strains (with the exception of Special G and Mortimer that failed to infect GH9 and EaBH, respectively) as well as closely related genera also commonly found on plants\\u2013 P. agglomerans (Deletoile et al., 2009) and P. vagans (Palmer et al., 2016; Table 2).\\nTable 2. Host range analysis of eight Agrican357virus bacteriophages.\\nOwing to the large nature of Agrican357virus bacteriophages, we investigated the burst size of bacteriophage Deimos-Minion on E. amylovora strain ATCC 29780. Burst size studies suggested that when infected at MOI of 100 Deimos-Minion has burst size of 4.6\\u20134.9 with latent period of 3\\u20134 h before the first burst (Figure 2) under the laboratory growth conditions used herein, consistent with their large size. As seen in Figure 2, a second burst is appearing at the end of this 6 h period. The observed burst size (\\u223c5) was confirmed with bacteriophage RAY (data not shown) and is consistent with other large Myoviridae in that P. aeruginosa bacteriophage KTN4 has a reported burst of 6\\u20138 and may be due to the need to build internal cellular structures for the Jumbo viruses to be built (Danis-Wlodarczyk et al., 2016), or due to sub-optimal assay conditions for proliferation.\\nFigure 2. Growth Curve of Deimos-Minion with host E. amylovora ATCC 29780 by plaque assays shows first burst at \\u223c4 h and second burst at \\u223c6 h.\\nGenomic and Evolutionary Characteristics\\nTo determine the overall genomic and proteomic similarity of our eight novel bacteriophages to available bacteriophages in GenBank, related bacteriophages were identified by BLAST (qblast) using each of the putative gene products encoded by RAY. The bacteriophages with e-values below 1.00E-04 and above 33% identity that were identified in three or more BLAST searches were then compared using Gepard dot plot (Krumsiek et al., 2007), average nucleotide identity (ANI analysis) (Lassmann and Sonnhammer, 2005), and BLAST alignment (Altschul et al., 1990). Dot plots were constructed using whole genome sequences, major capsid protein amino acid, and terminase amino acid sequences (Figures 3A\\u2013C, respectively). While looking at the results of the whole genome dot plot, all eight of our bacteriophages show no similarity to any other bacteriophages used in the dot plot except for very close similarity to Ea35-70 (KF806589) (Yagubi et al., 2014), an Erwinia bacteriophage isolated in Canada in 2014 (see Figure 3A). In addition, their average nucleotide identity (ANI) using Geneious (Kearse et al., 2012) was remarkably high >94% (see Supplementary Table S1). These results indicate that these eight bacteriophages Deimos-Minion, Simmy50, RAY, Special G, Bosolaphorus, Desertfox, MadMel, and Mortimer along with Ea35-70 make a distinct family of bacteriophages, consistent with the International committee on taxonomy of viruses\\u2019 classification as new species of a new genus Agrican357virus in the family Myoviridae of order Caudovirales (Adams et al., 2017).\\nFigure 3. Whole-genome nucleotide (A) and protein terminase (B) or major capsid protein (C) dot plot analysis reveals a fairly isolated cluster of bacteriophages that includes Deimos-Minion, Special G, RAY, Simmy50, Bosolaphorus, Desertfox, MadMel, Mortimer and Ea35-70. Dot plots were constructed using Gepard.\\nThe major capsid protein (MCP) and terminase proteins are two of the most conserved proteins in bacteriophage genomes and have been used to group bacteriophages in families by single gene analysis (Smith et al., 2013). In order to identify distant bacteriophage relatives, a proteomic comparison of these bacteriophages was performed using terminases (see Figure 3B) and MCPs (see Figure 3C) by Gepard dot plot (Krumsiek et al., 2007). The same bacteriophage order from the whole genome dot plot (Figure 3A) was used in these dot plots. Whole genome and terminase dot plots both displayed limited synteny between Agrican357virus bacteriophages and Erwinia bacteriophage phiEaH1 (4.00E-155 from blastp of terminase) indicating this bacteriophage as the closest known relative from Erwiniaceae. In contrast, little similarity to Pseudomonas bacteriophages phiKZ (8.00E-156), KTN4 (8.00E-156), phiPA3 (2.00E-149), 201phi2-1 (5.00E-140), OBP (2.00E-101), EL (3.00E-77), and Ralstonia bacteriophages RSF1 (9.00E-122) and RSL2 (3.00E-120) can be seen in the terminase dot plot which was not apparent in the whole genome and major capsid protein dot plots. All of these bacteriophages are distantly related jumbo Myoviridae.\\nThe two subunits of terminase protein; large and small, are an essential part of DNA packaging (Kuebler and Rao, 1998; Mesyanzhinov et al., 2002). All eight of our Agrican357virus bacteriophages have a putative terminase gene with identical amino acid sequences: Deimos-Minion gp189, Special G gp185, RAY gp183, Simmy50 gp186, Desertfox gp184, Bosolaphorus gp185, MadMel gp185, and Mortimer gp188. This protein is also present in Ea35-70 gp181. This indicates that it is a highly conserved protein for this family. Considering the similarity between these bacteriophages, it can be inferred that all nine bacteriophages of Agrican357virus may have headful packaging (Figure 4). In support of this conclusion, blastp results demonstrated a match with Pseudomonas bacteriophage phiKZ with an e-value of 8.00E-156, a terminase large subunit from Erwinia bacteriophage PhiEaH2 with an e-value of 6.00E-122 and a terminase large subunit of Pseudomonas bacteriophage 201phi2-1 with an e-value of 5.00E-140. Bacteriophages phiKZ, phiEaH2 and 201phi2-1 are all known to have headful packaging (Merrill et al., 2016). In addition to blastp, bacteriophage termini and packaging mode for six bacteriophages (excluding Deimos-Minion and Special G) was also determined using randomly fragmented next-generation sequencing (NGS) data with the help of software PhageTerm (Garneau et al., 2017)3. PhageTerm analysis indicated that RAY, MadMel, Desertfox, Bosolaphorus, Simmy50, and Mortimer have headful packaging without a pac site. Thus, the headful packaging strategy is supported by terminase homology and NGS sequencing data.\\nFigure 4. Phylogenetic analyses of phage terminase proteins supports the relationships depicted by dot plot analysis of the Agrican357virus bacteriophages. The evolutionary history was inferred using the Maximum Likelihood method and Poisson correction model in MEGA X. Branches corresponding to partitions reproduced in less than 50% bootstrap replicates are collapsed where 100 was set to be initial bootstrapping value.\\nProteomic Analysis of the Agrican357virus Family\\nDue to great similarity between these bacteriophages we randomly chose RAY as a representative for the protein classification. Proteomic analysis of RAY reveals the novel nature of these bacteriophages in that of 318 proteins, 202 proteins were considered to be novel with no BLAST hit (the e-value cutoff was <1.00E-04), 50 were hypothetical proteins with BLAST hits, and 67 were proteins with a putative function based on their BLAST hit (Supplementary Figure S1A). Thus, over half of the proteins had no BLAST hit outside of the Agrican357virus bacteriophages. These proteins represent a considerable proteomic \\u201cdark matter\\u201d (Hatfull, 2015), and underscore the vast biological richness harbored in bacteriophages. Of the 67 proteins with predicted function, a majority appear to be structural proteins (\\u223c41%), and DNA metabolism proteins (approximately 41%) (Supplementary Figure S1B).\\nThe computer program Phamerator (Cresawn et al., 2011) was used to compare the entire genomes of the nine Agrican357virus bacteriophages that infect E. amylovora: Deimos-Minion, Special G, RAY, Simmy50, Bosolaphorus, Desertfox, MadMel, Mortimer, and Ea35-70 (Figure 5). Despite their large size, these genomes display remarkable nucleotide sequence and proteomic conservation (>94% ANI, see Supplementary Table S1). The genomes encode recognizable structural and enzymatic bacteriophage proteins vital to the bacteriophage life cycle, including terminase proteins, major capsid proteins, and tail fiber proteins as well as proteins involved in DNA transcription and translation, such as helicase proteins, DNA polymerase, and RNA polymerase. Though the genomes of these nine bacteriophages are virtually identical, a few genes are differentially present across these bacteriophage genomes. Most of these are hypothetical proteins, however, HNH endonucleases also differed consistently between the Agrican357virus bacteriophages. HNH endonucleases are proteins that splice DNA and assist in the movement of introns and other intron-like sequences (Chevalier and Stoddard, 2001). Deimos-Minion has two such HNH endonucleases, gp93 and gp234 that do not appear to be homologs based on protein similarity. Protein BLAST results of gp93 show that the HNH endonuclease is also found in bacteriophages Bosolaphorus, Desertfox, MadMel, RAY, Simmy50, Special G and Ea35-70, and is similar to those found in some Pseudomonas bacteriophages (phiKZ and KTN4) as well as both Gram-negative and Gram-positive strains of bacteria. However, only the HNH endonuclease domain (\\u223camino acid 58-109 of bp93) is primarily conserved, the remaining 278 amino acid protein is not conserved in bacteria. On the other hand, homologs of HNH endonuclease gp234 are only found in Deimos-Minion and MadMel, as well as several Gram-positive and Gram-negative bacteria. Genomes of Deimos-Minion, Desertfox and MadMel also displayed a reversed order of two proteins (gp93-gp94 in Deimos-Minion, gp88-gp89 in Desertfox and gp90-gp91 in MadMel) when compared to similar proteins in other bacteriophages of this family. The proteins involved are HNH endonuclease and ribonucleotide reductase. To search for repetitive sequences in the genome which may be involved in recombination, MEME (Bailey et al., 2009) and FIMO (Grant et al., 2011) were used to locate motifs in the genomes of all eight of our Agrican357virus bacteriophages. Several common and unique motifs were discovered, however, they had poor e-values with little or no significance and were not followed further.\\nFigure 5. Whole genome Phamerator map of E. amylovora bacteriophages illustrates the high similarity of bacteriophages Mortimer, MadMel, Desertfox, Bosolaphorus, Deimos-Minion, Special G, RAY, Simmy50, and Ea35-70. Bacteriophages were mapped using Phamerator and arranged based on highest protein similarity. Violet shading between genomes indicates genome nucleotide homology (with standard e-value cutoff of 1.00E-04) and the ruler indicates genome base pairs, while white spaces indicate areas without significant nucleotide similarity. Boxes above and below the genome ruler indicate ORFs going in the forward and reverse direction, respectively. They are labeled with predicted function, occasionally numbered, and colored to indicate protein homologs between the bacteriophages.\\nDue to the large size of these bacteriophages, and their terminase similarity to bacteriophage phiKZ, these bacteriophages likely belong to the jumbo bacteriophages (Hertveldt et al., 2005; Yuan and Gao, 2017) making it no surprise that the structural proteins are found in other bacteriophages. Along with hypothetical proteins, the proteins that are conserved with other phiKZ-like jumbo bacteriophages include: RNA polymerase beta subunit, nuclease RtcB-like, SbcC like, helicase, virion structural proteins, tail fiber, tail sheath, lysozyme domain, terminase, and major capsid protein. A splitstree analysis showing the relationship of the related jumbo bacteriophages by protein conservation is displayed in Figure 6. This protein-based tree suggests seven groups of related jumbo Myoviridae bacteriophages, with the Agrican357virus group as the most distant group. It further confirms that proteins of Agrican357virus family are more similar to proteins from Pseudomonas bacteriophages EL and OBP and Ralstonia bacteriophage RSL2 than to other Enterobacteriales bacteriophages.\\nFigure 6. Protein-conservation analysis displayed by Splitstree of the Agrican357virus genus with related jumbo Myoviridae bacteriophages reveals Agrican357virus as a distant evolutionary group.\\nMass Spectrometry Validates 27 Hypothetical Proteins as Proteins of Unknown Function\\nFurther analysis of Deimos-Minion and RAY genomes via mass spectrometry (MS) detected several novel proteins, promoting the status of 27 proteins from hypothetical proteins to proteins of unknown function. In RAY and Deimos-Minion genomes collectively, MS analysis identified seventeen proteins with a putative function, eighteen novel hypothetical proteins specific to this bacteriophage family and nine hypothetical proteins (seven known bacteriophage proteins and two other) with blastp hits to other bacteriophages (Table 3). The majority of proteins found through MS are novel hypothetical proteins found only in this family, followed by putative bacteriophage structural proteins, hypothetical bacteriophage proteins, proteins with putative functions and other hypothetical proteins (see Table 3). This analysis agrees with our predicted conservation of proteins depicted through Phamerator analyses.\\nTable 3. Mass Spectrometry reveals 27 Hypothetical Proteins as Proteins of Unknown Function.\\nBiofilm Degradation (EPS) Assays Suggest Specificity for Pantoea\\nEnzymatic proteins like extracellular polysaccharide (EPS) depolymerase and phage-related lysozyme are few of the annotated proteins with putative functions which were also predicted via mass spectrometry. EPS depolymerase (Kim and Geider, 2000) is an enzyme that degrades EPS and phage-related lysozyme is shown to lyse the bacterial cell wall (Emrich and Streisinger, 1968). It has been shown that halo formation on the host could be a result of biofilm degradation activity (Cornelissen et al., 2012; Majkowska-Skrobek et al., 2016). The presence of halo zone after in infections of Agrican357virus family was first observed on P. vagans strain C9-1 (Figure 7A). To investigate further the EPS- depolymerase gene was cloned into a plasmid pJG1700, amplified using E. coli DH5\\u03b1, and spotted on P. vagans stain C-91 and E. amylovora strain ATCC 29780 (Figure 7B). Lysate from a similarly grown and prepared DH5\\u03b1 culture was used as a control. The clearing is indicative of EPS depolymerase activity on P. vagans. This activity was not seen on E. amylovora ATCC 29780.\\nFigure 7. Halo formation on P. vagans by RAY (A) and bio-film degradation activity of gp76 on P. vagans (B).\\nStructural Prediction Supports the Putative Function of Several Proteins\\nTo further understand Agrican357virus and verify their protein functions, we studied proteins involved in DNA metabolism (\\u223c45%), the largest group of functional proteins conserved in the Agrican357virus. Multiple mechanisms for DNA regulation and repair are evident with the presence of proteins that are hypothesized to aid DNA synthesis, repair, and recombination. These proteins may increase the stability and survival of these jumbo bacteriophages (Supplementary Table S2 and Supplementary Figure S2). In order to proliferate in host cells, bacteriophages need to be equipped with proteins that allow them to reproduce effectively. Although many bacteriophages harbor proteins for DNA damage repair and DNA reproduction inside a host bacteria cell, these large bacteriophages may require extremely viable progeny due to lower burst sizes (\\u223c4.6 functional virions compared to thousands reported for other bacteriophages).\\nTwo proteins with a conserved domain found in the nine Agrican357virus bacteriophages are SbcC and a SbcCD nuclease (see Supplementary Figure S2A). The ability of SbcC and SbcCD to regulate and repair DNA has been shown to be essential for the stability and proliferation of some bacteriophages (Connelly et al., 1998). During DNA replication, palindromic sequences will create hairpin-like structures that can inhibit the progression of DNA polymerase (Leach et al., 1997). SbcC and SbcCD proteins work together to cleave both double- stranded and single-stranded DNA, and have been shown to recognize and specifically cleave hairpin structures. This breaks down the replication fork, allowing the genome to be repaired through recombination, so replication can proceed (Leach et al., 1997; Connelly et al., 1998). The proteins SbcC and SbcCD nucleases preserve the viability of the genome by allowing replication without excising the palindromic sequences (Leach et al., 1997). There are many types of DNA damage that may occur within a genome, making recombination and repair of DNA important, such as mutations due to UV damage. UV damage creates kinks or abnormalities within a genome, and prohibits proliferation. Exodeoxyribonuclease VIII breaks double stranded DNA, and degrades a genome on both 5\\u2032 ends (Joseph and Kolodner, 1983; Kolodner et al., 1994). This allows the kinked and abnormal portions of a genome to be straightened and repaired through homologous recombination. Additionally, exodeoxyribonuclease VIII does not require ATP to perform DNA repair, enabling repair of the genome even in low-energy environments where the bacteriophage does not have access to ATP (Kolodner et al., 1994). We hypothesize that exodeoxyribonuclease VIII enables the bacteriophages to remain stable despite mutations from UV damage. However, unique from our other predicted structure alignments, the protein from RAY does not match up well with other exodeoxyribonuclease VIII homologs (see Supplementary Figure S2B). It is possible that since these proteins do not have the same protein folding and alignment, they may not have the same function but a related, adapted function.\\nIn the Agrican357virus bacteriophages, there are several encoded proteins with conserved domains of the thymidylate kinase and thymidine kinase (see Supplementary Table S2). Structural prediction and alignment confirm these proteins as likely thymidine kinases (see Supplementary Figures S2C,D), a necessary step due to the distant relationship (low e-values) of Agrican357virus bacteriophage proteins when compare to other biological entities. Thymidine kinase is an enzyme that catalyzes the phosphorylation of thymidine monophosphate (Kokoris and Black, 2002). Thymidylate kinase then catalyzes the phosphorylation of thymidine diphosphate (Doharey et al., 2016), which is an essential precursor for DNA (Chaudhary et al., 2013). Therefore, these proteins are regulatory enzymes that make bacteriophage cell growth and survival possible by aiding proliferation through the synthesis of DNA (Chaudhary et al., 2013; Cui et al., 2013; Doharey et al., 2016). Other proteins shown in Supplementary Figures S2E,F are putative UvsX recombinase and a putative SF2 helicase with conserved helicase domain known as UvsW, which finishes the recombination (Madej et al., 1995; Gajewski et al., 2011). UvsX and UvsW are proteins that have been known to work together to repair broken replication forks through homologous recombination (Kadyrov and Drake, 2004; Maher and Morrical, 2013). Homologous recombination is one of the most efficient ways to have error free DNA repair, and is beneficial to bacteriophages to have this repair mechanism. These repair mechanisms would be important to the bacteriophages because it would not only help repair broken replication forks but it would also help repair damaged or broken DNA (Kadyrov and Drake, 2004; Maher and Morrical, 2013). It has been shown that the absence of UvsX increases UV sensitivity (Kadyrov and Drake, 2004).\\nConclusion\\nAgrican357virus genus of bacteriophages are Myoviridae with dsDNA, large capsids, long contractile tails and high GC content. Their genomes are nearly identical (>94% ANI). All three dot plots (whole genome, major capsid protein, and terminase protein) show no close similarity between the Agrican357virus family and any of the other bacteriophages on NCBI (see Figures 3A\\u2013C). We have also found that the Agrican357virus cluster is more closely related to bacteriophages infecting Pseudomonas and Ralstonia, than those infecting E. amylovora. The contrast that we observe between this cluster of bacteriophages and the distantly related bacteriophage analyzed by dot plot contributes valuable information about evolutionary relationships between these other clusters (see Figure 3), suggesting the distant relationship may emphasize the importance of ecological niche, since most other Enterobacteriales bacteriophages isolated infect animal pathogens rather than plant pathogens. It may also, however, simply indicate the abundance of unstudied bacteriophages. The Agrican357virus family of bacteriophages is a novel family, with very low similarity to any other viruses, providing approximately 250 novel proteins to add to the viral dark matter that have no homolog by blastp (Hatfull, 2015). To understand a bacteriophage, it is vital to understand the encoded proteome. A bacteriophage\\u2019s proteins determine how effectively it can infect bacteria, and how stable and safe it would be to use in a phage cocktail (a mixture of bacteriophages used together for phage therapy). Of the proteins with predicted function, this family encodes primarily DNA metabolism and repair proteins. Since the bacteriophage host, E. amylovora, is found primarily on the blossoms of fruit trees of the Rosaceae family, these proteins may be particularly vital due to the onset of UV radiation including putative thymidine and thymidylate kinases which aid the production of the nucleotide thymine for DNA synthesis (Doharey et al., 2016), putative SbcC and SbcCD proteins which protect against DNA damage by cleaving harmful hairpin structures during replication (Connelly et al., 1998), putative exodeoxyribonuclease VIII which makes double stranded DNA breaks to help repair DNA damage at low energy (Joseph and Kolodner, 1983), and putative UvsX recombinase and putative SF2 helicase which aid in repair and recombination of DNA (Maher and Morrical, 2013). The small burst size we report herein for these jumbo bacteriophages (\\u223c4.6 functional virions), may require a high level of fidelity to ensure success in the environment.\\nA paper published in 2003 on evolutionary pathways of P. aeruginosa bacteria demonstrated that phiKZ-like bacteriophages have a very broad host range (Krylov et al., 2003). In 1995, Campbell et al. (Campbell et al., 1995) isolated bacteriophages from barley rhizosphere that infected Pseudomonas spp. other than P. aeruginosa. These bacteriophages displayed great morphological similarity to phiKZ-like bacteriophages despite low genomic similarity (Mesyanzhinov et al., 2002; Krylov et al., 2003; Sokolova et al., 2014). Similarly, Agrican357virus bacteriophages display proteomic similarity to phiKZ-like bacteriophages, particularly with their structural proteins, with little genomic synteny. These results suggest the phiKZ-like bacteriophages are highly divergent, derived from a common ancestor and successful in a wide range of ecological niches. It is highly likely that Agrican357virus family evolved through both mutational divergence and modular evolution (acquisition of larger regions of DNA, or modules), which is a common phenomenon in bacteriophages (Botstein, 1980), and yet there is extremely low variance in all isolates thus far (>94% ANI). Such high conservation in these large genomes may reflect selective forces on a majority of the genome, which is for the most part uncharacterized. The great challenge ahead is both the abundance of bacteriophages that are completely uncharacterized, and the abundance of novel proteins harbored in their genomes.\\nData Availability\\nAll datasets generated for this study are included in the manuscript and the Supplementary Files.\\nAuthor Contributions\\nRS designed and wrote the manuscript, performed genomic and proteomic analysis, executed host range, EPS depolymerase cloning, halo zone and burst size experiments, prepared samples for electron microscopy and mass spectrometry, and drew all the figures and tables. KB, BP, and OS performed the structural prediction of proteins and contributed to writing. TN and EY helped with Dotplots. DB contributed with funding and ideas and also edited the manuscript. JG is the corresponding author and principal investigator of this study and as such contributed to the design and execution of experiments, oversaw all students, edited the manuscript, and contributed with funding. All authors read and approved the submitted version.\\nFunding\\nDB was awarded a grant by the United States Dairy Association. Financial support by The University of Arizona, Brigham Young University, Department of Microbiology and Molecular Biology, and the College of Life Sciences in the form of student wage funds, support, and publication fees is gratefully acknowledged.\\nConflict of Interest Statement\\nJG is working on licensing a phage-based therapy for fire blight.\\nThe remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nThe authors extend a special thanks to the Science Education Alliance \\u2013 Phage Hunters Advancing Genomics and Evolutionary Science (SEA-PHAGES) for support and training on bacteriophage analysis, Steven Cresawn for training on Phamerator, Majid Ghassemian and BMPMSF at UCSD for carrying out mass spectrometry, George W. Sundin for donating E. amylovora strains, Edward Wilcox from the BYU Sequencing Center, Michael Standing from the BYU Microscopy Lab and Daniel Thompson for helping with phage microscopy, David Parady and Brandon Ekins from Life Science IT at BYU for hosting DNAmaster and PhamDB on BYU server, Tsz Ching Tam for media preparation, Daniel Arens for helping with cloning and Yomesh Sharma for helping with the formatting of the manuscript. The authors also like to thank the following undergraduates from the BYU Phage Hunters\\u2019 program for their contributions to isolation of the bacteriophages: Garrett Jensen, Jared Kruger, Madison Melville, Trevon Galbraith, Savannah Grossarth, Hannah Ferguson, and Austin Simister.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2019.01533/full#supplementary-material\\nFootnotes\\n^ http://cobamide2.bio.pitt.edu\\n^ https://cpt.tamu.edu/galaxy-pub/\\n^ https://galaxy.pasteur.fr\\nReferences\\nAbbasifar, R., Griffiths, M. W., Sabour, P. M., Ackermann, H. W., Vandersteegen, K., Lavigne, R., et al. (2014). Supersize me: Cronobacter sakazakii phage Gap32. Virology 460-461, 138\\u2013146. doi: 10.1016/j.virol.2014.05.003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAbedon, S. T., Thomas-Abedon, C., Thomas, A., and Mazure, H. (2011). Bacteriophage prehistory: is or is not Hankin, 1896, a phage reference? Bacteriophage 3, 174\\u2013178. doi: 10.4161/bact.1.3.16591\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAbramoff, M. D., Magalhaes, P. J., and Ram, S. J. (2004). Image processing with imageJ. Biophoton. Int. 11, 36\\u201342.\\nGoogle Scholar\\nAdams, M. J., Lefkowitz, E. J., King, A. M. Q., Harrach, B., Harrison, R. L., Knowles, N. J., et al. (2017). Changes to taxonomy and the international code of virus classification and nomenclature ratified by the international committee on taxonomy of viruses. Arch. Virol. 162, 2505\\u20132538.\\nGoogle Scholar\\nAltschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990). Basic local alignment search tool. J. Mol. Biol. 215, 403\\u2013410. doi: 10.1006/jmbi.1990.9999\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nArens, D. K., Brady, T. S., Carter, J. L., Pape, J. A., Robinson, D. M., Russell, K. A., et al. (2018). Characterization of two related erwinia myoviruses that are distant relatives of the phikz-like jumbo phages. PLoS One 13:e0200202. doi: 10.1371/journal.pone.0200202\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBailey, T. L., Boden, M., Buske, F. A., Frith, M., Grant, C. E., Clementi, L., et al. (2009). Meme suite: tools for motif discovery and searching. Nucleic Acids Res. 37, W202\\u2013W208. doi: 10.1093/nar/gkp335\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBerg, J. A., Merrill, B. D., Crockett, J. T., Esplin, K. P., Evans, M. R., Heaton, K. E., et al. (2016). Characterization of five novel brevibacillus bacteriophages and genomic comparison of Brevibacillus phages. PLoS One 11:e0156838. doi: 10.1371/journal.pone.0156838\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBhunchoth, A., Blanc-Mathieu, R., Mihara, T., Nishimura, Y., Askora, A., Phironrit, N., et al. (2016). Two Asian jumbo phages, varphirsl2 and varphirsf1, infect ralstonia solanacearum and show common features of varphikz-related phages. Virology 494, 56\\u201366. doi: 10.1016/j.virol.2016.03.028\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBollback, J. P., and Huelsenbeck, J. P. (2009). Parallel genetic evolution within and between bacteriophage species of varying degrees of divergence. Genetics 181, 225\\u2013234. doi: 10.1534/genetics.107.085225\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBoyd, E. F. (2012). Bacteriophage-encoded bacterial virulence factors and phage-pathogenicity island interactions. Adv. Virus Res. 82, 91\\u2013118. doi: 10.1016/B978-0-12-394621-8.00014-5\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nButtimer, C., Hendrix, H., Oliveira, H., Casey, A., Neve, H., McAuliffe, O., et al. (2017). Things are getting hairy: Enterobacteria bacteriophage Vb_Pcam_Cbb. Front. Microbiol. 8:44. doi: 10.3389/fmicb.2017.00044\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCampbell, J. I. A., Albrechtsen, M., and Sorensen, J. (1995). Large Pseudomonas phages isolated from barley rhizosphere. FEMS Microbiol. Ecol. 18, 63\\u201374. doi: 10.1016/0168-6496(95)00043-a\\nCrossRef Full Text | Google Scholar\\nCartagena, E., Colom, O. A., Neske, A., Valdez, J. C., and Bardon, A. (2007). Effects of plant lactones on the production of biofilm of Pseudomonas Aeruginosa. Chem. Pharm. Bull. 55, 22\\u201325. doi: 10.1248/cpb.55.22\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChaudhary, S. K., Jeyakanthan, J., and Sekar, K. (2013). Cloning, expression, purification, crystallization and preliminary X-Ray crystallographic study of thymidylate kinase (Ttha1607) from thermus thermophilus Hb8. Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun. 69(Pt 2), 118\\u2013121. doi: 10.1107/S1744309112050208\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChevalier, B. S., and Stoddard, B. L. (2001). Homing endonucleases: structural and functional insight into the catalysts of intron/intein mobility. Nucleic Acids Res. 29, 3757\\u20133774. doi: 10.1093/nar/29.18.3757\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nConnelly, J. C., Kirkham, L. A., and Leach, D. R. (1998). The sbccd nuclease of Escherichia Coli is a structural maintenance of chromosomes (Smc) family protein that cleaves hairpin DNA. Proc. Natl. Acad. Sci. U.S.A. 95, 7969\\u20137974. doi: 10.1073/pnas.95.14.7969\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nConway, H. F., Haynes, W. C., Jackson, R. W., Locke, J. M., Pridham, T. G., Sohns, V. E., et al. (1956). Pseudomonas aureofaciens kluyver and phenazine alpha-carboxylic acid, its characteristic pigment. J. Bacteriol. 72, 412\\u2013417.\\nGoogle Scholar\\nCornelissen, A., Ceyssens, P. J., Krylov, V. N., Noben, J. P., Volckaert, G., and Lavigne, R. (2012). Identification of Eps-degrading activity within the tail spikes of the novel Pseudomonas putida phage Af. Virology 434, 251\\u2013256. doi: 10.1016/j.virol.2012.09.030\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCresawn, S. G., Bogel, M., Day, N., Jacobs-Sera, D., Hendrix, R. W., and Hatfull, G. F. (2011). Phamerator: a bioinformatic tool for comparative bacteriophage genomics. BMC Bioinformatics 12:395. doi: 10.1186/1471-2105-12-395\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCui, Q., Shin, W. S., Luo, Y., Tian, J., Cui, H., and Yin, D. (2013). Thymidylate kinase: an old topic brings new perspectives. Curr. Med. Chem. 20, 1286\\u20131305. doi: 10.2174/0929867311320100006\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDanis-Wlodarczyk, K., Vandenheuvel, D., Jang, H. B., Briers, Y., Olszak, T., Arabski, M., et al. (2016). A proposed integrated approach for the preclinical evaluation of phage therapy in Pseudomonas infections. Sci. Rep. 6:28115. doi: 10.1038/srep28115\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDatsenko, K. A., and Wanner, B. L. (2000). \\u201cOne-step inactivation of chromosomal genes in Escherichia Coli K-12 using Pcr products. Proc. Natl. Acad. Sci. U.S.A. 97, 6640\\u20136645. doi: 10.1073/pnas.120163297\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDelbruck, M. (1945). The burst size distribution in the growth of bacterial viruses (Bacteriophages). J. Bacteriol. 50, 131\\u2013135.\\nGoogle Scholar\\nDeletoile, A., Decre, D., Courant, S., Passet, V., Audo, J., Grimont, P., et al. (2009). Phylogeny and Identification of Pantoea species and typing of pantoea agglomerans strains by multilocus gene sequencing. J. Clin. Microbiol. 47, 300\\u2013310. doi: 10.1128/JCM.01916-08\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nD\\u2019Herelle, F. (1917). Sur un microbe invisible antagoniste des bacilles dysent\\u00e9riques. CR Acad. 165, 373\\u2013375.\\nGoogle Scholar\\nD\\u2019Herelle, F. (2007). On an invisible microbe antagonistic toward dysenteric bacilli: brief note by Mr. F. D\\u2019herelle, presented by Mr. Roux. 1917. Res. Microbiol. 158, 553\\u2013554. doi: 10.1016/j.resmic.2007.07.005\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDoharey, P. K., Singh, S. K., Verma, P., Verma, A., Rathaur, S., and Saxena, J. K. (2016). Insights into the structure-function relationship of Brugia Malayi Thymidylate Kinase (Bmtmk). Int. J. Biol. Macromol. 88, 565\\u2013571. doi: 10.1016/j.ijbiomac.2016.04.004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nEffantin, G., Hamasaki, R., Kawasaki, T., Bacia, M., Moriscot, C., Weissenhorn, W., et al. (2013). Cryo-electron microscopy three-dimensional structure of the jumbo phage phirsl1 infecting the phytopathogen Ralstonia solanacearum. Structure 21, 298\\u2013305. doi: 10.1016/j.str.2012.12.017\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nEmrich, J., and Streisinger, G. (1968). The role of phage lysozyme in the life cycle of phage T4. Virology 36, 387\\u2013391. doi: 10.1016/0042-6822(68)90163-3\\nCrossRef Full Text | Google Scholar\\nEsplin, I. N. D., Berg, J. A., Sharma, R., Allen, R. C., Arens, D. K., Ashcroft, C. R., et al. (2017). Genome sequences of 19 novel erwinia amylovora bacteriophages. Genome Announc. 5:e00931-17. doi: 10.1128/genomeA.00931-17\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nFetherston, J. D., Schuetze, P., and Perry, R. D. (1992). Loss of the pigmentation phenotype in Yersinia pestis is due to the spontaneous deletion of 102 Kb of chromosomal DNA which is flanked by a repetitive element. Mol. Microbiol. 6, 2693\\u20132704. doi: 10.1111/j.1365-2958.1992.tb01446.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGajewski, S., Webb, M. R., Galkin, V., Egelman, E. H., Kreuzer, K. N., and White, S. W. (2011). \\u201cCrystal structure of the phage T4 recombinase Uvsx and its functional interaction with the T4 Sf2 helicase Uvsw. J. Mol. Biol. 405, 65\\u201376. doi: 10.1016/j.jmb.2010.10.004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGarneau, J. R., Florence, D., Louis-Charles, F., David, B., and Marc, M. (2017). Phageterm: a fast and user-friendly software to determine bacteriophage termini and packaging mode using randomly fragmented Ngs data. bioRxiv\\nGoogle Scholar\\nGibb, E. A., and Edgell, D. R. (2007). Multiple controls regulate the expression of mobe, an hnh homing endonuclease gene embedded within a ribonucleotide reductase gene of phage Aeh1. J. Bacteriol. 189, 4648\\u20134661. doi: 10.1128/jb.00321-07\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGicova, A., Orieskova, M., Oslanecova, L., Drahovska, H., and Kaclikova, E. (2014). Identification and characterization of cronobacter strains isolated from powdered infant foods. Lett. Appl. Microbiol. 58, 242\\u2013247. doi: 10.1111/lam.12179\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGill, J. J., Berry, J. D., Russell, W. K., Lessor, L., Escobar-Garcia, D. A., Hernandez, D., et al. (2012). The Caulobacter crescentus phage phicbk: genomics of a canonical phage. BMC Genomics 13:542. doi: 10.1186/1471-2164-13-542\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGong, S., Bearden, S. W., Geoffroy, V. A., Fetherston, J. D., and Perry, R. D. (2001). Characterization of the Yersinia pestis Yfu Abc inorganic iron transport system. Infect. Immun. 69, 2829\\u20132837. doi: 10.1128/iai.67.5.2829-2837.2001\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGuttman, M., Betts, G. N., Barnes, H., Ghassemian, M., van der Geer, P., and Komives, E. A. (2009). Interactions of the Npxy microdomains of the low density lipoprotein receptor-related protein 1. Proteomics 9, 5016\\u20135028. doi: 10.1002/pmic.200900457\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHertveldt, K., Lavigne, R., Pleteneva, E., Sernova, N., Kurochkina, L., Korchevskii, R., et al. (2005). \\u201cGenome comparison of Pseudomonas Aeruginosa large phages. J. Mol. Biol. 354, 536\\u2013545. doi: 10.1016/j.jmb.2005.08.075\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHockett, K. L., and Baltrus, D. A. (2017). Use of the soft-agar overlay technique to screen for bacterially produced inhibitory compounds. J. Vis. Exp. 119:55064. doi: 10.3791/55064\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nIshimaru, C. A., Klos, E. J., and Brubaker, R. R. (1987). Multiple antibiotic production by Erwinia herbicola. Phytopathology 78, 746\\u2013750.\\nGoogle Scholar\\nIversen, C., Lehner, A., Mullane, N., Bidlas, E., Cleenwerck, I., Marugg, J., et al. (2007). The taxonomy of Enterobacter sakazakii: proposal of a new genus cronobacter gen. nov. and descriptions of Cronobacter sakazakii comb. nov. Cronobacter sakazakii subsp. sakazakii, comb. nov., Cronobacter sakazakii subsp. Malonaticus subsp. nov., Cronobacter turicensis sp. nov., Cronobacter muytjensii sp. nov., Cronobacter dublinensis sp. nov. and cronobacter genomospecies 1. BMC Evol. Biol. 7:64. doi: 10.1186/1471-2148-7-64\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJoseph, J. W., and Kolodner, R. (1983). Exonuclease Viii of Escherichia Coli. I. purification and physical properties. J. Biol. Chem. 258, 10411\\u201310417.\\nPubMed Abstract | Google Scholar\\nKadyrov, F. A., and Drake, J. W. (2004). Uvsx recombinase and dda helicase rescue stalled bacteriophage T4 DNA replication forks in vitro. J. Biol. Chem. 279, 35735\\u201335740. doi: 10.1074/jbc.m403942200\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKatoh, K., Misawa, K., Kuma, K., and Miyata, T. (2002). Mafft: a novel method for rapid multiple sequence alignment based on fast fourier transform. Nucleic Acids Res. 30, 3059\\u20133066. doi: 10.1093/nar/gkf436\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKearse, M., Moir, R., Wilson, A., Stones-Havas, S., Cheung, M., Sturrock, S., et al. (2012). Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics 28, 1647\\u20131649. doi: 10.1093/bioinformatics/bts199\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKim, J. H., Son, J. S., Choi, Y. J., Choresca, C. H. Jr., Shin, S. P., Han, J. E., et al. (2012). Complete genome sequence and characterization of a broad-host range T4-Like bacteriophage phias5 infecting Aeromonas salmonicida Subsp. Salmonicida. Vet. Microbiol. 157, 164\\u2013171. doi: 10.1016/j.vetmic.2011.12.016\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKim, W. S., and Geider, K. (2000). Characterization of a viral eps-depolymerase, a potential tool for control of fire blight. Phytopathology 90, 1263\\u20131268. doi: 10.1094/PHYTO.2000.90.11.1263\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKokoris, M. S., and Black, M. E. (2002). Characterization of herpes simplex virus type 1 thymidine kinase mutants engineered for improved ganciclovir or acyclovir activity. Protein Sci. 11, 2267\\u20132272. doi: 10.1110/ps.2460102\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKolodner, R., Hall, S. D., and Luisi-DeLuca, C. (1994). Homologous pairing proteins encoded by the Escherichia Coli Rece and Rect Genes. Mol. Microbiol. 11, 23\\u201330. doi: 10.1111/j.1365-2958.1994.tb00286.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKrumsiek, J., Arnold, R., and Rattei, T. (2007). Gepard: a rapid and sensitive tool for creating dotplots on genome scale. Bioinformatics 23, 1026\\u20131028. doi: 10.1093/bioinformatics/btm039\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKrylov, V., Pleteneva, E., Bourkaltseva, M., Shaburova, O., Volckaert, G., Sykilinda, N., et al. (2003). Myoviridae bacteriophages of Pseudomonas Aeruginosa: a long and complex evolutionary pathway. Res. Microbiol. 154, 269\\u2013275. doi: 10.1016/s0923-2508(03)00070-6\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKuhn, J. H., Radoshitzky, S. R., Bavari, S., and Jahrling, P. B. (2013). The international code of virus classification and nomenclature (Icvcn): proposal for text changes for improved differentiation of viral taxa and viruses. Arch. Virol. 158, 1621\\u20131629. doi: 10.1007/s00705-012-1582-6\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKumar, S., Stecher, G., Li, M., Knyaz, C., and Tamura, K. (2018). Mega X: molecular evolutionary genetics analysis across computing platforms. Mol. Biol. Evol. 35, 1547\\u20131549. doi: 10.1093/molbev/msy096\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKurochkina, L. P., Semenyuk, P. I., Sykilinda, N. N., and Miroshnikov, K. A. (2018). The unique two-component tail sheath of giant Pseudomonas phage pabg. Virology 515, 46\\u201351. doi: 10.1016/j.virol.2017.12.010\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLamine, J. G., DeJong, R. J., and Nelesen, S. M. (2016). Phamdb: a web-based application for building phamerator databases. Bioinformatics 32, 2026\\u20132028. doi: 10.1093/bioinformatics/btw106\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLassmann, T., and Sonnhammer, E. L. (2005). Kalign\\u2013an accurate and fast multiple sequence alignment algorithm. BMC Bioinform. 6:298.\\nPubMed Abstract | Google Scholar\\nLavysh, D., Sokolova, M., Minakhin, L., Yakunina, M., Artamonova, T., Kozyavkin, S., et al. (2016). The genome of Ar9, a giant transducing bacillus phage encoding two multisubunit RNA polymerases. Virology 495, 185\\u2013196. doi: 10.1016/j.virol.2016.04.030\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLavysh, D., Sokolova, M., Slashcheva, M., Forstner, K. U., and Severinov, K. (2017). Transcription profiling of Bacillus subtilis cells infected with Ar9, a giant phage encoding two multisubunit RNA polymerases. MBio 8:e02041-16. doi: 10.1128/mBio.02041-16\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLeach, D. R., Okely, E. A., and Pinder, D. J. (1997). Repair by recombination of DNA containing a palindromic sequence. Mol. Microbiol. 26, 597\\u2013606. doi: 10.1046/j.1365-2958.1997.6071957.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLecoutere, E., Ceyssens, P. J., Miroshnikov, K. A., Mesyanzhinov, V. V., Krylov, V. N., Noben, J. P., et al. (2009). Identification and comparative analysis of the structural proteomes of phikz and El, two giant Pseudomonas Aeruginosa bacteriophages. Proteomics 9, 3215\\u20133219. doi: 10.1002/pmic.200800727\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLee, J. H., Bai, J., Shin, H., Kim, Y., Park, B., Heu, S., et al. (2016). \\u201cA novel bacteriophage targeting Cronobacter sakazakii is a potential biocontrol agent in foods. Appl. Environ. Microbiol. 82, 192\\u2013201. doi: 10.1128/AEM.01827-15\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMajkowska-Skrobek, G., Latka, A., Berisio, R., Maciejewska, B., Squeglia, F., Romano, M., et al. (2016). Capsule-targeting depolymerase, derived from klebsiella Kp36 phage, as a tool for the development of anti-virulent strategy. Viruses 8:E324.\\nPubMed Abstract | Google Scholar\\nMcCormack, A. L., Schieltz, D. M., Goode, B., Yang, S., Barnes, G., Drubin, D., et al. (1997). Direct analysis and identification of proteins in mixtures by Lc/Ms/Ms and database searching at the low-femtomole level. Anal. Chem. 69, 767\\u2013776. doi: 10.1021/ac960799q\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMcGhee, G. C., and Sundin, G. W. (2012). Erwinia amylovora crispr elements provide new tools for evaluating strain diversity and for microbial source tracking. PLoS One 7:e41706. doi: 10.1371/journal.pone.0041706\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMerrill, B. D., Ward, A. T., Grose, J. H., and Hope, S. (2016). Software-based analysis of bacteriophage genomes, physical ends, and packaging strategies. BMC Genomics 17:679. doi: 10.1186/s12864-016-3018-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMesyanzhinov, V. V., Robben, J., Grymonprez, B., Kostyuchenko, V. A., Bourkaltseva, M. V., Sykilinda, N. N., et al. (2002). The genome of bacteriophage phikz of Pseudomonas Aeruginosa. J. Mol. Biol. 317, 1\\u201319.\\nPubMed Abstract | Google Scholar\\nMiller, E. S., Heidelberg, J. F., Eisen, J. A., Nelson, W. C., Durkin, A. S., Ciecko, A., et al. (2003). Complete genome sequence of the broad-host-range vibriophage Kvp40: comparative genomics of a T4-related bacteriophage. J. Bacteriol. 185, 5220\\u20135233. doi: 10.1128/jb.185.17.5220-5233.2003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMonson, R., Foulds, I., Foweraker, J., Welch, M., and Salmond, G. P. (2011). The Pseudomonas Aeruginosa generalized transducing phage phipa3 is a new member of the phikz-like group of \\u2018jumbo\\u2019 phages, and infects model laboratory strains and clinical isolates from cystic fibrosis patients. Microbiology 157(Pt 3), 859\\u2013867. doi: 10.1099/mic.0.044701-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nNakamura, L. K., Roberts, M. S., and Cohan, F. M. (1999). Relationship of Bacillus subtilis clades associated with strains 168 and W23: a proposal for Bacillus Subtilis Subsp. Subtilis subsp. nov. and Bacillus Subtilis Subsp. Spizizenii Subsp. Nov. Int. J. Syst. Bacteriol. 49(Pt 3), 1211\\u20131215. doi: 10.1099/00207713-49-3-1211\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nOtman, J., Perugini, M. E., Tognim, M. C. B., and Vidotto, M. C. (2007). Atypical phenotypic characteristics of Klebsiella Pneumoniae isolates from an outbreak in a neonatal intensive care unit in Brazil. Brazil. J. Microbiol. 38, 273\\u2013277. doi: 10.1590/s1517-83822007000200016\\nCrossRef Full Text | Google Scholar\\nPalmer, M., de Maayer, P., Poulsen, M., Steenkamp, E. T., van Zyl, E., Coutinho, T. A., et al. (2016). Draft genome sequences of pantoea agglomerans and pantoea vagans isolates associated with termites. Stand Genomic Sci. 11:23. doi: 10.1186/s40793-016-0144-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRen, Y., Ren, Y., Zhou, Z., Guo, X., Li, Y., Feng, L., et al. (2010). Complete genome sequence of Enterobacter Cloacae subsp. cloacae type strain Atcc 13047. J. Bacteriol. 192, 2463\\u20132464. doi: 10.1128/JB.00067-10\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRitchie, D. F., and Klos, E. J. (1976). Isolation of Erwinia amylovora bacteriophage from aerial parts of apple trees. Phytopathology 67, 101\\u2013104.\\nGoogle Scholar\\nSebaihia, M., Bocsanczy, A. M., Biehl, B. S., Quail, M. A., Perna, N. T., Glasner, J. D., et al. (2010). Complete genome sequence of the plant pathogen Erwinia amylovora Strain Atcc 49946. J. Bacteriol. 192, 2020\\u20132021. doi: 10.1128/JB.00022-10\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nShaburova, O. V., Hertveldt, K., de la Crus, D. M., Krylov, S. V., Pleteneva, E. A., Burkaltseva, M. V., et al. (2006). Comparison of new giant bacteriophages Obp and Lu11 of soil pseudomonads with bacteriophages of phikz-supergroup of Pseudomonas Aeruginosa. Genetika 42, 1065\\u20131074.\\nPubMed Abstract | Google Scholar\\nSharma, R., Berg, J. A., Beatty, N. J., Choi, M. C., Cowger, A. E., and Cozzens, B. J. R. (2018). Genome sequences of nine Erwinia amylovora bacteriophages. Microbiol. Resour. Announc. 7:e00944-18. doi: 10.1128/MRA.00944-18\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nShivaswamy, V. C., Kalasuramath, S. B., Sadanand, C. K., Basavaraju, A. K., Ginnavaram, V., Bille, S., et al. (2015). Ability of bacteriophage in resolving wound infection caused by multidrug-resistant Acinetobacter Baumannii in uncontrolled diabetic rats. Microb. Drug Resist. 21, 171\\u2013177. doi: 10.1089/mdr.2014.0120\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSimoliunas, E., Kaliniene, L., Truncaite, L., Zajanckauskaite, A., Staniulis, J., Kaupinis, A., et al. (2013). Klebsiella phage Vb_Klem-Rak2 - a giant singleton virus of the family myoviridae. PLoS One 8:e60717. doi: 10.1371/journal.pone.0060717\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSmith, K. C., Castro-Nallar, E., Fisher, J. N., Breakwell, D. P., Grose, J. H., and Burnett, S. H. (2013). Phage cluster relationships identified through single gene analysis. BMC Genomics 14:410. doi: 10.1186/1471-2164-14-410\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSmits, T. H., Rezzonico, F., Kamber, T., Goesmann, A., Ishimaru, C. A., Stockwell, V. O., et al. (2010). Genome sequence of the biocontrol agent pantoea vagans strain C9-1. J. Bacteriol. 192, 6486\\u20136487. doi: 10.1128/JB.01122-10\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSokolova, O. S., Shaburova, O. V., Pechnikova, E. V., Shaytan, A. K., Krylov, S. V., Kiselev, N. A., et al. (2014). Genome packaging in El and Lin68, two giant phikz-like bacteriophages of P. Aeruginosa. Virology 468-470, 472\\u2013478. doi: 10.1016/j.virol.2014.09.002\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSulakvelidze, A. (2005). Phage therapy: an attractive option for dealing with antibiotic-resistant bacterial infections. Drug Discov. Today 10, 807\\u2013809. doi: 10.1016/s1359-6446(05)03441-0\\nCrossRef Full Text | Google Scholar\\nSullivan, M. B., Coleman, M. L., Weigele, P., Rohwer, F., and Chisholm, S. W. (2005). Three Prochlorococcus cyanophage genomes: signature features and ecological interpretations. PLoS Biol. 3:e144. doi: 10.1371/journal.pbio.0030144\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nThomas, J. A., Hardies, S. C., Rolando, M., Hayes, S. J., Lieman, K., Carroll, C. A., et al. (2007). Complete genomic sequence and mass spectrometric analysis of highly diverse, atypical bacillus thuringiensis phage 0305phi8-36. Virology 368, 405\\u2013421. doi: 10.1016/j.virol.2007.06.043\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nThomas, J. A., Rolando, M. R., Carroll, C. A., Shen, P. S., Belnap, D. M., Weintraub, S. T., et al. (2008). Characterization of Pseudomonas Chlororaphis myovirus 201varphi2-1 via genomic sequencing, mass spectrometry, and electron microscopy. Virology 376, 330\\u2013338. doi: 10.1016/j.virol.2008.04.004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nThomson, S. V. (2000). Epidemiology of Fire Blight. Fire Blight; the Disease and Its Causative Agent, Erwinia Amylovora, ed. J. Vanneste (Wallingford: CABI Publishing).\\nGoogle Scholar\\nTwort, F. W. (1915). An investigation on the nature of ultra-microscopic viruses. Lancet 186, 1241\\u20131243. doi: 10.1016/s0140-6736(01)20383-3\\nCrossRef Full Text | Google Scholar\\nYagubi, A. I., Castle, A. J., Kropinski, A. M., Banks, T. W., and Svircev, A. M. (2014). Complete genome sequence of Erwinia amylovora bacteriophage Vb_Eamm_Ea35-70. Genome Announc.2:e00413-14. doi: 10.1128/genomeA.00413-14\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYamada, T., Satoh, S., Ishikawa, H., Fujiwara, A., Kawasaki, T., Fujie, M., et al. (2010). A jumbo phage infecting the phytopathogen Ralstonia solanacearum defines a new lineage of the Myoviridae family. Virology 398, 135\\u2013147. doi: 10.1016/j.virol.2009.11.043\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYuan, Y., and Gao, M. (2016). Proteomic analysis of a novel bacillus jumbo phage revealing glycoside hydrolase as structural component. Front. Microbiol. 7:745. doi: 10.3389/fmicb.2016.00745\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nZuckerkandl, E., and Pauling, L. (1965). \\u201cEvolutionary Divergence and Convergence in Proteins,\\u201d in Evolving Genes and Proteins, eds V. Bryson and H. J. Vogel (New York, NY: Academic Press).\\nGoogle Scholar\\nZwet, T. V. D., and Keil, H. L. (1979). Fireblight: A Bacterial Disease of Rosaceous Plants. Washington, DC: USDA.\\nGoogle Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00202\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00202/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:35.867Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00202\",\n",
      "            \"title\": \"Frontiers | Identification and Characterization of T5-Like Bacteriophages Representing Two Novel Subgroups from Food Products\",\n",
      "            \"description\": \"During recent years, interest in the use of bacteriophages as biocontrol agents against foodborne pathogens has increased, particularly for members of the family Enterobacteriaceae, with pathogenic Escherichia coli, Shigella and Salmonella strains among them. Here, we report the isolation and characterisation of 12 novel T5-like bacteriophages from confiscated food samples. All bacterophages effectively lysed E. coli K-12 strains with an average burst size of 40 PFU/cell and were able to infect pathogenic E. coli strains representing enterohaemorrhagic (EHEC), enteropathogenic (EPEC), enterotoxigenic (ETEC) and enteroinvasive (EIEC) pathotypes, Shigella dysenteriae, S. sonnei strains, as well as multidrug-resistant (MDR) E. coli and multiple strains representing different Salmonella enterica serovars. All the bacteriophages exhibited Siphoviridae morphology. Whole genome sequencing of the novel T5-like bacteriophages showed that they represent two distinct groups, with the genome-based grouping correlating to the different host spectra. As these bacteriophages are of food origin, their stability and lack of any virulence genes, as well as their broad and mutually complementary host spectrum makes these new T5-like bacteriophages valuable candidates for use as biocontrol agents against food-borne pathogenic enterobacteria.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nSeveral members of the Enterobacteriaceae family are considered significant foodborne pathogens, with enterohemorrhagic Escherichia coli (EHEC) and Shigella strains capable of causing life-threatening disease even at a very small infectious dose (reviewed in Croxen et al., 2013; Anderson et al., 2016). Many of these infections are treated with antibiotic therapy, but this is threatened by the increasing occurrence of multidrug-resistant (MDR) strains (reviewed in Szmolka and Nagy, 2013). The development of novel antibiotics has significantly slowed down in the past decades, as it is becoming increasingly hard to find agents with new effect mechanisms or targets, and also because of limited economic returns (Hunter, 2012). More recently, attention has turned on the development of \\u201cevolution-proof\\u201d antibiotics (Bell and MacLean, 2017) and as an alternative answer to this challenge, there is also a renewed interest in the study of bacteriophages capable of lysing pathogenic bacteria.\\nAs biocontrol agents, phages can be applied either as a treatment of ongoing infections (phage therapy) as in the case of Staphylococcus aureus strains, or for foodborne pathogens, the treatment of alimentary food as a preventive measure (reviewed by Johnson et al., 2008 and Hagens and Loessner, 2010). There are already several examples of commercially available phage cocktails to be utilized against various foodborne pathogens. There is a phage cocktail aimed against E. coli O157 to be applied to food surfaces (Carter et al., 2012), as well as cocktails against Shigella (Soffer et al., 2017), Listeria monocytogenes and Salmonella enterica (reviewed by Abedon, 2017). There have also been clinical trials of phage cocktails against diarrheagenic E. coli on human subjects (Sarker et al., 2012, 2016), but these did not yield results regarding the efficacious use of phage cocktails for such infections.\\nBacteriophage T5 was one of the first enterobacterial phages to be characterized in-depth in the 1950s, and since then several similar bacteriophages have been reported. Their most unique feature is a two-step DNA transfer during infection (reviewed in Davison, 2015). Later, partly because of the renewed interest of utilizing bacteriophages as antibacterial agents, the bacteriophage T5 genome was fully sequenced (Wang et al., 2005) and new members of this bacteriophage group have also been characterized (Kim and Ryu, 2011; Golomidova et al., 2015). All T5 bacteriophages seem to share a common genomic structure where pre-early, early and late regions can be distinguished depending on the time of transcription during the infectious cycle (reviewed by Davison, 2015). In recent studies, several T5-like bacteriophages were proposed as potential candidates for use against various foodborne pathogens, including Salmonella enterica serovar Typhimurium (Kim and Ryu, 2011; Piya et al., 2015) as well as against E. coli O157:H7 strains (Niu et al., 2012; Hong et al., 2014), with promising in vivo results produced in live sheep with the phage CEV2 (Raya et al., 2011).\\nIn a previous study, we assessed the risk posed by foodborne pathogens present in illegally imported foodstuff in Europe, with a special emphasis on STEC (Nagy et al., 2015). We hypothesized that bacteriophages capable of lysing foodborne pathogenic bacteria should be also capable of surviving and replicating in the same food. Therefore, a potent source for bacteriophage candidates to be applied as biocontrol against foodborne bacteria could be the very same food samples. As pointed out by Abedon (2017), studies aiming toward phage therapy tend to focus on successes on therapeutic application, and neglect in-depth characterisation of phages. Studies on the potential therapeutic application of phages against enterobacterial infections (Sarker et al., 2016) suggest that a preventive approach, i.e., prior eradication of pathogens from food with phages before consumption could be a more succesful application.\\nIn light of these considerations we aimed to isolate bacteriophages from the food samples that are capable of lysing foodborne enteric pathogens to extensively characterize their host spectrum with other phenotypic features, and to characterize their genomes. This in-depth characterisation would serve to determine whether the isolated phages can be considered to serve as biocontrol agents against fooborne pathogenic Enterobacteria.\\nUsing E. coli K-12 strains for propagation, we isolated 12 new T5-like lytic bacteriophages from individual food samples, and proceeded with their detailed characterisation. Whole genome sequence analysis revealed that these bacteriophages do not harbor undesirable virulence-related genes. They represent two distinct new genotypes among T5-like bacteriophages and proved to be capable of lysing a wide array of pathogenic E. coli, Shigella, and Salmonella strains.\\nMaterials and Methods\\nIsolation of Bacteriophages\\nBacteriophages were isolated from food samples confiscated on the Hungarian border by the customs agents. Samples underwent the first steps of the ISO 16654:2001 method for isolating E. coli O157. Briefly, 5 g of the food samples were homogenized in 10 volumes of tryptic soy broth (TSB) supplemented with bile salts, and incubated for 24 h at 42\\u00b0C. Bacterium-free samples of these precultures were spread or spotted onto layered 0.7% soft agar plates containing the E. coli K-12 derivative C600 as propagating strain, as described by Strauch et al. (2001). After overnight incubation at 37\\u00b0C, single plaques were picked and purified by re-propagation on E. coli C600 as well as at E. coli MG1655 at least three times, until high titer (\\u22651011 PFU/ml) bacteriophage stocks were produced. The list of the isolated bacteriophages and their origin is given in Table 1.\\nTable 1. Origin and subgrouping of new T5-like bacteriophages.\\nTesting Host Specificity and Efficiency of Plating\\nThe host specificity of the isolated bacteriophages was investigated by spot assays on the strains listed in Table 2 as described before. In addition to reference strains and strains from our strain collection, extended-spectrum \\u03b2-lactamase (ESBL)-producing multidrug-resistant (MDR) E. coli strains isolated from human clinical samples were investigated. MDR strains were isolated at the University of Debrecen and Salmonella strains were kindly provided by L\\u00e1szl\\u00f3 Makrai (University of Veterinary Medicine, Budapest). Shigella flexneri M90T was kindly contributed by Zolt\\u00e1n Tigyi (University of P\\u00e9cs). Enteroinvasive E. coli (EIEC) and Shigella strains (except for S. sonnei 866-F) originated from the Hungarian National Collection of Medical Bacteria (HNCMB), Budapest.\\nTable 2. Host specificity and efficiency of plating of the two subgroups of new T5-like bacteriophages.\\nThe ESBL resistant E. coli strains were isolated from human extraintestinal infections, the Shigella flexneri strain was isolated from human feces and the Salmonella strains were isolated from food or from animals. These isolates were identified biochemically and were serotyped using O specific immunsera (data not shown).\\nEfficiency of plating (EOP) was determined by applying serial dilutions of bacteriophage suspensions in spot assays. The ratio of bacteriophage titer on each strain to the titer measured on E. coli MG1655 was considered the EOP of the bacteriophage on the given strain.\\nOne Step Growth Experiments\\nOne step growth experiment to determine burst size was conducted on the representative bacteriophages chee24 and chee130_1 on E. coli MG1655, according to the description of E. Bassiri1 with some modifications. Briefly, 5 \\u00d7 108 bacteria were mixed with 5 \\u00d7 106 bacteriophages in Luria broth (LB), setting the multiplicity of infection (MOI) to 0.01. The mixture was incubated for 10 min at room temperature for initial adsorption, then diluted 104-fold, and in a total volume of 10 ml incubated at 37\\u00b0C for 1 h. Hundred microliter aliquots were taken every 5 min and plated on layered soft agar for counting. Experiments were run at three independent times in two parallels each. Burst size was determined as a ratio of average bacteriophage count of the baseline and the average bacteriophage count after the burst.\\nHeat Stability Tests\\nHeat stability of the bacteriophages was tested as follows: 1 ml stocks containing a 100-fold dilution of a bacteriophage stock in LB were incubated for 1 h at 25\\u00b0C, 37\\u00b0C, 42\\u00b0C, and 80\\u00b0C. After incubation, the titer of the treated bacteriophage stocks was determined with spot assay on E. coli MG1655. Experiments were performed in two parallels on each temperature.\\npH Tolerance Tests\\nStability of the bacteriophages under different pH values was tested by incubating 100-fold dilutions of the bacteriophages for 1 h at 37\\u00b0C in 1 ml LB pH-adjusted to 3, 5, 7, and 9 with HCl and NaOH solutions. After incubation, the titer of the treated bacteriophage stocks was determined with spot assay on E. coli MG1655. Experiments were performed in two parallels on each pH value.\\nDetermination of Bacteriophage Morphology\\nBacteriophages were investigated by transmission electron microscopy (TEM). Briefly, drops of a high titer bacteriophage suspensions were placed on parafilm, absorbed onto carbon film, washed in TE buffer (10 mM TRIS, 1 mM EDTA, pH 6.9) and negatively-stained with 2% aqueous uranyl acetate, pH 5.0. Carbon film was collected with 300 mesh copper grids and access negative-stain was removed with filter paper and subsequentely air-dried. Samples were examined in a TEM 910 transmission electron microscope (Carl Zeiss, Oberkochen) at an acceleration voltage of 80 kV. Images were recorded digitally at calibrated magnifications with a Slow-Scan CCD-Camera (ProScan, 1024 \\u00d7 1024, Scheuring, Germany) with ITEM-Software (Olympus Soft Imaging Solutions, M\\u00fcnster, Germany). Contrast and brightness were adjusted with Adobe Photoshop CS3.\\nBacteriophage DNA Isolation\\nBacteriophage DNA was isolated from bacteriophage stocks with a concentration of at least 1011 PFU/ml. The phenol-chloroform method described by Sambrook et al. (1987) was used for isolation with the modifications outlined by T\\u00f3th et al. (2016). Briefly, to remove any non-bacteriophage related DNA and RNA from the 700 \\u03bcl bacteriophage suspension, the sample was treated with amplification grade DNase I and RNase A (Sigma Aldrich) in 10 \\u03bcg/ml final concentration for 30 min at 37\\u00b0C. For bacteriophage lysis, proteinase K (Sigma Aldrich) was added in 6.6 \\u03bcg/ml final concentration to the lysis buffer (50 mM Tris\\u2013HCl (pH 8.0), 10 mM EDTA, 0.5% SDS) and incubated at 65\\u00b0C for 30 min. The suspension was cooled to room temperature and mixed with 750 \\u03bcl of a 1:1 mixture of equilibrated phenol and chloroform. After 5 min incubation at room temperature, proteins were removed by centrifugation at 10,000 \\u00d7 g for 5 min. The aqueous phase was transferred to a clean Eppendorf tube and bacteriophage DNA was precipitated after adding 0.1 volume of 3 M potassium acetate, pH 5.5 with 0.7 volume of isopropanol at ice for 20 min. DNA was collected by centrifugation at 13,000 \\u00d7 g at 10 min, washed with 600 \\u03bc1 70% ethanol and dissolved in 30 \\u03bcl 10 mM Tris\\u2013HCl, 1 mM EDTA, pH 8.0 buffer.\\nBacteriophage Genome Sequencing and Analysis\\nGenomic DNA sequencing libraries were prepared using the Nextera XT kit (Illumina, Eindhoven, NL). Sequencing was performed using Nextseq Mid-output reagent kit v2 (2 \\u00d7 150 bp) on an Illumina NextSeq 500. Assembly was performed with SPAdes (Bankevich et al., 2012). The genome was annotated using the RAST server (Overbeek et al., 2014). The genes orf5c (T5.035) and the bacteriophage DNA polymerase (T5.122) were used for phylogenetic analysis. The DNA sequences of these two genes from all bacteriophages determined in this study and of T5 reference genomes (Supplementary Table 1) were aligned using Clustal Omega (Sievers et al., 2011). The phylogenetic tree was visualized using MEGA5.2 (Hall, 2013). For whole-bacteriophage sequence phylogeny, the program VICTOR (Meier-Kolthoff and Goeker, 2017) was used with default settings and all T5 reference genomes (Supplementary Table 1). Determination of sequence homologies and the identification of SNPs were conducted by using the BLAST tools available at the NCBI website. Genome alignments were visualized with Easyfig (Sullivan et al., 2011) and modified using Inkscape. The left and right repeats were defined using a pile-up analysis of raw sequencing reads after mapping with CLC workbench 9.0 (Qiagen, Hilden, Germany). The repeats were defined as >130% of average coverage at the particular site (30% more coverage than average coverage).\\nSequence Accession Numbers\\nThe whole genome sequences of all bacteriophages determined in the current study were deposited in GenBank database under accession nos. MF431730- MF431741.\\nResults\\nIsolation of Bacteriophages\\nBy using E. coli K-12 derivative strains MG1655 and C600 as indicator and propagating strain, we isolated 12 new bacteriophages from different food samples. The origin, as well as the subgrouping of the bacteriophages are summarized in Table 1. The bacteriophages were isolated from either meat, meat products (e.g., sausages) or cheese.\\nMorphology\\nAll the new isolated bacteriophages showed Siphoviridae morphology, with an average head length of ~85 nm, head width of ~75 nm and a flexible, non-contractile tail with a length of ~200 nm. As a representative, the morphology of T5-like bacteriophage chee24 is shown (Figure 1).\\nFigure 1. Transmission electron micrograph of bacteriophage 24 showing Siphoviridae morphology.\\nHost Spectrum, Efficiency of Plating, Burst Size, and Stability\\nTo assess the host specificity of the bacteriophages, we tested their lytic capacity on a large variety of enterobacterial pathogenic and ESBL multidrug-resistant clinical strains listed in Table 2. We found that the novel T5-like bacteriophages showed two distinct lysis patterns, which we denote by a representative bacteriophage of each: bacteriophage chee24 (group chee24) and bacteriophage chee130_1 (group chee130_1). The grouping of the bacteriophages is indicated in Table 1. EOP values were determined by spot assay for these two bacteriophages. The lysis spectrum and EOP values are given in Table 2.\\nApart from K-12 derivative strains, several E. coli, Shigella, and Salmonella strains representing different pathotypes and serovars proved to be susceptible to the bacteriophages. While bacteriophage chee130_1 lysed enteropathogenic E. coli (EPEC), EIEC, and Shigella sonnei strains, bacteriophage 24 induced lysis on Salmonella serovars, EHEC O103:H2, and Shigella dysenteriae strains. Both bacteriophages showed lysis on enterotoxigenic E. coli (ETEC) type strain H10407. It has to be noted however, that in each case EOP values were orders of magnitude lower when compared to K-12 strains, and on some wild type strains the plaque morphology was opaque, unlike the clear plaques of ~2 mm in diameter, observed with the strain MG1655.\\nBurst size was determined with one step growth experiments on E. coli MG1655 (Figure 2). The burst sizes of bacteriophage chee24 and chee130_1 proved to be different, in the case of the former, it was around 1,000 PFU/cell, while in the case of chee130_1, the burst size was around 44 PFU/cell. In both cases the burst occurred ~45 min after the end of the initial absorption.\\nFigure 2. One step growth curves of new T5-like bacteriophages on E. coli MG1655 strain. PFU values of bacteriophage chee24 are marked with triangles, those of bacteriophage chee130_1 are marked with squares. The error bars indicate standard deviations from the results of the three independent triplicate experiments.\\nHeat stability experiments showed that incubation of 1010 PFU bacteriophage stocks at 25\\u00b0C, 37\\u00b0C and 42\\u00b0C for 1 h did not significantly affect the EOP of the bacteriophages (data not shown). Incubation at 80\\u00b0C, however, caused 104-fold average decrease in the bacteriophage titer.\\nA pH value of 3 completely deactivated the bacteriophages, as no plaques were observable even in the spots of the concentrated suspensions. On the other hand, no significant decrease in titer could be observed at pH values 5, 7, and 9 after 1 h incubation (data not shown).\\nWhole Genome Sequence Analysis\\nTo examine the genomic structures and the potential diversity in the genomes, all the 12 newly isolated bacteriophage genome sequences were determined and detailed genomic comparison was conducted. Genome sequencing of the bacteriophages showed that all the bacteriophages have a linear double-stranded DNA genome, and belong to T5-like bacteriophages. The two subgroups with different host specificity also have minor genetic differences. The genome of the \\u201cgroup chee24\\u201d bacteriophages is 120,618 (bacteriophage pork27), 120,620 (bacteriophage saus111K), 120,622 (bacteriophage chee24, pork29 and saus47N) or 120,629 bp (bacteriophage poul124) long, and contains 165 (saus47N, saus111K, poul124), 166 (chee24, pork29) or 168 (pork27) protein-encoding CDS, as well as 18 tRNA genes, respectively. Its GC content is 39.3%. On the other hand, the genome size of the \\u2018group chee130_1\\u2019 bacteriophages is 121,986 bp, with 164 (saus132, poul149), 165 (cott162) or 167 (chee130_1, chee158, saus176N) protein-encoding CDS, 20 tRNA genes, and a GC content of 39.7%. These groups were congruent with the different host spectrum observable within the bacteriophages, indicating that the chee24-like and chee130_1-like bacteriophages form distinct genetic groups. The list of ORFs compared to each other and to the original T5 bacteriophage (GenBank AY543070.1) are shown in Supplementary Tables 2, 3. Members of both subgroups showed the characteristic functional regions in their genomes, which is a common feature of T5-like phages, having pre-early, early and late genomic regions named after the time of their transcription, as well as a terminal repeat region on the 5\\u2032 and 3\\u2032 end, which is 9,968 bp long in chee24-like phages, and 8,717 bp in group chee130_1 phages. For group chee24 bacteriophages, the bacteriophage pork27 differed by 1 SNP in the terminal repeat, while for group chee130_1 phages, all terminal repeats of other than chee130_1 differed from those present in the prototype, with saus132, chee158, cott162 and saus176N having 3 SNPs, and poul149 harboring 4 SNPs.\\nAt the nucleotide level, the newly identified T5-like bacteriophages not only have the same genome size within the two subgroups\\u2014group chee24 and group chee130_1-, but when compared to the \\u201ctype bacteriophage\\u201d of their groups as reference bacteriophage chee24 or chee130_1, respectively, they differ in only 2\\u20135 nucleotide positions. It has to be noted that in the case of bacteriophage pork27 and saus47N, one SNP causes a premature stop codon in bacteriophage protein genes of unknown function. In the case of bacteriophages saus132, poul149 and chee158 there are premature stop codons in a bacteriophage tail fiber gene. A summary of these genomic locations is given in Table 3.\\nTable 3. Single nucleotide polymorphisms shown by T5-like bacteriophages of the chee24-like subgroup (A) and the chee130_1-like subgroup (B) when compared to the type bacteriophage of each subgroup.\\nNone of the new characterized bacteriophage genomes contained any known virulence, toxin, antibiotic resistance-encoding and virulence regulator genes.\\nPhylogenetic Analysis\\nTo reveal the phylogenetic position of bacteriophages chee24 and chee130_1 among T5-like phages, several comparisons were conducted. Comparison based on the genes orf5c (T5.035) and the bacteriophage DNA polymerase (T5.122) of bacteriophage T5 (AY692264.1) showed that while bacteriophage 24 proved to be the closest relative of E. coli bacteriophages DT57C and DT571/2, bacteriophage chee130_1 are closely related to the Salmonella bacteriophages Stitch and SPC35 (Figure 3).\\nFigure 3. Phylogenetic tree of T5-like bacteriophages based on the Clustal Omega alignment of the genes orf5c (T5.035, A) and the phage DNA polymerase (T5.122, B) of bacteriophage T5 (AY692264.1). The scale represents homology %. The list of included T5-like phage genomes with their GenBank accession numbers is given in Supplementary Table 1.\\nWhole-genome based phylogeny of all T5-like bacteriophages clearly indicated that the phages isolated in the current study are members of the T5-like bacteriophages (Figure 4). The topology of this tree is entirely different from the one based on the single genes, putting the two representative phages next to the phage genomes that also proved closest homologs with the BLAST analysis. An alignment of T5-like genomes with mutual homologies including representatives of the two subgroups is presented in Figure 5, showing the 5\\u2032 end of the late region as the most variable, which contains mainly tail fiber genes.\\nFigure 4. Phylogenetic tree of T5-like bacteriophages made with the whole genome-based VICTOR analysis. The scale represents homology %. The list of included T5-like phage genomes with their GenBank accession numbers is given in Supplementary Table 1.\\nFigure 5. Alignment of T5-like bacteriophage whole genomes. GenBank accession numbers of the phages are given in Supplementary Table 1.Arrows with different colors represent ORFs associated with the genomic regions indicated at the top. The gray bar in the lower right corner shows the identity percentage associated with the color of the bars connecting homologous regions.\\nComparing the genomes of bacteriophage chee24 and chee130_1at the nucleotide-level, regions showing low homology to each other were the tail fiber region and an open reading frame encoding a bacteriophage-associated receptor binding protein gene. All these features seem to contribute to the differences observed in the host specificity of the two bacteriophage groups. At nucleotide level, the closest homolog to bacteriophage chee24 is bacteriophage SPC35 (GenBank no. HQ406778.). In addition, two hypothetical proteins, a thymidylate synthase gene, and two subunits of a HNH endonuclease, a receptor binding protein (CDS ADW80123.1) of SPC35 is also absent in chee24, suggesting different host specificity. Bacteriophage EPS7 (GenBank no. CP000917) is the closest homolog to bacteriophage chee130_1 of the previously sequenced T5-like bacteriophages, and carries homologs to all receptor-associated proteins, albeit the similarity is not 100% in each case. Furthermore, chee130_1 lacks a few hypothetical proteins and AGC_0146 putative tail protein as well. Genes showing low homology and those that are absent from bacteriophages of either the chee24 or chee130_1subgroup are listed in Table 4.\\nTable 4. List of genes and regions showing low homology or missing in chee24-like (A) and chee130_1-like (B) bacteriophages when compared to their closest homolog bacteriophages.\\nDiscussion\\nUsing E. coli K-12 C600 strain as the host bacterium, we isolated and characterized 12 novel T5-like bacteriophages from independent food samples of various animal origin from around the world These novel bacteriophages represented two subgroups based on their host specificity. While the most effective host of the bacteriophages were K-12 derivative E. coli strains, several strains representing important intestinal pathotypes of E. coli, Shigella, and Salmonella also proved susceptible to the bacteriophages. The Siphoviridae morphology and the genomic characteristics including the genome size, the genome architecture, the low GC content and the sequence homologies showed that the bacteriophages are T5-like bacteriophages.\\nAn important feature of these newly found phages is that they were isolated from food samples, unlike the previously isolated T5-like bacteriophages, which originated from sewage and fecal sources (Kim and Ryu, 2011; Golomidova et al., 2015). There is also one case where a T5-like phage was isolated and characterized from a commercially available bacteriophage cocktail designed against enteric pathogens (Piya et al., 2015). Our results indicate that food can be a rich source of bacteriophages effective against pathogenic bacteria potentially residing in the same environment. These phages maintain their infective capacity in this environment suggesting that foodborne phages are promising biocontrol candidates against foodborne bacteria.\\nOur results also show the huge diversity of bacteriophages, even from well-characterized groups such as T5-like bacteriophages, as both the chee24 and chee130_1 subgroups are different from the closest related bacteriophages. Our data indicate that the most crucial differences are within genes encoding receptor-binding proteins (Table 4, Figure 5). The closest colinear relative of bacteriophage chee24 is SPC35, which was reported to lyse Salmonella Typhimurium and Enteritidis strains, as well as a large set of K-12 derivative and commensal E. coli (Kim and Ryu, 2011). However, the host spectrum of group chee24 phages is particularly broad, and includes a strain representing one of the so-called \\u201cbig six\\u201d serotypes of EHEC (O103:H2), as well as an ESBL E. coli strain of serogroup O73 isolated from an extraintestinal infection. The differences in ORFs 145 and 149, annotated as \\u201cphage tail fibers\\u201d and \\u201creceptor binding protein\\u201d could account for the discrepancy.\\nWe wanted to test the efficiency of the new T5-like phages on target strains, and the one step growth on E. coli MG1655 yielded burst sizes comparable to those of the T5-like bacteriophage phiLLS (Amarillas et al., 2017).\\nThe closest homolog of the chee130_1 subgroup, EPS7 is able to lyse the same Salmonella serovars and seven unnamed E. coli strains (Hong et al., 2008), however, phage chee130_1 did not lyse Salmonella strains. On the other hand, phage chee130_1 was able to propagate on Shigella sonnei, EIEC and EPEC strains. All of these differences in host range underline the potential role that genes showing low homology to closely related bacteriophages (Table 4) might have importance in host specificity. In the case of chee130_1, ORF134 that is annotated as \\u201cphage tail fibers\\u201d has very low homology to the corresponding gene in EPS7 and seems to support this notion. It is interesting to note however, that in ORF133, another gene annotated as phage tail fiber, there are premature stop codons in the case of bacteriophages saus132, poul149 and chee158. These mutations nevertheless did not seem to have any effect on the bacterial host specificity of these bacteriophages when compared to other members of their subgroup (data not shown).\\nLee et al. (2016) suggested that besides bacteriophage cocktails, a solution for simultaneous biocontrol of several foodborne pathogens could be the cloning of several receptor binding genes into a single bacteriophage. For either purpose, obtaining knowledge about new bacteriophages capable of infecting these pathogens and assessing their host spectrum is essential.\\nBesides the above-cited studies of promising candidate phages to be used as biocontrol agents against pathogenic bacteria, there are already several commercially available phage cocktails aimed at various foodborne enterobacterial pathogens (reviewed by Abedon, 2017). However, according to our knowledge except for ShigaShield, which is aimed against Shigellae (Soffer et al., 2017), and the Microgen ColiProteus aimed against E. coli and Proteus (McCallin et al., 2013), none of the component phages have been sequenced. In order to assess the applicability of a phage either for biocontrol of for therapy, extensive knowledge of its genome is essential, to determine whether the phage carries \\u201cundesirable\\u201d genes. In-depth knowledge of genes determining host specificity or those contributing to phage stability in different environments are also valuable information when selecting bacteriophages for a given therapeutic or biocontrol purpose.\\nAs it has been noted by Br\\u00fcssow (2012), the development of an actual phage therapy treatment can take years from phage isolation to performing clinical trials. This process includes for exapmle the purification (Bourdin et al., 2014b), as well as characterisation of host specificity (Bourdin et al., 2014a) and testing the biological safety of the phages either in animals (Raya et al., 2011) or human subjects (Sarker et al., 2012, 2016). An extensive genetic database of phages isolated from different sources, representing different genuses and host spectra could speed up the process of selecting phages for clinical trials.\\nIn summary, we identified 12 new T5-like bacteriophages from food samples, which represent two new species within this bacteriophage group. Our study so far shows that they fulfill important criteria listed by Hagens and Loessner (2010) for bacteriophages to be used as biocontrol: their genome sequence was determined, they propagate well on a non-pathogenic host, they lack any virulence, virulence regulator, toxin, and antibiotic resistance genes. They also seem to be strictly lytic and were shown to be stable during storage and application temperatures, as well as a range of pH values usual in most of the food.\\nAll the above-mentioned properties, coupled with their food origin, as well as their broad and partially overlapping, supplementary host spectrum makes the new T5-like bacteriophages valuable candidates as effective bio-control agents against foodborne pathogenic enterobacteria.\\nAuthor Contributions\\nIT and TC conceived and elaborated the study. DS, LF, MR, JS, and IT conducted the phenotypic experiments. LF performed sequencing and provided bioinformatical tools. DS, LF, TC, and IT analyzed the results and composed the manuscript. All authors reviewed the manuscript.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nThis work was supported by the 7th EU Framework Programme PROMISE project (project n. 265877), the Hungarian National Research, Development and Innovative Office (NKFIH, grant 124335) and grants from the Bundesministerium fuer Bildung und Forschung (BMBF, Germany) within the German Center for Infection research (DZIF/grant number 8032808811 to TC).\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2018.00202/full#supplementary-material\\nFootnotes\\nReferences\\nAllu\\u00e9-Guardia, A., Garc\\u00eda-Aljaro, C., and Muniesa, M. (2011). Bacteriophage-encoding cytolethal distending toxin type V gene induced from nonclinical Escherichia coli isolates. Infect. Immun. 79, 3262\\u20133272. doi: 10.1128/IAI.05071-11\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAmarillas, L., Rub\\u00ed-Rangel, L., Chaidez, C., Gonz\\u00e1lez-Robles, A., Lightbourn-Rojas, L., and Le\\u00f3n-F\\u00e9lix, J. (2017). Isolation and characterization of phiLLS, a novel phage with potential biocontrol agent against multidrug-resistant Escherichia coli. Front. Microbiol. 8:1355. doi: 10.3389/fmicb.2017.01355\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAnderson, M., Sansonetti, P. J., and Marteyn, B. S. (2016). Shigella diversity and changing landscape: insights for the twenty-first century. Front. Cell. Infect. Microbiol. 6:45. doi: 10.3389/fcimb.2016.00045\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455\\u2013477. doi: 10.1089/cmb.2012.0021\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBlattner, F. R., Plunkett, G., Bloch, C. A., Perna, N. T., Burland, V., Riley, M., et al. (1997). The complete genome sequence of Escherichia coli K-12. Science 277, 1453\\u20131462. doi: 10.1126/science.277.5331.1453\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBourdin, G., Navarro, A., Sarker, S. A., Pittet, A.-C., Qadri, F., Sultana, S., et al. (2014a). Coverage of diarrhoea-associated Escherichia coli isolates from different origins with two types of phage cocktails. Microb. Biotechnol. 7, 165\\u2013176. doi: 10.1111/1751-7915.12113\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBourdin, G., Schmitt, B., Marvin Guy, L., Germond, J.-E., Zuber, S., Michot, L., et al. (2014b). Amplification and purification of T4-like Escherichia coli phages for phage therapy: from laboratory to pilot scale. Appl. Environ. Microbiol. 80, 1469\\u20131476. doi: 10.1128/AEM.03357-13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCarter, C. D., Parks, A., Abuladze, T., Li, M., Woolston, J., Magnone, J., et al. (2012). Bacteriophage cocktail significantly reduces Escherichia coli O157: H7 contamination of lettuce and beef, but does not protect against recontamination. Bacteriophage 2, 178\\u2013185. doi: 10.4161/bact.22825\\nCrossRef Full Text | Google Scholar\\nCroxen, M. A., Law, R. J., Scholz, R., Keeney, K. M., Wlodarska, M., and Finlay, B. B. (2013). Recent advances in understanding enteric pathogenic Escherichia coli. Clin. Microbiol. Rev. 26, 822\\u2013880. doi: 10.1128/CMR.00022-13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nEvans, D. G., Evans, D. J., and Pierce, N. F. (1973). Differences in the response of rabbit small intestine to heat-labile and heat-stable enterotoxins of Escherichia coli. Infect. Immun. 7, 873\\u2013880.\\nPubMed Abstract | Google Scholar\\nGolomidova, A. K., Kulikov, E. E., Prokhorov, N. S., Guerrero-Ferreira, R. C., Ksenzenko, V. N., Tarasyan, K. K., et al. (2015). Complete genome sequences of T5-related Escherichia coli bacteriophages DT57C and DT571/2 isolated from horse feces. Arch. Virol. 160, 3133\\u20133137. doi: 10.1007/s00705-015-2582-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHagens, S., and Loessner, M. J. (2010). Bacteriophage for biocontrol of foodborne pathogens: calculations and considerations. Curr. Pharm. Biotechnol. 11, 58\\u201368. doi: 10.2174/138920110790725429\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHayashi, T., Makino, K., Ohnishi, M., Kurokawa, K., Ishii, K., Yokoyama, K., et al. (2001). Complete genome sequence of enterohemorrhagic Escherichia coli O157:H7 and genomic comparison with a laboratory strain K-12. DNA Res. 8, 11\\u201322. doi: 10.1093/dnares/8.1.11\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHochhut, B., Wilde, C., Balling, G., Middendorf, B., Dobrindt, U., Brzuszkiewicz, E., et al. (2006). Role of pathogenicity island-associated integrases in the genome plasticity of uropathogenic Escherichia coli strain 536. Mol. Microbiol. 61, 584\\u2013595. doi: 10.1111/j.1365-2958.2006.05255.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHong, J., Kim, K.-P., Heu, S., Lee, S. J., Adhya, S., and Ryu, S. (2008). Identification of host receptor and receptor-binding module of a newly sequenced T5-like phage EPS7. FEMS Microbiol. Lett. 289, 202\\u2013209. doi: 10.1111/j.1574-6968.2008.01397.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHong, Y., Pan, Y., Harman, N. J., and Ebner, P. D. (2014). Complete genome sequences of two Escherichia coli O157:H7 phages effective in limiting contamination of food products. Genome Announc. 2:e00519\\u201314. doi: 10.1128/genomeA.00519-14\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHunter, P. (2012). Where next for antibiotics?: the immune system and the nature of pathogenicity are providing vital clues in the fight against antibiotic-resistant bacteria. EMBO Rep. 13, 680\\u2013683. doi: 10.1038/embor.2012.101\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nIguchi, A., Thomson, N. R., Ogura, Y., Saunders, D., Ooka, T., Henderson, I. R., et al. (2009). Complete genome sequence and comparative genome analysis of enteropathogenic Escherichia coli O127:H6 strain E2348/69. J. Bacteriol. 191, 347\\u2013354. doi: 10.1128/JB.01238-08\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJohnson, R. P., Gyles, C. L., Huff, W. E., Ojha, S., Huff, G. R., Rath, N. C., et al. (2008). Bacteriophages for prophylaxis and therapy in cattle, poultry and pigs. Anim. Health Res. Rev. 9, 201\\u2013215. doi: 10.1017/S1466252308001576\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKim, M., and Ryu, S. (2011). Characterization of a T5-like coliphage, SPC35, and differential development of resistance to SPC35 in Salmonella enterica serovar typhimurium and Escherichia coli. Appl. Environ. Microbiol. 77, 2042\\u20132050. doi: 10.1128/AEM.02504-10\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLee, H., Ku, H.-J., Lee, D.-H., Kim, Y.-T., Shin, H., Ryu, S., et al. (2016). Characterization and genomic study of the novel bacteriophage HY01 infecting both Escherichia coli O157:H7 and Shigella flexneri: potential as a biocontrol agent in food. PLoS ONE 11:e0168985. doi: 10.1371/journal.pone.0168985\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMarch\\u00e8s, O., Ledger, T. N., Boury, M., Ohara, M., Tu, X., Goffaux, F., et al. (2003). Enteropathogenic and enterohaemorrhagic Escherichia coli deliver a novel effector called Cif, which blocks cell cycle G2/M transition. Mol. Microbiol. 50, 1553\\u20131567. doi: 10.1046/j.1365-2958.2003.03821.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMcCallin, S., Sarker, S. A., Barretto, C., Sultana, S., Berger, B., Huq, S., et al. (2013). Safety analysis of a Russian phage cocktail: from MetaGenomic analysis to oral application in healthy humans. Virology 443, 187\\u2013196. doi: 10.1016/j.virol.2013.05.022\\nCrossRef Full Text | Google Scholar\\nMoriel, D. G., Bertoldi, I., Spagnuolo, A., Marchi, S., Rosini, R., Nesta, B., et al. (2010). Identification of protective and broadly conserved vaccine antigens from the genome of extraintestinal pathogenic Escherichia coli. Proc. Natl. Acad. Sci. U.S.A. 107, 9072\\u20139077. doi: 10.1073/pnas.0915077107\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nNagy, B., Szmolka, A., Smole Mo\\u017dina, S., Kova\\u010d, J., Strauss, A., Schlager, S., et al. (2015). Virulence and antimicrobial resistance determinants of verotoxigenic Escherichia coli (VTEC) and of multidrug-resistant E. coli from foods of animal origin illegally imported to the EU by flight passengers. Int. J. Food Microbiol. 209, 52\\u201359. doi: 10.1016/j.ijfoodmicro.2015.06.026\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nNiu, Y. D., Stanford, K., Kropinski, A. M., Ackermann, H.-W., Johnson, R. P., She, Y.-M., et al. (2012). Genomic, proteomic and physiological characterization of a T5-like bacteriophage for control of shiga toxin-producing Escherichia coli O157:H7. PLoS ONE 7:e34585. doi: 10.1371/journal.pone.0034585\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nOverbeek, R., Olson, R., Pusch, G. D., Olsen, G. J., Davis, J. J., Disz, T., et al. (2014). The SEED and the Rapid Annotation of microbial genomes using Subsystems Technology (RAST). Nucleic Acids Res. 42, D206\\u2013D214. doi: 10.1093/nar/gkt1226\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPerna, N. T., Plunkett, G., Burland, V., Mau, B., Glasner, J. D., Rose, D. J., et al. (2001). Genome sequence of enterohaemorrhagic Escherichia coli O157:H7. Nature 409, 529\\u2013533. doi: 10.1038/35054089\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPiya, D., Xie, Y., Hernandez Morales, A. C., and Kuty Everett, G. F. (2015). Complete genome sequence of Salmonella enterica serovar Typhimurium siphophage shivani. Genome Announc. 3:e01443\\u201314. doi: 10.1128/genomeA.01443-14\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRaya, R. R., Oot, R. A., Moore-Maley, B., Wieland, S., Callaway, T. R., Kutter, E. M., et al. (2011). Naturally resident and exogenously applied T4-like and T5-like bacteriophages can reduce Escherichia coli O157:H7 levels in sheep guts. Bacteriophage 1, 15\\u201324. doi: 10.4161/bact.1.1.14175\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSambrook, J., Maniatis, T., Fritsch, E. F., and Laboratory, C. S. H. (1987). Molecular Cloning : A Laboratory Manual, 2nd Edn. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press. Available online at: http://trove.nla.gov.au/work/13615226 (Accessed June 1, 2017).\\nSarker, S. A., McCallin, S., Barretto, C., Berger, B., Pittet, A.-C., Sultana, S., et al. (2012). Oral T4-like phage cocktail application to healthy adult volunteers from Bangladesh. Virology 434, 222\\u2013232. doi: 10.1016/j.virol.2012.09.002\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSarker, S. A., Sultana, S., Reuteler, G., Moine, D., Descombes, P., Charton, F., et al. (2016). Oral phage therapy of acute bacterial diarrhea with two coliphage preparations: a randomized trial in children from Bangladesh. EBioMedicine 4, 124\\u2013137. doi: 10.1016/j.ebiom.2015.12.023\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSievers, F., Wilm, A., Dineen, D., Gibson, T. J., Karplus, K., Li, W., et al. (2011). Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Mol. Syst. Biol. 7, 539\\u2013539. doi: 10.1038/msb.2011.75\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSoffer, N., Woolston, J., Li, M., Das, C., and Sulakvelidze, A. (2017). Bacteriophage preparation lytic for Shigella significantly reduces Shigella sonnei contamination in various foods. PLoS ONE 12:e0175256. doi: 10.1371/journal.pone.0175256\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nStrauch, E., Lurz, R., and Beutin, L. (2001). Characterization of a shiga toxin-encoding temperate bacteriophage of Shigella sonnei. Infect. Immun. 69, 7588\\u20137595. doi: 10.1128/IAI.69.12.7588-7595.2001\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nT\\u00f3th, I., H\\u00e9rault, F., Beutin, L., and Oswald, E. (2003). Production of cytolethal distending toxins by pathogenic Escherichia coli strains isolated from human and animal sources: establishment of the existence of a new cdt variant (Type IV). J. Clin. Microbiol. 41, 4285\\u20134291. doi: 10.1128/JCM.41.9.4285-4291.2003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nT\\u00f3th, I., Schmidt, H., Kardos, G., Lancz, Z., Creuzburg, K., Damjanova, I., et al. (2009). Virulence genes and molecular typing of different groups of Escherichia coli O157 strains in cattle. Appl. Environ. Microbiol. 75, 6282\\u20136291. doi: 10.1128/AEM.00873-09\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nT\\u00f3th, I., Sv\\u00e1b, D., B\\u00e1lint, B., Brown-Jaque, M., and Mar\\u00f3ti, G. (2016). Comparative analysis of the Shiga toxin converting bacteriophage first detected in Shigella sonnei. Infect. Genet. Evol. 37, 150\\u2013157. doi: 10.1016/j.meegid.2015.11.022\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWang, J., Jiang, Y., Vincent, M., Sun, Y., Yu, H., Wang, J., et al. (2005). Complete genome sequence of bacteriophage T5. Virology 332, 45\\u201365. doi: 10.1016/j.virol.2004.10.049\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWiles, S., Clare, S., Harker, J., Huett, A., Young, D., Dougan, G., et al. (2005). Organ-specificity, colonization and clearance dynamics in vivo following oral challenges with the murine pathogen Citrobacter rodentium. Cell. Microbiol. 7, 459\\u2013459. doi: 10.1111/j.1462-5822.2004.00479.x\\nCrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fped.2022.856520\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fped.2022.856520/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:37.185Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fped.2022.856520\",\n",
      "            \"title\": \"Frontiers | Optimisation and Application of a Novel Method to Identify Bacteriophages in Maternal Milk and Infant Stool Identifies Host-Phage Communities Within Preterm Infant Gut\",\n",
      "            \"description\": \"Human milk oligosaccharides, proteins such as lactoferrin, and bacteria represent just some of the bioactive components of mothers\\u2019 breast milk (BM). Bacteriophages (viruses that infect bacteria) are an often-overlooked component of BM that can cause major changes in microbial composition and metabolism. BM bacteriophage composition has been explored in term and healthy infants, suggesting vertical transmission of bacteriophages occurs between mothers and their infants. Several important differences between term and very preterm infants (< 30 weeks gestational age) may limit this phenomenon in the latter. To better understand the link between BM bacteriophages and gut microbiomes of very preterm infants in health and disease, standardised protocols are required for isolation and characterisation from BM. In this study we use isolated nucleic acid content, bacteriophage richness and Shannon diversity to validate several parameters applicable during bacteriophage isolation from precious BM samples. Parameters validated include sample volume required; centrifugal sedimentation of microbes; hydrolysis of milk samples with digestive enzymes; induction of temperate bacteriophages and concentration / purification of isolated bacteriophage particles in donor milk (DM). Our optimised method enables characterisation of bacteriophages from as little as 0.1 mL BM. We identify viral families that were exclusively identified with inclusion of induction of temperate bacteriophages (Inoviridae) and hydrolysis of milk lipid processes (Iridoviridae and Baculoviridae). Once applied to a small clinical cohort we demonstrate vertical transmission of bacteriophages from mothers BM to the gut of very preterm infants at the species level. This optimised method will enable future research characterising the bacteriophage composition of BM in very preterm infants to determine their clinical relevance in the development of a healthy preterm infant gut microbiome.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nIt is widely accepted that mother's breast milk (BM) is the gold standard of nutritional care for newborn infants (1, 2) due to the myriad of beneficial effects of BM for newborn infant growth and development (3, 4). Various studies have characterised the components present in BM, including human milk oligosaccharides (5), bioactive proteins, such as lactoferrin (6, 7), and bacteria (8\\u201310), to elucidate potential modes of the improved health outcomes in infants who receive BM. There are several key differences between term and preterm infants, including length of hospitalisation and exposure of preterms, to necessary clinical interventions that impact microbiome development (11). These include chemical interventions, such as antibiotics and antivirals, or mechanical interventions, such as invasive catheters and feeding tubes, as well as exposure to microbes resident within the neonatal unit environment. In addition to these neonate-specific factors, evidence has emerged of longitudinal change to the nutritional composition of BM particularly the quantity and quality of proteins and lipids. Preterm BM, therefore, differs substantially from that of term BM, both in nutritional and microbial composition.\\nAnother bioactive component of BM is the bacteriophage content. Initial analysis of vertical bacteriophage transmission between BM and the neonatal GI tract performed by Breitbart et al. (12) in 2008 found no shared bacteriophages between the single mother-infant pair studied despite large quantities of identifiable virus in infant stool. Since this pioneering study improvements in metagenomic sequencing technology and bacteriophage genome databases have facilitated further investigations. In 2017, Duranti et al. (13) reported the presence of Bifidobacterium phages in both mother's BM and their infant's stools demonstrating the potential of bacteriophage transmission in term mother-infant pairs. Pannaraj et al. (14) compared alpha and beta diversity of BM and stool samples from a separate cohort of term infants, finding a greater alpha diversity in BM viromes than stools as well as several shared taxonomic features between BM and stool from the same mother-infant pairs. However, these taxonomic analyses were capped at the family level which limits our ability to determine whether there is a transmission of bacteriophages from mother to infant via BM. A comparison between the virome of BM given to infants delivered at term or preterm found contrasting taxonomic compositions (15). This study identified several bacteriophage species with host specificity for common skin commensals. Preterm infants enrolled in the study were a minimum of 32 weeks gestation at birth although BM bacteriophages were not compared with infant stool bacteriophages. A knowledge gap still exists to elucidate bacteriophage transmission between BM and neonatal stool in very preterm (<30 weeks gestation) neonates.\\nAlthough bacteriophages have now been revealed as one of the active components of BM, understanding their full impact in preterm BM, including their interactions with microbes in the preterm infant GIT, remains enigmatic. A major obstacle to overcome when interpreting the results of each study investigating bacteriophages in BM is the methodological differences between studies (12, 14, 16). Indeed, some methods appear to have limitations that lead to selection bias during viral isolation (17). For instance, the application of glass beads during sample homogenisation can reduce non-enveloped bacteriophages, membrane filtration with 0.2 and/or 0.45 \\u03bcM filters may fail to recover giant viruses, and caesium chloride (CsCl) density-gradient ultracentrifugation fails to recover enveloped bacteriophages or those within particular buoyant densities (17). Such contrasting approaches in the literature prove an even greater issue when investigating preterm infant microbiome because sample volumes are so small and are therefore extremely precious.\\nThis study aims to develop a robust method for isolation of bacteriophages from the smallest possible volumes of BM from mothers to very preterm infants to promote accurate characterisation of these communities in future studies. We utilize pasteurised donor milk (DM) to validate bacteriophage isolation procedures, then apply the validated method to BM and neonatal stools collected from 4 mother-infant pairs to demonstrate its utility in identifying vertical transmission of bacteriophage species between clinical BM and stool samples.\\nMaterials and Methods\\nDevelopment and Validation of Bacteriophage Isolation Protocol\\nAll laboratory work was conducted in a Class II safety cabinet using nuclease-free or sterilised consumables and equipment. PROLACT HM\\u00ae human donor milk (Prolacta Bioscience Inc., USA) was used for method validation experiments. Donor milk (DM) was stored at \\u221280\\u00b0C until the day of processing, when samples were defrosted at 4\\u00b0C, gently mixed by inversion and dispensed into 1 mL aliquots. These were then further aliquoted, on ice, into 0.1, 0.2, 0.4, and 0.6 mL volumes to validate the required sample volume for bacteriophage isolation and DNA extraction. Six replicates of each sample volume were processed. DM aliquots were homogenised with cold sterile 1x PBS in a ratio of 1:5 (Supplementary Table 1).\\nExperiment 1: Sedimentation of Microbial Cells With Increased Centrifugation Speeds\\nTo assess the effect of centrifugal force in separating microbial cells for the induction of temperate bacteriophages, eight replicates of PBS-homogenised DM were split into three centrifugation groups (Supplementary Figure 1). In each group, samples were centrifuged at 400 xG at room temperature for 10 minutes to separate the DM lipid and skimmed milk, as previously described (18). The DM lipids were discarded, while the skimmed milk was centrifuged at 10, 15, and 20 k xG at 4\\u00b0C for 10 min. The resulting microbial cell pellets were incubated with norfloxacin (1 \\u03bcg/mL) at 37\\u00b0C for 1 h to induce temperate bacteriophages, followed by centrifugation at 15 k xG at 4\\u00b0C for 10 min to remove remaining microbial cell debris (19). Supernatants resulting from the first centrifugation (i.e., free bacteriophages) and the second centrifugation (i.e., induced temperate bacteriophages) were subjected to DNase (2 U/mL) and RNase (0.5 U/mL) treatments to remove human and microbial chromosomal DNA, as previously described (19). DNA of the isolated bacteriophages were then extracted using QIAamp MinElute Virus Spin Kit (QIAGEN, Germany), following the manufacturer's protocol. Equal volumes of autoclaved 1x PBS were included as a kit negative control in every extraction batch.\\nExperiment 2: Release of Bacteriophages in DM and Milk-Lipids With Digestive Enzymes\\nDigestive enzymes similar to those in infant guts (20) were used to hydrolyse milk fat globule membrane (MFGM) and release bacteria and bacteriophages that are associated with the MFGM. For hydrolysis of DM (Supplementary Figure 2), four replicates of PBS-homogenised DM were incubated with digestive enzymes (0.4 mg/mL of pepsin and 0.86 mg/mL of lipase) at 37\\u00b0C and shaking at 350 rpm for an hour. Hydrolysed samples were centrifuged at 400 xG at room temperature for 10 min to separate the DM lipid and skimmed milk, as previously described (18). The resulting DM lipids were carefully removed and were discarded, whilst the skimmed milk was centrifuged at 10 k xG at 4\\u00b0C for 10 min. As described in Experiment 1, the separated microbial cell pellets were then subjected to norfloxacin-mediated bacteriophage induction and removal of remaining microbial cell debris. Both supernatants of free and temperate bacteriophages were ready for the subsequent steps of DNase and RNase treatments and DNA extraction.\\nFor hydrolysis of DM lipids (Supplementary Figure 3), four replicates of PBS-homogenised DM samples were centrifuged at 400 xG and room temperature for 10 min. The resulting DM lipids were carefully separated and homogenised in 1 mL of cold sterile 1x PBS. Four further DM samples were frozen at \\u221220\\u00b0C for 30 min following centrifugation before separating the DM lipids and homogenising in cold sterile 1x PBS. These frozen lipid samples were used to validate freezing the DM lipid fraction during bacteriophage isolation. PBS-homogenised lipid suspensions were incubated with digestive enzymes at 37\\u00b0C and shaking at 350 rpm for an hour, then centrifuged at 10 k xG at 4\\u00b0C for 10 min for the separation of free bacteriophages and microbial cell pellets. As described upfront in Experiment 1, the separated microbial cell pellets were then subjected to norfloxacin-mediated bacteriophage induction and removal of remaining microbial cell debris. Both supernatants of free and temperate bacteriophages were ready for the subsequent steps of DNase and RNase treatments and DNA extraction.\\nIn addition, four replicates of PBS-homogenised DM samples without the input of digestive enzymes were processed as a control (Supplementary Figure 4).\\nExperiment 3: Concentrating and Purifying Isolated Bacteriophages\\nIsolated bacteriophages from the DM (Supplementary Figures 2, 4), were subjected to further concentration steps using 10 % PEG8000 and purification steps using Centriprep\\u00ae 50 kDa filter devices (Merck Millipore, USA), as previously described (21, 22).\\nIn addition, four replicates of PBS-homogenised DM samples without the application of 10 % PEG8000 and Centriprep\\u00ae 50 kDa filter devices were processed as a control (Supplementary Figure 4).\\nApplication of Validated Bacteriophage Isolation Protocol to Mother-Infant Pair Samples\\nThe pilot study utilised BM and stool samples from a total of four very preterm (<30 weeks gestation) mother-infant pairs enrolled on the SERVIS study at the Neonatal Intensive Care Unit, Royal Victoria Infirmary, Newcastle upon Tyne, UK, as previously described (8, 23). Samples were collected with ethical permission granted by NRES Committee North East\\u2013Newcastle & North Tyneside 2 (10/H0908/39) and analysed with written parental consent. Expressed BM and stool samples were stored at \\u221280\\u00b0C until processing. Bacteriophage isolation and DNA extraction were performed using the optimised protocol in Figure 4.\\nBacteriophage Metagenome Sequencing\\nThe concentration and purity of extracted bacteriophage DNA were determined using Qubit 2.0 Fluorometer (Thermo Fisher Scientific, USA) and NanoDrop\\u2122 1000 Spectrophotometer (Thermo Fisher Scientific, USA), respectively. Nucleic acid sequencing was performed by NU-OMICS DNA Sequencing Research Facility (Northumbria University, UK) using the Nextera XT kit and V3 600 cycle chemistry on the Illumina MiSeq (validation study) and V2.5 300 cycle mid-output chemistry on the Illumina NextSeq (pilot study) (Illumina Inc., USA). Raw sequencing data are available at ENA (study accession PRJEB49989).\\nInformatic Processing of Method Validation Data\\nRaw read data were de-multiplexed using MiSeq Reporter Software (Illumina, USA). Illumina adapter sequences were removed, sequences were trimmed with a sliding window (Q30) and any reads <50 nucleotides or >300 nucleotides were culled using FastQC (24) and Trimmomatic (25) in Galaxy (26). Trimmed paired and unpaired reads files were merged for further filtering with BBTools (27). Sequences identified in negative controls were culled from experimental samples. Low-complexity reads were removed using BBmask (entropy 0.7), and duplicated reads were removed with BBduk. Resulting reads were mapped to the human genome (GRCh38) using BBmap with high precision but low sensitivity parameter. Unmapped reads were aligned against NCBI viral reference database (accessed 16/07/2021 at https://ftp.ncbi.nlm.nih.gov/refseq/release/viral/) using the blastn option in BLAST (evalue 10-5, best-hit overhang 0.1, best-hit score edge 0.1). BLAST output reads were uploaded to MEGAN Community Edition (28), and taxonomy assignment was performed using a modified lowest common ancestor (LCA) algorithm (min score 100, E-value <10-5, min percent identity 80, min support 1, weighted 80). Enrichment of viral sequences in metagenomes was quantified using viromeQC (29).\\nInformatic Processing of Clinical Pilot Study\\nRaw paired end.fastq files were quality filtered with fastp (30) employing a 5 nucleotide sliding window and a minimum qscore >25. Reads <100 nucleotides were discarded. Kneaddata within the biobakery environment (31) was used to remove human reads. Contigs were assembled from cleaned reads using the default kmer steps in Megahit (32). Contigs with <5x coverage were culled with bwa mem (33), samtools (34), bamtools (35) and bedtools (35). Virsorter2 (36) was used to identify dsDNA phage, nucelocytoplasmic large-DNA viral, RNA viral and ssDNA viral contigs from the >5x contigs and CheckV (37) was used to estimate the completeness of viral assembled genomes. Taxonomic classification of >5x coverage viral contigs was performed with Kraken2 (38) by alignment to pre-built viral databases (accessed 04/01/2022, at https://benlangmead.github.io/aws-indexes/k2). Per-sample abundance of each classified viral contig was calculated using Bracken (39).\\nStatistical Analyses\\nValidation of bacteriophage isolation experiments was based on the concentration of DNA in nucleic acid extracts as well as the recovered viral richness and Shannon diversity index. R studio was used for all statistical analysis (40). The taxonomic richness and Shannon diversity measures of alpha diversity were calculated with the vegan package (41). Richness was calculated at a rarefied feature count below the minimum sample level where possible. Comparisons of continuous variables were performed by the Kruskal-Wallis test. Where significant (P > 0.05) differences were identified across experiment parameters the pairwise Wilcoxon test was used to perform pairwise comparisons between classes. Count and categorical data were compared by the Mann-Whitney test. In cases of multiple hypothesis testing P-values were adjusted using the Bonferroni method. The phyloseq (42) and ggplot (43) packages were used for data manipulation and visualisation.\\nResults\\nValidation of Bacteriophage Isolation\\nViral Composition of DM Samples\\nA median of 2.99 x 105 (IQR 8.82 x 104 \\u2013 4.81 x 105) raw reads were generated from 78 pasteurised DM samples (Supplementary Table 1). A total of 119 viral genera belonging to 18 families were identified (Supplementary Table 2). Bacteriophages accounted for 75 % of the total taxonomic reads. Predominant bacteriophage families included Siphoviridae (48%), Myoviridae (19%), Podoviridae (6%) and Herelleviridae (1%). The assigned bacteriophages are mainly associated with bacterial hosts Escherichia (17%), Salmonella (12%), Enterococcus (12 %), Klebsiella (7%), Staphylococcus (3%), Pseudomonas (3%), Enterobacter (3%), Acinetobacter (1%), Proteus (1%), Erwinia (1%), Lactococcus (1%) Shigella (1%) and Stenotrophomonas (1%) based on sequence homology. Unclassified bacteriophages constituted a further 1% of total reads.\\nEffect of Sample Volume on DM Bacteriophage Communities\\nNo significant difference in extracted DNA concentration (ng/\\u03bcL) (P = 0.96), genus richness (P = 0.91) or Shannon diversity (P = 0.86) was observed between DM sample volumes 0.1, 0.2, 0.4, and 0.6 mL (Figure 1).\\nFigure 1. Comparisons of DNA concentration, genus richness and Shannon diversity between different volumes of milk samples. Boxes represent the 1st and 3rd quartiles. Middle lines represent the median. Whiskers extend to the full range of data. Points represent individual samples.\\nCentrifugal Separation, Inclusion of Digestive Enzymes and Concentration/Purification of Viral Particles\\nNo significant differences in genus richness (P = 0.26) or Shannon diversity (P = 0.09) was observed in DM with increased centrifugation speed for microbial cell pelleting (Figure 2A). DNA concentration was significantly affected by centrifugation speed (P = 0.01) with significantly greater DNA concentrations observed in samples spun at 10 k xG (median 2.0 ng/\\u03bcL) than at 15 k xG (median 0.35 ng/\\u03bcL) (pairwise Wilcoxon: P = 0.02) (Figure 2A). Significantly lower Shannon diversity (P = 0.03) was observed in DM hydrolysed with digestive enzymes for lipid hydrolysis despite no difference in genus richness (P = 0.09) or DNA concentration (P = 0.54) (Figure 2B). The application of PEG8000 for concentration of isolated bacteriophages had no significant impact on DNA concentration (P = 0.32), genus richness (P = 0.92) or Shannon diversity (P = 0.92) of DM (Figure 2C). Concentrating and purifying bacteriophage isolates with Centriprep\\u00ae 50 kDa filter devices had no significant impact on observed genus richness (P = 0.25), or Shannon diversity (P = 1.00) (Figure 2D), but did marginally increase DNA concentration (P = 0.06) (Figure 2D). Separation of milk lipids from whole DM yielded no significant change in observed genus richness (P = 0.50), shannon diversity (P = 0.50), or isolated DNA concentration (P = 0.93) (Figure 2E).\\nFigure 2. Comparisons of DNA concentration, genus richness and Shannon diversity between centrifugal spin speeds for size-based separation (A), addition of digestive enzymes for milk hydrolysis (B), concentration of bacteriophage particles with PEG8000 (C), filtration of bacteriophage with Centriprep devices (D), and utilization of different milk fractions (E).\\nFurther optimisation of commercially available nucleic acid extraction kits found comparable results for both QIAamp MinElute Virus Spin Kit (QIAGEN, Germany) and Norgen Phage DNA Isolation Kits (Norgen Biotek Corp, Canada). Both yielded comparable bacteriophage richness and diversity (Supplementary Table 4, Supplementary Figure 5) (Figure 3A).\\nFigure 3. A higher viral richness was accounted for in samples at which (A) Norgen kit was used for DNA extraction, (B) norfloxacin-mediated phage induction was included or (C) digestive enzymes were added to the milk fat. The numbers within Venn diagrams represent the number of assigned families.\\nNorfloxacin-mediated induction of bacteriophages enabled identification of Inoviridae, which were not seen in any non-induced samples. Although hydrolysis with digestive enzymes had no significant impact on alpha diversity indices or isolated DNA concentration, an improved recovery of lipid-associated viruses, enabling exclusive identification of Iridoviridae and Baculoviridae was observed in samples subjected to lipid hydrolysis (Figure 3B). Freezing the lipid layer before separation enabled identification of Baculoviridae and Inoviridae but not Iridoviridae, suggesting lipid hydrolysis may be essential to isolate this family.\\nUnlike the pasteurised DM used here for method validation, bio-banked BM samples do not undergo pasteurisation, which disrupts interactions between DM fat globule membranes. We found that when processing bio-banked BM samples an extra step of intense horizontal agitation for 10 min at room temperature as proposed by Brewster and Paul (44) caused lipid aggregation against the side of sampling tubes which made removal of BM lipids easier prior to digestive enzyme treatment.\\nThe absence of any impact on DNA concentration, bacteriophage richness or diversity of increasing DM sample input volume validates our optimised protocol for bacteriophage community composition from as little as 0.1 mL DM. We found hydrolysis of DM lipids with digestive enzymes, 10 K xG centrifugal separation at 4\\u00b0C for 20 min and norfloxacin-mediated induction of temperate bacteriophages from the microbial cell pellets yields best results. We find no evidence for the inclusion of a freezing step prior to separation of DM lipids from the whey, nor preferential use of either QIAamp MinElute or Norgen Phage DNA isolation kits. Our optimised protocol is illustrated in Figure 4.\\nFigure 4. Optimized phage isolation protocol generated from the validation of different tested variables.\\nOptimized Method Application in Mother-Infant Pair Samples\\nTo demonstrate the utility of the validated isolation methods we applied it to a small cohort of 4 mother-infant pairs. All were inpatients at the RVI, NICU, Newcastle upon Tyne Hospitals, and received expressed MM feeds, supplemented with Labinic probiotics (Biofloratech Ltd., UK). One BM milk and one neonatal stool sample taken within 48 h of BM feed were analysed from each mother-infant pair. Clinical details of the cohort are provided in Table 1.\\nTable 1. Recorded characteristics of clinical cohort used in pilot study.\\nBM yielded greater raw and viral read count than neonatal stools however this did not translate to a greater number or length of assembled contigs with a minimum of 5 x coverage (Supplementary Table 3). Kit and sequencing negative control samples each yielded zero contigs.\\nSignificantly greater bacteriophage species richness (P = 0.04) and diversity (P = 0.04) were observed in neonatal stool samples than BM (Figure 5A). A total of 78 bacteriophage species belonging to 4 families were identified in stool with only 18 bacteriophage species from 4 families identified in BM (Figure 5B). Despite these differences, greater richness and diversity in maternal BM were often observed alongside greater richness and diversity in the paired neonatal stool samples (Figure 5A). Furthermore, we identify 6 bacteriophage species that were shared between BM and neonatal stools in two different mother-infant pairs (Figures 5C,D). All 6 species were putative Staphylococcus infecting bacteriophage, 4 of which belonged to the Siphoviridae and 2 to the Podoviridae families. Several bacteriophage species remain exclusively identifiable in either BM or neonatal stool (Figure 5D).\\nFigure 5. Bacteriophage community of mother-infant pair BM and stool samples. BM has lower species richness and diversity than stool (A) despite links between BM and neonatal stool richness and diversity as illustrated points, colored by mother-infant pair. Several bacteriophage taxonomic features are shared between BM and stool samples at the family level (B). Bacteriophage species composition of BM and neonatal stool is illustrated across all species observed via venn diagram (C) and in more detail as a heatmap containing only the top 25 most abundant & prevalent bacteriophage species (D). Greater relative abundance of species/families in heatmaps (B,D) is represented by greater color intensity [transformed on a log(2) scale].\\nDiscussion\\nCharacterising the mechanisms of action of bioactive components in BM is key to understanding its beneficial effects. To maximise the reliability and generalisability of studies investigating these components, it is important to validate laboratory processes utilized in large-scale surveys. Despite this, few studies exist exploring methods for the isolation of complex bacteriophage communities from human BM.\\nThis study represents a validation and optimisation of methods to reliably characterise the bacteriophage component of BM from mothers of very preterm infants by maximising the ability to observe the full diversity of bacteriophages within the sample while minimizing inherent biases, protocol cost and time. Specifically, we justify the use of very small sample volumes (as little as 0.1 mL) for very preterm BM phage community characterisation. Previous studies investigating the milk virome have utilized as much as 1.5\\u201310 mL of BM (13\\u201315). Reducing required sample volumes is of particular importance because preterm delivery can have a negative effect on lactogenesis and therefore salvageable volumes can be small to allow as much BM as possible to be available for the infant.\\nSimilar to previous studies which applied viral purification protocols using CsCl density gradient ultracentrifugation on term BM (14), we identify Mimiviridae (a giant group of viruses) in DM from our sequencing data that were not previously reported in BM studies using micron-sized membrane filters (16). Our proposed method omits the time-consuming CsCl density gradient purification which has previously been reported to have low reproducibility and is not well-suited for high-throughput studies (45, 46).\\nPEG has been used as an alternative to CsCl density gradients, proving to be a cost-effective way to substantially increase nucleic acid yields and give reproducible results compared to other methods for concentrating bacteriophages from 1 to 50 mL adult human stool samples (21, 45). Importantly we did not see differences between samples when isolating phage nucleic acids from DM with PEG or our proposed method. We also did not find any increased diversity or richness of phage genera using PEG concentration protocols. This is possible because more mature individuals have greater community diversity (47, 48), requiring substantially greater sample volumes to fully characterize the overall community structure. These same volumes would be impossible to salvage from very preterm infants and mothers. This is of note as the use of PEG concentrations to purify phage particles may introduce inherent biases by preferentially selecting for specific viral particle sizes or shapes, especially in complex or mixed bacteriophage communities (49).\\nOur protocol enabled us to identify 6 viral families (Adenoviridae, Baculoviridae, Herelleviridae, Papillomaviridae, Parvoviridae and Retroviridae), that were not previously observed in studies using micron-sized membrane filters to explore term BM viromes. The inclusion of these families, which may well have been filtered out during unnecessary pre-processing steps highlights the impact filtration can have on observable bacteriophage communities. Indeed, filtration has been shown to reduce viral particles isolated from stools by almost 50% (50).\\nPrevious studies of the viral composition of BM utilized isolation directly from whole or skimmed milk without upstream intervention (51\\u201353). Association of viruses with milk lipids are often excluded in these studies. Our data reveal the presence of bacteriophages belonging to the Inoviridae, and eukaryotic viruses belonging to the Iridoviridae and Baculoviridae families in the lipid fraction upon hydrolysis using digestive enzymes. This indicates the presence of microbes (particularly bacteriophages) associated with milk fat globule membranes that may have been previously overlooked. Furthermore, our data suggest norfloxacin-mediated induction of lysogens present in the microbial cell pellets following centrifugal separation is essential to identify the full complement of temperate bacteriophages within communities. In addition, we demonstrate the increased speed of centrifugal separation does not improve characterisation of the temperate bacteriophage community.\\nMany previous studies have utilized multiple displacement amplification (MDA) to increase nucleic acid yields for sequencing. MDA has previously been reported to preferentially amplify ssDNA phages introducing bias to community compositions by artificially inflating their numbers (54\\u201356). As such we did not include any MDA step prior to sequencing bacteriophage nucleic acids in this study but still achieve suitable nucleic acid concentrations for sequencing library preparation and identifying ssDNA phages from the Inoviridae family.\\nWe incorporate nucleases prior to nucleic acid isolation and include no RT step afterward which may overlook the role of any RNA viruses in this system (46), although similar to other gut virome studies, we still identify RNA viruses, such as members of the Retroviridae.\\nThis study highlights the importance of validating protocols specifically tailored to the sample type and matrices before conducting large-scale ecological surveys.\\nAfter validating a method for bacteriophage community characterisation in DM we applied this same protocol to BM and matched neonatal stool samples from mother-infant pairs highlighting six shared bacteriophage species between mother's BM and infant stools. The six shared species were all putative Staphylococcus infecting bacteriophages belonging to the Siphoviridae (4) and Podoviridae (2) families. Previous work by this group has identified Staphylococcus as the dominant member of bacterial communities of preterm infant's stool and mother's BM during early life (8, 57). Other highly abundant bacteriophages identified in this study, include Escherichia, Enterobacteria and Klebsiella phages. Although the host specificity of these phages is purely based on sequencing data, rather than host-range studies these results align with those of Liang et al. (16) who identified bacteriophage colonizing infant guts in the immediate post-natal period to be mostly comprised of temperate phages induced from bacterial community members. Previous studies have yielded conflicting results regarding the vertical transmission of bacteriophages between mothers and infants (12\\u201315). This study is the first to our knowledge to provide evidence for the potential of this phenomenon in very preterm infants who, importantly, receive much smaller feed volumes than term infants.\\nDespite being adopted in previous studies as a measure of viral recovery efficiency (58), our method validation data suggest DNA yield is a poor predictor of this. In two experiments the protocol yielding the greatest DNA concentration did not translate to greater bacteriophage richness or diversity. This is likely because despite multiple bacteriophage isolation and purification steps we still cannot confirm extracted nucleic acids are free from sources of other contaminating organisms. Alternative methods could spike known compositions of viral particles into sterile matrices, however, acquisition of sterile matrices with similar complexity to human milk proves extremely difficult. Our selection of pasteurized DM was initially intended to facilitate this, but our data demonstrate the persistence of bacteriophage nucleic acids despite pasteurisation. In addition, bacteriophage communities in DM and BM are not well studied so it is difficult to select bacteriophage species/strains for effective validation purposes (45).\\nThe persistence of bacteriophage nucleic acids post-pasteurisation of DM highlights the need for further work to discriminate between viable and non-viable bacteriophage communities associated with the very preterm infant GIT.\\nThis study developed an optimised protocol for the isolation of a broad range of bacteriophages from small volumes of BM samples. Using the validated bacteriophage isolation protocol, we have successfully identified viral communities in pasteurized DM, as well as in bio-banked preterm BM. Shared bacteriophage species between BM and stool samples from the same mother-infant pairs were also identified. These data will serve as baseline information toward future studies to develop greater understanding of how bacteriophages may be implicated in the development of the preterm gut microbiome.\\nData Availability Statement\\nThe original contributions presented in the study are publicly available. This data can be found here: https://www.ebi.ac.uk/ena, PRJEB49989.\\nEthics Statement\\nThe studies involving human participants were reviewed and approved by NRES Committee North East\\u2013Newcastle and North Tyneside 2 (10/H0908/39). Written informed consent to participate in this study was provided by the participants' legal guardian/next of kin.\\nAuthor Contributions\\nWY, AN, SB, and DS conceived the method validation study. WY performed laboratory and bioinformatic work for the validation study. GY and DS conceived the application in the mother-infant pairs study. GY performed laboratory and bioinformatic work for this. AN sequenced the libraries. JB and NE enrolled participants in the clinical study and arranged and carried out sampling. GY and WY wrote the manuscript. All authors reviewed and approved the manuscript before submission.\\nFunding\\nThis work was funded by Children's Charity Action Medical Research on grant number GN2730.\\nConflict of Interest\\nNE and JB declare research funding paid to their employing institution from Prolacta Biosciences US, and Danone Early Life Nutrition, and both declare lecture honoraria from Nestle Nutrition Institute.\\nThe remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nPublisher's Note\\nAll claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.\\nAcknowledgments\\nWe thank The Great North Neonatal Biobank, a collaboration between Newcastle University and Newcastle Upon Tyne Hospitals NHS Foundation Trust, for providing access to bio-banked human milk. We also thank the research nurses, clinical staff and all participant families at the Royal Victoria Infirmary, Newcastle upon Tyne, UK, for their help in collecting maternally expressed milk and preterm infant stool samples. DM was from Prolacta Biosciences and milk was provided for a separate research study where the milk was beyond the expiry date but remained frozen. Prolacta Bioscience had no involvement in this study.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fped.2022.856520/full#supplementary-material\\nReferences\\n1. Victora CG, Bahl R, Barros AJD, Fran\\u00e7a GVA, Horton S, Krasevec J, et al. Breastfeeding in the 21st century: epidemiology, mechanisms, and lifelong effect. Lancet. (2016) 387:475\\u201390. doi: 10.1016/S0140-6736(15)01024-7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n2. Royal College of Paediatrics and Child Health. Breastfeeding in the UK - Position Statement. (2017). p. 1\\u201310. \\nGoogle Scholar\\n4. Granger CL, Embleton ND, Palmer JM, Lamb CA, Berrington JE, Stewart CJ. Maternal breastmilk, infant gut microbiome and the impact on preterm infant health. Acta Paediatr. (2020) 110:450\\u20137. doi: 10.1111/apa.15534\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n5. Masi AC, Embleton ND, Lamb CA, Young G, Granger CL, Najera J, et al. Human milk oligosaccharide DSLNT and gut microbiome in preterm infants predicts necrotising enterocolitis. Gut. (2021) 70:2273\\u201382. doi: 10.1136/gutjnl-2020-322771\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n7. Embleton N, Berrington J, Cummings S, Dorling J, Ewer A, Frau A, et al. Lactoferrin impact on gut microbiota in preterm infants with late-onset sepsis or necrotising enterocolitis: the MAGPIE mechanisms of action study. Effic Mech Evaluat. (2021) 8:14. doi: 10.3310/eme08140\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n8. Young GR, van der Gast CJ, Smith DL, Berrington JE, Embleton ND, Lanyon C. Acquisition and development of the extremely preterm infant microbiota across multiple anatomical sites. J Pediatr Gastroenterol Nutr. (2020) 70:12\\u20139. doi: 10.1097/MPG.0000000000002549\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n10. Cabrera-Rubio R, Collado MC, Laitinen K, Salminen S, Isolauri E, Mira A. The human milk microbiome changes over lactation and is shaped by maternal weight and mode of delivery. Am J Clin Nutr. (2012) 96:544\\u201351. doi: 10.3945/ajcn.112.037382\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n11. Berrington JE, Stewart CJ, Embleton ND, Cummings SP. Gut microbiota in preterm infants: assessment and relevance to health and disease. Arch Dis Childhood-Fetal Neonatal Ed. (2013) 98:F286\\u201390. doi: 10.1136/archdischild-2012-302134\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n12. Breitbart M, Haynes M, Kelley S, Angly F, Edwards RA, Felts B, et al. Viral diversity and dynamics in an infant gut. Res Microbiol. (2008) 159:367\\u201373. doi: 10.1016/j.resmic.2008.04.006\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n13. Duranti S, Lugli GA, Mancabelli L, Armanini F, Turroni F, James K, et al. Maternal inheritance of bifidobacterial communities and bifidophages in infants through vertical transmission. Microbiome. (2017) 5:66. doi: 10.1186/s40168-017-0282-6\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n14. Pannaraj PS, Ly M, Cerini C, Saavedra M, Aldrovandi GM, Saboory AA, et al. Shared and distinct features of human milk and infant stool viromes. Front Microbiol. (2018) 9:1162. doi: 10.3389/fmicb.2018.01162\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n15. Dinleyici M, P\\u00e9rez-Brocal V, Arslanoglu S, Aydemir O, Sevuk Ozumut S, Tekin N, et al. Human milk virome analysis: changing pattern regarding mode of delivery, birth weight, and lactational stage. Nutr. (2021) 13:1779. doi: 10.3390/nu13061779\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n16. Liang G, Zhao C, Zhang H, Mattei L, Sherrill-Mix S, Bittinger K, et al. The stepwise assembly of the neonatal virome is modulated by breastfeeding. Nature. (2020) 581:470\\u20134. doi: 10.1038/s41586-020-2192-1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n18. Hamprecht K, Vochem M, Baumeister A, Boniek M, Speer CP, Jahn G. Detection of cytomegaloviral DNA in human milk cells and cell free milk whey by nested PCR1Parts of this study were presented at the 21ST herpesvirus workshop, Northern Illinois University, Dekalb, Illinois, USA, 1996.1. J Virol Methods. (1998) 70:167\\u201376. doi: 10.1016/S0166-0934(97)00179-1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n19. Tariq MA, Everest FLC, Cowley LA, De Soyza AD, Holt GS, Bridge SH, et al. A metagenomic approach to characterize temperate bacteriophage populations from cystic fibrosis and non-cystic fibrosis bronchiectasis patients. Front Microbiol. (2015) 6:97. doi: 10.3389/fmicb.2015.00097\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n20. Luo J, Wang Z, Li Y, Chen C, Ren F, Guo H. The simulated in vitro infant gastrointestinal digestion of droplets covered with milk fat globule membrane polar lipids concentrate. J Dairy Sci. (2019) 102:2879\\u201389. doi: 10.3168/jds.2018-15044\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n21. Castro-Mej\\u00eda JL, Muhammed MK, Kot W, Neve H, Franz CMAP, Hansen LH, et al. Optimizing protocols for extraction of bacteriophages prior to metagenomic analyses of phage communities in the human gut. Microbiome. (2015) 3:64. doi: 10.1186/s40168-015-0131-4\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n22. Rasmussen TS, de Vries L, Kot W, Hansen LH, Castro-Mej\\u00eda JL, Vogensen FK, et al. Mouse vendor influence on the bacterial and viral gut composition exceeds the effect of diet. Viruses. (2019) 11:435. doi: 10.3390/v11050435\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n23. Young GR, Smith DL, Embleton ND, Berrington JE, Schwalbe EC, Cummings SP, et al. Reducing viability bias in analysis of gut microbiota in preterm infants at risk of NEC and sepsis. Front Cell Infect Microbiol. (2017) 7:237.doi: 10.3389/fcimb.2017.00237\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n24. Andrews S. FastQC: A Quality Control Tool For High Throughput Sequence Data (2010). \\nGoogle Scholar\\n26. Afgan E, Baker D, van den Beek M, Blankenberg D, Bouvier D, Cech M, et al. The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update. Nucleic Acids Res. (2016) 44:W3\\u2013W10. doi: 10.1093/nar/gkw343\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n29. Zolfo M, Pinto F, Asnicar F, Manghi P, Tett A, Bushman FD, et al. Detecting contamination in viromes using viromeQC. Nat Biotechnol. (2019) 37:1408\\u201312. doi: 10.1038/s41587-019-0334-5\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n31. McIver LJ, Abu-Ali G, Franzosa EA, Schwager R, Morgan XC, Waldron L, et al. bioBakery: a meta'omic analysis environment. Bioinformatics. (2018) 34:1235\\u20137. doi: 10.1093/bioinformatics/btx754\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n32. Li D, Liu C-M, Luo R, Sadakane K, Lam T-W. MEGAHIT an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. Bioinformatics. (2015) 31:1674\\u20136. doi: 10.1093/bioinformatics/btv033\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n34. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The Sequence alignment/map format and SAMtools. Bioinformatics. (2009) 25:2078\\u20139. doi: 10.1093/bioinformatics/btp352\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n36. Guo J, Bolduc B, Zayed AA, Varsani A, Dominguez-Huerta G, Delmont TO, et al. VirSorter2: a multi-classifier, expert-guided approach to detect diverse DNA and RNA viruses. Microbiome. (2021) 9:37. doi: 10.1186/s40168-020-00990-y\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n37. Nayfach S, Camargo AP, Schulz F, Eloe-Fadrosh E, Roux S, Kyrpides NC. CheckV assesses the quality and completeness of metagenome-assembled viral genomes. Nat Biotechnol. (2021) 39:578\\u201385. doi: 10.1038/s41587-020-00774-7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n39. Lu J, Breitwieser FP, Thielen P, Salzberg SL. Bracken: estimating species abundance in metagenomics data. PeerJ Comput Sci. (2017) 3:e104. doi: 10.7717/peerj-cs.104 \\nCrossRef Full Text | Google Scholar\\n42. McMurdie PJ, Holmes S. Phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. PLoS ONE. (2013) 8:e61217. doi: 10.1371/journal.pone.0061217\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n44. Brewster JD, Paul M. Short communication: Improved method for centrifugal recovery of bacteria from raw milk applied to sensitive real-time quantitative PCR detection of Salmonella spp1. J Dairy Sci. (2016) 99:3375\\u20139. doi: 10.3168/jds.2015-9655\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n45. d'Humi\\u00e8res C, Touchon M, Dion S, Cury J, Ghozlane A, Garcia-Garcera M, et al. A simple, reproducible and cost-effective procedure to analyse gut phageome: from phage isolation to bioinformatic approach. Sci Rep. (2019) 9:11331. doi: 10.1038/s41598-019-47656-w\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n46. Callanan J, Stockdale SR, Shkoporov A, Draper LA, Ross RP, Hill C. Biases in Viral metagenomics-based detection, cataloguing and quantification of bacteriophage genomes in human faeces, a review. Microorg. (2021) 9:524. doi: 10.3390/microorganisms9030524\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n47. Stewart CJ, Ajami NJ, O'Brien JL, Hutchinson DS, Smith DP, Wong MC, et al. Temporal development of the gut microbiome in early childhood from the TEDDY study. Nature. (2018) 562:583\\u20138. doi: 10.1038/s41586-018-0617-x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n48. Yatsunenko T, Rey FE, Manary MJ, Trehan I, Dominguez-Bello MG, Contreras M, et al. Human gut microbiome viewed across age and geography. Nature. (2012) 486:222\\u20137. doi: 10.1038/nature11053\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n49. Carroll-Portillo A, Coffman CN, Varga MG, Alcock J, Singh SB, Lin HC. Standard bacteriophage purification procedures cause loss in numbers and activity. Viruses. (2021) 13:328. doi: 10.3390/v13020328\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n50. Hoyles L, McCartney AL, Neve H, Gibson GR, Sanderson JD, Heller KJ, et al. Characterization of virus-like particles associated with the human faecal and caecal microbiota. Res Microbiol. (2014) 165:803\\u201312. doi: 10.1016/j.resmic.2014.10.006\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n51. Lazar K, Rabe T, Goelz R, Hamprecht K. Human cytomegalovirus reactivation during lactation: impact of antibody kinetics and neutralization in blood and breast milk. Nutrients. (2020) 12:338. doi: 10.3390/nu12020338\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n52. Drago L, Toscano M, De Grandi R, Grossi E, Padovani EM, Peroni DG. Microbiota network and mathematic microbe mutualism in colostrum and mature milk collected in two different geographic areas: Italy versus Burundi. ISME J. (2017) 11:875\\u201384. doi: 10.1038/ismej.2016.183\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n54. Kyoung-Ho K, Jin-Woo B. Amplification methods bias metagenomic libraries of uncultured single-stranded and double-stranded DNA viruses. Appl Environ Microbiol. (2011) 77:7663\\u20138. doi: 10.1128/AEM.00289-11\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n55. D\\u017dunkov\\u00e1 M, Garcia-Garcer\\u00e0 M, Mart\\u00ednez-Priego L, D'Auria G, Calafell F, Moya A. Direct Sequencing from the minimal number of DNA molecules needed to fill a 454 picotiterplate. PLoS ONE. (2014) 9:e97379. doi: 10.1371/journal.pone.0097379\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n56. Yilmaz S, Allgaier M, Hugenholtz P. Multiple displacement amplification compromises quantitative analysis of metagenomes. Nat Methods. (2010) 7:943\\u20134. doi: 10.1038/nmeth1210-943\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n57. Stewart CJ, Embleton ND, Marrs ECL, Smith DP, Fofanova T, Nelson A, et al. Longitudinal development of the gut microbiome and metabolome in preterm neonates with late onset sepsis and healthy controls. Microbiome. (2017) 5:75. doi: 10.1186/s40168-017-0295-1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n58. Bergallo M, Costa C, Gribaudo G, Tarallo S, Baro S, Negro Ponzi A, et al. Evaluation of six methods for extraction and purification of viral DNA from urine and serum samples. New Microbiol. (2006) 29:111\\u20139.\\nPubMed Abstract | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fcimb.2022.1060825\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fcimb.2022.1060825/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:38.136Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fcimb.2022.1060825\",\n",
      "            \"title\": \"Frontiers | Bacteriophage-based decontamination to control environmental colonization by Staphylococcus capitis in neonatal intensive care units: An in vitro proof-of-concept\",\n",
      "            \"description\": \"In neonatal intensive care units (NICUs), the standard chemical-based disinfection procedures do not allow a complete eradication of pathogens from environmental surfaces. In particular, the clone Staphylococcus capitis NRCS-A, a significant pathogen in neonates, was shown to colonize neonatal incubators. The aim of this study was to evaluate the in vitro effect of a bacteriophage cocktail on NRCS-A eradication. Three bacteriophages were isolated, genetically characterized and assessed for their host range using a collection of representative clinical strains (n=31) belonging to the clone NRCS-A. The efficacy of a cocktail including these three bacteriophages to eradicate the reference strain S. capitis NRCS-A CR01 was determined in comparison or in combination with the chemical disinfectant Surfanios Premium on either dry inoculum or biofilm-embedded bacteria. The emergence of bacterial resistance against the bacteriophages alone or in cocktail was evaluated by growth kinetics. The three bacteriophages belonged to two families and genera, namely Herelleviridae/Kayvirus for V1SC01 and V1SC04 and Rountreeviridae/Andhravirus for V1SC05. They were active against 17, 25 and 16 of the 31 tested strains respectively. Bacteriophage cocktails decreased the bacterial inoculum of both dry spots and biofilms, with a dose dependent effect. The sequential treatment with bacteriophages then Surfanios Premium did not show enhanced efficacy. No bacterial resistance was observed when using the bacteriophage cocktail. This study established a proof-of-concept for the use of bacteriophages to fight against S. capitis NRCS-A. Further investigations are needed using a larger bacterial collection and in real-life conditions before being able to use such technology in NICUs.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nNosocomial infections in hospitalized patients are a well-known cause of mortality and morbidity all over the world (Sikora and Zahra, 2022). The prevalence of such infections is high, representing 6.5% of the patients in the European Economic Area and 3.2% patients in the United States (Magill et al., 2018; Suetens et al., 2018). Multidrug resistant bacteria from the hospital environment are frequently involved in nosocomial infections (Suleyman et al., 2018). Among the various hospital units, neonatal intensive care units (NICUs) are at high risk of nosocomial infections since preterm neonates are highly vulnerable given their global immaturity (Collins et al., 2018). Despite the strict hygiene measures implemented in NICUs, several studies have incriminated numerous pathogens in those infections including staphylococci, enterobacteria or enterococci originating from the environmental surfaces and notably from the incubators where preterm newborns are housed (Golan et al., 2005; Lin et al., 2017; Carter et al., 2018; Butin et al., 2019; Cadot et al., 2019; Ory et al., 2019; Chavignon et al., 2021).\\nThe clone Staphylococcus capitis NRCS-A is one of these pathogens. Its high prevalence in sepsis among low-birth weight preterm neonates in NICUs worldwide, its multidrug resistant profile and its high adaptation to the NICU environment have been reported (Butin et al., 2017; Wirth et al., 2020). This specialization has been related to its resistance pattern to antibiotics commonly used in NICUs including vancomycin or aminoglycosides (Rasigade et al., 2012; Wirth et al., 2020). Moreover, previous studies have highlighted the ability of NRCS-A to colonize inert surfaces, notably neonatal incubators, and to persist despite the usual chemical disinfection procedures fostering its continuous spread and dissemination after its initial introduction in a given NICU (Carter et al., 2018; Butin et al., 2019; Chavignon et al., 2021). Finally, the ability of S. capitis strains isolated from neonates or from the NICU environment to form biofilm has been highlighted and might explain, at least for a part, the ability of the strains to colonize abiotic surfaces (Cui et al., 2013; Carter et al., 2018; Qu et al., 2020). In this context, there is a need to develop alternative disinfection approaches to reduce the risk of environmental persistence of S. capitis and inter-patient transmission. One innovative approach could be the setup of bacteriophage-based decontamination. Bacteriophages are ubiquitous viruses able to specifically infect prokaryotic bacterial cells with a total innocuity for eucaryotic cells and so for humans (D\\u2019Accolti et al., 2021). Thus, lytic virulent bacteriophages present a bactericidal activity with a high target specificity at the species or infra-species level and constitute useful tools for biological control. The use of bacteriophages has already been considered as an alternative or a complementary method of decontamination in different contexts (D\\u2019Accolti et al., 2021). In food-processing industries, the use of bacteriophages as additive to control the contamination of food production by pathogenic bacteria has been already approved by the Food and Drug Administration (https://www.cfsanappsexternal.fda.gov/scripts/fdcc/?set=GRASNotices&sort=GRN_No&order=DESC&startrow=1&type=basic&search=phages, retrieved 9 August 2022). Bacteriophages have also been considered in decontamination of inanimate surfaces in the hospital environment. For instance, the use of an aerosolized bacteriophage suspension after the standard cleaning procedure (based on sodium hypochlorite 0.06% and alcohol 75%) in an intensive care unit in Taiwan has been reported to be able to decrease the rate of nosocomial infections caused by carbapenem-resistant Acinetobacter baumannii (Ho et al., 2016). In another study, D\\u2019Accolti et al. demonstrated the feasibility and effectiveness of hospital bathrooms nebulization with bacteriophages in addition to a probiotic-based sanitation to fight against Staphylococcus spp. contamination and nosocomial infections. Interestingly, daily nebulization of bacteriophages allowed a significant and rapid decrease of Staphylococcus on the surfaces that was up to 97% more effective than probiotic-based sanitation alone (D\\u2019Accolti et al., 2019).\\nDue to the difficulty to eradicate the clone S. capitis NRCS-A after its initial introduction in a NICU and to face its persistence, an in vitro preliminary study was conducted to determine if a bacteriophage-based decontamination could be an efficient approach. The aim of the present study was thus i) to isolate lytic bacteriophages presenting a bactericidal activity on clinical strains belonging to the clone S. capitis NRCS-A, ii) to explore the emergence of resistance against those bacteriophages, and finally iii) to evaluate the ability of those bacteriophages to impact biofilms or planktonic adhered bacteria in comparison with the usual disinfection molecule.\\nMaterials and methods\\nBacterial strains\\nThe Staphylococcus capitis strains used in this study were obtained from the collection of the French National Reference Centre for Staphylococci (Lyon, France). These strains were conserved in cryotubes containing glycerol at -20\\u00b0C. They were grown on Columbia agar + 5% sheep blood (BioM\\u00e9rieux, Marcy l\\u2019Etoile, France) for 24 hours at 37\\u00b0C. A collection of 31 strains was included in this study to isolate, produce and assess the host range of anti-S. capitis bacteriophages. These strains were selected to reflect the phylogenetic diversity of the S. capitis NRCS-A clone composed of three subgroups successively appeared: Proto-outbreak I, Proto-outbreak II and the most recent and specialized in NICU infections Outbreak (Table 1) (Wirth et al., 2020). The reference strain of the clone S. capitis NRCS-A, called CR01 (Lemriss et al., 2014) has been used to evaluate the effect of bacteriophages on dry spots and biofilms.\\nTable 1 Host range characterization of five anti-Staphylococcus capitis bacteriophages.\\nBacteriophages isolation and production\\nThree bacteriophages were isolated for the purpose of this study from three different wastewater samples in Lyon, France in 2020. Briefly, 5 mL of filtered water sample was incubated during 24 h at 37\\u00b0C under agitation (180 rpm) with 500 \\u00b5L of Tryptic Soy Broth (TSB) 10X (BD, Franklin Lakes, NJ, USA) and 10 \\u00b5L of an overnight culture of the strain P2SC02, chosen for the intensity of the lysis produced by the bacteriophages when applied to it. The culture supernatant was filtered using a 0.22-\\u00b5m syringe filter and diluted in double layer agars pouring a mix of 100 \\u00b5L of this supernatant, 250 \\u00b5L of P2SC02 culture and 5 mL of medium containing 0.75% of agar (called TSB soft-agar) over a Tryptic Soy Agar plate (BioM\\u00e9rieux, Marcy l\\u2019Etoile, France). Individual plaque forming units (PFU) were further purified by 5 rounds of serial passages and where required propagated in liquid medium. Strain P2SC16 was then used for bacteriophage production as amplification yields were higher with this strain. Bacteria were cultured in 50 mL of TSB in exponential phase then inoculated with bacteriophages to obtain a Multiplicity of Infection (MOI) of 10-3. Obtained bacteriophage lysates were filtered and stored at 4\\u00b0C.\\nBacteriophage genome sequencing and bio-informatic analysis\\nAfter isolation, a volume of 6 mL of each bacteriophage was centrifuged at 14,000 g for 5 h and pellets were re-suspended in 50 \\u00b5L of NaCl 0.9%. Enzymatic treatment with 100 mU of benzonase\\u00ae nuclease (Merk, Darmstadt, Germany) at 37\\u00b0C overnight was then used to degrade extracellular bacterial DNA, followed by benzonase heat-inactivation at 95\\u00b0C for 30 min, a treatment with 4 \\u00b5g of proteinase K (Merk, Darmstadt, Germany), and a proteinase K heat-inactivation at 95\\u00b0C for 30 min. Bacteriophage DNA was extracted using the DNA Extractor\\u00ae WB kit (Fujifilm, Osaka, Japan) and sequenced on an Illumina NextSeq 500 instrument using a 150-bp paired-end protocol. Reads were trimmed (cutadapt, v3.4; trimmomatic, v0.39) and the reads mapping against the genome of the P2SC16 strain used for the amplification of bacteriophages (bowtie2, v2.3.4.1; samtools, v1.15) were removed. The remaining reads were assembled (SPAdes, v3.13.0) and scaffolds smaller than 100 bp were removed. Taxonomic assignation was performed using kraken 1.1.1 with the minikraken database. Genomes were annotated using PATRIC (v3.6.12) with parameters for bacteriophage annotation. Finally, Abricate (v0.8.13) was used for resistance and virulence gene detection using all the databases available. The lytic nature of bacteriophages was assessed using the Phage AI repository (https://app.phage.ai/phages/). Bacteriophage genomes were deposited on the NCBI GenBank under the accession numbers OP323059, OP297178 and OP297179 for vB_ScaM-V1SC01, vB_ScaM-V1SC04, and vB_ScaM-V1SC05, respectively, which are named V1SC01, V1SC04, and V1SC05 in this study for simplification.\\nHost range assessment\\nHost range of each bacteriophage was assessed using the spot test assay on the panel of 31 S. capitis strains, selected as explained above (Table 1). Briefly, 5 \\u00b5L of serial ten-fold dilutions of bacteriophages were spotted on an agar plate prepared extemporaneously by mixing 30 mL of TSB soft agar and 500 \\u00b5L of S. capitis overnight culture in TSB broth. After overnight incubation at 37\\u00b0C, PFU were enumerated. The efficiency of plating ratio (EOP) was calculated dividing the bacteriophage titer obtained on a test strain by the titer of the same bacteriophage suspension on the reference production strain P2SC16. A bacteriophage was considered active if EOP was \\u2265 0.001 (Green et al., 2017). Experiments were performed in biological triplicates and mean EOP scores were calculated.\\nEvaluation of bacterial resistance emergence during bacteriophage treatment\\nGrowth kinetics were performed to determine if bacterial resistance occurred following the treatment with one bacteriophage alone compared to cocktails of the three bacteriophages. Briefly, CR01 was incubated in TSB until the exponential growth phase and then diluted in TSB to reach a concentration of 1.107 CFU/mL. A 96 well microplate (Thermo Fisher Scientific, Waltham, MA, USA) was then inoculated with 1.106 CFU of bacteria and either 1.106 PFU, 1.107 PFU or 1.108 PFU of each bacteriophage alone or in cocktail (to reach a MOI of respectively 1, 10 or 100) in a final TSB volume of 200 \\u00b5L. Bacterial growth alone without bacteriophage addition and absence of bacterial contamination in bacteriophage suspensions and in TSB were also controlled in the microplate. The outlines of the microplate were filled with water to prevent evaporation then the microplate was incubated 24 h in a microplate reader Tecan infinite\\u00ae 200 PRO (Tecan, M\\u00e4nnedorf, Swiss) under agitation at 37\\u00b0C with OD600 nm (optical density) measurement every 30 min. Emergence of bacterial resistance was considered when the OD600 nm exceeded the threshold of 0.01 (after TSB control media OD600 nm subtraction) that corresponded to the threshold of bacterial growth. The experiments were performed three times in technical triplicates and both bacteriophage titres and bacterial inoculum were verified each time.\\nActivity of a bacteriophage cocktail on dry spots and pre-formed biofilm of S. capitis NRCS-A\\nThe effect of a bacteriophage cocktail was tested on the reference strain CR01 in comparison with the disinfectant Surfanios Premium (SP) (composed of N-(3-aminopropyl)-N-dod\\u00e9cylpropane-1,3-diamine and chlorure de did\\u00e9cyldim\\u00e9thylammonium) (ANIOS, Lezennes, France). Both treatments were tested on dry spots and on pre-formed biofilms using a previously published method with some modifications (Stachler et al., 2021). For bacteriophage cocktails preparation, the three bacteriophages were associated by diluting each of them in equal measure in either TSB or Phosphate Buffered Saline (PBS) (Thermo Fisher Scientific, Waltham, MA, USA) to obtain cocktails at three different final concentrations (3.104 PFU/mL, 3.106 PFU/mL or 3.108 PFU/mL). Bacteria were exposed to the bacteriophage cocktail for 6 h in wet chamber at 37\\u00b0C. The chemical treatment consisted in exposition to SP at 0.25% in water at room temperature for 20 minutes, corresponding to the classical treatment used in NICUs for incubator disinfection (Butin et al., 2019). Untreated positive controls were treated with either TSB or PBS (controls for bacteriophage treatments) or water (control for SP treatment) alone.\\nDry spots were obtained by inoculating 10 \\u00b5L drops of a 1.107 CFU/mL bacterial PBS suspension in a 96 well microplate (Greiner Bio-One, Cap Horn, France). The drops were dried at room temperature during 4.5 h under microbiological safety workstation. The dry spots treatment consisted of covering the adhered bacteria with 20 \\u00b5L of bacteriophages cocktail, SP or control medium. Then, 100 \\u00b5L of Dey Engley Neutralizing broth (Merk, Darmstadt, Germany) was applied during 5 min to inactivate the SP action. This same neutralising treatment was applied to dry spots exposed to bacteriophages to avoid experimental variations. Finally, the neutralising broth was discarded and the surviving bacteria were suspended in 100 \\u00b5L of PBS by scraping and enumerated.\\nBiofilm was obtained by inoculating 100 \\u00b5L of a 1.107 CFU/mL bacterial suspension in TSB in a 96 well microplate that was incubated 24 h at 37\\u00b0C. After incubation, the biofilm was first rinsed using the steam method as previously described (Tasse et al., 2018) before being exposed to 150 \\u00b5L of bacteriophage cocktails or SP 0.25% as described above. After incubation, the supernatant was carefully removed and 200 \\u00b5L of Dey Engley Neutralizing broth was applied for 5 minutes. The biofilm was then rinsed with the steam method, suspended in 200 \\u00b5L of PBS by scraping and enumerated.\\nIn addition, a sequential treatment with a bacteriophage cocktail at the highest bacteriophages concentration (3.108 PFU/mL) followed by a SP treatment was also evaluated on bacterial biofilm. For this purpose, after the step of bacteriophage treatment as described above, an additional steam rinse was performed and then SP was applied followed by inactivation as previously described.\\nLog10 reduction values (LRV) for each treatment were calculated as follows: Log10(N/N0) with N corresponding to the number of bacteria in the treated well and N0 corresponding to the number of bacteria in the untreated control well. The experiments were performed three times in triplicate for the simple treatments and three times in sextuplicate for the sequential treatment. Bacteriophage cocktails titres were checked each time.\\nGraphic representation and statistical analyses\\nThe graphics were drawn on GraphPad Prism 8 (La Jolla, CA, USA) using the scatter dot plot representation with the mean and the standard deviation displayed for each condition. Statistical analyses were performed in GraphPad Prism 8 on the LRV values. Because of the small sample size in the different experiments (experiments performed three times in triplicate or in sextuplicate), the Mann-Whitney nonparametric test was performed, with an \\u03b1 risk of 0.05.\\nResults\\nBacteriophages characteristics and host range against S. capitis NRCSA\\nThe three bacteriophages belonged to two different families and genera, namely Herelleviridae/Kayvirus for V1SC01 and V1SC04 or Rountreeviridae/Andhravirus for V1SC05. VISC04 had the widest host range as it was active against 25 out of the 31 tested strains while V1SC01 and V1SC05 were active against 17 and 16 of the strains respectively (Table 1). Of note, activities of bacteriophages were complementary: 28 out of the 31 strains were susceptible to at least one bacteriophage. However, bacteriophage activity depended on the NRCS-A subgroup: all strains of the Outbreak and Proto-outbreak I groups were susceptible to 1 to 3 bacteriophages, while 3 out of the 6 Proto-outbreak II strains were resistant to all bacteriophages.\\nEmergence of resistances against bacteriophages\\nAfter 24 h of growth in presence of bacteriophages, growth curves indicated that CR01 did not develop resistance against the bacteriophage V1SC01 while resistance to V1SC04 at MOI 1 and MOI 100 and to V1SC05 at MOI 1, MOI 10 and MOI 100 were detected (Table 2). The emergence of these resistances revealed by bacterial regrowth always occurred in the late phase of the experiment (18 h to 24 h). Interestingly, no resistance was observed whatever the MOI when bacteriophage cocktail of the three bacteriophages was tested.\\nTable 2 Development of bacterial resistance against bacteriophages using bacteriophage alone or in cocktail.\\nBacteriophage cocktail activity compared to Surfanios Premium on bacterial dry spots\\nWhile the SP 0.25% treatment was able to completely eradicate bacteria on the dry spots in the tested conditions, bacteriophage cocktail treatment significantly decreased the bacterial inoculum on dry spots both in TSB and in PBS at 3.106 PFU/mL (1 LRV (p < 0.0001) and 0,5 LRV (p <0.0001) respectively) and at 3.108 PFU/mL (4.7 LRV (p < 0.0001) and 2.4 LRV (p < 0.0001) respectively) but not at 3.104 PFU/mL (Figure 1A).\\nFigure 1 Effect of a bacteriophage cocktail and Surfanios Premium on dry spots and pre-formed biofilms of S. capitis NRCS-A. Results are presented as Log10 reduction values (LRV) calculated as Log10(N/N0) with N the remaining inoculum after treatment and N0 the remaining inoculum after treatment with diluent (TSB or PBS for treatment with bacteriophages and water for treatment with SP). The treatments consisted either of i) bacteriophage cocktails at three concentrations (3.104 PFU/mL, 3.106 PFU/mL or 3.108 PFU/mL) in TSB or in PBS, ii) Surfanios Premium (SP) at 0.25% in water or iii) a sequential treatment with the bacteriophage cocktail at the highest concentration (3.108 PFU/mL) in TSB followed by SP treatment. (A) Effect of the bacteriophage cocktails or SP individually on bacterial dry spots. (B) Effect of the bacteriophage cocktails or SP individually or sequentially on pre-formed biofilms.\\nBacteriophage cocktail activity compared to SP and sequential treatment on biofilm\\nBacteriophage cocktail showed a bactericidal effect on biofilm only when incubated with TSB (Figure 1B). This bactericidal effect was significantly higher at 3.106 PFU/mL and 3.108 PFU/mL than at 3.104 PFU/mL with 1.4 LRV and 1.8 LRV respectively versus 0.5 LRV (p = 0.0005 and p < 0.0001 respectively). The efficacy of SP 0.25% was not significantly different to that of bacteriophage cocktail at 3.108 PFU/mL with 1.9 LRV (p = 0.6314).\\nThe sequential treatment, consisting in a first exposition to bacteriophage cocktail at 3.108 PFU/mL in TSB followed by an exposition to SP 0.25%, showed a 1.8 LRV (Figure 1B). This was not significantly different from the effect of either SP 0.25% alone or the bacteriophage cocktail alone at the highest concentration (p = 0.6959 and p = 0.6314 respectively).\\nDiscussion\\nThe present study highlights the ability of a cocktail of three lytic bacteriophages to significantly reduce the bacterial inoculum of S. capitis NRCS-A on dry spots or in biofilm. Our data highlights that no emergence of resistance to the bacteriophage cocktail occurred during exposition of S. capitis NRCS-A. The sequential exposition to bacteriophages then to chemical disinfectant did not improve the final bactericidal effect observed.\\nBacteriophages have to be considered in the perspective of environmental decontamination because they present advantages in comparison to chemical disinfectants. First, in contrast to bacteriophages, chemical products are generally toxic both for humans and the environment (Dhama et al., 2021). Thus, quaternary ammonium compounds (QACs), that are components of SP, have been previously shown to be harmful for frequently exposed healthcare workers by causing allergic contact dermatitis and asthma (Purohit et al., 2000; Suneja and Belsito, 2008). QACs exposition has also been shown to increase secretion of inflammatory cytokines, alter mitochondrial function, and disrupt cholesterol homeostasis in humans (Hrubec et al., 2021). All these features are of concerns, all the more when such disinfectants are used to clean incubators housing very low-birth-weight preterm neonates. Secondly, conventional disinfectants have a corrosive effect on abiotic surfaces (Song et al., 2019). Such damages may induce porosity on equipment surfaces which fosters biofilm formation and so might favor persistence of some pathogenic bacteria. In contrast, bacteriophages are usually presented as having no impact on human cells and so to be safe for human health in addition to be rapidly cleared from the human body, as previously shown in reports of bacteriophage therapy (Wright et al., 2009). Furthermore, due to their narrow spectrum of antibacterial activity, it seems that bacteriophages do not damage the beneficial microbiota and limit the selection of multidrug resistance (D\\u2019Accolti et al., 2021). However, it should be noted that despite these general statements, recent studies question the real neutrality of bacteriophages for eukaryotic organisms. Notably, due to their nature of protein-nucleic particle, bacteriophages were suspected to activate the immune system resulting in the production of anti-bacteriophages antibodies or the production of pro-inflammatory or anti-inflammatory cytokines (D\\u0105browska et al., 2014; Y\\u0131ld\\u0131zl\\u0131 et al., 2020; Podlacha et al., 2021). In addition, bacteriophages are able of transcytosis across confluent epithelial cell layers but also to interact with eukaryotic proteins, in particular protein misfolding and prion formation (Nguyen et al., 2017; Tetz and Tetz, 2017; Tetz and Tetz, 2018). Moreover, recent works suggested that bacteriophages could alter the normal microbiota, since Tetz et al. reported a high abundance of lytic Lactococcus bacteriophages associated with a reduction of Lactococcus in the microbiota of patients with Parkinson disease (Tetz and Tetz, 2018). For these reasons, in a perspective of using these bacteriophages for decontamination in a NICU, it would be necessary to carefully assess the safety of products containing bacteriophages.\\nThe present study has been motivated by the need to explore innovative procedures for disinfection in NICU settings. The aim was to determine if bacteriophages could be used to fight against S. capitis NRCS-A environmental spread/persistence in NICUs. Our data documented a bactericidal activity of bacteriophages on both bacterial dry spots and biofilms. This action was dose dependant, as reported in previous studies (Zhang and Hu, 2013; Stachler et al., 2021). For instance, the bacterial decrease represented 4.7 LRV and 1.8 LRV respectively on dry spots and biofilms at 3.108 PFU/mL in TSB while no bacterial loss on dry spots and only 0.47 LRV on biofilms was observed at 3.104 PFU/mL. Our data demonstrated that the medium used for treatment impacted the efficacy of the bacteriophage cocktail on S. capitis including a better efficacy in TSB than in PBS, particularly on biofilm. This is consistent with the mechanism of action of bacteriophages that are known to better infect active bacteria (Azeredo and Sutherland, 2008; Harper et al., 2014). A rich medium, as TSB, could induce the re-growth of the dormant bacteria inside the biofilm and thus foster bacteriophages infection and replication inducing bacterial lysis. In addition, when bacteriophages greatly outnumber bacteria, they are able to kill it without intrabacterial replication, a phenomenon previously described as \\u201clysis from without\\u201d (Abedon, 2011). This could explain the paradoxical efficacy of bacteriophages in PBS on dry spots at high bacteriophage concentration.\\nDespite all benefits of the use of bacteriophages, some points ought to be examined with particular attention. First, emergence of bacterial resistance against bacteriophages is frequent notably by avoiding bacteriophage adsorption or bacteriophage nucleic acid entry (based on alteration of bacteriophages nucleic acids or abortive infection systems) (Labrie et al., 2010; Oechslin, 2018). The emergence of bacterial resistance against bacteriophages was here prevented using a cocktail of three different bacteriophages, as already described in in vitro and in vivo studies (Yu et al., 2018; Yang et al., 2020). The choice of bacteriophages belonging to different genera (Herelleviridae/Kayvirus for V1SC01 and V1SC04 and Rountreeviridae/Andhravirus for V1SC05) may have been beneficial. Indeed, the Myoviridae including Kayvirus and the Podoviridae including Andhravirus are known to target different parts of wall teichoic acids in S. aureus (Moller et al., 2019). However, these are only hypotheses since in the present study, neither the mode of action of the bacteriophages nor their bacterial receptors have been investigated. It is likely that if these receptors are shared between two or three of these bacteriophages, it can increase the risk of bacterial resistance against the cocktail after a longer contact time than 24 h, as previously described (Oechslin, 2018). Moreover, our results demonstrate that the frequency of resistance increased for VISC05 used alone when the MOI increased. This result could be due to a faster selection of the mutant cells at higher bacteriophage concentration. This phenomenon has already been reported in vivo for Salmonella strains (Atterbury et al., 2007).\\nGiven that the bacteriophage cocktail at the higher concentration showed a similar effect than SP on S. capitis biofilm without resistance acquisition, its use in a protocol of decontamination seems achievable. However, because we observed only an incomplete effect on biofilm of either SP or bacteriophage cocktails, we aimed to potentiate the bactericidal effect by testing a sequential treatment using SP exposition followed by bacteriophage cocktail exposition. Unfortunately no additive effect was observed. We hypothesize that the biofilm structure offered a protection for bacteria against those two treatments, even when they were sequentially administered, as already described for antibiotics or disinfectant molecules (Smith and Hunter, 2008; Schilcher and Horswill, 2020). In a previous study, Melo et al. reported that an anti-Staphylococcus epidermidis bacteriophage was ineffective against the bacteria in biofilm whereas it showed efficacy on released biofilm cells and persisters, a subset of the biofilm population, highlighting the protective effect of the biofilm matrix against bacteriophages activity. Indeed, the authors demonstrated by confocal microscopy that bacteriophages had access only to biofilm cells not embedded in N-acetyl-D-glucosamine residues (PNAG) (Melo et al., 2020). The biofilm tridimensional structure and architecture was also shown to protect against bacteriophages in other bacterial species, i.e. Vibrio anguillarum (Tan et al., 2015). Contrariwise, many bacteriophages can also exert damages on biofilms notably thanks to enzymes that target components of the biofilm matrix (Harper et al., 2014; D\\u2019Accolti et al., 2021). For instance, exopolysaccharide degradation by a bacteriophage depolymerase was identified as the first step in the disruption of an Enterobacter agglomerans biofilm (Hughes et al., 1998). In that regard, the use of bacteriophages polysaccharide depolymerases in association with disinfectants was demonstrated to be effective to remove an Enterobacter biofilm (Tait et al., 2002). The combination of lytic bacteriophage with anti-biofilm bacteriophage could lead to a synergistic effect on biofilm. It should be noted that no genes corresponding to polysaccharide depolymerases were identified in the genomes of the three bacteriophages used in the present study (data not shown).\\nAccording to our results showing the absence of total eradication of biofilm-embedded bacteria by a bacteriophage cocktail, the necessity of using a rich medium that can lead to growth of other environmental bacteria and knowing their narrow antibacterial spectrum, bacteriophages alone would not be sufficient to eradicate bacteria from NICU surfaces. Thus, if used, bacteriophages must be combined with other complementary methods in a perspective of environmental disinfection of NICUs. Previous studies have evaluated the efficacy of bacteriophages in association with usual disinfectants in situations of hospital bacterial outbreaks and have shown the effectiveness of these combinations (Roy et al., 1993; Zhang and Hu, 2013; Stachler et al., 2021). Of note, when bacteriophages and disinfectant molecules are tested in combination, it is required to determine the effect of each disinfectant on bacteriophages activity and vice versa to avoid antagonistic effects (Ag\\u00fan et al., 2018; Stachler et al., 2021). Based on our data which did not show a better efficacy of the sequential treatment with bacteriophages then a chemical disinfectant compared to each treatment alone, but also due to the potential adverse effects of chemical molecules, other alternatives have to be explored. Disinfection of the incubators using steam pulverization is also a safe, efficient and simple method that has already been implemented in several NICU settings (Braux et al., 2008; Gillespie et al., 2017; Ory et al., 2019). If it seems more efficient than chemical disinfection, it did not allow for a complete decontamination of incubators. In that context, it might be interesting to develop a disinfection protocol including both the steam pulverization method, which allows non-selective killing of bacteria, and a bacteriophage cocktail to specifically target the persistent NRCS-A strains. However, the order in which the bacteriophage cocktail and steam should be used remains to be defined. Such a protocol deserves to be tested in future studies. Besides, in a perspective of using bacteriophages in real-life decontamination, additional questions concerning production and safety of bacteriophages have to be addressed in futures works. In particular, this project would require to setup a method allowing a large production of the three bacteriophages in controlled conditions ensuring the absence of bacterial residues as toxins and pyrogens in the final product. Another critical point to control is the stability of the bacteriophages that must be maintained during their storage; previous studies have reported that this stability would be obtained if bacteriophages are stored at high concentration (1.109 PFU/mL or more), or lyophilized in presence of sucrose or gelatin during months to years. (Pirnay et al., 2018; Manohar and Ramesh, 2019; Duyvejonck et al., 2021). Moreover, the safety of the treatment in the close environment of preterm neonates is another pivotal point but this will be easily performed since the product will be carefully rinsed after decontamination, to avoid the contact with neonates but also to avoid promoting the regrowth of other bacteria.\\nFinally, the present study presents several limitations. First, the tests confronting bacterial dry spots or biofilm to treatments were performed in a controlled laboratory setting using a single bacterial strain and a single type of material. Although CR01 is the reference strain for the NRCS-A clone, extended tests need to be performed to confirm the observed results on a larger collection of NRCS-A strains. Second, the comparison of the activity of bacteriophages and SP on dry spots has to be cautious given the differences in conditions during those two expositions. Indeed, during their treatment by bacteriophages, bacteria had grown in a rich medium (TSB) during 6 h versus water during 20 minutes for the SP treatment. Thus, the quantification of the effect using LRV is not fully comparable because the bacterial inoculum has evolved differently. Third, the high bacterial inoculum used for the treatments is not representative of the situation of the NICU settings and additional investigations are required before extrapolating our results to real-life practice although the likely lower bacterial inoculum in real life conditions could be in favor of a higher bacteriophage activity and bacterial eradication. Finally, the long-term effect of bacteriophages in the NICU environment but also their effect on eukaryotic cells could be interesting points to explore.\\nTo sum up, the present study constitutes a proof-of-concept for the use of bacteriophages on S. capitis NRCS-A environmental contamination. In the era of increasing resistance to antibiotics and disinfectants, the development of such innovative approach of decontamination using bacteriophages for biocontrol of specific pathogenic bacteria could find its full place in the landscape of infection prevention. Additional evaluations deserve to be performed to define the best and adequate use of biodecontamination and to evaluate its feasibility and efficacy in the NICU environment in conditions simulating real-life conditions.\\nData availability statement\\nThe data presented in the study are deposited in the data.world repository, following the link https://data.world/mariechavignon/phage. The genomes of bacteriophages vB_ScaM-V1SC01, vB_ScaM-V1SC04 and vB_ScaM-V1SC05 are deposited in the GenBank repository, accession number OP323059, OP297178 and OP297179 respectively.\\nAuthor contributions\\nMC and CK designed the study, analyzed the data and wrote the manuscript. MM, AT, FL, and MaB contributed to the design of the study. LB and TL contributed to the isolation, production and host range assessment of bacteriophages. M\\u00e9B performed the bioinformatics analysis. MaB and FL reviewed the manuscript. All authors contributed to the article and approved the submitted version.\\nFunding\\nThis study was supported by the ANR (Agence Nationale de la Recherche) as part of the project NeoSCap [grant number ANR 19-CE17-0004-01] and the project PHAG-ONE [grant number ANR 20-PAMR-0009]. The authors thank the ANR (Agence Nationale de la Recherche) which supported this work as part of the project NeoSCap and the project PHAG-ONE and the Hospices Civils de Lyon Foundation which financed the PHAGEinLYON project.\\nConflict of interest\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nPublisher\\u2019s note\\nAll claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.\\nAbbreviations\\nLRV, Log10 reduction values; MOI, Multiplicity of Infection; NICU, Neonatal Intensive Care Unit; SP, Surfanios Premium.\\nReferences\\nAg\\u00fan, S., Fern\\u00e1ndez, L., Gonz\\u00e1lez-Men\\u00e9ndez, E., Mart\\u00ednez, B., Rodr\\u00edguez, A., Garc\\u00eda, P. (2018). Study of the interactions between bacteriophage phiIPLA-RODI and four chemical disinfectants for the elimination of Staphylococcus aureus contamination. Viruses 10, E103. doi: 10.3390/v10030103\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAtterbury, R. J., Van Bergen, M. A. P., Ortiz, F., Lovell, M. A., Harris, J. A., De Boer, A., et al. (2007). Bacteriophage therapy to reduce Salmonella colonization of broiler chickens. App. Environ. Microbiol. 73, 4543\\u20134549. doi: 10.1128/AEM.00049-07\\nCrossRef Full Text | Google Scholar\\nBraux, C., Lagier, A., Andrini, P., Debillon, T., Croiz\\u00e9, J. (2008). Entretien des incubateurs de n\\u00e9onatalogie \\u00e0 l\\u2019aide d\\u2019un g\\u00e9n\\u00e9rateur de vapeur. Hygienes. XVI. 16, 241\\u2013247.\\nGoogle Scholar\\nButin, M., Dumont, Y., Monteix, A., Raphard, A., Roques, C., Martins Simoes, P., et al. (2019). Sources and reservoirs of Staphylococcus capitis NRCS-a inside a NICU. Antimicrob. Resist. Infect. Control. 8, 157. doi: 10.1186/s13756-019-0616-1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nButin, M., Martins-Sim\\u00f5es, P., Rasigade, J.-P., Picaud, J.-C., Laurent, F. (2017). Worldwide endemicity of a multidrug-resistant Staphylococcus capitis clone involved in neonatal sepsis. Emerg. Infect. Dis. 23, 538\\u2013539. doi: 10.3201/eid2303.160833\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCadot, L., Brugui\\u00e8re, H., Jumas-Bilak, E., Didelot, M.-N., Masnou, A., de Barry, G., et al. (2019). Extended spectrum beta-lactamase-producing Klebsiella pneumoniae outbreak reveals incubators as pathogen reservoir in neonatal care center. Eur. J. Pediatr. 178, 505\\u2013513. doi: 10.1007/s00431-019-03323-w\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCarter, G. P., Ussher, J. E., Da Silva, A. G., Baines, S. L., Heffernan, H., Riley, T. V., et al. (2018). Genomic analysis of multiresistant Staphylococcus capitis associated with neonatal sepsis. Antimicrob. Agents Chemother. 62, e00898\\u2013e00818. doi: 10.1128/AAC.00898-18\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChavignon, M., Reboux, M., Tasse, J., Tristan, A., Claris, O., Laurent, F., et al. (2021). Persistent microbial contamination of incubators despite disinfection. Pediatr. Res. 90, 1215\\u20131220. doi: 10.1038/s41390-021-01407-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCollins, A., Weitkamp, J.-H., Wynn, J. L. (2018). Why are preterm newborns at increased risk of infection? Arch. Dis. Child. - Fetal. Neonatal. Ed. 103, F391\\u2013F394. doi: 10.1136/archdischild-2017-313595\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCui, B., Smooker, P. M., Rouch, D. A., Daley, A. J., Deighton, M. A. (2013). Differences between two clinical Staphylococcus capitis subspecies as revealed by biofilm, antibiotic resistance, and pulsed-field gel electrophoresis profiling. J. Clin. Microbiol. 51, 9\\u201314. doi: 10.1128/JCM.05124-11\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nD\\u0105browska, K., Miernikiewicz, P., Piotrowicz, A., Hodyra, K., Owczarek, B., Lecion, D., et al. (2014). Immunogenicity studies of proteins forming the T4 phage head surface. J. Virol. 88, 12551\\u201312557. doi: 10.1128/JVI.02043-14\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nD\\u2019Accolti, M., Soffritti, I., Lanzoni, L., Bisi, M., Volta, A., Mazzacane, S., et al. (2019). Effective elimination of staphylococcal contamination from hospital surfaces by a bacteriophage-probiotic sanitation strategy: a monocentric study. Microb. Biotechnol. 12, 742\\u2013751. doi: 10.1111/1751-7915.13415\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nD\\u2019Accolti, M., Soffritti, I., Mazzacane, S., Caselli, E. (2021). Bacteriophages as a potential 360-degree pathogen control strategy. Microorganisms 9, 261. doi: 10.3390/microorganisms9020261\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDhama, K., Patel, S. K., Kumar, R., Masand, R., Rana, J., Yatoo, M. I., et al. (2021). The role of disinfectants and sanitizers during COVID-19 pandemic: advantages and deleterious effects on humans and the environment. Environ. Sci. pollut. Res. 28, 34211\\u201334228. doi: 10.1007/s11356-021-14429-w\\nCrossRef Full Text | Google Scholar\\nDuyvejonck, H., Merabishvili, M., Vaneechoutte, M., de Soir, S., Wright, R., Friman, V.-P., et al. (2021). Evaluation of the stability of bacteriophages in different solutions suitable for the production of magistral preparations in Belgium. Viruses 13, 865. doi: 10.3390/v13050865\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGillespie, E., Tabbara, L., Scott, C., Lovegrove, A., Kotsanas, D., Stuart, R. L. (2017). Microfiber and steam for a neonatal service: An improved and safe cleaning methodology. Am. J. Infect. Control. 45, 98\\u2013100. doi: 10.1016/j.ajic.2016.06.041\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGolan, Y., Doron, S., Sullivan, B., Snydman, D. R. (2005). Transmission of vancomycin-resistant Enterococcus in a neonatal intensive care unit. Pediatr. Infect. Dis. J. 24, 566\\u2013567. doi: 10.1097/01.inf.0000164762.03930.0a\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGreen, S. I., Kaelber, J. T., Ma, L., Trautner, B. W., Ramig, R. F., Maresso, A. W. (2017). Bacteriophages from ExPEC reservoirs kill pandemic multidrug-resistant strains of clonal group ST131 in animal models of bacteremia. Sci. Rep. 7, 46151. doi: 10.1038/srep46151\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHarper, D. R., Parracho, H. M. R. T., Walker, J., Sharp, R., Hughes, G., Werth\\u00e9n, M., et al. (2014). Bacteriophages and biofilms. Antibiotics 3, 270\\u2013284. doi: 10.3390/antibiotics3030270\\nCrossRef Full Text | Google Scholar\\nHo, Y.-H., Tseng, C.-C., Wang, L.-S., Chen, Y.-T., Ho, G.-J., Lin, T.-Y., et al. (2016). Application of bacteriophage-containing aerosol against nosocomial transmission of carbapenem-resistant Acinetobacter baumannii in an intensive care unit. PloS One 11, e0168380. doi: 10.1371/journal.pone.0168380\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHrubec, T. C., Seguin, R. P., Xu, L., Cortopassi, G. A., Datta, S., Hanlon, A. L., et al. (2021). Altered toxicological endpoints in humans from common quaternary ammonium compound disinfectant exposure. Toxicol. Rep. 8, 646\\u2013656. doi: 10.1016/j.toxrep.2021.03.006\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHughes, K. A., Sutherland, I. W., Jones, M. V. Y. (1998). Biofilm susceptibility to bacteriophage attack: the role of phage-borne polysaccharide depolymerase. Microbiology 144, 3039\\u20133047. doi: 10.1099/00221287-144-11-3039\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLemriss, H., Sim\\u00f5es, P. M., Lemriss, S., Butin, M., Ibrahimi, A., El Kabbaj, S., et al. (2014). Non-contiguous finished genome sequence of Staphylococcus capitis CR01 (pulsetype NRCS-a). Stand. Genom. Sci. 9, 1118\\u20131127. doi: 10.4056/sigs.5491045\\nCrossRef Full Text | Google Scholar\\nLin, D., Ou, Q., Lin, J., Peng, Y., Yao, Z. (2017). A meta-analysis of the rates of Staphylococcus aureus and methicillin-resistant S. aureus contamination on the surfaces of environmental objects that health care workers frequently touch. Am. J. Infect. Control. 45, 421\\u2013429. doi: 10.1016/j.ajic.2016.11.004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMagill, S. S., O\\u2019Leary, E., Janelle, S. J., Thompson, D. L., Dumyati, G., Nadle, J., et al. (2018). Changes in prevalence of health care\\u2013associated infections in U.S. hospitals. N. Engl. J. Med. 379, 1732\\u20131744. doi: 10.1056/NEJMoa1801550\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMelo, L. D. R., Pinto, G., Oliveira, F., Vilas-Boas, D., Almeida, C., Sillankorva, S., et al. (2020). The protective effect of Staphylococcus epidermidis biofilm matrix against phage predation. Viruses 12, 1076. doi: 10.3390/v12101076\\nCrossRef Full Text | Google Scholar\\nNguyen, S., Baker, K., Padman, B. S., Patwa, R., Dunstan, R. A., Weston, T. A., et al. (2017). Bacteriophage transcytosis provides a mechanism to cross epithelial cell layers. mBio 8, e01874\\u2013e01817. doi: 10.1128/mBio.01874-17\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nOechslin, F. (2018). Resistance development to bacteriophages occurring during bacteriophage therapy. Viruses 10, 351. doi: 10.3390/v10070351\\nCrossRef Full Text | Google Scholar\\nOry, J., Cazaban, M., Richaud-Morel, B., Maio, M. D., Dunyach-Remy, C., Pantel, A., et al. (2019). Successful implementation of infection control measure in a neonatal intensive care unit to combat the spread of pathogenic multidrug resistant Staphylococcus capitis. Antimicrob. Resist. Infect. Control. 8, 1\\u20136. doi: 10.1186/s13756-019-0512-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPirnay, J.-P., Merabishvili, M., Van Raemdonck, H., De Vos, D., Verbeken, G. (2018). \\u201cBacteriophage production in compliance with regulatory requirements,\\u201d in Bacteriophage therapy: From Lab to clinical practice methods in molecular biology. Eds. Azeredo, J., Sillankorva, S. (New York, NY: Springer), 233\\u2013252. doi: 10.1007/978-1-4939-7395-8_18\\nCrossRef Full Text | Google Scholar\\nPodlacha, M., Grabowski, \\u0141., Kosznik-Kaw\\u015bnicka, K., Zdrojewska, K., Stasi\\u0142oj\\u0107, M., W\\u0119grzyn, G., et al. (2021). Interactions of bacteriophages with animal and human organisms\\u2013safety issues in the light of phage therapy. Int. J. Mol. Sci. 22, 8937. doi: 10.3390/ijms22168937\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPurohit, A., Kopferschmitt-Kubler, M.-C., Moreau, C., Popin, E., Blaumeiser, M., Pauli, G. (2000). Quaternary ammonium compounds and occupational asthma. Int. Arch. Occup. Environ. Health 73, 423\\u2013427. doi: 10.1007/s004200000162\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nQu, Y., Li, Y., Cameron, D. R., Easton, C. D., Zhu, X., Zhu, M., et al. (2020). Hyperosmotic infusion and oxidized surfaces are essential for biofilm formation of Staphylococcus capitis from the neonatal intensive care unit. Front. Microbiol. 11. doi: 10.3389/fmicb.2020.00920\\nCrossRef Full Text | Google Scholar\\nRasigade, J.-P., Raulin, O., Picaud, J.-C., Tellini, C., Bes, M., Grando, J., et al. (2012). Methicillin-resistant Staphylococcus capitis with reduced vancomycin susceptibility causes late-onset sepsis in intensive care neonates. PloS One 7, e31548. doi: 10.1371/journal.pone.0031548\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRoy, B., Ackermann, H. W., Pandian, S., Picard, G., Goulet, J. (1993). Biological inactivation of adhering Listeria monocytogenes by listeriaphages and a quaternary ammonium compound. Appl. Environ. Microbiol. 59, 2914\\u20132917. doi: 10.1128/aem.59.9.2914-2917.1993\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSchilcher, K., Horswill, A. R. (2020). Staphylococcal biofilm development: structure, regulation, and treatment strategies. Microbiol. Mol. Bio. Rev. 84, e00026\\u2013e00019. doi: 10.1128/MMBR.00026-19\\nCrossRef Full Text | Google Scholar\\nSikora, A., Zahra, F. (2022). \\u201cNosocomial infections,\\u201d in StatPearls (Treasure Island (FL: StatPearls Publishing).\\nGoogle Scholar\\nSmith, K., Hunter, I. S. Y. (2008). Efficacy of common hospital biocides with biofilms of multi-drug resistant clinical isolates. J. Med. Microbiol. 57, 966\\u2013973. doi: 10.1099/jmm.0.47668-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSong, X., Vossebein, L., Zille, A. (2019). Efficacy of disinfectant-impregnated wipes used for surface disinfection in hospitals: a review. Antimicrob. Resist. Infect. Control. 8, 139. doi: 10.1186/s13756-019-0595-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nStachler, E., Kull, A., Julian, T. R. (2021). Bacteriophage treatment before chemical disinfection can enhance removal of plastic-surface-associated Pseudomonas aeruginosa. Appl. Environ. Microbiol. 87, e00980\\u2013e00921. doi: 10.1128/AEM.00980-21\\nCrossRef Full Text | Google Scholar\\nSuetens, C., Latour, K., K\\u00e4rki, T., Ricchizzi, E., Kinross, P., Moro, M. L., et al. (2018). Prevalence of healthcare-associated infections, estimated incidence and composite antimicrobial resistance index in acute care hospitals and long-term care facilities: results from two European point prevalence surveys 2016 to 2017. Euro. surveill. 23, 1800516. doi: 10.2807/1560-7917.ES.2018.23.46.1800516\\nCrossRef Full Text | Google Scholar\\nSuleyman, G., Alangaden, G., Bardossy, A. C. (2018). The role of environmental contamination in the transmission of nosocomial pathogens and healthcare-associated infections. Curr. Infect. Dis. Rep. 20, 12. doi: 10.1007/s11908-018-0620-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSuneja, T., Belsito, D. V. (2008). Occupational dermatoses in health care workers evaluated for suspected allergic contact dermatitis. Contact. Derm. 58, 285\\u2013290. doi: 10.1111/j.1600-0536.2007.01315.x\\nCrossRef Full Text | Google Scholar\\nTait, K., Skillman, L. C., Sutherland, I. W. (2002). The efficacy of bacteriophage as a method of biofilm eradication. Biofouling 18, 305\\u2013311. doi: 10.1080/0892701021000034418\\nCrossRef Full Text | Google Scholar\\nTan, D., Dahl, A., Middelboe, M. (2015). Vibriophages differentially influence biofilm formation by Vibrio anguillarum strains. Appl. Environ. Microbiol. 81, 4489\\u20134497. doi: 10.1128/AEM.00518-15\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTasse, J., Cara, A., Saglio, M., Villet, R., Laurent, F. (2018). A steam-based method to investigate biofilm. Sci. Rep. 8, 13040. doi: 10.1038/s41598-018-31437-y\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTetz, G., Tetz, V. (2018). Bacteriophages as new human viral pathogens. Microorganisms 6, 54. doi: 10.3390/microorganisms6020054\\nCrossRef Full Text | Google Scholar\\nWirth, T., Bergot, M., Rasigade, J.-P., Pichon, B., Barbier, M., Martins-Simoes, P., et al. (2020). Niche specialization and spread of Staphylococcus capitis involved in neonatal sepsis. Nat. Microbiol. 5, 735\\u2013745. doi: 10.1038/s41564-020-0676-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWright, A., Hawkins, C. H., \\u00c4ngg\\u00e5rd, E. E., Harper, D. R. (2009). A controlled clinical trial of a therapeutic bacteriophage preparation in chronic otitis due to antibiotic-resistant Pseudomonas aeruginosa; a preliminary report of efficacy. Clin. Otolaryngol. 34, 349\\u2013357. doi: 10.1111/j.1749-4486.2009.01973.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYang, Y., Shen, W., Zhong, Q., Chen, Q., He, X., Baker, J. L., et al. (2020). Development of a bacteriophage cocktail to constrain the emergence of phage-resistant Pseudomonas aeruginosa. Front. Microbiol. 11. doi: 10.3389/fmicb.2020.00327\\nCrossRef Full Text | Google Scholar\\nY\\u0131ld\\u0131zl\\u0131, G., Coral, G., Ayaz, F. (2020). Immunostimulatory activities of coliphages on in vitro activated mammalian macrophages. Inflammation 43, 595\\u2013604. doi: 10.1007/s10753-019-01140-9\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYu, L., Wang, S., Guo, Z., Liu, H., Sun, D., Yan, G., et al. (2018). A guard-killer phage cocktail effectively lyses the host and inhibits the development of phage-resistant strains of Escherichia coli. Appl. Microbiol. Biotechnol. 102, 971\\u2013983. doi: 10.1007/s00253-017-8591-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"http://www.sas.upenn.edu/LabManuals/biol275/Table_of_Contents_files/13-PhageGrowth.pdf\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"http://www.sas.upenn.edu/LabManuals/biol275/Table_of_Contents_files/13-PhageGrowth.pdf\",\n",
      "            \"depth\": 1,\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:52.298Z\",\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"html\": null,\n",
      "        \"text\": null,\n",
      "        \"markdown\": null,\n",
      "        \"metadata\": {\n",
      "            \"mimeType\": \"application/pdf\",\n",
      "            \"author\": null,\n",
      "            \"canonicalUrl\": \"http://www.sas.upenn.edu/LabManuals/biol275/Table_of_Contents_files/13-PhageGrowth.pdf\",\n",
      "            \"description\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": null,\n",
      "            \"title\": null\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"fileUrl\": \"https://api.apify.com/v2/key-value-stores/KVxLi5JzsdtaIIa9U/records/4a128da-13-PhageGrowth.pdf\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00875\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00875/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:57.568Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.00875\",\n",
      "            \"title\": \"Frontiers | Bacteriophage Infectivity Against Pseudomonas aeruginosa in Saline Conditions\",\n",
      "            \"description\": \"Pseudomonas aeruginosa is a ubiquitous member of marine biofilm, and reduces thiosulfate to produce toxic hydrogen sulfide gas. In this study, lytic bacteriophages of Podoviridae family were isolated and applied to inhibit the growth of P. aeruginosa in planktonic mode at different temperature, pH and salinity. Bacteriophages showed optimal infectivity at a multiplicity of infection of 10 in saline conditions, and demonstrated lytic abilities over all tested temperature (25, 30, 37 and 45 oC) and pH 6 to 9. Planktonic P. aeruginosa exhibited significantly longer lag phase and lower specific growth rates upon exposure to bacteriophages. Bacteriophages were subsequently applied to P. aeruginosa-enriched biofilm and were determined to lower the relative abundance of Pseudomonas-related taxa from 0.17-5.58% in controls to 0.01-0.61% in treated microbial communities. The relative abundance of Alphaproteobacteria, Pseudoalteromonas and Planococcaceae decreased, possibly due to the phage-induced disruption of the biofilm matrix. Lastly, when applied to mitigate biofouling of ultrafiltration membranes, bacteriophages were determined to reduce the transmembrane pressure increase by 18% when utilized alone, and by 49% when used in combination with citric acid. Collectively, the findings demonstrated that bacteriophages can be used as a biocidal agent to mitigate undesirable P. aeruginosa-associated problems in seawater applications.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nSeawater reverse osmosis (SWRO) desalination has had a great impact on the production of drinking water in the past 40 years (Goosen et al., 2005), and is particularly relied upon by sea-bordered countries that face water scarcity issues (Pe\\u00f1ate and Garc\\u00eda-Rodr\\u00edguez, 2012). Similar to all membrane-based technologies, SWRO is affected by biofouling. Biofilm formation on membrane detrimentally lowers the flux and salt rejection capacity. In order to maintain the desired desalination performance, plant operators usually apply higher transmembrane pressure which in turn increases energy consumption and economic costs (Fritzmann et al., 2007; Matin et al., 2011).\\nBacteria such as Pseudomonas, Bacillus, Mycobacterium, Acinetobacter are often detected on fouled RO membranes (Matin et al., 2011). In particular, Pseudomonas spp. and Desulfovibrio spp. were identified on fouled ultrafiltration (UF) and RO membranes sampled from a SWRO pilot plant located in the Arabian Gulf (Hong et al., 2016). Pseudomonas spp. create an optimal niche for Desulfovibrio spp. and other sulfate-reducing bacteria (SRB) by depleting oxygen. In addition, SRB and Pseudomonas aeruginosa are able to reduce sulfate and thiosulfate, respectively, to produce hydrogen sulfide, a corrosive and toxic gas (Hong et al., 2016). The presence of Pseudomonas spp. and SRB in SWRO are favored by the high sulfate concentration prevalent in seawater. In addition, sodium metabisulfite applied to neutralize chlorine within the SWRO (Hong et al., 2016) further provide a source of electron acceptors for these bacterial populations.\\nTo reduce both inorganic and organic foulants on membranes, oxidizing agents such as chlorine, permanganate, and ozone (Fritzmann et al., 2007; Gao et al., 2011) are used in large concentrations. However, the addition of these chemicals in seawater can result in the formation of carcinogenic and toxic disinfection byproducts that can result in public health concerns (Le Roux et al., 2015; Sanawar et al., 2017). Alternatively, regular cleaning of the membrane would have to be performed with acid cleaning (Greenlee et al., 2009). Citric acid cleaning is often used in the pretreatment stage of SWRO to chelate inorganic minerals and disrupt the stability of biofilm matrix attached on the pretreatment UF membranes (Lee et al., 2001; Porcelli and Judd, 2010). Despite their common usage, these biocidal agents have limited penetration through a biofilm matrix and are less effective against biofilm-associated bacteria compared to its effect on planktonic cells (Matin et al., 2011). Moreover, some chemicals e.g., chlorine, can detrimentally impact RO membrane integrity.\\nTo address the concerns arising from conventional biocides and cleaning agents, this study evaluates the use of lytic bacteriophages (i.e., viruses that infect bacteria at high host specificity) to inhibit the growth of planktonic P. aeruginosa in saline conditions. In addition, bacteriophages can also induce the release of depolymerases that degrade extracellular polymeric substances (EPS) and hence disrupt biofilm matrix (Harper et al., 2014). It is therefore hypothesized that bacteriophage would be effective to disrupt P. aeruginosa-associated biofilms formed on seawater filtration membranes. Although bacteriophage treatment against P. aeruginosa biofilm has been performed in therapeutic treatments (McVay et al., 2007; Fu et al., 2010; Alemayehu et al., 2012; Olszak et al., 2015), and as antifouling agents in water and wastewater membrane filtration systems (Zhang et al., 2013; Bhattacharjee et al., 2015), the conditions at which the bacteriophages were applied in those studies differ from those experienced in a SWRO system. Understanding how environmental conditions can impact bacteriophage infectivity is crucial for an effective application of bacteriophages in SWRO systems, where the conditions vary between high pH (i.e., the usual pH of seawater) to low pH (i.e., when citric acid cleaning is performed), high salinity (i.e., in raw seawater) to low salinity (i.e., after seawater is desalinated), and from low (i.e., in the raw seawater) to high temperatures (i.e., seawater retained within the desalination unit).\\nIn this study, the impact of various parameters, namely multiplicity of infection (MOI), pH, salinity and temperature, on the activity of seven lytic bacteriophages against planktonic P. aeruginosa cells was analyzed. Three bacteriophages were further applied against a P. aeruginosa-enriched seawater biofilm at various pH values and temperatures. Finally, a cocktail of the three bacteriophages was applied alone and in combination with citric acid cleaning to cross-flow seawater ultrafiltration setup, and the membrane modules were assessed for their transmembrane pressure and biofilm cell counts.\\nMaterials and Methods\\nBacteriophage Isolation\\nBacteriophages were isolated from an influent collected from a wastewater treatment plant located in KAUST. A 50 mL aliquot of the influent was centrifuged at 8,500 g for 20 min and thereafter supernatant was filtered through 0.22 \\u03bcm cellulose acetate syringe filter (VWR, Radnor, PA) to remove bacterial cells. The filtrate was mixed with 50 mL Lennox broth (LB) supplemented with 35 g/L NaCl and 4 mM Ca2+, and 50 mL of exponentially growing P. aeruginosa strain DSM1117. The mixed culture was incubated for 24 h at 37\\u00b0C. Thereafter, 1% v/v chloroform was added to lyse the bacterial cells, and incubated for 2 h at room temperature with constant agitation at 100 rpm. The culture was centrifuged at 8,500 g for 30 min at 4\\u00b0C and the supernatant was filtered through 0.22 \\u03bcm syringe filter to remove bacterial cells. Several dilutions, ranging from 10\\u22123 to 10\\u22125 fold, were performed in sodium magnesium (SM) buffer (5.8 g/L NaCl, 0.975 g/L MgSO4, 50 ml/L 50 mM Tris-Cl 7.5 pH) and 10 \\u03bcL of each diluted sample was mixed with 100 \\u03bcL of P. aeruginosa culture to detect the presence of plaques using the double layer plate method (Adams, 1959). Seven plaques were isolated with sterile inoculating loops, transferred to SM buffer and filtered through 0.22 \\u03bcm syringe filter to remove bacterial cells. The bacteriophages from the seven plaques were named P1 through P7 and propagated 10 more times using the above mentioned procedure.\\nInfection of Planktonic Cultures With Bacteriophages at Various Conditions\\nAll bacteriophages were tested for their infectivity against other bacterial species using the soft agar plaque assay method. All the bacteria used in this study are listed in Table 1. After verifying their host specificity, bacteriophage activity against P. aeruginosa was analyzed at various multiplicity of infection (MOI) values, namely 0.1, 1, and 10. MOI represents the ratio between the number of bacteriophage particles determine as phage forming units (PFU/mL) and bacterial cells determined as colony forming units (CFU/mL) in a mixture of 1:1 LB broth and SM buffer, pH 7. MOI was calculated for each bacteriophage based on the number of plaques formed on agar (Figure S1). For each MOI, one control was prepared without bacteriophages. The treated cultures and controls were incubated at 37\\u00b0C with constant shaking at 200 rpm for 24 h. Growth curves of P. aeruginosa were obtained with and without the presence of the seven phages. The best MOI value in terms of bacterial inhibition time (in h) and specific growth rate reduction (h\\u22121) compared with the control was selected for evaluating the impact of other parameters. Subsequently, bacteriophage infectivity against P. aeruginosa cultures was evaluated at different temperatures, pH, and salinity, with the same procedure described above at a constant MOI of 10. All experiments were performed in triplicates.\\nTable 1. Name and origin of the bacteria used in this study.\\nInfection of Biofilm With Bacteriophages at Various Conditions\\nTo examine the effect of bacteriophages on the biofilm structure at various temperatures (25, 30, 37, and 45\\u00b0C) and pH values (5, 6, 7, 8), 28 drip flow reactors (BioSurface Technologies Corp., Bozeman, MT) were assembled. Twenty eight cellulose acetate ultrafiltration membranes (Sterilitech Corporation, Kent, WA), each with surface area of 11.25 cm2, were secured on glass coupons and placed into the drip flow reactors to establish biofilm on the membranes. Reactors were covered with aluminum foil to avoid light exposure. Seawater from Red Sea was fed into each reactor at a continuous rate of 5 mL per minute. The feed was replaced with fresh seawater every 3 d. After 4 weeks, the feed solution was replaced with 5 L of P. aeruginosa-enriched seawater. Briefly, P. aeruginosa was cultivated overnight on Pseudomonas isolation agar (Sigma-Aldrich, St Louis, MO); single colonies were transferred into multiple tubes with 20 mL LB broth and grown at 37\\u00b0C with constant shaking (200 rpm) to reach OD600 of 0.7. After centrifugation at 4,100 g for 30 min, bacterial pellet was resuspended in 5 L seawater to achieve a cell density of 3.5 \\u00d7 107 CFU/mL of P. aeruginosa. This feed was also replaced every 3 d.\\nAfter 4 weeks, to calculate the starting number of Pseudomonas colonies, 4 membranes were removed from the reactor and washed twice in 1\\u00d7 phosphate-buffered saline (PBS) to remove loosely attached biofilm. Membranes were cut into 16 identical pieces and were placed individually into collection tubes containing 2 mL 1\\u00d7 PBS. Attached biofilm was sonicated for 5 min by a Q500 sonicator (Qsonica, Newton, CT, US) at 30% amplitude with 3 s pulsating steps to detach biomass from the membranes. The supernatant was serially diluted for spread plating on Pseudomonas isolation agar and incubated at 37\\u00b0C for 1 d for colony counting. The remaining membranes were removed, rinsed twice in 1\\u00d7 PBS to remove loosely attached biofilm, and aseptically cut into four pieces for a total of 96 identical fragments. Each piece was transferred to an individual 15 mL tube containing LB broth and SM buffer (each in 50% v/v, salinity 3.5%). Bacteriophages P1, P5, and P7 were propagated as described above and were diluted with SM buffer to the adequate number of PFU/mL to attain an MOI value of 10. Each bacteriophage was then used individually to infect attached biofilm on the membrane at four different temperatures (25, 30, 37, and 45\\u00b0C) and pH values (5, 6, 7, 8) for 10 h at constant shaking of 200 rpm. A control without bacteriophage inoculation was prepared for each temperature and pH condition. After 10 h of phage infection, biofilm was sonicated in 3.5 mL of 1\\u00d7 PBS and analyzed based on procedures as described in sections P. aeruginosa Colony and Bacteriophage Plaque Counts on Membranes and RNA Extraction and 16S rRNA Gene Based High-Throughput Sequencing. All experiments were conducted in triplicate.\\nPseudomonas aeruginosa Colony and Bacteriophage Plaque Counts on Membranes\\nA 100 \\u03bcL aliquot of bacterial suspension was used to evaluate the number of colony forming units (CFU)/cm2 on Pseudomonas isolation agar, while 1 mL was filtered through 0.22 \\u03bcm syringe filter to assess the number of plaque forming units (PFU)/cm2 recovered from the different tested conditions.\\nRNA Extraction and 16S rRNA Gene based High-Throughput Sequencing\\nA 2 mL aliquot of the biofilm suspension (as described in section Infection of Biofilm With Bacteriophages at Various Conditions) was utilized for RNA extraction. To avoid RNA degradation, 4 mL of RNAprotect\\u00ae Cell Reagent (Qiagen, Hilden, Germany) was added to each bacterial suspension immediately after sampling. The mixture was incubated at room temperature for 5 min and then centrifuged for 10 min at 5,400 g. After centrifugation, the supernatant was removed and the pellet was stored at \\u221280\\u00b0C for ~2 weeks until RNA extraction.\\nRNA extraction from the biomass pellet was performed using the RNeasy\\u00ae Midi Kit (Qiagen, Hilden, Germany) following manufacturer's protocol and RNA concentration was measured with the Invitrogen RNA HS Qubit\\u00ae 2.0 assay kit (Thermo Fisher Scientific, Carlsbad, CA). Extracted RNA was reverse transcribed into first-strand complementary DNA (cDNA) using the Invitrogen SuperScriptTM First-Strand Synthesis System (Thermo Fisher Scientific, Carlsbad, CA). The cDNA was then used as template to amplify for 16S rRNA genes with primer pair 515F (5\\u2032- Illumina overhang- GTG YCA GCM GCC GCG GTA A-3\\u2032) and 907R (5\\u2032- Illumina overhang- CCC CGY CAA TTC MTT TRA GT-3\\u2032) based on the procedure described earlier (Scarascia et al., 2017). Purified amplicons were submitted to KAUST Genomic Core lab for amplicon sequencing on Illumina MiSeq platform. All high-throughput sequencing files used in this study are deposited in the European Nucleotide Archive (ENA) and are accessible via accession number PRJEB23782.\\nBiofilm Microbial Community Data Analysis\\nAmplicon sequences were sorted on a Phred score >30 and primers, adaptors, and index sequences were removed. After removing any sequences longer than 280 nt, sequence chimeras were identified and removed by UCHIME (Edgar et al., 2011). Chimera-free sequences were split and a subset containing 100,000 sequences were further analyzed for each sample. Taxonomical assignment was obtained at 95% classification reliability level with copy number adjustment using the Ribosomal Database Project (RDP) Classifier (Wang et al., 2007). Relative abundance at genus and phylum level was calculated for each sample. Special emphasis was made to determine the relative abundance of Pseudomonas-related taxa in the presence of bacteriophages compared to the control. Chimera-removed sequence files were also sorted for unique operational taxonomic units (OTUs) at 97% 16S rRNA gene similarity [27]. Similarity percentages (SIMPER) analysis was performed using Primer-E version 7 to identify OTUs that showed a significant difference in their relative abundance between infected and non-infected biofilm (Clarke, 2015). The OTU identities were checked against the NCBI nucleotide database using the BLASTN algorithm.\\nEffect of Bacteriophage Treatment on UF Membrane Fouling\\nA UF filtration system was set up as illustrated in Figure S2. Five UF cellulose acetate membranes of pore size 8 kDa (Sterilitech Corporation, Kent, WA) were aseptically cut into dimensions of 10 by 2.5 cm each, soaked in deionized water for 24 h, and then rinsed in 80% ethanol. Sterile membranes were then individually placed inside cassette modules. Subsequently, deionized water was circulated through the modules to reach a stable trans-membrane pressure (TMP0). The modules were operated in cross-flow mode at room temperature with the concentrate recirculating into the feed tank. A constant flux of 12 L/m2/h (LMH) was maintained. Artificial seawater (26.29 g/L NaCl, 0.74 g/L KCl, 1.32 g/L CaCl2 dehydrated, 6.09 g/L MgCl2\\u00b77H2O, 1.92 g/L MgSO4, pH 7.8) was used as feed solution. Nutrient broth (HiMedia, Mumbai, India) was added to the feed solution to obtain a nutrient concentration of 24 mg/L as described previously (Oh et al., 2017). Feed bottle was inoculated with P. aeruginosa culture at a final concentration of 5 \\u00d7 107 CFU/mL and this feed was replaced every 3 d. Biofilm was established on each membrane for 6 d with a recording of the transmembrane pressure (TMP) every 8 h.\\nOn the 7th day, the filtration system was kept in offline mode for four different cleaning treatments: (i) 5 \\u00d7 108 PFU/mL phage cocktail application in SM buffer for 6 h followed by deionized water for 3 h, (ii) 0.1 M citric acid treatment alone for 3 h followed by deionized water for 6 h, (iii) 5 \\u00d7 108 PFU/mL phage cocktail application in SM buffer for 6 h followed by 0.1 M citric acid treatment for 3 h, and (iv) 0.1 M citric acid treatment for 3 h followed by 5 \\u00d7 108 PFU/mL phage cocktail application in SM buffer for 6 h. One membrane was used as non-treated control where deionized water was applied for the same 9 h duration as that of the treatments. All membranes were flushed for 10 min with deionized water at the end of the treatments, and the TMP monitoring was resumed after 1 h from the treatment. Cleaning cycles were repeated every 3 d for a total of three cycles.\\nAt the end of the third cycle, membranes were removed from the modules and three 2 \\u00d7 2 cm pieces were cut from each membrane sheet and placed in 2 mL 1\\u00d7 PBS to enumerate for the Pseudomonas colony counts and total cell counts. Cells were stained with SYBR\\u00ae green (Thermo Fisher Scientific, Waltham, MA, US) and counted by BD Accuri C6 flow cytometer (BD, Bioscience, NJ, US). The cleaning experiment on the filtration membranes was performed in duplicate as run 1 and run 2.\\nBacteriophages Morphology and Genome Size Characterization\\nPure bacteriophages P1, P5, and P7 in cultures were fixed with 2.5% v/v glutaraldehyde. A 5 \\u03bcL aliquot of each bacteriophage culture was deposited on carbon-coated copper grids and negatively stained with 1% w/v uranyl acetate (pH 4.5). Samples were washed with sterile water, air-dried, and visualized through transmission electron microscopy (TEM) (Tecnai Spirit TWIN, FEI) operated at 120 kV and equipped with an ORIUS SC1000 camera (Gaitan). Image analysis was carried out using ImageJ Software.\\nDirect phage plaques DNA was extracted for the three bacteriophages as described previously (Kot et al., 2014). Briefly, plaques were resuspended in DNase I buffer, filtered and treated for 30 min with DNase I; DNase was inactivated by 10 \\u03bcL of 50 mM EDTA and further treated with Proteinase K (Thermo Scientific, Waltham, USA) before DNA extraction using UltraClean Soil DNA Isolation Kit (MoBio Laboratories, Carlsbad, USA) based on manufacturer's protocol. Bacterial DNA contamination was checked through 16S rRNA gene amplification and all plaque DNA samples showed no amplification for 16S rRNA genes. Extracted DNA was used to assess genome size through pulsed field gel electrophoresis (PFGE). DNA samples were run on 1% w/v agarose gel using the CHEF-DR III system (Bio-Rad, Hercules, CA, USA) (18 h, 6 V cm\\u22121, 14\\u00b0C, angle 120\\u00b0, switch time 0.1\\u20131). Lambda DNA Ladder (New England, Biolabs, Ipswich, MA, USA) was used as size marker.\\nStatistical Analysis\\nOne-way ANOVA was performed to evaluate for statistical differences between treatments. Statistical differences were considered significant at 95% confidence level (p < 0.05).\\nResults\\nBacteriophage Infectivity Against Planktonic Pseudomonas aeruginosa\\nPlaques were only observed when isolated bacteriophages were infected against P. aeruginosa but not against all other tested bacterial hosts (Table 1). Infectivity of the seven isolated bacteriophages against P. aeruginosa growing in planktonic phase was systematically analyzed at various MOIs, temperatures, pH values, and salinities based on lag phase duration and specific growth rates. Lag phase of non-infected control cultures ranged from 1 to 4 h (Figures 1A\\u2013D), while specific growth rates ranged from 0.21 to 0.40 h\\u22121 (Table 2). In contrast, the lag phase duration of the infected bacterial culture was significantly longer compared to the control (p < 0.0001). The longer lag phase duration was observed regardless of the bacteriophage applied.\\nFigure 1. Average duration of the lag phase in infected and non-infected (control) planktonic Pseudomonas aeruginosa culture after phage treatment at various (A) multiplicities of infection (MOIs), (B) temperatures, (C) pH values, and (D) salinities. The standard test conditions were MOI 10, 37\\u00b0C, pH 7, and salinity 3.5% when these parameters were not varied. Bars indicate the standard deviation among the three biological replicates. Letter a indicates statistical difference between each treatment and the respective control at p < 0.0006.\\nTable 2. Average specific bacterial growth rate (h\\u22121) for infected Pseudomonas aeruginosa culture in the presence of bacteriophages (P1\\u2013P7) and in non-infected culture (C).\\nAt a MOI of 10, the difference in the lag phase duration between bacteriophage-treated culture and control was the highest (7.7 h; Figure 1A). Bacterial cultures infected with a MOI of 10 also showed significantly lower specific growth rates (p < 0.0003) compared with the control (Table 2). Infection at MOI 0.1 did not result in any specific growth rate reduction, while a MOI of 1 for phage P4 and P6 was able to reduce specific growth rates (Table 2). As such, subsequent experiments to test infectivity at various temperatures, pH values and salinities were conducted at a MOI value of 10.\\nLag phase was at least 3 times longer in bacteriophage-treated cultures compared with the control over all temperatures tested (p < 0.0001) (MOI 10, pH 7, salinity 3.5%), with particularly longer lag phase at 25 and 30\\u00b0C (Figure 1B). The lag phase was followed by a bacterial growth with lower specific rates compared with the control. Significant reductions in specific growth rates were observed at 25, 30, 37\\u00b0C (p < 0.0003) and 45\\u00b0C (p < 0.03) compared with the respective controls.\\nA longer lag phase of ca. 9 h was observed in the presence of bacteriophage regardless of the pH at a constant MOI of 10, 37\\u00b0C, and 3.5% salinity (p < 0.0001; Figure 1C), compared to the ca. 3 h lag phase for controls. Specific growth rate reduction was also observed when P. aeruginosa was infected at various pH values, with a higher decrease at pH 5 and 6 (p < 0.0005) compared with infection at pH 7 and 8 (p < 0.045; Table 2). Similarly, the lag phase duration was significantly prolonged to ca. 10 h (p < 0.0001) at various salinities in the presence of bacteriophages (Figure 1D). One-way ANOVA test showed significant reduction in specific growth rates in the presence of bacteriophages compared to controls at all tested salinities (p < 0.007).\\nBacteriophage Infectivity Against P. aeruginosa-Enriched Biofilm\\nPseudomonas aeruginosa-enriched seawater biofilm was infected separately with three bacteriophages (P1, P5, and P7) at four different temperatures (25, 30, 37, and 45\\u00b0C) and pH values (5, 6, 7, and 8) for 10 h. At all tested temperatures and upon infection with bacteriophages, the number of viable bacterial cells recovered from Pseudomonas isolation agar decreased significantly to an average of 2.8 \\u00d7 105 CFU/cm2 (p < 0.0001) compared with the initial number (1.37 \\u00d7 106 CFU/cm2) recovered from the membranes (Figure 2A). In contrast, the controls increased significantly to an average of 6.0 \\u00d7 106 CFU/cm2 and the number of bacterial colonies was significantly higher than the initial number (p < 0.04). Most of the bacteriophages, specifically P7, showed lower PFU recovery at high temperature of 45\\u00b0C compared with the other temperatures (Figure 2B). Bacteriophage P7, in particular, had significantly lower PFU counts than the other bacteriophages at 45\\u00b0C. The number of PFU counts for P7 was also significantly different from the PFU counts obtained after infection at 25, 30, and 37\\u00b0C (p < 0.0001 Figure 2B).\\nFigure 2. Effect of phage treatment against Pseudomonas aeruginosa enriched biofilm. The effect was evaluated in terms of (A) viable colony counts on Pseudomonas isolation agar at various temperatures, (B) recovery of plaque counts after 10 h bacteriophage exposure at various temperatures, (C) viable colony counts on Pseudomonas isolation agar at various pH values, and (D) the recovery of plaque counts after 10 h bacteriophage exposure at various pH values. Dashed red lines indicate the number of CFU before infection or the number of PFU spiked for biofilm infection. The standard test conditions were MOI 10, 37\\u00b0C, pH 7, and salinity 3.5% when these parameters were not varied. Bars indicate standard deviation among the three biological replicates. Letters indicate statistical difference between each treatment and the original spiked amount as indicated by red dash lines (a: p < 0.0006, b: p < 0.007, c: p < 0.05).\\nAt all tested pH values, the number of bacterial colonies in infected biofilm decreased significantly to an average of 4.9 \\u00d7 104 CFU/cm2 compared with the initial spiked numbers of 1.7 \\u00d7 106 CFU/cm2 (p < 0.001; 1.5\\u20132 logs reduction). This is in contrast with the bacterial cell count observed for the controls, where the number of bacterial colonies was significantly higher than the initial numbers (p < 0.0001; Figure 2C). Phages were able to replicate by 1.6\\u201313 times of the initial number (1.7 \\u00d7 107 PFU/cm2) at all tested pH values (p < 0.05; Figure 2D).\\nThe Effect of Phage Treatment on Biofilm Microbial Community Based on 16S rRNA Gene-Based Amplicon Sequencing Data Analysis\\nOverall Proteobacteria was the most abundant phylum with an average relative abundance of 95%. At genus level, Vibrio and unclassified Vibrionaceae together accounted for ca. 34.5\\u201365% of the microbial community regardless of the tested condition. Pseudoalteromonas was found to be present at high relative abundance of up to 37.3% of total microbial community, while Alteromonas, Arcobacter, and Thalassospira showed a relative abundance lower than 10%. Unclassified Pseudomonadaceae and Pseudomonas spp. were present in all samples at an average relative abundance of 0.75 and 0.62%, respectively. The relative abundance of these unclassified Pseudomonadaceae and Pseudomonas spp. in infected biofilm ranged from 0.01 to 0.61% of the total community, while in the controls they ranged from 0.17 to 5.58% (Table 3). Specifically, at 25, 30, and 45\\u00b0C, the relative abundance of both Pseudomonas-related taxa in bacteriophage-treated samples was significantly lower compared with the control (p < 0.05). Similarly, when the same experiment was carried out at various pH values, the average of relative abundance of both taxa was significantly lower in bacteriophage-infected samples compared with the controls at all tested pH values (p < 0.03).\\nTable 3. Average of the relative abundance in percentage among the three biological replicates of two Pseudomonas-related taxa (Unclassified Pseudomonadaceae and Pseudomonas).\\nFrom SIMPER analysis, the average dissimilarity between infected biofilm and control ranged from 13.8 to 17.4%. Unclassified Pseudomonadaceae and Pseudomonas accounted for the three main populations impacted by bacteriophages (Table S1). Other genera affected by bacteriophages were Pseudoalteromonas, unclassified Alphaproteobacteria, unclassified Planococcaceae, and unclassified Rhodobacteraceae (Table S1). An OTU-based analysis of biofilm infected at various temperatures and pH values further showed that the only OTU associated with genus Pseudomonas that was significantly impacted by the phage treatment shared at least 98% similarity with P. aeruginosa (Table S2). Finally, an OTU of at least 98% similarity with Pseudoalteromonas shioyasakiensis along with other OTUs related to Thalassospira, Alcanivorax, and Aestuatiibacter genera were also affected by bacteriophage application (Table S2).\\nEffects of Ultrafiltration (UF) Membrane Cleaning Treatments on Transmembrane Pressure (TMP)\\nNon-treated UF membrane showed a rapid increase in its TMP, and reached critical fouling after 130 h of operation. When the first cleaning cycle was implemented at 144 h, the membrane treated with citric acid alone showed a 30% reduction in TMP, while bacteriophage treatment alone resulted in 18% reduction. Bacteriophage in combination with citric acid, regardless of the sequence of treatment, resulted in a TMP decrease of ca. 49%. The same trend was observed for all subsequent cleaning cycles, albeit with decreasing efficacy toward TMP drop with every cycle (Figure 3A). The slopes of TMP increase exhibited by membranes subjected to bacteriophage cleaning alone for all cleaning cycles were significantly lower than those of other treatments (Paired t-test, P < 0.01, Figure S3). There was no statistical difference between the slopes of TMP increase for the respective treatment among the three cleaning cycles (Figure S4).\\nFigure 3. Run 1 of the ultrafiltration membrane experiment. (A) Transmembrane pressure (TMP) profiles of ultrafiltration membranes. TMP was normalized against the initial TMP (TMP0) for each membrane. Different treatments were applied, namely CA, Citric acid; PH, Phage; CA+PH, Citric acid followed by phage; PH+CA, Phage followed by citric acid; C, no treatment. Dashed red lines indicate the point at which treatment was applied over three different cycles. (B) Bacterial sell numbers from plate counts in terms of CFU/mL, (C) and from flow cytometry in terms of cell number/mL, were measured on three 2 \\u00d7 2 cm pieces for each membrane at the end of the experiment. Bars indicate standard deviation among the three biological replicates. Letters indicate statistical difference compared with the control (a: p < 0.0006, b: p < 0.007).\\nThe lower TMP was due to a lower average number of bacterial cells attached on the membranes after the cleaning treatment. To exemplify, the average number of bacterial cells on non-treated membrane fragment was 8.3 \\u00d7 106 CFU/cm2. Instead, all treatments significantly reduced both the bacterial colony counts and total cell counts (p < 0.0001), with the highest log reduction of 1.1-logs observed when the combined cleaning was conducted (Figures 3B,C). Lower TMP and cell counts were also observed in the duplicate experiment (Figure S5).\\nBacteriophage Genome Size and Morphology\\nThe genome sizes of bacteriophages P1, P5, and P7 were determined through pulse field gel electrophoresis (Figure S6), and were observed to share a similar genome size of approximately 48 kbp. This genome size fits in the reported range of both Podoviridae and Siphoviridae families (41.6\\u201379.4 and 34.5\\u201361.1 kbp, respectively; Pires et al., 2015).\\nBacteriophages P1, P5, and P7 were further characterized for their morphology through transmission electron microscopy (TEM). Bacteriophages P1 and P5 had similar morphology, both had an icosahedral head of ~55 and ~47 nm, respectively, and short tail of ~12 and ~9 nm, respectively (Figures S7, S8). Phage P7 was often found in agglomerates (Figure S9) and showed ~60 nm icosahedral head and with no easily identifiable tail (Figure S10). Following the Ackermann classification (Ackermann, 2006), these morphological traits suggest that the three bacteriophages could possibly belong to the order Caudovirales, family Podoviridae, as characterized by icosahedral head with short or no tail.\\nDiscussion\\nIn this study, the feasibility of applying bacteriophages to mitigate biofouling caused by P. aeruginosa in applications involving seawater is investigated. Infectivity against P. aeruginosa in both planktonic phase and biofilm were systematically studied under a range of salinity, temperature, and pH conditions that are representative of different stages of seawater desalination process. In addition, bacteriophages when applied in combination with conventional citric acid cleaning procedure were found to be effective in reducing the transmembrane pressure increment.\\nTo infect bacterial hosts, bacteriophages first attach to host cells and integrate their DNA into the host genome without killing the hosts. Subsequently, in response to environmental or other signal cues from the bacterial hosts (Echols, 1972), bacteriophages then enter into lytic infection mode and result in the death of bacterial hosts through the induction of a suite of proteins including holins, endolysins, and spanins. These proteins collectively disrupt the cell membranes and achieve cell lysis (Young, 2013, 2014). Furthermore, bacteriophages possess enzymes that degrade the extracellular polymeric substances of biofilm matrix (Harper et al., 2014), in turn dispersing the biofilm structure to increase the susceptibility of bacterial cells to biocides or chemical cleaning. This is especially relevant for tackling membrane fouling issues in SWRO plants since traditional chemical cleaning and chlorination have limited effect in eradicating bacteria such as Pseudomonas spp. and SRB (Khan et al., 2015; Hong et al., 2016), both of which have been found to be associated with membrane fouling and production of corrosive hydrogen sulfide (Hong et al., 2016).\\nMost host-specific bacteriophages like the ones isolated in this study exhibit lytic actions that are specific against a limited number of hosts, and the use of host-specific bacteriophages can potentially minimize unintentional ecological impacts on the indigenous microbial community present within the ecosystem. However, to facilitate the use of bacteriophages as antifouling agent for seawater applications, parameters such as bacteriophage-host ratio, temperature, pH, and salinity need to be taken in consideration. In natural marine environments, bacteriophages distribution and proliferation are determined by the productivity and density of the specific host population. In our study, the highest lag phase extension and slowest specific growth rate were also observed for P. aeruginosa infected at a MOI of 10 (Figure 1A and Table 2) and in saline media (Figure 1D and Table 2). In natural marine environments, bacteriophages distribution and proliferation are determined by the productivity and density of the specific host population. In this study, a MOI value of 10 was the optimal for bacteriophage infection. It is likely that this MOI value maximized encounter rates between bacteriophage and bacterial cell, hence ensuring that nearly all host cells are infected by at least one bacteriophage particle.\\nBesides an optimal bacteriophage-host ratio, temperature plays a fundamental role in governing the attachment, proliferation, and cell lysis efficiencies. Attachment is dependent on capsid proteins of bacteriophages and surface proteins on bacterial hosts, both of which are sensitive to high temperatures. The findings of this work suggest that the isolated bacteriophages were able to infect within a broad temperature range of 25\\u201345\\u00b0C. However, the highest tested temperature resulted in a lower reduction in specific growth rates of P. aeruginosa compared to the other temperatures (Table 2). The lower lytic efficiency at 45\\u00b0C can be accounted for by a lower bacterial growth and replication (Figure 2A), which may have provided lower surface area for bacteriophage adsorption (Sillankorva et al., 2004). Also, the high temperature of 45\\u00b0C affected bacteriophage replication as evidenced from the lower PFU recovery compared to other temperatures (Figure 2B). Among the isolated bacteriophages, P7 seemed to be particularly sensitive to high temperature of 45\\u00b0C. The PFU recovery for this bacteriophage showed a complex trend in which it first decreased at 30\\u00b0C compared with the one at 25\\u00b0C (p = 0.004) and it subsequently increased again at 37\\u00b0C (p = 0.04) before decreasing at 45\\u00b0C (p < 0.0001). The exact reasons to account for this cyclical behavior are not known. However, it may be possible that this bacteriophage was more affected than P1 and P5 by the different host metabolic activity at different temperatures. Bacterial density and metabolic activities are, in fact, key factors for an efficient bacteriophage proliferation (Chibani-Chennoufi et al., 2004). High temperatures can also result in lower infection efficiency due to alteration made in the capsid proteins and bacterial surface receptors, hence inhibiting the initial phage attachment (Sillankorva et al., 2004). Similarly, Kwiatek and colleagues observed a 1 order of magnitude reduction in the virus particles concentration when bacteriophages from Podoviridae family were maintained at 50\\u00b0C (Kwiatek et al., 2015).\\nIn addition to temperature, pH is another important parameter that affects bacteriophage activity. Environmental pH can change the charge state of the amino acids in bacteriophage capsids, which in turn lead to modifications in the capsid structure (Nap et al., 2014). Moreover, at pH close to the isoelectric point, virus aggregation was observed in MS2 bacteriophage (Langlet et al., 2007), which can contribute to significant bias during PFU enumeration. Despite this, the pH range analyzed in this study exhibited no apparent effect on bacteriophage infectivity against both planktonic cells (Figure 1C, Table 2) and biofilm (Figures 2C,D), suggesting the applied pH was feasible for application in SWRO, which is generally operated within this pH range.\\nIn an earlier study, bacteriophages were stored at a similar pH range and were determined to be stable (Kwiatek et al., 2015). Coincidentally, both in this and the earlier study, isolated lytic bacteriophages were possibly associated with the Podoviridae family. Podoviridae were shown to survive large temperature fluctuations compared to bacteriophages with smaller genome sizes (Jonczyk et al., 2011). A complete genome sequencing of the isolated bacteriophages is not done for this study and it was therefore not possible to provide better and definitive information about bacteriophage classification and infection mechanisms at different conditions. However, based on morphological traits, Podoviridae are generally characterized by a big icosahedral head and short tail, which were observed for the bacteriophages in this study. These phenotypic characteristics observed may have accounted for the stability of the bacteriophages observed over the wide range of temperatures and pH values. A complete genomic and taxonomical characterization for all bacteriophages used in this study would be necessary to confirm this hypothesis.\\nWhen evaluated for their host specificities, it was determined that the bacteriophages were not only able to delay the growth of planktonic cells, but were also able to statistically reduce the relative abundance of Pseudomonas-related taxa within the biofilm (Table 3). Specifically, OTU analysis showed that P. aeruginosa was detrimentally impacted by the bacteriophages (Table S2). However, it was also observed that other taxa such as unclassified Alphaproteobacteria and Rhodobacteraceae decreased in their relative abundance. Bacteriophages used in this study showed no infectivity against non-P. aeruginosa bacterial hosts. This means that the effect on other bacteria within the biofilm matrix was likely due to an indirect action rather than a direct virus infection. Indeed, these two taxa are within the most abundant Proteobacteria phylum, and the reduction may be due to the loss of bacterial cells upon bacteriophage-induced biofilm disruption. Pseudoalteromonas was also detrimentally impacted by bacteriophage application. This genus has been reported to play an important role in marine biofilm by contributing to the EPS production (Dheilly et al., 2010; Liu et al., 2016) and its decrease in relative abundance reiterate the effect of bacteriophages on biofilm matrix. Finally, the decrease in relative abundance of Gram-positive Planococcaceae could be due to a higher susceptibility to phage lysins since in the absence of outer membrane, those proteins can make direct contact with cell wall carbohydrates and peptidoglycans even if Planococcaceae is not the target host (Fischetti, 2005).\\nDeveloping resistance to bacteriophage remains to be of a concern. This was exemplified when bacteria eventually overcame the bacteriophage to grow albeit at a lower specific growth rate compared to the same host grown in absence of bacteriophage (Table 2). Similar trend was also observed in a separate study for P. aeruginosa infected with Podoviridae phages (Alves et al., 2016). The bacterial growth after a prolonged lag phase indicates the evolution of resistance to bacteriophage infection after a certain period of bacteria-virus interaction. Small gene mutation or change/loss of receptor proteins among a subpopulation of cells can result in the loss of bacteriophage infectivity (Ly-Chatain, 2014). Bacterial resistance to bacteriophage infection can also occur by the production of competitive inhibitors that bind to phage receptor or by preventing phage DNA entry (Labrie et al., 2010). It was observed that P. aeruginosa phage resistance can affect also the expression of virulence factors (Hosseinidoust et al., 2013a,b) but incurs a fitness cost for the bacteria in terms of specific growth rate and metabolic activity (Hall et al., 2012). This can potentially account for the lower specific bacterial growth rates observed in presence of bacteriophages (Table 2). Besides gaining resistance, bacterial regrowth could possibly be related to the establishment of phage lysogeny which leads to the presence of prophages in the bacterial cells instead of cell lysis, hence allowing bacterial regrowth. Further studies would have to be carried out to determine the cause of this bacterial regrowth observed in presence of bacteriophages.\\nSubsequently, when bacteriophages were applied to the P. aeruginosa-enriched biofilm attached on UF membranes, it was observed that the starting TMP at each initial point of cleaning cycle was higher than the previous cycle (Figures S4, S5). This suggests a slight loss in bacteriophage efficacy with each application. Since the slopes of TMP increase after each cleaning cycle were not significantly different, the increase of initial TMP at each cleaning cycle was most likely not due to changes in metabolic rates or infectivity rates. Instead, this can potentially be explained by an increased production of EPS that would provide a physical barrier between bacteriophages and their receptors (Labrie et al., 2010). Earlier studies have reported a higher reduction in TMP decline compared to this study (i.e., ca. 18% decrease). For instance, Goldman and colleagues observed a 47% reduction in the UF permeability drop after phage application compared with the control (Goldman et al., 2009). However, bacteriophages were introduced in the feed solution and not directly on the formed biofilm, and the higher reduction may be due to a combined effect on both planktonic and biofilm-associated bacteria. In a full-scale SWRO, direct application of bacteriophage to a large volume of seawater would be impractical and costly as this means a large dose of bacteriophage would be required to maintain infectivity at an optimal MOI. In another study, Bhattacharjee et al. observed a 53 and 78% flux recovery after 1 and 2 days of phage application, respectively (Bhattacharjee et al., 2015). However, hollow-fiber membranes were conditioned with nutrient rich medium instead of environmental waters, and both bacteriophages and host may be more metabolically active to facilitate interactions.\\nIsolation of new bacteriophages or application of phages cocktail can be adopted to mitigate problems associated with phage resistance and lysogeny. However, isolating new bacteriophages can be challenging and time-consuming for long-term operations. Alternatively, this study demonstrated the feasibility of combining bacteriophages application with the common membrane cleaning procedure. Approximately 49% TMP recovery and 1.1 log biofilm removal was observed when UF membrane was treated with bacteriophages and citric acid, regardless of the order of application (Figure 3). This confirms that bacteriophages are not affected by the acidic pH and that they can be easily integrated with the common membrane treatment procedure to enhance the cleaning efficacy. Citric acid chelates inorganic minerals and disrupts the stability of biofilm matrix attached on the pretreatment UF membranes (Lee et al., 2001; Porcelli and Judd, 2010). Bacteriophages are able to further disrupt the biofilm structure, hence allowing for a better cleaning efficacy. Similarly, bacteriophages were applied successfully in combination with chlorine treatment against P. aeruginosa biofilm (Zhang and Hu, 2013). These findings suggest that bacteriophage can be applied synergistically with existing biocides or cleaning strategies to effectively mitigate biofouling associated with P. aeruginosa. Alternatively, other bacteriophages specific against other host targets, for example Pseudoalteromonas which is typically present in high abundance on a marine biofilm layer, can also be isolated and verified for their lytic efficiency against this new host based on the procedure described in this study. Finally, it could be interesting to analyze the effect of bacteriophages treatment in preventing biofilm formation rather than removing an established biofilm structure. In this context, the lower cell density could represent a limitation for bacteriophages activity and infectivity. A cocktail of bacteriophages from different taxonomical families targeting different bacterial host could also represent a better approach to maximize the impact of the infection and to further mitigate seawater membrane biofouling in the future for bigger scale operating systems.\\nConclusion\\nThis study demonstrated the ability of bacteriophages to infect planktonic P. aeruginosa and to reduce biofilm formation. Infectivity at different environmental conditions was systematically evaluated and bacteriophages showed great versatility to infect P. aeruginosa over a wide temperature and pH range. Although the bacteriophages were able to effectively reduce the relative abundance of Pseudomonas-related taxa, other taxa including Pseudoalteromonas and predominant Proteobacteria within a marine biofilm were also affected, suggesting that the bacteriophages lyse P. aeruginosa and disrupt the biofilm matrix. Finally, bacteriophages were demonstrated to be feasible for reducing P. aeruginosa biofouling on a lab-scale UF membrane. Specifically, the best reduction in transmembrane pressure was obtained when bacteriophage treatment was combined with citric acid cleaning. Collectively, the findings demonstrate that bacteriophages can be used as a biocidal agent to mitigate undesirable P. aeruginosa-associated problems in seawater applications.\\nAuthor Contributions\\nGS and P-YH: Designed the experiment; GS and SY: Performed the experiments and analyzed the data; GS, P-YH, and AK: Wrote and revised the manuscript.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nThe research reported in this publication was supported by CRG funding URF/1/2982-01-01 from King Abdullah University of Science and Technology (KAUST) awarded to P-YH. AK thanks KAUST and CSIRO Land and Water for financial support.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2018.00875/full#supplementary-material\\nTable S1. Similarity percentages (SIMPER) analysis of the average composition of the microbial communities in phage infected and non-infected biofilms.\\nTable S2. OTUs affected by bacteriophage infection are listed together with the closest match identified by the BLASTN algorithm.\\nFigure S1. Phage forming units for each bacteriophage propagated with the double layer plate method. Bars indicate standard deviation.\\nFigure S2. Cross-flow ultrafiltration membrane setup.\\nFigure S3. Values of TMP/TMP0 increment slopes, averaged from the three cycles of UF membrane cleaning. a, b represent homogenous subgroups by two-tailed t-test.\\nFigure S4. Values of TMP/TMP0 increment slopes for each cycle of membrane cleaning.\\nFigure S5. Run 2 of the ultrafiltration membrane experiment. (A) Transmembrane pressure (TMP) profiles of ultrafiltration membranes. TMP was normalized against the initial TMP (TMP0) for each membrane. Different treatments were applied, namely CA, Citric acid; PH, Phage; CA+PH, Citric acid followed by phage; PH+CA, Phage followed by citric acid; C, no treatment. Dashed red lines indicate the point at which treatment was applied over three different cycles. (B) Cell numbers from plate counts in terms of CFU/mL, (C) and from flow cytometry in terms of cells number/mL, were measured on three 2 \\u00d7 2 cm pieces for each membrane at the end of the experiment. Bars indicate standard deviation among the three biological replicates. Letters indicate statistical difference compared with the control (a: p < 0.0006, b: p < 0.007 and c: p < 0.05).\\nFigure S6. Pulsed field gel electrophoresis images of P1, P5, and P7 genomes. \\u03bb and a indicate lambda and 1 kbp ladder respectively used as markers.\\nFigure S7. Transmission electron microscopy images of the isolated bacteriophage P1.\\nFigure S8. Transmission electron microscopy images of the isolated bacteriophage P5.\\nFigure S9. Transmission electron microscopy images of the isolated bacteriophage P7 in aggregate.\\nFigure S10. Transmission electron microscopy images of the isolated bacteriophage P7.\\nReferences\\nAckermann, H. W. (2006). \\u201cClassification of bacteriophages,\\u201d in The Bacteriophages, ed R. Calendar (New York, NY: Oxford University Press), 8\\u201316.\\nGoogle Scholar\\nAdams, M. H. (1959). Bacteriophages. New York, NY: Interscience Publishers Inc.\\nGoogle Scholar\\nAlemayehu, D., Casey, P. G., McAuliffe, O., Guinane, C. M., Martin, J. G., Shanahan, F., et al. (2012). Bacteriophages \\u03c6MR299-2 and \\u03c6NH-4 can eliminate Pseudomonas aeruginosa in the murine lung and on cystic fibrosis lung airway cells. MBio 3, e00029\\u2013e00012. doi: 10.1128/mBio.00029-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAl-Jassim, N., Ansari, M. I., Harb, M., and Hong, P.-Y. (2015). Removal of bacterial contaminants and antibiotic resistance genes by conventional wastewater treatment processes in Saudi Arabia: is the treated wastewater safe to reuse for agricultural irrigation? Water Res. 73, 277\\u2013290. doi: 10.1016/j.watres.2015.01.036\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAlves, D. R., Perez-Esteban, P., Kot, W., Bean, J., Arnot, T., Hansen, L. H., et al. (2016). A novel bacteriophage cocktail reduces and disperses Pseudomonas aeruginosa biofilms under static and flow conditions. Microb. Biotechnol. 9, 61\\u201374. doi: 10.1111/1751-7915.12316\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBhattacharjee, A. S., Choi, J., Motlagh, A. M., Mukherji, S. T., and Goel, R. (2015). Bacteriophage therapy for membrane biofouling in membrane bioreactors and antibiotic-resistant bacterial biofilms. Biotechnol. Bioeng. 112, 1644\\u20131654. doi: 10.1002/bit.25574\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChibani-Chennoufi, S., Bruttin, A., Dillmann, M.-L., and Br\\u00fcssow, H. (2004). Phage-host interaction: an ecological perspective. J. Bacteriol. 186, 3677\\u20133686. doi: 10.1128/JB.186.12.3677-3686.2004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nClarke, K. G. R. (2015). PRIMER Version 7: User Manual/Tutorial. Plymouth: PRIMER-E.\\nDheilly, A., Soum-Sout\\u00e9ra, E., Klein, G. L., Bazire, A., Comp\\u00e8re, C., Haras, D., et al. (2010). Antibiofilm activity of the marine bacterium Pseudoalteromonas sp. strain 3J6. Appl. Environ. Microbiol. 76, 3452\\u20133461. doi: 10.1128/AEM.02632-09\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nEdgar, R. C., Haas, B. J., Clemente, J. C., Quince, C., and Knight, R. (2011). UCHIME improves sensitivity and speed of chimera detection. Bioinformatics 27, 2194\\u20132200. doi: 10.1093/bioinformatics/btr381\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nFritzmann, C., L\\u00f6wenberg, J., Wintgens, T., and Melin, T. (2007). State-of-the-art of reverse osmosis desalination. Desalination 216, 1\\u201376. doi: 10.1016/j.desal.2006.12.009\\nCrossRef Full Text | Google Scholar\\nFu, W., Forster, T., Mayer, O., Curtin, J. J., Lehman, S. M., and Donlan, R. M. (2010). Bacteriophage cocktail for the prevention of biofilm formation by Pseudomonas aeruginosa on catheters in an in vitro model system. Antimicrob. Agents Chemother. 54, 397\\u2013404. doi: 10.1128/AAC.00669-09\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGao, W., Liang, H., Ma, J., Han, M., Chen, Z.-L., Han, Z.-S., et al. (2011). Membrane fouling control in ultrafiltration technology for drinking water production: a review. Desalination 272, 1\\u20138. doi: 10.1016/j.desal.2011.01.051\\nCrossRef Full Text | Google Scholar\\nGoldman, G., Starosvetsky, J., and Armon, R. (2009). Inhibition of biofilm formation on UF membrane by use of specific bacteriophages. J. Memb. Sci. 342, 145\\u2013152. doi: 10.1016/j.memsci.2009.06.036\\nCrossRef Full Text | Google Scholar\\nGoosen, M., Sablani, S., Al-Hinai, H., Al-Obeidani, S., Al-Belushi, R., and Jackson, D. (2005). Fouling of reverse osmosis and ultrafiltration membranes: a critical review. Sep. Sci. Technol. 39, 2261\\u20132297. doi: 10.1081/SS-120039343\\nCrossRef Full Text | Google Scholar\\nGreenlee, L. F., Lawler, D. F., Freeman, B. D., Marrot, B., and Moulin, P. (2009). Reverse osmosis desalination: water sources, technology, and today's challenges. Water Res. 43, 2317\\u20132348. doi: 10.1016/j.watres.2009.03.010\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHall, A. R., De Vos, D., Friman, V.-P., Pirnay, J.-P., and Buckling, A. (2012). Effects of sequential and simultaneous applications of bacteriophages on populations of Pseudomonas aeruginosa in vitro and in wax moth larvae. Appl. Environ. Microbiol. 78, 5646\\u20135652. doi: 10.1128/AEM.00757-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHarper, D. R., Parracho, H. M., Walker, J., Sharp, R., Hughes, G., Werth\\u00e9n, M., et al. (2014). Bacteriophages and biofilms. Antibiotics 3, 270\\u2013284. doi: 10.3390/antibiotics3030270\\nCrossRef Full Text | Google Scholar\\nHong, P.-Y., Moosa, N., and Mink, J. (2016). Dynamics of microbial communities in an integrated ultrafiltration\\u2013reverse osmosis desalination pilot plant located at the Arabian Gulf. Desalination Water Treat. 57, 16310\\u201316323. doi: 10.1080/19443994.2015.1083483\\nCrossRef Full Text | Google Scholar\\nHosseinidoust, Z., Tufenkji, N., and Van De Ven, T. G. (2013a). Predation in homogeneous and heterogeneous phage environments affects virulence determinants of Pseudomonas aeruginosa. Appl. Environ. Microbiol. 79, 2862\\u20132871. doi: 10.1128/AEM.03817-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHosseinidoust, Z., Van De Ven, T. G., and Tufenkji, N. (2013b). Evolution of Pseudomonas aeruginosa virulence as a result of phage predation. Appl. Environ. Microbiol. 79, 6110\\u20136116. doi: 10.1128/AEM.01421-13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKhan, M. T., Hong, P.-Y., Nada, N., and Croue, J. P. (2015). Does chlorination of seawater reverse osmosis membranes control biofouling? Water Res. 78, 84\\u201397. doi: 10.1016/j.watres.2015.03.029\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKot, W., Vogensen, F. K., S\\u00f8rensen, S. J., and Hansen, L. H. (2014). DPS\\u2013a rapid method for genome sequencing of DNA-containing bacteriophages directly from a single plaque. J. Virol. Methods 196, 152\\u2013156. doi: 10.1016/j.jviromet.2013.10.040\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKwiatek, M., Mizak, L., Parasion, S., Gryko, R., Olender, A., and Niemcewicz, M. (2015). Characterization of five newly isolated bacteriophages active against Pseudomonas aeruginosa clinical strains. Folia Microbiol. 60, 7\\u201314. doi: 10.1007/s12223-014-0333-3\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLanglet, J., Gaboriaud, F., and Gantzer, C. (2007). Effects of pH on plaque forming unit counts and aggregation of MS2 bacteriophage. J. Appl. Microbiol. 103, 1632\\u20131638. doi: 10.1111/j.1365-2672.2007.03396.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLee, H., Amy, G., Cho, J., Yoon, Y., Moon, S.-H., and Kim, I. S. (2001). Cleaning strategies for flux recovery of an ultrafiltration membrane fouled by natural organic matter. Water Res. 35, 3301\\u20133308. doi: 10.1016/S0043-1354(01)00063-X\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLe Roux, J., Nada, N., Khan, M. T., and Crou\\u00e9, J.-P. (2015). Tracing disinfection byproducts in full-scale desalination plants. Desalination 359, 141\\u2013148. doi: 10.1016/j.desal.2014.12.035\\nCrossRef Full Text | Google Scholar\\nLiu, A., Mi, Z.-H., Zheng, X.-Y., Yu, Y., Su, H.-N., Chen, X.-L., et al. (2016). Exopolysaccharides play a role in the swarming of the benthic bacterium Pseudoalteromonas sp. SM9913. Front. Microbiol. 7:473. doi: 10.3389/fmicb.2016.00473\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMatin, A., Khan, Z., Zaidi, S., and Boyce, M. (2011). Biofouling in reverse osmosis membranes for seawater desalination: phenomena and prevention. Desalination 281, 1\\u201316. doi: 10.1016/j.desal.2011.06.063\\nCrossRef Full Text | Google Scholar\\nMcVay, C. S., Vel\\u00e1squez, M., and Fralick, J. A. (2007). Phage therapy of Pseudomonas aeruginosa infection in a mouse burn wound model. Antimicrob. Agents Chemother. 51, 1934\\u20131938. doi: 10.1128/AAC.01028-06\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nNap, R. J., Bo\\u017di\\u010d, A. L., Szleifer, I., and Podgornik, R. (2014). The role of solution conditions in the bacteriophage PP7 capsid charge regulation. Biophys. J. 107, 1970\\u20131979. doi: 10.1016/j.bpj.2014.08.032\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nOh, H.-S., Tan, C. H., Low, J. H., Rzechowicz, M., Siddiqui, M. F., Winters, H., et al. (2017). Quorum quenching bacteria can be used to inhibit the biofouling of reverse osmosis membranes. Water Res. 112, 29\\u201337. doi: 10.1016/j.watres.2017.01.028\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nOlszak, T., Zarnowiec, P., Kaca, W., Danis-Wlodarczyk, K., Augustyniak, D., Drevinek, P., et al. (2015). In vitro and in vivo antibacterial activity of environmental bacteriophages against Pseudomonas aeruginosa strains from cystic fibrosis patients. Appl. Microbiol. Biotechnol. 99, 6021\\u20136033. doi: 10.1007/s00253-015-6492-6\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPe\\u00f1ate, B., and Garc\\u00eda-Rodr\\u00edguez, L. (2012). Current trends and future prospects in the design of seawater reverse osmosis desalination technology. Desalination 284, 1\\u20138. doi: 10.1016/j.desal.2011.09.010\\nCrossRef Full Text | Google Scholar\\nPires, D. P., Boas, D. V., Sillankorva, S., and Azeredo, J. (2015). Phage therapy: a step forward in the treatment of Pseudomonas aeruginosa infections. J. Virol. 89, 7449\\u20137456. doi: 10.1128/JVI.00385-15\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPorcelli, N., and Judd, S. (2010). Chemical cleaning of potable water membranes: a review. Sep. Purif. Technol. 71, 137\\u2013143. doi: 10.1016/j.seppur.2009.12.007\\nCrossRef Full Text | Google Scholar\\nSanawar, H., Xiong, Y., Alam, A., Croue, J.-P., and Hong, P.-Y. (2017). Chlorination or monochloramination: balancing the regulated trihalomethane formation and microbial inactivation in marine aquaculture waters. Aquaculture 480, 94\\u2013102. doi: 10.1016/j.aquaculture.2017.08.014\\nCrossRef Full Text | Google Scholar\\nScarascia, G., Cheng, H., Harb, M., and Hong, P.-Y. (2017). Application of hierarchical oligonucleotide primer extension (HOPE) to assess relative abundances of ammonia-and nitrite-oxidizing bacteria. BMC Microbiol. 17:85. doi: 10.1186/s12866-017-0998-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSillankorva, S., Oliveira, R., Vieira, M. J., Sutherland, I., and Azeredo, J. (2004). Pseudomonas fluorescens infection by bacteriophage \\u03a6S1: the influence of temperature, host growth phase and media. FEMS Microbiol. Lett. 241, 13\\u201320. doi: 10.1016/j.femsle.2004.06.058\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWang, Q., Garrity, G. M., Tiedje, J. M., and Cole, J. R. (2007). Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy. Appl. Environ. Microbiol. 73, 5261\\u20135267. doi: 10.1128/AEM.00062-07\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nZhang, Y., Hunt, H. K., and Hu, Z. (2013). Application of bacteriophages to selectively remove Pseudomonas aeruginosa in water and wastewater filtration systems. Water Res. 47, 4507\\u20134518. doi: 10.1016/j.watres.2013.05.014\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2020.580779\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2020.580779/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:49:58.786Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2020.580779\",\n",
      "            \"title\": \"Frontiers | Identification and Characterization of New Bacteriophages to Control Multidrug-Resistant Pseudomonas aeruginosa Biofilm on Endotracheal Tubes\",\n",
      "            \"description\": \"Studies involving antimicrobial-coated endotracheal tubes are scarce, and new approaches to control multidrug-resistant Pseudomonas aeruginosa biofilm on these devices should be investigated. In this study, five new P. aeruginosa bacteriophages from domestic sewage were isolated. All of them belong to the order Caudovirales, Myoviridae family. They are pH and heat stable and produce from 27 to 46 particles after a latent period of 30 min at 37oC. Their dsDNA genome (ranging from \\u205362 kb to \\u205365 kb), encode from 65 to 89 different putative proteins. They exhibit a broad lytic spectrum and infect 69.7% of the P. aeruginosa strains tested. All bacteriophages were able to reduce the growth of P. aeruginosa strains in the planktonic form. Bacteriophages were also able to reduce biofilm viability rates and metabolic activity of P. aeruginosa strains in a model of biofilms associated with endotracheal tubes. In addition, scanning electron microscopy micrographs showed disrupted biofilms and cell debris after treatment of bacteriophages revealing a remarkable biofilm reduction. The lytic activity on multidrug-resistant P. aeruginosa biofilm indicates that the isolated bacteriophages might be considered good candidates for therapeutic studies and for the application of bacteriophage-encoded products.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nMechanical ventilation through an endotracheal or tracheostomy tube is a supportive intervention to critically ill patients and represents a risk factor for the occurrence of ventilator-associated pneumonia (VAP; Haas et al., 2014). VAP is the most common healthcare-associated infection in critically ill patients, involves high financial costs, and has negative prognosis (Barbier et al., 2013; Zimlichman et al., 2013; Russotto et al., 2015). VAP etiologies have been widely discussed, and the factors associated with its development include endotracheal tube biofilm formation which plays an important role as a reservoir for microorganisms (Nseir et al., 2011).\\nKnowing the potential risk factor of multidrug-resistant Pseudomonas aeruginosa biofilms on endotracheal tubes, interest in the development of antimicrobial-coated endotracheal tubes has emerged. The presence of antimicrobial agents, such as silver particles and chlorhexidine, on the surface of endotracheal tubes demonstrated a decrease in biofilm formation on the device surface\\u2019s (Berra et al., 2004; Tokmaji et al., 2015). However, chemical compounds have been implicated in bacterial resistance (Cieplik et al., 2019) and frequently linked to potential cytotoxic hazards (Lessa et al., 2010; Mao et al., 2018). Therefore, new approaches to control multidrug-resistant P. aeruginosa biofilm on endotracheal tubes should be investigated.\\nThe applicability of bacteriophages in medical devices has aroused interest in several areas, including infections related to the urinary tract (Nzakizwanayo et al., 2015; Melo et al., 2016), venous catheters (Lungren et al., 2014), and orthopedic prostheses (Kaur et al., 2014; Barros et al., 2020). Recently, new lytic bacteriophages able to control multidrug-resistant P. aeruginosa planktonic cells and associated-biofilm forms were described (Yuan et al., 2019; Adnan et al., 2020). Although promising, there are few reports on bacteriophage applicability to control biofilms associated with endotracheal tubes (Aleshkin et al., 2016). Thus, we consider it essential to clarify whether newly isolated bacteriophages can be applied in order to control P. aeruginosa biofilms on the surface of endotracheal tubes. The null hypothesis of this study is that there would not be a difference in P. aeruginosa growth when challenged with bacteriophages.\\nMaterials and Methods\\nBacteriophage Isolation and Propagation\\nBacteriophages were isolated from domestic sewage, and P. aeruginosa (ATCC 27853) was used as the host strain. Isolation, sample enrichment, and purification methods followed the protocol described previously (Sillankorva et al., 2008). Briefly, 20 ml of domestic effluents, collected from March to November 2018 (Domestic Sewage Treatment Station, Ribeir\\u00e3o Preto, Brazil and Arax\\u00e1, Brazil), was mixed to 20 ml of double-strength tryptic soy broth (TSB; BD Difco, Sparks, MN, United States) and 100 \\u03bcl of exponentially grown P. aeruginosa. The presence of bacteriophage was confirmed by inhibition haloes on double-layer agar plating [Tryptic Soy Agar Soft (0.8% agar)\\u2014TSAS (BD Difco)] (Sambrook and Russell, 2001). Repeated double-layer agar plating was done until single-plaque morphology was observed. For propagation, one isolated plaque was collected through a sterile toothpick and diluted in 25 \\u03bcl of elution buffer [1 M Tris\\u2013HCl, pH 7.5 (Sigma Aldrich, Saint Louis, MO, United States), 10 mM MgSO4 (Sigma Aldrich), 10 mM NaCl (Din\\u00e2mica, Indaiatuba, SP, Brazil), 0.002% (w/v) gelatin (Din\\u00e2mica)]. The suspension was dropped onto a TSAS medium, with P. aeruginosa lawns, and incubated at 37\\u00b0C for 24 h. Afterward, 10 ml of elution buffer was added to each plate to elute the bacteriophages, which were purified with chloroform (Sigma Aldrich), titrated, and stored at 4\\u00b0C.\\nTransmission Electron Microscopy\\nOne milliliter of bacteriophage suspension was centrifuged at 21,800 g for 60 min at 4\\u00b0C. The sediment was washed twice in sterile ultrapure water, and 25 \\u03bcl of each bacteriophage was applied on an electron microscopy grid. The grids were stained with 2% uranyl acetate for 5 min and then analyzed by transmission electron microscopy (JEOL JEM-100 CXII Akishima, Tokyo, Japan; Melo et al., 2014).\\nAcid\\u2013Base and Thermal Stability\\nIn order to assess acid\\u2013base stability, 100 \\u03bcl of the bacteriophage suspension (108 plaque forming units per milliliter\\u2014PFU/ml) were added to 900 \\u03bcl of elution buffer, with pH adjusted to 3.0, 5.0, 7.0, 9.0, and 11.0. The samples were incubated for 2 h at 37\\u00b0C. To evaluate thermal stability, 100 \\u03bcl of the bacteriophage suspension (108 PFU/ml) was added to 900 \\u03bcl of elution buffer and incubated at different temperatures (37, 42, 50, 60, 70, and 80\\u00b0C) for 2 h (20). After the incubation period, the phage titer was determined through the double-layer agar method (Cui et al., 2017). The assays were carried out at three independent times in triplicate.\\nOne-Step Growth Curve\\nAn exponential-phase P. aeruginosa culture was centrifuged at 4,200 g for 5 min, and OD600 was adjusted to achieve \\u223c1011 colony-forming units per milliliter (CFU/ml). Subsequently, 5 ml of bacteria suspension was added to 5 ml of bacteriophage suspension at 5 \\u00d7 108 PFU/ml (multiplicity of infection\\u2014MOI: 0.005). The mixture was incubated (120 rpm, 37\\u00b0C, 5 min) and centrifuged (4,200 g, 5 min), and 10 ml of sterile TSB was added to the pellet. Samples were incubated (120 rpm, 37\\u00b0C), and aliquots were taken every 5 min until 60 min post-infection. Using the double-layer agar plating, the number of bacteriophage particles was determined at each time. Latent period was determined as the time required for the phage to increase in number and burst size as the ratio between the number of bacteriophages after reaching the plateau and the number of infected bacteria (Sillankorva et al., 2008). The assay was carried out at three independent times.\\nGenome Sequencing Analysis\\nOne milliliter of bacteriophage stock was incubated with 10 U DNAse I (Sigma Aldrich) and 10 \\u03bcl RNAse A (20 mg/ml; Sigma Aldrich) at 37\\u00b0C for 1 h. Enzymes were inactivated at 70\\u00b0C for 15 min. Then, the suspensions were incubated with extraction buffer [40 \\u03bcl EDTA 0.5 M (Sigma Aldrich), 5 \\u03bcl proteinase K 10 mg/ml (Sigma Aldrich), and 50 \\u03bcl SDS 10% (w/v; Sigma Aldrich)]. Extraction was performed with phenol/chloroform/isoamyl alcohol (25:24:1; Sigma Aldrich). Absence of bacterial contaminant DNA was confirmed by amplification of the 16S rRNA P. aeruginosa gene (Spilker et al., 2004).\\nLibraries were prepared using the NexteraXT kit (Illumina, San Diego, CA, United States), and genomes were sequenced using Illumina MiSeq paired-end sequencing (2 \\u00d7 300 bp/coverage 100\\u00d7), according to the manufacturer\\u2019s instructions. Reads were trimmed and assembled into a single contig using Velvet de novo tool (Illumina). Potential coding sequences (CDSs) were initially annotated using myRAST (Aziz et al., 2008), and alternative start codons or non-annotated CDSs were further analyzed using Geneious version 2019.2.1 (Biomatters Ltd., Auckland, New Zealand). Encoded proteins were queried against the National Center for Biotechnology Information (NCBI) non-redundant database with BLASTP (coverage > 80%; E-value \\u2264 10\\u20135; Altschul et al., 1990). Protein families, signatures, and structure prediction were searched with PFAM (Finn et al., 2014), InterPro (Finn et al., 2017), and HHpred (S\\u00f6ding, 2005), while the presence of transmembrane domains was checked using TMHMM (K\\u00e4ll and Sonnhammer, 2002), and Phobius (K\\u00e4ll et al., 2004). To predict signal peptide cleavage sites, SignalP (Petersen et al., 2011), and Spoctopus (Viklund et al., 2008) were used. Transfer RNAs (tRNAs) were scanned using tRNAscan-SE (Schattner et al., 2005).\\nHost Range Investigation and EOP Analysis\\nHost range was evaluated by the presence of clear or turbid inhibition haloes in a spot test on double-layer agar against 26 clinical isolates and seven multidrug-resistant P. aeruginosa strains (Panel ATCC\\u00ae MP-23TM; Table 1). The susceptibility profile of clinical isolates was determined by the disk diffusion method. Susceptibility test, antimicrobial classes, antibiotic concentration, and zone diameter breakpoint for P. aeruginosa strains were defined according to the Clinical and Laboratory Standards Institute (Clinical and Laboratory Standards Institute [CLSI], 2019).\\nTable 1. Lytic spectrum and efficiency of plating of the bacteriophages against sensible and multidrug-resistant Pseudomonas aeruginosa isolated from different sources.\\nEfficiency of plating (EOP) was determined for all P. aeruginosa strains positive for bacteriophage lytic activity. Tenfold dilutions of the bacteriophages were spotted on double-layer agar inoculated with P. aeruginosa strains. EOP values were calculated as the ratio between the lysis plaques produced in each susceptible strain and the number of plaques produced in P. aeruginosa ATCC 27853. EOP was recorded as high, low, and inefficient, representing >1, 0.1\\u20131, and <0.1%, respectively (Melo et al., 2014).\\nInfection of Planktonic Cells\\nTen clinical isolates with the highest EOP values, P. aeruginosa ATCC 27853, and multidrug-resistant P. aeruginosa ATCC 2108, ATCC 2110, ATCC 2112, and ATCC 2113 were selected for this assay. Then, 100 \\u03bcl of P. aeruginosa strains, OD600 adjusted to achieve \\u223c109 CFU/ml, was inoculated with 100 \\u03bcl of each bacteriophage at a concentration of 5 \\u00d7 107 PFU/ml (MOI: 0.05). The positive and negative control experiments were performed using a dilution buffer instead of bacteriophages and sterile TSB instead of bacterial suspension, respectively. The plates were incubated at 37\\u00b0C, 80 rpm. The OD600 was measured after 2, 4, 6, 8, and 24 h (Melo et al., 2014). The assays were carried out at three independent times in triplicate.\\nAntibiofilm Activity Measurement\\nThe same 15 P. aeruginosa strains applied in the infection of planktonic cells test were selected for this assay. The anti-biofilm activity of the bacteriophages was determined by means of biofilm viability rates (CFU/cm2), metabolic activity of the bacteria (XTT), and scanning electron microscopy (SEM). The assays were carried out at three independent times in triplicate.\\nFor biofilm infection, strains were grown in TSB medium (37\\u00b0C, 24 h), and OD600 was adjusted to \\u223c109 CFU/ml. Circular endotracheal tube specimens (\\u00d8 5 mm; R\\u00fcsh, Meridian, MS, United States) were sterilized by hydrogen peroxide plasma (Sterrad 100S, Irvine, CA, United States) and randomly distributed in 48-well tissue culture plates. Each well received 400 \\u03bcl of TSB medium containing the standardized cell suspension (\\u223c107 CFU/ml). Negative control was performed using sterile TSB. The plates were incubated for 48 h at 37\\u00b0C and 80 rpm. Afterward, the specimens were removed carefully from the plate, rinsed in 10 ml of phosphate-buffered saline (PBS), and transferred to a new plate with 1 ml of sterile culture medium and 108 PFU/ml of bacteriophages. The control samples received culture medium without bacteriophages. The plates were incubated for 24 h at 37\\u00b0C and 80 rpm. After the incubation time, the specimens were removed from the wells through sterilized tweezers and rinsed in 10 ml of PBS.\\nFor CFU quantification, the specimens were transferred to a tube containing 2 ml of PBS. In order to detach the remaining biofilm, the tubes were vortexed for 30 s, sonicated (200 W, 40 kHz; Altsonic, Clean 9CA, Ribeir\\u00e3o Preto, SP, Brazil) for 20 min, and vortexed again for 2 min. Afterward, 10-fold-dilution aliquots (100\\u201310\\u20136) were seeded in Petri dishes containing TSA medium. The Petri dishes were incubated at 37\\u00b0C for 24 h, and the number of colonies was registered and expressed as log CFU/cm2.\\nFor the evaluation of metabolic activity, the specimens were transferred to another 48-well plate containing 316 \\u03bcl PBS supplemented with 100 mM glucose (Sigma Aldrich), 80 \\u03bcl XTT 1 mg/ml (Sigma Aldrich), and 4 \\u03bcl of 0.4 mM menadione (Sigma Aldrich). The plates were statically incubated, protected from light, at 37\\u00b0C for 2 h, and the absorbance of the resulting solution was measured (492 nm).\\nFor SEM, P. aeruginosa ATCC 27853 was analyzed. The biofilms were removed from the plates and washed twice in distilled water. Samples were fixed with 2.5% glutaraldehyde for 60 min and then dehydrated in a graded ethanol series (30, 50, 70, 90, and 100%). After chemical drying (hexamethyldisilazane; Sigma Aldrich), the specimens were mounted on a metal stub and gold-coated. The surface morphology of the biofilms was examined under high vacuum with a JEOL JSM-35CF microscope (Tokyo, Japan).\\nStatistical Analysis\\nStatistical tests were performed through the IBM SPSS Statistics 25.0 software (IBM Corp Armonk, NY, United States). The significance level was set to 0.05. All data showed normal distribution and homogeneous variance and were analyzed using two-way ANOVA, with independent levels and Bonferroni post hoc test.\\nResults\\nBacteriophage Isolation and Propagation\\nFive new lytic bacteriophages against P. aeruginosa were isolated from domestic sewage and designated as vB_PaeM_USP_1, vB_PaeM_USP_2, vB_PaeM_USP_3, vB_PaeM_USP_18, and vB_PaeM_USP_25. All bacteriophages were able to form clear plaques, having well-defined boundaries. It was observed that none of the bacteriophages produced double haloes, and it was not possible to differentiate lysis plaques among the different isolated bacteriophages (Figures 1A\\u2013E).\\nFigure 1. (A\\u2013E) Representative image of phage isolates depicting different plaque morphologies of vB_PaeM_USP_1, vB_PaeM_USP_2, vB_PaeM_USP_3, vB_PaeM_USP_18, and vB_PaeM_USP_25. (F\\u2013J) Transmission electron micrographs of individual bacteriophages. (K) Panel of plaque characteristics, phage morphology, latent period, and burst size.\\nElectron Microscopy\\nThe morphological aspects were slightly different among the different isolated viruses (Figures 1F\\u2013J). According to the International Committee on Taxonomy of Viruses, all bacteriophages belong to the order Caudovirales and are members of the Myoviridae family.\\nAcid\\u2013Base and Thermal Stability\\nThe values of pH 3.0 and pH 11.0 contributed to reduced bacteriophage viability (p < 0.05). Neutral pH (7.0) and small variations in hydrogenionic potential (pH 5.0 and pH 9.0) did not cause significant changes in viability. vB_PaeM_USP_2 and vB_PaeM_USP_3 were the most resistant to pH variation (Figure 2A). Reduced bacteriophage viability after exposure to 70 and 80\\u00b0C was noticed. Bacteriophage vB_PaeM_USP_25 was the most resistant to temperature variation (Figure 2B).\\nFigure 2. Effects of different pH and temperature values (log10UFP/ml) on vB_PaeM_USP_1, vB_PaeM_USP_2, vB_PaeM_USP_3, vB_PaeM_USP_18, and vB_PaeM_USP_25.\\nOne-Step Growth Curve\\nA one-step growth curve revealed a similar behavior among the bacteriophages (Supplementary Figure 1). Latent period and yields at burst size are presented in Figure 1K.\\nGenome Sequencing Analysis\\nThe complete genome sequence of each bacteriophage was deposited in the NCBI database under the accession numbers MT491204\\u2013MT491208. Genome analysis revealed that the five bacteriophages are potentially virulent since they do not encode for any known protein associated to lysogeny. Moreover, no known virulence-associated or toxic proteins were identified, revealing that the isolated bacteriophages are potentially safe for therapeutic purposes. All bacteriophage genomes are composed of double-stranded DNA molecules, and no putative genes coding for tRNAs were found. The general genomic properties of the five bacteriophages are presented in Table 2. Two different subgroups of bacteriophages could be observed (Figure 3). The first one, composed of bacteriophages vB_PaeM_USP_1, vB_PaeM_USP_2, and vB_PaeM_USP_3, presents similar genome sizes (approximately 65 kb), G + C content (approximately 55.5%), a high nucleotide pairwise identity (94.6%), and proteome homology (sharing 83 protein homologs between them), with roughly 62% of the proteins with unknown function (Figure 3). Despite their similarity, they present some differences at the beginning (0\\u20135 kb) and at the end of their genome (53 kb end; Figure 4A). Their genomes share high nucleotide identity with other P. aeruginosa bacteriophage members of the Pbunavirus genus, namely, Pseudomonas virus Pa193, Epa22, vB_PaeM_PAO1_Ab27, and BrSP1, with identities above 96% and coverages of up to 98%.\\nTable 2. Genomic properties of Pseudomonas aeruginosa bacteriophages.\\nFigure 3. Genome map of the five sequenced bacteriophages depicting the predicted open reading frames colored according to their predicted function (black for hypothetical proteins, green for genes involved in DNA replication and transcription, blue for DNA packaging and morphogenesis genes, and red for cell lysis genes). The blue line represents the guanine-cytosine content (%).\\nFigure 4. Whole genome alignment (using progressive Mauve algorithm) of the bacteriophage genomes demonstrating the high identity between the two groups. (A) Mauve alignment of the group containing bacteriophages vB_PaeM_USP_1, vB_PaeM_USP_2, and vB_PaeM_USP_3. (B) Mauve alignment of the group containing bacteriophages vB_PaeM_USP_18 and vB_PaeM_USP_25.\\nThe second subgroup, composed of vB_PaeM_USP_18 and vB_PaeM_USP_25, has nucleotide pairwise identity of 97.1% and shares 65 homologs, ca. 74% of them without an assigned function (Figure 3). A slightly lower homology is observed in the genomic regions 3\\u20134.5, 12.5\\u201314, and mainly from 39\\u201346.5 kb (Figure 4B). The low homology with genomes from other phages deposited with the NCBI does not allow grouping these phage genomes with any known genus or subfamily.\\nHost Range Investigation and EOP Analysis\\nAll five bacteriophages exhibited a broad lytic spectrum and were able to infect 69.7% (n = 23) of the P. aeruginosa strains, of which seven were multidrug resistant. Furthermore, the bacteriophages showed lytic activity on clinical strains isolated from different infection sites (Table 1).\\nThe EOP values of vB_PaeM_USP_1 (high: 13; insufficient: 6; low: 4), vB_PaeM_USP_2 (high: 14; insufficient: 6; low: 3), and vB_PaeM_USP_3 (high: 13; insufficient: 7; low: 3) were higher than vB_PaeM_USP_18 (high: 8; insufficient: 11; low: 4) and vB_PaeM_USP_25 (high: 6; insufficient: 11; low: 6). The last two viruses had low or inefficient EOP for all multidrug-resistant strains (Table 1). Although during host range investigation the bacteriophages had lysed the same strains, inefficient EOP results indicate that the haloes observed previously refer to \\u201clysis from without events.\\u201d\\nInfection of Planktonic Cells\\nAll five bacteriophages reduced bacterial growth up to 8 h of incubation in 13 strains (p < 0.05; Figure 5A). The reduction was above 60% after 6 and 8 h of incubation. After 24 h of incubation, all bacteriophages were able to maintain lower OD values in three strains, but with reductions in bacterial growth of 45% (p < 0.05). In five other strains, the growth rates were reduced by different bacteriophages, and the growth of five strains was not affected by the bacteriophages. Concerning multidrug-resistant P. aeruginosa, the bacteriophages promoted alteration in growth rates in one of the two strains. vB_PaeM_USP_1 and vB_PaeM_USP_3 promoted the greatest growth reductions (Supplementary Tables 1\\u20135).\\nFigure 5. (A) Optical density of planktonic Pseudomonas aeruginosa strains after 2, 4, 6, 8, and 24 h of co-incubation with different bacteriophages. (B) Anti-biofilm activity of different bacteriophages on biofilm viability rates expressed in log10 CFU/cm2 (left y axis) and metabolic activity expressed in absorbance at 492 nm (right y axis). Inhibition of planktonic growth and anti-biofilm activity assays was carried out at three independent times in triplicate against 15 strains of P. aeruginosa. (C\\u2013H) Scanning electron micrographs after phage treatment: (C) control, (D) vB_PaeM_USP_1, (E) vB_PaeM_USP_2, (F) vB_PaeM_USP_3, (G) vB_PaeM_USP_18, and (H) vB_PaeM_USP_25. Scale bar = 100 nm. After the treatment of bacteriophages, it is possible to observe lower planktonic growth, biofilm viability rates, and metabolic activity. Disrupted biofilms and cell debris are also noticed in SEM images.\\nAnti-biofilm Activity Measurement\\nAll five bacteriophages reduced the biofilm viability rates in 12 strains (p < 0.05) in about 10-fold CFU/cm2 (Figure 5B). vB_PaeM_USP_2 and vB_PaeM_USP_18 presented increased anti-biofilm activity against two multidrug-resistant strains.\\nConcerning the metabolic activity (XTT), after treatment of the bacteriophages, the absorbance values reduced in about 50% in 11 strains (p < 0.05; Figure 5B). Bacteriophage vB_PaeM_USP_2 was able to significantly reduce the metabolic activity. The mean and the standard deviation values (CFU/cm2 and XTT) after exposure to different bacteriophages are shown in the Supplementary Data (Supplementary Tables 6, 7).\\nResults from SEM micrographs showed densely associated bacteria in the control biofilm (Figure 5C). In contrast, after exposure to bacteriophages, it is possible to observe disrupted biofilms and cell debris, demonstrating remarkable biofilm reduction (Figures 5D\\u2013H).\\nDiscussion\\nThis study comprises the isolation and the full characterization of five new P. aeruginosa bacteriophages isolated from domestic sewage. The bacteriophages showed a broad lytic spectrum, able to disrupt P. aeruginosa biofilms on endotracheal tube surfaces. Based on the results, the null hypothesis was rejected since there was a statistical difference on planktonic bacteria growth, viability rates, and metabolic activity of the biofilm.\\nThe vast majority of the already characterized bacteriophages are tailed (Hatfull and Hendrix, 2011). Through morphological and genomics analysis, here there is evidence that all the five bacteriophages have similar features, belonging to the order Caudovirales and Myoviridae family. Moreover, genome sequencing revealed that vB_PaeM_USP_1, vB_PaeM_USP_2, and vB_PaeM_USP_3 were most closely related to Epa22, sharing about 96% of identity. Their genomes are typically organized in clusters. Both guanine-cytosine content and genome length were similar to other Pbunaviruses as previously reported (de Melo et al., 2019; Yuan et al., 2019). Regarding vB_PaeM_USP_18 and vB_PaeM_USP_25, only ca. 20% of their genomes could be aligned with other phage genomes deposited at the NCBI. This low identity with other known genomes does not allow grouping these two phages with any known genus or subfamily. The inability to group these two genomes suggests that they belong to a new bacteriophage genus. Genomic bioinformatics analysis allowed identifying the endolysin gene on all bacteriophages. Commonly, this enzyme is responsible for the lytic ability of tailed bacteriophages (Wu et al., 2019). In addition, several open reading frames could only be annotated as coding for hypothetical proteins and thus with an unknown function, something that is common in bacteriophage studies (Guo et al., 2019).\\nThe one-step growth curve analysis revealed that the latent period was similar in the five bacteriophages (about 30 min; Figure 1H), with a progeny per infected cell varying from 22 to 46 bacteriophage particles. Regarding the latent period, the same result was reported previously. Nonetheless, comparing burst sizes with bacteriophages with a similar genome, we noticed that all bacteriophages produced lower burst sizes (Guo et al., 2019; Yuan et al., 2019). The reduction observed here could be explained by differences in bacterial growth rates since the burst size increases linearly with bacterial growth rate (Nabergoj et al., 2018). Adsorption and latent period also vary with bacterial growth rate, decreasing at higher growth rates.\\nThe infectivity of the bacteriophages remained stable for a pH range from 5.0 to 9.0 and temperature range from 37 to 60\\u00b0C (Figure 2). Although pH and temperature do not undergo abrupt variation in physiological systems, comprehending this aspect is important in order to maintain bacteriophage viability and infectivity. Even though new antimicrobial agents should be investigated, the large pH and temperature tolerance indicate a broad workability range to search and apply bacteriophage-based products to setups of different infection models. For instance, skin becomes more alkaline by bacterial colonization, and wounds show pH variations in relation to healthy skin (Schneider et al., 2007).\\nThe lytic activity of bacteriophages was tested against 37 strains (33 P. aeruginosa and four different bacterial species). Our results reveal that all bacteriophages could efficiently lyse 69.7% of the P. aeruginosa strains, of which seven were multidrug resistant. Infection of multidrug-resistant P. aeruginosa indicates that bacteriophages or bacteriophage-based products could potentially be applied for therapeutic purposes. The lytic rates presented here are higher than in other Pbunavirus (de Melo et al., 2019), which can be related to the origin and the number of tested strains. In fact, host broad range and species specificity are desirable for bacteriophage application in phage therapy. Moreover, bacteriophages should be strictly lytic in order to avoid the risk of potential transmission of virulent bacterial genes (Canchaya et al., 2003). Here five lytic bacteriophages had promising results in disrupting mature P. aeruginosa biofilms and can be further explored in studies involving phage therapy.\\nEfficiency of plating assays indicated a considerable variation from low to high between the different bacteriophages on the different strains. Moreover, the bacteriophages showed inefficient EOP for six strains. Differences in EOP and absence of lysis can be explained by the recognition process, which can diverge in different strains according to the expression of bacteriophage receptors on the bacterial outer membrane (Uchiyama et al., 2016). Furthermore, the localization, volume, and density of surface molecules can affect bacteriophage adsorption (Rakhuba et al., 2010). Considering the mentioned variation in molecules for bacteriophage adsorption, the application of a bacteriophage cocktail, which retains distinct adsorption mechanisms, is important to maintain bacteriophage infectivity (Forti et al., 2018). Apart from the recognition process, well-known bacterial resistance systems against bacteriophages could have led to low EOP values. Literature has pointed out that many antiviral mechanisms may be employed by the host cell to evade infection from bacteriophages, such as restriction nucleases and CRISPR (Seed, 2015).\\nAssessing the efficacy of bacteriophages in reducing P. aeruginosa growth in planktonic form, we observed significant reductions for up to 8 h of co-incubation for almost all investigated strains. However, following 24 h of co-incubation, only three strains showed reduced growth rates. Two multidrug-resistant P. aeruginosa did not show reduced growth rates at the evaluated times. Guo et al. (2019) demonstrated that two new phages at low MOI (0.01) had good performance both on preventing biofilm formation and eradicating preformed biofilms. Moreover, Melo et al. (2014), using a MOI of 0.05, demonstrated reduction in the optical density of planktonic cells in about 83% of clinical isolates after 24 h of infection. We considered that antiviral mechanisms might result in the emergence of bacteriophage-insensitive bacterial strains through blocking of bacteriophage receptors, production of extracellular matrix, production of competitive inhibitors, prevention of bacteriophage DNA entry, slicing of bacteriophage nucleic acids, and abortive infection mechanisms (Labrie et al., 2010).\\nWe observed a great reduction in planktonic growth and anti-biofilm activity after treatment of bacteriophages, and it was widely coincident in both assays. Slight differences were detected among the strains, which can be related to distinct analysis methods. Planktonic assays were performed by OD, while anti-biofilm activity was evaluated by CFU counts and activity metabolic measurement. Comparing OD and CFU results, only one strain showed a discordant outcome. Considering the three analysis methods (OD, CFU, and XTT), the results of six strains did not seem to be concordant. The lytic activity of the bacteriophages was relevant to biofilm control on the endotracheal tube surface. Although no depolymerase was identified in these bacteriophage genomes, the anti-biofilm action could be related to the penetration of viruses at the deeper layers of the biofilm by means of the degradation of the polysaccharide matrix (Lin et al., 2017).\\nAnti-biofilm results support the hypothesis that bacteriophages might be applied as a biocontrol tool against multidrug-resistant P. aeruginosa on the endotracheal tube surface in the near future. Systemic treatment with antibiotics did not fully inhibit biofilm development on endotracheal tube surfaces (Fern\\u00e1ndez-Barat et al., 2012). Therefore, direct administration of non-antibiotic approaches, such as antimicrobial agents (Guillon et al., 2018) and bacteriophages, seem to be a suitable option for the fight against persistent biofilms. Indeed bacteriophages could be potentially applied on endotracheal tube biofilm as prophylactic and therapeutic agent. Bacteriophage-coated tubes could be attended to inhibit bacterial adhesion, whereas direct application may be useful in reducing microbial bioburden. Aleshkin et al. (2016) reported the intragastric administration of a phage cocktail in patients on mechanical ventilation, aiming for the treatment of acute infections caused by nosocomial pathogens. The authors demonstrated an important reduction in bacterial counts after phage therapy, indicating it as an alternative purpose to antibiotics to treat healthcare-associated infections.\\nLiterature has presented that bacteriophages or the synergistic action of bacteriophage and antibiotics constitute promising strategies against multidrug-resistant P. aeruginosa biofilm (Akturk et al., 2019; Adnan et al., 2020). Given the ability of new lytic bacteriophages to disrupt and reduce the microbial load of endotracheal-associated biofilms in vitro, several aspects must be taken into consideration before the transfer of phage therapy to clinical practice. As the device is not in contact with vascularized tissues, potential bacteriophage administration routes should be discussed. Moreover, in clinical settings, it is highly unlikely that the endotracheal tubes will continuously be exposed to the phages, as demonstrated in this in vitro study. Thus, we suggest that phage therapy could be applied during endotracheal suctioning, as recommended for ventilated patients. The instillation of bacteriophages during the aspiration of pulmonary secretions could favor its direct application (Guillon et al., 2018). Inhalation of aerosol-containing antibiotics and phage formulations is another potential administration route.\\nIn addition, it should be considered that in vivo biofilms are different from the in vitro ones studied here, probably due to the presence of body fluids, host cells, and different interactions among microorganisms. In light of the aforementioned assumptions, future investigations considering mixed biofilm models and the synergistic action of phage and antibiotic therapy should be conducted. Although treatment of bacteriophage reduced the concentration of P. aeruginosa both on planktonic and biofilm forms, further research is required to determine the correct concentration and the possible combinations of different bacteriophages to establish a cocktail. Based on EOP and anti-biofilm activity and considering the similarities among the five bacteriophages, we suggest that, in further studies, a cocktail composed of vB_PaeM_USP_2 and vB_PaeM_USP_18 should be used. Moreover, only one administration with a single bacteriophage concentration to treat biofilms was evaluated. Assessing different times of infection, a strong lytic activity could be evidenced. In addition, special attention should be paid to the potential development of bacteriophage resistance. In view of this, we also consider it necessary to evaluate the phenotype of bacteriophage-insensitive mutants after a long exposure to bacteriophages.\\nConclusion\\nBiofilms associated with endotracheal tubes are critical concerns in the development of VAP. Since antibiotic therapy is not profitable to eliminate biofilms on the surface of medical devices, bacteriophage therapy offers a possible alternative to reduce bacterial colonization on the surface of endotracheal tubes. The studied bacteriophages were efficient in controlling the growth of P. aeruginosa and promoted biofilm disruption on the surface of endotracheal tubes, including multidrug-resistant strains. Thus, they might be considered as good candidates for therapeutic studies and the application of bacteriophage-encoded proteins.\\nData Availability Statement\\nThe datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found in the article/ Supplementary Material.\\nAuthor Contributions\\nEW contributed to conceptualization and funding acquisition. VO carried out the main body of research and wrote the original manuscript. FB and RM contributed to the phage growth experiments. AM contributed to data analysis. CS-L and HP supervised the work\\u2019s progress and edited the manuscript. LM and SS contributed to bioinformatics analysis and edited the manuscript. All authors have read and agreed to the published version of the manuscript.\\nFunding\\nThis study was supported by the S\\u00e3o Paulo Research Foundation (FAPESP) under grant numbers 2018/09757-0 and 2019/13271-9 and the National Council for Scientific and Technological Development (CNPq) under grant number 405622/2018-0.\\nConflict of Interest\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2020.580779/full#supplementary-material\\nReferences\\nAdnan, M., Shah, M. R. A., Jamal, M., Jalil, F., Andleeb, S., Nawaz, M. A., et al. (2020). Isolation and characterization of bacteriophage to control multidrug-resistant Pseudomonas aeruginosa planktonic cells and biofilm. Biologicals 63, 89\\u201396. doi: 10.1016/j.biologicals.2019.10.003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAkturk, E., Oliveira, H., Santos, S. B., Costa, S., Kuyumcu, S., Melo, L. D. R., et al. (2019). Synergistic action of phage and antibiotics: parameters to enhance the killing efficacy against mono and dual-species biofilms. Antibiotics 8:103. doi: 10.3390/antibiotics8030103\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAleshkin, A. V., Ershova, O. N., Volozhantsev, N. V., Svetoch, E. A., Popova, A. V., Rubalskii, E. O., et al. (2016). Phagebiotics in treatment and prophylaxis of healthcare-associated infections. Bacteriophage 6:e1251379. doi: 10.1080/21597081.2016.1251379\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAltschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990). Basic local alignment search tool. J. Mol. Biol. 215, 403\\u2013410. doi: 10.1016/S0022-2836(05)80360-2\\nCrossRef Full Text | Google Scholar\\nAziz, R. K., Bartels, D., Best, A. A., DeJongh, M., Disz, T., Edwards, R. A., et al. (2008). The RAST Server: rapid annotations using subsystems technology. BMC Genom. 9:75. doi: 10.1186/1471-2164-9-75\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBarbier, F., Andremont, A., Wolff, M., and Bouadma, L. (2013). Hospital-acquired pneumonia and ventilator-associated pneumonia: recent advances in epidemiology and management. Curr. Opin. Pulm. Med. 19, 216\\u2013228. doi: 10.1097/MCP.0b013e32835f27be\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBarros, J. A. R., Melo, L. D. R., Silva, R. A. R., Ferraz, M. P., Azeredo, J. C. V. R., Pinheiro, V. M. C., et al. (2020). Encapsulated bacteriophages in alginate-nanohydroxyapatite hydrogel as a novel delivery system to prevent orthopedic implant-associated infections. Nanomedicine 24:102145. doi: 10.1016/j.nano.2019.102145\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBerra, L., De Marchi, L., Yu, Z. X., Laquerriere, P., Baccarelli, A., and Kolobow, T. (2004). Endotracheal tubes coated with antiseptics decrease bacterial colonization of the ventilator circuits, lungs, and endotracheal tube. Anesthesiology 100, 1446\\u20131456. doi: 10.1097/00000542-200406000-00017\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCanchaya, C., Fournous, G., Chibani-Chennoufi, S., Dillmann, M. L., and Brussow, H. (2003). Phage as agents of lateral gene transfer. Curr. Opin. Microbiol. 6, 417\\u2013424. doi: 10.1016/s1369-5274(03)00086-9\\nCrossRef Full Text | Google Scholar\\nCieplik, F., Jakubovics, N. S., Buchalla, W., Maisch, T., Hellwig, E., and Al-Ahmad, A. (2019). Resistance toward Chlorhexidine in oral bacteria - is there cause for concern? Front. Microbiol. 10:587. doi: 10.3389/fmicb.2019.00587\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nClinical and Laboratory Standards Institute [CLSI] (2019). Performance Standards for Antimicrobial Susceptibility Testing, Twenty-Nine Informational Supplement. Wayne: CLSI.\\nGoogle Scholar\\nCui, Z. L., Feng, T. T., Gu, F. F., Li, Q. T., Dong, K., Zhang, Y., et al. (2017). Characterization and complete genome of the virulent Myoviridae phage JD007 active against a variety of Staphylococcus aureus isolates from different hospitals in Shanghai, China. Virol. J. 14:26. doi: 10.1186/s12985-017-0701-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nde Melo, A. C. C., Gomes, A. D., Melo, F. L., Ardisson-Araujo, D. M. P., de Vargas, A. P. C., Ely, V. L., et al. (2019). Characterization of a bacteriophage with broad host range against strains of Pseudomonas aeruginosa isolated from domestic animals. BMC Microbiol. 19:134. doi: 10.1186/s12866-019-1481-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nFern\\u00e1ndez-Barat, L., Ferrer, M., Sierra, J. M., Soy, D., Guerrero, L., Vila, J., et al. (2012). Linezolid limits burden of methicillin-resistant Staphylococcus aureus in biofilm of tracheal tubes. Crit. Care Med. 40, 2385\\u20132389. doi: 10.1097/CCM.0b013e31825332fc\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nFinn, R. D., Attwood, T. K., Babbitt, P. C., Bateman, A., Bork, P., Bridge, A. J., et al. (2017). InterPro in 2017-beyond protein family and domain annotations. Nucleic Acids Res. 45, 190\\u2013199. doi: 10.1093/nar/gkw1107\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nFinn, R. D., Bateman, A., Clements, J., Coggill, P., Eberhardt, R. Y., Eddy, S. R., et al. (2014). Pfam: the protein families database. Nucleic Acids Res. 42, 222\\u2013230. doi: 10.1093/nar/gkt1223\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nForti, F., Roach, D. R., Cafora, M., Pasini, M. E., Horner, D. S., Fiscarelli, E. V., et al. (2018). Design of a broad-range bacteriophage cocktail that reduces Pseudomonas aeruginosa biofilms and treats acute infections in two animal models. Antimicrob. Agents Chemother. 62:e02573-17. doi: 10.1128/AAC.02573-17\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGuillon, A., Fouquenet, D., Morello, E., Henry, C., Georgeault, S., Si-Tahar, M., et al. (2018). Treatment of Pseudomonas aeruginosa biofilm present in endotracheal tubes by Poly-l-Lysine. Antimicrob. Agents Chemother. 62:e0564-18.\\nGoogle Scholar\\nGuo, Y. Y. J., Chen, P., Lin, Z. L., and Wang, T. T. (2019). Characterization of two Pseudomonas aeruginosa viruses vB_PaeM_SCUT-S1 and vB_PaeM_SCUT-S2. Viruses 11:e318. doi: 10.3390/v11040318\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nK\\u00e4ll, L., Krogh, A., and Sonnhammer, E. L. L. (2004). A combined transmembrane topology and signal peptide prediction method. J. Mol. Biol. 338, 1027\\u20131036. doi: 10.1016/j.jmb.2004.03.016\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nK\\u00e4ll, L., and Sonnhammer, E. L. L. (2002). Reliability of transmembrane predictions in whole-genome data. FEBS Lett. 532, 415\\u2013418. doi: 10.1016/S0014-5793(02)03730-4\\nCrossRef Full Text | Google Scholar\\nKaur, S., Harjai, K., and Chhibber, S. (2014). Bacteriophage mediated killing of Staphylococcus aureus in vitro on orthopaedic K wires in presence of linezolid prevents implant colonization. PLoS One 9:e90411. doi: 10.1371/journal.pone.0090411\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLessa, F. C. R., Aranha, A. M. F., Nogueira, I., Giro, E. M. A., Hebling, J., and Costa, C. A. S. (2010). Toxicity of Chlorhexidine on odontoblast-like cells. J. Appl. Oral Sci. 18, 50\\u201358. doi: 10.1590/s1678-77572010000100010\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLin, D. M., Koskella, B., and Lin, H. C. (2017). Phage therapy: an alternative to antibiotics in the age of multi-drug resistance. World J. Gastrointest. Pharmacol. Ther. 8:162. doi: 10.4292/wjgpt.v8.i3.162\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLungren, M. P., Donlan, R. M., Kankotia, R., Paxton, B. E., Falk, I., Christensen, D., et al. (2014). Bacteriophage K antimicrobial-lock technique for treatment of Staphylococcus aureus central venous catheter-related infection: a leporine model efficacy analysis. J. Vasc. Interv. Radiol. 25, 1627\\u20131632. doi: 10.1016/j.jvir.2014.06.009\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMagiorakos, A. P., Srinivasan, A., Carey, R. B., Carmeli, Y., Falagas, M. E., Giske, C. G., et al. (2012). Multidrug-resistant, extensively drug-resistant and pandrug-resistant bacteria: an international expert proposal for interim standard definitions for acquired resistance. Clin. Microbiol. Infect. 18, 268\\u2013281. doi: 10.1111/j.1469-0691.2011.03570.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMao, B. H., Chen, Z. Y., Wang, Y. J., and Yan, S. J. (2018). Silver nanoparticles have lethal and sublethal adverse effects on development and longevity by inducing ROS-mediated stress responses. Sci. Rep. 8:2445. doi: 10.1038/s41598-018-20728-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMelo, L. D. R., Sillankorva, S., Ackermann, H. W., Kropinski, A. M., Azeredo, J., and Cerca, N. (2014). Isolation and characterization of a new Staphylococcus epidermidis broad-spectrum bacteriophage. J. Gen. Virol. 95, 506\\u2013515. doi: 10.1099/vir.0.060590-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMelo, L. D. R., Veiga, P., Cerca, N., Kropinski, A. M., Almeida, C., Azeredo, J., et al. (2016). Development of a phage cocktail to control Proteus mirabilis catheter-associated urinary tract infections. Front. Microbiol. 7:1024. doi: 10.3389/fmicb.2016.01024\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nNseir, S., Ader, F., Lubret, R., and Marquette, C. H. (2011). Pathophysiology of airway colonization in critically ill COPD patient. Curr. Drug Targets 12, 514\\u2013520. doi: 10.2174/138945011794751537\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nNzakizwanayo, J., Hanin, A., Alves, D. R., McCutcheon, B., Dedi, C., Salvage, J., et al. (2015). Bacteriophage can prevent encrustation and blockage of urinary catheters by Proteus mirabilis. Antimicrob. Agents Chemother. 60, 1530\\u20131536. doi: 10.1128/AAC.02685-15\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPetersen, T. N., Brunak, S., Heijne, G. V., and Nielsen, H. (2011). SignalP 4.0: discriminating signal peptides from transmembrane regions. Nat. Methods 8, 785\\u2013786. doi: 10.1038/nmeth.1701\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRakhuba, D. V., Kolomiets, E. I., Dey, E. S., and Novik, G. I. (2010). Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell. Pol. J. Microbiol. 59, 145\\u2013155. doi: 10.33073/pjm-2010-023\\nCrossRef Full Text | Google Scholar\\nRussotto, V., Cortegiani, A., Raineri, S. M., and Giarratano, A. (2015). Bacterial contamination of inanimate surfaces and equipment in the intensive care unit. J. Intensive Care 3:54. doi: 10.1186/s40560-015-0120-125\\nCrossRef Full Text | Google Scholar\\nSambrook, J., and Russell, D. W. (2001). Molecular Cloning: A Laboratory Manual. New York, NY: Cold Spring Harbor Laboratory Press.\\nGoogle Scholar\\nSchattner, P., Brooks, A. N., and Lowe, T. M. (2005). The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs. Nucleic Acids Res. 33, W686\\u2013W689. doi: 10.1093/nar/gki366\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSchneider, L. A., Korber, A., Grabbe, S., and Dissemond, J. (2007). Influence of pH on wound-healing: a new perspective for wound-therapy? Arch. Dermatol. Res. 298, 413\\u2013420. doi: 10.1007/s00403-006-0713-x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSillankorva, S., Neubauer, P., and Azeredo, J. (2008). Isolation and characterization of a T7-like lytic phage for Pseudomonas fluorescens. BMC Biotechnol. 8:80. doi: 10.1186/1472-6750-8-80\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSpilker, T., Coenye, T., Vandamme, P., and LiPuma, J. J. (2004). PCR-Based assay for differentiation of Pseudomonas aeruginosa from other Pseudomonas species recovered from cystic fibrosis patients. J. Clin. Microbiol. 42, 2074\\u20132079. doi: 10.1128/jcm.42.5.2074-2079.2004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTokmaji, G., Vermeulen, H., Muller, M. C. A., Kwakman, P. H. S., Schultz, M. J., and Zaat, S. A. J. (2015). Silver-coated endotracheal tubes for prevention of ventilator-associated pneumonia in critically ill patients. Cochrane Database Syst. Rev 8:CD009201. doi: 10.1002/14651858.CD009201\\nCrossRef Full Text | Google Scholar\\nUchiyama, J., Suzuki, M., Nishifuji, K., Kato, S. I., Miyata, R., Nasukawa, T., et al. (2016). Analyses of short-term antagonistic evolution of Pseudomonas aeruginosa strain PAO1 and phage KPP22 (Myoviridae Family, PB1-like virus Genus). Appl. Environ. Microbiol. 82, 4482\\u20134491. doi: 10.1128/AEM.00090-16\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nViklund, H., Bernsel, A., Skwark, M., and Elofsson, A. (2008). Spoctopus: a combined predictor of signal peptides and membrane protein topology. Bioinformatics 24, 2928\\u20132929. doi: 10.1093/bioinformatics/btn550\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWu, M. L., Hu, K. Y., Xie, Y. H., Liu, Y. L., Mu, D., Guo, H. M., et al. (2019). A novel phage PD-6A3, and its endolysin Ply6A3, with extended lytic activity against Acinetobacter baumannii. Front. Microbiol. 9:3302. doi: 10.3389/fmicb.2018.03302\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYuan, Y. Y., Qu, K. L., Tan, D. M., Li, X. Y., Wan, L. L., Cong, C., et al. (2019). Isolation and characterization of a bacteriophage and its potential to disrupt multi-drug resistant Pseudomonas aeruginosa biofilms. Microb. Pathog. 128, 329\\u2013336. doi: 10.1016/j.micpath.2019.01.032\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nZimlichman, E., Henderson, D., Tamir, O., Franz, C., Song, P., Yamin, C. K., et al. (2013). Health Care-associated infections a meta-analysis of costs and financial impact on the US health care system. JAMA Intern. Med. 173, 2039\\u20132046. doi: 10.1001/jamainternmed.2013.9763\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.02931\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.02931/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:50:15.677Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.02931\",\n",
      "            \"title\": \"Frontiers | Bacteriophages as an Up-and-Coming Alternative to the Use of Sulfur Dioxide in Winemaking\",\n",
      "            \"description\": \"Certain acetic and lactic acid bacteria are the major causes of quality defects in musts and wines, giving rise to defects such as \\\"vinegary\\\", \\\"sharp like nail polish remover\\\" or preventing alcoholic and/or malolactic fermentation. Sulfur dioxide is the major tool currently used in the control of these bacteria in wine. The aim of this work was to isolate bacteriophages from musts and wine of different grape varieties, able to eliminate lactic and acetic acid bacteria spoilages at laboratory scale. Musts obtained from grape-berries of Vitis vinifera cv. Chardonnay and Moscatel and a red wine made with Vitis vinifera cv. Tintilla de Rota were used to isolate bacteriophages. Bacteriophages were obtained from all musts and wine, belonging to the order Caudovirals and to the family Tectivirals. They were isolated by classical virology methods and identified by electron microscopy. Host-bacteria used in the study were the lactic acid bacteria of the species Lactobacillus hilgardii, L. plantarum and Oenococcus oeni and the acetic bacteria Acetobacter aceti. A comparative study was performed on musts and wines, previously inoculated with bacteria, by phage titration and SO2 addition to study the effectiveness of bacteriophages against bacteria. The comparative study showed that some bacteriophages were as effective as sulfur dioxide at low concentration.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nBacteria are part of the natural microbiota of wine, and they play a key role in winemaking by contributing to the aroma and flavor of wine while reducing acidity. Conversely, they can cause undesirable wine spoilage problems, reducing wine quality and value (Bartowsky, 2009). Some lactic acid and acetic acid bacteria species, such as Lactobacillus sp., Pediococcus sp., and Acetobacter sp., are primarily responsible for the loss of quality of musts and wines, leading to defects such as a \\u201cvinegary,\\u201d \\u201csharp, like nail polish-remover\\u201d taste or preventing alcoholic and/or malolactic fermentation (Bartowsky and Henschke, 2008; Bartowsky, 2009; Fumi et al., 2010; Sengun, 2017).\\nSulfur dioxide (SO2) is the most commonly used antimicrobial compound in wine production due to its efficacy and low cost, being able to affect a wide variety of spoilage microorganisms. Despite being active against bacteria and some yeasts, it is innocuous to enological yeasts, since sulfur dioxide is combined with pyruvate, 2-ketoglutarate, and other compounds that are produced during alcoholic fermentation (Ough and Were, 2005). Thus, the main advantage is that Saccharomyces cerevisiae, the most common yeast in fermentation, is not significantly affected by sulfur dioxide at the levels used, so that it can perform the process normally (Lisanti et al., 2019). Moreover, SO2 protects wines against oxidation, and in adequate doses, stabilizes the color of wine and provides a complex flavor, especially in wines made with grape varieties that offer little in a sensory way (Guerrero and Cantos-Villar, 2015; Lisanti et al., 2019). The free sulfur dioxide in wine is dissociated in three chemical forms: molecular sulfite, bisulfite ion, and sulfite ion. The balance between the different chemical forms depends on the pH, the alcoholic strength, and the storage temperature of the wine. The only active form in terms of protection against contaminating microorganisms (e.g., Brettanomyces, Lactobacillus) is molecular sulfur dioxide, which is responsible for the pH value (Ough and Were, 2005). The more acidic the wine, the more active the free SO2 will be (Sturm et al., 2013). The initial dose depends on different factors, such as the grape maturity, the sanitary state of the grape harvest, its temperature, and, mainly, the type of wine to be obtained. Due to the great diversity of wines, there is not a pre-established dose, but winemakers adjust it according to the raw materials, technology, and products they have available.\\nAccording to Commission Regulation (EU) No. 53/2011 of January 21, 2011, amending Regulation (EC) No. 606/2009, the European Commission established an authorized total SO2 dose (mg/L) according to the type of wine. However, the number of people who are sensitive or allergic to this compound is increasing and, consequently, the European Commission has proposed to limit its use until its complete prohibition. This fact presents a problem for the wine industry, which may suffer major quality losses due to the uncontrolled proliferation of harmful microorganisms in wine. There has been a great deal of research on possible alternatives that can carry out the major functions of sulfur dioxide but, until now, none have been completely successful (Guerrero and Cantos-Villar, 2015). Chemical compounds such as colloidal silver complexes (Izquierdo-Ca\\u00f1as et al., 2012; Garc\\u00eda-Ruiz et al., 2015) and dimethyl dicarbonate (Costa et al., 2008) and even natural compounds such as lysozyme, grapevine seeds and shoots, chitosan, and bacteriocins (Lasanta et al., 2010; Garde-Cerd\\u00e1n et al., 2013; Liburdi et al., 2014; Dias et al., 2015; Rold\\u00e1n et al., 2017; Valera et al., 2017; Raposo et al., 2018; Marchante et al., 2019) are some examples. The use of polyphenolic extracts has also been investigated. Specifically, the use of enological tannins combined with lysozyme could reduce the amount of SO2 in wines, although the volatile composition is significantly modified (Sonni et al., 2009, 2011). In white wines, the use of almond skin and eucalyptus leaf extracts has also been proposed to reduce the SO2 content during barrel aging (Gonz\\u00e1lez-Rompinelli et al., 2013).\\nBacteriophages, also known as phages, are viruses that infect bacteria and promote their lysis. Recently, they have been proposed as an alternative to the use of antibiotics in animals, as biopreservers of food, and as a tool for detecting pathogenic bacteria in food processing (Mathur et al., 2003; Clark and March, 2006; Premarathne et al., 2017). The characteristics of being innocuous to human cells and specific to their host range makes bacteriophages an innovative and safe way to replace the traditional chemical compounds used in the food industry, which can generate allergies and intolerances in humans (Garc\\u00eda et al., 2008). On the other hand, the production of these chemical agents, which include pesticides and herbicides, has an environmental impact on ecosystems, besides a socioeconomic impact, due to the tendency of the population to consume more natural products with fewer allergens and additive components. There is wide recognition that bacteriophage use as an antimicrobial poses new questions regarding its eventual impact on natural environments (Meaden and Koskella, 2013). It is important to evaluate the potential risk of altering the composition of natural microbial communities through the liberation of phages from medical or agricultural applications. This disruptive effect could be harmful to ecosystems, as the phages could alter the nutrient cycle (Fern\\u00e1ndez et al., 2019). However, these changes are expected to be much less severe than with synthetic antimicrobial agents due to the high specificity of bacteriophage activity.\\nIn this study, the aim was to search for bacteriophages that develop naturally in musts and wine. Thus, this research will allow us to identify phages able to hinder the development of the acetic and lactic acid bacteria that are responsible for quality losses in winemaking. If such viruses could be recognized as promising sulfur dioxide substitutes in combination with antioxidants, new strategies could be developed to deal with the wine spoilages caused by bacteria.\\nMaterials and Methods\\nStrains and Culture Media\\nLactic acid and acetic bacteria strains were obtained from the Colecci\\u00f3n Espa\\u00f1ola de Cultivos Tipo (CECT). Lactobacillus hilgardii (CECT 4786), Lactobacillus plantarum (CECT 748T), and Oenococcus oeni (CECT 4742) were selected as lactic acid strains, while Acetobacter aceti (CECT 473) were chosen as the acetic bacteria.\\nSpecific culture media were used for the growth of each strain following CECT recommendations. For lactic acid bacteria from genus Lactobacillus, MRS liquid/agar was used (peptone 1% w/v, meat extract 1% w/v, yeast extract 0.5 w/v, glucose 2% w/v, ammonium citrate 0.2 w/v, sodium acetate 5% w/v, K2HPO4 2% w/v, pH 6.2\\u20136.5). When agar was needed, 1.5% w/v (nutritive) and 0.75% w/v (layer) of agar was dissolved in 1 l of distilled water.\\nThe O. oeni strain was grown in MLO medium (tryptone 1% w/v, yeast extract 0.5% w/v, glucose 1% w/v, fructose 0.5% w/v, MgSO4\\u22c57H2O 0, 2% w/v, MnSO4\\u22c5H2O 0.05% w/v, diamonium citrate 0.35% w/v, L-cysteine hydrochloride 0.5% w/v, Tween 80 1 ml, pH 4.8). When agar was needed, 1.5% w/v (nutritive) and 0.75% w/v (layer) of agar was dissolved in 900 ml of distilled water.\\nIn the case of the acetic bacteria, two culture media were used: (i) GYC medium (glucose 2% w/v, yeast extract 1% w/v, CaCO3 2% w/v, pH 6.8) and (ii) M\\u00fceller-Hinton medium (meat extract 0.2% w/v, hydrolyzed casein 1.75% w/v, starch 0.15% w/v, agar 1.7% w/v, pH 7.3 \\u00b1 0.1). When agar was needed, 1.5% w/v (nutritive) and 0.75% w/v (layer) of agar was dissolved in 1 l of distilled water. The most recommended culture medium is GYC, in which the main compound is calcium carbonate, used to give opacity (Bordons and Reguant, 2013; Philippe et al., 2018). This opacity enables the bacterial growth to be monitored, as the microorganism dissolves carbonate when colonies are developed, producing holes or differently colored areas in the plate.\\nPhage Plaque Assays for Titration and Isolation\\nBacteriophage Plaque Assays\\nHost bacteria were grown in liquid cultures of their specific media until the exponential phase was reached. Conic tubes (Eurotubo\\u00ae) were inoculated with 15 ml of liquid media and 1 ml of fresh bacteria culture and were incubated at 30\\u00b0C for L. hilgardii and L. plantarum, 26\\u00b0C for O. oeni, and 28\\u00b0C for A. aceti for 24\\u201348 h for lactic acid bacteria and from 7 to 10 days for the acetic acid baterium.\\nThe presence of bacteriophages was determined by the double-layer agar (DLA) technique, as described by Sambrook and Russell (2001). In order to obtain the phage sample, a conic tube containing 25 ml of must or wine was centrifuged at 5000 rpm for 10 min and filtrated with a 0.2 \\u03bcm filter (Millipore, MA, United States). Assays were carried out for each bacterial strain in wines and musts from three different varieties of grape: wine from Vitis vinifera ssp. cv. Tintilla de Rota (pH 3.53, ethanol content 14.5% v/v) and musts from V. vinifera ssp. cv. Moscatel (pH 4.06) and cv. Chardonnay (pH 3.88). Tenfold dilutions (from 10\\u20131 to 10\\u20137) of the phage samples were prepared in sterile 1\\u00d7 PBS solution (NaCl 138 mM, KCl 3 mM, Na2HPO4 8.1 mM, KH2PO4 1.5 mM). Then, 100 \\u03bcl of diluted phage was added to 500 \\u03bcl of the host-bacteria culture, including a control sample without phage. They were incubated for 10 min at the growth temperature of each strain. Each phage/bacteria solution was then combined with 6 ml of layer agar in a sterile 15 ml conic tube, and they were mixed briefly. Each mixture was poured onto a Petri dish containing solid nutritive agar, avoiding air bubble formation, and left to dry for 10 min in a laminar flow hood. Finally, the Petri dishes were incubated at the growth temperature of each strain for 24 h for lactic acid bacteria and from 5 to 6 days for acetic bacteria.\\nPlaque-Forming Units (PFU) were counted for each type of bacteria by applying the formula:\\nPFUml=no\\u2062plates\\u2062\\u00d7\\u2062dilutionvolume\\u2062of\\u2062applied\\u2062virus\\u2062solution.\\nBacteriophage Isolation\\nBacteriophages were isolated from the lysis plaques formed in the Petri dishes after titration. These clear lysis zones on agar were cut with a sterile scalpel and placed in an Eppendorf microtube containing 1 ml of sterile 1\\u00d7 PBS solution. Three drops of chloroform were then added, and the mixture was shaken for 1 min using a vortex. It was centrifuged at 4\\u00b0C at 3300 rpm and filtered with a 0.2 \\u03bcm filter (Millipore, MA, United States). The process was repeated for the phage plaques obtained in each type of must and wine. The success of the isolation was checked by performing a second phage plaque assay following the same protocol as described before.\\nBacteriophage Characterization\\nTransmission Electron Microscopy (TEM)\\nThe sample preparation consisted of negative staining. A 2% solution of fosfotungstic acid (PTA, Electron Microscopy Sciences, Hatfield, PA, United States) was made up and adjusted to pH 8.1 using normal KOH. Ten microliters of each phage suspension in PBS was added to 10 \\u03bcl of PTA, and the solution was then poured onto carbon-coated 300 MESH copper grids. After 10 min, the grids were dried with filter paper and left to dry at room temperature. The preparations were examined in a TEM JEOL 2100.\\nSaturated Phenol DNA Extraction and Quantification\\nPhage stocks (obtained by the double-layer agar technique and recovered from the top layer) were enriched by adding 100 \\u03bcl of each virus to 20 mL of each host-bacteria culture to obtain high-titer (1010 to 1012 phage/ml) phage concentrates for DNA extraction. After 16 h of incubation at the growth temperature of each host bacterium, three drops of chloroform were added, and the cultures were then centrifuged at 14000 rpm at 4\\u00b0C for 30 min. The supernatant was collected and filtered with a 0.20 \\u03bcm Millipore filter.\\nA total of 400 \\u03bcl of the filtered phage suspension was added to a solution containing 20 \\u03bcl of EDTA (0.5 M Ethylenediaminetetraacetic Acid, pH 8, Sigma-Aldrich), 50 \\u03bcl of SDS 10% (Sodium Dodecyl Sulfate, Pronadisa, Madrid, Spain), and 5 \\u03bcl of proteinase K (Panreac, Glenview, IL, United States). The sample was incubated at 37\\u00b0C for 1 h. Subsequently, 400 \\u03bcl of saturated phenol in 0.5 M Tris\\u2013HCl (Sigma-Aldrich) was added and the mixture was centrifuged at 13000 rpm for 4 min.\\nThe aqueous phase was collected, and 200 \\u03bcl of saturated phenol in Tris\\u2013HCl and 200 \\u03bcl of phenol: chloroform:isoamyl alcohol (25:24:1) (Sigma-Aldrich) were added. They were briefly mixed and centrifuged at 13000 rpm for 4 min. The aqueous phase was transferred into a new Eppendorf microtube containing 400 \\u03bcl of phenol:chloroform:isoamyl alcohol and then centrifuged at 12000 rpm for 4 min. The aqueous phase was transferred back to another 2-mL Eppendorf microtube, and 200 \\u03bcl of potassium acetate (3 M, pH 7) and 600 \\u03bcl of isopropanol were added. It was then centrifuged at 12000 rpm for 10 min. The supernatant was discarded, and the pellet was resuspended in 200 \\u03bcl of cold ethanol 70% v/v. It was then centrifuged at 13000 rpm for 5 min, and ethanol was discarded. The pellet obtained was dehydrated at 45\\u00b0C for 20 min using a Speed-Vac concentrator (Eppendorf, Germany). Finally, the pellet was resuspended in 50 \\u03bcl of sterile ultrapure water. The extracted DNA concentration was measured using a Nanodrop spectrophotometer (Thermo Fisher Scientific, MA, United States) at a wavelength of 260/280.\\nRestriction Enzyme Digestion\\nThe phage genetic material was digested with restriction enzymes in order to determine its size. The enzymes used were EcoRI, EcoRV, HinfI, and HindIII, all of which were from Promega (Madison, WI, United States). Restricted DNA was electrophoresed on 0.7% (w/v) agarose gel in TAE buffer 1\\u00d7 (40 mM Tris-acetate, 1 mM EDTA) and visualized by UV photography after staining with ethidium bromide 5 \\u03bcg/mL. The Lambda phage digested with HindIII enzyme was used as a marker.\\nPulsed-Field Gel Electrophoresis (PFGE)\\nPulsed-field gel electrophoresis consisted of taking 1 \\u03bcg of non-digested viral DNA sample and heating it at 60\\u00b0C for 5 min to denaturalize cohesive ends. The genetic material was then cooled in ice and quickly added into a low fusion point agarose solution (cf: 0.6%, Pronadisa). It was then loaded in an agarose gel 1% in TBE 0.5\\u00d7 buffer (Tris 0.89 M, EDTA 0.02 M, and boric acid 0.89 M) in order to electrophorese it.\\nThe electrophoresis conditions were an initial pulse of 0.1 s and a final pulse of 8 s. An electric field of 0.6 V/cm was applied for 15 h at 14\\u00b0C and a 120\\u00b0 angle. After the electrophoresis had finished, the gel was stained with ethidium bromide 5 \\u03bcg/mL and visualized with an imaging system with a UV transilluminator (Biorad). The genome size was determined using the Lambda phage genome digested with the restriction enzyme HindIII as a reference.\\nIn vitro Assay of Bacteriophage Efficacy\\nAn in vitro assay in a 96-well multiplate (NuncTM 96-well polystyrene conical bottom MicrowellTM, Thermo Fisher, Denmark) was carried out in order to compare the action of the bacteriophage against host bacteria with the addition of sulfur dioxide following the procedure established by Rojo-Bezares et al. (2007) with some modifications and using the purified phages as the antimicrobial agents. Assays were done in 200 \\u03bcl of the different culture media according to the bacteria used (MRS, MLO, or GYC/MH). This assay was also carried out in musts and wines at the pH values mentioned above. The initial concentration of inoculated bacteria was 106 cells/ml after overnight growth in the indicated culture media. Pure bacteriophage solutions in PBS 1\\u00d7 were diluted from 10\\u20131 to 10\\u20138 UFP/mL and were co-inoculated with their host bacteria. On the other hand, sulfurous anhydrous dilutions were made from sodium metabisulfite (Sigma/Aldrich) from 100 to 450 mg/L, according to the doses recommended by Commission Regulation (EU) No. 53/2011, at pH 3.5 for Lactobacillus species and pH 4.5 for O. oenis. Blank and negative controls of infection were 200 \\u03bcl of the correspondent culture media and 100 \\u03bcl plus 100 \\u03bcl of cells, respectively. Microplates were sealed with Breathe-Easy membranes (Sigma, Saint Louis, MI, United States) and placed into an automatic microplate reader (Multiskan go reader, Thermo Fisher Scientific, Waltham, MA, United States). The optical density of the mixture at 600 nm was read every hour (1 min of shaking just before automatic reading by the microplate reader) at the growth temperature of each host bacteria over 48\\u201372 h. Assays were performed in triplicate, and the data obtained were processed with the software SkanIt RE for Multiskan GO 3.2. The absorbance data obtained were exported to Microsoft Excel for processing.\\nStatistical Analysis\\nAll statistical analysis was performed using SPSS software, version 25.0, and MS Excel for Windows. Data collected taken in triplicate and were analyzed using analysis of variance with Tukey\\u2019s multiple comparison test or by the non-parametric Mann\\u2013Whitney\\u2013Wilcoxon test.\\nResults\\nBacteriophage Titration and Isolation\\nPhage plaques were observed from L. plantarum as host bacteria from filtered samples of red wine (Tintilla de Rota) and musts (Moscatel and Chardonnay). In the case of L. hilgardii, bacteriophage plaques were only observed from red wine samples, while O. oeni showed to be infected by bacteriophages isolated from red wine and Moscatel and Chardonnay musts. Plaques from bacteriophages that infected lactic acid bacteria of genus Lactobacillus were small and displayed a uniform distribution (Figure 1A), while those associated with O. oeni showed a notable increase in the plaque size, and their distribution was very heterogeneous, occupying the entire plaque (Figure 1B).\\nFigure 1. Appearance of plaques formed by bacteriophages with the double layer agar (DLA) method. (A) Small size and homogeneous distribution of plaques from bacteriophages with L. plantarum as host bacteria and isolated from Chardonnay must. (B) Heterogeneous plaque formed by phage infecting O. oeni as host bacteria, isolated from red wine (Tintilla de Rota grape variety).\\nIn order to confirm the first results obtained, a second titration applying the DLA technique with purified phages isolated from the first screening was carried out. As a result, seven different bacteriophages were isolated and named according to their source, host bacteria, and plaque morphotype (Table 1). Significant differences were found between the PFU/ml in the titers made with the musts and the wine (p > 0.05). The number of plate-forming units (PFU) of the genus Lactobacillus titer was lower in red wine compared to white grape musts, which can be related to the acidity and ethanol content differences between wine and must (Table 1). As the optimum pH value for bacterial growth is near 4.5, musts formed a more suitable environment for the development of these bacterial populations and, therefore, for their predisposition to being infected by bacteriophages (Puertas, 1989; Bordons and Reguant, 2013).\\nTable 1. Count of plaques obtained in the second titration, applying the DLA technique.\\nRegarding A. aceti titration in GYC and MH media, lysis plaques were not observed using commercial vinegar samples, red wine (Tintilla de Rota), or white musts (Moscatel and Chardonnay) as the phage isolation source.\\nCharacterization of the Bacteriophages\\nTEM\\nThe bacteriophage morphology was analyzed by TEM, and they were taxonomically classified according to the International Committee on Taxonomy of Viruses (ICTV) (Ackermann, 2001). The images obtained were visually selected, and those with better resolution are featured in Figure 2.\\nFigure 2. Bacteriophages were taxonomically classified according to the International Committee on Taxonomy of Viruses (ICTV). Phages that infected genus Lactobacillus belonged to the order Caudovirals, which could be classified into three families: Myoviridae, Siphoviridae, and Podoviridae. Those that infected O.oeni belonged to poliedric phages and were classified into the Tectiviridae family.\\nPhages that infected genus Lactobacillus belonged to the order Caudovirals. According to Bradley\\u2019s classification (1987), three families have been described: Myoviridae (A), phages with a long contractile tail, Siphoviridae (B), phages with a long non-contractile tail, and Podoviridae (C), phages with a short tail (Figure 2). Each family has been divided into subgroups according to their head length: isometric (1), moderately elongated (2), and long-headed (3). Phages Tin1Lp and Tin1Lh belonged to the Siphoviridae (A) family, with morphotype 1 (isometric). Phages Mos1Lp and Char1Lp belonged to the Myoviridae (B) family, contractile tail phages, morphotype 1. On the other hand, those that infected O.oeni belonged to poliedric phages (D) and were classified into the Tectiviridae family (morphotype 4), phages with lipid vesicles and a pseudo-tail. The virus size went in decreasing order from the that which infected L. hilgardii, which was 330 nm in length, followed by those that infected L. plantarum, between 130 and 175 nm in length, and then those whose host was O. oeni, with a smaller size, between 80 and 100 nm length (Figure 2).\\nGenetic Material From Bacteriophages\\nAfter DNA extraction, the quantification showed high concentrations of viral genetic material (between 150 and 1000 ng/\\u03bcL). The genetic material was extracted with a high purity level, as indicated by the 260/280 ratio, whose value should vary between 1.8 and 2.0. Regarding the 260/230 ratio, it is considered that DNA is pure when its value varies from 1.7 and 2, so the quality of the extracted DNA was proved.\\nThe treatment with EcoRI, EcoRV, HinfI, and HindIII did not digest the genomes of the phages. Several gels were made, applying different conditions, first varying the voltage intensity and then the agarose concentration used to prepare the gel. None of the tests showed a good resolution of the DNA, either the complete genome or the digested one, of any of the bacteriophage samples. However, after applying the PFGE technique, bacteriophages that infected bacteria of the genus Lactobacillus showed a genome of approximately 1300 bp in size, while those corresponding to O. oeni were over 1000 bp (Figure 3).\\nFigure 3. Results obtained from pulsed-field electrophoresis. Bacteriophages that infected bacteria of the genus Lactobacillus showed a genome approximately 1300 bp in size, while those corresponding to O. oeni were over 1000 bp.\\nIn vitro Assay: Co-culturing Bacteriophages Against Host Bacteria Related to Different Sulfur Dioxide Doses\\nIn the growth curve corresponding to L. plantarum, significant differences (p > 0.01) were found between each dilution made from 1 \\u00d7 10\\u20131 to 1 \\u00d7 10\\u20138 and the controls in red wine and Moscatel and Chardonnay musts (Figure 4). L. plantarum growth values in the presence of bacteriophage were smaller than the sulfur dioxide effect when added in any concentration to the red wine (Figure 4A). This proves that the isolated bacteriophage is an effective infectious agent under the experimental conditions. Regarding Moscatel must, the growth values of L. plantarum with some bacteriophage dilutions were larger than or similar to the results obtained with sulfur dioxide (Figure 4B). The only favorable result obtained was 10\\u20131, which was able to decrease the bacterial growth to below that with anhydride addition, which indicates that the phage Mos1Lp would only be efficient in controlling a certain L. plantarum population. Regarding Chardonnay must, a 10\\u20137 concentration was enough to equate the action of sulfur dioxide, and the efficacy of the isolated bacteriophage was confirmed in all the minor dilutions tested (Figure 4C).\\nFigure 4. Effect of specific bacteriophages diluted from 10\\u2013 1 to 10\\u2013 8 UFP/mL on L. plantarum compared to sulfurous action. The viral efficacy was assessed by taking, as a reference, the host bacteria growth in red wine (A), Moscatel must (B), and Chardonnay must (C), both in the presence of bacteriophage and of sulfur dioxide. Each point is the mean \\u00a9 SD of three assays.\\nIn the case of the growth kinetics of L. hilgardii, the phage Tin1Lh showed an insufficient activity compared to the dose of SO2 for red wine (Figure 5). No significant differences between the different dilutions from 1 \\u00d7 10\\u20131 to 1 \\u00d7 10\\u20138 and the control (p > 0.01) were found in red wine (Figure 5). All of the bacteriophage concentrations applied (from 10\\u20131 to 10\\u20138 PFU/ml) followed the same trend as the control curve, so it was deduced that these concentrations lacked infective power under the experimental conditions.\\nFigure 5. Effect of bacteriophages diluted from 10\\u2013 1 to 10\\u2013 8 UFP/mL on L. hilgardii compared to sulfurous action. The viral efficacy was assessed by taking, as a reference, the host bacteria growth in the presence of either bacteriophage or sulfur dioxide in red wine. Each point is the mean \\u00a9 SD of three assays.\\nConcerning the results obtained for the kinetics of O. oeni in red wine, bacteriophages displayed certain inhibition activity on growth in all the dilutions applied. However, this growth decrease was not strong compared to the effect of sulfur dioxide applied in the doses used for this type of wine. Significant differences (p > 0.01) were found between each dilution made from 1 \\u00d7 10\\u20131 to 1 \\u00d7 10\\u20138 and the control in red wine and in Moscatel must (Figure 6). Phage samples continued to be out of range either dilution used (Figure 6A). Bacteriophages that infected O. oeni in Moscatel must also showed efficacy in decreasing bacterial growth (Figure 6B). Unlike red wine, there was major infective power, as the difference between the control curve and those corresponding to different dilutions was bigger. However, the kinetics graph obtained from O. oeni as the host bacterium in Chardonnay must did not show conclusive results data not shown. This may be due to the inability of the bacteria to grow in must, preventing the establishment of a relationship between the control growth and that associated with the phage. Consequently, it could not be compared with the sulfur dioxide effect in this grape variety must.\\nFigure 6. Effect of specific bacteriophages diluted from 10-1 to 10\\u2013 8 UFP/mL on O. oeni compared to sulfurous action. The viral efficacy was assessed by taking as a reference the host bacteria growth in red wine (A) and in Moscatel must (B), either in presence of bacteriophage or of sulfur dioxide. Each point is the mean \\u00b1 SD of three assays.\\nDiscussion\\nThe results obtained in this work evidenced the presence of antibacterial agents in the intrinsic microbiology of the grape musts and wine used under the experimental conditions harnessed in this research. This fact was confirmed with the appearance of phage plaques in titers made for lactic acid bacteria. As stated in the results, the plaques displayed diverse morphology and distributions, which indicated the existence of different infectious agents. The number of phage plaques ranged according to the source, wine or must.\\nThe Lactobacillus titer had a significantly higher PFU/ml value in white grape must than in red wine. This could be due to the ten-fold dilutions applied and to the host bacteria predisposition to grow and to be infected, as bacteriophages completely depend on their host cell, both for replicating their genome and for the production of new viral particles (Ackermann, 2003; Gonz\\u00e1lez-Garc\\u00eda, 2014). Red wine confers a suitable environment for the growth of lactic acid bacteria because of malolactic fermentation. However, transformations produced during the alcoholic fermentation cause a pH drop and an increase in the organic acid content (Oc\\u00f3n, 2014). This can trigger an increase of acidity, which can lead to bacterial incompetence and, consequently, the loss of infective power of their bacteriophages. In contrast, musts have higher pH values than wine, closer to the optimum value for Lactobacillus to grow (pH 4.5), which could explain a greater ease of bacteriophage infection of their host. In fact, the first isolation of wine bacteriophages in 1985 by Davis et al. (1985) confirmed that pH values lower than 3.5 inhibited their growth, as verified by Henick-Kling et al. (1986a). In contrast with the results obtained in Lactobacillus, bacteriophages of O. oeni were able to infect and lyse it at red wine pH values, as stated by Henick-Kling et al. (1986b). Regarding the ethanol content in red wine used in our experiments, LAB strains (including O. oeni) were able to grow at 14.5% (v/v) of ethanol. Phages are usually resistant to high ethanol concentrations; Ebrecht et al. (2010) applied ethanol at different concentrations (from 10 to 100% v/v) to control the number of virulent and temperate Lactobacillus delbrueckii phages, and only ethanol 100% was effective against the LAB phages. In other studies, a concentration of 75% of ethanol was effective against other species of LAB phages (Binetti and Reinheimer, 2000; Su\\u00e1rez and Reinheimer, 2002; Quiberoni et al., 2003). Thus, the phages isolated in our study seemed to also be resistant at 14.5% of ethanol.\\nThis study was limited, as no bacteriophages for acetic acid bacteria were isolated, though these are involved both in the production of vinegar and loss of wine quality. The acidity from the vinegar (pH < 2.8) used as a precautionary measure to avoid phages in the industrial field could be crucial to avoid the development of bacteriophages (Sellmer et al., 1992; Gullo and Giudici, 2008). Firstly, A. aceti showed some difficulties in growing properly. Unlike lactic acid bacteria, those from genus Acetobacter took more than a week to grow in the liquid medium and between 5 and 6 days in the solid one. When titrations were carried out with these cultures, changes were observed after 5 days, in contrast with the 24 h needed by bacteriophages that infected the lactic acid bacteria used in this work. Different colored areas appeared, bacteria colonies were not appreciated, that gave a first hint of the possible existence of bacteriophages. However, the use of GYC medium with calcium carbonate posed a problem for elucidating the main cause of the PFU formation, as they could be inhibition halos produced by phages or holes caused by carbonate dissolution by the bacterial metabolism while growing. As an alternative, M\\u00fceller\\u2013Hinton medium was used, since it can facilitate the observation of lysis plates because it is translucent; it has been used to show antimicrobial activity in previous acetic bacteria experiments (Du Toit and Lambrechts, 2002; Mateluna, 2006). Acetic bacteria cultures took 14 days to reach the exponential phase in the liquid medium and a week in the solid one, which makes the titre results slower than for lactic acid bacteria. These titrations did not show any lysis plate or inhibition haloes, which proved the inexistence of bacteriophages able to infect the selected strain of A. aceti in vinegar, wine, and musts, and allowed it to be substantiated that the plates in the GYC medium were due to the bacterial metabolism itself. However, expected results were not obtained in the titer for commercial vinegar, wine, and grape must samples. This could also be related to vinegar acid conditions or to the acidity generated by A. aceti. However, the existence of specific phages of acetic bacteria was documented in 1971, when Schocher et al. (1979) isolated the first bacteriophage that infected an Acetobacter suboxidans culture. Its host range was analyzed using 24 Acetobacter and Gluconobacter strains, where only four of the genus Acetobacter were susceptible to being infected. Taking into account that A. suboxidans turned into Gluconobacter suboxidans and due to the closeness of the two genuses, in 1981, Jucker and Ettlinger (1981) did a host range analysis of this phage and suggested that none of the Acetobacter strains but Gluconobacter, according to the new nomenclature, were susceptible. The presence of specific bacteriophages of acetic bacteria was also confirmed by a study of different alcohol vinegar factories in different places in Europe (France, Sweden, and Germany). Four bacteriophages were isolated and showed varied effects on the bacterial population, all of them related to fermentation breaks (Stamm et al., 1989).\\nBacteriophage morphology was observed by electron microscopy. According to the ICTV guidelines, bacteriophages that infected genus Lactobacillus were classified as Caudovirals or tailed phages. Phages Tin1Lp and Tin1Lh belong to the Siphoviridae family, while Mos1Lp and Char1Lp belong to Myoviridae. Both families were obtained in the same proportion, which is in agreement with previous studies where the majority of families were Siphoviridae and Myoviridae, leaving Podoviridae as very rare (Ackermann, 1998). All of the phages isolated were icosahedral, which is the most frequent morphotype in Caudovirals (Ackermann, 2001). On the other hand, the phages that infected O. oeni belonged to the Tectiviridae family within the polyhedral phages. Although tectiviruses usually infect enteric species, bacilli, and species of the Thermus genus, previous studies have already demonstrated the existence of this family of viruses in wine, such as bacteriophage GC1, a new tectivirus that infects Gluconobacter cerinus in dry white wine-making (Ackermann and DuBow, 1987; Philippe et al., 2018).\\nBacteriophage characterization was carried out using molecular microbiology techniques. Viral genetic material extraction achieved high concentrations, so it was the basis for deducing that the bacteriophages were constituted of DNA. Visualization by electronic microscopy with DNA extraction allowed the phages to be classified as double-stranded DNA, matching the majority of previous studies of wine or vinegar bacteriophages (Davis et al., 1985; Henick-Kling et al., 1986a, b; Hern\\u00e1ndez-Magad\\u00e1n, 2007; Dini, 2011 Philippe et al., 2018). Despite EcoRI, EcoRV, HinfI, and HindIII endonucleases having been used in similar studies of bacteriophages in dairy products and wine (Moineau et al., 1993; Ackermann, 1998; Spricigo et al., 2013), in our study, the RFLP analysis showed that these enzymes did not digest the genome of the phages isolated. It is well known that the restriction enzymes need very clean and concentrated genetic material; however, according to the extraction and quantification data obtained in our study, the concentration and ratios of DNA were sufficiently satisfactory. The phage resistance to endonuclease activity could be attributed to several factors. The most prevalent is the natural loss of restriction sites during evolution (Moineau et al., 1993; Gonz\\u00e1lez et al., 2005), although the integration of unusual bases in the bacteriophage genome (Jensen et al., 1998; Shivu et al., 2007) and a loss of validity when there is some methylation in phage DNA affecting the restriction sites that endonucleases could recognize (Sails et al., 1998; Dini, 2011) have also been described. Sequencing and comparative genomic analysis of the bacteriophages could provide us with further information about their genetic identity.\\nUnlike bacteria, there are no gene sequences conserved in any of the viruses or phages that can serve as universal primer sites for Polymerase Chain Reaction (PCR) amplification. PCR-based analyses of viral DNA must therefore target specific subsets of the total viral assemblage (Steward, 2001). Taking into account the results obtained by the RFLP, we used the PFGE as a tool for bacteriophage DNA fingerprinting. This molecular technique provides a quick and visual record of the genome size distribution that can be used for qualitative and quantitative comparisons among banding patterns of the viral DNA samples (Steward, 2001). Periodic changes of electric field orientation and duration were applied, and bands evidenced the presence of DNA and gave a phage genome size estimation. According to the size range of the Lambda phage marker fragments digested with the restriction enzyme HindIII, phages of lactic bacteria are about 1300 bp in size, larger than those corresponding to Oenococcus, which approach 1000 bp. It was found that the genome of the seven isolates was much smaller than that of other lactic phages, which usually vary between 18 and 55 Kb (Jarvis et al., 1991; Moineau et al., 1993).\\nRegarding the assays done to evaluate the efficacy of the bacteriophages compared to that of different doses of sulfur dioxide, the most promising results were obtained in the three bacteriophages of L. plantarum. In the three grape varieties tested, a very low phage concentration (10\\u20137) was able to substantially decrease the host-bacteria growth rate. Control curves followed a common structure, showing an initial exponential phase (initial culture) followed by a stationary phase, in which growth stabilized. In the case of red wine, the exponential phase was notably shorter than in Moscatel and Chardonnay musts. This could be due to the low availability of nutrients in fermented wine in contrast to the white musts, which contained sugars available to be used by host bacteria. In contrast to the results obtained for L. plantarum, the only isolated bacteriophage that infected L. hilgardii, Tin1Lh, did not show the same or comparable efficacy as the SO2 dose applied. Control wells, containing only bacteria, followed the same kinetic curve as all the bacteriophage doses, from 10\\u20131 to 10\\u20138, from which it was deduced these concentrations lacked infective power under the experimental conditions.\\nIn general, the results of the phage titers may vary between trials, due to the resistance that can be generated by bacteria to bacteriophage infection (Ackermann, 2003). Although consideration of the influence of \\u201cQuorum Sensing\\u201d (QS) mechanisms of the host bacteria itself is out of the scope of this work, it could provide a possible explanation for why the bacteriophages would act in the plaque titration assay and not in the 96-well plate (Waters and Bassler, 2005). Thereby, in the plaque assay, titration L. hilgardii could reach the required cellular density to activate QS mechanisms and, therefore, the bacteriophage would have used the self-inducers to start lysis, while in the 96-well plaque, the phages remained under the lysogenic cycle. However, genetic assays would be needed to prove the existence of receptors or self-inducer molecules shared by L. hilgardii and its specific bacteriophage. If this were not proved, it would be necessary to study other factors related to the host bacteria growth. Another question that remains to be addressed is the difference between the ability of different phages to capture the same host bacteria when immobilized on a substrate (Double Layer Agar) or in a liquid media (Calendar, 2006; Hosseinidoust et al., 2011a, b). Additional research needs to be conducted in order to fully characterize the observed phenomena in wine of decreasing infection of phages in a liquid environment in contrast to a solid one.\\nFinally, bacteriophages that infected O. oeni showed an inhibitory effect on the growth of their host bacteria. Control curves had the same behavior as those of L. plantarum, with a quicker growth stabilization observed in red wine than in musts. Both isolated phages, Tin1Oo and Mos1Oo, showed a certain infective power. Nevertheless, the difference in growth between the control well and those with phage dilutions was significantly bigger in musts, which indicates that the host bacteria found must conditions more suitable to growth, promoting the infectious effect of the phage. The effect of these bacteriophages could not be compared with sulfur dioxide, since the bacteria growth was still much lower when the chemical compound was applied. Consequently, since phages showed a lytic effect on the host bacteria, it would only be necessary to increase the phage concentration to obtain a growth inhibition comparable to the sulfurous action. Results of the molecular analysis of O. oeni have established direct relationships between different bacteria strains and their bacteriophages. In 2005, the first genome sequence of O. oeni strain PSU-1 was published, and currently, information is available on the complete genome of 13 different strains. It was discovered that the main differences between the ORF (open reading frame) contents of bacteria strains are due to the presence of multiple bacteriophage sequences located at different points. At least four lysogenic phages can be integrated as complete elements or fragments in the O. oeni genome associated with the transfer of RNA binding sites (Borneman et al., 2012, 2013). The study of the conditions suitable for the activation of the lytic cycle of phages integrated in the bacterium genome could enhance the inhibitory effect on its growth.\\nThe best results obtained in this work were those for bacteriophages that infected L. plantarum, whose effects were similar to sulfurous addition, followed by those of O. oeni, which showed less inhibition than obtained with SO2, while phages that infected L. hilgardii did not display any infective power under the experimental conditions. Although preliminary observations were obtained in this research, the presence of bacteriophages in the three grape varieties used highlights the potential application of a \\u201cphage cocktail\\u201d combining several infectious agents against a specific bacteria or different species or pollutants that require elimination in combination with a natural antioxidant. The fact that phages from the same family are able to infect different species allows savings to be proposed in the production of cocktails, in which the minimum number of bacteriophages could be used to prevent contaminating bacteria growth in the process of winemaking in a single treatment. In order to reduce the likelihood of resistant bacteria, cocktails could be designed that contain phages that use different receptors. In that way, if the mutant generated resistance to one of the phages in the cocktail, it would remain sensitive to the others. Currently, the use of lytic bacteriophages is a growing alternative for the biological control of pathogens in food and the food industry (Hermoso et al., 2007; Garc\\u00eda et al., 2008; Spricigo et al., 2013; Sulakvelidze, 2013). In the present study, the objective was not to remove pathogen microorganisms but those that cause losses in wine quality, such as lactic acid and acetic bacteria. Thus, the application of phage cocktails made by specific bacteriophages for every grape variety, vinification stage, and contaminating bacterium, together with a natural antioxidant, could be proposed in further studies at a larger scale.\\nData Availability Statement\\nAll datasets generated for this study are included in the article/supplementary material.\\nAuthor Contributions\\nGC-B, JM, and MR-C contributed to the conception and design of experiments. MR-C, MR-M, GC-B, and JM performed the experiments. MR-C, MR-M, GC-B, JM, and JC analyzed the data. MR-C and GC-B contributed to writing the manuscript. MR-M revised the English. All authors critically revised the manuscript before submission.\\nFunding\\nThis work was financed by the project UCA18DGUEII02.\\nConflict of Interest\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nWe thank Olga Aliseda, Juan Gonz\\u00e1lez, and Jos\\u00e9 Mar\\u00eda Gerald\\u00eda (Electron Microscopy Division, University of C\\u00e1diz, C\\u00e1diz, Spain) for assistance on the sample preparation, MET management, and image analysis.\\nReferences\\nAckermann, H. W., and DuBow, M. S. (1987). Viruses of Prokaryotes: General Properties of Bacteriophages, Vol. 1. Boca Raton, FL: CRC Press, 49\\u201385.\\nGoogle Scholar\\nAckermann, W. (2003). Bacteriophage observations and evolution. J. Microbiol. 154, 245\\u2013251. doi: 10.1016/S0923-2508(03)00067-6\\nCrossRef Full Text | Google Scholar\\nBinetti, A. G., and Reinheimer, J. A. (2000). Thermal and chemical inactivation of indigenous Streptococcus thermophilus bacteriophages isolated from Argentinian dairy plants. J. Food Prot. 64, 509\\u2013515. doi: 10.4315/0362-028X-63.4.509\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBordons, A., and Reguant, C. (2013). Bioqu\\u00edmica de las bacterias l\\u00e1cticas del vino y la fermentaci\\u00f3n malol\\u00e1ctica. Acenolog\\u00eda\\nGoogle Scholar\\nBorneman, A. R., McCarthy, J. M., Chambers, P. J., and Bartowsky, E. J. (2012). Comparative analysis of the Oenococcus oeni pan genome reveals genetic diversity in industrially-relevant pathways. BMC Genomics 13:373. doi: 10.1186/1471-2164-13-373\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCalendar, R. L. (2006). The Bacteriophages. New York, NY: Oxford University Press.\\nGoogle Scholar\\nCosta, A., Barata, A., Malfeito-Ferreira, M., and Loureiro, V. (2008). Evaluation of the inhibitory effect of dimethyl dicarbonate (DMDC) against wine microorganisms. Food Microbiol. 25, 422\\u2013427. doi: 10.1016/j.fm.2007.10.003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDavis, C., Silveira, N. F., and Fleet, G. H. (1985). Occurrence and properties of bacteriophages of Leuconostoc oenos in Australian wines. Appl. Environ. Microbiol. 50, 872\\u2013876.\\nPubMed Abstract | Google Scholar\\nDias, R., Volas-Boas, E., Campos, F. M., Hogg, T., and Couto, J. A. (2015). Activity of lysozyme on Lactobacillus hilgardii strains isolated from Port wines. Food Microbiol. 49, 6\\u201311. doi: 10.1016/j.fm.2015.01.007\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDini, C. (2011). Aislamiento y Caracterizaci\\u00f3n Molecular de Bacteri\\u00f3fagos de Bacterias Enteropat\\u00f3genas Para Biocontrol de Enfermedades Transmitidas por Alimentos (ETA). Doctoral dissertation, Universidad Nacional de la Plata, Buenos Aires.\\nGoogle Scholar\\nDu Toit, W. J., and Lambrechts, M. G. (2002). The enumeration and identification of acetic acid bacteria from South African red wine fermentations. Int. J. Food Microbiol. 74, 57\\u201364. doi: 10.1016/S0168-1605(01)00715-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nEbrecht, A., Guglielmotti, D. M., Tremmel, G., Reinheimer, J. A., and Su\\u00e1rez, V. B. (2010). Temperate and virulent Lactobacillus delbrueckii bacteriophages: comparison of their thermal and chemical resistance. Food Microbiol. 27, 515\\u2013520. doi: 10.1016/j.fm.2009.12.012\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nFern\\u00e1ndez, L., Guti\\u00e9rrez, D., Garc\\u00eda, P., and Rodr\\u00edguez, A. (2019). The perfect bacteriophage for therapeutic applications \\u2013 a quick guide. Antibiotics 8:E126. doi: 10.3390/antibiotics8030126\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nFumi, M. D., Krieger-Weber, S., D\\u00e9l\\u00e9ris-Bou, M., Angela, S., and du Toit, M. (2010). Lactobacillus: the next generation of malolactic fermentation starter cultures-an overview. Food Bioprocess Technol. 4, 876\\u2013906. doi: 10.1007/s11947-010-0448-448\\nCrossRef Full Text | Google Scholar\\nGarc\\u00eda, P., Mart\\u00ednez, B., Obeso, J. M., and Rodr\\u00edguez, A. (2008). Bacteriophages and their application in food safety. Lett. Appl. Microbiol. 47, 479\\u2013485. doi: 10.1111/j.1472765X.2008.02458.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGarc\\u00eda-Ruiz, A., Crespo, J., L\\u00f3pez-de-Luzuriaga, J. M., Olmos, M. E., Monge, M., Rodr\\u00edguez-\\u00c1lfaro, M. P., et al. (2015). Novel biocompatible silver nanoparticles for controlling the growth of lactic acid bacteria and acetic acid bacteria in wines. Food Control 50, 613\\u2013619. doi: 10.1016/j.foodcont.2014.09.035\\nCrossRef Full Text | Google Scholar\\nGarde-Cerd\\u00e1n, T., L\\u00f3pez, R., Garijo, P., Gonz\\u00e1lez-Arenzana, L., Guti\\u00e9rrez, A. R., L\\u00f3pez-Alfaro, I., et al. (2013). Application of colloidal silver versus sulfur dioxide during vinification and storage of Tempranillo red wines. Aust. J. Grape Wine Res. 20, 51\\u201361. doi: 10.1111/ajgw.12050\\nCrossRef Full Text | Google Scholar\\nGonz\\u00e1lez, \\u00c1, Hierro, N., Poblet, M., Mas, A., and Guillam\\u00f3n, J. M. (2005). Application of molecular methods to demonstrate species and strain evolution of acetic acid bacteria population during wine production. Int. J. Food Microbiol. 102, 295\\u2013304. doi: 10.1016/j.ijfoodmicro.2004.11.020\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nGonz\\u00e1lez-Garc\\u00eda, V. (2014). Mecanismos Moleculares Implicados en la Infecci\\u00f3n del Bacteri\\u00f3fago T7. Doctoral dissertation, Universidad Aut\\u00f3noma de Madrid, Espa\\u00f1a.\\nGoogle Scholar\\nGonz\\u00e1lez-Rompinelli, E. M., Rodr\\u00edguez-Bencomo, J. J., Garc\\u00eda-Ruiz, A., S\\u00e1nchez-Pat\\u00e1n, F., Mart\\u00edn-\\u00c1lvarez, P. J., Bartolom\\u00e9, B., et al. (2013). A winery-scale trial of the use of antimicrobial plant phenolic extracts as preservatives during wine ageing in barrels. Food Control 33, 440\\u2013447. doi: 10.1016/j.foodcont.2013.03.026\\nCrossRef Full Text | Google Scholar\\nGuerrero, R. F., and Cantos-Villar, E. (2015). Demonstrating the efficiency of sulfur dioxide replacements in wine: a parameter review. Trends Food Sci. Technol. 42, 27\\u201343. doi: 10.1016/j.tifs.2014.11.004\\nCrossRef Full Text | Google Scholar\\nGullo, M., and Giudici, P. (2008). Acetic acid bacteria in traditional balsamic vinegar: phenotypic traits relevant for starter cultures selection. Int. J. Food Microbiol. 125, 46\\u201353. doi: 10.1016/j.ijfoodmicro.2007.11.076\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHenick-Kling, T., Lee, T. H., and Nicholas, D. J. D. (1986a). Characterization of the lytic activity of bacteriophages of Leuconostoc oenos isolated from wine. J. Appl. Bacteriol. 61, 525\\u2013534. doi: 10.1111/j.1365-2672.1986.tb01725.x\\nCrossRef Full Text | Google Scholar\\nHenick-Kling, T., Lee, T. H., and Nicholas, D. J. D. (1986b). Inhibition of bacterial growth and malolactic fermentation in wine by bacteriophage. J. Appl. Bacteriol. 61, 287\\u2013293. doi: 10.1111/j.1365-2672.1986.tb04289.x\\nCrossRef Full Text | Google Scholar\\nHermoso, J., Garc\\u00eda, J., and Garc\\u00eda, P. (2007). Taking aim on bacterial pathogens: from phage therapy to enzybiotics. Curr. Opin. Microbiol. 10, 461\\u2013472. doi: 10.1016/j.mib.2007.08.002\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHern\\u00e1ndez-Magad\\u00e1n, A. (2007). Aislamiento, Caracterizaci\\u00f3n y Detecci\\u00f3n Precoz de Bacteri\\u00f3fagos de Streptococcus thermophilus en la Industria L\\u00e1ctea. Doctoral dissertation, Universidad de Oviedo, Oviedo.\\nGoogle Scholar\\nHosseinidoust, Z., Tufenkji, N., and van de Ven, T. G. M. (2011a). Predation in homogeneous and heterogeneous phage environments affects virulence determinants of Pseudomonas aeruginosa. Appl. Environ. Microbiol. 79, 2862\\u20132871. doi: 10.1128/AEM.03817-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHosseinidoust, Z., Van de Ven, T. G. M., and Tufenkji, N. (2011b). Bacterial capture efficiency and antimicrobial activity of phage-functionalized model surfaces. Langmuir 27, 5472\\u20135480. doi: 10.1021/la200102z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nIzquierdo-Ca\\u00f1as, P. M., Garc\\u00eda-Romero, E., Huertas-Nebreda, B., and G\\u00f3mez-Alonso, S. (2012). Colloidal silver complex as an alternative to sulphur dioxide in winemaking. Food Control 23, 73\\u201381. doi: 10.1016/j.foodcont.2011.06.014\\nCrossRef Full Text | Google Scholar\\nJarvis, A. W., Fitzgerald, G. F., Mata, M., Mercenier, A., Neve, H., Powell, I. B., et al. (1991). Species and type phages of lactococcal bacteriophages. Intervirology 32, 2\\u20139. doi: 10.1159/000150179\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nJensen, E. C., Schrader, H. S., Rieland, B., Thompson, T. L., Lee, K. W., Nickerson, K. W., et al. (1998). Prevalence of broad-host-range lytic bacteriophages of Sphaerotilus natans, Escherichia coli and Pseudomonas aeruginosa. Appl. Environ. Microbiol. 64, 575\\u2013580.\\nPubMed Abstract | Google Scholar\\nJucker, W., and Ettlinger, L. (1981). Host range of a bacteriophage of acetic acid bacteria. Int. J. Syst. Evol. Microbiol. 31, 245\\u2013246. doi: 10.1099/00207713-31-3-245\\nCrossRef Full Text | Google Scholar\\nLasanta, C., Rold\\u00e1n, A., Caro, I., P\\u00e9rez, L., and Palacios, V. (2010). Use of lysozyme for the prevention and treatment of heterolactic fermentation in the biological aging of sherry wines. Food Control 21, 1442\\u20131447. doi: 10.1016/j.foodcont.2010.03.013\\nCrossRef Full Text | Google Scholar\\nLiburdi, K., Benucci, I., and Esti, M. (2014). Lysozyme in wine: an overview of current and future applications. Compr. Rev. Food Sci. Food Saf. 13, 1062\\u20131073. doi: 10.1111/1541-4337.12102\\nCrossRef Full Text | Google Scholar\\nLisanti, M. T., Blaiotta, G., Nioi, C., and Moio, L. (2019). Alternative methods to so2 for microbiological stabilization of wine. Compr. Rev. Food Sci. Food Saf. 18, 455\\u2013479. doi: 10.1111/1541-4337.12422\\nCrossRef Full Text | Google Scholar\\nMarchante, L., Izquierdo-Ca\\u00f1as, P. M., G\\u00f3mez-Alonso, S., Ala\\u00f1\\u00f3n, M. E., Garc\\u00eda-Romero, E., P\\u00e9rez-Coello, M. S., et al. (2019). Oenological potential of extracts from winery and cooperage by-products in combination with colloidal silver as natural substitutes to Sulphur dioxide. Food Chem. 276, 485\\u2013493. doi: 10.1016/j.foodchem.2018.10.072\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMateluna, R. A. (2006). Estudio de Actividad Antibacteriana de Potenciales Biocontroles Sobre Bacterias Ac\\u00e9ticas Involucradas en la Pudrici\\u00f3n \\u00c1cida de la uva. Master\\u2019s thesis, Universidad de Santiago de Chile, Santiago.\\nGoogle Scholar\\nMathur, M., Vidhani, S., and Mehndiratta, P. (2003). Bacteriophage therapy: an alternative to conventional antibiotics. J. Assoc. Physicians India 51, 593\\u2013596.\\nGoogle Scholar\\nMoineau, S., Pandian, S., and Klaenhammer, T. R. (1993). Restriction/modification systems and restriction endonucleases are more effective on lactococcal bacteriophages that have emerged recently in the dairy industry. Appl. Environ. Microbiol. 59, 197\\u2013202.\\nPubMed Abstract | Google Scholar\\nOc\\u00f3n, E. (2014). Diversidad de Levaduras no-Saccharomyces en Diferentes Ecosistemas Vitivin\\u00edcolas. Doctoral dissertation, Universidad de La Rioja, Logro\\u00f1o.\\nGoogle Scholar\\nOugh, C. S., and Were, L. (2005). \\u201cSulfur dioxide and Sulfites,\\u201d in Antimicrobials in Food, eds P. M. Davinson, J. N. Sofos, and A. L. Branen (Boca Raton, FL: Taylor & Francis), 143\\u2013160.\\nGoogle Scholar\\nPhilippe, C., Krupovic, M., Jaomanjaka, F., Claisse, O., Petrel, M., and le Marrec, C. (2018). Bacteriophage GC1, a novel tectivirus infecting Gluconobacter cerinus, an acetic acid bacterium associated with wine-making. Viruses 10:39. doi: 10.3390/v10010039\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPremarathne, J. M. K. J. K., Thung, T. Y., New, C. Y., Huat, J. T. Y., Basri, D. F., Rukayadi, Y., et al. (2017). Distribution of bacteriophages in food and environment samples. Int. Food Res. J. 24, 888\\u2013896.\\nGoogle Scholar\\nPuertas, B. (1989). El Potencial Vit\\u00edcola y Enol\\u00f3gico de Quince Variedades Blancas de vid en la Zona de Jerez. Doctoral dissertation, Universidad de C\\u00e1diz, C\\u00e1diz.\\nGoogle Scholar\\nQuiberoni, A., Guglielmotti, D. M., and Reinheimer, J. A. (2003). Inactivation of Lactobacillus delbrueckii bacteriophages by heat and biocides. Int. J. Food Microbiol. 84, 51\\u201362. doi: 10.1016/s0168-1605(02)00394-x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRaposo, R., Chinnici, F., Ru\\u00edz-Romero, M. J., Puertas, B., Cuevas, F. J., Carb\\u00fa, M., et al. (2018). Sulfur free red wines through the use of grapevine shoots: impact on the wine quality. Food Chem. 243, 453\\u2013460. doi: 10.1016/j.foodchem.2017.09.111\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRojo-Bezares, B., S\\u00e1enz, Y., Zarazaga, M., Torres, C., and Ruiz-Larrea, F. (2007). Antimicrobial activity of nisin against Oenococcus oeni and other wine bacteria. Int. J. Food Microbiol. 116, 32\\u201336. doi: 10.1016/j.ijfoodmicro.2006.12.020\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRold\\u00e1n, A., Lloret, I., and Palacios, V. (2017). Use of a submerged yeast culture and lysozyme for the treatment of bacterial contamination during biological aging of sherry wines. Food Control 71, 42\\u201349. doi: 10.1016/j.foodcont.2016.06.016\\nCrossRef Full Text | Google Scholar\\nSails, D., Wareing, D. R., Bolton, F. J., Fox, J., and Curry, A. (1998). Characterization of 16 Campylobacter jejuni and E. coli typing bacteriophages. J. Med. Microbiol. 47, 123\\u2013128.\\nPubMed Abstract | Google Scholar\\nSambrook, J., and Russell, D. W. (2001). Molecular Cloning: A Laboratory Manual, 3rd Edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.\\nGoogle Scholar\\nSchocher, A. J., Kuhn, H., Schindler, B., Palleroni, N. J., Despreaux, C. W., Boublik, M., et al. (1979). Acetobacter bacteriophage A-1. Arch. Microbiol. 121, 193\\u2013197. doi: 10.1007/BF00689986\\nCrossRef Full Text | Google Scholar\\nSellmer, S., Sievers, M., and Teuber, M. (1992). Morphology, virulence and epidemiology of bacteriophage particles isolated from industrial vinegar fermentations. Syst. Appl. Microbiol. 15, 610\\u2013616. doi: 10.1016/S0723-2020(11)80123-2\\nCrossRef Full Text | Google Scholar\\nShivu, M. M., Rajeeva, B. C., Girisha, S. K., Karunasagar, I. I., and Krohne, G. (2007). Molecular characterization of Vibrio harveyi bacteriophages isolated from aquaculture environments along the coast of India. Environ. Microbiol. 9, 322\\u2013331. doi: 10.1111/j.1462-2920.2006.01140.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSonni, F., Cejudo Bastante, M. J., Chinnici, F., Natali, N., and Riponi, C. (2009). Replacement of sulfur dioxide by lysozyme and oenological tannins during fermentation: influence on volatile composition of white wines. J. Sci. Food Agric. 89, 688\\u2013696. doi: 10.1002/jsfa.3503\\nCrossRef Full Text | Google Scholar\\nSonni, F., Chinnici, F., Natali, N., and Riponi, C. (2011). Pre-fermentative replacement of sulphur dioxide by lysozyme and oenological tannins: effect on the formation and evolution of volatile compounds during the bottle storage of white wines. Food Chem. 129, 1193\\u20131200. doi: 10.1016/j.foodchem.2011.05.104\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSpricigo, D. A., Bardina, C., Cort\\u00e9s, P., and Llagostera, M. (2013). Use of a bacteriophage cocktail to control Salmonella in food and the food industry. Int. J. Microbiol. 165, 169\\u2013174. doi: 10.1016/j.ijfoodmicro.2013.05.009\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nStamm, W. W., Kittelmann, M., Follmann, H., and Tr\\u00fcper, H. G. (1989). The occurrence of bacteriophages in spirit vinegar fermentation. Appl. Microbiol. Biotechnol. 30, 41\\u201346. doi: 10.1007/BF00255994\\nCrossRef Full Text | Google Scholar\\nSteward, G. F. (2001). Fingerprinting viral assemblages by pulsed field gel electrophoresis (PFGE). Methods Microbiol. 30, 85\\u2013103. doi: 10.1016/s0580-9517(01)30041-7\\nCrossRef Full Text | Google Scholar\\nSturm, M. E., Arroyo-L\\u00f3pez, F. N., Garrido-Fern\\u00e1ndez, A., Querol, A., Mercado, L. A., Ram\\u00edrez, M. L., et al. (2013). Probabilistic model for the spoilage wine yeast Dekkera bruxellensis as a function of pH, ethanol and free SO2 using time as a dummy variable. Int. J. Food Microbiol. 170, 83\\u201390. doi: 10.1016/j.ijfoodmicro.2013.10.019\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSu\\u00e1rez, V. B., and Reinheimer, J. A. (2002). Effectiveness of thermal treatments and biocides in the inactivation of Argentinian Lactococcus lactis phages. J. Food Prot. 65, 1756\\u20131759. doi: 10.4315/0362-028X-65.11.1756\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSulakvelidze, A. (2013). Using lytic bacteriophages to eliminate or significantly reduce contamination of food by foodborne bacterial pathogens. J. Sci. Food Agric. 93, 3137\\u20133146. doi: 10.1002/jsfa.6222\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nValera, M. J., S\\u00e1inz, F., Mas, A., and Torija, M. J. (2017). Effect of chitosan and SO2 on viability of Acetobacter strains in wine. Int. J. Food Microbiol. 246, 1\\u20134. doi: 10.1016/j.ijfoodmicro.2017.01.0220168.1605\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWaters, C. M., and Bassler, B. L. (2005). Quorum sensing: cell-to-cell communication in bacteria. Ann. Rev. Cell Dev. Biol. 21, 319\\u2013346. doi: 10.1146/annurev.cellbio.21.012704.131001\\nCrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.02169\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.02169/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:50:30.641Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2018.02169\",\n",
      "            \"title\": \"Frontiers | Jumbo Bacteriophages Are Represented Within an Increasing Diversity of Environmental Viruses Infecting the Emerging Phytopathogen, Dickeya solani\",\n",
      "            \"description\": \"\\\\textit{Dickeya} species are economically important phytopathogens widespread in mainland Europe that can reduce crop yields by 25$\\\\%$. There are no effective environmentally-acceptable chemical systems available for diseases caused by \\\\textit{Dickeya}. Bacteriophages have been suggested for use in biocontrol of these pathogens in the field, and limited field trials have been conducted. To date the majority of bacteriophages capable of infecting \\\\textit{Dickeya solani}, one of the more aggressive species, are from the same family, the \\\\textit{Ackermannviridae}, many representatives of which have been shown to be unsuitable for use in the field due to their capacity for generalised transduction. Members of this family are also only capable of forming individual plaques on \\\\textit{Dickeya solani}. Here we describe novel bacteriophages from environmental sources isolated on \\\\textit{Dickeya solani}, including members of two other viral families; \\\\textit{Myoviridae} and \\\\textit{Podoviridae}, most of which are capable of forming plaques on multiple \\\\textit{Dickeya} species. Full genomic sequencing revealed that the \\\\textit{Myoviridae} family members form two novel clusters of jumbo bacteriophages with genomes over 250 kbp, with one cluster containing phages of another phytopathogen \\\\textit{Erwinia amylovora}. Transduction experiments showed that the majority of the new environmental bacteriophages are also capable of facilitating efficient horizontal gene transfer, however the single \\\\textit{Podoviridae} family member is not. This particular phage therefore has potential for use as a biocontrol agent against multiple species of \\\\textit{Dickeya}.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"1. Introduction\\nThe genus Dickeya, recently reclassified into the novel family Pectobacteriaceae (Adeolu et al., 2016), currently consists of 11 phytopathogenic species that can cause severe disease in economically important crops including tomato, orchid, and potato (Alic et al., 2017a). Until 2004, almost all European potato isolates of Dickeya were assigned as Dickeya dianthicola (Toth et al., 2011). In 2008/2009, a new clade of Dickeya in European potato isolates was identified (Laurila et al., 2008; Parkinson et al., 2009; S\\u0142awiak et al., 2009) and in 2014 a new species was proposed; Dickeya solani (van der Wolf et al., 2014).\\nDickeya solani is able to spread more easily through the plant vascular system and survive at higher temperatures than D. dianthicola (Toth et al., 2011). It is currently the predominant potato pathogen in Europe, with reductions in yield of up to 25% reported in potatoes exposed to Dickeya species (Tsror et al., 2009). Whilst there have been isolated cases of D. solani reported in England and Wales since 2007 (Cahill et al., 2010), these were all found in crops originating from outside of the UK (Toth et al., 2016). It is currently yet to become established in the UK, and to mitigate the significant economic cost inflicted by this virulent phytopathogen, the Scottish government has introduced specific legislation aimed at preventing the establishment of D. solani in its seed industry (Mansfield et al., 2012).\\nThe significant economic costs inflicted by Dickeya species have stimulated research interest in methods for control of these virulent phytopathogens. Bacterial viruses (bacteriophages; phages) have been suggested as potential tools for biocontrol due to their specificity, environmental persistence and biological \\u201corganic\\u201d nature (Iriarte et al., 2007; Czajkowski et al., 2017; Svircev et al., 2018). Several studies have isolated phages capable of infecting Dickeya species (Adriaenssens et al., 2012c; Czajkowski et al., 2014a,b, 2015a; Matilla et al., 2014; Ali\\u010d et al., 2017b; Day et al., 2017). Their potential use as biocontrol agents has been trialed both in the lab and in the field (Adriaenssens et al., 2012c) and these studies showed a partially \\u201ctherapeutic\\u201d outcome with reduced crop losses. There is a commercial product available, Biolyse\\u2122, from APS Biocontrol Ltd that is a phage cocktail able to target Pectobacterium as well as Dickeya species. Designed as a washing solution for potatoes during factory processing, to our knowledge it is the first, and currently the only, commercial Dickeya-targeting biocontrol product. It has been reported that Biolyse\\u2122 has been used by the UK supermarket chain Tesco (Branston, 2012). The identities of the phages contained within this cocktail however have not been reported.\\nAll of the Dickeya phages isolated so far, and 96% of all known phages, are members of the order Caudovirales (Fokine and Rossmann, 2014), which currently consists of four families. Apart from the Siphoviridae family member BF-CIM1/14 recently described by Ali\\u010d et al. (2017b) and three Podoviridae family members reported in our recent publication (Day et al., 2017), the vast majority of D. solani phages characterized so far share a high degree of similarity and have been designated members of the Ackermannviridae family (formerly known as the Vi1virus genus; Adriaenssens et al., 2018) based on morphology and genomic comparisons. A summary of these phages is shown in Table 1. This has generated research interest, as these phages have been isolated from both soil and water samples and in three separate European countries; Belgium, Poland, and the United Kingdom. Host range testing has shown that the phages isolated in Belgium and the majority isolated in the UK are capable of forming plaques on strains of D. solani only (Adriaenssens et al., 2012c; Day et al., 2017). The phages isolated in Poland are reported to infect multiple species of Dickeya and Pectobacterium (Czajkowski et al., 2014a,b, 2015a), however, host range testing to the level of individual plaque formation has not been reported\\u2014an important criterion that allows exclusion of false positives (Khan Mirzaei and Nilsson, 2015). The high degree of morphological and genomic similarity between these phages and the other Ackermannviridae family members makes the reported broader host range that spans genera an intriguing prospect, assuming plaque formation data supporting this broader host range can be confirmed.\\nTable 1. Members of the Ackermannviridae family isolated on Dickeya solani.\\nSixty-seven phages were described in our recent publication (Day et al., 2017), 59 of which were only capable of forming plaques on D. solani species. When two were genomically sequenced they showed a high degree of similarity with the previously published D. solani phages of the Ackermannviridae family. The remaining eight phages were capable of forming plaques on other species of Dickeya, including Dickeya zeae, Dickeya chrysanthemi, and Dickeya paradisiaca. These particular phages warranted further investigation, as an expanded host range can be helpful for further application in phage therapy. The aim of this study therefore was to genomically sequence these phages to determine their similarity to previously published D. solani phages and their suitability for use in phage therapy.\\n2. Results\\n2.1. Host Range and Plaque Morphology\\nThe plaques of the previously described D. solani phages that are members of the Ackermannviridae family have tended to be clear, defined, and easy to distinguish from the bacterial top lawn (Adriaenssens et al., 2012c; Czajkowski et al., 2015a). This is the case for the other Ackermannviridae family members isolated in this laboratory, and is also true of one of the broader host range phages JA10. However, the other seven have an indistinct, turbid plaque morphology that is often hard to distinguish from the bacterial top lawn (data not shown). We believe this to be the reason why the host range of the eight D. solani phages shown in Table 2 is different from the host range reported in the previous publication (Day et al., 2017). This first became apparent due to confusing results generated by related, unpublished experiments that suggested a variation in the host range from that published in our previous paper (Day et al., 2017). Rigorous retesting has confirmed that the host range data presented in Table 2 are accurate and that the previous interpretations were incorrect. The efficiency of plating data in Table 2 shows that most of the phages are able to adsorb at a similar efficiency to all species of Dickeya, apart from JA29 which has an efficiency 10\\u22124 lower on D. paradisiaca and D. dadantii subsp. dieffenbachiae.\\nTable 2. Broader host range of eight phages capable of infecting other species of Dickeya as well as Dickeya solani.\\n2.2. Morphological Classification\\nBacteriophages have traditionally been classified based on morphological characteristics viewed under electron microscopy (Ackermann, 2012). The majority of D. solani phages isolated to date are members of the Ackermannviridae family, which share common morphological characteristics. A representative of this family, XF4, is shown in Figure 1A. These phages possess an icosahedral head with a diameter of around 90 nm, a contractile tail around 110 nm in length and structures at the base of the tail that have been described as \\u201cstars\\u201d or \\u201cprongs\\u201d and have been identified as tail spikes (Adriaenssens et al., 2012a). Apart from these tail spikes, this is classical morphology of the phage family Myoviridae, therefore the combination of a contractile tail and tail spikes are the morphological markers of the family Ackermannviridae.\\nFigure 1. Transmission electron micrographs of six Dickeya solani phages. XF4 (A) is a member of the Ackermannviridae family, exhibiting an icosahedral head of around 90 nm, a tail length of 110 nm and tail spikes at the base of the tail. JA10 (B) is a member of the Podoviridae family, characterized by a short stubby tail. JA11, JA29, and AD1 (C\\u2013E) are members of the Myoviridae family with unclear tail appendages. AD2 (F) has a similar morphology to XF4 and has a partially-contracted tail.\\nAll eight of the expanded host range phages were viewed under transmission electron microscopy. Seven of them had an icosahedral head and long tail, with the structures at the base of the tail remaining unclear, with two representatives shown in Figures 1C,D. This marks them as either members of the Myoviridae or Ackermannviridae families, however, these phages were significantly larger than the previously viewed members of the Ackermannviridae family, which can be seen by comparing Figure 1A with Figure 1D. The head diameters were over 120 nm, with a tail length of around 150 nm, which suggested that these phages were not Ackermannviridae family members.\\nThe indistinct morphology of the tail appendages of the JA jumbo phages (best seen in Figure 1C) has also been identified in other phages. When first described in the Escherichia phage 121Q (Ackermann and Nguyen, 1983) this morphology was presumed to be an artifact of microscopy involving damage to the tail. It was also thought that the dimensions, at the time reported to be a head diameter of 150 nm and a tail length of 165 nm, were overstated. However, this morphology has since been directly reported in the Pseudomonas putida phage Lu11 (Adriaenssens et al., 2012b), the Pectobacterium carotovorum phage CBB (Buttimer et al., 2017), and the Erwinia amylovora phage Y3 (Buttimer et al., in press), and has been dubbed the \\u201chairy Myoviridae\\u201d morphology (Buttimer et al., 2017).\\nUnexpectedly, as can be seen in Figure 1B, the phage JA10 could be classified as a member of the Podoviridae family when imaged, characterized by an icosahedral head and a short non-contractile tail. Whilst this is not the first member of this family that we have isolated (Day et al., 2017), this is the first isolate we have studied in further depth. The genome of JA10 was therefore sequenced to investigate the similarity between it and previously published Dickeya-infecting Podoviridae family members (Ali\\u010d et al., 2017b).\\n2.3. Genome Sequence of Podoviridae Family Member JA10\\nThe genome of JA10 is 40,131 bp, has 50 predicted genes, and is shown in Figure 2. The closest match in the database is an as yet unpublished D. solani phage Ninurta (Genbank reference: MH059639) isolated from organic waste in Denmark that shares 95% DNA identity with JA10. The closest published phage is the Pectobacterium parmentieri phage PP74 isolated from potato washing waste water in Russia in 2015 (Kabanova et al., 2018), which shares less than 14% nucleotide identity. It shares no DNA sequence identity with the other sequenced Dickeya-infecting Podoviridae family member BF25/12 (Ali\\u010d et al., 2017b). PP74 has been designated as a T7-like virus and a member of the Autographivirinae subfamily, with a conserved core genome. A translated nucleotide comparison of JA10 with the type phage T7 showed that most of the predicted genes are conserved (data not shown), including almost all the genes with a proposed function, such as the T3/T7-like RNA polymerase, structural capsid genes and DNA packaging machinery. JA10 is therefore also a member of the Autographivirinae subfamily.\\nFigure 2. Map of the genome of JA10. The outer gray ring marks open reading frames, with those highlighted in black discussed in more detail in the text, whilst the inner ring categorizes the proposed functions of these genes. The genome map was generated using Circos.\\nThere are several putative proteins encoded in the JA10 genome that do not have significant homology in the T7 genome and these are highlighted in Figure 2. Four of these seven putative proteins, the S-adenosyl-L-methionine hydrolase, bacterial RNA polymerase inhibitor, minor capsid protein, and the first tail assembly protein, have a protein of similar function encoded at this position in T7. The variation between JA10 and T7 in these putative proteins is therefore likely a determinant of host specificity. The marked tail fiber protein, which share a common N-terminal region but differ at the C-terminus between the two phages, is also likely a contributor to host range specificity. The DNA ligase highlighted in Figure 2 neighbors a conserved ligase and consists of fewer than 200 amino acids. This is therefore a possible result of a recombination duplication event and may not be a functional protein. The final tail assembly protein, close to the end of the JA10 genome, has no functional homologue in T7.\\n2.4. Novel Jumbo \\u201cHairy Myoviridae\\u201d Phages\\nSequencing of the other seven genomes showed that, although the phages were isolated independently, several shared 100% identity at the nucleotide level. JA11, 31, and 32 grouped together, as did JA33 and 37. JA31, 32, and 37 were therefore excluded from further analysis. The genome size for the four remaining phages is between 253 and 256 kbp. A summary of the characteristics of these genomes, as well as JA10, is shown in Table 3. These phages are therefore jumbo phages, defined as phages with a genome over 200 kbp (Yuan and Gao, 2017). The genomes are significantly larger than the conserved size of the Ackermannviridae family genomes, which are around 150 kbp, and larger than most sequenced phages. As of July 2018, there were 9,351 recorded phage genome sequences in Genbank (http://millardlab.org/bioinformatics/bacteriophage-genomes/) and these JA phages would be the 62nd\\u201365th largest sequenced. Many of the over 300 predicted open reading frames in each genome do not match any known genes; the majority of those that do share identity with known genes are from the E. amylovora phages Yoloswag (Esplin et al., 2017) and Y3 (Buttimer et al., in press). These are largely structural genes and genes involved in DNA metabolism and replication.\\nTable 3. Summary of the genomes of the broader host range phages.\\n2.5. Variation Within the JA Phages\\nThe gene order of the four JA jumbo phages is largely conserved. Over three quarters of the predicted ORFs are annotated as encoding hypothetical proteins, and many of the differences between the phages are contained within these ORFs as shown in Figure 3. JA29 is the most dissimilar to the others, sharing 86% nucleotide identity with JA11, whereas the nucleotide identity between JA11 and JA13 is 95%. JA11 and JA33 are 99% identical, with the major difference being the insertion of 126 bp in both genomes at different positions, and of different sequences. These insertions are in non-coding regions however, therefore their biological relevance is unclear. The only other differences are in two putative proteins. One is a hypothetical protein, whilst the other contains a putative discoidin domain, with the substitution between JA11 and JA33 (alanine to threonine) in the middle of the domain. Discoidin domains are present in eukaryotic agglutination factors and therefore the possible biological role for this in a phage genome, and the effect of the substitution, is not immediately obvious.\\nFigure 3. Map of the genome of JA11. The outer gray ring marks open reading frames, with those highlighted in black discussed in more detail in the text. The second ring categorizes the proposed functions of these genes, whilst the inner rings highlight the areas of the genome that differ from the genomes of JA29, JA13, and JA33 (third to fifth ring, respectively). The genome map was generated using Circos.\\nJA11, 13, and 29 have differing host ranges, as listed in Table 2. To investigate whether this was caused by variations within the tail fibers of the phages, the amino acid sequences of each of the three putative tail fiber proteins was compared between the phages. JA11 and JA13 possess identical tail fibers, whereas JA29 shows variations of several amino acids in each protein, as listed in Table 4. Whilst these variations could explain the differing host range of JA11 and JA29, it does not explain the difference in host range observed for JA11 and JA13.\\nTable 4. Summary of the annotated genes which differ between JA11, JA13, and JA29.\\nWhilst most of the differences between the phages are located in genes with no predicted function, there are a few annotated that are present in all of the JA phages. These include encoding a DNA helicase, two potential transcription factors and one structural protein, all highlighted in Figure 3 and listed in Table 4.\\nThere are variations in two DNA related genes: a DNA primase and a DNA helicase. The helicase shows the most variation between the phages, as it appears to have undergone insertion or deletion between some of the phages. A comparison of this region of the genome can be seen in Figure 4. There are two ORFs annotated as putative helicases in JA11 and JA33, which both share homology with one ORF in JA13 and JA29. Whether the two ORFs are able to function independently as a helicase, or whether this duplication has rendered them non-functional, is unknown. A DksA/TraR family protein and a ssDNA binding protein, both likely transcription factors, both differ in one amino acid between JA11 and JA13, and share lower identity with the JA29 homolog, particularly the ssDNA binding protein, which differs in 32 positions. A VgrG-like family protein, a component of the T6SS thought to be phage-derived as it is capable of assembling into a structure similar to a phage tail spike (Cianfanelli et al., 2016) shares, at best, only 33% amino acid identity with the closest hit in the E. amylovora phage Y3, and so these may define a relatively novel VgrG-like protein group. JA11 and JA13 differ by one conservative substitution in this protein, whilst JA29 differs in 87 amino acids, 13 of which are conservative substitutions. It is possible that differences in this predicted protein are a contributing factor to the differing host range of these phages.\\nFigure 4. Translated nucleotide comparison of a putative DNA helicase between JA11 (top), JA33 (second), JA13 (third), and JA29 (bottom). Red bars mark areas of amino acid identity, with darker colors showing higher identity. Figure produced using the Artemis Comparison Tool.\\nThe biological significance of the differences observed between the four JA phages is currently unclear. It is somewhat surprising to find variations in genes involved in transcription initiation and DNA replication, as the genomes of these phages are relatively similar in both size and GC content, as summarized in Table 3. It is therefore possible that these differences do not significantly alter the function of these proteins. The variation in the VgrG-like proteins is more logical, as the different host ranges of these phages may be related to differences in tail spikes and other host recognition factors. To determine the impact of these differences, and to investigate why JA11 and JA13 have a different host range despite having apparently identical tail fibers, requires further experimental work.\\n2.6. More Recent Isolates: AD Phages\\nAll of the JA phages were isolated from the River Cam in November 2014. Isolation of XF phages from the same location a year earlier produced mainly members of the Ackermannviridae family and a few Podoviridae family members (Day et al., 2017). Whilst there is clearly some maintenance of viral populations, as members of the two families have been isolated on both occasions, the jumbo phages presented here are a novel grouping. To gain further insight into the viral populations in the River Cam, further samples were taken in October 2017. Two phages were isolated on D. solani and are named AD1 and AD2. Whilst both were only capable of forming plaques on D. solani and not on strains of other species, microscopy showed that they had differing morphologies. AD1 (Figure 1E) appears to have a \\u201chairy Myoviridae\\u201d morphology similar to that of the JA jumbo phages, with a head diameter of 120 nm, tail length of 150 nm, and unclear structures at the base of the tail. AD2 on the other hand (Figure 1F) has a head diameter of 90 nm and a (potentially partially-contracted) 70 nm tail, leading to a tentative classification as a member of the Ackermannviridae family. The structures at the end of the tail are inconclusive.\\nGenome sequencing of the two AD phages showed that, as suggested by microscopy, AD2 is a member of the Ackermannviridae family. It shares 99% nucleotide identity with previously published D. solani Ackermannviridae such as XF4 and LIMEstone1 (Day et al., 2017), although full coverage of the genome was not achieved (data not shown). AD1, as expected, has a large genome of 261,658 bp, confirming that it is a jumbo phage, shown in Figure 5. However, unexpectedly, it has low nucleotide sequence identity with the JA jumbo phages, despite sharing a similar gene order. In fact, a translated nucleotide comparison of JA11 and AD1, as shown in Figure 6C, shows that JA11 is about as similar to AD1 as it is to Y3 (Figure 6A), and a comparison of AD1 and Y3 (Figure 6B) shows them to be more similar to each other than to JA11 at the amino acid level.\\nFigure 5. Map of the genome of AD1. The outer gray ring marks open reading frames whilst the inner ring categorizes the proposed functions of these genes. The genome map was generated using Circos.\\nFigure 6. Translated nucleotide comparison of the genomes of JA11 (A), Y3 (B), and AD1 (C). Red bars mark areas of amino acid identity, with darker colors showing higher identity. Blue bars highlight areas of inversion. Figure produced using the Artemis Comparison Tool.\\n2.7. Phylogeny of the \\u201cHairy Myoviridae\\u201d Phages\\nIn their recent publication, Buttimer et al. discussed the phylogenetic position of Y3 considering its low level of nucleotide identity to existing genomes (Buttimer et al., in press). Two potential subfamilies within the \\u201chairy Myoviridae\\u201d have emerged; the Rak2-like phages, which includes the previously mentioned Pectobacterium phage CBB (Buttimer et al., 2017), and the as yet unnamed subfamily that encompasses the phage discussed here. This was established as it was found that Y3 had homologs including terminase, polymerase and helicase genes in several other phages reported or suspected to have the \\u201chairy Myoviridae\\u201d morphology. A comparison of the large terminase subunit of these phages with those reported here shows clear clustering, and can be seen in Figure 7. As expected, the three JA phages cluster tightly with little variation between them. As reported by Buttimer et al. the Pseudomonas-infecting phages PaBG (Sykilinda et al., 2014) and Lu11 (Adriaenssens et al., 2012b) form a clade, whilst the Ralstonia solanacearum phage phiRSL1 (Yamada et al., 2010) and the metagenomically-derived NCTB (Pfreundt et al., 2017) are single nodes within the tree. As suggested by the translated nucleotide comparison in Figure 6B, Y3 and AD1 form a clade that puts AD1 closer to Erwinia-infecting phages than to the other D. solani phages. Intriguingly, AD1 is placed closer phylogenetically to Y3 than the other E. amylovora-infecting phage Yoloswag (Esplin et al., 2017). The same clustering is seen when using the sequence of the tail sheath proteins of the phages. All of the phages except phiRSL1 have two annotated tail sheath proteins, and the same phylogeny is seen with both (data not shown).\\nFigure 7. Phylogenetic tree of the large terminase subunit constructed using the Maximum Likelihood method in MEGA. All positions containing gaps and missing data were eliminated, with 642 positions in the final dataset. The tree shown has the highest log likelihood (\\u22128677.52) and is drawn to scale, with branch lengths measured in the number of substitutions per site.\\nThe gene order between JA11, AD1, and Y3 is highly conserved. All three genomes contain over 300 open reading frames, with each containing only between one and three unique annotated genes. These unique genes are listed in Table 5 and are all DNA or metabolism-related. There are also five genes common to AD1 and Y3 that are not found in JA11. Whilst phylogenetic clustering, as shown in Figure 7, groups AD1 and Y3 closer than Y3 and Yoloswag, it is interesting to note that the two unique genes possessed by Y3 have homologs in Yoloswag. These two phages were both isolated from apple orchards using E. amylovora, therefore it is surprising that they differ phylogenetically. It is possible that these unique genes are a determinant of the host range of these phages. A phylogenetic comparison of three tail fiber genes found in each genome is shown in Figure 8. This again shows a definite separation between Yoloswag and the other two phages, particularly when comparing Yoloswag_102 with Y3_104 and AD1_102, which occupy the same genomic position. This also suggests the possibility that AD1 may be capable of forming plaques on Erwinia species, but we have not been able to test this as we do not have access to these hosts.\\nTable 5. Unique annotated genes found in the genomes of JA11, AD, and Y3, as well as genes common to AD1 and Y3 but not present in JA11.\\nFigure 8. Phylogenetic tree of three tail fiber proteins constructed using the Maximum Likelihood method in MEGA. All positions containing gaps and missing data were eliminated, with 158 positions in the final dataset. The tree shown has the highest log likelihood (\\u22122426.84) and is drawn to scale, with branch lengths measured in the number of substitutions per site.\\n2.8. Jumbo Phages Are Capable of Transduction\\nIn a recent publication (Day et al., 2017) we tested the ability of the JA phages to facilitate transduction of chromosomal markers and plasmids between Dickeya cells. We can reconfirm that all of the JA jumbos are capable of transduction of a chromosomal marker between D. solani cells at a frequency of between 1 \\u00d7 10\\u22126 and 3 \\u00d7 10\\u22124 and report that the AD phages are capable of transduction of chromosomal markers at similar frequencies. The broader host range of the JA jumbo phages also allows transduction of plasmids between Dickeya species, as shown for a representative of each host range group in Table 6. JA10, the member of the Podoviridae family, proved incapable of transduction under the conditions tested. This makes JA10 a more promising candidate for phage therapy, and suggests the other phages may not be suitable due to the potential risk of transduction in the field.\\nTable 6. Transduction frequencies of the plasmid pECA1039-Km3 from donor Dickeya solani cells into other Dickeya species.\\n3. Discussion\\nAll seed crops imported into Scotland, along with 10% of Scottish-origin crops, are tested for D. solani each year. In 2017, the most recent year for which there are data, 663 samples were tested and none were positive for Dickeya species (Scottish Government, 2017), which has been the finding since 2010 when rigorous testing began. Whilst there have been isolated cases of D. solani infection reported in England and Wales since 2007 (Cahill et al., 2010), these have all originated in crops from outside of the UK (Toth et al., 2016), therefore it is not thought that Dickeya is established in the UK. This begs the question as to why we have been able to isolate phages capable of infecting D. solani with relative ease from the River Cam. We would hypothesize that either D. solani is present in the environment, but has not been confirmed by testing, or there is another permissive, but currently unknown, environmental host(s) for these phage. A novel species of Dickeya, Dickeya aquatica, was isolated from waterways in England (Parkinson et al., 2014), and so far has only been identified in waterway environments. It is, therefore, a formal possibility that this species could be an environmental host for the phages isolated here, but this has not yet been tested.\\nPreviously isolated phages of D. solani have been found almost exclusively to be members of the Ackermannviridae family. This has been a consistent feature of isolations spanning multiple European countries across the last decade, including from both soil and water samples, as shown in Table 1. We questioned in our recent publication whether this indicated a special relationship between Ackermannviridae and D. solani (Day et al., 2017). The phages presented here confirm that that was the result of an extrapolation from a limited viral sample set. There are at least four groups of D. solani phages present in waterways around Cambridge, spanning three of the four Caudovirales families. Phages that have been isolated on other species of Dickeya that are capable of forming plaques on D. solani have also been recently described (Ali\\u010d et al., 2017b), including the fourth Caudovirales family, Siphoviridae. It is therefore apparent that we are only superficially defining the level of phage diversity present in the environment, consistent with the notion that double-stranded DNA phages alone have been predicted to outnumber their bacterial hosts by a factor of 10 to 1 (Chibani-Chennoufi et al., 2004).\\nAll the phages presented here, apart from JA10 and AD2, appear to be representatives of a recently described \\u201chairy Myoviridae\\u201d subfamily (Buttimer et al., in press). To the best of our knowledge, these are the first reported members of this subfamily isolated using Dickeya species. Many of the previously reported members of this subfamily were also isolated on plant-associated bacteria such as Pseudomonas putida (Adriaenssens et al., 2012b) and E. amylovora (Esplin et al., 2017; Buttimer et al., in press). Whether there is a link between this group of phages and plant-associated bacteria, or whether the recent increase in isolation of phages using phytopathogens is skewing this view remains to be determined. The proteins responsible for the \\u201chairs\\u201d that typify this grouping remain unknown, although the identified tail fiber proteins are likely candidates for further investigation.\\nInfection of seed crops with D. solani and related species inflicts a high economic burden, and therefore there is great interest in the use of virulent (lytic) bacteriophages as potential biocontrol agents. There have been multiple tests of the stability, environmental viability and efficacy of Dickeya phages (Adriaenssens et al., 2012c; Ali\\u010d et al., 2017b; Czajkowski et al., 2017) in which promising results have been reported. The Dickeya phages able to form individual plaques through productive lytic cycle replication on multiple Dickeya species, reported both here and by Ali\\u010d et al. (2017b), are potentially more promising for use as biocontrol agents, as they would be able to act against a wider set of phytopathogens. However, we have described here, and previously (Day et al., 2017), that the majority of the D. solani phages (including our phages and the LIMEstone phages) are able to facilitate generalized transduction between host cells. The phages isolated by Czajkowski et al. (2014a,b, 2015a) were not reported to have been tested for generalized transduction, but, due to their classification as Ackermannviridae family members, and the finding that all members of this family tested to date are capable of facilitating transduction, we predict that they are likely to be capable of doing so. Ali\\u010d et al. did not report testing of their phages for transduction capacity, and it is possible that they may not be transducers, but we would echo the caution of the European Medicines Agency, among others, who have stated that it is \\u201cimportant to ensure that therapeutic phages do not carry out generalized transduction\\u201d (Pelfrene et al., 2016). However, the results presented here do suggest that JA10, a podovirus capable of infecting three Dickeya species other than D. solani, could offer some promise as a potentially therapeutic candidate, as it has not shown transduction capabilities when tested.\\nCzajkowski et al. (2014a) have reported that phages D3 and D5 are capable of infecting multiple species of Dickeya. This conclusion was based on simple spot test assays in which undiluted spots of phage lysate were tested on bacterial top lawns and incubated, with any resultant clearing taken to show infection. It is known that applications of high titre lysates of phage to bacterial cells can cause the phenomenon of \\u201clysis-from-without,\\u201d in which cells lyse due to membrane disruption instead of productive phage infection (Khan Mirzaei and Nilsson, 2015). Consequently, confirmation of host range requires serial dilution of the phage lysate to visualize individual plaques on a host, and this confirmatory data would be helpful when assessing the reported host range of these phages. This reasonable caution is reinforced, particularly when we consider the genome identity of nearly 100 % with other Ackermannviridae family members.\\nThe phages PD10.3 and 23.1 (Czajkowski et al., 2015a) are also reported to infect both Dickeya and Pectobacterium species, although host range was determined by the same method as D3 and D5. However, adsorption and burst size data for both phages are reported on the two genera. The genomes of both phages have been sequenced and are reported as incomplete. Curiously, the largest scaffold of both is similar to the size of other Ackermannviridae family members (shown in Table 1) and these scaffolds share 99% identity with the full genome of LIMEstone1. The morphology of these two phages also clearly places them within the Ackermannviridae family. It is therefore intriguing that phages that are so similar have such different host ranges, and so further confirmatory data on the broader host range of these two phages could be biologically illuminating. Notwithstanding such observations, interpretation of Dickeya phage host range data should be treated with caution. We offer a salutary lesson, based on our own data, reported here, which more rigorously reinterprets previously reported host range data. Tests of the host range of our phages following the method of Czajkowski et al. have also suggested a much broader host range than we now know to be true (data not shown). Consequently, we would caution against assigning host range to phages without rigorous experimental data involving plaque formation in line with the comments of others (Khan Mirzaei and Nilsson, 2015).\\n4. Materials and Methods\\n4.1. Bacterial Strains, Phages, Culture Media, and Growth Conditions\\nAll bacterial strains used in this study are listed in Table 7. Dickeya species were routinely grown at 30\\u00b0C in Luria broth (LB) or on LB agar plates (1.5%, wt/vol, agar). Phages were stored at 4\\u00b0C in phage buffer (10 mM Tris-HCl, pH 7.4, 10 mM MgSO4, 0.01%, wt/vol, gelatine) over a few drops of NaHCO3 saturated chloroform.\\nTable 7. Bacterial strains and bacteriophage genomes used in this study.\\n4.2. Isolation of Phages\\nTreated sewage effluent was collected from a sewage treatment plant in Cambridge, United Kingdom (Matilla and Salmond, 2014). River water was collected from multiple sites along the River Cam. Samples were filter sterilized using a 0.22 \\u03bcm filter before 5 mL of the sample was added to 2x LB along with 500 \\u03bcL of an overnight culture of D. solani MK10. This mixture was incubated overnight in a 250 mL flask at 30\\u00b0C with shaking at 250 rpm. One milliliter of the enriched sample was mixed with 100 \\u03bcL of chloroform (saturated with NaHCO3) and vortexed to lyse bacterial cells. After sedimentation at 16,000 \\u00d7 g for 4 min, 10 \\u03bcL of a serial dilution series of the supernatant was mixed with 200 \\u03bcL of an overnight bacterial culture and 4 mL of LB top agar. This mixture was poured as an overlay on an LBA plate and incubated overnight at 30\\u00b0C. Single phage plaques were picked with a sterile toothpick, placed into 100 \\u03bcL phage buffer, and shaken with 40\\u03bcL of chloroform to kill any bacteria. The phages obtained were plaque purified three times. High-titer phage lysates were then obtained as described previously (Petty et al., 2006). Briefly, 10-fold serial dilutions of the phage were incubated overnight in an agar overlay. Those plates exhibiting confluent lysis (seen as a mosaic-like effect in which the plaques are close to merging) were used for lysate preparation. The top agar was removed from the plate, vortexed with chloroform before sedimentation at 2,200 \\u00d7 g for 20 min at 4\\u00b0C. The supernatant was removed and vortexed with a few drops of chloroform to produce the final lysate.\\n4.3. Determination of Host Range\\nThe host range of isolated phages was determined by plating out ten-fold serial dilutions of the phage lysates onto agar overlays containing host Dickeya cells and incubating overnight at 30\\u00b0C. Following best practice to avoid potential confusion with \\u201clysis from without,\\u201d only phages that produced individual plaques following serial dilution on three independent occasions were considered as being able to infect the respective host productively through a lytic cycle.\\n4.4. Transmission Electron Microscopy\\nHigh-titre lysates for transmission electron microscopy were obtained as described above using 0.35% (w/v) LB agarose instead of 0.35% (w/v) LB agar overlays. Ten \\u03bcL of high-titre phage lysates were adsorbed onto 400-mesh copper grids with holey carbon support films (Agar Scientific, Stansted, United Kingdom) for 2 min. The copper grids were discharged in a Quorum/Emitech K100X system (Quorum, Ringmer, United Kingdom) prior to use. Excess phage suspension was removed with filter paper and phage samples were negatively stained by placing the grids for 30 s in ten \\u03bcL of 2% uranyl acetate neutralized with NaOH. The grids were then blotted on filter paper to remove the excess solution and allowed to air dry. Phages were examined by transmission electron microscopy at Cambridge Advanced Imaging Centre (Department of Physiology, Development and Neuroscience, University of Cambridge) using an FEI Tecnai G2 transmission electron microscope (FEI, OR, USA). The accelerating voltage was 120.0 kV, and images were captured with an AMT XR60B digital camera running Deben software.\\n4.5. Genome Sequencing and Analysis\\nPhage genomes were sequenced on the Illumina MiSeq Sequencer at MicrobesNG (Birmingham, UK). The reads were trimmed using Trimmomatic (Bolger et al., 2014), assessed for quality using BWA-MEM (Li, 2013) and assembled using SPAdes 3.7.1 (Bankevich et al., 2012) with standard settings. Except for JA10 and AD2, all generated over 140,000 reads and had higher than 100x coverage of the full genome. JA10 generated 3,270 reads and had 26x coverage. AD2 generated 1,899 reads and had 4.79x coverage. All assembled into one contig except AD2. Annotation was performed using DNAMaster 5.23.2 (Lawrence, 2012). Genome maps were generated using Circos 0.69.6 (Krzywinski et al., 2009). Genomes were deposited in Genbank using BankIt (NCBI) and are available under accession numbers MH460459 (JA10), MH389777 (JA11), MH460460 (JA13), MH460461 (JA29), MH460462 (JA33), and MH460463 (AD1). Genomes were compared using NCBI Blast, MEGA 7.0.26 (Kumar et al., 2016) and the Artemis Comparison Tool 13.0.0 (Carver et al., 2005). Annotation tables are found in Tables S1\\u2013S6.\\n4.6. Transduction\\nTo test for transduction, phage lysates were generated as described above on donor bacterial strains carrying the desired plasmid or chromosomal marker. All the experiments used kanamycin as the antibiotic for selection. The chromosomal marker for the JA phages was a transposon stably inserted into a protease gene. Successful transduction was therefore confirmed by a protease-negative, kanamycin-resistant phenotype in the recipient cells. The chromosomal marker for the AD phages was a transposon stably inserted into the lacZ gene. Successful transduction was confirmed by kanamycin-resistant recipient colonies that were white on media containing X-gal. The plasmid marker was pECA1039-Km3 and successful transduction was confirmed by plasmid extraction and gel electrophoresis under standard conditions as described previously (Day et al., 2017).\\nTransduction was performed by mixing phage lysate with an overnight culture of the recipient cells to achieve a multiplicity of infection of 0.01, meaning that for each phage there were one hundred bacterial cells, apart from the transduction of pECA1039-Km3 into Dickeya dadantii subsp. dieffenbachiae, which required an MOI of 0.1. The mixture was left on the lab bench at room temperature for 20 min, followed by incubation on a rotary wheel at 30\\u00b0C for 30 min. The infected culture was then centrifuged and the bacterial pellets washed with LB twice to eliminate any remaining non-adsorbed phage. The bacterial pellets were resuspended in 1 mL LB and 100 \\u03bcL aliquots were spread onto LBA plates with selection for the chromosomal or plasmid marker. Appropriate standard controls, which were routinely negative, were used to score for any spontaneous resistance of the recipient strain. One hundred microliters of the phage lysate was also spread onto LBA plates to confirm lysate sterility.\\nAuthor Contributions\\nAD, JA, and GPCS conceived and designed the experiments, analyzed the data. AD and JA performed the experiments. AD wrote the paper. AD and GPCS edited the paper.\\nFunding\\nThis work was supported by the BBSRC, UK. AD was supported by a Cambridge Doctoral Training Partnership Award from the BBSRC, UK.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nSequencing was conducted by MicrobesNG (http://www.microbesng.uk), which is supported by the BBSRC (grant number BB/L024209/1). Microscopy was performed at Cambridge Advanced Imaging Centre with the help of Karin M\\u00fcller and Lyn Carter. We are grateful to Sonia Humphris and Ian Toth (James Hutton Institute, Scotland) for generous provision of Dickeya strains. This work was done under DEFRA license: 50864/197900/3. We thank Alison Rawlinson, Sarah Barker, and Diana Breitmaier for technical support and Rita Monson for helpful advice.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2018.02169/full#supplementary-material\\nReferences\\nAckermann, H.-W. (2012). \\u201cBacteriophage electron microscopy,\\u201d in Advances in Virus Research, eds M. Lobocka and W. T. Szybalski (Elsevier), 1\\u201332.\\nGoogle Scholar\\nAckermann, H. W., and Nguyen, T. M. (1983). Sewage coliphages studied by electron microscopy. Appl. Environ. Microbiol. 45, 1049\\u201359.\\nPubMed Abstract | Google Scholar\\nAdeolu, M., Alnajar, S., Naushad, S., and Gupta, R. S. (2016). Genome-based phylogeny and taxonomy of the \\u201cEnterobacteriales:\\u201d proposal for Enterobacterales ord. nov. divided into the families Enterobacteriaceae, Erwiniaceae fam. nov., Pectobacteriaceae fam. nov., Yersiniaceae fam. nov., Hafniaceae fam. nov., Morgane. Int. J. Syst. Evol. Microbiol. 66, 5575\\u20135599. doi: 10.1099/ijsem.0.001485\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAdriaenssens, E. M., Ackermann, H. W., Anany, H., Blasdel, B., Connerton, I. F., Goulding, D., et al. (2012a). A suggested new bacteriophage genus: \\u201cViunalikevirus.\\u201d Arch. Virol. 157, 2035\\u20132046. doi: 10.1007/s00705-012-1360-5\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAdriaenssens, E. M., Mattheus, W., Cornelissen, A., Shaburova, O., Krylov, V. N., Kropinski, A. M., et al. (2012b). Complete genome sequence of the giant Pseudomonas phage Lu11. J. Virol. 86, 6369\\u20136370. doi: 10.1128/JVI.00641-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAdriaenssens, E. M., Van Vaerenbergh, J., Vandenheuvel, D., Dunon, V., Ceyssens, P. J., De Proft, M., et al. (2012c). T4-related bacteriophage LIMEstone isolates for the control of soft rot on potato caused by \\u201cDickeya solani.\\u201d PLoS ONE 7:et002. doi: 10.1371/journal.pone.0033227.t002\\nCrossRef Full Text | Google Scholar\\nAdriaenssens, E. M., Wittmann, J., Kuhn, J. H., Turner, D., Sullivan, M. B., Dutilh, B. E., et al. (2018). Taxonomy of prokaryotic viruses: 2017 update from the ICTV Bacterial and Archaeal Viruses Subcommittee. Arch. Virol. 163, 1125\\u20131129. doi: 10.1007/s00705-018-3723-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAlic, \\u0160., Nagli\\u010d, T., Tu\\u0161ek-\\u017dnidari\\u010d, M., Peterka, M., Ravnikar, M., and Dreo, T. (2017a). Putative new species of the genus Dickeya as major soft rot pathogens in Phalaenopsis orchid production. Plant Pathol. 66, 1357\\u20131368. doi: 10.1111/ppa.12677\\nCrossRef Full Text | Google Scholar\\nAli\\u010d, \\u0160., Nagli\\u010d, T., Tu\\u0161ek-\\u017dnidari\\u010d, M., Ravnikar, M., Ra\\u010dki, N., Peterka, M., etal. (2017b). Newly isolated bacteriophages from the Podoviridae, Siphoviridae, and Myoviridae families have variable effects on putative novel Dickeya spp. Front. Microbiol. 8:1870. doi: 10.3389/fmicb.2017.01870\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455\\u2013477. doi: 10.1089/cmb.2012.0021\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nButtimer, C., Hendrix, H., Oliveira, H., Casey, A., Neve, H., McAuliffe, O., et al. (2017). Things are getting hairy: enterobacteria bacteriophage vB_PcaM_CBB. Front. Microbiol. 8:44. doi: 10.3389/fmicb.2017.00044\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nButtimer, C., Born, Y., Lucid, A., Loessner, M. J., Fieseler, L., and Coffey, A. (in press). Erwinia amylovora phage vB_EamM_Y3 represents another lineage of hairy Myoviridae. Res. Microbiol. doi: 10.1016/j.resmic.2018.04.006\\nPubMed Abstract | CrossRef Full Text\\nCahill, G., Fraser, K., Kowalewska, M. J., Kenyon, D. M., and Saddler, G. S. (2010). \\u201cRecent findings from the Dickeya survey and monitoring programme,\\u201d in Proceedings Crop Protection in Northern Britain (Edinburgh), 171\\u2013176.\\nGoogle Scholar\\nCarver, T. J., Rutherford, K. M., Berriman, M., Rajandream, M.-A., Barrell, B. G., and Parkhill, J. (2005). ACT: the artemis comparison tool. Bioinformatics 21, 3422\\u20133423. doi: 10.1093/bioinformatics/bti553\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChibani-Chennoufi, S., Bruttin, A., Dillmann, M.-L., and Brussow, H. (2004). Phage-host interaction: an ecological perspective. J. Bacteriol. 186, 3677\\u20133686. doi: 10.1128/JB.186.12.3677-3686.2004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCianfanelli, F. R., Alcoforado Diniz, J., Guo, M., De Cesare, V., Trost, M., and Coulthurst, S. J. (2016). VgrG and PAAR proteins define distinct versions of a functional type VI secretion system. PLOS Pathogens 12:e1005735. doi: 10.1371/journal.ppat.1005735\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCzajkowski, R., Ozymko, Z., de Jager, V., Siwinska, J., Smolarska, A., Ossowicki, A., et al. (2015a). Genomic, proteomic and morphological characterization of two novel broad host lytic bacteriophages \\u03a6PD10.3 and \\u03a6PD23.1 infecting pectinolytic Pectobacterium spp. and Dickeya spp. PloS ONE 10:e0119812. doi: 10.1371/journal.pone.0119812\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCzajkowski, R., Ozymko, Z., and Lojkowska, E. (2014a). Isolation and characterization of novel soilborne lytic bacteriophages infecting Dickeya spp. biovar 3 (\\u201cD. solani\\u201d). Plant Pathol. 63, 758\\u2013772. doi: 10.1111/ppa.12157\\nCrossRef Full Text | Google Scholar\\nCzajkowski, R., Ozymko, Z., Siwinska, J., Ossowicki, A., de Jager, V., Narajczyk, M., et al. (2015b). The complete genome, structural proteome, comparative genomics and phylogenetic analysis of a broad host lytic bacteriophage phiD3 infecting pectinolytic Dickeya spp. Stand. Genomic Sci. 10:68. doi: 10.1186/s40793-015-0068-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCzajkowski, R., Ozymko, Z., Zwirowski, S., and Lojkowska, E. (2014b). Complete genome sequence of a broad-host-range lytic Dickeya spp. bacteriophage \\u03a6D5. Arch. Virol. 159, 3153\\u20133155. doi: 10.1007/s00705-014-2170-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCzajkowski, R., Smolarska, A., and Ozymko, Z. (2017). The viability of lytic bacteriophage \\u03a6D5 in potato-associated environments and its effect on Dickeya solani in potato (Solanum tuberosum L.) plants. PLoS ONE 12:e0183200. doi: 10.1371/journal.pone.0183200\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDay, A., Ahn, J., Fang, X., and Salmond, G. P. C. (2017). Environmental bacteriophages of the emerging enterobacterial Phytopathogen, Dickeya solani, show genomic conservation and capacity for horizontal gene transfer between their bacterial hosts. Front. Microbiol. 8:1654. doi: 10.3389/fmicb.2017.01654\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nEsplin, I. N. D., Berg, J. A., Sharma, R., Allen, R. C., Arens, D. K., Ashcroft, C. R., et al. (2017). Genome sequences of 19 novel Erwinia amylovora bacteriophages. Genome Announc. 5:e00931\\u201317. doi: 10.1128/genomeA.00931-17\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nIriarte, F. B., Balogh, B., Momol, M. T., Smith, L. M., Wilson, M., and Jones, J. B. (2007). Factors affecting survival of bacteriophage on tomato leaf surfaces. Appl. Environ. Microbiol. 73, 1704\\u20131711. doi: 10.1128/AEM.02118-06\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKabanova, A., Shneider, M., Bugaeva, E., Ha, V. T. N., Miroshnikov, K., Korzhenkov, A., et al. (2018). Genomic characteristics of vB_PpaP_PP74, a T7-like Autographivirinae bacteriophage infecting a potato pathogen of the newly proposed species Pectobacterium parmentieri. Arch. Virol. 163, 1691\\u20131694. doi: 10.1007/s00705-018-3766-1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKhan Mirzaei, M., and Nilsson, A. S. (2015). Isolation of phages for phage therapy: a comparison of spot tests and efficiency of plating analyses for determination of host range and efficacy. PLoS ONE 10:e0118557. doi: 10.1371/journal.pone.0118557\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKrzywinski, M., Schein, J., Birol, I., Connors, J., Gascoyne, R., Horsman, D., et al. (2009). Circos: an information aesthetic for comparative genomics. Genome Res. 19, 1639\\u20131645. doi: 10.1101/gr.092759.109\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKumar, S., Stecher, G., and Tamura, K. (2016). MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets. Mol. Biol. Evol. 33:msw054. doi: 10.1093/molbev/msw054\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLaurila, J., Ahola, V., Lehtinen, A., Joutsjoki, T., Hannukkala, A., Rahkonen, A., et al. (2008). Characterization of Dickeya strains isolated from potato and river water samples in Finland. Eur. J. Plant Pathol. 122, 213\\u2013225. doi: 10.1007/s10658-008-9274-5\\nCrossRef Full Text | Google Scholar\\nLi, H. (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint arXiv.\\nGoogle Scholar\\nMansfield, J., Genin, S., Magori, S., Citovsky, V., Sriariyanum, M., Ronald, P., et al. (2012). Top 10 plant pathogenic bacteria in molecular plant pathology. Mol. Plant Pathol. 13, 614\\u201329. doi: 10.1111/j.1364-3703.2012.00804.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMatilla, M. A., Fang, X., and Salmond, G. P. C. (2014). Viunalikeviruses are environmentally common agents of horizontal gene transfer in pathogens and biocontrol bacteria. ISME J. 8, 2143\\u20132147. doi: 10.1038/ismej.2014.150\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMatilla, M. A., and Salmond, G. P. C. (2014). Bacteriophage phiMAM1, a viunalikevirus, is a broad-host-range, high-efficiency generalized transducer that infects environmental and clinical isolates of the enterobacterial genera Serratia and Kluyvera. Appl. Environ. Microbiol. 80, 6446\\u20136457. doi: 10.1128/AEM.01546-14\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nParkinson, N., DeVos, P., Pirhonen, M., and Elphinstone, J. (2014). Dickeya aquatica sp. nov., isolated from waterways. Int. J. Syst. Evol. Microbiol. 64(Pt 7), 2264\\u20132266. doi: 10.1099/ijs.0.058693-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nParkinson, N., Stead, D., Bew, J., Heeney, J., Tsror, L., and Elphinstone, J. (2009). Dickeya species relatedness and clade structure determined by comparison of recA sequences. Int. J. Syst. Evol. Microbiol. 59, 2388\\u20132393. doi: 10.1099/ijs.0.009258-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPelfrene, E., Willebrand, E., Cavaleiro Sanches, A., Sebris, Z., and Cavaleri, M. (2016). Bacteriophage therapy: a regulatory perspective. J. Antimicrob. Chemother. 71, 2071\\u20132074. doi: 10.1093/jac/dkw083\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPetty, N. K., Foulds, I. J., Pradel, E., Ewbank, J. J., and Salmond, G. P. C. (2006). A generalized transducing phage (phiIF3) for the genomically sequenced Serratia marcescens strain Db11: a tool for functional genomics of an opportunistic human pathogen. Microbiology 152, 1701\\u20131708. doi: 10.1099/mic.0.28712-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPfreundt, U., Spungin, D., Hou, S., Vo\\u00df, B., Berman-Frank, I., and Hess, W. R. (2017). Genome of a giant bacteriophage from a decaying Trichodesmium bloom. Mar. Genomics 33, 21\\u201325. doi: 10.1016/j.margen.2017.02.001\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPritchard, L., Humphris, S., Baeyen, S., Maes, M., Van Vaerenbergh, J., Elphinstone, J., et al. (2013a). Draft genome sequences of four Dickeya dianthicola and four Dickeya solani strains. Genome Announc. 1:e00087\\u201312. doi: 10.1128/genomeA.00087-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPritchard, L., Humphris, S., Saddler, G. S., Elphinstone, J. G., Pirhonen, M., and Toth, I. K. (2013b). Draft genome sequences of 17 isolates of the plant pathogenic bacterium Dickeya. Genome Announc. 1:e00978\\u201313. doi: 10.1128/genomeA.00978-13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nS\\u0142awiak, M., van Beckhoven, J. R. C. M., Speksnijder, A. G. C. L., Czajkowski, R., Grabe, G., and van der Wolf, J. M. (2009). Biochemical and genetical analysis reveal a new clade of biovar 3 Dickeya spp. strains isolated from potato in Europe. Eur. J. Plant Pathol. 125, 245\\u2013261. doi: 10.1007/s10658-009-9479-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSykilinda, N. N., Bondar, A. A., Gorshkova, A. S., Kurochkina, L. P., Kulikov, E. E., Shneider, M. M., et al. (2014). Complete genome sequence of the novel giant Pseudomonas phage PaBG. Genome Announc. 2:e00929\\u201313. doi: 10.1128/genomeA.00929-13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nToth, I. K., Cahill, G., Elphinstone, J. G., Humphris, S., and Wale, S. J. (2016). \\u201cAn update on the Potato Council/Scottish government-funded blackleg project - Year 2,\\u201d in Proceedings Crop Protection in Northern Britain 2016 (Edinburgh), 203\\u2013204.\\nGoogle Scholar\\nToth, I. K., van der Wolf, J. M., Saddler, G., Lojkowska, E., H\\u00e9lias, V., Pirhonen, M., et al. (2011). Dickeya species: an emerging problem for potato production in Europe. Plant Pathol. 60, 385\\u2013399. doi: 10.1111/j.1365-3059.2011.02427.x\\nCrossRef Full Text | Google Scholar\\nTsror, L., Erlich, O., Lebiush, S., Hazanovsky, M., Zig, U., Slawiak, M., et al. (2009). Assessment of recent outbreaks of Dickeya sp. (syn. Erwinia chrysanthemi) slow wilt in potato crops in Israel. Eur. J. Plant Pathol. 123, 311\\u2013320. doi: 10.1007/s10658-008-9368-0\\nCrossRef Full Text | Google Scholar\\nvan der Wolf, J. M., Nijhuis, E. H., Kowalewska, M. J., Saddler, G. S., Parkinson, N., Elphinstone, J. G., et al. (2014). Dickeya solani sp. nov., a pectinolytic plant-pathogenic bacterium isolated from potato (Solanum tuberosum). Int. J. Syst. Evol. Microbiol. 64(Pt 3), 768\\u2013774. doi: 10.1099/ijs.0.052944-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYamada, T., Satoh, S., Ishikawa, H., Fujiwara, A., Kawasaki, T., Fujie, M., et al. (2010). A jumbo phage infecting the phytopathogen Ralstonia solanacearum defines a new lineage of the Myoviridae family. Virology 398, 135\\u2013147. doi: 10.1016/j.virol.2009.11.043\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01654\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01654/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:50:28.875Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.01654\",\n",
      "            \"title\": \"Frontiers | Environmental Bacteriophages of the Emerging Enterobacterial Phytopathogen, Dickeya solani, Show Genomic Conservation and Capacity for Horizontal Gene Transfer between Their Bacterial Hosts\",\n",
      "            \"description\": \"Dickeya solani is an economically important phytopathogen widespread in mainland Europe that can reduce potato crop yields by 25%. There are no effective environmentally-acceptable chemical systems available for diseases caused by Dickeya. Bacteriophages have been suggested for use in biocontrol of this pathogen in the field, and limited field trials have been conducted. To date only a small number of bacteriophages capable of infecting Dickeya solani have been isolated and characterised, and so there is a need to expand the repertoire of phages that may have potential utility in phage therapy strategies. Here we describe 67 bacteriophages from environmental sources, the majority of which are members of the viral family Myoviridae. Full genomic sequencing of two isolates revealed a high degree of DNA identity with Dickeya solani bacteriophages isolated in Europe in the past five years, suggesting a wide ecological distribution of this phage family. Transduction experiments showed that the majority of the new environmental bacteriophages are capable of facilitating efficient horizontal gene transfer. The possible risk of unintentional transfer of virulence or antibiotic resistance genes between hosts susceptible to transducing phages cautions against their environmental use for biocontrol, until specific phages are fully tested for transduction capabilities.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"1. Introduction\\nThe enterobacterial genus, Dickeya, currently consists of six phytopathogenic species that can cause severe disease in economically important crops, including tomato, chicory, and potato (Reverchon and Nasser, 2013). The first report of Dickeya (previously known as Erwinia chrysanthemi) infecting European potatoes came from the Netherlands in the 1970s (Maas Geesteranus HP, 1972). Until 2004, almost all European potato isolates of Dickeya were assigned as Dickeya dianthicola, which has a broad host range across both nutritional and ornamental species (Toth et al., 2011). In the past 10 years however, three groups have independently identified a new clade of Dickeya in European potato isolates (Laurila et al., 2008; Parkinson et al., 2009; S\\u0142awiak et al., 2009). In 2014 this led to the classification of a new species; Dickeya solani (van der Wolf et al., 2014).\\nDickeya solani is more aggressive than other Dickeya species, able to spread more easily through the plant vascular system and survive at higher temperatures than D. dianthicola (Toth et al., 2011). It is currently the predominant potato pathogen in Europe and in 2010 Scotland became the first country to introduce specific legislation aimed at preventing the establishment of D. solani in its seed industry (Mansfield et al., 2012).\\nIn Israel a reduction in yield of up to 25% was observed in potatoes exposed to Dickeya species (Tsror et al., 2009). This imposes a significant economic cost, and consequently has led to research into methods for control of these virulent phytopathogens. In the absence of any effective chemical control systems, bacterial viruses (bacteriophages; phages) have been suggested as potential biocontrol tools and several studies have isolated phages capable of infecting Dickeya species (Adriaenssens et al., 2012b; Czajkowski et al., 2014, 2015; Matilla et al., 2014). Their potential use as biocontrol agents has been trialed both in the lab and in the field (Adriaenssens et al., 2012b) and these studies showed a \\u201ctherapeutic\\u201d outcome with an increase in yield of the potato crop. Because of the potential utility of specific phages as therapeutic agents in potato soft rot control experiments, there is value in investigating a wider range of Dickeya phages. However, prior work has shown that a previously isolated D. solani phage is capable of generalized transduction of both chromosomal and plasmid markers (Matilla et al., 2014). The European Medicines Agency, among others, has stated that it is \\u201cimportant to ensure that therapeutic phages do not carry out generalized transduction\\u201d (Pelfrene et al., 2016), and therefore this is an important consideration as some Dickeya phages may not have been fully tested for generalized transduction capacity before field trials. This study therefore aimed to isolate and characterize a larger repertoire of new environmental phages against D. solani and investigate their potential for generalized transduction.\\n2. Results\\n2.1. Isolation and Classification\\nSixty-seven phages were isolated using standard enrichment techniques from both treated sewage effluent and river water between 2013 and 2015 using D. solani MK10 as the host organism. Transmission electron microscopy (TEM) showed two different morphological groups, a selection of which are shown in Figure 1 alongside the previously characterized phage LIMEstone1 (Adriaenssens et al., 2012b).\\nFigure 1. Transmission electron micrographs of four Dickeya phages. LIMEstone1 is a previously characterized Dickeya phage (Adriaenssens et al., 2012b). \\u03d5JA15 and \\u03d5XF4 are phages of the Myoviridae family exhibiting tail spikes as opposed to tail fibers. \\u03d5XF28 is a phage of the Podoviridae family. Scale bars represent 100 nm.\\nOf 24 phages imaged, all possessed an icosahedral head and a tail, placing them in the order Caudovirales. Three possessed short tails, characteristic of the family Podoviridae (such as \\u03d5XF28 in the last panel of Figure 1) whilst the rest possessed longer contractile tails and belong to the family Myoviridae. The 21 Myoviridae members did not appear to possess the tail fibers characteristic of the family. Instead, short tail spikes were observed, (first three panels of Figure 1), and these are generally associated with the family Podoviridae.\\n2.2. Transduction\\nOther Dickeya phages with similar morphology have been described and were shown to be efficient generalized transducing phages (Matilla et al., 2014). Due to the transduction capability of certain phages shown by Matilla et al., the 67 newly-isolated phages were also tested for ability to affect horizontal gene transfer. Of these, 51 (including the 21 phages with the non-classical morphology) proved capable of transducing chromosomal markers. Twelve of the isolates, all of which had the non-classical morphology, were also tested for generalized transduction, and proved capable of transferring plasmids between Dickeya species. The results of transduction of the plasmid pBR322 by three of these phages are shown in Figure 2.\\nFigure 2. pBR322 was first used to transform Dickeya solani MK10 before the creation of lysates of \\u03d5JA13, \\u03d5JA29, or \\u03d5JA37 on the recombinant host. These lysates were then used to infect Dickeya solani, D. dieffenbachiae, or D. zeae as appropriate and transductants selected on LB agar containing ampicillin. The plasmids from these transductants were extracted and analyzed by gel electrophoresis. Co-migration of the plasmid DNA samples and absence of the plasmid from the wild type (WT) controls confirmed successful plasmid transduction.\\n2.3. Host Range\\nBased on the bacterial strains tested, LIMEstone1 is capable of forming plaques on strains of D. solani only (Adriaenssens et al., 2012b). The phages isolated during this study were tested against a variety of Dickeya strains, listed in Table 1, to determine their host range. The majority of the phages presented here exhibited the same host range as LIMEstone1 and were only capable of forming plaques on D. solani strains but not isolated representatives of other Dickeya species used in this study. However, eight of the phage isolates had a wider host range extending to species such as Dickeya dieffenbachiae, Dickeya paradisiaca, and Dickeya zeae (Table 2).\\nTable 1. Bacterial strains, bacteriophages, and primers used in this study.\\nTable 2. Extended host range of three groups of the isolated phages capable of infecting other species of Dickeya.\\n2.4. Genetic Comparisons\\nTwo key genes known to be conserved between these phages, those for DNA polymerase (DNAP) and tail spike protein 1 (TSP1), were sequenced for several of the newly-isolated phages. These nucleotide sequences were then compared to those of LIMEstone1 as shown in Table 3. All of the phages in Table 3 were able to form plaques on D. solani. The corresponding amino acid sequences were compared between these phages and phylogenetic trees were created as shown in Figure 3 (DNAP) and Figure 4 (TSP1). These show that, in agreement with Table 3, the DNAP genes of \\u03d5XF4 and \\u03d5XF11 grouped together away from the other phages with a branch length of 0.053. The other phages formed two clusters that differed in a single amino acid. The TSP1 genes formed two distinct clusters, with the genes from \\u03d5XF16 and \\u03d5JA1 forming their own cluster with a branch length of 0.092, whereas the other phages all had identical TSP1 genes. The final 20 amino acids were trimmed from the TSP1 genes as the sequencing data for some of the phages was insufficient for tree construction.\\nTable 3. Nucleotide comparison of two conserved genes between the previously characterized LIMEstone1 and a selection of isolated phages.\\nFigure 3. Phylogenetic tree of the DNA Polymerase (DNAP) gene from 10 phages of Dickeya solani. Tree was constructed using the Maximum Likelihood method with 1,134 positions in the final dataset, and the tree shown has the highest log likelihood (\\u22122033.82).\\nFigure 4. Phylogenetic tree of the Tail Spike Protein 1 (TSP1) gene from 10 phages of Dickeya solani. Tree was constructed using the Maximum Likelihood method with 1,593 positions in the final dataset, and the tree shown has the highest log likelihood (\\u22123105.91). The final 20 amino acids of the TSP1 gene were trimmed from the alignment as the sequencing data for some of the phages was insufficient.\\n2.5. Genomic Sequencing of Two New Dickeya solani Phages\\n\\u03d5XF4 and \\u03d5JA15 were isolated over a year apart yet they shared the same host range and PCR amplification and preliminary sequence analysis showed 100% DNA identity in the TSP1 genes, although they differ in their DNAP genes. The full genomes of both phages were then sequenced. Both consist of circular double-stranded DNA of 151,519 and 153,757 bp, respectively. The genome of \\u03d5XF4 has a G+C content of 49.4% and contained 185 predicted genes with lengths ranging between 116 and 4,838 nucleotides, as shown in Figure 5. \\u03d5JA15 has a G+C content of 49.2% and contained 188 predicted genes of lengths between 122 and 4,838 nucleotides, as shown in Figure 6. Full annotation tables for the two genomes can be found in Tables S1, S2, respectively.\\nFigure 5. Map of the genome of \\u03d5XF4. The outer gray ring marks open reading frames whilst the middle ring categorizes the proposed functions of these genes. The inner ring highlights the areas of the \\u03d5XF4 genome that differ from the genome of the LIMEstone1 phage. The genome map was generated using Circos.\\nFigure 6. Map of the genome of \\u03d5JA15. The outer gray ring marks open reading frames whilst the middle ring categorizes the proposed functions of these genes. The inner ring highlights the areas of the \\u03d5JA15 genome that differ from the genome of the LIMEstone1 phage. The genome map was generated using Circos.\\n2.6. Genomic Comparison\\nThe genomes of the two new phages shared 97% DNA identity, with the main areas of difference being regions encoding endonucleases and hypothetical proteins. A comparison of both of these phages with the previously published phage LIMEstone1, showed over 95% DNA identity, with the major areas of difference consisting of genes thought to be involved in DNA replication (such as homing endonucleases and polymerases) as well as introns located throughout all three genomes. These areas of difference are highlighted in Figures 5, 6.\\n3. Discussion\\nThe Scottish government tests all seed crops imported from outside Scotland plus 10% of Scottish-origin crops for D. solani and did not find any positive samples in 2016 (Scottish Government, 2016). These data are consistent with a view that D. solani is not yet environmentally widespread within the UK, although there have been isolated cases of D. solani reported in England and Wales since 2007 (Cahill et al., 2010) in crops originating from outside of the UK (Toth et al., 2016). The relative ease with which we have isolated environmental phages that infect D. solani therefore seems counter-intuitive given the reported paucity of the pathogen in the environment. In a restricted host range screen these phages did not form plaques on eight isolates of other Gram-negative laboratory strains such as Pectobacterium carotovorum, Pectobacterium atrosepticum, Serratia plymuthica, Serratia marcescens, Escherichia coli, and Pantoea agglomerans (data not shown). The apparently contradictory observations from phage isolations and host distribution beg the ecological question as to why D. solani phages are easy to find by simple enrichments. We would suggest that either D. solani is present in the environment around Cambridge and is not being detected, or that there is an, as yet unknown, alternative host(s) for these phages present in the environment.\\nAll but three of the phages imaged by TEM were morphologically characterized as Myoviridae due to the presence of an icosahedral head and a contractile tail. Classical Myoviridae members, such as the coliphage T4 possess long slender tail fibers attached to the baseplate that participate in adsorption of the phage to the bacterial host. The imaged phages do not appear to have tail fibers, but instead possess shorter, clustered structures more akin to the tail spikes present in members of the Podoviridae. Genome analysis of the phages \\u03d5XF4 and \\u03d5JA15 showed genes encoding potential tail spike proteins, which show 100% sequence identity to putative tail spike protein genes in LIMEstone1. This combination of a Myoviridae-like morphology with tail spikes has been reported previously as a feature of the novel viral genus termed viunalikevirus (Adriaenssens et al., 2012a), named after the ViI Salmonella typing phage, and includes virulent phages capable of infecting a wide range of Gram-negative hosts. Members of the genus share a high degree of genome order and identity, with the major region of divergence being the genes encoding the tail spike proteins. Although, we cannot state that all phages that exhibit this morphology are definitively members of the viunalikevirus genus, we conclude it is likely. This does pose the question of whether there is some particular connection between D. solani and viunalikeviruses, or whether the environment around Cambridge is a particularly abundant source of this genus of phages.\\nA comparison of two genes from a subset of the phages isolated here, along with LIMEstone1, showed that, whilst there were some differences in DNA polymerase and TSP1 genes, these did not translate into a difference in host range and that in general these phages are highly similar. Several D. solani phages have now been isolated from environmental sources, including the LIMEstone phages (Adriaenssens et al., 2012b) as well as \\u03d5D3 (Czajkowski et al., 2015) and \\u03d5D5 (Czajkowski et al., 2014). They are all remarkably similar upon comparison of their genomes, and the two phages discussed in this study show 95% DNA identity with LIMEstone1. Much of the variation comes within the many introns and homing endonucleases found throughout the genomes of the phages, whereas structural elements are largely conserved. These introns and homing endonucleases vary in their sequence, but their position within the genomes of the phages, and thus the gene order, remains the same. It is interesting that these phages have been isolated independently in countries across Europe, and even from different environments; soil (LIMEstone1 and 2, \\u03d5D3, and \\u03d5D5) and water (\\u03d5XF4 and \\u03d5JA15), and yet they share such conservation despite their wide geographical separation.\\nThe lytic activity of these phages, coupled with the high economic burden of D. solani crop phytopathogenesis, have highlighted the D. solani phages as potential biocontrol agents, and limited field trials have been performed (Adriaenssens et al., 2012b). Nevertheless, although these phages may have potential phage therapy features, we have shown that, 51 were capable of effecting horizontal gene transfer. We showed previously that eight candidate viunalikeviruses (including the LIMEstone phages and \\u03d5XF4) were efficient generalized transducers of plasmid markers (Matilla et al., 2014). Consequently, we proposed that generalized transduction capacity is a characteristic trait of the viunalikevirus genus. This feature is important if these virulent phages are to be used therapeutically, as there could be potential (albeit low) for collateral transfer of bacterial virulence genes or drug resistance plasmids into unintended hosts, with unknown consequences\\u2014depending on ecological selection pressures. We therefore consider it prudent to caution against further field trials until the specific phage(s) involved are fully tested for generalized transduction capabilities.\\n4. Materials and Methods\\n4.1. Bacterial Strains, Phages, Culture Media, and Growth Conditions\\nAll bacterial strains used in this study are listed in Table 1. Dickeya species were routinely grown at 30\\u00b0C in Luria broth (LB) or on LB agar plates (1.5%, wt/vol, agar). Phages were stored at 4\\u00b0C in phage buffer (10 mM Tris-HCl, pH 7.4, 10 mM MgSO4, 0.01%, wt/vol, gelatin) over a few drops of NaHCO3\\u2212 saturated chloroform.\\n4.2. Isolation of Phages\\nTreated sewage effluent was collected from a sewage treatment plant in Cambridge, United Kingdom (Matilla and Salmond, 2014). River water was collected from multiple sites along the River Cam. Samples were filter sterilized before 5 mL of the sample was added to 2x LB along with 500 \\u03bcL of an overnight culture of D. solani MK10. This mixture was incubated overnight in a 250 mL flask at 30\\u00b0C with shaking at 250 rpm. One milliliter of the enriched sample was mixed with 100 \\u03bcL of chloroform (saturated with sodium hydrogen carbonate) and vortexed to lyse bacterial cells. The sample was centrifuged at 16,000 x g for 4 min and 10 \\u03bcL of a serial dilution series of the supernatant was mixed with 200 \\u03bcL of an overnight bacterial culture and 4 ml of LB top agar. This mixture was poured as an overlay on an LBA plate and incubated overnight at 30\\u00b0C. Single phage plaques were picked with a sterile toothpick, placed into 100 \\u03bcL phage buffer, and shaken with 40 \\u03bcL of chloroform to kill any bacteria. The phages obtained were plaque purified three times. High-titre phage lysates were then obtained as described previously (Petty et al., 2006). Briefly, 10-fold serial dilutions of the phage were incubated overnight in an agar overlay as already described. Those plates exhibiting confluent lysis (seen as a mosaic-like effect in which the plaques are close to merging) were used for lysate preparation. The top agar was removed from the plate, vortexed with chloroform before sedimentation at 2,200 x g for 20 min at 4\\u00b0C. The supernatant was removed and vortexed with a few drops of chloroform to produce the final lysate.\\n4.3. Transmission Electron Microscopy\\nHigh-titre lysates for transmission electron microscopy were obtained as described previously (Petty et al., 2006) using 0.35% (w/v) LB agarose instead of 0.35% (w/v) LB agar overlays. Ten microliters of high-titer phage lysates were adsorbed onto 400-mesh copper grids with holey carbon support films (Agar Scientific, Stansted, United Kingdom) for 30 min. The copper grids were discharged in a Quorum/Emitech K100X system (Quorum, Ringmer, United Kingdom) prior to use. After 1 min, excess phage suspension was removed with filter paper and phage samples were negatively stained by placing the grids for 5 min in 10 \\u03bcL of 2% phosphotungstic acid (PTA) neutralized with sodium hydroxide, or with 10 \\u03bcL of 2% uranyl acetate for 2 min. The grids were then blotted on filter paper to remove the excess solution and allowed to air dry for 10 min. Phages were examined by transmission electron microscopy in the Multi-Imaging Centre (Department of Physiology, Development and Neuroscience, University of Cambridge) using an FEI Tecnai G2 transmission electron microscope (FEI, OR, USA). The accelerating voltage was 120.0 kV, and images were captured with anAMT XR60B digital camera running Deben software.\\n4.4. Determination of Host Range\\nThe host range of isolated phages was determined by plating out 10-fold serial dilutions of the phage lysates, onto agar overlays containing the six species of Dickeya listed in Table 1. To avoid potential confusion with \\u201clysis from without,\\u201d only phages that produced lysis at low dilution and individual plaques were considered as being able to infect the host.\\n4.5. Transduction\\nTo test for transduction, phage lysates were generated on donor bacterial strains carrying the desired plasmid or chromosomal marker as already described. Transduction was performed by mixing phage lysate with an overnight culture of the recipient host to achieve a multiplicity of infection of 0.1, meaning that for each phage there were 10 bacterial cells. The mixture was left on the lab bench at room temperature for 20 min, followed by incubation on a rotary wheel at 30\\u00b0C for 30 min. The infected culture was then centrifuged and the bacterial pellets washed with LB twice to eliminate any remaining non-adsorbed phage. The bacterial pellets were resuspended in 1 mL LB and 100 \\u03bcL aliquots were spread onto LBA plates with drug selection for the chromosomal or plasmid marker. Appropriate standard controls, which were routinely negative, were used to score for any spontaneous resistance of the recipient strain. One hundred microliters of the phage lysate was also spread onto LBA plates to confirm lysate sterility.\\n4.6. Gene Amplification and Analysis\\nGenomic DNA was extracted using Phase Lock Gel tubes (5 Prime, Hamburg, Germany) following manufacturer's instructions for isolation of Lambda DNA. DNA Polymerase (DNAP) and Tail Spike Protein 1 (TSP1) genes were amplified using Phusion polymerase (ThermoScientific, MA, USA) following standard protocols. TSP1 was amplified using the primers oJA1 and oJA2, and DNAP by the primers oJA14 and oJA15, listed in Table 1. PCR products were sequenced by GATC Biotech AG (Konstanz, Germany). Sequences were compared using NCBI Blast and the Artemis Comparison Tool (Carver et al., 2005). Phylogenetic trees were constructed using MEGA 7.0.26 (Kumar et al., 2016).\\n4.7. Genome Sequencing and Analysis\\n\\u03d5XF4 was sequenced on the Illumina Bench Top MiSeq Sequencer (Illumina, CA, USA) at the DNA Sequencing Facility, Department of Biochemistry, University of Cambridge, UK. The resulting 138,803 reads were trimmed, quality assessed and assembled using Geneious 7.1.5 (Biomatters Ltd.), leading to higher than 100x coverage of the full genome. Gaps or single nucleotide polymorphisms were further filled or verified by Sanger sequencing to produce one contig. \\u03d5JA15 was sequenced on the Illumina MiSeq Sequencer at MicrobesNG (Birmingham, UK). The 454,086 reads were trimmed using Trimmomatic (Bolger et al., 2014), assessed for quality using BWA-MEM (Li, 2013) and assembled using SPAdes 3.7.1 (Bankevich et al., 2012) with standard settings, leading to higher than 100x coverage of the full genome and producing one contig. Annotation of both genomes was performed using Prokka 1.11 (Seemann, 2014) using standard settings and LIMEstone1 (NC_019925.1) as a scaffold. Genome maps were generated using Circos (Krzywinski et al., 2009). Genomes were deposited in Genbank using Sequin (NCBI) and are available under accession numbers KY942057 (XF4) and KY942056 (JA15). Genomes were compared using NCBI Blast and the Artemis Comparison Tool (Carver et al., 2005).\\nAuthor Contributions\\nAnalyzed the data, conceived and designed the experiments: AD, JA, XF, and GS. Performed the experiments: AD, JA, and XF. Wrote the paper: AD and GS.\\nFunding\\nThis work was supported by the BBSRC, UK. AD was supported by a Cambridge Doctoral Training Partnership Award from the BBSRC, UK.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nAcknowledgments\\nSequencing of \\u03d5JA15 was conducted by MicrobesNG (http://www.microbesng.uk), which is supported by the BBSRC (grant number BB/L024209/1). We are grateful to Ian Toth (James Hutton Institute, Scotland) for generous provision of Dickeya strains. This work was done under DEFRA license: 50864/197900/3. We thank Alison Rawlinson for technical support and Rita Monson and Miguel Matilla for helpful advice.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/article/10.3389/fmicb.2017.01654/full#supplementary-material\\nReferences\\nAdriaenssens, E. M., Ackermann, H. W., Anany, H., Blasdel, B., Connerton, I. F., Goulding, D., et al. (2012a). A suggested new bacteriophage genus: \\u201cViunalikevirus\\u201d. Arch. Virol. 157, 2035\\u20132046. doi: 10.1007/s00705-012-1360-5\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAdriaenssens, E. M., van Vaerenbergh, J., Vandenheuvel, D., Dunon, V., Ceyssens, P. J., de Proft, M., et al. (2012b). T4-related bacteriophage LIMEstone isolates for the control of soft rot on potato caused by \\u2018Dickeya solani.\\u2019 PLoS ONE 7:e33227. doi: 10.1371/journal.pone.0033227\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBankevich, A., Nurk, S., Antipov, D., Gurevich, A. A., Dvorkin, M., Kulikov, A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455\\u2013477. doi: 10.1089/cmb.2012.0021\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCahill, G., Fraser, K., Kowalewska, M. J., Kenyon, D. M., and Saddler, G. S. (2010). \\u201cRecent findings from the Dickeya survey and monitoring programme,\\u201d in Proceedings Crop Protection in Northern Britain (Dundee), 171\\u2013176.\\nGoogle Scholar\\nCarver, T. J., Rutherford, K. M., Berriman, M., Rajandream, M.-A., Barrell, B. G., and Parkhill, J. (2005). ACT: the artemis comparison tool. Bioinformatics 21, 3422\\u20133423. doi: 10.1093/bioinformatics/bti553\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCzajkowski, R., Ozymko, Z., Siwinska, J., Ossowicki, A., de Jager, V., Narajczyk, M., et al. (2015). The complete genome, structural proteome, comparative genomics and phylogenetic analysis of a broad host lytic bacteriophage phiD3 infecting pectinolytic Dickeya spp. Stand. Genomic Sci. 10:68. doi: 10.1186/s40793-015-0068-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCzajkowski, R., Ozymko, Z., Zwirowski, S., and Lojkowska, E. (2014). Complete genome sequence of a broad-host-range lytic Dickeya spp. bacteriophage phiD5. Arch. Virol. 159, 3153\\u20133155. doi: 10.1007/s00705-014-2170-8\\nCrossRef Full Text | Google Scholar\\nKrzywinski, M., Schein, J., Birol, I., Connors, J., Gascoyne, R., Horsman, D., et al. (2009). Circos: an information aesthetic for comparative genomics. Genome Res. 19, 1639\\u20131645. doi: 10.1101/gr.092759.109\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKumar, S., Stecher, G., and Tamura, K. (2016). MEGA7: Molecular Evolutionary Genetics Analysis version 7.0 for bigger datasets. Mol. Biol. Evol. 33:msw054. doi: 10.1093/molbev/msw054\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLaurila, J., Ahola, V., Lehtinen, A., Joutsjoki, T., Hannukkala, A., Rahkonen, A., et al. (2008). Characterization of Dickeya strains isolated from potato and river water samples in Finland. Eur. J. Plant Pathol. 122, 213\\u2013225. doi: 10.1007/s10658-008-9274-5\\nCrossRef Full Text | Google Scholar\\nLi, H. (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint, arXiv:1303.3997.\\nMaas Geesteranus HP (1972). Natrot en zwartbenigheid bij aardappelen. Bedrijfsontwikkeling 3, 941\\u2013945.\\nMansfield, J., Genin, S., Magori, S., Citovsky, V., Sriariyanum, M., Ronald, P., et al. (2012). Top 10 plant pathogenic bacteria in molecular plant pathology. Mol. Plant Pathol. 13, 614\\u2013629. doi: 10.1111/j.1364-3703.2012.00804.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMatilla, M. A., Fang, X., and Salmond, G. P. C. (2014). Viunalikeviruses are environmentally common agents of horizontal gene transfer in pathogens and biocontrol bacteria. ISME J. 8, 2143\\u20132147. doi: 10.1038/ismej.2014.150\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMatilla, M. A., and Salmond, G. P. C. (2014). Bacteriophage phiMAM1, a viunalikevirus, is a broad-host-range, high-efficiency generalized transducer that infects environmental and clinical isolates of the enterobacterial genera Serratia and Kluyvera. Appl. Environ. Microbiol. 80, 6446\\u20136457. doi: 10.1128/AEM.01546-14\\nCrossRef Full Text | Google Scholar\\nParkinson, N., Stead, D., Bew, J., Heeney, J., Tsror, L., and Elphinstone, J. (2009). Dickeya species relatedness and clade structure determined by comparison of recA sequences. Int. J. Syst. Evol. Microbiol. 59, 2388\\u20132393. doi: 10.1099/ijs.0.009258-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPelfrene, E., Willebrand, E., Cavaleiro Sanches, A., Sebris, Z., and Cavaleri, M. (2016). Bacteriophage therapy: a regulatory perspective. J. Antimicrob. Chemother. 71, 2071\\u20132074. doi: 10.1093/jac/dkw083\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPetty, N. K., Foulds, I. J., Pradel, E., Ewbank, J. J., and Salmond, G. P. C. (2006). A generalized transducing phage (phiIF3) for the genomically sequenced Serratia marcescens strain Db11: a tool for functional genomics of an opportunistic human pathogen. Microbiology 152, 1701\\u20131708. doi: 10.1099/mic.0.28712-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPritchard, L., Humphris, S., Baeyen, S., Maes, M., Van Vaerenbergh, J., Elphinstone, J., et al. (2013a). Draft genome sequences of four Dickeya dianthicola and four Dickeya solani strains. Genome Announc. 1:e00087\\u201312. doi: 10.1128/genomeA.00087-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPritchard, L., Humphris, S., Saddler, G. S., Elphinstone, J. G., Pirhonen, M., and Toth, I. K. (2013b). Draft genome sequences of 17 isolates of the plant pathogenic bacterium Dickeya. Genome Announc. 1:e00978\\u201313. doi: 10.1128/genomeA.00978-13\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nS\\u0142awiak, M., van Beckhoven, J. R. C. M., Speksnijder, A. G. C. L., Czajkowski, R., Grabe, G., and van der Wolf, J. M. (2009). Biochemical and genetical analysis reveal a new clade of biovar 3 Dickeya spp. strains isolated from potato in Europe. Eur. J. Plant Pathol. 125, 245\\u2013261. doi: 10.1007/s10658-009-9479-2\\nCrossRef Full Text | Google Scholar\\nToth, I. K., Cahill, G., Elphinstone, J. G., Humphris, S., and Wale, S. J. (2016). \\u201cAn update on the Potato Council/Scottish government-funded blackleg project - Year 2,\\u201d in Proceedings Crop Protection in Northern Britain 2016 (Dundee), 203\\u2013204.\\nGoogle Scholar\\nToth, I. K., van der Wolf, J. M., Saddler, G., Lojkowska, E., H\\u00e9lias, V., Pirhonen, M., et al. (2011). Dickeya species: an emerging problem for potato production in Europe. Plant Pathol. 60, 385\\u2013399. doi: 10.1111/j.1365-3059.2011.02427.x\\nCrossRef Full Text | Google Scholar\\nTsror, L., Erlich, O., Lebiush, S., Hazanovsky, M., Zig, U., Slawiak, M., et al. (2009). Assessment of recent outbreaks of Dickeya sp. (syn. Erwinia chrysanthemi) slow wilt in potato crops in Israel. Eur. J. Plant Pathol. 123, 311\\u2013320. doi: 10.1007/s10658-008-9368-0\\nCrossRef Full Text | Google Scholar\\nvan der Wolf, J. M., Nijhuis, E. H., Kowalewska, M. J., Saddler, G. S., Parkinson, N., Elphinstone, J. G., et al. (2014). Dickeya solani sp. nov., a pectinolytic plant-pathogenic bacterium isolated from potato (Solanum tuberosum). Int. J. Syst. Evol. Microbiol. 64(Pt 3), 768\\u2013774. doi: 10.1099/ijs.0.052944-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fvets.2022.1058115\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fvets.2022.1058115/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:50:43.881Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fvets.2022.1058115\",\n",
      "            \"title\": \"Frontiers | A lytic bacteriophage isolate reduced Clostridium perfringens induced lesions in necrotic enteritis challenged broilers\",\n",
      "            \"description\": \"Background: Bacteriophages are viral predators of bacteria and occur commonly in nature. Their host-specific infections against specific bacteria make them an attractive natural agent to control bacterial pathogens. Interest in the potential of bacteriophages as antibacterial agents in the production animal industries has increased. Methods: A total of 18 bacteriophages were isolated from Australian commercial poultry environments, from which three highly active phages were chosen for enrichment. Sequencing libraries were prepared using a Nextera XT kit (Illumina) and sequenced on an Illumina MiSeq instrument using 2\\u00d7300 bp paired-end chemistry. The sequence data were then assembled and aligned with a2 bacteriophage as the reference. An animal trial was performed by oral gavaging Clostridium perfringens netB containing strain EHE-NE18 to the Ross 308 broiler chickens prior inoculation with Eimeria species. The chickens were raised following the management guide for Ross 308 from d 0 to d 21 and fed with starter and grower diets met the specific breed nutrient requirements. Body weight gain and feed intake were measured on d 9 and d 21 and FCR adjusted with mortality was calculated. Results: The isolated bacteriophages only had only 96.7% similarity to the most closely related, previously characterised, Clostridium bacteriophage indicated that they might represent a novel strain of bacteriophage. A \\u2018cocktail\\u2019 containing the three bacteriophages was capable of lysing four known disease-inducing C. perfringens strains in vitro. Oral administration of the bacteriophages cocktail to broilers challenged with necrotic enteritis markedly alleviated intestinal necrotic lesions in the duodenum and jejunum on day 16. The phage treatment significantly reduced the lesion scores of birds challenged with NE (P  0.05). However, no effect on the growth performance was apparent during the recorded period of days 9-21. Conclusions: These findings suggest that bacteriophage treatment is a promising approach to protect intestinal health from C. perfringens induced necrotic enteritis. Further research will be required on the dosing, route of administration, and large scale validation studies to further advance this approach to pathogen control.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nNecrotic enteritis (NE) is a bacterial enteric disease in poultry accountable for over US$6 billion profit loss annually to the industry worldwide in the treatments and containment measures (1). The etiology of NE has been well characterized and it is mainly due to the rapid proliferation of C. perfringens Type G (2) strains that express the NetB toxin (3), albeit NetB negative strains have also been reported to cause NE (4). Outbreaks of NE also depend on a number of predisposing factors that result in degradation of intestinal health and compromise of the host immune status (3, 5, 6). It is reported that change of feed, use of different cereal bases in diet formulation, i.e., non-viscous grains to viscous grains, the inclusion of low digestibility protein sources, Eimeria infection, and compromise of to the host immune status, all exacerbate the chances of NE outbreaks (7). Flocks affected by acute NE exhibit clinical symptoms coupled with a rapid increase in flock mortality and distress in the affected bird populations, but subclinical (mild) disease are more frequent with little mortality rate if any, and suboptimal growth performance. The subclinical form of necrotic enteritis leads to more significant economic consequences with prolonged deterioration of growth performance resulting from loss of feed efficiency over time (8).\\nFor decades, poultry and other animal production systems across the globe have depended on in-feed antibiotics to sustain growth performance and control enteric pathogens as they can effectively maintain intestinal microbiota uniformity and suppress bacterial infection (9). However, the emergence of antibiotic-resistant bacteria and the possibility of horizontal transfer of antibiotic resistance genes to human pathogens resulted in the ban of in-feed antibiotics and has ushered in antibiotic-free practices that require alternatives to conventional in-feed antibiotics to maintain flock performance and health. The suppression of specific intestinal bacterial populations is essential to manage opportunistic pathogens and stop them from reaching a disease-causing threshold (10). Following the 2006 EU's legislative ban on antibiotic use in animal feed, numerous countries across the globe have followed suit and are dealing with the challenges of running antibiotic-free production systems. Early efforts to overcome these challenges focused on the adoption of antibiotic-free measures that provided some protection against pathogens (11), but the end results have been inconsistent.\\nThe lack of consistency in the efficacy of the current alternatives for maintaining performance means that novel solutions continue to be explored. The interest in bacteriophages stems from the fact that they naturally target specific bacteria and have narrow host ranges. On exposure to specific bacteriophages, a host bacterium undergoes lysis resulting in the death of the bacterium and the release of bacteriophages that can continue to infect other host bacteria; this self-replicating process is a very different process compared to other alternatives to antibiotics. Hence, bacteriophages are an attractive and feasible option for pathogen control. The use of phage therapy is not new. In fact, the clinical application of bacteriophages was first implemented in the early 1920s, with extensive therapeutic application in human medicine across the eastern European countries, and it has continued to the present time (12). Many known attributes of bacteriophages fit the criteria for an effective antibacterial alternative, including automatic dosing capability through lytic propagation of bacteriophages progeny, minimal inherent toxicity, low disruption to commensal microbiota, biofilm clearing, and variable routes of administration (13). Bacteriophage absorption on the surface of susceptible bacteria is mediated by specific receptors on the bacteria cell wall (14). Different types of bacteriophages that infect any one species of bacteria may recognize different receptors. Hence, the combination of several bacteriophages in the form of a cocktail may improve their efficacy against a target bacteria. Indeed, the use of multivalent bacteriophage cocktails was reported to be successful in reducing mortality and improving performance in chickens infected with C. perfringens resulting in clinical NE (15).\\nHowever, only limited research has been reported on phage prophylactic treatment against C. perfringens induced subclinical necrotic enteritis in chickens. This study was conducted to isolate and purify bacteriophages from poultry production environments that could infect and lyse C. perfringens, determine their lytic profile against a range of known virulent C. perfringens strains capable of inducing NE, and then the isolated bacteriophage was evaluated for prophylactic properties in vivo using a necrotic enteritis challenge model.\\nMaterials and methods\\nSample collection, bacterial hosts, and bacteriophage preparation\\nSamples from various sources within poultry environments were aseptically collected for bacteriophage isolation, which included soil samples from backyard chicken sheds and yards located in Armidale and Tamworth, NSW, Australia, sewage and offal wash from seven individual chicken farms, and intestinal content of chickens from experiments conducted at UNE (Armidale, New South Wales, Australia).\\nAll collected samples were kept on ice until processed. C. perfringens were isolated by plating serial dilutions in 0.1% peptone onto Perfringens Agar Base (Oxoid, Hampshire, UK) supplemented with Tryptose Sulphite Cycloserine (TSC) (Oxoid, Hampshire, UK), followed by incubation under anaerobic conditions using an anaerobic sachet (Oxoid Australia) at 39oC for 48 h. Putative C. perfringens colonies were re-streaked twice (to ensure purity) on fresh Horse Blood Agar (HBA) (Oxoid, Hampshire, UK) and incubated at 39oC for 18 h. Bacteria isolates were picked and suspended in the Brain Heart Infusion (BHI, Oxoid, Hampshire, UK) broth with sterile glycerol addition to 40% (v/v) and stored at \\u221220oC until used.\\nSamples for bacteriophage isolation were prepared by gently shaking 1 g of solid samples (i.e., soil, gut content or litter) or 1 ml liquid sample (effluent) in 9 ml of SM buffer (0.1 M NaCl, 1 mM MgSO4, 0.2 M Tris-HCl, pH 7.5) for 12 h. The supernatant was then centrifuged at 11,000 \\u00d7 g at 4oC for 5 mins. The supernatant was then filtered through 0.22-\\u03bcm-pore-size Millipore filters (MF-Millipore\\u2122, MERCK, Australia). Bacteriophages were isolated as per the methods of Smith (16) using the isolated C. perfringens host strains. In brief, overnight growth of C. perfringens from HBA agar was initially mixed in 3 ml brain-heart infusion (BHI) broth, and then further BHI was added to adjust the turbidity to 220 \\u00b1 15 Nephelometric Turbidity Unit (NTU). Serial dilutions of bacteriophage filtrates were prepared in SM buffer and 100 \\u03bcl of each bacteriophage dilution was mixed with 200 \\u03bcl of C. perfringens and incubated at 37oC for 30 min. Each set of the bacteriophages-host mixture was mixed into 7 ml of 0.5% BHI soft agar (supplemented with 1 mM MgCl2 and 1 mM Ca Cl2) and plated as an overlay on 1% BHI base agar. Plates were incubated anaerobically at 39oC for 48 h. Well separated plaques were picked and resuspended in 1 ml of SM buffer, then stored at 4\\u00b0C overnight. Serial dilutions of the homogenized plaques were made and cultured with the susceptible host in three cycles of plaque purification (16, 17).\\nPhage stocks were prepared from three to five plates with confluent plaques from the third round of phage purification. SM buffer (5 ml) was added to confluent plates and shaken slowly at 10 rpm for 12 h to resuspend the bacteriophage. The supernatants were then filtrated through 0.22-\\u03bcm-pore-size Millipore filters (MF-Millipore\\u2122, MERCK, Australia). The resulting filtrates were centrifuged at 40,000 \\u00d7 g at 4\\u00b0C for 2h. The resulting bacteriophage pellets were resuspended in SM buffer to form 1 \\u00d7 109 pfu/ml bacteriophage stock and stored in the dark at 4\\u00b0C.\\nLytic profile of the bacteriophage\\nThe bacteriophages were tested for lytic ability against four pathogenic C. perfringens strains (NE14, NE18, NE21, and NE36) to investigate the bacteriophage's virulence against target hosts in vitro. The characteristics of the tested C. perfringens strains are detailed in Table 1. Bacterial lawns were prepared using turbidity-adjusted C. perfringens strains (220 \\u00b1 15 NTU) mixed with 7 ml of 0.5% BHI soft agar (supplemented with 5mM MgSO4 and 10mM CaCl2) and set on the prepared 1% BHI base agar. The bacteriophage lysates were titered, and all were within the range of 1\\u20137 \\u00d7 109pfu/ml. A volume of 20 \\u03bcl of each bacteriophage preparation was aseptically placed in duplicate on marked positions of the bacterial lawns and set to dry with minimal disturbance to retain the bacteriophage droplets at the designated positions. Plates were incubated anaerobically at 39\\u00b0C up to 48 h and inspected for plaque formation. All positive tests were replicated three times with the relevant C. perfringens host.\\nTable 1. Characteristics of C. perfringens strains and susceptibility to the bacteriophages.\\nDNA sequencing of the bacteriophage\\nThe bacteriophage stocks were concentrated by precipitation with polyethylene glycol, and DNA was isolated using a phenol/chloroform extraction method. Sequencing libraries of 5 bacteriophages from different sources (A2, C2, D1, E1, and H1) were prepared using a Nextera XT kit (Illumina). The libraries were sequenced on an Illumina MiSeq instrument using 2 \\u00d7 300 bp paired-end chemistry. The sequence data generated from the MiSeq were assembled using the A5-miseq assembly pipeline (21). The bacteriophage DNA sequences were aligned using Clone Manager Suite V8 (Scientific and Educational Software, NC) with a2 bacteriophage as the reference. Homology analysis of the bacteriophage a2 sequence was performed against the NCBI database using the BLASTn program. Gene prediction was performed by submitting the DNA sequence of bacteriophage a2 to the RAST server (https://rast.nmpdr.org/) for RAST annotations. The whole genome sequence data has been deposited in the NCBI database with the accesion numbers OP753449\\u2013OP753453.\\nPhage inoculants and in vivo study\\nEighteen bacteriophage isolates that had lytic activity against the C. perfringens EHE-NE18 strain were enriched for the preparation of a bacteriophage inoculate. Three selected bacteriophages were prepared in larger volumes using the procedure adopted for purification as described above. Pelleted bacteriophages were resuspended in SM buffer to a final concentration of 1 \\u00d7 106 pfu/ml for the in vivo experiment.\\nDay-old birds (Ross 308 broilers) were obtained from a commercial hatchery (Baiada Hatchery, Tamworth, NSW, Australia). Birds were housed in 18 pens, each containing 11 birds. The pens (0.9 \\u00d7 0.95 m) were equipped with feeders and drinkers and hardwood shavings were used as bedding materials. The temperature and lighting were controlled according to the Ross 308 broiler guideline. A standard wheat-soybean starter diet formulated to meet the 2014 Ross 308 nutrient specifications was fed to all birds from day 0 to 10 (Table 2). A grower diet consisting of the wheat-based formulation was fed from day 10 to the end of the trial on day 21 (Table 2). Feed and water were available ad libitum. Pen weight and feed intake were measured on days 9 and 21, and the feed conversion ratio (FCR) was calculated by pen feed intake divided by pen bird weight gain and adjusted with mortality during the rearing period of d 9 to d 21.\\nTable 2. Composition and nutrients of starter and grower.\\nThe experiment consisted of a control untreated and unchallenged group, and necrotic enteritis challenge groups with or without bacteriophage treatment. On day 9, the NE challenge groups were given 1 ml of a mixture of field strains of Eimeria species containing 5,000 oocysts each of E. acervulina and E. maxima, and 2,500 oocysts of E. brunetti (Eimeria Pty Ltd, Werribee, VIC, Australia) by oral gavage, and unchallenged control birds were orally gavaged with 1 ml phosphate buffered saline as a sham treatment. On days 14-15, birds in the challenged groups were inoculated by oral gavage with 1 \\u00d7 108 CFU of C. perfringens EHE-NE18 strain as described by Wu et al. (22). The bacteriophage treatment birds were orally gavaged with 1 \\u00d7 106 units of the C. perfringens specific bacteriophage inoculate approximately 1 h following C. perfringens gavage. On day 16, two birds per pen from all the groups were randomly selected, weighed, and euthanized by cervical dislocation to perform post-mortem analysis and intestinal lesion scoring at a 0\\u20136 scale according to Keyburn et al. (19). On day 21, the experiment was completed, and birds were euthanized and disposed of according to the approved ethics protocol.\\nStatistical analysis\\nData were analyzed by one-way ANOVA with the pen as the experimental unit. Paired comparisons of means were performed using Tukey's test when differences among treatments were detected by ANOVA. Significance at P < 0.05 was declared for paired comparison.\\nResults and discussion\\nLytic properties of C. perfringens specific bacteriophages and genomic characterization\\nThe current study isolated and purified eighteen C. perfringens bacteriophages isolates. Each bacteriophage isolate was capable of producing plaques (Figure 1A) on each of the C. perfringens hosts obtained from the same poultry environments (soil, sewage, offal wash or intestinal content) and corresponding geographic locations. Among 46 field C. perfringens isolates from the environment, only three C. perfringens isolates were susceptible to all the bacteriophages isolates and thus used as the hosts for the bacteriophages. Therefore, all 18 bacteriophages were subsequently enriched with these three common hosts for bacteriophage purification, and the resulting bacteriophage preparations were used to perform lytic tests individually against pathogenic C. perfringens strains derived from chickens with necrotic enteritis. Positive lytic activities were observed for all these bacteriophage strains against four previously described virulent C. perfringens strains (NE14, NE18, NE21, and NE36) in our spot-testing assay. However, other strains of C. perfringens (NE1, NE4, NE7, NE16, NE31, and NE38) were not susceptible to these bacteriophages (Table 1). As the bacteriophage preparation was lytic against NE18 strain, which has been reported to produce NetB toxin (3) and has been used in NE experimental disease induction models (23), this preparation was used as an inoculate in the subsequent in vivo experiment with EHE-NE18 as the challenge strain.\\nFigure 1. (A) Clostridium perfringens bacteriophage plaques. Arrows showing the plaques produced by the strains from sample E that was collected from chicken caecal content, (B) An evolutionary tree of Clostridium bacteriophage strains produced by NCBI online blast tools with the Fast Minimum Evolution tree method. The Clostridium phage A2 isolated in the current study is highlighted in yellow showing its close relation with Clostridium phage CPD4. The scale bar corresponds to 0.07 substitutions per nucleotide position.\\nA narrow spectrum of infectiousness against C. perfringens strains also presents the possibility of a controlled elimination of pathogenic C. perfringens strains but minimal harm to commensal C. perfringens. It has been demonstrated that C. perfringens exists as normal gut commensals with low abundance in the intestinal tract of healthy chickens (6).\\nFive bacteriophages genomes were randomly chosen for sequencing, and the BLAST analysis against the NCBI database indicated the bacteriophages isolated aligned with the Clostridium bacteriophages CPD4 with a homology of 97.7% which clustered closely as shown in Figure 1B and 77 genes were predicted, of which 19 had specific bacteriophage functions. The degree of homology to previously characterized bacteriophages indicates that the bacteriophages isolated in this study might represent a novel bacteriophage strain. C. perfringens specific bacteriophage isolates have been reported to belong to the family Siphoviridae or Podviridare, and bacteriophages endolysin has been shown to be responsible for the lytic activity (17, 24). The sequences of the bacteriophage isolates produced in the current study generated a single contig with the same GC content (34%) and showed a similar genome size with 99.98\\u2013100.00% identities (Table 3). The genomes of several C. perfringens specific bacteriophages have been characterized as reported previously, providing essential information for understanding the virulence of phages to their hosts and for downstream applications (17, 25).\\nTable 3. A5-miseq assemblies of the bacteriophages isolates.\\nBacteriophage treatment reduced NE induced lesions\\nNo mortality of birds due to NE and low lesion scores (0.8 out of a maximum of 6) were observed in the in vivo challenge experiment, indicating that subclinical NE had been induced. The lesion score results are shown in Table 4. Higher lesion scores were recorded in both the duodenum (P < 0.01) and jejunum (P < 0.01) of the challenged birds than those unchallenged. The phage treatment significantly reduced the lesion scores of birds challenged with NE (P < 0.01), such that the lesion scores between birds treated with the bacteriophages and the control unchallenged birds were not statistically different (P > 0.05). However, this significant effect of the bacteriophages on lesion scores did not fully manifest in the measures of bird performance. In the current study, the birds under NE challenge were significantly affected, showing reduced BWG, FI, and increased FCR in the challenged treatments compared to the control (Table 5). NE challenged birds with or without bacteriophage treatment had reduced BWG and FI and increased FCR during days 9 to 21 or 0 to 21 (P < 0.001 for all except FCR during d 0\\u201321 being P = 0.002). The challenge model used in the current study experimentally reproduced the conditions of possible NE outbreaks in the field, of which coccidiosis serves as a predisposing factor to damage intestinal epithelium before C. perfringens infection (7, 22). The current study was not set out to apportion the impact of the Eimeria per se, and therefore, it is difficult to determine with any certainty to what extent the lack of performance improvement by the bacteriophage therapy could be due to its inability to counter any damage that resulted from coccidiosis. Eimeria negatively affects nutrient utilization by interfering with the integrity of epithelium cells, leading to compromised growth of the animals (6, 26).\\nTable 4. Lesion scores of the birds under treatments at day 16*.\\nTable 5. Performance of the birds in response to the NE and bacteriophages treatments*.\\nAlthough bacteriophage treatment did not translate into improved bird performance, less severity of intestinal lesions in the treated birds suggests that bacteriophages appropriately selected against the NetB producing strain of C. perfringens may be worthy of further exploration.\\nThe current study focused on the performance of the birds during days 9\\u201321, when the NE challenge was applied, and the NE effect was expected. However, a prolonged rearing period might be useful to observe the bacteriophage treatment effect on later stage performance through improved intestinal health. In addition, the dosage and the route of bacteriophage administration may also be important factors for the effectiveness of the bacteriophage treatment to protect birds from NE infection. These will need to be addressed in future studies to maximize the protective effect of bacteriophages, such as water and feed delivery and higher doses. Further, more bacteriophage strains may be needed to control C. perfringens strains in chickens through the administration of a wider spectrum cocktail.\\nIn conclusion, 18 isolated C. perfringens specific bacteriophage strains were isolated. The finding that they had 96.7% similarity to the most closely related, previously characterized, Clostridium bacteriophage indicated that they represent a similar cluster of Clostridium bacteriophage. The result showed an encouraging protective effect of the bacteriophage application with alleviated lesions in the gut of NE-challenged birds despite the fact that no performance improvement was seen. Whether bacteriophage can be used in the poultry production to combat NE is yet to be assured. Further investigations are warranted to optimize the dosage and administration protocol and to isolate more NetB positive and even negative bacteriophage strains for a wider spectrum control of different C. perfringens pathogens.\\nData availability statement\\nThe data presented in the study are deposited in the NCBI GenBank repository, accession numbers OP753449 \\u2013 OP753453.\\nEthics statement\\nThe animal study was reviewed and approved by the Animal Ethics Committee of the University of New England (UNE), Armidale, Australia.\\nAuthor contributions\\nCK performed data collection and analysis, interpreted results, and wrote the manuscript. KM performed isolation of the bacteriophages and analysis. TV sequenced the bacteriophage genome, analyzed data, and read the manuscript critically. HC participated the design of the project, isolated bacteriophages, and reviewed the manuscript critically. RM participated the design of the project, data analysis, and reviewed the manuscript critically. MC participated the conceptualization of the project and reviewed the manuscript critically. S-BW conceptualized, designed the project, performed the animal experiment, analyzed data, and reviewed the manuscript critically. All authors contributed to the article and approved the submitted version.\\nFunding\\nThis study was supported by the Poultry CRC (#1-3-4).\\nConflict of interest\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nPublisher's note\\nAll claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.\\nReferences\\n1. Wade B, Keyburn, A. The true cost of necrotic enteritis. Poultry World. (2015) 31:16\\u201317. \\nGoogle Scholar\\n2. Rood, JI, Adams V, Lacey J, Lyras D, McClane BA, et al. Expansion of the Clostridium perfringens toxin-based typing scheme. Anaerobe. (2018) 53:5\\u201310. doi: 10.1016/j.anaerobe.2018.04.011\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n3. Keyburn AL, Boyce JD, Vaz P, Bannam TL, Ford ME, Parker D, et al. NetB, a new toxin that is associated with avian necrotic enteritis caused by Clostridium perfringens. PLoS Pathogens. (2008) 4:e26. doi: 10.1371/journal.ppat.0040026\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n4. Cooper KK, Songer JG. Virulence of Clostridium perfringens in an experimental model of poultry necrotic enteritis. Vet Microbiol. (2010) 142:323\\u20138. doi: 10.1016/j.vetmic.2009.09.065\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n5. Keyburn AL, Bannam TL, Moore RJ, Rood JI. NetB, a pore-forming toxin from necrotic enteritis strains of Clostridium perfringens. Toxins. (2010) 2:1913\\u201327. doi: 10.3390/toxins2071913\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n6. Van Immerseel F, Rood JI, Moore RJ, Titball RW. Rethinking our understanding of the pathogenesis of necrotic enteritis in chickens. Trends Microbiol. (2009) 17:32\\u20136. doi: 10.1016/j.tim.2008.09.005\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n8. Skinner JT, Bauer S, Young V, Pauling G, Wilson J. An economic analysis of the impact of subclinical (mild) necrotic enteritis in broiler chickens. Avian Dis. (2010) 54:1237\\u201340. doi: 10.1637/9399-052110-Reg.1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n9. Lanckriet A, Timbermont L, De Gussem M, Marien M, Vancraeynest D, Haesebrouck F, et al. The effect of commonly used anticoccidials and antibiotics in a subclinical necrotic enteritis model. Avian Pathol. (2010) 39:63\\u20138. doi: 10.1080/03079450903505771\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n10. Eeckhaut V, Wang J, Van Parys A, Haesebrouck F, Joossens M, Falony G, et al. The probiotic Butyricicoccus pullicaecorum reduces feed conversion and protects from potentially harmful intestinal microorganisms and necrotic enteritis in broilers. Front Microbiol. (2016) 7:1416. doi: 10.3389/fmicb.2016.01416\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n11. Cogliani C, Goossens H, Greko C. Restricting antimicrobial use in food animals: lessons from Europe. Microbe. (2011) 6:274. \\nGoogle Scholar\\n13. Karthik K, Muneeswaran NS, Manjunathachar HV, Gopi M, Elamurugan A, Kalaiyarasu S. Bacteriophages: effective alternative to antibiotics. Adv Anim Vet Sci. (2014) 2:1\\u20137. doi: 10.14737/journal.aavs/2014/2.3s.1.7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n14. Pleteneva EA, Shaburova OV, Sykilinda NN, Miroshnikov KA, Krylov SV, Mesianzhinov VV, Krylov, VN. Study of the diversity in a group of phages of Pseudomonas aeruginosa species PB1 (Myoviridae) and their behavior in adsorbtion-resistant bacterial mutants. Genetika. (2008) 44:185\\u201394. doi: 10.1134/S1022795408020051\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n15. Miller RW, Skinner J, Sulakvelidze A, Mathis GF, Hofacre, CL. Bacteriophage therapy for control of necrotic enteritis of broiler chickens experimentally infected with Clostridium perfringens. Avian Dis. (2010) 54:33\\u201340. doi: 10.1637/8953-060509-Reg.1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n17. Seal BS, Fouts DE, Simmons M, Garrish JK, Kuntz RL, Woolsey R, et al. Clostridium perfringens bacteriophages \\u03a6CP39O and \\u03a6CP26F: genomic organization and proteomic analysis of the virions. Arch Virol. (2011) 156:25\\u201335. doi: 10.1007/s00705-010-0812-z\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n18. Sheedy SA, Ingham AB, Rood JI, Moore RJ. Highly conserved alpha-toxin sequences of avian isolates of Clostridium perfringens. J Clin Microbiol. (2004) 42:1345\\u20137. doi: 10.1128/JCM.42.3.1345-1347.2003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n19. Keyburn AL, Sheedy SA, Ford ME, Williamson MM, Awad MM, Rood JI. Alpha-toxin of Clostridium perfringens is not an essential virulence factor in necrotic enteritis in chickens. Inf Immun. (2006) 74:6496\\u2013500. doi: 10.1128/IAI.00806-06\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n20. Keyburn AL, Portela RW, Sproat K, Ford ME, Bannam TL, Yan X, et al. Vaccination with recombinant NetB toxin partially protects broiler chickens from necrotic enteritis. Vet Res. (2013) 44:1. doi: 10.1186/1297-9716-44-54\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n21. Coil D, Jospin G, Darling AE. A5-miseq: an updated pipeline to assemble microbial genomes from Illumina MiSeq data. Bioinformatics. (2015) 31:587\\u20139. doi: 10.1093/bioinformatics/btu661\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n22. Wu S-B, Rodgers N, Choct M. Optimized necrotic enteritis model producing clinical and subclinical infection of Clostridium perfringens in broiler chickens. Avian Dis. (2010) 54:1058\\u201365. doi: 10.1637/9338-032910-Reg.1\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n23. Gharib-Naseri, K, Kheravii, S, Keerqin, C, Swick RA, Choct, M, Wu, S-B. Differential expression of intestinal genes in necrotic enteritis challenged broiler chickens with two different Clostridium perfringens strains. Poultry Sci. (2020) 100:100886. doi: 10.1016/j.psj.2020.11.063\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n25. Morales CA, Oakley BB, Garrish JK, Siragusa GR, Ard MB, Seal BS. Complete genome sequence of the podoviral bacteriophage \\u03a6CP24R, which is virulent for Clostridium perfringens. Arch Virol. (2012) 157:769\\u201372. doi: 10.1007/s00705-011-1218-2\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n26. Collier C, Hofacre C, Payne A, Anderson D, Kaiser P, Mackie R, et al. Coccidia-induced mucogenesis promotes the onset of necrotic enteritis by supporting Clostridium perfringens growth. Vet Immunol Immunopathol. (2008) 122:104\\u201315. doi: 10.1016/j.vetimm.2007.10.014\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fimmu.2018.01496\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fimmu.2018.01496/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:50:55.664Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fimmu.2018.01496\",\n",
      "            \"title\": \"Frontiers | Vectorized Delivery of Alpha-GalactosylCeramide and Tumor Antigen on Filamentous Bacteriophage fd Induces Protective Immunity by Enhancing Tumor-Specific T Cell Response\",\n",
      "            \"description\": \"We have exploited the properties of filamentous bacteriophage fd to deliver immunologically active lipids together with antigenic peptides. Filamentous bacteriophages resemble for size, capability to be permeable to blood vessels and high density antigen expression, a nature-made nanoparticle. In addition, their major coat protein pVIII, which is arranged to form a tubular shield surrounding the phage genome, has a high content of hydrophobic residues promoting lipid association. We conjugated bacteriophages to alpha-GalactosylCeramide (\\uf061-GalCer), a lipid antigen stimulating invariant Natural Killer T (iNKT) cells and capable of inducing their anti-tumoral activities. We found that bacteriophage fd/\\uf061-GalCer conjugates could repeatedly stimulate iNKT cells in vitro and in vivo, without inducing iNKT anergy. Moreover, co-delivery of alpha-GalactosylCeramide and a MHC class I restricted tumor associated antigenic determinant to APCs via bacteriophages strongly boosted adaptive CD8+ T cell response and efficiently delayed tumor progression. Co-delivery of a tumor antigen and iNKT-stimulatory lipid on the surface of filamentous bacteriophages is a novel approach to potentiate adaptive anti-cancer immune responses, overcoming the current limitations in the use of free alpha-GalactosylCeramide and may represent an attractive alternative to existing delivery methods, opening the path to a potential translational usage of this safe inexpensive and versatile tool.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nInvariant natural killer T (iNKT) cells represent a unique subpopulation of T lymphocytes with both innate-like and adaptive functions mainly found in spleen, liver, and bone marrow. iNKT cells express NK lineage receptors and a semi-invariant T cell receptor (TCR) composed of V\\u03b114\\u2013J\\u03b118 chain in mice and V\\u03b124\\u2013J\\u03b118 chain in humans, paired with \\u03b2 chains encoded by a limited number of V\\u03b2 genes (1).\\nInvariant natural killer T cells recognizes (glyco)lipid antigens (Ag) presented by the CD1d molecule expressed by several types of antigen-presenting cells (APCs), including dendritic cells (DCs) (2).\\nThe potential anti-tumoral function of iNKT cells was first discovered with the identification of alpha-GalactosylCeramide (\\u03b1-GalCer), a synthetic derivative of agelasphin, a glycolipid originally isolated from the marine sponge Agelas mauritianus, as a strong stimulatory ligand during screening of anti-tumor compounds from natural sources (3\\u20135).\\nIn response to \\u03b1-GalCer, iNKT cells rapidly secrete large quantities of cytokines, including IFN-\\u03b3, IL-2, IL-4, IL-6, IL-10, IL-12, IL-13, IL-17, IL-21, IL-22, TGF-\\u03b2, and TNF-\\u03b1, that in turn activate a variety of other cell types, including NK cells, DCs, B, and T cells (6\\u20139). Through this activation cascade, \\u03b1-GalCer showed to exert potent anti-tumor and adjuvant activities in vivo in mouse models (5, 10, 11), rendering it a powerful candidate for adjuvant therapy in cancer.\\nBased on these pioneering reports, attempts have been made to exploit the anti-tumor property of \\u03b1-GalCer. Although the drug was well tolerated, no or moderate clinical responses were observed among the patients repeatedly injected intravenously with \\u03b1-GalCer (12). Studies performed in a mouse model demonstrated that \\u03b1-GalCer induced a long-term anergy of iNKT cells, thus preventing proliferation and cytokine release upon a recall stimulation (13).\\nEven though the mechanism underlying iNKT cell anergy mediated by \\u03b1-GalCer is still unknown, it is common knowledge that co-stimulatory signals and cytokines provided by antigen-presenting DCs are considered crucial for avoiding anergy (14).\\nIndeed, mouse studies demonstrated that the injection of \\u03b1-GalCer-pulsed DCs induced a sustained cytokines production when compared with administration of free \\u03b1-GalCer, suggesting that the type of \\u03b1-GalCer administration is critical for iNKT cell stimulation (15).\\nA way to optimize iNKT cell responses may lie in actively directing \\u03b1-GalCer to the appropriate APC using suitable delivery systems (16).\\nIn previous studies, we found that the filamentous bacteriophage is an efficient antigen-delivery system because it is internalized by DCs and activates innate and adaptive immune responses in the absence of classical adjuvants (17\\u201319).\\nThe filamentous bacteriophage can be considered as a nature-made nanocarrier according to its nano-dimensions (5 nm in diameter and 1,000 nm in length), its capability to cross blood vessels and for the capacity of expressing very large amounts of recombinant protein antigen. Its major coat protein pVIII is present in 2,700 copies on the phage coat and is arranged to form a tubular shield surrounding the phage genome. The pVIII protein is composed of three specific domains: a hydrophobic core, an acidic N-terminal domain, and a basic C-terminal domain (20). Due to the high content of hydrophobic residues, the pVIII shows strong binding of lipids, a feature that we exploited by promoting association of the bacteriophage with \\u03b1-GalCer. Here, we show that \\u03b1-GalCer conjugated with bacteriophages could repeatedly stimulate iNKT cells in vitro and in vivo, without inducing iNKT cell anergy. Moreover, therapeutic vaccination with phages co-displaying \\u03b1-GalCer and a tumor antigen delayed tumor progression in mice, showing improved adjuvanticity of the phage particles and leading to enhanced vaccine efficacy. These results suggest a novel approach to potentiate the efficacy of the bacteriophage delivery system as anti-tumor vaccine.\\nMaterials and Methods\\nCells and Reagents\\nCD1d-restricted V\\u03b114i NKT hybridoma FF13 (21) cells were a kind gift of Dr. De Libero (Department of Biomedicine, University of Basel and University Hospital Basel, Basel, Switzerland) and were cultured in RPMI 1640 (Lonza) medium supplemented with 100 U/ml penicillin, 100 \\u00b5g/ml streptomycin, and 10% fetal calf serum (FCS) (all from GIBCO, Milan, Italy).\\nOTI hybridoma cell line, recognizing the OVA257\\u2013264 SIINFEKL determinant, was produced by infection of 54\\u03b617 hybridoma T cells (22) with a retrovirus encoding the OTI TCR V alpha 2 and V beta 5 chains. Recombinant retroviral particles were produced in HEK293T cells (ATCC CRL 1573) by transfection with pMXOTI, pEcotropic, and pVSVg plasmids. The virus-containing supernatant was collected 48 h after transfection. 5 \\u00d7 105 54\\u03b617 cells were infected with 10 ml of viral supernatant and TCR expression was verified after 5 days by APC-anti-CD3\\u03b5 (145-2C11, Biolegend) staining and FACS analysis. Positive cells were then sorted using a FACS ARIA (Becton Dickinson, Fullerton, CA, USA) and amplified in DMEM supplemented with 10% FCS, 25 U/ml penicillin G, 25 \\u00b5g/ml streptomycin, and 0.05 \\u00b5M \\u03b2-mercaptoethanol.\\nB16-OVA melanoma cells (H2Kb), stably expressing chicken ovalbumin, were a kind gift of Dr. Dellabona (San Raffaele Scientific Institute, Milan, Italy). Cells were cultured in RPMI 1640 medium supplemented with 100 U/ml penicillin, 100 \\u00b5g/ml streptomycin, 10% FCS, and 100 \\u00b5g/ml Hygromicin (Sigma-Aldrich).\\nThe synthetic peptide OVA257\\u2013264 (SIINFEKL) was purchased from Primm (Naples, Italy).\\nSynthetic \\u03b1-GalCer (KRN7000) (2S,3S,4R)-1-O-(\\u03b1-D-galactosyl)-N-hexacosanoyl-2-amino-1,3,4-octadecanetriol (BML-SL232-1000) was purchased from Vinci Biochem.\\nInternal standard (IS) for mass spectrometric analysis D-Galactosyl-\\u03b2-1,1\\u2019-N-Palmitoyl-D-erythro sphingosine was purchased from Avanti Polar Lipids.\\nMeOH and water for liquid chromatography\\u2013mass spectrometric analysis were purchased from Merck and were LC\\u2013MS grade.\\nMice\\nSix- to eight-week-old female C57BL/6 were purchased from Charles River (Lecco, Italy) and housed in IGB \\u201cA. Buzzati-Traverso\\u201d Animal House Facility under standard pathogen-free conditions abiding institutional guidelines.\\nPurification of Bacteriophages Particles and Conjugation to \\u03b1-GalCer\\nWild type (fdWT) and hybrid fdOVA (expressing the recombinant OVA257\\u2013264-pVIII proteins) filamentous bacteriophages were purified from the supernatant of transformed Escherichia coli TG1recO cells. Bacteria were grown in TY2X medium for 16 h and the bacteriophage virions were harvested from E. coli supernatant, Poly(ethylene glycol) 6000 (Sigma-Aldrich) precipitated, purified by ultracentrifugation (24,500 g) on cesium chloride (Sigma-Aldrich) gradient, and dialyzed against phosphate buffered saline (PBS) 1\\u00d7. Elimination of lipopolysaccharides (LPS) from phage particles was performed using Triton X-114 (Sigma-Aldrich). Briefly, Triton X-114 was mixed to the phage preparations to a final concentration of 1% by vigorous vortexing. The mixture was incubated at 4\\u00b0C for 5 min, then incubated for 5 min at 50\\u00b0C, and centrifuged (20,000 g, 10 min) at 25\\u00b0C. The upper aqueous phase containing the virions was carefully removed and subjected to Triton X-114 phase separation for more cycles. The resulting aqueous phase containing virions was subjected to cesium chloride gradient centrifugation, dialyzed against PBS 1\\u00d7, and assayed for LPS contamination using the Limulus Amebocyte Lysate assay (Limulus Amebocyte Lysate QCL-1000 chromogenic modification, Lonza), according to the manufacturer\\u2019s instructions.\\nThe expression of the recombinant OVA257\\u2013264-pVIII proteins was induced adding 0.1 mM isopropyl-beta-D-thiogalactopyranoside (Sigma-Aldrich) to the bacteria growing in TY2X medium.\\nThe number of copies of pVIII displaying the OVA257\\u2013264 peptide was estimated by N-terminal sequence analysis of the purified virions and resulted in 15\\u201320% for each phage preparation.\\nBacteriophages in PBS 1\\u00d7 pH 8 and KRN7000 in dimethyl sulfoxide (Sigma-Aldrich) were combined at a 10:1 ratio (\\u03bcg phages: \\u03bcg \\u03b1-GalCer) and stirred at 4\\u00b0C overnight. Virions were subjected to cesium chloride gradient ultracentrifugation (24,500 g), dialyzed against PBS 1\\u00d7 and the concentration of bacteriophages was determined using spectrophotometer. The presence of \\u03b1-GalCer in the phage preparations was determined by the in vitro biological assay and its conjugated amount determined by mass spectrometric analysis, as described below.\\nRelease of \\u03b1-GalCer From the Conjugated Bacteriophage by Solvent Extraction\\nA small aliquot (50 \\u00b5l) of a PBS solution containing the bacteriophage conjugated to \\u03b1-GalCer at a concentration of 1.5 mg/ml was diluted 1:10 to a final volume of 500 \\u00b5l with ultrapure water; 200 ng of IS in 20 \\u00b5l of methanol (MeOH) were added and the suspension was extracted with 2 ml of MeOH/CHCl3 (1:1) by sonication. After centrifugation, the organic phase was removed and the aqueous layer was re-extracted with MeOH/CHCl3. The combined organic phases were dried under nitrogen, reconstituted in 1 ml of MeOH, and subjected to LC\\u2013MS analysis.\\nQuantitative LC\\u2013MS/MS Analysis of \\u03b1-GalCer\\nA quantitative method was developed on a UPLC system (Acquity, Waters) coupled to a triple quadrupole mass spectrometer (API 3200, SCIEX). The chromatographic analysis was performed on an Acquity BEH Phenyl column (Waters, 100 \\u00d7 2.1 mm, 1.7 \\u00b5m), eluted with a short gradient program from 95:5 MeOH/H2O to 100% MeOH in 1 min followed by an isocratic elution at 100% MeOH for 4 min. Flow rate was set at 0.4 ml/min and column temperature at 40\\u00b0C. \\u03b1-GalCer eluted at a Rt of 1.59 min, IS at 1.1 min.\\nA calibration curve was prepared by using five calibration points of \\u03b1-GalCer standard (STD) (62.5, 125, 250, 500, and 1,000 ng/ml) spiked with 200 ng/ml IS and plotted as area ratio of STD/IS response vs concentration. Two MRM transitions were monitored for both STD and IS for quantitative purposes and to confirm analytical identification. The most intense transitions for each compound (i.e., m/z 856.7 > 178.9 for STD and m/z 698.5 > 89.2 for IS) were used as analytical responses.\\nIn Vitro \\u03b1-GalCer Presentation on CD1d and Stimulation of iNKT and OTI Hybridoma Cells\\nMouse bone marrow-derived dendritic cells (BMDCs) were generated from C57BL/6 mice according to Ref. (23). At 7 day of culture, BMDCs were incubated in RPMI medium supplemented with 10% FCS, 5 \\u00b5M 2-ME, 1 mM glutamine, and 1 mM sodium pyruvate for 2 h with different concentrations of free \\u03b1-GalCer, fdWT bacteriophages, or fd/\\u03b1-GalCer bacteriophages. The experiment OTI hybridoma cell experiment was performed by incubating BMDCs with different concentration of fdOVA, fdOVA/\\u03b1-GalCer, or OVA257\\u2013264 synthetic peptide. After the incubation, cells were washed and stained with PE-conjugated anti mouse \\u03b1-GalCer:CD1d complex (L363, Biolegend) or co-cultured (50,000/well) with the mouse V\\u03b114 iNKT hybridoma FF13 or OTI hybridoma (50,000/well) for 40 h.\\nPE-conjugated anti-mouse \\u03b1-GalCer:CD1d complex (L363, Biolegend) antibody was used to stain DCs and fluorescence of stained cells was analyzed by FACSCanto II flow-cytometer and DIVA (Data-Interpolating Variational Analysis) software (Becton Dickinson). The IL-2 released into cell co-culture supernatants was measured by ELISA. Supernatants (0.1 ml/well) were assayed in duplicate using mouse IL-2 ELISA MAX\\u2122 Standard (Biolegend), according to the manufacturer\\u2019s instructions.\\nMeasurement of In Vivo and In Vitro Recall Response to \\u03b1-GalCer\\nMice were injected intravenously (i.v.) or intratumorally (i.t.) with 100 \\u00b5l of PBS containing 5 \\u00b5g \\u03b1-GalCer, 50 \\u00b5g of fd/\\u03b1-GalCer bacteriophage conjugate, or with vehicle alone. Where indicated, 200 or 130 ng of \\u03b1-GalCer was also used. After 24 h mice were sacrificed, spleens were harvested, and single-cell suspensions were prepared. Spleen cell suspensions were plated in U-bottomed 96-well plate at 2 \\u00d7 105 cells per well in RPMI medium supplemented with 10% FCS, 5 \\u00b5M 2-ME, 1 mM glutamine, and 1 mM sodium pyruvate in the presence of indicated doses of free \\u03b1-GalCer or with medium alone. For proliferation assays, 1 \\u03bcCi of [3H] thymidine (PerkinElmer Life Sciences) was added to the wells after 60 h of culture, and cells were cultured for an additional 12 h. Cells were then harvested using a semi-automatic cell harvester FilterMate (PerkinElmer, CA, USA) and uptake of radioactivity was measured using the Top Count NTX microplate scintillation counter (PerkinElmer). Cell proliferation was expressed as a cpm fold increase vs unstimulated cells. IL-2 secretion was evaluated by ELISA using Mouse IL-2 ELISA MAX\\u2122 Standard, collecting cell culture supernatants (0.1 ml/well) after 60 h of culture.\\nAnalysis of iNKT and NK In Vivo Activation\\nMice (n = 3/group) were injected i.v. with 100 \\u00b5l of PBS containing 5 \\u00b5g free or 50 \\u00b5g of fd and fd/\\u03b1-GalCer. Mice were sacrificed 3 h post-treatment. Splenic iNKT cells were analyzed for intracellular IFN-\\u03b3 secretion by culturing 7.5 \\u00d7 106 spleen cells overnight in the presence of 10 \\u00b5g/ml of the secretion inhibitor Brefeldin-A (BFA, Sigma-Aldrich). Cells were then harvested and IFN-\\u03b3 production was evaluated by intracellular staining on gated CD3+NK1.1+ cells or CD3\\u2212NK1.1+ cells using APC-conjugated anti mouse CD3 monoclonal antibody (mAb) (17A2, Biolegend), PE-conjugated anti-mouse NK1.1 mAb (PK136, Biolegend), FITC-conjugated anti-mouse IFN-\\u03b3 mAb (XMG1.2, Biolegend), and Leucoperm fixation and permeabilization kit (AbD Serotec, Oxford, UK). The IFN-\\u03b3 release induced by 30 ng/ml of Phorbol 12-myristate 13-acetate (PMA, Sigma-Aldrich) plus 1 \\u00b5g/ml of ionomycin (Sigma-Aldrich) was used as a positive control. Cells were analyzed using FACSCanto II flow-cytometer and DIVA software.\\nIn Vivo OVA257\\u2013264 T Cell Response Evaluation\\nGroup of mice (n = 4) were primed (day 0) by subcutaneous injection with 100 \\u00b5l of PBS containing fdOVA (SIINFEKL peptide) bacteriophages or fdOVA/\\u03b1-GalCer bacteriophages (all delivering 1.5 \\u00b5g of OVA257\\u2013264 peptide) and boosted (day 14) with the same amount of fdOVA bacteriophages delivering or not the \\u03b1-GalCer. As control, mice were inoculated twice with vehicle alone (PBS).\\nAt day 21, splenocytes were isolated and the frequency of OVA257\\u2013264 specific CD8+ T cells was assessed using FITC or APC conjugated anti-mouse CD8a mAb (53\\u20136.7, Biolegend) and PE-H2Kb SIINFEKL MHC dextramers (Immudex) staining. Stained cells were analyzed by flow cytometry using a FACSCantoII. Results are expressed as the percentage of CD8+ gated cells that are positive for the MHC I/peptide dextramers.\\nIFN-\\u03b3-producing effector cells were evaluated by culturing 7.5 \\u00d7 106 spleen cells with OVA257\\u2013264 SIINFEKL synthetic peptide (10 \\u03bcg/ml) for 5 h in the presence of BFA (Sigma-Aldrich). Cells were then harvested and IFN-\\u03b3 production was evaluated by intracellular staining on gated CD8+ cells using FITC or APC conjugated anti-mouse CD8a mAb, PE conjugated anti-mouse IFN-\\u03b3 mAb (XMG1.2 Biolegend), and Leucoperm fixation and permeabilization kit. The IFN-\\u03b3 release induced by 30 ng/ml of PMA plus 1 \\u00b5g/ml of ionomycin was used as a positive control. Data were acquired on FACSCanto II flow-cytometer and DIVA software.\\nTherapeutic Vaccination Against B16 Tumor Cells\\nNa\\u00efve C57BL/6 mice were engrafted with 2 \\u00d7 105 B16 melanoma cells subcutaneously in the left flank. When tumors were palpable, mice were vaccinated with 100 \\u00b5l volumes containing PBS, 2.5 \\u00b5g \\u03b1-GalCer, 50 \\u00b5g of fdWT bacteriophages, or 50 \\u00b5g of fd/\\u03b1-GalCer. In the experiment with B16-OVA tumor cell line, mice were injected intratumorally twice, on day 0 and day 5, with 100 \\u00b5l volumes containing PBS, 2.5 \\u00b5g \\u03b1-GalCer, 50 \\u00b5g of fdOVA bacteriophages, 50 \\u00b5g of fd/\\u03b1-GalCer, or 50 \\u00b5g of fdOVA/\\u03b1-GalCer. Tumor growth was assessed three times weekly using caliper and recorded as tumor volume (mm3) according to the formula (d2 \\u00d7 D)/2, where d and D are the shortest and the longest diameters. Mice were culled once tumor size met or exceeded 1,500 mm3, in accordance with established guidelines. Survival was recorded as the percentage of surviving animals.\\nIn another set of experiments, mice were sacrificed on day 12 and tumors and spleens were collected and homogenized. Tumors and spleen cells were filtered through a 70-\\u00b5m cell strainer nylon mesh, and erythrocytes were lysed. Cells were then washed once with medium, resuspended in PBS + 5% FCS to a concentration of 1.5 \\u00d7 106 cells/ml and labeled with APC conjugated anti-mouse CD8a mAb and PE-H2Kb SIINFEKL MHC dextramers for flow cytometric analysis. 7-aminoactinomycin D (7-AAD) (BD) was used for live\\u2013dead cell discrimination.\\nStatistical Analysis\\nComparative analyses were performed using the analysis of variance (ANOVA) followed by a Dunnett\\u2019s multiple comparison test or Bonferroni post hoc comparisons. Analyses of survival were performed using the log-rank (Mantel\\u2013Cox) test. All the analyses were performed with the GraphPad Prism 5 program (GraphPad Software, CA, USA). In all cases, differences were considered statistically significant when p < 0.05.\\nResults\\n\\u03b1-GalCer Conjugation to fd Filamentous Bacteriophage\\nIn order to exploit the ability of the major coat protein pVIII of the filamentous bacteriophage fd to bind hydrophobic lipids, \\u03b1-GalCer was conjugated to LPS-purified bacteriophage particles (Figure 1A) in a 10:1 ratio (1.5 mg of bacteriophages: 150 \\u00b5g \\u03b1-GalCer), followed by ultracentrifugation on cesium chloride gradient to remove unbound lipids.\\nFigure 1. fd/alpha-GalactosylCeramide (\\u03b1-GalCer) conjugate characterization. (A) Schematic image representing filamentous bacteriophage engineered for the expression of antigenic peptide and \\u03b1-GalCer on the coat surface. (B) UPLC-multiple reaction monitoring profile of the organic extract of a solution containing 1.5 mg/ml of the bacteriophage conjugated with \\u03b1-GalCer; IS, internal standard. The calibration curve obtained from five calibration points of \\u03b1-GalCer STD (62.5, 125, 250, 500, and 1,000 ng/ml) spiked with fixed amount of IS (400 ng/ml) is reported along with equation parameters. (C,D) Dendritic cells (DCs) were cultured with free \\u03b1-GalCer or fd/\\u03b1-GalCer at different doses. After incubation, cells were stained with anti-\\u03b1-GalCer:CD1d antibody and analyzed by flow cytometry. (C) Representative histogram overlay of DCs incubated with free \\u03b1-GalCer or fd/\\u03b1-GalCer at a dose of 130 ng/ml \\u03b1-GalCer or fdWT. (D) Percentage of anti-\\u03b1-GalCer:CD1d positive cells. Mean \\u00b1 SD of two different experiments is reported. **p < 0.01 by unpaired two-tailed Student\\u2019s t-test. (E) The chart represents the IL-2 release of mouse V\\u03b114 invariant natural killer T (iNKT) hybridoma cell line FF13 to \\u03b1-GalCer delivered by phage particles. LPS-free filamentous bacteriophages were conjugated to \\u03b1-GalCer, and presented by mouse DCs to stimulate iNKT hybridoma cells. Free form of \\u03b1-GalCer was used as positive control. Supernatants were diluted 1:10 and assayed in duplicate. Mean \\u00b1 SD is reported, one representative experiment of four is shown.\\nAlpha-GalactosylCeramide rapidly distributed on the pVIII protein thanks to hydrophobic interactions between the hydrophobic domains of the protein and the acyl chains of the lipid. The amount of \\u03b1-GalCer conjugated to the bacteriophage vector was determined by quantitative mass analysis by a UPLC-MS/MS method. To this aim, the lipid was released from the conjugated vector by solvent extraction and the free glycosphingolipid was measured by multiple reaction monitoring (MRM) analysis. \\u03b2-galactosylpalmitoylsphingosine was added as IS before extraction. The organic extract obtained as reported in the Section \\u201cMaterials and Methods,\\u201d contained a measured absolute amount of \\u03b1-GalCer of 0.196 \\u00b5g/ml, indicating that 2.6% of the added galactosphingolipid was effectively loaded onto the phage particles (Figure 1B).\\nfd/\\u03b1-GalCer Ability to Activate iNKT Cells\\nWe assessed the ability of \\u03b1-GalCer loaded on phage particles to be presented by BMDCs and to activate iNKT cells in vitro. BMDCs were incubated with different doses of free \\u03b1-GalCer or bacteriophage particles delivering \\u03b1-GalCer. Cells were stained with a mAb recognizing the \\u03b1-GalCer:CD1d complex and analyzed by flow cytometry. We found that fd/\\u03b1-GalCer is able to be internalized by DCs and to efficiently induce \\u03b1-GalCer presentation on CD1d molecule. In addition, at the dose of 130 ng only the \\u03b1-GalCer carried by fd was displayed on CD1d molecule (Figures 1C,D).\\nMoreover, to investigate the efficiency of the antigen presentation of \\u03b1-GalCer delivered by phage particles on CD1d, we co-cultured fd/\\u03b1-GalCer pulsed BMDCs with the mouse V\\u03b114 iNKT hybridoma cell line FF13. Bacteriophage-vectorized \\u03b1-GalCer was presented by BMDCs, triggering activation of iNKT hybridoma, as assessed by IL-2 release (Figure 1E). According to mass analysis data, it is noteworthy that the highest used dose of bacteriophage (10,000 ng/ml) contains about 26 ng/ml of \\u03b1-GalCer. The IL-2 production by hybridoma cells was due to the \\u03b1-GalCer delivered on phage particles as demonstrated by the lack of capacity to induce IL-2 production from BMDCs pulsed with bacteriophage particles not-conjugated to the glycolipid (fdWT).\\nWe also tested the in vivo ability of bacteriophage/\\u03b1-GalCer conjugates to activate a response, by injecting mice intravenously with 50 \\u00b5g of fd/\\u03b1-GalCer or 5 \\u00b5g of free \\u03b1-GalCer. We found that 3 h after the injection of fd/\\u03b1-GalCer particles, CD3+NK1.1+ cells were activated, as measured by ex vivo analysis of IFN-\\u03b3 production (Figure 2A). It is noteworthy that similar response was induced by the injection of the free \\u03b1-GalCer and fd-conjugated \\u03b1-GalCer even though the amount of lipid administered with the phage particles was less with respect to the free \\u03b1-GalCer used. In fact, mass analysis demonstrated that approximately 130 ng of \\u03b1-GalCer resulted bound to 50 \\u00b5g of bacteriophages. Moreover, only the injection of vectorized glycosphingolipid was able to significantly increase the number of CD3+NK1.1+ in the spleen, compared to free \\u03b1-GalCer injection (Figure 2B).\\nFigure 2. In vitro recall response of mice to intravenous alpha-GalactosylCeramide (\\u03b1-GalCer) administration. Mice (n = 3/group) were injected intravenously with 50 \\u00b5g of fd bacteriophages or 5 \\u00b5g of free \\u03b1-GalCer/mouse or vehicle (phosphate buffered saline, PBS). After 3 h, mice were sacrificed, spleen cells were collected and stained with the reported antibodies. (A) Percentage of INF-\\u03b3-producing CD3+NK1.1+ cells; average + SEM is reported. p < 0.001 by one-way analysis of variance (ANOVA) followed by a Dunnett\\u2019s multiple comparison test. (B) Percentage of CD3+NK1.1+ cells 3 h after intravenous injection of above-mentioned reagents. Average + SEM is reported. p < 0.01 by one-way ANOVA followed by a Dunnett\\u2019s multiple comparison test. (C) Percentage of INF-\\u03b3-producing CD3\\u2212NK1.1+. Average + SEM is reported. p < 0.001 by one-way ANOVA followed by a Dunnett\\u2019s multiple comparison test. *p < 0.05, **p < 0.01, ***p < 0.001. Abbreviation: ns, not significant. (D) In vitro recall response of mice to \\u03b1-GalCer immunization. Mice (n = 3/group) were intravenously injected with 5 \\u00b5g or 200 ng of free \\u03b1-GalCer, 50 \\u00b5g of fd/\\u03b1-GalCer or vehicle alone (phosphate buffered saline, PBS). The day after, splenocytes were cultured with graded doses of \\u03b1-GalCer or medium. After 3 days, proliferation was assessed by [3H] thymidine incorporation. Average + SEM is reported. p < 0.001 by two-way ANOVA and the Bonferroni multiple comparison test. *p < 0.05, **p < 0.01, ***p < 0.001.\\nImportantly, notwithstanding the lower amount of lipid administered with bacteriophage particles, we found that fd/\\u03b1-GalCer induced a higher percentage of NK cells able to produce IFN-\\u03b3 compared to cells isolated from mice injected with free \\u03b1-GalCer (Figure 2C).\\nRecall Response to Intravenous \\u03b1-GalCer Administration\\nIt is known that free \\u03b1-GalCer administered intravenously to mice, causes TCR down regulation on iNKT, and splenocytes from \\u03b1-GalCer-injected mice loose their capacity to proliferate and produce cytokines upon in vitro re-stimulation with \\u03b1-GalCer (13). In agreement with these reports we found that spleen iNKT cells isolated from mice injected with 5 \\u00b5g of free \\u03b1-GalCer were unable to proliferate when re-stimulated in vitro with the same lipid (Figure 2D). Since mass experiments showed that the amounts of \\u03b1-GalCer bound to the phage particles is low, we also used a low dose of free \\u03b1-GalCer in these experiments and similar unresponsiveness was observed (Figure 2D). In contrast, we observed that splenocytes from mice injected with vectorized \\u03b1-GalCer, by conjugation to bacteriophages, were still responsive to \\u03b1-GalCer re-stimulation in vitro in a dose-dependent manner (Figure 2D, empty bars).\\nAntigen-Specific Immune Response Is Increased by the Adjuvant Effect of \\u03b1-GalCer/Bacteriophage Conjugate\\nWe then investigated the adjuvant effect of \\u03b1-GalCer delivered by bacteriophages on the immunogenicity of a displayed antigenic determinant. For this purpose, \\u03b1-GalCer was conjugated on the surface of fd particles displaying the ovalbumin (OVA) determinant SIINFEKL (OVA257\\u2013264).\\nIn Figures 3A,B we show that OVA antigen and \\u03b1-GalCer co-delivered by phage particles are presented by DCs. BMDCs were incubated with fdOVA/\\u03b1-GalCer conjugate, or with fdOVA alone. As controls, DCs were incubated with free \\u03b1-GalCer or synthetic OVA257\\u2013264 peptide in the same amount as estimated on phage surface (see methods). BMDCs pulsed with fdOVA/\\u03b1-GalCer are able to stimulate IL-2 production by either OTI hybridoma cells or iNKT hybridoma cells (Figures 3A,B). Of relevance, iNKT hybridoma cells produced higher amount of IL-2 when stimulated by DCs pulsed with fdOVA\\u03b1\\u2013GalCer with respect to free \\u03b1-GalCer.\\nFigure 3. Adjuvant effect of vectorized delivery of alpha-GalactosylCeramide (\\u03b1-GalCer) on bacteriophages in antigen-specific adaptive immune response. IL-2 release of mouse V\\u03b114 invariant natural killer T hybridoma cell line FF13 (A) or OTI hybridoma cell line (B) in response to \\u03b1-GalCer or OVA SIINFEKL peptide delivered by phage particles. LPS-free fdOVA filamentous bacteriophages were conjugated to \\u03b1-GalCer, and fdOVA or fdOVA/\\u03b1-GalCer were presented by mouse bone marrow-derived dendritic cells to stimulate FF13 or OTI hybridoma cells. Soluble form of \\u03b1-GalCer was used as positive control in (A). Synthetic OVA257\\u2013264 peptide was used as positive control in (B). Supernatants were diluted 1:10 (A) or left undiluted (B) and assayed in duplicate. Mean \\u00b1 SD is reported, one representative experiment of two is shown. (C,D) Group of mice (n = 4/group) were primed (day 0) and boosted (day 14) with fdOVA (SIINFEKL peptide) bacteriophages delivering or not the \\u03b1-GalCer as indicated on the x-axis. As control, mice were inoculated twice with vehicle alone (phosphate buffered saline, PBS). At day 21, splenocytes were isolated and percentage of OVA257\\u2013264-specific CD8+ T cells producing IFN-\\u03b3 (C) and percentage of H2Kb-SIINFEKL dextramer positive CD8+ T cells (D) were evaluated. Average + SEM is reported. Difference were statistical significant by one-way analysis of variance followed by a Dunnett\\u2019s multiple comparison test. *p < 0.05. Abbreviation: ns, not significant.\\nC57BL/6 mice were immunized with bacteriophages expressing OVA257\\u2013264 peptide (fdOVA) or with fdOVA delivering \\u03b1-GalCer (fdOVA/\\u03b1-GalCer) and after 14 days, mice were boosted with the same bacteriophage particles, delivering or not \\u03b1-GalCer. As control, mice were inoculated twice with only vehicle (PBS). On day 21, mice were culled and spleen cells were isolated and assayed for OVA257\\u2013264-specific cell response.\\nWe found that immunization with fdOVA/\\u03b1-GalCer particles developed an OVA257\\u2013264-specific IFN-\\u03b3-secreting T cell response. As illustrated in Figure 3, mice that received two fdOVA/\\u03b1-GalCer administrations, showed increased OVA-specific CD8+ T cells producing IFN-\\u03b3 compared to mice inoculated twice with fdOVA alone (Figures 3C,D).\\nInterestingly, the group of mice injected first with fdOVA and then with fdOVA/\\u03b1-GalCer, showed similar results to two fdOVA/aGalCer injections, indicating that an adjuvant effect of \\u03b1-GalCer administration via bacteriophage particles on the adaptive antigen-specific immune response can also be observed after a single administration of fd vectorized \\u03b1-GalCer. As control, we treated a group of mice first with fdOVA/\\u03b1-GalCer and then with fdOVA, obtaining comparable results to fdOVA/\\u03b1-GalCer-treated group, suggesting that the fdOVA/\\u03b1-GalCer administration either in priming or boosting is able to induce a higher CD8+ response.\\nTherapeutic Vaccination With Bacteriophages in B16 Tumor-Bearing Mice\\nWe also tested the ability of bacteriophage particles coated with \\u03b1-GalCer to mediate protection in a therapeutic anti-tumor vaccination setting. For this purpose, C57BL/6 mice were injected subcutaneously with B16 melanoma cells (day \\u221210), and when tumors were palpable (day 0), mice were treated intratumorally with free \\u03b1-GalCer, fd particles (fdWT), or fd/\\u03b1-GalCer bacteriophages (Figure 4A). Although intratumoral route of administration is not applicable to all the type of cancer, it has been demonstrated that intratumoral administration of drugs offers several advantages over traditional routes of immunization, as reduced systemic toxicity due to lower diffusion in the body and high local concentration, which permits use of smaller amount of drugs (24).\\nFigure 4. Invariant natural killer T response in mice injected intratumorally with fd-vectorized alpha-GalactosylCeramide (\\u03b1-GalCer). Mice were inoculated with B16 melanoma cells and when tumor was palpable, mice were intratumorally injected with free \\u03b1-GalCer, \\u03b1-GalCer delivered by phage particles or vehicle alone, and sacrificed after 24 h. (A) Schematic representation of the experiment schedule. (B) CD3+NK1.1+ cells were evaluated in freshly isolated spleen of injected mice by FACS analysis and staining with anti-NK1.1 and anti-CD3 antibodies. Percentage of NK1.1+CD3+ positive cells \\u00b1 SD is reported. p < 0.01 by one-way analysis of variance (ANOVA) followed by a Dunnett\\u2019s multiple comparison test. **p < 0.01. (C,D) Splenocytes were cultured with graded doses of \\u03b1-GalCer. After 3 days, proliferation was assessed by [3H] thymidine incorporation (C), and culture supernatants were evaluated for IL-2 release by ELISA (D). Average + SEM is reported. p < 0.001 by two-way ANOVA and the Bonferroni multiple comparison test. *p < 0.05, ***p < 0.001.\\nWe found that the intratumoral injection of \\u03b1-GalCer vectorized on phage particles (fd/\\u03b1-GalCer) was able to increase the number of iNKT cells, as demonstrated by FACS analysis on spleen cells 24 h later, in comparison with the number of iNKT cells observed in the spleens isolated from mice treated with free \\u03b1-GalCer (Figure 4B). We also found that spleen cells derived from mice primed in vivo with fd/\\u03b1-GalCer still retained the capability to proliferate and to produce cytokines like IL-2 when re-stimulated in vitro with increasing doses of free \\u03b1-GalCer (Figures 4C,D). This response was not observed with cells isolated from mice injected in vivo with free \\u03b1-GalCer, and also re-stimulated in vitro with free \\u03b1-GalCer. Moreover, we used for i.t. injection a dose of free \\u03b1-GalCer (130 ng), normalized according to the amount retained in the phage particles. Similar unresponsiveness to in vitro restimulation was observed (Figure 4D).\\nAn important anti-tumor effect was also clearly assessed. fd/\\u03b1-GalCer treatment resulted in a significant delay in the tumor growth, while administration of free \\u03b1-GalCer or fd wild-type bacteriophages were less efficient in delaying tumor growth (Figures 5A,B). No significant differences were noted between groups of mice treated with fdWT or free \\u03b1-GalCer. This protective effect was confirmed by the longer survival of mice receiving fd/\\u03b1-GalCer (Figure 5C).\\nFigure 5. Therapeutic vaccination with filamentous bacteriophages fd/alpha-GalactosylCeramide (\\u03b1-GalCer) is protective against a subcutaneous tumor challenge. C57BL/6 mice (n = 5/group) were engrafted with B16 melanoma cell line and when tumor was palpable, mice were injected two times subcutaneously into the tumor with phosphate buffered saline, free \\u03b1-GalCer, fdWT, or fd/\\u03b1-GalCer. (A) The chart shows the mean tumor size reached in each group. (B) Tumor volumes recorded at 9th day after vaccination are reported as box plot, showing the group median, quartiles, and extreme values. Median values are represented by the horizontal line. *p < 0.05 by one-way ANOVA followed by a Dunnett\\u2019s multiple comparison test. (C) Kaplan\\u2013Meier survival curves of B16-engrafted mice during the experiment. Differences among survival curves of \\u03b1-GalCer-treated and fd/\\u03b1-GalCer-treated mice are statistically different (p < 0.01) by log-rank (Mantel\\u2013Cox) test.\\nFinally, to investigate the capacity of fd/\\u03b1-GalCer to induce a tumor-specific adaptive response in a vaccination model, mice were injected with hybrid bacteriophages co-expressing the OVA257\\u2013264 SIINFEKL peptide in the presence or absence of conjugated \\u03b1-GalCer. C57BL/6 mice were engrafted (on day \\u221210), with B16 melanoma cells engineered for the expression of ovalbumin protein. After tumor engraftment, mice received intratumorally fdOVA, fdOVA/\\u03b1-GalCer, or fd/\\u03b1-GalCer on day 0. The injection was repeated on day 5 and tumor growth was assessed over time (Figure 6A).\\nFigure 6. Filamentous bacteriophage fd co-delivering tumor antigen and alpha-GalactosylCeramide (\\u03b1-GalCer) induces a stronger anti-tumor response. Mice (n = 5/group) were challenged with B16-OVA expressing melanoma cells. When tumor was palpable (day 0) mice were immunized intratumorally with fd/\\u03b1-GalCer, fdOVA/\\u03b1-GalCer, fdOVA, free \\u03b1-GalCer, or with phosphate buffered saline. The injection was repeated at day 5, and tumor volume was monitored with a caliper over time. (A) Schematic representation of the experiment. (B) Tumor growth recorded during experiment. The chart shows the mean tumor size reached in each group. Line for each group ends once more than half of the group is culled. (C) Box plot of tumor volumes recorded at day 12 post vaccination. Median values are represented by the horizontal line. p < 0.001 by one-way analysis of variance (ANOVA) followed by a Dunnett\\u2019s multiple comparison test. ***p < 0.001 (D) Kaplan\\u2013Meier survival curves of B16-engrafted mice during the experiment. p < 0.05 between fdOVA/\\u03b1-GalCer and free \\u03b1-GalCer-treated mice group by log-rank (Mantel\\u2013Cox) test. (E-H) Groups of mice treated as above were sacrificed at day 12 post vaccination and tumor infiltrating lymphocytes were evaluated. Mean percentages (E,F) and absolute numbers (G,H) of CD8+ T cells gated on 7-AAD negative cells (E,G) and of H2Kb-SIINFEKL dextramer positive gated on CD8+ T cells (F,H) are reported. p < 0.01 by one-way ANOVA followed by a Dunnett\\u2019s multiple comparison test. *p < 0.05, **p < 0.01. Abbreviation: ns, not significant.\\nPriming and boosting mice with fdOVA/\\u03b1-GalCer efficiently protected mice, as demonstrated by measuring tumor growth (Figures 6B,C). We observed a delay in tumor growth in mice immunized with fdOVA/\\u03b1-GalCer phage particles, in comparison with mice treated with fdOVA or free \\u03b1-GalCer.\\nMice engrafted with B16-OVA cells after 2 injections with fdOVA/\\u03b1-GalCer showed 80% survival rate at the end of the experiment, in comparison with 40% survival of mice treated twice with fd/\\u03b1-GalCer in the absence of the specific OVA257\\u2013264 peptide (Figure 6D). In this latter group of mice, a variable response was observed, suggesting the presence of responder and not responder mice (Figure S1 in Supplementary Material). No increased survival was observed in mouse groups treated with two injections of free \\u03b1-GalCer or fdOVA particles not delivering the galactosylceramide.\\nThe analysis of tumor-infiltrating cells at day 12 from intratumor vaccination showed the presence of an increased percentage and absolute number of CD8+ T cells within the tumor bed in mice injected with fdOVA/\\u03b1-GalCer compared to free \\u03b1-GalCer and fdOVA-injected groups (Figures 6E,G), while the percentage of OVA257\\u2013264 specific T cells, as stained by SIINFEKL-MHC dextramer, was the same in fdOVA or fdOVA/\\u03b1-GalCer-treated cells, both in tumors and in spleens (Figure 6F; Figure S2 in Supplementary Material). Importantly, the absolute number of tumor infiltrating OVA257\\u2013264-specific T cells was higher in mice treated with fdOVA/\\u03b1-GalCer with respect to mice which received fdOVA (Figure 6H). These data indicate that fdOVA/\\u03b1-GalCer bacteriophage particles are able to induce a general expansion of tumor-infiltrating T cells, probably due to the activity of \\u03b1-GalCer, and then, an expansion of OVA257\\u2013264-specific cells, which are boosted by the phage particles co-displaying the OVA257\\u2013264 antigenic peptide.\\nDiscussion\\nFilamentous bacteriophage is a well characterized, powerful antigen delivery system that has been demonstrated to evoke long and sustained adaptive immune responses toward the antigens displayed on its surface (17\\u201319). In addition, bacteriophage is able to activate also innate immune responses mainly via toll-like receptor pathways (23, 25).\\nTo further improve the immune responses elicited by bacteriophage particles delivering antigenic peptides, in this work we exploited the capacity of the partially hydrophobic pVIII structural protein expressed on the phage surface to interact and bind the immunostimulating glycolipid \\u03b1-GalCer. We found that the activation of iNKT via fd/\\u03b1-GalCer, also rapidly induced transactivation of NK cells, which is considered to enhance the anti-tumor effect of \\u03b1-GalCer treatment (26).\\nFurthermore, iNKT cells activated in vivo by fd/\\u03b1-GalCer remained responsive to repeated stimulations, in contrast to long-term anergy observed upon free \\u03b1-GalCer injection (13). The reason of iNKT unresponsiveness remains unclear, although it is known that injection of free \\u03b1-GalCer rapidly induces the overexpression of inhibitory co-stimulatory programmed death (PD)-1 receptor (27). Nevertheless, blockage of PD-1/PD-L1 interaction does not restore full iNKT functions, indicating the presence of additional inhibitory mechanisms.\\nCD1d is expressed by many types of APCs which might all stimulate iNKT cells in principle. However, the APCs expressing appropriate levels of costimulatory molecules are those more efficient in inducing full activation of iNKT cells (13, 15, 28, 29). Studies involving conditional depletion of CD1d molecule on selected types of APC have demonstrated that different APCs have different capability to prime iNKT cells to glycolipid-specific responses.\\nIn particular, when \\u03b1-GalCer was presented by B cells a state of unresponsiveness was observed in iNKT cells (13, 30). In contrast, DCs resulted to be very active in stimulating iNKT cells with \\u03b1-GalCer (31), and anergy could be further avoided by injection of \\u03b1-GalCer-pulsed autologous DCs (15, 32). However, the dose of the injected lipid may be relevant in the induction of iNKT unresponsiveness by administration of \\u03b1-GalCer-pulsed DCs (33). In addition, ex vivo manipulation of autologous human DCs appears to be a very expensive and time-consuming procedure.\\nFor this reason, vectorizing \\u03b1-GalCer on the surface of nanoparticles might be an interesting strategy to efficiently deliver the galactosylceramide to DCs in order to decrease the possibility of iNKT anergy. Indeed, nanoparticles are preferentially uptaken by DCs, and \\u03b1-GalCer-coupled nanoparticles have been described to activate DCs without inducing anergy (34). The filamentous bacteriophage is composed by repeated and ordered subunits and for its particulate nature can be assimilated to a nature-made nanoparticle, and its use prevent unresponsiveness of iNKT.\\nImportantly, the simultaneous delivery of glycolipid and a MHC class-I-presented peptide to the same APC enhanced CTL cross priming as compared to separate administration of \\u03b1-GalCer and protein antigen (35). We confirmed in our model that the co-delivery of tumor peptide and \\u03b1-GalCer by phage particles increased the induction of antigen-specific CD8+ T cells.\\nThe exact mechanism how \\u03b1-GalCer facilitates priming and expansion of OVA-specific CD8+ T cells remain to be investigated. We envisage two, non-alternative mechanisms. The first is due to the recruitment of iNKT cells next to DCs that internalized fdOVA/\\u03b1-GalCer. Recruited iNKT cells may immediately induce maturation of DCs and also the optimization of their Ag-presentation and priming capacities of CD8+ cells. These effects might be facilitated also by the large numbers of iNKT cells and their immediate response (within a few minutes) to \\u03b1-GalCer stimulation. The activation of iNKT cells might explain the reports in which it was found superior to those elicited by TLR agonists in inducing specific anti-tumor CD8+ T cell responses (36).\\nThe second mechanism may be ascribed to the fact that bacteriophage, after internalization localizes into the endocytic cellular compartments (23), where loading of glycolipids on CD1d molecule also occurs.\\nOverall, our findings suggest that filamentous bacteriophage is a powerful tool to deliver immunogenic lipid molecules together with antigenic peptides.\\nGrowing interest is focused on the role of lipids as key stimulatory molecules of the immune response. Either naturally derived non-self lipids, microbial lipids (i.e., peptidoglycan, lipoteichoic acid, and lipopolysaccharides) or self lipids (i.e., phospholipids, sphingomyelin, and oxidized lipoproteins) are known to trigger the immune system (37\\u201341), and co-administration of stimulatory lipid molecules and antigens of interest in vaccination seems a promising manner to formulate novel vaccines including anti-tumor vaccines.\\nThe use of filamentous bacteriophages complexed with lipid molecules might represent an efficient, fast, and cheap way to generate new vaccines capable of efficient priming and expansion of T cells and also devoid of chemical adjuvants. Our approach can be further improved by developing phage-based antigen delivery systems loaded with \\u03b1-GalCer and simultaneously expressing antigens target of the vaccine and molecules delivering the phage to specific cell subsets. Bacteriophages are generally considered safe (42, 43) and cheap to prepare and purify (44, 45), thereby they might represent new tools for the next generation of vaccines.\\nEthics Statement\\nThis study was carried out in accordance with European Union Laws and guidelines (European Directive 2010/63/EU). The study was approved by our institutional review board and the animal procedures (i.e., immunization, sacrifice) were performed according to rules approved by the ethics committee (permission no. 79/2014-PR).\\nAuthor Contributions\\nConceptualization: RS and PB. Methodology: RS, AC, and PB. Investigation: RS, LA, PB, DC, LG, and AC. Writing: RS and PB. Review and editing: LA and AC. Funding acquisition: PB. Supervision: PB. All authors read and approved the final version of the manuscript.\\nConflict of Interest Statement\\nRS and PB are inventors of the patent no. PCT/IB2018/050525 \\u201cPhage conjugates and uses thereof.\\u201d\\nAcknowledgments\\nThe IGB-IBP FACS Facility and the IGB \\u201cA. Buzzati-Traverso\\u201d Animal House Facility at CNR, Naples are gratefully acknowledged. We also thank Gennaro De Libero (Department of Biomedicine, University of Basel and University Hospital Basel, Basel, Switzerland) for providing V\\u03b114 iNKT hybridoma FF13 cells and for helpful scientific advices and for critically reviewing the manuscript and Angelo Fontana [Institute of Biomolecular Chemistry (ICB), CNR, Pozzuoli (NA), Italy] for helpful discussion.\\nFunding\\nThis work was supported by PO FESR Regione Campania: \\u201cBando realizzazione della rete delle biotecnologie in Campania\\u2013progetto TIMING.\\u201d\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at https://www.frontiersin.org/articles/10.3389/fimmu.2018.01496/full#supplementary-material.\\nReferences\\n1. Brennan PJ, Brigl M, Brenner MB. Invariant natural killer T cells: an innate activation scheme linked to diverse effector functions. Nat Rev Immunol (2013) 13(2):101\\u201317. doi:10.1038/nri3369\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n2. Kawano T, Cui J, Koezuka Y, Toura I, Kaneko Y, Motoki K, et al. CD1d-restricted and TCR-mediated activation of valpha14 NKT cells by glycosylceramides. Science (1997) 278(5343):1626\\u20139. doi:10.1126/science.278.5343.1626\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n3. Natori T, Morita M, Akimoto K, Koezuka Y. Agelasphins, novel antitumor and immunostimulatory cerebrosides from the sponge Agelas mauritianus. Tetrahedron (1994) 50:2771. doi:10.1016/S0040-4020(01)86991-X\\nCrossRef Full Text | Google Scholar\\n4. Morita M, Motoki K, Akimoto K, Natori T, Sakai T, Sawa E, et al. Structure-activity relationship of \\u03b1-galactosylceramides against B16-bearing mice. J Med Chem (1995) 38:2176. doi:10.1021/jm00012a018\\nCrossRef Full Text | Google Scholar\\n5. Kobayashi E, Motoki K, Uchida T, Fukushima H, Koezuka Y. KRN7000, a novel immunomodulator, and its antitumor activities. Oncol Res (1995) 7:529.\\nPubMed Abstract | Google Scholar\\n6. Michel ML, Keller AC, Paget C, Fujio M, Trottein F, Savage PB, et al. Identification of an IL-17-producing NK1.1(neg) iNKT cell population involved in airway neutrophilia. J Exp Med (2007) 204(5):995\\u20131001. doi:10.1084/jem.20061551\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n7. Coquet JM, Kyparissoudis K, Pellicci DG, Besra G, Berzins SP, Smyth MJ, et al. IL-21 is produced by NKT cells and modulates NKT cell activation and cytokine production. J Immunol (2007) 178(5):2827\\u201334. doi:10.4049/jimmunol.178.5.2827\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n8. Paget C, Ivanov S, Fontaine J, Renneson J, Blanc F, Pichavant M, et al. Interleukin-22 is produced by invariant natural killer T lymphocytes during influenza A virus infection: potential role in protection against lung epithelial damages. J Biol Chem (2012) 287(12):8816\\u201329. doi:10.1074/jbc.M111.304758\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n9. Cavallari M, Stallforth P, Kalinichenko A, Rathwell DC, Gronewold TM, Adibekian A, et al. A semisynthetic carbohydrate-lipid vaccine that protects against S. pneumoniae in mice. Nat Chem Biol (2014) 10(11):950\\u20136. doi:10.1038/nchembio.1650\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n10. Nakagawa R, Motoki K, Ueno H, Iijima R, Nakamura H, Kobayashi E, et al. Treatment of hepatic metastasis of the colon26 adenocarcinoma with an alpha-galactosylceramide, KRN7000. Cancer Res (1998) 58(6):1202\\u20137.\\nPubMed Abstract | Google Scholar\\n11. Hayakawa Y, Rovero S, Forni G, Smyth MJ. Alpha\\u2212galactosylceramide (KRN7000) suppression of chemical- and oncogene-dependent carcinogenesis. Proc Natl Acad Sci U S A (2003) 100(16):9464\\u20139. doi:10.1073/pnas.1630663100\\nCrossRef Full Text | Google Scholar\\n12. Giaccone G, Punt CJ, Ando Y, Ruijter R, Nishi N, Peters M, et al. A phase I study of the natural killer T-cell ligand alpha\\u2212galactosylceramide (KRN7000) in patients with solid tumors. Clin Cancer Res (2002) 8(12):3702\\u20139.\\nGoogle Scholar\\n13. Parekh VV, Wilson MT, Olivares-Villag\\u00f3mez D, Singh AK, Wu L, Wang C-RR, et al. Glycolipid antigen induces long-term natural killer T cell anergy in mice. J Clin Invest (2005) 115:2572\\u201383. doi:10.1172/JCI24762\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n15. Fujii S, Shimizu K, Kronenberg M, Steinman RM. Prolonged IFN-gamma\\u2212producing NKT response induced with alpha\\u2212galactosylceramide-loaded DCs. Nat Immunol (2002) 3(9):867\\u201374. doi:10.1038/ni827\\nCrossRef Full Text | Google Scholar\\n16. Faveeuw C, Trottein F. Optimization of natural killer T cell-mediated immunotherapy in cancer using cell-based and nanovector vaccines. Cancer Res (2014) 74(6):1632\\u20138. doi:10.1158/0008-5472.CAN-13-3504\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n17. Sartorius R, Bettua C, D\\u2019Apice L, Caivano A, Trovato M, Russo D, et al. Vaccination with filamentous bacteriophages targeting DEC-205 induces DC maturation and potent anti-tumor T-cell responses in the absence of adjuvants. Eur J Immunol (2011) 41(9):2573\\u201384. doi:10.1002/eji.201141526\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n18. De Berardinis P, Sartorius R, Caivano A, Mascolo D, Domingo GJ, Del Pozzo G, et al. Use of fusion proteins and procaryotic display systems for delivery of HIV-1 antigens: development of novel vaccines for HIV-1 infection. Curr HIV Res (2003) 1(4):441\\u20136. doi:10.2174/1570162033485168\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n19. Sartorius R, Pisu P, D\\u2019Apice L, Pizzella L, Romano C, Cortese G, et al. The use of filamentous bacteriophage fd to deliver MAGE-A10 or MAGE-A3 HLA\\u2212A2-restricted peptides and to induce strong antitumor CTL responses. J Immunol (2008) 180(6):3719\\u201328. doi:10.4049/jimmunol.180.6.3719\\nCrossRef Full Text | Google Scholar\\n20. Stopar D, Spruijt RB, Wolfs CJ, Hemminga MA. Protein-lipid interactions of bacteriophage M13 major coat protein. Biochim Biophys Acta (2003) 1611:5\\u201315. doi:10.1016/S0005-2736(03)00047-6\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n21. Sch\\u00fcmann J, Facciotti F, Panza L, Michieletti M, Compostella F, Collmann A, et al. Differential alteration of lipid antigen presentation to NKT cells due to imbalances in lipid metabolism. Eur J Immunol (2007) 37:1431\\u201341. doi:10.1002/eji.200737160\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n22. Caivano A, D\\u2019Apice L, Tiberio C, Prisco A, Acuto O, Guardiola J, et al. Design of cassette vectors permitting cloning of all types of human TCR variable alpha and beta regions. J Immunol Methods (2001) 255:125\\u201334. doi:10.1016/S0022-1759(01)00420-3\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n23. Sartorius R, D\\u2019Apice L, Trovato M, Cuccaro F, Costa V, De Leo MG, et al. Antigen delivery by filamentous bacteriophage fd displaying an anti-DEC-205 single-chain variable fragment confers adjuvanticity by triggering a TLR9-mediated immune response. EMBO Mol Med (2015) 7(7):973\\u201388. doi:10.15252/emmm.201404525\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n25. D\\u2019Apice L, Costa V, Sartorius R, Trovato M, Aprile M, De Berardinis P. Stimulation of innate and adaptive immunity by using filamentous bacteriophage fd targeted to DEC-205. J Immunol Res (2015) 2015:585078. doi:10.1155/2015/585078\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n26. Smyth MJ, Crowe NY, Pellicci DG, Kyparissoudis K, Kelly JM, Takeda K, et al. Sequential production of interferon-gamma by NK1.1(+) T cells and natural killer cells is essential for the antimetastatic effect of alpha-galactosylceramide. Blood (2002) 99(4):1259\\u201366. doi:10.1182/blood.V99.4.1259\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n27. Parekh VV, Lalani S, Kim S, Halder R, Azuma M, Yagita H, et al. PD-1/PD-L blockade prevents anergy induction and enhances the anti-tumor activities of glycolipid-activated invariant NKT cells. J Immunol (2009) 182(5):2816\\u201326. doi:10.4049/jimmunol.0803648\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n28. Fujii S, Shimizu K, Steinman RM, Dhodapkar MV. Detection and activation of human Valpha24+ natural killer T cells using alpha-galactosyl ceramide-pulsed dendritic cells. J Immunol Methods (2003) 272(1\\u20132):147\\u201359. doi:10.1016/S0022-1759(02)00497-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n29. Bezbradica JS, Stanic AK, Matsuki N, Bour-Jordan H, Bluestone JA, Thomas JW, et al. Distinct roles of dendritic cells and B cells in Va14Ja18 natural T cell activation in vivo. J Immunol (2005) 174(8):4696\\u2013705. doi:10.4049/jimmunol.174.8.4696\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n30. Bai L, Constantinides MG, Thomas SY, Reboulet R, Meng F, Koentgen F, et al. Distinct APCs explain the cytokine bias of \\u03b1-galactosylceramide variants in vivo. J Immunol (2012) 188(7):3053\\u201361. doi:10.4049/jimmunol.1102414\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n31. Chang DH, Osman K, Connolly J, Kukreja A, Krasovsky J, Pack M, et al. Sustained expansion of NKT cells and antigen-specific T cells after injection of alphagalactosyl-ceramide loaded mature dendritic cells in cancer patients. J Exp Med (2005) 201:1503\\u201317. doi:10.1084/jem.20042592 [Erratum in: J Exp Med (2007) 204:2487],\\nCrossRef Full Text | Google Scholar\\n32. Ishikawa A, Motohashi S, Ishikawa E, Fuchida H, Higashino K, Otsuji M, et al. A phase I study of alpha-galactosylceramide (KRN7000)-pulsed dendritic cells in patients with advanced and recurrent non-small cell lung cancer. Clin Cancer Res (2005) 11(5):1910\\u20137. doi:10.1158/1078-0432.CCR-04-1453\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n33. Iyoda T, Ushida M, Kimura Y, Minamino K, Hayuka A, Yokohata S, et al. Invariant NKT cell anergy is induced by a strong TCR-mediated signal plus co-stimulation. Int Immunol (2010) 22(11):905\\u201313. doi:10.1093/intimm/dxq444\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n34. Thapa P, Zhang G, Xia C, Gelbard A, Overwijk WW, Liu C, et al. Nanoparticle formulated alpha-galactosylceramide activates NKT cells without inducing anergy. Vaccine (2009) 27(25\\u201326):3484\\u20138. doi:10.1016/j.vaccine.2009.01.047\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n35. Semmling V, Lukacs-Kornek V, Thaiss CA, Quast T, Hochheiser K, Panzer U, et al. Alternative cross-priming through CCL17-CCR4-mediated attraction of CTLs toward NKT cell-licensed DCs. Nat Immunol (2010) 11(4):313\\u201320. doi:10.1038/ni.1848\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n36. D\\u00f6len Y, Kreutz M, Gileadi U, Tel J, Vasaturo A, van Dinther EA, et al. Co-delivery of PLGA encapsulated invariant NKT cell agonist with antigenic protein induce strong T cell-mediated antitumor immune responses. Oncoimmunology (2015) 5(1):e1068493. doi:10.1080/2162402X.2015.1068493\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n37. Dowds CM, Kornell SC, Blumberg RS, Zeissig S. Lipid antigens in immunity. Biol Chem (2014) 395(1):61\\u201381. doi:10.1515/hsz-2013-0220\\nCrossRef Full Text | Google Scholar\\n39. Manzo E, Cutignano A, Pagano D, Gallo C, Barra G, Nuzzo G, et al. A new marine-derived sulfoglycolipid triggers dendritic cell activation and immune adjuvant response. Sci Rep (2017) 7(1):6286. doi:10.1038/s41598-017-05969-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n40. Perrin-Cocon L, Diaz O, Andr\\u00e9 P, Lotteau V. Modified lipoproteins provide lipids that modulate dendritic cell immune function. Biochimie (2013) 95(1):103\\u20138. doi:10.1016/j.biochi.2012.08.006\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n41. Zanoni I, Tan Y, Di Gioia M, Broggi A, Ruan J, Shi J, et al. An endogenous caspase-11 ligand elicits interleukin-1 release from living dendritic cells. Science (2016) 352(6290):1232\\u20136. doi:10.1126/science.aaf3036\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\n43. Samoylova TI, Norris MD, Samoylov AM, Cochran AM, Wolfe KG, Petrenko VA, et al. Infective and inactivated filamentous phage as carriers for immunogenic peptides. J Virol Methods (2012) 183(1):63\\u20138. doi:10.1016/j.jviromet.2012.03.032\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02239\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02239/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:50:58.165Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2017.02239\",\n",
      "            \"title\": \"Frontiers | Prion-Like Domains in Phagobiota\",\n",
      "            \"description\": \"Prions are molecules characterized by self-propagation, which can undergo a conformational switch leading to the creation of new prions. Prion proteins have originally been associated with the development of mammalian pathologies; however, recently they have been shown to contribute to the environmental adaptation in a variety of prokaryotic and eukaryotic organisms. Bacteriophages are widespread and represent the important regulators of microbiota homeostasis and have been shown to be diverse across various bacterial families. Here, we examined whether bacteriophages contain prion-like proteins and whether these prion-like protein domains are involved in the regulation of homeostasis. We used a computational algorithm, prion-like amino acid composition (PLAAC), to detect prion-like domains in 370,617 publicly available bacteriophage protein sequences, which resulted in the identification of 5040 putative prions. We analyzed a set of these prion-like proteins, and observed regularities in their distribution across different phage families, associated with their interactions with the bacterial host cells. We found that prion-like domains could be found across all phages of various groups of bacteria and archeae. The results obtained in this study indicate that bacteriophage prion-like proteins are predominantly involved in the interactions between bacteriophages and bacterial cell, such as those associated with the attachment and penetration of bacteriophage in the cell, and the release of the phage progeny. These data allow the identification of phage prion-like proteins as novel regulators of the interactions between bacteriophages and bacterial cells.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nPrions were demonstrated to play important roles in eukaryotes and prokaryotes, and they have been intensively studied during the last decade (Alberti et al., 2009). Prion proteins (PrPs) are characterized by self-propagation, undergoing a conformational switch from one conformational state to another, which leads to the creation of new prions (Bolton et al., 1982; Telling et al., 1995). Pathologically, prions are characterized by a process in which the infectious form of prion (PrPSc) interacts with the endogenous PrPs, catalyzing the transformation of the endogenous molecule into misfolded PrPSc aggregates (Ma and Lindquist, 2002). This was first observed in mammals, and the pathological deposition of insoluble protein aggregates was shown to be associated with the development of a number of human diseases, including scrapie, Creutzfeldt-Jakob disease, Alzheimer\\u2019s disease, Parkinson\\u2019s disease, amyloidosis, and other (Collinge, 2001; Wemheuer et al., 2017).\\nOne of the best-known forms of PrPs are amyloid proteins that, in addition to the role of infectious agents, were shown to be an important component of many physiological processes in eukaryotes and prokaryotes (Westergard et al., 2007; Sabate et al., 2015; Walker et al., 2016).\\nAlthough the amino acid sequences and function of these evolutionarily unrelated proteins are diverse, all amyloid fibrils can be misfolded into distinct conformers, forming highly ordered cross-\\u03b2 structures (Cheng et al., 2012).\\nRecently, these highly ordered parallel or antiparallel amyloid \\u03b2-sheets were identified in yeasts and bacteria, suggesting that they may play an important physiological role (de Groot et al., 2012; Angarica et al., 2013; Malinovska et al., 2013). Unique mechanical and biological properties of amyloid sheets have multiple functions in a variety of bacterial species, and the secretion of amyloid proteins represents an important step during biofilm formation (Chapman et al., 2002; Fowler et al., 2007; Blanco et al., 2012). To discriminate between the physiological and pathological roles of amyloid, these forms that are involved in physiological process have been termed the functional amyloid (Chiti and Dobson, 2006). Therefore, the propagation of prions was shown to enable environmental adaptation, the activation of stress response, and in yeasts these molecules play important roles in the preservation of long-term memory (Shorter and Lindquist, 2005; Suzuki et al., 2012). The aggregation of PrPs is an amino-acid sequence-specific process, but the mechanism underlying the de novo appearance of prions remains unclear. PrPs are rich in asparagine (Q) and glutamine (N) domains, which was exploited for the development of an algorithm aimed at the identification of candidate prionogenic domains (PrDs) (Michelitsch and Weissman, 2000; Alberti et al., 2009). Currently, the most widespread PrD prediction algorithm is based on the hidden Markov model (HMM), the probabilistic sequence model based on the maximum likelihood estimation (Eddy, 1998; Angarica et al., 2013).\\nThe HMM can be found at the core of several bioinformatic approaches to the statistical representation of prion domains, which allow the scoring of protein sequences according to the likelihood that these proteins are prions (Toombs et al., 2010; Lancaster et al., 2014). To that end, the log-likelihood ratio (LLR) score is used to evaluate the sets of amino-acid interactions, with a large LLR indicating a high similarity between the examined interaction sets, whereas the LLR is nearing zero when comparing the sets of random interactions. Using these web-based algorithms, prion-forming domains were extensively studied in mammalian, bacterial, and fungal proteomes, demonstrating that prions are common in different organisms, and represent important global regulators (Iglesias et al., 2015; Yuan and Hochschild, 2017).\\nIn prokaryotes, PrPs were shown to play important roles in molecular transport, secretion, cell wall development, and other processes (Blanco et al., 2012; Yuan et al., 2014). Moreover, prion determinants were shown to participate in the inter-kingdom communication between prokaryotes and eukaryotes, leading to the alterations in the amyloidogenesis in Caenorhabditis elegans following its colonization with amyloid-producing bacteria (Chen et al., 2016).\\nHowever, the prion-like domains in viruses, in particular the bacterial viruses, bacteriophages, have not been thoroughly examined. Bacteriophages are ubiquitous organisms that are found almost everywhere, from the oceans, with the estimated 1031 viral particles, to mammalians, with over 1015 bacteriophages in the adult human gut (Dupuis et al., 2013; Dalmasso et al., 2014). The life cycle of bacteriophages can be lytic, resulting in bacterial death and de novo assembly of virions, or lysogenic, in which bacteriophage DNA is integrated into the bacterial genome. Both bacteriophage life cycles are important in the regulation of bacterial population. Phage-mediated horizontal gene transfer, occurring during the lysogenic cycle, is especially important for the bacterial ecological adaptation and is a common mechanism underlying the spreading of antibiotic resistance (Ochman et al., 2000). Moreover, we previously demonstrated that the bacteriophages, by altering mammalian gut flora, can induce an increase in gut permeability and facilitate endotoxemia, indirectly affecting the vital activities of microorganisms (Tetz and Tetz, 2016; Tetz et al., 2017). Previous studies revealed a correlation between the increased gut permeability and the development of a variety of human pathologies, including diabetes, neurodegenerative, and cardiovascular diseases. Therefore, the role of bacteriophages as the potential indirect causative agent inducing the development of these diseases requires further investigation (Bischoff et al., 2014).\\nHere, we studied in detail the putative prion domains in bacteriophages. We employed HHM algorithm for the analysis of bacteriophage proteomes, using the UniProtKB database. To the best of our knowledge, this is the most extensive effort aimed at the identification of candidate PrD sequences among bacteriophages. Furthermore, we analyzed the trends in the distribution of PrDs in different protein families and the correlations between bacteriophage families and the host bacterial host. The present predictive approach for the first time uncovers a large set of putative prionogenic proteins whose further experimental characterization might contribute significantly to understanding bacteriophages biology.\\nMaterials and Methods\\nProtein Sequences\\nTo identify the PrDs present in bacteriophage proteomes, protein sequences were obtained from the UniProt KnowledgeBase (Swiss-Prot and TrEMBL) (UniProt Consortium, 2012).\\nThe protein functions were manually curated using the information from the UniProt database, the National Center for Biotechnology Information (NCBI) protein sequence database1, and the literature.\\nIdentification of PrDs in Bacteriophage Proteomes\\nThe presence of PrDs was analyzed in all bacteriophage proteins using the prion-like amino acid composition (PLAAC) prion prediction program based on the HMM and trained using the known PrDs, by identifying the compositional bias toward N and Q residues. By employing a cutoff of 0.003 LLR, we identified 5040 PrDs (Supplementary Table S1). The enrichment values obtained for the proteins detected in the bacteriophage family subsets were compared with the total number of analyzed proteins.\\nWe analyzed the regularities in the likelihood of the identified PrDs to be prions, their distribution among bacteriophage families, host bacteria, and protein functions. The obtained data were used to generate a heatmap with R-statistical computing2, using the \\u201clevelplot\\u201d package. The color key indicates a range between the lowest (white) and the highest (red) LLR values.\\nStatistical Analyses\\nAll statistical analyses were performed using the statistics package Statistica for Windows (version 5.0). Data were compared between bacteriophage families by using a \\u03c72 test or the Fisher\\u2019s exact test. Significant differences were calculated using one-way analysis of variance (ANOVA) with multiple comparisons and a standard confidence interval of 95%. A value of p < 0.001 was considered statistically significant in all tests. Correlations between the size of bacteriophage proteome and PrD enrichment were calculated using the Spearman\\u2019s rank correlation coefficient test.\\nResults\\nIdentification of PrDs in Bacteriophage Proteomes\\nTo identify the PrDs in bacteriophages using the PLAAC algorithm we analyzed 2111 bacteriophage proteomes comprising a total of 370,617 proteins retrieved from the UniProtKB database. We detected 5040 PrPs with the LLRs higher than 0.003 (1.35% of the total bacteriophage protein dataset) (Lancaster et al., 2014).\\nIn Table 1, the trends in the PrD distribution across different bacteriophage families are presented. PrD frequency ranged from 1.16 to 4.49% when analyzing bacteriophage families with at least 400 sequenced proteins. Due to the low representation of archaeal phages, it was not possible to compare them with the bacteriophage proteins with sufficient reliability. Therefore, we focused on the analysis of the PrDs in bacteriophages. Within Caudovirales, the most well-studied order of phages, the highest numbers of proteins containing the PrDs when analyzing bacteriophage families with more than 10,000 sequenced proteins were found among Podoviridae, with over 1.74% of the total Podoviridae proteins were shown to harbor PrDs (p < 0.001), constituting 4.21% of the total detected PrDs among Caudovirales (Table 1 and Supplementary Table S1).\\nTABLE 1. Summary of the prion predicted in different bacteriophage families.\\nThe analysis shown in Supplementary Table S1 demonstrates an interesting trend of representation of prionogenic domains among different bacteriophages families and host organisms.\\nNext, we determined the patterns of correlation between bacteriophage families and LLR scores (Figure 1 and Supplementary Table S2).\\nFIGURE 1. Log-likelihood ratio (LLR) score showing the predicted putative PrDs across bacteriophage families. Results were analyzed using one-way ANOVA.\\nThe average numbers of LLRs between three most abundant bacteriophage families, Myoviridae, Podoviridae, and Siphoviridae, were similar, with the highest LLR scores obtained for Podoviridae family, with an average LLR of 7.78 (p < 0.001) (Supplementary Tables S3A\\u2013C).\\nUnder the same conditions, we compared the LLRs between the remaining bacteriophage families, and Myoviridae were shown to have the highest number of PrDs, with the LLR score over 30. We identified 114 PrDs with the medium LLR score over 20 within Myoviridae family, yielding 3.29% of the total number of PrDs identified in this bacteriophage family (Supplementary Figure S1).\\nThe same trend was observed when analyzing top 100 scoring PrDs. Over 60% of these 100 PrDs were derived from Myoviridae bacteriophages, with the medium LLR score of 40.55 (Supplementary Table S4).\\nAfterward, we aimed to select bacteriophage proteins in bacteria that may have a greater prion-forming potential. To this end, we analyzed the correlation between LLR score of the PrD proteins and the host bacteria. The average LLR score varied significantly between the bacteriophages of different bacterial species. The lowest mean LLR values were detected among Cellulophaga and Escherichia bacteriophages (3.49 and 3.98, respectively) and the highest LLR score was obtained for the Bacillus bacteriophages, with the average LLR of 8.96 (p < 0.001).\\nNo direct correlation was observed between the functions of proteins containing PrDs and the LLR score. One of both highest and lowest LLR scores were obtained for the single-stranded DNA binding protein of Listeria bacteriophage LMSP-25 (LLR score, 45.89; Myoviridae) and the Streptococcus bacteriophage 5093 (LLR 1.97; Siphoviridae), respectively (Figures 2A,B).\\nFIGURE 2. Characterization of the candidate bacteriophage PrDs in the single-stranded DNA binding proteins. (A)Graphical representation of the highest LLR value in this group of proteins identified in the Listeria phage LMSP-25. (B) Graphical representation of the lowest LLR value in this group of proteins in the Streptococcus phage 5093. The predicted PrDs are highlighted in red.\\nAdditionally, we analyzed the proteome enrichment with PrDs in the bacteriophages of different bacteria. The highest enrichment rate, with at least five PrDs per proteome, was found among the bacteriophages of Bacillus, Cronobacter, Lactobacillus, Synechococcus, Staphylococcus, and other bacteria (Figure 4 and Supplementary Table S5). The majority of these bacteriophages containing more than five PrDs per one proteome was shown to belong to the Myoviridae family. Since the members of this family have larger proteomes compared with those of other bacteriophage families, we analyzed and obtained a direct correlation between the number of PrDs per bacteriophage and the proteome (p < 0.05), but no correlations were observed within any bacteriophage family (Supplementary Table S6). Notably, the particular enrichment with PrDs was found across the majority of Prochlorococcus and Synechococcus cyanophages identified as PrD carriers in this study.\\nStructural Domains Associated with to Bacteriophage PrDs\\nTo analyze the functions of bacteriophage proteins with PrDs, we clustered the identified domains into five groups, according to the major steps during the process of bacteriophage interactions with its host cell (Marvin, 1998). The proteins were grouped based on their functions during the interactions with the bacterial host cells into the following groups: attachment and penetration, replication of bacteriophage DNA and protein synthesis, assembly, release, and unknown functions (Laemmli, 1970; Zylicz et al., 1989; Marvin, 1998; Rakhuba et al., 2010; Aksyuk and Rossmann, 2011). The obtained correlations between the LLR scores, bacterial and bacteriophage families, and protein functions were analyzed and are displayed as a heatmap in Figure 3.\\nFIGURE 3. Heatmap showing PrD distribution in bacteriophages. The correlation between LLR score of the identified PrDs, their distribution across bacteriophage families, host bacteria, and their function in the bacteriophage-bacterial interaction are presented. The likelihood that an identified PrD is a prion is represented by a color scale, from white to red (maximum).\\nThe most abundant (2046 PrPs; medium LLR score 4.86) functional group was shown to be the attachment and penetration group, which included the proteins associated with the host cell binding and genome injection. We identified receptor binding proteins, tail fiber proteins, peptidase_M23 baseplate wedge components, lysins, and other proteins known to have crucial functions in the various aspects of phage infection, including adsorption, peptidoglycan hydrolysis, cell wall penetration, and DNA ejection, but the PrDs were predominantly found among tail component, tail fiber, and baseplate proteins (Rodr\\u00edguez-Rubio et al., 2012; Stockdale et al., 2013). Furthermore, we identified a PrD in gp98/99 proteins that form the cell-puncturing device of Mycobacterium phages. Some of the identified proteins associated with surface structures, such as tail tube cap, are not involved in the direct contact with the bacterial cell, but they represent important structural components allowing the injection of DNA (Arisaka et al., 2003; Orlova, 2012). Heatmap analysis results showed that Podoviridae belonging to the Mycobacteriaceae family and Myoviridae of Listeriaceae have the highest LLR score (Figure 3).\\nThe second largest group was shown to contain 390 PrDs (medium LLR score, 7.93) involved in the replication of phage DNA and protein synthesis. We identified several Erf proteins, essential for bacteriophage DNA recombination, proteins belonging to glycosyltransferase family protecting bacteriophage DNA from degradation, and single-strand binding proteins (Markine-Goriaynoff et al., 2004). Other identified proteins included those involved in the regulation of host cell metabolism, such as serine/threonine phosphatases (Howard-Varona et al., 2017). The highest average LLR score was obtained for the members of Myoviridae family of Moraxellaceae and Listeriaceae (Figure 3).\\nFollowing this, we analyzed proteins with PrDs that are associated with the bacteriophage assembly in the host cells (170 proteins; medium LLR score, 6.16). These proteins were identified in all major families of Caudovirales. The highest average LLR score was obtained for the proteins of Myoviridae family of Moraxellaceae (Figure 3). These proteins are also associated with the organization and maturation of bacteriophages, and those involved in the tail assembly and DNA packaging (Linderoth et al., 1994; Boudko et al., 2002; Aksyuk and Rossmann, 2011).\\nThe identified PrDs in proteins that are involved in cell wall lysis and the release of bacteriophages from the host cells were less abundant and this group contained only 105 proteins (medium LLR score, 5.13). This group predominantly comprised lysins, including those with the amidase activity and proteins with peptidoglycan binding functions (hydrolases, D-ala-D-ala carboxypeptidase, endopeptidase CHAP domain protein, and others), which are involved in bacterial cell wall degradation, resulting in cell lysis and release of progeny viruses. They were predominantly detected among Siphoviridae infecting Streptococcaceae and Mycobacteriaceae. The highest average LLR score was obtained in Myoviridae family of Moraxellaceae (Figure 3).\\nFinally, we identified 2329 PrDs among the proteins with the still unknown functions. Among these proteins in Staphylococcus bacteriophages belonging to the Siphoviridae family, we identified Panton-Valentine leukocin (LukS-PV), a cytotoxin associated with the increased virulence of Staphylococcus aureus, which can induce a considerable tissue damage (Vandenesch et al., 2003). This is in agreement with previous observations that PrDs in viruses play important roles in the determination of virulence potential (Iglesias et al., 2015).\\nDiscussion\\nTo the best of our knowledge, this is the first analysis of PrDs among bacteriophages. The results of our study underline the necessity of characterizing bacteriophages and we propose here that the all bacteriophage genes should be known as the phagobiome.\\nHere, PrDs were more frequently found among archaeal phages, and the analysis of the three best-known Caudovirales families showed that these domains are more abundant among Podoviridae than Myoviridae or Siphoviridae.\\nFurthermore, our analyses demonstrated that approximately 50% of all sequenced bacteriophages available in public databases contain at least one PrD. The majority of bacteriophages contain less than five PrDs per proteome. Surprisingly, the vast majority of bacteriophages with more than five PrDs per proteome were shown to belong to the Myoviridae family. Since the total numbers of PrDs found in Myoviridae (2425) and Siphoviridae (1665) are of the same order of magnitude, these further supports the enrichment of PrDs in the Myoviridae proteomes. However, Myoviridae members have larger proteomes than the members of Siphoviridae family, which may explain the observed differences in the PrD enrichment (Sandaa, 2009). Further analyses indicated a direct correlation between the proteome size and the PrD enrichment, and therefore, we further explored whether the proteome size of bacteriophages in the Myoviridae family is related to the PrD enrichment. However, we have not observed a linear correlation between these parameters, suggesting that the PrD enrichment does not depend only on the size of a Myoviridae proteome.\\nMoreover, the majority of the top 100 scoring PrDs was also detected in the members of Myoviridae family, and among these, several short protein sequences were found to harbor a higher number of Q-rich domains than the longer ones. This indicates that the scores obtained in the analysis of protein sequences according to their prion-like characteristics do not correlate only with the size of a protein. However, the functional relevance of these findings remains unclear, since the majority of these proteins have not been characterized. The analysis of proteins with the known functions showed that the proteins involved in the replication of bacteriophage DNA and protein synthesis have the largest LLR scores.\\nMany PrD-containing proteins with known functions were identified among Bacillus phages that infect the spore-forming Bacillus spp. The identified PrDs were found in proteins responsible for attachment and penetration, replication of bacteriophage DNA, and release within the same Bacillus phages. Furthermore, the LLR scores of PrDs in the Bacillus phages were shown to have the highest meaning of the prion likelihood ratio compared with the phages associated with other bacterial genera. Previously, we reported a particular role of spore-forming bacteria in the global microbiota, which allowed their identification as the members of sporobiota (Tetz and Tetz, 2017). Taken together, previous results and the ones obtained in this study may suggest that the global distribution of sporobiota members and Bacillus spp., and their increased transfer between the ecological niches is associated with the presence of resistant transmissible spores due to their interactions with bacteriophages (von Wintersdorff et al., 2016).\\nHowever, the highest numbers of PrDs per proteome were found across Prochlorococcus and Synechococcus phages, whose hosts are represented by evolutionary ancient Cyanobacteria (Figure 4) (Schirrmeister et al., 2011). This finding along with the higher prevalence of PrDs in archaea suggests the possible association of the presence of PrDs and \\u201cevolutionary age,\\u201d however, this requires an independent analysis. Regardless, the enrichment of PrDs in the most abundant phages on Earth infecting marine Prochlorococcus and Synechococcus suggests the possible particular role of these elements (Sullivan et al., 2010).\\nFIGURE 4. Proteome enrichment for PrDs across bacteriophages. Bubble charts represent the medium enrichment rate with PrDs per bacteriophage proteome (phages with at least five PrDs per proteome have been considered) and the number of phages in which these PrDs were identified. The values in the bubbles are the median number of PrDs occurring per proteome in each bacteriophage group. Bigger bubble size indicates higher number of phages with at least five PrDs per proteome, and smaller size indicates lower number of phages with at least five PrDs per proteome.\\nTo determine the potential correlations between the functions of PrD-containing proteins, bacteriophage families, and host organisms, we generated a heatmap that showed that the highest LLRs were found among Myoviridae and Siphoviridae members of Moraxellaceae, Listeriaceae, and Pseudoalteromonadaceae, which were shown to be associated with replication of phage DNA, protein synthesis, and bacteriophage assembly. However, these data do not correspond to the frequency of PrD in bacteriophage proteomes. A vast majority of the bacteriophage PrD-containing proteins are involved in the interactions between a bacteriophage and bacterial cell wall, their attachment and penetration, and the release of progeny phages.\\nAttachment and penetration represent crucial steps in the bacteriophage\\u2013bacterium interaction, since these processes allow the entry of the bacteriophage DNA inside the host. We identified PrDs in the proteins forming different structural components of bacteriophages, including head, neck, sheath, baseplate and fibers, for tailed viruses, receptor-binding proteins for tailless Tectiviridae, and in the coat proteins III and IV of the filamentous Inoviridae (Hulo et al., 2017). This indicates the role of the identified PrDs in the formation of surface structures and specific and non-specific attachments to bacterial cells (Spinelli et al., 2014). Moreover, a similar trend in the distribution of prion-like domains in proteins responsible for interactions and virulence was defined by Gene Ontology GO terms across bacteria (Iglesias et al., 2015). Notably, PrDs were common across different bacterial and bacteriophage families, indicating the universal regulatory role of these putative prion domains (Iglesias et al., 2015).\\nFurthermore, since we identified PrDs in the interacting proteins in different bacteriophage families infecting a variety of bacterial hosts, this implies that these PrDs are conserved across species. However, since the PrDs in the surface proteins were not identified in all analyzed bacteriophages, and in those where they were identified, PrDs were found in some, but not all surface proteins of the given phage, this suggests that the presence of PrD-containing proteins may be beneficial, but it is not obligatory. The attachment of bacteriophages is a complex process, accompanied by a number of bacteriophage-resistance responses initiated in a bacterial cell (Labrie et al., 2010). Our findings may indicate that the proteins with the PrDs can alter their conformational states, adding further complexity and specificity in the phage-bacterial interactions, and this may serve as a mechanism to overcome some bacterial resistance mechanisms. Moreover, it can be speculated that the PrDs in the interacting proteins may be involved not only in the interactions with host bacteria, but also in the interactions of bacteriophages with other components of microbial communities. These observations agree with our recent studies showing that the oral administration of bacteriophages induces an increase in the intestinal permeability and endotoxemia in mammals, significantly changing microbiota composition, beyond the possible direct effect of bacteriophages, and because of this, it was suggested that these effects are exerted through yet unknown mechanisms (Tetz and Tetz, 2016; Tetz et al., 2017).\\nAdditionally, the results obtained in this study suggest that the presence of PrDs in bacteriophages may be associated with a number of mammalian diseases, in which the role of prions was demonstrated both in macroorganisms and in gut bacteria. Thus in the article by Chen et al. (2016), C. elegans fed with the prion-producing Escherichia coli was shown to have an enhanced prion aggregation in brain. It can be assumed that a similar process may be observed following the introduction of bacteriophages and that bacteriophages contribute to the observed processes.\\nProteins that are involved in the bacterial cell wall interactions and the release of progeny phages from the host were shown to harbor PrDs as well. The release stage involves bacterial cell wall degradation from within the cells with bacteriophage-encoded peptidoglycan hydrolases, synthesized at the end of the multiplication cycle. We identified PrDs in phage-encoded endolysins, and the majority of these proteins was found in Siphoviridae family, with only a few detected among the members of Podoviridae and Myoviridae.\\nSome of the proteins enriched in the PrDs are involved in the replication of bacteriophage DNA and protein synthesis. In eukaryotes, PrD-containing proteins were shown to play a role in the interactions and binding with the nucleic acids (King et al., 2012). Nucleic acids were shown to play a crucial role in the PrP conversion into PrPSc-like species as cofactors (Macedo and Cordeiro, 2017).\\nIn bacteriophages, we determined that the presence of PrDs can also be associated with the single-strand binding proteins, known as essential components of bacteriophage DNA synthesis. Therefore, this indicates that these PrDs can be easily misfolded into PrPSc through the interactions with the DNA, which was shown to be an important component inducing this misfolding (Cordeiro et al., 2001). This type of interaction with the host cell may represent an additional process through which the bacteriophages control bacterial intracellular functions (Wagner and Waldor, 2002; Davies and Davies, 2010).\\nBacteriophages are important vehicles facilitating the genetic exchange between microorganisms, including the spreading of virulence factors and the antibiotic resistant genes (Lina et al., 1999). Here, we identified a PrD in the LukS-PV protein of some Siphoviridae members and unclassified phages. This cytotoxin can be found among methicillin-resistant S. aureus and belongs to the group of \\u03b2-pore-forming toxins (Vandenesch et al., 2003). The presence of LukS-PV is associated with the increased virulence of S. aureus, leading to abscess formation and severe necrotizing pneumonia. The detected PrDs located in LukS-PV indicates the role of these domains in the virulence potential of the cytotoxins and their role in the pathogenicity. Our results may indicate that the presence of PrDs in LukS-PV affects its interactions with the membranes of the targeted eukaryotic cells and induce the pore-forming potential, which is supported by previous studies (Iglesias et al., 2015).\\nTaken together, we identified numerous putative PrD-containing proteins in bacteriophages. We observed consistent patterns of the distribution of PrDs across different bacteriophage families. However, since the infectious agents of some bacteria were shown to lack PrDs, this may indicate that the diversity of phagobiota and phogobiome is underestimated. Although bacteriophage genomes are significantly smaller than bacterial genomes, currently less than 2,500 phage genomes are deposited in the NCBI database, compared with almost 90,000 bacterial whole genome sequences.\\nAlthough several trends in the frequency of PrDs across bacteriophage families have been detected, future studies are required (Pourcel et al., 2017), to elucidate further the functions and presence of PrP-containing proteins in bacteriophages. The predictive approach employed here revealed for the first time a large set of putative PrPs, and further experimental characterization of these proteins may contribute to the understanding of bacteriophage biology.\\nAuthor Contributions\\nGT and VT designed and conducted the experiments and analyzed data, GT wrote the manuscript.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2017.02239/full#supplementary-material\\nTABLE S1 | LLR score showing the predicted putative PrDs across bacteriophages.\\nTABLE S2 | Summary of the LLR score of prion predictions across different phage families.\\nTABLE S3 | Least squares means for effect of bacteriophage family on LLR score.\\nTABLE S4 | Top 100 scoring PrDs across bacteriophages.\\nTABLE S5 | Proteome enrichment for PrDs across bacteriophages.\\nTABLE S6 | Correlation between the number of PrD per phage and the proteome size.\\nFootnotes\\n^ http://www.ncbi.nlm.nih.gov/\\n^ www.r-project.org\\nReferences\\nAlberti, S., Halfmann, R., King, O., Kapila, A., and Lindquist, S. (2009). A systematic survey identifies prions and illuminates sequence features of prionogenic proteins. Cell 137, 146\\u2013158. doi: 10.1016/j.cell.2009.02.044\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nAngarica, V. E., Ventura, S., and Sancho, J. (2013). Discovering putative prion sequences in complete proteomes using probabilistic representations of Q/N-rich domains. BMC Genomics 14:316. doi: 10.1186/1471-2164-14-316\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nArisaka, F., Kanamaru, S., Leiman, P., and Rossmann, M. G. (2003). The tail lysozyme complex of bacteriophage T4. Int. J. Biochem. Cell Biol. 35, 16\\u201321. doi: 10.1016/S1357-2725(02)00098-5\\nCrossRef Full Text | Google Scholar\\nBischoff, S. C., Barbara, G., Buurman, W., Ockhuizen, T., Schulzke, J. D., Serino, M., et al. (2014). Intestinal permeability\\u2013a new target for disease prevention and therapy. BMC Gastroenterol. 14:189. doi: 10.1186/s12876-014-0189-7\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBlanco, L. P., Evans, M. L., Smith, D. R., Badtke, M. P., and Chapman, M. R. (2012). Diversity, biogenesis and function of microbial amyloids. Trends microbial. 20, 66\\u201373. doi: 10.1016/j.tim.2011.11.005\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nBoudko, S. P., Londer, Y. Y., Letarov, A. V., Sernova, N. V., Engel, J., and Mesyanzhinov, V. V. (2002). Domain organization, folding and stability of bacteriophage T4 fibritin, a segmented coiled-coil protein. FEBS J. 269, 833\\u2013841. doi: 10.1046/j.1432-1033.2002.02734.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChapman, M. R., Robinson, L. S., Pinkner, J. S., Roth, R., Heuser, J., Hammar, M., et al. (2002). Role of E. coli curli operons in directing amyloid fiber formation. Science 295, 851\\u2013855. doi: 10.1126/science.1067484\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChen, S. G., Stribinskis, V., Rane, M. J., Demuth, D. R., Gozal, E., Roberts, A. M., et al. (2016). Exposure to the functional bacterial amyloid protein curli enhances alpha-synuclein aggregation in aged fischer 344 rats and Caenorhabditis elegans. Sci. Rep. 6:34477. doi: 10.1038/srep34477\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCheng, P. N., Liu, C., Zhao, M., Eisenberg, D., and Nowick, J. S. (2012). Amyloid \\u03b2-sheet mimics that antagonize protein aggregation and reduce amyloid toxicity. Nat. chem. 4, 927\\u2013933. doi: 10.1038/nchem.1433\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChiti, F., and Dobson, C. M. (2006). Protein misfolding, functional amyloid, and human disease. Annu. Rev. Biochem. 75, 333\\u2013366. doi: 10.1146/annurev.biochem.75.101304.123901\\nCrossRef Full Text | Google Scholar\\nCollinge, J. (2001). Prion diseases of humans and animals: their causes and molecular basis. Annu. Rev. Neurosci. 24, 519\\u2013550. doi: 10.1146/annurev.neuro.24.1.519\\nCrossRef Full Text | Google Scholar\\nCordeiro, Y., Machado, F., Juliano, L., Juliano, M. A., Brentani, R. R., Foguel, D., et al. (2001). DNA converts cellular prion protein into the \\u03b2-sheet conformation and inhibits prion peptide aggregation. J. Biol. Chem. 276, 49400\\u201349409. doi: 10.1074/jbc.M106707200\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nde Groot, N. S., Torrent, M., Villar-Piqu\\u00e9, A., Lang, B., Ventura, S., Gsponer, J., et al. (2012). Evolutionary selection for protein aggregation. Biochem. Soc. Trans. 40, 1032\\u20131037. doi: 10.1042/BST20120160\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nDupuis, M. \\u00c8., Villion, M., Magad\\u00e1n, A. H., and Moineau, S. (2013). CRISPR-Cas and restriction-modification systems are compatible and increase phage resistance. Nat. Commun. 4:2087. doi: 10.1038/ncomms3087\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHoward-Varona, C., Roux, S., Dore, H., Solonenko, N. E., Holmfeldt, K., Markillie, L. M., et al. (2017). Regulation of infection efficiency in a globally abundant marine Bacteriodetes virus. ISME J. 11, 284\\u2013285. doi: 10.1038/ismej.2016.81\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHulo, C., Masson, P., Toussaint, A., Osumi-Sutherland, D., de Castro, E., Auchincloss, A. H., et al. (2017). Bacterial virus ontology; coordinating across databases. Viruses 9:126. doi: 10.3390/v9060126\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nIglesias, V., de Groot, N. S., and Ventura, S. (2015). Computational analysis of candidate prion-like proteins in bacteria and their role. Front. Microbiol. 6:1123. doi: 10.3389/fmicb.2015.01123\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKing, O. D., Gitler, A. D., and Shorter, J. (2012). The tip of the iceberg: RNA-binding proteins with prion-like domains in neurodegenerative disease. Brain Res. 1462, 61\\u201380. doi: 10.1016/j.brainres.2012.01.016\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLaemmli, U. K. (1970). Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227, 680\\u2013685. doi: 10.1038/227680a0\\nCrossRef Full Text | Google Scholar\\nLancaster, A. K., Nutter-Upham, A., Lindquist, S., and King, O. D. (2014). PLAAC: a web and command-line application to identify proteins with prion-like amino acid composition. Bioinformatics 30, 2501\\u20132502. doi: 10.1093/bioinformatics/btu310\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLina, G., Piemont, Y., Godail-Gamot, F., Bes, M., Peter, M., Gauduchon, V., et al. (1999). Involvement of panton-valentine leukocidin-producing Staphylococcus aureus in primary skin infections and pneumonia. Clin. Infect. Dis. 29, 1128\\u20131132. doi: 10.1086/313461\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLinderoth, N. A., Julien, B., Flick, K. E., Calendar, R., and Christle, G. E. (1994). Molecular cloning and characterization of bacteriophage P2 genes R and S involved in tail completion. Virology 200, 347\\u2013359. doi: 10.1006/viro.1994.1199\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMalinovska, L., Kroschwald, S., and Alberti, S. (2013). Protein disorder, prion propensities, and self-organizing macromolecular collectives. Biochim. Biophys. Acta 1834, 918\\u2013931. doi: 10.1016/j.bbapap.2013.01.003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMarkine-Goriaynoff, N., Gillet, L., Van Etten, J. L., Korres, H., Verma, N., and Vanderplasschen, A. (2004). Glycosyltransferases encoded by viruses. J. Gen. Virol. 85, 2741\\u20132754. doi: 10.1099/vir.0.80320-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMarvin, D. A. (1998). Filamentous phage structure, infection and assembly. Curr. Opin. Struct. Biol. 8, 150\\u2013158. doi: 10.1016/S0959-440X(98)80032-8\\nCrossRef Full Text | Google Scholar\\nMichelitsch, M. D., and Weissman, J. S. (2000). A census of glutamine/asparagine-rich regions: implications for their conserved function and the prediction of novel prions. Proc. Natl. Acad. Sci. U.S.A. 97, 11910\\u201311915. doi: 10.1073/pnas.97.22.11910\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPourcel, C., Midoux, C., Vergnaud, G., and Latino, L. (2017). A carrier state is established in Pseudomonas aeruginosa by phage LeviOr01, a newly isolated ssRNA levivirus. J. Gen. Virol. 98, 2181\\u20132189. doi: 10.1099/jgv.0.000883\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRakhuba, D. V., Kolomiets, E. I., Dey, E. S., and Novik, G. I. (2010). Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell. Pol. J. Microbiol. 59, 145\\u2013155.\\nPubMed Abstract | Google Scholar\\nRodr\\u00edguez-Rubio, L., Guti\\u00e9rrez, D., Mart\\u00ednez, B., Rodr\\u00edguez, A., G\\u00f6tz, F., and Garc\\u00eda, P. (2012). The tape measure protein of the Staphylococcus aureus bacteriophage vB_SauS-phiIPLA35 has an active muramidase domain. Appl. Environ. Microbiol. 78, 6369\\u20136371. doi: 10.1128/AEM.01236-12\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSandaa, R. A. (2009). \\u201cViral pool, environmental,\\u201d in Encyclopedia in Microbiology, 3rd Edn, ed. M. Schaechter (Cambridge: Elsevier Ltd), 553\\u2013567.\\nSpinelli, S., Veesler, D., Bebeacua, C., and Cambillau, C. (2014). Structures and host-adhesion mechanisms of lactococcal siphophages. Front. Microb. 5:3. doi: 10.3389/fmicb.2014.00003\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nStockdale, S. R., Mahony, J., Courtin, P., Chapot-Chartier, M. P., Van Pijkeren, J. P., Britton, R. A., et al. (2013). The lactococcal phages Tuc2009 and TP901-1 incorporate two alternate forms of their tail fiber into their virions for infection specialization. J. Biol. Chem. 288, 5581\\u20135590. doi: 10.1074/jbc.M112.444901\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSullivan, M. B., Huang, K. H., Ignacio-Espinoza, J. C., Berlin, A. M., Kelly, L., Weigele, P. R., et al. (2010). Genomic analysis of oceanic cyanobacterial myoviruses compared with T4-like myoviruses from diverse hosts and environments. Environ. Microbiol. 12, 3035\\u20133056. doi: 10.1111/j.1462-2920.2010.02280.x\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSuzuki, G., Shimazu, N., and Tanaka, M. (2012). A yeast prion, Mod5, promotes acquired drug resistance and cell survival under environmental stress. Science 336, 355\\u2013359. doi: 10.1126/science.1219491\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTelling, G. C., Scott, M., Mastrianni, J., Gabizon, R., Torchia, M., Cohen, F. E., et al. (1995). Prion propagation in mice expressing human and chimeric PrP transgenes implicates the interaction of cellular PrP with another protein. Cell 83, 79\\u201390. doi: 10.1016/0092-8674(95)90236-8\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nTetz, G. V., Ruggles, K. V., Zhou, H., Heguy, A., Tsirigos, A., and Tetz, V. (2017). Bacteriophages as potential new mammalian pathogens. Sci. Rep. 7:7043. doi: 10.1038/s41598-017-07278-6\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nVandenesch, F., Naimi, T., Enright, M. C., Lina, G., Nimmo, G. R., and Heffernan, H. (2003). Community-acquired methicillin-resistant Staphylococcus aureus carrying Panton-Valentine leukocidin genes: worldwide emergence. Emerg. Infect. Dis. 9, 978\\u2013984. doi: 10.3201/eid0908.030089\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nvon Wintersdorff, C., Penders, J., van Niekerk, J., Mills, N., Majumder, S., van Alphen, L., et al. (2016). Dissemination of antimicrobial resistance in microbial ecosystems through horizontal gene transfer. Front. Microbiol. 7:173. doi: 10.3389/fmicb.2016.00173\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWagner, P. L., and Waldor, M. K. (2002). Bacteriophage control of bacterial virulence. Infect. Immun. 70, 3985\\u20133993. doi: 10.1128/IAI.70.8.3985-3993.2002\\nCrossRef Full Text | Google Scholar\\nWalker, L. C., Schelle, J., and Jucker, M. (2016). The prion-like properties of amyloid-\\u03b2 assemblies: implications for Alzheimer\\u2019s disease. Cold Spring Harb. Perspect. Med. 6:a024398. doi: 10.1101/cshperspect.a024398\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWemheuer, W. M., Wrede, A., and Schulz-Schaeffer, W. J. (2017). Types and strains: their essential role in understanding protein aggregation in neurodegenerative diseases. Front. Aging Neurosci. 9:187. doi: 10.3389/fnagi.2017.00187\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nWestergard, L., Christensen, H. M., and Harris, D. A. (2007). The cellular prion protein (PrP C): its physiological function and role in disease. Biochim. Biophys. Acta 1772, 629\\u2013644. doi: 10.1016/j.bbadis.2007.02.011\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nYuan, A. H., Garrity, S. J., Nako, E., and Hochschild, A. (2014). Prion propagation can occur in a prokaryote and requires the ClpB chaperone. Elife 3:e02949. doi: 10.7554/eLife.02949\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nZylicz, M., Ang, D., Liberek, K., and Georgopoulos, C. (1989). Initiation of lambda DNA replication with purified host-and bacteriophage-encoded proteins: the role of the dnaK, dnaJ and grpE heat shock proteins. EMBO J. 8, 1601\\u20131608.\\nPubMed Abstract | Google Scholar\"\n",
      "    },\n",
      "    {\n",
      "        \"url\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00574\",\n",
      "        \"crawl\": {\n",
      "            \"loadedUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00574/full\",\n",
      "            \"loadedTime\": \"2024-01-23T08:50:59.949Z\",\n",
      "            \"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\n",
      "            \"depth\": 1,\n",
      "            \"httpStatusCode\": 200\n",
      "        },\n",
      "        \"metadata\": {\n",
      "            \"canonicalUrl\": \"https://www.frontiersin.org/articles/10.3389/fmicb.2019.00574\",\n",
      "            \"title\": \"Frontiers | Therapeutic Characterization and Efficacy of Bacteriophage Cocktails Infecting Escherichia coli, Klebsiella pneumoniae, and Enterobacter Species\",\n",
      "            \"description\": \"Infections due to antibiotic resistant bacteria are increasing globally and this needs immediate attention. Bacteriophages are considered an effective alternative for the treatment of bacterial infections. The aim of this study was to isolate and characterize the bacteriophages that infect Escherichia coli, Klebsiella pneumoniae and Enterobacter species. For this, clinical bacterial isolates of the mentioned species were obtained from diagnostic centers located in Chennai, Tamil Nadu, India. The bacteriophages were isolated from sewage water samples collected from Tamil Nadu, India. Phage isolation was performed using the enrichment method and agar overlay method was used to confirm the presence of bacteriophages. All the phages were characterized for their life cycle parameters, genome analysis and in vitro phage cocktail activity. The three bacteriophages exhibited broad host range activity: Escherichia virus myPSH2311 infecting E. coli belonging to six different pathotypes, Klebsiella virus myPSH1235 infecting K. pneumoniae belonging to four different serotypes and Enterobacter virus myPSH1140 infecting four different species of Enterobacter. Morphological observations suggested that the bacteriophages belonged to, Phieco32virus (Escherichia virus myPSH2311), Podoviridae (Klebsiella virus myPSH1235) and Myoviridae (Enterobacter virus myPSH1140). The life cycles (adsorption, latent period and cell burst) of Escherichia virus myPSH2311, Klebsiella virus myPSH1235 and Enterobacter virus myPSH1140 were found to be 26 min, 40 min and 11 min respectively. Genomic analysis revealed that Escherichia virus myPSH2311 is closely related to Escherichia phage vB_EcoP_SU10, Klebsiella virus myPSH1235 is closely related to Klebsiella phage vB_KpnP_KpV48 and Enterobacter virus myPSH1140 is closely related to Enterobacter phage PG7 and Enterobacter phage CC31. When phage cocktail was used against multiple bacterial mixtures, there was a reduction in bacterial load from106 CFU/mL to 103 CFU/mL within 2 hours. All the three characterized phages were found to have a broad host range activity and the prepared phage cocktails were effective against a mixed bacterial population that is resistant to meropenem and colistin, two last resort antibiotics. Infections caused by drug resistant bacteria will be a serious threat in the future and the use of virulent bacteriophages in therapy may offer an effective solution.\",\n",
      "            \"author\": null,\n",
      "            \"keywords\": null,\n",
      "            \"languageCode\": \"en\"\n",
      "        },\n",
      "        \"screenshotUrl\": null,\n",
      "        \"text\": \"Introduction\\nBacteriophages are the viruses of bacteria that live in the same ecological niche, where their host bacteria are present (Rohwer, 2003). Phages are generally very specific (species-specific and strain-specific) to their bacterial host but some phages are polyvalent, and can infect more than one species or strain of bacteria (Chibani-Chennoufi et al., 2004). Phage therapy largely involves the treatment of bacterial infections using bacteriophages/phages (Levin and Bull, 2004). Phages with broad host range are mostly chosen for therapy, because of their broad spectrum host-range activity against multiple bacteria. The phages belonging to the order Caudovirales (Family-Myoviridae, Siphoviridae, and Podoviridae) with proteinaceous tail, that follow only lytic pathway, are preferred for therapy (Gill and Hyman, 2010). The use of bacteriophages for therapeutic purpose is an old concept that is re-emerging after about a century (Sulakvelidze et al., 2001). Antibiotic resistance has become a human health concern globally as the infections caused by resistant bacteria are becoming difficult to cure (World Health Organization [WHO], 2014; Ventola, 2015). Phage therapy can be one of the alternatives for combating antibiotic resistant bacterial infections (Rios et al., 2016).\\nEscherichia coli, Klebsiella pneumoniae, and Enterobacter cloacae are Gram-negative bacteria that belong to the family Enterobacteriaceae. All the three are enteric pathogens causing serious opportunistic infections in humans (Morens et al., 2004; Schlossberg, 2015). They cause hospital acquired and community acquired infections such as diarrhea, meningitis, urinary tract infections (UTIs), bacteremia, pneumonia, surgical site infections, and sepsis (Schlossberg, 2015). The increasing reports of resistance to carbapenem and colistin, two last resort drugs, among Enterobacteriaceae world over (World Health Organization [WHO], 2014) and more particularly so in the developing countries (Ventola, 2015) is a serious threat to their therapeutic use, which prompts search for alternative treatment options. Studies using bacteriophages as an antibacterial agent have shown promising outcomes in both in vitro and in vivo studies, and therefore phage therapy is being studied as a candidate to cure bacterial infections (McVay et al., 2007; Hung et al., 2011; Cao et al., 2015; Mirzaei and Nilsson, 2015). Phage cocktails have shown broad spectrum activity against many bacterial strains (Mendes et al., 2014; Yen et al., 2017). The characterization of phages for therapeutic purpose involves isolation of potential lytic phages, multi-step in vitro characterization, cocktail preparation and purification, dosing and in vivo studies. More than 50 Escherichia phages belonging to families Myoviridae, Siphoviridae, and Podoviridae have been reported with complete genome sequences1. Genome sequenced phages against Klebsiella (\\u224829) and Enterobacter (\\u224810) have been reported much lesser in number2. Here, we report characterization of three lytic bacteriophages that showed promising ability to lyse E. coli, K. pneumoniae, and Enterobacter species, report their host range specificity and also efficacy of phage cocktails made using these three phages in various permutations and combinations, in effectively killing combinations of host bacteria using in vitro phage killing assay.\\nMaterials and Methods\\nIsolation of Clinical Bacterial Strains for the Study\\nThis study does not include any human subjects; therefore, ethical approval was not required for this study according to national and institutional guidelines. The clinical isolates of Escherichia coli, Klebsiella pneumoniae, Enterobacter cloacae, E. hormaechei, E. asburiae, and E. aerogenes used in this study were collected from diagnostic centers in Chennai, Tiruchirappalli, and Madurai located in the state of Tamil Nadu in India, during December 2014- September 2016. All the isolates were preserved in 30% glycerol stocks at -20\\u00b0C. The clinical samples used for bacterial isolation were urine, sputum, pus, blood, wound swab, and bronchial aspirate. Bacterial identification was performed using VITEK identification system (bioM\\u00e8rieux, Inc., United States) and 16S rRNA analysis. Universal primers, 27F (5\\u2032-AGAGTTTGATCCTGGCTCAG-3\\u2032) and 1492R (5\\u2032-GGTTACCTTGTTACGACTT-3\\u2032), were adopted for gene amplification and sequences of 16S rRNA genes were deposited in GenBank. All the clinical isolates were studied for resistance against meropenem and colistin, two last resort antibiotics, using microbroth dilution method following Clinical and Laboratory Standards Institute [CLSI], 2018. For the study, a total of 150 non-repetitive, Gram-negative bacterial isolates belonging to three genera Escherichia, Klebsiella, and Enterobacter were used. The isolated clinical pathogens included 80 E. coli isolates, 44 Klebsiella pneumoniae isolates and in the case of Enterobacter isolates, there were four different species namely E. cloacae (n = 15), E. hormaechei (n = 4), E. asburiae (n = 4), and E. aerogenes (n = 3) (Table 1). The results for meropenem and colistin resistance screening of these isolates are presented in Supplementary Table S1.\\nTable 1. Host range infection and efficiency of plating (EOP) of the phages Escherichia virus myPSH2311, Klebsiella virus myPSH1235, and Enterobacter virus myPSH1140.\\nPhage Isolation and Enrichment\\nBacteriophages were isolated from water samples collected from Ganges River near Varanasi in Northern part of India and sewage water treatment plants (secondary treatment stage) from different locations in Chennai, Bangalore, Tirupathi, Vellore, Karur, and Trichy in Southern part of India. Initially, the isolated bacterial strains (one isolate at a time) were grown in Luria-Bertani broth (Himedia, India) medium and were used as a host for phage isolation. Briefly, to a 5 mL of exponentially grown bacterial culture (optical density at 600 nm = 0.6), 10 mL of phage containing water samples was added and incubated at 37\\u00b0C for 24 h in shaking incubator to enrich the phages against the host bacterium. This mixture was centrifuged for 15 min at 12,000 \\u00d7g and the supernatant was filtered in 0.22-\\u03bcm pore-sized membrane syringe filters for separation of phages from other contaminants. The filtrate was used for double-agar overlay method (Li et al., 2016a). Briefly, to the exponentially grown host bacterial culture (400 \\u03bcL), the phage filtrate (200 \\u03bcL) was added and incubated for 15 min. To the mixture, 3 mL of molten soft agar (0.75% agar) was added and over-laid onto prepared LB agar plate (1.5% agar). The plates were incubated at 37\\u00b0C for 10 h and the appearance of clear plaques indicated the presence of phages against the host bacterium. The phage plaques were picked-up from the plate for further purification and the phage titer was determined. For spot test method, the bacterial (using exponentially grown host bacterial culture) lawn was prepared in LB agar plates and 10\\u201350 \\u03bcL of phage filtrate was placed as a spot over the target bacterial lawn to evaluate the phage activity. The development of clear spots indicated the phage activity and the time taken for bacterial clearance indicated the lytic activity of phages. The bacterium initially used for phage isolation was deemed as a host bacterium against the phage. Multiplicity of infection (MOI) was calculated using the number of phage particles against the potent host bacteria (PFU/CFU).\\nPurification of Lytic Phages\\nThe isolated phage lysates were prepared in high titers using phage multiplication strategy (propagation on host bacterium). Briefly, the phages were multiplied using host bacterium for 24 h (day 1) and centrifuged at 6000 \\u00d7g for 15 min. The collected supernatant was mixed with exponentially grown host bacterium (day 2) and allowed to multiply. Similar passages were carried out for 5 days and evaluated for phage activity/ titer by spot test and double agar overlay method against E. coli, K. pneumoniae, and Enterobacter species. The obtained high titer phages were precipitated using 10% PEG 6000 (polyethylene glycol) and 1 M NaCl. Briefly, to the phage lysate 10% PEG 6000+1 M NaCl was added, mixed gently (did not vortex) and the mixture was stored at 4\\u00b0C for 24 h. The precipitated phage particles were centrifuged at 15,000 \\u00d7g for 45 min and the obtained pellet was resuspended in sterile SM buffer (For 1 L: 5.8 g, NaCl; 50 mL, 1 M Tris-HCl [pH 7.5]; 2 g, MgSO4.7H2O; 5 mL, 2% gelatin). The extraction was carried out by adding equal volume of chloroform and the aqueous phase was sedimented by centrifugation at 18,000 \\u00d7g for 80 min. The obtained phage particles were dialyzed against phosphate buffer saline (PBS) for 6 h by changing buffer every 2 h and the purified phage suspension was stored at 4\\u00b0C for further analysis.\\nElectron Microscopic Analysis\\nThe purified phage particles (105 PFU/mL) were negatively stained using phosphotungstic acid, PTA (2% [w/v], pH 7.0). Briefly, 10 \\u03bcL of phage lysate was added over the copper grid and the liquid was allowed to absorb for 10 min. The remaining liquid was removed using tissue paper and the prepared 2% PTA solution (staining solution) was added. After allowing it to stain for 5 min, the excess stain was removed and the grid was washed twice with sterile water. The negatively stained phage particles in the copper grid were allowed to dry at room temperature for 20\\u201330 min and visualized under Transmission Electron Microscopy (FEI-TECNAI G2-20 TWIN, Bionand, Spain). The phage morphology was determined and head/ tail lengths (10 measurements each) were measured using ImageJ software.\\nHost-Range Specificity Determination and Efficiency of Plating (EOP)\\nThe lytic activity of isolated phages was tested against the target bacteria. Accordingly, Escherichia phage was tested against 80 E. coli isolates belonging to pathotypes: EPEC (enteropathogenic E. coli), EHEC (enterohemorrhagic E. coli), ETEC (enterotoxigenic E. coli), EIEC (enteroinvasive E. coli), EAEC (enteroaggregative E. coli), and UPEC (uropathogenic E. coli). Klebsiella phage was tested against 37 K. pneumoniae isolates that belonged to serotypes K1, K2, K5 and Enterobacter phage was tested against 15 E. cloacae, 4 E. hormaechei, 4 E. asburiae, and 3 E. aerogenes isolates. Spot test was carried out to assess the host-range specificity of phages against the test bacteria and the resulting positive isolates were again tested for their plaque forming ability in double agar overlay method for calculating efficiency of plating (EOP) (Mirzaei and Nilsson, 2015). EOP was calculated using the number of virus particles infecting the test bacterium against the same titer of virus particles infecting the host bacterium. Accordingly, all the test bacterial strains were grown overnight (16 h) and the concentration of 106\\u2013109 (CFU/mL) was used for double agar overlay method. For the assay, 200 \\u03bcL of bacterial culture was mixed with 100 \\u03bcL of phage lysate (MOI = 0.01) and EOP was determined using the formula, plaque forming units (PFUs) on the test bacterium/PFU on the host bacterium, evaluated by double agar overlay method. EOP was classified as \\u2018High,\\u2019 \\u2018Moderate,\\u2019 and \\u2018Low\\u2019 based on the productive infection on the target bacterium. EOP was deemed as \\u2018High\\u2019 only if the phage-bacterium combination against the test bacterium had a productive infection of at least 50% (EOP \\u2265 0.5) compared to the host bacterium. EOP between >10 and <50% (EOP > 0.1- < 0.5) was considered \\u2018Moderate\\u2019 and EOP < 10% (EOP \\u2264 0.1) was recorded as \\u2018Low\\u2019 (Mirzaei and Nilsson, 2015).\\nCharacterization: Adsorption Rate, Latency Period, and Burst Size\\nExponentially grown bacterial cells were mixed with the respective phages at a MOI of 0.001 and incubated at 37\\u00b0C. Aliquots of 100 \\u03bcL were removed after every 4 min interval for 40 min and diluted in 4.4 mL LB broth and 0.5 mL of chloroform was added. After incubating the mixture for 30 min at 37\\u00b0C, the number of non-adsorbed phages was determined subsequently using double agar overlay method. The adsorption curve was constructed using the ratio of non-adsorbed phage particles at different time intervals to the number of initial phages. One-step growth experiment was performed to determine the latent period and burst size (14). Briefly, the bacterial cells (108 CFU/mL) were infected with the phage particles (MOI of 0.001) and allowed to adsorb (based on the adsorption time determined previously) at 37\\u00b0C. The mixture was then centrifuged at 12,000 \\u00d7g for 5 min and the pellet was resuspended in 10 mL of LB broth and the incubation was continued at 37\\u00b0C. The samples were taken at 5 min intervals for 80 min and titrated against the host bacterium. The latent-period was calculated as the duration between the phage adsorbed until the release of phage virions. The burst size of the phage was calculated using the final number of free phage particles to the initial number of phages.\\nPhage Stability Studies\\nStability studies were conducted at different pH and temperature. For thermal stability tests, phage lysates (108 PFU/mL) were incubated at 4, 20, 35, 45, 50, 55, 60, 70, and 80\\u00b0C for 60 min in temperature-controlled water bath and immediately transferred to the ice cold condition (-20\\u00b0C) which was further tested for phage activity using double agar overlay method. The pH stability studies were performed using SM buffer and pH was adjusted using 1N NaOH and 1N HCl. The phage lysates (108 PFU/mL) were incubated at pH 1\\u201314 for 60 min and the aliquots were removed for stability analysis. The results were expressed as phage viability in terms of percentage of initial viral counts. All the stability studies were tested using E. coli ec311, K. pneumoniae kp235, and E. cloacae el140, and the experiments were repeated in triplicates.\\nDNA Isolation, Genome Sequencing, and Analysis\\nThe phage DNA was extracted from purified phage particles using phenol-chloroform (24:1) method and precipitated using ethanol (100%). The purified phage DNA was visualized on 0.8% agarose gels and the PE libraries were prepared using Illumina TruSeq Nano DNA library Prep kit. The prepared libraries were sequenced using Illumina Nextseq 500 system (using 2 bp \\u00d7 150 bp chemistry) at Eurofins Genomics, Bangalore, India. The sequenced raw data was processed to obtain high quality clean reads using Trimmomaticv 0.35 to remove adapter sequences, ambiguous reads (reads with unknown nucleotides \\u201cN\\u201d larger than 5%), and low-quality sequences [reads with more than 10% quality threshold (QV) < 20 phred score]. The sequenced high quality reads were de novo assembled using CLC Genomics Workbench version 9.5.2. Protein-coding and tRNA genes were identified using the final assembly. The transfer-RNA (tRNA) genes were predicted using tRNAscan-SE 2.0 web server while the protein coding genes (CDS) were predicted using FGENESV web server. Functional annotation of the predicted proteins was performed using the amino acid sequences viaBLASTp program online against a custom database of viral proteins in NCBI. Gene ontology (GO) annotations of the genes were determined by the Blast2GO platform. Distribution of GO terms across the categories \\u2013 Biological Process, Molecular Function and Cellular Component was obtained through WEGOportal3. The NCBI sequence was downloaded from NCBI2 for sequence comparison and the scaffolds were then subjected to reference-based assembly via CONTIGuator2. The final assembly generated by CONTIGuator was validated based on sequence homology to known bacteriophage sequences in NCBI via BlastN.\\nComposition and Preparation of Phage Cocktails\\nPhage cocktails containing different compositions of isolated phages under study were prepared and evaluated for activity against target species. For cocktail preparation, two or three different phages were mixed together in equal proportions to obtain a concentration of 106 PFU/mL. Briefly, cocktail EK1 contained Escherichia phage plus Klebsiella phage; KL2 contained Klebsiella phage plus Enterobacter phage; EL3 contained Escherichia phage plus Enterobacter phage and EKL4 contained Escherichia phage plus Klebsiella phage plus Enterobacter phage. The prepared cocktails were tested for in vitro phage-killing assay against respective bacterial strains. The results were compared to the activity of phages in cocktails to the activity of phages alone, and one bacterium from each genus was used for this study. Accordingly, 1 mL of host bacterium (6 \\u00d7 107 CFU/mL) was diluted in LB broth to yield a final concentration of 6 \\u00d7 106 CFU/mL. For each study, 100 \\u03bcL (each phage at the concentration of 106 PFU/mL; MOI of 1.0) of bacteriophage suspension was added and the mixture was incubated at 37\\u00b0C and the aliquots (100 \\u03bcL) were removed at 0, 2, 4, 6, and 24 h to calculate the reduction in bacterial count. To test the activity of phage cocktails, bacteria were also used in combination similar to phage combinations (Supplementary Table S2). In the case of control experiments, bacteriophage buffer alone was used with the bacterial inoculum and bacterial growth was determined. All the cocktail studies were tested using E. coli ec311, K. pneumoniae kp235, and E. cloacae el140, and the experiments were repeated in triplicates.\\nResults\\nNomenclature and Morphological Characterization of Phages\\nThe phages were named as Escherichia virus myPSH2311, Klebsiella virus myPSH1235, and Enterobacter virus myPSH1140 following the bacteriophage nomenclature guidelines (Krupovic et al., 2016). Examination of phage morphology by TEM analysis showed that Escherichia virus myPSH2311 had an icosahedral head of 33 \\u00b1 3.0 nm, a non-contractile tail length of 65 \\u00b1 2.5 nm and belonged to genus Phieco32virus, Klebsiella virus myPSH1235 had the icosahedral head of 80 \\u00b1 4.5 nm and very short non-contractile tail that showed the phage belonged to family Podoviridae and Enterobacter virus myPSH1140 had an elongated head of approximately 80 \\u00b1 2.0 nm and long contractile tail of 101 \\u00b1 3.5 nm in length indicating that it belonged to the family Myoviridae (Figure 1).\\nFigure 1. Transmission electron microscopy (TEM) images of (A) Escherichia virus myPSH2311, (B) Enterobacter virus myPSH1140, (C) Klebsiella virus myPSH1235. The smaller images in (A,C) represent 100 nm.\\nHost-Range Activity Determination and Efficiency of Plating (EOP)\\nThe spot test assay showed that the Escherichia virus myPSH2311 had lytic activity against 73% of the tested E. coli isolates, Klebsiella virus myPSH1235 had activity against 52% K. pneumoniae isolates and Enterobacter virus myPSH1140 showed activity against 15 E. cloacae (n = 15), 3 E. hormaechei (n = 4), 2 E. asburiae (n = 4), and 2 E. aerogenes (n = 3) isolates. In the case of double agar overlay method, Escherichia virus myPSH2311 had plaques against 43/80 tested E. coli isolates, Klebsiella virus myPSH1235 had plaques against 17/44 K. pneumoniae isolates and Enterobacter virus myPSH1140 had plaques against 11/15 E. cloacae, 2/4 E. hormaechei, 2/4 E. asburiae, and 2/3 E. aerogenes isolates. The percentage activity difference between the spot test and double agar overlay method was found to be 29.7% for Escherichia virus myPSH2311, 30% for Klebsiella virus myPSH1235 and for Enterobacter virus myPSH1140, it was 30.7% for E. cloacae, 40% for E. hormaechei, 0% for both E. asburiae and E. aerogenes. The EOP analysis that was used to differentiate the phage infectivity between spot test and double agar overlay method, showed a different scenario (Table 1). Though, spot test results showed that Escherichia virus myPSH1311 produced clear zone (spot) against 58/80 tested E. coli isolates; the EOP analysis showed high productive infection in 34/80 E. coli isolates whereas against 9 E. coli isolates it was moderate or low productive infection and 15/80 E. coli isolates had no infection. Klebsiella virus myPSH1235 had high productive infection against 10/44 K. pneumoniae isolates with 23/44 in spot test assay and the percentage difference was 78.7%. Even if all the EOP results (High + Moderate + Low) were considered for Klebsiella phage myPSH1235, the number of isolates producing (17/44) plaques in double agar overlay assay was still lower than the spot test (23/44) results. Enterobacter virus myPSH1140 had high productive infection against 7/15 tested E. cloacae isolates compared to 15/15 in spot test assay. The same phage showed high productive infection against 2/4 E. hormaechei, 2/4 E. asburiae, and 2/3 E. aerogenes, respectively (Table 1).\\nPhage Characterization: One-Step and Stability Studies\\nThe multiplication capacity of phages was determined by one-step growth experiment to analyze the adsorption velocity, latency period and burst size (Figure 2). Accordingly, for Escherichia virus myPSH2311, the adsorption velocity was 1.1 \\u00d7 10-9mL/min with latency period of 26 min and the burst size of approximately 110 phages/infected cell. The adsorption velocity for Klebsiella virus myPSH1235 was 4.35 \\u00d7 10-9 and the latency period was 40 min with the burst size of 120 phages/infected cell. Enterobacter virus myPSH1140 had an adsorption velocity of 2.8 \\u00d7 10-9, a very short latency period of 11 min and a burst size of 135 phages/infected cell (Figure 2). When sensitivity of phages to different pH conditions was determined by exposing them to varying range of pH from 1 to 14 for 60 min, all the three phages were found to be viable from pH 4 up to pH 11, but the phages were inactivated at pH \\u2264 3 and \\u2265 12 (Figure 3). In the case of thermal stability, all the phages were found to uphold their activity up to 55\\u00b0C and reduction in activity was observed at higher temperatures (Figure 3). The complete characterization report is available in Supplementary Table S3.\\nFigure 2. (A) Adsorption rate of phages to their bacterial host, (A1) Escherichia virus myPSH2311, (A2) Enterobacter virus myPSH1140, (A3) Klebsiella virus myPSH1235, and (B) One-step growth curve experiment of phages, (B1) Escherichia virus myPSH2311, (B2) Enterobacter virus myPSH1140, (B3) Klebsiella virus myPSH1235. Legend within figure B: L-latency period; B-burst size.\\nFigure 3. (A) Stability of phages at varying temperatures; (A1) Escherichia virus myPSH2311, (A2) Enterobacter virus myPSH1140, (A3) Klebsiella virus myPSH1235 and (B) Stability of phages at varying pH at 37\\u00b0C, (B1) Escherichia virus myPSH2311, (B2) Enterobacter virus myPSH1140, (B3) Klebsiella virus myPSH1235.\\nGenomic Analysis and Annotation\\nThe genome of Escherichia virus myPSH2311 measured 68,712 bp in size with a GC content of 42.4%. The genome contains 89 proteins or coding sequences (CDS) and it includes 27 proteins of known putative function and 62 hypothetical proteins. A total of 1.01 Gb data was assembled into scaffolds using CLC workbench version 9.5.2, and the assembly size was 5,945,203 bp with the average scaffold size of 12,133 bp. The arranged complete genome of Escherichia virus myPSH2311 is closely related to Escherichia phage vB_EcoP_SU10 (88%) and Escherichia virus phiEco32 (72%) (Figure 4A and Supplementary Figures S1, S2A). The NCBI accession number for this sequence is MG976803. The complete list of all the proteins is available in Supplementary Table S4. Klebsiella virus myPSH1235 was found to have a genome size of 45,135 bp with a GC content of 53.7%. The genome contains 49 proteins or CDS, of which 21 were found to have known putative function and 28 were hypothetical proteins. The obtained 1.37 Gb data was assembled into scaffolds using CLC workbench version 9.5.2, and the assembly size was 5,740,807 bp with the average scaffold size of 1,321 bp. The genome was closely related to Klebsiella phage vB_KpnP_KpV48 (95%) (Figure 4B and Supplementary Figures S1, S2B). The complete genome of Klebsiella virus myPSH1235 is free of toxins or toxin-related genes, and none of the proteins representing a temperate or lysogenic lifestyle was detected (Supplementary Table S4). The NCBI accession number for this phage is MG972768. The complete list of all the proteins is available in Supplementary Table S5. Enterobacter virus myPSH1140 was having a genome size of 172,614 bp with a GC content of 39.9%. The gene annotation studies showed 102 proteins with known function and 138 proteins were hypothetical proteins. The obtained 1.22 Gb data was assembled into scaffolds using CLC workbench version 9.5.2, and the assembly size was 5,193,726 bp with the average scaffold size of 6,059 bp. The complete genome was having 92% similarity with Enterobacter phage CC31 and 90% similarity with Enterobacter phage PG7 (Figure 4C and Supplementary Figures S1, S2C). The NCBI accession number is MG999954. The complete list of all the proteins is available in Supplementary Table S6.\\nFigure 4. Phylogenetic tree of (A) Escherichia virus myPSH2311, (B) Klebsiella virus myPSH1235, and (C) Enterobacter virus myPSH1140 constructed based on the complete genome sequences of selected bacteriophages in NCBI-BLAST. The tree was produced using BLAST pairwise alignment using Neighbor-Joining method. The query sequence is highlighted in yellow.\\nIn vitro Activity of Phage Cocktail\\nPhage cocktails were prepared to evaluate the activity of phages against multiple bacterial strains. When the phage cocktail containing all the three phages was tested against the three meropenem and colistin resistant test bacteria, the growth declined after 2 h from 106 to <105 CFU/mL and at the end of 24 h the bacterial density reached to zero with no viable cells. For EK1 cocktail, >2 fold decrease in bacterial cell count (both E. coli and K. pneumoniae) was observed after 2 h, for KL2 cocktail, the bacterial cell count (both K. pneumoniae and E. cloacae) decreased from 106 to 103 CFU/mL within 2 h and for EL3 cocktail, twofold reduction of bacterial cells (both E. coli and E. cloacae) was observed after 4 h. In phage cocktail containing all the three phages, EKL4, a twofold decrease in bacterial count (E. coli, K. pneumoniae, E. cloacae) was observed in 2 h as comparable to the phages alone. In the case of prepared phage cocktails, all the combinations had similar to better activity in comparison to the phages alone (Figure 5). Our experiment proved the activity of prepared phage cocktails against multiple bacterial genera and showed promising results against meropenem and colistin resistant bacteria in vitro.\\nFigure 5. In vitro phage activity of prepared phage cocktails. (A) activity of Escherichia virus myPSH2311 and Klebsiella virus myPSH1235 and their combination (EK1); (B) activity of Klebsiella virus myPSH1235 and Enterobacter virus myPSH1140 and their combination (KL2); (C) activity of Escherichia virus myPSH2311 and Enterobacter virus myPSH1140 and their combination (EL3); (D) activity of Escherichia virus myPSH2311, Klebsiella virus myPSH1235, Enterobacter virus myPSH1140 and their combination (EKL4) (Control for all the experiments-bacterial growth without antibacterial agents and phages).\\nDiscussion\\nPhage therapy is currently gaining attention in clinical medicine, because the infections caused by \\u2018superbugs \\u2013 multi-drug resistant (MDR) bacteria\\u2019 are almost impossible to treat, using available antibiotics (Chan et al., 2013). From clinical point-of-view, bacteriophages are likely to be used in treatment, only when the patient is infected with multi-drug resistant bacteria or during the antibiotic crisis of non-curable infections (Lin et al., 2017). The increasing studies on bacteriophages and phage therapy could put forth phage therapy as one of the alternative treatment options to treat \\u2018superbug\\u2019 infections (Rios et al., 2016). This study is one such effort to characterize bacteriophages infecting Gram-negative bacteria, E. coli, K. pneumoniae, and Enterobacter species.\\nIn the present study, we characterized the phenotypic, morphological, and genomic properties of three bacteriophages that are independently specific in their activity against pathogenic E. coli, K. pneumoniae, and Enterobacter species. All the three characterized phages belong to order Caudovirales that are lytic phages which are generally considered for therapy (Podoviridae and Myoviridae). There are earlier studies on Escherichia phages that showed many of the studied phages are specific to a strain of E. coli (e.g., Escherichia phage OSYSP infecting only E. coli O157:H7) or had a narrow host range activity (Carter et al., 2012; Dalmasso et al., 2016; Baig et al., 2017; Lin et al., 2017). In our study, the characterized Escherichia virus myPSH2311 was found to have broad host range activity infecting E. coli isolates that belonged to six different pathotypes EPEC, EHEC, ETEC, EIEC, EAEC, and UPEC besides some unknown pathotypes. This is one of the rarest reports where an Escherichia phage infecting six different pathotypes of E. coli was characterized. But another study also report the broad host range activity of Escherichia phage against multiple pathotypes of E. coli (Manohar et al., 2018b). The studied Klebsiella virus myPSH1235 showed broad host range activity against K. pneumoniae belonging to serotypes K1, K2, K5 besides some unknown serotypes. An earlier study showed that a multi-host Klebsiella phage \\u03d5K64-1 was capable to infect Klebsiella capsular types K1, KN4, KN5, K11, K21, K25, K30, K35, K64, and K69. It is stated that the multi-host infectivity of the bacteriophages is due to the presence of multiple depolymerases in the tail-fibers (Pan et al., 2017). The role of polyvalent phages in phage therapy needs more detailed investigation. The studies on Enterobacter phage thus far are very minimal or reports of established Enterobacter phages capable of infecting various strains or species are scarce (Li et al., 2016a; Pereira et al., 2016). The characterized Enterobacter virus myPSH1140 in our study belonging to Myoviridae family showed activity against four species of Enterobacter E. cloacae, E. hormaechei, E. asburiae, and E. aerogenes. Our study and those of some others strongly suggest that there are bacteriophages that can have broad host range activity against different strains or species or genera of bacteria (Hamdi et al., 2017). A recent study showed that bacteriophages SH6 and SH7 were capable of infecting bacteria from different genera \\u2013 E. coli O157:H7, Shigella flexneri, Salmonella paratyphi, and Shigella dysenteriae (Hamdi et al., 2017). There are studies that showed \\u2018narrow\\u2019 host range specificity of bacteriophages mainly because of the use of standard isolation procedure where single host strain of bacteria is used (Li et al., 2016b; Ross et al., 2016; Hamdi et al., 2017). Future studies should focus on testing the host range activity of bacteriophages but it is also believed that host range activity is not a fixed property of bacteriophages and can evolve overtime (Ross et al., 2016; Hamdi et al., 2017). It should also be noted that we found dissimilarities in the results obtained between spot test and plaque assay in host range activity determination, therefore plaque assay is recommended in order to obtain productive infection for the determination of host range activity of phages (Mirzaei and Nilsson, 2015).\\nThe three characterized bacteriophages had different growth profile, which is one of the important characteristics of lytic bacteriophages. In this study, Escherichia virus myPSH2311 (Podovirus) had a growth profile with the latent period of 26 min and burst size of 110 phages/infected cell, which is beneficial for therapy. Earlier studies using Podoviridae phages showed latency period of \\u224815\\u201325 min and burst size of \\u224850\\u201380 phages/infected cell (Dalmasso et al., 2016). An earlier study on Podoviridae phages infecting Klebsiella species was found to have a growth profile with the latency period of 15 min and burst size of \\u224850\\u201360 phages/infected cell (Chaturongakul and Ounjai, 2014). In our study, Klebsiella virus myPSH1235 belonging to Podoviridae family was found to have a latent period of 40 min and burst size of 120 phages/infected cell, which is beneficial for therapy. It was very clear from this study that the growth profile of bacteriophages is not completely dependent on either the family or the host these bacteriophages infect. The bacteriophages belonging to Myoviridae family are known to have shorter life cycles and similarly the characterized Myoviridae phage (Enterobacter phage myPSH1140) had a latent period of 11 min and burst size of 135 phages/infected cell. It was also noted in this and earlier studies that the number of progeny phage particles (burst size) largely depends on the availability of host bacterial cells (Dalmasso et al., 2016). Life cycle parameters of bacteriophages will play significant role in determining both in vitro and in vivo phage activities (during therapy), because phage multiplication is directly proportional to reduction in bacteria.\\nThe genomes of all the three characterized phages showed 70\\u2013100% similarities to the already existing phage genomes in the database. Escherichia virus myPSH2311 (68,712 bp) was found to have 88% sequence similarity with Escherichia phage vB_EcoP_SU10 (77,327 bp). The endolysin gene (ORF 12) was found to have 98% sequence similarity with Escherichia virus phiEco32 and 97% sequence similarity with Escherichia phage vB_EcoP_SU10. The DNA injection protein (ORF 17) was detected to have 98% sequence similarity with Escherichia phage vB_EcoP_SU10. The genome does not have toxins or toxin-related genes, and none of the genomic markers representing a temperate or lysogenic lifestyle was detected (Supplementary Table S4). Escherichia phage vB_EcoP_SU10 is a C3 morphotype phage but Escherichia virus myPSH2311 does not have any elongated regions in its morphological characterization (Mirzaei et al., 2014) and related to T4-phages (Petrov et al., 2010). Klebsiella virus myPSH1235 was found to have 80% similarity with Klebsiella phage vB_KpnP_KpV48. Klebsiella virus myPSH1235 has a genome size of 45,135 bp with 49 CDS compared to Klebsiella phage vB_KpnP_KpV48, which has a genome size of 44,623 bp with 57 CDS. According to the genome morphology, phylogenetic analysis and sequence similarities, the Klebsiella virus myPSH1235 is classified within the genus Kp34virus, subfamily Autographivirinae, family Podoviridae. Klebsiella phage vB_KpnP_KpV48 has no capsular specificity and similarly, Klebsiella virus myPSH1235 was also found to infect K. pneumoniae of capsular types K1, K2 and K5 (Solovieva et al., 2018). From clinical point of view, the K. pneumoniae capsular types K1 and K2 are the most virulent strains and that the infections caused by these strains can be eliminated using Klebsiella virus myPSH1235 is a promising result, which needs further exploration. Enterobacter virus myPSH1140 was found to have a large genome size of 172,614 bp with 240 CDS with 92% similarity with Enterobacter phage CC31 (165,540 bp and 279 CDS) and 90% similarity to Enterobacter phage PG7 (173,276 and 294 bp). The genome contains 240 proteins or CDS and all the identified proteins were found to have 90\\u2013100% sequence similarity with the Enterobacter phage CC31 and Enterobacter phage PG7. Similarities in the phage genome showed that there is abundance of these phages in the environment and therefore they may be isolated for therapeutic purpose.\\nEarlier studies on phage cocktails showed promising results in reducing the bacterial load in both in vitro and in vivo models (Lin et al., 2017; Pan et al., 2017). In our in vitro experiments, a phage cocktail that was prepared using the three newly isolated bacteriophages infecting three different genera of bacteria (Escherichia virus myPSH2311 against E. coli, Klebsiella virus myPSH1235 against K. pneumoniae and Enterobacter virus myPSH1140 against Enterobacter species) showed promising results in totally suppressing the bacterial load within 24 h. The phage cocktails reported in this study consisting of multiple phage types could thus prove to be effective as clinical therapeutic agents (Chan et al., 2013; K\\u0119sik-Szeloch et al., 2013). Ours is one of the rarest studies to use phage cocktails against different genera of bacteria. In vivo activity of these phage cocktails also showed promising results and the results are published elsewhere (Manohar et al., 2018a).\\nThe strength of our study is that we established the lytic effectiveness of the phages and their cocktail against clinical bacterial isolates of three genera that are known to be pathogenic. Further we also demonstrated the lytic effectiveness of these phages and their cocktail against pathogenic bacteria that were resistant to two last resort antibiotics, meropenem, and colistin. Thus further experiments can be taken up for in vivo studies in search of a therapeutic treatment against bacterial infections resistant to these drugs.\\nConclusion\\nThe emergence of multi-drug resistant bacterial strains is a global threat which needs to be addressed by using alternative therapies, such as phage therapy. The three highlighted bacteriophages (Escherichia virus myPSH2311 against E. coli, Klebsiella virus myPSH1235 against K. pneumoniae and Enterobacter virus myPSH1140 against Enterobacter species) and their cocktails showed promising in vitro results that may be extended for use as biocontrol agents in some clinical applications. The development of phage cocktails could be considered for the effective treatment of bacterial infections consisting of broad targets. Hopefully, in future, availability of increased repertoire of phages may allow the development of multi-phage cocktail therapy possible.\\nData Availability\\nAll the data that support the findings of this study have been deposited in GenBank with the accession numbers MG976803 (Escherichia phage myPSH2311), MG972768 (Klebsiella phage myPSH1235), and MG999954 (Enterobacter phage myPSH1140).\\nAuthor Contributions\\nPM and RN collected the isolates and did bacteriophage experiments, undertook the laboratory work, and wrote the initial manuscript. AT, CL, and RN interpreted the data. CL and AT revised and edited the manuscript. All authors approved the final version of the manuscript.\\nFunding\\nThe authors would like to thank Vellore Institute of Technology (VIT) for providing research facilities to complete the work. This research was partly supported by Swedish Research Council with grants to CL (Grant Nos. 2012-02889 and 2017-01327). The authors would also like to thank Council of Scientific and Industrial Research (CSIR) for providing financial assistance (SRF) to support this research.\\nConflict of Interest Statement\\nThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.\\nSupplementary Material\\nThe Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fmicb.2019.00574/full#supplementary-material\\nFootnotes\\n^https://www.ncbi.nlm.nih.gov/genome/?term=escherichia+phage\\n^https://www.ncbi.nlm.nih.gov/\\n^http://wego.genomics.org.cn\\nReferences\\nBaig, A., Colom, J., Barrow, P., Schouler, C., Moodley, A., Lavigne, R., et al. (2017). Biology and genomics of an historic therapeutic Escherichia coli bacteriophage collection. Front. Microbiol. 8:1652. doi: 10.3389/fmicb.2017.01652\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCao, F., Wang, X., Wang, L., Li, Z., Che, J., Wang, L., et al. (2015). Evaluation of the efficacy of a bacteriophage in the treatment of pneumonia induced by multidrug resistance Klebsiella pneumoniae in mice. Biomed. Res. Int. 2015:752930. doi: 10.1155/2015/752930\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nCarter, C. D., Parks, A., Abuladze, T., Li, M., Woolston, J., Magnone, J., et al. (2012). Bacteriophage cocktail significantly reduces Escherichia coli O157: H7 contamination of lettuce and beef, but does not protect against recontamination. Bacteriophage 2, 178\\u2013185. doi: 10.4161/bact.22825\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nChaturongakul, S., and Ounjai, P. (2014). Phage\\u2013host interplay: examples from tailed phages and Gram-negative bacterial pathogens. Front. Microbiol. 5:442. doi: 10.3389/fmicb.2014.00442\\nCrossRef Full Text | Google Scholar\\nChibani-Chennoufi, S., Bruttin, A., Dillmann, M. L., and Br\\u00fcssow, H. (2004). Phage-host interaction: an ecological perspective. J. Bacteriol. 186, 3677\\u20133686. doi: 10.1128/JB.186.12.3677-3686.2004\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nClinical and Laboratory Standards Institute [CLSI] (2018). Performance Standards for Antimicrobial Susceptibility Testing; Twenty-Fifth Informational Supplements. Approved Standard M100-S28. Wayne, PA: Clinical and Laboratory Standards Institute [CLSI].\\nDalmasso, M., Strain, R., Neve, H., Franz, C. M., Cousin, F. J., Ross, R. P., et al. (2016). Three new Escherichia coli phages from the human gut show promising potential for phage therapy. PLoS One 11:e0156773. doi: 10.1371/journal.pone.0156773\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHamdi, S., Rousseau, G. M., Labrie, S. J., Tremblay, D. M., Kourda, R. S., Slama, K. B., et al. (2017). Characterization of two polyvalent phages infecting Enterobacteriaceae. Sci. Rep. 7:40349. doi: 10.1038/srep40349\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nHung, C. H., Kuo, C. F., Wang, C. H., Wu, C. M., and Tsao, N. (2011). Experimental phage therapy in treating Klebsiella pneumoniae-mediated liver abscesses and bacteremia in mice. Antimicrob. Agents Chemother. 55, 1358\\u20131365. doi: 10.1128/AAC.01123-10\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nK\\u0119sik-Szeloch, A., Drulis-Kawa, Z., Weber-D\\u0119abrowska, B., Kassner, J., Majkowska-Skrobek, G., Augustyniak, D., et al. (2013). Characterising the biology of novel lytic bacteriophages infecting multidrug resistant Klebsiella pneumoniae. Virol. J. 10:100. doi: 10.1186/1743-422X-10-100\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nKrupovic, M., Dutilh, B. E., Adriaenssens, E. M., Wittmann, J., Vogensen, F. K., Sullivan, M. B., et al. (2016). Taxonomy of prokaryotic viruses: update from the ICTV bacterial and archaeal viruses subcommittee. Arch. Virol. 161, 1095\\u20131099. doi: 10.1007/s00705-015-2728-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLi, E., Wei, X., Ma, Y., Yin, Z., Li, H., Lin, W., et al. (2016a). Isolation and characterization of a bacteriophage phiEap-2 infecting multidrug resistant Enterobacter aerogenes. Sci. Rep. 6:28338. doi: 10.1038/srep28338\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLi, E., Zhao, J., Ma, Y., Wei, X., Li, H., Lin, W., et al. (2016b). Characterization of a novel Achromobacter xylosoxidans specific siphoviruse: phiAxp-1. Sci. Rep. 6:21943. doi: 10.1038/srep21943\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nLin, D. M., Koskella, B., and Lin, H. C. (2017). Phage therapy: an alternative to antibiotics in the age of multi-drug resistance. World J. Gastrointest. Pharmacol. Therapeut. 8:162. doi: 10.4292/wjgpt.v8.i3.162\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nManohar, P., Nachimuthu, R., and Lopes, B. S. (2018a). The therapeutic potential of bacteriophages targeting gram-negative bacteria using Galleria mellonella infection model. BMC Microbiol. 18:97. doi: 10.1186/s12866-018-1234-4\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nManohar, P., Tamhankar, A. J., Lundborg, C. S., and Ramesh, N. (2018b). Isolation, characterization and in vivo efficacy of Escherichia phage myPSH1131. PLoS One 13:e0206278. doi: 10.1371/journal.pone.0206278\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMcVay, C. S., Vel\\u00e1squez, M., and Fralick, J. A. (2007). Phage therapy of Pseudomonas aeruginosa infection in a mouse burn wound model. Antimicrob. Agents Chemother. 51, 1934\\u20131938. doi: 10.1128/AAC.01028-06\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMendes, J. J., Leandro, C., Mottola, C., Barbosa, R., Silva, F. A., Oliveira, M., et al. (2014). In vitro design of a novel lytic bacteriophage cocktail with therapeutic potential against organisms causing diabetic foot infections. J. Med. Microbiol. 63, 1055\\u20131065. doi: 10.1099/jmm.0.071753-0\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMirzaei, M. K., Eriksson, H., Kasuga, K., Hagg\\u00e5rd-Ljungquist, E., and Nilsson, A. S. (2014). Genomic, proteomic, morphological, and phylogenetic analyses of vB_EcoP_SU10, a podoviridae phage with C3 morphology. PLoS One 9:e116294. doi: 10.1371/journal.pone.0116294\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nMirzaei, M. K., and Nilsson, A. S. (2015). Isolation of phages for phage therapy: a comparison of spot tests and efficiency of plating analyses for determination of host range and efficacy. PLoS One 10:e0118557. doi: 10.1371/journal.pone.0118557\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPan, Y. J., Lin, T. L., Chen, C. C., Tsai, Y. T., Cheng, Y. H., Chen, Y. Y., et al. (2017). Klebsiella phage \\u03a6K64-1 encodes multiple depolymerases for multiple host capsular types. J. Virol. 91:e2457\\u20136. doi: 10.1128/JVI.02457-16\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPereira, S., Pereira, C., Santos, L., Klumpp, J., and Almeida, A. (2016). Potential of phage cocktails in the inactivation of Enterobacter cloacae-an in vitro study in a buffer solution and in urine samples. Virus Res. 211, 199\\u2013208. doi: 10.1016/j.virusres.2015.10.025\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nPetrov, V. M., Ratnayaka, S., Nolan, J. M., Miller, E. S., and Karam, J. D. (2010). Genomes of the T4-related bacteriophages as windows on microbial genome evolution. Virol. J. 7:292. doi: 10.1186/1743-422X-7-292\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nRios, A. C., Moutinho, C. G., Pinto, F. C., Del Fiol, F. S., Jozala, A., Chaud, M. V., et al. (2016). Alternatives to overcoming bacterial resistances: state-of-the-art. Microbiol. Res. 191, 51\\u201380. doi: 10.1016/j.micres.2016.04.008\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nSchlossberg, D. (ed.) (2015). Clinical Infectious Disease. Cambridge: Cambridge University Press. doi: 10.1017/CBO9781139855952\\nCrossRef Full Text | Google Scholar\\nSolovieva, E. V., Myakinina, V. P., Kislichkina, A. A., Krasilnikova, V. M., Verevkin, V. V., Mochalov, V. V., et al. (2018). Comparative genome analysis of novel podoviruses lytic for hypermucoviscous Klebsiella pneumoniae of K1, K2, and K57 capsular types. Virus Res. 243, 10\\u201318. doi: 10.1016/j.virusres.2017.09.026\\nPubMed Abstract | CrossRef Full Text | Google Scholar\\nVentola, C. L. (2015). The antibiotic resistance crisis: part 1: causes and threats. Pharm. Therapeut. 40, 277\\u2013283.\\nPubMed Abstract | Google Scholar\\nYen, M., Cairns, L. S., and Camilli, A. (2017). A cocktail of three virulent bacteriophages prevents Vibrio cholerae infection in animal models. Nat. Commun. 8:14187. doi: 10.1038/ncomms14187\\nPubMed Abstract | CrossRef Full Text | Google Scholar\"\n",
      "    }\n",
      "]\n"
     ]
    }
   ],
   "source": [
    "json_data = jsonToDict(file_path)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Then use json loader \"JSONDataReader\" from llama_hub"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pathlib import Path\n",
    "from llama_index import download_loader\n",
    "\n",
    "JSONReader = download_loader(\"JSONReader\")\n",
    "\n",
    "loader = JSONReader(levels_back=0)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "json_file_path = '/workspaces/ml-learning/src/phages/notebooks/data/dataset_phages-microbiology-frontiers_2024-01-23_08-51-12-772.json'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "documents = loader.load_data(json_file_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'{\\n\"url\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n\"crawl\": {\\n\"loadedUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research#\",\\n\"loadedTime\": \"2024-01-23T08:48:06.942Z\",\\n\"referrerUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research\",\\n\"depth\": 0,\\n\"httpStatusCode\": 200\\n\"metadata\": {\\n\"canonicalUrl\": \"https://www.frontiersin.org/articles?domain=all&query=bacteriophages&type=original-research#\",\\n\"title\": \"Articles | Frontiers\",\\n\"description\": null,\\n\"author\": null,\\n\"keywords\": null,\\n\"languageCode\": \"en\"\\n\"screenshotUrl\": null,\\n\"text\": \"Original Research\\\\nPublished on 30 Apr 2018\\\\nTwo New Lytic Bacteriophages of the Myoviridae Family Against Carbapenem-Resistant Acinetobacter baumannii\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2018.00850\\\\n7,708 views \\\\n45 citations \\\\nOriginal Research\\\\nPublished on 4 Jan 2018\\\\nMetavirome Sequencing of the Termite Gut Reveals the Presence of an Unexplored Bacteriophage Community\\\\nChinmay V. Tikhe\\\\nClaudia Husseneder\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2017.02548\\\\n13,451 views \\\\n31 citations \\\\nOriginal Research\\\\nPublished on 28 Sep 2017\\\\nNewly Isolated Bacteriophages from the Podoviridae, Siphoviridae, and Myoviridae Families Have Variable Effects on Putative Novel Dickeya spp.\\\\n\\\\u0160pela Ali\\\\u010d\\\\nTina Nagli\\\\u010d\\\\nMagda Tu\\\\u0161ek-\\\\u017dnidari\\\\u010d\\\\nMaja Ravnikar\\\\nNejc Ra\\\\u010dki\\\\nMatja\\\\u017e Peterka\\\\nTanja Dreo\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2017.01870\\\\n9,561 views \\\\n27 citations \\\\nOriginal Research\\\\nPublished on 2 Apr 2019\\\\nDiscovery of Shiga Toxin-Producing Escherichia coli (STEC)-Specific Bacteriophages From Non-fecal Composts Using Genomic Characterization\\\\nYen-Te Liao\\\\nXincheng Sun\\\\nIrwin A. Quintela\\\\nDavid F. Bridges\\\\nFang Liu\\\\nYujie Zhang\\\\nAlexandra Salvador\\\\nVivian C. H. Wu\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.00627\\\\n4,555 views \\\\n13 citations \\\\nOriginal Research\\\\nPublished on 10 Mar 2021\\\\nIsolation and Characterization of Six Vibrio parahaemolyticus Lytic Bacteriophages From Seafood Samples\\\\nChia Wanq Tan\\\\nYaya Rukayadi\\\\nHanan Hasan\\\\nNoor-Azira Abdul-Mutalib\\\\nNuzul Noorahya Jambari\\\\nHirofumi Hara\\\\nTze Young Thung\\\\nEpeng Lee\\\\nSon Radu\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2021.616548\\\\n12,006 views \\\\n25 citations \\\\nOriginal Research\\\\nPublished on 23 Jul 2019\\\\nA Novel, Highly Related Jumbo Family of Bacteriophages That Were Isolated Against Erwinia\\\\nRuchira Sharma\\\\nBrittany A. Pielstick\\\\nKimberly A. Bell\\\\nTanner B. Nieman\\\\nOlivia A. Stubbs\\\\nEdward L. Yeates\\\\nDavid A. Baltrus\\\\nJulianne H. Grose\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.01533\\\\n16,064 views \\\\n34 citations \\\\nOriginal Research\\\\nPublished on 13 Feb 2018\\\\nIdentification and Characterization of T5-Like Bacteriophages Representing Two Novel Subgroups from Food Products\\\\nDomonkos Sv\\\\u00e1b\\\\nLinda Falgenhauer\\\\nManfred Rohde\\\\nJudit Szab\\\\u00f3\\\\nTrinad Chakraborty\\\\nIstv\\\\u00e1n T\\\\u00f3th\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2018.00202\\\\n6,673 views \\\\n32 citations \\\\nOriginal Research\\\\nPublished on 26 Apr 2022\\\\nOptimisation and Application of a Novel Method to Identify Bacteriophages in Maternal Milk and Infant Stool Identifies Host-Phage Communities Within Preterm Infant Gut\\\\nGregory R. Young\\\\nWen C. Yew\\\\nAndrew Nelson\\\\nSimon H. Bridge\\\\nJanet E. Berrington\\\\nNicholas D. Embleton\\\\nDarren L. Smith\\\\nFrontiers in Pediatrics\\\\ndoi 10.3389/fped.2022.856520\\\\n2,547 views \\\\n2 citations \\\\nOriginal Research\\\\nPublished on 16 Nov 2022\\\\nBacteriophage-based decontamination to control environmental colonization by Staphylococcus capitis in neonatal intensive care units: An in vitro proof-of-concept\\\\nMarie Chavignon\\\\nCamille Kolenda\\\\nMathieu Medina\\\\nM\\\\u00e9lanie Bonhomme\\\\nLeslie Blazere\\\\nTiphaine Legendre\\\\nAnne Tristan\\\\nFr\\\\u00e9d\\\\u00e9ric Laurent\\\\nMarine Butin\\\\nFrontiers in Cellular and Infection Microbiology\\\\ndoi 10.3389/fcimb.2022.1060825\\\\n2,238 views \\\\n1 citations \\\\nOriginal Research\\\\nPublished on 2 May 2018\\\\nBacteriophage Infectivity Against Pseudomonas aeruginosa in Saline Conditions\\\\nGiantommaso Scarascia\\\\nScott A. Yap\\\\nAnna H. Kaksonen\\\\nPei-Ying Hong\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2018.00875\\\\n8,420 views \\\\n11 citations \\\\nOriginal Research\\\\nPublished on 6 Oct 2020\\\\nIdentification and Characterization of New Bacteriophages to Control Multidrug-Resistant Pseudomonas aeruginosa Biofilm on Endotracheal Tubes\\\\nViviane C. Oliveira\\\\nFelipe L. Bim\\\\nRachel M. Monteiro\\\\nAna Paula Macedo\\\\nEmerson S. Santos\\\\nCl\\\\u00e1udia H. Silva-Lovato\\\\nHelena F. O. Paranhos\\\\nLu\\\\u00eds D. R. Melo\\\\nS\\\\u00edlvio B. Santos\\\\nEvandro Watanabe\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2020.580779\\\\n9,906 views \\\\n24 citations \\\\nOriginal Research\\\\nPublished on 23 Jan 2020\\\\nBacteriophages as an Up-and-Coming Alternative to the Use of Sulfur Dioxide in Winemaking\\\\nGustavo Cordero-Bueso\\\\nJavier Moraga\\\\nMar\\\\u00eda R\\\\u00edos-Carrasco\\\\nMarina Ruiz-Mu\\\\u00f1oz\\\\nJes\\\\u00fas Manuel Cantoral\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.02931\\\\n5,238 views \\\\n3 citations \\\\nOriginal Research\\\\nPublished on 30 Aug 2017\\\\nEnvironmental Bacteriophages of the Emerging Enterobacterial Phytopathogen, Dickeya solani, Show Genomic Conservation and Capacity for Horizontal Gene Transfer between Their Bacterial Hosts\\\\nAndrew Day\\\\nJiyoon Ahn\\\\nXinzhe Fang\\\\nGeorge P. C. Salmond\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2017.01654\\\\n3,267 views \\\\n15 citations \\\\nOriginal Research\\\\nPublished on 12 Sep 2018\\\\nJumbo Bacteriophages Are Represented Within an Increasing Diversity of Environmental Viruses Infecting the Emerging Phytopathogen, Dickeya solani\\\\nAndrew Day\\\\nJiyoon Ahn\\\\nGeorge P. C. Salmond\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2018.02169\\\\n4,563 views \\\\n17 citations \\\\nOriginal Research\\\\nPublished on 22 Dec 2022\\\\nA lytic bacteriophage isolate reduced Clostridium perfringens induced lesions in necrotic enteritis challenged broilers\\\\nChake Keerqin\\\\nKatherine McGlashan\\\\nThi Thu Hao Van\\\\nHelene N. Chinivasagam\\\\nRobert J. Moore\\\\nMingan Choct\\\\nShu-Biao Wu\\\\nFrontiers in Veterinary Science\\\\ndoi 10.3389/fvets.2022.1058115\\\\n1,860 views \\\\n2 citations \\\\nOriginal Research\\\\nPublished on 21 Mar 2019\\\\nTherapeutic Characterization and Efficacy of Bacteriophage Cocktails Infecting Escherichia coli, Klebsiella pneumoniae, and Enterobacter Species\\\\nPrasanth Manohar\\\\nAshok J. Tamhankar\\\\nCecilia Stalsby Lundborg\\\\nRamesh Nachimuthu\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.00574\\\\n20,572 views \\\\n108 citations \\\\nOriginal Research\\\\nPublished on 28 Jun 2018\\\\nVectorized Delivery of Alpha-GalactosylCeramide and Tumor Antigen on Filamentous Bacteriophage fd Induces Protective Immunity by Enhancing Tumor-Specific T Cell Response\\\\nRossella Sartorius\\\\nLuciana D\\\\u2019Apice\\\\nPasquale Barba\\\\nDeborah Cipria\\\\nLaura Grauso\\\\nAdele Cutignano\\\\nPiergiuseppe De Berardinis\\\\nFrontiers in Immunology\\\\ndoi 10.3389/fimmu.2018.01496\\\\n4,170 views \\\\n18 citations \\\\nOriginal Research\\\\nPublished on 15 Nov 2017\\\\nPrion-Like Domains in Phagobiota\\\\nGeorge Tetz\\\\nVictor Tetz\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2017.02239\\\\n8,010 views \\\\n27 citations \\\\nOriginal Research\\\\nPublished on 20 Apr 2016\\\\nGenomics of Three New Bacteriophages Useful in the Biocontrol of Salmonella\\\\nCarlota Bardina\\\\nJoan Colom\\\\nDenis A. Spricigo\\\\nJennifer Otero\\\\nMiquel S\\\\u00e1nchez-Osuna\\\\nPilar Cort\\\\u00e9s\\\\nMontserrat Llagostera\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2016.00545\\\\n10,249 views \\\\n38 citations \\\\nOriginal Research\\\\nPublished on 6 Apr 2022\\\\nThe Expansion of a Single Bacteriophage Leads to Bacterial Disturbance in Gut and Reduction of Larval Growth in Musca domestica\\\\nXinyu Zhang\\\\nShumin Wang\\\\nQian Zhang\\\\nKexin Zhang\\\\nWenjuan Liu\\\\nRuiling Zhang\\\\nZhong Zhang\\\\nFrontiers in Immunology\\\\ndoi 10.3389/fimmu.2022.885722\\\\n2,084 views \\\\n3 citations \\\\nOriginal Research\\\\nPublished on 21 Jul 2023\\\\nIsolation and characterization of a novel mycobacteriophage Kashi-VT1 infecting Mycobacterium species\\\\nTanmayee Nayak\\\\nAnuja Kakkar\\\\nRakesh Kumar Singh\\\\nLav Kumar Jaiswal\\\\nAnand Kumar Singh\\\\nLouise Temple\\\\nAnkush Gupta\\\\nFrontiers in Cellular and Infection Microbiology\\\\ndoi 10.3389/fcimb.2023.1173894\\\\n2,274 views \\\\nOriginal Research\\\\nPublished on 12 Mar 2013\\\\nAntiviral effect of cationic compounds on bacteriophages\\\\nMai Huong Chatain-Ly\\\\nSaliha Moussaoui\\\\nVeronique Rigobello\\\\nYann Demarigny\\\\nAnnabelle Vera\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2013.00046\\\\n8,076 views \\\\n47 citations \\\\nOriginal Research\\\\nPublished on 4 Apr 2019\\\\nBiodistribution of Liposome-Encapsulated Bacteriophages and Their Transcytosis During Oral Phage Therapy\\\\nJennifer Otero\\\\nAlba Garc\\\\u00eda-Rodr\\\\u00edguez\\\\nMary Cano-Sarabia\\\\nDaniel Maspoch\\\\nRicard Marcos\\\\nPilar Cort\\\\u00e9s\\\\nMontserrat Llagostera\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2019.00689\\\\n8,053 views \\\\n43 citations \\\\nOriginal Research\\\\nPublished on 8 Nov 2016\\\\nUltrastructure and Viral Metagenome of Bacteriophages from an Anaerobic Methane Oxidizing Methylomirabilis Bioreactor Enrichment Culture\\\\nLavinia Gambelli\\\\nGeert Cremers\\\\nRob Mesman\\\\nSimon Guerrero\\\\nBas E. Dutilh\\\\nMike S. M. Jetten\\\\nHuub J. M. Op den Camp\\\\nLaura van Niftrik\\\\nFrontiers in Microbiology\\\\ndoi 10.3389/fmicb.2016.01740\\\\n7,432 views \\\\n14 citations\"\\n}'"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "documents[0].text"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    " Parsing xml https://blog.apify.com/python-parse-xml/\n",
    "\n",
    " Pdf download https://www.frontiersin.org/articles/10.3389/fmicb.2018.00850/pdf?isPublishedV2=False\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\"\"\"Custom JSON Reader.\"\"\"\n",
    "\n",
    "import json\n",
    "import re\n",
    "from pathlib import Path\n",
    "from typing import Dict, Generator, List, Optional\n",
    "\n",
    "from llama_index.readers.base import BaseReader\n",
    "from llama_index.readers.schema.base import Document\n",
    "\n",
    "class CustomJSONReader(BaseReader):"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
